<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="e2e3216a-940d-4c07-8954-02476f88d99b" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867421006899;website:website:cell-site;ctype:string:Journal Content;issue:issue:pii\:S0092867420X00158;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Interconnecting solvent quality, transcription, and chromosome folding in Escherichia coli">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00689-9">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/c97a0d1a-f7ee-4287-9a9b-4e2a1eb07d3d/fx1.jpg">
    


        <meta property="og:description" content="Measurements of nucleoid mesh size and DNA concentration reveal that the cytoplasm
effectively behaves as a poor solvent for the chromosome, leading to chromosome compaction,
DNA density heterogeneity, and spatial density of ribosomes that is negatively correlated
with DNA density within the nucleoid. Transcription may contribute to the effective
poor solvent quality of the cytoplasm.">
    











        <meta name="twitter:title" content="Interconnecting solvent quality, transcription, and chromosome folding in Escherichia coli">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/c97a0d1a-f7ee-4287-9a9b-4e2a1eb07d3d/fx1.jpg">
    


        <meta name="twitter:description" content="Measurements of nucleoid mesh size and DNA concentration reveal that the cytoplasm
effectively behaves as a poor solvent for the chromosome, leading to chromosome compaction,
DNA density heterogeneity, and spatial density of ribosomes that is negatively correlated
with DNA density within the nucleoid. Transcription may contribute to the effective
poor solvent quality of the cytoplasm.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Interconnecting solvent quality, transcription, and chromosome folding in Escherichia coli: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Interconnecting solvent quality, transcription, and chromosome folding in Escherichia coli"> <meta name="citation_date" content="2021/07/08"> <meta name="citation_online_date" content="2021/06/28"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="14"> <meta name="citation_firstpage" content="3626"> <meta name="citation_lastpage" content="3642.e14"> <meta name="citation_pmid" content="34186018"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421006899"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421006899"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421006899"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.05.037"> <meta name="citation_pmid" content="34186018"> <meta name="citation_author" content="Yingjie Xiang"><meta name="citation_author_institution" content="Department of Mechanical Engineering and Materials Science, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author" content="Ivan V. Surovtsev"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA"><meta name="citation_author" content="Yunjie Chang"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author_institution" content="Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA"><meta name="citation_author" content="Sander K. Govers"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Department of Biology and Institute of Chemistry, Engineering and Medicine for Human Health, Stanford University, Palo Alto, CA 94305, USA"><meta name="citation_author" content="Bradley R. Parry"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA"><meta name="citation_author" content="Jun Liu"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author_institution" content="Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA"><meta name="citation_author" content="Christine Jacobs-Wagner"><meta name="citation_author_email" content="jacobs-wagner@stanford.edu"><meta name="citation_author_institution" content="Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA"><meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA"><meta name="citation_author_institution" content="Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA"><meta name="citation_author_institution" content="Department of Biology and Institute of Chemistry, Engineering and Medicine for Human Health, Stanford University, Palo Alto, CA 94305, USA"><meta name="citation_keywords" content="chromosome folding; nucleoid; solvent quality; chromosome compaction; domain organization; DNA mesh size; ribosome localization; RNA"><meta name="citation_reference" content="citation_title=Stochasticity and homeostasis in the E.&nbsp;coli replication and division cycle; citation_author=A. Adiciptaningrum; citation_author=M. Osella; citation_author=M.C. Moolman; citation_author=M. Cosentino Lagomarsino; citation_author=S.J. Tans; citation_journal_title=Sci. Rep.; citation_volume=5; citation_firstpage=18261; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187; citation_author=T. Alatossava; citation_author=H. Jütte; citation_author=A. Kuhn; citation_author=E. Kellenberger; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=162; citation_firstpage=413; citation_lastpage=419; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=On Interaction between Two Bodies Immersed in a Solution of Macromolecules; citation_author=S. Asakura; citation_author=F. Oosawa; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=22; citation_firstpage=1255; citation_lastpage=1256; citation_publication_date=1954; "><meta name="citation_reference" content="citation_title=A Multiscale Model for Solute Diffusion in Hydrogels; citation_author=E. Axpe; citation_author=D. Chan; citation_author=G.S. Offeddu; citation_author=Y. Chang; citation_author=D. Merida; citation_author=H.L. Hernandez; citation_author=E.A. Appel; citation_journal_title=Macromolecules; citation_volume=52; citation_firstpage=6889; citation_lastpage=6897; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Two types of localization of the DNA-binding proteins within the Escherichia coli nucleoid; citation_author=T.A. Azam; citation_author=S. Hiraga; citation_author=A. Ishihama; citation_journal_title=Genes Cells; citation_volume=5; citation_firstpage=613; citation_lastpage=626; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells; citation_author=S. Bakshi; citation_author=A. Siryaporn; citation_author=M. Goulian; citation_author=J.C. Weisshaar; citation_journal_title=Mol. Microbiol.; citation_volume=85; citation_firstpage=21; citation_lastpage=38; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes; citation_author=S. Bakshi; citation_author=H. Choi; citation_author=J. Mondal; citation_author=J.C. Weisshaar; citation_journal_title=Mol. Microbiol.; citation_volume=94; citation_firstpage=871; citation_lastpage=887; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Inflating bacterial cells by increased protein synthesis; citation_author=M. Basan; citation_author=M. Zhu; citation_author=X. Dai; citation_author=M. Warren; citation_author=D. Sévin; citation_author=Y.P. Wang; citation_author=T. Hwa; citation_journal_title=Mol. Syst. Biol.; citation_volume=11; citation_firstpage=836; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Ionic effects on the elasticity of single DNA molecules; citation_author=C.G. Baumann; citation_author=S.B. Smith; citation_author=V.A. Bloomfield; citation_author=C. Bustamante; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=94; citation_firstpage=6185; citation_lastpage=6190; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays; citation_author=J.A. Bernstein; citation_author=A.B. Khodursky; citation_author=P.H. Lin; citation_author=S. Lin-Chao; citation_author=S.N. Cohen; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=99; citation_firstpage=9697; citation_lastpage=9702; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION; citation_author=T.A. Bharat; citation_author=S.H. Scheres; citation_journal_title=Nat. Protoc.; citation_volume=11; citation_firstpage=2054; citation_lastpage=2065; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12; citation_author=F.R. Blattner; citation_author=G. Plunkett; citation_author=C.A. Bloch; citation_author=N.T. Perna; citation_author=V. Burland; citation_author=M. Riley; citation_author=J. Collado-Vides; citation_author=J.D. Glasner; citation_author=C.K. Rode; citation_author=G.F. Mayhew; citation_author=et al.; citation_journal_title=Science; citation_volume=277; citation_firstpage=1453; citation_lastpage=1462; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Dependence of DNA conformation on the concentration of salt; citation_author=N. Borochov; citation_author=H. Eisenberg; citation_author=Z. Kam; citation_journal_title=Biopolymers; citation_volume=20; citation_firstpage=231; citation_lastpage=235; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=Structure of the Escherichia coli Ribosome; citation_inbook_title=Molecular cytology of Escherichia coli; citation_author=M. Boublik; citation_publication_date=1985; citation_publisher=Academic Press; "><meta name="citation_reference" content="citation_title=Intracellular transport by active diffusion; citation_author=C.P. Brangwynne; citation_author=G.H. Koenderink; citation_author=F.C. MacKintosh; citation_author=D.A. Weitz; citation_journal_title=Trends Cell Biol.; citation_volume=19; citation_firstpage=423; citation_lastpage=427; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=A protein that binds to the P1 origin core and the oriC 13mer region in a methylation-specific fashion is the product of the host seqA gene; citation_author=T. Brendler; citation_author=A. Abeles; citation_author=S. Austin; citation_journal_title=EMBO J.; citation_volume=14; citation_firstpage=4083; citation_lastpage=4089; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Mammalian reovirus nonstructural protein microNS forms large inclusions and colocalizes with reovirus microtubule-associated protein micro2 in transfected cells; citation_author=T.J. Broering; citation_author=J.S.L. Parker; citation_author=P.L. Joyce; citation_author=J. Kim; citation_author=M.L. Nibert; citation_journal_title=J.&nbsp;Virol.; citation_volume=76; citation_firstpage=8285; citation_lastpage=8297; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Carboxyl-proximal regions of reovirus nonstructural protein muNS necessary and sufficient for forming factory-like inclusions; citation_author=T.J. Broering; citation_author=M.M. Arnold; citation_author=C.L. Miller; citation_author=J.A. Hurt; citation_author=P.L. Joyce; citation_author=M.L. Nibert; citation_journal_title=J.&nbsp;Virol.; citation_volume=79; citation_firstpage=6194; citation_lastpage=6206; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues; citation_author=J.E. Cabrera; citation_author=D.J. Jin; citation_journal_title=Mol. Microbiol.; citation_volume=50; citation_firstpage=1493; citation_lastpage=1505; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Active transcription of rRNA operons condenses the nucleoid in Escherichia coli: examining the effect of transcription on nucleoid structure in the absence of transertion; citation_author=J.E. Cabrera; citation_author=C. Cagliero; citation_author=S. Quan; citation_author=C.L. Squires; citation_author=D.J. Jin; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=191; citation_firstpage=4180; citation_lastpage=4185; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Mobility of Nonsticky Nanoparticles in Polymer Liquids; citation_author=L.H. Cai; citation_author=S. Panyukov; citation_author=M. Rubinstein; citation_journal_title=Macromolecules; citation_volume=44; citation_firstpage=7853; citation_lastpage=7863; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Biopolymer folding driven nanoparticle reorganization in bio-nanocomposites; citation_author=F. Carn; citation_author=F. Boué; citation_author=M. Djabourov; citation_author=N. Steunou; citation_author=T. Coradin; citation_author=J. Livage; citation_author=S. Floquet; citation_author=E. Cadot; citation_author=E. Buhler; citation_journal_title=Soft Matter; citation_volume=8; citation_firstpage=2930; citation_lastpage=2944; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Some statistical properties of flexible ring polymers; citation_author=E.F. Casassa; citation_journal_title=J.&nbsp;Polym. Sci. A1; citation_volume=3; citation_firstpage=605; citation_lastpage=614; citation_publication_date=1965; "><meta name="citation_reference" content="citation_title=Semiflexible Chain Condensation by Neutral Depleting Agents: Role of Correlations between Depletants; citation_author=M. Castelnovo; citation_author=W.M. Gelbart; citation_journal_title=Macromolecules; citation_volume=37; citation_firstpage=3510; citation_lastpage=3517; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo; citation_author=S. Cayley; citation_author=B.A. Lewis; citation_author=H.J. Guttman; citation_author=M.T. Record; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=222; citation_firstpage=281; citation_lastpage=300; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Genome-wide study of mRNA degradation and transcript elongation in Escherichia coli; citation_author=H. Chen; citation_author=K. Shiroguchi; citation_author=H. Ge; citation_author=X.S. Xie; citation_journal_title=Mol. Syst. Biol.; citation_volume=11; citation_firstpage=781; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant; citation_author=P.P. Cherepanov; citation_author=W. Wackernagel; citation_journal_title=Gene; citation_volume=158; citation_firstpage=9; citation_lastpage=14; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Mechanism of transcriptional bursting in bacteria; citation_author=S. Chong; citation_author=C. Chen; citation_author=H. Ge; citation_author=X.S. Xie; citation_journal_title=Cell; citation_volume=158; citation_firstpage=314; citation_lastpage=326; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Diverse bacterial microcompartment organelles; citation_author=C. Chowdhury; citation_author=S. Sinha; citation_author=S. Chun; citation_author=T.O. Yeates; citation_author=T.A. Bobik; citation_journal_title=Microbiol. Mol. Biol. Rev.; citation_volume=78; citation_firstpage=438; citation_lastpage=468; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Feedback regulation of rRNA synthesis in Escherichia coli. Requirement for initiation factor IF2; citation_author=J.R. Cole; citation_author=C.L. Olsson; citation_author=J.W. Hershey; citation_author=M. Grunberg-Manago; citation_author=M. Nomura; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=198; citation_firstpage=383; citation_lastpage=392; citation_publication_date=1987; "><meta name="citation_reference" content="citation_title=Chromosome replication and the division cycle of Escherichia coli B/r; citation_author=S. Cooper; citation_author=C.E. Helmstetter; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=31; citation_firstpage=519; citation_lastpage=540; citation_publication_date=1968; "><meta name="citation_reference" content="citation_title=Probe diffusion in polymer solutions in the dilute/semi-dilute crossover regime: 1. Poly(ethylene oxide); citation_author=E.C. Cooper; citation_author=P. Johnson; citation_author=A.M. Donald; citation_journal_title=Polymer (Guildf.); citation_volume=32; citation_firstpage=2815; citation_lastpage=2822; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Methods of Digital Video Microscopy for Colloidal Studies; citation_author=J.C. Crocker; citation_author=D.G. Grier; citation_journal_title=J.&nbsp;Colloid Interface Sci.; citation_volume=179; citation_firstpage=298; citation_lastpage=310; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Polymer-mediated compaction and internal dynamics of isolated Escherichia coli nucleoids; citation_author=S. Cunha; citation_author=C.L. Woldringh; citation_author=T. Odijk; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=136; citation_firstpage=53; citation_lastpage=66; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth; citation_author=X. Dai; citation_author=M. Zhu; citation_author=M. Warren; citation_author=R. Balakrishnan; citation_author=V. Patsalo; citation_author=H. Okano; citation_author=J.R. Williamson; citation_author=K. Fredrick; citation_author=Y.P. Wang; citation_author=T. Hwa; citation_journal_title=Nat. Microbiol.; citation_volume=2; citation_firstpage=16231; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Chromosome organization in bacteria: mechanistic insights into genome structure and function; citation_author=R.T. Dame; citation_author=F.M. Rashid; citation_author=D.C. Grainger; citation_journal_title=Nat. Rev. Genet.; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Microrheology of polyethylene oxide using diffusing wave spectroscopy and single scattering; citation_author=B.R. Dasgupta; citation_author=S.-Y. Tee; citation_author=J.C. Crocker; citation_author=B.J. Frisken; citation_author=D.A. Weitz; citation_journal_title=Phys. Rev. E Stat. Nonlin. Soft Matter Phys.; citation_volume=65; citation_firstpage=051505; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Scaling concepts in polymer physics; citation_author=P.G. de Gennes; citation_publication_date=1979; citation_publisher=Cornell University Press; "><meta name="citation_reference" content="citation_title=Depletion-induced instability in protein-DNA mixtures: Influence of protein charge and size; citation_author=R. de Vries; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=125; citation_firstpage=014905; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=DNA condensation in bacteria: Interplay between macromolecular crowding and nucleoid proteins; citation_author=R. de Vries; citation_journal_title=Biochimie; citation_volume=92; citation_firstpage=1715; citation_lastpage=1721; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Organization of supercoil domains and their reorganization by transcription; citation_author=S. Deng; citation_author=R.A. Stein; citation_author=N.P. Higgins; citation_journal_title=Mol. Microbiol.; citation_volume=57; citation_firstpage=1511; citation_lastpage=1521; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Orientation relaxation of DNA restriction fragments and the internal mobility of the double helix; citation_author=S. Diekmann; citation_author=W. Hillen; citation_author=B. Morgeneyer; citation_author=R.D. Wells; citation_author=D. Pörschke; citation_journal_title=Biophys. Chem.; citation_volume=15; citation_firstpage=263; citation_lastpage=270; citation_publication_date=1982; "><meta name="citation_reference" content="citation_title=Bacterial nucleoid-associated proteins, nucleoid structure and gene expression; citation_author=S.C. Dillon; citation_author=C.J. Dorman; citation_journal_title=Nat. Rev. Microbiol.; citation_volume=8; citation_firstpage=185; citation_lastpage=195; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=DNA supercoiling and transcription in bacteria: a two-way street; citation_author=C.J. Dorman; citation_journal_title=BMC Mol. Cell Biol.; citation_volume=20; citation_firstpage=26; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Effect of rifampin on the structure and membrane attachment of the nucleoid of Escherichia coli; citation_author=P. Dworsky; citation_author=M. Schaechter; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=116; citation_firstpage=1364; citation_lastpage=1374; citation_publication_date=1973; "><meta name="citation_reference" content="citation_title=Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of Escherichia coli 15; citation_author=J. Forchhammer; citation_author=L. Lindahl; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=55; citation_firstpage=563; citation_lastpage=568; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=The structural basis for release-factor activation during translation termination revealed by time-resolved cryogenic electron microscopy; citation_author=Z. Fu; citation_author=G. Indrisiunaite; citation_author=S. Kaledhonkar; citation_author=B. Shah; citation_author=M. Sun; citation_author=B. Chen; citation_author=R.A. Grassucci; citation_author=M. Ehrenberg; citation_author=J. Frank; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=2579; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=UCSF ChimeraX: Meeting modern challenges in visualization and analysis; citation_author=T.D. Goddard; citation_author=C.C. Huang; citation_author=E.C. Meng; citation_author=E.F. Pettersen; citation_author=G.S. Couch; citation_author=J.H. Morris; citation_author=T.E. Ferrin; citation_journal_title=Protein Sci.; citation_volume=27; citation_firstpage=14; citation_lastpage=25; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria; citation_author=W.T. Gray; citation_author=S.K. Govers; citation_author=Y. Xiang; citation_author=B.R. Parry; citation_author=M. Campos; citation_author=S. Kim; citation_author=C. Jacobs-Wagner; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1632; citation_lastpage=1648.e1620; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Probing the stochastic, motor-driven properties of the cytoplasm using force spectrum microscopy; citation_author=M. Guo; citation_author=A.J. Ehrlicher; citation_author=M.H. Jensen; citation_author=M. Renz; citation_author=J.R. Moore; citation_author=R.D. Goldman; citation_author=J. Lippincott-Schwartz; citation_author=F.C. Mackintosh; citation_author=D.A. Weitz; citation_journal_title=Cell; citation_volume=158; citation_firstpage=822; citation_lastpage=832; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Variation of the folding and dynamics of the Escherichia coli chromosome with growth conditions; citation_author=N. Hadizadeh Yazdi; citation_author=C.C. Guet; citation_author=R.C. Johnson; citation_author=J.F. Marko; citation_journal_title=Mol. Microbiol.; citation_volume=86; citation_firstpage=1318; citation_lastpage=1333; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Array programming with NumPy; citation_author=C.R. Harris; citation_author=K.J. Millman; citation_author=S.J. van der Walt; citation_author=R. Gommers; citation_author=P. Virtanen; citation_author=D. Cournapeau; citation_author=E. Wieser; citation_author=J. Taylor; citation_author=S. Berg; citation_author=N.J. Smith; citation_author=et al.; citation_journal_title=Nature; citation_volume=585; citation_firstpage=357; citation_lastpage=362; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Dynamic Escherichia coli SeqA complexes organize the newly replicated DNA at a considerable distance from the replisome; citation_author=E. Helgesen; citation_author=S. Fossum-Raunehaug; citation_author=F. Sætre; citation_author=K.O. Schink; citation_author=K. Skarstad; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=2730; citation_lastpage=2743; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Chromosome replication and segregation govern the biogenesis and inheritance of inorganic polyphosphate granules; citation_author=J.T. Henry; citation_author=S. Crosson; citation_journal_title=Mol. Biol. Cell; citation_volume=24; citation_firstpage=3177; citation_lastpage=3186; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Guanosine tetraphosphate (ppGpp) dependence of the growth rate control of rrnB P1 promoter activity in Escherichia coli; citation_author=V.J. Hernandez; citation_author=H. Bremer; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=265; citation_firstpage=11605; citation_lastpage=11614; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=emClarity: software for high-resolution cryo-electron tomography and subtomogram averaging; citation_author=B.A. Himes; citation_author=P. Zhang; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=955; citation_lastpage=961; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Shape and fine structure of nucleoids observed on sections of ultrarapidly frozen and cryosubstituted bacteria; citation_author=J.A. Hobot; citation_author=W. Villiger; citation_author=J. Escaig; citation_author=M. Maeder; citation_author=A. Ryter; citation_author=E. Kellenberger; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=162; citation_firstpage=960; citation_lastpage=971; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=Design of a hyperstable 60-subunit protein dodecahedron. [corrected]; citation_author=Y. Hsia; citation_author=J.B. Bale; citation_author=S. Gonen; citation_author=D. Shi; citation_author=W. Sheffler; citation_author=K.K. Fong; citation_author=U. Nattermann; citation_author=C. Xu; citation_author=P.S. Huang; citation_author=R. Ravichandran; citation_author=et al.; citation_journal_title=Nature; citation_volume=535; citation_firstpage=136; citation_lastpage=139; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Matplotlib: A 2D Graphics Environment; citation_author=J.D. Hunter; citation_journal_title=Comput. Sci. Eng.; citation_volume=9; citation_firstpage=90; citation_lastpage=95; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The Escherichia coli K-12 “wild types” W3110 and MG1655 have an rph frameshift mutation that leads to pyrimidine starvation due to low pyrE expression levels; citation_author=K.F. Jensen; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=175; citation_firstpage=3401; citation_lastpage=3407; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Effects of molecular crowding and confinement on the spatial organization of a biopolymer; citation_author=C. Jeon; citation_author=Y. Jung; citation_author=B.-Y. Ha; citation_journal_title=Soft Matter; citation_volume=12; citation_firstpage=9436; citation_lastpage=9450; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Compaction of bacterial genomic DNA: clarifying the concepts; citation_author=M. Joyeux; citation_journal_title=J.&nbsp;Phys. Condens. Matter; citation_volume=27; citation_firstpage=383001; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=In&nbsp;vivo compaction dynamics of bacterial DNA: A fingerprint of DNA/RNA demixing?; citation_author=M. Joyeux; citation_journal_title=Curr. Opin. Colloid Interface Sci.; citation_volume=26; citation_firstpage=17; citation_lastpage=27; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Chromosome, cell cycle, and entropy; citation_author=S. Jun; citation_journal_title=Biophys. J.; citation_volume=108; citation_firstpage=785; citation_lastpage=786; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Dependence of laser light scattering of DNA on NaCl concentration; citation_author=Z. Kam; citation_author=N. Borochov; citation_author=H. Eisenberg; citation_journal_title=Biopolymers; citation_volume=20; citation_firstpage=2671; citation_lastpage=2690; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=Extracellular RNA constitutes a natural procoagulant cofactor in blood coagulation; citation_author=C. Kannemeier; citation_author=A. Shibamiya; citation_author=F. Nakazawa; citation_author=H. Trusheim; citation_author=C. Ruppert; citation_author=P. Markart; citation_author=Y. Song; citation_author=E. Tzima; citation_author=E. Kennerknecht; citation_author=M. Niepmann; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_firstpage=6388; citation_lastpage=6393; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Electron microscope study of DNA-containing plasms. II. Vegetative and mature phage DNA as compared with normal bacterial nucleoids in different physiological states; citation_author=E. Kellenberger; citation_author=A. Ryter; citation_author=J. Sechaud; citation_journal_title=J.&nbsp;Biophys. Biochem. Cytol.; citation_volume=4; citation_firstpage=671; citation_lastpage=678; citation_publication_date=1958; "><meta name="citation_reference" content="citation_title=A polymer in a crowded and confined space: effects of crowder size and poly-dispersity; citation_author=J. Kim; citation_author=C. Jeon; citation_author=H. Jeong; citation_author=Y. Jung; citation_author=B.Y. Ha; citation_journal_title=Soft Matter; citation_volume=11; citation_firstpage=1877; citation_lastpage=1888; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The Behavior of Macromolecules in Inhomogeneous Flow; citation_author=H.A. Kramers; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=14; citation_firstpage=415; citation_lastpage=424; citation_publication_date=1946; "><meta name="citation_reference" content="citation_title=Computer visualization of three-dimensional image data using IMOD; citation_author=J.R. Kremer; citation_author=D.N. Mastronarde; citation_author=J.R. McIntosh; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=116; citation_firstpage=71; citation_lastpage=76; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Productive phage infection in Escherichia coli with reduced internal levels of the major cations; citation_author=A. Kuhn; citation_author=E. Kellenberger; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=163; citation_firstpage=906; citation_lastpage=912; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=Regulation of ribosomal and transfer RNA synthesis; citation_author=C.G. Kurland; citation_author=O. Maaloe; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=4; citation_firstpage=193; citation_lastpage=210; citation_publication_date=1962; "><meta name="citation_reference" content="citation_title=Transcription rate and transcript length drive formation of chromosomal interaction domain boundaries; citation_author=T.B. Le; citation_author=M.T. Laub; citation_journal_title=EMBO J.; citation_volume=35; citation_firstpage=1582; citation_lastpage=1595; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=High-resolution mapping of the spatial organization of a bacterial chromosome; citation_author=T.B. Le; citation_author=M.V. Imakaev; citation_author=L.A. Mirny; citation_author=M.T. Laub; citation_journal_title=Science; citation_volume=342; citation_firstpage=731; citation_lastpage=734; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Bacterial partition complexes segregate within the volume of the nucleoid; citation_author=A. Le Gall; citation_author=D.I. Cattoni; citation_author=B. Guilhas; citation_author=C. Mathieu-Demazière; citation_author=L. Oudjedi; citation_author=J.-B. Fiche; citation_author=J. Rech; citation_author=S. Abrahamsson; citation_author=H. Murray; citation_author=J.-Y. Bouet; citation_author=M. Nollmann; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=12107; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Factors affecting chemical-based purification of DNA from Saccharomyces cerevisiae; citation_author=C.K. Lee; citation_author=N. Araki; citation_author=D.S. Sowersby; citation_author=L.K. Lewis; citation_journal_title=Yeast; citation_volume=29; citation_firstpage=73; citation_lastpage=80; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Compartmentalization of transcription and translation in Bacillus subtilis; citation_author=P.J. Lewis; citation_author=S.D. Thaker; citation_author=J. Errington; citation_journal_title=EMBO J.; citation_volume=19; citation_firstpage=710; citation_lastpage=718; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources; citation_author=G.W. Li; citation_author=D. Burkhardt; citation_author=C. Gross; citation_author=J.S. Weissman; citation_journal_title=Cell; citation_volume=157; citation_firstpage=624; citation_lastpage=635; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Membrane protein expression triggers chromosomal locus repositioning in bacteria; citation_author=E.A. Libby; citation_author=M. Roggiani; citation_author=M. Goulian; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=7445; citation_lastpage=7450; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins; citation_author=V.S. Lioy; citation_author=A. Cournac; citation_author=M. Marbouty; citation_author=S. Duigou; citation_author=J. Mozziconacci; citation_author=O. Espéli; citation_author=F. Boccard; citation_author=R. Koszul; citation_journal_title=Cell; citation_volume=172; citation_firstpage=771; citation_lastpage=783.e18; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Intact flagellar motor of Borrelia burgdorferi revealed by cryo-electron tomography: evidence for stator ring curvature and rotor/C-ring assembly flexion; citation_author=J. Liu; citation_author=T. Lin; citation_author=D.J. Botkin; citation_author=E. McCrum; citation_author=H. Winkler; citation_author=S.J. Norris; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=191; citation_firstpage=5026; citation_lastpage=5036; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Translation inhibitors stabilize Escherichia coli mRNAs independently of ribosome protection; citation_author=P.J. Lopez; citation_author=I. Marchand; citation_author=O. Yarchuk; citation_author=M. Dreyfus; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=95; citation_firstpage=6067; citation_lastpage=6072; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=SeqA: a negative modulator of replication initiation in E.&nbsp;coli; citation_author=M. Lu; citation_author=J.L. Campbell; citation_author=E. Boye; citation_author=N. Kleckner; citation_journal_title=Cell; citation_volume=77; citation_firstpage=413; citation_lastpage=426; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Magnesium and the growth of Escherichia coli; citation_author=J.E. Lusk; citation_author=R.J.P. Williams; citation_author=E.P. Kennedy; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=243; citation_firstpage=2618; citation_lastpage=2624; citation_publication_date=1968; "><meta name="citation_reference" content="citation_title=DNA supercoiling during transcription; citation_author=J. Ma; citation_author=M.D. Wang; citation_journal_title=Biophys. Rev.; citation_volume=8; citation_number=Suppl 1; citation_firstpage=75; citation_lastpage=87; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Conformational analysis and estimation of the persistence length of DNA using atomic force microscopy in solution; citation_author=S. Mantelli; citation_author=P. Muller; citation_author=S. Harlepp; citation_author=M. Maaloum; citation_journal_title=Soft Matter; citation_volume=7; citation_firstpage=3412; citation_lastpage=3416; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms; citation_author=M. Marbouty; citation_author=A. Cournac; citation_author=J.F. Flot; citation_author=H. Marie-Nelly; citation_author=J. Mozziconacci; citation_author=R. Koszul; citation_journal_title=eLife; citation_volume=3; citation_firstpage=e03318; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging; citation_author=M. Marbouty; citation_author=A. Le Gall; citation_author=D.I. Cattoni; citation_author=A. Cournac; citation_author=A. Koh; citation_author=J.B. Fiche; citation_author=J. Mozziconacci; citation_author=H. Murray; citation_author=R. Koszul; citation_author=M. Nollmann; citation_journal_title=Mol. Cell; citation_volume=59; citation_firstpage=588; citation_lastpage=602; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Nuclear division as observed in live bacteria by a new technique; citation_author=D.J. Mason; citation_author=D.M. Powelson; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=71; citation_firstpage=474; citation_lastpage=479; citation_publication_date=1956; "><meta name="citation_reference" content="citation_title=Automated electron microscope tomography using robust prediction of specimen movements; citation_author=D.N. Mastronarde; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=152; citation_firstpage=36; citation_lastpage=51; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Data Structures for Statistical Computing in Python; citation_author=W. McKinney; citation_journal_title=Proceedings of the 9th Python in Science Conference; citation_doi=10.25080/Majora-92bf1922-00a; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Functional mapping of the E.&nbsp;coli translational machinery using single-molecule tracking; citation_author=S. Mohapatra; citation_author=J.C. Weisshaar; citation_journal_title=Mol. Microbiol.; citation_volume=110; citation_firstpage=262; citation_lastpage=282; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Replication fork and SeqA focus distributions in Escherichia coli suggest a replication hyperstructure dependent on nucleotide metabolism; citation_author=F. Molina; citation_author=K. Skarstad; citation_journal_title=Mol. Microbiol.; citation_volume=52; citation_firstpage=1597; citation_lastpage=1612; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=High magnesium content of Escherichia coli B; citation_author=M.L.J. Moncany; citation_author=E. Kellenberger; citation_journal_title=Experientia; citation_volume=37; citation_firstpage=846; citation_lastpage=847; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=Entropy-based mechanism of ribosome-nucleoid segregation in E.&nbsp;coli cells; citation_author=J. Mondal; citation_author=B.P. Bratton; citation_author=Y. Li; citation_author=A. Yethiraj; citation_author=J.C. Weisshaar; citation_journal_title=Biophys. J.; citation_volume=100; citation_firstpage=2605; citation_lastpage=2613; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Colloid-polymer mixtures in the protein limit; citation_author=K.J. Mutch; citation_author=J.S. van Duijneveldt; citation_author=J. Eastoe; citation_journal_title=Soft Matter; citation_volume=3; citation_firstpage=155; citation_lastpage=167; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The ionic strength dependence of the second virial coefficient of low molar mass DNA fragments in aqueous solutions; citation_author=T. Nicolai; citation_author=M. Mandel; citation_journal_title=Macromolecules; citation_volume=22; citation_firstpage=438; citation_lastpage=444; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=In-cell architecture of an actively transcribing-translating expressome; citation_author=F.J. O’Reilly; citation_author=L. Xue; citation_author=A. Graziadei; citation_author=L. Sinn; citation_author=S. Lenz; citation_author=D. Tegunov; citation_author=C. Blötz; citation_author=N. Singh; citation_author=W.J.H. Hagen; citation_author=P. Cramer; citation_author=et al.; citation_journal_title=Science; citation_volume=369; citation_firstpage=554; citation_lastpage=557; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Osmotic compaction of supercoiled DNA into a bacterial nucleoid; citation_author=T. Odijk; citation_journal_title=Biophys. Chem.; citation_volume=73; citation_firstpage=23; citation_lastpage=29; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis; citation_author=A. Paintdakhi; citation_author=B. Parry; citation_author=M. Campos; citation_author=I. Irnov; citation_author=J. Elf; citation_author=I. Surovtsev; citation_author=C. Jacobs-Wagner; citation_journal_title=Mol. Microbiol.; citation_volume=99; citation_firstpage=767; citation_lastpage=777; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity; citation_author=B.R. Parry; citation_author=I.V. Surovtsev; citation_author=M.T. Cabeen; citation_author=C.S. O’Hern; citation_author=E.R. Dufresne; citation_author=C. Jacobs-Wagner; citation_journal_title=Cell; citation_volume=156; citation_firstpage=183; citation_lastpage=194; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Messenger ribonucleic acid synthesis and degradation in Escherichia coli during inhibition of translation; citation_author=M.L. Pato; citation_author=P.M. Bennett; citation_author=K. von Meyenburg; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=116; citation_firstpage=710; citation_lastpage=718; citation_publication_date=1973; "><meta name="citation_reference" content="citation_title=Rapid purification of the estrogen receptor by sequence-specific DNA affinity chromatography; citation_author=F.V. Peale; citation_author=Y. Ishibe; citation_author=C.M. Klinge; citation_author=S. Zain; citation_author=R. Hilf; citation_author=R.A. Bambara; citation_journal_title=Biochemistry; citation_volume=28; citation_firstpage=8671; citation_lastpage=8675; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Physical manipulation of the Escherichia coli chromosome reveals its soft nature; citation_author=J. Pelletier; citation_author=K. Halvorsen; citation_author=B.Y. Ha; citation_author=R. Paparcone; citation_author=S.J. Sandler; citation_author=C.L. Woldringh; citation_author=W.P. Wong; citation_author=S. Jun; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=E2649; citation_lastpage=E2656; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=RNA molecules bound to the folded bacterial genome stabilize DNA folds and segregate domains of supercoiling; citation_author=D.E. Pettijohn; citation_author=R. Hecht; citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.; citation_volume=38; citation_firstpage=31; citation_lastpage=41; citation_publication_date=1974; "><meta name="citation_reference" content="citation_title=Polyribosomes of Escherichia coli. II. Experiments to determine the in&nbsp;vivo distribution of polysomes, ribosomes and ribosomal subunits; citation_author=L.A. Phillips; citation_author=B. Hotham-Iglewski; citation_author=R.M. Franklin; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=45; citation_firstpage=23; citation_lastpage=38; citation_publication_date=1969; "><meta name="citation_reference" content="citation_title=Zytologische Untersuchungen an Paratyphusund Coli Bakterien; citation_author=G. Piekarski; citation_journal_title=Arch. Mikrobiol.; citation_volume=8; citation_firstpage=428; citation_lastpage=429; citation_publication_date=1937; "><meta name="citation_reference" content="citation_title=Structure and dynamics of double helices in solution: modes of DNA bending; citation_author=D. Porschke; citation_journal_title=J.&nbsp;Biomol. Struct. Dyn.; citation_volume=4; citation_firstpage=373; citation_lastpage=389; citation_publication_date=1986; "><meta name="citation_reference" content="citation_title=Persistence length and bending dynamics of DNA from electrooptical measurements at high salt concentrations; citation_author=D. Porschke; citation_journal_title=Biophys. Chem.; citation_volume=40; citation_firstpage=169; citation_lastpage=179; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Topological domain structure of the Escherichia coli chromosome; citation_author=L. Postow; citation_author=C.D. Hardy; citation_author=J. Arsuaga; citation_author=N.R. Cozzarelli; citation_journal_title=Genes Dev.; citation_volume=18; citation_firstpage=1766; citation_lastpage=1779; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Inter-DNA attraction mediated by divalent counterions; citation_author=X. Qiu; citation_author=K. Andresen; citation_author=L.W. Kwok; citation_author=J.S. Lamb; citation_author=H.Y. Park; citation_author=L. Pollack; citation_journal_title=Phys. Rev. Lett.; citation_volume=99; citation_firstpage=038104; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Polyphosphate granule biogenesis is temporally and functionally tied to cell cycle exit during starvation in Pseudomonas aeruginosa; citation_author=L.R. Racki; citation_author=E.I. Tocheva; citation_author=M.G. Dieterle; citation_author=M.C. Sullivan; citation_author=G.J. Jensen; citation_author=D.K. Newman; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=114; citation_firstpage=E2440; citation_lastpage=E2449; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Entropy driven phase transitions in colloid–polymer suspensions: Tests of depletion theories; citation_author=S. Ramakrishnan; citation_author=M. Fuchs; citation_author=K.S. Schweizer; citation_author=C.F. Zukoski; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=116; citation_firstpage=2201; citation_lastpage=2212; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=A simple model for gene targeting; citation_author=T. Ratilainen; citation_author=P. Lincoln; citation_author=B. Nordén; citation_journal_title=Biophys. J.; citation_volume=81; citation_firstpage=2876; citation_lastpage=2885; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=High-copy bacterial plasmids diffuse in the nucleoid-free space, replicate stochastically and are randomly partitioned at cell division; citation_author=R. Reyes-Lamothe; citation_author=T. Tran; citation_author=D. Meas; citation_author=L. Lee; citation_author=A.M. Li; citation_author=D.J. Sherratt; citation_author=M.E. Tolmasky; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=1042; citation_lastpage=1051; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The bacterial nucleoid revisited; citation_author=C. Robinow; citation_author=E. Kellenberger; citation_journal_title=Microbiol. Rev.; citation_volume=58; citation_firstpage=211; citation_lastpage=232; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Perturbation of anion balance during inhibition of growth of Escherichia coli by weak acids; citation_author=A.J. Roe; citation_author=D. McLaggan; citation_author=I. Davidson; citation_author=C. O’Byrne; citation_author=I.R. Booth; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=180; citation_firstpage=767; citation_lastpage=772; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Polymer physics; citation_author=M. Rubinstein; citation_publication_date=2003; citation_publisher=Oxford University Press; "><meta name="citation_reference" content="citation_title=Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid; citation_author=A. Sanamrad; citation_author=F. Persson; citation_author=E.G. Lundius; citation_author=D. Fange; citation_author=A.H. Gynnå; citation_author=J. Elf; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=11413; citation_lastpage=11418; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Direct observation of fusion of bacterial nuclei; citation_author=M. Schaechter; citation_author=V.O. Laing; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=81; citation_firstpage=667; citation_lastpage=668; citation_publication_date=1961; "><meta name="citation_reference" content="citation_title=The antibiotic kasugamycin mimics mRNA nucleotides to destabilize tRNA binding and inhibit canonical translation initiation; citation_author=F. Schluenzen; citation_author=C. Takemoto; citation_author=D.N. Wilson; citation_author=T. Kaminishi; citation_author=J.M. Harms; citation_author=K. Hanawa-Suetsugu; citation_author=W. Szaflarski; citation_author=M. Kawazoe; citation_author=M. Shirouzu; citation_author=K.H. Nierhaus; citation_author=et al.; citation_journal_title=Nat. Struct. Mol. Biol.; citation_volume=13; citation_firstpage=871; citation_lastpage=878; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Translation and mRNA decay; citation_author=E. Schneider; citation_author=M. Blundell; citation_author=D. Kennell; citation_journal_title=Mol. Gen. Genet.; citation_volume=160; citation_firstpage=121; citation_lastpage=129; citation_publication_date=1978; "><meta name="citation_reference" content="citation_title=Cation transport in Escherichia coli. II. Intracellular chloride concentration; citation_author=S.G. Schultz; citation_author=N.L. Wilson; citation_author=W. Epstein; citation_journal_title=J.&nbsp;Gen. Physiol.; citation_volume=46; citation_firstpage=159; citation_lastpage=166; citation_publication_date=1962; "><meta name="citation_reference" content="citation_title=Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation; citation_author=D.W. Selinger; citation_author=R.M. Saxena; citation_author=K.J. Cheung; citation_author=G.M. Church; citation_author=C. Rosenow; citation_journal_title=Genome Res.; citation_volume=13; citation_firstpage=216; citation_lastpage=223; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Bacillus subtilis cell cycle as studied by fluorescence microscopy: constancy of cell length at initiation of DNA replication and evidence for active nucleoid partitioning; citation_author=M.E. Sharpe; citation_author=P.M. Hauser; citation_author=R.G. Sharpe; citation_author=J. Errington; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=180; citation_firstpage=547; citation_lastpage=555; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Simulating the entropic collapse of coarse-grained chromosomes; citation_author=T.N. Shendruk; citation_author=M. Bertrand; citation_author=H.W. de Haan; citation_author=J.L. Harden; citation_author=G.W. Slater; citation_journal_title=Biophys. J.; citation_volume=108; citation_firstpage=810; citation_lastpage=820; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Measurements of attractive forces between proteins and end-grafted poly(ethylene glycol) chains; citation_author=S.R. Sheth; citation_author=D. Leckband; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=94; citation_firstpage=8399; citation_lastpage=8404; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Micromechanical analysis of the binding of DNA-bending proteins HMGB1, NHP6A, and HU reveals their ability to form highly stable DNA-protein complexes; citation_author=D. Skoko; citation_author=B. Wong; citation_author=R.C. Johnson; citation_author=J.F. Marko; citation_journal_title=Biochemistry; citation_volume=43; citation_firstpage=13867; citation_lastpage=13874; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=E.&nbsp;coli SeqA protein binds oriC in two different methyl-modulated reactions appropriate to its roles in DNA replication initiation and origin sequestration; citation_author=S. Slater; citation_author=S. Wold; citation_author=M. Lu; citation_author=E. Boye; citation_author=K. Skarstad; citation_author=N. Kleckner; citation_journal_title=Cell; citation_volume=82; citation_firstpage=927; citation_lastpage=936; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=High-throughput, subpixel precision analysis of bacterial morphogenesis and intracellular spatio-temporal dynamics; citation_author=O. Sliusarenko; citation_author=J. Heinritz; citation_author=T. Emonet; citation_author=C. Jacobs-Wagner; citation_journal_title=Mol. Microbiol.; citation_volume=80; citation_firstpage=612; citation_lastpage=627; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Effects of Na+ on the persistence length and excluded volume of T7 bacteriophage DNA; citation_author=E.S. Sobel; citation_author=J.A. Harpst; citation_journal_title=Biopolymers; citation_volume=31; citation_firstpage=1559; citation_lastpage=1564; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Super-resolution imaging of Escherichia coli nucleoids reveals highly structured and asymmetric segregation during fast growth; citation_author=C. Spahn; citation_author=U. Endesfelder; citation_author=M. Heilemann; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=185; citation_firstpage=243; citation_lastpage=249; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A toolbox for multiplexed super-resolution imaging of the E.&nbsp;coli nucleoid and membrane using novel PAINT labels; citation_author=C.K. Spahn; citation_author=M. Glaesmann; citation_author=J.B. Grimm; citation_author=A.X. Ayala; citation_author=L.D. Lavis; citation_author=M. Heilemann; citation_journal_title=Sci. Rep.; citation_volume=8; citation_firstpage=14768; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Lethal overproduction of the Escherichia coli nucleoid protein H-NS: ultramicroscopic and molecular autopsy; citation_author=R. Spurio; citation_author=M. Dürrenberger; citation_author=M. Falconi; citation_author=A. La Teana; citation_author=C.L. Pon; citation_author=C.O. Gualerzi; citation_journal_title=Mol. Gen. Genet.; citation_volume=231; citation_firstpage=201; citation_lastpage=211; citation_publication_date=1992; "><meta name="citation_reference" content="citation_title=DNA-protein interactions and bacterial chromosome architecture; citation_author=J. Stavans; citation_author=A. Oppenheim; citation_journal_title=Phys. Biol.; citation_volume=3; citation_firstpage=R1; citation_lastpage=R10; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Live-cell superresolution microscopy reveals the organization of RNA polymerase in the bacterial nucleoid; citation_author=M. Stracy; citation_author=C. Lesterlin; citation_author=F. Garza de Leon; citation_author=S. Uphoff; citation_author=P. Zawadzki; citation_author=A.N. Kapanidis; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E4390; citation_lastpage=E4399; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Effects of perturbing nucleoid structure on nucleoid occlusion-mediated toporegulation of FtsZ ring assembly; citation_author=Q. Sun; citation_author=W. Margolin; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=186; citation_firstpage=3951; citation_lastpage=3959; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Subcellular Organization: A Critical Feature of Bacterial Cell Replication; citation_author=I.V. Surovtsev; citation_author=C. Jacobs-Wagner; citation_journal_title=Cell; citation_volume=172; citation_firstpage=1271; citation_lastpage=1293; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Actions of aurintricarboxylate, kasugamycin, and pactamycin on Escherichia coli polysomes; citation_author=P.C. Tai; citation_author=B.J. Wallace; citation_author=B.D. Davis; citation_journal_title=Biochemistry; citation_volume=12; citation_firstpage=616; citation_lastpage=620; citation_publication_date=1973; "><meta name="citation_reference" content="citation_title=Micro-organization and visco-elasticity of the interphase nucleus revealed by particle nanotracking; citation_author=Y. Tseng; citation_author=J.S. Lee; citation_author=T.P. Kole; citation_author=I. Jiang; citation_author=D. Wirtz; citation_journal_title=J.&nbsp;Cell Sci.; citation_volume=117; citation_firstpage=2159; citation_lastpage=2167; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A checkpoint control orchestrates the replication of the two chromosomes of Vibrio cholerae; citation_author=M.E. Val; citation_author=M. Marbouty; citation_author=F. de Lemos Martins; citation_author=S.P. Kennedy; citation_author=H. Kemble; citation_author=M.J. Bland; citation_author=C. Possoz; citation_author=R. Koszul; citation_author=O. Skovgaard; citation_author=D. Mazel; citation_journal_title=Sci. Adv.; citation_volume=2; citation_firstpage=e1501914; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Chloramphenicol causes fusion of separated nucleoids in Escherichia coli K-12 cells and filaments; citation_author=J.M. van Helvoort; citation_author=J. Kool; citation_author=C.L. Woldringh; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=178; citation_firstpage=4289; citation_lastpage=4293; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Dual architectural roles of HU: formation of flexible hinges and rigid filaments; citation_author=J. van Noort; citation_author=S. Verbrugge; citation_author=N. Goosen; citation_author=C. Dekker; citation_author=R.T. Dame; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=6969; citation_lastpage=6974; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Ribosome patterns in Escherichia coli growing at various rates; citation_author=F. Varricchio; citation_author=R. Monier; citation_journal_title=J.&nbsp;Bacteriol.; citation_volume=108; citation_firstpage=105; citation_lastpage=110; citation_publication_date=1971; "><meta name="citation_reference" content="citation_title=Architecture of the Escherichia coli nucleoid; citation_author=S.C. Verma; citation_author=Z. Qian; citation_author=S.L. Adhya; citation_journal_title=PLoS Genet.; citation_volume=15; citation_firstpage=e1008456; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Investigation of the 50&nbsp;S ribosomal subunit by electron microscopy and image analysis; citation_author=A. Verschoor; citation_author=J. Frank; citation_author=M. Boublik; citation_journal_title=J.&nbsp;Ultrastruct. Res.; citation_volume=92; citation_firstpage=180; citation_lastpage=189; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=SciPy 1.0: fundamental algorithms for scientific computing in Python; citation_author=P. Virtanen; citation_author=R. Gommers; citation_author=T.E. Oliphant; citation_author=M. Haberland; citation_author=T. Reddy; citation_author=D. Cournapeau; citation_author=E. Burovski; citation_author=P. Peterson; citation_author=W. Weckesser; citation_author=J. Bright; citation_author=SciPy 1.0 Contributors; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=17; citation_firstpage=261; citation_lastpage=272; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Replication fork movement and methylation govern SeqA binding to the Escherichia coli chromosome; citation_author=T. Waldminghaus; citation_author=C. Weigel; citation_author=K. Skarstad; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=5465; citation_lastpage=5476; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The Synchronization of Replication and Division Cycles in Individual E.&nbsp;coli Cells; citation_author=M. Wallden; citation_author=D. Fange; citation_author=E.G. Lundius; citation_author=Ö. Baltekin; citation_author=J. Elf; citation_journal_title=Cell; citation_volume=166; citation_firstpage=729; citation_lastpage=739; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis; citation_author=X. Wang; citation_author=T.B. Le; citation_author=B.R. Lajoie; citation_author=J. Dekker; citation_author=M.T. Laub; citation_author=D.Z. Rudner; citation_journal_title=Genes Dev.; citation_volume=29; citation_firstpage=1661; citation_lastpage=1675; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Quantitative Localization Microscopy Reveals a Novel Organization of a High-Copy Number Plasmid; citation_author=Y. Wang; citation_author=P. Penkul; citation_author=J.N. Milstein; citation_journal_title=Biophys. J.; citation_volume=111; citation_firstpage=467; citation_lastpage=479; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus; citation_author=X. Wang; citation_author=H.B. Brandão; citation_author=T.B. Le; citation_author=M.T. Laub; citation_author=D.Z. Rudner; citation_journal_title=Science; citation_volume=355; citation_firstpage=524; citation_lastpage=527; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Compaction of isolated Escherichia coli nucleoids: Polymer and H-NS protein synergetics; citation_author=A.S. Wegner; citation_author=K. Wintraecken; citation_author=R. Spurio; citation_author=C.L. Woldringh; citation_author=R. de Vries; citation_author=T. Odijk; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=194; citation_firstpage=129; citation_lastpage=137; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Strong intranucleoid interactions organize the Escherichia coli chromosome into a nucleoid filament; citation_author=P.A. Wiggins; citation_author=K.C. Cheveralls; citation_author=J.S. Martin; citation_author=R. Lintner; citation_author=J. Kondev; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=107; citation_firstpage=4991; citation_lastpage=4995; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=3D reconstruction and processing of volumetric data in cryo-electron tomography; citation_author=H. Winkler; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=157; citation_firstpage=126; citation_lastpage=137; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Tomographic subvolume alignment and subvolume classification applied to myosin V and SIV envelope spikes; citation_author=H. Winkler; citation_author=P. Zhu; citation_author=J. Liu; citation_author=F. Ye; citation_author=K.H. Roux; citation_author=K.A. Taylor; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=165; citation_firstpage=64; citation_lastpage=77; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Quantitative and spatio-temporal features of protein aggregation in Escherichia coli and consequences on protein quality control and cellular ageing; citation_author=J. Winkler; citation_author=A. Seybert; citation_author=L. König; citation_author=S. Pruggnaller; citation_author=U. Haselmann; citation_author=V. Sourjik; citation_author=M. Weiss; citation_author=A.S. Frangakis; citation_author=A. Mogk; citation_author=B. Bukau; citation_journal_title=EMBO J.; citation_volume=29; citation_firstpage=910; citation_lastpage=923; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The role of co-transcriptional translation and protein translocation (transertion) in bacterial chromosome segregation; citation_author=C.L. Woldringh; citation_journal_title=Mol. Microbiol.; citation_volume=45; citation_firstpage=17; citation_lastpage=29; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Structure and partitioning of bacterial DNA: determined by a balance of compaction and expansion forces?; citation_author=C.L. Woldringh; citation_author=P.R. Jensen; citation_author=H.V. Westerhoff; citation_journal_title=FEMS Microbiol. Lett.; citation_volume=131; citation_firstpage=235; citation_lastpage=242; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Anomalous diffusion probes microstructure dynamics of entangled F-actin networks; citation_author=I.Y. Wong; citation_author=M.L. Gardel; citation_author=D.R. Reichman; citation_author=E.R. Weeks; citation_author=M.T. Valentine; citation_author=A.R. Bausch; citation_author=D.A. Weitz; citation_journal_title=Phys. Rev. Lett.; citation_volume=92; citation_firstpage=178101; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Cell Boundary Confinement Sets the Size and Position of the E.&nbsp;coli Chromosome; citation_author=F. Wu; citation_author=P. Swain; citation_author=L. Kuijpers; citation_author=X. Zheng; citation_author=K. Felter; citation_author=M. Guurink; citation_author=J. Solari; citation_author=S. Jun; citation_author=T.S. Shimizu; citation_author=D. Chaudhuri; citation_author=et al.; citation_journal_title=Curr. Biol.; citation_volume=29; citation_firstpage=2131; citation_lastpage=2144.e4; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Transcription and translation contribute to gene locus relocation to the nucleoid periphery in E.&nbsp;coli; citation_author=S. Yang; citation_author=S. Kim; citation_author=D.K. Kim; citation_author=H. Jeon An; citation_author=J. Bae Son; citation_author=A. Hedén Gynnå; citation_author=N. Ki Lee; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=5131; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid; citation_author=D. Yang; citation_author=J. Männik; citation_author=S.T. Retterer; citation_author=J. Männik; citation_journal_title=Mol. Microbiol.; citation_volume=113; citation_firstpage=1022; citation_lastpage=1037; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A new assay method for DNase by fluorescence polarization and fluorescence intensity using DNA-ethidium bromide complex as a sensitive substrate; citation_author=K. Yonemura; citation_author=H. Maeda; citation_journal_title=J.&nbsp;Biochem.; citation_volume=92; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=1982; "><meta name="citation_reference" content="citation_title=Compaction of DNA Induced by Like-Charge Protein: Opposite Salt-Effect against the Polymer-Salt-Induced Condensation with Neutral Polymer; citation_author=K. Yoshikawa; citation_author=S. Hirota; citation_author=N. Makita; citation_author=Y. Yoshikawa; citation_journal_title=J.&nbsp;Phys. Chem. Lett.; citation_volume=1; citation_firstpage=1763; citation_lastpage=1766; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Gctf: Real-time CTF determination and correction; citation_author=K. Zhang; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=193; citation_firstpage=1; citation_lastpage=12; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Macromolecular crowding induced elongation and compaction of single DNA molecules confined in a nanochannel; citation_author=C. Zhang; citation_author=P.G. Shao; citation_author=J.A. van Kan; citation_author=J.R. van der Maarel; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=106; citation_firstpage=16651; citation_lastpage=16656; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Cryoelectron tomography reveals the sequential assembly of bacterial flagella in Borrelia burgdorferi; citation_author=X. Zhao; citation_author=K. Zhang; citation_author=T. Boquoi; citation_author=B. Hu; citation_author=M.A. Motaleb; citation_author=K.A. Miller; citation_author=M.E. James; citation_author=N.W. Charon; citation_author=M.D. Manson; citation_author=S.J. Norris; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=14390; citation_lastpage=14395; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy; citation_author=S.Q. Zheng; citation_author=E. Palovcak; citation_author=J.P. Armache; citation_author=K.A. Verba; citation_author=Y. Cheng; citation_author=D.A. Agard; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=331; citation_lastpage=332; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Three-dimensional reconstruction with contrast transfer function correction from energy-filtered cryoelectron micrographs: procedure and application to the 70S Escherichia coli ribosome; citation_author=J. Zhu; citation_author=P.A. Penczek; citation_author=R. Schröder; citation_author=J. Frank; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=118; citation_firstpage=197; citation_lastpage=219; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=The Dimensions of Chain Molecules Containing Branches and Rings; citation_author=B.H. Zimm; citation_author=W.H. Stockmayer; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=17; citation_firstpage=1301; citation_lastpage=1314; citation_publication_date=1949; "><meta name="citation_reference" content="citation_title=Macromolecular crowding effects on macromolecular interactions: some implications for genome structure and function; citation_author=S.B. Zimmerman; citation_journal_title=Biochim. Biophys. Acta; citation_volume=1216; citation_firstpage=175; citation_lastpage=185; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Toroidal nucleoids in Escherichia coli exposed to chloramphenicol; citation_author=S.B. Zimmerman; citation_journal_title=J.&nbsp;Struct. Biol.; citation_volume=138; citation_firstpage=199; citation_lastpage=206; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Macromolecular crowding: biochemical, biophysical, and physiological consequences; citation_author=S.B. Zimmerman; citation_author=A.P. Minton; citation_journal_title=Annu. Rev. Biophys. Biomol. Struct.; citation_volume=22; citation_firstpage=27; citation_lastpage=65; citation_publication_date=1993; "><meta name="citation_abstract" content="<h2>Summary</h2><p>All cells fold their genomes, including bacterial cells, where the chromosome is compacted into a domain-organized meshwork called the nucleoid. How compaction and domain organization arise is not fully understood. Here, we describe a method to estimate the average mesh size of the nucleoid in <i>Escherichia coli</i>. Using nucleoid mesh size and DNA concentration estimates, we find that the cytoplasm behaves as a poor solvent for the chromosome when the cell is considered as a simple semidilute polymer solution. Monte Carlo simulations suggest that a poor solvent leads to chromosome compaction and DNA density heterogeneity (i.e., domain formation) at physiological DNA concentration. Fluorescence microscopy reveals that the heterogeneous DNA density negatively correlates with ribosome density within the nucleoid, consistent with cryoelectron tomography data. Drug experiments, together with past observations, suggest the hypothesis that RNAs contribute to the poor solvent effects, connecting chromosome compaction and domain formation to transcription and intracellular organization.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=10474&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9&amp;be=3843&amp;fe=10153&amp;dc=5582&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749590612416,%22n%22:0,%22f%22:3179,%22dn%22:3180,%22dne%22:3180,%22c%22:3180,%22s%22:3180,%22ce%22:3180,%22rq%22:3180,%22rp%22:3816,%22rpe%22:4792,%22dl%22:3822,%22di%22:5575,%22ds%22:5581,%22de%22:5961,%22dc%22:10152,%22l%22:10152,%22le%22:10433%7D,%22navigation%22:%7B%7D%7D&amp;fp=4356&amp;fcp=4356&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style>stream {
  position: relative;
  display: block;
}
</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-msqrt {
  display: inline-block;
  text-align: left;
}

mjx-root {
  display: inline-block;
  white-space: nowrap;
}

mjx-surd {
  display: inline-block;
  vertical-align: top;
}

mjx-sqrt {
  display: inline-block;
  padding-top: .075em;
}

mjx-sqrt > mjx-box {
  border-top: .075em solid;
  padding-left: .03em;
  margin-left: -.03em;
}

mjx-sqrt.mjx-tall > mjx-box {
  padding-left: .3em;
  margin-left: -.3em;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-munderover {
  display: inline-block;
  text-align: left;
}

mjx-munderover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-munderover:not([limits="false"]) > * {
  display: block;
}

mjx-mroot {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D709 {
  padding: 0.697em 0.446em 0.205em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c221A {
  padding: 0.04em 0.853em 0.96em 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c36 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D44F {
  padding: 0.694em 0.429em 0.011em 0;
}

mjx-c.mjx-c1D708 {
  padding: 0.442em 0.524em 0 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c28.MM-S4 {
  padding: 1.146em 0.597em 0.646em 0;
}

mjx-c.mjx-c38 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c1D464 {
  padding: 0.442em 0.716em 0.011em 0;
}

mjx-c.mjx-c1D70B {
  padding: 0.431em 0.57em 0.011em 0;
}

mjx-c.mjx-c1D441 {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c29.MM-S4 {
  padding: 1.146em 0.597em 0.646em 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D6FC {
  padding: 0.442em 0.64em 0.011em 0;
}

mjx-c.mjx-c1D445 {
  padding: 0.683em 0.759em 0.022em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c1D70E {
  padding: 0.431em 0.571em 0.011em 0;
}

mjx-c.mjx-c1D70C {
  padding: 0.442em 0.517em 0.216em 0;
}

mjx-c.mjx-c1D43F {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c1D703 {
  padding: 0.705em 0.469em 0.011em 0;
}

mjx-c.mjx-c2217 {
  padding: 0.462em 0.5em 0 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c28.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c29.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c5B.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c28.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c29.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c5D.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c1D446 {
  padding: 0.705em 0.645em 0.022em 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D70F {
  padding: 0.431em 0.511em 0.012em 0;
}

mjx-c.mjx-c3C {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c2032.MM-V {
  padding: 0.549em 0.407em 0 0;
}

mjx-c.mjx-c1D449 {
  padding: 0.683em 0.769em 0.022em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D43C {
  padding: 0.683em 0.497em 0 0;
}

mjx-c.mjx-c221A.MM-SO {
  padding: 0.85em 1.02em 0.35em 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c1D458 {
  padding: 0.694em 0.521em 0.011em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c1D702 {
  padding: 0.442em 0.497em 0.216em 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D457 {
  padding: 0.661em 0.412em 0.205em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00689-9"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-542286" aria-expanded="false" aria-controls="login-links-542286" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-542286" id="login-links--542286" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-542286" data-opens="#links-dialog-542286" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-542286" aria-modal="true"><button data-closes="#links-dialog-542286" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900689-9" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.05.037">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00689-9" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0015-8"><span>Volume 184</span>, <span>Issue 14</span></a></span><span class="meta-panel__pages"><span>p3626-3642.e14</span></span><span class="meta-panel__onlineDate">July 08, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=14&amp;pii=S0092-8674%2820%29X0015-8" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Interconnecting solvent quality, transcription, and chromosome folding in <i>Escherichia coli</i></h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Yingjie</span> <span property="familyName">Xiang</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yingjie</span> <span property="familyName">Xiang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Mechanical Engineering and Materials Science, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Xiang/Yingjie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Ivan V.</span> <span property="familyName">Surovtsev</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ivan V.</span> <span property="familyName">Surovtsev</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Surovtsev/Ivan+V" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Yunjie</span> <span property="familyName">Chang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yunjie</span> <span property="familyName">Chang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chang/Yunjie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Sander K.</span> <span property="familyName">Govers</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sander K.</span> <span property="familyName">Govers</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology and Institute of Chemistry, Engineering and Medicine for Human Health, Stanford University, Palo Alto, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Govers/Sander+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Bradley R.</span> <span property="familyName">Parry</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bradley R.</span> <span property="familyName">Parry</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Parry/Bradley+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Jun</span> <span property="familyName">Liu</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jun</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Jun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Christine</span> <span property="familyName">Jacobs-Wagner</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christine</span> <span property="familyName">Jacobs-Wagner</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:jacobs-wagner@stanford.edu" property="email" title="Send email to Christine Jacobs-Wagner">jacobs-wagner@stanford.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology and Institute of Chemistry, Engineering and Medicine for Human Health, Stanford University, Palo Alto, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jacobs-Wagner/Christine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a role="doc-noteref">7</a></sup> <a id="eadd0010" href="mailto:jacobs-wagner@stanford.edu" property="email" title="Send email to Christine Jacobs-Wagner">jacobs-wagner@stanford.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Mechanical Engineering and Materials Science, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Microbial Sciences Institute, Yale University, West Haven, CT 06516, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Howard Hughes Medical Institute, Yale University, New Haven, CT 06520, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Microbial Pathogenesis, Yale School of Medicine, New Haven, CT 06510, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Department of Biology and Institute of Chemistry, Engineering and Medicine for Human Health, Stanford University, Palo Alto, CA 94305, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received June 5, 2020</span>; <span>Revised December 9, 2020</span>; <span>Accepted May 25, 2021</span>; <span>Published online June 28, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.05.037" property="sameAs" target="_blank">10.1016/j.cell.2021.05.037 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421006899" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.05.037" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0015-8"><img src="/cms/asset/adaca0ef-bc3c-428e-afb9-ebccecf9ad40/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 14"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900689-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900689-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.05.037">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900689-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.05.037&amp;pii=S0092-8674%2821%2900689-9" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900689-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.05.037">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.05.037" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421006899&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421006899&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=14&amp;pii=S0092-8674%2820%29X0015-8" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00625-5" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00628-0" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">The apparent average mesh size of the <i>Escherichia coli</i> chromosome is around 50&nbsp;nm</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">The cytoplasm effectively acts as a poor solvent for the chromosome</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Ribosome and DNA densities are negatively correlated within the nucleoid region</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">RNA/DNA segregation is associated with chromosome compaction</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">All cells fold their genomes, including bacterial cells, where the chromosome is compacted into a domain-organized meshwork called the nucleoid. How compaction and domain organization arise is not fully understood. Here, we describe a method to estimate the average mesh size of the nucleoid in <i>Escherichia coli</i>. Using nucleoid mesh size and DNA concentration estimates, we find that the cytoplasm behaves as a poor solvent for the chromosome when the cell is considered as a simple semidilute polymer solution. Monte Carlo simulations suggest that a poor solvent leads to chromosome compaction and DNA density heterogeneity (i.e., domain formation) at physiological DNA concentration. Fluorescence microscopy reveals that the heterogeneous DNA density negatively correlates with ribosome density within the nucleoid, consistent with cryoelectron tomography data. Drug experiments, together with past observations, suggest the hypothesis that RNAs contribute to the poor solvent effects, connecting chromosome compaction and domain formation to transcription and intracellular organization.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/dd7d3d88-ee57-4bba-ae62-562f37037552/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.05.037/asset/c97a0d1a-f7ee-4287-9a9b-4e2a1eb07d3d/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/dd7d3d88-ee57-4bba-ae62-562f37037552/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;chromosome+folding&quot;" alt="Search on this keyword" rel="nofollow">chromosome folding</a></li><li><a href="/action/doSearch?AllField=&quot;nucleoid&quot;" alt="Search on this keyword" rel="nofollow">nucleoid</a></li><li><a href="/action/doSearch?AllField=&quot;solvent+quality&quot;" alt="Search on this keyword" rel="nofollow">solvent quality</a></li><li><a href="/action/doSearch?AllField=&quot;chromosome+compaction&quot;" alt="Search on this keyword" rel="nofollow">chromosome compaction</a></li><li><a href="/action/doSearch?AllField=&quot;domain+organization&quot;" alt="Search on this keyword" rel="nofollow">domain organization</a></li><li><a href="/action/doSearch?AllField=&quot;DNA+mesh+size&quot;" alt="Search on this keyword" rel="nofollow">DNA mesh size</a></li><li><a href="/action/doSearch?AllField=&quot;ribosome+localization&quot;" alt="Search on this keyword" rel="nofollow">ribosome localization</a></li><li><a href="/action/doSearch?AllField=&quot;RNA&quot;" alt="Search on this keyword" rel="nofollow">RNA</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">All cells, regardless of their origin, must package and organize their genome in a small volume. Proper folding of the chromosomes is critical, as it affects many cellular processes, including gene expression, DNA repair, and chromosome segregation. Unlike eukaryotic cells, bacterial cells lack a nuclear membrane and do not package their chromosomal DNA into repeating structural units akin to nucleosomes. However, they still fold and concentrate their chromosomal material into a dynamic and organized DNA meshwork known as the nucleoid. In many bacterial species, including <i>Escherichia coli</i>, the nucleoid is found within a portion of the cytoplasmic space (<a id="crosref0125" href="#fig1">Figure&nbsp;1</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Hobot et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Hobot, J.A. ∙ Villiger, W. ∙ Escaig, J. ...</div><div><strong>Shape and fine structure of nucleoids observed on sections of ultrarapidly frozen and cryosubstituted bacteria</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:960-971</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.3.960-971.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3922958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.3.960-971.1985&amp;pmid=3922958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Kellenberger et&nbsp;al., 1958</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Kellenberger, E. ∙ Ryter, A. ∙ Sechaud, J.</div><div><strong>Electron microscope study of DNA-containing plasms. II. Vegetative and mature phage DNA as compared with normal bacterial nucleoids in different physiological states</strong></div><div><em>J.&nbsp;Biophys. Biochem. Cytol.</em> 1958; <strong>4</strong>:671-678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.4.6.671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13610928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.4.6.671&amp;pmid=13610928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Mason and Powelson, 1956</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Mason, D.J. ∙ Powelson, D.M.</div><div><strong>Nuclear division as observed in live bacteria by a new technique</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1956; <strong>71</strong>:474-479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.71.4.474-479.1956" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13319263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.71.4.474-479.1956&amp;pmid=13319263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Piekarski, 1937</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Piekarski, G.</div><div><strong>Zytologische Untersuchungen an Paratyphusund Coli Bakterien</strong></div><div><em>Arch. Mikrobiol.</em> 1937; <strong>8</strong>:428-429</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00407212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0000625722" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00407212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-1" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-1" aria-controls="sref117-1">Robinow and Kellenberger, 1994</a><div class="dropBlock__holder" data-db-target-of="sref117-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Robinow, C. ∙ Kellenberger, E.</div><div><strong>The bacterial nucleoid revisited</strong></div><div><em>Microbiol. Rev.</em> 1994; <strong>58</strong>:211-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/mr.58.2.211-232.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7521510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fmr.58.2.211-232.1994&amp;pmid=7521510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This indicates that the compaction of the bacterial chromosome is not simply dictated by the physical confinement created by the cell envelope. Furthermore, the chromosome displays higher-order organization across multiple length scales (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Dame et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Dame, R.T. ∙ Rashid, F.M. ∙ Grainger, D.C.</div><div><strong>Chromosome organization in bacteria: mechanistic insights into genome structure and function</strong></div><div><em>Nat. Rev. Genet.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31767998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31767998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref146" href="#" role="doc-biblioref" data-xml-rid="bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146" aria-controls="sref146">Verma et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref146"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Verma, S.C. ∙ Qian, Z. ∙ Adhya, S.L.</div><div><strong>Architecture of the Escherichia coli nucleoid</strong></div><div><em>PLoS Genet.</em> 2019; <strong>15</strong>:e1008456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1008456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85076457375" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31830036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1008456&amp;pmid=31830036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For instance, high-resolution chromosome conformation capture (Hi-C) studies have demonstrated that the bacterial chromosome is organized into various chromosomal interaction domains (CIDs) within which nearby gene loci interact more frequently with each other than with those outside the domain (<span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Le et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Le, T.B. ∙ Imakaev, M.V. ∙ Mirny, L.A. ...</div><div><strong>High-resolution mapping of the spatial organization of a bacterial chromosome</strong></div><div><em>Science.</em> 2013; <strong>342</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1242059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84887322004" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24158908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1242059&amp;pmid=24158908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Lioy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Lioy, V.S. ∙ Cournac, A. ∙ Marbouty, M. ...</div><div><strong>Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:771-783.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85041112985" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.027&amp;pmid=29358050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Marbouty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Marbouty, M. ∙ Cournac, A. ∙ Flot, J.F. ...</div><div><strong>Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e03318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.03318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.03318&amp;pmid=25517076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">2015</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref142-1" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-1" aria-controls="sref142-1">Val et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref142-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Val, M.E. ∙ Marbouty, M. ∙ de Lemos Martins, F. ...</div><div><strong>A checkpoint control orchestrates the replication of the two chromosomes of Vibrio cholerae</strong></div><div><em>Sci. Adv.</em> 2016; <strong>2</strong>:e1501914</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1501914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84991009833" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27152358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1501914&amp;pmid=27152358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref151-1" href="#" role="doc-biblioref" data-xml-rid="bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151-1" aria-controls="sref151-1">Wang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref151-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Wang, X. ∙ Le, T.B. ∙ Lajoie, B.R. ...</div><div><strong>Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis</strong></div><div><em>Genes Dev.</em> 2015; <strong>29</strong>:1661-1675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.265876.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26253537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.265876.115&amp;pmid=26253537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref153-1" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-1" aria-controls="sref153-1">2017</a><div class="dropBlock__holder" data-db-target-of="sref153-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Wang, X. ∙ Brandão, H.B. ∙ Le, T.B. ...</div><div><strong>Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus</strong></div><div><em>Science.</em> 2017; <strong>355</strong>:524-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai8982" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85011362856" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28154080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai8982&amp;pmid=28154080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/141b225b-1bcb-4dae-8046-5e48fe926b12/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.05.037/asset/ff5bfad5-7ca5-4382-98bd-8273a2f2df32/main.assets/gr1.jpg" height="385" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/141b225b-1bcb-4dae-8046-5e48fe926b12/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Estimation of the DNA concentration within the nucleoid region</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic showing the <i>E.&nbsp;coli</i> chromosome folded into a meshwork structure (nucleoid) with mesh size <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="xi"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. The cytoplasm is viewed as a semidilute polymer solution in which the DNA is the polymer and the rest is the solvent. The nucleoid region (dotted line) defines the volume used to determine the DNA concentration <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>.</div><div id="fspara0020" role="paragraph">(B) Fluorescence images of representative DAPI-stained <i>E.&nbsp;coli</i> CJW6324 cells (n&nbsp;= 19,510) in B, C, or D period. Cell (green) and nucleoid (yellow) contours are shown. See <a id="crosref2750" href="#figs1">Figure&nbsp;S1</a>A for the corresponding phase-contrast images.</div><div id="fspara0025" role="paragraph">(C) Plot showing the relative area of the SeqA-mCherry signal as a function of cell area. The contour lines (from light to dark color) represent the 25%, 50%, and 75% probability envelopes of the data.</div><div id="fspara0030" role="paragraph">(D) Probability density function of the estimated nucleoid volume in B-period cells.</div><div id="fspara0035" role="paragraph">(E) Probability density function of the average DNA concentration calculated based on the total mass of a single <i>E.&nbsp;coli</i> chromosome and the nucleoid volume measurements shown in (D).</div></div></div></div></figcaption></figure></div><div id="p0035" role="paragraph">Apart from its compacted and organized structure, another important, though less appreciated, aspect of the nucleoid is its mesh size (<a id="crosref0205" href="#fig1">Figure&nbsp;1</a>A), as it impacts the mobility of cytoplasmic components and thereby their intracellular organization. For example, the enrichment of ribosomes at the cell poles and between segregated nucleoids in <i>E.&nbsp;coli</i> and other bacteria is often attributed to nucleoid exclusion due to their large sizes (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Azam et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Azam, T.A. ∙ Hiraga, S. ∙ Ishihama, A.</div><div><strong>Two types of localization of the DNA-binding proteins within the Escherichia coli nucleoid</strong></div><div><em>Genes Cells.</em> 2000; <strong>5</strong>:613-626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2443.2000.00350.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0033869982" target="_blank" aria-label="Open Scopus in new tab">Scopus (169)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10947847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2443.2000.00350.x&amp;pmid=10947847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Lewis et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Lewis, P.J. ∙ Thaker, S.D. ∙ Errington, J.</div><div><strong>Compartmentalization of transcription and translation in Bacillus subtilis</strong></div><div><em>EMBO J.</em> 2000; <strong>19</strong>:710-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/19.4.710" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10675340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2F19.4.710&amp;pmid=10675340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-2" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-2" aria-controls="sref117-2">Robinow and Kellenberger, 1994</a><div class="dropBlock__holder" data-db-target-of="sref117-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Robinow, C. ∙ Kellenberger, E.</div><div><strong>The bacterial nucleoid revisited</strong></div><div><em>Microbiol. Rev.</em> 1994; <strong>58</strong>:211-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/mr.58.2.211-232.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7521510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fmr.58.2.211-232.1994&amp;pmid=7521510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120-1" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-1" aria-controls="sref120-1">Sanamrad et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref120-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Sanamrad, A. ∙ Persson, F. ∙ Lundius, E.G. ...</div><div><strong>Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11413-11418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1411558111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84905669647" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1411558111&amp;pmid=25056965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It is, however, difficult to predict which cytoplasmic components are impacted by the nucleoid, as its average mesh size is unknown.</div><div id="p0040" role="paragraph">What are the biophysical principles that help explain the compacted size, domain organization, and mesh size of the bacterial nucleoid? Multiple factors are thought to be at play (<span class="dropBlock reference-citations"><a id="body-ref-sref136" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136" aria-controls="sref136">Stavans and Oppenheim, 2006</a><div class="dropBlock__holder" data-db-target-of="sref136"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Stavans, J. ∙ Oppenheim, A.</div><div><strong>DNA-protein interactions and bacterial chromosome architecture</strong></div><div><em>Phys. Biol.</em> 2006; <strong>3</strong>:R1-R10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1088/1478-3975/3/4/R01" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34247517469" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17200598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1088%2F1478-3975%2F3%2F4%2FR01&amp;pmid=17200598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref139" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139" aria-controls="sref139">Surovtsev and Jacobs-Wagner, 2018</a><div class="dropBlock__holder" data-db-target-of="sref139"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Surovtsev, I.V. ∙ Jacobs-Wagner, C.</div><div><strong>Subcellular Organization: A Critical Feature of Bacterial Cell Replication</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:1271-1293</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2018.01.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2018.01.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85043302867" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29522747/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.01.014&amp;pmid=29522747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). They include biochemical processes that modulate DNA supercoiling (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Dorman, 2019</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Dorman, C.J.</div><div><strong>DNA supercoiling and transcription in bacteria: a two-way street</strong></div><div><em>BMC Mol. Cell Biol.</em> 2019; <strong>20</strong>:26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12860-019-0211-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85070094224" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31319794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12860-019-0211-6&amp;pmid=31319794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Ma and Wang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Ma, J. ∙ Wang, M.D.</div><div><strong>DNA supercoiling during transcription</strong></div><div><em>Biophys. Rev.</em> 2016; <strong>8</strong>:75-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12551-016-0215-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85003554279" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28275417/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12551-016-0215-9&amp;pmid=28275417" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Bacteria also carry a number of DNA-binding proteins, known as nucleoid-associated proteins (NAPs) and structural maintenance of chromosome (SMC) proteins, that alter the structure of the DNA locally or over long molecular distances (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Dame et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Dame, R.T. ∙ Rashid, F.M. ∙ Grainger, D.C.</div><div><strong>Chromosome organization in bacteria: mechanistic insights into genome structure and function</strong></div><div><em>Nat. Rev. Genet.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31767998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31767998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Dillon and Dorman, 2010</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Dillon, S.C. ∙ Dorman, C.J.</div><div><strong>Bacterial nucleoid-associated proteins, nucleoid structure and gene expression</strong></div><div><em>Nat. Rev. Microbiol.</em> 2010; <strong>8</strong>:185-195</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro2261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-76949089832" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20140026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro2261&amp;pmid=20140026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While the direct contribution of these proteins to DNA compaction is not entirely clear (<span class="dropBlock reference-citations"><a id="body-ref-sref135" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135" aria-controls="sref135">Spurio et&nbsp;al., 1992</a><div class="dropBlock__holder" data-db-target-of="sref135"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Spurio, R. ∙ Dürrenberger, M. ∙ Falconi, M. ...</div><div><strong>Lethal overproduction of the Escherichia coli nucleoid protein H-NS: ultramicroscopic and molecular autopsy</strong></div><div><em>Mol. Gen. Genet.</em> 1992; <strong>231</strong>:201-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00279792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1310520/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00279792&amp;pmid=1310520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref162-1" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref162-1" aria-controls="sref162-1">Wu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref162-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">161.</div><div>Wu, F. ∙ Swain, P. ∙ Kuijpers, L. ...</div><div><strong>Cell Boundary Confinement Sets the Size and Position of the E.&nbsp;coli Chromosome</strong></div><div><em>Curr. Biol.</em> 2019; <strong>29</strong>:2131-2144.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cub.2019.05.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cub.2019.05.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065894263" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31155353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2019.05.015&amp;pmid=31155353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), NAPs and SMC protein complexes are known to play important roles in the organization and regulation of chromosome architecture at the level of individual genes and CIDs (<span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Dame et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Dame, R.T. ∙ Rashid, F.M. ∙ Grainger, D.C.</div><div><strong>Chromosome organization in bacteria: mechanistic insights into genome structure and function</strong></div><div><em>Nat. Rev. Genet.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31767998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31767998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Other factors have been proposed to affect nucleoid size in an opposing fashion. Transertion, which is the coupling between transcription, translation, and insertion of nascent proteins into the cytoplasmic membrane, is thought to provide an expanding force on the nucleoid (<span class="dropBlock reference-citations"><a id="body-ref-sref7-1" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-1" aria-controls="sref7-1">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref159" href="#" role="doc-biblioref" data-xml-rid="bib159" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref159" aria-controls="sref159">Woldringh, 2002</a><div class="dropBlock__holder" data-db-target-of="sref159"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">158.</div><div>Woldringh, C.L.</div><div><strong>The role of co-transcriptional translation and protein translocation (transertion) in bacterial chromosome segregation</strong></div><div><em>Mol. Microbiol.</em> 2002; <strong>45</strong>:17-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2958.2002.02993.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036050398" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12100545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2958.2002.02993.x&amp;pmid=12100545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref160" href="#" role="doc-biblioref" data-xml-rid="bib160" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160" aria-controls="sref160">Woldringh et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref160"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">159.</div><div>Woldringh, C.L. ∙ Jensen, P.R. ∙ Westerhoff, H.V.</div><div><strong>Structure and partitioning of bacterial DNA: determined by a balance of compaction and expansion forces?</strong></div><div><em>FEMS Microbiol. Lett.</em> 1995; <strong>131</strong>:235-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1574-6968.1995.tb07782.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7557335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1574-6968.1995.tb07782.x&amp;pmid=7557335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, macromolecular crowding is frequently proposed to generate a compacting force through steric (size-dependent) effects (<span class="dropBlock reference-citations"><a id="body-ref-sref7-2" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-2" aria-controls="sref7-2">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Cunha et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Cunha, S. ∙ Woldringh, C.L. ∙ Odijk, T.</div><div><strong>Polymer-mediated compaction and internal dynamics of isolated Escherichia coli nucleoids</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2001; <strong>136</strong>:53-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.2001.4420" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0035782861" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11858707/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.2001.4420&amp;pmid=11858707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">de Vries, 2010</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>de Vries, R.</div><div><strong>DNA condensation in bacteria: Interplay between macromolecular crowding and nucleoid proteins</strong></div><div><em>Biochimie.</em> 2010; <strong>92</strong>:1715-1721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.biochi.2010.06.024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-78649882848" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20615449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biochi.2010.06.024&amp;pmid=20615449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Jeon et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Jeon, C. ∙ Jung, Y. ∙ Ha, B.-Y.</div><div><strong>Effects of molecular crowding and confinement on the spatial organization of a biopolymer</strong></div><div><em>Soft Matter.</em> 2016; <strong>12</strong>:9436-9450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C6SM01184E" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27834427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC6SM01184E&amp;pmid=27834427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Jun, 2015</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Jun, S.</div><div><strong>Chromosome, cell cycle, and entropy</strong></div><div><em>Biophys. J.</em> 2015; <strong>108</strong>:785-786</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.12.032&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.12.032&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84923241784" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25692581/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2014.12.032&amp;pmid=25692581" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Kim et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Kim, J. ∙ Jeon, C. ∙ Jeong, H. ...</div><div><strong>A polymer in a crowded and confined space: effects of crowder size and poly-dispersity</strong></div><div><em>Soft Matter.</em> 2015; <strong>11</strong>:1877-1888</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C4SM02198C" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535704/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC4SM02198C&amp;pmid=25535704" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Mondal et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Mondal, J. ∙ Bratton, B.P. ∙ Li, Y. ...</div><div><strong>Entropy-based mechanism of ribosome-nucleoid segregation in E.&nbsp;coli cells</strong></div><div><em>Biophys. J.</em> 2011; <strong>100</strong>:2605-2613</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79960325276" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21641305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2011.04.030&amp;pmid=21641305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Odijk, 1998</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Odijk, T.</div><div><strong>Osmotic compaction of supercoiled DNA into a bacterial nucleoid</strong></div><div><em>Biophys. Chem.</em> 1998; <strong>73</strong>:23-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0301-4622(98)00115-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0032514107" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9697298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0301-4622%2898%2900115-X&amp;pmid=9697298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Pelletier et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Pelletier, J. ∙ Halvorsen, K. ∙ Ha, B.Y. ...</div><div><strong>Physical manipulation of the Escherichia coli chromosome reveals its soft nature</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:E2649-E2656</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1208689109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84867042373" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22984156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1208689109&amp;pmid=22984156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref127" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127" aria-controls="sref127">Shendruk et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref127"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Shendruk, T.N. ∙ Bertrand, M. ∙ de Haan, H.W. ...</div><div><strong>Simulating the entropic collapse of coarse-grained chromosomes</strong></div><div><em>Biophys. J.</em> 2015; <strong>108</strong>:810-820</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.11.3487&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.11.3487&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84923248039" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25692586/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2014.11.3487&amp;pmid=25692586" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref154" href="#" role="doc-biblioref" data-xml-rid="bib154" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref154" aria-controls="sref154">Wegner et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref154"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Wegner, A.S. ∙ Wintraecken, K. ∙ Spurio, R. ...</div><div><strong>Compaction of isolated Escherichia coli nucleoids: Polymer and H-NS protein synergetics</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>194</strong>:129-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2016.02.009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958672778" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26868106/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2016.02.009&amp;pmid=26868106" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref162-2" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref162-2" aria-controls="sref162-2">Wu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref162-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">161.</div><div>Wu, F. ∙ Swain, P. ∙ Kuijpers, L. ...</div><div><strong>Cell Boundary Confinement Sets the Size and Position of the E.&nbsp;coli Chromosome</strong></div><div><em>Curr. Biol.</em> 2019; <strong>29</strong>:2131-2144.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cub.2019.05.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cub.2019.05.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065894263" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31155353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2019.05.015&amp;pmid=31155353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref164-1" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164-1" aria-controls="sref164-1">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref164-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Yang, D. ∙ Männik, J. ∙ Retterer, S.T. ...</div><div><strong>The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid</strong></div><div><em>Mol. Microbiol.</em> 2020; <strong>113</strong>:1022-1037</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85078903920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31961016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14467&amp;pmid=31961016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref166" href="#" role="doc-biblioref" data-xml-rid="bib166" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref166" aria-controls="sref166">Yoshikawa et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref166"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Yoshikawa, K. ∙ Hirota, S. ∙ Makita, N. ...</div><div><strong>Compaction of DNA Induced by Like-Charge Protein: Opposite Salt-Effect against the Polymer-Salt-Induced Condensation with Neutral Polymer</strong></div><div><em>J.&nbsp;Phys. Chem. Lett.</em> 2010; <strong>1</strong>:1763-1766</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jz100569e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-77953791541" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjz100569e" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref168" href="#" role="doc-biblioref" data-xml-rid="bib168" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref168" aria-controls="sref168">Zhang et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref168"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Zhang, C. ∙ Shao, P.G. ∙ van Kan, J.A. ...</div><div><strong>Macromolecular crowding induced elongation and compaction of single DNA molecules confined in a nanochannel</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:16651-16656</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0904741106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-70349728576" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19805352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0904741106&amp;pmid=19805352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref173" href="#" role="doc-biblioref" data-xml-rid="bib173" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref173" aria-controls="sref173">Zimmerman, 1993</a><div class="dropBlock__holder" data-db-target-of="sref173"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">172.</div><div>Zimmerman, S.B.</div><div><strong>Macromolecular crowding effects on macromolecular interactions: some implications for genome structure and function</strong></div><div><em>Biochim. Biophys. Acta.</em> 1993; <strong>1216</strong>:175-185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0167-4781(93)90142-Z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0027496418" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8241257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0167-4781%2893%2990142-Z&amp;pmid=8241257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref175" href="#" role="doc-biblioref" data-xml-rid="bib175" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref175" aria-controls="sref175">Zimmerman and Minton, 1993</a><div class="dropBlock__holder" data-db-target-of="sref175"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">174.</div><div>Zimmerman, S.B. ∙ Minton, A.P.</div><div><strong>Macromolecular crowding: biochemical, biophysical, and physiological consequences</strong></div><div><em>Annu. Rev. Biophys. Biomol. Struct.</em> 1993; <strong>22</strong>:27-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.bb.22.060193.000331" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7688609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.bb.22.060193.000331&amp;pmid=7688609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The idea is that entropic forces drive depletion of crowders (e.g., proteins and ribosomes) from the nucleoid to reduce the excluded volume. The imbalance in crowder concentration between the nucleoid region and the rest of the cytoplasm results in an effective osmotic pressure that pushes DNA segments closer to each other. The magnitude of the entropic (depletion) forces, which remains to be determined inside cells, will depend on the abundance, size, and polydispersity of the crowders (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Kim et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Kim, J. ∙ Jeon, C. ∙ Jeong, H. ...</div><div><strong>A polymer in a crowded and confined space: effects of crowder size and poly-dispersity</strong></div><div><em>Soft Matter.</em> 2015; <strong>11</strong>:1877-1888</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C4SM02198C" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535704/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC4SM02198C&amp;pmid=25535704" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref164-2" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164-2" aria-controls="sref164-2">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref164-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Yang, D. ∙ Männik, J. ∙ Retterer, S.T. ...</div><div><strong>The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid</strong></div><div><em>Mol. Microbiol.</em> 2020; <strong>113</strong>:1022-1037</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85078903920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31961016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14467&amp;pmid=31961016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While steric interactions are primarily emphasized in current models of chromosome compaction, DNA and other cytoplasmic components are not chemically inert, and non-steric interactions may also contribute to chromosome folding <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Joyeux, 2015</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Joyeux, M.</div><div><strong>Compaction of bacterial genomic DNA: clarifying the concepts</strong></div><div><em>J.&nbsp;Phys. Condens. Matter.</em> 2015; <strong>27</strong>:383001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1088/0953-8984/27/38/383001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84941695861" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26345139/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1088%2F0953-8984%2F27%2F38%2F383001&amp;pmid=26345139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0045" role="paragraph">In this study, we adopt a simple polymer physics perspective of the cell by considering the cytoplasm as a polymer solution in which the polymer is the DNA and the solvent is everything else in the cytoplasm (e.g., water, metabolites, proteins, NAPs, ribosomes, and RNAs) (<a id="crosref0400" href="#fig1">Figure&nbsp;1</a>A). That is, rather than describing how each potential factor structures the chromosome mechanistically, we account for this at the coarse-grained level by describing all effects of known and unknown factors that shape the chromosome as an effective solvent. It is well established that the three-dimensional (3D) conformation of a polymer is directly affected by the interaction between the polymer and its solvent (<span class="dropBlock reference-citations"><a id="body-ref-sref39-1" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-1" aria-controls="sref39-1">de Gennes, 1979</a><div class="dropBlock__holder" data-db-target-of="sref39-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>de Gennes, P.G.</div><div><strong>Scaling concepts in polymer physics</strong></div><div>Cornell University Press, 1979</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.G.de+GennesScaling+concepts+in+polymer+physics1979Cornell+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-1" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-1" aria-controls="sref119-1">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Based on the polymer-solvent interaction, the quality of a solvent is broadly classified into three types: good, ideal, and poor. In a good solvent, interactions between the polymer and the solvent are favored over interactions between polymer segments. Conversely, in a poor solvent, interactions between polymer segments are favored over their interactions with the solvent. When the repulsive and attractive interactions are balanced out (zero net interaction), the solvent is said to be ideal. How the bacterial cytoplasm (or eukaryotic nucleoplasm) behaves as a solvent for the chromosome remains an open question.</div><div id="p0050" role="paragraph">Here, we develop a live-cell experimental approach to estimate the apparent average mesh size of the nucleoid and infer the effective solvent quality of the cytoplasm in <i>E.&nbsp;coli</i>. We then describe experiments and simulations that connect this macroscopic description of the cytoplasm to chromosome folding, transcription, and intracellular organization.</div></section><section id="sec-2"><h2>Results</h2><div id="p0055" role="paragraph">Here, we considered the cell as a simple semidilute (i.e., concentrated) polymer solution in which the chromosome is assumed to be a random coil polymer and its solvent is everything in the cytoplasm except the DNA (<a id="crosref0415" href="#fig1">Figure&nbsp;1</a>A). In polymer physics, a polymer solution is said to be semidilute when the polymer concentration is high enough that the polymer segments overlap to form a meshwork. The average mesh size (or correlation length) <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="xi"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> of such a meshwork depends not only on properties of the polymer (concentration, rigidity, molecular weight and monomer size) but also on the quality of the solvent (<span class="dropBlock reference-citations"><a id="body-ref-sref39-2" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-2" aria-controls="sref39-2">de Gennes, 1979</a><div class="dropBlock__holder" data-db-target-of="sref39-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>de Gennes, P.G.</div><div><strong>Scaling concepts in polymer physics</strong></div><div>Cornell University Press, 1979</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.G.de+GennesScaling+concepts+in+polymer+physics1979Cornell+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-2" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-2" aria-controls="sref119-2">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as shown in <a id="crosref0430" href="#fd1">Equation 1</a> (see <a id="crosref0435" href="#sec-4">STAR Methods</a> for derivation).<div id="fd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi equals StartFraction StartRoot 3 EndRoot Over 6 EndFraction b Superscript StartFraction nu minus 1 Over 3 nu minus 1 EndFraction Baseline left parenthesis StartFraction 18 StartRoot 3 EndRoot upper M Subscript w Baseline Over pi c upper N Subscript upper A Baseline l Subscript b p Baseline EndFraction right parenthesis Superscript StartFraction nu Over 3 nu minus 1 EndFraction" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(61 0 1 (60 (5 (3 2) 4) 58 (19 6 (18 (10 7 8 9) (17 (16 11 15 12) 13 14))) 59 (57 (47 20 (45 (29 21 27 (23 22) 28 (26 24 25)) (44 30 41 31 42 (34 32 33) 43 (40 35 (39 36 38 37)))) 46) (56 48 (55 (54 49 53 50) 51 52)))))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="61" data-semantic-children="0,60" data-semantic-content="1" data-semantic-owns="0 1 60" data-semantic-speech="xi equals StartFraction StartRoot 3 EndRoot Over 6 EndFraction b Superscript StartFraction nu minus 1 Over 3 nu minus 1 EndFraction Baseline left parenthesis StartFraction 18 StartRoot 3 EndRoot upper M Subscript w Baseline Over pi c upper N Subscript upper A Baseline l Subscript b p Baseline EndFraction right parenthesis Superscript StartFraction nu Over 3 nu minus 1 EndFraction"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="61"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="61" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="60" data-semantic-children="5,19,57" data-semantic-content="58,59" data-semantic-parent="61" data-semantic-owns="5 58 19 59 57"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="60" data-semantic-owns="3 4"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msqrt size="s" data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="3" data-semantic-children="2" data-semantic-parent="5" data-semantic-owns="2"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="60" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="6,18" data-semantic-parent="60" data-semantic-owns="6 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="19"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.396em;"><mjx-mfrac size="s" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="18" data-semantic-children="10,17" data-semantic-parent="19" data-semantic-owns="10 17"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-children="7,9" data-semantic-content="8" data-semantic-parent="18" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="10"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-parent="10" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="17" data-semantic-children="16,14" data-semantic-content="13" data-semantic-parent="18" data-semantic-owns="16 13 14"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="16" data-semantic-children="11,12" data-semantic-content="15" data-semantic-parent="17" data-semantic-owns="11 15 12"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="16"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="16"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-parent="17" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="60" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="57" data-semantic-children="47,56" data-semantic-parent="60" data-semantic-owns="47 56"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="47" data-semantic-children="45" data-semantic-content="20,46" data-semantic-parent="57" data-semantic-owns="20 45 46"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="20" data-semantic-parent="47" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S4">(</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="45" data-semantic-children="29,44" data-semantic-parent="47" data-semantic-owns="29 44"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-children="21,23,26" data-semantic-content="27,28" data-semantic-parent="45" data-semantic-owns="21 27 23 28 26"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="29"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="23" data-semantic-children="22" data-semantic-parent="29" data-semantic-owns="22"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="29" data-semantic-owns="24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="44" data-semantic-children="30,31,34,40" data-semantic-content="41,42,43" data-semantic-parent="45" data-semantic-owns="30 41 31 42 34 43 40"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="44"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="44"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="32,33" data-semantic-parent="44" data-semantic-owns="32 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="34"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="40" data-semantic-children="35,39" data-semantic-parent="44" data-semantic-owns="35 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="40"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="39" data-semantic-children="36,37" data-semantic-content="38" data-semantic-parent="40" data-semantic-owns="36 38 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="39"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="39"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="46" data-semantic-parent="47" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S4">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.873em;"><mjx-mfrac size="s" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="56" data-semantic-children="48,55" data-semantic-parent="57" data-semantic-owns="48 55"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="56"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="55" data-semantic-children="54,52" data-semantic-content="51" data-semantic-parent="56" data-semantic-owns="54 51 52"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="54" data-semantic-children="49,50" data-semantic-content="53" data-semantic-parent="55" data-semantic-owns="49 53 50"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="54"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="54"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="51" data-semantic-parent="55" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="55"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div><div class="label">(1)</div></div>In this equation, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N Subscript upper A" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper N Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is Avogadro’s number and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> represents the average concentration of the polymer. In the context of the DNA, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M Subscript w" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper M Subscript w"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l Subscript b p" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="l Subscript b p"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are the average molecular weight (650 g/mol) and the base pair size (0.34&nbsp;nm), respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Diekmann et&nbsp;al., 1982</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Diekmann, S. ∙ Hillen, W. ∙ Morgeneyer, B. ...</div><div><strong>Orientation relaxation of DNA restriction fragments and the internal mobility of the double helix</strong></div><div><em>Biophys. Chem.</em> 1982; <strong>15</strong>:263-270</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(82)80009-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6288135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2882%2980009-4&amp;pmid=6288135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Lee et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Lee, C.K. ∙ Araki, N. ∙ Sowersby, D.S. ...</div><div><strong>Factors affecting chemical-based purification of DNA from Saccharomyces cerevisiae</strong></div><div><em>Yeast.</em> 2012; <strong>29</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/yea.1918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84856594171" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fyea.1918&amp;pmid=22134898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104-1" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-1" aria-controls="sref104-1">Peale et&nbsp;al., 1989</a><div class="dropBlock__holder" data-db-target-of="sref104-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Peale, Jr., F.V. ∙ Ishibe, Y. ∙ Klinge, C.M. ...</div><div><strong>Rapid purification of the estrogen receptor by sequence-specific DNA affinity chromatography</strong></div><div><em>Biochemistry.</em> 1989; <strong>28</strong>:8671-8675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00448a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0024450125" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2605215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00448a001&amp;pmid=2605215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref115-1" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-1" aria-controls="sref115-1">Ratilainen et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref115-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Ratilainen, T. ∙ Lincoln, P. ∙ Nordén, B.</div><div><strong>A simple model for gene targeting</strong></div><div><em>Biophys. J.</em> 2001; <strong>81</strong>:2876-2885</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11606298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2801%2975928-0&amp;pmid=11606298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref165-1" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref165-1" aria-controls="sref165-1">Yonemura and Maeda, 1982</a><div class="dropBlock__holder" data-db-target-of="sref165-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>Yonemura, K. ∙ Maeda, H.</div><div><strong>A new assay method for DNase by fluorescence polarization and fluorescence intensity using DNA-ethidium bromide complex as a sensitive substrate</strong></div><div><em>J.&nbsp;Biochem.</em> 1982; <strong>92</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a134048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0020309501" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7174645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a134048&amp;pmid=7174645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The term, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b comma" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="2" data-semantic-children="0,1" data-semantic-content="1" data-semantic-owns="0 1" data-semantic-speech="b comma"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="1" data-semantic-parent="2" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> is the Kuhn length of the polymer, which is a measure of its rigidity (twice the persistence length). Based on <i>in&nbsp;vitro</i> measurements (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Baumann et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Baumann, C.G. ∙ Smith, S.B. ∙ Bloomfield, V.A. ...</div><div><strong>Ionic effects on the elasticity of single DNA molecules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:6185-6190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.12.6185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030947539" target="_blank" aria-label="Open Scopus in new tab">Scopus (771)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9177192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.12.6185&amp;pmid=9177192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Mantelli et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Mantelli, S. ∙ Muller, P. ∙ Harlepp, S. ...</div><div><strong>Conformational analysis and estimation of the persistence length of DNA using atomic force microscopy in solution</strong></div><div><em>Soft Matter.</em> 2011; <strong>7</strong>:3412-3416</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c0sm01160f" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955812312" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc0sm01160f" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109-1" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-1" aria-controls="sref109-1">Porschke, 1986</a><div class="dropBlock__holder" data-db-target-of="sref109-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Porschke, D.</div><div><strong>Structure and dynamics of double helices in solution: modes of DNA bending</strong></div><div><em>J.&nbsp;Biomol. Struct. Dyn.</em> 1986; <strong>4</strong>:373-389</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/07391102.1986.10506356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2908425/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F07391102.1986.10506356&amp;pmid=2908425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" aria-label="Reference 1991" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">1991</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Porschke, D.</div><div><strong>Persistence length and bending dynamics of DNA from electrooptical measurements at high salt concentrations</strong></div><div><em>Biophys. Chem.</em> 1991; <strong>40</strong>:169-179</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(91)87006-Q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0025911729" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1653052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2891%2987006-Q&amp;pmid=1653052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the concentration of Mg<sup>2+</sup> in the <i>E.&nbsp;coli</i> cytoplasm (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Alatossava et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alatossava, T. ∙ Jütte, H. ∙ Kuhn, A. ...</div><div><strong>Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.1.413-419.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2984182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.1.413-419.1985&amp;pmid=2984182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">Cayley et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cayley, S. ∙ Lewis, B.A. ∙ Guttman, H.J. ...</div><div><strong>Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1991; <strong>222</strong>:281-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(91)90212-O" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026409841" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1960728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2891%2990212-O&amp;pmid=1960728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Lusk et&nbsp;al., 1968</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Lusk, J.E. ∙ Williams, R.J.P. ∙ Kennedy, E.P.</div><div><strong>Magnesium and the growth of Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1968; <strong>243</strong>:2618-2624</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4968384/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2993417-4&amp;pmid=4968384" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we estimate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> to be ∼60&nbsp;nm (<a id="crosref0500" href="#sec-4">STAR Methods</a>). The remaining term <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> in Equation 1 is the Flory exponent, which relates to the solvent quality. It is equal to 0.5 for an ideal solvent and is smaller than 0.5 for a poor solvent and greater than 0.5 for a good solvent (<span class="dropBlock reference-citations"><a id="body-ref-sref119-3" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-3" aria-controls="sref119-3">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Given our simplified view of the cell and the chromosome (<a id="crosref0510" href="#fig1">Figure&nbsp;1</a>A), <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> reflects the effective solvent quality of the cytoplasm for the chromosome. This parameter can be deduced by determining the concentration of the chromosomal DNA within the nucleoid region <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis c right parenthesis" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis c right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and the average mesh size of the nucleoid <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis xi right parenthesis" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis xi right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div><section id="sec-2-1"><h3>Average DNA concentration within the nucleoid</h3><div id="p0060" role="paragraph">To determine the average concentration of DNA within the nucleoid, we took advantage that in nutrient-poor environments (e.g., glycerol as a carbon source), <i>E.&nbsp;coli</i> cells exhibit discrete B, C, and D periods, corresponding to the cell-cycle phases before, during, and after DNA replication, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Cooper and Helmstetter, 1968</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Cooper, S. ∙ Helmstetter, C.E.</div><div><strong>Chromosome replication and the division cycle of Escherichia coli B/r</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1968; <strong>31</strong>:519-540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(68)90425-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4866337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2868%2990425-7&amp;pmid=4866337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In B-period cells, the DNA mass can be unambiguously determined, as each nucleoid in these cells consists of a single, non-replicating chromosome (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Cooper and Helmstetter, 1968</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Cooper, S. ∙ Helmstetter, C.E.</div><div><strong>Chromosome replication and the division cycle of Escherichia coli B/r</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1968; <strong>31</strong>:519-540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(68)90425-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4866337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2868%2990425-7&amp;pmid=4866337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Using a previously described method (<span class="dropBlock reference-citations"><a id="body-ref-sref50-3" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-3" aria-controls="sref50-3">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we identified B-period cells within a population by examining cell size and the spatial pattern of the DNA replication marker SeqA fused to mCherry (<a id="crosref0530" href="#fig1">Figures 1</a>B, 1C, and <a id="crosref0535" href="#figs1">S1</a>A; <a id="crosref0540" href="#sec-4">STAR Methods</a>). We found that DAPI-stained nucleoids consisting of one chromosome have an average volume of ∼0.7&nbsp;μm<sup>3</sup> (<a id="crosref0545" href="#fig1">Figure&nbsp;1</a>D). The mass of the <i>E.&nbsp;coli</i> chromosome is ∼5&nbsp;× 10<sup>−12</sup> mg given that this chromosome is made of ∼4.6&nbsp;× 10<sup>6</sup> base pairs and that the average molecular weight of a base pair is 650 g/mol (<span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Lee et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Lee, C.K. ∙ Araki, N. ∙ Sowersby, D.S. ...</div><div><strong>Factors affecting chemical-based purification of DNA from Saccharomyces cerevisiae</strong></div><div><em>Yeast.</em> 2012; <strong>29</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/yea.1918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84856594171" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fyea.1918&amp;pmid=22134898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104-2" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-2" aria-controls="sref104-2">Peale et&nbsp;al., 1989</a><div class="dropBlock__holder" data-db-target-of="sref104-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Peale, Jr., F.V. ∙ Ishibe, Y. ∙ Klinge, C.M. ...</div><div><strong>Rapid purification of the estrogen receptor by sequence-specific DNA affinity chromatography</strong></div><div><em>Biochemistry.</em> 1989; <strong>28</strong>:8671-8675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00448a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0024450125" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2605215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00448a001&amp;pmid=2605215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref115-2" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-2" aria-controls="sref115-2">Ratilainen et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref115-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Ratilainen, T. ∙ Lincoln, P. ∙ Nordén, B.</div><div><strong>A simple model for gene targeting</strong></div><div><em>Biophys. J.</em> 2001; <strong>81</strong>:2876-2885</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11606298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2801%2975928-0&amp;pmid=11606298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From this, we determined the average DNA concentration within the nucleoid region to be 7.1 ± 1.0&nbsp;mg/mL (mean ± SD), whether cells were grown at 30°C or 37°C (<a id="crosref0565" href="#fig1">Figures 1</a>E and <a id="crosref0570" href="#figs1">S1</a>B).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/657ca581-59c3-45af-8abc-1be54785d948/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.05.037/asset/94e121f5-a605-4054-9d4e-85c3b21a7222/main.assets/figs1.jpg" height="362" width="499" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/657ca581-59c3-45af-8abc-1be54785d948/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Estimation of the average DNA concentration within the nucleoid region, related to <a id="crosref2765" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0220" role="paragraph">A. Example of phase contrast images with Oufti outlines in the B, C or D periods.</div><div id="fspara0225" role="paragraph">B. Probability density functions of the average DNA concentrations estimated within the nucleoid region in B-period cells (CJW6324) grown at either 30°C (green) or 37°C (red).</div><div id="fspara0230" role="paragraph">C. Probability density functions of SCF between the DNA and ribosome signals in untreated <i>E.&nbsp;coli</i> cells (CJW7039) in the B, C or D period and in cells (CJW7324) treated with rifampicin (300&nbsp;μg/ml) for 40&nbsp;min. The DNA and ribosome signals were calculated using the software package Oufti. Cells were classified into the B, C and D periods based on the cell size and the spatial pattern of the SeqA-mCherry. The SCF between the DNA and ribosome signals was also calculated as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref50-4" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-4" aria-controls="sref50-4">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There is no large difference in nucleoid exclusion for ribosomes between cells in B, C and D period as shown by the strongly overlapping distribution of SCF values between nucleoid and ribosomes. The SCF measures the pixel correlation between signals of fluorescently labeled nucleoid and ribosomes. Positive values represent mixing between the DNA and ribosomes, and negative values indicate that ribosomes are enriched outside the nucleoid. The fact that the SCF values in B-period cells are primarily negative indicate that polysomes are present and thus that transcription is ongoing. If transcription was not ongoing, the SCF values would be positive, as observed when transcription is inhibited by rifampicin.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">When we converted bulk measurements of DNA mass per cell made under three nutrient-rich growth conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Basan et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Basan, M. ∙ Zhu, M. ∙ Dai, X. ...</div><div><strong>Inflating bacterial cells by increased protein synthesis</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156178" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84945905016" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26519362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156178&amp;pmid=26519362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to DNA concentrations within the nucleoid volume <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis c right parenthesis" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis c right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> by taking into consideration changes in cell and nucleoid volumes, we obtained <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 7.4 ± 0.2&nbsp;mg/mL (mean ± SD; <a id="crosref0580" href="#sec-4">STAR Methods</a>). The agreement between this estimation and our single-cell measurements (<a id="crosref0585" href="#fig1">Figures 1</a>E and <a id="crosref0590" href="#figs1">S1</a>B) indicates that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> remains largely constant across nutrient-poor and nutrient-rich conditions. This is consistent with both DNA content per cell and nucleoid size scaling with cell size across growth conditions (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Basan et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Basan, M. ∙ Zhu, M. ∙ Dai, X. ...</div><div><strong>Inflating bacterial cells by increased protein synthesis</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156178" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84945905016" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26519362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156178&amp;pmid=26519362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-5" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-5" aria-controls="sref50-5">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126" aria-controls="sref126">Sharpe et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref126"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Sharpe, M.E. ∙ Hauser, P.M. ∙ Sharpe, R.G. ...</div><div><strong>Bacillus subtilis cell cycle as studied by fluorescence microscopy: constancy of cell length at initiation of DNA replication and evidence for active nucleoid partitioning</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1998; <strong>180</strong>:547-555</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.180.3.547-555.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9457856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.180.3.547-555.1998&amp;pmid=9457856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-2"><h3>Apparent average nucleoid mesh size in <i>E.&nbsp;coli</i> cells</h3><div id="p0070" role="paragraph">Given a <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> ≈ 7&nbsp;mg/mL, the next variable to determine was the mesh size <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis xi right parenthesis" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis xi right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Studies on biopolymer solutions and hydrogels have demonstrated that particles of size much smaller than the average mesh size diffuse freely through the meshwork (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Axpe et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Axpe, E. ∙ Chan, D. ∙ Offeddu, G.S. ...</div><div><strong>A Multiscale Model for Solute Diffusion in Hydrogels</strong></div><div><em>Macromolecules.</em> 2019; <strong>52</strong>:6889-6897</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.macromol.9b00753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85072667338" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31579160/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.macromol.9b00753&amp;pmid=31579160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Cai et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Cai, L.H. ∙ Panyukov, S. ∙ Rubinstein, M.</div><div><strong>Mobility of Nonsticky Nanoparticles in Polymer Liquids</strong></div><div><em>Macromolecules.</em> 2011; <strong>44</strong>:7853-7863</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma201583q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-80053467043" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22058573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma201583q&amp;pmid=22058573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Carn et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Carn, F. ∙ Boué, F. ∙ Djabourov, M. ...</div><div><strong>Biopolymer folding driven nanoparticle reorganization in bio-nanocomposites</strong></div><div><em>Soft Matter.</em> 2012; <strong>8</strong>:2930-2944</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c2sm06666a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84857701999" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc2sm06666a" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref161-1" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref161-1" aria-controls="sref161-1">Wong et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref161-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Wong, I.Y. ∙ Gardel, M.L. ∙ Reichman, D.R. ...</div><div><strong>Anomalous diffusion probes microstructure dynamics of entangled F-actin networks</strong></div><div><em>Phys. Rev. Lett.</em> 2004; <strong>92</strong>:178101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.92.178101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942551253" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15169197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.92.178101&amp;pmid=15169197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). When the probe size is close to the average mesh size, the probe dynamics becomes impacted by the polymer mesh. This impact exacerbates with increasing probe size. In the context of the cell, this implies that probes much smaller than the nucleoid mesh size will diffuse freely in the cytoplasm without any apparent hindrance from the nucleoid (assuming that the particles do not interact with the nucleoid). Such probes will have an equal probability of diffusing inside or outside the nucleoid. In contrast, probes larger than the average mesh size will have a higher probability of diffusing outside the nucleoid, leading to greater nucleoid exclusion with increasing probe size.</div><div id="p0075" role="paragraph">To probe the nucleoid mesh size, we first used an <i>E.&nbsp;coli</i> strain (CJW6340) that expresses a GFP-tagged, artificially designed protein that self-assembles into a 60-subunit dodecahedron nanocage of 25&nbsp;nm (<a id="crosref0630" href="#fig2">Figure&nbsp;2</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977663424" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27309817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18010&amp;pmid=27309817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Basal expression of nanocage-GFP proteins (i.e., leaky synthesis) was sufficient to produce a single nanocage-GFP particle per cell in a fraction of the population (<a id="crosref0640" href="#fig2">Figure&nbsp;2</a>B). Single-particle tracking experiments revealed no evidence of steric hindrance by the nucleoid, as the nanocage-GFP particle appeared to diffuse freely throughout the cytoplasm (<a id="crosref0645" href="#fig2">Figure&nbsp;2</a>C; <a id="crosref0650" href="#mmc1">Video S1</a>). The slope of the ensemble-averaged mean squared displacement (MSD) curve (n&nbsp;= 85 particle trajectories) was close to 1 in the log-log scale (<a id="crosref0655" href="#fig2">Figure&nbsp;2</a>D), consistent with Brownian motion (note that the deviation of the MSD from the linearity at long-timescale is due to cell confinement). Furthermore, ensemble calculation of the relative positions of nanocage-GFP particles inside cells demonstrated that the probability density of nanocage-GFP localization is uniform throughout the cytoplasm (<a id="crosref0660" href="#fig2">Figure&nbsp;2</a>E; n&nbsp;= 2,500 localizations). Thus, these 25-nm probes were neither trapped nor excluded by the nucleoid, consistent with their diffusion being Brownian. This result indicates that the apparent average nucleoid mesh size is &gt;25&nbsp;nm.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/cf747cda-c12e-4dd0-b65a-593c96dee549/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.05.037/asset/490a2c49-974c-494d-835c-897b45728b90/main.assets/gr2.jpg" height="615" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cf747cda-c12e-4dd0-b65a-593c96dee549/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Estimation of the apparent average nucleoid mesh size</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">(A) Schematic of an assembled nanocage-GFP particle.</div><div id="fspara0050" role="paragraph">(B) Fluorescence images of nanocage-GFP particles assembled in <i>E.&nbsp;coli</i> cells (CJW6340).</div><div id="fspara0055" role="paragraph">(C) Example of nanocage-GFP particle trajectory in a cell.</div><div id="fspara0060" role="paragraph">(D) Log-log ensemble-averaged MSD based on individual trajectories of nanocage-GFP particles (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="n"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 85). The slope <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis alpha right parenthesis" tabindex="0" ctxtmenu_counter="20" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis alpha right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> was fitted based on the first three time delays.</div><div id="fspara0065" role="paragraph">(E) Probability density map of the position of nanocage-GFP particles.</div><div id="fspara0070" role="paragraph">(F) Probability density maps of the position of GFP-μNS particles (n&nbsp;= 133,692) as a function of their size.</div><div id="fspara0075" role="paragraph">(G) Exclusion scores of fluorescent probes (nanocage-GFP and GFP-μNS particles) with different sizes.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph"><div id="mmc1" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc1" data-type="mp4"><span id="video_stream_mmc1_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5MzAwMGQxMzU0ODkwYWI2YzA4Zjc1MGY4ZmU2NDlmYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.VU8ors54yi41OEg5Y0ciZgPkZTmpYWIdh20IN5NolO9tZxPvPrNXE4eNrcfv3VnNOulWtoGQdFn57vKA9OhxPB3Bp8os29oD2DEU5p46WtUyMdyyWsCiwE1kNBd3P0mRPvlfNUmn69A4ChabbY4mLdbr0-GU5QCpD3yy-j1-OapymPU7QLhRElxFYwUjojrXxIZ1cnCtFlSzx8UUqKru2HSWR09d2LBpzpdbM7fIC5VQGZz-FsqO2Q8xUR321Oq6D65genjYrIczXFW_gPWdIFYy0MWgUNGJaOHsOLI8rWbTmURDbKnA3rhdrKfpT3ALFynjpNpyJW7R9F81bR3EAA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5MzAwMGQxMzU0ODkwYWI2YzA4Zjc1MGY4ZmU2NDlmYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.VU8ors54yi41OEg5Y0ciZgPkZTmpYWIdh20IN5NolO9tZxPvPrNXE4eNrcfv3VnNOulWtoGQdFn57vKA9OhxPB3Bp8os29oD2DEU5p46WtUyMdyyWsCiwE1kNBd3P0mRPvlfNUmn69A4ChabbY4mLdbr0-GU5QCpD3yy-j1-OapymPU7QLhRElxFYwUjojrXxIZ1cnCtFlSzx8UUqKru2HSWR09d2LBpzpdbM7fIC5VQGZz-FsqO2Q8xUR321Oq6D65genjYrIczXFW_gPWdIFYy0MWgUNGJaOHsOLI8rWbTmURDbKnA3rhdrKfpT3ALFynjpNpyJW7R9F81bR3EAA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc1" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 42.3026%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI5MzAwMGQxMzU0ODkwYWI2YzA4Zjc1MGY4ZmU2NDlmYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.VU8ors54yi41OEg5Y0ciZgPkZTmpYWIdh20IN5NolO9tZxPvPrNXE4eNrcfv3VnNOulWtoGQdFn57vKA9OhxPB3Bp8os29oD2DEU5p46WtUyMdyyWsCiwE1kNBd3P0mRPvlfNUmn69A4ChabbY4mLdbr0-GU5QCpD3yy-j1-OapymPU7QLhRElxFYwUjojrXxIZ1cnCtFlSzx8UUqKru2HSWR09d2LBpzpdbM7fIC5VQGZz-FsqO2Q8xUR321Oq6D65genjYrIczXFW_gPWdIFYy0MWgUNGJaOHsOLI8rWbTmURDbKnA3rhdrKfpT3ALFynjpNpyJW7R9F81bR3EAA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/bb86a3ef-f183-43c9-9c37-72d667289344/mmc1.mp4" download="mmc1.mp4" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Video (9.74 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Video S1. Diffusion of a 25-nm nanocage-GFP particle, related to Figure&nbsp;2</div><div id="espara0010" role="paragraph">This video shows an example trajectory of nanocage-GFP particle diffusing in the cytoplasm of an <i>E.&nbsp;coli</i> cell (CJW6340). The left panel shows the original GFP fluorescence signal from the particle, and the right panel shows the additional annotation based on the single-particle tracking results: the particle center was denoted by the red point, and the yellow tail represents a short trajectory indicating the particle positions within the last 5 frames.</div></div></div></div><div id="p0085" role="paragraph">Therefore, we switched to GFP-μNS particles, which are larger probes that our laboratory previously developed to probe the material properties of the cytoplasm (<span class="dropBlock reference-citations"><a id="body-ref-sref102-1" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-1" aria-controls="sref102-1">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GFP-μNS is a green fluorescent protein fused to a mammalian reovirus protein that self-assembles into a globular complex (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Broering et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Broering, T.J. ∙ Parker, J.S.L. ∙ Joyce, P.L. ...</div><div><strong>Mammalian reovirus nonstructural protein microNS forms large inclusions and colocalizes with reovirus microtubule-associated protein micro2 in transfected cells</strong></div><div><em>J.&nbsp;Virol.</em> 2002; <strong>76</strong>:8285-8297</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.76.16.8285-8297.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036316297" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12134034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.76.16.8285-8297.2002&amp;pmid=12134034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" aria-label="Reference 2005" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">2005</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Broering, T.J. ∙ Arnold, M.M. ∙ Miller, C.L. ...</div><div><strong>Carboxyl-proximal regions of reovirus nonstructural protein muNS necessary and sufficient for forming factory-like inclusions</strong></div><div><em>J.&nbsp;Virol.</em> 2005; <strong>79</strong>:6194-6206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.79.10.6194-6206.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-18144383857" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15858004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.79.10.6194-6206.2005&amp;pmid=15858004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). When expressed from the chromosome of <i>E.&nbsp;coli</i> (CJW4617) following IPTG induction (<a id="crosref0680" href="#sec-4">STAR Methods</a>), GFP-μNS assembled into fluorescent particles, usually one per cell (<span class="dropBlock reference-citations"><a id="body-ref-sref102-2" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-2" aria-controls="sref102-2">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The size of the GFP-μNS particles, which was deduced by establishing a calibration between the particle sizes and their corresponding fluorescence intensities (<span class="dropBlock reference-citations"><a id="body-ref-sref102-3" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-3" aria-controls="sref102-3">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0695" href="#sec-4">STAR Methods</a>), ranged from ∼50 to ∼200&nbsp;nm based on the level of protein expression in each cell (<a id="crosref0700" href="#figs2">Figure&nbsp;S2</a>).</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/aefb65b3-399c-43ef-9829-86c95d6da77f/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.05.037/asset/016215b4-20a2-42d7-9b97-7d26661b95c3/main.assets/figs2.jpg" height="181" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/aefb65b3-399c-43ef-9829-86c95d6da77f/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Example GFP-μNS particles of various sizes, related to <a id="crosref2775" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0240" role="paragraph">Each panel shows a GFP fluorescence image of an <i>E.</i> coli cell (CJW4617). The GFP-μNS particles appeared as distinct diffraction-limited fluorescent spots. The particle size is indicated in the upper right corner. The green and yellow outlines are the cell and nucleoid contours detected by the Oufti software package.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">We imaged GFP-μNS particles (n&nbsp;= 133,692) and calculated their relative positions in the corresponding cells from static images. We then binned all GFP-μNS particles by size and constructed probability density maps that indicate the likelihood of finding a GFP-μNS particle at a specific intracellular location across cells. For particles with the smallest average particle size (51&nbsp;nm), the probability density map appeared almost indistinguishable from that obtained for nanocage-GFP particles (<a id="optu2Wb0P0kpO" href="#fig2">Figures 2</a>E and 2F). This pattern changed in the next size bin, as probes of an average size of 58&nbsp;nm started to display an increased probability to be localized at the cell pole regions (i.e., away from the nucleoid). This enrichment in probability density of localization at the cell poles continued to increase with probe size (<a id="crosref0710" href="#fig2">Figures 2</a>F and 2G), with ever-larger particles becoming less and less likely to be found within the nucleoid region. Particles with the largest average size (150&nbsp;nm) were almost completely excluded from the nucleoid, as shown by the high probability density of localization at the cell poles and the near-zero probability density elsewhere in the cell. While these ensemble results cannot provide an exact value, they estimate the apparent average nucleoid mesh size to be around 50&nbsp;nm.</div></section><section id="sec-2-3"><h3>The <i>E.&nbsp;coli</i> cytoplasm behaves as a poor solvent for the chromosome</h3><div id="p0095" role="paragraph">Based on <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="21" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> ≈ 7&nbsp;mg/mL, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="22" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> ≈6 0&nbsp;nm, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi" tabindex="0" ctxtmenu_counter="23" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="xi"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> ≈ 50&nbsp;nm, we found that the corresponding <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="24" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> would be ∼0.36 (<a id="crosref0715" href="#fd1">Equation 1</a>). Such a small exponent (&lt; 0.5) implies that the <i>E.&nbsp;coli</i> cytoplasm effectively behaves as a poor solvent for the chromosome. Since probes can start “feeling” a meshwork at sizes lower than the mesh size (<span class="dropBlock reference-citations"><a id="body-ref-sref161-2" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref161-2" aria-controls="sref161-2">Wong et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref161-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Wong, I.Y. ∙ Gardel, M.L. ∙ Reichman, D.R. ...</div><div><strong>Anomalous diffusion probes microstructure dynamics of entangled F-actin networks</strong></div><div><em>Phys. Rev. Lett.</em> 2004; <strong>92</strong>:178101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.92.178101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942551253" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15169197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.92.178101&amp;pmid=15169197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), our <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi" tabindex="0" ctxtmenu_counter="25" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="xi"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> value may be underestimated. However, a mesh size greater than 50&nbsp;nm would result in an even smaller <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="26" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> value. The conclusion that the cytoplasm acts as a poor solvent is also robust against variability in DNA concentration across cells (<a id="crosref0725" href="#fig1">Figure&nbsp;1</a>E) or nutrient-poor or -rich growth conditions (see above). For instance, a DNA concentration of 8&nbsp;mg/mL (or higher) would correspond to a smaller <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="27" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> (<a id="crosref0730" href="#figs3">Figure&nbsp;S3</a>A). At DNA concentrations of 6&nbsp;mg/mL or as low as 5&nbsp;mg/mL,<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="28" style="font-size: 130%;"><mjx-math data-semantic-structure="1" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> would remain well below 0.5 (<a id="crosref0735" href="#figs3">Figure&nbsp;S3</a>A). In addition, our conclusion appears robust to reasonable variations in DNA Kuhn length due to potential fluctuations in cytoplasmic ionic strength. In our model, to achieve an apparent ∼50-nm mesh size in an ideal (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="29" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.5) or good (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="30" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.58) solvent, the Kuhn length of a polymer at an average concentration of 7&nbsp;mg/mL would have to be 26&nbsp;nm or 13&nbsp;nm, respectively (i.e., persistence length ≤13&nbsp;nm). Such a small value has not been observed experimentally even in solutions of extremely high ionic strength (e.g., 3–4&nbsp;M NaCl) (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Borochov et&nbsp;al., 1981</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Borochov, N. ∙ Eisenberg, H. ∙ Kam, Z.</div><div><strong>Dependence of DNA conformation on the concentration of salt</strong></div><div><em>Biopolymers.</em> 1981; <strong>20</strong>:231-235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.1981.360200116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33327671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.1981.360200116&amp;pmid=33327671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Kam et&nbsp;al., 1981</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Kam, Z. ∙ Borochov, N. ∙ Eisenberg, H.</div><div><strong>Dependence of laser light scattering of DNA on NaCl concentration</strong></div><div><em>Biopolymers.</em> 1981; <strong>20</strong>:2671-2690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.1981.360201213" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7034800/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.1981.360201213&amp;pmid=7034800" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref132" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132" aria-controls="sref132">Sobel and Harpst, 1991</a><div class="dropBlock__holder" data-db-target-of="sref132"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Sobel, E.S. ∙ Harpst, J.A.</div><div><strong>Effects of Na+ on the persistence length and excluded volume of T7 bacteriophage DNA</strong></div><div><em>Biopolymers.</em> 1991; <strong>31</strong>:1559-1564</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.360311311" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1814504/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.360311311&amp;pmid=1814504" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or in the presence of multivalent ions such as Mg<sup>2+</sup> (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Baumann et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Baumann, C.G. ∙ Smith, S.B. ∙ Bloomfield, V.A. ...</div><div><strong>Ionic effects on the elasticity of single DNA molecules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:6185-6190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.12.6185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030947539" target="_blank" aria-label="Open Scopus in new tab">Scopus (771)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9177192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.12.6185&amp;pmid=9177192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Mantelli et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Mantelli, S. ∙ Muller, P. ∙ Harlepp, S. ...</div><div><strong>Conformational analysis and estimation of the persistence length of DNA using atomic force microscopy in solution</strong></div><div><em>Soft Matter.</em> 2011; <strong>7</strong>:3412-3416</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c0sm01160f" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955812312" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc0sm01160f" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109-2" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-2" aria-controls="sref109-2">Porschke, 1986</a><div class="dropBlock__holder" data-db-target-of="sref109-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Porschke, D.</div><div><strong>Structure and dynamics of double helices in solution: modes of DNA bending</strong></div><div><em>J.&nbsp;Biomol. Struct. Dyn.</em> 1986; <strong>4</strong>:373-389</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/07391102.1986.10506356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2908425/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F07391102.1986.10506356&amp;pmid=2908425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" aria-label="Reference 1991" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">1991</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Porschke, D.</div><div><strong>Persistence length and bending dynamics of DNA from electrooptical measurements at high salt concentrations</strong></div><div><em>Biophys. Chem.</em> 1991; <strong>40</strong>:169-179</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(91)87006-Q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0025911729" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1653052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2891%2987006-Q&amp;pmid=1653052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/65a4e1c7-86b5-4dbd-88b0-97dd6ab29cb4/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.05.037/asset/7c45c482-e82e-4820-9115-f504425cd676/main.assets/figs3.jpg" height="240" width="507" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/65a4e1c7-86b5-4dbd-88b0-97dd6ab29cb4/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Relationship between variables in <a id="crosref2780" href="#fd1">Equation 1</a> and size scaling in simulated <i>E.&nbsp;coli</i> chromosomes, related to <a id="crosref2785" href="#fd1">Equation 1</a> and <a id="crosref2790" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0250" role="paragraph">A. Plot showing the average mesh size of the nucleoid as a function of the Flory exponent, with a DNA Kuhn length of 60&nbsp;nm and various DNA concentrations (5, 6, 7 or 8&nbsp;mg/ml), as calculated using Equation 1.</div><div id="fspara0255" role="paragraph">B. Scaling between the number of segments and the normalized <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="31" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> of simulated chromosomes in different types of solvent (under non-confined conditions). For a better comparison in the scaling, the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="32" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> of the simulated chromosomes of different numbers of segments were normalized by the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="33" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> of the chromosome with 10 segments. Each point is based on the average of 200 simulated chromosomes. In the main plot, the scaling is shown for small length scales as interactions become screened for polymers at large length scales (<span class="dropBlock reference-citations"><a id="body-ref-sref119-4" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-4" aria-controls="sref119-4">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As a result, the scaling exponent becomes 0.5, regardless of the solvent quality, as shown in the inset.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4"><h3>A poor solvent may promote chromosome domain formation</h3><div id="p0100" role="paragraph">Intriguingly, for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="34" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> ≈ 60&nbsp;nm and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="35" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> ≈ 7&nbsp;mg/mL, <a id="crosref0775" href="#fd1">Equation 1</a> indicates that the average mesh size decreases as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="36" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> increases (i.e., as the solvent quality improves) (<a id="crosref0780" href="#figs3">Figure&nbsp;S3</a>A). To probe the potential origin of this dependence, we performed 3D Monte Carlo simulations of chromosome conformation (<a id="crosref0785" href="#sec-4">STAR Methods</a>). For all simulations, we modeled the <i>E.&nbsp;coli</i> chromosome as a freely jointed phantom chain. The solvent quality was simply modeled by tuning the constraint on the range of angles between neighboring DNA segments (<a id="crosref0790" href="#sec-4">STAR Methods</a>). These constraints were empirically chosen so that the power law scaling between the polymer radius of gyration and the number of segments had an exponent equal to <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="37" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.36 (poor), 0.50 (ideal), and 0.58 (good) at small length scales (<a id="crosref0795" href="#figs3">Figure&nbsp;S3</a>B). Note that in a semidilute polymer solution, at large length scales, the interactions among monomers become screened such that the scaling exponent converges to 0.5 (<span class="dropBlock reference-citations"><a id="body-ref-sref119-5" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-5" aria-controls="sref119-5">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); this crossover also occurs in our simulations (<a id="crosref0805" href="#figs3">Figure&nbsp;S3</a>B, inset). To achieve the average DNA concentration observed in actual nucleoids (7&nbsp;mg/mL), we confined the entire chromosome within the average nucleoid volume (0.70&nbsp;μm<sup>3</sup>) in B-period cells (<a id="crosref0810" href="#fig1">Figure&nbsp;1</a>D).</div><div id="p0105" role="paragraph">The simulations revealed a drastic difference in chromosomal conformation across the “solvent quality” conditions (<a id="crosref0815" href="#mmc2">Video S2</a>). The DNA density appeared to be much more spatially heterogeneous in the poor solvent condition relative to the ideal and good solvent conditions. This was evident in cross-sectional slices of simulated chromosomes in which each dot represents a DNA segment crossing the plane (<a id="crosref0820" href="#fig3">Figure&nbsp;3</a>A). The spatial heterogeneity of DNA density in the poor solvent condition compared to the other conditions was also apparent in the two-dimensional (2D) histograms showing the probability density of finding a DNA segment inside the nucleoid region (<a id="crosref0825" href="#fig3">Figure&nbsp;3</a>B). Gratifyingly, this result is consistent with the spatial heterogeneity of DNA density observed in super-resolution fluorescence images of bacterial nucleoids (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Le Gall et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Le Gall, A. ∙ Cattoni, D.I. ∙ Guilhas, B. ...</div><div><strong>Bacterial partition complexes segregate within the volume of the nucleoid</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:12107</div><div>12107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms12107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977271741" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27377966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms12107&amp;pmid=27377966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Marbouty et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref133-1" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-1" aria-controls="sref133-1">Spahn et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref133-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Spahn, C. ∙ Endesfelder, U. ∙ Heilemann, M.</div><div><strong>Super-resolution imaging of Escherichia coli nucleoids reveals highly structured and asymmetric segregation during fast growth</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2014; <strong>185</strong>:243-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2014.01.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84894410667" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24473063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2014.01.007&amp;pmid=24473063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref134-1" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" aria-label="Reference 2018" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-1" aria-controls="sref134-1">2018</a><div class="dropBlock__holder" data-db-target-of="sref134-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Spahn, C.K. ∙ Glaesmann, M. ∙ Grimm, J.B. ...</div><div><strong>A toolbox for multiplexed super-resolution imaging of the E.&nbsp;coli nucleoid and membrane using novel PAINT labels</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:14768</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-33052-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85054290920" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30282984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-33052-3&amp;pmid=30282984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref137-1" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-1" aria-controls="sref137-1">Stracy et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref137-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Stracy, M. ∙ Lesterlin, C. ∙ Garza de Leon, F. ...</div><div><strong>Live-cell superresolution microscopy reveals the organization of RNA polymerase in the bacterial nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4390-E4399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507592112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26224838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507592112&amp;pmid=26224838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/42e5bfd0-54c7-4006-9976-3c18e9db413a/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.05.037/asset/a7680832-9157-48c2-bf0d-a6817a6b4ad9/main.assets/gr3.jpg" height="647" width="657" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/42e5bfd0-54c7-4006-9976-3c18e9db413a/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Simulations of the <i>E.&nbsp;coli</i> chromosome conformation in different types of solvent</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0085" role="paragraph">(A) Example slices of simulated chromosomes in a poor, ideal, or good solvent. Each dot represents a DNA segment going in or out of the slices (XY or YZ plane).</div><div id="fspara0090" role="paragraph">(B) Density maps of the slices shown in (A). The maps were constructed as the normalized 2D histograms of positions of the simulated DNA segments.</div><div id="fspara0095" role="paragraph">(C) Distance maps indicating the spatial separation between any two loci along the simulated chromosomes in the poor, ideal, or good solvent. Each map is a result of averaging 10 simulated chromosomes.</div><div id="fspara0100" role="paragraph">(D) Distance maps as in (C), except for showing only the first 500-kb region of the simulated chromosomes.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph"><div id="mmc2" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc2" data-type="mp4"><span id="video_stream_mmc2_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2MDcwYTEyNDBjMGJjMGE1MDhkYTI5ZTQ1ZGYzMTFhYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.dxbqJc4iDHREpieQCt590IE_BKFsdDDSbUEXjpvIX48uEf2-Um6Y2jsfkGVJpiEU2SjUvijWiF-dIavg3Hs0pnyv2Q1RNce2SnMqnn988l16V8DpG8TuMx1POe9HmUOpdVuE25v7mFhOAeoiEJEMsjd81o6SA7IBhZhn8ekImSidgL96Y8hZz3g6xB4erQspCIVqT4iz10F5QVIi0Nh6WAk7qnNieMnW7RruyGYeWIHiWart1wt0QtleO66e3slkp7zYlmYiRrGPrYoBms9yHoU2qbOvyR2nyl2jsqspmrZmPUQ_IGfzpG2hv8vkFIaU0Tu6t1zAB_Kp7dW1dWAlYA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2MDcwYTEyNDBjMGJjMGE1MDhkYTI5ZTQ1ZGYzMTFhYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.dxbqJc4iDHREpieQCt590IE_BKFsdDDSbUEXjpvIX48uEf2-Um6Y2jsfkGVJpiEU2SjUvijWiF-dIavg3Hs0pnyv2Q1RNce2SnMqnn988l16V8DpG8TuMx1POe9HmUOpdVuE25v7mFhOAeoiEJEMsjd81o6SA7IBhZhn8ekImSidgL96Y8hZz3g6xB4erQspCIVqT4iz10F5QVIi0Nh6WAk7qnNieMnW7RruyGYeWIHiWart1wt0QtleO66e3slkp7zYlmYiRrGPrYoBms9yHoU2qbOvyR2nyl2jsqspmrZmPUQ_IGfzpG2hv8vkFIaU0Tu6t1zAB_Kp7dW1dWAlYA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc2" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 33.3324%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI2MDcwYTEyNDBjMGJjMGE1MDhkYTI5ZTQ1ZGYzMTFhYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.dxbqJc4iDHREpieQCt590IE_BKFsdDDSbUEXjpvIX48uEf2-Um6Y2jsfkGVJpiEU2SjUvijWiF-dIavg3Hs0pnyv2Q1RNce2SnMqnn988l16V8DpG8TuMx1POe9HmUOpdVuE25v7mFhOAeoiEJEMsjd81o6SA7IBhZhn8ekImSidgL96Y8hZz3g6xB4erQspCIVqT4iz10F5QVIi0Nh6WAk7qnNieMnW7RruyGYeWIHiWart1wt0QtleO66e3slkp7zYlmYiRrGPrYoBms9yHoU2qbOvyR2nyl2jsqspmrZmPUQ_IGfzpG2hv8vkFIaU0Tu6t1zAB_Kp7dW1dWAlYA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/dd23174b-b7c5-4af3-8cd8-71e24dd8551b/mmc2.mp4" download="mmc2.mp4" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Video (42.66 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Video S2. Simulated chromosomes in different types of solvent conditions, related to Figure&nbsp;3</div><div id="espara0015" role="paragraph">This video shows a 3D overview of chromosomes simulated in confinement of a spherocylinder with a volume equal to that of an average nucleoid (∼0.7&nbsp;μm<sup>3</sup>). From left to right, the solvent was assumed to be poor (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="38" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.36), ideal (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="39" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.50) and good (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="40" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.58). In all three cases, the Kuhn length of the DNA was assumed to be 60&nbsp;nm.</div></div></div></div><div id="p0115" role="paragraph">In the poor solvent simulations, DNA segments were often closer to each other, leading to the formation of regions with high DNA density interspersed with regions of low DNA density. This spatial heterogeneity in DNA density created “holes” (<a id="crosref0855" href="#fig3">Figures 3</a>A and 3B; <a id="crosref0860" href="#mmc2">Video S2</a>), which may increase the apparent average mesh size and allow larger objects to pass through the nucleoid. Meanwhile, the DNA density of the nucleoid was more spatially homogeneous in the ideal and good solvent conditions (<a id="crosref0865" href="#fig3">Figures 3</a>A and 3B; <a id="crosref0870" href="#mmc2">Video S2</a>), likely contributing to the smaller average mesh size.</div><div id="p0120" role="paragraph">The denser, domain-like regions of the nucleoid in the poor solvent simulations reminded us of CIDs reported in Hi-C studies in <i>E.&nbsp;coli</i> and other bacteria (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Le et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Le, T.B. ∙ Imakaev, M.V. ∙ Mirny, L.A. ...</div><div><strong>High-resolution mapping of the spatial organization of a bacterial chromosome</strong></div><div><em>Science.</em> 2013; <strong>342</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1242059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84887322004" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24158908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1242059&amp;pmid=24158908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Lioy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Lioy, V.S. ∙ Cournac, A. ∙ Marbouty, M. ...</div><div><strong>Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:771-783.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85041112985" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.027&amp;pmid=29358050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Marbouty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Marbouty, M. ∙ Cournac, A. ∙ Flot, J.F. ...</div><div><strong>Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e03318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.03318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.03318&amp;pmid=25517076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref89-3" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-3" aria-controls="sref89-3">2015</a><div class="dropBlock__holder" data-db-target-of="sref89-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref142-2" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-2" aria-controls="sref142-2">Val et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref142-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Val, M.E. ∙ Marbouty, M. ∙ de Lemos Martins, F. ...</div><div><strong>A checkpoint control orchestrates the replication of the two chromosomes of Vibrio cholerae</strong></div><div><em>Sci. Adv.</em> 2016; <strong>2</strong>:e1501914</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1501914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84991009833" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27152358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1501914&amp;pmid=27152358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref151-2" href="#" role="doc-biblioref" data-xml-rid="bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151-2" aria-controls="sref151-2">Wang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref151-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Wang, X. ∙ Le, T.B. ∙ Lajoie, B.R. ...</div><div><strong>Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis</strong></div><div><em>Genes Dev.</em> 2015; <strong>29</strong>:1661-1675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.265876.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26253537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.265876.115&amp;pmid=26253537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref153-2" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-2" aria-controls="sref153-2">2017</a><div class="dropBlock__holder" data-db-target-of="sref153-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Wang, X. ∙ Brandão, H.B. ∙ Le, T.B. ...</div><div><strong>Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus</strong></div><div><em>Science.</em> 2017; <strong>355</strong>:524-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai8982" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85011362856" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28154080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai8982&amp;pmid=28154080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To examine this more closely, we created average maps that show the average distances (over 10 simulations) between any given pair of DNA loci along the entire simulated chromosomes (<a id="crosref0910" href="#fig3">Figure&nbsp;3</a>C) or a 500-kb region (<a id="crosref0915" href="#fig3">Figure&nbsp;3</a>D) under a poor, ideal, or good solvent condition. Dark patches along the diagonal in the distance maps demarcate individual domain-like structures, in which DNA loci separated by a relatively large genomic distance remain in close spatial proximity to each other. Large domain-like structures were only seen under the poor solvent condition (<a id="crosref0920" href="#fig3">Figures 3</a>C and 3D). Our results suggest that by itself, an effective poor solvent quality of the cytoplasm can promote the formation of chromosomal domains that are reminiscent of CIDs seen in Hi-C experiments (see <a id="crosref0925" href="#sec-3">Discussion</a> for further comments and additional simulations).</div></section><section id="sec-2-5"><h3>Spatial heterogeneity in ribosome density within the nucleoid correlates negatively with DNA density</h3><div id="p0125" role="paragraph">If an effective poor solvent quality promotes spatial heterogeneity of DNA density in the cell, this may, in turn, affect the spatial distribution of other cytoplasmic components, even within the nucleoid region. Ribosomes, which exist mostly in polyribosome form, are well known to be enriched outside of the nucleoid region (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Azam et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Azam, T.A. ∙ Hiraga, S. ∙ Ishihama, A.</div><div><strong>Two types of localization of the DNA-binding proteins within the Escherichia coli nucleoid</strong></div><div><em>Genes Cells.</em> 2000; <strong>5</strong>:613-626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2443.2000.00350.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0033869982" target="_blank" aria-label="Open Scopus in new tab">Scopus (169)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10947847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2443.2000.00350.x&amp;pmid=10947847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-6" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-6" aria-controls="sref50-6">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-3" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-3" aria-controls="sref117-3">Robinow and Kellenberger, 1994</a><div class="dropBlock__holder" data-db-target-of="sref117-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Robinow, C. ∙ Kellenberger, E.</div><div><strong>The bacterial nucleoid revisited</strong></div><div><em>Microbiol. Rev.</em> 1994; <strong>58</strong>:211-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/mr.58.2.211-232.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7521510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fmr.58.2.211-232.1994&amp;pmid=7521510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120-2" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-2" aria-controls="sref120-2">Sanamrad et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref120-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Sanamrad, A. ∙ Persson, F. ∙ Lundius, E.G. ...</div><div><strong>Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11413-11418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1411558111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84905669647" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1411558111&amp;pmid=25056965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We hypothesized that ribosomes may also be heterogeneously distributed throughout the nucleoid based on its uneven DNA density. To test this hypothesis, we performed cryoelectron tomography (cryo-ET) after cryo-focused ion beam (cryo-FIB) milling on frozen-hydrated samples of <i>E.&nbsp;coli</i> cells (MG1655) (<a id="crosref0955" href="#fig4">Figures 4</a>A and <a id="crosref0960" href="#figs4">S4</a>; <a id="crosref0965" href="#sec-4">STAR Methods</a>). Tomographic reconstruction and ribosome localization (<a id="crosref0970" href="#mmc3">Videos S3</a> and <a id="crosref0975" href="#mmc4">S4</a>; <a id="crosref0980" href="#sec-4">STAR Methods</a>) revealed that the density of ribosomes was highly heterogeneous across lamellae (<a id="crosref0985" href="#fig4">Figures 4</a>B–4E and see <a id="crosref0990" href="#figs5">Figure&nbsp;S5</a>A for statistics on tomograms). This was readily apparent after the Gaussian smoothing of the ribosome density map (<a id="crosref0995" href="#fig4">Figures 4</a>F and <a id="crosref1000" href="#figs5">S5</a>B). In addition to the expected accumulation of ribosomes at the cell periphery, ribosomes displayed heterogeneity in density across the cytoplasm, including within the central region of the cell (i.e., nucleoid region).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/e8ed7571-1115-45e5-8141-61ddd658a6e3/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.05.037/asset/205a0a01-a085-4c94-b5ce-537b90eb1685/main.assets/gr4.jpg" height="409" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/e8ed7571-1115-45e5-8141-61ddd658a6e3/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Spatial distribution of ribosomes in <i>E.&nbsp;coli</i> tomograms</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">(A) Schematic showing the preparation procedure for the cryo-ET samples of <i>E.&nbsp;coli</i> cells (MG1655).</div><div id="fspara0115" role="paragraph">(B) Example section of a tomogram.</div><div id="fspara0120" role="paragraph">(C) Comparison between the reference ribosome structure (gray) and the averaged structure (yellow) derived from ribosomes localized in the lamella tomogram shown in (B). An <i>E.&nbsp;coli</i> 70S ribosome structure (EMD-20173) was used as the reference (<span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Fu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Fu, Z. ∙ Indrisiunaite, G. ∙ Kaledhonkar, S. ...</div><div><strong>The structural basis for release-factor activation during translation termination revealed by time-resolved cryogenic electron microscopy</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2579</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10608-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85067294268" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31189921/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10608-z&amp;pmid=31189921" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0125" role="paragraph">(D) Maps showing localized ribosome structures (yellow) with the positions and orientations determined in the tomogram shown in (B).</div><div id="fspara0130" role="paragraph">(E) Probability density map of the localizations of ribosomes determined from the tomogram in (B). The map was constructed as a 2D histogram of the ribosome positions in the XY plane.</div><div id="fspara0135" role="paragraph">(F) Gaussian-smoothed (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="sigma" tabindex="0" ctxtmenu_counter="41" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="sigma"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 1.2 pixel, 21&nbsp;nm) density map of (E) to more clearly show subcellular regions with high and low density in ribosomes.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/cfe8608a-ac41-4ddb-aaa1-ea61b47abcef/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.05.037/asset/05c99845-7c97-44d9-82a8-9f640b82d007/main.assets/figs4.jpg" height="536" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cfe8608a-ac41-4ddb-aaa1-ea61b47abcef/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Localization of ribosomes using cryo-ET, related to <a id="crosref2800" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0265" role="paragraph">A. Representative scanning electron microscope (SEM) image showing the <i>E.&nbsp;coli</i> cells on the grid before the cryo-FIB milling.</div><div id="fspara0270" role="paragraph">B. Top view of the SEM image showing a lamella after the final polishing in cryo-FIB milling.</div><div id="fspara0275" role="paragraph">C. From top to bottom, each panel represents the same tomogram section overlaid with the ribosomes identified by the initial 3D template search, after the removal of structures in the false class (see panel D), and after the removal of duplicate structures. In each case, <i>n</i> is the remaining number of localized ribosome structures, the positions of which are marked by the green circles.</div><div id="fspara0280" role="paragraph">D. Representative structures in the true and false classes during classification of the candidate 70S ribosome structures found in the 3D template search.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/324691b1-4b6a-4c55-bbda-5162c3abf59b/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.05.037/asset/460ce7af-3489-4a19-94f4-24179a186a06/main.assets/figs5.jpg" height="617" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/324691b1-4b6a-4c55-bbda-5162c3abf59b/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Ribosome statistics and classification of polyribosomes, related to <a id="crosref2805" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0290" role="paragraph">A. Statistics on the lamellae and the localized ribosomes. Note that the ribosome density within a lamella does not necessarily reflect the density of ribosomes per cell given the non-uniform distribution of ribosomes within cells.</div><div id="fspara0295" role="paragraph">B. Additional examples of probability density maps of ribosome structures localized in tomograms. All maps were Gaussian smoothed (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="sigma" tabindex="0" ctxtmenu_counter="42" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="sigma"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 1.2 pixel, 21&nbsp;nm) for easier visualization of the ribosome regions of low and high densities.</div><div id="fspara0300" role="paragraph">C. Average structures of ribosomes classified as polyribosomes. Using a similar method described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref99-1" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-1" aria-controls="sref99-1">O’Reilly et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref99-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>O’Reilly, F.J. ∙ Xue, L. ∙ Graziadei, A. ...</div><div><strong>In-cell architecture of an actively transcribing-translating expressome</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:554-557</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb3758" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85088883448" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32732422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb3758&amp;pmid=32732422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), polyribosomes were classified using the software RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Bharat and Scheres, 2016</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bharat, T.A. ∙ Scheres, S.H.</div><div><strong>Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:2054-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.124" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84992573769" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27685097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.124&amp;pmid=27685097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). See <a id="crosref2820" href="#sec-4">STAR Methods</a> for details.</div><div id="fspara0305" role="paragraph">D. Scatter plots showing positions of ribosomes classified as polyribosomes (left column) and those that were not (right column). Every scatter point is colored based on the number of its neighbors within a radius of 100&nbsp;nm. On average, only around 20% of the ribosomes were classified as polyribosomes. This is likely an underestimation, as it is known that around 70%–80% of the ribosomes are bound to mRNA (<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Dai, X. ∙ Zhu, M. ∙ Warren, M. ...</div><div><strong>Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>2</strong>:16231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85005950833" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27941827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.231&amp;pmid=27941827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Forchhammer and Lindahl, 1971</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Forchhammer, J. ∙ Lindahl, L.</div><div><strong>Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of Escherichia coli 15</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:563-568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90337-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0015222649" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4927947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990337-8&amp;pmid=4927947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107-1" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-1" aria-controls="sref107-1">Phillips et&nbsp;al., 1969</a><div class="dropBlock__holder" data-db-target-of="sref107-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Phillips, L.A. ∙ Hotham-Iglewski, B. ∙ Franklin, R.M.</div><div><strong>Polyribosomes of Escherichia coli. II. Experiments to determine the in&nbsp;vivo distribution of polysomes, ribosomes and ribosomal subunits</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1969; <strong>45</strong>:23-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(69)90207-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4186447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2869%2990207-1&amp;pmid=4186447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref145-1" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-1" aria-controls="sref145-1">Varricchio and Monier, 1971</a><div class="dropBlock__holder" data-db-target-of="sref145-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Varricchio, F. ∙ Monier, R.</div><div><strong>Ribosome patterns in Escherichia coli growing at various rates</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1971; <strong>108</strong>:105-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.108.1.105-110.1971" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5001192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.108.1.105-110.1971&amp;pmid=5001192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This suggests that the classification method by <span class="dropBlock reference-citations"><a id="body-ref-sref99-2" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-2" aria-controls="sref99-2">O’Reilly et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref99-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>O’Reilly, F.J. ∙ Xue, L. ∙ Graziadei, A. ...</div><div><strong>In-cell architecture of an actively transcribing-translating expressome</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:554-557</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb3758" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85088883448" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32732422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb3758&amp;pmid=32732422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> identifies only a subset of polyribosomes in our samples, i.e., those that have back-to-back ribosomes in a specific relative orientation. This underestimation is not surprising, as many mRNAs are likely to have differentially spaced ribosomes along the transcript due to differences in rates of translation initiation and ribosome elongation. This is consistent with the high variability in ribosome densities across transcripts (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Li et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Li, G.W. ∙ Burkhardt, D. ∙ Gross, C. ...</div><div><strong>Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources</strong></div><div><em>Cell.</em> 2014; <strong>157</strong>:624-635</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.02.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.02.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84899550455" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24766808/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.02.033&amp;pmid=24766808" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhM2QyYjc1Y2QwOTI4YjA4ZGE5Zjc0YzZhOGM4N2U5YSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.bMYgM3qYmwECM63mhGdSog4vLszqIqliVW23IBucTvPBk-9OaYoCCqIyuAuk77jUCDbW7J_vFth-QUk2v-hS3lYNzqJU2GkU9Amoq0UeAM5sQFUK1vNyFLpv7L575zkWiXiZ6A7DM7IkvKd0ht6quRS8BDCXtoDlHtokzM55bGRfHwfIGLzs1BEngUv-WYYl6lclM8bBxh1qz3iv_eqh2f65mCYF3OkRlUfUlbtB0-boy8JyRAh1APrWoFfl2Bj0fWwrhLaxs59LTezU90isqOIMsh9MzDdKRTX4LgVhyImq5Dvm0sahAZbOqVAUScYiwnDsPfUxVMPsWulDUrgdCA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhM2QyYjc1Y2QwOTI4YjA4ZGE5Zjc0YzZhOGM4N2U5YSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.bMYgM3qYmwECM63mhGdSog4vLszqIqliVW23IBucTvPBk-9OaYoCCqIyuAuk77jUCDbW7J_vFth-QUk2v-hS3lYNzqJU2GkU9Amoq0UeAM5sQFUK1vNyFLpv7L575zkWiXiZ6A7DM7IkvKd0ht6quRS8BDCXtoDlHtokzM55bGRfHwfIGLzs1BEngUv-WYYl6lclM8bBxh1qz3iv_eqh2f65mCYF3OkRlUfUlbtB0-boy8JyRAh1APrWoFfl2Bj0fWwrhLaxs59LTezU90isqOIMsh9MzDdKRTX4LgVhyImq5Dvm0sahAZbOqVAUScYiwnDsPfUxVMPsWulDUrgdCA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJhM2QyYjc1Y2QwOTI4YjA4ZGE5Zjc0YzZhOGM4N2U5YSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.bMYgM3qYmwECM63mhGdSog4vLszqIqliVW23IBucTvPBk-9OaYoCCqIyuAuk77jUCDbW7J_vFth-QUk2v-hS3lYNzqJU2GkU9Amoq0UeAM5sQFUK1vNyFLpv7L575zkWiXiZ6A7DM7IkvKd0ht6quRS8BDCXtoDlHtokzM55bGRfHwfIGLzs1BEngUv-WYYl6lclM8bBxh1qz3iv_eqh2f65mCYF3OkRlUfUlbtB0-boy8JyRAh1APrWoFfl2Bj0fWwrhLaxs59LTezU90isqOIMsh9MzDdKRTX4LgVhyImq5Dvm0sahAZbOqVAUScYiwnDsPfUxVMPsWulDUrgdCA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/b128ac07-bf77-4ffd-a786-d30ec549466c/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (54.38 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S3. Tomogram of the <i>E.&nbsp;coli</i> lamella, related to Figure&nbsp;4</div><div id="espara0020" role="paragraph">This video shows the tomogram of an <i>E.&nbsp;coli</i> lamella obtained after the FIB milling. The lamella had an average thickness of 155&nbsp;nm, and 5,028 ribosomes were detected in it. Based on their corresponding positions and orientations identified in the analysis, the ribosome structures (colored yellow) were placed back into the tomogram for visual inspection. The video shows individual tomogram sections along the Z axis.</div></div></div></div><div id="p0135" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMTY4NDRmMTUzNTZmZWY5MzdlMTZiMDI0NTEwYWQwMSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.htIaG1HuqLNCTHceCZCTIK3gFqnxY94PB703ZD7QucHvGM98yUpjQUQ-HKb6pc_PevsZuugwFbt_VYKPXz3E7Qkppvpd8IoJxImIWJeCCABvDkfFL9TOWU3_4sBSklUlDqQ75nxDOjumCgxtHIObGHNJSfiHVOygOBoy54st8jJLukFj7SqYXzQ3U0wXrGoVlJua52oJ_XEVxGTDr4hELoBCZNpzEczOKkpuXz7P3QEzxAYFpbGmOqNv5mMhRsY0sxCzZoySbCPXudCUFChB4jtNY5PzoiSoj0Bq4-Y_UCiJZLb1z-YuoHUB7eUVqvobMO5sJhBvPsgTl9gDBdHTfw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMTY4NDRmMTUzNTZmZWY5MzdlMTZiMDI0NTEwYWQwMSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.htIaG1HuqLNCTHceCZCTIK3gFqnxY94PB703ZD7QucHvGM98yUpjQUQ-HKb6pc_PevsZuugwFbt_VYKPXz3E7Qkppvpd8IoJxImIWJeCCABvDkfFL9TOWU3_4sBSklUlDqQ75nxDOjumCgxtHIObGHNJSfiHVOygOBoy54st8jJLukFj7SqYXzQ3U0wXrGoVlJua52oJ_XEVxGTDr4hELoBCZNpzEczOKkpuXz7P3QEzxAYFpbGmOqNv5mMhRsY0sxCzZoySbCPXudCUFChB4jtNY5PzoiSoj0Bq4-Y_UCiJZLb1z-YuoHUB7eUVqvobMO5sJhBvPsgTl9gDBdHTfw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMTY4NDRmMTUzNTZmZWY5MzdlMTZiMDI0NTEwYWQwMSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.htIaG1HuqLNCTHceCZCTIK3gFqnxY94PB703ZD7QucHvGM98yUpjQUQ-HKb6pc_PevsZuugwFbt_VYKPXz3E7Qkppvpd8IoJxImIWJeCCABvDkfFL9TOWU3_4sBSklUlDqQ75nxDOjumCgxtHIObGHNJSfiHVOygOBoy54st8jJLukFj7SqYXzQ3U0wXrGoVlJua52oJ_XEVxGTDr4hELoBCZNpzEczOKkpuXz7P3QEzxAYFpbGmOqNv5mMhRsY0sxCzZoySbCPXudCUFChB4jtNY5PzoiSoj0Bq4-Y_UCiJZLb1z-YuoHUB7eUVqvobMO5sJhBvPsgTl9gDBdHTfw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/75ab8221-9a1f-413f-a4e0-b6275cd96525/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (14.89 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S4. Comparison between the reference and the average structure of localized 70S ribosomes, related to Figure&nbsp;4</div><div id="espara0025" role="paragraph">This video shows a 3D comparison between the low-pass filtered (∼3&nbsp;nm resolution) reference structure (gray) obtained from (Fu et&nbsp;al., 2019) and the average structure (yellow) based on all ribosomes localized in the tomogram shown in Figure&nbsp;4B and Video S3.</div></div></div></div><div id="p0140" role="paragraph">Since DNA cannot be directly visualized in the tomograms, we turned to fluorescence microscopy to examine whether the spatial heterogeneity in ribosome density within the nucleoid region is associated with differences in DNA densities. We imaged DAPI-stained <i>E.&nbsp;coli</i> cells (CJW7020) producing 50S ribosomal protein L1 tagged with monomeric superfolder GFP (<a id="crosref1005" href="#fig5">Figures 5</a>A and 5B). As before, in each cell (n&nbsp;= 1,126), we identified the nucleoid outlines based on the DAPI signal (dotted lines in <a id="crosref1010" href="#fig5">Figures 5</a>A and 5B). When calculating the correlation between the DNA and ribosome fluorescence signals, we considered only image pixels well within the nucleoid region (at least two pixels away from the nucleoid outline; third row images in <a id="crosref1015" href="#fig5">Figures 5</a>A and 5B) to prevent any bias in the correlation analysis due to the well-known enrichment of ribosomes outside the nucleoid region. We found that even well within the nucleoid, the ribosome fluorescence signal correlates negatively with the DNA signal, as shown by two cell examples (<a id="crosref1020" href="#fig5">Figures 5</a>C and 5D). This was also observed at the population level (n&nbsp;= 1,126 cells, <a id="crosref1025" href="#fig5">Figure&nbsp;5</a>E). These findings are consistent with the idea that the effective poor solvent quality of the cytoplasm contributes to the uneven DNA density, which, in turn, affects ribosome localization (or the other way around).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/1c23f037-5dda-400e-a132-1369ea01a5df/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.05.037/asset/7f23ef86-b1f2-4494-9f58-36c385145cf8/main.assets/gr5.jpg" height="525" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/1c23f037-5dda-400e-a132-1369ea01a5df/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Negative correlation between the density of ribosomes and DNA within the nucleoid region</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0145" role="paragraph">(A) Fluorescence micrographs showing an example cell (CJW7020) with a strong negative correlation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho" tabindex="0" ctxtmenu_counter="43" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="rho"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= −0.89) between the DNA and ribosome fluorescence signals (shown in a gray or color scale). The cell outlines are depicted as white solid lines, whereas the nucleoid outlines are shown as either black or white dashed lines. The bottom row shows only the pixels enclosed well within the nucleoid (at least two pixels away from the nucleoid outline). These pixels were used for the correlation calculation.</div><div id="fspara0150" role="paragraph">(B) Similar to (A), except showing an example cell with a more common negative correlation (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho" tabindex="0" ctxtmenu_counter="44" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="rho"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= −0.60) between DNA and ribosome fluorescence signals.</div><div id="fspara0155" role="paragraph">(C) Scatter plot showing the pixel intensity correlation between DAPI and L1-msfGFP signals obtained in the cell shown in (A).</div><div id="fspara0160" role="paragraph">(D) Similar to (C), except for showing the corresponding pixel intensity correlation in the cell shown in (B).</div><div id="fspara0165" role="paragraph">(E) Probability density function of pixel intensity correlations calculated among the cell population (n&nbsp;= 1,126).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-6"><h3>An effective poor solvent quality contributes to significant chromosome compaction</h3><div id="p0145" role="paragraph">A polymer is intrinsically more compact in a poor solvent, because polymer segments preferentially interact with themselves rather than with the solvent. In our previous Monte Carlo&nbsp;simulations, the chromosome was modeled within the experimentally determined nucleoid volume (0.7&nbsp;μm<sup>3</sup>) to keep the DNA concentration constant (i.e., 7&nbsp;mg/mL) across solvent types. If our simulation approach reproduces expected solvent effects, then performing the same Monte Carlo simulations, this time in an unbounded space to allow the DNA concentration to change (<a id="crosref1030" href="#sec-4">STAR Methods</a>), should result in simulated chromosomes being the most compact in the poor solvent condition. Simulation results validated this expectation (<a id="crosref1035" href="#mmc5">Video S5</a>) and showed that the effective poor solvent quality of the cytoplasm can result in up to ∼60-fold chromosome compaction compared to a good solvent (<a id="crosref1040" href="#fig6">Figure&nbsp;6</a>). We note that this solvent effect alone cannot explain the full compaction of a single chromosome inside cells, as the actual volume of a single-chromosome nucleoid in cells (<a id="crosref1045" href="#fig1">Figure&nbsp;1</a>D) is still ∼10 times smaller (<a id="crosref1050" href="#fig6">Figure&nbsp;6</a>, inset). This is not surprising given the simplicity of our model; it highlights the complexity of the cell and the multifactorial origin of chromosome compaction (see <a id="crosref1055" href="#sec-3">Discussion</a>).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/2d2bd7a5-b741-4e19-86bc-339154df78dc/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.05.037/asset/bb703af3-4891-4b31-84cb-9b3c5a253d63/main.assets/gr6.jpg" height="377" width="309" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/2d2bd7a5-b741-4e19-86bc-339154df78dc/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Simulation results of chromosome size in different types of solvent</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">Bar graph showing the mean volume of 1,000 simulated chromosomes in each type of solvent. The error bars represent the 95% confidence interval of the mean. The inset shows a comparison between the experimentally determined volume ("Exp") of the nucleoid observed in the B-period cells (<a id="crosref2760" href="#fig1">Figure&nbsp;1</a>D) and the volume calculated for simulated chromosomes (n&nbsp;= 1,000) in the poor solvent.</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkMGYzOGUxNDMwNDlmNmM0MjdjMjVkYjAyZmJmZGY1ZSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.cXrnPiVcjSggGD07K3WMGa-ME5oMibDKnuW6cbMzVpItWqgAatHEz14buF4UNBoLEwnoiW_EXJ-9Q-uMr4ifhhu_CyQ70o-H7xcq-0WdMXDflxSZfo1Nsk2zTwUREm_l-yS4mAahpiGX1J0FVjfKv0n_VHxWsdSU1sX__JayMUJR97rPRFpXl9jpfUYjPX2Gx2zAv-NKYqExU-gwDVvr0dbkrkqanJarS0ZVn2PuvGKM6E2ZxH7Xho_tv0jAq1A2SlRhN6FZn6WbQunD8153HAKQ2jZrwPGomqtrlQjxUfeRyO4wCvdtikBJLhgSNvf94I5c91RE7JC2eQOX7ZFceA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkMGYzOGUxNDMwNDlmNmM0MjdjMjVkYjAyZmJmZGY1ZSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.cXrnPiVcjSggGD07K3WMGa-ME5oMibDKnuW6cbMzVpItWqgAatHEz14buF4UNBoLEwnoiW_EXJ-9Q-uMr4ifhhu_CyQ70o-H7xcq-0WdMXDflxSZfo1Nsk2zTwUREm_l-yS4mAahpiGX1J0FVjfKv0n_VHxWsdSU1sX__JayMUJR97rPRFpXl9jpfUYjPX2Gx2zAv-NKYqExU-gwDVvr0dbkrkqanJarS0ZVn2PuvGKM6E2ZxH7Xho_tv0jAq1A2SlRhN6FZn6WbQunD8153HAKQ2jZrwPGomqtrlQjxUfeRyO4wCvdtikBJLhgSNvf94I5c91RE7JC2eQOX7ZFceA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 195.651%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkMGYzOGUxNDMwNDlmNmM0MjdjMjVkYjAyZmJmZGY1ZSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.cXrnPiVcjSggGD07K3WMGa-ME5oMibDKnuW6cbMzVpItWqgAatHEz14buF4UNBoLEwnoiW_EXJ-9Q-uMr4ifhhu_CyQ70o-H7xcq-0WdMXDflxSZfo1Nsk2zTwUREm_l-yS4mAahpiGX1J0FVjfKv0n_VHxWsdSU1sX__JayMUJR97rPRFpXl9jpfUYjPX2Gx2zAv-NKYqExU-gwDVvr0dbkrkqanJarS0ZVn2PuvGKM6E2ZxH7Xho_tv0jAq1A2SlRhN6FZn6WbQunD8153HAKQ2jZrwPGomqtrlQjxUfeRyO4wCvdtikBJLhgSNvf94I5c91RE7JC2eQOX7ZFceA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/2e4dcf70-c741-46e5-8b29-fe43d1e97360/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (5.57 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S5. Simulated chromosomes in free space in different types of solvent, related to Figure&nbsp;6</div><div id="espara0030" role="paragraph">This video shows a 3D overview of chromosomes simulated in free (unbound) space in different types of solvent. From top to bottom, the solvent was assumed to be poor (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="45" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.36), ideal (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="46" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.50) and good (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="47" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.58). In all three cases, the Kuhn length of the DNA was assumed to be 60&nbsp;nm.</div></div></div></div></section><section id="sec-2-7"><h3>Transcription may contribute to the effective poor solvent quality of the cytoplasm</h3><div id="p0155" role="paragraph">We expect that a combination of factors contributes to the observed poor solvent quality of the cytoplasm for the chromosome (see <a id="crosref1060" href="#sec-3">Discussion</a>). Among these factors, we find RNAs to be attractive candidates. Indeed, treatment of <i>E.&nbsp;coli</i> with rifampicin (Rif), which blocks transcription initiation and leads to mRNA depletion due to the short lifetime of most bacterial mRNAs (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Bernstein et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bernstein, J.A. ∙ Khodursky, A.B. ∙ Lin, P.H. ...</div><div><strong>Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:9697-9702</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.112318199" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0037162469" target="_blank" aria-label="Open Scopus in new tab">Scopus (571)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12119387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.112318199&amp;pmid=12119387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Chen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Chen, H. ∙ Shiroguchi, K. ∙ Ge, H. ...</div><div><strong>Genome-wide study of mRNA degradation and transcript elongation in Escherichia coli</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:781</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20145794" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84922888717" target="_blank" aria-label="Open Scopus in new tab">Scopus (105)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25583150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20145794&amp;pmid=25583150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref125" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125" aria-controls="sref125">Selinger et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref125"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Selinger, D.W. ∙ Saxena, R.M. ∙ Cheung, K.J. ...</div><div><strong>Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:216-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.912603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0037315564" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12566399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.912603&amp;pmid=12566399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), is known to result in nucleoid expansion (<span class="dropBlock reference-citations"><a id="body-ref-sref6-3" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-3" aria-controls="sref6-3">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref7-3" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-3" aria-controls="sref7-3">2014</a><div class="dropBlock__holder" data-db-target-of="sref7-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Cabrera et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cabrera, J.E. ∙ Cagliero, C. ∙ Quan, S. ...</div><div><strong>Active transcription of rRNA operons condenses the nucleoid in Escherichia coli: examining the effect of transcription on nucleoid structure in the absence of transertion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:4180-4185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01707-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67649394401" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19395497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01707-08&amp;pmid=19395497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Cabrera and Jin, 2003</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Cabrera, J.E. ∙ Jin, D.J.</div><div><strong>The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues</strong></div><div><em>Mol. Microbiol.</em> 2003; <strong>50</strong>:1493-1505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2958.2003.03805.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0346887121" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14651633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2958.2003.03805.x&amp;pmid=14651633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Dworsky and Schaechter, 1973</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Dworsky, P. ∙ Schaechter, M.</div><div><strong>Effect of rifampin on the structure and membrane attachment of the nucleoid of Escherichia coli</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1973; <strong>116</strong>:1364-1374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.116.3.1364-1374.1973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4584813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.116.3.1364-1374.1973&amp;pmid=4584813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Pettijohn and Hecht, 1974</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Pettijohn, D.E. ∙ Hecht, R.</div><div><strong>RNA molecules bound to the folded bacterial genome stabilize DNA folds and segregate domains of supercoiling</strong></div><div><em>Cold Spring Harb. Symp. Quant. Biol.</em> 1974; <strong>38</strong>:31-41</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/SQB.1974.038.01.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0016000696" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4598638/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2FSQB.1974.038.01.006&amp;pmid=4598638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref138-1" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-1" aria-controls="sref138-1">Sun and Margolin, 2004</a><div class="dropBlock__holder" data-db-target-of="sref138-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Sun, Q. ∙ Margolin, W.</div><div><strong>Effects of perturbing nucleoid structure on nucleoid occlusion-mediated toporegulation of FtsZ ring assembly</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2004; <strong>186</strong>:3951-3959</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.186.12.3951-3959.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942547429" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15175309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.186.12.3951-3959.2004&amp;pmid=15175309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref164-3" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164-3" aria-controls="sref164-3">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref164-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Yang, D. ∙ Männik, J. ∙ Retterer, S.T. ...</div><div><strong>The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid</strong></div><div><em>Mol. Microbiol.</em> 2020; <strong>113</strong>:1022-1037</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85078903920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31961016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14467&amp;pmid=31961016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, treatment with chloramphenicol (Cpl), which inhibits ribosome translocation and stabilizes mRNAs (<span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Lopez et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Lopez, P.J. ∙ Marchand, I. ∙ Yarchuk, O. ...</div><div><strong>Translation inhibitors stabilize Escherichia coli mRNAs independently of ribosome protection</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1998; <strong>95</strong>:6067-6072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.95.11.6067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0032568544" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9600918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.95.11.6067&amp;pmid=9600918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103-1" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-1" aria-controls="sref103-1">Pato et&nbsp;al., 1973</a><div class="dropBlock__holder" data-db-target-of="sref103-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Pato, M.L. ∙ Bennett, P.M. ∙ von Meyenburg, K.</div><div><strong>Messenger ribonucleic acid synthesis and degradation in <i>Escherichia coli</i> during inhibition of translation</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1973; <strong>116</strong>:710-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.116.2.710-718.1973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4583248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.116.2.710-718.1973&amp;pmid=4583248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref123-1" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-1" aria-controls="sref123-1">Schneider et&nbsp;al., 1978</a><div class="dropBlock__holder" data-db-target-of="sref123-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Schneider, E. ∙ Blundell, M. ∙ Kennell, D.</div><div><strong>Translation and mRNA decay</strong></div><div><em>Mol. Gen. Genet.</em> 1978; <strong>160</strong>:121-129</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00267473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/349350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00267473&amp;pmid=349350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), is associated with nucleoid compaction (<span class="dropBlock reference-citations"><a id="body-ref-sref6-4" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-4" aria-controls="sref6-4">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref7-4" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-4" aria-controls="sref7-4">2014</a><div class="dropBlock__holder" data-db-target-of="sref7-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Cabrera et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cabrera, J.E. ∙ Cagliero, C. ∙ Quan, S. ...</div><div><strong>Active transcription of rRNA operons condenses the nucleoid in Escherichia coli: examining the effect of transcription on nucleoid structure in the absence of transertion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:4180-4185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01707-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67649394401" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19395497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01707-08&amp;pmid=19395497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-1" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-1" aria-controls="sref143-1">van Helvoort et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref143-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>van Helvoort, J.M. ∙ Kool, J. ∙ Woldringh, C.L.</div><div><strong>Chloramphenicol causes fusion of separated nucleoids in Escherichia coli K-12 cells and filaments</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1996; <strong>178</strong>:4289-4293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.178.14.4289-4293.1996" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029978680" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8763959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.178.14.4289-4293.1996&amp;pmid=8763959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref174-1" href="#" role="doc-biblioref" data-xml-rid="bib174" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref174-1" aria-controls="sref174-1">Zimmerman, 2002</a><div class="dropBlock__holder" data-db-target-of="sref174-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Zimmerman, S.B.</div><div><strong>Toroidal nucleoids in Escherichia coli exposed to chloramphenicol</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2002; <strong>138</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1047-8477(02)00036-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036420081" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12217658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1047-8477%2802%2900036-9&amp;pmid=12217658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These observations have been interpreted as a consequence of a loss or gain of polyribosome crowding in Rif- or Cpl-treated cells, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref6-5" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-5" aria-controls="sref6-5">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref7-5" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-5" aria-controls="sref7-5">2014</a><div class="dropBlock__holder" data-db-target-of="sref7-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Mondal et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Mondal, J. ∙ Bratton, B.P. ∙ Li, Y. ...</div><div><strong>Entropy-based mechanism of ribosome-nucleoid segregation in E.&nbsp;coli cells</strong></div><div><em>Biophys. J.</em> 2011; <strong>100</strong>:2605-2613</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79960325276" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21641305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2011.04.030&amp;pmid=21641305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The idea is that the large size of polyribosomes leads to entropy-driven polyribosome/DNA segregation and chromosome compaction through excluded volume effects. However, Rif and Cpl treatments also affect the levels of mRNAs in opposite direction. Furthermore, rRNA synthesis is also impacted by these drug treatments. rRNA synthesis stops with transcription inhibition (Rif), while it persists when proteins synthesis (Cpl) is inhibited (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Cole et&nbsp;al., 1987</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Cole, J.R. ∙ Olsson, C.L. ∙ Hershey, J.W. ...</div><div><strong>Feedback regulation of rRNA synthesis in Escherichia coli. Requirement for initiation factor IF2</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1987; <strong>198</strong>:383-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(87)90288-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2448483/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2887%2990288-9&amp;pmid=2448483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Hernandez and Bremer, 1990</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Hernandez, V.J. ∙ Bremer, H.</div><div><strong>Guanosine tetraphosphate (ppGpp) dependence of the growth rate control of rrnB P1 promoter activity in Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1990; <strong>265</strong>:11605-11614</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2819%2938441-8&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2819%2938441-8&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2114400/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2938441-8&amp;pmid=2114400" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-1" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-1" aria-controls="sref73-1">Kurland and Maaloe, 1962</a><div class="dropBlock__holder" data-db-target-of="sref73-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Kurland, C.G. ∙ Maaloe, O.</div><div><strong>Regulation of ribosomal and transfer RNA synthesis</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1962; <strong>4</strong>:193-210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(62)80051-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14460744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2862%2980051-5&amp;pmid=14460744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, another mutually nonexclusive interpretation of the drug-induced phenotypes is that the chemical nature of the RNAs, and not exclusively the size of the polyribosomes, plays a role in controlling nucleoid size by affecting the effective solvent quality for the chromosome.</div><div id="p0160" role="paragraph">If polyribosome crowding is the primary contributor of nucleoid compaction, then this hypothesis predicts that a reduced abundance in polyribosomes will lead to nucleoid expansion even when transcription is unaffected. To test this possibility, we treated <i>E.&nbsp;coli</i> cells with 5&nbsp;mg/mL kasugamycin (Ksg), a translation initiation inhibitor of canonical RNAs (i.e., transcripts with a leader sequence) (<span class="dropBlock reference-citations"><a id="body-ref-sref122" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122" aria-controls="sref122">Schluenzen et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref122"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Schluenzen, F. ∙ Takemoto, C. ∙ Wilson, D.N. ...</div><div><strong>The antibiotic kasugamycin mimics mRNA nucleotides to destabilize tRNA binding and inhibit canonical translation initiation</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2006; <strong>13</strong>:871-878</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb1145" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33749531968" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16998488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb1145&amp;pmid=16998488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref140" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140" aria-controls="sref140">Tai et&nbsp;al., 1973</a><div class="dropBlock__holder" data-db-target-of="sref140"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Tai, P.C. ∙ Wallace, B.J. ∙ Davis, B.D.</div><div><strong>Actions of aurintricarboxylate, kasugamycin, and pactamycin on Escherichia coli polysomes</strong></div><div><em>Biochemistry.</em> 1973; <strong>12</strong>:616-620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00728a008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4570848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00728a008&amp;pmid=4570848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At this concentration, Ksg has been shown to reduce the abundance of polyribosomes by converting a significant fraction of them into free ribosomal subunits and ribosome-free mRNAs (<span class="dropBlock reference-citations"><a id="body-ref-sref7-6" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-6" aria-controls="sref7-6">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Contrary to prediction, we found that DAPI-stained cells do not expand their nucleoids after incubation (40&nbsp;min) with Ksg, unlike with Rif (<a id="crosref1205" href="#fig7">Figures 7</a>A and <a id="crosref1210" href="#figs6">S6</a>A). In fact, most nucleoids in Ksg-treated cells appeared more compact, consistent with a previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref7-7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-7" aria-controls="sref7-7">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Quantification of a time-course experiment in which cells were imaged after Rif or Ksg treatment for 20 to 80&nbsp;min revealed a large overlap of size distributions between drug-treated and untreated cells (<a id="crosref1220" href="#fig7">Figure&nbsp;7</a>B). The nucleocytoplasmic (NC) ratio (nucleoid area divided by cell area) increased following Rif treatment (<a id="crosref1225" href="#fig7">Figure&nbsp;7</a>C), as expected for nucleoid expansion. In contrast, the NC ratio decreased over time in Ksg-treated cells (<a id="crosref1230" href="#fig7">Figure&nbsp;7</a>C), indicative of nucleoid compaction. These results conflict with the polyribosome crowding hypothesis.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/e0bcebff-e3eb-480b-8184-6675e9f261d1/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.05.037/asset/a76731be-f1f2-48ee-9d99-b541578be9d4/main.assets/gr7.jpg" height="500" width="752" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/e0bcebff-e3eb-480b-8184-6675e9f261d1/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">RNA/DNA segregation and nucleoid compaction</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0185" role="paragraph">(A) Example micrographs of DAPI-stained <i>E.&nbsp;coli</i> cells (CJW7324) untreated or treated with Rif or Ksg for 40&nbsp;min.</div><div id="fspara0190" role="paragraph">(B) Violin plots showing the distributions of cell area for untreated CJW7324 cells and cells treated with Rif or Ksg for 20–80&nbsp;min.</div><div id="fspara0195" role="paragraph">(C) Probability density functions of NC ratios of cells described in (B).</div><div id="fspara0200" role="paragraph">(D) Probability density functions of signal correlation factor (SCF) between DAPI and RNASelect for cells in (B).</div><div id="fspara0205" role="paragraph">(E) Scatter plot showing the correlation between the NC ratio and the RNASelect fluorescence density for cells treated with Ksg for 40&nbsp;min. The scatter points are colored by the corresponding SCF between the DAPI and RNASelect signals. The Spearman's correlation coefficient between the NC ratio and the RNASelect fluorescence density is shown in the top right corner.</div><div id="fspara0210" role="paragraph">(F) Demographs of the DAPI signal in untreated, Rif-treated, and Ksg-treated cells. Each demograph represents a concatenation of normalized DAPI signal profiles obtained from individual cells (normalized for cell length), which were sorted by cell length in ascending order from top to bottom. The number of cells (<i>n</i>) used to construct each demograph is shown at the bottom.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/3c1b6e2a-a45d-4c7c-b8db-c8aad58a7690/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.05.037/asset/1d5b4764-967a-4978-bb62-24618276c577/main.assets/figs6.jpg" height="411" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/3c1b6e2a-a45d-4c7c-b8db-c8aad58a7690/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Effect of kasugamycin treatment on the nucleoid compaction and RNA localization, related to <a id="crosref2855" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0315" role="paragraph">A. Similar to <a id="crosref2860" href="#fig7">Figure&nbsp;7</a>A but with a larger field view, showing example micrographs of <i>E.&nbsp;coli</i> cells (CJW7324) untreated or treated with Ksg (5&nbsp;mg/ml) for 40&nbsp;min.</div><div id="fspara0320" role="paragraph">B. Same as <a id="crosref2865" href="#fig7">Figure&nbsp;7</a>E, but for untreated cells. Scatter plot showing the correlation between the NC ratio and the RNASelect fluorescence density for untreated cells (i.e., no antibiotics). The scatter points are colored by the corresponding SCF between the DAPI and RNASelect signals. The correlation coefficient (Spearman’s <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho" tabindex="0" ctxtmenu_counter="48" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="rho"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) is shown in the top right corner.</div><div id="fspara0325" role="paragraph">C. Control experiment showing the NC ratios of untreated cells sampled after incubation of RNASelect for the same amount of time as for cells treated with Ksg (<a id="crosref2870" href="#fig7">Figure&nbsp;7</a>C). The plot shows that the NC ratios do not decrease, indicating that the nucleoid compaction observed in the presence of Ksg (<a id="crosref2875" href="#fig7">Figure&nbsp;7</a>C) is not due to the mere presence of RNASelect in the culture.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">In the drug experiments, we also imaged RNAs using SYTO RNASelect (<a id="crosref1235" href="#fig7">Figures 7</a>A and <a id="crosref1240" href="#figs6">S6</a>A), a fluorogenic RNA-specific dye that has been validated for use in live <i>E.&nbsp;coli</i> cells (<span class="dropBlock reference-citations"><a id="body-ref-sref7-8" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-8" aria-controls="sref7-8">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the untreated cell population, we found that the correlation between the RNASelect and DAPI signals varies across cells (<a id="crosref1250" href="#fig7">Figure&nbsp;7</a>D), as quantified by the signal correlation factor (SCF) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-7" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-7" aria-controls="sref50-7">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As expected (<span class="dropBlock reference-citations"><a id="body-ref-sref7-9" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-9" aria-controls="sref7-9">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), transcription inhibition by Rif led to a decrease in RNASelect signal (<a id="crosref1265" href="#fig7">Figure&nbsp;7</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref7-10" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-10" aria-controls="sref7-10">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), consistent with a reduced level of mRNAs due to their common short lifetime (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Bernstein et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bernstein, J.A. ∙ Khodursky, A.B. ∙ Lin, P.H. ...</div><div><strong>Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:9697-9702</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.112318199" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0037162469" target="_blank" aria-label="Open Scopus in new tab">Scopus (571)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12119387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.112318199&amp;pmid=12119387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The remaining RNASelect signal was positively correlated with the DAPI staining in Rif-treated cells (<a id="crosref1280" href="#fig7">Figure&nbsp;7</a>D). In striking contrast, in Ksg-treated cells where the levels of mRNAs and rRNAs, respectively depleted of ribosomes or ribosomal proteins, are expected to increase, the RNASelect and DAPI signals appeared to segregate from each other (<a id="crosref1285" href="#fig7">Figures 7</a>A and <a id="crosref1290" href="#figs6">S6</a>A). This segregation gave rise to strongly negative SCF values (<a id="crosref1295" href="#fig7">Figure&nbsp;7</a>D). A potential role for RNAs in compacting the nucleoid was also supported at&nbsp;the single-cell level. This was suggested by (1) the negative correlation between the RNASelect signal and the NC ratio across Ksg-treated cells (<a id="crosref1300" href="#fig7">Figure&nbsp;7</a>E) and (2) the trend for cells with smaller NC ratios to have more negative SCF values between the RNASelect and DAPI signals (<a id="crosref1305" href="#fig7">Figure&nbsp;7</a>E). We observed the same trends in untreated cells, albeit to a lesser degree (<a id="crosref1310" href="#figs6">Figure&nbsp;S6</a>B). These results may suggest the higher the RNA level, the greater the segregation between RNAs and DNA and the more compacted the nucleoid.</div><div id="p0170" role="paragraph">We also noted a significant reduction in the fraction of cells with two discernable nucleoids in the Rif- and Ksg-treated populations relative to the untreated condition, as shown in demographs of cell populations (<a id="crosref1315" href="#fig7">Figure&nbsp;7</a>F). For the Rif condition, this was easily explained by the expansion of the two nucleoids into the previously DNA-free internucleoid space. Such a trivial explanation could not apply to the Ksg condition, since Ksg compacts nucleoids over time. A reduced fraction in cells with two discernable nucleoids could also not be explained by a difference in cell size distribution (<a id="crosref1320" href="#fig7">Figure&nbsp;7</a>B). To examine the origin of the phenotype, we performed time-lapse imaging on agarose pads containing Ksg. Nucleoids were visualized using a mCherry fusion to the NAP HU. We found that when cells started with two&nbsp;nucleoids, Ksg treatment often resulted in the nucleoids seemingly moving toward each other (<a id="crosref1325" href="#mmc6">Video S6</a>) and almost coalescing (<a id="crosref1330" href="#mmc7">Video S7</a>). This coalescence likely minimizes the interfacial interaction and promotes the separation between the nucleoid and RNA. Altogether, our results support the hypothesis that RNAs contribute to the effective poor solvent quality of the cytoplasm for the chromosome through an energetically unfavorable (i.e., repulsive) interaction between DNA and RNA.</div><div id="p0175" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIyOGZlNGM1MWY5MTc5YjAwN2ZkY2Q1ZTgwYmNiZGFlNiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.iwlw9_XTo7MZzHsZ9yi1BoHul5jZOzLkRZpom7CFtGJQWsDZ6uBGz5pNC4ieNbqqDrxqTnFHa2VJSXqkeJJF2RxwG3ZExnGzdc3QWl0NyAbBHuEAixdvWsAKJf9BG3TTXzPYB31t4Z3L6bapnU-BgMcim_fJriOygbs761i5ny1NIQ9akM23gGcmKSEWwB3xRzF875vHHP7-m-bCBlCYc_qnShi_tOhhA0X-zLgjNjLpzQi-Vx0D06_oveGMEBeK0rRFUJFdfleVkIj1Vh6rwmQmsHOmCGa8nXaFuzmYVJSBAW5FN9Fdy4bVwpWwMau0sMJHTSUMB6ie1w_ifpwpFA</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIyOGZlNGM1MWY5MTc5YjAwN2ZkY2Q1ZTgwYmNiZGFlNiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.iwlw9_XTo7MZzHsZ9yi1BoHul5jZOzLkRZpom7CFtGJQWsDZ6uBGz5pNC4ieNbqqDrxqTnFHa2VJSXqkeJJF2RxwG3ZExnGzdc3QWl0NyAbBHuEAixdvWsAKJf9BG3TTXzPYB31t4Z3L6bapnU-BgMcim_fJriOygbs761i5ny1NIQ9akM23gGcmKSEWwB3xRzF875vHHP7-m-bCBlCYc_qnShi_tOhhA0X-zLgjNjLpzQi-Vx0D06_oveGMEBeK0rRFUJFdfleVkIj1Vh6rwmQmsHOmCGa8nXaFuzmYVJSBAW5FN9Fdy4bVwpWwMau0sMJHTSUMB6ie1w_ifpwpFA" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 87.8046%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIyOGZlNGM1MWY5MTc5YjAwN2ZkY2Q1ZTgwYmNiZGFlNiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.iwlw9_XTo7MZzHsZ9yi1BoHul5jZOzLkRZpom7CFtGJQWsDZ6uBGz5pNC4ieNbqqDrxqTnFHa2VJSXqkeJJF2RxwG3ZExnGzdc3QWl0NyAbBHuEAixdvWsAKJf9BG3TTXzPYB31t4Z3L6bapnU-BgMcim_fJriOygbs761i5ny1NIQ9akM23gGcmKSEWwB3xRzF875vHHP7-m-bCBlCYc_qnShi_tOhhA0X-zLgjNjLpzQi-Vx0D06_oveGMEBeK0rRFUJFdfleVkIj1Vh6rwmQmsHOmCGa8nXaFuzmYVJSBAW5FN9Fdy4bVwpWwMau0sMJHTSUMB6ie1w_ifpwpFA" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/021805c7-45eb-4757-8f4e-da6e5b3a85cd/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (1003.50 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S6. Nucleoid dynamics in an <i>E.&nbsp;coli</i> cell in the presence of kasugamycin, related to Figure&nbsp;7</div><div id="espara0035" role="paragraph">This video shows a time-lapse sequence of the localization of two nucleoids labeled by the HU-mCherry (yellow signal) in an <i>E.&nbsp;coli</i> CJW7323 cell (phase-contrast signal) spotted on an agarose pad containing Ksg in addition to growth medium. The nucleoids appear to compact and move toward each other over time, resulting a smaller NC ratio.</div></div></div></div><div id="p0180" role="paragraph"><div id="mmc7" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc7" data-type="mp4"><span id="video_stream_mmc7_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkMzM5ZDMyZWU4YjM2OTExNzY0NTk1NjE3OWU4YmZmYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.R1jY2M18dW3766GOnWTlUtfDDUiWu4dZOilWRpGMPJlmM4Y5UHG_gc4Qaxfk3itV_tbsbEtxQgQ4LC823xRE_IwshiB7iauqHJev25sXFXE-dGoai-3drntsAdn9Mp99hwZbgJIP_vMUqkErFEBVM8VgjXuW_NGA_yhbbZIFExNzk11vTkqgyPyYD1vZt22IElOqRusddeRJCKZEroek5rD9AFP8MOpH3WC9_0YW5VrcMrw-2mxI0HVGiG97uCKX2uDDT2y4wM-M78B2UtiIxBi5Fdhii-qW623PA_UGw0UatOOl9rp8no_hVEncDunAkUltoOuVucJjLBERo8Nb6A</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkMzM5ZDMyZWU4YjM2OTExNzY0NTk1NjE3OWU4YmZmYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.R1jY2M18dW3766GOnWTlUtfDDUiWu4dZOilWRpGMPJlmM4Y5UHG_gc4Qaxfk3itV_tbsbEtxQgQ4LC823xRE_IwshiB7iauqHJev25sXFXE-dGoai-3drntsAdn9Mp99hwZbgJIP_vMUqkErFEBVM8VgjXuW_NGA_yhbbZIFExNzk11vTkqgyPyYD1vZt22IElOqRusddeRJCKZEroek5rD9AFP8MOpH3WC9_0YW5VrcMrw-2mxI0HVGiG97uCKX2uDDT2y4wM-M78B2UtiIxBi5Fdhii-qW623PA_UGw0UatOOl9rp8no_hVEncDunAkUltoOuVucJjLBERo8Nb6A" class="cloudflare-stream-player ready tracked" id="video_stream_mmc7" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 149.377%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkMzM5ZDMyZWU4YjM2OTExNzY0NTk1NjE3OWU4YmZmYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjEyMjE3fQ.R1jY2M18dW3766GOnWTlUtfDDUiWu4dZOilWRpGMPJlmM4Y5UHG_gc4Qaxfk3itV_tbsbEtxQgQ4LC823xRE_IwshiB7iauqHJev25sXFXE-dGoai-3drntsAdn9Mp99hwZbgJIP_vMUqkErFEBVM8VgjXuW_NGA_yhbbZIFExNzk11vTkqgyPyYD1vZt22IElOqRusddeRJCKZEroek5rD9AFP8MOpH3WC9_0YW5VrcMrw-2mxI0HVGiG97uCKX2uDDT2y4wM-M78B2UtiIxBi5Fdhii-qW623PA_UGw0UatOOl9rp8no_hVEncDunAkUltoOuVucJjLBERo8Nb6A" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.05.037/attachment/6ff28389-7b4b-435b-85db-12ef95f9fb40/mmc7.mp4" download="mmc7.mp4" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Video (897.10 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Video S7. Nucleoid dynamics in an <i>E.&nbsp;coli</i> cell in the presence of kasugamycin, related to Figure&nbsp;7</div><div id="espara0040" role="paragraph">Similar to Video S6, this video shows a time-lapse sequence of the localization of two nucleoids labeled by the HU-mCherry (yellow signal) in an <i>E.&nbsp;coli</i> CJW7323 cell (phase-contrast signal) spotted on an agarose pad containing Ksg in addition to growth medium. The nucleoids appear to compact and move toward each other, coalescing in time.</div></div></div></div></section></section><section id="sec-3"><h2>Discussion</h2><section id="sec-3-1"><h3>What may contribute to the effective poor solvent quality of the cytoplasm?</h3><div id="p0185" role="paragraph">Several factors are likely to contribute to the effective poor solvent quality of the cytoplasm. In our simplified view of the cell, the solvent is everything in the cytoplasm apart from the DNA polymer. In this context, NAPs are part of the solvent and therefore may be one of the factors due to their high abundance and well-known interactions with the chromosome. The potential role of NAPs may be direct or indirect. Some NAPs have been shown to bridge DNA segments or generate loops (<span class="dropBlock reference-citations"><a id="body-ref-sref37-4" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-4" aria-controls="sref37-4">Dame et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Dame, R.T. ∙ Rashid, F.M. ∙ Grainger, D.C.</div><div><strong>Chromosome organization in bacteria: mechanistic insights into genome structure and function</strong></div><div><em>Nat. Rev. Genet.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31767998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31767998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Dillon and Dorman, 2010</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Dillon, S.C. ∙ Dorman, C.J.</div><div><strong>Bacterial nucleoid-associated proteins, nucleoid structure and gene expression</strong></div><div><em>Nat. Rev. Microbiol.</em> 2010; <strong>8</strong>:185-195</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro2261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-76949089832" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20140026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro2261&amp;pmid=20140026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which may be macroscopically described as increasing polymer interaction. In addition, we speculate that NAPs, and potentially other DNA-binding proteins (such as transcriptional regulators), may act as an interface that indirectly modulates the chemical interaction between the chromosome and other components of the cytoplasm. Protein binding to the anionic DNA polymer could, for instance, locally alter the electrostatic potential of the chromosome. The effects may alter the compaction and mesh size of the nucleoid. It is important to note that NAP binding may also reduce the DNA Kuhn length, for instance, by bending, twisting or screening the DNA (<span class="dropBlock reference-citations"><a id="body-ref-sref37-5" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-5" aria-controls="sref37-5">Dame et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Dame, R.T. ∙ Rashid, F.M. ∙ Grainger, D.C.</div><div><strong>Chromosome organization in bacteria: mechanistic insights into genome structure and function</strong></div><div><em>Nat. Rev. Genet.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31767998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31767998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44-3" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-3" aria-controls="sref44-3">Dillon and Dorman, 2010</a><div class="dropBlock__holder" data-db-target-of="sref44-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Dillon, S.C. ∙ Dorman, C.J.</div><div><strong>Bacterial nucleoid-associated proteins, nucleoid structure and gene expression</strong></div><div><em>Nat. Rev. Microbiol.</em> 2010; <strong>8</strong>:185-195</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro2261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-76949089832" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20140026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro2261&amp;pmid=20140026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In other instances, some NAPs may increase the DNA Kuhn length depending on their (local) concentration. For example, <i>in&nbsp;vitro</i> studies have shown that at low concentration, HU bends the DNA, locally increasing its flexibility (<span class="dropBlock reference-citations"><a id="body-ref-sref129" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129" aria-controls="sref129">Skoko et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref129"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Skoko, D. ∙ Wong, B. ∙ Johnson, R.C. ...</div><div><strong>Micromechanical analysis of the binding of DNA-bending proteins HMGB1, NHP6A, and HU reveals their ability to form highly stable DNA-protein complexes</strong></div><div><em>Biochemistry.</em> 2004; <strong>43</strong>:13867-13874</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi048428o" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-7244245362" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15504049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi048428o&amp;pmid=15504049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref144" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144" aria-controls="sref144">van Noort et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref144"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>van Noort, J. ∙ Verbrugge, S. ∙ Goosen, N. ...</div><div><strong>Dual architectural roles of HU: formation of flexible hinges and rigid filaments</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:6969-6974</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0308230101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2342454974" target="_blank" aria-label="Open Scopus in new tab">Scopus (233)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15118104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0308230101&amp;pmid=15118104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, at high concentration, HU stiffens the DNA by coating it and forming a rigid filament.</div><div id="p0190" role="paragraph">The high ionic strength of the cytoplasm (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Alatossava et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alatossava, T. ∙ Jütte, H. ∙ Kuhn, A. ...</div><div><strong>Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.1.413-419.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2984182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.1.413-419.1985&amp;pmid=2984182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">Cayley et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cayley, S. ∙ Lewis, B.A. ∙ Guttman, H.J. ...</div><div><strong>Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1991; <strong>222</strong>:281-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(91)90212-O" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026409841" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1960728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2891%2990212-O&amp;pmid=1960728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Kuhn and Kellenberger, 1985</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Kuhn, A. ∙ Kellenberger, E.</div><div><strong>Productive phage infection in Escherichia coli with reduced internal levels of the major cations</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>163</strong>:906-912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.163.3.906-912.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3161872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.163.3.906-912.1985&amp;pmid=3161872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Lusk et&nbsp;al., 1968</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Lusk, J.E. ∙ Williams, R.J.P. ∙ Kennedy, E.P.</div><div><strong>Magnesium and the growth of Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1968; <strong>243</strong>:2618-2624</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4968384/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2993417-4&amp;pmid=4968384" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Moncany and Kellenberger, 1981</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Moncany, M.L.J. ∙ Kellenberger, E.</div><div><strong>High magnesium content of Escherichia coli B</strong></div><div><em>Experientia.</em> 1981; <strong>37</strong>:846-847</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF01985672" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0019401975" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7026272/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF01985672&amp;pmid=7026272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref118-1" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-1" aria-controls="sref118-1">Roe et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref118-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Roe, A.J. ∙ McLaggan, D. ∙ Davidson, I. ...</div><div><strong>Perturbation of anion balance during inhibition of growth of Escherichia coli by weak acids</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1998; <strong>180</strong>:767-772</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.180.4.767-772.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9473028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.180.4.767-772.1998&amp;pmid=9473028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref124-1" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-1" aria-controls="sref124-1">Schultz et&nbsp;al., 1962</a><div class="dropBlock__holder" data-db-target-of="sref124-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Schultz, S.G. ∙ Wilson, N.L. ∙ Epstein, W.</div><div><strong>Cation transport in Escherichia coli. II. Intracellular chloride concentration</strong></div><div><em>J.&nbsp;Gen. Physiol.</em> 1962; <strong>46</strong>:159-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1085/jgp.46.1.159" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13909522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1085%2Fjgp.46.1.159&amp;pmid=13909522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) may also contribute to the poor solvent effects of the cytoplasm as well as the Kuhn length of the DNA. <i>In&nbsp;vitro</i> studies have shown that the net interaction between DNA fragments becomes less repulsive with increasing salt concentrations (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Nicolai and Mandel, 1989</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Nicolai, T. ∙ Mandel, M.</div><div><strong>The ionic strength dependence of the second virial coefficient of low molar mass DNA fragments in aqueous solutions</strong></div><div><em>Macromolecules.</em> 1989; <strong>22</strong>:438-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma00191a080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma00191a080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Divalent Mg<sup>2+</sup> has been found to not only screen charges on DNA but also induce attraction between DNA segments (<span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Qiu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Qiu, X. ∙ Andresen, K. ∙ Kwok, L.W. ...</div><div><strong>Inter-DNA attraction mediated by divalent counterions</strong></div><div><em>Phys. Rev. Lett.</em> 2007; <strong>99</strong>:038104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.99.038104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34547432557" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17678334/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.99.038104&amp;pmid=17678334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0195" role="paragraph">Another important factor to consider is the crowded nature of the cytoplasm. Indeed, molecular dynamics simulations suggest that macromolecular crowding can be viewed as effectively reducing the quality of the solvent (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Jeon et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Jeon, C. ∙ Jung, Y. ∙ Ha, B.-Y.</div><div><strong>Effects of molecular crowding and confinement on the spatial organization of a biopolymer</strong></div><div><em>Soft Matter.</em> 2016; <strong>12</strong>:9436-9450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C6SM01184E" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27834427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC6SM01184E&amp;pmid=27834427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Polyribosomes or proteins are generally evoked as the principal cytoplasmic crowders that compact the nucleoid (<span class="dropBlock reference-citations"><a id="body-ref-sref96-3" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-3" aria-controls="sref96-3">Mondal et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref96-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Mondal, J. ∙ Bratton, B.P. ∙ Li, Y. ...</div><div><strong>Entropy-based mechanism of ribosome-nucleoid segregation in E.&nbsp;coli cells</strong></div><div><em>Biophys. J.</em> 2011; <strong>100</strong>:2605-2613</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79960325276" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21641305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2011.04.030&amp;pmid=21641305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Odijk, 1998</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Odijk, T.</div><div><strong>Osmotic compaction of supercoiled DNA into a bacterial nucleoid</strong></div><div><em>Biophys. Chem.</em> 1998; <strong>73</strong>:23-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0301-4622(98)00115-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0032514107" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9697298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0301-4622%2898%2900115-X&amp;pmid=9697298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Pelletier et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Pelletier, J. ∙ Halvorsen, K. ∙ Ha, B.Y. ...</div><div><strong>Physical manipulation of the Escherichia coli chromosome reveals its soft nature</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:E2649-E2656</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1208689109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84867042373" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22984156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1208689109&amp;pmid=22984156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Both of them are plausible crowders because polyribosomes are large in size compared to proteins, but proteins are far more abundant (<span class="dropBlock reference-citations"><a id="body-ref-sref69-3" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-3" aria-controls="sref69-3">Kim et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref69-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Kim, J. ∙ Jeon, C. ∙ Jeong, H. ...</div><div><strong>A polymer in a crowded and confined space: effects of crowder size and poly-dispersity</strong></div><div><em>Soft Matter.</em> 2015; <strong>11</strong>:1877-1888</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C4SM02198C" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535704/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC4SM02198C&amp;pmid=25535704" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref164-4" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164-4" aria-controls="sref164-4">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref164-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Yang, D. ∙ Männik, J. ∙ Retterer, S.T. ...</div><div><strong>The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid</strong></div><div><em>Mol. Microbiol.</em> 2020; <strong>113</strong>:1022-1037</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85078903920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31961016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14467&amp;pmid=31961016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Both size and abundance are important determinants of depletion forces associated with excluded volume effects (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Asakura and Oosawa, 1954</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Asakura, S. ∙ Oosawa, F.</div><div><strong>On Interaction between Two Bodies Immersed in a Solution of Macromolecules</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1954; <strong>22</strong>:1255-1256</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1740347" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1740347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While macromolecular crowding is widely believed to be play a major, if not the predominant, role in chromosome compaction, it is worth noting that the interpretations of the experimental data supporting this view are not always clear cut. As mentioned earlier, the well-known opposite effects of Rif and Cpl treatment on nucleoid size may be due to a change in not only polyribosome abundance but also RNA levels. According to the crowding hypothesis, Ksg treatment, which reduces polyribosome abundance (<span class="dropBlock reference-citations"><a id="body-ref-sref7-11" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-11" aria-controls="sref7-11">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), should result in nucleoid expansion. Instead, Ksg treatment leads to nucleoid compaction over time (<a id="crosref1450" href="#fig7">Figures 7</a>A, 7C, and <a id="crosref1455" href="#figs6">S6</a>A; <a id="crosref1460" href="#mmc6">Videos S6</a> and <a id="crosref1465" href="#mmc7">S7</a>). As for proteins, their proposed role as crowders is primarily based on <i>in&nbsp;vitro</i> experiments showing that flexible uncharged polymers such as poly(ethylene glycol) (PEG) condense isolated nucleoids (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Cunha et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Cunha, S. ∙ Woldringh, C.L. ∙ Odijk, T.</div><div><strong>Polymer-mediated compaction and internal dynamics of isolated Escherichia coli nucleoids</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2001; <strong>136</strong>:53-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.2001.4420" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0035782861" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11858707/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.2001.4420&amp;pmid=11858707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-3" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-3" aria-controls="sref105-3">Pelletier et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref105-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Pelletier, J. ∙ Halvorsen, K. ∙ Ha, B.Y. ...</div><div><strong>Physical manipulation of the Escherichia coli chromosome reveals its soft nature</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:E2649-E2656</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1208689109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84867042373" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22984156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1208689109&amp;pmid=22984156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This DNA condensation occurs at molecular weights and concentrations (packing fractions) of PEG that are biologically relevant for proteins. However, PEG is not chemically inert and has been shown to mediate non-steric interactions (<span class="dropBlock reference-citations"><a id="body-ref-sref128" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128" aria-controls="sref128">Sheth and Leckband, 1997</a><div class="dropBlock__holder" data-db-target-of="sref128"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Sheth, S.R. ∙ Leckband, D.</div><div><strong>Measurements of attractive forces between proteins and end-grafted poly(ethylene glycol) chains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:8399-8404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.16.8399" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030831255" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9237988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.16.8399&amp;pmid=9237988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, flexible polymers like PEG are thought to be more efficient at condensing DNA than globular proteins due to differences in translational entropy and interaction surface (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Castelnovo and Gelbart, 2004</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Castelnovo, M. ∙ Gelbart, W.M.</div><div><strong>Semiflexible Chain Condensation by Neutral Depleting Agents: Role of Correlations between Depletants</strong></div><div><em>Macromolecules.</em> 2004; <strong>37</strong>:3510-3517</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma0354795" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2442678013" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma0354795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">de Vries, 2006</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>de Vries, R.</div><div><strong>Depletion-induced instability in protein-DNA mixtures: Influence of protein charge and size</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2006; <strong>125</strong>:014905</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.2209683" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33745802542" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16863331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.2209683&amp;pmid=16863331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To be clear, we do not dispute a role for polyribosomes or proteins in nucleoid compaction through excluded volume effects. We merely suggest that non-steric interactions and other cytoplasmic components may also be involved.</div><div id="p0200" role="paragraph">For instance, RNAs could play such a role. As mentioned above, global inhibition of RNA synthesis by Rif treatment results in nucleoid expansion (<a id="crosref1495" href="#fig7">Figures 7</a>A, 7C, and <a id="crosref1500" href="#figs6">S6</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref6-6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-6" aria-controls="sref6-6">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref7-12" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-12" aria-controls="sref7-12">2014</a><div class="dropBlock__holder" data-db-target-of="sref7-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21-3" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-3" aria-controls="sref21-3">Cabrera et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref21-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cabrera, J.E. ∙ Cagliero, C. ∙ Quan, S. ...</div><div><strong>Active transcription of rRNA operons condenses the nucleoid in Escherichia coli: examining the effect of transcription on nucleoid structure in the absence of transertion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:4180-4185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01707-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67649394401" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19395497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01707-08&amp;pmid=19395497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Cabrera and Jin, 2003</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Cabrera, J.E. ∙ Jin, D.J.</div><div><strong>The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues</strong></div><div><em>Mol. Microbiol.</em> 2003; <strong>50</strong>:1493-1505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2958.2003.03805.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0346887121" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14651633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2958.2003.03805.x&amp;pmid=14651633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Dworsky and Schaechter, 1973</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Dworsky, P. ∙ Schaechter, M.</div><div><strong>Effect of rifampin on the structure and membrane attachment of the nucleoid of Escherichia coli</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1973; <strong>116</strong>:1364-1374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.116.3.1364-1374.1973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4584813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.116.3.1364-1374.1973&amp;pmid=4584813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Pettijohn and Hecht, 1974</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Pettijohn, D.E. ∙ Hecht, R.</div><div><strong>RNA molecules bound to the folded bacterial genome stabilize DNA folds and segregate domains of supercoiling</strong></div><div><em>Cold Spring Harb. Symp. Quant. Biol.</em> 1974; <strong>38</strong>:31-41</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/SQB.1974.038.01.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0016000696" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4598638/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2FSQB.1974.038.01.006&amp;pmid=4598638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref138-2" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-2" aria-controls="sref138-2">Sun and Margolin, 2004</a><div class="dropBlock__holder" data-db-target-of="sref138-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Sun, Q. ∙ Margolin, W.</div><div><strong>Effects of perturbing nucleoid structure on nucleoid occlusion-mediated toporegulation of FtsZ ring assembly</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2004; <strong>186</strong>:3951-3959</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.186.12.3951-3959.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942547429" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15175309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.186.12.3951-3959.2004&amp;pmid=15175309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref164-5" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164-5" aria-controls="sref164-5">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref164-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Yang, D. ∙ Männik, J. ∙ Retterer, S.T. ...</div><div><strong>The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid</strong></div><div><em>Mol. Microbiol.</em> 2020; <strong>113</strong>:1022-1037</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85078903920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31961016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14467&amp;pmid=31961016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, nucleoid decondensation also occurs when rRNA synthesis is inhibited through induction of the stringent response (<span class="dropBlock reference-citations"><a id="body-ref-sref20-3" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-3" aria-controls="sref20-3">Cabrera and Jin, 2003</a><div class="dropBlock__holder" data-db-target-of="sref20-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Cabrera, J.E. ∙ Jin, D.J.</div><div><strong>The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues</strong></div><div><em>Mol. Microbiol.</em> 2003; <strong>50</strong>:1493-1505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2958.2003.03805.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0346887121" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14651633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2958.2003.03805.x&amp;pmid=14651633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conversely, inhibition of translation elongation by Cpl is associated with nucleoid compaction (<span class="dropBlock reference-citations"><a id="body-ref-sref6-7" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-7" aria-controls="sref6-7">Bakshi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref6-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref7-13" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-13" aria-controls="sref7-13">2014</a><div class="dropBlock__holder" data-db-target-of="sref7-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21-4" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-4" aria-controls="sref21-4">Cabrera et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref21-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Cabrera, J.E. ∙ Cagliero, C. ∙ Quan, S. ...</div><div><strong>Active transcription of rRNA operons condenses the nucleoid in Escherichia coli: examining the effect of transcription on nucleoid structure in the absence of transertion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:4180-4185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01707-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67649394401" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19395497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01707-08&amp;pmid=19395497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-2" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-2" aria-controls="sref143-2">van Helvoort et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref143-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>van Helvoort, J.M. ∙ Kool, J. ∙ Woldringh, C.L.</div><div><strong>Chloramphenicol causes fusion of separated nucleoids in Escherichia coli K-12 cells and filaments</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1996; <strong>178</strong>:4289-4293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.178.14.4289-4293.1996" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029978680" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8763959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.178.14.4289-4293.1996&amp;pmid=8763959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref174-2" href="#" role="doc-biblioref" data-xml-rid="bib174" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref174-2" aria-controls="sref174-2">Zimmerman, 2002</a><div class="dropBlock__holder" data-db-target-of="sref174-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Zimmerman, S.B.</div><div><strong>Toroidal nucleoids in Escherichia coli exposed to chloramphenicol</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2002; <strong>138</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1047-8477(02)00036-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036420081" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12217658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1047-8477%2802%2900036-9&amp;pmid=12217658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This effect may be linked to an increase in RNA levels, since Clp treatment results in stabilization of mRNAs (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Lopez et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Lopez, P.J. ∙ Marchand, I. ∙ Yarchuk, O. ...</div><div><strong>Translation inhibitors stabilize Escherichia coli mRNAs independently of ribosome protection</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1998; <strong>95</strong>:6067-6072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.95.11.6067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0032568544" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9600918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.95.11.6067&amp;pmid=9600918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">Pato et&nbsp;al., 1973</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Pato, M.L. ∙ Bennett, P.M. ∙ von Meyenburg, K.</div><div><strong>Messenger ribonucleic acid synthesis and degradation in <i>Escherichia coli</i> during inhibition of translation</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1973; <strong>116</strong>:710-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.116.2.710-718.1973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4583248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.116.2.710-718.1973&amp;pmid=4583248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref123-2" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-2" aria-controls="sref123-2">Schneider et&nbsp;al., 1978</a><div class="dropBlock__holder" data-db-target-of="sref123-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Schneider, E. ∙ Blundell, M. ∙ Kennell, D.</div><div><strong>Translation and mRNA decay</strong></div><div><em>Mol. Gen. Genet.</em> 1978; <strong>160</strong>:121-129</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00267473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/349350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00267473&amp;pmid=349350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, there is probably an accumulation of “free” rRNAs (i.e., depleted of ribosomal proteins), as rRNA synthesis persists in the absence of protein production (<span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Cole et&nbsp;al., 1987</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Cole, J.R. ∙ Olsson, C.L. ∙ Hershey, J.W. ...</div><div><strong>Feedback regulation of rRNA synthesis in Escherichia coli. Requirement for initiation factor IF2</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1987; <strong>198</strong>:383-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(87)90288-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2448483/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2887%2990288-9&amp;pmid=2448483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Kurland and Maaloe, 1962</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Kurland, C.G. ∙ Maaloe, O.</div><div><strong>Regulation of ribosomal and transfer RNA synthesis</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1962; <strong>4</strong>:193-210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(62)80051-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14460744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2862%2980051-5&amp;pmid=14460744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, inhibition of translation initiation by Ksg, which increases the concentration of ribosome-depleted mRNAs, compacts the nucleoid (<a id="crosref1600" href="#fig7">Figures 7</a>A, 7C, and <a id="crosref1605" href="#figs6">S6</a>A). Moreover, as with any form of translation inhibition (<span class="dropBlock reference-citations"><a id="body-ref-sref31-3" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-3" aria-controls="sref31-3">Cole et&nbsp;al., 1987</a><div class="dropBlock__holder" data-db-target-of="sref31-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Cole, J.R. ∙ Olsson, C.L. ∙ Hershey, J.W. ...</div><div><strong>Feedback regulation of rRNA synthesis in Escherichia coli. Requirement for initiation factor IF2</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1987; <strong>198</strong>:383-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(87)90288-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2448483/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2887%2990288-9&amp;pmid=2448483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Hernandez and Bremer, 1990</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Hernandez, V.J. ∙ Bremer, H.</div><div><strong>Guanosine tetraphosphate (ppGpp) dependence of the growth rate control of rrnB P1 promoter activity in Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1990; <strong>265</strong>:11605-11614</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2819%2938441-8&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2819%2938441-8&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2114400/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2938441-8&amp;pmid=2114400" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-3" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-3" aria-controls="sref73-3">Kurland and Maaloe, 1962</a><div class="dropBlock__holder" data-db-target-of="sref73-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Kurland, C.G. ∙ Maaloe, O.</div><div><strong>Regulation of ribosomal and transfer RNA synthesis</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1962; <strong>4</strong>:193-210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(62)80051-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14460744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2862%2980051-5&amp;pmid=14460744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Ksg treatment is expected to cause an accumulation of free rRNAs, and this is associated with compaction and coalescence of separated nucleoids (<a id="crosref1625" href="#fig7">Figure&nbsp;7</a>F; <a id="crosref1630" href="#mmc7">Video S7</a>). Interestingly, nucleoid fusion was also reported for Cpl-treated cells over six decades ago (<span class="dropBlock reference-citations"><a id="body-ref-sref121" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121" aria-controls="sref121">Schaechter and Laing, 1961</a><div class="dropBlock__holder" data-db-target-of="sref121"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Schaechter, M. ∙ Laing, V.O.</div><div><strong>Direct observation of fusion of bacterial nuclei</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1961; <strong>81</strong>:667-668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.81.4.667-668.1961" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0001515687" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13747158/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.81.4.667-668.1961&amp;pmid=13747158" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143-3" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143-3" aria-controls="sref143-3">van Helvoort et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref143-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>van Helvoort, J.M. ∙ Kool, J. ∙ Woldringh, C.L.</div><div><strong>Chloramphenicol causes fusion of separated nucleoids in Escherichia coli K-12 cells and filaments</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1996; <strong>178</strong>:4289-4293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.178.14.4289-4293.1996" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029978680" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8763959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.178.14.4289-4293.1996&amp;pmid=8763959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, in both untreated and Ksg-treated cells, a high RNA signal level is correlated with greater RNA/DNA segregation and nucleoid compaction (<a id="crosref1645" href="#fig7">Figures 7</a>E and <a id="crosref1650" href="#figs6">S6</a>B).</div><div id="p0205" role="paragraph">Other observations are consistent with RNAs contributing to the effective poor solvent quality of the cytoplasm. For instance, gene loci have been shown to move toward the nucleoid periphery during transcription (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Libby et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Libby, E.A. ∙ Roggiani, M. ∙ Goulian, M.</div><div><strong>Membrane protein expression triggers chromosomal locus repositioning in bacteria</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:7445-7450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1109479109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84860828333" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22529375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1109479109&amp;pmid=22529375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref163-1" href="#" role="doc-biblioref" data-xml-rid="bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref163-1" aria-controls="sref163-1">Yang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref163-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">162.</div><div>Yang, S. ∙ Kim, S. ∙ Kim, D.K. ...</div><div><strong>Transcription and translation contribute to gene locus relocation to the nucleoid periphery in E.&nbsp;coli</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:5131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-13152-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85074899720" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31719538/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-13152-y&amp;pmid=31719538" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This has led to a model in which actively transcribed gene loci with their associated nascent mRNAs segregate away from the nucleoid bulk. Gene relocation is driven by transcription, though translation enhances it (<span class="dropBlock reference-citations"><a id="body-ref-sref163-2" href="#" role="doc-biblioref" data-xml-rid="bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref163-2" aria-controls="sref163-2">Yang et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref163-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">162.</div><div>Yang, S. ∙ Kim, S. ∙ Kim, D.K. ...</div><div><strong>Transcription and translation contribute to gene locus relocation to the nucleoid periphery in E.&nbsp;coli</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:5131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-13152-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85074899720" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31719538/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-13152-y&amp;pmid=31719538" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These observations support the idea that the interaction between RNA and DNA is energetically unfavorable. A tendency for DNA/RNA to demix may also explain why nucleoids appear more structured by fluorescence microscopy when cells are grown under nutrient-rich conditions (e.g., see <span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Hadizadeh Yazdi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Hadizadeh Yazdi, N. ∙ Guet, C.C. ∙ Johnson, R.C. ...</div><div><strong>Variation of the folding and dynamics of the Escherichia coli chromosome with growth conditions</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>86</strong>:1318-1333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84871006430" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23078205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12071&amp;pmid=23078205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Under these fast-growth conditions, a large fraction of transcription is dedicated to rRNA synthesis at a few chromosomal locations. We speculate that this results in stronger compacting forces at sites of rRNA transcription relative to other chromosomal locations, producing unevenly shaped nucleoids. Comparatively, under slow-growth conditions, rRNA synthesis is reduced and transcription activities are more homogeneously distributed across the chromosome, potentially contributing to the smoother, more uniform nucleoid shape observed under these conditions.</div><div id="p0210" role="paragraph">Among all of the potential factors involved, we find transcription particularly interesting for several reasons: (1) transcription is dictated by the sequence of the DNA, meaning that the polymer itself may be able to affect the quality of its own solvent; (2) transcription is tunable, which may contribute to differences in chromosome folding across environmental conditions and bacterial species; and (3) both DNA and RNA are strong polyanions. Thus, in addition of repelling each other for steric reasons, their segregation may involve electrostatic interactions, in a phenomenon called “segregative phase separation” (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Joyeux, 2016</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Joyeux, M.</div><div><strong>In&nbsp;vivo compaction dynamics of bacterial DNA: A fingerprint of DNA/RNA demixing?</strong></div><div><em>Curr. Opin. Colloid Interface Sci.</em> 2016; <strong>26</strong>:17-27</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.cocis.2016.08.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84988693705" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cocis.2016.08.005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the cytoplasmic environment, where the concentration of cations is high (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Alatossava et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alatossava, T. ∙ Jütte, H. ∙ Kuhn, A. ...</div><div><strong>Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.1.413-419.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2984182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.1.413-419.1985&amp;pmid=2984182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-3" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-3" aria-controls="sref26-3">Cayley et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref26-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cayley, S. ∙ Lewis, B.A. ∙ Guttman, H.J. ...</div><div><strong>Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1991; <strong>222</strong>:281-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(91)90212-O" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026409841" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1960728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2891%2990212-O&amp;pmid=1960728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Kuhn and Kellenberger, 1985</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Kuhn, A. ∙ Kellenberger, E.</div><div><strong>Productive phage infection in Escherichia coli with reduced internal levels of the major cations</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>163</strong>:906-912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.163.3.906-912.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3161872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.163.3.906-912.1985&amp;pmid=3161872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-3" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-3" aria-controls="sref85-3">Lusk et&nbsp;al., 1968</a><div class="dropBlock__holder" data-db-target-of="sref85-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Lusk, J.E. ∙ Williams, R.J.P. ∙ Kennedy, E.P.</div><div><strong>Magnesium and the growth of Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1968; <strong>243</strong>:2618-2624</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4968384/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2993417-4&amp;pmid=4968384" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Moncany and Kellenberger, 1981</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Moncany, M.L.J. ∙ Kellenberger, E.</div><div><strong>High magnesium content of Escherichia coli B</strong></div><div><em>Experientia.</em> 1981; <strong>37</strong>:846-847</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF01985672" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0019401975" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7026272/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF01985672&amp;pmid=7026272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref118-2" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-2" aria-controls="sref118-2">Roe et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref118-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Roe, A.J. ∙ McLaggan, D. ∙ Davidson, I. ...</div><div><strong>Perturbation of anion balance during inhibition of growth of Escherichia coli by weak acids</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1998; <strong>180</strong>:767-772</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.180.4.767-772.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9473028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.180.4.767-772.1998&amp;pmid=9473028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref124-2" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-2" aria-controls="sref124-2">Schultz et&nbsp;al., 1962</a><div class="dropBlock__holder" data-db-target-of="sref124-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Schultz, S.G. ∙ Wilson, N.L. ∙ Epstein, W.</div><div><strong>Cation transport in Escherichia coli. II. Intracellular chloride concentration</strong></div><div><em>J.&nbsp;Gen. Physiol.</em> 1962; <strong>46</strong>:159-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1085/jgp.46.1.159" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13909522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1085%2Fjgp.46.1.159&amp;pmid=13909522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), most negative charges on nucleic acids are screened. Therefore, any electrostatic repulsion would have to occur at short range (much like steric interactions). The tethering of RNAs to the DNA via RNA polymerases during transcription may facilitate short-range electrostatic interaction by keeping DNA and RNAs in proximity. It is also possible that other forms of non-steric interaction are involved. We hope that our work will motivate future studies that will test these hypotheses.</div></section><section id="sec-3-2"><h3>Transcription and chromosome domain organization</h3><div id="p0215" role="paragraph">The bacterial chromosome is not simply compacted into cells; it is instead partitioned into various domains (CIDs) in which DNA loci preferentially interact with each other (<span class="dropBlock reference-citations"><a id="body-ref-sref75-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-3" aria-controls="sref75-3">Le et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref75-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Le, T.B. ∙ Imakaev, M.V. ∙ Mirny, L.A. ...</div><div><strong>High-resolution mapping of the spatial organization of a bacterial chromosome</strong></div><div><em>Science.</em> 2013; <strong>342</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1242059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84887322004" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24158908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1242059&amp;pmid=24158908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-3" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-3" aria-controls="sref81-3">Lioy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref81-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Lioy, V.S. ∙ Cournac, A. ∙ Marbouty, M. ...</div><div><strong>Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:771-783.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85041112985" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.027&amp;pmid=29358050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-3" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-3" aria-controls="sref88-3">Marbouty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref88-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Marbouty, M. ∙ Cournac, A. ∙ Flot, J.F. ...</div><div><strong>Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e03318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.03318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.03318&amp;pmid=25517076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-4" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-4" aria-controls="sref89-4">Marbouty et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref89-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref142-3" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-3" aria-controls="sref142-3">Val et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref142-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Val, M.E. ∙ Marbouty, M. ∙ de Lemos Martins, F. ...</div><div><strong>A checkpoint control orchestrates the replication of the two chromosomes of Vibrio cholerae</strong></div><div><em>Sci. Adv.</em> 2016; <strong>2</strong>:e1501914</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1501914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84991009833" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27152358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1501914&amp;pmid=27152358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref153-3" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-3" aria-controls="sref153-3">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref153-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Wang, X. ∙ Brandão, H.B. ∙ Le, T.B. ...</div><div><strong>Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus</strong></div><div><em>Science.</em> 2017; <strong>355</strong>:524-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai8982" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85011362856" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28154080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai8982&amp;pmid=28154080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref151-3" href="#" role="doc-biblioref" data-xml-rid="bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151-3" aria-controls="sref151-3">Wang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref151-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Wang, X. ∙ Le, T.B. ∙ Lajoie, B.R. ...</div><div><strong>Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis</strong></div><div><em>Genes Dev.</em> 2015; <strong>29</strong>:1661-1675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.265876.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26253537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.265876.115&amp;pmid=26253537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our simulations suggest that the effective poor solvent quality of the cytoplasm may promote the spontaneous formation of chromosomal domains (<a id="crosref1750" href="#fig3">Figure&nbsp;3</a>; <a id="crosref1755" href="#mmc2">Video S2</a>).</div><div id="p0220" role="paragraph">In these simulations, the DNA has no sequence information. Therefore, domains can form anywhere along the DNA polymer, as shown by randomly averaging 10 different simulated chromosomes (<a id="crosref1760" href="#figs7">Figure&nbsp;S7</a>A). As a result of this stochasticity, boundaries between domains blurred with each other and became undefined when 100 distance maps of simulated chromosomes were averaged (<a id="crosref1765" href="#figs7">Figure&nbsp;S7</a>B). Note that the presence of domains remained apparent in the poor solvent relative to the good or ideal solvent, as shown by the thicker signal along the diagonal of the averaged map. However, the blurring of the domain boundaries in the 100-chromosome averaged map is in contrast with Hi-C contact maps in which boundaries of CIDs remain visible despite averaging over billions of cells. In light of our findings, we propose that domain boundaries may result, at least in part, from energetically unfavorable interactions between DNA and RNAs, particularly at sites of highly expressed genes where transcripts are abundant. Consistent with this hypothesis, high transcriptional activity is an important determinant of CID boundaries (<span class="dropBlock reference-citations"><a id="body-ref-sref75-4" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-4" aria-controls="sref75-4">Le et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref75-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Le, T.B. ∙ Imakaev, M.V. ∙ Mirny, L.A. ...</div><div><strong>High-resolution mapping of the spatial organization of a bacterial chromosome</strong></div><div><em>Science.</em> 2013; <strong>342</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1242059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84887322004" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24158908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1242059&amp;pmid=24158908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Le and Laub, 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Le, T.B. ∙ Laub, M.T.</div><div><strong>Transcription rate and transcript length drive formation of chromosomal interaction domain boundaries</strong></div><div><em>EMBO J.</em> 2016; <strong>35</strong>:1582-1595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201593561" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84978298560" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27288403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201593561&amp;pmid=27288403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-4" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-4" aria-controls="sref81-4">Lioy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref81-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Lioy, V.S. ∙ Cournac, A. ∙ Marbouty, M. ...</div><div><strong>Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:771-783.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85041112985" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.027&amp;pmid=29358050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-5" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-5" aria-controls="sref89-5">Marbouty et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref89-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In our simulations, the domains and their boundaries correspond to DNA regions of high and low densities, respectively. This also appears to be true inside cells, as the DNA at CID boundaries is less condensed (more extensible) than the DNA within CIDs (<span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Le and Laub, 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Le, T.B. ∙ Laub, M.T.</div><div><strong>Transcription rate and transcript length drive formation of chromosomal interaction domain boundaries</strong></div><div><em>EMBO J.</em> 2016; <strong>35</strong>:1582-1595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201593561" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84978298560" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27288403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201593561&amp;pmid=27288403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At the domain boundary, high transcriptional activity leads to an accumulation of nascent RNAs. Since these RNAs are still bound to the DNA via RNA polymerases, they cannot diffuse away. We speculate that in order to minimize the contact with these bound RNAs, nearby DNA strands retract to the flanking sides, reducing their local density at the boundary while forming denser DNA regions (domains) on both sides of the boundary (<a id="crosref1795" href="#figs7">Figure&nbsp;S7</a>C). This would be mainly driven by the unfavorable interaction between DNA and RNAs and not by the steric blocking of ribosomes, as high transcription drives the local establishment of domain boundaries independently of translation (<span class="dropBlock reference-citations"><a id="body-ref-sref74-3" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-3" aria-controls="sref74-3">Le and Laub, 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Le, T.B. ∙ Laub, M.T.</div><div><strong>Transcription rate and transcript length drive formation of chromosomal interaction domain boundaries</strong></div><div><em>EMBO J.</em> 2016; <strong>35</strong>:1582-1595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201593561" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84978298560" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27288403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201593561&amp;pmid=27288403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.05.037/asset/cf6800cb-ea3b-423d-be07-31e86c320a2f/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.05.037/asset/e02b7a19-b959-4fc0-8d22-32cf7cf93b75/main.assets/figs7.jpg" height="760" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cf6800cb-ea3b-423d-be07-31e86c320a2f/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Distance maps for simulated <i>E.&nbsp;coli</i> chromosomes, related to <a id="crosref2880" href="#fig3">Figures 3</a> and <a id="crosref2885" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0335" role="paragraph">A. Three examples of distance maps indicating the spatial separation between any two loci along simulated chromosomes in a poor solvent (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="49" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.36). Each map is the result of averaging 10 randomly selected simulated chromosomes.</div><div id="fspara0340" role="paragraph">B. Distance maps obtained by averaging 100 simulated chromosomes in a poor (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="50" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.36), ideal (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="51" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.50) and good solvent (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="52" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.58) conditions. Same as <a id="crosref2890" href="#fig3">Figure&nbsp;3</a>D, except that 100 simulated chromosomes were averaged instead of 10.</div><div id="fspara0345" role="paragraph">C. Cartoon showing a hypothetical repulsive interaction between DNA and RNA at the site of a highly expressed gene. In this working model, the local accumulation of RNA is proposed to cause a segregation between DNA-rich and RNA-rich regions (dotted lines), leading to the formation of a chromosomal domain boundary. To minimize the DNA/RNA interaction, a segment of DNA of length <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L" tabindex="0" ctxtmenu_counter="53" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper L"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, which likely depends on the length of the transcript, would relax to minimize the local DNA density. The degree of relaxation, modeled by <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="theta" tabindex="0" ctxtmenu_counter="54" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="theta"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> (see <a id="crosref2895" href="#sec-4">STAR Methods</a>), may depend on multiple factors, such as the extent of the unfavorable interaction and the rigidity of the DNA and RNA polymers.</div><div id="fspara0350" role="paragraph">D. Average distance maps calculated based on 1,000 <i>E.&nbsp;coli</i> chromosomes simulated in a poor solvent condition (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="55" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 0.36). DNA relaxation of different lengths (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L" tabindex="0" ctxtmenu_counter="56" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper L"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) and degrees <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis theta right parenthesis" tabindex="0" ctxtmenu_counter="57" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis theta right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> were simulated for selected regions centered at 1-, 2-, 3- and 4-Mb coordinates (see <a id="crosref2900" href="#sec-4">STAR Methods</a>). The larger relaxation length and the higher relaxation degree resulted in sharper boundaries.</div></div></div></div></figcaption></figure></div><div id="p0225" role="paragraph">To model the reduced local DNA density at a boundary, we performed additional simulations in which we varied the length <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper L right parenthesis" tabindex="0" ctxtmenu_counter="58" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis upper L right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and degree <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis theta right parenthesis" tabindex="0" ctxtmenu_counter="59" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis theta right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of DNA relaxation at regions arbitrarily centered at the 1-, 2-, 3- and 4-Mb coordinates (<a id="crosref1805" href="#sec-4">STAR Methods</a>). Results based on averaging 1,000 simulated chromosomes showed that domain boundaries gradually appear and become increasingly sharper as the selected regions become longer (larger <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L" tabindex="0" ctxtmenu_counter="60" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper L"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) and more relaxed (lower <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="theta" tabindex="0" ctxtmenu_counter="61" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="theta"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) (<a id="crosref1810" href="#figs7">Figure&nbsp;S7</a>D). These results corroborate the findings that both the rate of transcription and the length of the transcripts determine the formation of chromosomal domain boundaries (<span class="dropBlock reference-citations"><a id="body-ref-sref74-4" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-4" aria-controls="sref74-4">Le and Laub, 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Le, T.B. ∙ Laub, M.T.</div><div><strong>Transcription rate and transcript length drive formation of chromosomal interaction domain boundaries</strong></div><div><em>EMBO J.</em> 2016; <strong>35</strong>:1582-1595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201593561" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84978298560" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27288403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201593561&amp;pmid=27288403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Besides transcription, other factors, such as NAP binding and membrane attachment, also contribute to the predominance of specific domain boundaries across cells (<span class="dropBlock reference-citations"><a id="body-ref-sref81-5" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-5" aria-controls="sref81-5">Lioy et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref81-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Lioy, V.S. ∙ Cournac, A. ∙ Marbouty, M. ...</div><div><strong>Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:771-783.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85041112985" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.027&amp;pmid=29358050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-4" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-4" aria-controls="sref88-4">Marbouty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref88-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Marbouty, M. ∙ Cournac, A. ∙ Flot, J.F. ...</div><div><strong>Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e03318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.03318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.03318&amp;pmid=25517076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-3-3"><h3>Chromosome-dependent intracellular organization and dynamics</h3><div id="p0230" role="paragraph">Bacteria lack nuclear membranes. As a result, the chromosome effectively functions as a mesh-like physical barrier that can affect the diffusion and spatial distribution of cytoplasmic components. Therefore, the mesh size of the nucleoid is an important length scale to consider. By assessing the degree of nucleoid exclusion for fluorescent probes of varying sizes (25–150&nbsp;nm), we approximate the apparent average mesh size of the nucleoid to be around 50&nbsp;nm in exponentially growing <i>E.&nbsp;coli</i> cells (<a id="crosref1830" href="#fig2">Figure&nbsp;2</a>). This length scale has several physiological implications. For instance, it predicts that the diffusion and spatial distribution of cytoplasmic components close or larger than this estimate are impacted by the presence of the nucleoid in the cytoplasm. We anticipate that some (possibly most) polyribosome species belong to this category. This reasoning is consistent with the single-molecule studies showing that ribosomes, which are mostly found as polyribosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Dai, X. ∙ Zhu, M. ∙ Warren, M. ...</div><div><strong>Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>2</strong>:16231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85005950833" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27941827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.231&amp;pmid=27941827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Forchhammer and Lindahl, 1971</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Forchhammer, J. ∙ Lindahl, L.</div><div><strong>Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of Escherichia coli 15</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:563-568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90337-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0015222649" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4927947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990337-8&amp;pmid=4927947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107-2" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-2" aria-controls="sref107-2">Phillips et&nbsp;al., 1969</a><div class="dropBlock__holder" data-db-target-of="sref107-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Phillips, L.A. ∙ Hotham-Iglewski, B. ∙ Franklin, R.M.</div><div><strong>Polyribosomes of Escherichia coli. II. Experiments to determine the in&nbsp;vivo distribution of polysomes, ribosomes and ribosomal subunits</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1969; <strong>45</strong>:23-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(69)90207-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4186447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2869%2990207-1&amp;pmid=4186447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref145-2" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-2" aria-controls="sref145-2">Varricchio and Monier, 1971</a><div class="dropBlock__holder" data-db-target-of="sref145-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Varricchio, F. ∙ Monier, R.</div><div><strong>Ribosome patterns in Escherichia coli growing at various rates</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1971; <strong>108</strong>:105-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.108.1.105-110.1971" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5001192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.108.1.105-110.1971&amp;pmid=5001192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), display timescale-dependent subdiffusive dynamics (<span class="dropBlock reference-citations"><a id="body-ref-sref50-8" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-8" aria-controls="sref50-8">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as expected for objects that experience caging and uncaging events (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Brangwynne et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Brangwynne, C.P. ∙ Koenderink, G.H. ∙ MacKintosh, F.C. ...</div><div><strong>Intracellular transport by active diffusion</strong></div><div><em>Trends Cell Biol.</em> 2009; <strong>19</strong>:423-427</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.tcb.2009.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.tcb.2009.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-69549118397" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19699642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tcb.2009.04.004&amp;pmid=19699642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Cai et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Cai, L.H. ∙ Panyukov, S. ∙ Rubinstein, M.</div><div><strong>Mobility of Nonsticky Nanoparticles in Polymer Liquids</strong></div><div><em>Macromolecules.</em> 2011; <strong>44</strong>:7853-7863</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma201583q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-80053467043" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22058573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma201583q&amp;pmid=22058573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Guo et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Guo, M. ∙ Ehrlicher, A.J. ∙ Jensen, M.H. ...</div><div><strong>Probing the stochastic, motor-driven properties of the cytoplasm using force spectrum microscopy</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:822-832</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.06.051&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.06.051&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84907369302" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25126787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.06.051&amp;pmid=25126787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref141" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141" aria-controls="sref141">Tseng et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref141"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">140.</div><div>Tseng, Y. ∙ Lee, J.S. ∙ Kole, T.P. ...</div><div><strong>Micro-organization and visco-elasticity of the interphase nucleus revealed by particle nanotracking</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2004; <strong>117</strong>:2159-2167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.01073" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942516225" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15090601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.01073&amp;pmid=15090601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref161-3" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref161-3" aria-controls="sref161-3">Wong et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref161-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Wong, I.Y. ∙ Gardel, M.L. ∙ Reichman, D.R. ...</div><div><strong>Anomalous diffusion probes microstructure dynamics of entangled F-actin networks</strong></div><div><em>Phys. Rev. Lett.</em> 2004; <strong>92</strong>:178101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.92.178101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942551253" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15169197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.92.178101&amp;pmid=15169197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Once polyribosomes escape the nucleoid, they are less likely to get back inside the nucleoid due to their large size. Using a cryo-ET classification method based on the distances and orientations of ribosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Bharat and Scheres, 2016</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bharat, T.A. ∙ Scheres, S.H.</div><div><strong>Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:2054-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.124" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84992573769" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27685097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.124&amp;pmid=27685097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99-3" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-3" aria-controls="sref99-3">O’Reilly et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref99-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>O’Reilly, F.J. ∙ Xue, L. ∙ Graziadei, A. ...</div><div><strong>In-cell architecture of an actively transcribing-translating expressome</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:554-557</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb3758" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85088883448" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32732422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb3758&amp;pmid=32732422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1895" href="#figs5">Figure&nbsp;S5</a>C), we indeed found that ribosomes classified as polyribosomes are considerably enriched at the cell periphery (<a id="crosref1900" href="#figs5">Figure&nbsp;S5</a>D). This is consistent with analyses from single-molecule tracking studies (<span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Mohapatra and Weisshaar, 2018</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Mohapatra, S. ∙ Weisshaar, J.C.</div><div><strong>Functional mapping of the E.&nbsp;coli translational machinery using single-molecule tracking</strong></div><div><em>Mol. Microbiol.</em> 2018; <strong>110</strong>:262-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85054160957" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30107639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14103&amp;pmid=30107639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120-3" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-3" aria-controls="sref120-3">Sanamrad et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref120-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Sanamrad, A. ∙ Persson, F. ∙ Lundius, E.G. ...</div><div><strong>Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11413-11418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1411558111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84905669647" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1411558111&amp;pmid=25056965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We expect that the motion and therefore spatial distribution of large cellular components, such as storage granules, plasmids, microcompartments, and stress-induced protein aggregates, are also impacted by the nucleoid, for which there is evidence (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Chowdhury et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Chowdhury, C. ∙ Sinha, S. ∙ Chun, S. ...</div><div><strong>Diverse bacterial microcompartment organelles</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 2014; <strong>78</strong>:438-468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MMBR.00009-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84907216686" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25184561/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMMBR.00009-14&amp;pmid=25184561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Henry and Crosson, 2013</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Henry, J.T. ∙ Crosson, S.</div><div><strong>Chromosome replication and segregation govern the biogenesis and inheritance of inorganic polyphosphate granules</strong></div><div><em>Mol. Biol. Cell.</em> 2013; <strong>24</strong>:3177-3186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E13-04-0182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84885642867" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23985321/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E13-04-0182&amp;pmid=23985321" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Racki et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Racki, L.R. ∙ Tocheva, E.I. ∙ Dieterle, M.G. ...</div><div><strong>Polyphosphate granule biogenesis is temporally and functionally tied to cell cycle exit during starvation in <i>Pseudomonas aeruginosa</i></strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:E2440-E2449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1615575114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85016092632" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28265086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1615575114&amp;pmid=28265086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Reyes-Lamothe et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Reyes-Lamothe, R. ∙ Tran, T. ∙ Meas, D. ...</div><div><strong>High-copy bacterial plasmids diffuse in the nucleoid-free space, replicate stochastically and are randomly partitioned at cell division</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:1042-1051</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84893258556" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24137005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt918&amp;pmid=24137005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref152" href="#" role="doc-biblioref" data-xml-rid="bib152" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref152" aria-controls="sref152">Wang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref152"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">151.</div><div>Wang, Y. ∙ Penkul, P. ∙ Milstein, J.N.</div><div><strong>Quantitative Localization Microscopy Reveals a Novel Organization of a High-Copy Number Plasmid</strong></div><div><em>Biophys. J.</em> 2016; <strong>111</strong>:467-479</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2016.06.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2016.06.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84982994278" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27508432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2016.06.033&amp;pmid=27508432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref158" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158" aria-controls="sref158">Winkler et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref158"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">157.</div><div>Winkler, J. ∙ Seybert, A. ∙ König, L. ...</div><div><strong>Quantitative and spatio-temporal features of protein aggregation in Escherichia coli and consequences on protein quality control and cellular ageing</strong></div><div><em>EMBO J.</em> 2010; <strong>29</strong>:910-923</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/emboj.2009.412" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-77649293067" target="_blank" aria-label="Open Scopus in new tab">Scopus (186)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20094032/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Femboj.2009.412&amp;pmid=20094032" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0235" role="paragraph">Conversely, the nucleoid allows cytoplasmic components of sizes well below 50&nbsp;nm (e.g., metabolites, proteins, and most protein complexes) to diffuse across the nucleoid, facilitating biochemical reactions. Since the sizes of free ribosomal subunits are ∼23–26&nbsp;nm (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Boublik, 1985</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Boublik, M.</div><div><strong>Structure of the Escherichia coli Ribosome</strong></div><div><div>Nanninga, N. (Editor)</div></div><div><strong>Molecular cytology of Escherichia coli</strong></div><div>Academic Press, 1985</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BoublikStructure+of+the+Escherichia+coli+RibosomeN.NanningaMolecular+cytology+of+Escherichia+coli1985Academic+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref147" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147" aria-controls="sref147">Verschoor et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref147"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Verschoor, A. ∙ Frank, J. ∙ Boublik, M.</div><div><strong>Investigation of the 50&nbsp;S ribosomal subunit by electron microscopy and image analysis</strong></div><div><em>J.&nbsp;Ultrastruct. Res.</em> 1985; <strong>92</strong>:180-189</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0889-1605(85)90045-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0022381381" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3913778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0889-1605%2885%2990045-X&amp;pmid=3913778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref171" href="#" role="doc-biblioref" data-xml-rid="bib171" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref171" aria-controls="sref171">Zhu et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref171"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">170.</div><div>Zhu, J. ∙ Penczek, P.A. ∙ Schröder, R. ...</div><div><strong>Three-dimensional reconstruction with contrast transfer function correction from energy-filtered cryoelectron micrographs: procedure and application to the 70S Escherichia coli ribosome</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 1997; <strong>118</strong>:197-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.1997.3845" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030960710" target="_blank" aria-label="Open Scopus in new tab">Scopus (132)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9169230/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.1997.3845&amp;pmid=9169230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the nucleoid mesh enables the coupling between transcription and translation by allowing free ribosomal subunits to reach nascent mRNAs within the nucleoid through diffusion, as previously suggested from ensemble single-molecule tracking experiments (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Mohapatra and Weisshaar, 2018</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Mohapatra, S. ∙ Weisshaar, J.C.</div><div><strong>Functional mapping of the E.&nbsp;coli translational machinery using single-molecule tracking</strong></div><div><em>Mol. Microbiol.</em> 2018; <strong>110</strong>:262-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85054160957" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30107639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14103&amp;pmid=30107639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120-4" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-4" aria-controls="sref120-4">Sanamrad et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref120-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Sanamrad, A. ∙ Persson, F. ∙ Lundius, E.G. ...</div><div><strong>Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11413-11418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1411558111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84905669647" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1411558111&amp;pmid=25056965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0240" role="paragraph">Note that our mesh size estimation reflects an average obtained from ensemble measurements. DNA density is known to be spatially heterogeneous within individual nucleoids (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Le Gall et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Le Gall, A. ∙ Cattoni, D.I. ∙ Guilhas, B. ...</div><div><strong>Bacterial partition complexes segregate within the volume of the nucleoid</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:12107</div><div>12107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms12107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977271741" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27377966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms12107&amp;pmid=27377966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-6" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-6" aria-controls="sref89-6">Marbouty et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref89-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref133-2" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-2" aria-controls="sref133-2">Spahn et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref133-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Spahn, C. ∙ Endesfelder, U. ∙ Heilemann, M.</div><div><strong>Super-resolution imaging of Escherichia coli nucleoids reveals highly structured and asymmetric segregation during fast growth</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2014; <strong>185</strong>:243-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2014.01.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84894410667" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24473063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2014.01.007&amp;pmid=24473063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref134-2" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" aria-label="Reference 2018" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-2" aria-controls="sref134-2">2018</a><div class="dropBlock__holder" data-db-target-of="sref134-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Spahn, C.K. ∙ Glaesmann, M. ∙ Grimm, J.B. ...</div><div><strong>A toolbox for multiplexed super-resolution imaging of the E.&nbsp;coli nucleoid and membrane using novel PAINT labels</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:14768</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-33052-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85054290920" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30282984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-33052-3&amp;pmid=30282984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref137-2" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-2" aria-controls="sref137-2">Stracy et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref137-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Stracy, M. ∙ Lesterlin, C. ∙ Garza de Leon, F. ...</div><div><strong>Live-cell superresolution microscopy reveals the organization of RNA polymerase in the bacterial nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4390-E4399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507592112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26224838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507592112&amp;pmid=26224838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), consistent with our poor solvent simulations (<a id="crosref1995" href="#fig3">Figure&nbsp;3</a>). We therefore expect the nucleoid to have a spectrum of mesh sizes with an average around 50&nbsp;nm. We speculate that regions with smaller mesh sizes (i.e., more compacted DNA) represent regions that are less active in gene expression, while regions with larger mesh sizes (i.e., less compacted DNA) are more active. This is consistent with the negative correlation between ribosome and DNA densities at the nucleoid level (<a id="crosref2000" href="#fig5">Figure&nbsp;5</a>).</div></section><section id="sec-3-4"><h3>Limitations of the study</h3><div id="p0245" role="paragraph">This study only used <i>E.&nbsp;coli</i> cells in exponential phase. Our method should provide a mean to examine different conditions (e.g., stationary phase and antibiotic exposure) in various bacteria, as long as probes of varying size (e.g., GFP-μNS) can be expressed. Our simulation approach offers high computational efficiency and effectively reproduces the scaling of polymer size at different length scales (<a id="crosref2005" href="#figs3">Figure&nbsp;S3</a>; <a id="crosref2010" href="#sec-4">STAR Methods</a>) and the DNA density heterogeneity across the nucleoid (<a id="crosref2015" href="#fig3">Figure&nbsp;3</a>). However, it does not explicitly model the molecular forces among the monomers or interaction between distant monomers, which will be important to do in future work. In addition, we inferred the effective solvent quality of the cytoplasm for the chromosome by considering that the chromosome is a random coil polymer and the solvent is everything else in the cytoplasm. This simplification provides a macroscopic description of the cytoplasm that interconnect various aspects of cell and chromosome biology, from chromosome compaction (<a id="crosref2020" href="#fig6">Figure&nbsp;6</a>) and transcription (<a id="crosref2025" href="#fig7">Figure&nbsp;7</a>) to chromosome domain organization (<a id="crosref2030" href="#fig3">Figure&nbsp;3</a>) and intracellular organization (<a id="crosref2035" href="#fig2">Figures 2</a>, <a id="crosref2040" href="#fig4">4</a>, and <a id="crosref2045" href="#fig5">5</a>). In reality, cells are far more complex than simple polymer solutions. The chromosome does not behave as simply as a random coiled polymer (<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Hadizadeh Yazdi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Hadizadeh Yazdi, N. ∙ Guet, C.C. ∙ Johnson, R.C. ...</div><div><strong>Variation of the folding and dynamics of the Escherichia coli chromosome with growth conditions</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>86</strong>:1318-1333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84871006430" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23078205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12071&amp;pmid=23078205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref155" href="#" role="doc-biblioref" data-xml-rid="bib155" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref155" aria-controls="sref155">Wiggins et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref155"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">154.</div><div>Wiggins, P.A. ∙ Cheveralls, K.C. ∙ Martin, J.S. ...</div><div><strong>Strong intranucleoid interactions organize the Escherichia coli chromosome into a nucleoid filament</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2010; <strong>107</strong>:4991-4995</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0912062107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-77950455981" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20194778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0912062107&amp;pmid=20194778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), in part due to enzymatic activity leading to DNA supercoiling (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Deng et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Deng, S. ∙ Stein, R.A. ∙ Higgins, N.P.</div><div><strong>Organization of supercoil domains and their reorganization by transcription</strong></div><div><em>Mol. Microbiol.</em> 2005; <strong>57</strong>:1511-1521</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2005.04796.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-25144495578" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16135220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2005.04796.x&amp;pmid=16135220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref111" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111" aria-controls="sref111">Postow et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref111"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Postow, L. ∙ Hardy, C.D. ∙ Arsuaga, J. ...</div><div><strong>Topological domain structure of the Escherichia coli chromosome</strong></div><div><em>Genes Dev.</em> 2004; <strong>18</strong>:1766-1779</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1207504" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-3142774839" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15256503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1207504&amp;pmid=15256503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As mentioned above, the proposed poor solvent effects likely originate from interspersed and superimposed molecular interactions associated with the complex and active nature of the cytoplasm. Further experimental and theoretical work will be required to disentangle the contribution of each factor.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0255" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 (F-lambda- <i>ilvG</i>- <i>rfb</i>-50 <i>rph</i>-1)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Jensen, 1993</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Jensen, K.F.</div><div><strong>The Escherichia coli K-12 “wild types” W3110 and MG1655 have an rph frameshift mutation that leads to pyrimidine starvation due to low pyrE expression levels</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1993; <strong>175</strong>:3401-3407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.175.11.3401-3407.1993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0027251266" target="_blank" aria-label="Open Scopus in new tab">Scopus (377)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8501045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.175.11.3401-3407.1993&amp;pmid=8501045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>MG1655</td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 Δ<i>lacZYA</i>::<i>gfp-μNS</i></td><td><span class="dropBlock reference-citations"><a id="body-ref-sref102-4" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-4" aria-controls="sref102-4">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CJW4617</td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 <i>seqA</i>::<i>seqA</i>-<i>mcherry</i>-<i>frt</i>-<i>cat</i>-<i>frt</i></td><td>This work</td><td>CJW6320</td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 <i>seqA</i>::<i>seqA-mcherry ftsZ</i>::<i>ftsZ</i>-<i>venus</i><sup><i>SW</i></sup></td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-9" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-9" aria-controls="sref50-9">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CJW6324</td></tr><tr><td><i>E.&nbsp;coli</i> BL21 Star (DE3)/pET29b+ I3-01-sfgfp</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977663424" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27309817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18010&amp;pmid=27309817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CJW6340</td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 <i>rplA</i>::<i>rplA</i>-<i>msfgfp</i></td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-10" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-10" aria-controls="sref50-10">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CJW7020</td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 <i>rplA</i>::<i>rplA</i>-<i>msfgfp seqA</i>::<i>seqA</i>-<i>mcherry</i></td><td>This work</td><td>CJW7039</td></tr><tr><td><i>E.&nbsp;coli</i> MG1655 <i>hupA</i>::<i>hupA</i>-<i>mcherry rplA</i>::<i>rplA</i>-<i>gfp</i></td><td>William Gray and Yashna Thappeta</td><td>CJW7323</td></tr><tr class="rowsep"><td><i>E.&nbsp;coli</i> MG1655 <i>rplA</i>::<i>rplA</i>-<i>mcherry</i></td><td>Alexandros Papagiannakis and Yashna Thappeta</td><td>CJW7324</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Agarose</td><td>AmericanBio</td><td>Cat#AB00972-00500</td></tr><tr><td>4’,6-Diamidine-2′-phenylindole dihydrochloride (DAPI) fluorescent dye</td><td>Thermo Fisher Scientific</td><td>Cat#D1306</td></tr><tr><td>SYTO RNASelect Green Fluorescent cell Stain - 5&nbsp;mM Solution in DMSO</td><td>Invitrogen</td><td>Cat#S32703</td></tr><tr><td>FluoSpheres Carboxylate-Modified Microspheres, 0.1&nbsp;μm, yellow-green fluorescent (505/515), 2% solids</td><td>Invitrogen</td><td>Cat#F8803</td></tr><tr><td>Isopropyl β-D-1-thiogalactopyranoside (IPTG)</td><td>Sigma Aldrich</td><td>Cat#I5502</td></tr><tr><td>Rifampicin</td><td>Sigma Aldrich</td><td>Cat#R3501</td></tr><tr class="rowsep"><td>Kasugamycin hydrochloride from <i>Streptomyces kasugaensis</i></td><td>Sigma Aldrich</td><td>Cat#K4013</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td><i>E.&nbsp;coli</i> tomogram from cryo-ET experiments</td><td>Electron Microscopy Public Image Archive</td><td>EMPIAR-10589</td></tr><tr><td>Average ribosome structure</td><td>Electron Microscopy Data Bank</td><td>EMD-22877</td></tr><tr class="rowsep"><td>Average polyribosome structure</td><td>Electron Microscopy Data Bank</td><td>EMD-22878</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Oufti</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref101-1" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-1" aria-controls="sref101-1">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0010" href="https://oufti.org" target="_blank">https://oufti.org</a></td></tr><tr><td>SpotFinder</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref131-1" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-1" aria-controls="sref131-1">Sliusarenko et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref131-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Sliusarenko, O. ∙ Heinritz, J. ∙ Emonet, T. ...</div><div><strong>High-throughput, subpixel precision analysis of bacterial morphogenesis and intracellular spatio-temporal dynamics</strong></div><div><em>Mol. Microbiol.</em> 2011; <strong>80</strong>:612-627</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2011.07579.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955024764" target="_blank" aria-label="Open Scopus in new tab">Scopus (340)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21414037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2011.07579.x&amp;pmid=21414037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>MATLAB</td><td>MathWorks</td><td><a id="intref0015" href="https://www.mathworks.com" target="_blank">https://www.mathworks.com</a></td></tr><tr><td>Python</td><td>Python Software Foundation</td><td><a id="intref0020" href="https://www.python.org" target="_blank">https://www.python.org</a></td></tr><tr><td>Numpy</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Harris et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2649-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85091129687" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32939066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2649-2&amp;pmid=32939066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0025" href="https://numpy.org" target="_blank">https://numpy.org</a></td></tr><tr><td>Scipy</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref148-1" href="#" role="doc-biblioref" data-xml-rid="bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148-1" aria-controls="sref148-1">Virtanen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref148-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Virtanen, P. ∙ Gommers, R. ∙ Oliphant, T.E. ..., SciPy 1.0 Contributors</div><div><strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:261-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0686-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85079119586" target="_blank" aria-label="Open Scopus in new tab">Scopus (3027)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32015543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0686-2&amp;pmid=32015543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://scipy.org" target="_blank">https://scipy.org</a></td></tr><tr><td>Matplotlib</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Hunter, 2007</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics Environment</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/MCSE.2007.55" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34247493236" target="_blank" aria-label="Open Scopus in new tab">Scopus (9418)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FMCSE.2007.55" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://matplotlib.org" target="_blank">https://matplotlib.org</a></td></tr><tr><td>pandas</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">McKinney, 2010</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>McKinney, W.</div><div><strong>Data Structures for Statistical Computing in Python</strong></div><div><em>Proceedings of the 9th Python in Science Conference.</em> 2010;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.25080/Majora-92bf1922-00a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.25080%2FMajora-92bf1922-00a" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://pandas.pydata.org" target="_blank">https://pandas.pydata.org</a></td></tr><tr><td>RELION</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref12-3" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-3" aria-controls="sref12-3">Bharat and Scheres, 2016</a><div class="dropBlock__holder" data-db-target-of="sref12-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bharat, T.A. ∙ Scheres, S.H.</div><div><strong>Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:2054-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.124" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84992573769" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27685097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.124&amp;pmid=27685097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="https://www3.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page" target="_blank">https://www3.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page</a></td></tr><tr><td>UCSF ChimeraX</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Goddard et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Goddard, T.D. ∙ Huang, C.C. ∙ Meng, E.C. ...</div><div><strong>UCSF ChimeraX: Meeting modern challenges in visualization and analysis</strong></div><div><em>Protein Sci.</em> 2018; <strong>27</strong>:14-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3235" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85039039173" target="_blank" aria-label="Open Scopus in new tab">Scopus (840)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28710774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3235&amp;pmid=28710774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="https://www.cgl.ucsf.edu/chimera" target="_blank">https://www.cgl.ucsf.edu/chimera</a></td></tr><tr><td>emClarity</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Himes and Zhang, 2018</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Himes, B.A. ∙ Zhang, P.</div><div><strong>emClarity: software for high-resolution cryo-electron tomography and subtomogram averaging</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:955-961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0167-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85055493329" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30349041/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0167-z&amp;pmid=30349041" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://github.com/bHimes/emClarity/wiki" target="_blank">https://github.com/bHimes/emClarity/wiki</a></td></tr><tr><td>MotionCor2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref170-1" href="#" role="doc-biblioref" data-xml-rid="bib170" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref170-1" aria-controls="sref170-1">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref170-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">169.</div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85014129582" target="_blank" aria-label="Open Scopus in new tab">Scopus (1982)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4193&amp;pmid=28250466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>IMOD</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Kremer et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Kremer, J.R. ∙ Mastronarde, D.N. ∙ McIntosh, J.R.</div><div><strong>Computer visualization of three-dimensional image data using IMOD</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 1996; <strong>116</strong>:71-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.1996.0013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029879295" target="_blank" aria-label="Open Scopus in new tab">Scopus (3066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8742726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.1996.0013&amp;pmid=8742726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://emcore.ucsf.edu/ucsf-software" target="_blank">https://emcore.ucsf.edu/ucsf-software</a></td></tr><tr><td>Gctf</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref167-1" href="#" role="doc-biblioref" data-xml-rid="bib167" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref167-1" aria-controls="sref167-1">Zhang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref167-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2015.11.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84955216953" target="_blank" aria-label="Open Scopus in new tab">Scopus (1297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2015.11.003&amp;pmid=26592709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/zhang-software/" target="_blank">https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/zhang-software/</a></td></tr><tr class="rowsep"><td>i3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref156-1" href="#" role="doc-biblioref" data-xml-rid="bib156" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref156-1" aria-controls="sref156-1">Winkler, 2007</a><div class="dropBlock__holder" data-db-target-of="sref156-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div><div>Winkler, H.</div><div><strong>3D reconstruction and processing of volumetric data in cryo-electron tomography</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2007; <strong>157</strong>:126-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2006.07.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33845310367" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16973379/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2006.07.014&amp;pmid=16973379" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref157-1" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157-1" aria-controls="sref157-1">Winkler et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref157-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Winkler, H. ∙ Zhu, P. ∙ Liu, J. ...</div><div><strong>Tomographic subvolume alignment and subvolume classification applied to myosin V and SIV envelope spikes</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2009; <strong>165</strong>:64-77</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2008.10.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-58349085125" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19032983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2008.10.004&amp;pmid=19032983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0070" href="http://www.electrontomography.org" target="_blank">http://www.electrontomography.org</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>QUANTIFOIL® holey carbon films</td><td>Electron Microscopy Sciences</td><td>Cat#Q2100CR1</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900689-9" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0260" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Christine Jacobs-Wagner (<a id="intref0075" href="mailto:jacobs-wagner@stanford.edu">jacobs-wagner@stanford.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0265" role="paragraph">Reagents generated in this study are available upon request.</div></section><section id="sec-4-2-3"><h4>Data and code availability</h4><div id="p0270" role="paragraph">All tomogram data obtained from the cryo-ET experiments are available at the Electron Microscopy Public Image Archive (EMPIAR-10589). The corresponding average structures of ribosomes and polyribosomes were deposited in the Electron Microscopy Data Bank (EMD-22877 and EMD-22878). Code used for analysis in this study can be found at <a id="intref0080" href="https://github.com/JacobsWagnerLab" target="_blank">https://github.com/JacobsWagnerLab</a>.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Strains</h4><div id="p0275" role="paragraph">This study used the following published strains. MG1655 (F-lambda- <i>ilvG</i>- <i>rfb</i>-50 <i>rph</i>-1) was used as a wild-type <i>E.&nbsp;coli</i> strain (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Jensen, 1993</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Jensen, K.F.</div><div><strong>The Escherichia coli K-12 “wild types” W3110 and MG1655 have an rph frameshift mutation that leads to pyrimidine starvation due to low pyrE expression levels</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1993; <strong>175</strong>:3401-3407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.175.11.3401-3407.1993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0027251266" target="_blank" aria-label="Open Scopus in new tab">Scopus (377)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8501045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.175.11.3401-3407.1993&amp;pmid=8501045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Strain CJW6324 (MG1655 <i>seqA</i>::<i>seqA-mcherry ftsZ</i>::<i>ftsZ</i>-<i>venus</i><sup><i>SW</i></sup>) was used to separate the <i>E.&nbsp;coli</i> cell population into three distinct groups that corresponded to the cells in the B, C and D periods (<span class="dropBlock reference-citations"><a id="body-ref-sref50-11" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-11" aria-controls="sref50-11">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Strain CJW6340 (BL21 Star (DE3) pET29b+ I3-01-sfgfp) was obtained as a kind gift from David Baker’s laboratory at the University of Washington (<span class="dropBlock reference-citations"><a id="body-ref-sref59-3" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-3" aria-controls="sref59-3">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref59-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977663424" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27309817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18010&amp;pmid=27309817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this strain, the artificially designed protein I3-01 fused to superfolder GFP (I3-01(ctGFP)) was produced from the pET29b+ plasmid in BL21 Star (DE3) <i>E.&nbsp;coli</i> cells. I3-01(ctGFP) assembles into dodecahedron nanocage particles (<span class="dropBlock reference-citations"><a id="body-ref-sref59-4" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-4" aria-controls="sref59-4">Hsia et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref59-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977663424" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27309817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18010&amp;pmid=27309817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Strain CJW4617 (MG1655 Δ<i>lacZYA</i>::<i>gfp-μNS</i>) produces GFP-μNS particles of tunable sizes in response to IPTG induction (<span class="dropBlock reference-citations"><a id="body-ref-sref102-5" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-5" aria-controls="sref102-5">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Strain CJW7020 (MG1655 <i>rplA</i>::<i>rplA</i>-<i>msfgfp</i>) was used to investigate the pixel intensity correlation between DNA and ribosome fluorescence signals (<span class="dropBlock reference-citations"><a id="body-ref-sref50-12" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-12" aria-controls="sref50-12">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0280" role="paragraph">Strain CJW7324 (MG1655 <i>rplA</i>::<i>rplA</i>-<i>mcherry</i>), which was a kind gift from Dr. Alexandros Papagiannakis and Yashna Thappeta, was used to assess the impact of antibiotic treatments (Rif and Ksg) on the compaction of the nucleoid and the spatial pattern of the RNASelect signal.</div><div id="p0285" role="paragraph">Strain CJW7323 (MG1655 <i>hupA</i>::<i>hupA</i>-<i>mcherry rplA</i>::<i>rplA</i>-<i>gfp</i>) was used in the time lapse experiment to visualize the nucleoid dynamics in cells on Ksg-containing agarose pads. The <i>hupA</i>-<i>mcherry</i>-<i>frt</i>-<i>kanR</i>-<i>frt</i> cassette was P1-transduced from CJW5158 (BW25113 <i>hupA</i>::<i>hupA</i>-<i>mcherry</i>-<i>frt</i>-<i>kanR</i>-<i>frt</i>) into CJW4677 (MG1655 <i>rplA</i>::<i>rplA</i>-<i>gfp</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-13" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-13" aria-controls="sref50-13">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), yielding strain CJW5159 (MG1655 <i>hupA</i>::<i>hupA</i>-<i>mcherry</i>-<i>frt</i>-<i>kanR</i>-<i>frt rplA</i>::<i>rplA</i>-<i>gfp</i>). Strain CJW5159 was then transiently equipped with plasmid pCP20 expressing the Flp site-specific recombinase (<span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Cherepanov and Wackernagel, 1995</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Cherepanov, P.P. ∙ Wackernagel, W.</div><div><strong>Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant</strong></div><div><em>Gene.</em> 1995; <strong>158</strong>:9-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0378-1119(95)00193-A" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029065955" target="_blank" aria-label="Open Scopus in new tab">Scopus (1285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7789817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0378-1119%2895%2900193-A&amp;pmid=7789817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which excised the kanamycin resistance cassette, yielding strain CJW7323.</div><div id="p0290" role="paragraph">Strain CJW6320 (MG1655 <i>seqA</i>::<i>seqA-mcherry-frt-cat-frt</i>) was constructed using <i>E.&nbsp;coli</i> K12 MG1655 as a parental strain. PCR amplification with oligonucleotides SG5 and SG6 (<span class="dropBlock reference-citations"><a id="body-ref-sref50-14" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-14" aria-controls="sref50-14">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was performed using pKD3-mCherry (<span class="dropBlock reference-citations"><a id="body-ref-sref50-15" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-15" aria-controls="sref50-15">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as a template to generate an <i>mCherry</i>-FRT-<i>cat</i>-FRT amplicon flanked by short (50&nbsp;bp) nucleotide sequences homologous to their target regions (50 nt on both sides of the <i>seqA</i> stop codon). This amplicon was used to replace the stop codon of the native <i>seqA</i> gene, yielding a chromosomally encoded <i>seqA-mcherry</i> fusion.</div><div id="p0295" role="paragraph">Strain CJW7039 (MG1655 <i>rplA</i>::<i>rplA</i>-<i>msfgfp seqA</i>::<i>seqA</i>-<i>mcherry</i>) was used to compare the SCF between DAPI and RplA-msfGFP signals among untreated <i>E.&nbsp;coli</i> cells in the B, C, D periods. Using P1 lysate prepared from CJW6320 (MG1655 <i>seqA</i>::<i>seqA</i>-<i>mcherry</i>-<i>frt</i>-<i>cat</i>-<i>frt</i>), <i>seqA</i>-<i>mcherry</i>-<i>frt</i>-<i>cat</i>-<i>frt</i> was introduced into CJW7020 (MG1655 <i>rplA</i>::<i>rplA</i>-<i>msfgfp</i>) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-16" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-16" aria-controls="sref50-16">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This strain was then transiently equipped with plasmid pCP20 expressing the Flp site-specific recombinase, which excised the chloramphenicol resistance cassette (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Cherepanov and Wackernagel, 1995</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Cherepanov, P.P. ∙ Wackernagel, W.</div><div><strong>Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant</strong></div><div><em>Gene.</em> 1995; <strong>158</strong>:9-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0378-1119(95)00193-A" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029065955" target="_blank" aria-label="Open Scopus in new tab">Scopus (1285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7789817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0378-1119%2895%2900193-A&amp;pmid=7789817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), resulting in strain CJW7039.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Cultures</h4><div id="p0300" role="paragraph">To obtain steady-state growth conditions, a sample of an overnight liquid cell culture in stationary phase was diluted by a factor of at least 10,000 in the corresponding fresh growth medium (see below for the composition of the growth medium used for each experiment). The culture was then allowed to grow in an incubator shaker (New Brunswick Innova 44), at the indicated temperature, to reach an optical density at 600&nbsp;nm (OD<sub>600</sub>) of 0.1-0.3 prior to microscopy. Details about growth conditions are described for each method section below.</div></section><section id="sec-4-4-2"><h4>Light microscopy and image analysis</h4><div id="p0305" role="paragraph">Unless otherwise specified, all exponentially growing cells were imaged on 1% agarose pads supplemented with growth medium. When appropriate, cultures were incubated with 1&nbsp;μg/ml 4′,6-diamidino-2-phenylindole (DAPI) for 15&nbsp;min in growth medium prior to imaging on agarose pads. Phase contrast and epifluorescence imaging were performed on a Nikon Eclipse Ti-E microscope equipped with a phase contrast Nikon CFI Plan Apo DM Lambda 100x oil objective (NA&nbsp;= 1.45), and a SOLA light engine (Lumencor). A Ph3 phase contrast ring was used. The following Chroma filters were used for epifluorescence imaging: DAPI (excitation D390/22x, dichroic T425lpxr, emission ET460/50 m), GFP (excitation ET470/40x, dichroic T495lpxr, emission ET525/50 m), mCherry/TexasRed (excitation ET560/40x, dichroic T585lp, emission ET630/75 m). The microscope was controlled by the NIS-Elements AR software.</div><div id="p0310" role="paragraph">Using the open source software Oufti (<span class="dropBlock reference-citations"><a id="body-ref-sref101-2" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-2" aria-controls="sref101-2">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), cell and nucleoid outlines were constructed based on the phase contrast and epifluorescence (e.g., DAPI) images, respectively. When appropriate, cells were classified into the B, C and D cell-cycle periods based on two cell features: SeqA-mCherry pattern and cell area, as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref50-17" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-17" aria-controls="sref50-17">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-3"><h4>Derivation of the nucleoid mesh size</h4><div id="p0315" role="paragraph">The correlation length <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi" tabindex="0" ctxtmenu_counter="62" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="xi"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> (i.e., mesh size) of a semidilute polymer solution is defined as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi equals upper R Subscript g Baseline left parenthesis c Superscript asterisk Baseline divided by c right parenthesis Superscript nu divided by left parenthesis 3 nu minus 1 right parenthesis" tabindex="0" ctxtmenu_counter="63" style="font-size: 130%;"><mjx-math data-semantic-structure="(30 0 1 (29 (4 2 3) 28 (27 (13 5 (11 (8 6 7) 9 10) 12) (26 14 15 (25 16 (23 (22 17 21 18) 19 20) 24)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="30" data-semantic-children="0,29" data-semantic-content="1" data-semantic-owns="0 1 29" data-semantic-speech="xi equals upper R Subscript g Baseline left parenthesis c Superscript asterisk Baseline divided by c right parenthesis Superscript nu divided by left parenthesis 3 nu minus 1 right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="30"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="30" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-children="4,27" data-semantic-content="28" data-semantic-parent="30" data-semantic-owns="4 28 27"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="29" data-semantic-owns="2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="27" data-semantic-children="13,26" data-semantic-parent="29" data-semantic-owns="13 26"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="13" data-semantic-children="11" data-semantic-content="5,12" data-semantic-parent="27" data-semantic-owns="5 11 12"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="5" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="11" data-semantic-children="8,10" data-semantic-content="9" data-semantic-parent="13" data-semantic-owns="8 9 10"><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="11" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="9" data-semantic-parent="11" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="12" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.477em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="26" data-semantic-children="14,25" data-semantic-content="15" data-semantic-parent="27" data-semantic-owns="14 15 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="26"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="15" data-semantic-parent="26" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="25" data-semantic-children="23" data-semantic-content="16,24" data-semantic-parent="26" data-semantic-owns="16 23 24"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="16" data-semantic-parent="25" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="23" data-semantic-children="22,20" data-semantic-content="19" data-semantic-parent="25" data-semantic-owns="22 19 20"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="22" data-semantic-children="17,18" data-semantic-content="21" data-semantic-parent="23" data-semantic-owns="17 21 18"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="22"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="22"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="19" data-semantic-parent="23" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="23"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="24" data-semantic-parent="25" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container> (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Cooper et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cooper, E.C. ∙ Johnson, P. ∙ Donald, A.M.</div><div><strong>Probe diffusion in polymer solutions in the dilute/semi-dilute crossover regime: 1. Poly(ethylene oxide)</strong></div><div><em>Polymer (Guildf.).</em> 1991; <strong>32</strong>:2815-2822</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0032-3861(91)90114-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026368173" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0032-3861%2891%2990114-X" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Dasgupta et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Dasgupta, B.R. ∙ Tee, S.-Y. ∙ Crocker, J.C. ...</div><div><strong>Microrheology of polyethylene oxide using diffusing wave spectroscopy and single scattering</strong></div><div><em>Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</em> 2002; <strong>65</strong>:051505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevE.65.051505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-37649031119" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12059562/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevE.65.051505&amp;pmid=12059562" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39-3" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39-3" aria-controls="sref39-3">de Gennes, 1979</a><div class="dropBlock__holder" data-db-target-of="sref39-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>de Gennes, P.G.</div><div><strong>Scaling concepts in polymer physics</strong></div><div>Cornell University Press, 1979</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.G.de+GennesScaling+concepts+in+polymer+physics1979Cornell+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-6" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-6" aria-controls="sref119-6">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="64" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the radius of gyration, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="65" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the polymer concentration, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu" tabindex="0" ctxtmenu_counter="66" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="nu"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the Flory exponent. The overlap concentration <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c Superscript asterisk" tabindex="0" ctxtmenu_counter="67" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="c Superscript asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container>, above which polymer segments overlap to form a mesh can be written as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c Superscript asterisk Baseline equals 3 upper M Subscript w Baseline upper N b divided by left parenthesis 4 pi upper R Subscript g Superscript 3 Baseline upper N Subscript upper A Baseline l Subscript b a s e Baseline right parenthesis" tabindex="0" ctxtmenu_counter="68" style="font-size: 130%;"><mjx-math data-semantic-structure="(44 (2 0 1) 3 (43 (42 4 39 (7 5 6) 40 8 41 9) 10 (38 11 (36 12 32 13 33 (18 (17 14 15) 16) 34 (21 19 20) 35 (31 22 (30 23 27 24 28 25 29 26))) 37)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="44" data-semantic-children="2,43" data-semantic-content="3" data-semantic-owns="2 3 43" data-semantic-speech="c Superscript asterisk Baseline equals 3 upper M Subscript w Baseline upper N b divided by left parenthesis 4 pi upper R Subscript g Superscript 3 Baseline upper N Subscript upper A Baseline l Subscript b a s e Baseline right parenthesis"><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="44" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo></mjx-script></mjx-msup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="44" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="43" data-semantic-children="42,38" data-semantic-content="10" data-semantic-parent="44" data-semantic-owns="42 10 38"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="42" data-semantic-children="4,7,8,9" data-semantic-content="39,40,41" data-semantic-parent="43" data-semantic-owns="4 39 7 40 8 41 9"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="42"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="42" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="42"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="42"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="10" data-semantic-parent="43" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="38" data-semantic-children="36" data-semantic-content="11,37" data-semantic-parent="43" data-semantic-owns="11 36 37"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="38" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="36" data-semantic-children="12,13,18,21,31" data-semantic-content="32,33,34,35" data-semantic-parent="38" data-semantic-owns="12 32 13 33 18 34 21 35 31"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="36"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="36"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="14,15,16" data-semantic-parent="36" data-semantic-collapsed="(18 (17 14 15) 16)" data-semantic-owns="14 15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="18"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="18"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="18"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="36" data-semantic-owns="19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="22,30" data-semantic-parent="36" data-semantic-owns="22 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="31"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="30" data-semantic-children="23,24,25,26" data-semantic-content="27,28,29" data-semantic-parent="31" data-semantic-owns="23 27 24 28 25 29 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="30"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="30"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="30"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="29" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="30"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="37" data-semantic-parent="38" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Dasgupta et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Dasgupta, B.R. ∙ Tee, S.-Y. ∙ Crocker, J.C. ...</div><div><strong>Microrheology of polyethylene oxide using diffusing wave spectroscopy and single scattering</strong></div><div><em>Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</em> 2002; <strong>65</strong>:051505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevE.65.051505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-37649031119" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12059562/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevE.65.051505&amp;pmid=12059562" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Mutch et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Mutch, K.J. ∙ van Duijneveldt, J.S. ∙ Eastoe, J.</div><div><strong>Colloid-polymer mixtures in the protein limit</strong></div><div><em>Soft Matter.</em> 2007; <strong>3</strong>:155-167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/B611137H" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32680259/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FB611137H&amp;pmid=32680259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Ramakrishnan et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Ramakrishnan, S. ∙ Fuchs, M. ∙ Schweizer, K.S. ...</div><div><strong>Entropy driven phase transitions in colloid–polymer suspensions: Tests of depletion theories</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2002; <strong>116</strong>:2201-2212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1426413" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036470201" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1426413" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M Subscript w" tabindex="0" ctxtmenu_counter="69" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper M Subscript w"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the average molecular weight of the base pair (650 g/mol) (<span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Lee et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Lee, C.K. ∙ Araki, N. ∙ Sowersby, D.S. ...</div><div><strong>Factors affecting chemical-based purification of DNA from Saccharomyces cerevisiae</strong></div><div><em>Yeast.</em> 2012; <strong>29</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/yea.1918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84856594171" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fyea.1918&amp;pmid=22134898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104-3" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-3" aria-controls="sref104-3">Peale et&nbsp;al., 1989</a><div class="dropBlock__holder" data-db-target-of="sref104-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Peale, Jr., F.V. ∙ Ishibe, Y. ∙ Klinge, C.M. ...</div><div><strong>Rapid purification of the estrogen receptor by sequence-specific DNA affinity chromatography</strong></div><div><em>Biochemistry.</em> 1989; <strong>28</strong>:8671-8675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00448a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0024450125" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2605215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00448a001&amp;pmid=2605215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref115-3" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-3" aria-controls="sref115-3">Ratilainen et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref115-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Ratilainen, T. ∙ Lincoln, P. ∙ Nordén, B.</div><div><strong>A simple model for gene targeting</strong></div><div><em>Biophys. J.</em> 2001; <strong>81</strong>:2876-2885</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11606298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2801%2975928-0&amp;pmid=11606298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="70" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the number of polymer segments, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="71" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the Kuhn length, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l Subscript b a s e" tabindex="0" ctxtmenu_counter="72" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="l Subscript b a s e"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the size of a base pair (0.34&nbsp;nm) (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Diekmann et&nbsp;al., 1982</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Diekmann, S. ∙ Hillen, W. ∙ Morgeneyer, B. ...</div><div><strong>Orientation relaxation of DNA restriction fragments and the internal mobility of the double helix</strong></div><div><em>Biophys. Chem.</em> 1982; <strong>15</strong>:263-270</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(82)80009-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6288135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2882%2980009-4&amp;pmid=6288135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref165-2" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref165-2" aria-controls="sref165-2">Yonemura and Maeda, 1982</a><div class="dropBlock__holder" data-db-target-of="sref165-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>Yonemura, K. ∙ Maeda, H.</div><div><strong>A new assay method for DNase by fluorescence polarization and fluorescence intensity using DNA-ethidium bromide complex as a sensitive substrate</strong></div><div><em>J.&nbsp;Biochem.</em> 1982; <strong>92</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a134048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0020309501" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7174645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a134048&amp;pmid=7174645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N Subscript upper A" tabindex="0" ctxtmenu_counter="73" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper N Subscript upper A"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the Avogadro’s number. For a circular polymer, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g Baseline equals left parenthesis StartRoot 3 EndRoot divided by 6 right parenthesis b upper N Superscript nu" tabindex="0" ctxtmenu_counter="74" style="font-size: 130%;"><mjx-math data-semantic-structure="(19 (2 0 1) 3 (18 (11 4 (9 (6 5) 7 8) 10) 16 12 17 (15 13 14)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="19" data-semantic-children="2,18" data-semantic-content="3" data-semantic-owns="2 3 18" data-semantic-speech="upper R Subscript g Baseline equals left parenthesis StartRoot 3 EndRoot divided by 6 right parenthesis b upper N Superscript nu"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="19" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="19" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="11,12,15" data-semantic-content="16,17" data-semantic-parent="19" data-semantic-owns="11 16 12 17 15"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="11" data-semantic-children="9" data-semantic-content="4,10" data-semantic-parent="18" data-semantic-owns="4 9 10"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="11" data-semantic-owns="6 7 8"><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="6" data-semantic-children="5" data-semantic-parent="9" data-semantic-owns="5"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="10" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="18"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="18" data-semantic-owns="13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="15"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em; margin-left: 0.054em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container> (<span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Casassa, 1965</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Casassa, E.F.</div><div><strong>Some statistical properties of flexible ring polymers</strong></div><div><em>J.&nbsp;Polym. Sci. A1.</em> 1965; <strong>3</strong>:605-614</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.CasassaSome+statistical+properties+of+flexible+ring+polymersJ.%C2%A0Polym.+Sci.+A131965605614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Kramers, 1946</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Kramers, H.A.</div><div><strong>The Behavior of Macromolecules in Inhomogeneous Flow</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1946; <strong>14</strong>:415-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1724163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1724163" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-7" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-7" aria-controls="sref119-7">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref172-1" href="#" role="doc-biblioref" data-xml-rid="bib172" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref172-1" aria-controls="sref172-1">Zimm and Stockmayer, 1949</a><div class="dropBlock__holder" data-db-target-of="sref172-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">171.</div><div>Zimm, B.H. ∙ Stockmayer, W.H.</div><div><strong>The Dimensions of Chain Molecules Containing Branches and Rings</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1949; <strong>17</strong>:1301-1314</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1747157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1747157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Plugging both the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="75" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c Superscript asterisk" tabindex="0" ctxtmenu_counter="76" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="c Superscript asterisk"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mo size="s" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c2217">∗</mjx-c></mjx-mo></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> into the above definition, we have <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi equals left parenthesis StartRoot 3 EndRoot divided by 6 right parenthesis b upper N Superscript nu Baseline left bracket 18 StartRoot 3 EndRoot upper M Subscript w Baseline upper N b divided by left parenthesis pi b cubed upper N Superscript 3 nu Baseline upper N Subscript upper A Baseline l Subscript b a s e Baseline c right parenthesis right bracket Superscript nu divided by left parenthesis 3 nu minus 1 right parenthesis" tabindex="0" ctxtmenu_counter="77" style="font-size: 130%;"><mjx-math data-semantic-structure="(83 0 1 (82 (9 2 (7 (4 3) 5 6) 8) 79 10 80 (13 11 12) 81 (78 (64 14 (62 (61 15 57 (17 16) 58 (20 18 19) 59 21 60 22) 23 (56 24 (54 25 49 (28 26 27) 50 (34 29 (33 30 32 31)) 51 (37 35 36) 52 (47 38 (46 39 43 40 44 41 45 42)) 53 48) 55)) 63) (77 65 66 (76 67 (74 (73 68 72 69) 70 71) 75)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="83" data-semantic-children="0,82" data-semantic-content="1" data-semantic-owns="0 1 82" data-semantic-speech="xi equals left parenthesis StartRoot 3 EndRoot divided by 6 right parenthesis b upper N Superscript nu Baseline left bracket 18 StartRoot 3 EndRoot upper M Subscript w Baseline upper N b divided by left parenthesis pi b cubed upper N Superscript 3 nu Baseline upper N Subscript upper A Baseline l Subscript b a s e Baseline c right parenthesis right bracket Superscript nu divided by left parenthesis 3 nu minus 1 right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="83"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="83" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="82" data-semantic-children="9,10,13,78" data-semantic-content="79,80,81" data-semantic-parent="83" data-semantic-owns="9 79 10 80 13 81 78"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="2,8" data-semantic-parent="82" data-semantic-owns="2 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="7" data-semantic-children="4,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="4" data-semantic-children="3" data-semantic-parent="7" data-semantic-owns="3"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="79" data-semantic-parent="82" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="82"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="80" data-semantic-parent="82" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="82" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em; margin-left: 0.054em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="81" data-semantic-parent="82" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="78" data-semantic-children="64,77" data-semantic-parent="82" data-semantic-owns="64 77"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="64" data-semantic-children="62" data-semantic-content="14,63" data-semantic-parent="78" data-semantic-owns="14 62 63"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="14" data-semantic-parent="64" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="62" data-semantic-children="61,56" data-semantic-content="23" data-semantic-parent="64" data-semantic-owns="61 23 56"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="61" data-semantic-children="15,17,20,21,22" data-semantic-content="57,58,59,60" data-semantic-parent="62" data-semantic-owns="15 57 17 58 20 59 21 60 22"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="61"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="57" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="17" data-semantic-children="16" data-semantic-parent="61" data-semantic-owns="16"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="61" data-semantic-owns="18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="61"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="60" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="61"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="23" data-semantic-parent="62" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="56" data-semantic-children="54" data-semantic-content="24,55" data-semantic-parent="62" data-semantic-owns="24 54 55"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="24" data-semantic-parent="56" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="54" data-semantic-children="25,28,34,37,47,48" data-semantic-content="49,50,51,52,53" data-semantic-parent="56" data-semantic-owns="25 49 28 50 34 51 37 52 47 53 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="54"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="49" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="54" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="50" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="29,33" data-semantic-parent="54" data-semantic-owns="29 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="34"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em; margin-left: 0.054em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="33" data-semantic-children="30,31" data-semantic-content="32" data-semantic-parent="34" data-semantic-owns="30 32 31"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="33"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="35,36" data-semantic-parent="54" data-semantic-owns="35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="52" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="47" data-semantic-children="38,46" data-semantic-parent="54" data-semantic-owns="38 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="47"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="46" data-semantic-children="39,40,41,42" data-semantic-content="43,44,45" data-semantic-parent="47" data-semantic-owns="39 43 40 44 41 45 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="46"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="46"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="44" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="46"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="46"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="53" data-semantic-parent="54" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="54"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="55" data-semantic-parent="56" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="63" data-semantic-parent="64" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.702em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="77" data-semantic-children="65,76" data-semantic-content="66" data-semantic-parent="78" data-semantic-owns="65 66 76"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="77"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="66" data-semantic-parent="77" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="76" data-semantic-children="74" data-semantic-content="67,75" data-semantic-parent="77" data-semantic-owns="67 74 75"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="67" data-semantic-parent="76" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="74" data-semantic-children="73,71" data-semantic-content="70" data-semantic-parent="76" data-semantic-owns="73 70 71"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="73" data-semantic-children="68,69" data-semantic-content="72" data-semantic-parent="74" data-semantic-owns="68 72 69"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="73"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="72" data-semantic-parent="73" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="73"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="70" data-semantic-parent="74" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="74"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="75" data-semantic-parent="76" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, which can be simplified to yield Equation 1 above: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi equals left parenthesis StartRoot 3 EndRoot divided by 6 right parenthesis b Superscript left parenthesis nu minus 1 right parenthesis divided by left parenthesis 3 nu minus 1 right parenthesis Baseline left bracket 18 StartRoot 3 EndRoot upper M Subscript w Baseline divided by left parenthesis pi c upper N Subscript upper A Baseline l Subscript b a s e Baseline right parenthesis right bracket Superscript nu divided by left parenthesis 3 nu minus 1 right parenthesis" tabindex="0" ctxtmenu_counter="78" style="font-size: 130%;"><mjx-math data-semantic-structure="(84 0 1 (83 (9 2 (7 (4 3) 5 6) 8) 81 (30 10 (29 (17 11 (15 12 13 14) 16) 18 (28 19 (26 (25 20 24 21) 22 23) 27))) 82 (80 (66 31 (64 (63 32 61 (34 33) 62 (37 35 36)) 38 (60 39 (58 40 55 41 56 (44 42 43) 57 (54 45 (53 46 50 47 51 48 52 49))) 59)) 65) (79 67 68 (78 69 (76 (75 70 74 71) 72 73) 77)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="84" data-semantic-children="0,83" data-semantic-content="1" data-semantic-owns="0 1 83" data-semantic-speech="xi equals left parenthesis StartRoot 3 EndRoot divided by 6 right parenthesis b Superscript left parenthesis nu minus 1 right parenthesis divided by left parenthesis 3 nu minus 1 right parenthesis Baseline left bracket 18 StartRoot 3 EndRoot upper M Subscript w Baseline divided by left parenthesis pi c upper N Subscript upper A Baseline l Subscript b a s e Baseline right parenthesis right bracket Superscript nu divided by left parenthesis 3 nu minus 1 right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="84"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="84" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="83" data-semantic-children="9,30,80" data-semantic-content="81,82" data-semantic-parent="84" data-semantic-owns="9 81 30 82 80"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="2,8" data-semantic-parent="83" data-semantic-owns="2 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="7" data-semantic-children="4,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="4" data-semantic-children="3" data-semantic-parent="7" data-semantic-owns="3"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="81" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="30" data-semantic-children="10,29" data-semantic-parent="83" data-semantic-owns="10 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="30"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="29" data-semantic-children="17,28" data-semantic-content="18" data-semantic-parent="30" data-semantic-owns="17 18 28"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="15" data-semantic-content="11,16" data-semantic-parent="29" data-semantic-owns="11 15 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="15" data-semantic-children="12,14" data-semantic-content="13" data-semantic-parent="17" data-semantic-owns="12 13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-parent="15" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="18" data-semantic-parent="29" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="28" data-semantic-children="26" data-semantic-content="19,27" data-semantic-parent="29" data-semantic-owns="19 26 27"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="19" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-children="25,23" data-semantic-content="22" data-semantic-parent="28" data-semantic-owns="25 22 23"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="25" data-semantic-children="20,21" data-semantic-content="24" data-semantic-parent="26" data-semantic-owns="20 24 21"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="25"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="25"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="26"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="27" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="82" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="80" data-semantic-children="66,79" data-semantic-parent="83" data-semantic-owns="66 79"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="66" data-semantic-children="64" data-semantic-content="31,65" data-semantic-parent="80" data-semantic-owns="31 64 65"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="31" data-semantic-parent="66" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="64" data-semantic-children="63,60" data-semantic-content="38" data-semantic-parent="66" data-semantic-owns="63 38 60"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="63" data-semantic-children="32,34,37" data-semantic-content="61,62" data-semantic-parent="64" data-semantic-owns="32 61 34 62 37"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="63"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="61" data-semantic-parent="63" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="34" data-semantic-children="33" data-semantic-parent="63" data-semantic-owns="33"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="62" data-semantic-parent="63" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="35,36" data-semantic-parent="63" data-semantic-owns="35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="38" data-semantic-parent="64" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="60" data-semantic-children="58" data-semantic-content="39,59" data-semantic-parent="64" data-semantic-owns="39 58 59"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="39" data-semantic-parent="60" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="58" data-semantic-children="40,41,44,54" data-semantic-content="55,56,57" data-semantic-parent="60" data-semantic-owns="40 55 41 56 44 57 54"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="58"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="58" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="58"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="56" data-semantic-parent="58" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="44" data-semantic-children="42,43" data-semantic-parent="58" data-semantic-owns="42 43"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="44"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="44"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="57" data-semantic-parent="58" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="54" data-semantic-children="45,53" data-semantic-parent="58" data-semantic-owns="45 53"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="54"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="53" data-semantic-children="46,47,48,49" data-semantic-content="50,51,52" data-semantic-parent="54" data-semantic-owns="46 50 47 51 48 52 49"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="53"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="50" data-semantic-parent="53" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="53"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="51" data-semantic-parent="53" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="53"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="52" data-semantic-parent="53" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="53"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="59" data-semantic-parent="60" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="65" data-semantic-parent="66" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.702em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="79" data-semantic-children="67,78" data-semantic-content="68" data-semantic-parent="80" data-semantic-owns="67 68 78"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="79"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="68" data-semantic-parent="79" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="78" data-semantic-children="76" data-semantic-content="69,77" data-semantic-parent="79" data-semantic-owns="69 76 77"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="69" data-semantic-parent="78" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="76" data-semantic-children="75,73" data-semantic-content="72" data-semantic-parent="78" data-semantic-owns="75 72 73"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="75" data-semantic-children="70,71" data-semantic-content="74" data-semantic-parent="76" data-semantic-owns="70 74 71"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="75"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="74" data-semantic-parent="75" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="75"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="72" data-semantic-parent="76" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="73" data-semantic-parent="76"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="77" data-semantic-parent="78" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Note that the numerical prefactors (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartRoot 3 EndRoot divided by 6" tabindex="0" ctxtmenu_counter="79" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (1 0) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-owns="1 2 3" data-semantic-speech="StartRoot 3 EndRoot divided by 6"><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="1" data-semantic-children="0" data-semantic-parent="4" data-semantic-owns="0"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="1"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="18 StartRoot 3 EndRoot" tabindex="0" ctxtmenu_counter="80" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 3 (2 1))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="4" data-semantic-children="0,2" data-semantic-content="3" data-semantic-owns="0 3 2" data-semantic-speech="18 StartRoot 3 EndRoot"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="2" data-semantic-children="1" data-semantic-parent="4" data-semantic-owns="1"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="pi" tabindex="0" ctxtmenu_counter="81" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="pi"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) in the derived equation purely come from the geometry of chromosome and the definition of the overlap concentration. To verify the&nbsp;dimension of our result, we note the term <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="18 StartRoot 3 EndRoot upper M Subscript w divided by left parenthesis pi c upper N Subscript upper A Baseline l Subscript b a s e Baseline right parenthesis" tabindex="0" ctxtmenu_counter="82" style="font-size: 130%;"><mjx-math data-semantic-structure="(32 (31 0 29 (2 1) 30 (5 3 4)) 6 (28 7 (26 8 23 9 24 (12 10 11) 25 (22 13 (21 14 18 15 19 16 20 17))) 27))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="32" data-semantic-children="31,28" data-semantic-content="6" data-semantic-owns="31 6 28" data-semantic-speech="18 StartRoot 3 EndRoot upper M Subscript w divided by left parenthesis pi c upper N Subscript upper A Baseline l Subscript b a s e Baseline right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="31" data-semantic-children="0,2,5" data-semantic-content="29,30" data-semantic-parent="32" data-semantic-owns="0 29 2 30 5"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="31"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c38" style="padding-top: 0.666em;">8</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="29" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="2" data-semantic-children="1" data-semantic-parent="31" data-semantic-owns="1"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.163em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="31" data-semantic-owns="3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.074em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D464">𝑤</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="6" data-semantic-parent="32" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="28" data-semantic-children="26" data-semantic-content="7,27" data-semantic-parent="32" data-semantic-owns="7 26 27"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="26" data-semantic-children="8,9,12,22" data-semantic-content="23,24,25" data-semantic-parent="28" data-semantic-owns="8 23 9 24 12 25 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="26"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="26"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="26" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.153em; margin-left: -0.078em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="22" data-semantic-children="13,21" data-semantic-parent="26" data-semantic-owns="13 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="22"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="21" data-semantic-children="14,15,16,17" data-semantic-content="18,19,20" data-semantic-parent="22" data-semantic-owns="14 18 15 19 16 20 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="21"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="21"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="21"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="21"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="27" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> has a dimension of area. Since <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="83" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> has a dimension of length and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis nu minus 1 right parenthesis divided by left parenthesis 3 nu minus 1 right parenthesis plus 2 nu divided by left parenthesis 3 nu minus 1 right parenthesis equals 1" tabindex="0" ctxtmenu_counter="84" style="font-size: 130%;"><mjx-math data-semantic-structure="(39 (37 (34 (6 0 (4 1 2 3) 5) 7 (17 8 (15 (14 9 13 10) 11 12) 16)) 18 (38 (36 19 35 20) 21 (31 22 (29 (28 23 27 24) 25 26) 30))) 32 33)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="39" data-semantic-children="37,33" data-semantic-content="32" data-semantic-owns="37 32 33" data-semantic-speech="left parenthesis nu minus 1 right parenthesis divided by left parenthesis 3 nu minus 1 right parenthesis plus 2 nu divided by left parenthesis 3 nu minus 1 right parenthesis equals 1"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="37" data-semantic-children="34,38" data-semantic-content="18" data-semantic-parent="39" data-semantic-owns="34 18 38"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="34" data-semantic-children="6,17" data-semantic-content="7" data-semantic-parent="37" data-semantic-owns="6 7 17"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="4" data-semantic-content="0,5" data-semantic-parent="34" data-semantic-owns="0 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="34" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="15" data-semantic-content="8,16" data-semantic-parent="34" data-semantic-owns="8 15 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="8" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="15" data-semantic-children="14,12" data-semantic-content="11" data-semantic-parent="17" data-semantic-owns="14 11 12"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="14" data-semantic-children="9,10" data-semantic-content="13" data-semantic-parent="15" data-semantic-owns="9 13 10"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="14"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="14"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="11" data-semantic-parent="15" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="18" data-semantic-parent="37" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="38" data-semantic-children="36,31" data-semantic-content="21" data-semantic-parent="37" data-semantic-owns="36 21 31"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="36" data-semantic-children="19,20" data-semantic-content="35" data-semantic-parent="38" data-semantic-owns="19 35 20"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="36"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="36"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="21" data-semantic-parent="38" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="31" data-semantic-children="29" data-semantic-content="22,30" data-semantic-parent="38" data-semantic-owns="22 29 30"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="22" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="29" data-semantic-children="28,26" data-semantic-content="25" data-semantic-parent="31" data-semantic-owns="28 25 26"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="28" data-semantic-children="23,24" data-semantic-content="27" data-semantic-parent="29" data-semantic-owns="23 27 24"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="28"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="28"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="25" data-semantic-parent="29" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="29"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="30" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="32" data-semantic-parent="39" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="39"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="xi" tabindex="0" ctxtmenu_counter="85" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="xi"><mjx-c class="mjx-c1D709">𝜉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> correctly has a dimension of length.</div></section><section id="sec-4-4-4"><h4>Single-particle tracking experiments</h4><div id="p0320" role="paragraph">GFP-tagged nanocage particles were produced through basal expression of the GFP-tagged protein subunit I3-01 in <i>E.&nbsp;coli</i> cells (CJW6340) grown at 30°C in M9&nbsp;minimal medium supplemented with glycerol (0.2%), casamino acids (0.1%) and thiamine (1&nbsp;μg/ml). Phase contrast images were taken first to define cell outlines. Time-lapse fluorescence images were then acquired at a frame rate of 100&nbsp;ms. Nanocage particles were localized and tracked using the MATLAB class SPT (see supplementary code). Briefly, an image was first smoothed by a band-pass filter. Potential regions containing the particles were then segmented based on an intensity threshold. Based on the region sizes, background noise pixels or particle clusters were filtered out. Within each of the remaining segmented regions, the brightest pixel was located. On the brightest pixel and its eight surrounding pixels, the pixel intensities were fitted by a 2D Gaussian distribution, whose center was taken as the estimated particle position. The particle locations were linked into trajectories using an algorithm described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Crocker and Grier, 1996</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Crocker, J.C. ∙ Grier, D.G.</div><div><strong>Methods of Digital Video Microscopy for Colloidal Studies</strong></div><div><em>J.&nbsp;Colloid Interface Sci.</em> 1996; <strong>179</strong>:298-310</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jcis.1996.0217" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029977418" target="_blank" aria-label="Open Scopus in new tab">Scopus (2486)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjcis.1996.0217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The 2D mean squared displacements (MSD) plot were calculated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M upper S upper D left parenthesis tau right parenthesis equals less than left parenthesis r left parenthesis t plus tau right parenthesis minus r left parenthesis t right parenthesis right parenthesis squared greater than" tabindex="0" ctxtmenu_counter="86" style="font-size: 130%;"><mjx-math data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="40" data-semantic-children="39,34" data-semantic-content="7" data-semantic-owns="39 7 34" data-semantic-structure="(40 (39 0 37 1 38 (36 2 35 (6 4 3 5))) 7 (34 32 (31 (29 27 (26 (25 8 24 (15 13 (12 9 10 11) 14)) 16 (23 17 22 (21 19 18 20))) 28) 30) 33))" data-semantic-speech="upper M upper S upper D left parenthesis tau right parenthesis equals less than left parenthesis r left parenthesis t plus tau right parenthesis minus r left parenthesis t right parenthesis right parenthesis squared greater than" breakable="true" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="39" data-semantic-children="0,1,36" data-semantic-content="37,38" data-semantic-parent="40" data-semantic-owns="0 37 1 38 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="39"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="37" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="39"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-children="2,6" data-semantic-content="35,2" data-semantic-parent="39" data-semantic-owns="2 35 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="36" data-semantic-operator="appl"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="35" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="6" data-semantic-children="3" data-semantic-content="4,5" data-semantic-parent="36" data-semantic-owns="4 3 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D70F">𝜏</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="40" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="inequality" data-semantic-id="34" data-semantic-children="31" data-semantic-content="32,33" data-semantic-parent="40" data-semantic-owns="32 31 33" inline-breaks="true"><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="32" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="34" data-semantic-owns="29 30"><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="29" data-semantic-children="26" data-semantic-content="27,28" data-semantic-parent="31" data-semantic-owns="27 26 28"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="27" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-children="25,23" data-semantic-content="16" data-semantic-parent="29" data-semantic-owns="25 16 23"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="25" data-semantic-children="8,15" data-semantic-content="24,8" data-semantic-parent="26" data-semantic-owns="8 24 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="25" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="12" data-semantic-content="13,14" data-semantic-parent="25" data-semantic-owns="13 12 14"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="12" data-semantic-children="9,11" data-semantic-content="10" data-semantic-parent="15" data-semantic-owns="9 10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="12"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="10" data-semantic-parent="12" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D70F">𝜏</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="16" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-children="17,21" data-semantic-content="22,17" data-semantic-parent="26" data-semantic-owns="17 22 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="23" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="21" data-semantic-children="18" data-semantic-content="19,20" data-semantic-parent="23" data-semantic-owns="19 18 20"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="19" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="21"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="20" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-break size="4"></mjx-break><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="tau" tabindex="0" ctxtmenu_counter="87" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="tau"><mjx-c class="mjx-c1D70F">𝜏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is a given time delay and<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r left parenthesis t right parenthesis" tabindex="0" ctxtmenu_counter="88" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 1 6 (5 2 3 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="1,5" data-semantic-content="6,1" data-semantic-owns="1 6 5" data-semantic-speech="r left parenthesis t right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="2,4" data-semantic-parent="7" data-semantic-owns="2 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r left parenthesis t plus tau right parenthesis" tabindex="0" ctxtmenu_counter="89" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 8 (7 1 (5 2 3 4) 6))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="0,7" data-semantic-content="8" data-semantic-owns="0 8 7" data-semantic-speech="r left parenthesis t plus tau right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="7" data-semantic-children="5" data-semantic-content="1,6" data-semantic-parent="9" data-semantic-owns="1 5 6"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-parent="7" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D70F">𝜏</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> denote the particle positions before and after the given time delay. Similarly, GFP-μNS particles were produced through IPTG induction (50-200&nbsp;μM) for a duration of 30-120&nbsp;min, in <i>E.&nbsp;coli</i> cells (CJW4617) grown at 30°C in M9&nbsp;minimal medium supplemented with glycerol (0.2%), casamino acids (0.1%) and thiamine (1&nbsp;μg/ml). GFP-μNS particles were detected using SpotFinder (<span class="dropBlock reference-citations"><a id="body-ref-sref131-2" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-2" aria-controls="sref131-2">Sliusarenko et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref131-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Sliusarenko, O. ∙ Heinritz, J. ∙ Emonet, T. ...</div><div><strong>High-throughput, subpixel precision analysis of bacterial morphogenesis and intracellular spatio-temporal dynamics</strong></div><div><em>Mol. Microbiol.</em> 2011; <strong>80</strong>:612-627</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2011.07579.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955024764" target="_blank" aria-label="Open Scopus in new tab">Scopus (340)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21414037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2011.07579.x&amp;pmid=21414037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Particle positions of both nanocage-GFP and GFP-μNS particles relative to the cell coordinates were calculated using the function projectToMesh in Oufti and normalized by cell width and length. The relative particle positions were used to generate histograms to demonstrate the probability density of relative particle locations inside the cell.</div></section><section id="sec-4-4-5"><h4>Monte Carlo simulation of chromosome conformation</h4><div id="p0325" role="paragraph">The entire <i>E.&nbsp;coli</i> chromosome (4.6 million base pairs, contour length&nbsp;= 1564&nbsp;μm) was modeled as a freely-jointed phantom chain (zero volume, no self-avoidance) of 26,066 segments, each of which had a length of 60&nbsp;nm (to reflect the estimated Kuhn length). In each simulation instance, the first segment of the chromosome was constructed by linking the origin (x&nbsp;= 0, y&nbsp;= 0, z&nbsp;= 0) and a random point radially a Kuhn-length (i.e., 60&nbsp;nm) away. Similarly, the remaining segments were constructed iteratively by linking the current end of the chromosome chain with another randomly sampled point that was radially a Kuhn-length away. The solvent quality was modeled by the constraint on the range of angles between consecutive segments. In the ideal-solvent case, there was no restriction on the inter-segmental angles <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis theta right parenthesis" tabindex="0" ctxtmenu_counter="90" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis theta right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, as there was no preference for a segment to interact with other parts of the polymer or the solvent. Meanwhile, for a polymer in a poor or good solvent, the segments on the polymer chain was set to ‘prefer’ interacting among themselves (poor solvent) or with the solvent (good solvent). To model the attraction and repulsion, restrictions were applied to the range of possible <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="theta" tabindex="0" ctxtmenu_counter="91" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="theta"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. By verifying the power-law scaling between the number of segments and the radius of gyration of the simulated chromosomes, the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="theta" tabindex="0" ctxtmenu_counter="92" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="theta"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> was constrained within empirically determined ranges (<a id="crosref2330" href="#figs3">Figure&nbsp;S3</a>B). Specifically, we found that the Flory exponents (0.36 and 0.58) could be achieved when the inter-segmental angles were constrained to be greater than 11π/18 and smaller than π/2, respectively. A limitation of our simulation scheme is that it does not explicitly model the molecular forces among the monomers and only considers interaction between adjacent monomers. However, it is computationally efficient for modeling a large polymer like the bacterial chromosome and effectively reproduces the scaling of polymer size at two length scales. Specifically, at small length scales, the conformation (and thus <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="93" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>) of a small section of the polymer chain in a semidilute solution behaves similarly to that in a dilute solution at a given solvent quality (<a id="crosref2335" href="#figs3">Figure&nbsp;S3</a>B), as monomers are surrounded mostly by the solvent at small length scales (<span class="dropBlock reference-citations"><a id="body-ref-sref119-8" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-8" aria-controls="sref119-8">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At larger length scales (<a id="crosref2345" href="#figs3">Figure&nbsp;S3</a>B, inset), interactions between distant monomers become screened and the polymer behaves as a random walk, leading to a scaling exponent of 0.50 regardless of the solvent quality (<span class="dropBlock reference-citations"><a id="body-ref-sref119-9" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-9" aria-controls="sref119-9">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0330" role="paragraph">In order to compare the degree of chromosome compaction as a result of different types of solvent, the <i>E.&nbsp;coli</i> chromosome was simulated in free space 1,000 times for each type of the solvent. The radius of gyration of each simulated chromosome was calculated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g Superscript 2 Baseline equals StartFraction 1 Over upper N EndFraction sigma summation Underscript i Overscript upper N Endscripts left parenthesis r Subscript i Baseline minus r Subscript m e a n Baseline right parenthesis squared" tabindex="0" ctxtmenu_counter="94" style="font-size: 130%;"><mjx-math data-semantic-structure="(36 (4 (3 0 1) 2) 5 (35 (8 6 7) 34 (33 (12 9 10 11) (32 (30 13 (28 (16 14 15) 17 (27 18 (26 19 23 20 24 21 25 22))) 29) 31))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="36" data-semantic-children="4,35" data-semantic-content="5" data-semantic-owns="4 5 35" data-semantic-speech="upper R Subscript g Superscript 2 Baseline equals StartFraction 1 Over upper N EndFraction sigma summation Underscript i Overscript upper N Endscripts left parenthesis r Subscript i Baseline minus r Subscript m e a n Baseline right parenthesis squared"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="36" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.297em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="36" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="35" data-semantic-children="8,33" data-semantic-content="34" data-semantic-parent="36" data-semantic-owns="8 34 33"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="35" data-semantic-owns="6 7"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="33" data-semantic-children="12,32" data-semantic-content="9" data-semantic-parent="35" data-semantic-owns="12 32"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="12" data-semantic-children="9,10,11" data-semantic-parent="33" data-semantic-owns="9 10 11" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="9" data-semantic-parent="12" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.295em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-munderover><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="32" data-semantic-children="30,31" data-semantic-parent="33" data-semantic-owns="30 31"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="13,29" data-semantic-parent="32" data-semantic-owns="13 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="28" data-semantic-children="16,27" data-semantic-content="17" data-semantic-parent="30" data-semantic-owns="16 17 27"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="28" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="17" data-semantic-parent="28" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="27" data-semantic-children="18,26" data-semantic-parent="28" data-semantic-owns="18 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="27"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="26" data-semantic-children="19,20,21,22" data-semantic-content="23,24,25" data-semantic-parent="27" data-semantic-owns="19 23 20 24 21 25 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="26"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="26"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="26"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="26"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="32"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="95" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the radius of gyration, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript i" tabindex="0" ctxtmenu_counter="96" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is position of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="97" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>-th segment, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript m e a n" tabindex="0" ctxtmenu_counter="98" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="r Subscript m e a n"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the average position of all segments and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="99" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the total number of segments. Given a fixed contour length, the geometric constraint of being circular forces the chromosome to adopt a smaller average size compared to that of a linear chain. Quantitatively, the average radius of gyration of a circular polymer is a factor of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartRoot 2 EndRoot" tabindex="0" ctxtmenu_counter="100" style="font-size: 130%;"><mjx-math data-semantic-structure="(1 0)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="1" data-semantic-children="0" data-semantic-owns="0" data-semantic-speech="StartRoot 2 EndRoot"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.174em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="1"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-math></mjx-container> smaller than that of its linear counterpart (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Casassa, 1965</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Casassa, E.F.</div><div><strong>Some statistical properties of flexible ring polymers</strong></div><div><em>J.&nbsp;Polym. Sci. A1.</em> 1965; <strong>3</strong>:605-614</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.CasassaSome+statistical+properties+of+flexible+ring+polymersJ.%C2%A0Polym.+Sci.+A131965605614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Kramers, 1946</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Kramers, H.A.</div><div><strong>The Behavior of Macromolecules in Inhomogeneous Flow</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1946; <strong>14</strong>:415-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1724163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1724163" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-10" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-10" aria-controls="sref119-10">Rubinstein, 2003</a><div class="dropBlock__holder" data-db-target-of="sref119-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref172-2" href="#" role="doc-biblioref" data-xml-rid="bib172" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref172-2" aria-controls="sref172-2">Zimm and Stockmayer, 1949</a><div class="dropBlock__holder" data-db-target-of="sref172-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">171.</div><div>Zimm, B.H. ∙ Stockmayer, W.H.</div><div><strong>The Dimensions of Chain Molecules Containing Branches and Rings</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1949; <strong>17</strong>:1301-1314</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1747157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1747157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, to account for the circular geometry of the <i>E.&nbsp;coli</i> chromosome, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R Subscript g" tabindex="0" ctxtmenu_counter="101" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper R Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> was divided by a factor of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartRoot 2 EndRoot" tabindex="0" ctxtmenu_counter="102" style="font-size: 130%;"><mjx-math data-semantic-structure="(1 0)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="1" data-semantic-children="0" data-semantic-owns="0" data-semantic-speech="StartRoot 2 EndRoot"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.174em;"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="1"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-math></mjx-container> to get <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper R prime Subscript g" tabindex="0" ctxtmenu_counter="103" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 (3 0 1) 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2" data-semantic-speech="upper R prime Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="punctuation" data-semantic-role="prime" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c2032 MM-V">′</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.365em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. The volume of the simulated nucleoid in each type of solvent was calculated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper V equals four thirds pi left parenthesis upper R prime Subscript g right parenthesis cubed" tabindex="0" ctxtmenu_counter="104" style="font-size: 130%;"><mjx-math data-semantic-structure="(19 0 1 (18 (4 2 3) 16 5 17 (15 (13 6 (11 (10 7 8) 9) 12) 14)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="19" data-semantic-children="0,18" data-semantic-content="1" data-semantic-owns="0 1 18" data-semantic-speech="upper V equals four thirds pi left parenthesis upper R prime Subscript g right parenthesis cubed"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="19"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="19" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="4,5,15" data-semantic-content="16,17" data-semantic-parent="19" data-semantic-owns="4 16 5 17 15"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="vulgar" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="18" data-semantic-owns="2 3"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="18"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13,14" data-semantic-parent="18" data-semantic-owns="13 14"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="13" data-semantic-children="11" data-semantic-content="6,12" data-semantic-parent="15" data-semantic-owns="6 11 12"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="6" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="7,8,9" data-semantic-parent="13" data-semantic-collapsed="(11 (10 7 8) 9)" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mo size="s" data-semantic-type="punctuation" data-semantic-role="prime" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c2032 MM-V">′</mjx-c></mjx-mo><mjx-spacer style="margin-top: 0.365em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="12" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.574em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0335" role="paragraph">To examine chromosome conformation across solvents under the experimentally determined average DNA concentration within the nucleoid region (<a id="crosref2375" href="#fig3">Figures 3</a>, <a id="crosref2380" href="#figs7">S7</a>A, and S7B), we performed the simulations described above, except that the entire chromosome chain was confined within a spherocylindrical space with a volume (0.70&nbsp;μm<sup>3</sup>) equal to the experimentally observed average nucleoid volume in B-period cells.</div><div id="p0340" role="paragraph">To model the DNA relaxation at the domain boundary in a poor solvent (<a id="crosref2385" href="#figs7">Figure&nbsp;S7</a>D), the restriction on <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="theta" tabindex="0" ctxtmenu_counter="105" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="theta"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> was altered to a different range among a number <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis n right parenthesis" tabindex="0" ctxtmenu_counter="106" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis n right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of DNA segments surrounding the 1-, 2-, 3- and 4-Mb coordinates of the chromosome during simulation. The variables <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="theta" tabindex="0" ctxtmenu_counter="107" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="theta"><mjx-c class="mjx-c1D703">𝜃</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n" tabindex="0" ctxtmenu_counter="108" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="n"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> therefore set the degree and length <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper L right parenthesis" tabindex="0" ctxtmenu_counter="109" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis upper L right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of the DNA relaxation, respectively.</div></section><section id="sec-4-4-6"><h4>Preparation of the cryo-EM grids</h4><div id="p0345" role="paragraph"><i>E.&nbsp;coli</i> wild-type cells (MG1655) were grown at 37°C in M9&nbsp;minimal medium supplemented with glucose (0.2%), casamino acids (0.1%) and thiamine (1&nbsp;μg/ml). Nutrient-rich growth conditions were chosen over nutrient-poor condition to increase the intracellular concentration of ribosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref36-3" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-3" aria-controls="sref36-3">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref36-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Dai, X. ∙ Zhu, M. ∙ Warren, M. ...</div><div><strong>Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>2</strong>:16231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85005950833" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27941827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.231&amp;pmid=27941827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-3" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-3" aria-controls="sref47-3">Forchhammer and Lindahl, 1971</a><div class="dropBlock__holder" data-db-target-of="sref47-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Forchhammer, J. ∙ Lindahl, L.</div><div><strong>Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of Escherichia coli 15</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:563-568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90337-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0015222649" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4927947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990337-8&amp;pmid=4927947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107-3" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-3" aria-controls="sref107-3">Phillips et&nbsp;al., 1969</a><div class="dropBlock__holder" data-db-target-of="sref107-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Phillips, L.A. ∙ Hotham-Iglewski, B. ∙ Franklin, R.M.</div><div><strong>Polyribosomes of Escherichia coli. II. Experiments to determine the in&nbsp;vivo distribution of polysomes, ribosomes and ribosomal subunits</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1969; <strong>45</strong>:23-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(69)90207-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4186447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2869%2990207-1&amp;pmid=4186447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref145-3" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-3" aria-controls="sref145-3">Varricchio and Monier, 1971</a><div class="dropBlock__holder" data-db-target-of="sref145-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Varricchio, F. ∙ Monier, R.</div><div><strong>Ribosome patterns in Escherichia coli growing at various rates</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1971; <strong>108</strong>:105-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.108.1.105-110.1971" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5001192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.108.1.105-110.1971&amp;pmid=5001192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) thereby decreasing the likelihood that any observed heterogeneity in ribosome distribution in the cell is due to random clustering. Exponentially growing cells were then harvested and concentrated between 30- and 150-fold by centrifugation (6,000&nbsp;g for 5&nbsp;min). The concentrated culture sample was deposited onto freshly glow-discharged QUANTIFOIL holey carbon grids (Electron Microscopy Sciences). The grids were then blotted with the filter paper for ∼3-5&nbsp;s before getting plunge-frozen into liquid ethane, using a custom gravity-driven plunger apparatus described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Liu et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Liu, J. ∙ Lin, T. ∙ Botkin, D.J. ...</div><div><strong>Intact flagellar motor of Borrelia burgdorferi revealed by cryo-electron tomography: evidence for stator ring curvature and rotor/C-ring assembly flexion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:5026-5036</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.00340-09" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67749093018" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19429612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.00340-09&amp;pmid=19429612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref169" href="#" role="doc-biblioref" data-xml-rid="bib169" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref169" aria-controls="sref169">Zhao et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref169"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">168.</div><div>Zhao, X. ∙ Zhang, K. ∙ Boquoi, T. ...</div><div><strong>Cryoelectron tomography reveals the sequential assembly of bacterial flagella in Borrelia burgdorferi</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:14390-14395</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1308306110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84883311301" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23940315/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1308306110&amp;pmid=23940315" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-7"><h4>Cryo-FIB milling</h4><div id="p0350" role="paragraph">The plunge-frozen grids were clipped into cryo-FIB AutoGrids and mounted into the specimen shuttle under liquid nitrogen. An Aquilos cryo-FIB system (Thermo Fisher Scientific) was used to mill the samples to produce thin lamellae. The samples were first sputter-coated with Pt (1 kV, 15 mA, 15 s) to improve the overall sample conductivity, and then were deposited by an organometallic Pt layer (4-5&nbsp;μm thick) using the gas injection system for the sample protection. Lamellae were produced using the gallium ion beam at 30 kV with stage tilt angle around 17°. The ion beam current was reduced according to the lamella thickness (t) during the milling process: 0.5 nA for t ≥ 3&nbsp;μm, 0.3 nA for t ≥ 1&nbsp;μm, 0.1 nA for t ≥ 700&nbsp;nm, 0.05/0.03 nA for final polishing. Afterward, a thin Pt layer was sputter-coated (1 kV, 10 mA, 5 s) on the lamella to prevent possible charging issue during the cryo-ET imaging.</div></section><section id="sec-4-4-8"><h4>Cryo-ET data acquisition and tomogram reconstruction</h4><div id="p0355" role="paragraph">The cryo-FIB lamellae were transferred to a 300 kV Titan Krios electron microscope (Thermo Fisher Scientific) equipped with a Direct Electron Detector and energy filter (Gatan). The program SerialEM (<span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Mastronarde, 2005</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Mastronarde, D.N.</div><div><strong>Automated electron microscope tomography using robust prediction of specimen movements</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2005; <strong>152</strong>:36-51</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2005.07.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-25644458666" target="_blank" aria-label="Open Scopus in new tab">Scopus (1782)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16182563/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2005.07.007&amp;pmid=16182563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to collect single-axis tilt series around −6&nbsp;μm defocus, with a cumulative dose of ∼100 e<sup>-</sup>/Å covering angles from −60° to 60° (2° tilt step). Images were acquired at 26,000&nbsp;× magnification with an effective pixel size of 5.457&nbsp;Å at the specimen level. All recorded images were first drift corrected by the software MotionCor2 (<span class="dropBlock reference-citations"><a id="body-ref-sref170-2" href="#" role="doc-biblioref" data-xml-rid="bib170" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref170-2" aria-controls="sref170-2">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref170-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">169.</div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85014129582" target="_blank" aria-label="Open Scopus in new tab">Scopus (1982)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4193&amp;pmid=28250466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and then stacked by the software package IMOD (<span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Kremer et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Kremer, J.R. ∙ Mastronarde, D.N. ∙ McIntosh, J.R.</div><div><strong>Computer visualization of three-dimensional image data using IMOD</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 1996; <strong>116</strong>:71-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.1996.0013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029879295" target="_blank" aria-label="Open Scopus in new tab">Scopus (3066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8742726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.1996.0013&amp;pmid=8742726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All tilt series were then aligned by IMOD with the patching tracking method. The program Gctf (<span class="dropBlock reference-citations"><a id="body-ref-sref167-2" href="#" role="doc-biblioref" data-xml-rid="bib167" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref167-2" aria-controls="sref167-2">Zhang, 2016</a><div class="dropBlock__holder" data-db-target-of="sref167-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2015.11.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84955216953" target="_blank" aria-label="Open Scopus in new tab">Scopus (1297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2015.11.003&amp;pmid=26592709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to determine the defocus of each tilt image in the aligned stacks, and the function “ctfphaseflip” in IMOD was used to perform the contrast transfer function (CTF) correction for the tilt images. Tomograms were then reconstructed in IMOD using the CTF-corrected and aligned stacks. In total, we collected 11 tomograms and selected those that have the following two characteristics: 1) a cell-like shape (i.e., the lamella spans the length of the cell), and 2) least amount of iced dirt and crystalline ice.</div></section><section id="sec-4-4-9"><h4>Antibiotic treatments</h4><div id="p0360" role="paragraph"><i>E.&nbsp;coli</i> cells (CJW7324) were grown at 37°C in M9&nbsp;minimal medium supplemented with glycerol (0.2%), casamino acids (0.1%) and thiamine (1&nbsp;μg/ml). Exponentially growing cells were stained with SYTO RNASelect (Invitrogen) with a final concentration of 500&nbsp;nM for 15&nbsp;min. SYTO RNASelect was shown to bind more specifically to the RNA (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Chong et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Chong, S. ∙ Chen, C. ∙ Ge, H. ...</div><div><strong>Mechanism of transcriptional bursting in bacteria</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:314-326</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.05.038&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.05.038&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84904582175" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25036631/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.05.038&amp;pmid=25036631" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), even in the presence of DNA (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Kannemeier et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Kannemeier, C. ∙ Shibamiya, A. ∙ Nakazawa, F. ...</div><div><strong>Extracellular RNA constitutes a natural procoagulant cofactor in blood coagulation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:6388-6393</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0608647104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34447322451" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17405864/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0608647104&amp;pmid=17405864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were incubated with SYTO RNASelect prior to the addition of Rif (300&nbsp;μg/ml, Sigma-Aldrich) and Ksg (5&nbsp;mg/ml, Sigma-Aldrich) to separate cultures because no cell staining was observed when RNASelect was added after drug treatment, presumably due to a change in membrane permeability for the dye. Our RNASelect staining protocol did not significantly affect the NC ratios, as shown in control experiments (<a id="crosref2450" href="#figs6">Figure&nbsp;S6</a>C). The antibiotic concentrations were selected to match the concentrations used in a previous study (<span class="dropBlock reference-citations"><a id="body-ref-sref7-14" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-14" aria-controls="sref7-14">Bakshi et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref7-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The nucleoid was stained with DAPI for 15&nbsp;min prior to imaging.</div></section><section id="sec-4-4-10"><h4>Time lapse experiment <i>E.&nbsp;coli</i> cells on Ksg-containing agarose pads</h4><div id="p0365" role="paragraph"><i>E.&nbsp;coli</i> cells (CJW7323) were grown at 37°C in M9&nbsp;minimal medium supplemented with glycerol (0.2%), casamino acids (0.1%), thiamine (1&nbsp;μg/ml). Exponentially growing cells were spotted onto the agarose pad containing 5&nbsp;mg/ml Ksg in addition to the growth medium, and images were taken with neutral-density filters every minute.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>Quantification of cell and nucleoid characteristics</h4><div id="p0370" role="paragraph">Cell characteristics (cell size, nucleoid size, fluorescence signal intensities etc.) were obtained by analyzing phase and fluorescence images using the software package Oufti (<span class="dropBlock reference-citations"><a id="body-ref-sref101-3" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-3" aria-controls="sref101-3">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and further processed in MATLAB. All sample statistics (e.g., mean, standard deviation) were also calculated in MATLAB using built functions mean and std.</div></section><section id="sec-4-5-2"><h4>Calculation of correlation coefficients</h4><div id="p0375" role="paragraph">Correlation coefficients (Spearman’s <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho" tabindex="0" ctxtmenu_counter="110" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="rho"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) between variables (i.e., DAPI pixel intensity versus GFP pixel intensity, NC ratio versus RNASelect fluorescence density) were calculated using the built-in MATLAB function corr.</div></section><section id="sec-4-5-3"><h4>Estimation of the DNA Kuhn length</h4><div id="p0380" role="paragraph">The Kuhn length of a polymer, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="111" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, is equivalent to twice its persistence length, which is a measure of rigidity. DNA is a semiflexible polymer. Its rigidity stems from base stacking interactions within the DNA duplex and electrostatic repulsion among phosphate groups in the DNA backbone. The persistence length of the DNA is often assumed to be 50&nbsp;nm (i.e., Kuhn length <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="112" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 100&nbsp;nm). However, <i>in&nbsp;vitro</i> measurements of DNA rigidity indicate that the presence of multivalent cations, including Mg<sup>2+</sup>, the most abundant divalent cation in <i>E.&nbsp;coli</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref2-4" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-4" aria-controls="sref2-4">Alatossava et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref2-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alatossava, T. ∙ Jütte, H. ∙ Kuhn, A. ...</div><div><strong>Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.1.413-419.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2984182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.1.413-419.1985&amp;pmid=2984182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-4" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-4" aria-controls="sref26-4">Cayley et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref26-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cayley, S. ∙ Lewis, B.A. ∙ Guttman, H.J. ...</div><div><strong>Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1991; <strong>222</strong>:281-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(91)90212-O" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026409841" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1960728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2891%2990212-O&amp;pmid=1960728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Kuhn and Kellenberger, 1985</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Kuhn, A. ∙ Kellenberger, E.</div><div><strong>Productive phage infection in Escherichia coli with reduced internal levels of the major cations</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>163</strong>:906-912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.163.3.906-912.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3161872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.163.3.906-912.1985&amp;pmid=3161872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-4" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-4" aria-controls="sref85-4">Lusk et&nbsp;al., 1968</a><div class="dropBlock__holder" data-db-target-of="sref85-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Lusk, J.E. ∙ Williams, R.J.P. ∙ Kennedy, E.P.</div><div><strong>Magnesium and the growth of Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1968; <strong>243</strong>:2618-2624</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4968384/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2993417-4&amp;pmid=4968384" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-3" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-3" aria-controls="sref95-3">Moncany and Kellenberger, 1981</a><div class="dropBlock__holder" data-db-target-of="sref95-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Moncany, M.L.J. ∙ Kellenberger, E.</div><div><strong>High magnesium content of Escherichia coli B</strong></div><div><em>Experientia.</em> 1981; <strong>37</strong>:846-847</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF01985672" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0019401975" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7026272/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF01985672&amp;pmid=7026272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), can decrease the DNA persistence length down to 25-35&nbsp;nm (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Baumann et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Baumann, C.G. ∙ Smith, S.B. ∙ Bloomfield, V.A. ...</div><div><strong>Ionic effects on the elasticity of single DNA molecules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:6185-6190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.12.6185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030947539" target="_blank" aria-label="Open Scopus in new tab">Scopus (771)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9177192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.12.6185&amp;pmid=9177192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-3" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-3" aria-controls="sref87-3">Mantelli et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref87-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Mantelli, S. ∙ Muller, P. ∙ Harlepp, S. ...</div><div><strong>Conformational analysis and estimation of the persistence length of DNA using atomic force microscopy in solution</strong></div><div><em>Soft Matter.</em> 2011; <strong>7</strong>:3412-3416</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c0sm01160f" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955812312" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc0sm01160f" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109-3" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-3" aria-controls="sref109-3">Porschke, 1986</a><div class="dropBlock__holder" data-db-target-of="sref109-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Porschke, D.</div><div><strong>Structure and dynamics of double helices in solution: modes of DNA bending</strong></div><div><em>J.&nbsp;Biomol. Struct. Dyn.</em> 1986; <strong>4</strong>:373-389</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/07391102.1986.10506356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2908425/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F07391102.1986.10506356&amp;pmid=2908425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref110-3" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" aria-label="Reference 1991" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-3" aria-controls="sref110-3">1991</a><div class="dropBlock__holder" data-db-target-of="sref110-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Porschke, D.</div><div><strong>Persistence length and bending dynamics of DNA from electrooptical measurements at high salt concentrations</strong></div><div><em>Biophys. Chem.</em> 1991; <strong>40</strong>:169-179</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(91)87006-Q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0025911729" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1653052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2891%2987006-Q&amp;pmid=1653052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Given the millimolar concentration of Mg<sup>2+</sup> in the <i>E.&nbsp;coli</i> cytoplasm (<span class="dropBlock reference-citations"><a id="body-ref-sref2-5" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-5" aria-controls="sref2-5">Alatossava et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref2-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Alatossava, T. ∙ Jütte, H. ∙ Kuhn, A. ...</div><div><strong>Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.1.413-419.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2984182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.1.413-419.1985&amp;pmid=2984182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26-5" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-5" aria-controls="sref26-5">Cayley et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref26-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Cayley, S. ∙ Lewis, B.A. ∙ Guttman, H.J. ...</div><div><strong>Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1991; <strong>222</strong>:281-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(91)90212-O" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026409841" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1960728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2891%2990212-O&amp;pmid=1960728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-5" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-5" aria-controls="sref85-5">Lusk et&nbsp;al., 1968</a><div class="dropBlock__holder" data-db-target-of="sref85-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Lusk, J.E. ∙ Williams, R.J.P. ∙ Kennedy, E.P.</div><div><strong>Magnesium and the growth of Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1968; <strong>243</strong>:2618-2624</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4968384/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2993417-4&amp;pmid=4968384" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="113" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is approximated to be 60&nbsp;nm (i.e., persistence length ∼30&nbsp;nm) inside cells.</div></section><section id="sec-4-5-4"><h4>Estimation of the average DNA concentration within nucleoids of <i>E.&nbsp;coli</i> in the B period</h4><div id="p0385" role="paragraph"><i>E.&nbsp;coli</i> strain CJW6324 was grown at 37°C in M9&nbsp;minimal medium supplemented with glycerol (0.2%). To estimate the average DNA concentration within the nucleoid regions, B-period cells (with a single chromosome) were first identified using a method based on cell size and the localization of the SeqA DNA replication marker (<span class="dropBlock reference-citations"><a id="body-ref-sref50-18" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-18" aria-controls="sref50-18">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). During DNA replication (C period), SeqA binds the newly synthesized hemi-methylated DNA (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Brendler et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Brendler, T. ∙ Abeles, A. ∙ Austin, S.</div><div><strong>A protein that binds to the P1 origin core and the oriC 13mer region in a methylation-specific fashion is the product of the host seqA gene</strong></div><div><em>EMBO J.</em> 1995; <strong>14</strong>:4083-4089</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1995.tb00080.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7664748/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1995.tb00080.x&amp;pmid=7664748" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Lu et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Lu, M. ∙ Campbell, J.L. ∙ Boye, E. ...</div><div><strong>SeqA: a negative modulator of replication initiation in E.&nbsp;coli</strong></div><div><em>Cell.</em> 1994; <strong>77</strong>:413-426</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2894%2990156-2&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2894%2990156-2&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0028219947" target="_blank" aria-label="Open Scopus in new tab">Scopus (403)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8011018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2894%2990156-2&amp;pmid=8011018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref130" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130" aria-controls="sref130">Slater et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref130"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Slater, S. ∙ Wold, S. ∙ Lu, M. ...</div><div><strong>E.&nbsp;coli SeqA protein binds oriC in two different methyl-modulated reactions appropriate to its roles in DNA replication initiation and origin sequestration</strong></div><div><em>Cell.</em> 1995; <strong>82</strong>:927-936</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2895%2990272-4&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2895%2990272-4&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029147297" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7553853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2895%2990272-4&amp;pmid=7553853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref149" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149" aria-controls="sref149">Waldminghaus et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref149"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">148.</div><div>Waldminghaus, T. ∙ Weigel, C. ∙ Skarstad, K.</div><div><strong>Replication fork movement and methylation govern SeqA binding to the Escherichia coli chromosome</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:5465-5476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks187" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84863227289" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22373925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks187&amp;pmid=22373925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This property leads to the formation of fluorescent foci in cells actively replicating DNA (<a id="crosref2550" href="#fig1">Figure&nbsp;1</a>B), as shown before (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Adiciptaningrum et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Adiciptaningrum, A. ∙ Osella, M. ∙ Moolman, M.C. ...</div><div><strong>Stochasticity and homeostasis in the E.&nbsp;coli replication and division cycle</strong></div><div><em>Sci. Rep.</em> 2015; <strong>5</strong>:18261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep18261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26671779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep18261&amp;pmid=26671779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref50-19" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-19" aria-controls="sref50-19">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Helgesen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Helgesen, E. ∙ Fossum-Raunehaug, S. ∙ Sætre, F. ...</div><div><strong>Dynamic Escherichia coli SeqA complexes organize the newly replicated DNA at a considerable distance from the replisome</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:2730-2743</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84937538742" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv146&amp;pmid=25722374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Molina and Skarstad, 2004</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Molina, F. ∙ Skarstad, K.</div><div><strong>Replication fork and SeqA focus distributions in Escherichia coli suggest a replication hyperstructure dependent on nucleotide metabolism</strong></div><div><em>Mol. Microbiol.</em> 2004; <strong>52</strong>:1597-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2004.04097.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-3042513379" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15186411/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2004.04097.x&amp;pmid=15186411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref150" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150" aria-controls="sref150">Wallden et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref150"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">149.</div><div>Wallden, M. ∙ Fange, D. ∙ Lundius, E.G. ...</div><div><strong>The Synchronization of Replication and Division Cycles in Individual E.&nbsp;coli Cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:729-739</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2016.06.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2016.06.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84979642298" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27471967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.052&amp;pmid=27471967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In contrast, SeqA-mCherry displays a diffuse distribution in cells before and after DNA replication (i.e., during B and D periods, respectively) (<a id="crosref2580" href="#fig1">Figure&nbsp;1</a>B). We used these changes in spatial distribution of SeqA-mCherry signal and the knowledge that cells grow in size during the cell cycle to classify cells by cell-cycle period (<a id="crosref2585" href="#fig1">Figure&nbsp;1</a>C). Once cells with a single chromosome were identified, we calculated their DNA mass. Given that the average molecular weight of a nucleotide base pair is 650 g/mol (<span class="dropBlock reference-citations"><a id="body-ref-sref77-4" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-4" aria-controls="sref77-4">Lee et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref77-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Lee, C.K. ∙ Araki, N. ∙ Sowersby, D.S. ...</div><div><strong>Factors affecting chemical-based purification of DNA from Saccharomyces cerevisiae</strong></div><div><em>Yeast.</em> 2012; <strong>29</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/yea.1918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84856594171" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fyea.1918&amp;pmid=22134898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104-4" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-4" aria-controls="sref104-4">Peale et&nbsp;al., 1989</a><div class="dropBlock__holder" data-db-target-of="sref104-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Peale, Jr., F.V. ∙ Ishibe, Y. ∙ Klinge, C.M. ...</div><div><strong>Rapid purification of the estrogen receptor by sequence-specific DNA affinity chromatography</strong></div><div><em>Biochemistry.</em> 1989; <strong>28</strong>:8671-8675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00448a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0024450125" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2605215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00448a001&amp;pmid=2605215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref115-4" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-4" aria-controls="sref115-4">Ratilainen et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref115-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Ratilainen, T. ∙ Lincoln, P. ∙ Nordén, B.</div><div><strong>A simple model for gene targeting</strong></div><div><em>Biophys. J.</em> 2001; <strong>81</strong>:2876-2885</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11606298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2801%2975928-0&amp;pmid=11606298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the <i>E.&nbsp;coli</i> genome size is about 4.6 million base pairs (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Blattner et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Blattner, F.R. ∙ Plunkett, 3rd, G. ∙ Bloch, C.A. ...</div><div><strong>The complete genome sequence of Escherichia coli K-12</strong></div><div><em>Science.</em> 1997; <strong>277</strong>:1453-1462</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.277.5331.1453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-15444350252" target="_blank" aria-label="Open Scopus in new tab">Scopus (5721)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9278503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.277.5331.1453&amp;pmid=9278503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the total DNA mass in a single chromosome is around 5 femtograms. The nucleoid volume was estimated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper V Subscript n u c l e o i d Baseline equals upper V Subscript c e l l Baseline left parenthesis upper A Subscript n u c l e o i d Baseline divided by upper A Subscript c e l l Baseline right parenthesis Superscript 3 divided by 2" tabindex="0" ctxtmenu_counter="114" style="font-size: 130%;"><mjx-math data-semantic-structure="(69 (17 0 (16 1 9 2 10 3 11 4 12 5 13 6 14 7 15 8)) 18 (68 (28 19 (27 20 24 21 25 22 26 23)) 67 (66 (61 29 (59 (47 30 (46 31 39 32 40 33 41 34 42 35 43 36 44 37 45 38)) 48 (58 49 (57 50 54 51 55 52 56 53))) 60) (65 62 63 64))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="69" data-semantic-children="17,68" data-semantic-content="18" data-semantic-owns="17 18 68" data-semantic-speech="upper V Subscript n u c l e o i d Baseline equals upper V Subscript c e l l Baseline left parenthesis upper A Subscript n u c l e o i d Baseline divided by upper A Subscript c e l l Baseline right parenthesis Superscript 3 divided by 2"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="0,16" data-semantic-parent="69" data-semantic-owns="0 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="17"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="1,2,3,4,5,6,7,8" data-semantic-content="9,10,11,12,13,14,15" data-semantic-parent="17" data-semantic-owns="1 9 2 10 3 11 4 12 5 13 6 14 7 15 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="16"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="16"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="16"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="16"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="16"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="16"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="16"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="18" data-semantic-parent="69" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="68" data-semantic-children="28,66" data-semantic-content="67" data-semantic-parent="69" data-semantic-owns="28 67 66"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="19,27" data-semantic-parent="68" data-semantic-owns="19 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="28"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-children="20,21,22,23" data-semantic-content="24,25,26" data-semantic-parent="28" data-semantic-owns="20 24 21 25 22 26 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="27"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="27"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="27"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="27"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="67" data-semantic-parent="68" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="66" data-semantic-children="61,65" data-semantic-parent="68" data-semantic-owns="61 65"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="61" data-semantic-children="59" data-semantic-content="29,60" data-semantic-parent="66" data-semantic-owns="29 59 60"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="29" data-semantic-parent="61" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="59" data-semantic-children="47,58" data-semantic-content="48" data-semantic-parent="61" data-semantic-owns="47 48 58"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="47" data-semantic-children="30,46" data-semantic-parent="59" data-semantic-owns="30 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="47"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="46" data-semantic-children="31,32,33,34,35,36,37,38" data-semantic-content="39,40,41,42,43,44,45" data-semantic-parent="47" data-semantic-owns="31 39 32 40 33 41 34 42 35 43 36 44 37 45 38"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="46"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="46"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="46"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="46"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="46"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="46"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="44" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="46"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="46"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="48" data-semantic-parent="59" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="58" data-semantic-children="49,57" data-semantic-parent="59" data-semantic-owns="49 57"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="58"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="57" data-semantic-children="50,51,52,53" data-semantic-content="54,55,56" data-semantic-parent="58" data-semantic-owns="50 54 51 55 52 56 53"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="57"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="54" data-semantic-parent="57" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="57"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="57" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="57"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="56" data-semantic-parent="57" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="57"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="60" data-semantic-parent="61" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.477em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="65" data-semantic-children="62,64" data-semantic-content="63" data-semantic-parent="66" data-semantic-owns="62 63 64"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="62" data-semantic-parent="65"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="63" data-semantic-parent="65" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="65"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper V Subscript n u c l e o i d" tabindex="0" ctxtmenu_counter="115" style="font-size: 130%;"><mjx-math data-semantic-structure="(17 0 (16 1 9 2 10 3 11 4 12 5 13 6 14 7 15 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="0,16" data-semantic-owns="0 16" data-semantic-speech="upper V Subscript n u c l e o i d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="17"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="1,2,3,4,5,6,7,8" data-semantic-content="9,10,11,12,13,14,15" data-semantic-parent="17" data-semantic-owns="1 9 2 10 3 11 4 12 5 13 6 14 7 15 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="16"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="16"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="16"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="16"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="16"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="16"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="16"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper V Subscript c e l l" tabindex="0" ctxtmenu_counter="116" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="upper V Subscript c e l l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A Subscript n u c l e o i d" tabindex="0" ctxtmenu_counter="117" style="font-size: 130%;"><mjx-math data-semantic-structure="(17 0 (16 1 9 2 10 3 11 4 12 5 13 6 14 7 15 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="0,16" data-semantic-owns="0 16" data-semantic-speech="upper A Subscript n u c l e o i d"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="17"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="1,2,3,4,5,6,7,8" data-semantic-content="9,10,11,12,13,14,15" data-semantic-parent="17" data-semantic-owns="1 9 2 10 3 11 4 12 5 13 6 14 7 15 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="16"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="16"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="16"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="16"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="16"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="16"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="16"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A Subscript c e l l" tabindex="0" ctxtmenu_counter="118" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="upper A Subscript c e l l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are the volume and cross-sectional area of the nucleoid and cell. The cell volume was calculated by accumulating the volume of cylindrical slices along the cell center line. Each cylindrical slice was defined by the rotation of consecutive pairs of points in the cell outlines constructed by the&nbsp;Oufti software (<span class="dropBlock reference-citations"><a id="body-ref-sref101-4" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-4" aria-controls="sref101-4">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To construct the nucleoid outlines, Oufti first convolves the background-subtracted fluorescence image with a Laplacian of Gaussian. The convolution product was up-sampled using a cubic spline. The up-sampled pixels with zero-crossing components are then connected to form polygons that define the nucleoid contours (<span class="dropBlock reference-citations"><a id="body-ref-sref101-5" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-5" aria-controls="sref101-5">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The nucleoid and cell areas were calculated by summing up the areas of the polygons defined by the nucleoid and cell outlines. The power 3/2 was used to convert the estimated nucleoid area fraction into a volume fraction. The nucleoid volume estimation was performed at the single-cell level, as was the DNA concentration calculation, which was simply a division between the estimated DNA mass and the nucleoid volume. Note that the exclusion of the largest GFP-μNS probes (150&nbsp;nm) from the nucleoid gave us an independent method to estimate the nucleocytoplasmic (NC) ratio <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper A Subscript n u c l e o i d Baseline divided by upper A Subscript c e l l Baseline right parenthesis" tabindex="0" ctxtmenu_counter="119" style="font-size: 130%;"><mjx-math data-semantic-structure="(32 0 (30 (18 1 (17 2 10 3 11 4 12 5 13 6 14 7 15 8 16 9)) 19 (29 20 (28 21 25 22 26 23 27 24))) 31)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="32" data-semantic-children="30" data-semantic-content="0,31" data-semantic-owns="0 30 31" data-semantic-speech="left parenthesis upper A Subscript n u c l e o i d Baseline divided by upper A Subscript c e l l Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="30" data-semantic-children="18,29" data-semantic-content="19" data-semantic-parent="32" data-semantic-owns="18 19 29"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="1,17" data-semantic-parent="30" data-semantic-owns="1 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="18"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-children="2,3,4,5,6,7,8,9" data-semantic-content="10,11,12,13,14,15,16" data-semantic-parent="18" data-semantic-owns="2 10 3 11 4 12 5 13 6 14 7 15 8 16 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="17"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="17"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="17"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="17"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="17"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="17"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="17"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="19" data-semantic-parent="30" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="29" data-semantic-children="20,28" data-semantic-parent="30" data-semantic-owns="20 28"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="29"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="28" data-semantic-children="21,22,23,24" data-semantic-content="25,26,27" data-semantic-parent="29" data-semantic-owns="21 25 22 26 23 27 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="28"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="28"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="28"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="28"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="31" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Based on the cumulative distribution function of the GFP-μNS localizations, cell regions with the highest 30% localization density was considered as the cell pole and periphery. The total area of these regions was subtracted from the cell area to obtain the estimated nucleoid area. Using this method, we estimated that NC ratio&nbsp;= 0.67, which is close to the value obtained when the nucleoid area is defined by the Oufti outline of the DAPI staining of the cell. To verify that B-period cells are transcriptionally active, we used a strain (CJW7039) in which ribosomes are fluorescently labeled. CJW7039 cells were grown at 37°C in M9&nbsp;minimal medium supplemented with glycerol (0.2%). Following DAPI staining and imaging, we did not see any large differences in nucleoid exclusion for ribosomes between cells in B, C and D periods as shown by the strongly overlapping distribution of the SCF values between nucleoid and ribosomes (<a id="crosref2620" href="#figs1">Figure&nbsp;S1</a>C). The SCF measures the pixel correlation between signals of fluorescently labeled nucleoid and ribosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref50-20" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-20" aria-controls="sref50-20">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Positive values represent mixing between the DNA and ribosomes, and negative values indicate that ribosomes are enriched outside the nucleoid. The fact that the SCF values in B-period cells are primarily negative indicates that polysomes are present and thus that transcription is ongoing. If transcription was not ongoing, the SCF values would be positive, as observed when transcription is inhibited by rifampicin (<a id="crosref2630" href="#figs1">Figure&nbsp;S1</a>C).</div></section><section id="sec-4-5-5"><h4>Estimation of the average DNA concentration within nucleoids of <i>E.&nbsp;coli</i> in nutrient-rich conditions</h4><div id="p0390" role="paragraph">Bulk studies using the diphenylamine colorimetric assay and cell counting reported that the average DNA mass per cell under nutrient-rich conditions, glucose, glucose plus casamino acids and rich defined medium plus glucose as carbon sources, were around 10, 14 and 30 fg, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref9-3" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-3" aria-controls="sref9-3">Basan et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref9-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Basan, M. ∙ Zhu, M. ∙ Dai, X. ...</div><div><strong>Inflating bacterial cells by increased protein synthesis</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156178" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84945905016" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26519362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156178&amp;pmid=26519362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The average cell sizes among these growth conditions (i.e., 2.32, 3.33, 6.60&nbsp;μm<sup>3</sup>, respectively) were also shown to scale linearly with the average DNA mass per cell (<span class="dropBlock reference-citations"><a id="body-ref-sref9-4" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-4" aria-controls="sref9-4">Basan et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref9-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Basan, M. ∙ Zhu, M. ∙ Dai, X. ...</div><div><strong>Inflating bacterial cells by increased protein synthesis</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156178" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84945905016" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26519362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156178&amp;pmid=26519362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Taking into consideration of the strong scaling between the nucleoid size and cell size (<span class="dropBlock reference-citations"><a id="body-ref-sref50-21" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-21" aria-controls="sref50-21">Gray et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref50-21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we estimated the average DNA concentration within the nucleoid to be 7.3, 7.1 and 7.7&nbsp;mg/ml under these three nutrient-rich growth conditions. Therefore, when taken together, the DNA concentration within the nucleoid among these nutrient-rich conditions is 7.4 ± 0.2&nbsp;mg/ml.</div></section><section id="sec-4-5-6"><h4>Estimation of GFP-μNS particle sizes</h4><div id="p0395" role="paragraph">The size estimation of the GFP- μNS particles was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref102-6" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-6" aria-controls="sref102-6">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This method establishes a relation between the fluorescence intensity <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper I right parenthesis" tabindex="0" ctxtmenu_counter="120" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis upper I right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of GFP-μNS particles and their absolute sizes <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis d right parenthesis" tabindex="0" ctxtmenu_counter="121" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis d right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. The particle fluorescence intensity <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper I right parenthesis" tabindex="0" ctxtmenu_counter="122" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis upper I right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> was calculated by integrating the volume of the fitting 2D Gaussian distribution that was used to determine the particle location: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I equals 2 pi upper A sigma squared" tabindex="0" ctxtmenu_counter="123" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 1 (11 2 8 3 9 4 10 (7 5 6)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="12" data-semantic-children="0,11" data-semantic-content="1" data-semantic-owns="0 1 11" data-semantic-speech="upper I equals 2 pi upper A sigma squared"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="12"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="12" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="2,3,4,7" data-semantic-content="8,9,10" data-semantic-parent="12" data-semantic-owns="2 8 3 9 4 10 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="11" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A" tabindex="0" ctxtmenu_counter="124" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper A"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="sigma" tabindex="0" ctxtmenu_counter="125" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="sigma"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> are the height and width of the fitting distribution, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref131-3" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-3" aria-controls="sref131-3">Sliusarenko et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref131-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Sliusarenko, O. ∙ Heinritz, J. ∙ Emonet, T. ...</div><div><strong>High-throughput, subpixel precision analysis of bacterial morphogenesis and intracellular spatio-temporal dynamics</strong></div><div><em>Mol. Microbiol.</em> 2011; <strong>80</strong>:612-627</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2011.07579.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955024764" target="_blank" aria-label="Open Scopus in new tab">Scopus (340)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21414037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2011.07579.x&amp;pmid=21414037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Assuming the total fluorescence intensity of a particle is proportional to its volume, and the volume is a cubic function of its size (e.g., for a sphere), we further calculated the cubic root of the fluorescence intensity: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="RootIndex 3 StartRoot upper I EndRoot" tabindex="0" ctxtmenu_counter="126" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 1 0)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mroot data-semantic-type="root" data-semantic-role="unknown" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="1 0" data-semantic-speech="RootIndex 3 StartRoot upper I EndRoot"><mjx-root style="vertical-align: 0.47em; width: 0px;"><mjx-mn size="ss" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2" style="padding-left: 0.524em;"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-root><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.166em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-box></mjx-sqrt></mjx-mroot></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0400" role="paragraph">The absolute sizes of GFP-μNS particles were estimated as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref102-7" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-7" aria-controls="sref102-7">Parry et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, cells expressing GFP-μNS were lysed with T7 phages, resulting in the release of the GFP-μNS particles in solution. Cell debris were removed by centrifugation at 4,000 g. Released GFP-μNS particles were then mixed with fluorescent microspheres of known nominal size (110&nbsp;nm, FluoSpheres F8803, Invitrogen), and their diffusion coefficients were determined using single particle tracking microscopy. We defined the fluorescence-derived relative particle size <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="d Subscript r e l Baseline equals RootIndex 3 StartRoot upper I divided by upper I Subscript f l u o Baseline EndRoot" tabindex="0" ctxtmenu_counter="127" style="font-size: 130%;"><mjx-math data-semantic-structure="(24 (7 0 (6 1 4 2 5 3)) 8 (23 (22 10 11 (21 12 (20 13 17 14 18 15 19 16))) 9))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="24" data-semantic-children="7,23" data-semantic-content="8" data-semantic-owns="7 8 23" data-semantic-speech="d Subscript r e l Baseline equals RootIndex 3 StartRoot upper I divided by upper I Subscript f l u o Baseline EndRoot"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-parent="24" data-semantic-owns="0 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="24" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mroot space="4" data-semantic-type="root" data-semantic-role="unknown" data-semantic-id="23" data-semantic-children="9,22" data-semantic-parent="24" data-semantic-owns="22 9"><mjx-root style="vertical-align: 0.366em; width: 0px;"><mjx-mn size="ss" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="23" style="padding-left: 0.724em;"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-root><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A MM-SO">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.085em;"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="22" data-semantic-children="10,21" data-semantic-content="11" data-semantic-parent="23" data-semantic-owns="10 11 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="22"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="11" data-semantic-parent="22" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="12,20" data-semantic-parent="22" data-semantic-owns="12 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="21"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="13,14,15,16" data-semantic-content="17,18,19" data-semantic-parent="21" data-semantic-owns="13 17 14 18 15 19 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="20"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="20"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-box></mjx-sqrt></mjx-mroot></mjx-mrow></mjx-math></mjx-container> for the GFP-μNS particles, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript f l u o" tabindex="0" ctxtmenu_counter="128" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="upper I Subscript f l u o"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> denotes the average fluorescence intensity of the fluorescent microspheres. Based on the Stokes-Einstein equation, the relative particle size and the corresponding diffusion coefficient is inversely proportional: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper D equals k upper T divided by left parenthesis 3 pi eta d Subscript r e l Baseline right parenthesis" tabindex="0" ctxtmenu_counter="129" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 0 1 (25 (24 2 23 3) 4 (22 5 (20 6 17 7 18 8 19 (16 9 (15 10 13 11 14 12))) 21)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="26" data-semantic-children="0,25" data-semantic-content="1" data-semantic-owns="0 1 25" data-semantic-speech="upper D equals k upper T divided by left parenthesis 3 pi eta d Subscript r e l Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="26"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="26" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="25" data-semantic-children="24,22" data-semantic-content="4" data-semantic-parent="26" data-semantic-owns="24 4 22"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="24" data-semantic-children="2,3" data-semantic-content="23" data-semantic-parent="25" data-semantic-owns="2 23 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="24"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="24"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="4" data-semantic-parent="25" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="22" data-semantic-children="20" data-semantic-content="5,21" data-semantic-parent="25" data-semantic-owns="5 20 21"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="5" data-semantic-parent="22" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="6,7,8,16" data-semantic-content="17,18,19" data-semantic-parent="22" data-semantic-owns="6 17 7 18 8 19 16"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="20"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="20"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="20"><mjx-c class="mjx-c1D702">𝜂</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="9,15" data-semantic-parent="20" data-semantic-owns="9 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="16"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-children="10,11,12" data-semantic-content="13,14" data-semantic-parent="16" data-semantic-owns="10 13 11 14 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="15"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="21" data-semantic-parent="22" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k" tabindex="0" ctxtmenu_counter="130" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="k"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the Boltzmann constant, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T" tabindex="0" ctxtmenu_counter="131" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper T"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the absolute temperature, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="eta" tabindex="0" ctxtmenu_counter="132" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="eta"><mjx-c class="mjx-c1D702">𝜂</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the viscosity of the solution. The absolute sizes of the GFP-μNS particles were estimated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="d equals d Subscript f l u o Baseline upper D Subscript f l u o Baseline alpha divided by upper D" tabindex="0" ctxtmenu_counter="133" style="font-size: 130%;"><mjx-math data-semantic-structure="(29 0 1 (28 (27 (11 2 (10 3 7 4 8 5 9 6)) 25 (21 12 (20 13 17 14 18 15 19 16)) 26 22) 23 24))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="29" data-semantic-children="0,28" data-semantic-content="1" data-semantic-owns="0 1 28" data-semantic-speech="d equals d Subscript f l u o Baseline upper D Subscript f l u o Baseline alpha divided by upper D"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="29"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="29" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="28" data-semantic-children="27,24" data-semantic-content="23" data-semantic-parent="29" data-semantic-owns="27 23 24"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-children="11,21,22" data-semantic-content="25,26" data-semantic-parent="28" data-semantic-owns="11 25 21 26 22"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="2,10" data-semantic-parent="27" data-semantic-owns="2 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="11"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="10" data-semantic-children="3,4,5,6" data-semantic-content="7,8,9" data-semantic-parent="11" data-semantic-owns="3 7 4 8 5 9 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="10"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="10"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="10"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="12,20" data-semantic-parent="27" data-semantic-owns="12 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="21"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="13,14,15,16" data-semantic-content="17,18,19" data-semantic-parent="21" data-semantic-owns="13 17 14 18 15 19 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="20"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="20"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="27"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="23" data-semantic-parent="28" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="28"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="d Subscript f l u o" tabindex="0" ctxtmenu_counter="134" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="d Subscript f l u o"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;= 110&nbsp;nm and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha" tabindex="0" ctxtmenu_counter="135" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="alpha"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="d divided by d Subscript r e l" tabindex="0" ctxtmenu_counter="136" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 (8 3 6 4 7 5)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="d divided by d Subscript r e l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="2,8" data-semantic-parent="10" data-semantic-owns="2 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="3,4,5" data-semantic-content="6,7" data-semantic-parent="9" data-semantic-owns="3 6 4 7 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is a calibration constant, which was found to be 685&nbsp;nm.</div></section><section id="sec-4-5-7"><h4>Construction of probability density maps for nanocage-GFP and GFP-μNS particles</h4><div id="p0405" role="paragraph">To construct the probability density maps of nanocage-GFP particle positions, we first calculated the particle positions (n&nbsp;= 2,500) relative to the cell outlines. All relative particle positions were then aggregated together and plotted as a normalized 2D histogram. For GFP-μNS particles, the particle positions (n&nbsp;= 133,692) were first grouped together (i.e., binned) based on the corresponding estimated particle sizes. For a given group of positions obtained from particles of similar estimated sizes, we calculated the relative positions to the cell outlines. For each group, we then plotted the particle positions as a normalized 2D histogram, representing a probability density map of the positions.</div></section><section id="sec-4-5-8"><h4>Estimation of exclusion scores for nanocage-GFP and GFP-μNS particles</h4><div id="p0410" role="paragraph">Based on the cumulative distribution function of the GFP-μNS localizations in the bin of the largest particle size (150&nbsp;nm), cell regions with the highest 30% localization density was assumed to be the non-nucleoid region (i.e., the cell pole and periphery), and the rest was assumed to be the nucleoid region. The average probability density of localizing the nanocage-GFP and GFP-μNS particles of different sizes within <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis f Subscript i n Baseline right parenthesis" tabindex="0" ctxtmenu_counter="137" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 (6 1 (5 2 4 3)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="0,7" data-semantic-owns="0 6 7" data-semantic-speech="left parenthesis f Subscript i n Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="1,5" data-semantic-parent="8" data-semantic-owns="1 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and outside <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis f Subscript o u t Baseline right parenthesis" tabindex="0" ctxtmenu_counter="138" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 (8 1 (7 2 5 3 6 4)) 9)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="0,9" data-semantic-owns="0 8 9" data-semantic-speech="left parenthesis f Subscript o u t Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="1,7" data-semantic-parent="10" data-semantic-owns="1 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="8" data-semantic-owns="2 5 3 6 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> the assumed nucleoid regions were calculated. The exclusion score was calculated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="1 minus left parenthesis f Subscript i n Baseline divided by f Subscript o u t Baseline right parenthesis" tabindex="0" ctxtmenu_counter="139" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 0 1 (20 2 (18 (8 3 (7 4 6 5)) 9 (17 10 (16 11 14 12 15 13))) 19))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="21" data-semantic-children="0,20" data-semantic-content="1" data-semantic-owns="0 1 20" data-semantic-speech="1 minus left parenthesis f Subscript i n Baseline divided by f Subscript o u t Baseline right parenthesis"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="21"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="21" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="20" data-semantic-children="18" data-semantic-content="2,19" data-semantic-parent="21" data-semantic-owns="2 18 19"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="20" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="18" data-semantic-children="8,17" data-semantic-content="9" data-semantic-parent="20" data-semantic-owns="8 9 17"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="18" data-semantic-owns="3 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="9" data-semantic-parent="18" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="10,16" data-semantic-parent="18" data-semantic-owns="10 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="17"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="11,12,13" data-semantic-content="14,15" data-semantic-parent="17" data-semantic-owns="11 14 12 15 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="16"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="16"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="19" data-semantic-parent="20" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div></section><section id="sec-4-5-9"><h4>Pixel intensity correlation between the DNA and ribosome fluorescence signals</h4><div id="p0415" role="paragraph"><i>E.&nbsp;coli</i> cells (CJW7020) were grown at 37°C in M9&nbsp;minimal medium supplemented with glucose (0.2%), casamino acids (0.1%) and thiamine (1&nbsp;μg/ml). The nucleoid was visualized through DAPI straining. Fluorescence signals from DAPI and L1-msfGFP were acquired. The nucleoid outlines were constructed using objectDetection in the software package Oufti (<span class="dropBlock reference-citations"><a id="body-ref-sref101-6" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-6" aria-controls="sref101-6">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In order to exclusively quantify the pixel intensity correlation <i>within</i> the nucleoid region, the nucleoid outlines were shrunk by 2 pixels from the original boundaries using the built-in MATLAB function polybuffer. The shrunk nucleoid outlines enclosed, on average, about 30% of the inner most pixels defined by the original outlines. Image masks were constructed using the shrunk nucleoid outlines, and pixels from both the DAPI and GFP images were extracted. Correlation scores (Spearman’s <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="rho" tabindex="0" ctxtmenu_counter="140" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="rho"><mjx-c class="mjx-c1D70C">𝜌</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>) were then calculated between the intensities of DAPI and GFP pixels using the built-in MATLAB function corr.</div></section><section id="sec-4-5-10"><h4>Construction of DAPI demographs</h4><div id="p0420" role="paragraph">The signal profiles of DAPI were calculated for individual cells using the software package Oufti (<span class="dropBlock reference-citations"><a id="body-ref-sref101-7" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-7" aria-controls="sref101-7">Paintdakhi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref101-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For each cell, the DAPI signal profile was normalized as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript n o r m Baseline equals left parenthesis upper I minus upper I Subscript m i n Baseline right parenthesis divided by left parenthesis upper I Subscript m a x Baseline minus upper I Subscript m i n Baseline right parenthesis" tabindex="0" ctxtmenu_counter="141" style="font-size: 130%;"><mjx-math data-semantic-structure="(48 (9 0 (8 1 5 2 6 3 7 4)) 10 (47 (24 11 (23 12 13 (21 14 (20 15 18 16 19 17))) 22) 25 (46 26 (44 (34 27 (33 28 31 29 32 30)) 35 (43 36 (42 37 40 38 41 39))) 45)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="48" data-semantic-children="9,47" data-semantic-content="10" data-semantic-owns="9 10 47" data-semantic-speech="upper I Subscript n o r m Baseline equals left parenthesis upper I minus upper I Subscript m i n Baseline right parenthesis divided by left parenthesis upper I Subscript m a x Baseline minus upper I Subscript m i n Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-parent="48" data-semantic-owns="0 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="10" data-semantic-parent="48" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="47" data-semantic-children="24,46" data-semantic-content="25" data-semantic-parent="48" data-semantic-owns="24 25 46"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="24" data-semantic-children="23" data-semantic-content="11,22" data-semantic-parent="47" data-semantic-owns="11 23 22"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="24" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="23" data-semantic-children="12,21" data-semantic-content="13" data-semantic-parent="24" data-semantic-owns="12 13 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="23"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-parent="23" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="14,20" data-semantic-parent="23" data-semantic-owns="14 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="21"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="15,16,17" data-semantic-content="18,19" data-semantic-parent="21" data-semantic-owns="15 18 16 19 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="20"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="20"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="22" data-semantic-parent="24" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="25" data-semantic-parent="47" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="46" data-semantic-children="44" data-semantic-content="26,45" data-semantic-parent="47" data-semantic-owns="26 44 45"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="26" data-semantic-parent="46" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="44" data-semantic-children="34,43" data-semantic-content="35" data-semantic-parent="46" data-semantic-owns="34 35 43"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="27,33" data-semantic-parent="44" data-semantic-owns="27 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="34"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="33" data-semantic-children="28,29,30" data-semantic-content="31,32" data-semantic-parent="34" data-semantic-owns="28 31 29 32 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="33"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="33"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="33"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="35" data-semantic-parent="44" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="43" data-semantic-children="36,42" data-semantic-parent="44" data-semantic-owns="36 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="43"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="42" data-semantic-children="37,38,39" data-semantic-content="40,41" data-semantic-parent="43" data-semantic-owns="37 40 38 41 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="42"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="42"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="42"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="45" data-semantic-parent="46" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I" tabindex="0" ctxtmenu_counter="142" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper I"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript n o r m" tabindex="0" ctxtmenu_counter="143" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="upper I Subscript n o r m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript m i n" tabindex="0" ctxtmenu_counter="144" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 (6 1 4 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-owns="0 6" data-semantic-speech="upper I Subscript m i n"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper I Subscript m a x" tabindex="0" ctxtmenu_counter="145" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 0 (6 1 4 2 5 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="0,6" data-semantic-owns="0 6" data-semantic-speech="upper I Subscript m a x"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D43C">𝐼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.057em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="7" data-semantic-owns="1 4 2 5 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are the raw, normalized, minimum and maximum values of the signal, respectively. The normalized signal profiles were further interpolated linearly using the built-in MATLAB function interp1 to obtain a fixed signal profile length. The signal profiles then sorted based on the corresponding cell length. All signal profiles were then concatenated to construct the final demograph plots.</div></section><section id="sec-4-5-11"><h4>Implementation and quantification of Monte Carlo simulations</h4><div id="p0425" role="paragraph">The simulation of chromosomes was implemented and quantified in Python 3.6, with the additional libraries NumPy (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Harris et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2649-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85091129687" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32939066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2649-2&amp;pmid=32939066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), SciPy (<span class="dropBlock reference-citations"><a id="body-ref-sref148-2" href="#" role="doc-biblioref" data-xml-rid="bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148-2" aria-controls="sref148-2">Virtanen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref148-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Virtanen, P. ∙ Gommers, R. ∙ Oliphant, T.E. ..., SciPy 1.0 Contributors</div><div><strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:261-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0686-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85079119586" target="_blank" aria-label="Open Scopus in new tab">Scopus (3027)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32015543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0686-2&amp;pmid=32015543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Matplotlib (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Hunter, 2007</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics Environment</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/MCSE.2007.55" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34247493236" target="_blank" aria-label="Open Scopus in new tab">Scopus (9418)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FMCSE.2007.55" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and pandas (<span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">McKinney, 2010</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>McKinney, W.</div><div><strong>Data Structures for Statistical Computing in Python</strong></div><div><em>Proceedings of the 9th Python in Science Conference.</em> 2010;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.25080/Majora-92bf1922-00a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.25080%2FMajora-92bf1922-00a" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Specifically, the simulation was implemented using base functions in Python 3.6 and NumPy library. SciPy and pandas were used to analyze all simulations and provide the corresponding statistics (e.g., confidence interval, mean radius of gyration) relevant to the simulated chromosomes. Matplotlib was used to graph the simulated chromosome data.</div></section><section id="sec-4-5-12"><h4>Construction of distance maps for simulated DNA segments</h4><div id="p0430" role="paragraph">The simulated chromosomes were binned by approximately 10 kb (56 DNA segments), and the average positions of the segments within each bin were calculated. The binning was applied to reduce the computational complexity of calculating the pair distance between DNA segments by more than three orders of magnitude. A matrix of pair distances, where the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="146" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>-th row and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="147" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>-th column represent the Euclidean distance between the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="148" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>-th and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="j" tabindex="0" ctxtmenu_counter="149" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="j"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>-th bins, was constructed for every simulated chromosome (obtained from independent simulation runs). A chosen number (10, 100 or 1000) of such matrices were averaged together to generate distance maps representing the average pair distances between the binned segments, which were then visualized as images in which the individual pixels correspond to the distances.</div></section><section id="sec-4-5-13"><h4>Ribosome localization in the cryo-ET tomograms</h4><div id="p0435" role="paragraph">The software package emClarity (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Himes and Zhang, 2018</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Himes, B.A. ∙ Zhang, P.</div><div><strong>emClarity: software for high-resolution cryo-electron tomography and subtomogram averaging</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:955-961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0167-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85055493329" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30349041/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0167-z&amp;pmid=30349041" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to perform the 3D template search to identify ribosomes from the tomograms. A low-pass filtered (∼4-nm resolution) <i>E.&nbsp;coli</i> 70S ribosome structure (<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Fu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Fu, Z. ∙ Indrisiunaite, G. ∙ Kaledhonkar, S. ...</div><div><strong>The structural basis for release-factor activation during translation termination revealed by time-resolved cryogenic electron microscopy</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2579</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10608-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85067294268" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31189921/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10608-z&amp;pmid=31189921" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used as a reference for the template search (<a id="crosref2705" href="#figs4">Figure&nbsp;S4</a>C). Initially, up to 15,000-20,000 sub-tomograms were initially selected from each tomogram. Subsequently, the sub-tomograms (80&nbsp;× 80&nbsp;× 80 voxel) were extracted from the CTF-corrected tomograms with no binning and the 3D classification and alignment were applied to all sub-tomograms by the software package i3 (<span class="dropBlock reference-citations"><a id="body-ref-sref156-2" href="#" role="doc-biblioref" data-xml-rid="bib156" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref156-2" aria-controls="sref156-2">Winkler, 2007</a><div class="dropBlock__holder" data-db-target-of="sref156-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div><div>Winkler, H.</div><div><strong>3D reconstruction and processing of volumetric data in cryo-electron tomography</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2007; <strong>157</strong>:126-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2006.07.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33845310367" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16973379/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2006.07.014&amp;pmid=16973379" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref157-2" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157-2" aria-controls="sref157-2">Winkler et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref157-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Winkler, H. ∙ Zhu, P. ∙ Liu, J. ...</div><div><strong>Tomographic subvolume alignment and subvolume classification applied to myosin V and SIV envelope spikes</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2009; <strong>165</strong>:64-77</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2008.10.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-58349085125" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19032983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2008.10.004&amp;pmid=19032983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with no binning. Sub-tomograms of the particles in the false class were removed (<a id="crosref2720" href="#figs4">Figure&nbsp;S4</a>D). After that, for every pair of localized structures, the separation distance between their center mass was calculated. Two structures in a pair were considered duplicates, if the distance between their centers of mass was smaller than 20&nbsp;nm, and the structure that had a lower cross-correlation score with the global average structure was removed. This process repeated itself until no duplicate structure was found. The program UCSF ChimeraX (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Goddard et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Goddard, T.D. ∙ Huang, C.C. ∙ Meng, E.C. ...</div><div><strong>UCSF ChimeraX: Meeting modern challenges in visualization and analysis</strong></div><div><em>Protein Sci.</em> 2018; <strong>27</strong>:14-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3235" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85039039173" target="_blank" aria-label="Open Scopus in new tab">Scopus (840)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28710774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3235&amp;pmid=28710774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used for the 3D surface rendering of subtomogram average structure of ribosome (<a id="crosref2730" href="#fig4">Figure&nbsp;4</a>C; <a id="crosref2735" href="#mmc4">Video S4</a>). The accuracy of the localization of ribosomes was verified by both visual inspection and comparison of the average structure obtained from the localized ribosomes to the reference structure. Ribosome statistics for each analyzed lamella are provided in <a id="crosref2740" href="#figs5">Figure&nbsp;S5</a>A.</div></section><section id="sec-4-5-14"><h4>Classification of polyribosomes in tomograms</h4><div id="p0440" role="paragraph">The information on ribosome positions and orientations generated by i3 from the previous localization steps were first transferred to RELION (<span class="dropBlock reference-citations"><a id="body-ref-sref12-4" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-4" aria-controls="sref12-4">Bharat and Scheres, 2016</a><div class="dropBlock__holder" data-db-target-of="sref12-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bharat, T.A. ∙ Scheres, S.H.</div><div><strong>Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:2054-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.124" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84992573769" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27685097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.124&amp;pmid=27685097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All sub-tomograms were re-extracted from the CTF-corrected tomograms with a larger size (200&nbsp;× 200&nbsp;× 200 voxel) to include the potential neighboring ribosomes. The CTF volumes in RELION were used for missing wedge compensation and dose weighting, but no phase flip. A spherical mask with ∼80&nbsp;nm diameter was used for the 3D classification. The polyribosome classes were sorted out after ∼20 iterations of classification.</div></section></section><section id="sec-4-6"><h3>Additional resources</h3><div id="p0445" role="paragraph">This study did not generate any additional resources.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0450" role="paragraph">We are grateful to Dr. David Baker for the nanocage-producing strain and Drs. Marco Cosentino-Lagomarsino, Eric Dufresne, Thierry Emonet, Terry Hwa, Suckjoon Jun, Sangjin Kim, and Corey O’Hern for valuable feedback. We thank Dr. Alexandros Papagiannakis, Dr. William Gray, and Yashna Thappeta for sharing strains CJW7323 and CJW7324. We also thank the members of the Jacobs-Wagner laboratory and Dr. Chase Broedersz for fruitful discussions and critical reading of the manuscript. This work was partially supported by the National Institute of Allergy and Infectious Diseases (R01AI087946 and R01AI132818 to J.L.). C.J.-W. is an investigator of the Howard Hughes Medical Institute.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0455" role="paragraph">Conceptualization, Y.X. and C.J.-W.; methodology, Y.X., I.V.S., Y.C., S.K.G., B.R.P., and C.J.-W.; software, Y.X.; formal analysis, Y.X., I.V.S., Y.C., and B.R.P.; investigation, Y.X., I.V.S., Y.C., S.K.G., and B.R.P.; data curation, Y.X., I.V.S., Y.C., S.K.G., and B.R.P.; writing – original draft, Y.X. and C.J.-W.; writing – review &amp; editing, Y.X., I.V.S., Y.C., S.K.G., B.R.P., J.L., and C.J.-W; visualization, Y.X., I.V.S., Y.C., and C.J.-W.; supervision, J.L. and C.J.-W.; project administration, C.J.-W.; funding acquisition, C.J.-W.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0460" role="paragraph">The authors declare no competing interests.</div></section></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Adiciptaningrum, A. ∙ Osella, M. ∙ Moolman, M.C. ...</div><div><strong>Stochasticity and homeostasis in the E.&nbsp;coli replication and division cycle</strong></div><div><em>Sci. Rep.</em> 2015; <strong>5</strong>:18261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/srep18261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26671779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsrep18261&amp;pmid=26671779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Alatossava, T. ∙ Jütte, H. ∙ Kuhn, A. ...</div><div><strong>Manipulation of intracellular magnesium content in polymyxin B nonapeptide-sensitized Escherichia coli by ionophore A23187</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.1.413-419.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2984182/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.1.413-419.1985&amp;pmid=2984182" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Asakura, S. ∙ Oosawa, F.</div><div><strong>On Interaction between Two Bodies Immersed in a Solution of Macromolecules</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1954; <strong>22</strong>:1255-1256</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1740347" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1740347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Axpe, E. ∙ Chan, D. ∙ Offeddu, G.S. ...</div><div><strong>A Multiscale Model for Solute Diffusion in Hydrogels</strong></div><div><em>Macromolecules.</em> 2019; <strong>52</strong>:6889-6897</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.macromol.9b00753" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85072667338" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31579160/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.macromol.9b00753&amp;pmid=31579160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Azam, T.A. ∙ Hiraga, S. ∙ Ishihama, A.</div><div><strong>Two types of localization of the DNA-binding proteins within the Escherichia coli nucleoid</strong></div><div><em>Genes Cells.</em> 2000; <strong>5</strong>:613-626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2443.2000.00350.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0033869982" target="_blank" aria-label="Open Scopus in new tab">Scopus (169)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10947847/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2443.2000.00350.x&amp;pmid=10947847" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Bakshi, S. ∙ Siryaporn, A. ∙ Goulian, M. ...</div><div><strong>Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>85</strong>:21-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2012.08081.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84862765893" target="_blank" aria-label="Open Scopus in new tab">Scopus (265)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22624875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2012.08081.x&amp;pmid=22624875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7-1" title="View in article">7.</a></div><div>Bakshi, S. ∙ Choi, H. ∙ Mondal, J. ...</div><div><strong>Time-dependent effects of transcription- and translation-halting drugs on the spatial distributions of the Escherichia coli chromosome and ribosomes</strong></div><div><em>Mol. Microbiol.</em> 2014; <strong>94</strong>:871-887</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84927148189" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25250841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12805&amp;pmid=25250841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">8.</a></div><div>Basan, M. ∙ Zhu, M. ∙ Dai, X. ...</div><div><strong>Inflating bacterial cells by increased protein synthesis</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:836</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20156178" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84945905016" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26519362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20156178&amp;pmid=26519362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">9.</a></div><div>Baumann, C.G. ∙ Smith, S.B. ∙ Bloomfield, V.A. ...</div><div><strong>Ionic effects on the elasticity of single DNA molecules</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:6185-6190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.12.6185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030947539" target="_blank" aria-label="Open Scopus in new tab">Scopus (771)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9177192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.12.6185&amp;pmid=9177192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">10.</a></div><div>Bernstein, J.A. ∙ Khodursky, A.B. ∙ Lin, P.H. ...</div><div><strong>Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:9697-9702</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.112318199" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0037162469" target="_blank" aria-label="Open Scopus in new tab">Scopus (571)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12119387/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.112318199&amp;pmid=12119387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">11.</a></div><div>Bharat, T.A. ∙ Scheres, S.H.</div><div><strong>Resolving macromolecular structures from electron cryo-tomography data using subtomogram averaging in RELION</strong></div><div><em>Nat. Protoc.</em> 2016; <strong>11</strong>:2054-2065</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2016.124" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84992573769" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27685097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2016.124&amp;pmid=27685097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">12.</a></div><div>Blattner, F.R. ∙ Plunkett, 3rd, G. ∙ Bloch, C.A. ...</div><div><strong>The complete genome sequence of Escherichia coli K-12</strong></div><div><em>Science.</em> 1997; <strong>277</strong>:1453-1462</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.277.5331.1453" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-15444350252" target="_blank" aria-label="Open Scopus in new tab">Scopus (5721)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9278503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.277.5331.1453&amp;pmid=9278503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">13.</a></div><div>Borochov, N. ∙ Eisenberg, H. ∙ Kam, Z.</div><div><strong>Dependence of DNA conformation on the concentration of salt</strong></div><div><em>Biopolymers.</em> 1981; <strong>20</strong>:231-235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.1981.360200116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33327671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.1981.360200116&amp;pmid=33327671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">14.</a></div><div>Boublik, M.</div><div><strong>Structure of the Escherichia coli Ribosome</strong></div><div><div>Nanninga, N. (Editor)</div></div><div><strong>Molecular cytology of Escherichia coli</strong></div><div>Academic Press, 1985</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BoublikStructure+of+the+Escherichia+coli+RibosomeN.NanningaMolecular+cytology+of+Escherichia+coli1985Academic+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">15.</a></div><div>Brangwynne, C.P. ∙ Koenderink, G.H. ∙ MacKintosh, F.C. ...</div><div><strong>Intracellular transport by active diffusion</strong></div><div><em>Trends Cell Biol.</em> 2009; <strong>19</strong>:423-427</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.tcb.2009.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.tcb.2009.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-69549118397" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19699642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tcb.2009.04.004&amp;pmid=19699642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">16.</a></div><div>Brendler, T. ∙ Abeles, A. ∙ Austin, S.</div><div><strong>A protein that binds to the P1 origin core and the oriC 13mer region in a methylation-specific fashion is the product of the host seqA gene</strong></div><div><em>EMBO J.</em> 1995; <strong>14</strong>:4083-4089</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/j.1460-2075.1995.tb00080.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7664748/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fj.1460-2075.1995.tb00080.x&amp;pmid=7664748" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">17.</a></div><div>Broering, T.J. ∙ Parker, J.S.L. ∙ Joyce, P.L. ...</div><div><strong>Mammalian reovirus nonstructural protein microNS forms large inclusions and colocalizes with reovirus microtubule-associated protein micro2 in transfected cells</strong></div><div><em>J.&nbsp;Virol.</em> 2002; <strong>76</strong>:8285-8297</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.76.16.8285-8297.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036316297" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12134034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.76.16.8285-8297.2002&amp;pmid=12134034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">18.</a></div><div>Broering, T.J. ∙ Arnold, M.M. ∙ Miller, C.L. ...</div><div><strong>Carboxyl-proximal regions of reovirus nonstructural protein muNS necessary and sufficient for forming factory-like inclusions</strong></div><div><em>J.&nbsp;Virol.</em> 2005; <strong>79</strong>:6194-6206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.79.10.6194-6206.2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-18144383857" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15858004/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.79.10.6194-6206.2005&amp;pmid=15858004" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">19.</a></div><div>Cabrera, J.E. ∙ Jin, D.J.</div><div><strong>The distribution of RNA polymerase in Escherichia coli is dynamic and sensitive to environmental cues</strong></div><div><em>Mol. Microbiol.</em> 2003; <strong>50</strong>:1493-1505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2958.2003.03805.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0346887121" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14651633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2958.2003.03805.x&amp;pmid=14651633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">20.</a></div><div>Cabrera, J.E. ∙ Cagliero, C. ∙ Quan, S. ...</div><div><strong>Active transcription of rRNA operons condenses the nucleoid in Escherichia coli: examining the effect of transcription on nucleoid structure in the absence of transertion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:4180-4185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.01707-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67649394401" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19395497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.01707-08&amp;pmid=19395497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">21.</a></div><div>Cai, L.H. ∙ Panyukov, S. ∙ Rubinstein, M.</div><div><strong>Mobility of Nonsticky Nanoparticles in Polymer Liquids</strong></div><div><em>Macromolecules.</em> 2011; <strong>44</strong>:7853-7863</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma201583q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-80053467043" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22058573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma201583q&amp;pmid=22058573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">22.</a></div><div>Carn, F. ∙ Boué, F. ∙ Djabourov, M. ...</div><div><strong>Biopolymer folding driven nanoparticle reorganization in bio-nanocomposites</strong></div><div><em>Soft Matter.</em> 2012; <strong>8</strong>:2930-2944</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c2sm06666a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84857701999" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc2sm06666a" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">23.</a></div><div>Casassa, E.F.</div><div><strong>Some statistical properties of flexible ring polymers</strong></div><div><em>J.&nbsp;Polym. Sci. A1.</em> 1965; <strong>3</strong>:605-614</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.F.CasassaSome+statistical+properties+of+flexible+ring+polymersJ.%C2%A0Polym.+Sci.+A131965605614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">24.</a></div><div>Castelnovo, M. ∙ Gelbart, W.M.</div><div><strong>Semiflexible Chain Condensation by Neutral Depleting Agents: Role of Correlations between Depletants</strong></div><div><em>Macromolecules.</em> 2004; <strong>37</strong>:3510-3517</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma0354795" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2442678013" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma0354795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">25.</a></div><div>Cayley, S. ∙ Lewis, B.A. ∙ Guttman, H.J. ...</div><div><strong>Characterization of the cytoplasm of Escherichia coli K-12 as a function of external osmolarity. Implications for protein-DNA interactions in&nbsp;vivo</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1991; <strong>222</strong>:281-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(91)90212-O" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026409841" target="_blank" aria-label="Open Scopus in new tab">Scopus (326)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1960728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2891%2990212-O&amp;pmid=1960728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">26.</a></div><div>Chen, H. ∙ Shiroguchi, K. ∙ Ge, H. ...</div><div><strong>Genome-wide study of mRNA degradation and transcript elongation in Escherichia coli</strong></div><div><em>Mol. Syst. Biol.</em> 2015; <strong>11</strong>:781</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/msb.20145794" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84922888717" target="_blank" aria-label="Open Scopus in new tab">Scopus (105)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25583150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fmsb.20145794&amp;pmid=25583150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">27.</a></div><div>Cherepanov, P.P. ∙ Wackernagel, W.</div><div><strong>Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant</strong></div><div><em>Gene.</em> 1995; <strong>158</strong>:9-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0378-1119(95)00193-A" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029065955" target="_blank" aria-label="Open Scopus in new tab">Scopus (1285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7789817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0378-1119%2895%2900193-A&amp;pmid=7789817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">28.</a></div><div>Chong, S. ∙ Chen, C. ∙ Ge, H. ...</div><div><strong>Mechanism of transcriptional bursting in bacteria</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:314-326</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.05.038&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.05.038&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84904582175" target="_blank" aria-label="Open Scopus in new tab">Scopus (199)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25036631/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.05.038&amp;pmid=25036631" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">29.</a></div><div>Chowdhury, C. ∙ Sinha, S. ∙ Chun, S. ...</div><div><strong>Diverse bacterial microcompartment organelles</strong></div><div><em>Microbiol. Mol. Biol. Rev.</em> 2014; <strong>78</strong>:438-468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MMBR.00009-14" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84907216686" target="_blank" aria-label="Open Scopus in new tab">Scopus (121)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25184561/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMMBR.00009-14&amp;pmid=25184561" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">30.</a></div><div>Cole, J.R. ∙ Olsson, C.L. ∙ Hershey, J.W. ...</div><div><strong>Feedback regulation of rRNA synthesis in Escherichia coli. Requirement for initiation factor IF2</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1987; <strong>198</strong>:383-392</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(87)90288-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2448483/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2887%2990288-9&amp;pmid=2448483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">31.</a></div><div>Cooper, S. ∙ Helmstetter, C.E.</div><div><strong>Chromosome replication and the division cycle of Escherichia coli B/r</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1968; <strong>31</strong>:519-540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(68)90425-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4866337/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2868%2990425-7&amp;pmid=4866337" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">32.</a></div><div>Cooper, E.C. ∙ Johnson, P. ∙ Donald, A.M.</div><div><strong>Probe diffusion in polymer solutions in the dilute/semi-dilute crossover regime: 1. Poly(ethylene oxide)</strong></div><div><em>Polymer (Guildf.).</em> 1991; <strong>32</strong>:2815-2822</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0032-3861(91)90114-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0026368173" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0032-3861%2891%2990114-X" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">33.</a></div><div>Crocker, J.C. ∙ Grier, D.G.</div><div><strong>Methods of Digital Video Microscopy for Colloidal Studies</strong></div><div><em>J.&nbsp;Colloid Interface Sci.</em> 1996; <strong>179</strong>:298-310</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jcis.1996.0217" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029977418" target="_blank" aria-label="Open Scopus in new tab">Scopus (2486)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjcis.1996.0217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">34.</a></div><div>Cunha, S. ∙ Woldringh, C.L. ∙ Odijk, T.</div><div><strong>Polymer-mediated compaction and internal dynamics of isolated Escherichia coli nucleoids</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2001; <strong>136</strong>:53-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.2001.4420" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0035782861" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11858707/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.2001.4420&amp;pmid=11858707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">35.</a></div><div>Dai, X. ∙ Zhu, M. ∙ Warren, M. ...</div><div><strong>Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth</strong></div><div><em>Nat. Microbiol.</em> 2016; <strong>2</strong>:16231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmicrobiol.2016.231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85005950833" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27941827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmicrobiol.2016.231&amp;pmid=27941827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">36.</a></div><div>Dame, R.T. ∙ Rashid, F.M. ∙ Grainger, D.C.</div><div><strong>Chromosome organization in bacteria: mechanistic insights into genome structure and function</strong></div><div><em>Nat. Rev. Genet.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31767998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=31767998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">37.</a></div><div>Dasgupta, B.R. ∙ Tee, S.-Y. ∙ Crocker, J.C. ...</div><div><strong>Microrheology of polyethylene oxide using diffusing wave spectroscopy and single scattering</strong></div><div><em>Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</em> 2002; <strong>65</strong>:051505</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevE.65.051505" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-37649031119" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12059562/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevE.65.051505&amp;pmid=12059562" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39-1" title="View in article">38.</a></div><div>de Gennes, P.G.</div><div><strong>Scaling concepts in polymer physics</strong></div><div>Cornell University Press, 1979</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.G.de+GennesScaling+concepts+in+polymer+physics1979Cornell+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">39.</a></div><div>de Vries, R.</div><div><strong>Depletion-induced instability in protein-DNA mixtures: Influence of protein charge and size</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2006; <strong>125</strong>:014905</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.2209683" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33745802542" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16863331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.2209683&amp;pmid=16863331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">40.</a></div><div>de Vries, R.</div><div><strong>DNA condensation in bacteria: Interplay between macromolecular crowding and nucleoid proteins</strong></div><div><em>Biochimie.</em> 2010; <strong>92</strong>:1715-1721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.biochi.2010.06.024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-78649882848" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20615449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biochi.2010.06.024&amp;pmid=20615449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">41.</a></div><div>Deng, S. ∙ Stein, R.A. ∙ Higgins, N.P.</div><div><strong>Organization of supercoil domains and their reorganization by transcription</strong></div><div><em>Mol. Microbiol.</em> 2005; <strong>57</strong>:1511-1521</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2005.04796.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-25144495578" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16135220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2005.04796.x&amp;pmid=16135220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">42.</a></div><div>Diekmann, S. ∙ Hillen, W. ∙ Morgeneyer, B. ...</div><div><strong>Orientation relaxation of DNA restriction fragments and the internal mobility of the double helix</strong></div><div><em>Biophys. Chem.</em> 1982; <strong>15</strong>:263-270</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(82)80009-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6288135/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2882%2980009-4&amp;pmid=6288135" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">43.</a></div><div>Dillon, S.C. ∙ Dorman, C.J.</div><div><strong>Bacterial nucleoid-associated proteins, nucleoid structure and gene expression</strong></div><div><em>Nat. Rev. Microbiol.</em> 2010; <strong>8</strong>:185-195</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrmicro2261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-76949089832" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20140026/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrmicro2261&amp;pmid=20140026" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">44.</a></div><div>Dorman, C.J.</div><div><strong>DNA supercoiling and transcription in bacteria: a two-way street</strong></div><div><em>BMC Mol. Cell Biol.</em> 2019; <strong>20</strong>:26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12860-019-0211-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85070094224" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31319794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12860-019-0211-6&amp;pmid=31319794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">45.</a></div><div>Dworsky, P. ∙ Schaechter, M.</div><div><strong>Effect of rifampin on the structure and membrane attachment of the nucleoid of Escherichia coli</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1973; <strong>116</strong>:1364-1374</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.116.3.1364-1374.1973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4584813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.116.3.1364-1374.1973&amp;pmid=4584813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">46.</a></div><div>Forchhammer, J. ∙ Lindahl, L.</div><div><strong>Growth rate of polypeptide chains as a function of the cell growth rate in a mutant of Escherichia coli 15</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1971; <strong>55</strong>:563-568</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(71)90337-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0015222649" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4927947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2871%2990337-8&amp;pmid=4927947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#fig4" title="View in article">47.</a></div><div>Fu, Z. ∙ Indrisiunaite, G. ∙ Kaledhonkar, S. ...</div><div><strong>The structural basis for release-factor activation during translation termination revealed by time-resolved cryogenic electron microscopy</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2579</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10608-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85067294268" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31189921/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10608-z&amp;pmid=31189921" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">48.</a></div><div>Goddard, T.D. ∙ Huang, C.C. ∙ Meng, E.C. ...</div><div><strong>UCSF ChimeraX: Meeting modern challenges in visualization and analysis</strong></div><div><em>Protein Sci.</em> 2018; <strong>27</strong>:14-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3235" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85039039173" target="_blank" aria-label="Open Scopus in new tab">Scopus (840)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28710774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3235&amp;pmid=28710774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">49.</a></div><div>Gray, W.T. ∙ Govers, S.K. ∙ Xiang, Y. ...</div><div><strong>Nucleoid Size Scaling and Intracellular Organization of Translation across Bacteria</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1632-1648.e1620</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2019.05.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065873701" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31150626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.017&amp;pmid=31150626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">50.</a></div><div>Guo, M. ∙ Ehrlicher, A.J. ∙ Jensen, M.H. ...</div><div><strong>Probing the stochastic, motor-driven properties of the cytoplasm using force spectrum microscopy</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:822-832</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.06.051&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.06.051&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84907369302" target="_blank" aria-label="Open Scopus in new tab">Scopus (281)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25126787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.06.051&amp;pmid=25126787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">51.</a></div><div>Hadizadeh Yazdi, N. ∙ Guet, C.C. ∙ Johnson, R.C. ...</div><div><strong>Variation of the folding and dynamics of the Escherichia coli chromosome with growth conditions</strong></div><div><em>Mol. Microbiol.</em> 2012; <strong>86</strong>:1318-1333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.12071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84871006430" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23078205/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.12071&amp;pmid=23078205" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">52.</a></div><div>Harris, C.R. ∙ Millman, K.J. ∙ van der Walt, S.J. ...</div><div><strong>Array programming with NumPy</strong></div><div><em>Nature.</em> 2020; <strong>585</strong>:357-362</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2649-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85091129687" target="_blank" aria-label="Open Scopus in new tab">Scopus (800)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32939066/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2649-2&amp;pmid=32939066" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">53.</a></div><div>Helgesen, E. ∙ Fossum-Raunehaug, S. ∙ Sætre, F. ...</div><div><strong>Dynamic Escherichia coli SeqA complexes organize the newly replicated DNA at a considerable distance from the replisome</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:2730-2743</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84937538742" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv146&amp;pmid=25722374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">54.</a></div><div>Henry, J.T. ∙ Crosson, S.</div><div><strong>Chromosome replication and segregation govern the biogenesis and inheritance of inorganic polyphosphate granules</strong></div><div><em>Mol. Biol. Cell.</em> 2013; <strong>24</strong>:3177-3186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E13-04-0182" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84885642867" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23985321/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E13-04-0182&amp;pmid=23985321" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">55.</a></div><div>Hernandez, V.J. ∙ Bremer, H.</div><div><strong>Guanosine tetraphosphate (ppGpp) dependence of the growth rate control of rrnB P1 promoter activity in Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1990; <strong>265</strong>:11605-11614</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2819%2938441-8&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2819%2938441-8&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2114400/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2819%2938441-8&amp;pmid=2114400" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">56.</a></div><div>Himes, B.A. ∙ Zhang, P.</div><div><strong>emClarity: software for high-resolution cryo-electron tomography and subtomogram averaging</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:955-961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0167-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85055493329" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30349041/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0167-z&amp;pmid=30349041" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">57.</a></div><div>Hobot, J.A. ∙ Villiger, W. ∙ Escaig, J. ...</div><div><strong>Shape and fine structure of nucleoids observed on sections of ultrarapidly frozen and cryosubstituted bacteria</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>162</strong>:960-971</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.162.3.960-971.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3922958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.162.3.960-971.1985&amp;pmid=3922958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">58.</a></div><div>Hsia, Y. ∙ Bale, J.B. ∙ Gonen, S. ...</div><div><strong>Design of a hyperstable 60-subunit protein dodecahedron. [corrected]</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:136-139</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977663424" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27309817/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18010&amp;pmid=27309817" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">59.</a></div><div>Hunter, J.D.</div><div><strong>Matplotlib: A 2D Graphics Environment</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:90-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/MCSE.2007.55" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34247493236" target="_blank" aria-label="Open Scopus in new tab">Scopus (9418)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FMCSE.2007.55" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">60.</a></div><div>Jensen, K.F.</div><div><strong>The Escherichia coli K-12 “wild types” W3110 and MG1655 have an rph frameshift mutation that leads to pyrimidine starvation due to low pyrE expression levels</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1993; <strong>175</strong>:3401-3407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.175.11.3401-3407.1993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0027251266" target="_blank" aria-label="Open Scopus in new tab">Scopus (377)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8501045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.175.11.3401-3407.1993&amp;pmid=8501045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">61.</a></div><div>Jeon, C. ∙ Jung, Y. ∙ Ha, B.-Y.</div><div><strong>Effects of molecular crowding and confinement on the spatial organization of a biopolymer</strong></div><div><em>Soft Matter.</em> 2016; <strong>12</strong>:9436-9450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C6SM01184E" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27834427/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC6SM01184E&amp;pmid=27834427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">62.</a></div><div>Joyeux, M.</div><div><strong>Compaction of bacterial genomic DNA: clarifying the concepts</strong></div><div><em>J.&nbsp;Phys. Condens. Matter.</em> 2015; <strong>27</strong>:383001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1088/0953-8984/27/38/383001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84941695861" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26345139/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1088%2F0953-8984%2F27%2F38%2F383001&amp;pmid=26345139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">63.</a></div><div>Joyeux, M.</div><div><strong>In&nbsp;vivo compaction dynamics of bacterial DNA: A fingerprint of DNA/RNA demixing?</strong></div><div><em>Curr. Opin. Colloid Interface Sci.</em> 2016; <strong>26</strong>:17-27</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.cocis.2016.08.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84988693705" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cocis.2016.08.005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">64.</a></div><div>Jun, S.</div><div><strong>Chromosome, cell cycle, and entropy</strong></div><div><em>Biophys. J.</em> 2015; <strong>108</strong>:785-786</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.12.032&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.12.032&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84923241784" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25692581/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2014.12.032&amp;pmid=25692581" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">65.</a></div><div>Kam, Z. ∙ Borochov, N. ∙ Eisenberg, H.</div><div><strong>Dependence of laser light scattering of DNA on NaCl concentration</strong></div><div><em>Biopolymers.</em> 1981; <strong>20</strong>:2671-2690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.1981.360201213" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7034800/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.1981.360201213&amp;pmid=7034800" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">66.</a></div><div>Kannemeier, C. ∙ Shibamiya, A. ∙ Nakazawa, F. ...</div><div><strong>Extracellular RNA constitutes a natural procoagulant cofactor in blood coagulation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:6388-6393</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0608647104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34447322451" target="_blank" aria-label="Open Scopus in new tab">Scopus (369)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17405864/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0608647104&amp;pmid=17405864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">67.</a></div><div>Kellenberger, E. ∙ Ryter, A. ∙ Sechaud, J.</div><div><strong>Electron microscope study of DNA-containing plasms. II. Vegetative and mature phage DNA as compared with normal bacterial nucleoids in different physiological states</strong></div><div><em>J.&nbsp;Biophys. Biochem. Cytol.</em> 1958; <strong>4</strong>:671-678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.4.6.671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13610928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.4.6.671&amp;pmid=13610928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">68.</a></div><div>Kim, J. ∙ Jeon, C. ∙ Jeong, H. ...</div><div><strong>A polymer in a crowded and confined space: effects of crowder size and poly-dispersity</strong></div><div><em>Soft Matter.</em> 2015; <strong>11</strong>:1877-1888</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/C4SM02198C" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535704/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FC4SM02198C&amp;pmid=25535704" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">69.</a></div><div>Kramers, H.A.</div><div><strong>The Behavior of Macromolecules in Inhomogeneous Flow</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1946; <strong>14</strong>:415-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1724163" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1724163" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">70.</a></div><div>Kremer, J.R. ∙ Mastronarde, D.N. ∙ McIntosh, J.R.</div><div><strong>Computer visualization of three-dimensional image data using IMOD</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 1996; <strong>116</strong>:71-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.1996.0013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029879295" target="_blank" aria-label="Open Scopus in new tab">Scopus (3066)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8742726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.1996.0013&amp;pmid=8742726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">71.</a></div><div>Kuhn, A. ∙ Kellenberger, E.</div><div><strong>Productive phage infection in Escherichia coli with reduced internal levels of the major cations</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1985; <strong>163</strong>:906-912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.163.3.906-912.1985" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3161872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.163.3.906-912.1985&amp;pmid=3161872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73-1" title="View in article">72.</a></div><div>Kurland, C.G. ∙ Maaloe, O.</div><div><strong>Regulation of ribosomal and transfer RNA synthesis</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1962; <strong>4</strong>:193-210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-2836(62)80051-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14460744/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-2836%2862%2980051-5&amp;pmid=14460744" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">73.</a></div><div>Le, T.B. ∙ Laub, M.T.</div><div><strong>Transcription rate and transcript length drive formation of chromosomal interaction domain boundaries</strong></div><div><em>EMBO J.</em> 2016; <strong>35</strong>:1582-1595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embj.201593561" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84978298560" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27288403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembj.201593561&amp;pmid=27288403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">74.</a></div><div>Le, T.B. ∙ Imakaev, M.V. ∙ Mirny, L.A. ...</div><div><strong>High-resolution mapping of the spatial organization of a bacterial chromosome</strong></div><div><em>Science.</em> 2013; <strong>342</strong>:731-734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1242059" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84887322004" target="_blank" aria-label="Open Scopus in new tab">Scopus (327)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24158908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1242059&amp;pmid=24158908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">75.</a></div><div>Le Gall, A. ∙ Cattoni, D.I. ∙ Guilhas, B. ...</div><div><strong>Bacterial partition complexes segregate within the volume of the nucleoid</strong></div><div><em>Nat. Commun.</em> 2016; <strong>7</strong>:12107</div><div>12107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms12107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84977271741" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27377966/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms12107&amp;pmid=27377966" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">76.</a></div><div>Lee, C.K. ∙ Araki, N. ∙ Sowersby, D.S. ...</div><div><strong>Factors affecting chemical-based purification of DNA from Saccharomyces cerevisiae</strong></div><div><em>Yeast.</em> 2012; <strong>29</strong>:73-80</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/yea.1918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84856594171" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22134898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fyea.1918&amp;pmid=22134898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">77.</a></div><div>Lewis, P.J. ∙ Thaker, S.D. ∙ Errington, J.</div><div><strong>Compartmentalization of transcription and translation in Bacillus subtilis</strong></div><div><em>EMBO J.</em> 2000; <strong>19</strong>:710-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/19.4.710" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10675340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2F19.4.710&amp;pmid=10675340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">78.</a></div><div>Li, G.W. ∙ Burkhardt, D. ∙ Gross, C. ...</div><div><strong>Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources</strong></div><div><em>Cell.</em> 2014; <strong>157</strong>:624-635</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.02.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2014.02.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84899550455" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24766808/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.02.033&amp;pmid=24766808" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">79.</a></div><div>Libby, E.A. ∙ Roggiani, M. ∙ Goulian, M.</div><div><strong>Membrane protein expression triggers chromosomal locus repositioning in bacteria</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:7445-7450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1109479109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84860828333" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22529375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1109479109&amp;pmid=22529375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">80.</a></div><div>Lioy, V.S. ∙ Cournac, A. ∙ Marbouty, M. ...</div><div><strong>Multiscale Structuring of the E.&nbsp;coli Chromosome by Nucleoid-Associated and Condensin Proteins</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:771-783.e18</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2017.12.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85041112985" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29358050/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.12.027&amp;pmid=29358050" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">81.</a></div><div>Liu, J. ∙ Lin, T. ∙ Botkin, D.J. ...</div><div><strong>Intact flagellar motor of Borrelia burgdorferi revealed by cryo-electron tomography: evidence for stator ring curvature and rotor/C-ring assembly flexion</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2009; <strong>191</strong>:5026-5036</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.00340-09" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-67749093018" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19429612/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.00340-09&amp;pmid=19429612" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">82.</a></div><div>Lopez, P.J. ∙ Marchand, I. ∙ Yarchuk, O. ...</div><div><strong>Translation inhibitors stabilize Escherichia coli mRNAs independently of ribosome protection</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1998; <strong>95</strong>:6067-6072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.95.11.6067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0032568544" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9600918/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.95.11.6067&amp;pmid=9600918" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">83.</a></div><div>Lu, M. ∙ Campbell, J.L. ∙ Boye, E. ...</div><div><strong>SeqA: a negative modulator of replication initiation in E.&nbsp;coli</strong></div><div><em>Cell.</em> 1994; <strong>77</strong>:413-426</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2894%2990156-2&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2894%2990156-2&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0028219947" target="_blank" aria-label="Open Scopus in new tab">Scopus (403)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8011018/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2894%2990156-2&amp;pmid=8011018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">84.</a></div><div>Lusk, J.E. ∙ Williams, R.J.P. ∙ Kennedy, E.P.</div><div><strong>Magnesium and the growth of Escherichia coli</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1968; <strong>243</strong>:2618-2624</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=abstract&amp;site=jbc-site" target="_blank" aria-label="Open Abstract in new tab">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0021-9258%2818%2993417-4&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4968384/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0021-9258%2818%2993417-4&amp;pmid=4968384" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">85.</a></div><div>Ma, J. ∙ Wang, M.D.</div><div><strong>DNA supercoiling during transcription</strong></div><div><em>Biophys. Rev.</em> 2016; <strong>8</strong>:75-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s12551-016-0215-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85003554279" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28275417/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs12551-016-0215-9&amp;pmid=28275417" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">86.</a></div><div>Mantelli, S. ∙ Muller, P. ∙ Harlepp, S. ...</div><div><strong>Conformational analysis and estimation of the persistence length of DNA using atomic force microscopy in solution</strong></div><div><em>Soft Matter.</em> 2011; <strong>7</strong>:3412-3416</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/c0sm01160f" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955812312" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2Fc0sm01160f" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">87.</a></div><div>Marbouty, M. ∙ Cournac, A. ∙ Flot, J.F. ...</div><div><strong>Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms</strong></div><div><em>eLife.</em> 2014; <strong>3</strong>:e03318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.03318" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25517076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.03318&amp;pmid=25517076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">88.</a></div><div>Marbouty, M. ∙ Le Gall, A. ∙ Cattoni, D.I. ...</div><div><strong>Condensin- and Replication-Mediated Bacterial Chromosome Folding and Origin Condensation Revealed by Hi-C and Super-resolution Imaging</strong></div><div><em>Mol. Cell.</em> 2015; <strong>59</strong>:588-602</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.molcel.2015.07.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84939550260" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26295962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2015.07.020&amp;pmid=26295962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">89.</a></div><div>Mason, D.J. ∙ Powelson, D.M.</div><div><strong>Nuclear division as observed in live bacteria by a new technique</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1956; <strong>71</strong>:474-479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.71.4.474-479.1956" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13319263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.71.4.474-479.1956&amp;pmid=13319263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">90.</a></div><div>Mastronarde, D.N.</div><div><strong>Automated electron microscope tomography using robust prediction of specimen movements</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2005; <strong>152</strong>:36-51</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2005.07.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-25644458666" target="_blank" aria-label="Open Scopus in new tab">Scopus (1782)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16182563/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2005.07.007&amp;pmid=16182563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">91.</a></div><div>McKinney, W.</div><div><strong>Data Structures for Statistical Computing in Python</strong></div><div><em>Proceedings of the 9th Python in Science Conference.</em> 2010;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.25080/Majora-92bf1922-00a" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.25080%2FMajora-92bf1922-00a" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">92.</a></div><div>Mohapatra, S. ∙ Weisshaar, J.C.</div><div><strong>Functional mapping of the E.&nbsp;coli translational machinery using single-molecule tracking</strong></div><div><em>Mol. Microbiol.</em> 2018; <strong>110</strong>:262-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85054160957" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30107639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14103&amp;pmid=30107639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">93.</a></div><div>Molina, F. ∙ Skarstad, K.</div><div><strong>Replication fork and SeqA focus distributions in Escherichia coli suggest a replication hyperstructure dependent on nucleotide metabolism</strong></div><div><em>Mol. Microbiol.</em> 2004; <strong>52</strong>:1597-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2004.04097.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-3042513379" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15186411/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2004.04097.x&amp;pmid=15186411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">94.</a></div><div>Moncany, M.L.J. ∙ Kellenberger, E.</div><div><strong>High magnesium content of Escherichia coli B</strong></div><div><em>Experientia.</em> 1981; <strong>37</strong>:846-847</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF01985672" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0019401975" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7026272/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF01985672&amp;pmid=7026272" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">95.</a></div><div>Mondal, J. ∙ Bratton, B.P. ∙ Li, Y. ...</div><div><strong>Entropy-based mechanism of ribosome-nucleoid segregation in E.&nbsp;coli cells</strong></div><div><em>Biophys. J.</em> 2011; <strong>100</strong>:2605-2613</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2011.04.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79960325276" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21641305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2011.04.030&amp;pmid=21641305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">96.</a></div><div>Mutch, K.J. ∙ van Duijneveldt, J.S. ∙ Eastoe, J.</div><div><strong>Colloid-polymer mixtures in the protein limit</strong></div><div><em>Soft Matter.</em> 2007; <strong>3</strong>:155-167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1039/B611137H" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32680259/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1039%2FB611137H&amp;pmid=32680259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">97.</a></div><div>Nicolai, T. ∙ Mandel, M.</div><div><strong>The ionic strength dependence of the second virial coefficient of low molar mass DNA fragments in aqueous solutions</strong></div><div><em>Macromolecules.</em> 1989; <strong>22</strong>:438-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ma00191a080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fma00191a080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">98.</a></div><div>O’Reilly, F.J. ∙ Xue, L. ∙ Graziadei, A. ...</div><div><strong>In-cell architecture of an actively transcribing-translating expressome</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:554-557</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb3758" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85088883448" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32732422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb3758&amp;pmid=32732422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">99.</a></div><div>Odijk, T.</div><div><strong>Osmotic compaction of supercoiled DNA into a bacterial nucleoid</strong></div><div><em>Biophys. Chem.</em> 1998; <strong>73</strong>:23-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0301-4622(98)00115-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0032514107" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9697298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0301-4622%2898%2900115-X&amp;pmid=9697298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101-1" title="View in article">100.</a></div><div>Paintdakhi, A. ∙ Parry, B. ∙ Campos, M. ...</div><div><strong>Oufti: an integrated software package for high-accuracy, high-throughput quantitative microscopy analysis</strong></div><div><em>Mol. Microbiol.</em> 2016; <strong>99</strong>:767-777</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.13264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958061161" target="_blank" aria-label="Open Scopus in new tab">Scopus (146)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26538279/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.13264&amp;pmid=26538279" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102-1" title="View in article">101.</a></div><div>Parry, B.R. ∙ Surovtsev, I.V. ∙ Cabeen, M.T. ...</div><div><strong>The bacterial cytoplasm has glass-like properties and is fluidized by metabolic activity</strong></div><div><em>Cell.</em> 2014; <strong>156</strong>:183-194</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2013.11.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84892828598" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24361104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.11.028&amp;pmid=24361104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-1" title="View in article">102.</a></div><div>Pato, M.L. ∙ Bennett, P.M. ∙ von Meyenburg, K.</div><div><strong>Messenger ribonucleic acid synthesis and degradation in <i>Escherichia coli</i> during inhibition of translation</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1973; <strong>116</strong>:710-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.116.2.710-718.1973" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4583248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.116.2.710-718.1973&amp;pmid=4583248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104-1" title="View in article">103.</a></div><div>Peale, Jr., F.V. ∙ Ishibe, Y. ∙ Klinge, C.M. ...</div><div><strong>Rapid purification of the estrogen receptor by sequence-specific DNA affinity chromatography</strong></div><div><em>Biochemistry.</em> 1989; <strong>28</strong>:8671-8675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00448a001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0024450125" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2605215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00448a001&amp;pmid=2605215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">104.</a></div><div>Pelletier, J. ∙ Halvorsen, K. ∙ Ha, B.Y. ...</div><div><strong>Physical manipulation of the Escherichia coli chromosome reveals its soft nature</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:E2649-E2656</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1208689109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84867042373" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22984156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1208689109&amp;pmid=22984156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106-1" title="View in article">105.</a></div><div>Pettijohn, D.E. ∙ Hecht, R.</div><div><strong>RNA molecules bound to the folded bacterial genome stabilize DNA folds and segregate domains of supercoiling</strong></div><div><em>Cold Spring Harb. Symp. Quant. Biol.</em> 1974; <strong>38</strong>:31-41</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/SQB.1974.038.01.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0016000696" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4598638/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2FSQB.1974.038.01.006&amp;pmid=4598638" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">106.</a></div><div>Phillips, L.A. ∙ Hotham-Iglewski, B. ∙ Franklin, R.M.</div><div><strong>Polyribosomes of Escherichia coli. II. Experiments to determine the in&nbsp;vivo distribution of polysomes, ribosomes and ribosomal subunits</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1969; <strong>45</strong>:23-38</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0022-2836(69)90207-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4186447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0022-2836%2869%2990207-1&amp;pmid=4186447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">107.</a></div><div>Piekarski, G.</div><div><strong>Zytologische Untersuchungen an Paratyphusund Coli Bakterien</strong></div><div><em>Arch. Mikrobiol.</em> 1937; <strong>8</strong>:428-429</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00407212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0000625722" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00407212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">108.</a></div><div>Porschke, D.</div><div><strong>Structure and dynamics of double helices in solution: modes of DNA bending</strong></div><div><em>J.&nbsp;Biomol. Struct. Dyn.</em> 1986; <strong>4</strong>:373-389</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/07391102.1986.10506356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2908425/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F07391102.1986.10506356&amp;pmid=2908425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">109.</a></div><div>Porschke, D.</div><div><strong>Persistence length and bending dynamics of DNA from electrooptical measurements at high salt concentrations</strong></div><div><em>Biophys. Chem.</em> 1991; <strong>40</strong>:169-179</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0301-4622(91)87006-Q" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0025911729" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1653052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0301-4622%2891%2987006-Q&amp;pmid=1653052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111" title="View in article">110.</a></div><div>Postow, L. ∙ Hardy, C.D. ∙ Arsuaga, J. ...</div><div><strong>Topological domain structure of the Escherichia coli chromosome</strong></div><div><em>Genes Dev.</em> 2004; <strong>18</strong>:1766-1779</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1207504" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-3142774839" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15256503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1207504&amp;pmid=15256503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">111.</a></div><div>Qiu, X. ∙ Andresen, K. ∙ Kwok, L.W. ...</div><div><strong>Inter-DNA attraction mediated by divalent counterions</strong></div><div><em>Phys. Rev. Lett.</em> 2007; <strong>99</strong>:038104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.99.038104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34547432557" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17678334/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.99.038104&amp;pmid=17678334" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">112.</a></div><div>Racki, L.R. ∙ Tocheva, E.I. ∙ Dieterle, M.G. ...</div><div><strong>Polyphosphate granule biogenesis is temporally and functionally tied to cell cycle exit during starvation in <i>Pseudomonas aeruginosa</i></strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2017; <strong>114</strong>:E2440-E2449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1615575114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85016092632" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28265086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1615575114&amp;pmid=28265086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">113.</a></div><div>Ramakrishnan, S. ∙ Fuchs, M. ∙ Schweizer, K.S. ...</div><div><strong>Entropy driven phase transitions in colloid–polymer suspensions: Tests of depletion theories</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2002; <strong>116</strong>:2201-2212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1426413" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036470201" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1426413" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115-1" title="View in article">114.</a></div><div>Ratilainen, T. ∙ Lincoln, P. ∙ Nordén, B.</div><div><strong>A simple model for gene targeting</strong></div><div><em>Biophys. J.</em> 2001; <strong>81</strong>:2876-2885</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2FS0006-3495%2801%2975928-0&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11606298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0006-3495%2801%2975928-0&amp;pmid=11606298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">115.</a></div><div>Reyes-Lamothe, R. ∙ Tran, T. ∙ Meas, D. ...</div><div><strong>High-copy bacterial plasmids diffuse in the nucleoid-free space, replicate stochastically and are randomly partitioned at cell division</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:1042-1051</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt918" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84893258556" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24137005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt918&amp;pmid=24137005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117-1" title="View in article">116.</a></div><div>Robinow, C. ∙ Kellenberger, E.</div><div><strong>The bacterial nucleoid revisited</strong></div><div><em>Microbiol. Rev.</em> 1994; <strong>58</strong>:211-232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/mr.58.2.211-232.1994" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7521510/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fmr.58.2.211-232.1994&amp;pmid=7521510" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118-1" title="View in article">117.</a></div><div>Roe, A.J. ∙ McLaggan, D. ∙ Davidson, I. ...</div><div><strong>Perturbation of anion balance during inhibition of growth of Escherichia coli by weak acids</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1998; <strong>180</strong>:767-772</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.180.4.767-772.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9473028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.180.4.767-772.1998&amp;pmid=9473028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119-1" title="View in article">118.</a></div><div>Rubinstein, M.</div><div><strong>Polymer physics</strong></div><div><div>Colby, R.H. (Editor)</div></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oso/9780198520597.001.0001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foso%2F9780198520597.001.0001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120-1" title="View in article">119.</a></div><div>Sanamrad, A. ∙ Persson, F. ∙ Lundius, E.G. ...</div><div><strong>Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11413-11418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1411558111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84905669647" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056965/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1411558111&amp;pmid=25056965" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121" title="View in article">120.</a></div><div>Schaechter, M. ∙ Laing, V.O.</div><div><strong>Direct observation of fusion of bacterial nuclei</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1961; <strong>81</strong>:667-668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.81.4.667-668.1961" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0001515687" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13747158/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.81.4.667-668.1961&amp;pmid=13747158" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122" title="View in article">121.</a></div><div>Schluenzen, F. ∙ Takemoto, C. ∙ Wilson, D.N. ...</div><div><strong>The antibiotic kasugamycin mimics mRNA nucleotides to destabilize tRNA binding and inhibit canonical translation initiation</strong></div><div><em>Nat. Struct. Mol. Biol.</em> 2006; <strong>13</strong>:871-878</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nsmb1145" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33749531968" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16998488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnsmb1145&amp;pmid=16998488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123-1" title="View in article">122.</a></div><div>Schneider, E. ∙ Blundell, M. ∙ Kennell, D.</div><div><strong>Translation and mRNA decay</strong></div><div><em>Mol. Gen. Genet.</em> 1978; <strong>160</strong>:121-129</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00267473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/349350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00267473&amp;pmid=349350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124-1" title="View in article">123.</a></div><div>Schultz, S.G. ∙ Wilson, N.L. ∙ Epstein, W.</div><div><strong>Cation transport in Escherichia coli. II. Intracellular chloride concentration</strong></div><div><em>J.&nbsp;Gen. Physiol.</em> 1962; <strong>46</strong>:159-166</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1085/jgp.46.1.159" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13909522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1085%2Fjgp.46.1.159&amp;pmid=13909522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref125" title="View in article">124.</a></div><div>Selinger, D.W. ∙ Saxena, R.M. ∙ Cheung, K.J. ...</div><div><strong>Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:216-223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.912603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0037315564" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12566399/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.912603&amp;pmid=12566399" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref126" title="View in article">125.</a></div><div>Sharpe, M.E. ∙ Hauser, P.M. ∙ Sharpe, R.G. ...</div><div><strong>Bacillus subtilis cell cycle as studied by fluorescence microscopy: constancy of cell length at initiation of DNA replication and evidence for active nucleoid partitioning</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1998; <strong>180</strong>:547-555</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.180.3.547-555.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9457856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.180.3.547-555.1998&amp;pmid=9457856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127" title="View in article">126.</a></div><div>Shendruk, T.N. ∙ Bertrand, M. ∙ de Haan, H.W. ...</div><div><strong>Simulating the entropic collapse of coarse-grained chromosomes</strong></div><div><em>Biophys. J.</em> 2015; <strong>108</strong>:810-820</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.11.3487&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2014.11.3487&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84923248039" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25692586/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2014.11.3487&amp;pmid=25692586" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">127.</a></div><div>Sheth, S.R. ∙ Leckband, D.</div><div><strong>Measurements of attractive forces between proteins and end-grafted poly(ethylene glycol) chains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:8399-8404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.16.8399" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030831255" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9237988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.16.8399&amp;pmid=9237988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129" title="View in article">128.</a></div><div>Skoko, D. ∙ Wong, B. ∙ Johnson, R.C. ...</div><div><strong>Micromechanical analysis of the binding of DNA-bending proteins HMGB1, NHP6A, and HU reveals their ability to form highly stable DNA-protein complexes</strong></div><div><em>Biochemistry.</em> 2004; <strong>43</strong>:13867-13874</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi048428o" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-7244245362" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15504049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi048428o&amp;pmid=15504049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130" title="View in article">129.</a></div><div>Slater, S. ∙ Wold, S. ∙ Lu, M. ...</div><div><strong>E.&nbsp;coli SeqA protein binds oriC in two different methyl-modulated reactions appropriate to its roles in DNA replication initiation and origin sequestration</strong></div><div><em>Cell.</em> 1995; <strong>82</strong>:927-936</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2895%2990272-4&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2F0092-8674%2895%2990272-4&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029147297" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7553853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2895%2990272-4&amp;pmid=7553853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref131-1" title="View in article">130.</a></div><div>Sliusarenko, O. ∙ Heinritz, J. ∙ Emonet, T. ...</div><div><strong>High-throughput, subpixel precision analysis of bacterial morphogenesis and intracellular spatio-temporal dynamics</strong></div><div><em>Mol. Microbiol.</em> 2011; <strong>80</strong>:612-627</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2958.2011.07579.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-79955024764" target="_blank" aria-label="Open Scopus in new tab">Scopus (340)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21414037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2958.2011.07579.x&amp;pmid=21414037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132" title="View in article">131.</a></div><div>Sobel, E.S. ∙ Harpst, J.A.</div><div><strong>Effects of Na+ on the persistence length and excluded volume of T7 bacteriophage DNA</strong></div><div><em>Biopolymers.</em> 1991; <strong>31</strong>:1559-1564</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bip.360311311" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1814504/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbip.360311311&amp;pmid=1814504" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib133" class="citations"><div id="sref133" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">132.</a></div><div>Spahn, C. ∙ Endesfelder, U. ∙ Heilemann, M.</div><div><strong>Super-resolution imaging of Escherichia coli nucleoids reveals highly structured and asymmetric segregation during fast growth</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2014; <strong>185</strong>:243-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2014.01.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84894410667" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24473063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2014.01.007&amp;pmid=24473063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134-1" title="View in article">133.</a></div><div>Spahn, C.K. ∙ Glaesmann, M. ∙ Grimm, J.B. ...</div><div><strong>A toolbox for multiplexed super-resolution imaging of the E.&nbsp;coli nucleoid and membrane using novel PAINT labels</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:14768</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-33052-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85054290920" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30282984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-33052-3&amp;pmid=30282984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib135" class="citations"><div id="sref135" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref135" title="View in article">134.</a></div><div>Spurio, R. ∙ Dürrenberger, M. ∙ Falconi, M. ...</div><div><strong>Lethal overproduction of the Escherichia coli nucleoid protein H-NS: ultramicroscopic and molecular autopsy</strong></div><div><em>Mol. Gen. Genet.</em> 1992; <strong>231</strong>:201-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF00279792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1310520/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF00279792&amp;pmid=1310520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136" title="View in article">135.</a></div><div>Stavans, J. ∙ Oppenheim, A.</div><div><strong>DNA-protein interactions and bacterial chromosome architecture</strong></div><div><em>Phys. Biol.</em> 2006; <strong>3</strong>:R1-R10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1088/1478-3975/3/4/R01" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-34247517469" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17200598/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1088%2F1478-3975%2F3%2F4%2FR01&amp;pmid=17200598" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137-1" title="View in article">136.</a></div><div>Stracy, M. ∙ Lesterlin, C. ∙ Garza de Leon, F. ...</div><div><strong>Live-cell superresolution microscopy reveals the organization of RNA polymerase in the bacterial nucleoid</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4390-E4399</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1507592112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26224838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1507592112&amp;pmid=26224838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138-1" title="View in article">137.</a></div><div>Sun, Q. ∙ Margolin, W.</div><div><strong>Effects of perturbing nucleoid structure on nucleoid occlusion-mediated toporegulation of FtsZ ring assembly</strong></div><div><em>J.&nbsp;Bacteriol.</em> 2004; <strong>186</strong>:3951-3959</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.186.12.3951-3959.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942547429" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15175309/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.186.12.3951-3959.2004&amp;pmid=15175309" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref139" title="View in article">138.</a></div><div>Surovtsev, I.V. ∙ Jacobs-Wagner, C.</div><div><strong>Subcellular Organization: A Critical Feature of Bacterial Cell Replication</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:1271-1293</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2018.01.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2018.01.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85043302867" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29522747/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.01.014&amp;pmid=29522747" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib140" class="citations"><div id="sref140" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref140" title="View in article">139.</a></div><div>Tai, P.C. ∙ Wallace, B.J. ∙ Davis, B.D.</div><div><strong>Actions of aurintricarboxylate, kasugamycin, and pactamycin on Escherichia coli polysomes</strong></div><div><em>Biochemistry.</em> 1973; <strong>12</strong>:616-620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/bi00728a008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4570848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fbi00728a008&amp;pmid=4570848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib141" class="citations"><div id="sref141" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref141" title="View in article">140.</a></div><div>Tseng, Y. ∙ Lee, J.S. ∙ Kole, T.P. ...</div><div><strong>Micro-organization and visco-elasticity of the interphase nucleus revealed by particle nanotracking</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2004; <strong>117</strong>:2159-2167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.01073" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942516225" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15090601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.01073&amp;pmid=15090601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib142" class="citations"><div id="sref142" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref142-1" title="View in article">141.</a></div><div>Val, M.E. ∙ Marbouty, M. ∙ de Lemos Martins, F. ...</div><div><strong>A checkpoint control orchestrates the replication of the two chromosomes of Vibrio cholerae</strong></div><div><em>Sci. Adv.</em> 2016; <strong>2</strong>:e1501914</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1501914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84991009833" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27152358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1501914&amp;pmid=27152358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib143" class="citations"><div id="sref143" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref143-1" title="View in article">142.</a></div><div>van Helvoort, J.M. ∙ Kool, J. ∙ Woldringh, C.L.</div><div><strong>Chloramphenicol causes fusion of separated nucleoids in Escherichia coli K-12 cells and filaments</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1996; <strong>178</strong>:4289-4293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jb.178.14.4289-4293.1996" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0029978680" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8763959/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjb.178.14.4289-4293.1996&amp;pmid=8763959" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib144" class="citations"><div id="sref144" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144" title="View in article">143.</a></div><div>van Noort, J. ∙ Verbrugge, S. ∙ Goosen, N. ...</div><div><strong>Dual architectural roles of HU: formation of flexible hinges and rigid filaments</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:6969-6974</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0308230101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2342454974" target="_blank" aria-label="Open Scopus in new tab">Scopus (233)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15118104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0308230101&amp;pmid=15118104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib145" class="citations"><div id="sref145" class="citation"><div class="citation-content"><div class="label"><a href="#figs5" title="View in article">144.</a></div><div>Varricchio, F. ∙ Monier, R.</div><div><strong>Ribosome patterns in Escherichia coli growing at various rates</strong></div><div><em>J.&nbsp;Bacteriol.</em> 1971; <strong>108</strong>:105-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JB.108.1.105-110.1971" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5001192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJB.108.1.105-110.1971&amp;pmid=5001192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib146" class="citations"><div id="sref146" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146" title="View in article">145.</a></div><div>Verma, S.C. ∙ Qian, Z. ∙ Adhya, S.L.</div><div><strong>Architecture of the Escherichia coli nucleoid</strong></div><div><em>PLoS Genet.</em> 2019; <strong>15</strong>:e1008456</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1008456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85076457375" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31830036/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1008456&amp;pmid=31830036" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib147" class="citations"><div id="sref147" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref147" title="View in article">146.</a></div><div>Verschoor, A. ∙ Frank, J. ∙ Boublik, M.</div><div><strong>Investigation of the 50&nbsp;S ribosomal subunit by electron microscopy and image analysis</strong></div><div><em>J.&nbsp;Ultrastruct. Res.</em> 1985; <strong>92</strong>:180-189</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0889-1605(85)90045-X" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0022381381" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3913778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0889-1605%2885%2990045-X&amp;pmid=3913778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib148" class="citations"><div id="sref148" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148-1" title="View in article">147.</a></div><div>Virtanen, P. ∙ Gommers, R. ∙ Oliphant, T.E. ..., SciPy 1.0 Contributors</div><div><strong>SciPy 1.0: fundamental algorithms for scientific computing in Python</strong></div><div><em>Nat. Methods.</em> 2020; <strong>17</strong>:261-272</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0686-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85079119586" target="_blank" aria-label="Open Scopus in new tab">Scopus (3027)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32015543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0686-2&amp;pmid=32015543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib149" class="citations"><div id="sref149" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref149" title="View in article">148.</a></div><div>Waldminghaus, T. ∙ Weigel, C. ∙ Skarstad, K.</div><div><strong>Replication fork movement and methylation govern SeqA binding to the Escherichia coli chromosome</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:5465-5476</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gks187" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84863227289" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22373925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgks187&amp;pmid=22373925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib150" class="citations"><div id="sref150" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref150" title="View in article">149.</a></div><div>Wallden, M. ∙ Fange, D. ∙ Lundius, E.G. ...</div><div><strong>The Synchronization of Replication and Division Cycles in Individual E.&nbsp;coli Cells</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:729-739</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2016.06.052&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cell.2016.06.052&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84979642298" target="_blank" aria-label="Open Scopus in new tab">Scopus (143)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27471967/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.052&amp;pmid=27471967" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib151" class="citations"><div id="sref151" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151-1" title="View in article">150.</a></div><div>Wang, X. ∙ Le, T.B. ∙ Lajoie, B.R. ...</div><div><strong>Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis</strong></div><div><em>Genes Dev.</em> 2015; <strong>29</strong>:1661-1675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.265876.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26253537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.265876.115&amp;pmid=26253537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib152" class="citations"><div id="sref152" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref152" title="View in article">151.</a></div><div>Wang, Y. ∙ Penkul, P. ∙ Milstein, J.N.</div><div><strong>Quantitative Localization Microscopy Reveals a Novel Organization of a High-Copy Number Plasmid</strong></div><div><em>Biophys. J.</em> 2016; <strong>111</strong>:467-479</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2016.06.033&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.bpj.2016.06.033&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84982994278" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27508432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2016.06.033&amp;pmid=27508432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib153" class="citations"><div id="sref153" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref153-1" title="View in article">152.</a></div><div>Wang, X. ∙ Brandão, H.B. ∙ Le, T.B. ...</div><div><strong>Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus</strong></div><div><em>Science.</em> 2017; <strong>355</strong>:524-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai8982" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85011362856" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28154080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai8982&amp;pmid=28154080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib154" class="citations"><div id="sref154" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref154" title="View in article">153.</a></div><div>Wegner, A.S. ∙ Wintraecken, K. ∙ Spurio, R. ...</div><div><strong>Compaction of isolated Escherichia coli nucleoids: Polymer and H-NS protein synergetics</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>194</strong>:129-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2016.02.009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84958672778" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26868106/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2016.02.009&amp;pmid=26868106" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib155" class="citations"><div id="sref155" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref155" title="View in article">154.</a></div><div>Wiggins, P.A. ∙ Cheveralls, K.C. ∙ Martin, J.S. ...</div><div><strong>Strong intranucleoid interactions organize the Escherichia coli chromosome into a nucleoid filament</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2010; <strong>107</strong>:4991-4995</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0912062107" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-77950455981" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20194778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0912062107&amp;pmid=20194778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib156" class="citations"><div id="sref156" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref156-1" title="View in article">155.</a></div><div>Winkler, H.</div><div><strong>3D reconstruction and processing of volumetric data in cryo-electron tomography</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2007; <strong>157</strong>:126-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2006.07.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-33845310367" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16973379/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2006.07.014&amp;pmid=16973379" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib157" class="citations"><div id="sref157" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref157-1" title="View in article">156.</a></div><div>Winkler, H. ∙ Zhu, P. ∙ Liu, J. ...</div><div><strong>Tomographic subvolume alignment and subvolume classification applied to myosin V and SIV envelope spikes</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2009; <strong>165</strong>:64-77</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2008.10.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-58349085125" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19032983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2008.10.004&amp;pmid=19032983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib158" class="citations"><div id="sref158" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref158" title="View in article">157.</a></div><div>Winkler, J. ∙ Seybert, A. ∙ König, L. ...</div><div><strong>Quantitative and spatio-temporal features of protein aggregation in Escherichia coli and consequences on protein quality control and cellular ageing</strong></div><div><em>EMBO J.</em> 2010; <strong>29</strong>:910-923</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/emboj.2009.412" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-77649293067" target="_blank" aria-label="Open Scopus in new tab">Scopus (186)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20094032/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Femboj.2009.412&amp;pmid=20094032" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib159" class="citations"><div id="sref159" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref159" title="View in article">158.</a></div><div>Woldringh, C.L.</div><div><strong>The role of co-transcriptional translation and protein translocation (transertion) in bacterial chromosome segregation</strong></div><div><em>Mol. Microbiol.</em> 2002; <strong>45</strong>:17-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1046/j.1365-2958.2002.02993.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036050398" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12100545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1046%2Fj.1365-2958.2002.02993.x&amp;pmid=12100545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib160" class="citations"><div id="sref160" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref160" title="View in article">159.</a></div><div>Woldringh, C.L. ∙ Jensen, P.R. ∙ Westerhoff, H.V.</div><div><strong>Structure and partitioning of bacterial DNA: determined by a balance of compaction and expansion forces?</strong></div><div><em>FEMS Microbiol. Lett.</em> 1995; <strong>131</strong>:235-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1574-6968.1995.tb07782.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7557335/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1574-6968.1995.tb07782.x&amp;pmid=7557335" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib161" class="citations"><div id="sref161" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref161-1" title="View in article">160.</a></div><div>Wong, I.Y. ∙ Gardel, M.L. ∙ Reichman, D.R. ...</div><div><strong>Anomalous diffusion probes microstructure dynamics of entangled F-actin networks</strong></div><div><em>Phys. Rev. Lett.</em> 2004; <strong>92</strong>:178101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.92.178101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-2942551253" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15169197/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.92.178101&amp;pmid=15169197" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib162" class="citations"><div id="sref162" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref162-1" title="View in article">161.</a></div><div>Wu, F. ∙ Swain, P. ∙ Kuijpers, L. ...</div><div><strong>Cell Boundary Confinement Sets the Size and Position of the E.&nbsp;coli Chromosome</strong></div><div><em>Curr. Biol.</em> 2019; <strong>29</strong>:2131-2144.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cub.2019.05.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=10.1016%2Fj.cub.2019.05.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85065894263" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31155353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2019.05.015&amp;pmid=31155353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib163" class="citations"><div id="sref163" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref163-1" title="View in article">162.</a></div><div>Yang, S. ∙ Kim, S. ∙ Kim, D.K. ...</div><div><strong>Transcription and translation contribute to gene locus relocation to the nucleoid periphery in E.&nbsp;coli</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:5131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-13152-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85074899720" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31719538/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-13152-y&amp;pmid=31719538" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib164" class="citations"><div id="sref164" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref164-1" title="View in article">163.</a></div><div>Yang, D. ∙ Männik, J. ∙ Retterer, S.T. ...</div><div><strong>The effects of polydisperse crowders on the compaction of the Escherichia coli nucleoid</strong></div><div><em>Mol. Microbiol.</em> 2020; <strong>113</strong>:1022-1037</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/mmi.14467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85078903920" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31961016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fmmi.14467&amp;pmid=31961016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib165" class="citations"><div id="sref165" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref165-1" title="View in article">164.</a></div><div>Yonemura, K. ∙ Maeda, H.</div><div><strong>A new assay method for DNase by fluorescence polarization and fluorescence intensity using DNA-ethidium bromide complex as a sensitive substrate</strong></div><div><em>J.&nbsp;Biochem.</em> 1982; <strong>92</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/oxfordjournals.jbchem.a134048" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0020309501" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7174645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Foxfordjournals.jbchem.a134048&amp;pmid=7174645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib166" class="citations"><div id="sref166" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref166" title="View in article">165.</a></div><div>Yoshikawa, K. ∙ Hirota, S. ∙ Makita, N. ...</div><div><strong>Compaction of DNA Induced by Like-Charge Protein: Opposite Salt-Effect against the Polymer-Salt-Induced Condensation with Neutral Polymer</strong></div><div><em>J.&nbsp;Phys. Chem. Lett.</em> 2010; <strong>1</strong>:1763-1766</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/jz100569e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-77953791541" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fjz100569e" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib167" class="citations"><div id="sref167" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref167-1" title="View in article">166.</a></div><div>Zhang, K.</div><div><strong>Gctf: Real-time CTF determination and correction</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2016; <strong>193</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jsb.2015.11.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84955216953" target="_blank" aria-label="Open Scopus in new tab">Scopus (1297)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jsb.2015.11.003&amp;pmid=26592709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib168" class="citations"><div id="sref168" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168" title="View in article">167.</a></div><div>Zhang, C. ∙ Shao, P.G. ∙ van Kan, J.A. ...</div><div><strong>Macromolecular crowding induced elongation and compaction of single DNA molecules confined in a nanochannel</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:16651-16656</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0904741106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-70349728576" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19805352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0904741106&amp;pmid=19805352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib169" class="citations"><div id="sref169" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref169" title="View in article">168.</a></div><div>Zhao, X. ∙ Zhang, K. ∙ Boquoi, T. ...</div><div><strong>Cryoelectron tomography reveals the sequential assembly of bacterial flagella in Borrelia burgdorferi</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:14390-14395</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1308306110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-84883311301" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23940315/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1308306110&amp;pmid=23940315" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib170" class="citations"><div id="sref170" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref170-1" title="View in article">169.</a></div><div>Zheng, S.Q. ∙ Palovcak, E. ∙ Armache, J.P. ...</div><div><strong>MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:331-332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4193" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-85014129582" target="_blank" aria-label="Open Scopus in new tab">Scopus (1982)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4193&amp;pmid=28250466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib171" class="citations"><div id="sref171" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref171" title="View in article">170.</a></div><div>Zhu, J. ∙ Penczek, P.A. ∙ Schröder, R. ...</div><div><strong>Three-dimensional reconstruction with contrast transfer function correction from energy-filtered cryoelectron micrographs: procedure and application to the 70S Escherichia coli ribosome</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 1997; <strong>118</strong>:197-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/jsbi.1997.3845" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0030960710" target="_blank" aria-label="Open Scopus in new tab">Scopus (132)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9169230/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fjsbi.1997.3845&amp;pmid=9169230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib172" class="citations"><div id="sref172" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref172-1" title="View in article">171.</a></div><div>Zimm, B.H. ∙ Stockmayer, W.H.</div><div><strong>The Dimensions of Chain Molecules Containing Branches and Rings</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1949; <strong>17</strong>:1301-1314</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.1747157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.1747157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib173" class="citations"><div id="sref173" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref173" title="View in article">172.</a></div><div>Zimmerman, S.B.</div><div><strong>Macromolecular crowding effects on macromolecular interactions: some implications for genome structure and function</strong></div><div><em>Biochim. Biophys. Acta.</em> 1993; <strong>1216</strong>:175-185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0167-4781(93)90142-Z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0027496418" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8241257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0167-4781%2893%2990142-Z&amp;pmid=8241257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib174" class="citations"><div id="sref174" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref174-1" title="View in article">173.</a></div><div>Zimmerman, S.B.</div><div><strong>Toroidal nucleoids in Escherichia coli exposed to chloramphenicol</strong></div><div><em>J.&nbsp;Struct. Biol.</em> 2002; <strong>138</strong>:199-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S1047-8477(02)00036-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.05.037&amp;key=2-s2.0-0036420081" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12217658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1047-8477%2802%2900036-9&amp;pmid=12217658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib175" class="citations"><div id="sref175" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref175" title="View in article">174.</a></div><div>Zimmerman, S.B. ∙ Minton, A.P.</div><div><strong>Macromolecular crowding: biochemical, biophysical, and physiological consequences</strong></div><div><em>Annu. Rev. Biophys. Biomol. Struct.</em> 1993; <strong>22</strong>:27-65</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.bb.22.060193.000331" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7688609/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.bb.22.060193.000331&amp;pmid=7688609" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/ff5bfad5-7ca5-4382-98bd-8273a2f2df32/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/94e121f5-a605-4054-9d4e-85c3b21a7222/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/490a2c49-974c-494d-835c-897b45728b90/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/016215b4-20a2-42d7-9b97-7d26661b95c3/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/7c45c482-e82e-4820-9115-f504425cd676/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/a7680832-9157-48c2-bf0d-a6817a6b4ad9/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/205a0a01-a085-4c94-b5ce-537b90eb1685/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/05c99845-7c97-44d9-82a8-9f640b82d007/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/460ce7af-3489-4a19-94f4-24179a186a06/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/7f23ef86-b1f2-4494-9f58-36c385145cf8/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/bb703af3-4891-4b31-84cb-9b3c5a253d63/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/a76731be-f1f2-48ee-9d99-b541578be9d4/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/1d5b4764-967a-4978-bb62-24618276c577/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.05.037/asset/e02b7a19-b959-4fc0-8d22-32cf7cf93b75/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867421006899%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00158%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">56</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">109</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">2</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">27</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.05.037&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421006899" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.05.037","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Interconnecting solvent quality, transcription, and chromosome folding in Escherichia coli</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.05.037/asset/ff5bfad5-7ca5-4382-98bd-8273a2f2df32/main.assets/gr1.jpg" height="385" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/141b225b-1bcb-4dae-8046-5e48fe926b12/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.05.037/asset/94e121f5-a605-4054-9d4e-85c3b21a7222/main.assets/figs1.jpg" height="362" width="499" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/657ca581-59c3-45af-8abc-1be54785d948/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.05.037/asset/490a2c49-974c-494d-835c-897b45728b90/main.assets/gr2.jpg" height="615" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cf747cda-c12e-4dd0-b65a-593c96dee549/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.05.037/asset/016215b4-20a2-42d7-9b97-7d26661b95c3/main.assets/figs2.jpg" height="181" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/aefb65b3-399c-43ef-9829-86c95d6da77f/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.05.037/asset/7c45c482-e82e-4820-9115-f504425cd676/main.assets/figs3.jpg" height="240" width="507" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/65a4e1c7-86b5-4dbd-88b0-97dd6ab29cb4/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.05.037/asset/a7680832-9157-48c2-bf0d-a6817a6b4ad9/main.assets/gr3.jpg" height="647" width="657" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/42e5bfd0-54c7-4006-9976-3c18e9db413a/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.05.037/asset/205a0a01-a085-4c94-b5ce-537b90eb1685/main.assets/gr4.jpg" height="409" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/e8ed7571-1115-45e5-8141-61ddd658a6e3/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.05.037/asset/05c99845-7c97-44d9-82a8-9f640b82d007/main.assets/figs4.jpg" height="536" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cfe8608a-ac41-4ddb-aaa1-ea61b47abcef/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.05.037/asset/460ce7af-3489-4a19-94f4-24179a186a06/main.assets/figs5.jpg" height="617" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/324691b1-4b6a-4c55-bbda-5162c3abf59b/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.05.037/asset/7f23ef86-b1f2-4494-9f58-36c385145cf8/main.assets/gr5.jpg" height="525" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/1c23f037-5dda-400e-a132-1369ea01a5df/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.05.037/asset/bb703af3-4891-4b31-84cb-9b3c5a253d63/main.assets/gr6.jpg" height="377" width="309" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/2d2bd7a5-b741-4e19-86bc-339154df78dc/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.05.037/asset/a76731be-f1f2-48ee-9d99-b541578be9d4/main.assets/gr7.jpg" height="500" width="752" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/e0bcebff-e3eb-480b-8184-6675e9f261d1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.05.037/asset/1d5b4764-967a-4978-bb62-24618276c577/main.assets/figs6.jpg" height="411" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/3c1b6e2a-a45d-4c7c-b8db-c8aad58a7690/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.05.037/asset/e02b7a19-b959-4fc0-8d22-32cf7cf93b75/main.assets/figs7.jpg" height="760" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cf6800cb-ea3b-423d-be07-31e86c320a2f/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.05.037/asset/ff5bfad5-7ca5-4382-98bd-8273a2f2df32/main.assets/gr1.jpg" height="385" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/141b225b-1bcb-4dae-8046-5e48fe926b12/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.05.037/asset/94e121f5-a605-4054-9d4e-85c3b21a7222/main.assets/figs1.jpg" height="362" width="499" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/657ca581-59c3-45af-8abc-1be54785d948/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.05.037/asset/490a2c49-974c-494d-835c-897b45728b90/main.assets/gr2.jpg" height="615" width="753" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cf747cda-c12e-4dd0-b65a-593c96dee549/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.05.037/asset/016215b4-20a2-42d7-9b97-7d26661b95c3/main.assets/figs2.jpg" height="181" width="506" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/aefb65b3-399c-43ef-9829-86c95d6da77f/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.05.037/asset/7c45c482-e82e-4820-9115-f504425cd676/main.assets/figs3.jpg" height="240" width="507" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/65a4e1c7-86b5-4dbd-88b0-97dd6ab29cb4/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.05.037/asset/a7680832-9157-48c2-bf0d-a6817a6b4ad9/main.assets/gr3.jpg" height="647" width="657" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/42e5bfd0-54c7-4006-9976-3c18e9db413a/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.05.037/asset/205a0a01-a085-4c94-b5ce-537b90eb1685/main.assets/gr4.jpg" height="409" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/e8ed7571-1115-45e5-8141-61ddd658a6e3/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.05.037/asset/05c99845-7c97-44d9-82a8-9f640b82d007/main.assets/figs4.jpg" height="536" width="657" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cfe8608a-ac41-4ddb-aaa1-ea61b47abcef/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.05.037/asset/460ce7af-3489-4a19-94f4-24179a186a06/main.assets/figs5.jpg" height="617" width="657" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/324691b1-4b6a-4c55-bbda-5162c3abf59b/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.05.037/asset/7f23ef86-b1f2-4494-9f58-36c385145cf8/main.assets/gr5.jpg" height="525" width="753" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/1c23f037-5dda-400e-a132-1369ea01a5df/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.05.037/asset/bb703af3-4891-4b31-84cb-9b3c5a253d63/main.assets/gr6.jpg" height="377" width="309" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/2d2bd7a5-b741-4e19-86bc-339154df78dc/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.05.037/asset/a76731be-f1f2-48ee-9d99-b541578be9d4/main.assets/gr7.jpg" height="500" width="752" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/e0bcebff-e3eb-480b-8184-6675e9f261d1/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.05.037/asset/1d5b4764-967a-4978-bb62-24618276c577/main.assets/figs6.jpg" height="411" width="506" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/3c1b6e2a-a45d-4c7c-b8db-c8aad58a7690/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.05.037/asset/e02b7a19-b959-4fc0-8d22-32cf7cf93b75/main.assets/figs7.jpg" height="760" width="657" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.05.037/asset/cf6800cb-ea3b-423d-be07-31e86c320a2f/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00689-9';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749590617324","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"14","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421006899","coverDate":"202107","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Interconnecting solvent quality transcription and chromosome folding in emEscherichia coliem","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbee45280c4edd',t:'MTc0OTU5MDYxNi4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbee45280c4edd&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.05.037&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>