<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="78ee74cd-21d4-4ece-b316-1bcc0151ae83" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867422008479;journal:journal:cell;pageGroup:string:Article View;issue:issue:pii\:S0092867421X00177;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Glioblastoma hijacks neuronal mechanisms for brain invasion">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00847-9">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/5c59ff00-fe74-4fb9-9634-55055bc004ba/fx1.jpg">
    


        <meta property="og:description" content="Glioblastoma cells with neuronal features drive brain invasion.">
    











        <meta name="twitter:title" content="Glioblastoma hijacks neuronal mechanisms for brain invasion">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/5c59ff00-fe74-4fb9-9634-55055bc004ba/fx1.jpg">
    


        <meta name="twitter:description" content="Glioblastoma cells with neuronal features drive brain invasion.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Glioblastoma hijacks neuronal mechanisms for brain invasion: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Glioblastoma hijacks neuronal mechanisms for brain invasion"> <meta name="citation_date" content="2022/08/04"> <meta name="citation_online_date" content="2022/07/31"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="16"> <meta name="citation_firstpage" content="2899"> <meta name="citation_lastpage" content="2917.e31"> <meta name="citation_pmid" content="35914528"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422008479"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422008479"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422008479"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.06.054"> <meta name="citation_pmid" content="35914528"> <meta name="citation_author" content="Varun Venkataramani"><meta name="citation_author_email" content="varun.venkataramani@med.uni-heidelberg.de"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Yvonne Yang"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Marc Cicero Schubert"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Ekin Reyhan"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Svenja Kristin Tetzlaff"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Niklas Wißmann"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Michael Botz"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Stella Judith Soyka"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Carlo Antonio Beretta"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Rangel Lyubomirov Pramatarov"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Laura Fankhauser"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Luciano Garofano"><meta name="citation_author_institution" content="Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA"><meta name="citation_author" content="Alexander Freudenberg"><meta name="citation_author_institution" content="Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany"><meta name="citation_author" content="Julia Wagner"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Dimitar Ivanov Tanev"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Miriam Ratliff"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Neurosurgery Clinic, University Hospital Mannheim, 68167 Mannheim, Germany"><meta name="citation_author" content="Ruifan Xie"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Tobias Kessler"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Dirk C. Hoffmann"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Faculty of Biosciences, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Ling Hai"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Yvette Dörflinger"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Simone Hoppe"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Yahaya A. Yabo"><meta name="citation_author_institution" content="NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg"><meta name="citation_author" content="Anna Golebiewska"><meta name="citation_author_institution" content="NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg"><meta name="citation_author" content="Simone P. Niclou"><meta name="citation_author_institution" content="NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg"><meta name="citation_author" content="Felix Sahm"><meta name="citation_author_institution" content="Department of Neuropathology, Institute of Pathology, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neuropathology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Anna Lasorella"><meta name="citation_author_institution" content="Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA"><meta name="citation_author" content="Martin Slowik"><meta name="citation_author_institution" content="Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany"><meta name="citation_author" content="Leif Döring"><meta name="citation_author_institution" content="Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany"><meta name="citation_author" content="Antonio Iavarone"><meta name="citation_author_institution" content="Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA"><meta name="citation_author" content="Wolfgang Wick"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_author" content="Thomas Kuner"><meta name="citation_author_email" content="thomas.kuner@uni-heidelberg.de"><meta name="citation_author_institution" content="Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany"><meta name="citation_author" content="Frank Winkler"><meta name="citation_author_email" content="frank.winkler@med.uni-heidelberg.de"><meta name="citation_author_institution" content="Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany"><meta name="citation_author_institution" content="Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany"><meta name="citation_keywords" content="glioblastoma; tumor microtubes; tumor cell networks; cancer neuroscience; neurogliomal synapses; neuron-tumor interaction; cell heterogeneity; glutamatergic synapse"><meta name="citation_reference" content="citation_title=On the estimation of the extreme-value index and large quantile estimation; citation_author=L.M.D. Arnold; citation_author=H. Laurens De; citation_journal_title=Ann. Stat.; citation_volume=17; citation_firstpage=1795; citation_lastpage=1832; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=ilastik: interactive machine learning for (bio)image analysis; citation_author=S. Berg; citation_author=D. Kutra; citation_author=T. Kroeger; citation_author=C.N. Straehle; citation_author=B.X. Kausler; citation_author=C. Haubold; citation_author=M. Schiegg; citation_author=J. Ales; citation_author=T. Beier; citation_author=M. Rudy; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=1226; citation_lastpage=1232; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Outer radial glia-like cancer stem cells contribute to heterogeneity of glioblastoma; citation_author=A. Bhaduri; citation_author=E. Di Lullo; citation_author=D.N. Jung; citation_author=S. Müller; citation_author=E.E. Crouch; citation_author=C.S. Espinosa; citation_author=T. Ozawa; citation_author=B. Alvarado; citation_author=J. Spatazza; citation_author=C.R. Cadwell; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=26; citation_firstpage=48; citation_lastpage=63.e6; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Three-dimensional Ca(2+) imaging advances understanding of astrocyte biology; citation_author=E. Bindocci; citation_author=I. Savtchouk; citation_author=N. Liaudet; citation_author=D. Becker; citation_author=G. Carriero; citation_author=A. Volterra; citation_journal_title=Science; citation_volume=356; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Gap junctions in hematopoietic stroma control proliferation and differentiation of blood cell precursors; citation_author=E. Bodi; citation_author=S.P. Hurtado; citation_author=M.A. Carvalho; citation_author=R. Borojevic; citation_author=A.C.C. De Carvalho; citation_journal_title=An. Acad. Bras. Cienc.; citation_volume=76; citation_firstpage=743; citation_lastpage=756; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A simple general approach to inference about the tail of a distribution; citation_author=M.H. Bruce; citation_journal_title=Ann. Stat.; citation_volume=3; citation_firstpage=1163; citation_lastpage=1174; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Parameter estimation for fractional Poisson processes; citation_author=D.O. Cahoy; citation_author=V.V. Uchaikin; citation_author=W.A. Woyczynski; citation_journal_title=J.&nbsp;Stat. Plan. Inference; citation_volume=140; citation_firstpage=3106; citation_lastpage=3120; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The single-cell transcriptional landscape of mammalian organogenesis; citation_author=J. Cao; citation_author=M. Spielmann; citation_author=X. Qiu; citation_author=X. Huang; citation_author=D.M. Ibrahim; citation_author=A.J. Hill; citation_author=F. Zhang; citation_author=S. Mundlos; citation_author=L. Christiansen; citation_author=F.J. Steemers; citation_author=et al.; citation_journal_title=Nature; citation_volume=566; citation_firstpage=496; citation_lastpage=502; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=DNA methylation-based classification of central nervous system tumours; citation_author=D. Capper; citation_author=D.T.W. Jones; citation_author=M. Sill; citation_author=V. Hovestadt; citation_author=D. Schrimpf; citation_author=D. Sturm; citation_author=C. Koelsche; citation_author=F. Sahm; citation_author=L. Chavez; citation_author=D.E. Reuss; citation_author=et al.; citation_journal_title=Nature; citation_volume=555; citation_firstpage=469; citation_lastpage=474; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=TrakEM2 software for neural circuit reconstruction; citation_author=A. Cardona; citation_author=S. Saalfeld; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=S. Preibisch; citation_author=M. Longair; citation_author=P. Tomancak; citation_author=V. Hartenstein; citation_author=R.J. Douglas; citation_journal_title=PLoS One; citation_volume=7; citation_firstpage=e38011; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states; citation_author=R. Chaligne; citation_author=F. Gaiti; citation_author=D. Silverbush; citation_author=J.S. Schiffman; citation_author=H.R. Weisman; citation_author=L. Kluegel; citation_author=S. Gritsch; citation_author=S.D. Deochand; citation_author=L.N. Gonzalez Castro; citation_author=A.R. Richman; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=53; citation_firstpage=1469; citation_lastpage=1479; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Perampanel inhibition of AMPA receptor currents in cultured hippocampal neurons; citation_author=C.Y. Chen; citation_author=L. Matt; citation_author=J.W. Hell; citation_author=M.A. Rogawski; citation_journal_title=PLoS One; citation_volume=9; citation_firstpage=e108021; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A neurocentric perspective on glioma invasion; citation_author=V.A. Cuddapah; citation_author=S. Robel; citation_author=S. Watkins; citation_author=H. Sontheimer; citation_journal_title=Nat. Rev. Neurosci.; citation_volume=15; citation_firstpage=455; citation_lastpage=465; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=High-performance calcium sensors for imaging activity in neuronal populations and microcompartments; citation_author=H. Dana; citation_author=Y. Sun; citation_author=B. Mohar; citation_author=B.K. Hulse; citation_author=A.M. Kerlin; citation_author=J.P. Hasseman; citation_author=G. Tsegaye; citation_author=A. Tsang; citation_author=A. Wong; citation_author=R. Patel; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=649; citation_lastpage=657; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Stem cell-associated heterogeneity in Glioblastoma results from intrinsic tumor plasticity shaped by the microenvironment; citation_author=A. Dirkse; citation_author=A. Golebiewska; citation_author=T. Buder; citation_author=P.V. Nazarov; citation_author=A. Muller; citation_author=S. Poovathingal; citation_author=N.H.C. Brons; citation_author=S. Leite; citation_author=N. Sauvageot; citation_author=D. Sarkisjan; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=1787; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Targeted three-dimensional immunohistochemistry reveals localization of presynaptic proteins Bassoon and Piccolo in the rat calyx of Held before and after the onset of hearing; citation_author=A. Dondzillo; citation_author=K. Sätzler; citation_author=H. Horstmann; citation_author=W.D. Altrock; citation_author=E.D. Gundelfinger; citation_author=T. Kuner; citation_journal_title=J.&nbsp;Comp. Neurol.; citation_volume=518; citation_firstpage=1008; citation_lastpage=1029; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Connexin-43 type gap-junctions mediate communication between bone-marrow stromal cells; citation_author=K. Dorshkind; citation_author=L. Green; citation_author=A. Godwin; citation_author=W.H. Fletcher; citation_journal_title=Blood; citation_volume=82; citation_firstpage=38; citation_lastpage=45; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Extensive brainstem infiltration, not mass effect, is a common feature of end-stage cerebral glioblastomas; citation_author=M.R. Drumm; citation_author=K.S. Dixit; citation_author=S. Grimm; citation_author=P. Kumthekar; citation_author=R.V. Lukas; citation_author=J.J. Raizer; citation_author=R. Stupp; citation_author=M.G. Chheda; citation_author=K.L. Kam; citation_author=M. McCord; citation_author=et al.; citation_journal_title=Neuro. Oncol; citation_volume=22; citation_firstpage=470; citation_lastpage=479; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=moco: fast Motion Correction for Calcium Imaging; citation_author=A. Dubbs; citation_author=J. Guevara; citation_author=R. Yuste; citation_journal_title=Front. Neuroinform.; citation_volume=10; citation_firstpage=6; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Intellicount: High-Throughput Quantification of Fluorescent Synaptic Protein Puncta by Machine Learning; citation_author=J.A. Fantuzzo; citation_author=V.R. Mirabella; citation_author=A.H. Hamod; citation_author=A.H. Hart; citation_author=J.D. Zahn; citation_author=Z.P. Pang; citation_journal_title=eNeuro; citation_volume=4; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Malignant gliomas: simplifying the complexity; citation_author=H.A. Fine; citation_journal_title=Cancer Discov.; citation_volume=9; citation_firstpage=1650; citation_lastpage=1652; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A method for the three-dimensional reconstruction of Neurobiotin™-filled neurons and the location of their synaptic inputs; citation_author=M.J. Fogarty; citation_author=L.A. Hammond; citation_author=R. Kanjhan; citation_author=M.C. Bellingham; citation_author=P.G. Noakes; citation_journal_title=Front. Neural Circuits; citation_volume=7; citation_firstpage=153; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Classifying collective cancer cell invasion; citation_author=P. Friedl; citation_author=J. Locker; citation_author=E. Sahai; citation_author=J.E. Segall; citation_journal_title=Nat. Cell Biol.; citation_volume=14; citation_firstpage=777; citation_lastpage=783; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities; citation_author=L. Garofano; citation_author=S. Migliozzi; citation_author=Y.T. Oh; citation_author=F. D'Angelo; citation_author=R.D. Najac; citation_author=A. Ko; citation_author=B. Frangaj; citation_author=F.P. Caruso; citation_author=K. Yu; citation_author=J. Yuan; citation_author=et al.; citation_journal_title=Nat. Cancer; citation_volume=2; citation_firstpage=141; citation_lastpage=156; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=ShinyGO: a graphical gene-set enrichment tool for animals and plants; citation_author=S.X. Ge; citation_author=D. Jung; citation_author=R. Yao; citation_journal_title=Bioinformatics; citation_volume=36; citation_firstpage=2628; citation_lastpage=2629; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Regular Variation, Extensions and Tauberian Theorems; citation_author=J.L. Geluk; citation_author=L.F.M.d. Haan; citation_publication_date=1987; citation_publisher=CWI; "><meta name="citation_reference" content="citation_title=Doublecortin is a microtubule-associated protein and is expressed widely by migrating neurons; citation_author=J.G. Gleeson; citation_author=P.T. Lin; citation_author=L.A. Flanagan; citation_author=C.A. Walsh; citation_journal_title=Neuron; citation_volume=23; citation_firstpage=257; citation_lastpage=271; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Stochastic models that separate fractal dimension and the Hurst effect; citation_author=T. Gneiting; citation_author=M. Schlather; citation_journal_title=SIAM Rev.; citation_volume=46; citation_firstpage=269; citation_lastpage=282; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology; citation_author=A. Golebiewska; citation_author=A.C. Hau; citation_author=A. Oudin; citation_author=D. Stieber; citation_author=Y.A. Yabo; citation_author=V. Baus; citation_author=V. Barthelemy; citation_author=E. Klein; citation_author=S. Bougnaud; citation_author=O. Keunen; citation_author=et al.; citation_journal_title=Acta Neuropathol.; citation_volume=140; citation_firstpage=919; citation_lastpage=949; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Cyclic-AMP stimulates somatostatin gene-transcription by phosphorylation of Creb at Serine-133; citation_author=G.A. Gonzalez; citation_author=M.R. Montminy; citation_journal_title=Cell; citation_volume=59; citation_firstpage=675; citation_lastpage=680; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Recapitulating in&nbsp;vivo-like plasticity of glioma cell invasion along blood vessels and in astrocyte-rich stroma; citation_author=P. Gritsenko; citation_author=W. Leenders; citation_author=P. Friedl; citation_journal_title=Histochem. Cell Biol.; citation_volume=148; citation_firstpage=395; citation_lastpage=406; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=p120-catenin-dependent collective brain infiltration by glioma cell networks; citation_author=P.G. Gritsenko; citation_author=N. Atlasy; citation_author=C.E.J. Dieteren; citation_author=A.C. Navis; citation_author=J.H. Venhuizen; citation_author=C. Veelken; citation_author=D. Schubert; citation_author=A. Acker-Palmer; citation_author=B.A. Westerman; citation_author=T. Wurdinger; citation_author=et al.; citation_journal_title=Nat. Cell Biol.; citation_volume=22; citation_firstpage=97; citation_lastpage=107; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A connectivity signature for glioblastoma; citation_author=L. Hai; citation_author=D.C. Hoffmann; citation_author=H. Mandelbaum; citation_author=R. Xie; citation_author=J. Ito; citation_author=E. Jung; citation_author=S. Weil; citation_author=P. Sievers; citation_author=V. Venkataramani; citation_author=D.D. Azorin; citation_author=et al.; citation_journal_title=bioRxiv.; citation_doi=10.1101/2021.11.07.465791; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma; citation_author=T. Hara; citation_author=R. Chanoch-Myers; citation_author=N.D. Mathewson; citation_author=C. Myskiw; citation_author=L. Atta; citation_author=L. Bussema; citation_author=S.W. Eichhorn; citation_author=A.C. Greenwald; citation_author=G.S. Kinker; citation_author=C. Rodman; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=39; citation_firstpage=779; citation_lastpage=792.e11; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Serial section scanning electron microscopy (S3EM) on silicon wafers for ultra-structural volume imaging of cells and tissues; citation_author=H. Horstmann; citation_author=C. Körber; citation_author=K. Sätzler; citation_author=D. Aydin; citation_author=T. Kuner; citation_journal_title=PLoS One; citation_volume=7; citation_firstpage=e35172; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Levy-like movement patterns of metastatic cancer cells revealed in microfabricated systems and implicated in&nbsp;vivo; citation_author=S. Huda; citation_author=B. Weigelin; citation_author=K. Wolf; citation_author=K.V. Tretiakov; citation_author=K. Polev; citation_author=G. Wilk; citation_author=M. Iwasa; citation_author=F.S. Emami; citation_author=J.W. Narojczyk; citation_author=M. Banaszak; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=4539; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Foraging success of biological Levy flights recorded in situ; citation_author=N.E. Humphries; citation_author=H. Weimerskirch; citation_author=N. Queiroz; citation_author=E.J. Southall; citation_author=D.W. Sims; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=7169; citation_lastpage=7174; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Epithelial-mesenchymal transition in glioblastoma progression; citation_author=Y. Iwadate; citation_journal_title=Oncol. Lett.; citation_volume=11; citation_firstpage=1615; citation_lastpage=1620; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Tweety-homolog 1 drives brain colonization of gliomas; citation_author=E. Jung; citation_author=M. Osswald; citation_author=J. Blaes; citation_author=B. Wiestler; citation_author=F. Sahm; citation_author=T. Schmenger; citation_author=G. Solecki; citation_author=K. Deumelandt; citation_author=F.T. Kurz; citation_author=R. Xie; citation_author=et al.; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=37; citation_firstpage=6837; citation_lastpage=6850; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Tumor cell plasticity, heterogeneity, and resistance in crucial microenvironmental niches in glioma; citation_author=E. Jung; citation_author=M. Osswald; citation_author=M. Ratliff; citation_author=H. Dogan; citation_author=R. Xie; citation_author=S. Weil; citation_author=D.C. Hoffmann; citation_author=F.T. Kurz; citation_author=T. Kessler; citation_author=S. Heiland; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=12; citation_firstpage=1014; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A practical guide to the synthesis and use of membrane-permeant acetoxymethyl esters of caged inositol polyphosphates; citation_author=S. Kantevari; citation_author=G.R.J. Gordon; citation_author=B.A. MacVicar; citation_author=G.C.R. Ellis-Davies; citation_journal_title=Nat. Protoc.; citation_volume=6; citation_firstpage=327; citation_lastpage=337; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Is mesenchyme a syncytium?; citation_author=W.H. Lewis; citation_journal_title=Anat. Rec.; citation_volume=23; citation_firstpage=177; citation_lastpage=184; citation_publication_date=1922; "><meta name="citation_reference" content="citation_title=MARCKSL1 promotes the proliferation, migration and invasion of lung adenocarcinoma cells; citation_author=W. Liang; citation_author=R. Gao; citation_author=M. Yang; citation_author=X. Wang; citation_author=K. Cheng; citation_author=X. Shi; citation_author=C. He; citation_author=Y. Li; citation_author=Y. Wu; citation_author=L. Shi; citation_author=et al.; citation_journal_title=Oncol. Lett.; citation_volume=19; citation_firstpage=2272; citation_lastpage=2280; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=BiNGO: a cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks; citation_author=S. Maere; citation_author=K. Heymans; citation_author=M. Kuiper; citation_journal_title=Bioinformatics; citation_volume=21; citation_firstpage=3448; citation_lastpage=3449; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Inhibition by levetiracetam of a non-GABA(A) receptor-associated epileptiform effect of bicuculline in rat hippocampus; citation_author=D.G. Margineanu; citation_author=E. Wülfert; citation_journal_title=Br. J. Pharmacol.; citation_volume=122; citation_firstpage=1146; citation_lastpage=1150; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Guiding neuronal cell migrations; citation_author=O. Marín; citation_author=M. Valiente; citation_author=X. Ge; citation_author=L.H. Tsai; citation_journal_title=Cold Spring Harbor Perspect. Biol.; citation_volume=2; citation_firstpage=a001834; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Biased selection of leading process branches mediates chemotaxis during tangential neuronal migration; citation_author=F.J. Martini; citation_author=M. Valiente; citation_author=G. López Bendito; citation_author=G. Szabó; citation_author=F. Moya; citation_author=M. Valdeolmillos; citation_author=O. Marín; citation_journal_title=Development; citation_volume=136; citation_firstpage=41; citation_lastpage=50; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Heterogeneity of astrocyte resting membrane potentials and intercellular coupling revealed by whole-cell and gramicidin-perforated patch recordings from cultured neocortical and hippocampal slice astrocytes; citation_author=G.M. McKhann; citation_author=R. D'Ambrosio; citation_author=D. Janigro; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=17; citation_firstpage=6850; citation_lastpage=6863; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Methods for cell and particle tracking; citation_author=E. Meijering; citation_author=O. Dzyubachyk; citation_author=I. Smal; citation_journal_title=Methods Enzymol.; citation_volume=504; citation_firstpage=183; citation_lastpage=200; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The random walk's guide to anomalous diffusion: a fractional dynamics approach; citation_author=R. Metzler; citation_author=J. Klafter; citation_journal_title=Phys. Rep.; citation_volume=339; citation_firstpage=1; citation_lastpage=77; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=CREB is necessary for synaptic maintenance and learning-induced changes of the ampa receptor GluA1 subunit; citation_author=S. Middei; citation_author=G. Houeland; citation_author=V. Cavallucci; citation_author=M. Ammassari-Teule; citation_author=M. D'Amelio; citation_author=H. Marie; citation_journal_title=Hippocampus; citation_volume=23; citation_firstpage=488; citation_lastpage=499; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Neuronal migration in the developing cerebral cortex: observations based on real-time imaging; citation_author=B. Nadarajah; citation_author=P. Alifragis; citation_author=R.O. Wong; citation_author=J.G. Parnavelas; citation_journal_title=Cereb. Cortex; citation_volume=13; citation_firstpage=607; citation_lastpage=611; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Two modes of radial migration in early development of the cerebral cortex; citation_author=B. Nadarajah; citation_author=J.E. Brunstrom; citation_author=J. Grutzendler; citation_author=R.O. Wong; citation_author=A.L. Pearlman; citation_journal_title=Nat. Neurosci.; citation_volume=4; citation_firstpage=143; citation_lastpage=150; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=An integrative model of cellular states, plasticity, and genetics for glioblastoma; citation_author=C. Neftel; citation_author=J. Laffy; citation_author=M.G. Filbin; citation_author=T. Hara; citation_author=M.E. Shore; citation_author=G.J. Rahme; citation_author=A.R. Richman; citation_author=D. Silverbush; citation_author=M.L. Shaw; citation_author=C.M. Hebert; citation_author=et al.; citation_journal_title=Cell; citation_volume=178; citation_firstpage=835; citation_lastpage=849.e21; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Sulforhodamine 101 as a specific marker of astroglia in the neocortex in&nbsp;vivo; citation_author=A. Nimmerjahn; citation_author=F. Kirchhoff; citation_author=J.N.D. Kerr; citation_author=F. Helmchen; citation_journal_title=Nat. Methods; citation_volume=1; citation_firstpage=31; citation_lastpage=37; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Synaptic transmission from subplate neurons controls radial migration of neocortical neurons; citation_author=C. Ohtaka-Maruyama; citation_author=M. Okamoto; citation_author=K. Endo; citation_author=M. Oshima; citation_author=N. Kaneko; citation_author=K. Yura; citation_author=H. Okado; citation_author=T. Miyata; citation_author=N. Maeda; citation_journal_title=Science; citation_volume=360; citation_firstpage=313; citation_lastpage=317; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Brain tumour cells interconnect to a functional and resistant network; citation_author=M. Osswald; citation_author=E. Jung; citation_author=F. Sahm; citation_author=G. Solecki; citation_author=V. Venkataramani; citation_author=J. Blaes; citation_author=S. Weil; citation_author=H. Horstmann; citation_author=B. Wiestler; citation_author=M. Syed; citation_author=et al.; citation_journal_title=Nature; citation_volume=528; citation_firstpage=93; citation_lastpage=98; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Protocol for derivation of organoids and patient-derived orthotopic xenografts from glioma patient tumors; citation_author=A. Oudin; citation_author=V. Baus; citation_author=V. Barthelemy; citation_author=C. Fabian; citation_author=E. Klein; citation_author=M. Dieterle; citation_author=M. Wantz; citation_author=A.C. Hau; citation_author=C. Dording; citation_author=A. Bernard; citation_author=et al.; citation_journal_title=Star Protoc.; citation_volume=2; citation_firstpage=100534; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Sample drift correction following 4D confocal time-lapse imaging; citation_author=A. Parslow; citation_author=A. Cardona; citation_author=R.J. Bryson-Richardson; citation_journal_title=J.&nbsp;Vis. Exp.; citation_volume=12; citation_firstpage=51086; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=An anatomic transcriptional atlas of human glioblastoma; citation_author=R.B. Puchalski; citation_author=N. Shah; citation_author=J. Miller; citation_author=R. Dalley; citation_author=S.R. Nomura; citation_author=J.G. Yoon; citation_author=K.A. Smith; citation_author=M. Lankerovich; citation_author=D. Bertagnolli; citation_author=K. Bickley; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=660; citation_lastpage=663; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=YK-4-279 inhibits ERG and ETV1 mediated prostate cancer cell invasion; citation_author=S. Rahim; citation_author=E.M. Beauchamp; citation_author=Y. Kong; citation_author=M.L. Brown; citation_author=J.A. Toretsky; citation_author=A. Üren; citation_journal_title=PLoS One; citation_volume=6; citation_firstpage=e19343; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Dynamical robustness of Levy search strategies; citation_author=E.P. Raposo; citation_author=S.V. Buldyrev; citation_author=M.G. da Luz; citation_author=M.C. Santos; citation_author=H.E. Stanley; citation_author=G.M. Viswanathan; citation_journal_title=Phys. Rev. Lett.; citation_volume=91; citation_firstpage=240601; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The use of lead citrate at high pH as an electron-opaque stain in electron microscopy; citation_author=E.S. Reynolds; citation_journal_title=J.&nbsp;Cell Biol.; citation_volume=17; citation_firstpage=208; citation_lastpage=212; citation_publication_date=1963; "><meta name="citation_reference" content="citation_title=Gradient of Developmental and Injury Response transcriptional states defines functional vulnerabilities underpinning glioblastoma heterogeneity; citation_author=L.M. Richards; citation_author=O.K.N. Whitley; citation_author=G. MacLeod; citation_author=F.M.G. Cavalli; citation_author=F.J. Coutinho; citation_author=J.E. Jaramillo; citation_author=N. Svergun; citation_author=M. Riverin; citation_author=D.C. Croucher; citation_author=M. Kushida; citation_author=et al.; citation_journal_title=Nat. Cancer; citation_volume=2; citation_firstpage=157; citation_lastpage=173; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Addressing diffuse glioma as a systemic brain disease with single-cell analysis; citation_author=F. Sahm; citation_author=D. Capper; citation_author=A. Jeibmann; citation_author=A. Habel; citation_author=W. Paulus; citation_author=D. Troost; citation_author=A. von Deimling; citation_journal_title=Arch. Neurol.; citation_volume=69; citation_firstpage=523; citation_lastpage=526; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The highly expressed 5'isomiR of hsa-miR-140-3p contributes to the tumor-suppressive effects of miR-140 by reducing breast cancer proliferation and migration; citation_author=O. Salem; citation_author=N. Erdem; citation_author=J. Jung; citation_author=E. Münstermann; citation_author=A. Wörner; citation_author=H. Wilhelm; citation_author=S. Wiemann; citation_author=C. Körner; citation_journal_title=BMC Genomics; citation_volume=17; citation_firstpage=566; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Doublecortin induces mitotic microtubule catastrophe and inhibits glioma cell invasion; citation_author=M. Santra; citation_author=S. Santra; citation_author=C. Roberts; citation_author=R.L. Zhang; citation_author=M. Chopp; citation_journal_title=J.&nbsp;Neurochem.; citation_volume=108; citation_firstpage=231; citation_lastpage=245; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Automated quantification of synapses by fluorescence microscopy; citation_author=P. Schätzle; citation_author=R. Wuttke; citation_author=U. Ziegler; citation_author=P. Sonderegger; citation_journal_title=J. Neurosci. Methods; citation_volume=204; citation_firstpage=144; citation_lastpage=149; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=A high-level 3D visualization API for Java and ImageJ; citation_author=B. Schmid; citation_author=J. Schindelin; citation_author=A. Cardona; citation_author=M. Longair; citation_author=M. Heisenberg; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=274; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=NIH Image to ImageJ: 25 years of image analysis; citation_author=C.A. Schneider; citation_author=W.S. Rasband; citation_author=K.W. Eliceiri; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=671; citation_lastpage=675; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; citation_author=P. Shannon; citation_author=A. Markiel; citation_author=O. Ozier; citation_author=N.S. Baliga; citation_author=J.T. Wang; citation_author=D. Ramage; citation_author=N. Amin; citation_author=B. Schwikowski; citation_author=T. Ideker; citation_journal_title=Genome Res.; citation_volume=13; citation_firstpage=2498; citation_lastpage=2504; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance; citation_author=F.A. Siebzehnrubl; citation_author=D.J. Silver; citation_author=B. Tugertimur; citation_author=L.P. Deleyrolle; citation_author=D. Siebzehnrubl; citation_author=M.R. Sarkisian; citation_author=K.G. Devers; citation_author=A.T. Yachnis; citation_author=M.D. Kupper; citation_author=D. Neal; citation_author=et al.; citation_journal_title=EMBO Mol. Med.; citation_volume=5; citation_firstpage=1196; citation_lastpage=1212; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Hierarchical random walks in trace fossils and the origin of optimal search behavior; citation_author=D.W. Sims; citation_author=A.M. Reynolds; citation_author=N.E. Humphries; citation_author=E.J. Southall; citation_author=V.J. Wearmouth; citation_author=B. Metcalfe; citation_author=R.J. Twitchett; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=11073; citation_lastpage=11078; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Minimizing errors in identifying Levy flight behaviour of organisms; citation_author=D.W. Sims; citation_author=D. Righton; citation_author=J.W. Pitchford; citation_journal_title=J.&nbsp;Anim. Ecol.; citation_volume=76; citation_firstpage=222; citation_lastpage=229; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Scaling laws of marine predator search behaviour; citation_author=D.W. Sims; citation_author=E.J. Southall; citation_author=N.E. Humphries; citation_author=G.C. Hays; citation_author=C.J. Bradshaw; citation_author=J.W. Pitchford; citation_author=A. James; citation_author=M.Z. Ahmed; citation_author=A.S. Brierley; citation_author=M.A. Hindell; citation_author=et al.; citation_journal_title=Nature; citation_volume=451; citation_firstpage=1098; citation_lastpage=1102; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Astrocytes promote glioma invasion via the Gap junction protein connexin43; citation_author=W.C. Sin; citation_author=Q. Aftab; citation_author=J.F. Bechberger; citation_author=J.H. Leung; citation_author=H. Chen; citation_author=C.C. Naus; citation_journal_title=Oncogene; citation_volume=35; citation_firstpage=1504; citation_lastpage=1516; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The GeneCards Suite: from gene data mining to disease genome sequence analyses; citation_author=G. Stelzer; citation_author=N. Rosen; citation_author=I. Plaschkes; citation_author=S. Zimmerman; citation_author=M. Twik; citation_author=S. Fishilevich; citation_author=T.I. Stein; citation_author=R. Nudel; citation_author=I. Lieder; citation_author=Y. Mazor; citation_author=et al.; citation_journal_title=Curr. Protoc. Bioinformatics; citation_volume=54; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Comprehensive integration of single-cell data; citation_author=T. Stuart; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=C. Hafemeister; citation_author=E. Papalexi; citation_author=W.M. Mauck; citation_author=Y. Hao; citation_author=M. Stoeckius; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1888; citation_lastpage=1902.e21; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Identification of conserved isotype-defining variable region sequences for four vertebrate beta tubulin polypeptide classes; citation_author=K.F. Sullivan; citation_author=D.W. Cleveland; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=83; citation_firstpage=4327; citation_lastpage=4331; citation_publication_date=1986; "><meta name="citation_reference" content="citation_title=Multipolar migration: the third mode of radial neuronal migration in the developing cerebral cortex; citation_author=H. Tabata; citation_author=K. Nakajima; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=23; citation_firstpage=9996; citation_lastpage=10001; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Astrocyte-mediated control of cerebral blood flow; citation_author=T. Takano; citation_author=G.F. Tian; citation_author=W.G. Peng; citation_author=N.H. Lou; citation_author=W. Libionka; citation_author=X.N. Han; citation_author=M. Nedergaard; citation_journal_title=Nat. Neurosci.; citation_volume=9; citation_firstpage=260; citation_lastpage=267; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=MALAT-1 enhances cell motility of lung adenocarcinoma cells by influencing the expression of motility-related genes; citation_author=K. Tano; citation_author=R. Mizuno; citation_author=T. Okada; citation_author=R. Rakwal; citation_author=J. Shibato; citation_author=Y. Masuo; citation_author=K. Ijiri; citation_author=N. Akimitsu; citation_journal_title=FEBS Lett.; citation_volume=584; citation_firstpage=4575; citation_lastpage=4580; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Optimized ratiometric calcium sensors for functional in&nbsp;vivo imaging of neurons and T lymphocytes; citation_author=T. Thestrup; citation_author=J. Litzlbauer; citation_author=I. Bartholomäus; citation_author=M. Mues; citation_author=L. Russo; citation_author=H. Dana; citation_author=Y. Kovalchuk; citation_author=Y. Liang; citation_author=G. Kalamakis; citation_author=Y. Laukat; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=11; citation_firstpage=175; citation_lastpage=182; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Oligodendrocyte precursors migrate along vasculature in the developing nervous system; citation_author=H.H. Tsai; citation_author=J. Niu; citation_author=R. Munji; citation_author=D. Davalos; citation_author=J. Chang; citation_author=H. Zhang; citation_author=A.C. Tien; citation_author=C.J. Kuo; citation_author=J.R. Chan; citation_author=R. Daneman; citation_author=et al.; citation_journal_title=Science; citation_volume=351; citation_firstpage=379; citation_lastpage=384; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Migration of cortical interneurons relies on branched leading process dynamics; citation_author=M. Valiente; citation_author=F.J. Martini; citation_journal_title=Cell Adh. Migr.; citation_volume=3; citation_firstpage=278; citation_lastpage=280; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Human mesenchymal stem cells make cardiac connexins and form functional gap junctions; citation_author=V. Valiunas; citation_author=S. Doronin; citation_author=L. Valiuniene; citation_author=I. Potapova; citation_author=J. Zuckerman; citation_author=B. Walcott; citation_author=R.B. Robinson; citation_author=M.R. Rosen; citation_author=P.R. Brink; citation_author=I.S. Cohen; citation_journal_title=J.&nbsp;Physiol. Lond.; citation_volume=555; citation_firstpage=617; citation_lastpage=626; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A scalable SCENIC workflow for single-cell gene regulatory network analysis; citation_author=B. Van de Sande; citation_author=C. Flerin; citation_author=K. Davie; citation_author=M. De Waegeneer; citation_author=G. Hulselmans; citation_author=S. Aibar; citation_author=R. Seurinck; citation_author=W. Saelens; citation_author=R. Cannoodt; citation_author=Q. Rouchon; citation_author=et al.; citation_journal_title=Nat. Protoc.; citation_volume=15; citation_firstpage=2247; citation_lastpage=2276; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Metastatic properties and genomic amplification of the tyrosine kinase gene ACK1; citation_author=E.H. van der Horst; citation_author=Y.Y. Degenhardt; citation_author=A. Strelow; citation_author=A. Slavin; citation_author=L. Chinn; citation_author=J. Orf; citation_author=M. Rong; citation_author=S. Li; citation_author=L.H. See; citation_author=K.Q. Nguyen; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=15901; citation_lastpage=15906; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Glioma progression is shaped by genetic evolution and microenvironment interactions; citation_author=F.S. Varn; citation_author=K.C. Johnson; citation_author=J. Martinek; citation_author=J.T. Huse; citation_author=M.P. Nasrallah; citation_author=P. Wesseling; citation_author=L.A.D. Cooper; citation_author=T.M. Malta; citation_author=T.E. Wade; citation_author=T.S. Sabedot; citation_author=et al.; citation_journal_title=Cell; citation_volume=185; citation_firstpage=2184; citation_lastpage=2199.e16; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Glutamatergic synaptic input to glioma cells drives brain tumour progression; citation_author=V. Venkataramani; citation_author=D.I. Tanev; citation_author=C. Strahle; citation_author=A. Studier-Fischer; citation_author=L. Fankhauser; citation_author=T. Kessler; citation_author=C. Körber; citation_author=M. Kardorff; citation_author=M. Ratliff; citation_author=R. Xie; citation_author=et al.; citation_journal_title=Nature; citation_volume=573; citation_firstpage=532; citation_lastpage=538; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Neuronal activity promotes glioma growth through Neuroligin-3 secretion; citation_author=H.S. Venkatesh; citation_author=T.B. Johung; citation_author=V. Caretti; citation_author=A. Noll; citation_author=Y. Tang; citation_author=S. Nagaraja; citation_author=E.M. Gibson; citation_author=C.W. Mount; citation_author=J. Polepalli; citation_author=S.S. Mitra; citation_author=et al.; citation_journal_title=Cell; citation_volume=161; citation_firstpage=803; citation_lastpage=816; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Electrical and synaptic integration of glioma into neural circuits; citation_author=H.S. Venkatesh; citation_author=W. Morishita; citation_author=A.C. Geraghty; citation_author=D. Silverbush; citation_author=S.M. Gillespie; citation_author=M. Arzt; citation_author=L.T. Tam; citation_author=C. Espenel; citation_author=A. Ponnuswami; citation_author=L. Ni; citation_author=et al.; citation_journal_title=Nature; citation_volume=573; citation_firstpage=539; citation_lastpage=545; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Targeting neuronal activity-regulated neuroligin-3 dependency in high-grade glioma; citation_author=H.S. Venkatesh; citation_author=L.T. Tam; citation_author=P.J. Woo; citation_author=J. Lennon; citation_author=S. Nagaraja; citation_author=S.M. Gillespie; citation_author=J. Ni; citation_author=D.Y. Duveau; citation_author=P.J. Morris; citation_author=J.J. Zhao; citation_author=et al.; citation_journal_title=Nature; citation_volume=549; citation_firstpage=533; citation_lastpage=537; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Optimizing the success of random searches; citation_author=G.M. Viswanathan; citation_author=S.V. Buldyrev; citation_author=S. Havlin; citation_author=M.G.E. da Luz; citation_author=E.P. Raposo; citation_author=H.E. Stanley; citation_journal_title=Nature; citation_volume=401; citation_firstpage=911; citation_lastpage=914; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Lévy flights and superdiffusion in the context of biological encounters and random searches; citation_author=G.M. Viswanathan; citation_author=E.P. Raposo; citation_author=M.G.E. da Luz; citation_journal_title=Phys. Life Rev.; citation_volume=5; citation_firstpage=133; citation_lastpage=150; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The phenotypes of proliferating glioblastoma cells reside on a single axis of variation; citation_author=L. Wang; citation_author=H. Babikir; citation_author=S. Müller; citation_author=G. Yagnik; citation_author=K. Shamardani; citation_author=F. Catalan; citation_author=G. Kohanbash; citation_author=B. Alvarado; citation_author=E. Di Lullo; citation_author=A. Kriegstein; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=9; citation_firstpage=1708; citation_lastpage=1719; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Accurate quantification of astrocyte and neurotransmitter fluorescence dynamics for single-cell and population-level physiology; citation_author=Y. Wang; citation_author=N.V. DelRosso; citation_author=T.V. Vaidyanathan; citation_author=M.K. Cahill; citation_author=M.E. Reitman; citation_author=S. Pittolo; citation_author=X. Mi; citation_author=G. Yu; citation_author=K.E. Poskanzer; citation_journal_title=Nat. Neurosci.; citation_volume=22; citation_firstpage=1936; citation_lastpage=1944; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Synaptic input as a directional cue for migrating interneuron precursors; citation_author=A.K. Wefers; citation_author=C. Haberlandt; citation_author=N.B. Tekin; citation_author=D.A. Fedorov; citation_author=A. Timmermann; citation_author=J.J.L. van der Want; citation_author=F.A. Chaudhry; citation_author=C. Steinhäuser; citation_author=K. Schilling; citation_author=R. Jabs; citation_journal_title=Development; citation_volume=144; citation_firstpage=4125; citation_lastpage=4136; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Content-aware image restoration: pushing the limits of fluorescence microscopy; citation_author=M. Weigert; citation_author=U. Schmidt; citation_author=T. Boothe; citation_author=A. Müller; citation_author=A. Dibrov; citation_author=A. Jain; citation_author=B. Wilhelm; citation_author=D. Schmidt; citation_author=C. Broaddus; citation_author=S. Culley; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=1090; citation_lastpage=1097; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas; citation_author=S. Weil; citation_author=M. Osswald; citation_author=G. Solecki; citation_author=J. Grosch; citation_author=E. Jung; citation_author=D. Lemke; citation_author=M. Ratliff; citation_author=D. Hänggi; citation_author=W. Wick; citation_author=F. Winkler; citation_journal_title=Neuro. Oncol; citation_volume=19; citation_firstpage=1316; citation_lastpage=1326; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Treatment of glioblastoma in adults; citation_author=W. Wick; citation_author=M. Osswald; citation_author=A. Wick; citation_author=F. Winkler; citation_journal_title=Ther. Adv. Neurol. Diso; citation_volume=11; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Targeted in&nbsp;vivo expression of proteins in the calyx of Held; citation_author=V.C. Wimmer; citation_author=T. Nevian; citation_author=T. Kuner; citation_journal_title=Pflugers Arch.; citation_volume=449; citation_firstpage=319; citation_lastpage=333; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Tumor cell network integration in glioma represents a stemness feature; citation_author=R. Xie; citation_author=T. Kessler; citation_author=J. Grosch; citation_author=L. Hai; citation_author=V. Venkataramani; citation_author=L. Huang; citation_author=D.C. Hoffmann; citation_author=G. Solecki; citation_author=M. Ratliff; citation_author=M. Schlesner; citation_author=et al.; citation_journal_title=Neuro. Oncol.; citation_volume=23; citation_firstpage=757; citation_lastpage=769; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies; citation_author=K. Yu; citation_author=Y.Q. Hu; citation_author=F. Wu; citation_author=Q.F. Guo; citation_author=Z.H. Qian; citation_author=W.E. Hu; citation_author=J. Chen; citation_author=K.Y. Wang; citation_author=X.Y. Fan; citation_author=X.L. Wu; citation_author=et al.; citation_journal_title=Natl. Sci. Rev.; citation_volume=7; citation_firstpage=1306; citation_lastpage=1318; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Levy walks; citation_author=V. Zaburdaev; citation_author=S. Denisov; citation_author=J. Klafter; citation_journal_title=Rev. Mod. Phys.; citation_volume=87; citation_firstpage=483; citation_lastpage=530; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Expression of a cleaved brain-specific extracellular matrix protein mediates glioma cell invasion in&nbsp;vivo; citation_author=H. Zhang; citation_author=G. Kelly; citation_author=C. Zerillo; citation_author=D.M. Jaworski; citation_author=S. Hockfield; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=18; citation_firstpage=2370; citation_lastpage=2376; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Direct gap junction communication between malignant glioma cells and astrocytes; citation_author=W. Zhang; citation_author=W.T. Couldwell; citation_author=M.F. Simard; citation_author=H. Song; citation_author=J.H. Lin; citation_author=M. Nedergaard; citation_journal_title=Cancer Res.; citation_volume=59; citation_firstpage=1994; citation_lastpage=2003; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=AMPA receptor-mediated, calcium-dependent CREB phosphorylation in a subpopulation of auditory neurons surviving activity deprivation; citation_author=L. Zirpel; citation_author=M.A. Janowiak; citation_author=C.A. Veltri; citation_author=T.N. Parks; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=20; citation_firstpage=6267; citation_lastpage=6275; citation_publication_date=2000; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Glioblastomas are incurable tumors infiltrating the brain. A subpopulation of glioblastoma cells forms a functional and therapy-resistant tumor cell network interconnected by tumor microtubes (TMs). Other subpopulations appear unconnected, and their biological role remains unclear. Here, we demonstrate that whole-brain colonization is fueled by glioblastoma cells that lack connections with other tumor cells and astrocytes yet receive synaptic input from neurons. This subpopulation corresponds to neuronal and neural-progenitor-like tumor cell states, as defined by single-cell transcriptomics, both in mouse models and in the human disease. Tumor cell invasion resembled neuronal migration mechanisms and adopted a Lévy-like movement pattern of probing the environment. Neuronal activity induced complex calcium signals in glioblastoma cells followed by the <i>de novo</i> formation of TMs and increased invasion speed. Collectively, superimposing molecular and functional single-cell data revealed that neuronal mechanisms govern glioblastoma cell invasion on multiple levels. This explains how glioblastoma's dissemination and cellular heterogeneity are closely interlinked.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7855&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9&amp;be=4746&amp;fe=7527&amp;dc=7213&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749594028028,%22n%22:0,%22f%22:2814,%22dn%22:2815,%22dne%22:2815,%22c%22:2815,%22s%22:2815,%22ce%22:2815,%22rq%22:2815,%22rp%22:4729,%22rpe%22:5829,%22dl%22:4735,%22di%22:6423,%22ds%22:7213,%22de%22:7476,%22dc%22:7527,%22l%22:7527,%22le%22:7601%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style>stream {
  position: relative;
  display: block;
}
</style><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><script src="https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/dynamic/double-struck.js" charset="UTF-8"></script><script src="https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/dynamic/script.js" charset="UTF-8"></script><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-munderover {
  display: inline-block;
  text-align: left;
}

mjx-munderover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-munderover:not([limits="false"]) > * {
  display: block;
}

mjx-mover {
  display: inline-block;
  text-align: left;
}

mjx-mover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-mover:not([limits="false"]) > * {
  display: block;
  text-align: left;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D44B {
  padding: 0.683em 0.851em 0 0;
}

mjx-c.mjx-c27E8 {
  padding: 0.75em 0.389em 0.25em 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c27E9 {
  padding: 0.75em 0.389em 0.25em 0;
}

mjx-c.mjx-c223C {
  padding: 0.366em 0.773em 0 0;
}

mjx-c.mjx-c1D6FC {
  padding: 0.442em 0.64em 0.011em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2192 {
  padding: 0.51em 1em 0.01em 0;
}

mjx-c.mjx-c221E {
  padding: 0.442em 1em 0.011em 0;
}

mjx-c.mjx-c2265 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D53C.MM-DS {
  padding: 0.683em 0.611em 0 0;
}

mjx-c.mjx-c5B.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c5D.MM-S3 {
  padding: 0.975em 0.444em 0.475em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c1D6E5 {
  padding: 0.716em 0.833em 0 0;
}

mjx-c.mjx-c1D441 {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c2208 {
  padding: 0.543em 0.667em 0.043em 0;
}

mjx-c.mjx-c2115.MM-DS {
  padding: 0.683em 0.722em 0 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c1D458 {
  padding: 0.694em 0.521em 0.011em 0;
}

mjx-c.mjx-c28.MM-S4 {
  padding: 1.146em 0.597em 0.646em 0;
}

mjx-c.mjx-c29.MM-S4 {
  padding: 1.146em 0.597em 0.646em 0;
}

mjx-c.mjx-c2248 {
  padding: 0.457em 0.773em 0 0;
}

mjx-c.mjx-c302 {
  padding: 0.734em 0 0 0;
  width: 0;
}

mjx-c.mjx-c1D6FE {
  padding: 0.442em 0.543em 0.215em 0;
}

mjx-c.mjx-c1D43B {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c28.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c67 {
  padding: 0.453em 0.5em 0.206em 0;
}

mjx-c.mjx-c3A {
  padding: 0.431em 0.278em 0 0;
}

mjx-c.mjx-c29.MM-SO {
  padding: 0.797em 0.422em 0.297em 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c1D438 {
  padding: 0.68em 0.764em 0 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c28.MM-S7 {
  padding: 1.745em 0.875em 1.245em 0;
}

mjx-c.mjx-c29.MM-S7 {
  padding: 1.745em 0.875em 1.245em 0;
}

mjx-c.mjx-c1D706 {
  padding: 0.694em 0.583em 0.013em 0;
}

mjx-c.mjx-c1D708 {
  padding: 0.442em 0.524em 0 0;
}

mjx-c.mjx-c5D {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D443 {
  padding: 0.683em 0.754em 0 0;
}

mjx-c.mjx-c1D6FF {
  padding: 0.712em 0.452em 0.011em 0;
}

mjx-c.mjx-c1D44D {
  padding: 0.683em 0.723em 0 0;
}

mjx-c.mjx-c1D707 {
  padding: 0.442em 0.603em 0.216em 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c5B.MM-SO {
  padding: 0.8em 0.34em 0.3em 0;
}

mjx-c.mjx-c5D.MM-SO {
  padding: 0.8em 0.34em 0.3em 0;
}

mjx-c.mjx-c221E.MM-I {
  padding: 0.456em 0.942em 0.025em 0;
}

mjx-c.mjx-c5B.MM-S4 {
  padding: 1.15em 0.472em 0.65em 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c5D.MM-S4 {
  padding: 1.15em 0.472em 0.65em 0;
}

mjx-c.mjx-c28.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c1D457 {
  padding: 0.661em 0.412em 0.205em 0;
}

mjx-c.mjx-c29.MM-S3 {
  padding: 0.972em 0.523em 0.472em 0;
}

mjx-c.mjx-c22C5 {
  padding: 0.303em 0.278em 0 0;
}

mjx-c.mjx-c5B {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D6E4 {
  padding: 0.68em 0.721em 0 0;
}

mjx-c.mjx-c28.MM-S5 {
  padding: 1.296em 0.663em 0.796em 0;
}

mjx-c.mjx-c302.MM-S7 {
  padding: 0.749em 0 0 0;
  width: 1.896em;
}

mjx-c.mjx-c29.MM-S5 {
  padding: 1.296em 0.663em 0.796em 0;
}

mjx-c.mjx-c302.MM-SO {
  padding: 0.746em 0 0 0;
  width: 0.644em;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c7C {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D448 {
  padding: 0.683em 0.76em 0.022em 0;
}

mjx-c.mjx-c73 {
  padding: 0.448em 0.394em 0.011em 0;
}

mjx-c.mjx-c69 {
  padding: 0.657em 0.278em 0 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c1D70B {
  padding: 0.431em 0.57em 0.011em 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D4B0.MM-S {
  padding: 0.727em 0.907em 0.006em 0;
}

mjx-c.mjx-c1D435 {
  padding: 0.683em 0.759em 0 0;
}

mjx-c.mjx-c65 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c78 {
  padding: 0.431em 0.528em 0 0;
}

mjx-c.mjx-c70 {
  padding: 0.442em 0.556em 0.194em 0;
}

mjx-c.mjx-c28.MM-S6 {
  padding: 1.446em 0.736em 0.946em 0;
}

mjx-c.mjx-c222B {
  padding: 0.805em 0.665em 0.306em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c64 {
  padding: 0.694em 0.556em 0.011em 0;
}

mjx-c.mjx-c29.MM-S6 {
  padding: 1.446em 0.736em 0.946em 0;
}

mjx-c.mjx-c211D.MM-DS {
  padding: 0.683em 0.639em 0 0;
}

mjx-c.mjx-c6D {
  padding: 0.442em 0.833em 0 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00847-9"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-59435" aria-expanded="false" aria-controls="login-links-59435" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-59435" id="login-links--59435" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-59435" data-opens="#links-dialog-59435" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-59435" aria-modal="true"><button data-closes="#links-dialog-59435" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900847-9" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.06.054">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00847-9" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0017-7"><span>Volume 185</span>, <span>Issue 16</span></a></span><span class="meta-panel__pages"><span>p2899-2917.e31</span></span><span class="meta-panel__onlineDate">August 04, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=16&amp;pii=S0092-8674%2821%29X0017-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Glioblastoma hijacks neuronal mechanisms for brain invasion</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Varun</span> <span property="familyName">Venkataramani</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Varun</span> <span property="familyName">Venkataramani</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:varun.venkataramani@med.uni-heidelberg.de" property="email" title="Send email to Varun Venkataramani">varun.venkataramani@med.uni-heidelberg.de</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Venkataramani/Varun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup> <a id="eadd0010" href="mailto:varun.venkataramani@med.uni-heidelberg.de" property="email" title="Send email to Varun Venkataramani">varun.venkataramani@med.uni-heidelberg.de</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Yvonne</span> <span property="familyName">Yang</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yvonne</span> <span property="familyName">Yang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Yang/Yvonne" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Marc Cicero</span> <span property="familyName">Schubert</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marc Cicero</span> <span property="familyName">Schubert</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Schubert/Marc+Cicero" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">11</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Ekin</span> <span property="familyName">Reyhan</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ekin</span> <span property="familyName">Reyhan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Reyhan/Ekin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Svenja Kristin</span> <span property="familyName">Tetzlaff</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Svenja Kristin</span> <span property="familyName">Tetzlaff</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Tetzlaff/Svenja+Kristin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Niklas</span> <span property="familyName">Wißmann</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Niklas</span> <span property="familyName">Wißmann</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Wi%C3%9Fmann/Niklas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Michael</span> <span property="familyName">Botz</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael</span> <span property="familyName">Botz</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Botz/Michael" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Stella Judith</span> <span property="familyName">Soyka</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stella Judith</span> <span property="familyName">Soyka</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Soyka/Stella+Judith" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Carlo Antonio</span> <span property="familyName">Beretta</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Carlo Antonio</span> <span property="familyName">Beretta</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Beretta/Carlo+Antonio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Rangel Lyubomirov</span> <span property="familyName">Pramatarov</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rangel Lyubomirov</span> <span property="familyName">Pramatarov</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Pramatarov/Rangel+Lyubomirov" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Laura</span> <span property="familyName">Fankhauser</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Laura</span> <span property="familyName">Fankhauser</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Fankhauser/Laura" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Luciano</span> <span property="familyName">Garofano</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Luciano</span> <span property="familyName">Garofano</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Garofano/Luciano" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Alexander</span> <span property="familyName">Freudenberg</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alexander</span> <span property="familyName">Freudenberg</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Freudenberg/Alexander" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Julia</span> <span property="familyName">Wagner</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Julia</span> <span property="familyName">Wagner</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Wagner/Julia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Dimitar Ivanov</span> <span property="familyName">Tanev</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dimitar Ivanov</span> <span property="familyName">Tanev</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Tanev/Dimitar+Ivanov" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Miriam</span> <span property="familyName">Ratliff</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Miriam</span> <span property="familyName">Ratliff</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Neurosurgery Clinic, University Hospital Mannheim, 68167 Mannheim, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Ratliff/Miriam" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Ruifan</span> <span property="familyName">Xie</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ruifan</span> <span property="familyName">Xie</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Xie/Ruifan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Tobias</span> <span property="familyName">Kessler</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tobias</span> <span property="familyName">Kessler</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Kessler/Tobias" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Dirk C.</span> <span property="familyName">Hoffmann</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dirk C.</span> <span property="familyName">Hoffmann</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Faculty of Biosciences, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Hoffmann/Dirk+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Ling</span> <span property="familyName">Hai</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ling</span> <span property="familyName">Hai</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Hai/Ling" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Yvette</span> <span property="familyName">Dörflinger</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yvette</span> <span property="familyName">Dörflinger</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/D%C3%B6rflinger/Yvette" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Simone</span> <span property="familyName">Hoppe</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Simone</span> <span property="familyName">Hoppe</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Hoppe/Simone" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Yahaya A.</span> <span property="familyName">Yabo</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yahaya A.</span> <span property="familyName">Yabo</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg</span></div></div><div class="core-author-link"><a href="/authored-by/Yabo/Yahaya+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Anna</span> <span property="familyName">Golebiewska</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anna</span> <span property="familyName">Golebiewska</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg</span></div></div><div class="core-author-link"><a href="/authored-by/Golebiewska/Anna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Simone P.</span> <span property="familyName">Niclou</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Simone P.</span> <span property="familyName">Niclou</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg</span></div></div><div class="core-author-link"><a href="/authored-by/Niclou/Simone+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Felix</span> <span property="familyName">Sahm</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Felix</span> <span property="familyName">Sahm</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuropathology, Institute of Pathology, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neuropathology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Sahm/Felix" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Anna</span> <span property="familyName">Lasorella</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anna</span> <span property="familyName">Lasorella</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lasorella/Anna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Martin</span> <span property="familyName">Slowik</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Martin</span> <span property="familyName">Slowik</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Slowik/Martin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au29" aria-controls="axel_author_au29"><span property="givenName">Leif</span> <span property="familyName">Döring</span></a><div id="au29" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Leif</span> <span property="familyName">Döring</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/D%C3%B6ring/Leif" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au30" aria-controls="axel_author_au30"><span property="givenName">Antonio</span> <span property="familyName">Iavarone</span></a><div id="au30" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Antonio</span> <span property="familyName">Iavarone</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Iavarone/Antonio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au31" aria-controls="axel_author_au31"><span property="givenName">Wolfgang</span> <span property="familyName">Wick</span></a><div id="au31" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Wolfgang</span> <span property="familyName">Wick</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Wick/Wolfgang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au32" aria-controls="axel_author_au32"><span property="givenName">Thomas</span> <span property="familyName">Kuner</span></a><div id="au32" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Thomas</span> <span property="familyName">Kuner</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:thomas.kuner@uni-heidelberg.de" property="email" title="Send email to Thomas Kuner">thomas.kuner@uni-heidelberg.de</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Kuner/Thomas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">12</a></sup> <a id="eadd0015" href="mailto:thomas.kuner@uni-heidelberg.de" property="email" title="Send email to Thomas Kuner">thomas.kuner@uni-heidelberg.de</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au33" aria-controls="axel_author_au33"><span property="givenName">Frank</span> <span property="familyName">Winkler</span></a><div id="au33" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Frank</span> <span property="familyName">Winkler</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:frank.winkler@med.uni-heidelberg.de" property="email" title="Send email to Frank Winkler">frank.winkler@med.uni-heidelberg.de</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Winkler/Frank" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">12</a>,<a role="doc-noteref">13</a></sup> <a id="eadd0020" href="mailto:frank.winkler@med.uni-heidelberg.de" property="email" title="Send email to Frank Winkler">frank.winkler@med.uni-heidelberg.de</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Neurology Clinic and National Center for Tumor Diseases, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Functional Neuroanatomy, Institute for Anatomy and Cell Biology, Heidelberg University, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Clinical Cooperation Unit Neurooncology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10027, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Institute of Mathematics, University of Mannheim, 68131 Mannheim, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Neurosurgery Clinic, University Hospital Mannheim, 68167 Mannheim, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Faculty of Biosciences, Heidelberg University, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">NORLUX Neuro-Oncology Laboratory, Department of Cancer Research, Luxembourg Institute of Health, 1526 Luxembourg, Luxembourg</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Neuropathology, Institute of Pathology, University Hospital Heidelberg, 69120 Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Clinical Cooperation Unit Neuropathology, German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received November 12, 2021</span>; <span>Revised April 10, 2022</span>; <span>Accepted June 28, 2022</span>; <span>Published online July 31, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.06.054" property="sameAs" target="_blank">10.1016/j.cell.2022.06.054 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422008479" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.06.054" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0017-7"><img src="/cms/asset/ef1b48a8-ec76-4113-ac5b-c6ee9bc3a720/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 16"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900847-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (2)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900847-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.06.054">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900847-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.06.054&amp;pii=S0092-8674%2822%2900847-9" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900847-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.06.054">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.06.054" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422008479&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422008479&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=16&amp;pii=S0092-8674%2821%29X0017-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00850-9" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00723-1" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (2)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Molecular glioblastoma cell states are related to cell functions</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Neuronal-like, single-glioblastoma cells drive brain invasion</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Glioblastoma invasion resembles multiple traits of neuronal development</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Neurogliomal synapses stimulate neurite-like tumor microtubes crucial for invasion</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Glioblastomas are incurable tumors infiltrating the brain. A subpopulation of glioblastoma cells forms a functional and therapy-resistant tumor cell network interconnected by tumor microtubes (TMs). Other subpopulations appear unconnected, and their biological role remains unclear. Here, we demonstrate that whole-brain colonization is fueled by glioblastoma cells that lack connections with other tumor cells and astrocytes yet receive synaptic input from neurons. This subpopulation corresponds to neuronal and neural-progenitor-like tumor cell states, as defined by single-cell transcriptomics, both in mouse models and in the human disease. Tumor cell invasion resembled neuronal migration mechanisms and adopted a Lévy-like movement pattern of probing the environment. Neuronal activity induced complex calcium signals in glioblastoma cells followed by the <i>de novo</i> formation of TMs and increased invasion speed. Collectively, superimposing molecular and functional single-cell data revealed that neuronal mechanisms govern glioblastoma cell invasion on multiple levels. This explains how glioblastoma’s dissemination and cellular heterogeneity are closely interlinked.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/0684004d-f29c-419b-b4a9-b3bd3e4bd632/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.06.054/asset/5c59ff00-fe74-4fb9-9634-55055bc004ba/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/0684004d-f29c-419b-b4a9-b3bd3e4bd632/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;glioblastoma&quot;" alt="Search on this keyword" rel="nofollow">glioblastoma</a></li><li><a href="/action/doSearch?AllField=&quot;tumor+microtubes&quot;" alt="Search on this keyword" rel="nofollow">tumor microtubes</a></li><li><a href="/action/doSearch?AllField=&quot;tumor+cell+networks&quot;" alt="Search on this keyword" rel="nofollow">tumor cell networks</a></li><li><a href="/action/doSearch?AllField=&quot;cancer+neuroscience&quot;" alt="Search on this keyword" rel="nofollow">cancer neuroscience</a></li><li><a href="/action/doSearch?AllField=&quot;neurogliomal+synapses&quot;" alt="Search on this keyword" rel="nofollow">neurogliomal synapses</a></li><li><a href="/action/doSearch?AllField=&quot;neuron-tumor+interaction&quot;" alt="Search on this keyword" rel="nofollow">neuron-tumor interaction</a></li><li><a href="/action/doSearch?AllField=&quot;cell+heterogeneity&quot;" alt="Search on this keyword" rel="nofollow">cell heterogeneity</a></li><li><a href="/action/doSearch?AllField=&quot;glutamatergic+synapse&quot;" alt="Search on this keyword" rel="nofollow">glutamatergic synapse</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0050" role="paragraph">Glioblastomas (GBs) are incurable brain tumors characterized by their cellular heterogeneity (<span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), invasion, and colonization of the entire brain (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Drumm et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Drumm, M.R. ∙ Dixit, K.S. ∙ Grimm, S. ...</div><div><strong>Extensive brainstem infiltration, not mass effect, is a common feature of end-stage cerebral glioblastomas</strong></div><div><em>Neuro. Oncol.</em> 2020; <strong>22</strong>:470-479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noz216" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85083544250" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31711239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoz216&amp;pmid=31711239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Sahm et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Sahm, F. ∙ Capper, D. ∙ Jeibmann, A. ...</div><div><strong>Addressing diffuse glioma as a systemic brain disease with single-cell analysis</strong></div><div><em>Arch. Neurol.</em> 2012; <strong>69</strong>:523-526</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/archneurol.2011.2910" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84859963131" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22158715/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Farchneurol.2011.2910&amp;pmid=22158715" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), rendering these tumors incurable. GBs also show considerable resistance against standard-of-care treatment with radio- and chemotherapy as well as surgical resection (<span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Wick et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Wick, W. ∙ Osswald, M. ∙ Wick, A. ...</div><div><strong>Treatment of glioblastoma in adults</strong></div><div><em>Ther. Adv. Neurol. Diso.</em> 2018; <strong>11</strong></div><div>1756286418790452</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30083233/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=30083233" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A large subpopulation of GB cells (GBCs) can organize in networks where they are connected with each other via ultralong and thin membrane protrusions, tumor microtubes (TMs), that are the anatomical basis of a highly functional tumor cell network coupled via gap junctions. This network is a key mediator of therapeutic resistance (<span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-1" aria-controls="sref98-1">Weil et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref98-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Weil, S. ∙ Osswald, M. ∙ Solecki, G. ...</div><div><strong>Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas</strong></div><div><em>Neuro. Oncol.</em> 2017; <strong>19</strong>:1316-1326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/nox070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85029925185" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28419303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnox070&amp;pmid=28419303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, neuronal activity—via paracrine signaling and glutamatergic synapses that are formed onto GBC networks—can activate these networks and are associated with increased tumor cell proliferation and invasion (<span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Venkatesh et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Venkatesh, H.S. ∙ Johung, T.B. ∙ Caretti, V. ...</div><div><strong>Neuronal activity promotes glioma growth through Neuroligin-3 secretion</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:803-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2015.04.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2015.04.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84928918997" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25913192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.012&amp;pmid=25913192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">2017</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Venkatesh, H.S. ∙ Tam, L.T. ∙ Woo, P.J. ...</div><div><strong>Targeting neuronal activity-regulated neuroligin-3 dependency in high-grade glioma</strong></div><div><em>Nature.</em> 2017; <strong>549</strong>:533-537</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85030622967" target="_blank" aria-label="Open Scopus in new tab">Scopus (228)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28959975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24014&amp;pmid=28959975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">2019</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Venkatesh, H.S. ∙ Morishita, W. ∙ Geraghty, A.C. ...</div><div><strong>Electrical and synaptic integration of glioma into neural circuits</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:539-545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1563-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072686777" target="_blank" aria-label="Open Scopus in new tab">Scopus (407)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1563-y&amp;pmid=31534222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This could imply that mutually connected GBCs also drive invasion (<span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Gritsenko et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Gritsenko, P.G. ∙ Atlasy, N. ∙ Dieteren, C.E.J. ...</div><div><strong>p120-catenin-dependent collective brain infiltration by glioma cell networks</strong></div><div><em>Nat. Cell Biol.</em> 2020; <strong>22</strong>:97-107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-019-0443-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85077546068" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31907411/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-019-0443-x&amp;pmid=31907411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), similar to collective migration in other cancer entities (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Friedl et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Friedl, P. ∙ Locker, J. ∙ Sahai, E. ...</div><div><strong>Classifying collective cancer cell invasion</strong></div><div><em>Nat. Cell Biol.</em> 2012; <strong>14</strong>:777-783</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84864886120" target="_blank" aria-label="Open Scopus in new tab">Scopus (635)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22854810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2548&amp;pmid=22854810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Alternative current models describe GBC invasion along blood vessels, nerve and astrocytic tracts, and diffusely through the extracellular matrix (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Cuddapah et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Cuddapah, V.A. ∙ Robel, S. ∙ Watkins, S. ...</div><div><strong>A neurocentric perspective on glioma invasion</strong></div><div><em>Nat. Rev. Neurosci.</em> 2014; <strong>15</strong>:455-465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn3765" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84903198110" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24946761/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn3765&amp;pmid=24946761" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In light of these data, two important questions remain: what are the mechanisms of GB brain invasion? And what is the role of GBCs not connected to other tumor cells? How the latter contribute to tumor progression and interact with other cells of the tumor microenvironment, such as astrocytes and neurons, remains unclear.</div><div id="p0055" role="paragraph">The heterogeneous molecular composition of GBCs has been recently classified using pathway-based and gene-expression-based cell states (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Richards et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Richards, L.M. ∙ Whitley, O.K.N. ∙ MacLeod, G. ...</div><div><strong>Gradient of Developmental and Injury Response transcriptional states defines functional vulnerabilities underpinning glioblastoma heterogeneity</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:157-173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00154-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85098800036" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35122077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00154-9&amp;pmid=35122077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-1" aria-controls="sref94-1">Wang et&nbsp;al., 2019a</a><div class="dropBlock__holder" data-db-target-of="sref94-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, L. ∙ Babikir, H. ∙ Müller, S. ...</div><div><strong>The phenotypes of proliferating glioblastoma cells reside on a single axis of variation</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1708-1719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-0329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075958233" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31554641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-0329&amp;pmid=31554641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It remains an open question how these molecular cell states are related to tumor biology, which includes the recently discovered neuronal GBC state (<span class="dropBlock reference-citations"><a id="body-ref-sref22-3" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-3" aria-controls="sref22-3">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Particularly, how those GBC states are functionally related to tumor cell connectivity and ability to invade the normal brain is unclear. In addition, it is still controversial what kind of trajectories brain tumor cell states follow (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Chaligne et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chaligne, R. ∙ Gaiti, F. ∙ Silverbush, D. ...</div><div><strong>Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states</strong></div><div><em>Nat. Genet.</em> 2021; <strong>53</strong>:1469-1479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-021-00927-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85116387369" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34594037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-021-00927-7&amp;pmid=34594037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-2" aria-controls="sref94-2">Wang et&nbsp;al., 2019a</a><div class="dropBlock__holder" data-db-target-of="sref94-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, L. ∙ Babikir, H. ∙ Müller, S. ...</div><div><strong>The phenotypes of proliferating glioblastoma cells reside on a single axis of variation</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1708-1719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-0329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075958233" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31554641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-0329&amp;pmid=31554641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and how these cell states might change with disease progression.</div><div id="p0060" role="paragraph">Here, we integrate molecular and functional states of GBCs to lay the ground for insights into multidimensional tumor cell heterogeneity and its relation to multicellular connectivity and propensity of brain invasion. Longitudinal time-lapse imaging of migrating GBCs <i>in&nbsp;vivo</i> revealed that single GBCs lacking connections to other GBCs and astrocytes are main drivers of diffuse brain tumor invasion. These GBCs were enriched for neuronal, neural-progenitor-like, and non-mesenchymal-like (MES-like) cell states. Infiltrative regions enriched with such tumor cell and astrocyte-unconnected, invasive GBCs transition over time into regions with predominantly tumor cell and astrocyte-connected, stable GBC networks. GBC invasion resembled mechanisms of neuronal migration during development. Lastly, activation of glutamatergic neurogliomal synapses increased GBC invasiveness by stimulating the formation and dynamics of TMs. In summary, three layers of neuronal features governing GBC invasion illustrate parallels between brain development and incurable GB.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>GB networks are functionally connected to astrocytic networks</h3><div id="p0065" role="paragraph">To comprehensively characterize intratumoral heterogeneity, we established a workflow integrating longitudinal 3D <i>in&nbsp;vivo</i> imaging of patient-derived human GBCs in the living mouse with single-cell RNA sequencing (scRNA-seq) data (<a id="crosref0390" href="#fig1">Figure&nbsp;1</a>A, patient-derived gliobastoma xenograft [PDX] models and human GB from resected tissue; <a id="crosref0395" href="#figs1">Figure&nbsp;S1</a>A; <a id="crosref0400" href="#mmc1">Table&nbsp;S1</a>). GBCs were separated into two groups based on their <i>in&nbsp;vivo</i> uptake of the gap junction-permeable dye SR101 (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Nimmerjahn et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Nimmerjahn, A. ∙ Kirchhoff, F. ∙ Kerr, J.N.D. ...</div><div><strong>Sulforhodamine 101 as a specific marker of astroglia in the neocortex in&nbsp;vivo</strong></div><div><em>Nat. Methods.</em> 2004; <strong>1</strong>:31-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-18744409106" target="_blank" aria-label="Open Scopus in new tab">Scopus (636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15782150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth706&amp;pmid=15782150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): GBCs that were anatomically connected with each other via TMs and gap junctions (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) took up SR101 and dynamically spread it across the tumor cell network, while GBCs lacking anatomical connections barely took up SR101 (<a id="crosref0415" href="#fig1">Figures&nbsp;1</a>B and 1C). These mutually connected GBCs showed multicellular co-active calcium events indicating functional connectivity across this GBC subpopulation, while calcium events remained local in GBCs not exhibiting connecting TMs suggesting that they are disconnected from other GBCs (<a id="crosref0420" href="#figs1">Figures&nbsp;S1</a>B–S1D). SR101 accumulated rapidly in astrocytes after intraperitoneal injection while a subpopulation of neighboring GBCs was stained after a delay (<a id="crosref0425" href="#figs1">Figures&nbsp;S1</a>E–S1H). After injection into tumor microregions, SR101 was taken up both by astrocytes and GBCs, which was inhibited by pharmacological gap junction blockade (<a id="crosref0430" href="#fig1">Figures&nbsp;1</a>D and 1E). To further confirm gap junctional coupling of GBCs with each other, with astrocytes, or with both (from here on termed “connected<sup>TUM/AC</sup>”), single GBCs expressing GFP were filled with Alexa 594 via whole-cell patch-clamp recordings performed in co-cultures. This gap junction-permeable dye was dynamically distributed to GBCs and to morphologically distinct astrocytes not expressing GFP (<a id="crosref0435" href="#fig1">Figures&nbsp;1</a>F, <a id="crosref0440" href="#figs1">S1</a>I, and S1J). Consistent with&nbsp;gap junctional coupling, connected<sup>TUM/AC</sup> GBCs had a significantly decreased input resistance as compared with unconnected<sup>TUM/AC</sup> GBCs (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">McKhann et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>McKhann, 2nd, G.M. ∙ D'Ambrosio, R. ∙ Janigro, D.</div><div><strong>Heterogeneity of astrocyte resting membrane potentials and intercellular coupling revealed by whole-cell and gramicidin-perforated patch recordings from cultured neocortical and hippocampal slice astrocytes</strong></div><div><em>J.&nbsp;Neurosci.</em> 1997; <strong>17</strong>:6850-6863</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.17-18-06850.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9278520/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.17-18-06850.1997&amp;pmid=9278520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0450" href="#figs1">Figure&nbsp;S1</a>K). This coupling was reflected in TM-mediated morphological connectivity between GBCs and astrocytes <i>in&nbsp;vitro</i> and <i>in&nbsp;vivo</i> (<a id="crosref0455" href="#fig1">Figures&nbsp;1</a>G and <a id="crosref0460" href="#figs1">S1</a>L). To test whether these two cell populations were also functionally coupled, astrocytes and GBCs were loaded with Rhod-2AM applied at the brain surface. Subsequent intravital calcium imaging revealed co-active events in astrocytes and GBCs (<a id="crosref0465" href="#fig1">Figures&nbsp;1</a>H and 1I), indicating functional coupling of a GBC subpopulation with astrocytes. In summary, GBCs are not only gap-junction-connected among themselves, but also with astrocytes. The discovery of this GB-astrocyte network extends the tumor network concept: from a purely homotypic to an even larger, heterotypic, connected<sup>TUM/AC</sup> network.</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/a11b6971-fa32-472c-a2a8-c7d8fd5b67f6/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.054/asset/9f4bd696-b37c-4fff-82d5-6e6db3a9dbde/main.assets/gr1.jpg" height="679" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/a11b6971-fa32-472c-a2a8-c7d8fd5b67f6/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure 1</span> <span class="figure__title__text">GBCs unconnected to other tumor cells or astrocytes drive tumor invasion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Scheme of the multidimensional analysis workflow.</div><div id="fspara0020" role="paragraph">(B) Intravital two-photon microscopy (IV2PM) of tumor-connected GBCs (yellow, SR101 and GFP), tumor-unconnected GBCs (green, only GFP) and astrocytes (red, only SR101). Post-processed with enhance.ai.</div><div id="fspara0025" role="paragraph">(C) Normalized SR101 gray values per dataset as shown in (B) (n = 125 GBCs from 4 mice, Mann-Whitney test).</div><div id="fspara0030" role="paragraph">(D) IV2PM of SR101 uptake after microinjection into the brain without (left) and with (right) simultaneous carbenoxolone (CBX) microinjection (circle). Arrows = SR101+/connected<sup>TUM/AC</sup> GBCs (yellow), arrowheads = SR101−/unconnected<sup>TUM/AC</sup> GBCs (green), asterisks = brain astrocytes (red).</div><div id="fspara0035" role="paragraph">(E) Normalized SR101 gray values per dataset as shown in (D) (n = 2,374 GBCs and n = 807 astrocytes from 3 mice, Mann-Whitney test).</div><div id="fspara0040" role="paragraph">(F) Confocal image of GFP-positive GBCs in neuron/astrocyte co-culture. Unconnected<sup>TUM/AC</sup> (green), connected<sup>TUM/AC</sup> GBCs (yellow), astrocytes (red), Alexa-594-filled micropipette (dashed lines).</div><div id="fspara0045" role="paragraph">(G) IV2PM 3D rendering (z dimension = 90 μm). Tumor cell somata (blue), astrocyte somata (yellow), tumor-astrocyte connections via TM (green, arrows), tumor-tumor connections via TM (light blue), astrocytic protrusions (gray).</div><div id="fspara0050" role="paragraph">(H) IV2PM time-lapse calcium imaging of GBCs and astrocytes. GBCs (green, 1 and 2), astrocytes (red, 3 and 4), calcium transients in astrocytes (arrows), calcium transients in GBCs (arrowheads).</div><div id="fspara0055" role="paragraph">(I) Representative calcium traces of a co-active calcium wave in tumor cells (1, 2) and astrocytes (3, 4) from (H).</div><div id="fspara0060" role="paragraph">(J) 61h IV2PM time-lapse imaging of a morphologically unconnected<sup>TUM/AC</sup> GBC (green, arrow), blood vessels (red), TM (arrowheads), invasion route (dashed line). Representative of 5 experiments in 5 mice. Post-processed with denoise.ai and “remove outliers” function in ImageJ/Fiji.</div><div id="fspara0065" role="paragraph">(K) 59h IV2PM time-lapse of a morphologically connected<sup>TUM/AC</sup> stationary GBC (arrow, green).</div><div id="fspara0070" role="paragraph">(L) GBC invasion speed over 4 h for SR101+/connected<sup>TUM/AC</sup> and SR101−/unconnected<sup>TUM/AC</sup> GBCs (n = 167 unconnected<sup>TUM/AC</sup> and n = 40 connected<sup>TUM/AC</sup> GBCs from 9 mice, Mann-Whitney test).</div><div id="fspara0075" role="paragraph">(C, E, and L) Data represented as mean ± SEM. S24 GBCs used for all experiments.</div><div id="fspara0080" role="paragraph">See also <a id="crosref2130" href="#figs1">Figure S1</a> and <a id="crosref2135" href="#mmc1">Table S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/28eb2f52-878e-40bd-8224-5c562dabcacc/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.054/asset/1a276d00-d15e-4718-9df8-eb0cdd6dbfab/main.assets/figs1.jpg" height="941" width="720" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/28eb2f52-878e-40bd-8224-5c562dabcacc/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Functional differences between connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> brain tumor cells, related to <a id="crosref2210" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0645" role="paragraph">(A) Representative images of morphologically different tumor cell subtypes with different number of TMs in resected human GB with anti-Nestin immunostaining (far left, <i>ex&nbsp;vivo</i> imaging with confocal microscopy) and in three different PDX models using <i>in&nbsp;vivo</i> two-photon microscopy expressing endogenous mGFP.</div><div id="fspara0650" role="paragraph">(B and C) Intracellular calcium events are seen in tumor-cell-unconnected GBCs whereas tumor-cell-connected GBCs (connectivity based on morphology) show multicellular calcium events.</div><div id="fspara0655" role="paragraph">(B) Arrowheads point to a subcellular calcium event in a TM.</div><div id="fspara0660" role="paragraph">(C) Arrowheads point to a multicellular calcium event in tumor-cell-connected GBCs.</div><div id="fspara0665" role="paragraph">(D) Quantification of the multicellular event frequency in connected<sup>TUM/AC</sup> GBCs and unconnected<sup>TUM/AC</sup> GBCs (n&nbsp;= 37 cells from 5 S24 PDX mice, Mann-Whitney test, connectivity based on morphology).</div><div id="fspara0670" role="paragraph">(E) Exemplary time-lapse IV2PM of SR101 uptake dynamic in S24 PDX GBC <i>in&nbsp;vivo</i> after intraperitoneal application at t&nbsp;= 0&nbsp;min. Astrocyte shown in red (arrowhead), GBC shown in green (arrow). Raw images were post-processed with denoise.ai. Background of SR101 signal was subtracted.</div><div id="fspara0675" role="paragraph">(F) Mean SR101 intensity uptake in GBC (blue) and astrocytes (red) measured in single slices of a z stack over time (n&nbsp;= 17 GBCs and n&nbsp;= 18 astrocytes from 3 S24 PDX mice).</div><div id="fspara0680" role="paragraph">(G) IV2PM time-lapse of SR101 distribution in connected<sup>TUM/AC</sup> GBCs (connectivity based on SR101 uptake) showing how the dye is dynamically distributed between GBCs after intraperitoneal SR101 application at t&nbsp;= 0&nbsp;min.</div><div id="fspara0685" role="paragraph">(H) Fluorescence intensity traces showing SR101 signal intensity at the regions of interest (ROIs) shown in (A)&nbsp;of four GBCs (left panel of G).</div><div id="fspara0690" role="paragraph">(I) Confocal <i>in&nbsp;vitro</i> co-culture time-lapse images of S24 connected<sup>TUM/AC</sup> GBCs with endogenous GFP fluorescence (green). One connected<sup>TUM/AC</sup> GBC is filled with Alexa 594 (red), and the dye gets distributed to surrounding astrocytes (red) and GBC (yellow) indicating connections to GBCs and astrocytes.</div><div id="fspara0695" role="paragraph">(J) Confocal <i>in&nbsp;vitro</i> co-culture images of the exemplary connected<sup>TUM/AC</sup> GBC of <a id="crosref2215" href="#fig1">Figure&nbsp;1</a>F (top) and an exemplary unconnected <sup>TUM/AC</sup> GBC (bottom) with the left panel showing the endogenous GFP fluorescence of the S24 GBCs (green), the middle panel showing the astrocytes and GBCs filled with Alexa 594 (red) and right panel showing the composite images. The Alexa 594 dye remains mostly within the unconnected<sup>TUM/AC</sup> S24 GBC and gets distributed to the astrocytes in the connected<sup>TUM/AC</sup> GBCs.</div><div id="fspara0700" role="paragraph">(K) The electrophysiologically measured input resistance of connected<sup>TUM/AC</sup> GBCs in co-cultures is significantly lower than the input resistance of unconnected<sup>TUM/AC</sup> GBCs indicative of extensive gap junctional coupling of connected<sup>TUM/AC</sup> GBCs, connectivity was morphologically determined (n&nbsp;= 15 connected<sup>TUM/AC</sup> and 8 unconnected<sup>TUM/AC</sup> S24 GBCs, Mann-Whitney test). Data represented as mean ± SEM.</div><div id="fspara0705" role="paragraph">(L) An exemplary <i>ex&nbsp;vivo</i> 3D Rendering of a PDX model showing an anatomical connection between a GBC (green) and an astrocyte (red) (n&nbsp;= 4 connected<sup>TUM/AC</sup> GBCs in n&nbsp;= 3 S24 PDX mice) through a combined staining with antibodies against EAAT2/S100B.</div><div id="fspara0710" role="paragraph">(M) IV2PM time-lapse of an unconnected<sup>TUM/AC</sup>, invasive cell that does not take up SR101 (above) and non-invasive tumor cell network taking up SR101 (below). Raw images were post-processed with enhance.ai.</div><div id="fspara0715" role="paragraph">(N) TMs per GBC acquired with IV2PM in connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs showing significantly more TMs in connected<sup>TUM/AC</sup> GBCs (n&nbsp;= 121 cells in 9 S24 PDX mice, Mann-Whitney test, connectivity based on SR101 uptake).</div><div id="fspara0720" role="paragraph">(O) GBC invasion speed compared for a timespan over 4&nbsp;h for connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs (n&nbsp;= 394 cells from 9 S24 PDX mice, Mann-Whitney test, connectivity based on morphology).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Unconnected<sup>TUM/AC</sup> GBCs are the main drivers of tumor invasion</h3><div id="p0070" role="paragraph">By functionally separating connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs, we first sought to define which of them drive diffuse brain colonization. Time-lapse intravital two-photon microscopy revealed that unconnected<sup>TUM/AC</sup> GBCs are invasive, whereas connected<sup>TUM/AC</sup> GBCs remained mostly stationary (<a id="crosref0470" href="#fig1">Figures&nbsp;1</a>J and 1K). Migration speed significantly differed in connected<sup>TUM/AC</sup> (as evidenced by SR101 uptake or by morphological connectivity) versus unconnected<sup>TUM/AC</sup> GBCs (<a id="crosref0475" href="#fig1">Figures 1</a>L and <a id="crosref0480" href="#figs1">S1</a>M–S1O), demonstrating that GB invasion is dominated by unconnected<sup>TUM/AC</sup> GBCs.</div></section><section id="sec-2-3"><h3>Invasive GBCs harbor neural gene expression signatures</h3><div id="p0075" role="paragraph">To molecularly characterize these functionally distinct tumor cell subpopulations, GFP/SR101-labeled GBCs were isolated using cell sorting and submitted to scRNA-seq (<span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">Hai et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hai, L. ∙ Hoffmann, D.C. ∙ Mandelbaum, H. ...</div><div><strong>A connectivity signature for glioblastoma</strong></div><div>Preprint at <em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.11.07.465791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85121392461" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.11.07.465791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs could be distinguished by their gene expression profiles in three PDX models (<a id="crosref0490" href="#fig2">Figures&nbsp;2</a>A and <a id="crosref0495" href="#figs2">S2</a>A–S2D). To quantitatively characterize changes in cellular transcriptional programs in the context of potential temporal trajectories between connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs, they were sorted according to pseudotime (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1050)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In all three PDX models used here,&nbsp;transcriptional changes appear along the axis from unconnected<sup>TUM/AC</sup> to connected<sup>TUM/AC</sup> GBCs (<a id="crosref0505" href="#fig2">Figures&nbsp;2</a>B, 2C, and <a id="crosref0510" href="#figs2">S2</a>E–S2H). However, this approach does not allow to infer a directionality of the respective trajectories. Therefore, longitudinal intravital two-photon imaging (IV2PM) was used to follow invasion and network formation in a defined brain area at the rim of the tumor (<a id="crosref0515" href="#fig2">Figure&nbsp;2</a>D). Quantitative analysis confirmed an increase of connected<sup>TUM/AC</sup> cells over time and with increasing tumor cell density (<a id="crosref0520" href="#fig2">Figures&nbsp;2</a>E and <a id="crosref0525" href="#figs2">S2</a>I–S2Q; <a id="crosref0530" href="#mmc3">Video S1</a>). Taken together, combined scRNA-seq and IV2PM suggest a concept of how brain colonization is achieved by brain tumors: unconnected<sup>TUM/AC</sup> GBCs are taking the lead, colonize new brain regions, and over time interconnect with other GBCs and astrocytes, transitioning to connected<sup>TUM/AC</sup> GBCs.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/7dde6d90-bba8-4523-9b1e-e3283af8151e/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.054/asset/e5faae35-c0bf-4f01-84c9-863638c2827d/main.assets/gr2.jpg" height="908" width="759" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7dde6d90-bba8-4523-9b1e-e3283af8151e/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Neuronal-like cell states govern brain tumor invasion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0090" role="paragraph">(A) tSNE plot of cells sorted according to SR101 uptake into connected<sup>TUM/AC</sup> or unconnected<sup>TUM/AC</sup> GBCs.</div><div id="fspara0095" role="paragraph">(B) UMAP plot comparing connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs (left) and pseudotime (right).</div><div id="fspara0100" role="paragraph">(C) Bar plot of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs based on pseudotime.</div><div id="fspara0105" role="paragraph">(D) Probability maps of IV2PM of same region over 4&nbsp;weeks. Unconnected<sup>TUM/AC</sup> (green, asterisks, SR101-negative) and connected<sup>TUM/AC</sup> (yellow, arrowhead, SR101-positive) S24 GBCs, astrocytes (red).</div><div id="fspara0110" role="paragraph">(E) Bar plot of average distribution of unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> S24 GBCs in regions from (D)&nbsp;over 4&nbsp;weeks (n&nbsp;= 3,169 GBCs in 24 datasets of 6 S24 PDX mice).</div><div id="fspara0115" role="paragraph">(F) River plots from pathway-based cell states including cells analyzed for unconnected <sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> GBCs.</div><div id="fspara0120" role="paragraph">(G) Distribution of neuronal cell state in core versus rim.</div><div id="fspara0125" role="paragraph">(H) Pie chart for gene-based cell states for connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> cells.</div><div id="fspara0130" role="paragraph">(I) CD44 gene expression in Neftel dataset classified according to the gene-based cell states (n&nbsp;= 6,576 cells).</div><div id="fspara0135" role="paragraph">(J) Confocal maximum intensity projection (z&nbsp;= 15&nbsp;μm) of S24 PDX with CD44 staining (red) and mGFP (green) in mostly unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> GBC-rich regions. Comparison of normalized staining intensity (n&nbsp;= 164 GBCs from 5 S24 PDX mice, Mann-Whitney test, data are represented as mean ± SEM).</div><div id="fspara0140" role="paragraph">(K) MES-like GBC distribution in human GB core compared with rim.</div><div id="fspara0145" role="paragraph">(L and M) tSNE plots based on their neurodevelopmental and injury response transcriptional signatures with connectivity (L)&nbsp;and plotted against the pseudotime (M).</div><div id="fspara0150" role="paragraph">(N) Neurodevelopmental and injury response signatures in comparison of core and rim.</div><div id="fspara0155" role="paragraph">(O) Invasivity score in single cells of core and rim.</div><div id="fspara0160" role="paragraph">(P) Invasivity score classified according to pathway-based cell states.</div><div id="fspara0165" role="paragraph">Connected<sup>TUM/AC</sup> GBCs determined by their SR101 uptake (A–F), (H), (L), or by morphological connectivity (J).</div><div id="fspara0170" role="paragraph">(A, F, H, and L) PDX models (S24, P3XX, T269, n&nbsp;= 35,822 GBCs).</div><div id="fspara0175" role="paragraph">(B, C, and M) S24 PDX model (n&nbsp;= 15,269 GBCs).</div><div id="fspara0180" role="paragraph">(G, K, N, O, and P) Yu dataset (n&nbsp;= 2,795 GBCs).</div><div id="fspara0185" role="paragraph">See also <a id="crosref2140" href="#figs2">Figures&nbsp;S2</a>, <a id="crosref2145" href="#figs3">S3</a>, and <a id="crosref2150" href="#figs4">S4</a>, <a id="crosref2155" href="#mmc2">Table&nbsp;S2</a>, and <a id="crosref2160" href="#mmc3">Video S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/d2eba235-997f-4314-8d83-f947e418ce85/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.054/asset/45149dc6-9ed6-4f96-a0b4-c21ced647489/main.assets/figs2.jpg" height="823" width="761" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/d2eba235-997f-4314-8d83-f947e418ce85/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Single-cell RNA sequencing analyses of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> brain tumor cells, related to <a id="crosref2220" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0730" role="paragraph">(A–D) GO over-representation analysis (biological process: GO:BP; molecular function: GO:MF; cellular component: GO:CC) of differentially expressed genes of S24 (left) and T269 (right) PDX(min.pct&nbsp;= 0.25, logfc.threshold&nbsp;= 0.25) in each cluster using gProfiler (FDR&nbsp;&lt;&nbsp;0.01). Representative pathways are reported.</div><div id="fspara0735" role="paragraph">(B–D) Enrichment of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> cells in each cluster seen in (A)&nbsp;and (C). Reported are the standardized residuals from the chi-squared test. Enrichment of the four functional subtypes in each cluster is shown. Reported are the standardized residuals from the chi-squared test. <sup>∗</sup>&nbsp;= standardized residuals&nbsp;&gt;&nbsp;1.5. S24 PDX (B), T269 PDX (D).</div><div id="fspara0740" role="paragraph">(E) Uniform manifold approximation projection (UMAP) plot of the T269 PDX model in which GBCs are colored based on their connectivity<sup>TUM/AC</sup> (n&nbsp;= 17,178 cells, connectivity based on SR101 uptake.).</div><div id="fspara0745" role="paragraph">(F) UMAP plot of the T269 PDX model in which GBCs are colored based on their pseudotime (n&nbsp;= 17,178 cells, connectivity based on SR101 uptake.).</div><div id="fspara0750" role="paragraph">(G) UMAP plot of the P3XX PDX model, cells are colored based on their connectivity <sup>TUM/AC</sup> (n&nbsp;= 3,015 cells, connectivity based on SR101 uptake.).</div><div id="fspara0755" role="paragraph">(H) UMAP plot of the P3XX PDX model, cells are colored based on their pseudotime (n&nbsp;= 3,015 cells, connectivity based on SR101 uptake.).</div><div id="fspara0760" role="paragraph">(I) Distribution of connected<sup>TUM/AC</sup> and unconnected <sup>TUM/AC</sup> T269 GBCs plotted based on their pseudotime (n&nbsp;= 17,178 cells, connectivity based on SR101 uptake.).</div><div id="fspara0765" role="paragraph">(J) Distribution of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> P3XX GBCs plotted based on their pseudotime (n&nbsp;= 3,015 cells, connectivity based on SR101 uptake.).</div><div id="fspara0770" role="paragraph">(K) Panel shows the fold change of unconnected<sup>TUM/AC</sup> cells, determined by normalization of the percentage over 4&nbsp;weeks to the value determined in the first week, connectivity based on SR101 uptake (n&nbsp;= 1,592 cells in 16 datasets of 4 S24 PDX mice, connectivity based on morphology).</div><div id="fspara0775" role="paragraph">(L) Distribution of unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> S24 GBCs plotted based on their tumor cell density, connectivity based on morphology (n&nbsp;= 4,425 cells in 47 datasets in 18 S24 PDX mice). Data points are plotted with a smoothing function.</div><div id="fspara0780" role="paragraph">(M) IV2PM images with different tumor cell density of connected<sup>TUM/AC</sup> (arrowhead) and unconnected<sup>TUM/AC</sup> S24 GBCs. Connected <sup>TUM/AC</sup> GBCs take up SR101 (red). Raw images were post-processed with enhance.ai.</div><div id="fspara0785" role="paragraph">(N) Distribution of unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> S24 GBCs plotted based on their tumor cell density (n&nbsp;= 4,425 cells in 47 datasets in 18 S24 PDX mice). Data points are plotted with a smoothing function (stat_smooth() in R with a span of 0.55).</div><div id="fspara0790" role="paragraph">(O) Intravital imaging of identical tumor microregions at days 0, 9, 15, and 22 of S24 PDX mice (MIP [z&nbsp;= 30&nbsp;μm]). GBCs are colored in green while blood vessels are colored in blue. Unconnected<sup>TUM/AC</sup> cells are marked with arrows, connectivity based on morphology. Representative series out of 5 such series taken in 5 S24 PDX mice. Raw images were restored in enhance.ai and the vessel channel was post-processed with the “remove outliers” function in ImageJ/Fiji.</div><div id="fspara0795" role="paragraph">(P) Average distribution of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> cells (n&nbsp;= 3,502 cells) in the same region at 2–6 time points in 5 S24 PDX mice, connectivity based on morphology.</div><div id="fspara0800" role="paragraph">(Q) Bar plot of the fold change of connected<sup>TUM/AC</sup> S24 GBCs in tumor regions followed up over 4&nbsp;weeks (n&nbsp;= 3,169 cells in 24 datasets of 6 S24 PDX mice). Datasets are normalized by division of the value at the first time point.</div><div id="fspara0805" role="paragraph">(R) River plots of three PDX models (S24, T269, and P3XX) of each cell line for the pathway-based classification (glycolytic/plurimetabolic [GPM, red], mitochondrial [MTC, green], neuronal [NEU, dark blue], and proliferative/progenitor [PPR, light blue]) for connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs.</div><div id="fspara0810" role="paragraph">(S) Distribution of gene-based cell states for S24 (n&nbsp;= 15,629 cells), T269 (n&nbsp;= 17,178 cells), and P3XX (n&nbsp;= 3,015 cells), connectivity based on SR101 uptake.</div><div id="fspara0815" role="paragraph">(T) Gene expression score along the developmental/injury axis in three PDX datasets, connectivity based on SR101 uptake (n&nbsp;= 35,822 cells).</div><div id="fspara0820" role="paragraph">(S and T) Enrichment of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> cells in each cluster. Reported are the standardized residuals from the chi-squared test. Enrichment of the four functional subtypes in each cluster. Reported are the standardized residuals from the chi-squared test. <sup>∗</sup>&nbsp;= standardized residuals&nbsp;&gt;&nbsp;1.5. S24 PDX (S), T269 PDX (T).</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJiYzdjMDI4YzY5YTcwYzBiMzUzNjc1YWIxZGZkZDVlYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.THNSNiBQ6TauLzaAHffEQJsBWn0rxr7B7HSX29I4vXTbL7jnaOMYxdiQ08AbEssPjvsWpaQqIJQQwcxyMDFWBCxrcJcl9egkU-iOZOW5rU29jKT50GpmrhBfFvB-t94LmB-kHs89XTLl6rNkU9o18ZFcajqCQsL3nJhpDpJSfslHR0oDsIrVt8pMAgJBZgax5RmgAHy2VFJ-2vdNw8puSH5VVu0N7LphvFAm2KiW-6iUTQhVyXck_g5V6HFa1TUk8VObNlKwbgm7APSQpoK5Y_UmMKnqKcPJAuOjXHvNT9ZkO0vZd-1oMQP6eSRYIsQTrGSCQyQGZXoHROoKPIWHgQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJiYzdjMDI4YzY5YTcwYzBiMzUzNjc1YWIxZGZkZDVlYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.THNSNiBQ6TauLzaAHffEQJsBWn0rxr7B7HSX29I4vXTbL7jnaOMYxdiQ08AbEssPjvsWpaQqIJQQwcxyMDFWBCxrcJcl9egkU-iOZOW5rU29jKT50GpmrhBfFvB-t94LmB-kHs89XTLl6rNkU9o18ZFcajqCQsL3nJhpDpJSfslHR0oDsIrVt8pMAgJBZgax5RmgAHy2VFJ-2vdNw8puSH5VVu0N7LphvFAm2KiW-6iUTQhVyXck_g5V6HFa1TUk8VObNlKwbgm7APSQpoK5Y_UmMKnqKcPJAuOjXHvNT9ZkO0vZd-1oMQP6eSRYIsQTrGSCQyQGZXoHROoKPIWHgQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJiYzdjMDI4YzY5YTcwYzBiMzUzNjc1YWIxZGZkZDVlYSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.THNSNiBQ6TauLzaAHffEQJsBWn0rxr7B7HSX29I4vXTbL7jnaOMYxdiQ08AbEssPjvsWpaQqIJQQwcxyMDFWBCxrcJcl9egkU-iOZOW5rU29jKT50GpmrhBfFvB-t94LmB-kHs89XTLl6rNkU9o18ZFcajqCQsL3nJhpDpJSfslHR0oDsIrVt8pMAgJBZgax5RmgAHy2VFJ-2vdNw8puSH5VVu0N7LphvFAm2KiW-6iUTQhVyXck_g5V6HFa1TUk8VObNlKwbgm7APSQpoK5Y_UmMKnqKcPJAuOjXHvNT9ZkO0vZd-1oMQP6eSRYIsQTrGSCQyQGZXoHROoKPIWHgQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.06.054/attachment/b1d9e47d-3bac-46d7-a165-d4656e593125/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (71.19 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S1. Evolution in GBC and the malignant TM-network over time through intravital SR101 imaging, related to Figure&nbsp;2</div><div id="espara0010" role="paragraph"><i>In&nbsp;vivo</i> imaging of unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> over the course of four weeks. GBCs are shown here in the S24 PDX model. Raw data were trained in ilastik (Berg et&nbsp;al., 2019) and the probability map is shown here with an object segmentation of the SR101 signal.</div></div></div></div><div id="p0085" role="paragraph">Next, we superimposed intravital imaging and scRNA-seq results to identify key molecular and functional cellular states driving tumor progression. Unconnected<sup>TUM/AC</sup> cells were predominately enriched for the neuronal (NEU) (<a id="crosref0535" href="#fig2">Figures&nbsp;2</a>F, 2G, and <a id="crosref0540" href="#figs2">S2</a>R; <span class="dropBlock reference-citations"><a id="body-ref-sref22-4" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-4" aria-controls="sref22-4">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the oligodendrocyte precursor-like (OPC-like)/neural precursor-like (NPC-like) cell states (<a id="crosref0550" href="#fig2">Figures&nbsp;2</a>H–2K and <a id="crosref0555" href="#figs2">S2</a>S; <span class="dropBlock reference-citations"><a id="body-ref-sref52-3" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-3" aria-controls="sref52-3">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and neurodevelopmental transcriptional signatures (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Richards et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Richards, L.M. ∙ Whitley, O.K.N. ∙ MacLeod, G. ...</div><div><strong>Gradient of Developmental and Injury Response transcriptional states defines functional vulnerabilities underpinning glioblastoma heterogeneity</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:157-173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00154-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85098800036" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35122077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00154-9&amp;pmid=35122077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0570" href="#fig2">Figures&nbsp;2</a>F, 2L–2N, and <a id="crosref0575" href="#figs2">S2</a>T). In contrast, connected<sup>TUM/AC</sup> GBCs were enriched for the non-neuronal cell states (glycolytic/plurimetabolic [GPM], mitochondrial [MTC], proliferative/progenitor [PPR]) (<span class="dropBlock reference-citations"><a id="body-ref-sref22-5" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-5" aria-controls="sref22-5">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0585" href="#fig2">Figure&nbsp;2</a>F), and astrocyte-like (AC-like)/MES-like cell states (<span class="dropBlock reference-citations"><a id="body-ref-sref52-4" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-4" aria-controls="sref52-4">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0595" href="#fig2">Figures&nbsp;2</a>H–2K and <a id="crosref0600" href="#figs2">S2</a>S). Furthermore, connected<sup>TUM/AC</sup> GBCs exhibited an injury response transcriptional signature (<span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Richards et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Richards, L.M. ∙ Whitley, O.K.N. ∙ MacLeod, G. ...</div><div><strong>Gradient of Developmental and Injury Response transcriptional states defines functional vulnerabilities underpinning glioblastoma heterogeneity</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:157-173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00154-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85098800036" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35122077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00154-9&amp;pmid=35122077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in three different PDX models, confirmed in human scRNA-seq datasets of GB core and infiltration zones (<a id="crosref0610" href="#fig2">Figures&nbsp;2</a>L–2N and <a id="crosref0615" href="#figs2">S2</a>T). The predominance of the neuronal cell state in unconnected<sup>TUM/AC</sup> GBCs is consistent with an enriched neuronal cell state in the tumor rim versus the core in human GB (<span class="dropBlock reference-citations"><a id="body-ref-sref22-6" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-6" aria-controls="sref22-6">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102-1" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-1" aria-controls="sref102-1">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref102-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Yu, K. ∙ Hu, Y.Q. ∙ Wu, F. ...</div><div><strong>Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies</strong></div><div><em>Natl. Sci. Rev.</em> 2020; <strong>7</strong>:1306-1318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwaa099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85092585099" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34692159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwaa099&amp;pmid=34692159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0630" href="#fig2">Figure&nbsp;2</a>G). Together, the neuronal cell state coincides with invasion at the infiltration zone.</div><div id="p0090" role="paragraph">Accordingly, the mesenchymal marker CD44 (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Hara et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Hara, T. ∙ Chanoch-Myers, R. ∙ Mathewson, N.D. ...</div><div><strong>Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:779-792.e11</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.ccell.2021.05.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.ccell.2021.05.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34087162/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.05.002&amp;pmid=34087162" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-5" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-5" aria-controls="sref52-5">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) stained high in densely connected<sup>TUM/AC</sup> GBC regions, but not unconnected<sup>TUM/AC</sup> ones (<a id="crosref0645" href="#fig2">Figure&nbsp;2</a>J), and also in the GB core versus rim (<a id="crosref0650" href="#fig2">Figure&nbsp;2</a>K), consistent with an enrichment in MES-like and non-neuronal cell states (<a id="crosref0655" href="#fig2">Figures&nbsp;2</a>I and <a id="crosref0660" href="#figs3">S3</a>A–S3D). This contrasts with work suggesting a major role for the mesenchymal cell state in GBC invasion (<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Iwadate, 2016</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Iwadate, Y.</div><div><strong>Epithelial-mesenchymal transition in glioblastoma progression</strong></div><div><em>Oncol. Lett.</em> 2016; <strong>11</strong>:1615-1620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3892/ol.2016.4113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84958047483" target="_blank" aria-label="Open Scopus in new tab">Scopus (175)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26998052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3892%2Fol.2016.4113&amp;pmid=26998052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) but agrees with early observations that mesenchymal development was accompanied by multicellular network formation via long membrane protrusions (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Lewis, 1922</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Lewis, W.H.</div><div><strong>Is mesenchyme a syncytium?</strong></div><div><em>Anat. Rec.</em> 1922; <strong>23</strong>:177-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ar.1090230204" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84944483553" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Far.1090230204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, the gap junction gene GJA1 was also enriched in non-neuronal cell states (<a id="crosref0675" href="#figs3">Figures&nbsp;S3</a>E–S3H) and in MES/AC-like cell states, consistent with broad gap junctional coupling found in mesenchymal cells and astrocytes (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Bodi et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bodi, E. ∙ Hurtado, S.P. ∙ Carvalho, M.A. ...</div><div><strong>Gap junctions in hematopoietic stroma control proliferation and differentiation of blood cell precursors</strong></div><div><em>An. Acad. Bras. Cienc.</em> 2004; <strong>76</strong>:743-756</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1590/S0001-37652004000400009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15558154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1590%2FS0001-37652004000400009&amp;pmid=15558154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Dorshkind et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Dorshkind, K. ∙ Green, L. ∙ Godwin, A. ...</div><div><strong>Connexin-43 type gap-junctions mediate communication between bone-marrow stromal cells</strong></div><div><em>Blood.</em> 1993; <strong>82</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood.V82.1.38.bloodjournal82138" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8391877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood.V82.1.38.bloodjournal82138&amp;pmid=8391877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Valiunas et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Valiunas, V. ∙ Doronin, S. ∙ Valiuniene, L. ...</div><div><strong>Human mesenchymal stem cells make cardiac connexins and form functional gap junctions</strong></div><div><em>J.&nbsp;Physiol. Lond.</em> 2004; <strong>555</strong>:617-626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1113/jphysiol.2003.058719" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-12144286490" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1113%2Fjphysiol.2003.058719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/2d24d472-2208-4468-a95b-37919e99de0e/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.054/asset/bfff812d-f7df-417d-9038-b9e90ecff931/main.assets/figs3.jpg" height="698" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/2d24d472-2208-4468-a95b-37919e99de0e/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Single-cell RNA sequencing analyses of glioblastoma cell states and relation to the invasivity score, related to <a id="crosref2225" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0845" role="paragraph">(A) Violin plot displaying CD44 gene expression of GBCs in the Yu dataset split by gene-based cell state assignment (n&nbsp;= 2,795 cells).</div><div id="fspara0850" role="paragraph">(B) Violin plot displaying CD44 gene expression of GBCs in the Neftel dataset split by gene-based cell state assignment (n&nbsp;= 6,576 cells).</div><div id="fspara0855" role="paragraph">(C) Violin plot displaying CD44 gene expression of GBCs in the Yu dataset split by the pathway-based cell state assignment (n&nbsp;= 2,795 cells).</div><div id="fspara0860" role="paragraph">(D) Violin plot displaying CD44 gene expression of GBCs in the Yu dataset split by sample localization of rim and core regions (n&nbsp;= 2,795 cells).</div><div id="fspara0865" role="paragraph">(E) Violin plot displaying GJA1 gene expression of GBCs in the Neftel dataset split by gene-based cell state assignment (n&nbsp;= 6,576 cells).</div><div id="fspara0870" role="paragraph">(F) Violin plot displaying GJA1 gene expression of GBCs in the Yu dataset split by gene-based cell state assignment (n&nbsp;= 2,795 cells).</div><div id="fspara0875" role="paragraph">(G) Violin plot displaying GJA1 gene expression of GBCs in the Neftel dataset split by pathway-based cell state assignment (n&nbsp;= 6,576 cells).</div><div id="fspara0880" role="paragraph">(H) Violin plot displaying GJA1 gene expression of GBCs in the Yu dataset split by pathway-based cell state assignment (n&nbsp;= 2,795 cells).</div><div id="fspara0885" role="paragraph">(I) Schematic of the analysis workflow with which the invasivity score for GBCs was determined.</div><div id="fspara0890" role="paragraph">(J and K) UMAP plot of GBCs in the Yu dataset. GBCs are colored based on their localization in the core and rim of the tumor (J). A UMAP plot of GBCs of the Yu dataset is shown in which cells are colored based on their invasivity score (K) (n&nbsp;= 2,795 cells).</div><div id="fspara0895" role="paragraph">(L) Violin plot of GBCs in the PDX dataset displaying the invasivity score expression split by the gene-based cell state assignment (n&nbsp;= 35,822 cells).</div><div id="fspara0900" role="paragraph">(M) Violin plot of GBCs in the Neftel dataset displaying the invasivity score expression split by the pathway-based cell state assignment (n&nbsp;= 35,822 cells).</div><div id="fspara0905" role="paragraph">(N) Violin plot in the Yu dataset displaying the invasivity score expression split by the gene-based cell state assignment (n&nbsp;= 6,576 cells).</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">In addition, a transcriptional gradient between neurodevelopmental and injury response was linked to molecular GB heterogeneity (<span class="dropBlock reference-citations"><a id="body-ref-sref62-4" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-4" aria-controls="sref62-4">Richards et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Richards, L.M. ∙ Whitley, O.K.N. ∙ MacLeod, G. ...</div><div><strong>Gradient of Developmental and Injury Response transcriptional states defines functional vulnerabilities underpinning glioblastoma heterogeneity</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:157-173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00154-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85098800036" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35122077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00154-9&amp;pmid=35122077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In line with findings showing that connected<sup>TUM/AC</sup> GBCs increased TM formation and self-repair after surgical lesions, photoablation and radio/chemotherapy (<span class="dropBlock reference-citations"><a id="body-ref-sref55-3" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-3" aria-controls="sref55-3">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-2" aria-controls="sref98-2">Weil et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref98-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Weil, S. ∙ Osswald, M. ∙ Solecki, G. ...</div><div><strong>Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas</strong></div><div><em>Neuro. Oncol.</em> 2017; <strong>19</strong>:1316-1326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/nox070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85029925185" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28419303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnox070&amp;pmid=28419303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), single-cell RNA-seq in PDX models showed a high transcriptional injury signature in connected<sup>TUM/AC</sup> GBCs, whereas unconnected<sup>TUM/AC</sup> cells were enriched for a neurodevelopmental transcriptional signature (<a id="crosref0710" href="#fig2">Figures&nbsp;2</a>L–2N and <a id="crosref0715" href="#figs2">S2</a>T).</div><div id="p0100" role="paragraph">In conclusion, neuronal, OPC/NPC-like cell states and a neurodevelopmental transcriptional signature were enriched in unconnected<sup>TUM/AC</sup> GBCs, whereas MES-like, non-neuronal cell states, and a transcriptional injury signature were enriched in connected<sup>TUM/AC</sup> GBCs.</div><div id="p0105" role="paragraph">To define molecular changes occurring during tumor evolution from unconnected<sup>TUM/AC</sup> to connected<sup>TUM/AC</sup> cellular states, we derived an invasivity gene expression signature from three PDX models based on genes changing as a function of pseudotime from unconnected<sup>TUM/AC</sup> to connected<sup>TUM/AC</sup> cells (<a id="crosref0720" href="#figs3">Figure&nbsp;S3</a>I; <a id="crosref0725" href="#mmc2">Table&nbsp;S2</a>). This signature was enriched in the tumor rim versus core of tissue obtained from GB patients (<a id="crosref0730" href="#fig2">Figures&nbsp;2</a>O, <a id="crosref0735" href="#figs3">S3</a>J, and S3K), as well as in the neuronal cell state and OPC/NPC-like cell states (<a id="crosref0740" href="#fig2">Figures&nbsp;2</a>P and <a id="crosref0745" href="#figs3">S3</a>L–S3N). Furthermore, the invasivity score correlated well with the distribution of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBC states, the distribution of the neuronal cell state, and morphological invasiveness (<a id="crosref0750" href="#figs4">Figures S4</a>A–S4F). Lastly, the gene signatures of the invasivity score were positively correlated with genes and GO terms of neuronal features, migration, and invasion (<a id="crosref0755" href="#figs4">Figures&nbsp;S4</a>G–S4J; <a id="crosref0760" href="#mmc2">Table&nbsp;S2</a>; <a id="crosref0765" href="#sec-4">STAR Methods</a>).</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/b9f258a1-a5ab-43ab-9781-9098795cf401/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.054/asset/ffe3ff99-cd30-41e2-98ae-8b300a718a96/main.assets/figs4.jpg" height="849" width="761" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/b9f258a1-a5ab-43ab-9781-9098795cf401/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Characterization of the invasivity score, related to <a id="crosref2230" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0915" role="paragraph">(A) Percentage of connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> in P3XX, S24, and T269. Connected<sup>TUM/AC</sup> GBCs were based on their SR101 uptake (n&nbsp;= 3,015, n&nbsp;= 15,629, and n&nbsp;= 17,178 cells, respectively).</div><div id="fspara0920" role="paragraph">(B) Percentage of NEU cells in S24 and T269 (n&nbsp;= 3,015, n&nbsp;= 15,629, and n&nbsp;= 17,178 cells, respectively), based on the pathway-based classification.</div><div id="fspara0925" role="paragraph">(C) Representative <i>ex&nbsp;vivo</i> confocal imaging of the three different PDX cell lines expressing mGFP (maximum intensity projection of z&nbsp;= 46&nbsp;μm). Raw images were post-processed with denoise.ai and the “remove outliers” function in Fiji/ImageJ.</div><div id="fspara0930" role="paragraph">(D) Invasivity score based on different phenotypes of PDXs models (n&nbsp;= 35,822 cells from 3 PDX models [P3xx, S24, T269]).</div><div id="fspara0935" role="paragraph">(E) Histopathological characteristics of PDOX models. Hematoxylin and eosin, human-specific stainings with antibodies against vimentin and mouse-specific CD31 stainings are shown for PDXs representative for three histopathological types on the left side: angiogenic (P13), intermediate (T16), and invasive (T347). Scale bars represent 1&nbsp;mm (black) and 100&nbsp;μm (white).</div><div id="fspara0940" role="paragraph">(F) Invasivity score based on different phenotypes of PDOXs models, exemplary stainings are shown in (E) (n&nbsp;= 8,837 cells from 9 PDOX models [P13, P3, P8, T101, T16, T192, T233, T347, T470]).</div><div id="fspara0945" role="paragraph">(G) Top GO terms of high invasivity genes (n&nbsp;= 30 GO terms). Enriched GO terms were determined using ShinyGO.</div><div id="fspara0950" role="paragraph">(H) Dot plot showing the expression of top 20 high invasivity genes along pseudotime in the PDX dataset (n&nbsp;= 35,822).</div><div id="fspara0955" role="paragraph">(I) Correlation of genes to pseudotime, top 20 anticorrelating genes are shown (PDX dataset n&nbsp;= 35,822 cells).</div><div id="fspara0960" role="paragraph">(J) Percentage of GeneCards descriptions of high and low invasivity genes that contain the terms “migration,” “motility,” or “movement” in the GeneCards “summaries”-table (n&nbsp;= 172 genes).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4"><h3>DeepISTI: A workflow to study TM dynamics in high resolution</h3><div id="p0110" role="paragraph">As neuronal and neural progenitor-like cell states were associated with invasivity of GBCs, we next wanted to understand how cellular neuronal features relate to brain tumor invasion. GBCs stably expressing mGFP to reveal finest structural details such as filopodia were intravitally time-lapse-imaged at reduced phototoxicity while maintaining spatial and temporal resolution using deep learning image restoration (<span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Weigert et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Weigert, M. ∙ Schmidt, U. ∙ Boothe, T. ...</div><div><strong>Content-aware image restoration: pushing the limits of fluorescence microscopy</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:1090-1097</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0216-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85057342134" target="_blank" aria-label="Open Scopus in new tab">Scopus (422)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0216-7&amp;pmid=30478326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We refer to this approach as deep-learning-enabled intravital subcellular time-lapse imaging (DeepISTI, <a id="crosref0775" href="#fig3">Figures&nbsp;3</a>A, 3B, and <a id="crosref0780" href="#figs5">S5</a>A; <a id="crosref0785" href="#mmc4">Video S2</a>; see <a id="crosref0790" href="#sec-4">STAR Methods</a>). The signal-to-noise ratio strongly increased with DeepISTI, providing a basis for subsequent automated image analyses (<a id="crosref0795" href="#fig3">Figures&nbsp;3</a>B, 3C, <a id="crosref0800" href="#figs5">S5</a>A, and S5B).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/311aed84-0248-4ee8-963e-c4a1685549a1/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.054/asset/ab438768-8f5c-4b5e-b845-dd9171532c33/main.assets/gr3.jpg" height="940" width="714" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/311aed84-0248-4ee8-963e-c4a1685549a1/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure 3</span> <span class="figure__title__text">Heterogeneous cell soma and TM mechanisms drive GBC invasion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0195" role="paragraph">(A) Scheme of the DeepISTI workflow.</div><div id="fspara0200" role="paragraph">(B) Raw IV2PM image compared with restored, enhance.ai (green), enhance.ai merged with probability map (white).</div><div id="fspara0205" role="paragraph">(C) Signal-to-noise ratio at different steps of the image processing workflow and different IV2PM time points of the same GBC.</div><div id="fspara0210" role="paragraph">(D) 3D renderings of a segmented human GBC resected from a GB patient (left, confocal microscopy) and segmented GBCs from PDX model (right, IV2PM). TMs (green), GBC somata (blue).</div><div id="fspara0215" role="paragraph">(E–G) Characterization of TM versus soma volume per GBC, surface area per TMs and GBC soma, as well as relative contribution of surface area compared with volume from TMs to somata (n = 79 GBCs from 10 patients, Wilcoxon matched-pairs signed rank test).</div><div id="fspara0220" role="paragraph">(H–J) Characterization of TMs versus somata volume per GBC (H), surface area per TM and GBC soma (I), as well as the relative contribution of surface area compared with volume from TMs to somata (J) in the S24 PDX model (n = 123 GBCs from 7 S24 PDX mice, Wilcoxon matched-pairs signed rank test).</div><div id="fspara0225" role="paragraph">(K) Probability map of <i>ex vivo</i> confocal imaging, PDX mouse, brain slices stained with anti-beta-III-tubulin antibody, a pan-neuronal marker (red, average intensity projection). 3D rendering of the same region, segmented GBC (green), contact areas with neurons (red).</div><div id="fspara0230" role="paragraph">(L) Morphometry of the relative contact surface area between GBC and neurons in TMs versus somata (n = 50 GBCs from 3 mice, Wilcoxon matched-pairs signed rank test).</div><div id="fspara0235" role="paragraph">(M) IV2PM of TM dynamics in somatokinetic and non-somatokinetic cells in S24 PDX. GBC soma (arrowhead), blind-ending TM (arrows).</div><div id="fspara0240" role="paragraph">(N) Comparison of soma movement with process turnover in somatokinetic GBC (n = 11 GBCs with their respective soma and processes from 5 mice, Wilcoxon test).</div><div id="fspara0245" role="paragraph">(O) As in (H) but non-somatokinetic GBC (n = 10 GBCs with their respective soma and processes from 3 mice, paired t test).</div><div id="fspara0250" role="paragraph">(P) Comparison of TM turnover per TM in somatokinetic versus non-somatokinetic GBCs (n = 103 TMs from 4 mice, Mann-Whitney test).</div><div id="fspara0255" role="paragraph">(Q) IV2PM of TM dynamics in blind-ending and connected TMs.</div><div id="fspara0260" role="paragraph">(R) TM turnover in blind-ending versus connected TMs per TM (n = 586 blind TMs versus n = 110 connected TMs from 9 mice, Mann-Whitney test).</div><div id="fspara0265" role="paragraph">(S) TM turnover in unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> GBCs (n = 245 TMs of unconnected<sup>TUM/AC</sup> and n = 125 of connected<sup>TUM/AC</sup> GBCs from 9 mice, Mann-Whitney test). Connectivity based on morphology.</div><div id="fspara0270" role="paragraph">(T) IV2PM of TM dynamics in blind-ending TM (arrowhead) of GBC (green, mGFP) in 4 h.</div><div id="fspara0275" role="paragraph">(G, J, L, N–P, R, and S) Data represented as mean ± SEM.</div><div id="fspara0280" role="paragraph">(M, Q, and T) Post-processed with enhance.ai and the “remove outliers” function in ImageJ/Fiji.</div><div id="fspara0285" role="paragraph">S24 GBCs used for all experiments.</div><div id="fspara0290" role="paragraph">See also <a id="crosref2165" href="#figs5">Figure S5</a> and <a id="crosref2170" href="#mmc4">Video S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/cee47c56-5760-4cd1-8f1b-9b7bc62c3b13/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.054/asset/c137927c-0d1e-4def-9997-442b40b99e21/main.assets/figs5.jpg" height="907" width="761" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/cee47c56-5760-4cd1-8f1b-9b7bc62c3b13/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Mechanistic characterization of subcellular components driving GBC invasion, related to <a id="crosref2235" href="#fig3">Figures&nbsp;3</a> and <a id="crosref2240" href="#fig4">4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0970" role="paragraph">(A) Comparison of raw IV2PM images to restored images with enhance.ai and subsequent processing with a pixel classification workflow in ilastik.</div><div id="fspara0975" role="paragraph">(B) TM tracks (left side) that were manually tracked by two independent investigators projected onto an exemplary IV2PM image exemplifying the interrater variability and robustness of the TM tracking workflow (see <a id="crosref2245" href="#sec-4">STAR Methods</a>). Quantification of interrater variability for manual TM tracking of intravitally imaged GBCs on the right side.</div><div id="fspara0980" role="paragraph">(C) 3D rendering of segmented GBCs from two PDX models (BG7, BG5) imaged with IV2PM. TMs are shown in green while the GBC somata are shown in blue.</div><div id="fspara0985" role="paragraph">(D–F) Protrusion, retraction and branching examples of TMs in the BG5 PDX model captured with IV2PM. Blood vessels are shown in blue while the GBCs are green. Arrow heads indicate dynamic blind-ending TMs. Raw images were post-processed with enhance.ai, and background was removed with the “remove outlier” function in ImageJ/Fiji.</div><div id="fspara0990" role="paragraph">(G) Mean TM growth speed of TMs with normal growth compared with the subgroup of TMs with TM genesis (n&nbsp;= 72 TMs from 31 somatokinetic GBCs from 4 S24 PDX mice, Mann-Whitney test).</div><div id="fspara0995" role="paragraph">(H) TM turnover of GBCs with TM genesis as compared with GBCs without TM genesis (n&nbsp;= 51 cells from 4 S24 PDX mice, Mann-Whitney test).</div><div id="fspara1000" role="paragraph">(I) Invasion speed of GBCs with TM genesis compared with GBCs without TM genesis (n&nbsp;= 55 cells from 4 S24 PDX mice, Mann-Whitney test).</div><div id="fspara1005" role="paragraph">(J) Representative confocal microscopy of branching (above) and non-branching (below) GBC in resected patient material. Arrowheads point to the branching points.</div><div id="fspara1010" role="paragraph">(K) Relative abundance of branching versus non-branching GBC (n&nbsp;= 54 uni- or bipolar cells from 6 patients). Data in (G)&nbsp;and (K)–(M) are represented as mean ± SEM.</div><div id="fspara1015" role="paragraph">(L and M) Diameter (L)&nbsp;and maximum length (M)&nbsp;comparison between the three different small process classes (n&nbsp;= 291 processes from 8 S24 PDX mice, Kruskal-Wallis test).</div><div id="fspara1020" role="paragraph">(N) Transition plot of newly developing GBC processes (n&nbsp;= 210 processes from 8 S24 PDX mice).</div><div id="fspara1025" role="paragraph">(O) Scheme showing different classes of small process GBC dynamics.</div><div id="fspara1030" role="paragraph">(P) Quantification of TM growth efficiency (netto growth divided by overall traveled track distance) between TMs and small processes (n&nbsp;= 138 TMs from 8 S24 PDX mice, Mann-Whitney test).</div><div id="fspara1035" role="paragraph">(Q and R) IV2PM example of invasion phenotypes in BG7 GBCs (green) over 4 h. Blood vessels are colored in blue. Raw images were post-processed with enhance.ai, and background signal was removed with the “remove outliers” plugin in ImageJ/Fiji.</div><div id="fspara1040" role="paragraph">(S and T) Branching migration examples in two GBC lines (S24 in S and BG5 in T) in co-cultures with astrocytes and neurons. Raw images were post-processed with denoise.ai.</div><div id="fspara1045" role="paragraph">(U) Quantitative comparison of different invasive mechanisms <i>in&nbsp;vivo</i> of cumulative displacement of somata, netto TM dynamics, TM length of the leading TM and displacement of the TM length and the histogram of the displacement steps of exemplary S24 GBCs.</div><div id="fspara1050" role="paragraph">(V) Percentages of somatokinetic GBCs and distribution of invasion phenotypes for the BG7 and BG5 PDX model (BG7: n&nbsp;= 108 non-somatokinetic cells and 25 somatokinetic GBCs in 2 mice. BG5: n&nbsp;= 60 non-somatokinetic cells and n&nbsp;= 8 somatokinetic GBCs in 2 mice). Data in (C)&nbsp;are represented as mean ± SEM.</div><div id="fspara1055" role="paragraph">(W and X) Histogram of the angle of directionality change in the leading TM (W)&nbsp;as compared with the histogram of the branching angles (X) (n&nbsp;= 66 TMs [M] and n&nbsp;= 85 TMs [N]).</div><div id="fspara1060" role="paragraph">(Y) <i>In&nbsp;vitro</i> live cell time-lapse imaging of a S24 branching migrating GBC with a directionality change mediated by branching. The timespan shown here covers 500&nbsp;min. Raw images were post-processed with denoise.ai.</div></div></div></div></figcaption></figure></div><div id="p0115" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkYWM2N2Q1Y2QyZGQwMzM4YjBlMzU2NWQxNjhlNzVjMiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.D8y7PRc0LXdkrX4sZMAMckchI0ytEVaN6nuiV5irPbSNq0jFBOudBIdcm7SD3ArEDxuTRRHBAjJXqKZ3V8tLvO5V-6oWebu6HLt55RbgXoh0ub3ppuVR_zdUONrcQcNxem-nGGDV6KCU6OgD6D6W_0Atzchb14jMmH-7-CX6GqNQ5hV6MhcC9rpSHFatGEQLsPhfpXhWwsTU8hhwFqC2oF8AdUKLb26hva7Q9qpgnm8Z81ru7o_eZffay9VAIkNLOfGvY_iUgHAFINAyrfZPgVr8Me1PEEGdNUHHQcNuWzuwWegX20_LcuRfbhQUTQd59HPLQzXqeP1kzCioY6xV8w</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkYWM2N2Q1Y2QyZGQwMzM4YjBlMzU2NWQxNjhlNzVjMiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.D8y7PRc0LXdkrX4sZMAMckchI0ytEVaN6nuiV5irPbSNq0jFBOudBIdcm7SD3ArEDxuTRRHBAjJXqKZ3V8tLvO5V-6oWebu6HLt55RbgXoh0ub3ppuVR_zdUONrcQcNxem-nGGDV6KCU6OgD6D6W_0Atzchb14jMmH-7-CX6GqNQ5hV6MhcC9rpSHFatGEQLsPhfpXhWwsTU8hhwFqC2oF8AdUKLb26hva7Q9qpgnm8Z81ru7o_eZffay9VAIkNLOfGvY_iUgHAFINAyrfZPgVr8Me1PEEGdNUHHQcNuWzuwWegX20_LcuRfbhQUTQd59HPLQzXqeP1kzCioY6xV8w" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJkYWM2N2Q1Y2QyZGQwMzM4YjBlMzU2NWQxNjhlNzVjMiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.D8y7PRc0LXdkrX4sZMAMckchI0ytEVaN6nuiV5irPbSNq0jFBOudBIdcm7SD3ArEDxuTRRHBAjJXqKZ3V8tLvO5V-6oWebu6HLt55RbgXoh0ub3ppuVR_zdUONrcQcNxem-nGGDV6KCU6OgD6D6W_0Atzchb14jMmH-7-CX6GqNQ5hV6MhcC9rpSHFatGEQLsPhfpXhWwsTU8hhwFqC2oF8AdUKLb26hva7Q9qpgnm8Z81ru7o_eZffay9VAIkNLOfGvY_iUgHAFINAyrfZPgVr8Me1PEEGdNUHHQcNuWzuwWegX20_LcuRfbhQUTQd59HPLQzXqeP1kzCioY6xV8w" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.06.054/attachment/7ca420af-9162-44b3-a5b5-e068f96fa9c4/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (14.28 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S2. DeepISTI workflow and effects on signal-to-noise ratio for intravital imaging, related to Figures&nbsp;3 and 4</div><div id="espara0025" role="paragraph">In the first part, the DeepISTI workflow is illustrated with <i>in vivo</i> two-photon microscopy data and the respective signal-to-noise ratio (SNR) is calculated. This is displayed together with the respective 3D rendering of multiple GBCs with their TMs in the S24 PDX model. In the second part, the three basic mechanisms of TM dynamics are shown in this time-lapse video of single GBC imaged with <i>in vivo</i> two-photon microscopy. For each mechanism (TM branching, TM protrusion and TM retraction) a time-lapse video is shown from S24 PDX mice. In the third part, examples of the dynamic behavior in three classes of small processes (filopodia, TM-like, lamellipodia) are shown here as 3D renderings of <i>in&nbsp;vivo</i> time-lapse imaging of BG5 and BG7 PDX mice.</div></div></div></div><div id="p0120" role="paragraph">Before using DeepISTI, we determined the contribution of cellular compartments to cell volume and surface because these parameters govern the interaction of GBC with the brain parenchyma. A quantitative morphometric analysis showed that in single GBCs, TMs contributed mainly to GBC volume and surface area and served as the predominant site of interaction with neurons in the brain (<a id="crosref0805" href="#fig3">Figures&nbsp;3</a>D–3L and <a id="crosref0810" href="#figs5">S5</a>C).</div></section><section id="sec-2-5"><h3>GBCs use neuronal cellular mechanisms for migration and invasion</h3><div id="p0125" role="paragraph">To assess the dynamics of GBCs and their processes, we used DeepISTI to intravitally follow individual cells over several hours in PDX models. <a id="crosref0815" href="#fig3">Figure&nbsp;3</a>M shows an unconnected<sup>TUM/AC</sup> GBC that extended two processes and moved the soma in the same direction, while retracting the trailing process. In contrast, the soma remained at the same position in non-somatokinetic cells while the TM extended further only little (<a id="crosref0820" href="#fig3">Figure&nbsp;3</a>M). Somatokinetic cells exhibited a higher TM turnover (<a id="crosref0825" href="#fig3">Figures&nbsp;3</a>N–3P). To understand whether all TMs contribute to the invasive process, we subclassified them based on whether they are connected to other TMs or whether they are blind-ending (<a id="crosref0830" href="#fig3">Figure&nbsp;3</a>Q). We found that blind-ending TMs contribute largely to TM invasion, as they showed a much higher turnover than tumor-connected TMs (<a id="crosref0835" href="#fig3">Figure&nbsp;3</a>R). As expected, unconnected<sup>TUM/AC</sup> GBCs showed a higher TM turnover than connected<sup>TUM/AC</sup> ones (<a id="crosref0840" href="#fig3">Figure&nbsp;3</a>S). When focusing on invasive, blind-ending TMs we found three main mechanisms of TM movement patterns consisting of protrusion, retraction, and TM generation via TM branching (<a id="crosref0845" href="#fig3">Figures&nbsp;3</a>T and <a id="crosref0850" href="#figs5">S5</a>D–S5K; <a id="crosref0855" href="#mmc4">Video S2</a>). Interestingly, these mechanisms were reminiscent of dynamics occurring during migration of immature neurons (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Marín et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Marín, O. ∙ Valiente, M. ∙ Ge, X. ...</div><div><strong>Guiding neuronal cell migrations</strong></div><div><em>Cold Spring Harbor Perspect. Biol.</em> 2010; <strong>2</strong>:a001834</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a001834" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77957277593" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20182622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a001834&amp;pmid=20182622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Martini et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Martini, F.J. ∙ Valiente, M. ∙ López Bendito, G. ...</div><div><strong>Biased selection of leading process branches mediates chemotaxis during tangential neuronal migration</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:41-50</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.025502" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-60749096087" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19060332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.025502&amp;pmid=19060332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). On a smaller spatial scale we found filopodial extensions (<a id="crosref0870" href="#fig4">Figures&nbsp;4</a>A–4C, <a id="crosref0875" href="#figs5">S5</a>L, and S5M), with only a few converting into TM-like protrusions (<a id="crosref0880" href="#fig4">Figures&nbsp;4</a>A, 4D, and <a id="crosref0885" href="#figs5">S5</a>N–S5P; <a id="crosref0890" href="#mmc4">Video S2</a>). TM-like protrusions were thicker and longer than filopodia, and more stable, yet about 50% of them were eliminated (<a id="crosref0895" href="#fig4">Figures&nbsp;4</a>B and 4D; <a id="crosref0900" href="#figs5">Figure&nbsp;S5</a>L). Finally, lamellipodial protrusions could transform into TM-like protrusions, persist, or get eliminated at equal chance (<a id="crosref0905" href="#fig4">Figures&nbsp;4</a>C and 4D). We also identified these small processes on the ultrastructural level (<a id="crosref0910" href="#fig4">Figures&nbsp;4</a>E and 4F). When compared with classical TMs, small protrusions were shorter, more dynamic, and less efficient in their growth (<a id="crosref0915" href="#figs5">Figures&nbsp;S5</a>M and S5P). In summary, we suggest that TMs and associated small membrane protrusions mediate the dynamic scanning of the brain microenvironment.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/0d63dd00-3217-4a79-92e8-75c485801798/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.054/asset/a190164f-05e7-48bc-8693-52e01f4d2969/main.assets/gr4.jpg" height="699" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/0d63dd00-3217-4a79-92e8-75c485801798/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure 4</span> <span class="figure__title__text">Neuronal-like cellular mechanisms of brain tumor invasion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0300" role="paragraph">(A–C) IV2PM post-processed using probability maps showing small process dynamics of GBCs (green).</div><div id="fspara0305" role="paragraph">(D) Transition plot of GBC protrusions over 4 h analyzed with DeepISTI (n = 226 processes from 7 mice).</div><div id="fspara0310" role="paragraph">(E and F) Electron microscopy images of small protrusions (E) (n = 45 small protrusions in n = 11 GBCs from 7 PDX mice) and resected GB patient material (F) (n = 13 small protrusions in n = 4 GBCs).</div><div id="fspara0315" role="paragraph">(G) IV2PM of the tip of a TM with projected TM track indicating dynamics over 4 h (asterisk, stationary phases; arrow head, invasive phases).</div><div id="fspara0320" role="paragraph">(H) Mean-squared displacement of GB TM step lengths (<i>in vivo</i> TM tracking). Red line: same in control paradigm. Gray line: best-fit linear regression. Dotted: mean-squared displacement of a diffusive process.</div><div id="fspara0325" role="paragraph">(I) Hill plot of TM track step sizes.</div><div id="fspara0330" role="paragraph">(J) Schematic and examples of 3 neuronal-like mechanisms of GBC invasion in PDX mice. Leading edge of TM (arrowhead). Post-processed with enhance.ai.</div><div id="fspara0335" role="paragraph">(K) Total TM turnover per GBC of different invasion mechanisms (n = 140 GBCs of 16 mice, Kruskal-Wallis test).</div><div id="fspara0340" role="paragraph">(L) <i>In vivo</i> invasion speed of different migration patterns (n = 140 GBCs from 16 mice, Kruskal-Wallis test).</div><div id="fspara0345" role="paragraph">(M) Percentage of somatokinetic cells and relative abundance of distinct invasion mechanisms in somatokinetic GBCs (n = 400 GBCs, of these n = 140 invasive GBCs from 16 mice).</div><div id="fspara0350" role="paragraph">(N) <i>In vivo</i> example of a GBC showing different leading TM angles.</div><div id="fspara0355" role="paragraph">(O) Comparison of leading TM angles during elongation and branching TM angles (n = 151 TM from 7 mice, Mann-Whitney test).</div><div id="fspara0360" role="paragraph">(P) Example of a GBC <i>in vivo</i> with branching migration and directionality change during 24 h. Soma (arrow), TM tree (arrow head), migration path (dashed line).</div><div id="fspara0365" role="paragraph">(K, L, and O) Data represented as mean ± SEM.</div><div id="fspara0370" role="paragraph">S24 GBCs used for all experiments.</div><div id="fspara0375" role="paragraph">See also <a id="crosref2175" href="#figs6">Figure S6</a> and <a id="crosref2180" href="#mmc4">Video S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">When TMs were manually tracked over time (<a id="crosref0920" href="#fig4">Figure&nbsp;4</a>G), the analysis revealed small steps interspersed with bigger steps, characteristic of Lévy-like movement (<span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Huda et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Huda, S. ∙ Weigelin, B. ∙ Wolf, K. ...</div><div><strong>Levy-like movement patterns of metastatic cancer cells revealed in microfabricated systems and implicated in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06563-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85055771401" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30382086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06563-w&amp;pmid=30382086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103-1" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-1" aria-controls="sref103-1">Zaburdaev et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref103-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Zaburdaev, V. ∙ Denisov, S. ∙ Klafter, J.</div><div><strong>Levy walks</strong></div><div><em>Rev. Mod. Phys.</em> 2015; <strong>87</strong>:483-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/RevModPhys.87.483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84937578864" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FRevModPhys.87.483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Several mathematical approaches confirmed that GB TM growth follows a stochastic process with a heavy-tailed step-length distribution. This can be interpreted as Lévy-like (<a id="crosref0935" href="#fig4">Figures&nbsp;4</a>H and 4I, <a id="crosref0940" href="#sec-4">STAR Methods</a>), a pattern that has been described in animal predators looking for scarce sources of food (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Humphries et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Humphries, N.E. ∙ Weimerskirch, H. ∙ Queiroz, N. ...</div><div><strong>Foraging success of biological Levy flights recorded <i>in situ</i></strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:7169-7174</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1121201109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84860816827" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22529349/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1121201109&amp;pmid=22529349" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Sims et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Sims, D.W. ∙ Southall, E.J. ∙ Humphries, N.E. ...</div><div><strong>Scaling laws of marine predator search behaviour</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:1098-1102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06518" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-39849092398" target="_blank" aria-label="Open Scopus in new tab">Scopus (726)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18305542/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06518&amp;pmid=18305542" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">2014</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Sims, D.W. ∙ Reynolds, A.M. ∙ Humphries, N.E. ...</div><div><strong>Hierarchical random walks in trace fossils and the origin of optimal search behavior</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11073-11078</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1405966111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84905042767" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25024221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1405966111&amp;pmid=25024221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and may reflect an optimal search strategy for sparsely distributed targets (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Raposo et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Raposo, E.P. ∙ Buldyrev, S.V. ∙ da Luz, M.G. ...</div><div><strong>Dynamical robustness of Levy search strategies</strong></div><div><em>Phys. Rev. Lett.</em> 2003; <strong>91</strong>:240601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.91.240601" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0742321825" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14683101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.91.240601&amp;pmid=14683101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Viswanathan et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Viswanathan, G.M. ∙ Buldyrev, S.V. ∙ Havlin, S. ...</div><div><strong>Optimizing the success of random searches</strong></div><div><em>Nature.</em> 1999; <strong>401</strong>:911-914</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/44831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0033613429" target="_blank" aria-label="Open Scopus in new tab">Scopus (1137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10553906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F44831&amp;pmid=10553906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" aria-label="Reference 2008" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">2008</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Viswanathan, G.M. ∙ Raposo, E.P. ∙ da Luz, M.G.E.</div><div><strong>Lévy flights and superdiffusion in the context of biological encounters and random searches</strong></div><div><em>Phys. Life Rev.</em> 2008; <strong>5</strong>:133-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.plrev.2008.03.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-49849093656" target="_blank" aria-label="Open Scopus in new tab">Scopus (189)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.plrev.2008.03.002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, TM invasion follows a search-efficient mechanism.</div><div id="p0135" role="paragraph">To understand patterns of GBC invasion in more depth, we further examined the dynamics of TMs and somata of invading, unconnected<sup>TUM/AC</sup>, somatokinetic GBCs using DeepISTI. We found three major invasion mechanisms of GBCs: branching migration, locomotion, and translocation (<a id="crosref0975" href="#fig4">Figures&nbsp;4</a>J and <a id="crosref0980" href="#figs5">S5</a>Q–S5T). Branching migration showed extensive branching, protrusion and retraction of the TMs during the invasion of these cells, whereas locomotion appeared in uni- or bipolar cells where the leading TM was protruding, and the soma was following the leading TM. Lastly, translocation occurred with the TM stably anchored and the soma translocating with the length of the TM shortening during this process (<a id="crosref0985" href="#fig4">Figures&nbsp;4</a>J and <a id="crosref0990" href="#figs5">S5</a>U). These three mechanisms of GBC invasion were comparable to neuronal progenitor (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Nadarajah et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Nadarajah, B. ∙ Brunstrom, J.E. ∙ Grutzendler, J. ...</div><div><strong>Two modes of radial migration in early development of the cerebral cortex</strong></div><div><em>Nat. Neurosci.</em> 2001; <strong>4</strong>:143-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/83967" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0035144045" target="_blank" aria-label="Open Scopus in new tab">Scopus (545)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11175874/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F83967&amp;pmid=11175874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" aria-label="Reference 2003" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">2003</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Nadarajah, B. ∙ Alifragis, P. ∙ Wong, R.O. ...</div><div><strong>Neuronal migration in the developing cerebral cortex: observations based on real-time imaging</strong></div><div><em>Cereb. Cortex.</em> 2003; <strong>13</strong>:607-611</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/cercor/13.6.607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0038662777" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12764035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcercor%2F13.6.607&amp;pmid=12764035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Tabata and Nakajima, 2003</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Tabata, H. ∙ Nakajima, K.</div><div><strong>Multipolar migration: the third mode of radial neuronal migration in the developing cerebral cortex</strong></div><div><em>J.&nbsp;Neurosci.</em> 2003; <strong>23</strong>:9996-10001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.23-31-09996.2003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14602813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.23-31-09996.2003&amp;pmid=14602813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and other neural progenitor cells (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Tsai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Tsai, H.H. ∙ Niu, J. ∙ Munji, R. ...</div><div><strong>Oligodendrocyte precursors migrate along vasculature in the developing nervous system</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:379-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad3839" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84964306884" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26798014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad3839&amp;pmid=26798014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) migrating during brain development. Quantitative analyses of these processes revealed that branching migration was the mechanism with the highest TM turnover, and also the predominant mechanism of GB invasion (<a id="crosref1015" href="#fig4">Figures&nbsp;4</a>K–4M and <a id="crosref1020" href="#figs5">S5</a>V). We hypothesized that TM branching during GBC invasion may induce directionality changes during invasion. Indeed, GBCs followed newly formed leading TM branches during migration (<a id="crosref1025" href="#fig4">Figures&nbsp;4</a>N–4P and <a id="crosref1030" href="#figs5">S5</a>W–S5Y). Furthermore, the directionality change of leading TMs during protrusion and retraction was significantly lower than in branched TMs (<a id="crosref1035" href="#fig4">Figures&nbsp;4</a>O and <a id="crosref1040" href="#figs5">S5</a>W–S5Y). This mechanism differs from axonal growth cone pathfinding but resembles interneuron movement during tangential migration (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Martini et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Martini, F.J. ∙ Valiente, M. ∙ López Bendito, G. ...</div><div><strong>Biased selection of leading process branches mediates chemotaxis during tangential neuronal migration</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:41-50</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.025502" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-60749096087" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19060332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.025502&amp;pmid=19060332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Valiente and Martini, 2009</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Valiente, M. ∙ Martini, F.J.</div><div><strong>Migration of cortical interneurons relies on branched leading process dynamics</strong></div><div><em>Cell Adh. Migr.</em> 2009; <strong>3</strong>:278-280</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/cam.3.3.8832" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-68249083818" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19535916/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Fcam.3.3.8832&amp;pmid=19535916" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, GBCs adopt migration strategies initially described for neuronal progenitor cells during development, a feature that is likely related to their neuronal- and neural progenitor-like transcriptional states.</div></section><section id="sec-2-6"><h3>Neuronal activity drives TM formation and TM growth</h3><div id="p0140" role="paragraph">Next, we asked how neurogliomal synapses (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Venkatesh et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Venkatesh, H.S. ∙ Morishita, W. ∙ Geraghty, A.C. ...</div><div><strong>Electrical and synaptic integration of glioma into neural circuits</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:539-545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1563-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072686777" target="_blank" aria-label="Open Scopus in new tab">Scopus (407)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1563-y&amp;pmid=31534222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) would impact the formation and growth of TMs and tumor invasion. For this, neuronal activity was modulated either by varying anesthesia depth or by <i>in&nbsp;vivo</i> optogenetic activation of neurons (<a id="crosref1065" href="#figs6">Figures&nbsp;S6</a>A–S6C). We found that neuronal activity increased the number of TM branching events and TM turnover (<a id="crosref1070" href="#fig5">Figures&nbsp;5</a>A–5F), implying that neuronal input drives TM dynamics in the context of invasion. Neuronal activity also increased the average step length of TM movements per GBC (<a id="crosref1075" href="#fig5">Figures&nbsp;5</a>G and 5H), and Lévy-like movements were significantly accelerated (<a id="crosref1080" href="#fig5">Figures&nbsp;5</a>I, 5J, and <a id="crosref1085" href="#figs6">S6</a>D–S6K). In effect, these mechanisms lead to a neuronal-activity-related significant increase in invasion speed of GBCs (<a id="crosref1090" href="#fig5">Figures&nbsp;5</a>K and 5L).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/fb2d90d2-7e9b-40db-bcd9-755a212145b2/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.054/asset/46a6cf0e-1997-4c43-8415-cc01cd45c7bf/main.assets/gr5.jpg" height="941" width="734" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/fb2d90d2-7e9b-40db-bcd9-755a212145b2/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Neuronal activity stimulates genesis and growth of invasive TMs</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0385" role="paragraph">(A) Comparison of unconnected<sup>TUM/AC</sup> GBC under light versus deep anesthesia with IV2PM. GBC (green), blood vessels (blue). Dynamic TMs and TM genesis of the unconnected<sup>TUM/AC</sup> GBCs under light anesthesia (arrow heads). Post-processed with enhance.ai and “remove outliers” function in ImageJ/Fiji.</div><div id="fspara0390" role="paragraph">(B) Branching events per GBC <i>in&nbsp;vivo</i> under isoflurane anesthesia (n&nbsp;= 122 GBCs under deep and n&nbsp;= 158 GBCs under light anesthesia from 6 versus 6 mice, Mann-Whitney test).</div><div id="fspara0395" role="paragraph">(C) As in (B)&nbsp;but neuronal channelrhodopsin stimulation compared with control (n&nbsp;= 45 GBCs under mCherry control conditions and n&nbsp;= 57 GBCs under mCherry-ChR2 conditions from 3 versus 4 mice, Mann-Whitney test).</div><div id="fspara0400" role="paragraph">(D) TM turnover per GBC and TM under isoflurane anesthesia (n&nbsp;= 122 GBCs under deep and n&nbsp;= 158 GBCs under light anesthesia from 6 versus 6 mice, Mann-Whitney test).</div><div id="fspara0405" role="paragraph">(E) As in (D), but neuronal channelrhodopsin stimulation as compared with control (n&nbsp;= 45 GBCs under mCherry control conditions and n&nbsp;= 57 GBCs under mCherry-ChR2 conditions from 3 versus 4 mice, Mann-Whitney test).</div><div id="fspara0410" role="paragraph">(F) Exemplary time-lapse IV2PM of somatokinetic S24 GBCs after neuronal channelrhodopsin stimulation and control conditions.</div><div id="fspara0415" role="paragraph">(G) Average step length of TM dynamics per GBC under isoflurane anesthesia (n&nbsp;=122 GBCs under deep and n&nbsp;= 158 under light anesthesia GBCs from 6 versus 6 mice, Mann-Whitney test).</div><div id="fspara0420" role="paragraph">(H) As in (F), but neuronal channelrhodopsin stimulation as compared with control (n&nbsp;= 45 GBCs under mCherry control conditions and n&nbsp;= 57 GBCs under mCherry-ChR2 conditions from 3 versus 4 mice, Mann-Whitney test).</div><div id="fspara0425" role="paragraph">(I) Hill plot of TM movement steps of neuronal channelrhodopsin stimulation compared with mCherry control.</div><div id="fspara0430" role="paragraph">(J) Mean-squared displacement of TM movement steps of a neuronal channelrhodopsin stimulation paradigm compared with control conditions. The black lines illustrate the best-fit linear regression.</div><div id="fspara0435" role="paragraph">(K) Invasion speed of somatokinetic GBCs under isoflurane anesthesia n&nbsp;= 26 under deep and n&nbsp;= 60 GBCs under light anesthesia from 6 versus 6 mice, Mann-Whitney test).</div><div id="fspara0440" role="paragraph">(L) Invasion speed of somatokinetic GBCs after neuronal channelrhodopsin stimulation versus control (n&nbsp;= 19 GBCs under mCherry control conditions and n&nbsp;= 29 GBCs under mCherry-ChR2 conditions from 3 versus 4 mice, Mann-Whitney test).</div><div id="fspara0445" role="paragraph">(M) Unconnected<sup>TUM/AC</sup> GBC identified using <i>ex&nbsp;vivo</i> correlative light and electron microscopy. Pre-synaptic axons are shown in sepia, and post-synaptic GBC TMs are shown in green. Confocal images post-processed with denoise.ai.</div><div id="fspara0450" role="paragraph">(N) Confocal image of unconnected<sup>TUM/AC</sup> GBC (red, expression of tdTomato) in co-culture. Cell filled with Alexa488 (green), no connected<sup>TUM/AC</sup> cells detectable.</div><div id="fspara0455" role="paragraph">(O) Representative spontaneous EPSC recorded from unconnected<sup>TUM/AC</sup> GBC shown in (M) (n&nbsp;= 12 GBCs from 7 mice).</div><div id="fspara0460" role="paragraph">(B–E, G, H, K, and L) Data represented as mean ± SEM.</div><div id="fspara0465" role="paragraph">S24 GBCs used for all experiments.</div><div id="fspara0470" role="paragraph">See also <a id="crosref2185" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/7a13b7b2-1e60-4881-aff2-fb489205395d/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.054/asset/5797308e-caac-435d-953f-d758bb3874a4/main.assets/figs6.jpg" height="914" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7a13b7b2-1e60-4881-aff2-fb489205395d/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Neuronal activity-driven TM dynamics and neurogliomal synaptic input, related to <a id="crosref2250" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara1070" role="paragraph">(A) Exemplary monitoring of deep and light anesthesia with isoflurane based on the breathing rate (n&nbsp;= 49 time points in 10 PDX mice).</div><div id="fspara1075" role="paragraph">(B) Validation of the channelrhodopsin construct (mChR2-H134) in AAV-infected neurons in co-cultures with astrocytes and GBCs stimulated with blue light followed by currents produced by the opening of voltage-gated channels in the course of an action potential after optogenetic stimulation. Upper panels: images of neurons in co-cultures with astrocytes and GBCs infected with AAV-mCherry-ChR2-H134 virus. On the left side infected neurons expressing mCherry can be seen. On the right side, a patched neuron filled via the patch pipette with Alexa 488 can be seen. Lower panel: the red bars indicate light pulses and the trace below shows the current trace of the neuron that was patched in voltage-clamp mode reliably showing currents produced by the opening of voltage-gated channels in the course of an action potential after optogenetic stimulation with 10&nbsp;Hz.</div><div id="fspara1080" role="paragraph">(C) Expression of CaMKIIahChR2(H134R)-mCherry (AAV2) in the infiltration zone. Confocal maximum intensity projection (z&nbsp;= 10&nbsp;μm) of a representative S24 PDX mouse brain slice infected with CaMKIIahChR2(H134R)-mCherry (AAV2) (red). The tumor cells express mGFP (green), neurons were stained with NeuN (white), and all nuclei were stained with DAPI (blue) (n&nbsp;= 3 S24 PDX mice).</div><div id="fspara1085" role="paragraph">(D) Mean-squared displacement of TM movement steps under light and deep anesthesia conditions. Red line: mean-squared displacement of GBC TM tracks in the light isoflurane paradigm, indicating a superdiffusive behavior. Gray line: best-fit linear regression. Black: mean-squared displacement of a diffusive process.</div><div id="fspara1090" role="paragraph">(E) Mean-squared variation plot for the control and light isoflurane paradigm. The black lines indicate the expected behavior under independent and stationary increments.</div><div id="fspara1095" role="paragraph">(F) Dekkers-Einmal-de Haan plot of TM movement steps under deep and light anesthesia conditions.</div><div id="fspara1100" role="paragraph">(G) Tail functions of positive travel distance under light and deep anesthesia conditions.</div><div id="fspara1105" role="paragraph">(H) Hill plot of TM movement steps under deep and light anesthesia conditions in S24 PDX mice.</div><div id="fspara1110" role="paragraph">(I) Mean-squared variation plot for the mCherry control and the ChR2 stimulation paradigm. The black lines indicate the expected behavior under independent and stationary increments.</div><div id="fspara1115" role="paragraph">(J) Dekkers-Einmal-de Haan plot of TM movement steps under neuronal channelrhodopsin-mCherry-stimulated and control conditions.</div><div id="fspara1120" role="paragraph">(K) Tail functions of positive travel distance of TM movement steps under neuronal channelrhodopsin-mCherry-stimulated and control conditions.</div><div id="fspara1125" role="paragraph">(L) Confocal image of neuronal/astrocytic co-culture with S24 GBCs (green). A pipette filled with Alexa 594 (red) is used for patch-clamp of GBC. The unconnected<sup>TUM/AC</sup> GBC is filled with Alexa 594 (red) while other GBCs stay green, and no astrocytes are filled with the red dye.</div><div id="fspara1130" role="paragraph">(M) The left panel shows correlative light and electron microscopy (CLEM) of two connected<sup>TUM/AC</sup> GBCs (mGFP-positive GBCs shown in green and nuclei stained with DAPI in blue) with 3D-reconstruction of the connection and scanning electron microscopy image of the connection. One GBC is colored in green, the other one in magenta. The right panel shows CLEM imaging of a connected<sup>TUM/AC</sup> GBC with a 3D reconstruction of the cell using manual segmentation of 153 2D-EM-sections. The lower part of the right panel shows series of 2D-sections of a synaptic contact on the connected<sup>TUM/AC</sup> GBC with zoom-in and 3D-EM-reconstruction. 3D reconstructions were performed in TrakEM2 and visualized in Autodesk 3ds max.</div><div id="fspara1135" role="paragraph">(N) Confocal image of neuronal/astrocytic co-culture with S24 GBCs (green). A pipette filled with Alexa 594 (red) is used for patch-clamp of GBC. The connected<sup>TUM/AC</sup> GBC is connected to other tumor cells and astrocytes, which are filled with the gap-junction-permeable Alexa 594.</div><div id="fspara1140" role="paragraph">(O) Scanning electron microscopy image of a synaptic contact on an unconnected<sup>TUM/AC</sup> GBC, connectivity based on morphology. GBC is shown in green, neuronal presynaptic bouton in yellow (left side). Scanning electron microscopy image of a synaptic contact on a connected<sup>TUM/AC</sup> GBC, connectivity based on morphology (right side).</div><div id="fspara1145" role="paragraph">(P) Scanning electron microscopy image of a healthy neuron-neuron-synapse, presynaptic part colored in yellow, post-synaptic part in blue.</div><div id="fspara1150" role="paragraph">(Q–V) Quantification of morphometric parameters of synapses on unconnected<sup>TUM/AC</sup> versus connected<sup>TUM/AC</sup> GBCs versus neuron-neuron synapses (NNSs). Unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> GBCs were identified via correlative light and electron microscopy. Unconnected<sup>TUM/AC</sup> GBCs do not have connections to other tumor cells. Connected<sup>TUM/AC</sup> GBCs exhibit at least one connection to other GBCs. (n&nbsp;= 17 synapses on 5 unconnected<sup>TUM/AC</sup> GBCs from 3 S24 PDX mice; n&nbsp;=31 synapses on 8 connected<sup>TUM/AC</sup> GBCs from 3 S24 PDX mice, connectivity based on morphology; n&nbsp;= 55 neuron-neuron synapses from 4 PDX S24 mice; Kruskal-Wallis test).</div><div id="fspara1155" role="paragraph">(W) A somatokinetic and a non-somatokinetic S24 PDX GBC <i>in&nbsp;vivo</i> are shown together with putative neurogliomal presynaptic bouton. Putative neurogliomal presynaptic boutons are shown in blue (arrows) that are transient over time. Stable synaptic boutons are shown in yellow (arrowheads).</div><div id="fspara1160" role="paragraph">(Y) Percentage of transient putative neurogliomal presynaptic boutons (see <a id="crosref2255" href="#sec-4">STAR Methods</a>) compared between somatokinetic and non-somatokinetic PDX GBCs (n&nbsp;= 14 somatokinetic GBCs and n&nbsp;= 15 non-somatokinetic GBCs and of 3 PDX mice, Mann-Whitney test).</div><div id="fspara1165" role="paragraph">(Z) Percentage of transient putative neurogliomal presynaptic boutons compared between somatokinetic and non-somatokinetic S24 GBCs in co-culture with neurons and astrocytes (n&nbsp;= 13 somatokinetic GBCs and n&nbsp;= 7 non-somatokinetic GBCs, unpaired t test).</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">To identify contributing cellular mechanisms, we tested whether TMs of unconnected<sup>TUM/AC</sup> GBCs receive direct synaptic input. Using correlative light and electron microscopy (CLEM) to create 3D reconstructions of GBC processes, we unequivocally identified unconnected<sup>TUM/AC</sup> GBCs in xenograft models and found neurogliomal synapses on these cells (<a id="crosref1095" href="#fig5">Figure&nbsp;5</a>M). Functionality of synaptic inputs was established <i>ex&nbsp;vivo</i> and&nbsp;<i>in&nbsp;vitro</i> via whole-cell patch-clamp recordings from unconnected<sup>TUM/AC</sup> GBCs, identified by dye filling (<a id="crosref1100" href="#fig5">Figure&nbsp;5</a>N). Spontaneous excitatory post-synaptic currents (EPSCs) demonstrated functional synaptic input to unconnected<sup>TUM/AC</sup> GBCs (<a id="crosref1105" href="#fig5">Figures 5</a>O and <a id="crosref1110" href="#figs6">S6</a>L). Additionally, using the same approach, we could show that also connected<sup>TUM/AC</sup> GBCs can receive synaptic input (<a id="crosref1115" href="#figs6">Figure&nbsp;S6</a>M–S6V). Lastly, we studied the dynamics of neurogliomal synapses on unconnected<sup>TUM/AC</sup> GBCs. We found that neurogliomal synaptic boutons are predominately transient on invasive, unconnected<sup>TUM/AC</sup> GBCs (<a id="crosref1120" href="#figs6">Figures&nbsp;S6</a>W–S6Z). Together, these findings demonstrate that synaptic input stimulates unconnected<sup>TUM/AC</sup> GBCs to invade the brain.</div></section><section id="sec-2-7"><h3>Calcium transients in GBCs control TM dynamics and invasion speed</h3><div id="p0150" role="paragraph">To understand whether neuronal activity was also functionally relevant for putative downstream mechanisms such as calcium transients, we established 3D <i>in&nbsp;vivo</i> calcium imaging of GBCs, similar to <span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Bindocci et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bindocci, E. ∙ Savtchouk, I. ∙ Liaudet, N. ...</div><div><strong>Three-dimensional Ca(2+) imaging advances understanding of astrocyte biology</strong></div><div><em>Science.</em> 2017; <strong>356</strong></div><div>eaai8185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai8185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85019912597" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28522470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai8185&amp;pmid=28522470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> (<a id="crosref1130" href="#fig6">Figure&nbsp;6</a>A; <a id="crosref1135" href="#mmc5">Video S3</a>). We found complex calcium signals including microdomains in TMs and somata (<a id="crosref1140" href="#figs7">Figures&nbsp;S7</a>A–S7F). Neuronal activity drove these complex calcium dynamics in single GBCs. Optogenetic activation of neurons <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> glutamate puffing in co-cultures increased calcium event area and frequency in GBCs (<a id="crosref1145" href="#fig6">Figures 6</a>B–6E and <a id="crosref1150" href="#figs7">S7</a>G–S7J). To identify functional consequences of these single-cell calcium dynamics increased by neuronal activity, calcium events were ablated in GBCs using the calcium chelator BAPTA-AM (1,2-bis(<i>o</i>-aminophenoxy)ethane-<i>N</i>,<i>N</i>,<i>N′</i>,<i>N′</i>-tetraacetic acid). TM generation as well as GBC invasion were significantly reduced (<a id="crosref1155" href="#fig6">Figures&nbsp;6</a>F–6I and <a id="crosref1160" href="#figs7">S7</a>K–S7O). To identify downstream mechanisms, a transcription factor analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref102-2" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-2" aria-controls="sref102-2">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref102-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Yu, K. ∙ Hu, Y.Q. ∙ Wu, F. ...</div><div><strong>Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies</strong></div><div><em>Natl. Sci. Rev.</em> 2020; <strong>7</strong>:1306-1318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwaa099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85092585099" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34692159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwaa099&amp;pmid=34692159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) revealed cAMP response element-binding protein (CREB) as a potential neuronal-activity-dependent and calcium-regulated target (<a id="crosref1170" href="#fig6">Figures&nbsp;6</a>J, <a id="crosref1175" href="#figs7">S7</a>P, and S7Q). CREB was targeted with the inhibitor 666-15 (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Middei et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Middei, S. ∙ Houeland, G. ∙ Cavallucci, V. ...</div><div><strong>CREB is necessary for synaptic maintenance and learning-induced changes of the ampa receptor GluA1 subunit</strong></div><div><em>Hippocampus.</em> 2013; <strong>23</strong>:488-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/hipo.22108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84878192270" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23504989/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fhipo.22108&amp;pmid=23504989" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Zirpel et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Zirpel, L. ∙ Janowiak, M.A. ∙ Veltri, C.A. ...</div><div><strong>AMPA receptor-mediated, calcium-dependent CREB phosphorylation in a subpopulation of auditory neurons surviving activity deprivation</strong></div><div><em>J.&nbsp;Neurosci.</em> 2000; <strong>20</strong>:6267-6275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.20-16-06267.2000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10934277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.20-16-06267.2000&amp;pmid=10934277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), resulting in reduced TM generation and GBC invasion (<a id="crosref1190" href="#fig6">Figures&nbsp;6</a>K–6M). Thus, these analyses together with the discovery of neurogliomal synapses on unconnected<sup>TUM/AC</sup> tumor cells provide a crucial link between neuronal activity, calcium signaling, and TM-dependent tumor invasion, connecting three key biological factors of brain tumor biology.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/b623e600-0975-428f-a8a7-485405a7ec48/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.054/asset/9a73eef6-9005-448a-a6a2-abc169dfc4a9/main.assets/gr6.jpg" height="511" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/b623e600-0975-428f-a8a7-485405a7ec48/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Distinct calcium transient patterns mediate GBC invasion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0480" role="paragraph">(A) 3D rendering of <i>in&nbsp;vivo</i> 3D calcium imaging of a GBC.</div><div id="fspara0485" role="paragraph">(B) Frequency of calcium events after intravital neuronal channelrhodopsin stimulation compared with baseline.</div><div id="fspara0490" role="paragraph">(C) As in (B), but area of calcium events (n&nbsp;= 157 GBCs in 6 mice [2 with neuronal ChR2(H134R)-mCherry and 4 with neuronal ChrimsonR-mScarlet] in B and C, two-tailed Wilcoxon test).</div><div id="fspara0495" role="paragraph">(D) Mean frequency of calcium events in GBCs, <i>in&nbsp;vitro</i> neuron/astrocyte co-cultures after puffing of glutamate versus aCSF only (n&nbsp;= 17 S24 GBCs, two-tailed Wilcoxon test).</div><div id="fspara0500" role="paragraph">(E) As in (D), but mean area of calcium events (n&nbsp;= 17 S24 GBCs, two-tailed Wilcoxon test).</div><div id="fspara0505" role="paragraph">(F–I) S24 GBCs in neuron/astrocyte co-cultures with neuronal stimulation via gabazine (Stim, left) or gabazine and preincubated with BAPTA-AM (Stim&nbsp;+ BAPTA-AM, right). (F) Confocal time-lapse imaging series of representative GBCs. Post-processed with denoise.ai. (G) GBC invasion speed, (H) TM branching events per GBC per hour, and (I)&nbsp;small process turnover per GBC per hour (n&nbsp;=170 in control and n&nbsp;= 335 S24 GBCs under BAPTA-AM preincubated conditions in n&nbsp;= 4 independent experiments, Mann-Whitney test).</div><div id="fspara0510" role="paragraph">(J) Marker regulons of neuronal rim cells compared with other cells in the Yu dataset.</div><div id="fspara0515" role="paragraph">Transcription factors previously described to be responsive upon neuronal activity and mediators of neuronal morphology highlighted.</div><div id="fspara0520" role="paragraph">(K–M) GBC invasion speed (K), TM branching events per GBC per hour (L)&nbsp;and small process turnover per GBC per hour (M)&nbsp;under 666-15 (CREB inhibitor) preincubation and additional neuronal stimulation with gabazine compared with stimulated control in co-cultures (n&nbsp;= 61 cells in control and n&nbsp;= 120 S24 GBCs under 666-15 preincubation in n&nbsp;= 3 independent experiments, Mann-Whitney test).</div><div id="fspara0525" role="paragraph">(B, C, G–I, K, L, and M) Data represented as mean ± SEM.</div><div id="fspara0530" role="paragraph">See also <a id="crosref2190" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref2195" href="#mmc5">Video S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/20613082-f643-4299-9726-569c7c3049a5/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2022.06.054/asset/bc6b9e53-87ea-47a1-9af0-52cba9783c87/main.assets/figs7.jpg" height="940" width="701" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/20613082-f643-4299-9726-569c7c3049a5/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Classification of calcium events in GBC <i>in&nbsp;vivo</i> with 3D calcium imaging, related to <a id="crosref2260" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara1175" role="paragraph">(A) Example of a calcium event restricted to a TM as seen with <i>in&nbsp;vivo</i> 3D calcium imaging.</div><div id="fspara1180" role="paragraph">(B) Example of a calcium event restricted to a GBC soma as seen with <i>in&nbsp;vivo</i> 3D calcium imaging.</div><div id="fspara1185" role="paragraph">(C) Example of a calcium event restricted to TM and GBC soma as seen with <i>in&nbsp;vivo</i> 3D calcium imaging.</div><div id="fspara1190" role="paragraph">(D) Example of a calcium event encompassing the whole cell as seen with <i>in&nbsp;vivo</i> 3D calcium imaging.</div><div id="fspara1195" role="paragraph">(E) Example of a calcium event in the GBC network termed multicellular events (CME) as seen with <i>in&nbsp;vivo</i> 3D calcium imaging.</div><div id="fspara1200" role="paragraph">(F) Distribution of classes of calcium events in GBCs <i>in&nbsp;vivo</i> shown in <a id="crosref2265" href="#figs6">Figure&nbsp;S6</a> with 3D calcium imaging (n&nbsp;= 272 GBCs from 4 S24 PDX mice).</div><div id="fspara1205" role="paragraph">(G) Mean event area of calcium transients in BG5 GBCs <i>in&nbsp;vitro</i> during glutamate puffing and aCSF puffing (n&nbsp;= 45 pairs, two-tailed Wilcoxon test).</div><div id="fspara1210" role="paragraph">(H) Mean frequency of calcium transients in BG5 GBCs <i>in&nbsp;vitro</i> during glutamate puffing and aCSF puffing (n&nbsp;= 45 pairs, two-tailed Wilcoxon test).</div><div id="fspara1215" role="paragraph">(I and J) Mean event area and mean event frequency in S24 GBCs, baseline glutamate puffing compared with BAPTA-AM wash-in (n&nbsp;= 27 cells).</div><div id="fspara1220" role="paragraph">(K) Confocal image of a neuron in co-culture filled with Alexa 488. Patch pipette is indicated by dashed lines (left). Representative recordings of neurons in co-culture before and after 4&nbsp;μM gabazine wash-in (right).</div><div id="fspara1225" role="paragraph">(L) Quantification of AP frequency before and after wash-in of gabazine (two-sided Wilcoxon matched-pairs test, n&nbsp;= 6 cells).</div><div id="fspara1230" role="paragraph">(M–O) Invasion speed (M), TM branching events (N), and small process dynamics (O)&nbsp;per GBC per hour under gabazine treated conditions compared with control conditions with DMSO (n&nbsp;= 147 control and n&nbsp;= 163 gabazine treated conditions in n&nbsp;= 4 independent experiments, Mann-Whitney test).</div><div id="fspara1235" role="paragraph">(P) Enriched GO terms for the genes that are regulated by the CREB1 regulon. pySCENIC analyses are based on the Yu dataset (n&nbsp;= 30). Terms are sorted and colored by enrichment.</div><div id="fspara1240" role="paragraph">(Q) Confocal images of neuronal and astrocytic co-cultures with S24 GBCs (green) stained against phosphorylated CREB (pCREB) under control conditions, neuronal stimulation with gabazine and incubation with CNQX and TTX (left). Quantification of normalized mean gray value under different conditions (right) (n&nbsp;= 159 GBCs under control, n&nbsp;= 199 GBCs under gabazine, and n&nbsp;= 157 GBCs under CNQX/TTX condition, Kruskal-Wallis test).</div><div id="fspara1245" role="paragraph">Data in (L)–(N) and (Q)&nbsp;are represented as mean ± SEM.</div></div></div></div></figcaption></figure></div><div id="p0155" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIwNmU5Mjk4NTU0NDM5YTBiMjk4YTI1MjcwZTAxOTQyNiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.QcC6ppHjEjLyh-7FfHYCVH-D-iDaLR5yOlfidQHkh9NexSCKE5YctjDgdxvkI4ksPJPqUUX5KhuGQt1INJA5dxq9UvETcrbOlmMTJrv7VeUDJWRlXr7n7XW2Z51xS52cSb4K0U5nK-hSZs6iiKLnVg44CZZpO-1fqTu-7Njr9TflFs61KxQlZZoXO37SSee-tbIFL9HQKB35IH53sNOUCKp3lH2whQPGlj75vdkb-bCZd_WHc2B0-mSpwaFWvlUcY7W6NVXTBCDpAvcMZreLMlVfx8ENJHUJM-YvAGpkR38K5A0gIQgLRU0pdtgVsljBlF1vrCVdzb2HbT7Ectf_xg</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIwNmU5Mjk4NTU0NDM5YTBiMjk4YTI1MjcwZTAxOTQyNiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.QcC6ppHjEjLyh-7FfHYCVH-D-iDaLR5yOlfidQHkh9NexSCKE5YctjDgdxvkI4ksPJPqUUX5KhuGQt1INJA5dxq9UvETcrbOlmMTJrv7VeUDJWRlXr7n7XW2Z51xS52cSb4K0U5nK-hSZs6iiKLnVg44CZZpO-1fqTu-7Njr9TflFs61KxQlZZoXO37SSee-tbIFL9HQKB35IH53sNOUCKp3lH2whQPGlj75vdkb-bCZd_WHc2B0-mSpwaFWvlUcY7W6NVXTBCDpAvcMZreLMlVfx8ENJHUJM-YvAGpkR38K5A0gIQgLRU0pdtgVsljBlF1vrCVdzb2HbT7Ectf_xg" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIwNmU5Mjk4NTU0NDM5YTBiMjk4YTI1MjcwZTAxOTQyNiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NjMzfQ.QcC6ppHjEjLyh-7FfHYCVH-D-iDaLR5yOlfidQHkh9NexSCKE5YctjDgdxvkI4ksPJPqUUX5KhuGQt1INJA5dxq9UvETcrbOlmMTJrv7VeUDJWRlXr7n7XW2Z51xS52cSb4K0U5nK-hSZs6iiKLnVg44CZZpO-1fqTu-7Njr9TflFs61KxQlZZoXO37SSee-tbIFL9HQKB35IH53sNOUCKp3lH2whQPGlj75vdkb-bCZd_WHc2B0-mSpwaFWvlUcY7W6NVXTBCDpAvcMZreLMlVfx8ENJHUJM-YvAGpkR38K5A0gIQgLRU0pdtgVsljBlF1vrCVdzb2HbT7Ectf_xg" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.06.054/attachment/b87adc39-3d7b-4bf2-8b4c-fa3b01a73ec7/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (44.40 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S3. <i>In&nbsp;vivo</i> 3D calcium imaging of GBC reveals complex calcium event classes, related to Figure&nbsp;6</div><div id="espara0020" role="paragraph">Five distinct classes of calcium dynamics are revealed with 3D calcium imaging <i>in&nbsp;vivo</i>. These calcium transient classes consist of TM calcium transients, soma calcium transients, calcium transients that involve both parts of TM and GBC soma, whole-cell calcium transients and multicellular calcium events where at least parts of multiple GBCs show co-active calcium transients and are connected with each other via direct contacts.</div></div></div></div></section><section id="sec-2-8"><h3>Neurogliomal synapses containing AMPA-type glutamate receptors drive TM generation</h3><div id="p0160" role="paragraph">To understand how neuronal synaptic input to GBCs affects TM dynamics, we investigated the spatial distribution of α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptor-type glutamate receptors (AMPARs) (<span class="dropBlock reference-citations"><a id="body-ref-sref88-3" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-3" aria-controls="sref88-3">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-3" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-3" aria-controls="sref90-3">Venkatesh et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref90-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Venkatesh, H.S. ∙ Morishita, W. ∙ Geraghty, A.C. ...</div><div><strong>Electrical and synaptic integration of glioma into neural circuits</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:539-545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1563-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072686777" target="_blank" aria-label="Open Scopus in new tab">Scopus (407)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1563-y&amp;pmid=31534222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An AMPAR gene expression score was applied to bulk RNA-seq data (<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Puchalski et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Puchalski, R.B. ∙ Shah, N. ∙ Miller, J. ...</div><div><strong>An anatomic transcriptional atlas of human glioblastoma</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:660-663</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2666" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85046671790" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29748285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2666&amp;pmid=29748285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as well as scRNA-seq data of spatially defined GB regions (<span class="dropBlock reference-citations"><a id="body-ref-sref102-3" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-3" aria-controls="sref102-3">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref102-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Yu, K. ∙ Hu, Y.Q. ∙ Wu, F. ...</div><div><strong>Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies</strong></div><div><em>Natl. Sci. Rev.</em> 2020; <strong>7</strong>:1306-1318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwaa099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85092585099" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34692159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwaa099&amp;pmid=34692159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Both analyses showed that AMPAR gene expression is enriched at the tumor rim compared with the core (<a id="crosref1215" href="#fig7">Figures&nbsp;7</a>A–7C and <a id="crosref1220" href="#figs8">S8</a>A), supporting a key role of neurogliomal synapses for tumor invasion. In line, AMPAR gene expression scores are significantly enriched in unconnected<sup>TUM/AC</sup> GBCs compared with connected<sup>TUM/AC</sup> GBCs (<a id="crosref1225" href="#fig7">Figures&nbsp;7</a>D and <a id="crosref1230" href="#figs8">S8</a>B; <span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">Hai et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hai, L. ∙ Hoffmann, D.C. ∙ Mandelbaum, H. ...</div><div><strong>A connectivity signature for glioblastoma</strong></div><div>Preprint at <em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.11.07.465791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85121392461" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.11.07.465791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Correlating these analyses back to the gene- and pathway-based cell states discussed before unveiled enriched AMPAR gene expression in neuronal-like, non-MES-like, and neurodevelopmental cell states (<a id="crosref1240" href="#fig7">Figures&nbsp;7</a>E–7I and <a id="crosref1245" href="#figs8">S8</a>C–S8G). Lastly, we found two cell clusters in the rim of human GB to be enriched for the AMPAR gene expression score, synaptic and cell projection GO terms and associated with a high invasivity gene expression score (<a id="crosref1250" href="#figs8">Figures&nbsp;S8</a>H–S8J). In summary, AMPAR expression in unconnected<sup>TUM/AC</sup> GBCs of the neuronal and non-MES-like cell states is associated with activity-dependent TM dynamics and invasion.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/7f0c3b05-75b9-473b-aabe-d3849f5e8745/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2022.06.054/asset/eaa4aade-2aeb-4767-ab13-699c4e530b63/main.assets/gr7.jpg" height="941" width="715" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7f0c3b05-75b9-473b-aabe-d3849f5e8745/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Pharmacological and genetic perturbation of AMPA receptors inhibits TM-related mechanisms of invasivity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0540" role="paragraph">(A) AMPAR gene expression score (see <a id="crosref2200" href="#sec-4">STAR Methods</a>) in different tumor regions (Ivy GB Atlas, n&nbsp;= 270 human GB samples).</div><div id="fspara0545" role="paragraph">(B) AMPAR gene expression score in GB core versus rim regions (Yu dataset).</div><div id="fspara0550" role="paragraph">(C) UMAP plots with invasivity expression score (left) and AMPAR expression score (right).</div><div id="fspara0555" role="paragraph">(D) Violin plot displaying AMPAR gene expression scores between connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs in the PDX dataset (n&nbsp;= 35,822 cells).</div><div id="fspara0560" role="paragraph">(E) Violin plot displaying the AMPAR gene expression score in Yu dataset sorted by the pathway-based cell states.</div><div id="fspara0565" role="paragraph">(F) AMPAR gene expression score in the neuronal cell state compared with all non-neuronal cell states.</div><div id="fspara0570" role="paragraph">(G) Violin plot displaying AMPAR expression in the neuronal cell state of the GB core as compared with the GB rim in the Yu dataset.</div><div id="fspara0575" role="paragraph">(H) AMPAR gene expression score in the MES-like cell state compared with all non-MES-like cell states.</div><div id="fspara0580" role="paragraph">(I) AMPAR gene expression score in injury response transcriptional signature-enriched GBCs compared with neurodevelopmental transcriptional signature-enriched GBCs.</div><div id="fspara0585" role="paragraph">(J and K) Comparison of EPSC-positive S24 GBCs compared with GBCs not receiving neuronal input with regard to total TM length per cell (J)&nbsp;and branching points (K) (n&nbsp;= 44 GBCs, Mann-Whitney test).</div><div id="fspara0590" role="paragraph">(L) Small processes per cell of EPSC-positive S24 GBCs compared with GBCs not receiving neuronal input (n&nbsp;= 44 GBCs, Mann-Whitney test).</div><div id="fspara0595" role="paragraph">(M and N) Mean frequency (M)&nbsp;and mean event area (N)&nbsp;of calcium transients in BG5 GBCs <i>in&nbsp;vitro</i> during glutamate puffing and effect of cyanquixaline (CNQX) in a subpopulation (n&nbsp;= 70 GBCs, Friedman test).</div><div id="fspara0600" role="paragraph">(O) IV2PM of S24 tdTomato GBCs expressing dominant-negative AMPARs (yellow, arrow heads) and S24 tdTomato GBCs that do not express them (red).</div><div id="fspara0605" role="paragraph">(P and Q) TM length per cell (P) (n&nbsp;= 718 TMs from 6 S24 PDX mice, Mann-Whitney test) and TMs per cell (Q)&nbsp;of control versus dominant-negative S24 GBCs (n&nbsp;= 593 TMs from 6 S24 PDX mice, Mann-Whitney test).</div><div id="fspara0610" role="paragraph">(R) IV2PM of tumor microregions of S24 PDX mice under control and perampanel-treated conditions.</div><div id="fspara0615" role="paragraph">(S and T) TM length per cell (S) (n&nbsp;= 243 TMs from 6 S24 PDX mice, Mann-Whitney test) and number of TMs per cell (T) (n&nbsp;= 142 TMs from 6 S24 PDX mice, Mann-Whitney test) under control and perampanel conditions.</div><div id="fspara0620" role="paragraph">(U) IV2PM of single GBC in control versus perampanel treatment in S24 PDX mice. Arrows at branching points of TMs.</div><div id="fspara0625" role="paragraph">(V) Branching points per GBC under control and perampanel conditions (n&nbsp;= 184 GBCs from 7 S24 PDX mice, Mann-Whitney test).</div><div id="fspara0630" role="paragraph">(B–F) (Yu dataset, n&nbsp;= 2,795 cells). (J–L, P,Q, S, T, and V) Data represented as mean ± SEM.</div><div id="fspara0635" role="paragraph">See also <a id="crosref2205" href="#figs8">Figure&nbsp;S8</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs8" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.054/asset/524c6a37-182f-4b76-9086-bdc537a07506/main.assets/figs8_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs8" src="/cms/10.1016/j.cell.2022.06.054/asset/6eae3291-d537-4fe4-8289-bad021178dc0/main.assets/figs8.jpg" height="941" width="723" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/524c6a37-182f-4b76-9086-bdc537a07506/main.assets/figs8_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs8-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S8</span> <span class="figure__title__text">AMPA receptors inhibition <i>in&nbsp;vivo</i> and in co-cultures reduces TM generation and TM growth, related to <a id="crosref2270" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs8-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara1255" role="paragraph">(A) Heatmap displaying gene expression of AMPAR genes in Ivy Glioblastoma dataset (n&nbsp;= 270 samples) (left panel). Heatmap displaying gene expression of AMPAR genes in Neftel dataset, grouped based on the pathway-based classification (n&nbsp;= 6,576 cells) (middle panel). Heatmap displaying gene expression of AMPAR genes in Neftel dataset, grouped based on gene-expression-based classification (n&nbsp;= 6,576 cells) (right panel).</div><div id="fspara1260" role="paragraph">(B) AMPAR gene expression score in connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs of the S24 PDX dataset (n&nbsp;= 15,629 cells).</div><div id="fspara1265" role="paragraph">(C) tSNE plot of all PDX cells. Cells are colored based on the pathway-based cell state classification (n&nbsp;= 35,822 cells).</div><div id="fspara1270" role="paragraph">(D) Expression of AMPAR gene expression scores on a tSNE plot of PDX cells (n&nbsp;= 35,822 cells).</div><div id="fspara1275" role="paragraph">(E) Violin plot displaying the AMPAR gene expression score in the Neftel dataset sorted by the gene-based cell states (n&nbsp;= 6,576 cells).</div><div id="fspara1280" role="paragraph">(F) Violin plot displaying the AMPAR gene expression score in the Yu dataset sorted by the gene-based cell states (n&nbsp;= 2,795 cells).</div><div id="fspara1285" role="paragraph">(G) Violin plot displaying AMPAR gene expression scores in the PDX dataset sorted by the gene-based cell states (n&nbsp;= 35,822 cells).</div><div id="fspara1290" role="paragraph">(H) Invasivity signature score in clusters of rim cells in the Yu dataset (n&nbsp;= 239 cells).</div><div id="fspara1295" role="paragraph">(I) Expression of AMPAR score signature in clusters of rim cells of Yu dataset (n&nbsp;= 239 cells).</div><div id="fspara1300" role="paragraph">(J) Enriched GO terms of marker genes of rim cluster 1 (n&nbsp;= 10), (as seen in J and K).</div><div id="fspara1305" role="paragraph">In all panels, connectivity is based on SR101 uptake.</div><div id="fspara1310" role="paragraph">(K) Exemplary images of S24 GBCs in co-cultures with astrocytes and neurons of EPSC-positive (top) and EPSC-negative cells (bottom). Arrows show branching points. Dotted line indicates the patch pipette.</div><div id="fspara1315" role="paragraph">(L) Mean event area of BG5 GBCs under glutamate puffing, CNQX wash-in and wash-out (n&nbsp;= 70 cells, Friedman test).</div><div id="fspara1320" role="paragraph">(M) Exemplary IV2PM images of control S24 GBCs and dominant-negative AMPAR construct expression. Arrows show branching points in S24 control GBCs only expressing tdTomato without the dominant-negative AMPAR construct.</div><div id="fspara1325" role="paragraph">(N) Quantification of branching points per GBC of control S24 GBCs and dominant-negative AMPAR construct expressing cells <i>in&nbsp;vivo</i> (n&nbsp;= 211 from 4 S24 PDX mice, Mann-Whitney test).</div><div id="fspara1330" role="paragraph">(O) Representative images of BG5 GBC under control conditions and treatment with perampanel <i>in&nbsp;vivo</i>.</div><div id="fspara1335" role="paragraph">(P–R) (P) TM length (n&nbsp;= 1,426 TMs from 6 BG5 PDX mice, Mann-Whitney test), (Q) TMs per cell (n&nbsp;= 429 TMs from 4 BG5 PDX mice, Mann-Whitney test), and (R) TM length per cell (n&nbsp;= 62 GBCs from 4 BG5 PDX mice, unpaired t test) under control conditions and perampanel treatment. Data in (E)&nbsp;and (G)–(I) are represented as mean ± SEM.</div><div id="fspara1340" role="paragraph">(S) Representative recordings of neurons in co-culture under 4-μM gabazine (left) and under gabazine plus 50-μM perampanel (right).</div><div id="fspara1345" role="paragraph">(T) Quantification of AP frequency under gabazine and under gabazine plus perampanel (two-sided Wilcoxon matched-pairs test, n&nbsp;= 6 cells).</div><div id="fspara1350" role="paragraph">(U–X) Invasion speed per hour (U), TM branching events per hour (V), primary TM genesis events per hour (W), and small process turnover per hour (X)&nbsp;compared between S24tdTomato (S24-WT) and S24-tdTomato-EGFP- dominant-negative GluA2 (S24-DN) under baseline condition (n&nbsp;= 137 GBCs in S24-WT and n&nbsp;= 112 GBCs in S24-DB in n&nbsp;= 3 independent experiments, Mann-Whitney test).</div><div id="fspara1355" role="paragraph">(Y) Invasion speed per hour, TM branching events per hour, primary TM genesis events per hour, and small process turnover per hour in S24-WT after treatment with perampanel (n&nbsp;= 137 GBCs in S24-WT in n&nbsp;= 3 independent experiments, Wilcoxon test).</div><div id="fspara1360" role="paragraph">(Z) Invasion speed per hour, TM branching events per hour, primary TM genesis events per hour and small process turnover per hour in S24-DN after treatment with perampanel (n&nbsp;= 112 GBCs in S24-DN in n&nbsp;= 3 independent experiments, Wilcoxon test). Effects of perampanel were largely abrogated in S24-DN GBCs as compared to S24-WT GBCs.</div><div id="fspara1365" role="paragraph">Data in (C)–(N) are represented as mean ± SEM.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">To link AMPAR-mediated responses to changes in cell morphology, co-cultures of neurons and GBCs were investigated by electrophysiology and confocal microscopy. GBCs exhibiting spontaneous EPSCs when co-cultured with neurons had longer TMs and more branches/filopodia than GBCs without neuronal input, suggesting that synaptic input drove TM formation and growth (<a id="crosref1255" href="#fig7">Figures&nbsp;7</a>J–7L and <a id="crosref1260" href="#figs8">S8</a>K). The frequency and magnitude of GBC calcium events elicited by glutamate puffing was reversibly decreased by CNQX in a GBC subpopulation (<a id="crosref1265" href="#fig7">Figures&nbsp;7</a>M, 7N, and <a id="crosref1270" href="#figs8">S8</a>L). To further support a contribution of AMPARs to TM formation and growth, we expressed dominant-negative AMPAR in GBCs and evaluated the structural consequences by IV2PM (<a id="crosref1275" href="#fig7">Figure&nbsp;7</a>O). The total length and number of TMs and number of branching points were significantly reduced (<a id="crosref1280" href="#fig7">Figures&nbsp;7</a>P, 7Q, <a id="crosref1285" href="#figs8">S8</a>M, and S8N), consistently demonstrating that synaptic input via AMPAR is required for TM formation and growth. As a first step toward clinical translation, mice received the FDA-approved AMPAR inhibiting antiepileptic drug perampanel, and comparable tumor regions of treated and control mice were assessed intravitally (<a id="crosref1290" href="#fig7">Figure&nbsp;7</a>R). TM length and branching points were significantly reduced by perampanel treatment (<a id="crosref1295" href="#fig7">Figures&nbsp;7</a>S–7V and <a id="crosref1300" href="#figs8">S8</a>O–S8Z).</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0170" role="paragraph">Our work demonstrates that a distinct subpopulation of GBCs, transcriptionally residing in neuronal- and neural-progenitor-like cell states and not connected to other tumor cells or astrocytes, but receiving neuronal synaptic input, is driving brain invasion via adaptation of cellular neuronal mechanisms. This mechanism of tumor cell migration allows effective brain colonization and thus contributes to GB’s incurability.</div><div id="p0175" role="paragraph">In this study, we connected recently described gene- and pathway-based classifications (<span class="dropBlock reference-citations"><a id="body-ref-sref22-7" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-7" aria-controls="sref22-7">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52-6" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-6" aria-controls="sref52-6">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with their functional relevance for GBC invasion of the brain. Previous work was focused on tumor-connected GBCs and their networks connected via TMs (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Jung et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Jung, E. ∙ Osswald, M. ∙ Blaes, J. ...</div><div><strong>Tweety-homolog 1 drives brain colonization of gliomas</strong></div><div><em>J.&nbsp;Neurosci.</em> 2017; <strong>37</strong>:6837-6850</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3532-16.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85025161941" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28607172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3532-16.2017&amp;pmid=28607172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55-4" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-4" aria-controls="sref55-4">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98-3" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-3" aria-controls="sref98-3">Weil et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref98-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Weil, S. ∙ Osswald, M. ∙ Solecki, G. ...</div><div><strong>Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas</strong></div><div><em>Neuro. Oncol.</em> 2017; <strong>19</strong>:1316-1326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/nox070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85029925185" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28419303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnox070&amp;pmid=28419303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-1" aria-controls="sref101-1">Xie et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Xie, R. ∙ Kessler, T. ∙ Grosch, J. ...</div><div><strong>Tumor cell network integration in glioma represents a stemness feature</strong></div><div><em>Neuro. Oncol.</em> 2021; <strong>23</strong>:757-769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noaa275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100835529" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33320195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoaa275&amp;pmid=33320195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with therapeutic resistance governed by this homotypic tumor cell network (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Jung et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Jung, E. ∙ Osswald, M. ∙ Ratliff, M. ...</div><div><strong>Tumor cell plasticity, heterogeneity, and resistance in crucial microenvironmental niches in glioma</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:1014</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-21117-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100832371" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33579922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-21117-3&amp;pmid=33579922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55-5" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-5" aria-controls="sref55-5">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98-4" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-4" aria-controls="sref98-4">Weil et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref98-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Weil, S. ∙ Osswald, M. ∙ Solecki, G. ...</div><div><strong>Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas</strong></div><div><em>Neuro. Oncol.</em> 2017; <strong>19</strong>:1316-1326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/nox070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85029925185" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28419303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnox070&amp;pmid=28419303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we extended this concept of the “GB connectome” to astrocytes <i>in&nbsp;vivo</i>, with direct gap junction-mediated heterotypic contacts between both cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Sin et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Sin, W.C. ∙ Aftab, Q. ∙ Bechberger, J.F. ...</div><div><strong>Astrocytes promote glioma invasion via the Gap junction protein connexin43</strong></div><div><em>Oncogene.</em> 2016; <strong>35</strong>:1504-1516</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/onc.2015.210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84957900747" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26165844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fonc.2015.210&amp;pmid=26165844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Zhang et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Zhang, W. ∙ Couldwell, W.T. ∙ Simard, M.F. ...</div><div><strong>Direct gap junction communication between malignant glioma cells and astrocytes</strong></div><div><em>Cancer Res.</em> 1999; <strong>59</strong>:1994-2003</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10213512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=10213512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0180" role="paragraph">In contrast to previous work suggesting collective network invasion of GBCs (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Gritsenko et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Gritsenko, P.G. ∙ Atlasy, N. ∙ Dieteren, C.E.J. ...</div><div><strong>p120-catenin-dependent collective brain infiltration by glioma cell networks</strong></div><div><em>Nat. Cell Biol.</em> 2020; <strong>22</strong>:97-107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-019-0443-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85077546068" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31907411/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-019-0443-x&amp;pmid=31907411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and our own hypotheses (<span class="dropBlock reference-citations"><a id="body-ref-sref55-6" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-6" aria-controls="sref55-6">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found that unconnected<sup>TUM/AC</sup> GBCs receiving neurogliomal synaptic input drive brain invasion and colonization. This could be explained by <i>in&nbsp;vitro</i> models that were previously used (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Gritsenko et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Gritsenko, P. ∙ Leenders, W. ∙ Friedl, P.</div><div><strong>Recapitulating in&nbsp;vivo-like plasticity of glioma cell invasion along blood vessels and in astrocyte-rich stroma</strong></div><div><em>Histochem. Cell Biol.</em> 2017; <strong>148</strong>:395-406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00418-017-1604-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85027881035" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28825130/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00418-017-1604-2&amp;pmid=28825130" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, a sufficient temporal and spatial resolution was obligatory to uncover the role of unconnected<sup>TUM/AC</sup> GBCs, as these cell states transition into connected<sup>TUM/AC</sup> GBCs. Thus, unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> GBCs represent opposite ends of a cell state gradient.</div><div id="p0185" role="paragraph">As connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs could be functionally separated and subsequently characterized with scRNA-seq, we could overlay their tumor biological differences with the molecular cell states for gene- and pathway-based classification. The neuronal, OPC/NPC and neurodevelopmental-like cell states were enriched in the unconnected<sup>TUM/AC</sup> GBCs, whereas the connected<sup>TUM/AC</sup> GBCs were enriched for AC/MES-like and non-NEU cell states as well as those associated with an injury response. In contrast to previous work suggesting a role for epithelial-to-mesenchymal transition in invasion of GB (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Iwadate, 2016</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Iwadate, Y.</div><div><strong>Epithelial-mesenchymal transition in glioblastoma progression</strong></div><div><em>Oncol. Lett.</em> 2016; <strong>11</strong>:1615-1620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3892/ol.2016.4113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84958047483" target="_blank" aria-label="Open Scopus in new tab">Scopus (175)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26998052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3892%2Fol.2016.4113&amp;pmid=26998052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Siebzehnrubl et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Siebzehnrubl, F.A. ∙ Silver, D.J. ∙ Tugertimur, B. ...</div><div><strong>The ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance</strong></div><div><em>EMBO Mol. Med.</em> 2013; <strong>5</strong>:1196-1212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/emmm.201302827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84881540768" target="_blank" aria-label="Open Scopus in new tab">Scopus (275)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23818228/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Femmm.201302827&amp;pmid=23818228" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found that MES-like cell states were enriched in the core of GB samples where tumor networks prevail.</div><div id="p0190" role="paragraph">This is in line with a recent publication demonstrating that upregulation of neuronal signaling programs is associated with brain tumor invasiveness at recurrence in isocitrate dehydrogenase (IDH)-wild-type gliomas, including GBs (<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Varn et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Varn, F.S. ∙ Johnson, K.C. ∙ Martinek, J. ...</div><div><strong>Glioma progression is shaped by genetic evolution and microenvironment interactions</strong></div><div><em>Cell.</em> 2022; <strong>185</strong>:2184-2199.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2022.04.038&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2022.04.038&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85131903367" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35649412/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2022.04.038&amp;pmid=35649412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0195" role="paragraph">We also demonstrate a transition of cell states in GB: with expansion of the tumor mass and as tumor cellularity increases, unconnected<sup>TUM/AC</sup> GBCs were progressively integrating into a tumor cell/astrocyte-connected network, suggesting an ordered evolution of cell states during the progression of GB. This context-dependent transition of GBC states in the infiltration zone adds another layer of complexity to the recent debate of brain tumor cell state transitions (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Chaligne et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chaligne, R. ∙ Gaiti, F. ∙ Silverbush, D. ...</div><div><strong>Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states</strong></div><div><em>Nat. Genet.</em> 2021; <strong>53</strong>:1469-1479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-021-00927-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85116387369" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34594037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-021-00927-7&amp;pmid=34594037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Fine, 2019</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Fine, H.A.</div><div><strong>Malignant gliomas: simplifying the complexity</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1650-1652</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075914397" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31792122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-1081&amp;pmid=31792122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-3" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-3" aria-controls="sref94-3">Wang et&nbsp;al., 2019a</a><div class="dropBlock__holder" data-db-target-of="sref94-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Wang, L. ∙ Babikir, H. ∙ Müller, S. ...</div><div><strong>The phenotypes of proliferating glioblastoma cells reside on a single axis of variation</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1708-1719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-0329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075958233" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31554641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-0329&amp;pmid=31554641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, the framework developed here for GB points toward fruitful opportunities for other tumor entities where molecular intratumoral heterogeneity is increasingly well understood, but without clear links to tumor biology yet.</div><div id="p0200" role="paragraph">Analogous mechanisms to neuronal migration during neurodevelopment demonstrate a second layer of neuronal features. The three different mechanisms of GB single-cell invasion <i>in&nbsp;vivo</i> reported here enhance current ideas of GB invasion obtained <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Bhaduri et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bhaduri, A. ∙ Di Lullo, E. ∙ Jung, D.N. ...</div><div><strong>Outer radial glia-like cancer stem cells contribute to heterogeneity of glioblastoma</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>26</strong>:48-63.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.stem.2019.11.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.stem.2019.11.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85076910021" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31901251/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.11.015&amp;pmid=31901251" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This extends the concept of how basic neurodevelopmental pathways are hijacked to further brain tumor progression.</div><div id="p0205" role="paragraph">The last layer of neuronal features was the observation that neuron-GBC synaptic activity can generate new TMs and also makes their brain scanning behavior more efficient, ultimately driving GB invasion and progression. In addition to previous work suggesting synapses mainly on tumor-connected GBCs (<span class="dropBlock reference-citations"><a id="body-ref-sref88-4" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-4" aria-controls="sref88-4">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-4" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-4" aria-controls="sref90-4">Venkatesh et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref90-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Venkatesh, H.S. ∙ Morishita, W. ∙ Geraghty, A.C. ...</div><div><strong>Electrical and synaptic integration of glioma into neural circuits</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:539-545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1563-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072686777" target="_blank" aria-label="Open Scopus in new tab">Scopus (407)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1563-y&amp;pmid=31534222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we now found that glutamatergic synaptic contacts are present on unconnected<sup>TUM/AC</sup> GBCs as well; here, neurogliomal synapses drive efficient migration and brain colonization of GBCs. Together this advances our understanding of how heterotypic multicellular networks in brain tumors are differentially regulating tumor biological functions. Finally, targeting specific subcellular mechanisms of TM dynamics and TM formation decreased overall invasion in GB by mechanisms downstream of AMPAR signaling. Synaptic input stimulating the migration of unconnected<sup>TUM/AC</sup> GBCs is mirrored by synaptic input driving neuronal progenitor cell migration during brain development (<span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Ohtaka-Maruyama et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Ohtaka-Maruyama, C. ∙ Okamoto, M. ∙ Endo, K. ...</div><div><strong>Synaptic transmission from subplate neurons controls radial migration of neocortical neurons</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:313-317</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar2866" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85045556191" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29674592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar2866&amp;pmid=29674592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Wefers et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Wefers, A.K. ∙ Haberlandt, C. ∙ Tekin, N.B. ...</div><div><strong>Synaptic input as a directional cue for migrating interneuron precursors</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:4125-4136</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.154096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85034447129" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29061636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.154096&amp;pmid=29061636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0210" role="paragraph">In conclusion, we present a model of how three layers of neuronal features drive GBC invasion, reaching from molecular cell states over cellular mechanisms that resemble neuronal progenitor-like migration to functional effects of synaptic communication of neurons with migrating tumor cells. This explains how dissemination and cellular heterogeneity of brain tumors, two crucial hallmarks of the disease, are closely interlinked and identifies specific vulnerabilities that can be clinically tested, such as the dependency of migrating tumor cells on AMPAR signaling.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0215" role="paragraph">In this study, the nature of tumor progression and evolution of cell states was analyzed using multiple PDX models. However, the effects of therapeutic pressures including radio- and chemotherapy and effects of surgical resection were not analyzed. Thus, to completely characterize tumor heterogeneity, its biological functions, its interrelations with invasion, proliferation, and therapeutic resistance will require further studies. As the IV2PM cell state analysis showed that unconnected<sup>TUM/AC</sup> GBCs evolve over time into connected<sup>TUM/AC</sup> GBCs, it will be important to further understand whether and how many intermediary cell states might exist, and how they might interchange during brain tumor progression. Furthermore, it is yet unclear which factors drive this cell state evolution, including the role of neurogliomal synapses. Although neurogliomal synapses exist on both unconnected<sup>TUM/AC</sup> and connected<sup>TUM/AC</sup> GBCs, the role for the latter is less well understood; it is reasonable to assume that synaptic input activates intercellular Ca<sup>2+</sup> waves and promotes growth and potentially resistance of the main tumor mass. Interestingly, on a broad gene expression level, unconnected<sup>TUM/AC</sup> GBCs are enriched for synaptic gene signatures such as the AMPAR gene expression score. However, this is likely due to the large heterogeneity of connected<sup>TUM/AC</sup> GBCs. Therefore, the characterization of GBCs receiving neurogliomal synapses with respect to their connectivity to other GBCs and astrocytes will be an important line of future investigations. Furthermore, the timescale on which synaptic input effects tumor biology and cell state transitions remains unclear. For this purpose, further technological innovation that can directly link connectivity, molecular, and functional analyses on a single-cell level is needed. It will be interesting to learn whether additional cell types of the brain tumor microenvironment can interconnect with GBCs and heterotypic tumor cell networks. Lastly, the clinical translation of the basic research findings reported here will require further studies in GB patient cohorts and further technological development to monitor the diffuse brain tumor infiltration zone with clinical imaging.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0220" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Rabbit polyclonal anti-CD44</td><td>Abcam</td><td>Cat#ab157107; RRID:AB_2847859</td></tr><tr><td>Chicken polyclonal anti-GFP</td><td>Abcam</td><td>Cat#ab13970<br>RRID:AB_300798</td></tr><tr><td>Rabbit polyclonal anti-phospho-CREB</td><td>Millipore</td><td>Cat#06-519:<br>RRID:AB_310153</td></tr><tr><td>Mouse monoclonal anti-Nestin</td><td>Abcam</td><td>Cat#ab22035<br>RRID:AB_446723</td></tr><tr><td>Biotinylated anti-mouse antibody</td><td>Abcam</td><td>Cat#ab6788<br>RRID:AB_954885</td></tr><tr><td>Chicken Alexa488</td><td>Invitrogen</td><td>1458638<br>and 2304258<br>RRID:AB_2534096</td></tr><tr><td>Rabbit Alexa647</td><td>Invitrogen</td><td>1981173 and 2299231<br>RRID:AB_2535813</td></tr><tr><td>Guinea pig Alexa647</td><td>Invitrogen</td><td>Cat#A-21450<br>RRID:AB_141882</td></tr><tr><td>Chicken Alexa647</td><td>Invitrogen</td><td>Cat#A-21449<br>RRID:AB_2535866</td></tr><tr><td>Chicken polyclonal anti-S100B</td><td>Synaptic Systems</td><td>Cat#287006, RRID:AB_2713986</td></tr><tr><td>Rabbit polyclonal anti-EAAT2</td><td>Synaptic Systems</td><td>Cat#250203, RRID:AB_11042312</td></tr><tr><td>Guinea pig polyclonal anti-NeuN</td><td>Synaptic Systems</td><td>Cat#266004,<br>RRID:AB_2619988</td></tr><tr class="rowsep"><td>Mouse monoclonal anti-beta-III-tubulin</td><td>Abcam</td><td>Cat#ab7751, RRID:AB_306045</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr><td>pAAV-CamKIIa-ChrimsonR-mScarlet-KV2.1 (AAV 9)</td><td>Addgene</td><td>124651-AAV9, RRID:Addgene_124651</td></tr><tr><td>pAAV-CaMKIIahChR2(H134R)-mCherry (AAV2)</td><td>UNC Vector Core</td><td>N/A</td></tr><tr><td>pAAV-CaMKIIa-mCherry (AAV2)</td><td>Addgene</td><td>114469- AAV2, RRID:Addgene_114469</td></tr><tr class="rowsep"><td>pAAV-CAG-Synaptophysin-mCherry (AAV1/2)</td><td>analogous to <span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Wimmer et&nbsp;al. (2004)</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Wimmer, V.C. ∙ Nevian, T. ∙ Kuner, T.</div><div><strong>Targeted in&nbsp;vivo expression of proteins in the calyx of Held</strong></div><div><em>Pflugers Arch.</em> 2004; <strong>449</strong>:319-333</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15452710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=15452710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr class="rowsep"><td>Patient-derived xenografts (PDX)</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>BAPTA-AM</td><td>Sigma-Aldrich</td><td>A1076;<br>CAS: 126150-97-8</td></tr><tr><td>666-15</td><td>TOCRIS</td><td>71566396;<br>CAS: 1433286-70-4</td></tr><tr><td>Gabazine</td><td>Abcam</td><td>Ab120042;<br>CAS: 104104-50-9</td></tr><tr><td>CNQX</td><td>Abcam</td><td>Ab12004;<br>CAS: 479347-85-8</td></tr><tr><td>Tetrodotoxin citrate</td><td>HelloBio</td><td>HB1035;<br>CAS: 18660-81-6</td></tr><tr><td>Perampanel</td><td>Eisai</td><td>N/A</td></tr><tr><td>Perampanel</td><td>BioCrick</td><td>BCC1847;<br>CAS: 380917-97-5</td></tr><tr><td>0.5% Trypsin-EDTA (10x)</td><td>Gibco</td><td>15400-054</td></tr><tr><td>Alexa Fluor 594 hydrazide sodium salt</td><td>Invitrogen</td><td>A10442</td></tr><tr><td>Glutamate</td><td>Sigma-Aldrich</td><td>49449-100G</td></tr><tr><td>NaCl</td><td>Sigma-Aldrich</td><td>S7653</td></tr><tr><td>KCL</td><td>Fluka</td><td>60129</td></tr><tr><td>NaHCO3</td><td>Fluka</td><td>71627</td></tr><tr><td>NaH2PO4</td><td>Fluka</td><td>71496</td></tr><tr><td>L-glutamine (GlutaMAX™-I (100x))</td><td>Gibco</td><td>35050-038</td></tr><tr><td>B-27 Supplement for neuronal co-culture</td><td>Gibco</td><td>17504-044 10ml</td></tr><tr><td>Poly-L-lysine solution</td><td>Sigma-Aldrich</td><td>P4832-50ml</td></tr><tr><td>Rhod-2,AM</td><td>Invitrogen/ ThermoFisher Scientific</td><td>R1245MP</td></tr><tr><td>DMEM/D12 Medium</td><td>Life Technologies</td><td>11330032</td></tr><tr><td>Insulin solution human</td><td>Sigma</td><td>I9278-5ML</td></tr><tr><td>Heparin</td><td>Sigma</td><td>H4784-1G</td></tr><tr><td>B27 Supplement without vitamin A for GBC culture</td><td>Gibco</td><td>12587-010 10ml</td></tr><tr><td>EGF</td><td>Biotechne</td><td>236-EG-200</td></tr><tr><td>FGF Recombinant Protein</td><td>Life Technologies</td><td>PHG0021</td></tr><tr><td>FGF Recombinant Protein 1mg</td><td>Life Technologies</td><td>PHG0023</td></tr><tr><td>Dulbecco's PBS (1x) w/o Ca&amp;Mg, w/o Phenol Red</td><td>Capricorn scientific</td><td>CP21-4014</td></tr><tr><td>Dulbecco's Phosphate Buffered Saline</td><td>Sigma</td><td>RNBJ5140</td></tr><tr><td>Accutase Solution</td><td>Sigma</td><td>SLCH7391</td></tr><tr><td>Polybrene/ Hexadimethrine bromide</td><td>Sigma</td><td>H9268-10G</td></tr><tr><td>FBS</td><td>Life</td><td>10500064</td></tr><tr><td>Triton™ X-100</td><td>Sigma-Aldrich</td><td>T9284-100ML</td></tr><tr><td>DAB+Chromogen</td><td>Dako</td><td>K3468</td></tr><tr><td>Sucrose</td><td>Sigma</td><td>S0389</td></tr><tr><td>Vectastain ABC-kit</td><td>Linaris/Vector</td><td>PK-4000</td></tr><tr><td>Alpha-D-Glucose</td><td>Serva</td><td>070937</td></tr><tr><td>Glucose oxidase</td><td>Serva</td><td>22778.01</td></tr><tr><td>Cacodylic acid-Na-salt-3H2O</td><td>Serva</td><td>15540.1</td></tr><tr><td>Potassium ferricyanide</td><td>Serva/Sigma</td><td>107H3450</td></tr><tr><td>Osmium tetroxide</td><td>Serva</td><td>31253.04</td></tr><tr><td>Glycid Ether 100 for electron microscopy</td><td>Serva</td><td>21045.02</td></tr><tr><td>MNA hardener</td><td>Carl Roth</td><td>8639.2</td></tr><tr><td>DMP-30</td><td>Carl Roth</td><td>8621.1</td></tr><tr><td>DBA Härter</td><td>Carl Roth</td><td>8623.2</td></tr><tr><td>DDSA</td><td>SERVA</td><td>20755.02</td></tr><tr><td>Propylenoxide</td><td>VWR Chemicals</td><td>27165.295</td></tr><tr><td>Uranyl acetate</td><td>Serva</td><td>77870</td></tr><tr><td>SEM pin stub (0,5”/6mm length)</td><td>Agar scientific</td><td>#G301F</td></tr><tr><td>Acheson silver</td><td>Plano</td><td>#3692</td></tr><tr><td>DAPI</td><td>Sigma</td><td>D9542</td></tr><tr><td>PFA 4%</td><td>Roth</td><td>P087.3</td></tr><tr><td>PFA 4.5%</td><td>Roth</td><td>2212.6</td></tr><tr><td>Sulforhodamine 101 (SR101)</td><td>Sigma Aldrich</td><td>S7635</td></tr><tr><td>Sulforhodamine 101 (SR101)</td><td>Invitrogen</td><td>S359</td></tr><tr><td>Tetramethylrhodamine isothiocyanate dextran (TRITC)</td><td>Sigma Aldrich</td><td>52194-1G</td></tr><tr class="rowsep"><td>SlowFade™ Gold Antifade Mountant</td><td>ThermoFischer</td><td>S36936</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Patient-derived xenograft glioblastoma cell lines and patient-derived organoid xenograft models</b></td></tr><tr><td>S24</td><td>This paper</td><td>N/A</td></tr><tr><td>BG5</td><td>This paper</td><td>N/A</td></tr><tr><td>BG7</td><td>This paper</td><td>N/A</td></tr><tr><td>P3XX</td><td>This paper</td><td>N/A</td></tr><tr><td>T269</td><td>This paper</td><td>N/A</td></tr><tr><td>P8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>P3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>P13</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-3" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-3" aria-controls="sref27-3">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>T101</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-4" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-4" aria-controls="sref27-4">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>T16</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-5" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-5" aria-controls="sref27-5">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>T347</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-6" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-6" aria-controls="sref27-6">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>T470</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-7" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-7" aria-controls="sref27-7">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>T192</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-8" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-8" aria-controls="sref27-8">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>T233</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-9" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-9" aria-controls="sref27-9">Golebiewska et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>NMRI-Foxn1 nu/nu</td><td>Charles River and Janvier</td><td>BL210203171</td></tr><tr class="rowsep"><td>WISTAR</td><td>Janvier</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Plasmid: mGFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Dondzillo et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Dondzillo, A. ∙ Sätzler, K. ∙ Horstmann, H. ...</div><div><strong>Targeted three-dimensional immunohistochemistry reveals localization of presynaptic proteins Bassoon and Piccolo in the rat calyx of Held before and after the onset of hearing</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2010; <strong>518</strong>:1008-1029</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.22260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77949344465" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20127803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.22260&amp;pmid=20127803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Plasmid: GCamp7-tdTomato</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Dana et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Dana, H. ∙ Sun, Y. ∙ Mohar, B. ...</div><div><strong>High-performance calcium sensors for imaging activity in neuronal populations and microcompartments</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:649-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0435-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85067851088" target="_blank" aria-label="Open Scopus in new tab">Scopus (421)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0435-6&amp;pmid=31209382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Plasmid: Twitch3A</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Thestrup et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Thestrup, T. ∙ Litzlbauer, J. ∙ Bartholomäus, I. ...</div><div><strong>Optimized ratiometric calcium sensors for functional in&nbsp;vivo imaging of neurons and T lymphocytes</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:175-182</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2773" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84893256698" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24390440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2773&amp;pmid=24390440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Plasmid: GFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55-7" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-7" aria-controls="sref55-7">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Plasmid: tdTomato</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55-8" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-8" aria-controls="sref55-8">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>Plasmid: dominant negative GluA2 tdTomato</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref88-5" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-5" aria-controls="sref88-5">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>ImageJ/Fiji</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Schneider et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (35949)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0010" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>NIS-Elements AR Analysis 5.30.01 64-bit</td><td>Nikon</td><td><a id="intref0015" href="https://www.microscope.healthcare.nikon.com/products/software" target="_blank">https://www.microscope.healthcare.nikon.com/products/software</a></td></tr><tr><td>Ilastik 1.3.3 post3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Berg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (831)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0020" href="https://www.ilastik.org/development.html" target="_blank">https://www.ilastik.org/development.html</a></td></tr><tr><td>Arivis Vision4D 3.5.0</td><td>arivis AG, Munich, Germany</td><td><a id="intref0025" href="https://imaging.arivis.com/en/imaging-science/arivis-vision4d" target="_blank">https://imaging.arivis.com/en/imaging-science/arivis-vision4d</a></td></tr><tr><td>R Studio 1.4</td><td>RStudio Team, 2020</td><td>N/A</td></tr><tr><td>AquA</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Wang et al., 2019b</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Wang, Y. ∙ DelRosso, N.V. ∙ Vaidyanathan, T.V. ...</div><div><strong>Accurate quantification of astrocyte and neurotransmitter fluorescence dynamics for single-cell and population-level physiology</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1936-1944</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0492-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85074279015" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0492-2&amp;pmid=31570865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>PATCHMASTER Igor Pro 6.21</td><td>HEKA</td><td>N/A</td></tr><tr><td>GraphPad Prism Version 9.2.0</td><td>GraphPad</td><td>N/A</td></tr><tr><td>Adobe Illustrator 25.4.1</td><td>Adobe</td><td>N/A</td></tr><tr><td>Adobe Photoshop</td><td>Adobe</td><td>N/A</td></tr><tr><td>ProCreate</td><td>Apple</td><td>N/A</td></tr><tr><td>DaVinciResolve 17</td><td>Blackmagicdesign</td><td><a id="intref0030" href="https://www.blackmagicdesign.com/de/products/davinciresolve/" target="_blank">https://www.blackmagicdesign.com/de/products/davinciresolve/</a></td></tr><tr><td>Autodesk 3ds max</td><td>Autodesk</td><td><a id="intref0035" href="https://www.autodesk.de/products/3ds-max/overview" target="_blank">https://www.autodesk.de/products/3ds-max/overview</a></td></tr><tr><td>Leica Application Suite X</td><td>Leica Microsystems CMS GmbH</td><td><a id="intref0040" href="https://www.leica-microsystems.com/de/produkte/mikroskop-software/p/leica-las-x-ls/" target="_blank">https://www.leica-microsystems.com/de/produkte/mikroskop-software/p/leica-las-x-ls/</a></td></tr><tr><td>Zen Blue 3.5</td><td>Zeiss</td><td><a id="intref0045" href="https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html" target="_blank">https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html</a></td></tr><tr><td>Zen Black 2.3 SP1</td><td>Zeiss</td><td><a id="intref0050" href="https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html" target="_blank">https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html</a></td></tr><tr><td>IMOD 4.11</td><td>University of Colorado, Boulder</td><td><a id="intref0055" href="https://bio3d.colorado.edu/imod/" target="_blank">https://bio3d.colorado.edu/imod/</a></td></tr><tr><td>Image J 3D Viewer</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Schmid et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Schmid, B. ∙ Schindelin, J. ∙ Cardona, A. ...</div><div><strong>A high-level 3D visualization API for Java and ImageJ</strong></div><div><em>BMC Bioinformatics.</em> 2010; <strong>11</strong>:274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-11-274" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77952447610" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20492697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-11-274&amp;pmid=20492697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>TrakEM</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Cardona et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Cardona, A. ∙ Saalfeld, S. ∙ Schindelin, J. ...</div><div><strong>TrakEM2 software for neural circuit reconstruction</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e38011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0038011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862556352" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0038011&amp;pmid=22723842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>Atlas 5.3.0.25</td><td>Zeiss</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Single-cell and bulk RNA-sequencing datasets</b></td></tr><tr><td>Human single-cell RNA-sequencing glioblastoma dataset</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref52-7" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-7" aria-controls="sref52-7">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GSE131928</td></tr><tr><td>Human single-cell RNA-sequencing glioblastoma dataset</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref102-4" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-4" aria-controls="sref102-4">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref102-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Yu, K. ∙ Hu, Y.Q. ∙ Wu, F. ...</div><div><strong>Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies</strong></div><div><em>Natl. Sci. Rev.</em> 2020; <strong>7</strong>:1306-1318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwaa099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85092585099" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34692159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwaa099&amp;pmid=34692159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GSE117891</td></tr><tr><td>Human bulk RNA-sequencing glioblastoma datasets</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Puchalski et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Puchalski, R.B. ∙ Shah, N. ∙ Miller, J. ...</div><div><strong>An anatomic transcriptional atlas of human glioblastoma</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:660-663</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2666" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85046671790" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29748285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2666&amp;pmid=29748285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GSE107558</td></tr><tr class="rowsep"><td>Xenograft single-cell RNA-sequencing gliolastoma datasets</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref31-3" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-3" aria-controls="sref31-3">Hai et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hai, L. ∙ Hoffmann, D.C. ∙ Mandelbaum, H. ...</div><div><strong>A connectivity signature for glioblastoma</strong></div><div>Preprint at <em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.11.07.465791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85121392461" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.11.07.465791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>FACSAria Fusion 2 Bernhard Shoor</td><td>BD</td><td>N/A</td></tr><tr><td>FACSAria Fusion Richard Sweet</td><td>BD</td><td>N/A</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900847-9" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0225" role="paragraph">Further information and requests for resources should be directed to and will be fulfilled by the lead contact, Frank Winkler (<a id="intref0060" href="mailto:frank.winkler@med.uni-heidelberg.de">frank.winkler@med.uni-heidelberg.de</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0230" role="paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><div id="p0235" role="paragraph">Male NMRI nude mice and NSG (NOD scid gamma) mice (age 8-12&nbsp;weeks) were used for all animal studies involving human patient-derived primary GBCs. All animal procedures were performed in accordance with the European Directive on animal experimentation (2010/63/EU) and institutional laboratory animal research guidelines after approval of the Regierungspräsidium Karlsruhe, Germany and the Animal Welfare Structure of the Luxembourg Institute of Health. Efforts were made to minimize animal suffering and to reduce the number of animals used according to the 3R principles. Mice were clinically scored and if they showed marked neurological symptoms or weight loss exceeding 10-20%, experiments were terminated. No maximum tumor size was defined for the invasive brain tumor models. Human tissues were used from patients having given informed consent and after approval of the local regulatory authorities (Ethic Committees at the Mannheim and Heidelberg Medical Faculty of the University Heidelberg, protocols (S-206/2005, S-207/2005, S-306/2019, 2018-614N-MA, 2018-843R-MA), the National Committee for Ethics in Research (CNER) Luxembourg (201201/06) and the regionale komiteer for medisinsk og helsefaglig forskningsetikk at the Helse Bergen (protocol 2013/720/REK vest). Diagnoses were molecularly confirmed with IDH1-R132H and ATRX stainings and the Illumina 850k methylation array (Department of Neuropathology, University of Heidelberg). Human patient samples were pseudonymized manually.</div><section id="sec-4-3-1"><h4>Patient-derived primary GBC and Illumina 850k methylation array characterization</h4><div id="p0240" role="paragraph">Patient-derived tumor cell lines from resected GB were cultivated as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Jung et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Jung, E. ∙ Osswald, M. ∙ Blaes, J. ...</div><div><strong>Tweety-homolog 1 drives brain colonization of gliomas</strong></div><div><em>J.&nbsp;Neurosci.</em> 2017; <strong>37</strong>:6837-6850</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3532-16.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85025161941" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28607172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3532-16.2017&amp;pmid=28607172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55-9" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-9" aria-controls="sref55-9">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-6" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-6" aria-controls="sref88-6">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98-5" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-5" aria-controls="sref98-5">Weil et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref98-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Weil, S. ∙ Osswald, M. ∙ Solecki, G. ...</div><div><strong>Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas</strong></div><div><em>Neuro. Oncol.</em> 2017; <strong>19</strong>:1316-1326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/nox070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85029925185" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28419303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnox070&amp;pmid=28419303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in DMEM/F-12 under serum-free, non-adherent, ‘stem-like’ conditions, which includes B27 supplement (12587-010, Gibco), insulin, heparin, epidermal growth factor, and fibroblast growth factor as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref55-10" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-10" aria-controls="sref55-10">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The molecular classification of glioblastoma xenograft models used in this study can be found in <a id="crosref1575" href="#mmc1">Table&nbsp;S1</a>. The Illumina Infinium Methylation EPIC kit was used to obtain the DNA methylation status at &gt;850,000 CpG sites in all GBC lines, according to the manufacturer’s instructions at the Genomics and Proteomics Core Facility of the German Cancer Research Center in Heidelberg, Germany, as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Capper et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Capper, D. ∙ Jones, D.T.W. ∙ Sill, M. ...</div><div><strong>DNA methylation-based classification of central nervous system tumours</strong></div><div><em>Nature.</em> 2018; <strong>555</strong>:469-474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature26000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85044262739" target="_blank" aria-label="Open Scopus in new tab">Scopus (1269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29539639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature26000&amp;pmid=29539639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The GB stem-cell lines were transduced with lentiviral vectors for membrane-bound GFP with the pLego-T2-mGFP construct based on <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Dondzillo et&nbsp;al. (2010)</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Dondzillo, A. ∙ Sätzler, K. ∙ Horstmann, H. ...</div><div><strong>Targeted three-dimensional immunohistochemistry reveals localization of presynaptic proteins Bassoon and Piccolo in the rat calyx of Held before and after the onset of hearing</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2010; <strong>518</strong>:1008-1029</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.22260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77949344465" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20127803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.22260&amp;pmid=20127803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, for calcium imaging with the pLego-T2-GCaMP7b-tdTomato based on <span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Dana et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Dana, H. ∙ Sun, Y. ∙ Mohar, B. ...</div><div><strong>High-performance calcium sensors for imaging activity in neuronal populations and microcompartments</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:649-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0435-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85067851088" target="_blank" aria-label="Open Scopus in new tab">Scopus (421)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0435-6&amp;pmid=31209382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, Twitch3A as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref55-11" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-11" aria-controls="sref55-11">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and for dominant-negative AMPA receptor expression with the EGFP-dnGluA2 construct (<span class="dropBlock reference-citations"><a id="body-ref-sref88-7" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-7" aria-controls="sref88-7">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-5" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-5" aria-controls="sref90-5">Venkatesh et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref90-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Venkatesh, H.S. ∙ Morishita, W. ∙ Geraghty, A.C. ...</div><div><strong>Electrical and synaptic integration of glioma into neural circuits</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:539-545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1563-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072686777" target="_blank" aria-label="Open Scopus in new tab">Scopus (407)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1563-y&amp;pmid=31534222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Transduced cells were sorted regularly by FACS with either FACSAria Fusion 2 Bernhard Shoor or FACSAria Fusion Richard Sweet. The BL 530/30 filter was used for FACS-sorting GFP and Twitch3A. For tdTomato, the YG 586/15 filter was used.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Surgical procedures</h4><div id="p0245" role="paragraph">Surgical procedures were performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref55-12" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-12" aria-controls="sref55-12">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-8" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-8" aria-controls="sref88-8">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cranial window implantation in mice was done in a modification of what we had previously described, including a custom-made titanium ring for painless head fixation during imaging. 500-900 nl of AAV viral particle for either one of the following constructs were injected cortically in 450&nbsp;μm depth: neuronal channelrhodopsin expression (pAAV-CaMKIIa-hChR2(H134R)-mCherry (AAV2), UNC Vector Core or pAAV-CamKIIa-ChrimsonR-mScarlet-KV2.1 (AAV9), Addgene 124651), a viral construct used as control (pAAV-CaMKIIa-mCherry (AAV2), Addgene 114469) or pAAVCAG-Synaptophysin-mCherry (AAV1/2) (analogous to <span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Wimmer et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Wimmer, V.C. ∙ Nevian, T. ∙ Kuner, T.</div><div><strong>Targeted in&nbsp;vivo expression of proteins in the calyx of Held</strong></div><div><em>Pflugers Arch.</em> 2004; <strong>449</strong>:319-333</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15452710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=15452710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) 1 to 3&nbsp;weeks after cranial window implantation, 50.000-100.000 tumor cells were stereotactically injected into the mouse cortex at an approximate depth of 500&nbsp;μm. Alternatively, the tumor cell injection was done in one procedure with the cranial window implantation.</div></section><section id="sec-4-4-2"><h4>General <i>in&nbsp;vivo</i> multiphoton laser scanning microscopy (MPLSM) and analysis</h4><div id="p0250" role="paragraph">For MPLSM imaging, male NMRI nude mice older than 8&nbsp;weeks (Charles River and Janvier) were operated to implant chronic cranial windows and were injected with tumor cells as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref55-13" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-13" aria-controls="sref55-13">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-9" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-9" aria-controls="sref88-9">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The tumors were intravitally observed from 3&nbsp;weeks after tumor implantation with a Zeiss 7MP microscope (Zeiss), a Zeiss LSM 980 with Airyscan2 and a TriM Scope II microscope (LaVision BioTec GmbH). All microscopes were equipped with a pulsed Ti:Sapphire laser (Chameleon II ultra; Coherent). SR101, GFP, tdTomato and TRITC dextran were imaged using 850&nbsp;nm and 960&nbsp;nm wavelengths respectively. Twitch3A reporter GBCs were imaged at an excitation wavelength of 860&nbsp;nm. Emitted fluorescence was split with a 560&nbsp;nm dichroic. The Zeiss 7MP and the Zeiss LSM 980 setup were equipped with bandpass filter sets of 500 - 550&nbsp;nm and 575 - 610&nbsp;nm. For the TriM Scope II setup, filter sets with bandpass 500 - 570&nbsp;nm and longpass 590&nbsp;nm were used. For FRET imaging, only the TriM Scope II was used. Emitted fluorescence was split with a 495&nbsp;nm dichroic and filtered with bandpass 460 - 500&nbsp;nm and bandpass 500 - 570&nbsp;nm. A 20x, 1.0 NA, apochromatic, 1.7&nbsp;mm working distance, water immersion objective (Zeiss) was used for imaging at the Zeiss 7MP and the Zeiss LSM 980 setup. The TriM Scope II set up contained a 16x, 0.8 NA, apochromatic, 3&nbsp;mm working distance, water immersion objective and a 25 x, 1.1 NA, apochromatic 2&nbsp;mm working distance, water immersion objective (both Nikon). Fluorescence emission was detected with low-noise high-sensitivity photomultiplier tubes.</div><div id="p0255" role="paragraph">Isoflurane gas was diluted in 100 % O2 to a concentration between 0.5 - 4.0% depending on the experiment. For the induction of anesthesia, the mouse was exposed to 3-5% isoflurane, which was lowered to 0.5-3% depending on the experiment for anesthesia maintenance and monitored based on the breathing rate of the mice. Eye cream was applied to the mice after anesthesia induction. During imaging, the body temperature of the mice was monitored and kept at 37°C using a temperature sensor and a heating plate. Anesthesia was regularly evaluated during image acquisition by checking postures and breathing rates. TRITC-dextran (fluorescent conjugated tetramethylrhodamine isothiocyanate-dextran, 500.000 g/mol) was dissolved in 0.9% NaCl-solution at 10&nbsp;mg/ml. Before imaging, 100&nbsp;μl of TRITC solution was injected into the lateral tail vein for blood vessel visualization. Connected<sup>TUM/AC</sup> GBCs were classified as such if there was at least one end-to-end connected TM to another GBC manually analyzed in 3D or showed uptake of SR101. Otherwise, GBCs were classified as unconnected<sup>TUM/AC</sup> GBCs.</div></section><section id="sec-4-4-3"><h4>Deep learning-enabled intravital subcellular timelapse imaging (DeepISTI) workflow</h4><div id="p0260" role="paragraph">An imaging and image processing pipeline was designed to enable high temporal and spatial resolution at low phototoxicity for three-dimensional <i>in&nbsp;vivo</i> two-photon time-lapse imaging. This workflow was developed to visualize and quantify <i>in&nbsp;vivo</i> GBC invasion on a subcellular level. The workflow consists of three layers: deep learning-based restoration microscopy, machine learning-based segmentation and subsequent post-processing analyses for further image enhancement and intensity normalization over time and subsequent quantification of GBC dynamics. For the deep learning-based image restoration we used the Nikon enhance.ai deep learning tool, analogous to the previously published CARE algorithms (<span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Weigert et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Weigert, M. ∙ Schmidt, U. ∙ Boothe, T. ...</div><div><strong>Content-aware image restoration: pushing the limits of fluorescence microscopy</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:1090-1097</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0216-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85057342134" target="_blank" aria-label="Open Scopus in new tab">Scopus (422)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0216-7&amp;pmid=30478326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To train the enhance.ai model, pairs of 3D image stacks were acquired with low and high signal-to-noise ratios. High signal-to-noise ratio imaging stacks were imaged with a pixel dwell time of 3.14&nbsp;μs at a pixel size of 0.59&nbsp;μm whereas low signal-to-noise ratio imaging stacks were imaged with the same resolution, with a pixel dwell time of 0.79&nbsp;μs. For high signal-to-noise-ratio imaging stacks laser powers between 5-17 mW were measured at the objective. The laser powers used were adapted according to the z-depth. For the low signal-to-noise ratios the laser powers were reduced by 10 %. These data were used in the NIS-Elements AR Analysis software suite to train the enhance.ai model with 1000 iterations. Similar imaging conditions to the low signal-to-noise ratio imaging stacks were then routinely acquired as time-lapse imaging experiments and the enhance.ai inference was applied to restore the low signal-to-noise image stacks. Further improvement of the signal-to-noise ratio and intensity normalization over time was achieved by training the ilastik pixel classification classifier using a combination of raw images and the restored data. Indeed, both pairs of images were used to train the pixel classification workflow and obtained as a prediction the probability map of the foreground pixels (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Berg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (831)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The ilastik probability maps trained on the combination of raw and enhanced images show the highest signal-to-noise ratio and allow analysis of subcellular structures such as TM or even small processes of GBCs <i>in&nbsp;vivo</i> over time. These data can then be further processed and for instance be used for tracking TM dynamics or to perform somatokinetic measurements as described below.</div></section><section id="sec-4-4-4"><h4><i>In&nbsp;vivo</i> 3D time-lapse experiments with isoflurane anesthesia experimental paradigms using MPLSM</h4><div id="p0265" role="paragraph">Tail veins were injected with TRITC-dextran under isoflurane anesthesia. Before time-lapse imaging, a stable respiratory rate (RR) was established by manual monitoring of breathing over 15 seconds with an application (Tap BPM <a id="intref0065" href="https://www.beatsperminuteonline.com/" target="_blank">https://www.beatsperminuteonline.com/</a>). Deep anesthesia corresponded to a mean respiratory rate&nbsp;&lt;&nbsp;60&nbsp;bpm, while light anesthesia corresponded to a mean respiratory rate &gt;&nbsp;100&nbsp;bpm. Once a stable respiratory rate was established, a field of view with a volume of 607.28μm x 607.28μm x 100μm, was selected and the image stack was repetitively imaged over 4h every 5&nbsp;min resulting in a total of 48 time points. The laser power was chosen at 90% of the optimal amount according to the built-in range indicator function of Zen to reduce laser power and avoid phototoxicity. Visible drift during time-lapse imaging was corrected manually after image acquisition of each stack in xyz directions. The respiratory rate was monitored after each stack as described above and the isoflurane concentration was adapted if needed to adjust the respective breathing rates as readout for anesthesia depth. After imaging, the mice were observed carefully until they woke up and injected s.c. with 1&nbsp;ml 0.9% NaCl-Glucose-solution for the prevention of dehydration.</div><div id="p0270" role="paragraph">The 48 stacks for each time point were hyperstacked and registered in Fiji via the vessel signal and by using the plugin Correct 3D drift (<span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Parslow et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Parslow, A. ∙ Cardona, A. ∙ Bryson-Richardson, R.J.</div><div><strong>Sample drift correction following 4D confocal time-lapse imaging</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2014; <strong>12</strong>:51086</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ParslowA.CardonaR.J.Bryson-RichardsonSample+drift+correction+following+4D+confocal+time-lapse+imagingJ.%C2%A0Vis.+Exp.12201451086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-5"><h4>Intravital neuronal channelrhodopsin stimulation and GBC imaging</h4><div id="p0275" role="paragraph">To analyze the effect of neuronal stimulation on TM dynamics and GBCs, neurons infected with AAV particles expressing CaMKIIahChR2-mCherry (experimental group) or CaMKIIa-mCherry (control group) (see ‘Surgical procedures’) were stimulated with 473&nbsp;nm laser light (Shanghai Laser &amp; Optics Century Co. Ltd, China) via an optical fiber (Laser Components, Item 3016347). Cells were stimulated 15 times at 20&nbsp;Hz for 30&nbsp;s using 90&nbsp;s pause intervals as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Venkatesh et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Venkatesh, H.S. ∙ Johung, T.B. ∙ Caretti, V. ...</div><div><strong>Neuronal activity promotes glioma growth through Neuroligin-3 secretion</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:803-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2015.04.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2015.04.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84928918997" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25913192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.012&amp;pmid=25913192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The PMT shutter of the TriM II MPLSM was closed during stimulation. Stimulation and shutter were controlled using a PulsePal device and a custom written MATLAB script (<span class="dropBlock reference-citations"><a id="body-ref-sref88-10" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-10" aria-controls="sref88-10">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Afterwards, the same tumor microregions were repetitively imaged every five minutes for four hours in analogy to the above-described experimental paradigm using isoflurane anesthesia.</div></section></section><section id="sec-4-5"><h3>SR101 <i>in&nbsp;vivo</i> imaging</h3><div id="p0280" role="paragraph">For all SR101 <i>in&nbsp;vivo</i> imaging, SR101 dissolved in 1x phosphate buffered saline (Sigma) was injected intraperitoneally at a dosage of 0.12&nbsp;mg per g bodyweight as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref101-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-2" aria-controls="sref101-2">Xie et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Xie, R. ∙ Kessler, T. ∙ Grosch, J. ...</div><div><strong>Tumor cell network integration in glioma represents a stemness feature</strong></div><div><em>Neuro. Oncol.</em> 2021; <strong>23</strong>:757-769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noaa275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100835529" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33320195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoaa275&amp;pmid=33320195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To determine SR101 uptake in GBCs, four experimental paradigms were used. In the first experimental paradigm, <i>in&nbsp;vivo</i> 3D time-lapse stacks of GBCs over the course of four hours were acquired (see ‘<i>In&nbsp;vivo</i> 3D time-lapse experiments with isoflurane anesthesia experimental paradigms using MPLSM’). SR101 was intraperitoneally injected directly after the experiment and a 3D stack of the respective tumor microregion was acquired after six to eight hours. The 3D stack (3.15&nbsp;μs pixel dwell time, 0.59&nbsp;μm pixel size) was acquired in the same field of view (FoV) as for the previous time-lapse experiment. These stacks were then concatenated with the time-lapse images from the invasion experiment to clearly identify the same cells for subsequent analysis.</div><div id="p0285" role="paragraph">To analyze dynamic SR101 uptake in GBCs and glial cells, SR101 was injected into the mice at the same dosage stated above and after approximately 1.5h, the chosen FoV was imaged once every 5&nbsp;min for 4&nbsp;h (volume of FoV&nbsp;= 607.28 um x 607.28&nbsp;μm x 100&nbsp;μm, pixel size 0.59&nbsp;μm, 0.79&nbsp;μs pixel dwell time). A mean respiratory rate of &gt;100/min was aimed for and the respiratory rate was monitored after each stack. The isoflurane concentration was adapted accordingly. After imaging, mice were injected s.c. with 1&nbsp;ml 0.9% NaCl-Glucose-Solution and observed until awake. The stacks were hyperstacked and registered as described above.</div><div id="p0290" role="paragraph">In the third experimental paradigm, uptake of SR101 in GBCs was weekly determined. SR101 was injected each week as described above six to ten hours prior to two-photon microscopy imaging.</div><div id="p0295" role="paragraph">In the fourth experimental paradigm, SR101 uptake in GBCs and its inhibition by the gap junction inhibitor carbenoxolone was determined by direct injection of SR101 into tumor regions.</div><div id="p0300" role="paragraph">Specifically, 50 nl of 100&nbsp;μM SR101 and 100&nbsp;μM carbenoxolone (CBX, Sigma-Aldrich, C4790) were injected under isoflurane anesthesia through a thin glass pipette into the selected tumor regions with similar tumor density&nbsp;&gt;&nbsp;90&nbsp;days after tumor injection. For the control experiments, CBX was omitted. The images were acquired 120&nbsp;minutes after the microinjection.</div><div id="p0305" role="paragraph">SR101 signal intensity was measured in 3D image stacks by selecting the cell soma boundary in Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with the Polygon selection-tool in a slice with maximum extension in the x/y-axis and measuring the mean grey value in the SR101 channel.</div><div id="p0310" role="paragraph">For the SR101 intensity analyses, the results measured over time for GBCs and astrocytes were either normalized by subtracting the mean background intensity per time point, mean SR101 intensity values were normalized per dataset by dividing by the mean background intensity measured in various positions of the 3D image stack or by subtracting a filtered image of the raw data (Gaussian Blur with a sigma (radius)=20) from the raw data.</div><section id="sec-4-5-1"><h4><i>In&nbsp;vivo</i> imaging of synaptophysin and GBCs</h4><div id="p0315" role="paragraph">For the synaptophysin punctae tracking, 500-900 nl of AAV particles expressing the presynaptic vesicle marker synaptophysin (pAAV-CAG-Synaptophysin-mCherry (AAV1/2)) were injected into the mouse cortex at a depth of 400&nbsp;μm prior to tumor cell injection during the chronic cranial window implantation as described above. Synaptophysin is a synaptic vesicle marker and allows to reliably identify presynaptic boutons that can be distinguished with confocal microscopy as punctae in the range of 0.4-4&nbsp;μm (<span class="dropBlock reference-citations"><a id="body-ref-optdySj1QVrQC" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optdySj1QVrQC" aria-controls="optdySj1QVrQC">Fantuzzo et al., 2017</a><div class="dropBlock__holder" data-db-target-of="optdySj1QVrQC"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Fantuzzo, J.A. ∙ Mirabella, V.R. ∙ Hamod, A.H. ...</div><div><strong><i>Intellicount</i>: High-Throughput Quantification of Fluorescent Synaptic Protein Puncta by Machine Learning</strong></div><div><em>eNeuro.</em> 2017; <strong>4</strong>, ENEURO.0219-17.2017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0219-17.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85039731634" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29218324/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0219-17.2017&amp;pmid=29218324" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Three to six weeks after surgery, experiments were performed. Imaging of GBCs and respective synaptophysin punctae was repeated, with a minimal time interval of 1 hour between the first and the last time point (pixel size&nbsp;= 0.21&nbsp;μm, pixel dwell time&nbsp;= 4.73&nbsp;μs).</div><div id="p0320" role="paragraph">After image acquisition, the images were denoised using Denoise.ai within Nikon NIS- Elements AR Analysis. To determine putative neurogliomal synapses, we first segmented GBC that were fluorescently labeled with mGFP as well synaptic boutons with interactive machine learning (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Berg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (831)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and a subsequent watershed algorithm in Arivis Vision 4D. Next, we used the direct overlap of signal coming from the segmented synaptic boutons and GBCs which allows the identification of putative neurogliomal synapses. Usually, distance thresholds commonly used for determining synaptic connectivity in <i>ex&nbsp;vivo</i> analyses with confocal imaging 0.7-2.5&nbsp;μm (<span class="dropBlock reference-citations"><a id="body-ref-optO7kvXMytHi" href="#" role="doc-biblioref" data-xml-rid="bib108 bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optO7kvXMytHi" aria-controls="optO7kvXMytHi">Fogarty et al., 2013; Schätzle et al., 2012</a><div class="dropBlock__holder" data-db-target-of="optO7kvXMytHi"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Fogarty, M.J. ∙ Hammond, L.A. ∙ Kanjhan, R. ...</div><div><strong>A method for the three-dimensional reconstruction of Neurobiotin™-filled neurons and the location of their synaptic inputs</strong></div><div><em>Front. Neural Circuits.</em> 2013; <strong>7</strong>:153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncir.2013.00153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24101895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncir.2013.00153&amp;pmid=24101895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Schätzle, P. ∙ Wuttke, R. ∙ Ziegler, U. ...</div><div><strong>Automated quantification of synapses by fluorescence microscopy</strong></div><div><em>J. Neurosci. Methods.</em> 2012; <strong>204</strong>:144-149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jneumeth.2011.11.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-82355161288" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22108140/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jneumeth.2011.11.010&amp;pmid=22108140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, the chosen workflow should enable a more reliable identification of putative neurogliomal synapses.</div></section><section id="sec-4-5-2"><h4><i>In&nbsp;vivo</i> calcium imaging with Rhod2-AM</h4><div id="p0325" role="paragraph">For simultaneous <i>in&nbsp;vivo</i> calcium imaging of GBCs and astrocytes, the following small molecule calcium indicators were applied to the brain surface for 45&nbsp;minutes as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref55-14" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-14" aria-controls="sref55-14">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) based on previous published protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Kantevari et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Kantevari, S. ∙ Gordon, G.R.J. ∙ MacVicar, B.A. ...</div><div><strong>A practical guide to the synthesis and use of membrane-permeant acetoxymethyl esters of caged inositol polyphosphates</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:327-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2010.194" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-79952402173" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21372813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2010.194&amp;pmid=21372813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Takano et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Takano, T. ∙ Tian, G.F. ∙ Peng, W.G. ...</div><div><strong>Astrocyte-mediated control of cerebral blood flow</strong></div><div><em>Nat. Neurosci.</em> 2006; <strong>9</strong>:260-267</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn1623" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-31544456102" target="_blank" aria-label="Open Scopus in new tab">Scopus (866)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16388306/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn1623&amp;pmid=16388306" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): for GFP-transfected tumor cells, 2 mM Rhod-2AM (Life Technologies, R-1244); for RFP-transfected, 2 mM Fluo-4AM (Life Technologies, F-14201). Subsequent <i>in&nbsp;vivo</i> calcium imaging was performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref55-15" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-15" aria-controls="sref55-15">Osswald et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-3"><h4><i>In&nbsp;vivo</i> imaging of dominant-negative AMPAR and control GBCs</h4><div id="p0330" role="paragraph"><i>In&nbsp;vivo</i> imaging of dominant-negative AMPAR and control GBCs was performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref88-11" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-11" aria-controls="sref88-11">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0335" role="paragraph">Previously, we validated the dominant-negative AMPAR in HEK cells transfected with GluA2-eGFP and dominant-negative GluA2-tdTomato as compared to cells transfected with GluA2-eGFP alone. As previously described, glutamate-induced peak currents were significantly inhibited when HEK cell were transfected with both GluA2-eGFP and dominant-negative GluA2-tdTomato (<span class="dropBlock reference-citations"><a id="body-ref-sref88-12" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-12" aria-controls="sref88-12">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-4"><h4>3D <i>in&nbsp;vivo</i> calcium imaging with intravital MPLSM and image analysis</h4><div id="p0340" role="paragraph">3D <i>in&nbsp;vivo</i> calcium imaging was performed to cover whole GBC calcium dynamics with a sufficient time resolution to capture calcium microdomains, whole cell and multicellular calcium events. Cells were centered in the field of view and their maximum z range was determined. The piezoelectric objective z-actuator was used to repetitively acquire z stacks over a range of 15-25&nbsp;μm with a z step size between 3-5&nbsp;μm and a frequency of 0.3-0.5&nbsp;Hz.</div><div id="p0345" role="paragraph">To analyze the effect of neuronal stimulation on GBC calcium dynamics, neurons expressing CaMKIIahChR2-mCherry or CamKIIa-ChrimsonR-mScarlet after AAV infection (see ‘Surgical procedures’) were stimulated with 473&nbsp;nm laser light (Shanghai Laser &amp; Optics Century Co. Ltd, China) via an optical fiber (Laser Components, Item 3016347). For 3D <i>in&nbsp;vivo</i> calcium imaging, regions were stimulated at 40&nbsp;Hz for 1&nbsp;s with 1&nbsp;min intervals between stimulations for five minutes. Baseline activity was recorded for five minutes beforehand. Pulsed illumination at 40&nbsp;Hz for 1&nbsp;s was an effective trigger for calcium transients in GBCs as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref88-13" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-13" aria-controls="sref88-13">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0350" role="paragraph">The maximum intensity projection (MIP) of the z-stack was used for further analysis. The time-lapse MIP stack was motion corrected using the ImageJ/Fiji plugin moco (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Dubbs et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Dubbs, A. ∙ Guevara, J. ∙ Yuste, R.</div><div><strong>moco: fast Motion Correction for Calcium Imaging</strong></div><div><em>Front. Neuroinform.</em> 2016; <strong>10</strong>:6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fninf.2016.00006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26909035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffninf.2016.00006&amp;pmid=26909035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Calcium events in motion-corrected stacks were analyzed using AquA (Astrocyte Quantification and Analysis) (<span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Wang et&nbsp;al., 2019b</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Wang, Y. ∙ DelRosso, N.V. ∙ Vaidyanathan, T.V. ...</div><div><strong>Accurate quantification of astrocyte and neurotransmitter fluorescence dynamics for single-cell and population-level physiology</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1936-1944</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0492-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85074279015" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0492-2&amp;pmid=31570865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-5"><h4>General image processing</h4><div id="p0355" role="paragraph">Image processing was primarily performed in ImageJ/Fiji (e.g. to reduce and remove unspecific background by subtraction of different channels, filtering with a median filter or the ‘Remove Outlier’) (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0360" role="paragraph">Arivis Vision 4D and ImageJ/Fiji were used for 3D and 4D Image visualization. Image restoration was performed using the enhance.ai tool within the Nikon NIS-Elements AR Analysis software v5.30.01 (Nikon GmbH Germany/Laboratory Imaging) as described above. Confocal Laser Scanning Microscopy (CLSM) images and <i>in&nbsp;vivo</i> imaging data were denoised using the denoise.ai pretrained&nbsp;model and enhance.ai (see ‘Deep learning-enabled intravital subcellular time-lapse imaging (DeepISTI) workflow’) in the Nikon NIS-Elements AR software v5.30.01 (Nikon GmbH Germany/Laboratory Imaging). Videos were produced in DaVinci Resolve 17.</div></section><section id="sec-4-5-6"><h4>Dynamics measurement and directionality change analysis of GBC processes</h4><div id="p0365" role="paragraph">Tumor microtube dynamics were tracked using MTrackJ (<span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Meijering et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Meijering, E. ∙ Dzyubachyk, O. ∙ Smal, I.</div><div><strong>Methods for cell and particle tracking</strong></div><div><em>Methods Enzymol.</em> 2012; <strong>504</strong>:183-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/B978-0-12-391857-4.00009-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84855958004" target="_blank" aria-label="Open Scopus in new tab">Scopus (1017)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22264535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FB978-0-12-391857-4.00009-4&amp;pmid=22264535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each tumor cell process was tracked separately for all time points, and subsequent TM dynamics quantification was performed using custom-written scripts in RStudio (Version 1.2.5033). The directionality change of the leading TM was calculated by comparing the direction of the leading TM at time point 0 and after 4 hours. The directions of the TMs were used as vectors. The angle between the two vectors was determined with ImageJ/Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref66-3" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-3" aria-controls="sref66-3">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref66-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-7"><h4>Somatokinesis measurement</h4><div id="p0370" role="paragraph">Measurement of somatokinesis was performed in ImageJ/Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref66-4" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-4" aria-controls="sref66-4">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref66-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were cropped and registered using the 3D Drift Correction plugin in ImageJ/Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Parslow et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Parslow, A. ∙ Cardona, A. ∙ Bryson-Richardson, R.J.</div><div><strong>Sample drift correction following 4D confocal time-lapse imaging</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2014; <strong>12</strong>:51086</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ParslowA.CardonaR.J.Bryson-RichardsonSample+drift+correction+following+4D+confocal+time-lapse+imagingJ.%C2%A0Vis.+Exp.12201451086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The cell somata were outlined and the center point of the selection was determined using the centroid function. The distance between the center points of two time points was divided by the experimental observation time to calculate the somatokinetic speed.</div></section><section id="sec-4-5-8"><h4>Measurement of TM turnover</h4><div id="p0375" role="paragraph">The tips of TMs were manually tracked over time with MTrackJ (<span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Meijering et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Meijering, E. ∙ Dzyubachyk, O. ∙ Smal, I.</div><div><strong>Methods for cell and particle tracking</strong></div><div><em>Methods Enzymol.</em> 2012; <strong>504</strong>:183-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/B978-0-12-391857-4.00009-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84855958004" target="_blank" aria-label="Open Scopus in new tab">Scopus (1017)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22264535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FB978-0-12-391857-4.00009-4&amp;pmid=22264535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Subsequently, the steps between each time point of these tracks were extracted. The TM turnover per single TM was determined by adding up the distances between the single steps and dividing this number by the experimental observation time and referred to as TM turnover per TM to sum up the dynamics of single TMs. The TM turnover per GBC was determined by tracking all TMs of a cell, adding single steps up and divide this number by the experimental observation time and the number of TMs. The total TM turnover per GBC was determined by tracking all TMs of a cell, adding single steps up and divide this number by the experimental observation time.</div><div id="p0380" role="paragraph">TM turnover per TM was used to investigate the heterogeneity of TM dynamics in blind TMs as compared to 4 connected TMs. To characterise the TM turnover of the three different invasion phenotypes, the total TM turnover per GBC was analyzed. For this purpose, the TM turnover of all TMs per cell was added up. To determine the effects of neuronal stimulation, effects on TM dynamics of whole GBCs were determined. As neuronal stimulation also increased branching TM generation, TM turnover was normalized in addition to the TMs per cell.</div></section><section id="sec-4-5-9"><h4>Classification of TM</h4><div id="p0385" role="paragraph">The TMs of GBCs were analyzed in three-dimensional imaging stacks to determine whether they were connected or ended blindly: Only end-to-end connections between TMs or TM connected with GBC somata were classified as connected TMs, all other TMs were classified as blindly ending TMs.</div></section><section id="sec-4-5-10"><h4>Classification of GBC small processes</h4><div id="p0390" role="paragraph">Filopodial structures were defined as structures shorter than 10&nbsp;μm and thinner than 1.25&nbsp;μm. Structures that were shorter than normal TMs (10&nbsp;μm), but exhibiting TM diameters from 1.25&nbsp;μm to 3&nbsp;μm were classified as TM-like structures. Short structures that were thicker than normal TM (&gt; 3&nbsp;μm) were classified as lamellipodia like structures.</div></section><section id="sec-4-5-11"><h4>FACS sorting and scRNA Seq</h4><div id="p0395" role="paragraph">To distinguish between connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> GBCs in fluorescent (GFP) PDX models, 0.12&nbsp;mg/g body weight SR101 injected i.p. into tumor bearing mice prior to perfusion as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref101-3" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-3" aria-controls="sref101-3">Xie et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Xie, R. ∙ Kessler, T. ∙ Grosch, J. ...</div><div><strong>Tumor cell network integration in glioma represents a stemness feature</strong></div><div><em>Neuro. Oncol.</em> 2021; <strong>23</strong>:757-769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noaa275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100835529" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33320195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoaa275&amp;pmid=33320195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The perfused brain was dissolved into a single cell suspension by using the brain tumor dissociation kit and device by Miltenyi as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref101-4" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-4" aria-controls="sref101-4">Xie et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Xie, R. ∙ Kessler, T. ∙ Grosch, J. ...</div><div><strong>Tumor cell network integration in glioma represents a stemness feature</strong></div><div><em>Neuro. Oncol.</em> 2021; <strong>23</strong>:757-769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noaa275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100835529" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33320195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoaa275&amp;pmid=33320195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The samples were stained with a dead cell marker (TO-PRO3) and a live cell marker (Calcein Violet 450 AM) to identify the vital cell population. The single cell suspension was sorted with a FACSAria Fusion 2 Bernhard Shoor or a FACSAria Fusion Richard Sweet (BD Biosciences). Tumor cells were identified by the GFP signal measured with the filter BL-530/30 and the SR101 signal was determined with the filter YG-586/15. Lasers with wavelengths of 488&nbsp;nm, 561&nbsp;nm and 640&nbsp;nm were used for this purpose. Tumor cells were gated into connected<sup>TUM/AC</sup> (SR101 high) and unconnected<sup>TUM/AC</sup> (SR101 low) groups as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref101-5" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-5" aria-controls="sref101-5">Xie et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Xie, R. ∙ Kessler, T. ∙ Grosch, J. ...</div><div><strong>Tumor cell network integration in glioma represents a stemness feature</strong></div><div><em>Neuro. Oncol.</em> 2021; <strong>23</strong>:757-769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noaa275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100835529" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33320195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoaa275&amp;pmid=33320195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-12"><h4>Xenograft single-cell RNA-seq data generation and processing</h4><div id="p0400" role="paragraph">Xenograft single-cell RNA-sequencing data were taken from <span class="dropBlock reference-citations"><a id="body-ref-sref31-4" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-4" aria-controls="sref31-4">Hai et&nbsp;al. (2021)</a><div class="dropBlock__holder" data-db-target-of="sref31-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hai, L. ∙ Hoffmann, D.C. ∙ Mandelbaum, H. ...</div><div><strong>A connectivity signature for glioblastoma</strong></div><div>Preprint at <em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.11.07.465791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85121392461" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.11.07.465791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Gene expression count matrices were generated and aligned with Cell Ranger (v.2.1.1, 10X Genomics). Quality controls were performed as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref31-5" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-5" aria-controls="sref31-5">Hai et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Hai, L. ∙ Hoffmann, D.C. ∙ Mandelbaum, H. ...</div><div><strong>A connectivity signature for glioblastoma</strong></div><div>Preprint at <em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.11.07.465791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85121392461" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.11.07.465791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-13"><h4>Human scRNA-seq datasets</h4><div id="p0405" role="paragraph">The Yu dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref102-5" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-5" aria-controls="sref102-5">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref102-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Yu, K. ∙ Hu, Y.Q. ∙ Wu, F. ...</div><div><strong>Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies</strong></div><div><em>Natl. Sci. Rev.</em> 2020; <strong>7</strong>:1306-1318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwaa099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85092585099" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34692159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwaa099&amp;pmid=34692159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was obtained from the Gene Ontology Omnibus (GSE117891). Cells that were classified as tumor cells as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref22-8" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-8" aria-controls="sref22-8">Garofano et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were used for the subsequent analysis presented in this manuscript. Data was normalized and scaled in Seurat (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (4645)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Principal components were calculated using RunPCA. The Neftel dataset (<span class="dropBlock reference-citations"><a id="body-ref-sref52-8" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-8" aria-controls="sref52-8">Neftel et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was obtained from the Gene Ontology Omnibus (GSE131928).</div></section><section id="sec-4-5-14"><h4>Definition of single-cell invasivity signature and AMPAR scores</h4><div id="p0410" role="paragraph">Cell cycle scoring was performed on the three PDX models and their scores were regressed out in Seurat. After scaling and centering, principal component analysis was performed. Data was visualized with UMAP using the first ten principal components. Using&nbsp;Monocle3 (1.0.0) (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Cao et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1050)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), cells were clustered and the graph was learned based on the reduced dimensions using the learn graph function. Subsequently, cells were ordered using the order_cells function where root nodes were set to the unconnected<sup>TUM/AC</sup> cells of each cell line. Pseudotime was inferred for each cell. For all genes, expression was correlated to pseudotime using the stats::cor function. For each cell line, genes whose absolute value of their correlation coefficient was higher than 0.2 were further analyzed. Genes that were correlated or anticorrelated in the same direction in two cell lines were kept. The expression levels for correlated and anticorrelated genes were calculated separately by using the AddModuleScore function for each group of genes. For each cell, the score calculated for the correlated genes was subtracted from the score calculated for the anticorrelated genes to compute the invasivity score. For further validation of the gene sets defining the invasivity score, GeneCards information was downloaded from GeneCards using batch inquiries at GeneALaCart (<span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Stelzer et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Stelzer, G. ∙ Rosen, N. ∙ Plaschkes, I. ...</div><div><strong>The GeneCards Suite: from gene data mining to disease genome sequence analyses</strong></div><div><em>Curr. Protoc. Bioinformatics.</em> 2016; <strong>54</strong></div><div>1 30 31–31 30 33</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpbi.5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85016156181" target="_blank" aria-label="Open Scopus in new tab">Scopus (1361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27322403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpbi.5&amp;pmid=27322403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0415" role="paragraph">For the AMPAR gene expression score all four AMPAR genes GRIA1, GRIA2, GRIA3 and GRIA4 were used using the AddModuleScore function in <span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Stuart et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (4645)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section><section id="sec-4-5-15"><h4>Validation of the Invasivity Score</h4><div id="p0420" role="paragraph">Nine patient-derived orthotopic xenograft (PDOX) models representing invasive, intermediate and angiogenic growth phenotypes were derived as described in <span class="dropBlock reference-citations"><a id="body-ref-sref27-10" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-10" aria-controls="sref27-10">Golebiewska et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref27-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> following the protocol presented in <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Oudin et&nbsp;al. (2021)</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Oudin, A. ∙ Baus, V. ∙ Barthelemy, V. ...</div><div><strong>Protocol for derivation of organoids and patient-derived orthotopic xenografts from glioma patient tumors</strong></div><div><em>Star Protoc.</em> 2021; <strong>2</strong>:100534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.xpro.2021.100534" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85105598446" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34027491/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.xpro.2021.100534&amp;pmid=34027491" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Shortly, primary organoids were derived from mechanically minced IDH-wildtype GB tumor tissue. Organoids were cultured shortly in DMEM medium, 10% FBS, 2mM L-Glutamine, 0.4&nbsp;mM NEAA, and 100&nbsp;U/ml Pen–Strep (all from Lonza) for maximum 12&nbsp;days. Organoids were implanted intracranially to immunodeficient mice and tumor development was monitored by MRI. Tumor development was validated by IHC using an anti-vimentin antibody (Thermo Fisher Scientific, Mab3400), vasculature was monitored with mouse-specific anti-CD31 antibody (Millipore, CBL1337). Tumors were dissociated with the MACS Neural Dissociation kit (Miltenyi Biotec). Human tumor cells were purified with FACS (as human-specific CD90<sup>+</sup> cells) or MACS with Mouse Cell Depletion kit (Miltenyi Biotec). Single cell RNA-seq based on Drop-seq or 10xGenomics technology was performed on purified human tumor cells as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Dirkse et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Dirkse, A. ∙ Golebiewska, A. ∙ Buder, T. ...</div><div><strong>Stem cell-associated heterogeneity in Glioblastoma results from intrinsic tumor plasticity shaped by the microenvironment</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1787</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09853-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85064545608" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30992437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09853-z&amp;pmid=30992437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and according to the manufacturer’s protocol. scRNA-seq data is available at Omnibus repository (<a id="intref0070" href="https://www.ncbi.nlm.nih.gov/geo/" target="_blank">https://www.ncbi.nlm.nih.gov/geo/</a>) under accession number GSE128195.</div></section><section id="sec-4-5-16"><h4>GO Term Analysis and GO Term Clustering</h4><div id="p0425" role="paragraph">Gene sets of differentially expressed genes based on anatomic regions were used from previously published work (<span class="dropBlock reference-citations"><a id="body-ref-sref58-3" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-3" aria-controls="sref58-3">Puchalski et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref58-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Puchalski, R.B. ∙ Shah, N. ∙ Miller, J. ...</div><div><strong>An anatomic transcriptional atlas of human glioblastoma</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:660-663</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2666" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85046671790" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29748285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2666&amp;pmid=29748285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gene sets were analyzed in Cytoscape (Version 3.8.2, Java 11.0.6) (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Shannon et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Shannon, P. ∙ Markiel, A. ∙ Ozier, O. ...</div><div><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:2498-2504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.1239303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0242490780" target="_blank" aria-label="Open Scopus in new tab">Scopus (26124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.1239303&amp;pmid=14597658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the Biological Networks Gene Ontology tool (BiNGO, Version 3.0.4) (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Maere et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Maere, S. ∙ Heymans, K. ∙ Kuiper, M.</div><div><strong>BiNGO: a cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:3448-3449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bti551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-24044440971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15972284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbti551&amp;pmid=15972284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Overrepresented GO Terms within the Biological Process Ontology were calculated by using a hypergeometric test as statistical test and the Benjamini &amp; Hochberg False Discovery Rate (FDR) Correction to correct for multiple testing. Additional GO term analyses were performed using ShinyGO (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Ge et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Ge, S.X. ∙ Jung, D. ∙ Yao, R.</div><div><strong>ShinyGO: a graphical gene-set enrichment tool for animals and plants</strong></div><div><em>Bioinformatics.</em> 2020; <strong>36</strong>:2628-2629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz931" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85084025997" target="_blank" aria-label="Open Scopus in new tab">Scopus (667)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz931&amp;pmid=31882993" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-17"><h4>Transcription factor analysis using SCENIC</h4><div id="p0430" role="paragraph">Transcription factor analysis was performed using the python implementation of SCENIC (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Van de Sande et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Van de Sande, B. ∙ Flerin, C. ∙ Davie, K. ...</div><div><strong>A scalable SCENIC workflow for single-cell gene regulatory network analysis</strong></div><div><em>Nat. Protoc.</em> 2020; <strong>15</strong>:2247-2276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-020-0336-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85086586785" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32561888/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-020-0336-2&amp;pmid=32561888" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gene regulatory networks were inferred using the GRNBoost2 algorithm of SCENIC. Enriched motifs were identified and regulon prediction was performed using the CLI ctx function. The enrichment of regulon activity for each cell was calculated using the AUCELL algorithm. Results were integrated into Seurat (<span class="dropBlock reference-citations"><a id="body-ref-sref76-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-3" aria-controls="sref76-3">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref76-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (4645)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-18"><h4><i>In&nbsp;vitro</i> co-culture</h4><div id="p0435" role="paragraph">Primary cultures of rat cortical neurons and astrocytes were prepared as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref88-14" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-14" aria-controls="sref88-14">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, the cells were cultivated from E19 embryos and plated at a density of 90,000 cells/cm<sup>2</sup> on 12&nbsp;mm coverslips in 24-well plates coated with poly-L-lysine and maintained in neurobasal medium (Invitrogen) supplemented with B27 supplement (50x) (2% v/v) and L-glutamine (0.5&nbsp;mM). After 7&nbsp;days <i>in&nbsp;vitro</i> (DIV), mechanically dissociated, patient-derived primary GBCs (1000 per well) were added and co-cultured in medium. For electrophysiology and live-cell imaging GBCs (DIV 4-28) were used.</div></section><section id="sec-4-5-19"><h4>Drug treatment in co-cultures with neurons, astrocytes and GBCs</h4><div id="p0440" role="paragraph">After seeding 1000 GBC onto E19 rat cortex neurons (DIV 7) in 24 well plates and waiting 4-7&nbsp;days, they were incubated with BAPTA-AM (Sigma-Aldrich; reaching a culture medium concentration of 1,2&nbsp;μM), 666-15 (Tocris, reaching a culture medium concentration of 1μM) or DMSO as control. After 12-hour incubation, well plates were imaged using a LSM 710 confocal microscope with a 10x (NA 0.3) objective in 37 degrees Celsius and 5% CO<sub>2</sub> every 10&nbsp;minutes for a period of 9-12 hours.</div><div id="p0445" role="paragraph">For live-cell imaging experiments, gabazine (Abcam, reaching a culture medium concentration of 4μM) was added prior to imaging to inhibit GABA<sub>A</sub> receptor activity, thus disinhibiting neuronal activity and leading to neuronal stimulation and epileptiform neuronal activity (<span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Margineanu and Wulfert, 1997</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Margineanu, D.G. ∙ Wülfert, E.</div><div><strong>Inhibition by levetiracetam of a non-GABA(A) receptor-associated epileptiform effect of bicuculline in rat hippocampus</strong></div><div><em>Br. J. Pharmacol.</em> 1997; <strong>122</strong>:1146-1150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sj.bjp.0701476" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0030734833" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9401779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsj.bjp.0701476&amp;pmid=9401779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0450" role="paragraph">Seeding of S24 GBCs was done as described before, with S24tdTomato and S24GluA2Q dominant negative tdTomato cells (<span class="dropBlock reference-citations"><a id="body-ref-sref88-15" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-15" aria-controls="sref88-15">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 5-10&nbsp;days after seeding, cells were imaged in baseline live cell (5% CO<sub>2</sub>, 37 degrees Celsius) conditions using a Zeiss LSM 710 confocal microscope with a 10x (NA 0.3) objective. After 4 hours, perampanel (BioCrick, reaching a culture medium concentration of 60μM) was applied and imaged for the same time period.</div><div id="p0455" role="paragraph">All images were acquired with a pixel size of 346nm in xy with a fully open pinhole and subsequently analyzed manually with ImageJ/Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref66-5" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-5" aria-controls="sref66-5">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref66-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-20"><h4>Simultaneous <i>in&nbsp;vitro</i> co-culture calcium imaging and time-lapse imaging of GBC</h4><div id="p0460" role="paragraph">For <i>in&nbsp;vitro</i> calcium imaging the genetically encoded calcium indicator jGCaMP7b (<span class="dropBlock reference-citations"><a id="body-ref-sref14-3" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-3" aria-controls="sref14-3">Dana et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref14-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Dana, H. ∙ Sun, Y. ∙ Mohar, B. ...</div><div><strong>High-performance calcium sensors for imaging activity in neuronal populations and microcompartments</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:649-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0435-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85067851088" target="_blank" aria-label="Open Scopus in new tab">Scopus (421)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0435-6&amp;pmid=31209382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used. GBCs in our co-cultures were identified by the tdTomato signal.</div><div id="p0465" role="paragraph">For live cell imaging, co-culture coverslips (DIV 5-12) were immersed in 2ml neurobasal medium supplemented with L-glutamine (0.5&nbsp;mM) and B27 supplement (50x). Coverslips were imaged for 100&nbsp;minutes at 37 degrees Celsius with 5% CO2 using a 20x objective of Leica TCS SP8 confocal microscope with a numerical aperture of 0.75. Images were acquired with a pixel size of 445,63&nbsp;nm in xy with a fully open pinhole and an image frequency of 1&nbsp;Hz in a bidirectional acquisition mode.</div></section><section id="sec-4-5-21"><h4>Glutamate puffing and simultaneous calcium imaging</h4><div id="p0470" role="paragraph">1000 GBCs were seeded onto rat cortical neurons (DIV 7). After 5-12&nbsp;days, coverslips were transferred to an upright confocal setup and perfused with carbonated aCSF at 35°C. Simultaneous calcium imaging and puffing was performed. For image acquisition, a TCS SP5 Leica microscope equipped with a 20x water immersion objective (NA 1,0) and a Luigs and Neumann stage was used. Images were acquired with a pixel size of 0,2&nbsp;μm and 1,3&nbsp;μm with a frequency of 1,56&nbsp;Hz in a bidirectional xyt acquisition mode. For simultaneous puffing and calcium imaging, Patchmaster software (HEKA) was used for triggered puff recordings, with a puff applied every 45 s. Puffs were generated at 10-15 PSI using a Picospritzer. Pipettes for puffing were fabricated from borosilicate capillaries (World Precision Instruments) and had resistances of 2-7 MΩ. For all co-culture experiments, aCSF with 1&nbsp;μM tetrodotoxin (TTX) was used to exclude that action potentials of neurons are evoked by the puffing of glutamate. Puffing was performed as follows: Glutamate puffing, aCSF puffing, drug wash-in for 7&nbsp;min 30 s, i.e. CNQX (Abcam; reaching an aCSF concentration of 20&nbsp;μM) /TTX (HelloBio; reaching an aCSF concentration of 1&nbsp;μM) or BAPTA-AM (Sigma-Aldrich; reaching an aCSF concentration of 1,2&nbsp;μM) /TTX (HelloBio; reaching an aCSF concentration of 1&nbsp;μM)) respectively, calcium imaging under pharmacologically altered conditions, CNQX/TTX wash-out (7min 30 s) and in the case of CNQX/TTX calcium imaging after wash out with continuous glutamate puffing after aCSF puffing control. Total recording time per imaging series was 3&nbsp;min 45&nbsp;s with 5 puff stimulations. Data were analyzed using ImageJ/Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref66-6" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-6" aria-controls="sref66-6">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref66-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and AQuA (<span class="dropBlock reference-citations"><a id="body-ref-sref95-3" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-3" aria-controls="sref95-3">Wang et&nbsp;al., 2019b</a><div class="dropBlock__holder" data-db-target-of="sref95-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Wang, Y. ∙ DelRosso, N.V. ∙ Vaidyanathan, T.V. ...</div><div><strong>Accurate quantification of astrocyte and neurotransmitter fluorescence dynamics for single-cell and population-level physiology</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1936-1944</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0492-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85074279015" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0492-2&amp;pmid=31570865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Correlated calcium transients were defined as transients occurring within 20&nbsp;s or less after glutamate stimulation. GBCs in which ACSF puffing could evoke calcium transients were excluded from the analysis.</div></section><section id="sec-4-5-22"><h4><i>In&nbsp;vitro</i> co-culture imaging of synaptophysin and GBCs</h4><div id="p0475" role="paragraph">For the <i>in&nbsp;vitro</i> synaptophysin punctae tracking, cortical rat neurons (DIV 7) were transduced with 1μl of AAV particles expressing the presynaptic vesicle marker synaptophysin (pAAV-CAG-Synaptophysin-mCherry (AAV1/2)). Three days after neuronal infection, S24GFP GBCs were seeded onto cortical neurons.</div><div id="p0480" role="paragraph">Seven days after seeding, these co-cultures were imaged in glass bottom dishes using a TCS SP8, Leica with a 40x objective (NA&nbsp;1.1) and pixel size of 94,7&nbsp;nm in xy. They were observed over a period of two hours with a 15-minute interval.</div><div id="p0485" role="paragraph">Image processing and analysis were done as described above for the <i>in&nbsp;vivo</i> imaging analysis (see ‘<i>In&nbsp;vivo</i> imaging of synaptophysin and GBCs’).</div></section><section id="sec-4-5-23"><h4>Sample preparation, microscopy and analysis for correlative light and scanning electron microscopy (SEM)</h4><div id="p0490" role="paragraph">For <i>ex&nbsp;vivo</i> analyses of PDX models the mice were anesthetized with either ketamine/xylazine or pentobarbital i.p. First, mice were perfused transcardially with PBS followed by 4-4,5% PFA (w/v) in 1x PBS. After the removal of the brain, it was postfixed in 4% PFA overnight and kept in PBS at 4°C. Serial sections of 70-100&nbsp;μm were cut with a semiautomatic vibratome (Leica VT1000s). The expression of fluorescence (mGFP or cytosolic GFP) in the GBCs of the PDX models was checked under a wide field fluorescence microscope (Leica DM6000) and slabs of ∼400&nbsp;μm x 400&nbsp;μm in size were dissected.</div><div id="p0495" role="paragraph">For the nestin antibody staining, the human slices were put into 10% (w/v) sucrose (Sigma) dissolved in PBS for 10&nbsp;minutes. Subsequently, the concentration was increased and slices were stored in 30% sucrose solution for 12-15h. To prepare the infiltration of the antibody, slices were freeze-thawed in liquid nitrogen twice for 5 seconds. As a blocking solution, 5% FBS in PBS was used for 1 hour at RT. The slices were incubated overnight at 4°C with human-specific mouse anti-nestin antibody in blocking solution and washed three times in PBS. The samples were stained in 3,3’-diaminobenzidine (DAB) precipitate. For the DAB precipitate labeling, slices were incubated for 12-15h at 4°C with secondary AB, a biotinylated anti-mouse AB (Abcam (ab6788), 1:500, in blocking solution). Slices were washed in PBS three times and then incubated in Vectastain ABC-kit (Linaris) for 1h at RT. Incubation in glucose-DAB-solution (glucose: 2mg/ml, DAB: 1.4mg/ml, dissolved) followed. After another hour of incubation in glucose-DAB oxidase solution (glucose oxidase: 0.1&nbsp;mg/ml, Serva) the electron dense precipitate could be assessed under a wide field light microscope.</div><div id="p0500" role="paragraph">For correlative light and electron microscopy (CLEM), the sections were stained with DAPI (1:1000) for 1 hour and subsequently imaged under a Leica TCS SP8 confocal microscope with a 63X objective (NA 1.4) and a 20X (NA 0.75) a with a pixel size of 200&nbsp;nm in a z-stack taking an image at every 520&nbsp;nm with a scanning speed of 400-600Hz. The sections were washed in PBS and in cacodylic buffer (100&nbsp;mM, pH 7.4, (Serva)) (SERVA#15540.1 Cacodylic acid-Na-salt-3H2O, 100mM, pH 7.4) at 4°C. For post-fixation and contrasting it was incubated in 1.5% potassium ferricyanide (Serva) and 2% osmium tetroxide (Serva) for 1h on ice. Next, the sections were rinsed three times with dH2O and then put on a spinning wheel for 30&nbsp;minutes. The sections were dehydrated in solutions of increasing ethanol concentrations (70%/95%/100%). The samples were subsequently embedded in resin as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Horstmann et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Horstmann, H. ∙ Körber, C. ∙ Sätzler, K. ...</div><div><strong>Serial section scanning electron microscopy (S3EM) on silicon wafers for ultra-structural volume imaging of cells and tissues</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e35172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0035172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84859813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22523574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0035172&amp;pmid=22523574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The self-produced Epon consisted of 455.8&nbsp;mg/ml Glycidether 100/E812, 308.6&nbsp;mg/ml DDSA, 235.4&nbsp;mg/ml MNA hardener and 0,015&nbsp;ml/ml DMP-30/BDMA. Sections were then infiltrated in a 1:1 mixture of propylenoxide and Epon (Serva) for 12-15h at 4°C, followed by three times rinsing in fresh Epon 100% and embedding in epoxy resin. The sample blocks were polymerized at 60°C for at least 24 hours. Afterwards, the blocks were trimmed with the Leica EM-trim and fine trimming into a trapezoid form was performed with a razor blade. For CLEM, the whole tissue block was cut with a custom-modified ultramicrotome (Ultracut S, Leica) using a diamond knife (DIATOME Ultra 45°) (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Horstmann et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Horstmann, H. ∙ Körber, C. ∙ Sätzler, K. ...</div><div><strong>Serial section scanning electron microscopy (S3EM) on silicon wafers for ultra-structural volume imaging of cells and tissues</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e35172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0035172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84859813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22523574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0035172&amp;pmid=22523574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Serial sectioning of the whole sample (70-100 um) with 40-70&nbsp;nm slices resulted in up to 30 wafers with up to 100 sections per wafer. While cutting, the bottom side of the trapezoid-shaped sample was coated with a thin layer of Patex glue (Henkel, Düsseldorf, Germany) and Xylene (1:3) to give the long ribbon more stability and connectivity on the water surface in the knife boat.</div><div id="p0505" role="paragraph">After drying, the wafers were poststained with uranyl acetate (3%) in a closed dark reaction tube for 16&nbsp;minutes followed by rinsing shortly in dH2O for three times. A modified Reynolds-procedure was used as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref33-3" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-3" aria-controls="sref33-3">Horstmann et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref33-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Horstmann, H. ∙ Körber, C. ∙ Sätzler, K. ...</div><div><strong>Serial section scanning electron microscopy (S3EM) on silicon wafers for ultra-structural volume imaging of cells and tissues</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e35172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0035172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84859813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22523574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0035172&amp;pmid=22523574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Reynolds, 1963</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Reynolds, E.S.</div><div><strong>The use of lead citrate at high pH as an electron-opaque stain in electron microscopy</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1963; <strong>17</strong>:208-212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.17.1.208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13986422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.17.1.208&amp;pmid=13986422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-16" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-16" aria-controls="sref88-16">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, wafers were incubated with a lead citrate solution for 8&nbsp;minutes at room temperature in a closed reaction tube to avoid contact with carbon dioxide. Subsequently, the wafers were dried at 60°C for 30&nbsp;min. The wafers were mounted on SEM pin stubs (0,5”/6mm length #G301F; agar scientific, Essex UK) with Acheson silver (#3692, Plano, Wetzlar, Germany).</div><div id="p0510" role="paragraph">For the correlation of GBCs with light microscopy and electron microscopy, overview images of single sections were acquired with scanning electron microscopy to locate the corresponding level in the light microscopy stack. Fluorescent GBCs were identified by the cell/nucleus morphology and the spatial arrangement of the DAPI signal. The exact light microscopic level was unequivocally identified with an overlay of EM and LM with TrakEM2 (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Cardona et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Cardona, A. ∙ Saalfeld, S. ∙ Schindelin, J. ...</div><div><strong>TrakEM2 software for neural circuit reconstruction</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e38011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0038011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862556352" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0038011&amp;pmid=22723842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0515" role="paragraph">For image acquisition, a LEO Gemini 1530 scanning electron microscope (Zeiss) and an Auriga (Zeiss) scanning electron microscope were used in combination with an ATLAS scan generator (Zeiss). The working distance was set to 2-3&nbsp;mm and the aperture to 20&nbsp;μm. The acceleration voltage was set to 2&nbsp;kV. For the 3D reconstructions, images were taken at the same spot in consecutive&nbsp;slices. A stack was created with extensions in z with steps of 70-280&nbsp;nm. Pixel sizes between 2 and 5&nbsp;nm, a pixel dwell time of 12.8&nbsp;μs and FoVs ranging from 100&nbsp;μm<sup>2</sup> to 2400&nbsp;μm<sup>2</sup> were used as imaging parameters. The acquisition time differed from 3 to 75&nbsp;minutes. For larger overview images, the FoV size was increased up to 1&nbsp;mm<sup>2</sup> with a pixel size of 30&nbsp;nm. Connected<sup>TUM/AC</sup> and unconnected<sup>TUM/AC</sup> cells were defined based on morphological connectivity. Direct synaptic contacts formed onto GBCs were counted as such if 2 out of 3 criteria were met: (1) synaptic vesicle cluster present, (2) synaptic cleft visible, (3) postsynaptic density (PSD) apparent. For the analysis of the synapses, EM-sections were reacquired at 2&nbsp;nm pixel size on each 70nm-section. Manual segmentation of the synaptic bouton was performed in TrakEM2 (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Cardona et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Cardona, A. ∙ Saalfeld, S. ∙ Schindelin, J. ...</div><div><strong>TrakEM2 software for neural circuit reconstruction</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e38011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0038011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862556352" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0038011&amp;pmid=22723842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Ultrastructural morphometric synaptic features were analyzed using custom-written scripts in ImageJ/Fiji. The diameter of the synaptic vesicles was measured at the maximum extent in the 2D-section. The boundaries of the cells/synaptic boutons were manually segmented and imported as area lists on consecutive EM-sections in TrakEM2 (<span class="dropBlock reference-citations"><a id="body-ref-sref10-4" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-4" aria-controls="sref10-4">Cardona et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref10-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Cardona, A. ∙ Saalfeld, S. ∙ Schindelin, J. ...</div><div><strong>TrakEM2 software for neural circuit reconstruction</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e38011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0038011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862556352" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0038011&amp;pmid=22723842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 3D visualization of the structures was performed in Fiji 3D Viewer (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Schmid et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Schmid, B. ∙ Schindelin, J. ∙ Cardona, A. ...</div><div><strong>A high-level 3D visualization API for Java and ImageJ</strong></div><div><em>BMC Bioinformatics.</em> 2010; <strong>11</strong>:274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-11-274" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77952447610" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20492697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-11-274&amp;pmid=20492697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Meshes were smoothed and the areas were interpolated in z-dimension. The smoothed model was exported to Autodesk 3DS MAX for further visualization.</div></section><section id="sec-4-5-24"><h4>Sample preparation and immunohistochemistry for confocal imaging of PDX mice</h4><div id="p0520" role="paragraph">Following deep anesthesia with isoflurane, xenograft mice were transcardially perfused with 4% (w/v) PFA dissolved in 1x phosphate buffered saline (PBS, Sigma or Gibco) after &gt;3&nbsp;months of tumor growth or showing of symptoms. Brains were collected and postfixed in 4% PFA overnight before transfer to a DAPI (1:10000) solution in PBS. Preparation of serial 80&nbsp;μm thick tissue sections was performed using a Leica VT000S vibratome.</div><div id="p0525" role="paragraph">Permeabilization of sections was performed with 5% (v/v) fetal bovine serum (FBS) and 1% (v/v) Triton X-100, shaking, for 2 hours at RT. Following, primary antibodies were applied in a 1% FBS and 0,2% Triton X-100 blocking solution and incubation at 4° Celsius overnight, shaking. Before applying appropriate secondary antibodies, 3x washing steps for 15&nbsp;min with a 2% FBS solution at RT was performed. Secondary antibodies were then coupled to chicken Alexa488 (Invitrogen) and rabbit Alexa647 (Invitrogen) with a dilution of 1:500 (same solution as the primary antibodies) and incubated for 3 hours. Before mounting, the sections were washed first 3x with a 1% FBS solution and then three times with PBS for 10&nbsp;minutes at RT. Mounting was performed with “SlowFade Gold” solution.</div><div id="p0530" role="paragraph">Images were acquired using a 63x immersion oil objective with a numerical aperture of 1.4 at a confocal laser-scanning microscope (TCS SP8, Leica microsystems and LSM710 ConfoCor3, Zeiss). The pixel size of the acquired images was 141&nbsp;nm or 220&nbsp;nm and the z step size was 300&nbsp;nm.</div></section><section id="sec-4-5-25"><h4>Validation of CREB phosphorylation in co-cultures</h4><div id="p0535" role="paragraph">To validate the phosphorylation of CREB (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Gonzalez and Montminy, 1989</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Gonzalez, G.A. ∙ Montminy, M.R.</div><div><strong>Cyclic-AMP stimulates somatostatin gene-transcription by phosphorylation of Creb at Serine-133</strong></div><div><em>Cell.</em> 1989; <strong>59</strong>:675-680</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2F0092-8674%2889%2990013-5&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2F0092-8674%2889%2990013-5&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0024445798" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2573431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2889%2990013-5&amp;pmid=2573431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in GBCs after neuronal stimulation, DIV5-12 GBC co-cultures were incubated with gabazine (Abcam, reaching a medium concentration of 4μM), CNQX (Abcam, reaching a medium concentration of 20μM) together with TTX (HelloBio, reaching a medium concentration of 1μM) or DMSO for 4 hours. Next, the coverslips were fixated in methanol at -20° Celsius for 5&nbsp;minutes and subsequently blocked in 5% (v/v) fetal bovine serum FBS solution for 10&nbsp;minutes at RT. Following,the primary antibody (anti-phospho-CREB, Millipore, 1:100 dilution) was applied in a 5% FBS solution and incubated for 1 hour at RT, shaking. Before applying appropriate secondary antibodies, 3x washing steps for 5&nbsp;min with a 5% FBS solution at RT were performed. 1:500 diluted appropriate secondary antibodies were applied in a 5% FBS solution and incubated for 1 hour at RT, shaking. Before mounting, the coverslips were washed first 3x with a 5% FBS solution and then three times with PBS for 5&nbsp;minutes at RT, shaking. Mounting was also performed with “SlowFade Gold” and DAPI (1:10000 dilution in PBS). Confocal images were acquired as described for PDX brain sections.</div></section><section id="sec-4-5-26"><h4>Sample preparation of human tissue for immunohistochemistry</h4><div id="p0540" role="paragraph">Human biopsy tissue was immersion fixed with 4% PFA overnight as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref88-17" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-17" aria-controls="sref88-17">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Afterwards, the tissue was washed in PBS and subsequently serially cut to 100&nbsp;μm thick tissue sections using a Leica VT000S vibratome and stored in PBS. The staining of the sections was done as described above. To visualize tumor cells a mouse anti-nestin antibody (Abcam, ab22035) was used. A secondary antibody coupled with Alexa 488 against mouse was subsequently used. Sections were stained with DAPI to simultaneously visualize nuclei. Images were acquired using a 40x immersion oil objective of a confocal laser scanning microscope (LSM710 ConfoCor3, Zeiss) with a numerical aperture of 1.3. Images were acquired with a pixel size of 350&nbsp;nm and a z step size of 390&nbsp;nm.</div></section><section id="sec-4-5-27"><h4>Morphometric analysis of somata and TM in human tissue and PDX mice</h4><div id="p0545" role="paragraph">3D rendering and analysis of the <i>in&nbsp;vivo</i> two photon microscopy images of PDX mice and confocal microscopy images of fixed human GB tissue were done using Arivis Vision 4D. Tumor cells were semiautomatically segmented using the ‘Magic Wand’ function. Somata and TMs were manually split and annotated using the ‘Splitting Tool’. Features, such as volume and surface area, were exported for further morphometric comparison.</div></section><section id="sec-4-5-28"><h4>Morphometric interaction analysis of somata and TMs with neurons</h4><div id="p0550" role="paragraph">First, each channel of the confocal images was segmented with a machine-learning based pixel classification using ilastik (<span class="dropBlock reference-citations"><a id="body-ref-sref2-4" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-4" aria-controls="sref2-4">Berg et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref2-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (831)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 3D Rendering and image analysis were done using Arivis Vision 4D. A custom pipeline was built from existing analysis features to automatically threshold and segment the beta-III-tubulin signal, a pan-neuronal marker (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Sullivan and Cleveland, 1986</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Sullivan, K.F. ∙ Cleveland, D.W.</div><div><strong>Identification of conserved isotype-defining variable region sequences for four vertebrate beta tubulin polypeptide classes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1986; <strong>83</strong>:4327-4331</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.83.12.4327" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3459176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.83.12.4327&amp;pmid=3459176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), from immunohistochemistry. The tumor cells of interest were segmented manually into somata and TMs, and annotated accordingly as described above. Afterwards, the direct overlap of tumor cell signal to neuron signal was automatically determined. Features such as volume and surface area of the detected contacts were exported for further comparisons.</div></section><section id="sec-4-5-29"><h4>Acute brain slice preparation</h4><div id="p0555" role="paragraph">Brain tumor-bearing mice were deeply anesthetized, rapidly decapitated, and the brain was removed in ice-cold slicing solution containing (in mM) 125 NaCl, 2.5 KCl, 25 NaHCO3, 1.25 NaH2PO4, 10 glucose, 75 sucrose, 0.5 CaCl2, 7 MgCl2, adjusted to ∼340 mOsm, and aerated with carbogen (5% CO2in O2). 300&nbsp;μm thick coronal slices were prepared on a vibratome (VT1200S; Leica) and stored in the same solution for 30&nbsp;min at 37°C. Afterwards, acute slices were transferred into a custom-made incubation system, where slices were slowly cooled down to 16°C and the circulating slicing solution was irradiated with UV light, to extend slice viability as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref88-18" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-18" aria-controls="sref88-18">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 30&nbsp;minutes before electrophysiological recordings, slices were warmed up to RT in artificial cerebrospinal fluid (aCSF) containing (in mM) 87 NaCl, 2.5 KCl, 25 NaHCO3, 1.25 NaH2PO4, 13 glucose, 2 CaCl2, 1 MgCl2, aerated with carbogen (5% CO2in O2).</div></section><section id="sec-4-5-30"><h4>Electrophysiology</h4><div id="p0560" role="paragraph">All experiments were performed on a Leica TCS SP5 Confocal Microscope equipped with a Luigs and Neumann stage and a water-immersion objective (20x, NA 1.0) as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref88-19" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-19" aria-controls="sref88-19">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GBCs were unequivocally identified by their endogenous EGFP or tdTomato-fluorescence, respectively, and subsequently targeted for whole-cell patch-clamp recordings. Acute brain slices and co-cultured plates were constantly perfused with artificial cerebrospinal fluid (aCSF), containing (in mM): 125 NaCl, 2.5 KCl, 25 NaHCO3, 1.25 NaH2PO4, 13 glucose, 2 CaCl2, 1 MgCl2. The aCSF was permanently aerated with carbogen and heated to 35°C. For the analysis of epileptiform, neuronal activity in co-cultures of neurons, astrocytes and GBCs, gabazine (4&nbsp;μM) was perfused in aCSF and subsequently neurons were targeted with whole-cell patch-clamp electrophysiology. For the analysis of anti-epileptic effects of perampanel (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Chen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Chen, C.Y. ∙ Matt, L. ∙ Hell, J.W. ...</div><div><strong>Perampanel inhibition of AMPA receptor currents in cultured hippocampal neurons</strong></div><div><em>PLoS One.</em> 2014; <strong>9</strong>:e108021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0108021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84907220225" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25229608/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0108021&amp;pmid=25229608" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on neuronal activity in co-culture, epilepsy induced by 4&nbsp;μM gabazine was compared to neuronal activity under 4&nbsp;μM gabazine together with 50&nbsp;μM perampanel.</div><div id="p0565" role="paragraph">Patch pipettes were fabricated from borosilicate capillaries (World Precision Instruments) and mounted on Luigs and Neumann-micromanipulators. Pipettes had resistances ranging from 3 to 7 MΩ when put in aCSF and filled with an internal solution containing (in mM): 130 K gluconate, 4 KCl, 5 Na-phosphocreatine, 10 HEPES, 0.1 EGTA, 4 Mg-ATP (pH set to 7.2).</div><div id="p0570" role="paragraph">An EPC10 amplifier (HEKA Elektronik, Lambrecht, Germany) controlled by Patchmaster software (HEKA) was used for acquisition of electrophysiological recordings. Currents were digitized at sampling rates of 25 kHz, filtered at 10 kHz, and Bessel-filtered at 2.9 kHz. The holding potential in the voltage clamp mode was set to -71&nbsp;mV to correct for the liquid junction potential. Recordings were performed without series resistance compensation.</div><div id="p0575" role="paragraph">Electrophysiological data were analyzed using custom written macros in MATLAB and Igor Pro (Wavemetric Inc., Lage Oswego, OR, USA).</div></section><section id="sec-4-5-31"><h4>Electrophysiological validation of neuronal channelrhodopsin activation</h4><div id="p0580" role="paragraph">To validate the ability of neuronal channelrhodopsin to reliably elicit action potentials, electrophysiological recordings of neurons in co-culture with astrocytes and GBCs were performed. For this purpose, neurons (DIV 7) were infected with 1&nbsp;μl AAV particles (pAAV-CaMKIIa-mCherry (AAV2)) prior the addition of GBCs to the co-culture model. Infected neurons were subsequently targeted with whole-cell patch-clamp recordings. Neurons were stimulated with 473&nbsp;nm laser light (Shanghai Laser&nbsp;&amp;&nbsp;Optics Century Co. Ltd, China) via an optical fiber (Laser Components, Item 3016347) with blue light at a frequency of 10&nbsp;Hz for 1 - 2 s. Currents produced by the opening of voltage-gated channels in the course of an action potential could be reliably elicited.</div></section><section id="sec-4-5-32"><h4>Perampanel treatment effects on GBC morphology over time</h4><div id="p0585" role="paragraph">Perampanel treatment in two xenograft models was performed as described before (<span class="dropBlock reference-citations"><a id="body-ref-sref88-20" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-20" aria-controls="sref88-20">Venkataramani et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Ten to 100&nbsp;days prior to imaging, S24 Twitch3A cells or BG5 GFP cells were injected according to the method described above. In&nbsp;vivo MPLSM image z-stacks were acquired at 3 time points (0, 7, 14d). Identical tumor microregions, identified by vascular and other landmarks, were recorded at all 3 time points in every mouse. Imaging parameters for stacks were set to a pixel size of 0.59&nbsp;μm and to a field of view with a volume of 607.28&nbsp;μm x 607.28&nbsp;μm x 240&nbsp;μm with 81 slices at an interval of 3&nbsp;μm. On day 0, food pellets of the mice in the treatment group were exchanged by pellets containing 320&nbsp;mg per kg perampanel (Eisai) and offered ad libitum throughout the time lapse over fourteen days. In a dose escalation study for the GBC line BG5, we applied several amounts of perampanel ranging from 320&nbsp;mg per kg to 960&nbsp;mg per kg. Mice in the control group were offered their normal food ad libitum.</div></section><section id="sec-4-5-33"><h4>Validation of the invasivity score</h4><div id="p0590" role="paragraph">For further validation of the invasivity score, we looked at the genes and gene sets that were enriched in the invasivity score. Here, we could clearly see that genes associated with physiological migration of neurons as well as genes associated with tumor cell motility could be found in the 20 genes with the highest invasivity score (<a id="crosref1980" href="#figs4">Figure&nbsp;S4</a>). Interestingly, the gene for the protein doublecortin (DCX) was among the high invasivity score genes. Doublecortin is a master regulator of physiological neuronal migration (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Gleeson et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Gleeson, J.G. ∙ Lin, P.T. ∙ Flanagan, L.A. ...</div><div><strong>Doublecortin is a microtubule-associated protein and is expressed widely by migrating neurons</strong></div><div><em>Neuron.</em> 1999; <strong>23</strong>:257-271</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2FS0896-6273%2800%2980778-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2FS0896-6273%2800%2980778-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0033152450" target="_blank" aria-label="Open Scopus in new tab">Scopus (1069)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10399933/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2800%2980778-3&amp;pmid=10399933" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) but has also been already described to be involved in GB migration (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Santra et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Santra, M. ∙ Santra, S. ∙ Roberts, C. ...</div><div><strong>Doublecortin induces mitotic microtubule catastrophe and inhibits glioma cell invasion</strong></div><div><em>J.&nbsp;Neurochem.</em> 2009; <strong>108</strong>:231-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1471-4159.2008.05758.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-57249105391" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19094064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1471-4159.2008.05758.x&amp;pmid=19094064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Another example is the gene MARCKSL1 encoding for the MARCKS-like protein 1. MARCKS-like protein 1 is predominately expressed in the immature brain. Genetic disruption resulted in defects of cranial neural tube closure and neural tube defects where mechanisms such as neuronal migration are critical. Furthermore, MARCKS-like protein 1 has been also reported to play a role for cancer cell migration in breast cancer and lung adenocarcinoma (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Liang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Liang, W. ∙ Gao, R. ∙ Yang, M. ...</div><div><strong>MARCKSL1 promotes the proliferation, migration and invasion of lung adenocarcinoma cells</strong></div><div><em>Oncol. Lett.</em> 2020; <strong>19</strong>:2272-2280</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32194726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=32194726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Salem et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Salem, O. ∙ Erdem, N. ∙ Jung, J. ...</div><div><strong>The highly expressed 5'isomiR of hsa-miR-140-3p contributes to the tumor-suppressive effects of miR-140 by reducing breast cancer proliferation and migration</strong></div><div><em>BMC Genomics.</em> 2016; <strong>17</strong>:566</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12864-016-2869-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84981294724" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27502506/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12864-016-2869-x&amp;pmid=27502506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This further substantiates the concept of parallels between neuronal progenitor migration and tumor cell invasion. In addition, several genes associated with high invasivity have been shown to be involved in either brain tumor cell motility (e.g. BCAN; <span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Zhang et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Zhang, H. ∙ Kelly, G. ∙ Zerillo, C. ...</div><div><strong>Expression of a cleaved brain-specific extracellular matrix protein mediates glioma cell invasion in&nbsp;vivo</strong></div><div><em>J.&nbsp;Neurosci.</em> 1998; <strong>18</strong>:2370-2376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.18-07-02370.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9502798/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.18-07-02370.1998&amp;pmid=9502798" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or tumor cell motility across various entities (e.g. MALAT1 [<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Tano et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Tano, K. ∙ Mizuno, R. ∙ Okada, T. ...</div><div><strong>MALAT-1 enhances cell motility of lung adenocarcinoma cells by influencing the expression of motility-related genes</strong></div><div><em>FEBS Lett.</em> 2010; <strong>584</strong>:4575-4580</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.febslet.2010.10.008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-78249258719" target="_blank" aria-label="Open Scopus in new tab">Scopus (383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20937273/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.febslet.2010.10.008&amp;pmid=20937273" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>], ETV1 [<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Rahim et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Rahim, S. ∙ Beauchamp, E.M. ∙ Kong, Y. ...</div><div><strong>YK-4-279 inhibits ERG and ETV1 mediated prostate cancer cell invasion</strong></div><div><em>PLoS One.</em> 2011; <strong>6</strong>:e19343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0019343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-79955779109" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21559405/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0019343&amp;pmid=21559405" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>], TNK2 [<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">van der Horst et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>van der Horst, E.H. ∙ Degenhardt, Y.Y. ∙ Strelow, A. ...</div><div><strong>Metastatic properties and genomic amplification of the tyrosine kinase gene ACK1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15901-15906</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0508014102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-27644436744" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16247015/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0508014102&amp;pmid=16247015" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]).</div><div id="p0595" role="paragraph">Further analysis revealed that the gene set for the invasion score is enriched for GO terms involving migration, motility and movement (<a id="crosref2025" href="#figs4">Figure&nbsp;S4</a>).</div></section><section id="sec-4-5-34"><h4>Lévy-like walk analysis of GB motility</h4><div id="p0600" role="paragraph">In the past decades, Lévy walks have been used to analyze a wide range of mechanisms, e.g., in finance, biology, ecology or physics (<span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">Zaburdaev et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Zaburdaev, V. ∙ Denisov, S. ∙ Klafter, J.</div><div><strong>Levy walks</strong></div><div><em>Rev. Mod. Phys.</em> 2015; <strong>87</strong>:483-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/RevModPhys.87.483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84937578864" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FRevModPhys.87.483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, it has been controversely discussed if Lévy walks can be associated with an efficient search pattern in animal foraging (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Sims et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Sims, D.W. ∙ Righton, D. ∙ Pitchford, J.W.</div><div><strong>Minimizing errors in identifying Levy flight behaviour of organisms</strong></div><div><em>J.&nbsp;Anim. Ecol.</em> 2007; <strong>76</strong>:222-229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2656.2006.01208.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-33846950846" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17302829/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2656.2006.01208.x&amp;pmid=17302829" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103-3" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-3" aria-controls="sref103-3">Zaburdaev et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref103-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Zaburdaev, V. ∙ Denisov, S. ∙ Klafter, J.</div><div><strong>Levy walks</strong></div><div><em>Rev. Mod. Phys.</em> 2015; <strong>87</strong>:483-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/RevModPhys.87.483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84937578864" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FRevModPhys.87.483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recently, this movement pattern has been hypothesized to be a characteristic property of metastatic cancer cells (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Huda et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Huda, S. ∙ Weigelin, B. ∙ Wolf, K. ...</div><div><strong>Levy-like movement patterns of metastatic cancer cells revealed in microfabricated systems and implicated in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06563-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85055771401" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30382086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06563-w&amp;pmid=30382086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the statistical analysis of stochastic processes, the differences between processes in discrete time and in continuous time should be carefully considered. While it can be useful to use a discrete index set to account for the finite number of sampling times and disregard the behavior in between these observations, there is no reason for most biological settings to assume that the actual process does not move continuously in time. This can lead to profound consequences. For example, it is sometimes attempted to determine the distribution of a process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper X"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> by estimating the exponential parameter of the mean-squared displacement (MSD) <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left angle bracket upper X left parenthesis t right parenthesis right angle bracket tilde t Superscript alpha" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (9 0 (7 1 6 (5 2 3 4)) 8) 10 (13 11 12))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="9,13" data-semantic-content="10" data-semantic-owns="9 10 13" data-semantic-speech="left angle bracket upper X left parenthesis t right parenthesis right angle bracket tilde t Superscript alpha"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="0,8" data-semantic-parent="14" data-semantic-owns="0 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c27E8">⟨</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="1,5" data-semantic-content="6,1" data-semantic-parent="9" data-semantic-owns="1 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="appl"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="2,4" data-semantic-parent="7" data-semantic-owns="2 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c27E9">⟩</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="10" data-semantic-parent="14" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="14" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha greater than 0" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="alpha greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t right arrow normal infinity" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="arrow" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t right arrow normal infinity"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,→"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-font="normal" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c221E">∞</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, which, even within the class of Gaussian processes, does not uniquely determine the process (<span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">Gneiting and Schlather, 2004</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Gneiting, T. ∙ Schlather, M.</div><div><strong>Stochastic models that separate fractal dimension and the Hurst effect</strong></div><div><em>SIAM Rev.</em> 2004; <strong>46</strong>:269-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1137/S0036144501394387" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-3142710385" target="_blank" aria-label="Open Scopus in new tab">Scopus (316)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1137%2FS0036144501394387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103-4" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-4" aria-controls="sref103-4">Zaburdaev et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref103-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Zaburdaev, V. ∙ Denisov, S. ∙ Klafter, J.</div><div><strong>Levy walks</strong></div><div><em>Rev. Mod. Phys.</em> 2015; <strong>87</strong>:483-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/RevModPhys.87.483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84937578864" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FRevModPhys.87.483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We accommodate the two model settings by introducing a time-continuous Lévy walk, which also allows the existence of second moments. In general, it should be noted that the sole analysis of the MSD as a metric to determine the distribution of a stochastic process is insufficient and has significant weaknesses. While it can be used to decide if the process diffuses anomalously (i.e., slower of faster than a Brownian motion) and is a useful tool to analyze the scaling behavior of self-similar processes, the scaling parameter <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="alpha"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> in general does not determine the behavior of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper X"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> on short time scales and for Lévy walks and for stable Lévy processes with stability index smaller than 2 the MSD is not even finite.</div><div id="p0605" role="paragraph">Moreover, the MSD relies on a sufficient number of data points at each time step. Specifically, due to the complexity ofdata collection, our dataset comprises a small number of tracks which persist through the whole time horizon. This is a possible explanation for the deteriorating fit of the log-log-linear regression for later stages in the experiment (see <a id="crosref2060" href="#figs6">Figures&nbsp;S6</a>D–S6K). Furthermore, it is sometimes criticized that, from a visual point of view, the log-log scale makes relationships seem more linear (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Sims et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Sims, D.W. ∙ Righton, D. ∙ Pitchford, J.W.</div><div><strong>Minimizing errors in identifying Levy flight behaviour of organisms</strong></div><div><em>J.&nbsp;Anim. Ecol.</em> 2007; <strong>76</strong>:222-229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2656.2006.01208.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-33846950846" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17302829/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2656.2006.01208.x&amp;pmid=17302829" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0610" role="paragraph">As an alternative, we consider the case of an anomalously diffusive Lévy process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper X Subscript t Baseline right parenthesis Subscript t greater than or equals 0" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (5 0 (3 1 2) 4) (9 6 7 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="5,9" data-semantic-owns="5 9" data-semantic-speech="left parenthesis upper X Subscript t Baseline right parenthesis Subscript t greater than or equals 0"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-parent="10" data-semantic-owns="0 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.283em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> with a stability parameter <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha greater than 0" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="alpha greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> which moves through time with an acceleration factor <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c greater than 0" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="c greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>. Then,<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="double struck upper E left bracket left parenthesis upper X Subscript c t Baseline minus upper X 0 right parenthesis squared right bracket equals left parenthesis c t right parenthesis Superscript alpha Baseline period" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="(34 (33 (32 0 31 (19 1 (17 (15 2 (13 (8 3 (7 4 6 5)) 9 (12 10 11)) 14) 16) 18)) 20 (29 (27 21 (25 22 24 23) 26) 28)) 30)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="34" data-semantic-children="33,30" data-semantic-content="30" data-semantic-owns="33 30" data-semantic-speech="double struck upper E left bracket left parenthesis upper X Subscript c t Baseline minus upper X 0 right parenthesis squared right bracket equals left parenthesis c t right parenthesis Superscript alpha Baseline period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="33" data-semantic-children="32,29" data-semantic-content="20" data-semantic-parent="34" data-semantic-owns="32 20 29"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="32" data-semantic-children="0,19" data-semantic-content="31,0" data-semantic-parent="33" data-semantic-owns="0 31 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="32" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="19" data-semantic-children="17" data-semantic-content="1,18" data-semantic-parent="32" data-semantic-owns="1 17 18"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="19" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="19" data-semantic-owns="15 16"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="2,14" data-semantic-parent="17" data-semantic-owns="2 13 14"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-children="8,12" data-semantic-content="9" data-semantic-parent="15" data-semantic-owns="8 9 12"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="13" data-semantic-owns="3 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="9" data-semantic-parent="13" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="13" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="18" data-semantic-parent="19" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="20" data-semantic-parent="33" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="33" data-semantic-owns="27 28"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="27" data-semantic-children="25" data-semantic-content="21,26" data-semantic-parent="29" data-semantic-owns="21 25 26"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="21" data-semantic-parent="27" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="25" data-semantic-children="22,23" data-semantic-content="24" data-semantic-parent="27" data-semantic-owns="22 24 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="26" data-semantic-parent="27" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="30" data-semantic-parent="34" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0615" role="paragraph">For equidistant observation times <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t 0 equals 0 comma t 1 equals upper Delta comma period period period comma t Subscript upper N Baseline equals upper N upper Delta" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="(25 (20 (2 0 1) 3 4) 5 (21 (8 6 7) 9 10) 11 12 13 (24 (16 14 15) 17 (23 18 22 19)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="25" data-semantic-children="20,5,21,11,12,13,24" data-semantic-content="5,11,13" data-semantic-owns="20 5 21 11 12 13 24" data-semantic-speech="t 0 equals 0 comma t 1 equals upper Delta comma period period period comma t Subscript upper N Baseline equals upper N upper Delta"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="20" data-semantic-children="2,4" data-semantic-content="3" data-semantic-parent="25" data-semantic-owns="2 3 4"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="20" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="20" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="20"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="21" data-semantic-children="8,10" data-semantic-content="9" data-semantic-parent="25" data-semantic-owns="8 9 10"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="21" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="21" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="21"><mjx-c class="mjx-c1D6E5">𝛥</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="25"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="13" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="24" data-semantic-children="16,23" data-semantic-content="17" data-semantic-parent="25" data-semantic-owns="16 17 23"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="24" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="17" data-semantic-parent="24" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="23" data-semantic-children="18,19" data-semantic-content="22" data-semantic-parent="24" data-semantic-owns="18 22 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="23"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="23"><mjx-c class="mjx-c1D6E5">𝛥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N element of double struck upper N comma upper Delta greater than 0 comma" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (8 0 1 2) 3 (9 4 5 6) 7)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="8,3,9,7" data-semantic-content="3,7" data-semantic-owns="8 3 9 7" data-semantic-speech="upper N element of double struck upper N comma upper Delta greater than 0 comma"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="8" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="10" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="8"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="numbersetletter" data-semantic-font="double-struck" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c2115 MM-DS">ℕ</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="9" data-semantic-children="4,6" data-semantic-content="5" data-semantic-parent="10" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D6E5">𝛥</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> it then holds that<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartFraction 1 Over n EndFraction double struck upper E left bracket sigma summation Underscript i equals 1 Overscript n Endscripts left parenthesis upper X Subscript t Sub Subscript i Subscript Baseline minus upper X Subscript t Sub Subscript i minus 1 Subscript Baseline right parenthesis squared right bracket equals upper Delta Superscript alpha Baseline comma" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="(45 (44 (43 (2 0 1) 42 (41 3 40 (34 4 (32 (11 5 (9 6 7 8) 10) (31 (29 12 (27 (17 13 (16 14 15)) 18 (26 19 (25 20 (24 21 22 23)))) 28) 30)) 33))) 35 (38 36 37)) 39)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="45" data-semantic-children="44,39" data-semantic-content="39" data-semantic-owns="44 39" data-semantic-speech="StartFraction 1 Over n EndFraction double struck upper E left bracket sigma summation Underscript i equals 1 Overscript n Endscripts left parenthesis upper X Subscript t Sub Subscript i Subscript Baseline minus upper X Subscript t Sub Subscript i minus 1 Subscript Baseline right parenthesis squared right bracket equals upper Delta Superscript alpha Baseline comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="44" data-semantic-children="43,38" data-semantic-content="35" data-semantic-parent="45" data-semantic-owns="43 35 38"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="43" data-semantic-children="2,41" data-semantic-content="42" data-semantic-parent="44" data-semantic-owns="2 42 41"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="43" data-semantic-owns="0 1"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="42" data-semantic-parent="43" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="41" data-semantic-children="3,34" data-semantic-content="40,3" data-semantic-parent="43" data-semantic-owns="3 40 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="41" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="40" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="34" data-semantic-children="32" data-semantic-content="4,33" data-semantic-parent="41" data-semantic-owns="4 32 33"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="32" data-semantic-children="11,31" data-semantic-content="5" data-semantic-parent="34" data-semantic-owns="11 31"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="11" data-semantic-children="5,9,10" data-semantic-parent="32" data-semantic-owns="5 9 10" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.284em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="11" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="32" data-semantic-owns="29 30"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="29" data-semantic-children="27" data-semantic-content="12,28" data-semantic-parent="31" data-semantic-owns="12 27 28"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="27" data-semantic-children="17,26" data-semantic-content="18" data-semantic-parent="29" data-semantic-owns="17 18 26"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="13,16" data-semantic-parent="27" data-semantic-owns="13 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="17"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="17" data-semantic-owns="14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="18" data-semantic-parent="27" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="19,25" data-semantic-parent="27" data-semantic-owns="19 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="26"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="20,24" data-semantic-parent="26" data-semantic-owns="20 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="25"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-children="21,23" data-semantic-content="22" data-semantic-parent="25" data-semantic-owns="21 22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="24"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-parent="24" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="28" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="33" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="35" data-semantic-parent="44" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="38" data-semantic-children="36,37" data-semantic-parent="44" data-semantic-owns="36 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c1D6E5">𝛥</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="39" data-semantic-parent="45" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n equals 1 comma period period period comma upper N" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (7 0 1 2) 3 4 5 6)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="8" data-semantic-children="7,3,4,5,6" data-semantic-content="3,5" data-semantic-owns="7 3 4 5 6" data-semantic-speech="n equals 1 comma period period period comma upper N"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="7" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="8" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="7"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. If the underlying process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper X"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is indeed a Lévy process, we get for independent trajectories <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X Superscript left parenthesis 1 right parenthesis Baseline comma period period period comma upper X Superscript left parenthesis upper M right parenthesis" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 (5 0 (4 1 2 3)) 6 7 8 (14 9 (13 10 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="15" data-semantic-children="5,6,7,8,14" data-semantic-content="6,8" data-semantic-owns="5 6 7 8 14" data-semantic-speech="upper X Superscript left parenthesis 1 right parenthesis Baseline comma period period period comma upper X Superscript left parenthesis upper M right parenthesis"><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="15" data-semantic-owns="0 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em; margin-left: 0.051em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="15" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="15"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="8" data-semantic-parent="15" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="9,13" data-semantic-parent="15" data-semantic-owns="9 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="14"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em; margin-left: 0.051em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="13" data-semantic-children="11" data-semantic-content="10,12" data-semantic-parent="14" data-semantic-owns="10 11 12"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="10" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="12" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper X"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> through the law of large numbers<div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartFraction 1 Over upper M EndFraction sigma summation Underscript k equals 1 Overscript upper M Endscripts StartFraction 1 Over upper N EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts left parenthesis upper X Subscript t Sub Subscript i Subscript Superscript left parenthesis k right parenthesis Baseline minus upper X Subscript t Sub Subscript i minus 1 Subscript Superscript left parenthesis k right parenthesis Baseline right parenthesis squared almost equals upper Delta Superscript alpha Baseline comma" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="(62 (61 (60 (2 0 1) 59 (58 (9 3 (7 4 5 6) 8) (57 (12 10 11) 56 (55 (19 13 (17 14 15 16) 18) (49 (47 20 (45 (30 (29 21 (24 22 23)) (28 25 26 27)) 31 (44 (43 32 (38 33 (37 34 35 36))) (42 39 40 41))) 46) 48))))) 50 (53 51 52)) 54)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="62" data-semantic-children="61,54" data-semantic-content="54" data-semantic-owns="61 54" data-semantic-speech="StartFraction 1 Over upper M EndFraction sigma summation Underscript k equals 1 Overscript upper M Endscripts StartFraction 1 Over upper N EndFraction sigma summation Underscript i equals 1 Overscript upper N Endscripts left parenthesis upper X Subscript t Sub Subscript i Subscript Superscript left parenthesis k right parenthesis Baseline minus upper X Subscript t Sub Subscript i minus 1 Subscript Superscript left parenthesis k right parenthesis Baseline right parenthesis squared almost equals upper Delta Superscript alpha Baseline comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="61" data-semantic-children="60,53" data-semantic-content="50" data-semantic-parent="62" data-semantic-owns="60 50 53"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="60" data-semantic-children="2,58" data-semantic-content="59" data-semantic-parent="61" data-semantic-owns="2 59 58"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="60" data-semantic-owns="0 1"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="60" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="58" data-semantic-children="9,57" data-semantic-content="3" data-semantic-parent="60" data-semantic-owns="9 57"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="9" data-semantic-children="3,7,8" data-semantic-parent="58" data-semantic-owns="3 7 8" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.272em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="7" data-semantic-children="4,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="57" data-semantic-children="12,55" data-semantic-content="56" data-semantic-parent="58" data-semantic-owns="12 56 55"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="57" data-semantic-owns="10 11"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="56" data-semantic-parent="57" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="55" data-semantic-children="19,49" data-semantic-content="13" data-semantic-parent="57" data-semantic-owns="19 49"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="19" data-semantic-children="13,17,18" data-semantic-parent="55" data-semantic-owns="13 17 18" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="13" data-semantic-parent="19" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.291em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="17" data-semantic-children="14,16" data-semantic-content="15" data-semantic-parent="19" data-semantic-owns="14 15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="15" data-semantic-parent="17" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="49" data-semantic-children="47,48" data-semantic-parent="55" data-semantic-owns="47 48"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="47" data-semantic-children="45" data-semantic-content="20,46" data-semantic-parent="49" data-semantic-owns="20 45 46"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="20" data-semantic-parent="47" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S4">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="45" data-semantic-children="30,44" data-semantic-content="31" data-semantic-parent="47" data-semantic-owns="30 31 44"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="30" data-semantic-children="21,24,28" data-semantic-parent="45" data-semantic-collapsed="(30 (29 21 24) 28)" data-semantic-owns="21 24 28"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="30"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.269em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="28" data-semantic-children="26" data-semantic-content="25,27" data-semantic-parent="30" data-semantic-owns="25 26 27" style="margin-left: 0.105em;"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="25" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="27" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="22,23" data-semantic-parent="30" data-semantic-owns="22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-script></mjx-msubsup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="31" data-semantic-parent="45" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msubsup space="3" data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="44" data-semantic-children="32,38,42" data-semantic-parent="45" data-semantic-collapsed="(44 (43 32 38) 42)" data-semantic-owns="32 38 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="44"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.269em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="42" data-semantic-children="40" data-semantic-content="39,41" data-semantic-parent="44" data-semantic-owns="39 40 41" style="margin-left: 0.105em;"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="39" data-semantic-parent="42" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="42"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="41" data-semantic-parent="42" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="38" data-semantic-children="33,37" data-semantic-parent="43" data-semantic-owns="33 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="44"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="37" data-semantic-children="34,36" data-semantic-content="35" data-semantic-parent="38" data-semantic-owns="34 35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="37"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="35" data-semantic-parent="37" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="46" data-semantic-parent="47" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S4">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.873em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="49"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="50" data-semantic-parent="61" data-semantic-operator="relseq,≈"><mjx-c class="mjx-c2248">≈</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="53" data-semantic-children="51,52" data-semantic-parent="61" data-semantic-owns="51 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="53"><mjx-c class="mjx-c1D6E5">𝛥</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="53"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="54" data-semantic-parent="62" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>with a higher number of independent data points involved in the calculation. This quantity can be interpreted as an approximation of the mean-squared variation. By plugging in the MSD estimates for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="c"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="alpha"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, we can compare the two approaches. We see in see <a id="crosref2070" href="#figs6">Figures&nbsp;S6</a>D–S6K that the estimation is rather rough and the mean-squared variation does not converge, indicating that the assumption of independent, stationary increments might be violated.</div><div id="p0620" role="paragraph">Due to a lack of observable drivers of TM growth, we do not specify a stochastic process to model the expansion of TMs. In particular, there is no physiological justification to assume self-similarity. We observe that there is little to no movement in the TMs for a significant fraction of the observed sampling times of the process, interspersed with time frames of large growth. Since the empirical step size distribution is not atomless, a maximum-likelihood approach would yield flawed results. Instead, we rely on nonparametric methods from the field of extreme value theory to capture the behavior of the process and use estimators for the generalized extreme value index to demonstrate that the flight distribution between sampling times is indeed heavy-tailed. We depict the Hill plot generated by the Hill estimator (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Bruce, 1975</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Bruce, M.H.</div><div><strong>A simple general approach to inference about the tail of a distribution</strong></div><div><em>Ann. Stat.</em> 1975; <strong>3</strong>:1163-1174</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.H.BruceA+simple+general+approach+to+inference+about+the+tail+of+a+distributionAnn.+Stat.3197511631174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)<div id="ufd4" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper H right parenthesis Baseline equals StartFraction 1 Over k EndFraction sigma summation Underscript i equals 0 Overscript k minus 1 Endscripts left parenthesis log left parenthesis upper X Subscript left parenthesis n minus i right parenthesis colon n Baseline right parenthesis minus log left parenthesis upper X Subscript left parenthesis n minus k right parenthesis colon n Baseline right parenthesis right parenthesis comma k equals 1 comma period period period comma n comma" tabindex="0" ctxtmenu_counter="20" style="font-size: 130%;"><mjx-math data-semantic-structure="(85 (81 (12 (11 (2 0 1) (6 3 4 5)) (10 7 8 9)) 13 (80 (16 14 15) 79 (78 (26 17 (21 18 19 20) (25 22 23 24)) (67 27 (65 (64 28 63 (43 29 (41 30 (40 (37 31 (35 32 33 34) 36) 38 39)) 42)) 44 (62 45 61 (60 46 (58 47 (57 (54 48 (52 49 50 51) 53) 55 56)) 59))) 66)))) 68 (84 69 (82 70 71 72)) 73 74 75 76 77)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="85" data-semantic-children="81,68,84,73,74,75,76,77" data-semantic-content="68,73,75,77" data-semantic-owns="81 68 84 73 74 75 76 77" data-semantic-speech="ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper H right parenthesis Baseline equals StartFraction 1 Over k EndFraction sigma summation Underscript i equals 0 Overscript k minus 1 Endscripts left parenthesis log left parenthesis upper X Subscript left parenthesis n minus i right parenthesis colon n Baseline right parenthesis minus log left parenthesis upper X Subscript left parenthesis n minus k right parenthesis colon n Baseline right parenthesis right parenthesis comma k equals 1 comma period period period comma n comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="81" data-semantic-children="12,80" data-semantic-content="13" data-semantic-parent="85" data-semantic-owns="12 13 80"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="12" data-semantic-children="2,6,10" data-semantic-parent="81" data-semantic-collapsed="(12 (11 2 6) 10)" data-semantic-owns="2 6 10"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="12" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.523em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.317em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="7,9" data-semantic-parent="12" data-semantic-owns="7 8 9" style="margin-left: 0.108em;"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="3,4,5" data-semantic-content="4" data-semantic-parent="12" data-semantic-owns="3 4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="13" data-semantic-parent="81" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="80" data-semantic-children="16,78" data-semantic-content="79" data-semantic-parent="81" data-semantic-owns="16 79 78"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="80" data-semantic-owns="14 15"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="79" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="78" data-semantic-children="26,67" data-semantic-content="17" data-semantic-parent="80" data-semantic-owns="26 67"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="26" data-semantic-children="17,21,25" data-semantic-parent="78" data-semantic-owns="17 21 25" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="17" data-semantic-parent="26" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="25" data-semantic-children="22,24" data-semantic-content="23" data-semantic-parent="26" data-semantic-owns="22 23 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="23" data-semantic-parent="25" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-spacer style="margin-top: 0.233em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="21" data-semantic-children="18,20" data-semantic-content="19" data-semantic-parent="26" data-semantic-owns="18 19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="21"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="19" data-semantic-parent="21" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="67" data-semantic-children="65" data-semantic-content="27,66" data-semantic-parent="78" data-semantic-owns="27 65 66"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="27" data-semantic-parent="67" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="65" data-semantic-children="64,62" data-semantic-content="44" data-semantic-parent="67" data-semantic-owns="64 44 62"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="64" data-semantic-children="28,43" data-semantic-content="63,28" data-semantic-parent="65" data-semantic-owns="28 63 43"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="28" data-semantic-parent="64" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="63" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="43" data-semantic-children="41" data-semantic-content="29,42" data-semantic-parent="64" data-semantic-owns="29 41 42"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="29" data-semantic-parent="43" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="41" data-semantic-children="30,40" data-semantic-parent="43" data-semantic-owns="30 40"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="41"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.175em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="40" data-semantic-children="37,38,39" data-semantic-content="38" data-semantic-parent="41" data-semantic-owns="37 38 39"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="37" data-semantic-children="35" data-semantic-content="31,36" data-semantic-parent="40" data-semantic-owns="31 35 36"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="31" data-semantic-parent="37" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="35" data-semantic-children="32,34" data-semantic-content="33" data-semantic-parent="37" data-semantic-owns="32 33 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="35"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="33" data-semantic-parent="35" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="35"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="36" data-semantic-parent="37" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="38" data-semantic-parent="40" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="40"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="42" data-semantic-parent="43" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="44" data-semantic-parent="65" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="62" data-semantic-children="45,60" data-semantic-content="61,45" data-semantic-parent="65" data-semantic-owns="45 61 60"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="45" data-semantic-parent="62" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="61" data-semantic-parent="62" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="60" data-semantic-children="58" data-semantic-content="46,59" data-semantic-parent="62" data-semantic-owns="46 58 59"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="46" data-semantic-parent="60" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="58" data-semantic-children="47,57" data-semantic-parent="60" data-semantic-owns="47 57"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="58"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.175em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="57" data-semantic-children="54,55,56" data-semantic-content="55" data-semantic-parent="58" data-semantic-owns="54 55 56"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="54" data-semantic-children="52" data-semantic-content="48,53" data-semantic-parent="57" data-semantic-owns="48 52 53"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="48" data-semantic-parent="54" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="52" data-semantic-children="49,51" data-semantic-content="50" data-semantic-parent="54" data-semantic-owns="49 50 51"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="52"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="50" data-semantic-parent="52" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="52"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="53" data-semantic-parent="54" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="55" data-semantic-parent="57" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="57"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="59" data-semantic-parent="60" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="66" data-semantic-parent="67" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="68" data-semantic-parent="85" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="84" data-semantic-children="69,82" data-semantic-parent="85" data-semantic-collapsed="(84 (c 83) 69 82)" data-semantic-owns="69 82"><mjx-mspace data-semantic-type="text" data-semantic-role="space" data-semantic-annotation="clearspeak:unit" data-semantic-id="69" data-semantic-parent="84" style="width: 1em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="82" data-semantic-children="70,72" data-semantic-content="71" data-semantic-parent="84" data-semantic-owns="70 71 72"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="82"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="71" data-semantic-parent="82" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="82"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="73" data-semantic-parent="85" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="85"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="75" data-semantic-parent="85" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="85"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="77" data-semantic-parent="85" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>and the Dekkers-Einmal-de Haan plot generated by the corresponding estimator (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Arnold and Laurens De, 1989</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Arnold, L.M.D. ∙ Laurens De, H.</div><div><strong>On the estimation of the extreme-value index and large quantile estimation</strong></div><div><em>Ann. Stat.</em> 1989; <strong>17</strong>:1795-1832</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.M.D.ArnoldH.Laurens+DeOn+the+estimation+of+the+extreme-value+index+and+large+quantile+estimationAnn.+Stat.17198917951832" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)<div id="ufd5" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper D upper E d upper H Baseline equals ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper H right parenthesis Baseline plus 1 minus one half left parenthesis 1 minus StartStartFraction left parenthesis ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper H right parenthesis Baseline right parenthesis squared OverOver StartFraction 1 Over k EndFraction sigma summation Underscript i equals 0 Overscript k minus 1 Endscripts left parenthesis log left parenthesis upper X Subscript left parenthesis n minus i right parenthesis colon n Baseline right parenthesis minus log left parenthesis upper X Subscript left parenthesis n minus k right parenthesis colon n Baseline right parenthesis right parenthesis squared EndEndFraction right parenthesis Superscript negative 1 Baseline comma k equals 1 comma period period period comma n comma" tabindex="0" ctxtmenu_counter="21" style="font-size: 130%;"><mjx-math data-semantic-structure="(145 (141 (18 (17 (2 0 1) (6 3 4 5)) (16 7 (15 8 12 9 13 10 14 11))) 19 (140 (137 (32 (31 (22 20 21) (26 23 24 25)) (30 27 28 29)) 33 34) 35 (139 (38 36 37) 138 (126 (122 39 (120 40 41 (119 (59 (57 42 (55 (54 (45 43 44) (49 46 47 48)) (53 50 51 52)) 56) 58) (118 (62 60 61) 117 (116 (72 63 (67 64 65 66) (71 68 69 70)) (115 (113 73 (111 (110 74 109 (89 75 (87 76 (86 (83 77 (81 78 79 80) 82) 84 85)) 88)) 90 (108 91 107 (106 92 (104 93 (103 (100 94 (98 95 96 97) 99) 101 102)) 105))) 112) 114))))) 121) (125 123 124))))) 127 (144 128 (142 129 130 131)) 132 133 134 135 136)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="145" data-semantic-children="141,127,144,132,133,134,135,136" data-semantic-content="127,132,134,136" data-semantic-owns="141 127 144 132 133 134 135 136" data-semantic-speech="ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper D upper E d upper H Baseline equals ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper H right parenthesis Baseline plus 1 minus one half left parenthesis 1 minus StartStartFraction left parenthesis ModifyingAbove gamma With ˆ Subscript k comma n Superscript left parenthesis upper H right parenthesis Baseline right parenthesis squared OverOver StartFraction 1 Over k EndFraction sigma summation Underscript i equals 0 Overscript k minus 1 Endscripts left parenthesis log left parenthesis upper X Subscript left parenthesis n minus i right parenthesis colon n Baseline right parenthesis minus log left parenthesis upper X Subscript left parenthesis n minus k right parenthesis colon n Baseline right parenthesis right parenthesis squared EndEndFraction right parenthesis Superscript negative 1 Baseline comma k equals 1 comma period period period comma n comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="141" data-semantic-children="18,140" data-semantic-content="19" data-semantic-parent="145" data-semantic-owns="18 19 140"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="18" data-semantic-children="2,6,16" data-semantic-parent="141" data-semantic-collapsed="(18 (17 2 6) 16)" data-semantic-owns="2 6 16"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="18" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.523em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.317em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="16" data-semantic-children="7,15" data-semantic-content="7" data-semantic-parent="18" data-semantic-owns="7 15" style="margin-left: 0.108em;"><mjx-mo data-semantic-type="punctuation" data-semantic-role="openfence" data-semantic-id="7" data-semantic-parent="18" data-semantic-operator="punctuated"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-children="8,9,10,11" data-semantic-content="12,13,14" data-semantic-parent="16" data-semantic-owns="8 12 9 13 10 14 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="15"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="15"><mjx-c class="mjx-c1D438">𝐸</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="15"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="15"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="3,4,5" data-semantic-content="4" data-semantic-parent="18" data-semantic-owns="3 4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="19" data-semantic-parent="141" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="140" data-semantic-children="137,139" data-semantic-content="35" data-semantic-parent="141" data-semantic-owns="137 35 139"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="137" data-semantic-children="32,34" data-semantic-content="33" data-semantic-parent="140" data-semantic-owns="32 33 34"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="32" data-semantic-children="22,26,30" data-semantic-parent="137" data-semantic-collapsed="(32 (31 22 26) 30)" data-semantic-owns="22 26 30"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="22" data-semantic-children="20,21" data-semantic-parent="32" data-semantic-owns="20 21"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.523em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="21" data-semantic-parent="22" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="22"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.317em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="27,29" data-semantic-parent="32" data-semantic-owns="27 28 29" style="margin-left: 0.108em;"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="27" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="30"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="26" data-semantic-children="23,24,25" data-semantic-content="24" data-semantic-parent="32" data-semantic-owns="23 24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="26"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="24" data-semantic-parent="26" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="33" data-semantic-parent="137" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="137"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="35" data-semantic-parent="140" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="139" data-semantic-children="38,126" data-semantic-content="138" data-semantic-parent="140" data-semantic-owns="38 138 126"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="vulgar" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-children="36,37" data-semantic-parent="139" data-semantic-owns="36 37"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="138" data-semantic-parent="139" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="126" data-semantic-children="122,125" data-semantic-parent="139" data-semantic-owns="122 125"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="122" data-semantic-children="120" data-semantic-content="39,121" data-semantic-parent="126" data-semantic-owns="39 120 121"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="39" data-semantic-parent="122" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S7">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="120" data-semantic-children="40,119" data-semantic-content="41" data-semantic-parent="122" data-semantic-owns="40 41 119"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="120"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="41" data-semantic-parent="120" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mfrac space="3" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="119" data-semantic-children="59,118" data-semantic-parent="120" data-semantic-owns="59 118"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msup size="s" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="59" data-semantic-children="57,58" data-semantic-parent="119" data-semantic-owns="57 58"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="57" data-semantic-children="55" data-semantic-content="42,56" data-semantic-parent="59" data-semantic-owns="42 55 56"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="42" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S4">(</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="55" data-semantic-children="45,49,53" data-semantic-parent="57" data-semantic-collapsed="(55 (54 45 49) 53)" data-semantic-owns="45 49 53"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="45" data-semantic-children="43,44" data-semantic-parent="55" data-semantic-owns="43 44"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.523em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="44" data-semantic-parent="45" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="45"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-script style="vertical-align: -0.317em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="53" data-semantic-children="51" data-semantic-content="50,52" data-semantic-parent="55" data-semantic-owns="50 51 52" style="margin-left: 0.108em;"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="50" data-semantic-parent="53" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="53"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="52" data-semantic-parent="53" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="49" data-semantic-children="46,47,48" data-semantic-content="47" data-semantic-parent="55" data-semantic-owns="46 47 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="49"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="47" data-semantic-parent="49" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="49"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="56" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S4">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.873em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="59"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="118" data-semantic-children="62,116" data-semantic-content="117" data-semantic-parent="119" data-semantic-owns="62 117 116"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="62" data-semantic-children="60,61" data-semantic-parent="118" data-semantic-owns="60 61"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="62"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="62"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="117" data-semantic-parent="118" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="116" data-semantic-children="72,115" data-semantic-content="63" data-semantic-parent="118" data-semantic-owns="72 115"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="72" data-semantic-children="63,67,71" data-semantic-parent="116" data-semantic-owns="63 67 71"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="63" data-semantic-parent="72" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="71" data-semantic-children="68,70" data-semantic-content="69" data-semantic-parent="72" data-semantic-owns="68 69 70"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="71"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="69" data-semantic-parent="71" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="71"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-spacer style="margin-top: 0.233em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="67" data-semantic-children="64,66" data-semantic-content="65" data-semantic-parent="72" data-semantic-owns="64 65 66"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="67"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="65" data-semantic-parent="67" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="67"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="115" data-semantic-children="113,114" data-semantic-parent="116" data-semantic-owns="113 114"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="113" data-semantic-children="111" data-semantic-content="73,112" data-semantic-parent="115" data-semantic-owns="73 111 112"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="73" data-semantic-parent="113" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="111" data-semantic-children="110,108" data-semantic-content="90" data-semantic-parent="113" data-semantic-owns="110 90 108"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="110" data-semantic-children="74,89" data-semantic-content="109,74" data-semantic-parent="111" data-semantic-owns="74 109 89"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="74" data-semantic-parent="110" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="109" data-semantic-parent="110" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="89" data-semantic-children="87" data-semantic-content="75,88" data-semantic-parent="110" data-semantic-owns="75 87 88"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="75" data-semantic-parent="89" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="87" data-semantic-children="76,86" data-semantic-parent="89" data-semantic-owns="76 86"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="87"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.175em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="86" data-semantic-children="83,84,85" data-semantic-content="84" data-semantic-parent="87" data-semantic-owns="83 84 85"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="83" data-semantic-children="81" data-semantic-content="77,82" data-semantic-parent="86" data-semantic-owns="77 81 82"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="77" data-semantic-parent="83" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="81" data-semantic-children="78,80" data-semantic-content="79" data-semantic-parent="83" data-semantic-owns="78 79 80"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="81"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="79" data-semantic-parent="81" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="80" data-semantic-parent="81"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="82" data-semantic-parent="83" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="84" data-semantic-parent="86" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="86"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="88" data-semantic-parent="89" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="90" data-semantic-parent="111" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="108" data-semantic-children="91,106" data-semantic-content="107,91" data-semantic-parent="111" data-semantic-owns="91 107 106"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="91" data-semantic-parent="108" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="107" data-semantic-parent="108" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="106" data-semantic-children="104" data-semantic-content="92,105" data-semantic-parent="108" data-semantic-owns="92 104 105"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="92" data-semantic-parent="106" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="104" data-semantic-children="93,103" data-semantic-parent="106" data-semantic-owns="93 103"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="93" data-semantic-parent="104"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.175em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="103" data-semantic-children="100,101,102" data-semantic-content="101" data-semantic-parent="104" data-semantic-owns="100 101 102"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="100" data-semantic-children="98" data-semantic-content="94,99" data-semantic-parent="103" data-semantic-owns="94 98 99"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="94" data-semantic-parent="100" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="98" data-semantic-children="95,97" data-semantic-content="96" data-semantic-parent="100" data-semantic-owns="95 96 97"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="95" data-semantic-parent="98"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="96" data-semantic-parent="98" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="97" data-semantic-parent="98"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="99" data-semantic-parent="100" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="101" data-semantic-parent="103" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="103"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="105" data-semantic-parent="106" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="112" data-semantic-parent="113" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.524em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="114" data-semantic-parent="115"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="121" data-semantic-parent="122" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S7">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 1.474em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="125" data-semantic-children="124" data-semantic-content="123" data-semantic-parent="126" data-semantic-owns="123 124"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="123" data-semantic-parent="125" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="124" data-semantic-parent="125"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="127" data-semantic-parent="145" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="144" data-semantic-children="128,142" data-semantic-parent="145" data-semantic-collapsed="(144 (c 143) 128 142)" data-semantic-owns="128 142"><mjx-mspace data-semantic-type="text" data-semantic-role="space" data-semantic-annotation="clearspeak:unit" data-semantic-id="128" data-semantic-parent="144" style="width: 1em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="142" data-semantic-children="129,131" data-semantic-content="130" data-semantic-parent="144" data-semantic-owns="129 130 131"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="129" data-semantic-parent="142"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="130" data-semantic-parent="142" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="131" data-semantic-parent="142"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="132" data-semantic-parent="145" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="133" data-semantic-parent="145"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="134" data-semantic-parent="145" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="135" data-semantic-parent="145"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="136" data-semantic-parent="145" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>in see <a id="crosref2085" href="#figs6">Figures&nbsp;S6</a>D–S6K, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X 1 comma period period period comma upper X Subscript n Baseline" tabindex="0" ctxtmenu_counter="22" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (2 0 1) 3 4 5 (8 6 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="2,3,4,5,8" data-semantic-content="3,5" data-semantic-owns="2 3 4 5 8" data-semantic-speech="upper X 1 comma period period period comma upper X Subscript n Baseline"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> are the step sizes and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X Subscript k colon n" tabindex="0" ctxtmenu_counter="23" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper X Subscript k colon n"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="1,2,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> denotes the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="k" tabindex="0" ctxtmenu_counter="24" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="k"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>-th order statistic. Though we are not able to determine a high-confidence estimate for the generalized extreme value index of the max-domain of attraction, it becomes clear that the step sizes follow a heavy-tailed distribution. Furthermore, the empirical autocorrelation function for the step sizes in <a id="crosref2090" href="#fig4">Figure&nbsp;4</a>H seems to support the hypothesis of long-range dependence. We show the positive part of the empirical tail function (i.e., the excess tail function in 0) in see <a id="crosref2095" href="#figs6">Figures&nbsp;S6</a>D–S6K, which shows a polynomial decay.</div></section><section id="sec-4-5-35"><h4>Sensitivity analysis of Lévy-like walk analysis</h4><div id="p0625" role="paragraph">As discussed above, we do not assume the structure of a specific stochastic process. Still, it is clear from our analysis that the analyzed TMs exhibit an anomalous diffusion behavior. Since the collection of <i>in&nbsp;vivo</i> data is complex, we rely on a limited number of TM observations, that is, between 50 and 100 tracks per setting. To analyze the effect of small sample sizes on the estimation, we simulate different stochastic processes with a diffusion behavior which is similar to the observed data. We use moment estimators to derive estimates of the stability parameter and the acceleration factor.</div><div id="p0630" role="paragraph">In the following, we first analyze several stochastic processes which diffuse anomalously with an (approximately) log-log-linear mean squared displacement. Afterwards, we simulate independent copies of these processes and analyze the behavior of the moment estimates, in particular in the setting of small sample sizes. While for certain processes consistency results for these estimators can be shown, we restrict ourselves to numerical experiments.</div></section><section id="sec-4-5-36"><h4>Fractional Compound Poisson Process</h4><div id="p0635" role="paragraph">Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper N left parenthesis t right parenthesis comma t greater than or equals 0 right parenthesis" tabindex="0" ctxtmenu_counter="25" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 0 (13 (11 1 10 (5 2 3 4)) 6 (12 7 8 9)) 14)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="0,14" data-semantic-owns="0 13 14" data-semantic-speech="left parenthesis upper N left parenthesis t right parenthesis comma t greater than or equals 0 right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="13" data-semantic-children="11,6,12" data-semantic-content="6" data-semantic-parent="15" data-semantic-owns="11 6 12"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-children="1,5" data-semantic-content="10,1" data-semantic-parent="13" data-semantic-owns="1 10 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="2,4" data-semantic-parent="11" data-semantic-owns="2 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="13" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="12" data-semantic-children="7,9" data-semantic-content="8" data-semantic-parent="13" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="8" data-semantic-parent="12" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="12"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> be a fractional Poisson process with intensity <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda greater than 0" tabindex="0" ctxtmenu_counter="26" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="lamda greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and stability parameter <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu element of left parenthesis 0 comma 1 right bracket" tabindex="0" ctxtmenu_counter="27" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 1 (5 2 3 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="6" data-semantic-children="0,5" data-semantic-content="1" data-semantic-owns="0 1 5" data-semantic-speech="nu element of left parenthesis 0 comma 1 right bracket"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="2,4" data-semantic-parent="6" data-semantic-owns="2 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c2C" noic="true" style="padding-top: 0.666em;">,</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> as defined in <span class="dropBlock reference-citations"><a id="body-ref-sref7-1" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-1" aria-controls="sref7-1">Cahoy et&nbsp;al. (2010)</a><div class="dropBlock__holder" data-db-target-of="sref7-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Cahoy, D.O. ∙ Uchaikin, V.V. ∙ Woyczynski, W.A.</div><div><strong>Parameter estimation for fractional Poisson processes</strong></div><div><em>J.&nbsp;Stat. Plan. Inference.</em> 2010; <strong>140</strong>:3106-3120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jspi.2010.04.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77954030633" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jspi.2010.04.016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Furthermore, let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper X Subscript k Baseline right parenthesis Subscript k element of double struck upper N 0" tabindex="0" ctxtmenu_counter="28" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 (5 0 (3 1 2) 4) (11 6 7 (10 8 9)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="5,11" data-semantic-owns="5 11" data-semantic-speech="left parenthesis upper X Subscript k Baseline right parenthesis Subscript k element of double struck upper N 0"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-parent="12" data-semantic-owns="0 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.283em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="11" data-semantic-children="6,10" data-semantic-content="7" data-semantic-parent="12" data-semantic-owns="6 7 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="element" data-semantic-id="7" data-semantic-parent="11" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="numbersetletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="11" data-semantic-owns="8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="numbersetletter" data-semantic-font="double-struck" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c2115 MM-DS">ℕ</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> be a sequence of independent, identically distributed random variables with probability distribution <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P Subscript upper X 1 Baseline equals one half delta Subscript negative 1 Baseline plus one half delta 1" tabindex="0" ctxtmenu_counter="29" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (4 0 (3 1 2)) 5 (25 (22 (8 6 7) 21 (13 9 (12 10 11))) 14 (24 (17 15 16) 23 (20 18 19))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="26" data-semantic-children="4,25" data-semantic-content="5" data-semantic-owns="4 5 25" data-semantic-speech="upper P Subscript upper X 1 Baseline equals one half delta Subscript negative 1 Baseline plus one half delta 1"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-parent="26" data-semantic-owns="0 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.112em;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="26" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="25" data-semantic-children="22,24" data-semantic-content="14" data-semantic-parent="26" data-semantic-owns="22 14 24"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="8,13" data-semantic-content="21" data-semantic-parent="25" data-semantic-owns="8 21 13"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="vulgar" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="22" data-semantic-owns="6 7"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="13" data-semantic-children="9,12" data-semantic-parent="22" data-semantic-owns="9 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="13"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-children="11" data-semantic-content="10" data-semantic-parent="13" data-semantic-owns="10 11"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-parent="12" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="25" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="17,20" data-semantic-content="23" data-semantic-parent="25" data-semantic-owns="17 23 20"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="vulgar" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="24" data-semantic-owns="15 16"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="24" data-semantic-owns="18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D6FF">𝛿</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.008em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, independent of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="30" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. Define the fractional compound Poisson process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis upper Z Subscript t Baseline right parenthesis Subscript t greater than or equals 0" tabindex="0" ctxtmenu_counter="31" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 (5 0 (3 1 2) 4) (9 6 7 8))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="5,9" data-semantic-owns="5 9" data-semantic-speech="left parenthesis upper Z Subscript t Baseline right parenthesis Subscript t greater than or equals 0"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="0,4" data-semantic-parent="10" data-semantic-owns="0 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="5" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.283em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> with drift <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu greater than 0" tabindex="0" ctxtmenu_counter="32" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="mu greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Z Subscript t Baseline equals mu t Superscript nu divided by 2 Baseline plus sigma summation Underscript i equals 0 Overscript upper N left parenthesis t right parenthesis Endscripts upper X Subscript k" tabindex="0" ctxtmenu_counter="33" style="font-size: 130%;"><mjx-math data-semantic-structure="(32 (2 0 1) 3 (31 (30 4 29 (10 5 (9 6 7 8))) 11 (28 (24 12 (16 13 14 15) (23 17 22 (21 18 19 20))) (27 25 26))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="32" data-semantic-children="2,31" data-semantic-content="3" data-semantic-owns="2 3 31" data-semantic-speech="upper Z Subscript t Baseline equals mu t Superscript nu divided by 2 Baseline plus sigma summation Underscript i equals 0 Overscript upper N left parenthesis t right parenthesis Endscripts upper X Subscript k"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="32" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="32" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="31" data-semantic-children="30,28" data-semantic-content="11" data-semantic-parent="32" data-semantic-owns="30 11 28"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="30" data-semantic-children="4,10" data-semantic-content="29" data-semantic-parent="31" data-semantic-owns="4 29 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="30"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="29" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="5,9" data-semantic-parent="30" data-semantic-owns="5 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="11" data-semantic-parent="31" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="28" data-semantic-children="24,27" data-semantic-content="12" data-semantic-parent="31" data-semantic-owns="24 27"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="24" data-semantic-children="12,16,23" data-semantic-parent="28" data-semantic-owns="12 16 23"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="12" data-semantic-parent="24" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-mrow size="s" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-children="17,21" data-semantic-content="22,17" data-semantic-parent="24" data-semantic-owns="17 22 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="23" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="21" data-semantic-children="19" data-semantic-content="18,20" data-semantic-parent="23" data-semantic-owns="18 19 20"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="18" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="20" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="16" data-semantic-children="13,15" data-semantic-content="14" data-semantic-parent="24" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="14" data-semantic-parent="16" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="27" data-semantic-children="25,26" data-semantic-parent="28" data-semantic-owns="25 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="27"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="27"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. It then holds that<div id="ufd6" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket equals sigma summation Underscript n equals 0 Overscript infinity Endscripts double struck upper E left bracket left parenthesis mu t Superscript nu divided by 2 Baseline plus sigma summation Underscript k equals 1 Overscript n Endscripts upper X Subscript k Baseline right parenthesis squared 1 Subscript upper N left parenthesis t right parenthesis equals n Baseline right bracket equals mu t Superscript nu Baseline plus sigma summation Underscript n equals 0 Overscript infinity Endscripts double struck upper E left bracket left parenthesis 2 mu t Superscript nu divided by 2 Baseline sigma summation Underscript k equals 1 Overscript n Endscripts upper X Subscript k Baseline plus sigma summation Underscript j comma k equals 1 Overscript n Endscripts upper X Subscript k Baseline upper X Subscript j Baseline right parenthesis 1 Subscript upper N left parenthesis t right parenthesis equals n Baseline right bracket equals mu t Superscript nu Baseline plus sigma summation Underscript n equals 0 Overscript infinity Endscripts upper P left parenthesis upper N left parenthesis t right parenthesis equals n right parenthesis dot n dot double struck upper E left bracket upper X 1 squared right bracket equals mu t Superscript nu Baseline plus double struck upper E left bracket upper N left parenthesis t right parenthesis right bracket equals left parenthesis StartFraction lamda Over upper Gamma left parenthesis nu plus 1 right parenthesis EndFraction plus mu right parenthesis t Superscript nu Baseline period" tabindex="0" ctxtmenu_counter="34" style="font-size: 130%;"><mjx-math data-semantic-structure="(260 (67 (66 0 65 (8 1 (6 (5 2 3) 4) 7)) 9 (64 (16 10 (14 11 12 13) 15) (63 17 62 (61 18 (59 (45 (43 19 (41 (40 20 39 (26 21 (25 22 23 24))) 27 (38 (34 28 (32 29 30 31) 33) (37 35 36))) 42) 44) 58 (57 46 (56 (55 47 54 (51 48 49 50)) 52 53))) 60)))) 256 (153 149 68 (152 (151 69 150 (72 70 71)) 73 (148 (80 74 (78 75 76 77) 79) (147 81 146 (145 82 (143 (129 83 (127 (126 84 123 85 124 (91 86 (90 87 88 89)) 125 (122 (98 92 (96 93 94 95) 97) (101 99 100))) 102 (121 (112 103 (110 104 105 (109 106 107 108)) 111) (120 (115 113 114) 119 (118 116 117)))) 128) 142 (141 130 (140 (139 131 138 (135 132 133 134)) 136 137))) 144))))) 257 (203 199 154 (202 (201 155 200 (158 156 157)) 159 (198 (166 160 (164 161 162 163) 165) (197 (196 167 195 (180 168 (178 (177 169 176 (173 170 171 172)) 174 175) 179)) 181 182 183 (194 184 193 (192 185 (190 (189 186 187) 188) 191)))))) 258 (227 223 204 (226 (225 205 224 (208 206 207)) 209 (222 210 221 (220 211 (218 212 217 (216 213 214 215)) 219)))) 259 (255 (254 251 228 (253 (246 229 (244 (241 230 (240 231 239 (238 232 (236 233 234 235) 237))) 242 243) 245) 252 (249 247 248))) 250))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="260" data-semantic-children="67,153,203,227,255" data-semantic-content="256,257,258,259" data-semantic-owns="67 256 153 257 203 258 227 259 255" data-semantic-speech="double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket equals sigma summation Underscript n equals 0 Overscript infinity Endscripts double struck upper E left bracket left parenthesis mu t Superscript nu divided by 2 Baseline plus sigma summation Underscript k equals 1 Overscript n Endscripts upper X Subscript k Baseline right parenthesis squared 1 Subscript upper N left parenthesis t right parenthesis equals n Baseline right bracket equals mu t Superscript nu Baseline plus sigma summation Underscript n equals 0 Overscript infinity Endscripts double struck upper E left bracket left parenthesis 2 mu t Superscript nu divided by 2 Baseline sigma summation Underscript k equals 1 Overscript n Endscripts upper X Subscript k Baseline plus sigma summation Underscript j comma k equals 1 Overscript n Endscripts upper X Subscript k Baseline upper X Subscript j Baseline right parenthesis 1 Subscript upper N left parenthesis t right parenthesis equals n Baseline right bracket equals mu t Superscript nu Baseline plus sigma summation Underscript n equals 0 Overscript infinity Endscripts upper P left parenthesis upper N left parenthesis t right parenthesis equals n right parenthesis dot n dot double struck upper E left bracket upper X 1 squared right bracket equals mu t Superscript nu Baseline plus double struck upper E left bracket upper N left parenthesis t right parenthesis right bracket equals left parenthesis StartFraction lamda Over upper Gamma left parenthesis nu plus 1 right parenthesis EndFraction plus mu right parenthesis t Superscript nu Baseline period"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="67" data-semantic-children="66,64" data-semantic-content="9" data-semantic-parent="260" data-semantic-owns="66 9 64"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="66" data-semantic-children="0,8" data-semantic-content="65,0" data-semantic-parent="67" data-semantic-owns="0 65 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="66" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="65" data-semantic-parent="66" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="1,7" data-semantic-parent="66" data-semantic-owns="1 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-SO">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.26em; margin-left: -0.04em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6" style="margin-left: 0.13em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-SO">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="67" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="64" data-semantic-children="16,63" data-semantic-content="10" data-semantic-parent="67" data-semantic-owns="16 63"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="16" data-semantic-children="10,14,15" data-semantic-parent="64" data-semantic-owns="10 14 15" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="10" data-semantic-parent="16" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c221E MM-I">∞</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.274em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="12" data-semantic-parent="16" data-semantic-owns="11 12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="12" data-semantic-parent="14" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="63" data-semantic-children="17,61" data-semantic-content="62,17" data-semantic-parent="64" data-semantic-owns="17 62 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="63" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="62" data-semantic-parent="63" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="61" data-semantic-children="59" data-semantic-content="18,60" data-semantic-parent="63" data-semantic-owns="18 59 60"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="18" data-semantic-parent="61" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S4">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="59" data-semantic-children="45,57" data-semantic-content="58" data-semantic-parent="61" data-semantic-owns="45 58 57"><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="45" data-semantic-children="43,44" data-semantic-parent="59" data-semantic-owns="43 44"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="43" data-semantic-children="41" data-semantic-content="19,42" data-semantic-parent="45" data-semantic-owns="19 41 42"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="19" data-semantic-parent="43" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="41" data-semantic-children="40,38" data-semantic-content="27" data-semantic-parent="43" data-semantic-owns="40 27 38"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="40" data-semantic-children="20,26" data-semantic-content="39" data-semantic-parent="41" data-semantic-owns="20 39 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="40"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-parent="40" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="21,25" data-semantic-parent="40" data-semantic-owns="21 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="26"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="25" data-semantic-children="22,24" data-semantic-content="23" data-semantic-parent="26" data-semantic-owns="22 23 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="23" data-semantic-parent="25" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="27" data-semantic-parent="41" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="38" data-semantic-children="34,37" data-semantic-content="28" data-semantic-parent="41" data-semantic-owns="34 37"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="34" data-semantic-children="28,32,33" data-semantic-parent="38" data-semantic-owns="28 32 33" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="28" data-semantic-parent="34" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.264em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="32" data-semantic-children="29,31" data-semantic-content="30" data-semantic-parent="34" data-semantic-owns="29 30 31"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="32"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="30" data-semantic-parent="32" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="32"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="35,36" data-semantic-parent="38" data-semantic-owns="35 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="42" data-semantic-parent="43" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.62em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="45"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="59" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="integer" data-semantic-id="57" data-semantic-children="46,56" data-semantic-parent="59" data-semantic-owns="46 56"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="57"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-script style="vertical-align: -0.175em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="56" data-semantic-children="55,53" data-semantic-content="52" data-semantic-parent="57" data-semantic-owns="55 52 53"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-children="47,51" data-semantic-content="54,47" data-semantic-parent="56" data-semantic-owns="47 54 51"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="47" data-semantic-parent="55" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="54" data-semantic-parent="55" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="51" data-semantic-children="49" data-semantic-content="48,50" data-semantic-parent="55" data-semantic-owns="48 49 50"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="48" data-semantic-parent="51" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="51"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="50" data-semantic-parent="51" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="52" data-semantic-parent="56" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="56"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="60" data-semantic-parent="61" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S4">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="256" data-semantic-parent="260" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="153" data-semantic-children="149,152" data-semantic-content="68" data-semantic-parent="260" data-semantic-owns="149 68 152"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="149" data-semantic-parent="153"></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="68" data-semantic-parent="153" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="152" data-semantic-children="151,148" data-semantic-content="73" data-semantic-parent="153" data-semantic-owns="151 73 148"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="151" data-semantic-children="69,72" data-semantic-content="150" data-semantic-parent="152" data-semantic-owns="69 150 72"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="69" data-semantic-parent="151"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="150" data-semantic-parent="151" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="72" data-semantic-children="70,71" data-semantic-parent="151" data-semantic-owns="70 71"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="70" data-semantic-parent="72"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="72"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="73" data-semantic-parent="152" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="148" data-semantic-children="80,147" data-semantic-content="74" data-semantic-parent="152" data-semantic-owns="80 147"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="80" data-semantic-children="74,78,79" data-semantic-parent="148" data-semantic-owns="74 78 79" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="74" data-semantic-parent="80" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-id="79" data-semantic-parent="80"><mjx-c class="mjx-c221E MM-I">∞</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.274em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="78" data-semantic-children="75,77" data-semantic-content="76" data-semantic-parent="80" data-semantic-owns="75 76 77"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="75" data-semantic-parent="78"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="76" data-semantic-parent="78" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="78"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="147" data-semantic-children="81,145" data-semantic-content="146,81" data-semantic-parent="148" data-semantic-owns="81 146 145"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-parent="147" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="146" data-semantic-parent="147" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="145" data-semantic-children="143" data-semantic-content="82,144" data-semantic-parent="147" data-semantic-owns="82 143 144"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="82" data-semantic-parent="145" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="143" data-semantic-children="129,141" data-semantic-content="142" data-semantic-parent="145" data-semantic-owns="129 142 141"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="129" data-semantic-children="127" data-semantic-content="83,128" data-semantic-parent="143" data-semantic-owns="83 127 128"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="83" data-semantic-parent="129" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="127" data-semantic-children="126,121" data-semantic-content="102" data-semantic-parent="129" data-semantic-owns="126 102 121"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="126" data-semantic-children="84,85,91,122" data-semantic-content="123,124,125" data-semantic-parent="127" data-semantic-owns="84 123 85 124 91 125 122"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="84" data-semantic-parent="126"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="123" data-semantic-parent="126" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="85" data-semantic-parent="126"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="124" data-semantic-parent="126" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="91" data-semantic-children="86,90" data-semantic-parent="126" data-semantic-owns="86 90"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="86" data-semantic-parent="91"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="90" data-semantic-children="87,89" data-semantic-content="88" data-semantic-parent="91" data-semantic-owns="87 88 89"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="87" data-semantic-parent="90"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="88" data-semantic-parent="90" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="89" data-semantic-parent="90"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="125" data-semantic-parent="126" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="122" data-semantic-children="98,101" data-semantic-content="92" data-semantic-parent="126" data-semantic-owns="98 101"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="98" data-semantic-children="92,96,97" data-semantic-parent="122" data-semantic-owns="92 96 97" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="92" data-semantic-parent="98" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="97" data-semantic-parent="98"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.264em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="96" data-semantic-children="93,95" data-semantic-content="94" data-semantic-parent="98" data-semantic-owns="93 94 95"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="93" data-semantic-parent="96"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="94" data-semantic-parent="96" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="95" data-semantic-parent="96"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="101" data-semantic-children="99,100" data-semantic-parent="122" data-semantic-owns="99 100"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="101"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="100" data-semantic-parent="101"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="102" data-semantic-parent="127" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="121" data-semantic-children="112,120" data-semantic-content="103" data-semantic-parent="127" data-semantic-owns="112 120"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="112" data-semantic-children="103,110,111" data-semantic-parent="121" data-semantic-owns="103 110 111" limits="false"><mjx-mrow><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="103" data-semantic-parent="112" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="111" data-semantic-parent="112"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.264em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="110" data-semantic-children="104,105,109" data-semantic-content="105" data-semantic-parent="112" data-semantic-owns="104 105 109"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="104" data-semantic-parent="110"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="105" data-semantic-parent="110" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="109" data-semantic-children="106,108" data-semantic-content="107" data-semantic-parent="110" data-semantic-owns="106 107 108"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="106" data-semantic-parent="109"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="107" data-semantic-parent="109" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="108" data-semantic-parent="109"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="120" data-semantic-children="115,118" data-semantic-content="119" data-semantic-parent="121" data-semantic-owns="115 119 118"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="115" data-semantic-children="113,114" data-semantic-parent="120" data-semantic-owns="113 114"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="113" data-semantic-parent="115"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="114" data-semantic-parent="115"><mjx-c class="mjx-c1D458">𝑘</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="119" data-semantic-parent="120" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="118" data-semantic-children="116,117" data-semantic-parent="120" data-semantic-owns="116 117"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="116" data-semantic-parent="118"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="117" data-semantic-parent="118"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="128" data-semantic-parent="129" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="142" data-semantic-parent="143" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="integer" data-semantic-id="141" data-semantic-children="130,140" data-semantic-parent="143" data-semantic-owns="130 140"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="130" data-semantic-parent="141"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-script style="vertical-align: -0.175em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="140" data-semantic-children="139,137" data-semantic-content="136" data-semantic-parent="141" data-semantic-owns="139 136 137"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="139" data-semantic-children="131,135" data-semantic-content="138,131" data-semantic-parent="140" data-semantic-owns="131 138 135"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="131" data-semantic-parent="139" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="138" data-semantic-parent="139" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="135" data-semantic-children="133" data-semantic-content="132,134" data-semantic-parent="139" data-semantic-owns="132 133 134"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="132" data-semantic-parent="135" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="133" data-semantic-parent="135"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="134" data-semantic-parent="135" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="136" data-semantic-parent="140" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="137" data-semantic-parent="140"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="144" data-semantic-parent="145" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="257" data-semantic-parent="260" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="203" data-semantic-children="199,202" data-semantic-content="154" data-semantic-parent="260" data-semantic-owns="199 154 202"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="199" data-semantic-parent="203"></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="154" data-semantic-parent="203" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="202" data-semantic-children="201,198" data-semantic-content="159" data-semantic-parent="203" data-semantic-owns="201 159 198"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="201" data-semantic-children="155,158" data-semantic-content="200" data-semantic-parent="202" data-semantic-owns="155 200 158"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="155" data-semantic-parent="201"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="200" data-semantic-parent="201" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="158" data-semantic-children="156,157" data-semantic-parent="201" data-semantic-owns="156 157"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="156" data-semantic-parent="158"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="157" data-semantic-parent="158"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="159" data-semantic-parent="202" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="198" data-semantic-children="166,197" data-semantic-content="160" data-semantic-parent="202" data-semantic-owns="166 197"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="166" data-semantic-children="160,164,165" data-semantic-parent="198" data-semantic-owns="160 164 165" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="160" data-semantic-parent="166" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-id="165" data-semantic-parent="166"><mjx-c class="mjx-c221E MM-I">∞</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.274em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="164" data-semantic-children="161,163" data-semantic-content="162" data-semantic-parent="166" data-semantic-owns="161 162 163"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="161" data-semantic-parent="164"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="162" data-semantic-parent="164" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="163" data-semantic-parent="164"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="197" data-semantic-children="196,182,194" data-semantic-content="181,183" data-semantic-parent="198" data-semantic-owns="196 181 182 183 194"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="196" data-semantic-children="167,180" data-semantic-content="195,167" data-semantic-parent="197" data-semantic-owns="167 195 180"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="167" data-semantic-parent="196" data-semantic-operator="appl"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="195" data-semantic-parent="196" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="180" data-semantic-children="178" data-semantic-content="168,179" data-semantic-parent="196" data-semantic-owns="168 178 179"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="168" data-semantic-parent="180" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="178" data-semantic-children="177,175" data-semantic-content="174" data-semantic-parent="180" data-semantic-owns="177 174 175"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="177" data-semantic-children="169,173" data-semantic-content="176,169" data-semantic-parent="178" data-semantic-owns="169 176 173"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="169" data-semantic-parent="177" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="176" data-semantic-parent="177" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="173" data-semantic-children="171" data-semantic-content="170,172" data-semantic-parent="177" data-semantic-owns="170 171 172"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="170" data-semantic-parent="173" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="171" data-semantic-parent="173"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="172" data-semantic-parent="173" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="174" data-semantic-parent="178" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="175" data-semantic-parent="178"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="179" data-semantic-parent="180" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="181" data-semantic-parent="197" data-semantic-operator="infixop,⋅"><mjx-c class="mjx-c22C5">⋅</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="182" data-semantic-parent="197"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="183" data-semantic-parent="197" data-semantic-operator="infixop,⋅"><mjx-c class="mjx-c22C5">⋅</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="194" data-semantic-children="184,192" data-semantic-content="193,184" data-semantic-parent="197" data-semantic-owns="184 193 192"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="184" data-semantic-parent="194" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="193" data-semantic-parent="194" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="192" data-semantic-children="190" data-semantic-content="185,191" data-semantic-parent="194" data-semantic-owns="185 190 191"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="185" data-semantic-parent="192" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-SO">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="190" data-semantic-children="186,187,188" data-semantic-parent="192" data-semantic-collapsed="(190 (189 186 187) 188)" data-semantic-owns="186 187 188"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="186" data-semantic-parent="190"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.288em; margin-left: -0.023em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="188" data-semantic-parent="190" style="margin-left: 0.105em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="187" data-semantic-parent="190"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="191" data-semantic-parent="192" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-SO">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="258" data-semantic-parent="260" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="227" data-semantic-children="223,226" data-semantic-content="204" data-semantic-parent="260" data-semantic-owns="223 204 226"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="223" data-semantic-parent="227"></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="204" data-semantic-parent="227" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="226" data-semantic-children="225,222" data-semantic-content="209" data-semantic-parent="227" data-semantic-owns="225 209 222"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="225" data-semantic-children="205,208" data-semantic-content="224" data-semantic-parent="226" data-semantic-owns="205 224 208"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="205" data-semantic-parent="225"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="224" data-semantic-parent="225" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="208" data-semantic-children="206,207" data-semantic-parent="225" data-semantic-owns="206 207"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="206" data-semantic-parent="208"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="207" data-semantic-parent="208"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="209" data-semantic-parent="226" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="222" data-semantic-children="210,220" data-semantic-content="221,210" data-semantic-parent="226" data-semantic-owns="210 221 220"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="210" data-semantic-parent="222" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="221" data-semantic-parent="222" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="220" data-semantic-children="218" data-semantic-content="211,219" data-semantic-parent="222" data-semantic-owns="211 218 219"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="211" data-semantic-parent="220" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="218" data-semantic-children="212,216" data-semantic-content="217,212" data-semantic-parent="220" data-semantic-owns="212 217 216"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="212" data-semantic-parent="218" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="217" data-semantic-parent="218" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="216" data-semantic-children="214" data-semantic-content="213,215" data-semantic-parent="218" data-semantic-owns="213 214 215"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="213" data-semantic-parent="216" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="214" data-semantic-parent="216"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="215" data-semantic-parent="216" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="219" data-semantic-parent="220" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="259" data-semantic-parent="260" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="255" data-semantic-children="254,250" data-semantic-content="250" data-semantic-parent="260" data-semantic-owns="254 250"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="254" data-semantic-children="251,253" data-semantic-content="228" data-semantic-parent="255" data-semantic-owns="251 228 253"><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="251" data-semantic-parent="254"></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="228" data-semantic-parent="254" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="253" data-semantic-children="246,249" data-semantic-content="252" data-semantic-parent="254" data-semantic-owns="246 252 249"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="246" data-semantic-children="244" data-semantic-content="229,245" data-semantic-parent="253" data-semantic-owns="229 244 245"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="229" data-semantic-parent="246" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="244" data-semantic-children="241,243" data-semantic-content="242" data-semantic-parent="246" data-semantic-owns="241 242 243"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="241" data-semantic-children="230,240" data-semantic-parent="244" data-semantic-owns="230 240"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="230" data-semantic-parent="241"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="240" data-semantic-children="231,238" data-semantic-content="239" data-semantic-parent="241" data-semantic-owns="231 239 238"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="231" data-semantic-parent="240"><mjx-c class="mjx-c1D6E4">𝛤</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="239" data-semantic-parent="240" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="238" data-semantic-children="236" data-semantic-content="232,237" data-semantic-parent="240" data-semantic-owns="232 236 237"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="232" data-semantic-parent="238" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="236" data-semantic-children="233,235" data-semantic-content="234" data-semantic-parent="238" data-semantic-owns="233 234 235"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="233" data-semantic-parent="236"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="234" data-semantic-parent="236" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="235" data-semantic-parent="236"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="237" data-semantic-parent="238" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="242" data-semantic-parent="244" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="243" data-semantic-parent="244"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="245" data-semantic-parent="246" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="252" data-semantic-parent="253" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="249" data-semantic-children="247,248" data-semantic-parent="253" data-semantic-owns="247 248"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="247" data-semantic-parent="249"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="248" data-semantic-parent="249"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="250" data-semantic-parent="255" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0640" role="paragraph">Therefore, we get that<div id="ufd7" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="log left parenthesis double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket right parenthesis equals nu log left parenthesis t right parenthesis plus log left parenthesis StartFraction lamda Over upper Gamma left parenthesis nu plus 1 right parenthesis EndFraction plus mu right parenthesis period" tabindex="0" ctxtmenu_counter="35" style="font-size: 130%;"><mjx-math data-semantic-structure="(53 (52 (48 0 47 (14 1 (12 2 11 (10 3 (8 (7 4 5) 6) 9)) 13)) 15 (51 (50 16 49 (46 17 45 (21 18 19 20))) 22 (44 23 43 (41 24 (39 (36 25 (35 26 34 (33 27 (31 28 29 30) 32))) 37 38) 40)))) 42)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="53" data-semantic-children="52,42" data-semantic-content="42" data-semantic-owns="52 42" data-semantic-speech="log left parenthesis double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket right parenthesis equals nu log left parenthesis t right parenthesis plus log left parenthesis StartFraction lamda Over upper Gamma left parenthesis nu plus 1 right parenthesis EndFraction plus mu right parenthesis period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="52" data-semantic-children="48,51" data-semantic-content="15" data-semantic-parent="53" data-semantic-owns="48 15 51"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="48" data-semantic-children="0,14" data-semantic-content="47,0" data-semantic-parent="52" data-semantic-owns="0 47 14"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="0" data-semantic-parent="48" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="1,13" data-semantic-parent="48" data-semantic-owns="1 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="12" data-semantic-children="2,10" data-semantic-content="11,2" data-semantic-parent="14" data-semantic-owns="2 11 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="12" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="3,9" data-semantic-parent="12" data-semantic-owns="3 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-SO">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="4,5,6" data-semantic-parent="10" data-semantic-collapsed="(8 (7 4 5) 6)" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.26em; margin-left: -0.04em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8" style="margin-left: 0.13em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-SO">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="15" data-semantic-parent="52" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="51" data-semantic-children="50,44" data-semantic-content="22" data-semantic-parent="52" data-semantic-owns="50 22 44"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="50" data-semantic-children="16,46" data-semantic-content="49" data-semantic-parent="51" data-semantic-owns="16 49 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="50"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="49" data-semantic-parent="50" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-children="17,21" data-semantic-content="45,17" data-semantic-parent="50" data-semantic-owns="17 45 21"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="17" data-semantic-parent="46" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="21" data-semantic-children="19" data-semantic-content="18,20" data-semantic-parent="46" data-semantic-owns="18 19 20"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="18" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="20" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="22" data-semantic-parent="51" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="44" data-semantic-children="23,41" data-semantic-content="43,23" data-semantic-parent="51" data-semantic-owns="23 43 41"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="23" data-semantic-parent="44" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="41" data-semantic-children="39" data-semantic-content="24,40" data-semantic-parent="44" data-semantic-owns="24 39 40"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="24" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="39" data-semantic-children="36,38" data-semantic-content="37" data-semantic-parent="41" data-semantic-owns="36 37 38"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="36" data-semantic-children="25,35" data-semantic-parent="39" data-semantic-owns="25 35"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="36"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="35" data-semantic-children="26,33" data-semantic-content="34" data-semantic-parent="36" data-semantic-owns="26 34 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="35"><mjx-c class="mjx-c1D6E4">𝛤</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="33" data-semantic-children="31" data-semantic-content="27,32" data-semantic-parent="35" data-semantic-owns="27 31 32"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="27" data-semantic-parent="33" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="31" data-semantic-children="28,30" data-semantic-content="29" data-semantic-parent="33" data-semantic-owns="28 29 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="31"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="29" data-semantic-parent="31" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="32" data-semantic-parent="33" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="37" data-semantic-parent="39" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="39"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="40" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="42" data-semantic-parent="53" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0645" role="paragraph">Fixing either <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda greater than 0" tabindex="0" ctxtmenu_counter="36" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="lamda greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu greater than 0" tabindex="0" ctxtmenu_counter="37" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="mu greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, we obtain a model for which moment estimators are derivable and for a fixed number of observations <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t 0 comma period period period comma t Subscript n Baseline" tabindex="0" ctxtmenu_counter="38" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 (2 0 1) 3 4 5 (8 6 7))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="2,3,4,5,8" data-semantic-content="3,5" data-semantic-owns="2 3 4 5 8" data-semantic-speech="t 0 comma period period period comma t Subscript n Baseline"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="9" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn space="2" data-semantic-type="number" data-semantic-role="float" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.106em;">.</mjx-c><mjx-c class="mjx-c2E" style="padding-top: 0.106em;">.</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> we set<div id="ufd8" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove nu With ˆ equals StartFraction 1 Over n EndFraction sigma summation Underscript i equals 1 Overscript n Endscripts StartFraction log left parenthesis ModifyingAbove double struck upper E left bracket upper Z Subscript t Sub Subscript i Subscript Superscript 2 Baseline right bracket With ˆ right parenthesis minus log left parenthesis ModifyingAbove double struck upper E left bracket upper Z Subscript t Sub Subscript i minus 1 Subscript Superscript 2 Baseline right bracket With ˆ right parenthesis Over log left parenthesis t Subscript i Baseline right parenthesis minus log left parenthesis t Subscript i minus 1 Baseline right parenthesis EndFraction" tabindex="0" ctxtmenu_counter="39" style="font-size: 130%;"><mjx-math data-semantic-structure="(88 (2 0 1) 3 (87 (6 4 5) 86 (85 (13 7 (11 8 9 10) 12) (84 (60 (59 14 58 (32 15 (30 (28 16 27 (26 17 (24 (23 18 (21 19 20)) 22) 25)) 29) 31)) 33 (57 34 56 (55 35 (53 (51 36 50 (49 37 (47 (46 38 (44 39 (43 40 41 42))) 45) 48)) 52) 54))) (83 (82 61 81 (67 62 (65 63 64) 66)) 68 (80 69 79 (78 70 (76 71 (75 72 73 74)) 77)))))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="88" data-semantic-children="2,87" data-semantic-content="3" data-semantic-owns="2 3 87" data-semantic-speech="ModifyingAbove nu With ˆ equals StartFraction 1 Over n EndFraction sigma summation Underscript i equals 1 Overscript n Endscripts StartFraction log left parenthesis ModifyingAbove double struck upper E left bracket upper Z Subscript t Sub Subscript i Subscript Superscript 2 Baseline right bracket With ˆ right parenthesis minus log left parenthesis ModifyingAbove double struck upper E left bracket upper Z Subscript t Sub Subscript i minus 1 Subscript Superscript 2 Baseline right bracket With ˆ right parenthesis Over log left parenthesis t Subscript i Baseline right parenthesis minus log left parenthesis t Subscript i minus 1 Baseline right parenthesis EndFraction"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="88" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.561em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="88" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="87" data-semantic-children="6,85" data-semantic-content="86" data-semantic-parent="88" data-semantic-owns="6 86 85"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="87" data-semantic-owns="4 5"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="86" data-semantic-parent="87" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="85" data-semantic-children="13,84" data-semantic-content="7" data-semantic-parent="87" data-semantic-owns="13 84"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="13" data-semantic-children="7,11,12" data-semantic-parent="85" data-semantic-owns="7 11 12" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="7" data-semantic-parent="13" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.284em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="11" data-semantic-children="8,10" data-semantic-content="9" data-semantic-parent="13" data-semantic-owns="8 9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="11" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mfrac space="2" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="84" data-semantic-children="60,83" data-semantic-parent="85" data-semantic-owns="60 83"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="60" data-semantic-children="59,57" data-semantic-content="33" data-semantic-parent="84" data-semantic-owns="59 33 57"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="59" data-semantic-children="14,32" data-semantic-content="58,14" data-semantic-parent="60" data-semantic-owns="14 58 32"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="14" data-semantic-parent="59" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="58" data-semantic-parent="59" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="32" data-semantic-children="30" data-semantic-content="15,31" data-semantic-parent="59" data-semantic-owns="15 30 31"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S5">(</mjx-c></mjx-mo><mjx-mover data-semantic-type="overscore" data-semantic-role="simple function" data-semantic-id="30" data-semantic-children="28,29" data-semantic-parent="32" data-semantic-owns="28 29"><mjx-over style="padding-bottom: 0.105em; padding-left: 1.411em; margin-bottom: -0.569em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="29" data-semantic-parent="30" style="width: 0px; margin-left: -0.948em;"><mjx-c class="mjx-c302 MM-S7">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="28" data-semantic-children="16,26" data-semantic-content="27,16" data-semantic-parent="30" data-semantic-owns="16 27 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="28" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="26" data-semantic-children="24" data-semantic-content="17,25" data-semantic-parent="28" data-semantic-owns="17 24 25"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="17" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="18,21,22" data-semantic-parent="26" data-semantic-collapsed="(24 (23 18 21) 22)" data-semantic-owns="18 21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="24"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.269em; margin-left: -0.04em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24" style="margin-left: 0.13em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="24" data-semantic-owns="19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.307em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="25" data-semantic-parent="26" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-base></mjx-mover><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="31" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S5">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="33" data-semantic-parent="60" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="57" data-semantic-children="34,55" data-semantic-content="56,34" data-semantic-parent="60" data-semantic-owns="34 56 55"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="34" data-semantic-parent="57" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="56" data-semantic-parent="57" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="55" data-semantic-children="53" data-semantic-content="35,54" data-semantic-parent="57" data-semantic-owns="35 53 54"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="35" data-semantic-parent="55" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S5">(</mjx-c></mjx-mo><mjx-mover data-semantic-type="overscore" data-semantic-role="simple function" data-semantic-id="53" data-semantic-children="51,52" data-semantic-parent="55" data-semantic-owns="51 52"><mjx-over style="padding-bottom: 0.105em; padding-left: 1.863em; margin-bottom: -0.569em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="52" data-semantic-parent="53" style="width: 0px; margin-left: -0.948em;"><mjx-c class="mjx-c302 MM-S7">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="51" data-semantic-children="36,49" data-semantic-content="50,36" data-semantic-parent="53" data-semantic-owns="36 50 49"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="51" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="50" data-semantic-parent="51" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="49" data-semantic-children="47" data-semantic-content="37,48" data-semantic-parent="51" data-semantic-owns="37 47 48"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="37" data-semantic-parent="49" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-S3">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="47" data-semantic-children="38,44,45" data-semantic-parent="49" data-semantic-collapsed="(47 (46 38 44) 45)" data-semantic-owns="38 44 45"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="47"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.269em; margin-left: -0.04em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="47" style="margin-left: 0.13em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="44" data-semantic-children="39,43" data-semantic-parent="46" data-semantic-owns="39 43"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="47"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.312em;"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="43" data-semantic-children="40,42" data-semantic-content="41" data-semantic-parent="44" data-semantic-owns="40 41 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="43"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="41" data-semantic-parent="43" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="43"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="48" data-semantic-parent="49" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-S3">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-base></mjx-mover><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="54" data-semantic-parent="55" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S5">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="83" data-semantic-children="82,80" data-semantic-content="68" data-semantic-parent="84" data-semantic-owns="82 68 80"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="82" data-semantic-children="61,67" data-semantic-content="81,61" data-semantic-parent="83" data-semantic-owns="61 81 67"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="61" data-semantic-parent="82" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="81" data-semantic-parent="82" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="67" data-semantic-children="65" data-semantic-content="62,66" data-semantic-parent="82" data-semantic-owns="62 65 66"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="62" data-semantic-parent="67" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="65" data-semantic-children="63,64" data-semantic-parent="67" data-semantic-owns="63 64"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="65"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="65"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="66" data-semantic-parent="67" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="68" data-semantic-parent="83" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="80" data-semantic-children="69,78" data-semantic-content="79,69" data-semantic-parent="83" data-semantic-owns="69 79 78"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="69" data-semantic-parent="80" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="79" data-semantic-parent="80" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="78" data-semantic-children="76" data-semantic-content="70,77" data-semantic-parent="80" data-semantic-owns="70 76 77"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="70" data-semantic-parent="78" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="76" data-semantic-children="71,75" data-semantic-parent="78" data-semantic-owns="71 75"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="71" data-semantic-parent="76"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="75" data-semantic-children="72,74" data-semantic-content="73" data-semantic-parent="76" data-semantic-owns="72 73 74"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="75"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="73" data-semantic-parent="75" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="75"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="77" data-semantic-parent="78" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>and estimate the constant <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c equals log left parenthesis lamda divided by upper Gamma left parenthesis nu plus 1 right parenthesis plus mu right parenthesis" tabindex="0" ctxtmenu_counter="40" style="font-size: 130%;"><mjx-math data-semantic-structure="(24 0 1 (23 2 22 (21 3 (19 4 5 (18 (17 6 16 (13 7 (11 8 9 10) 12)) 14 15)) 20)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="24" data-semantic-children="0,23" data-semantic-content="1" data-semantic-owns="0 1 23" data-semantic-speech="c equals log left parenthesis lamda divided by upper Gamma left parenthesis nu plus 1 right parenthesis plus mu right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="24"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="24" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="23" data-semantic-children="2,21" data-semantic-content="22,2" data-semantic-parent="24" data-semantic-owns="2 22 21"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="2" data-semantic-parent="23" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="21" data-semantic-children="19" data-semantic-content="3,20" data-semantic-parent="23" data-semantic-owns="3 19 20"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="19" data-semantic-children="4,18" data-semantic-content="5" data-semantic-parent="21" data-semantic-owns="4 5 18"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="19"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="5" data-semantic-parent="19" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="18" data-semantic-children="17,15" data-semantic-content="14" data-semantic-parent="19" data-semantic-owns="17 14 15"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-children="6,13" data-semantic-content="16" data-semantic-parent="18" data-semantic-owns="6 16 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="17"><mjx-c class="mjx-c1D6E4">𝛤</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="13" data-semantic-children="11" data-semantic-content="7,12" data-semantic-parent="17" data-semantic-owns="7 11 12"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="11" data-semantic-children="8,10" data-semantic-content="9" data-semantic-parent="13" data-semantic-owns="8 9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="9" data-semantic-parent="11" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="12" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="18" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="18"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="20" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> through plug-in estimation. For fixed <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda greater than 0" tabindex="0" ctxtmenu_counter="41" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="lamda greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, we then use the estimator <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="ModifyingAbove mu With ˆ equals e Superscript ModifyingAbove c With ˆ Baseline minus lamda divided by upper Gamma left parenthesis ModifyingAbove nu With ˆ plus 1 right parenthesis" tabindex="0" ctxtmenu_counter="42" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (2 0 1) 3 (25 (8 4 (7 5 6)) 9 (24 10 11 (23 12 22 (21 13 (19 (16 14 15) 17 18) 20)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="26" data-semantic-children="2,25" data-semantic-content="3" data-semantic-owns="2 3 25" data-semantic-speech="ModifyingAbove mu With ˆ equals e Superscript ModifyingAbove c With ˆ Baseline minus lamda divided by upper Gamma left parenthesis ModifyingAbove nu With ˆ plus 1 right parenthesis"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="26" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.349em; margin-bottom: -0.572em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="1" data-semantic-parent="2" style="width: 0px; margin-left: -0.322em;"><mjx-c class="mjx-c302 MM-SO">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="26" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="25" data-semantic-children="8,24" data-semantic-content="9" data-semantic-parent="26" data-semantic-owns="8 9 24"><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="4,7" data-semantic-parent="25" data-semantic-owns="4 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mover size="s" data-semantic-type="overscore" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.582em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="6" data-semantic-parent="7" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-script></mjx-msup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="9" data-semantic-parent="25" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="24" data-semantic-children="10,23" data-semantic-content="11" data-semantic-parent="25" data-semantic-owns="10 11 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="24"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="11" data-semantic-parent="24" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="23" data-semantic-children="12,21" data-semantic-content="22" data-semantic-parent="24" data-semantic-owns="12 22 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="23"><mjx-c class="mjx-c1D6E4">𝛤</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="21" data-semantic-children="19" data-semantic-content="13,20" data-semantic-parent="23" data-semantic-owns="13 19 20"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="19" data-semantic-children="16,18" data-semantic-content="17" data-semantic-parent="21" data-semantic-owns="16 17 18"><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="19" data-semantic-owns="14 15"><mjx-over style="padding-bottom: 0.105em; padding-left: 0.561em; margin-bottom: -0.587em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:unknown" data-semantic-id="15" data-semantic-parent="16" style="width: 0px; margin-left: 0px;"><mjx-c class="mjx-c302">̂</mjx-c></mjx-mo></mjx-over><mjx-base><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-base></mjx-mover><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="17" data-semantic-parent="19" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="20" data-semantic-parent="21" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0650" role="paragraph">Following <span class="dropBlock reference-citations"><a id="body-ref-sref7-2" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7-2" aria-controls="sref7-2">Cahoy et&nbsp;al. (2010)</a><div class="dropBlock__holder" data-db-target-of="sref7-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Cahoy, D.O. ∙ Uchaikin, V.V. ∙ Woyczynski, W.A.</div><div><strong>Parameter estimation for fractional Poisson processes</strong></div><div><em>J.&nbsp;Stat. Plan. Inference.</em> 2010; <strong>140</strong>:3106-3120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jspi.2010.04.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77954030633" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jspi.2010.04.016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, we simulate the inter-arrival times of the fractional Poisson process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="43" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> with rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda greater than 0" tabindex="0" ctxtmenu_counter="44" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="lamda greater than 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> by<div id="ufd9" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper T equals Overscript d Endscripts StartFraction StartAbsoluteValue log left parenthesis upper U 1 right parenthesis EndAbsoluteValue Superscript 1 divided by nu Baseline Over lamda Superscript 1 divided by nu Baseline EndFraction StartFraction sine left parenthesis nu pi upper U 2 right parenthesis left parenthesis sine left parenthesis left parenthesis 1 minus nu right parenthesis pi upper U 2 right parenthesis right parenthesis Superscript 1 divided by nu minus 1 Baseline Over left parenthesis sine left parenthesis pi upper U 2 right parenthesis right parenthesis Superscript 1 divided by nu Baseline StartAbsoluteValue log left parenthesis upper U 3 right parenthesis EndAbsoluteValue Superscript 1 divided by nu minus 1 Baseline EndFraction comma" tabindex="0" ctxtmenu_counter="45" style="font-size: 130%;"><mjx-math data-semantic-structure="(122 (121 0 (3 1 2) (120 (27 (20 (15 4 (13 5 12 (11 6 (9 7 8) 10)) 14) (19 16 17 18)) (26 21 (25 22 23 24))) 119 (117 (74 (72 28 71 (39 29 (37 30 35 31 36 (34 32 33)) 38)) 73 (70 (62 40 (60 41 59 (58 42 (56 (49 43 (47 44 45 46) 48) 54 50 55 (53 51 52)) 57)) 61) (69 63 64 (68 65 66 67)))) (116 (94 (89 75 (87 76 86 (85 77 (83 78 82 (81 79 80)) 84)) 88) (93 90 91 92)) 115 (114 (106 95 (104 96 103 (102 97 (100 98 99) 101)) 105) (113 107 108 (112 109 110 111))))))) 118)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="122" data-semantic-children="121,118" data-semantic-content="118" data-semantic-owns="121 118" data-semantic-speech="upper T equals Overscript d Endscripts StartFraction StartAbsoluteValue log left parenthesis upper U 1 right parenthesis EndAbsoluteValue Superscript 1 divided by nu Baseline Over lamda Superscript 1 divided by nu Baseline EndFraction StartFraction sine left parenthesis nu pi upper U 2 right parenthesis left parenthesis sine left parenthesis left parenthesis 1 minus nu right parenthesis pi upper U 2 right parenthesis right parenthesis Superscript 1 divided by nu minus 1 Baseline Over left parenthesis sine left parenthesis pi upper U 2 right parenthesis right parenthesis Superscript 1 divided by nu Baseline StartAbsoluteValue log left parenthesis upper U 3 right parenthesis EndAbsoluteValue Superscript 1 divided by nu minus 1 Baseline EndFraction comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="121" data-semantic-children="0,120" data-semantic-content="3" data-semantic-parent="122" data-semantic-owns="0 3 120"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="121"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mover space="4" data-semantic-type="overscore" data-semantic-role="equality" data-semantic-embellished="relation" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="121" data-semantic-operator="relseq,=" data-semantic-owns="1 2"><mjx-over style="padding-bottom: 0.105em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-over><mjx-base style="padding-left: 0.053em;"><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo></mjx-base></mjx-mover><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="120" data-semantic-children="27,117" data-semantic-content="119" data-semantic-parent="121" data-semantic-owns="27 119 117"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="27" data-semantic-children="20,26" data-semantic-parent="120" data-semantic-owns="20 26"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-msup size="s" data-semantic-type="superscript" data-semantic-role="neutral" data-semantic-id="20" data-semantic-children="15,19" data-semantic-parent="27" data-semantic-owns="15 19"><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="15" data-semantic-children="13" data-semantic-content="4,14" data-semantic-parent="20" data-semantic-owns="4 13 14"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="4" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="13" data-semantic-children="5,11" data-semantic-content="12,5" data-semantic-parent="15" data-semantic-owns="5 12 11"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="5" data-semantic-parent="13" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="11" data-semantic-children="9" data-semantic-content="6,10" data-semantic-parent="13" data-semantic-owns="6 9 10"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="6" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="11" data-semantic-owns="7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="10" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.477em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="19" data-semantic-children="16,18" data-semantic-content="17" data-semantic-parent="20" data-semantic-owns="16 17 18"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="19"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="17" data-semantic-parent="19" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msup size="s" data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="26" data-semantic-children="21,25" data-semantic-parent="27" data-semantic-owns="21 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="26"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.289em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="25" data-semantic-children="22,24" data-semantic-content="23" data-semantic-parent="26" data-semantic-owns="22 23 24"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="23" data-semantic-parent="25" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="25"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="119" data-semantic-parent="120" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="117" data-semantic-children="74,116" data-semantic-parent="120" data-semantic-owns="74 116"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="74" data-semantic-children="72,70" data-semantic-content="73" data-semantic-parent="117" data-semantic-owns="72 73 70"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="72" data-semantic-children="28,39" data-semantic-content="71,28" data-semantic-parent="74" data-semantic-owns="28 71 39"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="28" data-semantic-parent="72" data-semantic-operator="appl"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.657em;">s</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.657em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="71" data-semantic-parent="72" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="39" data-semantic-children="37" data-semantic-content="29,38" data-semantic-parent="72" data-semantic-owns="29 37 38"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="29" data-semantic-parent="39" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="37" data-semantic-children="30,31,34" data-semantic-content="35,36" data-semantic-parent="39" data-semantic-owns="30 35 31 36 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="37"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="37"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="32,33" data-semantic-parent="37" data-semantic-owns="32 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="34"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="34"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="38" data-semantic-parent="39" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="73" data-semantic-parent="74" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="70" data-semantic-children="62,69" data-semantic-parent="74" data-semantic-owns="62 69"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="62" data-semantic-children="60" data-semantic-content="40,61" data-semantic-parent="70" data-semantic-owns="40 60 61"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="40" data-semantic-parent="62" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="60" data-semantic-children="41,58" data-semantic-content="59,41" data-semantic-parent="62" data-semantic-owns="41 59 58"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="41" data-semantic-parent="60" data-semantic-operator="appl"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.657em;">s</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.657em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="59" data-semantic-parent="60" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="58" data-semantic-children="56" data-semantic-content="42,57" data-semantic-parent="60" data-semantic-owns="42 56 57"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="42" data-semantic-parent="58" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="56" data-semantic-children="49,50,53" data-semantic-content="54,55" data-semantic-parent="58" data-semantic-owns="49 54 50 55 53"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="49" data-semantic-children="47" data-semantic-content="43,48" data-semantic-parent="56" data-semantic-owns="43 47 48"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="43" data-semantic-parent="49" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="47" data-semantic-children="44,46" data-semantic-content="45" data-semantic-parent="49" data-semantic-owns="44 45 46"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="47"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="45" data-semantic-parent="47" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="46" data-semantic-parent="47"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="48" data-semantic-parent="49" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="54" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="56"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="53" data-semantic-children="51,52" data-semantic-parent="56" data-semantic-owns="51 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="53"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="53"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="57" data-semantic-parent="58" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="61" data-semantic-parent="62" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="69" data-semantic-children="63,68" data-semantic-content="64" data-semantic-parent="70" data-semantic-owns="63 64 68"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="69"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="64" data-semantic-parent="69" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="68" data-semantic-children="65,67" data-semantic-content="66" data-semantic-parent="69" data-semantic-owns="65 66 67"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="65" data-semantic-parent="68"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="66" data-semantic-parent="68" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="68"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="116" data-semantic-children="94,114" data-semantic-content="115" data-semantic-parent="117" data-semantic-owns="94 115 114"><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="94" data-semantic-children="89,93" data-semantic-parent="116" data-semantic-owns="89 93"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="89" data-semantic-children="87" data-semantic-content="75,88" data-semantic-parent="94" data-semantic-owns="75 87 88"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="75" data-semantic-parent="89" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="87" data-semantic-children="76,85" data-semantic-content="86,76" data-semantic-parent="89" data-semantic-owns="76 86 85"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="76" data-semantic-parent="87" data-semantic-operator="appl"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.657em;">s</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.657em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="86" data-semantic-parent="87" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="85" data-semantic-children="83" data-semantic-content="77,84" data-semantic-parent="87" data-semantic-owns="77 83 84"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="77" data-semantic-parent="85" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="83" data-semantic-children="78,81" data-semantic-content="82" data-semantic-parent="85" data-semantic-owns="78 82 81"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="78" data-semantic-parent="83"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="82" data-semantic-parent="83" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="81" data-semantic-children="79,80" data-semantic-parent="83" data-semantic-owns="79 80"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="79" data-semantic-parent="81"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="80" data-semantic-parent="81"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="84" data-semantic-parent="85" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="88" data-semantic-parent="89" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="93" data-semantic-children="90,92" data-semantic-content="91" data-semantic-parent="94" data-semantic-owns="90 91 92"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="90" data-semantic-parent="93"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="91" data-semantic-parent="93" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="92" data-semantic-parent="93"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="115" data-semantic-parent="116" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="neutral" data-semantic-id="114" data-semantic-children="106,113" data-semantic-parent="116" data-semantic-owns="106 113"><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="106" data-semantic-children="104" data-semantic-content="95,105" data-semantic-parent="114" data-semantic-owns="95 104 105"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="95" data-semantic-parent="106" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="104" data-semantic-children="96,102" data-semantic-content="103,96" data-semantic-parent="106" data-semantic-owns="96 103 102"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="96" data-semantic-parent="104" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="103" data-semantic-parent="104" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="102" data-semantic-children="100" data-semantic-content="97,101" data-semantic-parent="104" data-semantic-owns="97 100 101"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="97" data-semantic-parent="102" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="100" data-semantic-children="98,99" data-semantic-parent="102" data-semantic-owns="98 99"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="98" data-semantic-parent="100"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="99" data-semantic-parent="100"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="101" data-semantic-parent="102" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="105" data-semantic-parent="106" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.477em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="113" data-semantic-children="107,112" data-semantic-content="108" data-semantic-parent="114" data-semantic-owns="107 108 112"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="107" data-semantic-parent="113"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="108" data-semantic-parent="113" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="112" data-semantic-children="109,111" data-semantic-content="110" data-semantic-parent="113" data-semantic-owns="109 110 111"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="109" data-semantic-parent="112"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="110" data-semantic-parent="112" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="111" data-semantic-parent="112"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="118" data-semantic-parent="122" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>for independent <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper U 1 comma upper U 2 comma upper U 3 tilde script upper U left bracket 0 comma 1 right bracket" tabindex="0" ctxtmenu_counter="46" style="font-size: 130%;"><mjx-math data-semantic-structure="(20 (2 0 1) 3 (6 4 5) 7 (19 (10 8 9) 11 (18 12 17 (16 13 14 15))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="20" data-semantic-children="2,3,6,7,19" data-semantic-content="3,7" data-semantic-owns="2 3 6 7 19" data-semantic-speech="upper U 1 comma upper U 2 comma upper U 3 tilde script upper U left bracket 0 comma 1 right bracket"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="20" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="20" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="20" data-semantic-owns="4 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="20" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="19" data-semantic-children="10,18" data-semantic-content="11" data-semantic-parent="20" data-semantic-owns="10 11 18"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="19" data-semantic-owns="8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.077em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c33">3</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="11" data-semantic-parent="19" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-children="12,16" data-semantic-content="17,12" data-semantic-parent="19" data-semantic-owns="12 17 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="script" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="18" data-semantic-operator="appl"><mjx-c class="mjx-c1D4B0 MM-S">𝒰</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="16" data-semantic-children="14" data-semantic-content="13,15" data-semantic-parent="18" data-semantic-owns="13 14 15"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="13" data-semantic-parent="16" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c2C" noic="true" style="padding-top: 0.666em;">,</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="15" data-semantic-parent="16" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. We use this fact to simulate the paths of a fractional compound Poisson process.</div><div id="p0655" role="paragraph">Note that the log-log-linear MSD of a compound Poisson process follows as a special case with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="nu equals 1 period" tabindex="0" ctxtmenu_counter="47" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 (4 0 1 2) 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="5" data-semantic-children="4,3" data-semantic-content="3" data-semantic-owns="4 3" data-semantic-speech="nu equals 1 period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="4" data-semantic-children="0,2" data-semantic-content="1" data-semantic-parent="5" data-semantic-owns="0 1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D708">𝜈</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></section><section id="sec-4-5-37"><h4>Fractional Brownian Motion</h4><div id="p0660" role="paragraph">Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper B Superscript upper H Baseline equals left parenthesis upper B Subscript t Superscript upper H Baseline right parenthesis Subscript t greater than or equals 0" tabindex="0" ctxtmenu_counter="48" style="font-size: 130%;"><mjx-math data-semantic-structure="(17 (2 0 1) 3 (16 (11 4 (9 (8 5 6) 7) 10) (15 12 13 14)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="17" data-semantic-children="2,16" data-semantic-content="3" data-semantic-owns="2 3 16" data-semantic-speech="upper B Superscript upper H Baseline equals left parenthesis upper B Subscript t Superscript upper H Baseline right parenthesis Subscript t greater than or equals 0"><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="17" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-script></mjx-msup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="17" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="leftright" data-semantic-id="16" data-semantic-children="11,15" data-semantic-parent="17" data-semantic-owns="11 15"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="11" data-semantic-children="9" data-semantic-content="4,10" data-semantic-parent="16" data-semantic-owns="4 9 10"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="5,6,7" data-semantic-parent="11" data-semantic-collapsed="(9 (8 5 6) 7)" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.26em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="10" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: -0.332em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="15" data-semantic-children="12,14" data-semantic-content="13" data-semantic-parent="16" data-semantic-owns="12 13 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="15"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="13" data-semantic-parent="15" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="15"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> be a fractional Brownian motion with Hurst parameter <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper H element of left parenthesis 0 comma 1 right parenthesis" tabindex="0" ctxtmenu_counter="49" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 1 (5 2 3 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="6" data-semantic-children="0,5" data-semantic-content="1" data-semantic-owns="0 1 5" data-semantic-speech="upper H element of left parenthesis 0 comma 1 right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="2,4" data-semantic-parent="6" data-semantic-owns="2 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="general:basenumber;clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c2C" noic="true" style="padding-top: 0.666em;">,</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Then the MSD of the process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Z Subscript t Baseline equals mu t Superscript upper H Baseline plus upper B Subscript t" tabindex="0" ctxtmenu_counter="50" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 (2 0 1) 3 (14 (13 4 12 (7 5 6)) 8 (11 9 10)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="15" data-semantic-children="2,14" data-semantic-content="3" data-semantic-owns="2 3 14" data-semantic-speech="upper Z Subscript t Baseline equals mu t Superscript upper H Baseline plus upper B Subscript t"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="15" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.04em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="15" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="14" data-semantic-children="13,11" data-semantic-content="8" data-semantic-parent="15" data-semantic-owns="13 8 11"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="13" data-semantic-children="4,7" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="4 12 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="13"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="13" data-semantic-owns="5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="8" data-semantic-parent="14" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="14" data-semantic-owns="9 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t greater than or equals 0" tabindex="0" ctxtmenu_counter="51" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t greater than or equals 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> is given by<div id="ufd10" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket equals mu squared t Superscript 2 upper H Baseline plus 2 mu t Superscript upper H Baseline double struck upper E left bracket upper B Subscript t Baseline right bracket plus double struck upper E left bracket upper B Subscript t Superscript 2 Baseline right bracket equals left parenthesis mu squared plus 1 right parenthesis t Superscript 2 upper H Baseline period" tabindex="0" ctxtmenu_counter="52" style="font-size: 130%;"><mjx-math data-semantic-structure="(75 (74 (64 0 63 (8 1 (6 (5 2 3) 4) 7)) 9 (71 (66 (12 10 11) 65 (18 13 (17 14 16 15))) 19 (70 20 67 21 68 (24 22 23) 69 (62 25 61 (31 26 (29 27 28) 30))) 32 (60 33 59 (41 34 (39 (38 35 36) 37) 40))) 42 (73 (51 43 (49 (46 44 45) 47 48) 50) 72 (57 52 (56 53 55 54)))) 58)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="75" data-semantic-children="74,58" data-semantic-content="58" data-semantic-owns="74 58" data-semantic-speech="double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket equals mu squared t Superscript 2 upper H Baseline plus 2 mu t Superscript upper H Baseline double struck upper E left bracket upper B Subscript t Baseline right bracket plus double struck upper E left bracket upper B Subscript t Superscript 2 Baseline right bracket equals left parenthesis mu squared plus 1 right parenthesis t Superscript 2 upper H Baseline period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="74" data-semantic-children="64,71,73" data-semantic-content="9,42" data-semantic-parent="75" data-semantic-owns="64 9 71 42 73"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="64" data-semantic-children="0,8" data-semantic-content="63,0" data-semantic-parent="74" data-semantic-owns="0 63 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="64" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="63" data-semantic-parent="64" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="1,7" data-semantic-parent="64" data-semantic-owns="1 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-SO">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.26em; margin-left: -0.04em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6" style="margin-left: 0.13em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-SO">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="74" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="71" data-semantic-children="66,70,60" data-semantic-content="19,32" data-semantic-parent="74" data-semantic-owns="66 19 70 32 60"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="66" data-semantic-children="12,18" data-semantic-content="65" data-semantic-parent="71" data-semantic-owns="12 65 18"><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="66" data-semantic-owns="10 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="65" data-semantic-parent="66" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="13,17" data-semantic-parent="66" data-semantic-owns="13 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="18"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="17" data-semantic-children="14,15" data-semantic-content="16" data-semantic-parent="18" data-semantic-owns="14 16 15"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="19" data-semantic-parent="71" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="70" data-semantic-children="20,21,24,62" data-semantic-content="67,68,69" data-semantic-parent="71" data-semantic-owns="20 67 21 68 24 69 62"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="70"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="67" data-semantic-parent="70" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="70"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="68" data-semantic-parent="70" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="22,23" data-semantic-parent="70" data-semantic-owns="22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="69" data-semantic-parent="70" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="62" data-semantic-children="25,31" data-semantic-content="61,25" data-semantic-parent="70" data-semantic-owns="25 61 31"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="62" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="61" data-semantic-parent="62" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="31" data-semantic-children="29" data-semantic-content="26,30" data-semantic-parent="62" data-semantic-owns="26 29 30"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="26" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="31" data-semantic-owns="27 28"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="29"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="30" data-semantic-parent="31" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="32" data-semantic-parent="71" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="60" data-semantic-children="33,41" data-semantic-content="59,33" data-semantic-parent="71" data-semantic-owns="33 59 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="60" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="59" data-semantic-parent="60" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="41" data-semantic-children="39" data-semantic-content="34,40" data-semantic-parent="60" data-semantic-owns="34 39 40"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="34" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-SO">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="39" data-semantic-children="35,36,37" data-semantic-parent="41" data-semantic-collapsed="(39 (38 35 36) 37)" data-semantic-owns="35 36 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="39"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.26em; margin-left: 0px;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="39"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="39"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="40" data-semantic-parent="41" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-SO">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="42" data-semantic-parent="74" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="73" data-semantic-children="51,57" data-semantic-content="72" data-semantic-parent="74" data-semantic-owns="51 72 57"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="51" data-semantic-children="49" data-semantic-content="43,50" data-semantic-parent="73" data-semantic-owns="43 49 50"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="43" data-semantic-parent="51" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-SO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="49" data-semantic-children="46,48" data-semantic-content="47" data-semantic-parent="51" data-semantic-owns="46 47 48"><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="46" data-semantic-children="44,45" data-semantic-parent="49" data-semantic-owns="44 45"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="46"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="45" data-semantic-parent="46"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msup><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="47" data-semantic-parent="49" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="48" data-semantic-parent="49"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="50" data-semantic-parent="51" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-SO">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="72" data-semantic-parent="73" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup space="2" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="57" data-semantic-children="52,56" data-semantic-parent="73" data-semantic-owns="52 56"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="57"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="56" data-semantic-children="53,54" data-semantic-content="55" data-semantic-parent="57" data-semantic-owns="53 55 54"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="56"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="55" data-semantic-parent="56" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="54" data-semantic-parent="56"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="58" data-semantic-parent="75" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div></div><div id="p0665" role="paragraph">Therefore, the MSD of the process <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Z" tabindex="0" ctxtmenu_counter="53" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper Z"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is again log-log-linear. We simulate trajectories of this process by generating a sample of a multivariate normal distribution with the appropriate covariance structure.</div></section><section id="sec-4-5-38"><h4>Other Gaussian processes</h4><div id="p0670" role="paragraph">In general, the Karamata Representation Theorem (see, e.g., Theorem 1.5 in <span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Geluk and Haan, 1987</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Geluk, J.L. ∙ Haan, L.F.M.d.</div><div><strong>Regular Variation, Extensions and Tauberian Theorems</strong></div><div>CWI, 1987</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.GelukL.F.M.d.HaanRegular+Variation%2C+Extensions+and+Tauberian+Theorems1987CWI" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) yields that for any process with deterministic start point <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X 0" tabindex="0" ctxtmenu_counter="54" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper X 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and second moments of the form<div id="ufd11" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket equals c left parenthesis t right parenthesis exp left parenthesis integral Subscript 0 Superscript t Baseline StartFraction a left parenthesis s right parenthesis Over s EndFraction d s right parenthesis comma" tabindex="0" ctxtmenu_counter="55" style="font-size: 130%;"><mjx-math data-semantic-structure="(50 (49 (46 0 45 (8 1 (6 (5 2 3) 4) 7)) 9 (48 (44 10 43 (14 11 12 13)) 47 (42 15 41 (39 16 (37 (20 17 18 19) (29 (27 21 26 (25 22 23 24)) 28) (36 31 32)) 38)))) 40)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="50" data-semantic-children="49,40" data-semantic-content="40" data-semantic-owns="49 40" data-semantic-speech="double struck upper E left bracket upper Z Subscript t Superscript 2 Baseline right bracket equals c left parenthesis t right parenthesis exp left parenthesis integral Subscript 0 Superscript t Baseline StartFraction a left parenthesis s right parenthesis Over s EndFraction d s right parenthesis comma"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="49" data-semantic-children="46,48" data-semantic-content="9" data-semantic-parent="50" data-semantic-owns="46 9 48"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="46" data-semantic-children="0,8" data-semantic-content="45,0" data-semantic-parent="49" data-semantic-owns="0 45 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="double-struck" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="46" data-semantic-operator="appl"><mjx-c class="mjx-c1D53C MM-DS">𝔼</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="1,7" data-semantic-parent="46" data-semantic-owns="1 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5B MM-SO">[</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="8" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D44D">𝑍</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.26em; margin-left: -0.04em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6" style="margin-left: 0.13em;"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c5D MM-SO">]</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="9" data-semantic-parent="49" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="48" data-semantic-children="44,42" data-semantic-content="47" data-semantic-parent="49" data-semantic-owns="44 47 42"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-children="10,14" data-semantic-content="43,10" data-semantic-parent="48" data-semantic-owns="10 43 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="44" data-semantic-operator="appl"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="11,13" data-semantic-parent="44" data-semantic-owns="11 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="42" data-semantic-children="15,39" data-semantic-content="41,15" data-semantic-parent="48" data-semantic-owns="15 41 39"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-annotation="general:text" data-semantic-id="15" data-semantic-parent="42" data-semantic-operator="appl"><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.448em;">e</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.448em;">x</mjx-c><mjx-c class="mjx-c70" style="padding-top: 0.448em;">p</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="39" data-semantic-children="37" data-semantic-content="16,38" data-semantic-parent="42" data-semantic-owns="16 37 38"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="16" data-semantic-parent="39" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-S6">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="integral" data-semantic-role="integral" data-semantic-id="37" data-semantic-children="20,29,36" data-semantic-content="17" data-semantic-parent="39" data-semantic-owns="20 29 36"><mjx-munderover data-semantic-type="limboth" data-semantic-role="integral" data-semantic-id="20" data-semantic-children="17,18,19" data-semantic-parent="37" data-semantic-owns="17 18 19"><mjx-over style="padding-bottom: 0.192em; padding-left: 0.295em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-over><mjx-box><mjx-munder><mjx-row><mjx-base><mjx-mo data-semantic-type="largeop" data-semantic-role="integral" data-semantic-id="17" data-semantic-parent="20" data-semantic-operator="integral"><mjx-c class="mjx-c222B">∫</mjx-c></mjx-mo></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em; padding-left: 0.066em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-under></mjx-row></mjx-munder></mjx-box></mjx-munderover><mjx-mrow space="2"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="37" data-semantic-owns="27 28"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-children="21,25" data-semantic-content="26,21" data-semantic-parent="29" data-semantic-owns="21 26 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="27" data-semantic-operator="appl"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="25" data-semantic-children="23" data-semantic-content="22,24" data-semantic-parent="27" data-semantic-owns="22 23 24"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="22" data-semantic-parent="25" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="24" data-semantic-parent="25" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="integral" data-semantic-font="normal" data-semantic-id="36" data-semantic-children="32" data-semantic-content="31" data-semantic-parent="37" data-semantic-owns="31 32"><mjx-mtext data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text" data-semantic-id="31" data-semantic-parent="36" data-semantic-operator="prefixop,d"><mjx-c class="mjx-c64">d</mjx-c></mjx-mtext><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="36"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="38" data-semantic-parent="39" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-S6">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="40" data-semantic-parent="50" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>for all <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t greater than or equals 0" tabindex="0" ctxtmenu_counter="56" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t greater than or equals 0"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,≥"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> and functions <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c comma a colon left parenthesis 0 comma infinity right parenthesis right arrow double struck upper R" tabindex="0" ctxtmenu_counter="57" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 1 2 3 (13 (10 4 (8 5 6 7) 9) 11 12))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="14" data-semantic-children="0,1,2,3,13" data-semantic-content="1,3" data-semantic-owns="0 1 2 3 13" data-semantic-speech="c comma a colon left parenthesis 0 comma infinity right parenthesis right arrow double struck upper R"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="1" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="14"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="3" data-semantic-parent="14" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="arrow" data-semantic-id="13" data-semantic-children="10,12" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="10 11 12"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="4,9" data-semantic-parent="13" data-semantic-owns="4 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="8" data-semantic-children="5,6,7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="5 6 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c221E MM-I">∞</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="relseq,→"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="numbersetletter" data-semantic-font="double-struck" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c211D MM-DS">ℝ</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container> with existing limits <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="limit Subscript t right arrow normal infinity Baseline c left parenthesis t right parenthesis element of left parenthesis 0 comma normal infinity right parenthesis comma limit Subscript t right arrow normal infinity Baseline a left parenthesis t right parenthesis equals colon alpha greater than 0" tabindex="0" ctxtmenu_counter="58" style="font-size: 130%;"><mjx-math data-semantic-structure="(48 (40 (39 (5 0 (4 1 2 3)) 38 (37 6 36 (10 7 8 9))) 11 (18 12 (16 13 14 15) 17)) 19 (46 (44 (25 20 (24 21 22 23)) 43 (42 26 41 (30 27 28 29))) 31 45) 32 (47 33 34 35))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="48" data-semantic-children="40,19,46,32,47" data-semantic-content="19,32" data-semantic-owns="40 19 46 32 47" data-semantic-speech="limit Subscript t right arrow normal infinity Baseline c left parenthesis t right parenthesis element of left parenthesis 0 comma normal infinity right parenthesis comma limit Subscript t right arrow normal infinity Baseline a left parenthesis t right parenthesis equals colon alpha greater than 0"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="40" data-semantic-children="39,18" data-semantic-content="11" data-semantic-parent="48" data-semantic-owns="39 11 18"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="39" data-semantic-children="5,37" data-semantic-content="38" data-semantic-parent="40" data-semantic-owns="5 38 37"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="39" data-semantic-owns="0 4"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6D" style="padding-top: 0.694em;">m</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="arrow" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="relseq,→"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="infty" data-semantic-font="normal" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c221E">∞</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-children="6,10" data-semantic-content="36,6" data-semantic-parent="39" data-semantic-owns="6 36 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="37" data-semantic-operator="appl"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="7,9" data-semantic-parent="37" data-semantic-owns="7 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="11" data-semantic-parent="40" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="18" data-semantic-children="16" data-semantic-content="12,17" data-semantic-parent="40" data-semantic-owns="12 16 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="16" data-semantic-children="13,14,15" data-semantic-content="14" data-semantic-parent="18" data-semantic-owns="13 14 15"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="14" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-font="normal" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c221E">∞</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="19" data-semantic-parent="48" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="46" data-semantic-children="44,45" data-semantic-content="31" data-semantic-parent="48" data-semantic-owns="44 31 45"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="44" data-semantic-children="25,42" data-semantic-content="43" data-semantic-parent="46" data-semantic-owns="25 43 42"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="25" data-semantic-children="20,24" data-semantic-parent="44" data-semantic-owns="20 24"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-parent="25"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6D" style="padding-top: 0.694em;">m</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="arrow" data-semantic-id="24" data-semantic-children="21,23" data-semantic-content="22" data-semantic-parent="25" data-semantic-owns="21 22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="24"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="22" data-semantic-parent="24" data-semantic-operator="relseq,→"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="infty" data-semantic-font="normal" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c221E">∞</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-children="26,30" data-semantic-content="41,26" data-semantic-parent="44" data-semantic-owns="26 41 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="42" data-semantic-operator="appl"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="27,29" data-semantic-parent="42" data-semantic-owns="27 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="27" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="30"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="31" data-semantic-parent="46" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="45" data-semantic-parent="46"></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="32" data-semantic-parent="48" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="47" data-semantic-children="33,35" data-semantic-content="34" data-semantic-parent="48" data-semantic-owns="33 34 35"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="33" data-semantic-parent="47"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="34" data-semantic-parent="47" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="47"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, the MSD is asymptotically mononomial with exponent <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha" tabindex="0" ctxtmenu_counter="59" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="alpha"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, i.e., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left angle bracket upper X left parenthesis t right parenthesis right angle bracket tilde t Superscript alpha" tabindex="0" ctxtmenu_counter="60" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (9 0 (7 1 6 (5 2 3 4)) 8) 10 (13 11 12))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="9,13" data-semantic-content="10" data-semantic-owns="9 10 13" data-semantic-speech="left angle bracket upper X left parenthesis t right parenthesis right angle bracket tilde t Superscript alpha"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="0,8" data-semantic-parent="14" data-semantic-owns="0 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c27E8">⟨</mjx-c></mjx-mo><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="1,5" data-semantic-content="6,1" data-semantic-parent="9" data-semantic-owns="1 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="appl"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="5" data-semantic-children="3" data-semantic-content="2,4" data-semantic-parent="7" data-semantic-owns="2 3 4"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c27E9">⟩</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="10" data-semantic-parent="14" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="14" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t right arrow normal infinity" tabindex="0" ctxtmenu_counter="61" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="arrow" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t right arrow normal infinity"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="arrow" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,→"><mjx-c class="mjx-c2192">→</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="infty" data-semantic-font="normal" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c221E">∞</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, the characteristic of anomalous diffusion (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Metzler and Klafter, 2000</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Metzler, R. ∙ Klafter, J.</div><div><strong>The random walk's guide to anomalous diffusion: a fractional dynamics approach</strong></div><div><em>Phys. Rep.</em> 2000; <strong>339</strong>:1-77</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0370-1573(00)00070-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0370-1573%2800%2900070-3" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, it is straightforward to construct superdiffusive processes for which the MSD on a limited time scale is not log-log-linear. Yet, the empirically observed process has the desired property on the observation interval and, using the models from <span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">Gneiting and Schlather (2004)</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Gneiting, T. ∙ Schlather, M.</div><div><strong>Stochastic models that separate fractal dimension and the Hurst effect</strong></div><div><em>SIAM Rev.</em> 2004; <strong>46</strong>:269-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1137/S0036144501394387" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-3142710385" target="_blank" aria-label="Open Scopus in new tab">Scopus (316)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1137%2FS0036144501394387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, we investigate the estimation stability for the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha" tabindex="0" ctxtmenu_counter="62" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="alpha"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> index in small sampling sizes if the long-term scaling limit differs from the short-term behavior.</div></section><section id="sec-4-5-39"><h4>Numerical Experiments</h4><div id="p0675" role="paragraph">For the numerical analysis of the moment estimators for the MSD, we simulated 50 trajectories of the different stochastic processes and calculated the estimates for the intercept and slope in the log-log-linear relationship. This was repeated 1000 times and the mean and standard deviation of the respective values were calculated. The number of time steps 40 and the number of trajectories 50 were chosen to reflect the worst-case settings of the real-world experiment. We illustrate the results of the analysis in <a id="crosref2125" href="#fig4">Figure&nbsp;4</a>H. In general, it can be observed that for the processes in consideration the average intercept and slope only mildly deviates from the true values. The empirical standard deviation, however, depends on the respective process, with a larger variance for the estimates for the fractional compound Poisson process.</div></section></section><section id="sec-4-6"><h3>Quantification and statistical analysis</h3><div id="p0680" role="paragraph">The results of all quantifications were transferred to GraphPad Prism (GraphPad Software) or R to test the statistical significance with the appropriate tests (data were tested for normality using the D’Agostino &amp; Pearson normality). Statistical significance was assessed by the two-sided Students’ t-test for normally distributed data. Otherwise, a Mann–Whitney test was used for non-normal distributions. For more than 2 groups a one-way ANOVA test was used for normally distributed data and a Kruskal-Wallis test or the Friedman test (paired data) for non-normal distributed data. Results were considered statistically significant if the P value was below 0.05. Quantifications were done blinded by 2 independent investigators. Animal group sizes were as low as possible and empirically chosen and longitudinal measurements allowed a reduction of animal numbers by maintaining an adequate power. No statistical methods were used to predetermine sample size. Quantifications were depicted as dot plots with mean ± standard error of means or as violin plots.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0685" role="paragraph">V.V. received financial support from the German Research Foundation (DFG: VE1373/2-1), the Else Kröner-Fresenius-Stiftung (2020-EKEA.135) and the Medical Faculty of Heidelberg University (Physician-Scientist-Program, Krebs-und Scharlachstiftung). Y.Y., M.C.S., S.K.T., E.R., and D.I.T. were supported by the Deutsche Krebshilfe/German Cancer Aid (Mildred-Scheel-Scholarship for MD students). V.V., T. Kessler, W.W., and F.W. were supported by a grant from the German Research Foundation (DFG: SFB 1389). Y.A.Y., A.G., and S.P.N. were supported by GLIOTRAIN ITN H2020 grant no. 766069. We gratefully acknowledge the data storage service SDS@hd supported by the Ministry of Science, Research, and the Arts Baden-Württemberg (MWK).</div><div id="p0690" role="paragraph">We thank M. Kaiser, M. Schmitt, C. Kocksch, D. Domínguez Azorín, M. Karreman, S. Wendler, and C. Löb for technical assistance. We thank T. Chopurian and C. Strahle for initial contributions. We thank K. Kandere-Grzybowska and K. Polev for initial discussions. We thank F. Blum, T. Rubner, M. Eich, K. Hexel, and S. Schmitt from the Flow Cytometry Core Facility of the German Cancer Research Center for their assistance in performing FACS experiments. We thank K. Becker, K. Dell, K. Schmidt, and A. Berdel for providing support and assistance in animal care and design of animal experiments. We thank the EM Core Facility of the Heidelberg University, the EM and Microscopy Core Facility of the German Cancer Research center for their support.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0695" role="paragraph">Supervision, V.V., L.D., A.I., W.W., T. Kuner, and F.W.; conceptualization, V.V., T. Kuner, and F.W.; methodology, V.V., Y.Y., M.C.S., S.K.T., C.A.B., N.W., and M.R.; software, V.V., M.C.S., C.A.B., and N.W.; project administration, V.V.; investigation, V.V., Y.Y., M.C.S., S.K.T., M.B., L.F., E.R., N.W., R.L.P., J.W., S.J.S., D.C.H., D.I.T., Y.A.Y., A.G., S.P.N., Y.D., and S.H.; formal analysis, V.V., Y.Y., M.C.S., S.K.T., M.B., C.A.B., L.F., L.G., E.R., N.W., R.L.P., J.W., S.J.S., D.C.H., L.H., T. Kessler, D.I.T., Y.A.Y., A.G., S.P.N., Y.D., S.H., A.L., M.S., L.D., and A.I.; resources, M.R., W.W., T. Kuner, and F.W.; visualization, V.V., Y.Y., M.C.S., S.K.T., M.B., L.F., E.R., N.W., R.L.P., J.W., S.J.S., and D.I.T.; writing – original draft, V.V., T. Kuner, and F.W.; writing – review &amp; editing, V.V., Y.Y., M.C.S., S.K.T., M.B., L.F., E.R., N.W., R.L.P., J.W., S.J.S., D.I.T., M.S., W.W., A.I., T. Kuner, and F.W.; funding acquisition, V.V., W.W., T. Kuner, and F.W.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0700" role="paragraph">F.W. and W.W. report the patent (WO2017020982A1) “Agents for use in the treatment of glioma.” F.W. is co-founder of DC Europa Ltd (a company trading under the name Divide &amp; Conquer) that is developing new medicines for the treatment of glioma. Divide &amp; Conquer also provides research funding to F.W.’s lab under a research collaboration agreement.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (2)</h2><div id="p0710" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.054/attachment/66f8c130-d441-463d-ac9e-38bac7a6e1ec/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (10.84 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table&nbsp;S1. Molecular characterization of patient-derived cell lines and PDOX models used in this study, related to Figure&nbsp;1</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.054/attachment/2a20f685-5c53-4f52-98fb-6d3ed686764d/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (4.97 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table&nbsp;S2. Further molecular characterization of the invasivity score, related to Figure&nbsp;2</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0035" role="paragraph">Single-cell RNA-seq data used in this study are listed with accession numbers in the <a id="crosref0275" href="#sec-4-1">key resources table</a>.</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0040" role="paragraph">This paper does not report original code.</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0045" role="paragraph">Any additional information that support the findings of this study are available on request from the <a id="crosref0280" href="#sec-4-2-1">lead contact</a>.</div></div></div></div></div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Arnold, L.M.D. ∙ Laurens De, H.</div><div><strong>On the estimation of the extreme-value index and large quantile estimation</strong></div><div><em>Ann. Stat.</em> 1989; <strong>17</strong>:1795-1832</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.M.D.ArnoldH.Laurens+DeOn+the+estimation+of+the+extreme-value+index+and+large+quantile+estimationAnn.+Stat.17198917951832" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Berg, S. ∙ Kutra, D. ∙ Kroeger, T. ...</div><div><strong>ilastik: interactive machine learning for (bio)image analysis</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1226-1232</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0582-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075429634" target="_blank" aria-label="Open Scopus in new tab">Scopus (831)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0582-9&amp;pmid=31570887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Bhaduri, A. ∙ Di Lullo, E. ∙ Jung, D.N. ...</div><div><strong>Outer radial glia-like cancer stem cells contribute to heterogeneity of glioblastoma</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>26</strong>:48-63.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.stem.2019.11.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.stem.2019.11.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85076910021" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31901251/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.11.015&amp;pmid=31901251" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Bindocci, E. ∙ Savtchouk, I. ∙ Liaudet, N. ...</div><div><strong>Three-dimensional Ca(2+) imaging advances understanding of astrocyte biology</strong></div><div><em>Science.</em> 2017; <strong>356</strong></div><div>eaai8185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai8185" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85019912597" target="_blank" aria-label="Open Scopus in new tab">Scopus (174)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28522470/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai8185&amp;pmid=28522470" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bodi, E. ∙ Hurtado, S.P. ∙ Carvalho, M.A. ...</div><div><strong>Gap junctions in hematopoietic stroma control proliferation and differentiation of blood cell precursors</strong></div><div><em>An. Acad. Bras. Cienc.</em> 2004; <strong>76</strong>:743-756</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1590/S0001-37652004000400009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15558154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1590%2FS0001-37652004000400009&amp;pmid=15558154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Bruce, M.H.</div><div><strong>A simple general approach to inference about the tail of a distribution</strong></div><div><em>Ann. Stat.</em> 1975; <strong>3</strong>:1163-1174</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.H.BruceA+simple+general+approach+to+inference+about+the+tail+of+a+distributionAnn.+Stat.3197511631174" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7-1" title="View in article">7.</a></div><div>Cahoy, D.O. ∙ Uchaikin, V.V. ∙ Woyczynski, W.A.</div><div><strong>Parameter estimation for fractional Poisson processes</strong></div><div><em>J.&nbsp;Stat. Plan. Inference.</em> 2010; <strong>140</strong>:3106-3120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jspi.2010.04.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77954030633" target="_blank" aria-label="Open Scopus in new tab">Scopus (55)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jspi.2010.04.016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Cao, J. ∙ Spielmann, M. ∙ Qiu, X. ...</div><div><strong>The single-cell transcriptional landscape of mammalian organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:496-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0969-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85062419120" target="_blank" aria-label="Open Scopus in new tab">Scopus (1050)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0969-x&amp;pmid=30787437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Capper, D. ∙ Jones, D.T.W. ∙ Sill, M. ...</div><div><strong>DNA methylation-based classification of central nervous system tumours</strong></div><div><em>Nature.</em> 2018; <strong>555</strong>:469-474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature26000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85044262739" target="_blank" aria-label="Open Scopus in new tab">Scopus (1269)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29539639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature26000&amp;pmid=29539639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Cardona, A. ∙ Saalfeld, S. ∙ Schindelin, J. ...</div><div><strong>TrakEM2 software for neural circuit reconstruction</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e38011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0038011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862556352" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0038011&amp;pmid=22723842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Chaligne, R. ∙ Gaiti, F. ∙ Silverbush, D. ...</div><div><strong>Epigenetic encoding, heritability and plasticity of glioma transcriptional cell states</strong></div><div><em>Nat. Genet.</em> 2021; <strong>53</strong>:1469-1479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-021-00927-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85116387369" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34594037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-021-00927-7&amp;pmid=34594037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Chen, C.Y. ∙ Matt, L. ∙ Hell, J.W. ...</div><div><strong>Perampanel inhibition of AMPA receptor currents in cultured hippocampal neurons</strong></div><div><em>PLoS One.</em> 2014; <strong>9</strong>:e108021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0108021" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84907220225" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25229608/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0108021&amp;pmid=25229608" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Cuddapah, V.A. ∙ Robel, S. ∙ Watkins, S. ...</div><div><strong>A neurocentric perspective on glioma invasion</strong></div><div><em>Nat. Rev. Neurosci.</em> 2014; <strong>15</strong>:455-465</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrn3765" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84903198110" target="_blank" aria-label="Open Scopus in new tab">Scopus (506)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24946761/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrn3765&amp;pmid=24946761" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Dana, H. ∙ Sun, Y. ∙ Mohar, B. ...</div><div><strong>High-performance calcium sensors for imaging activity in neuronal populations and microcompartments</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:649-657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0435-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85067851088" target="_blank" aria-label="Open Scopus in new tab">Scopus (421)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31209382/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0435-6&amp;pmid=31209382" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Dirkse, A. ∙ Golebiewska, A. ∙ Buder, T. ...</div><div><strong>Stem cell-associated heterogeneity in Glioblastoma results from intrinsic tumor plasticity shaped by the microenvironment</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:1787</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-09853-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85064545608" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30992437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-09853-z&amp;pmid=30992437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Dondzillo, A. ∙ Sätzler, K. ∙ Horstmann, H. ...</div><div><strong>Targeted three-dimensional immunohistochemistry reveals localization of presynaptic proteins Bassoon and Piccolo in the rat calyx of Held before and after the onset of hearing</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2010; <strong>518</strong>:1008-1029</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.22260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77949344465" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20127803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.22260&amp;pmid=20127803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Dorshkind, K. ∙ Green, L. ∙ Godwin, A. ...</div><div><strong>Connexin-43 type gap-junctions mediate communication between bone-marrow stromal cells</strong></div><div><em>Blood.</em> 1993; <strong>82</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood.V82.1.38.bloodjournal82138" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8391877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood.V82.1.38.bloodjournal82138&amp;pmid=8391877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Drumm, M.R. ∙ Dixit, K.S. ∙ Grimm, S. ...</div><div><strong>Extensive brainstem infiltration, not mass effect, is a common feature of end-stage cerebral glioblastomas</strong></div><div><em>Neuro. Oncol.</em> 2020; <strong>22</strong>:470-479</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noz216" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85083544250" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31711239/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoz216&amp;pmid=31711239" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Dubbs, A. ∙ Guevara, J. ∙ Yuste, R.</div><div><strong>moco: fast Motion Correction for Calcium Imaging</strong></div><div><em>Front. Neuroinform.</em> 2016; <strong>10</strong>:6</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fninf.2016.00006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26909035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffninf.2016.00006&amp;pmid=26909035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="optdySj1QVrQC" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optdySj1QVrQC" title="View in article">20.</a></div><div>Fantuzzo, J.A. ∙ Mirabella, V.R. ∙ Hamod, A.H. ...</div><div><strong><i>Intellicount</i>: High-Throughput Quantification of Fluorescent Synaptic Protein Puncta by Machine Learning</strong></div><div><em>eNeuro.</em> 2017; <strong>4</strong>, ENEURO.0219-17.2017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/ENEURO.0219-17.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85039731634" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29218324/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FENEURO.0219-17.2017&amp;pmid=29218324" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">21.</a></div><div>Fine, H.A.</div><div><strong>Malignant gliomas: simplifying the complexity</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1650-1652</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075914397" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31792122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-1081&amp;pmid=31792122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="optW0r1Xl9RvZ" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optO7kvXMytHi" title="View in article">22.</a></div><div>Fogarty, M.J. ∙ Hammond, L.A. ∙ Kanjhan, R. ...</div><div><strong>A method for the three-dimensional reconstruction of Neurobiotin™-filled neurons and the location of their synaptic inputs</strong></div><div><em>Front. Neural Circuits.</em> 2013; <strong>7</strong>:153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncir.2013.00153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24101895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncir.2013.00153&amp;pmid=24101895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">23.</a></div><div>Friedl, P. ∙ Locker, J. ∙ Sahai, E. ...</div><div><strong>Classifying collective cancer cell invasion</strong></div><div><em>Nat. Cell Biol.</em> 2012; <strong>14</strong>:777-783</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2548" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84864886120" target="_blank" aria-label="Open Scopus in new tab">Scopus (635)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22854810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2548&amp;pmid=22854810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">24.</a></div><div>Garofano, L. ∙ Migliozzi, S. ∙ Oh, Y.T. ...</div><div><strong>Pathway-based classification of glioblastoma uncovers a mitochondrial subtype with therapeutic vulnerabilities</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:141-156</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00159-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100027397" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33681822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00159-4&amp;pmid=33681822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">25.</a></div><div>Ge, S.X. ∙ Jung, D. ∙ Yao, R.</div><div><strong>ShinyGO: a graphical gene-set enrichment tool for animals and plants</strong></div><div><em>Bioinformatics.</em> 2020; <strong>36</strong>:2628-2629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz931" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85084025997" target="_blank" aria-label="Open Scopus in new tab">Scopus (667)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz931&amp;pmid=31882993" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">26.</a></div><div>Geluk, J.L. ∙ Haan, L.F.M.d.</div><div><strong>Regular Variation, Extensions and Tauberian Theorems</strong></div><div>CWI, 1987</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.GelukL.F.M.d.HaanRegular+Variation%2C+Extensions+and+Tauberian+Theorems1987CWI" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">27.</a></div><div>Gleeson, J.G. ∙ Lin, P.T. ∙ Flanagan, L.A. ...</div><div><strong>Doublecortin is a microtubule-associated protein and is expressed widely by migrating neurons</strong></div><div><em>Neuron.</em> 1999; <strong>23</strong>:257-271</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2FS0896-6273%2800%2980778-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2FS0896-6273%2800%2980778-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0033152450" target="_blank" aria-label="Open Scopus in new tab">Scopus (1069)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10399933/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0896-6273%2800%2980778-3&amp;pmid=10399933" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">28.</a></div><div>Gneiting, T. ∙ Schlather, M.</div><div><strong>Stochastic models that separate fractal dimension and the Hurst effect</strong></div><div><em>SIAM Rev.</em> 2004; <strong>46</strong>:269-282</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1137/S0036144501394387" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-3142710385" target="_blank" aria-label="Open Scopus in new tab">Scopus (316)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1137%2FS0036144501394387" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">29.</a></div><div>Golebiewska, A. ∙ Hau, A.C. ∙ Oudin, A. ...</div><div><strong>Patient-derived organoids and orthotopic xenografts of primary and recurrent gliomas represent relevant patient avatars for precision oncology</strong></div><div><em>Acta Neuropathol.</em> 2020; <strong>140</strong>:919-949</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00401-020-02226-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85091876218" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33009951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00401-020-02226-7&amp;pmid=33009951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">30.</a></div><div>Gonzalez, G.A. ∙ Montminy, M.R.</div><div><strong>Cyclic-AMP stimulates somatostatin gene-transcription by phosphorylation of Creb at Serine-133</strong></div><div><em>Cell.</em> 1989; <strong>59</strong>:675-680</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2F0092-8674%2889%2990013-5&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2F0092-8674%2889%2990013-5&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0024445798" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2573431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0092-8674%2889%2990013-5&amp;pmid=2573431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">31.</a></div><div>Gritsenko, P. ∙ Leenders, W. ∙ Friedl, P.</div><div><strong>Recapitulating in&nbsp;vivo-like plasticity of glioma cell invasion along blood vessels and in astrocyte-rich stroma</strong></div><div><em>Histochem. Cell Biol.</em> 2017; <strong>148</strong>:395-406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00418-017-1604-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85027881035" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28825130/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00418-017-1604-2&amp;pmid=28825130" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">32.</a></div><div>Gritsenko, P.G. ∙ Atlasy, N. ∙ Dieteren, C.E.J. ...</div><div><strong>p120-catenin-dependent collective brain infiltration by glioma cell networks</strong></div><div><em>Nat. Cell Biol.</em> 2020; <strong>22</strong>:97-107</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-019-0443-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85077546068" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31907411/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-019-0443-x&amp;pmid=31907411" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">33.</a></div><div>Hai, L. ∙ Hoffmann, D.C. ∙ Mandelbaum, H. ...</div><div><strong>A connectivity signature for glioblastoma</strong></div><div>Preprint at <em>bioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.11.07.465791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85121392461" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.11.07.465791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">34.</a></div><div>Hara, T. ∙ Chanoch-Myers, R. ∙ Mathewson, N.D. ...</div><div><strong>Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma</strong></div><div><em>Cancer Cell.</em> 2021; <strong>39</strong>:779-792.e11</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.ccell.2021.05.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.ccell.2021.05.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34087162/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccell.2021.05.002&amp;pmid=34087162" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">35.</a></div><div>Horstmann, H. ∙ Körber, C. ∙ Sätzler, K. ...</div><div><strong>Serial section scanning electron microscopy (S3EM) on silicon wafers for ultra-structural volume imaging of cells and tissues</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e35172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0035172" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84859813369" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22523574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0035172&amp;pmid=22523574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">36.</a></div><div>Huda, S. ∙ Weigelin, B. ∙ Wolf, K. ...</div><div><strong>Levy-like movement patterns of metastatic cancer cells revealed in microfabricated systems and implicated in&nbsp;vivo</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:4539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06563-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85055771401" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30382086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06563-w&amp;pmid=30382086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">37.</a></div><div>Humphries, N.E. ∙ Weimerskirch, H. ∙ Queiroz, N. ...</div><div><strong>Foraging success of biological Levy flights recorded <i>in situ</i></strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:7169-7174</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1121201109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84860816827" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22529349/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1121201109&amp;pmid=22529349" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">38.</a></div><div>Iwadate, Y.</div><div><strong>Epithelial-mesenchymal transition in glioblastoma progression</strong></div><div><em>Oncol. Lett.</em> 2016; <strong>11</strong>:1615-1620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3892/ol.2016.4113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84958047483" target="_blank" aria-label="Open Scopus in new tab">Scopus (175)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26998052/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3892%2Fol.2016.4113&amp;pmid=26998052" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">39.</a></div><div>Jung, E. ∙ Osswald, M. ∙ Blaes, J. ...</div><div><strong>Tweety-homolog 1 drives brain colonization of gliomas</strong></div><div><em>J.&nbsp;Neurosci.</em> 2017; <strong>37</strong>:6837-6850</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.3532-16.2017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85025161941" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28607172/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.3532-16.2017&amp;pmid=28607172" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">40.</a></div><div>Jung, E. ∙ Osswald, M. ∙ Ratliff, M. ...</div><div><strong>Tumor cell plasticity, heterogeneity, and resistance in crucial microenvironmental niches in glioma</strong></div><div><em>Nat. Commun.</em> 2021; <strong>12</strong>:1014</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-021-21117-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100832371" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33579922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-021-21117-3&amp;pmid=33579922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">41.</a></div><div>Kantevari, S. ∙ Gordon, G.R.J. ∙ MacVicar, B.A. ...</div><div><strong>A practical guide to the synthesis and use of membrane-permeant acetoxymethyl esters of caged inositol polyphosphates</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:327-337</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2010.194" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-79952402173" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21372813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2010.194&amp;pmid=21372813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">42.</a></div><div>Lewis, W.H.</div><div><strong>Is mesenchyme a syncytium?</strong></div><div><em>Anat. Rec.</em> 1922; <strong>23</strong>:177-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ar.1090230204" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84944483553" target="_blank" aria-label="Open Scopus in new tab">Scopus (2)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Far.1090230204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">43.</a></div><div>Liang, W. ∙ Gao, R. ∙ Yang, M. ...</div><div><strong>MARCKSL1 promotes the proliferation, migration and invasion of lung adenocarcinoma cells</strong></div><div><em>Oncol. Lett.</em> 2020; <strong>19</strong>:2272-2280</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32194726/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=32194726" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">44.</a></div><div>Maere, S. ∙ Heymans, K. ∙ Kuiper, M.</div><div><strong>BiNGO: a cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:3448-3449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bti551" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-24044440971" target="_blank" aria-label="Open Scopus in new tab">Scopus (3127)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15972284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbti551&amp;pmid=15972284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">45.</a></div><div>Margineanu, D.G. ∙ Wülfert, E.</div><div><strong>Inhibition by levetiracetam of a non-GABA(A) receptor-associated epileptiform effect of bicuculline in rat hippocampus</strong></div><div><em>Br. J. Pharmacol.</em> 1997; <strong>122</strong>:1146-1150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sj.bjp.0701476" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0030734833" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9401779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsj.bjp.0701476&amp;pmid=9401779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">46.</a></div><div>Marín, O. ∙ Valiente, M. ∙ Ge, X. ...</div><div><strong>Guiding neuronal cell migrations</strong></div><div><em>Cold Spring Harbor Perspect. Biol.</em> 2010; <strong>2</strong>:a001834</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/cshperspect.a001834" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77957277593" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20182622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fcshperspect.a001834&amp;pmid=20182622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">47.</a></div><div>Martini, F.J. ∙ Valiente, M. ∙ López Bendito, G. ...</div><div><strong>Biased selection of leading process branches mediates chemotaxis during tangential neuronal migration</strong></div><div><em>Development.</em> 2009; <strong>136</strong>:41-50</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.025502" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-60749096087" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19060332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.025502&amp;pmid=19060332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">48.</a></div><div>McKhann, 2nd, G.M. ∙ D'Ambrosio, R. ∙ Janigro, D.</div><div><strong>Heterogeneity of astrocyte resting membrane potentials and intercellular coupling revealed by whole-cell and gramicidin-perforated patch recordings from cultured neocortical and hippocampal slice astrocytes</strong></div><div><em>J.&nbsp;Neurosci.</em> 1997; <strong>17</strong>:6850-6863</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.17-18-06850.1997" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9278520/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.17-18-06850.1997&amp;pmid=9278520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">49.</a></div><div>Meijering, E. ∙ Dzyubachyk, O. ∙ Smal, I.</div><div><strong>Methods for cell and particle tracking</strong></div><div><em>Methods Enzymol.</em> 2012; <strong>504</strong>:183-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/B978-0-12-391857-4.00009-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84855958004" target="_blank" aria-label="Open Scopus in new tab">Scopus (1017)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22264535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FB978-0-12-391857-4.00009-4&amp;pmid=22264535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">50.</a></div><div>Metzler, R. ∙ Klafter, J.</div><div><strong>The random walk's guide to anomalous diffusion: a fractional dynamics approach</strong></div><div><em>Phys. Rep.</em> 2000; <strong>339</strong>:1-77</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0370-1573(00)00070-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0370-1573%2800%2900070-3" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">51.</a></div><div>Middei, S. ∙ Houeland, G. ∙ Cavallucci, V. ...</div><div><strong>CREB is necessary for synaptic maintenance and learning-induced changes of the ampa receptor GluA1 subunit</strong></div><div><em>Hippocampus.</em> 2013; <strong>23</strong>:488-499</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/hipo.22108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84878192270" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23504989/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fhipo.22108&amp;pmid=23504989" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">52.</a></div><div>Nadarajah, B. ∙ Alifragis, P. ∙ Wong, R.O. ...</div><div><strong>Neuronal migration in the developing cerebral cortex: observations based on real-time imaging</strong></div><div><em>Cereb. Cortex.</em> 2003; <strong>13</strong>:607-611</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/cercor/13.6.607" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0038662777" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12764035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcercor%2F13.6.607&amp;pmid=12764035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">53.</a></div><div>Nadarajah, B. ∙ Brunstrom, J.E. ∙ Grutzendler, J. ...</div><div><strong>Two modes of radial migration in early development of the cerebral cortex</strong></div><div><em>Nat. Neurosci.</em> 2001; <strong>4</strong>:143-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/83967" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0035144045" target="_blank" aria-label="Open Scopus in new tab">Scopus (545)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11175874/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F83967&amp;pmid=11175874" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">54.</a></div><div>Neftel, C. ∙ Laffy, J. ∙ Filbin, M.G. ...</div><div><strong>An integrative model of cellular states, plasticity, and genetics for glioblastoma</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:835-849.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.06.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85071345728" target="_blank" aria-label="Open Scopus in new tab">Scopus (790)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.06.024&amp;pmid=31327527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">55.</a></div><div>Nimmerjahn, A. ∙ Kirchhoff, F. ∙ Kerr, J.N.D. ...</div><div><strong>Sulforhodamine 101 as a specific marker of astroglia in the neocortex in&nbsp;vivo</strong></div><div><em>Nat. Methods.</em> 2004; <strong>1</strong>:31-37</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-18744409106" target="_blank" aria-label="Open Scopus in new tab">Scopus (636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15782150/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth706&amp;pmid=15782150" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">56.</a></div><div>Ohtaka-Maruyama, C. ∙ Okamoto, M. ∙ Endo, K. ...</div><div><strong>Synaptic transmission from subplate neurons controls radial migration of neocortical neurons</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:313-317</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar2866" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85045556191" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29674592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar2866&amp;pmid=29674592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">57.</a></div><div>Osswald, M. ∙ Jung, E. ∙ Sahm, F. ...</div><div><strong>Brain tumour cells interconnect to a functional and resistant network</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:93-98</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16071" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84949293502" target="_blank" aria-label="Open Scopus in new tab">Scopus (557)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26536111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16071&amp;pmid=26536111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">58.</a></div><div>Oudin, A. ∙ Baus, V. ∙ Barthelemy, V. ...</div><div><strong>Protocol for derivation of organoids and patient-derived orthotopic xenografts from glioma patient tumors</strong></div><div><em>Star Protoc.</em> 2021; <strong>2</strong>:100534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.xpro.2021.100534" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85105598446" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34027491/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.xpro.2021.100534&amp;pmid=34027491" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">59.</a></div><div>Parslow, A. ∙ Cardona, A. ∙ Bryson-Richardson, R.J.</div><div><strong>Sample drift correction following 4D confocal time-lapse imaging</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2014; <strong>12</strong>:51086</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ParslowA.CardonaR.J.Bryson-RichardsonSample+drift+correction+following+4D+confocal+time-lapse+imagingJ.%C2%A0Vis.+Exp.12201451086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">60.</a></div><div>Puchalski, R.B. ∙ Shah, N. ∙ Miller, J. ...</div><div><strong>An anatomic transcriptional atlas of human glioblastoma</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:660-663</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaf2666" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85046671790" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29748285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaf2666&amp;pmid=29748285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">61.</a></div><div>Rahim, S. ∙ Beauchamp, E.M. ∙ Kong, Y. ...</div><div><strong>YK-4-279 inhibits ERG and ETV1 mediated prostate cancer cell invasion</strong></div><div><em>PLoS One.</em> 2011; <strong>6</strong>:e19343</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0019343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-79955779109" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21559405/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0019343&amp;pmid=21559405" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">62.</a></div><div>Raposo, E.P. ∙ Buldyrev, S.V. ∙ da Luz, M.G. ...</div><div><strong>Dynamical robustness of Levy search strategies</strong></div><div><em>Phys. Rev. Lett.</em> 2003; <strong>91</strong>:240601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/PhysRevLett.91.240601" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0742321825" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14683101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FPhysRevLett.91.240601&amp;pmid=14683101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">63.</a></div><div>Reynolds, E.S.</div><div><strong>The use of lead citrate at high pH as an electron-opaque stain in electron microscopy</strong></div><div><em>J.&nbsp;Cell Biol.</em> 1963; <strong>17</strong>:208-212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1083/jcb.17.1.208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13986422/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1083%2Fjcb.17.1.208&amp;pmid=13986422" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">64.</a></div><div>Richards, L.M. ∙ Whitley, O.K.N. ∙ MacLeod, G. ...</div><div><strong>Gradient of Developmental and Injury Response transcriptional states defines functional vulnerabilities underpinning glioblastoma heterogeneity</strong></div><div><em>Nat. Cancer.</em> 2021; <strong>2</strong>:157-173</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s43018-020-00154-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85098800036" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35122077/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs43018-020-00154-9&amp;pmid=35122077" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">65.</a></div><div>Sahm, F. ∙ Capper, D. ∙ Jeibmann, A. ...</div><div><strong>Addressing diffuse glioma as a systemic brain disease with single-cell analysis</strong></div><div><em>Arch. Neurol.</em> 2012; <strong>69</strong>:523-526</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/archneurol.2011.2910" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84859963131" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22158715/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Farchneurol.2011.2910&amp;pmid=22158715" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">66.</a></div><div>Salem, O. ∙ Erdem, N. ∙ Jung, J. ...</div><div><strong>The highly expressed 5'isomiR of hsa-miR-140-3p contributes to the tumor-suppressive effects of miR-140 by reducing breast cancer proliferation and migration</strong></div><div><em>BMC Genomics.</em> 2016; <strong>17</strong>:566</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12864-016-2869-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84981294724" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27502506/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12864-016-2869-x&amp;pmid=27502506" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">67.</a></div><div>Santra, M. ∙ Santra, S. ∙ Roberts, C. ...</div><div><strong>Doublecortin induces mitotic microtubule catastrophe and inhibits glioma cell invasion</strong></div><div><em>J.&nbsp;Neurochem.</em> 2009; <strong>108</strong>:231-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1471-4159.2008.05758.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-57249105391" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19094064/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1471-4159.2008.05758.x&amp;pmid=19094064" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="optO7kvXMytHi" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optO7kvXMytHi" title="View in article">68.</a></div><div>Schätzle, P. ∙ Wuttke, R. ∙ Ziegler, U. ...</div><div><strong>Automated quantification of synapses by fluorescence microscopy</strong></div><div><em>J. Neurosci. Methods.</em> 2012; <strong>204</strong>:144-149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jneumeth.2011.11.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-82355161288" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22108140/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jneumeth.2011.11.010&amp;pmid=22108140" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">69.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (31257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">70.</a></div><div>Schmid, B. ∙ Schindelin, J. ∙ Cardona, A. ...</div><div><strong>A high-level 3D visualization API for Java and ImageJ</strong></div><div><em>BMC Bioinformatics.</em> 2010; <strong>11</strong>:274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-11-274" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-77952447610" target="_blank" aria-label="Open Scopus in new tab">Scopus (370)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20492697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-11-274&amp;pmid=20492697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">71.</a></div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (35949)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">72.</a></div><div>Shannon, P. ∙ Markiel, A. ∙ Ozier, O. ...</div><div><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks</strong></div><div><em>Genome Res.</em> 2003; <strong>13</strong>:2498-2504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.1239303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0242490780" target="_blank" aria-label="Open Scopus in new tab">Scopus (26124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.1239303&amp;pmid=14597658" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">73.</a></div><div>Siebzehnrubl, F.A. ∙ Silver, D.J. ∙ Tugertimur, B. ...</div><div><strong>The ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance</strong></div><div><em>EMBO Mol. Med.</em> 2013; <strong>5</strong>:1196-1212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/emmm.201302827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84881540768" target="_blank" aria-label="Open Scopus in new tab">Scopus (275)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23818228/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Femmm.201302827&amp;pmid=23818228" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">74.</a></div><div>Sims, D.W. ∙ Reynolds, A.M. ∙ Humphries, N.E. ...</div><div><strong>Hierarchical random walks in trace fossils and the origin of optimal search behavior</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:11073-11078</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1405966111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84905042767" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25024221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1405966111&amp;pmid=25024221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">75.</a></div><div>Sims, D.W. ∙ Righton, D. ∙ Pitchford, J.W.</div><div><strong>Minimizing errors in identifying Levy flight behaviour of organisms</strong></div><div><em>J.&nbsp;Anim. Ecol.</em> 2007; <strong>76</strong>:222-229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1365-2656.2006.01208.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-33846950846" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17302829/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1365-2656.2006.01208.x&amp;pmid=17302829" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">76.</a></div><div>Sims, D.W. ∙ Southall, E.J. ∙ Humphries, N.E. ...</div><div><strong>Scaling laws of marine predator search behaviour</strong></div><div><em>Nature.</em> 2008; <strong>451</strong>:1098-1102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature06518" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-39849092398" target="_blank" aria-label="Open Scopus in new tab">Scopus (726)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18305542/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature06518&amp;pmid=18305542" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">77.</a></div><div>Sin, W.C. ∙ Aftab, Q. ∙ Bechberger, J.F. ...</div><div><strong>Astrocytes promote glioma invasion via the Gap junction protein connexin43</strong></div><div><em>Oncogene.</em> 2016; <strong>35</strong>:1504-1516</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/onc.2015.210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84957900747" target="_blank" aria-label="Open Scopus in new tab">Scopus (100)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26165844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fonc.2015.210&amp;pmid=26165844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">78.</a></div><div>Stelzer, G. ∙ Rosen, N. ∙ Plaschkes, I. ...</div><div><strong>The GeneCards Suite: from gene data mining to disease genome sequence analyses</strong></div><div><em>Curr. Protoc. Bioinformatics.</em> 2016; <strong>54</strong></div><div>1 30 31–31 30 33</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpbi.5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85016156181" target="_blank" aria-label="Open Scopus in new tab">Scopus (1361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27322403/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpbi.5&amp;pmid=27322403" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">79.</a></div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (4645)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">80.</a></div><div>Sullivan, K.F. ∙ Cleveland, D.W.</div><div><strong>Identification of conserved isotype-defining variable region sequences for four vertebrate beta tubulin polypeptide classes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1986; <strong>83</strong>:4327-4331</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.83.12.4327" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3459176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.83.12.4327&amp;pmid=3459176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">81.</a></div><div>Tabata, H. ∙ Nakajima, K.</div><div><strong>Multipolar migration: the third mode of radial neuronal migration in the developing cerebral cortex</strong></div><div><em>J.&nbsp;Neurosci.</em> 2003; <strong>23</strong>:9996-10001</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.23-31-09996.2003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14602813/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.23-31-09996.2003&amp;pmid=14602813" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">82.</a></div><div>Takano, T. ∙ Tian, G.F. ∙ Peng, W.G. ...</div><div><strong>Astrocyte-mediated control of cerebral blood flow</strong></div><div><em>Nat. Neurosci.</em> 2006; <strong>9</strong>:260-267</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn1623" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-31544456102" target="_blank" aria-label="Open Scopus in new tab">Scopus (866)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16388306/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn1623&amp;pmid=16388306" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">83.</a></div><div>Tano, K. ∙ Mizuno, R. ∙ Okada, T. ...</div><div><strong>MALAT-1 enhances cell motility of lung adenocarcinoma cells by influencing the expression of motility-related genes</strong></div><div><em>FEBS Lett.</em> 2010; <strong>584</strong>:4575-4580</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.febslet.2010.10.008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-78249258719" target="_blank" aria-label="Open Scopus in new tab">Scopus (383)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20937273/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.febslet.2010.10.008&amp;pmid=20937273" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">84.</a></div><div>Thestrup, T. ∙ Litzlbauer, J. ∙ Bartholomäus, I. ...</div><div><strong>Optimized ratiometric calcium sensors for functional in&nbsp;vivo imaging of neurons and T lymphocytes</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:175-182</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2773" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84893256698" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24390440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2773&amp;pmid=24390440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">85.</a></div><div>Tsai, H.H. ∙ Niu, J. ∙ Munji, R. ...</div><div><strong>Oligodendrocyte precursors migrate along vasculature in the developing nervous system</strong></div><div><em>Science.</em> 2016; <strong>351</strong>:379-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad3839" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84964306884" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26798014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad3839&amp;pmid=26798014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">86.</a></div><div>Valiente, M. ∙ Martini, F.J.</div><div><strong>Migration of cortical interneurons relies on branched leading process dynamics</strong></div><div><em>Cell Adh. Migr.</em> 2009; <strong>3</strong>:278-280</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/cam.3.3.8832" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-68249083818" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19535916/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Fcam.3.3.8832&amp;pmid=19535916" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">87.</a></div><div>Valiunas, V. ∙ Doronin, S. ∙ Valiuniene, L. ...</div><div><strong>Human mesenchymal stem cells make cardiac connexins and form functional gap junctions</strong></div><div><em>J.&nbsp;Physiol. Lond.</em> 2004; <strong>555</strong>:617-626</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1113/jphysiol.2003.058719" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-12144286490" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1113%2Fjphysiol.2003.058719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">88.</a></div><div>Van de Sande, B. ∙ Flerin, C. ∙ Davie, K. ...</div><div><strong>A scalable SCENIC workflow for single-cell gene regulatory network analysis</strong></div><div><em>Nat. Protoc.</em> 2020; <strong>15</strong>:2247-2276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-020-0336-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85086586785" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32561888/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-020-0336-2&amp;pmid=32561888" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">89.</a></div><div>van der Horst, E.H. ∙ Degenhardt, Y.Y. ∙ Strelow, A. ...</div><div><strong>Metastatic properties and genomic amplification of the tyrosine kinase gene ACK1</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:15901-15906</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0508014102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-27644436744" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16247015/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0508014102&amp;pmid=16247015" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">90.</a></div><div>Varn, F.S. ∙ Johnson, K.C. ∙ Martinek, J. ...</div><div><strong>Glioma progression is shaped by genetic evolution and microenvironment interactions</strong></div><div><em>Cell.</em> 2022; <strong>185</strong>:2184-2199.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2022.04.038&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2022.04.038&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85131903367" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/35649412/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2022.04.038&amp;pmid=35649412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">91.</a></div><div>Venkataramani, V. ∙ Tanev, D.I. ∙ Strahle, C. ...</div><div><strong>Glutamatergic synaptic input to glioma cells drives brain tumour progression</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:532-538</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1564-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072687457" target="_blank" aria-label="Open Scopus in new tab">Scopus (365)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1564-x&amp;pmid=31534219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">92.</a></div><div>Venkatesh, H.S. ∙ Johung, T.B. ∙ Caretti, V. ...</div><div><strong>Neuronal activity promotes glioma growth through Neuroligin-3 secretion</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:803-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2015.04.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=10.1016%2Fj.cell.2015.04.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84928918997" target="_blank" aria-label="Open Scopus in new tab">Scopus (363)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25913192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.04.012&amp;pmid=25913192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">93.</a></div><div>Venkatesh, H.S. ∙ Morishita, W. ∙ Geraghty, A.C. ...</div><div><strong>Electrical and synaptic integration of glioma into neural circuits</strong></div><div><em>Nature.</em> 2019; <strong>573</strong>:539-545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1563-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85072686777" target="_blank" aria-label="Open Scopus in new tab">Scopus (407)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31534222/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1563-y&amp;pmid=31534222" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">94.</a></div><div>Venkatesh, H.S. ∙ Tam, L.T. ∙ Woo, P.J. ...</div><div><strong>Targeting neuronal activity-regulated neuroligin-3 dependency in high-grade glioma</strong></div><div><em>Nature.</em> 2017; <strong>549</strong>:533-537</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85030622967" target="_blank" aria-label="Open Scopus in new tab">Scopus (228)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28959975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24014&amp;pmid=28959975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">95.</a></div><div>Viswanathan, G.M. ∙ Buldyrev, S.V. ∙ Havlin, S. ...</div><div><strong>Optimizing the success of random searches</strong></div><div><em>Nature.</em> 1999; <strong>401</strong>:911-914</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/44831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-0033613429" target="_blank" aria-label="Open Scopus in new tab">Scopus (1137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10553906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F44831&amp;pmid=10553906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">96.</a></div><div>Viswanathan, G.M. ∙ Raposo, E.P. ∙ da Luz, M.G.E.</div><div><strong>Lévy flights and superdiffusion in the context of biological encounters and random searches</strong></div><div><em>Phys. Life Rev.</em> 2008; <strong>5</strong>:133-150</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.plrev.2008.03.002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-49849093656" target="_blank" aria-label="Open Scopus in new tab">Scopus (189)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.plrev.2008.03.002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94-1" title="View in article">97.</a></div><div>Wang, L. ∙ Babikir, H. ∙ Müller, S. ...</div><div><strong>The phenotypes of proliferating glioblastoma cells reside on a single axis of variation</strong></div><div><em>Cancer Discov.</em> 2019; <strong>9</strong>:1708-1719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1158/2159-8290.CD-19-0329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85075958233" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31554641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1158%2F2159-8290.CD-19-0329&amp;pmid=31554641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">98.</a></div><div>Wang, Y. ∙ DelRosso, N.V. ∙ Vaidyanathan, T.V. ...</div><div><strong>Accurate quantification of astrocyte and neurotransmitter fluorescence dynamics for single-cell and population-level physiology</strong></div><div><em>Nat. Neurosci.</em> 2019; <strong>22</strong>:1936-1944</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41593-019-0492-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85074279015" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31570865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41593-019-0492-2&amp;pmid=31570865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">99.</a></div><div>Wefers, A.K. ∙ Haberlandt, C. ∙ Tekin, N.B. ...</div><div><strong>Synaptic input as a directional cue for migrating interneuron precursors</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:4125-4136</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.154096" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85034447129" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29061636/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.154096&amp;pmid=29061636" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">100.</a></div><div>Weigert, M. ∙ Schmidt, U. ∙ Boothe, T. ...</div><div><strong>Content-aware image restoration: pushing the limits of fluorescence microscopy</strong></div><div><em>Nat. Methods.</em> 2018; <strong>15</strong>:1090-1097</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-018-0216-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85057342134" target="_blank" aria-label="Open Scopus in new tab">Scopus (422)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-018-0216-7&amp;pmid=30478326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98-1" title="View in article">101.</a></div><div>Weil, S. ∙ Osswald, M. ∙ Solecki, G. ...</div><div><strong>Tumor microtubes convey resistance to surgical lesions and chemotherapy in gliomas</strong></div><div><em>Neuro. Oncol.</em> 2017; <strong>19</strong>:1316-1326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/nox070" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85029925185" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28419303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnox070&amp;pmid=28419303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">102.</a></div><div>Wick, W. ∙ Osswald, M. ∙ Wick, A. ...</div><div><strong>Treatment of glioblastoma in adults</strong></div><div><em>Ther. Adv. Neurol. Diso.</em> 2018; <strong>11</strong></div><div>1756286418790452</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30083233/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=30083233" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">103.</a></div><div>Wimmer, V.C. ∙ Nevian, T. ∙ Kuner, T.</div><div><strong>Targeted in&nbsp;vivo expression of proteins in the calyx of Held</strong></div><div><em>Pflugers Arch.</em> 2004; <strong>449</strong>:319-333</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15452710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=15452710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101-1" title="View in article">104.</a></div><div>Xie, R. ∙ Kessler, T. ∙ Grosch, J. ...</div><div><strong>Tumor cell network integration in glioma represents a stemness feature</strong></div><div><em>Neuro. Oncol.</em> 2021; <strong>23</strong>:757-769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/neuonc/noaa275" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85100835529" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33320195/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fneuonc%2Fnoaa275&amp;pmid=33320195" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102-1" title="View in article">105.</a></div><div>Yu, K. ∙ Hu, Y.Q. ∙ Wu, F. ...</div><div><strong>Surveying brain tumor heterogeneity by single-cell RNA-sequencing of multi-sector biopsies</strong></div><div><em>Natl. Sci. Rev.</em> 2020; <strong>7</strong>:1306-1318</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nsr/nwaa099" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-85092585099" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34692159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnsr%2Fnwaa099&amp;pmid=34692159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-1" title="View in article">106.</a></div><div>Zaburdaev, V. ∙ Denisov, S. ∙ Klafter, J.</div><div><strong>Levy walks</strong></div><div><em>Rev. Mod. Phys.</em> 2015; <strong>87</strong>:483-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1103/RevModPhys.87.483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.054&amp;key=2-s2.0-84937578864" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1103%2FRevModPhys.87.483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">107.</a></div><div>Zhang, H. ∙ Kelly, G. ∙ Zerillo, C. ...</div><div><strong>Expression of a cleaved brain-specific extracellular matrix protein mediates glioma cell invasion in&nbsp;vivo</strong></div><div><em>J.&nbsp;Neurosci.</em> 1998; <strong>18</strong>:2370-2376</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.18-07-02370.1998" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9502798/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.18-07-02370.1998&amp;pmid=9502798" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">108.</a></div><div>Zhang, W. ∙ Couldwell, W.T. ∙ Simard, M.F. ...</div><div><strong>Direct gap junction communication between malignant glioma cells and astrocytes</strong></div><div><em>Cancer Res.</em> 1999; <strong>59</strong>:1994-2003</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10213512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=10213512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">109.</a></div><div>Zirpel, L. ∙ Janowiak, M.A. ∙ Veltri, C.A. ...</div><div><strong>AMPA receptor-mediated, calcium-dependent CREB phosphorylation in a subpopulation of auditory neurons surviving activity deprivation</strong></div><div><em>J.&nbsp;Neurosci.</em> 2000; <strong>20</strong>:6267-6275</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.20-16-06267.2000" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10934277/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.20-16-06267.2000&amp;pmid=10934277" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (15)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure 1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/9f4bd696-b37c-4fff-82d5-6e6db3a9dbde/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/1a276d00-d15e-4718-9df8-eb0cdd6dbfab/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/e5faae35-c0bf-4f01-84c9-863638c2827d/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/45149dc6-9ed6-4f96-a0b4-c21ced647489/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/bfff812d-f7df-417d-9038-b9e90ecff931/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/ffe3ff99-cd30-41e2-98ae-8b300a718a96/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure 3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/ab438768-8f5c-4b5e-b845-dd9171532c33/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/c137927c-0d1e-4def-9997-442b40b99e21/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure 4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/a190164f-05e7-48bc-8693-52e01f4d2969/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/46a6cf0e-1997-4c43-8415-cc01cd45c7bf/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/5797308e-caac-435d-953f-d758bb3874a4/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/9a73eef6-9005-448a-a6a2-abc169dfc4a9/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/bc6b9e53-87ea-47a1-9af0-52cba9783c87/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/eaa4aade-2aeb-4767-ab13-699c4e530b63/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs8" aria-label="Go to Figure&amp;nbsp;S8"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/asset/6eae3291-d537-4fe4-8289-bad021178dc0/main.assets/figs8.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867422008479%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00177%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (2)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/attachment/66f8c130-d441-463d-ac9e-38bac7a6e1ec/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (10.84 KB)</span></a><div class="core-description">Table&nbsp;S1. Molecular characterization of patient-derived cell lines and PDOX models used in this study, related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.054/attachment/2a20f685-5c53-4f52-98fb-6d3ed686764d/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (4.97 MB)</span></a><div class="core-description">Table&nbsp;S2. Further molecular characterization of the invasivity score, related to Figure&nbsp;2</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422008479" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.06.054","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Glioblastoma hijacks neuronal mechanisms for brain invasion</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.054/asset/9f4bd696-b37c-4fff-82d5-6e6db3a9dbde/main.assets/gr1.jpg" height="679" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/a11b6971-fa32-472c-a2a8-c7d8fd5b67f6/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.054/asset/1a276d00-d15e-4718-9df8-eb0cdd6dbfab/main.assets/figs1.jpg" height="941" width="720" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/28eb2f52-878e-40bd-8224-5c562dabcacc/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.054/asset/e5faae35-c0bf-4f01-84c9-863638c2827d/main.assets/gr2.jpg" height="908" width="759" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7dde6d90-bba8-4523-9b1e-e3283af8151e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.054/asset/45149dc6-9ed6-4f96-a0b4-c21ced647489/main.assets/figs2.jpg" height="823" width="761" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/d2eba235-997f-4314-8d83-f947e418ce85/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.054/asset/bfff812d-f7df-417d-9038-b9e90ecff931/main.assets/figs3.jpg" height="698" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/2d24d472-2208-4468-a95b-37919e99de0e/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.054/asset/ffe3ff99-cd30-41e2-98ae-8b300a718a96/main.assets/figs4.jpg" height="849" width="761" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/b9f258a1-a5ab-43ab-9781-9098795cf401/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.054/asset/ab438768-8f5c-4b5e-b845-dd9171532c33/main.assets/gr3.jpg" height="940" width="714" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/311aed84-0248-4ee8-963e-c4a1685549a1/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.054/asset/c137927c-0d1e-4def-9997-442b40b99e21/main.assets/figs5.jpg" height="907" width="761" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/cee47c56-5760-4cd1-8f1b-9b7bc62c3b13/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.054/asset/a190164f-05e7-48bc-8693-52e01f4d2969/main.assets/gr4.jpg" height="699" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/0d63dd00-3217-4a79-92e8-75c485801798/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.054/asset/46a6cf0e-1997-4c43-8415-cc01cd45c7bf/main.assets/gr5.jpg" height="941" width="734" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/fb2d90d2-7e9b-40db-bcd9-755a212145b2/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.054/asset/5797308e-caac-435d-953f-d758bb3874a4/main.assets/figs6.jpg" height="914" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7a13b7b2-1e60-4881-aff2-fb489205395d/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.054/asset/9a73eef6-9005-448a-a6a2-abc169dfc4a9/main.assets/gr6.jpg" height="511" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/b623e600-0975-428f-a8a7-485405a7ec48/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.06.054/asset/bc6b9e53-87ea-47a1-9af0-52cba9783c87/main.assets/figs7.jpg" height="940" width="701" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/20613082-f643-4299-9726-569c7c3049a5/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.06.054/asset/eaa4aade-2aeb-4767-ab13-699c4e530b63/main.assets/gr7.jpg" height="941" width="715" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7f0c3b05-75b9-473b-aabe-d3849f5e8745/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs8" src="/cms/10.1016/j.cell.2022.06.054/asset/6eae3291-d537-4fe4-8289-bad021178dc0/main.assets/figs8.jpg" height="941" width="723" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/524c6a37-182f-4b76-9086-bdc537a07506/main.assets/figs8_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure 1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.054/asset/9f4bd696-b37c-4fff-82d5-6e6db3a9dbde/main.assets/gr1.jpg" height="679" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/a11b6971-fa32-472c-a2a8-c7d8fd5b67f6/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.054/asset/1a276d00-d15e-4718-9df8-eb0cdd6dbfab/main.assets/figs1.jpg" height="941" width="720" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/28eb2f52-878e-40bd-8224-5c562dabcacc/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.054/asset/e5faae35-c0bf-4f01-84c9-863638c2827d/main.assets/gr2.jpg" height="908" width="759" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7dde6d90-bba8-4523-9b1e-e3283af8151e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.054/asset/45149dc6-9ed6-4f96-a0b4-c21ced647489/main.assets/figs2.jpg" height="823" width="761" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/d2eba235-997f-4314-8d83-f947e418ce85/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.054/asset/bfff812d-f7df-417d-9038-b9e90ecff931/main.assets/figs3.jpg" height="698" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/2d24d472-2208-4468-a95b-37919e99de0e/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.054/asset/ffe3ff99-cd30-41e2-98ae-8b300a718a96/main.assets/figs4.jpg" height="849" width="761" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/b9f258a1-a5ab-43ab-9781-9098795cf401/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.054/asset/ab438768-8f5c-4b5e-b845-dd9171532c33/main.assets/gr3.jpg" height="940" width="714" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/311aed84-0248-4ee8-963e-c4a1685549a1/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.054/asset/c137927c-0d1e-4def-9997-442b40b99e21/main.assets/figs5.jpg" height="907" width="761" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/cee47c56-5760-4cd1-8f1b-9b7bc62c3b13/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.054/asset/a190164f-05e7-48bc-8693-52e01f4d2969/main.assets/gr4.jpg" height="699" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/0d63dd00-3217-4a79-92e8-75c485801798/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.054/asset/46a6cf0e-1997-4c43-8415-cc01cd45c7bf/main.assets/gr5.jpg" height="941" width="734" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/fb2d90d2-7e9b-40db-bcd9-755a212145b2/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.054/asset/5797308e-caac-435d-953f-d758bb3874a4/main.assets/figs6.jpg" height="914" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7a13b7b2-1e60-4881-aff2-fb489205395d/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.054/asset/9a73eef6-9005-448a-a6a2-abc169dfc4a9/main.assets/gr6.jpg" height="511" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/b623e600-0975-428f-a8a7-485405a7ec48/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.06.054/asset/bc6b9e53-87ea-47a1-9af0-52cba9783c87/main.assets/figs7.jpg" height="940" width="701" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/20613082-f643-4299-9726-569c7c3049a5/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.06.054/asset/eaa4aade-2aeb-4767-ab13-699c4e530b63/main.assets/gr7.jpg" height="941" width="715" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/7f0c3b05-75b9-473b-aabe-d3849f5e8745/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S8</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs8" src="/cms/10.1016/j.cell.2022.06.054/asset/6eae3291-d537-4fe4-8289-bad021178dc0/main.assets/figs8.jpg" height="941" width="723" aria-labelledby="figs8-title figs8-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.054/asset/524c6a37-182f-4b76-9086-bdc537a07506/main.assets/figs8_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00847-9';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749594033756","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"16","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422008479","coverDate":"202208","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Glioblastoma hijacks neuronal mechanisms for brain invasion","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc41a54e924ee9',t:'MTc0OTU5NDAzMi4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc41a54e924ee9&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.06.054&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>