<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="c3b20f82-8cbc-4e9d-88f5-399833292f93" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867421002397;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;issue:issue:pii\:S0092867420X00092;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="An early cell shape transition drives evolutionary expansion of the human forebrain">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00239-7">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/14928b4f-0dca-4bc9-983b-87346f06a4b9/fx1.jpg">
    


        <meta property="og:description" content="Cerebral organoid models reveal that differences in the duration of a developmental
transitional state driven by the factor ZEB2 underlie the basis of brain expansion
in humans in comparison to great apes.">
    











        <meta name="twitter:title" content="An early cell shape transition drives evolutionary expansion of the human forebrain">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/14928b4f-0dca-4bc9-983b-87346f06a4b9/fx1.jpg">
    


        <meta name="twitter:description" content="Cerebral organoid models reveal that differences in the duration of a developmental
transitional state driven by the factor ZEB2 underlie the basis of brain expansion
in humans in comparison to great apes.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>An early cell shape transition drives evolutionary expansion of the human forebrain: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="An early cell shape transition drives evolutionary expansion of the human forebrain"> <meta name="citation_date" content="2021/04/15"> <meta name="citation_online_date" content="2021/03/24"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="8"> <meta name="citation_firstpage" content="2084"> <meta name="citation_lastpage" content="2102.e19"> <meta name="citation_pmid" content="33765444"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421002397"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421002397"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421002397"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.02.050"> <meta name="citation_pmid" content="33765444"> <meta name="citation_author" content="Silvia Benito-Kwiecinski"><meta name="citation_author_orcid" content="0000-0002-0821-3858"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_author" content="Stefano L. Giandomenico"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_author" content="Magdalena Sutcliffe"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_author" content="Erlend S. Riis"><meta name="citation_author_orcid" content="0000-0001-9656-4123"><meta name="citation_author_institution" content="Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Wilberforce Road, Cambridge CB3 0WA, UK"><meta name="citation_author" content="Paula Freire-Pritchett"><meta name="citation_author_orcid" content="0000-0002-7090-3016"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_author" content="Iva Kelava"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_author" content="Stephanie Wunderlich"><meta name="citation_author_institution" content="Leibniz Research Laboratories for Biotechnology and Artificial Organs (LEBAO), REBIRTH-Research Center for Translational and Regenerative Medicine, Hannover Medical School, 30625 Hannover, Germany"><meta name="citation_author_institution" content="Biomedical Research in Endstage and Obstructive Lung Disease (BREATH), Member of the German Center for Lung Research (DZL), Hannover Medical School, 30625 Hannover, Germany"><meta name="citation_author" content="Ulrich Martin"><meta name="citation_author_institution" content="Leibniz Research Laboratories for Biotechnology and Artificial Organs (LEBAO), REBIRTH-Research Center for Translational and Regenerative Medicine, Hannover Medical School, 30625 Hannover, Germany"><meta name="citation_author_institution" content="Biomedical Research in Endstage and Obstructive Lung Disease (BREATH), Member of the German Center for Lung Research (DZL), Hannover Medical School, 30625 Hannover, Germany"><meta name="citation_author" content="Gregory A. Wray"><meta name="citation_author_institution" content="Department of Biology, Duke University, Biological Sciences Building, 124 Science Drive, Durham, NC 27708, USA"><meta name="citation_author" content="Kate McDole"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_author" content="Madeline A. Lancaster"><meta name="citation_author_email" content="madeline.lancaster@mrc-lmb.cam.ac.uk"><meta name="citation_author_institution" content="MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK"><meta name="citation_keywords" content="brain; evolution; cell shape; organoids; brain expansion; neuroepithelium; neural stem cells; ZEB2; gorilla; chimpanzee"><meta name="citation_reference" content="citation_title=Loss of occludin and functional tight junctions, but not ZO-1, during neural tube closure--remodeling of the neuroepithelium prior to neurogenesis; citation_author=E. Aaku-Saraste; citation_author=A. Hellwig; citation_author=W.B. Huttner; citation_journal_title=Dev. Biol.; citation_volume=180; citation_firstpage=664; citation_lastpage=679; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Cell-cycle-dependent TGFβ-BMP antagonism regulates neural tube closure by modulating tight junctions; citation_author=S. Amarnath; citation_author=S. Agarwala; citation_journal_title=J.&nbsp;Cell Sci.; citation_volume=130; citation_firstpage=119; citation_lastpage=131; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Efficient processing and analysis of large-scale light-sheet microscopy data; citation_author=F. Amat; citation_author=B. Höckendorf; citation_author=Y. Wan; citation_author=W.C. Lemon; citation_author=K. McDole; citation_author=P.J. Keller; citation_journal_title=Nat. Protoc.; citation_volume=10; citation_firstpage=1679; citation_lastpage=1696; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=HTSeq--a Python framework to work with high-throughput sequencing data; citation_author=S. Anders; citation_author=P.T. Pyl; citation_author=W. Huber; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=166; citation_lastpage=169; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Differential behavior of E-cadherin and occludin in their colocalization with ZO-1 during the establishment of epithelial cell polarity; citation_author=Y. Ando-Akatsuka; citation_author=S. Yonemura; citation_author=M. Itoh; citation_author=M. Furuse; citation_author=S. Tsukita; citation_journal_title=J.&nbsp;Cell. Physiol.; citation_volume=179; citation_firstpage=115; citation_lastpage=125; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=FastQC: a quality control tool for high throughput sequence data; citation_author=S. Andrews; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Check-list of the data on the gestation length of primates; citation_author=G. Ardito; citation_journal_title=J.&nbsp;Hum. Evol.; citation_volume=5; citation_firstpage=213; citation_lastpage=222; citation_publication_date=1976; "><meta name="citation_reference" content="citation_title=Genetic changes shaping the human brain; citation_author=B.I. Bae; citation_author=D. Jayaraman; citation_author=C.A. Walsh; citation_journal_title=Dev. Cell; citation_volume=32; citation_firstpage=423; citation_lastpage=434; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Timetables of neurogenesis in the human brain based on experimentally determined patterns in the rat; citation_author=S.A. Bayer; citation_author=J. Altman; citation_author=R.J. Russo; citation_author=X. Zhang; citation_journal_title=Neurotoxicology; citation_volume=14; citation_firstpage=83; citation_lastpage=144; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Optimized inducible shRNA and CRISPR/Cas9 platforms for in&nbsp;vitro studies of human development using hPSCs; citation_author=A. Bertero; citation_author=M. Pawlowski; citation_author=D. Ortmann; citation_author=K. Snijders; citation_author=L. Yiangou; citation_author=M.C. De Brito; citation_author=S. Brown; citation_author=W.G. Bernard; citation_author=J.D. Cooper; citation_author=E. Giacomelli; citation_author=et al.; citation_journal_title=Development; citation_volume=143; citation_firstpage=4405; citation_lastpage=4418; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Human-chimpanzee differences in a FZD8 enhancer alter cell-cycle dynamics in the developing neocortex; citation_author=J.L. Boyd; citation_author=S.L. Skove; citation_author=J.P. Rouanet; citation_author=L.J. Pilaz; citation_author=T. Bepler; citation_author=R. Gordân; citation_author=G.A. Wray; citation_author=D.L. Silver; citation_journal_title=Curr. Biol.; citation_volume=25; citation_firstpage=772; citation_lastpage=779; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Alternative splicing and the evolution of phenotypic novelty; citation_author=S.J. Bush; citation_author=L. Chen; citation_author=J.M. Tovar-Corona; citation_author=A.O. Urrutia; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=372; citation_firstpage=20150474; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Gradients in cytoarchitectural landscapes of the isocortex: Diprotodont marsupials in comparison to eutherian mammals; citation_author=C.J. Charvet; citation_author=C.D. Stimpson; citation_author=Y.D. Kim; citation_author=M.A. Raghanti; citation_author=A.H. Lewandowski; citation_author=P.R. Hof; citation_author=A. Gómez-Robles; citation_author=F.M. Krienen; citation_author=C.C. Sherwood; citation_journal_title=J.&nbsp;Comp. Neurol.; citation_volume=525; citation_firstpage=1811; citation_lastpage=1826; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=SIP1 mediates cell-fate decisions between neuroectoderm and mesendoderm in human pluripotent stem cells; citation_author=Z. Chng; citation_author=A. Teo; citation_author=R.A. Pedersen; citation_author=L. Vallier; citation_journal_title=Cell Stem Cell; citation_volume=6; citation_firstpage=59; citation_lastpage=70; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Lulu regulates Shroom-induced apical constriction during neural tube closure; citation_author=C.W. Chu; citation_author=E. Gerstenzang; citation_author=O. Ossipova; citation_author=S.Y. Sokol; citation_journal_title=PLoS ONE; citation_volume=8; citation_firstpage=e81854; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Comparative cytoarchitectural analyses of striate and extrastriate areas in hominoids; citation_author=A.A. de Sousa; citation_author=C.C. Sherwood; citation_author=A. Schleicher; citation_author=K. Amunts; citation_author=C.E. MacLeod; citation_author=P.R. Hof; citation_author=K. Zilles; citation_journal_title=Cereb. Cortex; citation_volume=20; citation_firstpage=966; citation_lastpage=981; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Microstructure of the neocortex: comparative aspects; citation_author=J. DeFelipe; citation_author=L. Alonso-Nanclares; citation_author=J.I. Arellano; citation_journal_title=J.&nbsp;Neurocytol.; citation_volume=31; citation_firstpage=299; citation_lastpage=316; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=The outer subventricular zone and primate-specific cortical complexification; citation_author=C. Dehay; citation_author=H. Kennedy; citation_author=K.S. Kosik; citation_journal_title=Neuron; citation_volume=85; citation_firstpage=683; citation_lastpage=694; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Functional genomics, proteomics, and regulatory DNA analysis in isogenic settings using zinc finger nuclease-driven transgenesis into a safe harbor locus in the human genome; citation_author=R.C. DeKelver; citation_author=V.M. Choi; citation_author=E.A. Moehle; citation_author=D.E. Paschon; citation_author=D. Hockemeyer; citation_author=S.H. Meijsing; citation_author=Y. Sancak; citation_author=X. Cui; citation_author=E.J. Steine; citation_author=J.C. Miller; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=1133; citation_lastpage=1142; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Quantitative assessment of prefrontal cortex in humans relative to nonhuman primates; citation_author=C.J. Donahue; citation_author=M.F. Glasser; citation_author=T.M. Preuss; citation_author=J.K. Rilling; citation_author=D.C. Van Essen; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E5183; citation_lastpage=E5192; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=XSIP1, a member of two-handed zinc finger proteins, induced anterior neural markers in Xenopus laevis animal cap; citation_author=A. Eisaki; citation_author=H. Kuroda; citation_author=A. Fukui; citation_author=M. Asashima; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=271; citation_firstpage=151; citation_lastpage=157; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Role of Zeb2/Sip1 in neuronal development; citation_author=E. Epifanova; citation_author=A. Babaev; citation_author=A.G. Newman; citation_author=V. Tarabykin; citation_journal_title=Brain Res.; citation_volume=1705; citation_firstpage=24; citation_lastpage=31; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Integrating diverse datasets improves developmental enhancer prediction; citation_author=G.D. Erwin; citation_author=N. Oksenberg; citation_author=R.M. Truty; citation_author=D. Kostka; citation_author=K.K. Murphy; citation_author=N. Ahituv; citation_author=K.S. Pollard; citation_author=J.A. Capra; citation_journal_title=PLoS Comput. Biol.; citation_volume=10; citation_firstpage=e1003677; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The crucial role of ZEB2: From development to epithelial-to-mesenchymal transition and cancer complexity; citation_author=M. Fardi; citation_author=M. Alivand; citation_author=B. Baradaran; citation_author=M. Farshdousti Hagh; citation_author=S. Solali; citation_journal_title=J.&nbsp;Cell. Physiol.; citation_volume=234; citation_firstpage=14783; citation_lastpage=14799; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Human-Specific NOTCH2NL Genes Affect Notch Signaling and Cortical Neurogenesis; citation_author=I.T. Fiddes; citation_author=G.A. Lodewijk; citation_author=M. Mooring; citation_author=C.M. Bosworth; citation_author=A.D. Ewing; citation_author=G.L. Mantalas; citation_author=A.M. Novak; citation_author=A. van den Bout; citation_author=A. Bishara; citation_author=J.L. Rosenkrantz; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=1356; citation_lastpage=1369.e22; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=OSVZ progenitors of human and ferret neocortex are epithelial-like and expand by integrin signaling; citation_author=S.A. Fietz; citation_author=I. Kelava; citation_author=J. Vogt; citation_author=M. Wilsch-Bräuninger; citation_author=D. Stenzel; citation_author=J.L. Fish; citation_author=D. Corbeil; citation_author=A. Riehn; citation_author=W. Distler; citation_author=R. Nitsch; citation_author=W.B. Huttner; citation_journal_title=Nat. Neurosci.; citation_volume=13; citation_firstpage=690; citation_lastpage=699; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A digital pcr-based method for efficient and highly specific screening of genome edited cells; citation_author=S.D. Findlay; citation_author=K.M. Vincent; citation_author=J.R. Berman; citation_author=L.M. Postovit; citation_journal_title=PLoS ONE; citation_volume=11; citation_firstpage=e0153901; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Human-specific gene ARHGAP11B promotes basal progenitor amplification and neocortex expansion; citation_author=M. Florio; citation_author=M. Albert; citation_author=E. Taverna; citation_author=T. Namba; citation_author=H. Brandl; citation_author=E. Lewitus; citation_author=C. Haffner; citation_author=A. Sykes; citation_author=F.K. Wong; citation_author=J. Peters; citation_author=et al.; citation_journal_title=Science; citation_volume=347; citation_firstpage=1465; citation_lastpage=1470; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A panel of induced pluripotent stem cells from chimpanzees: a resource for comparative functional genomics; citation_author=I. Gallego Romero; citation_author=B.J. Pavlovic; citation_author=I. Hernando-Herraez; citation_author=X. Zhou; citation_author=M.C. Ward; citation_author=N.E. Banovich; citation_author=C.L. Kagan; citation_author=J.E. Burnett; citation_author=C.H. Huang; citation_author=A. Mitrano; citation_author=et al.; citation_journal_title=eLife; citation_volume=4; citation_firstpage=e07103; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Neuroimaging findings in Mowat-Wilson syndrome: a study of 54 patients; citation_author=L. Garavelli; citation_author=I. Ivanovski; citation_author=S.G. Caraffi; citation_author=D. Santodirocco; citation_author=M. Pollazzon; citation_author=D.M. Cordelli; citation_author=E. Abdalla; citation_author=P. Accorsi; citation_author=M.P. Adam; citation_author=C. Baldo; citation_author=et al.; citation_journal_title=Genet. Med.; citation_volume=19; citation_firstpage=691; citation_lastpage=700; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=An iCRISPR platform for rapid, multiplexable, and inducible genome editing in human pluripotent stem cells; citation_author=F. González; citation_author=Z. Zhu; citation_author=Z.D. Shi; citation_author=K. Lelli; citation_author=N. Verma; citation_author=Q.V. Li; citation_author=D. Huangfu; citation_journal_title=Cell Stem Cell; citation_volume=15; citation_firstpage=215; citation_lastpage=226; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The cell biology of neurogenesis; citation_author=M. Götz; citation_author=W.B. Huttner; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=6; citation_firstpage=777; citation_lastpage=788; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Mechanical impact of epithelial-mesenchymal transition on epithelial morphogenesis in Drosophila; citation_author=M. Gracia; citation_author=S. Theis; citation_author=A. Proag; citation_author=G. Gay; citation_author=C. Benassayag; citation_author=M. Suzanne; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=2951; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Shroom induces apical constriction and is required for hingepoint formation during neural tube closure; citation_author=S.L. Haigo; citation_author=J.D. Hildebrand; citation_author=R.M. Harland; citation_author=J.B. Wallingford; citation_journal_title=Curr. Biol.; citation_volume=13; citation_firstpage=2125; citation_lastpage=2137; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Neurogenic radial glia in the outer subventricular zone of human neocortex; citation_author=D.V. Hansen; citation_author=J.H. Lui; citation_author=P.R.L. Parker; citation_author=A.R. Kriegstein; citation_journal_title=Nature; citation_volume=464; citation_firstpage=554; citation_lastpage=561; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The human brain in numbers: a linearly scaled-up primate brain; citation_author=S. Herculano-Houzel; citation_journal_title=Front. Hum. Neurosci.; citation_volume=3; citation_firstpage=31; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The remarkable, yet not extraordinary, human brain as a scaled-up primate brain and its associated cost; citation_author=S. Herculano-Houzel; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_number=Suppl 1; citation_firstpage=10661; citation_lastpage=10668; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Cellular scaling rules for rodent brains; citation_author=S. Herculano-Houzel; citation_author=B. Mota; citation_author=R. Lent; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=12138; citation_lastpage=12143; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Shroom, a PDZ domain-containing actin-binding protein, is required for neural tube morphogenesis in mice; citation_author=J.D. Hildebrand; citation_author=P. Soriano; citation_journal_title=Cell; citation_volume=99; citation_firstpage=485; citation_lastpage=497; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Comparative analysis of cortical layering and supragranular layer enlargement in rodent carnivore and primate species; citation_author=J.J. Hutsler; citation_author=D.G. Lee; citation_author=K.K. Porter; citation_journal_title=Brain Res.; citation_volume=1052; citation_firstpage=71; citation_lastpage=81; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Self-organization of axial polarity, inside-out layer pattern, and species-specific progenitor dynamics in human ES cell-derived neocortex; citation_author=T. Kadoshima; citation_author=H. Sakaguchi; citation_author=T. Nakano; citation_author=M. Soen; citation_author=S. Ando; citation_author=M. Eiraku; citation_author=Y. Sasai; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=20284; citation_lastpage=20289; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Neocortical Expansion Due to Increased Proliferation of Basal Progenitors Is Linked to Changes in Their Morphology; citation_author=N. Kalebic; citation_author=C. Gilardi; citation_author=B. Stepien; citation_author=M. Wilsch-Bräuninger; citation_author=K.R. Long; citation_author=T. Namba; citation_author=M. Florio; citation_author=B. Langen; citation_author=B. Lombardot; citation_author=A. Shevchenko; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=24; citation_firstpage=535; citation_lastpage=550; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Organoid single-cell genomic atlas uncovers human-specific features of brain development; citation_author=S. Kanton; citation_author=M.J. Boyle; citation_author=Z. He; citation_author=M. Santel; citation_author=A. Weigert; citation_author=F. Sanchís-Calleja; citation_author=P. Guijarro; citation_author=L. Sidow; citation_author=J.S. Fleck; citation_author=D. Han; citation_author=et al.; citation_journal_title=Nature; citation_volume=574; citation_firstpage=418; citation_lastpage=422; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=HISAT: a fast spliced aligner with low memory requirements; citation_author=D. Kim; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=357; citation_lastpage=360; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The glial nature of embryonic and adult neural stem cells; citation_author=A. Kriegstein; citation_author=A. Alvarez-Buylla; citation_journal_title=Annu. Rev. Neurosci.; citation_volume=32; citation_firstpage=149; citation_lastpage=184; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Trim Galore! v0.4.0; citation_author=F. Krueger; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The Human Transcription Factors; citation_author=S.A. Lambert; citation_author=A. Jolma; citation_author=L.F. Campitelli; citation_author=P.K. Das; citation_author=Y. Yin; citation_author=M. Albu; citation_author=X. Chen; citation_author=J. Taipale; citation_author=T.R. Hughes; citation_author=M.T. Weirauch; citation_journal_title=Cell; citation_volume=172; citation_firstpage=650; citation_lastpage=665; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Guided self-organization and cortical plate formation in human brain organoids; citation_author=M.A. Lancaster; citation_author=N.S. Corsini; citation_author=S. Wolfinger; citation_author=E.H. Gustafson; citation_author=A.W. Phillips; citation_author=T.R. Burkard; citation_author=T. Otani; citation_author=F.J. Livesey; citation_author=J.A. Knoblich; citation_journal_title=Nat. Biotechnol.; citation_volume=35; citation_firstpage=659; citation_lastpage=666; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Cerebral organoids model human brain development and microcephaly; citation_author=M.A. Lancaster; citation_author=M. Renner; citation_author=C.A. Martin; citation_author=D. Wenzel; citation_author=L.S. Bicknell; citation_author=M.E. Hurles; citation_author=T. Homfray; citation_author=J.M. Penninger; citation_author=A.P. Jackson; citation_author=J.A. Knoblich; citation_journal_title=Nature; citation_volume=501; citation_firstpage=373; citation_lastpage=379; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Shroom family proteins regulate γ-tubulin distribution and microtubule architecture during epithelial cell shape change; citation_author=C. Lee; citation_author=H.M. Scherr; citation_author=J.B. Wallingford; citation_journal_title=Development; citation_volume=134; citation_firstpage=1431; citation_lastpage=1441; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=A high-resolution map of human evolutionary constraint using 29 mammals; citation_author=K. Lindblad-Toh; citation_author=M. Garber; citation_author=O. Zuk; citation_author=M.F. Lin; citation_author=B.J. Parker; citation_author=S. Washietl; citation_author=P. Kheradpour; citation_author=J. Ernst; citation_author=G. Jordan; citation_author=E. Mauceli; citation_author=Broad Institute Sequencing Platform and Whole Genome Assembly Team; citation_author=Baylor College of Medicine Human Genome Sequencing Center Sequencing Team; citation_author=Genome Institute at Washington University; citation_author=et al.; citation_journal_title=Nature; citation_volume=478; citation_firstpage=476; citation_lastpage=482; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Extension of cortical synaptic development distinguishes humans from chimpanzees and macaques; citation_author=X. Liu; citation_author=M. Somel; citation_author=L. Tang; citation_author=Z. Yan; citation_author=X. Jiang; citation_author=S. Guo; citation_author=Y. Yuan; citation_author=L. He; citation_author=A. Oleksiak; citation_author=Y. Zhang; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=611; citation_lastpage=622; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Development and evolution of the human neocortex; citation_author=J.H. Lui; citation_author=D.V. Hansen; citation_author=A.R. Kriegstein; citation_journal_title=Cell; citation_volume=146; citation_firstpage=18; citation_lastpage=36; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Species-specific maturation profiles of human, chimpanzee and bonobo neural cells; citation_author=M.C. Marchetto; citation_author=B. Hrvoj-Mihic; citation_author=B.E. Kerman; citation_author=D.X. Yu; citation_author=K.C. Vadodaria; citation_author=S.B. Linker; citation_author=I. Narvaiza; citation_author=R. Santos; citation_author=A.M. Denli; citation_author=A.P.D. Mendes; citation_author=et al.; citation_journal_title=eLife; citation_volume=8; citation_firstpage=e37527; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; citation_author=M. Martin; citation_journal_title=EMBnet. J.; citation_volume=17; citation_firstpage=10; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Controlled expression of transgenes introduced by in&nbsp;vivo electroporation; citation_author=T. Matsuda; citation_author=C.L. Cepko; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_firstpage=1027; citation_lastpage=1032; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=In Toto Imaging and Reconstruction of Post-Implantation Mouse Development at the Single-Cell Level; citation_author=K. McDole; citation_author=L. Guignard; citation_author=F. Amat; citation_author=A. Berger; citation_author=G. Malandain; citation_author=L.A. Royer; citation_author=S.C. Turaga; citation_author=K. Branson; citation_author=P.J. Keller; citation_journal_title=Cell; citation_volume=175; citation_firstpage=859; citation_lastpage=876.e33; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Signaling-Dependent Control of Apical Membrane Size and Self-Renewal in Rosette-Stage Human Neuroepithelial Stem Cells; citation_author=J.P. Medelnik; citation_author=K. Roensch; citation_author=S. Okawa; citation_author=A. Del Sol; citation_author=O. Chara; citation_author=L. Mchedlishvili; citation_author=E.M. Tanaka; citation_journal_title=Stem Cell Reports; citation_volume=10; citation_firstpage=1751; citation_lastpage=1765; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Zeb2 regulates the balance between retinal interneurons and Müller glia by inhibition of BMP-Smad signaling; citation_author=Y. Menuchin-Lasowski; citation_author=B. Dagan; citation_author=A. Conidi; citation_author=M. Cohen-Gulkar; citation_author=A. David; citation_author=M. Ehrlich; citation_author=P.O. Giladi; citation_author=B.S. Clark; citation_author=S. Blackshaw; citation_author=K. Shapira; citation_author=et al.; citation_journal_title=Dev. Biol.; citation_volume=468; citation_firstpage=80; citation_lastpage=92; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Complementary expression pattern of Zfhx1 genes Sip1 and deltaEF1 in the mouse embryo and their genetic interaction revealed by compound mutants; citation_author=T. Miyoshi; citation_author=M. Maruhashi; citation_author=T. Van De Putte; citation_author=H. Kondoh; citation_author=D. Huylebroeck; citation_author=Y. Higashi; citation_journal_title=Dev. Dyn.; citation_volume=235; citation_firstpage=1941; citation_lastpage=1952; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Differences and similarities between human and chimpanzee neural progenitors during cerebral cortex development; citation_author=F. Mora-Bermúdez; citation_author=F. Badsha; citation_author=S. Kanton; citation_author=J.G. Camp; citation_author=B. Vernot; citation_author=K. Köhler; citation_author=B. Voigt; citation_author=K. Okita; citation_author=T. Maricic; citation_author=Z. He; citation_author=et al.; citation_journal_title=eLife; citation_volume=5; citation_firstpage=e18683; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Actin cytoskeletal control during epithelial to mesenchymal transition: focus on the pancreas and intestinal tract; citation_author=H.T. Morris; citation_author=L.M. Machesky; citation_journal_title=Br. J. Cancer; citation_volume=112; citation_firstpage=613; citation_lastpage=620; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Mowat-Wilson syndrome; citation_author=D.R. Mowat; citation_author=M.J. Wilson; citation_author=M. Goossens; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=40; citation_firstpage=305; citation_lastpage=310; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Distinct behaviors of neural stem and progenitor cells underlie cortical neurogenesis; citation_author=S.C. Noctor; citation_author=V. Martínez-Cerdeño; citation_author=A.R. Kriegstein; citation_journal_title=J.&nbsp;Comp. Neurol.; citation_volume=508; citation_firstpage=28; citation_lastpage=44; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Transformation of the Radial Glia Scaffold Demarcates Two Stages of Human Cerebral Cortex Development; citation_author=T.J. Nowakowski; citation_author=A.A. Pollen; citation_author=C. Sandoval-Espinosa; citation_author=A.R. Kriegstein; citation_journal_title=Neuron; citation_volume=91; citation_firstpage=1219; citation_lastpage=1227; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A Primate-Specific Isoform of PLEKHG6 Regulates Neurogenesis and Neuronal Migration; citation_author=A.C. O’Neill; citation_author=C. Kyrousi; citation_author=J. Klaus; citation_author=R.J. Leventer; citation_author=E.P. Kirk; citation_author=A. Fry; citation_author=D.T. Pilz; citation_author=T. Morgan; citation_author=Z.A. Jenkins; citation_author=M. Drukker; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=25; citation_firstpage=2729; citation_lastpage=2741.e6; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Downregulation of Notch mediates the seamless transition of individual Drosophila neuroepithelial progenitors into optic medullar neuroblasts during prolonged G1; citation_author=M. Orihara-Ono; citation_author=M. Toriya; citation_author=K. Nakao; citation_author=H. Okano; citation_journal_title=Dev. Biol.; citation_volume=351; citation_firstpage=163; citation_lastpage=175; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=2D and 3D Stem Cell Models of Primate Cortical Development Identify Species-Specific Differences in Progenitor Behavior Contributing to Brain Size; citation_author=T. Otani; citation_author=M.C. Marchetto; citation_author=F.H. Gage; citation_author=B.D. Simons; citation_author=F.J. Livesey; citation_journal_title=Cell Stem Cell; citation_volume=18; citation_firstpage=467; citation_lastpage=480; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Ntf3 acts downstream of Sip1 in cortical postmitotic neurons to control progenitor cell fate through feedback signaling; citation_author=S. Parthasarathy; citation_author=S. Srivatsa; citation_author=A. Nityanandam; citation_author=V. Tarabykin; citation_journal_title=Development; citation_volume=141; citation_firstpage=3324; citation_lastpage=3330; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture; citation_author=A.M. Paşca; citation_author=S.A. Sloan; citation_author=L.E. Clarke; citation_author=Y. Tian; citation_author=C.D. Makinson; citation_author=N. Huber; citation_author=C.H. Kim; citation_author=J.Y. Park; citation_author=N.A. O’Rourke; citation_author=K.D. Nguyen; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=671; citation_lastpage=678; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=EMT Transition States during Tumor Progression and Metastasis; citation_author=I. Pastushenko; citation_author=C. Blanpain; citation_journal_title=Trends Cell Biol.; citation_volume=29; citation_firstpage=212; citation_lastpage=226; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Snail, Zeb and bHLH factors in tumour progression: an alliance against the epithelial phenotype?; citation_author=H. Peinado; citation_author=D. Olmeda; citation_author=A. Cano; citation_journal_title=Nat. Rev. Cancer; citation_volume=7; citation_firstpage=415; citation_lastpage=428; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Forces shaping the fastest evolving regions in the human genome; citation_author=K.S. Pollard; citation_author=S.R. Salama; citation_author=B. King; citation_author=A.D. Kern; citation_author=T. Dreszer; citation_author=S. Katzman; citation_author=A. Siepel; citation_author=J.S. Pedersen; citation_author=G. Bejerano; citation_author=R. Baertsch; citation_author=et al.; citation_journal_title=PLoS Genet.; citation_volume=2; citation_firstpage=e168; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Establishing Cerebral Organoids as Models of Human-Specific Brain Evolution; citation_author=A.A. Pollen; citation_author=A. Bhaduri; citation_author=M.G. Andrews; citation_author=T.J. Nowakowski; citation_author=O.S. Meyerson; citation_author=M.A. Mostajo-Radji; citation_author=E. Di Lullo; citation_author=B. Alvarado; citation_author=M. Bedolli; citation_author=M.L. Dougherty; citation_author=et al.; citation_journal_title=Cell; citation_volume=176; citation_firstpage=743; citation_lastpage=756.e17; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Generation of human brain region-specific organoids using a miniaturized spinning bioreactor; citation_author=X. Qian; citation_author=F. Jacob; citation_author=M.M. Song; citation_author=H.N. Nguyen; citation_author=H. Song; citation_author=G.L. Ming; citation_journal_title=Nat. Protoc.; citation_volume=13; citation_firstpage=565; citation_lastpage=580; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Cell diversity and network dynamics in photosensitive human brain organoids; citation_author=G. Quadrato; citation_author=T. Nguyen; citation_author=E.Z. Macosko; citation_author=J.L. Sherwood; citation_author=S. Min Yang; citation_author=D.R. Berger; citation_author=N. Maria; citation_author=J. Scholvin; citation_author=M. Goldman; citation_author=J.P. Kinney; citation_author=et al.; citation_journal_title=Nature; citation_volume=545; citation_firstpage=48; citation_lastpage=53; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing; citation_author=R Core Team; citation_publication_date=2018; citation_publisher=R Foundation for Statistical Computing, Vienna, Austria; "><meta name="citation_reference" content="citation_title=Specification of cerebral cortical areas; citation_author=P. Rakic; citation_journal_title=Science; citation_volume=241; citation_firstpage=170; citation_lastpage=176; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=A small step for the cell, a giant leap for mankind: a hypothesis of neocortical expansion during evolution; citation_author=P. Rakic; citation_journal_title=Trends Neurosci.; citation_volume=18; citation_firstpage=383; citation_lastpage=388; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=The radial edifice of cortical architecture: from neuronal silhouettes to genetic engineering; citation_author=P. Rakic; citation_journal_title=Brain Res. Brain Res. Rev.; citation_volume=55; citation_firstpage=204; citation_lastpage=219; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Genome engineering using the CRISPR-Cas9 system; citation_author=F.A. Ran; citation_author=P.D. Hsu; citation_author=J. Wright; citation_author=V. Agarwala; citation_author=D.A. Scott; citation_author=F. Zhang; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=2281; citation_lastpage=2308; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=g:Profiler--a web-based toolset for functional profiling of gene lists from large-scale experiments; citation_author=J. Reimand; citation_author=M. Kull; citation_author=H. Peterson; citation_author=J. Hansen; citation_author=J. Vilo; citation_journal_title=Nucleic Acids Res.; citation_volume=35; citation_firstpage=W193-200; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The basic uniformity in structure of the neocortex; citation_author=A.J. Rockel; citation_author=R.W. Hiorns; citation_author=T.P.S. Powell; citation_journal_title=Brain; citation_volume=103; citation_firstpage=221; citation_lastpage=244; citation_publication_date=1980; "><meta name="citation_reference" content="citation_title=A catenin-dependent balance between N-cadherin and E-cadherin controls neuroectodermal cell fate choices; citation_author=C.D. Rogers; citation_author=L.K. Sorrells; citation_author=M.E. Bronner; citation_journal_title=Mech. Dev.; citation_volume=152; citation_firstpage=44; citation_lastpage=56; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Sub-circuits of a gene regulatory network control a developmental epithelial-mesenchymal transition; citation_author=L.R. Saunders; citation_author=D.R. McClay; citation_journal_title=Development; citation_volume=141; citation_firstpage=1503; citation_lastpage=1513; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Sip1 regulates sequential fate decisions by feedback signaling from postmitotic neurons to progenitors; citation_author=E. Seuntjens; citation_author=A. Nityanandam; citation_author=A. Miquelajauregui; citation_author=J. Debruyn; citation_author=A. Stryjewska; citation_author=S. Goebbels; citation_author=K.A. Nave; citation_author=D. Huylebroeck; citation_author=V. Tarabykin; citation_journal_title=Nat. Neurosci.; citation_volume=12; citation_firstpage=1373; citation_lastpage=1380; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Polarity transitions during neurogenesis and germinal zone exit in the developing central nervous system; citation_author=S. Singh; citation_author=D.J. Solecki; citation_journal_title=Front. Cell. Neurosci.; citation_volume=9; citation_firstpage=62; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Evolution of the Human Nervous System Function, Structure, and Development; citation_author=A.M.M. Sousa; citation_author=K.A. Meyer; citation_author=G. Santpere; citation_author=F.O. Gulden; citation_author=N. Sestan; citation_journal_title=Cell; citation_volume=170; citation_firstpage=226; citation_lastpage=247; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Sip1 downstream Effector ninein controls neocortical axonal growth, ipsilateral branching, and microtubule growth and stability; citation_author=S. Srivatsa; citation_author=S. Parthasarathy; citation_author=Z. Molnár; citation_author=V. Tarabykin; citation_journal_title=Neuron; citation_volume=85; citation_firstpage=998; citation_lastpage=1012; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Non-redundant functions of EMT transcription factors; citation_author=M.P. Stemmler; citation_author=R.L. Eccles; citation_author=S. Brabletz; citation_author=T. Brabletz; citation_journal_title=Nat. Cell Biol.; citation_volume=21; citation_firstpage=102; citation_lastpage=112; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Zeb2 regulates cell fate at the exit from epiblast state in mouse embryonic stem cells; citation_author=A. Stryjewska; citation_author=R. Dries; citation_author=T. Pieters; citation_author=G. Verstappen; citation_author=A. Conidi; citation_author=K. Coddens; citation_author=A. Francis; citation_author=L. Umans; citation_author=W.F.J. van IJcken; citation_author=G. Berx; citation_author=et al.; citation_journal_title=Stem Cells; citation_volume=35; citation_firstpage=611; citation_lastpage=625; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Dynamic behaviour of human neuroepithelial cells in the developing forebrain; citation_author=L. Subramanian; citation_author=M. Bershteyn; citation_author=M.F. Paredes; citation_author=A.R. Kriegstein; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=14167; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Human-Specific NOTCH2NL Genes Expand Cortical Neurogenesis through Delta/Notch Regulation; citation_author=I.K. Suzuki; citation_author=D. Gacquer; citation_author=R. Heurck; citation_author=Van; citation_author=F. Polleux; citation_author=V. Detours; citation_author=P. Vanderhaeghen; citation_journal_title=Cell; citation_volume=173; citation_firstpage=1370; citation_lastpage=1384.e16; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=SIP1/ZEB2 induces EMT by repressing genes of different epithelial cell-cell junctions; citation_author=C. Vandewalle; citation_author=J. Comijn; citation_author=B. De Craene; citation_author=P. Vermassen; citation_author=E. Bruyneel; citation_author=H. Andersen; citation_author=E. Tulchinsky; citation_author=F. Van Roy; citation_author=G. Berx; citation_journal_title=Nucleic Acids Res.; citation_volume=33; citation_firstpage=6566; citation_lastpage=6578; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=ZEB2 Represses the Epithelial Phenotype and Facilitates Metastasis in Ewing Sarcoma; citation_author=E.T. Wiles; citation_author=R. Bell; citation_author=D. Thomas; citation_author=M. Beckerle; citation_author=S.L. Lessnick; citation_journal_title=Genes Cancer; citation_volume=4; citation_firstpage=486; citation_lastpage=500; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Mouse primitive streak forms in&nbsp;situ by initiation of epithelial to mesenchymal transition without migration of a cell population; citation_author=M. Williams; citation_author=C. Burdsal; citation_author=A. Periasamy; citation_author=M. Lewandoski; citation_author=A. Sutherland; citation_journal_title=Dev. Dyn.; citation_volume=241; citation_firstpage=270; citation_lastpage=283; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Multi-view light-sheet imaging and tracking with the MaMuT software reveals the cell lineage of a direct developing arthropod limb; citation_author=C. Wolff; citation_author=J.Y. Tinevez; citation_author=T. Pietzsch; citation_author=E. Stamataki; citation_author=B. Harich; citation_author=L. Guignard; citation_author=S. Preibisch; citation_author=S. Shorte; citation_author=P.J. Keller; citation_author=P. Tomancak; citation_author=A. Pavlopoulos; citation_journal_title=eLife; citation_volume=7; citation_firstpage=e34410; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=TCseq: Time course sequencing data analysis. R package version 1.12.0; citation_author=M. Wu; citation_author=L. Gu; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Primate iPS cells as tools for evolutionary analyses; citation_author=S. Wunderlich; citation_author=M. Kircher; citation_author=B. Vieth; citation_author=A. Haase; citation_author=S. Merkert; citation_author=J. Beier; citation_author=G. Göhring; citation_author=S. Glage; citation_author=A. Schambach; citation_author=E.C. Curnow; citation_author=et al.; citation_journal_title=Stem Cell Res. (Amst.); citation_volume=12; citation_firstpage=622; citation_lastpage=629; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Regulation of trunk neural crest delamination by δEF1 and Sip1 in the chicken embryo; citation_author=T. Yasumi; citation_author=M. Inoue; citation_author=M. Maruhashi; citation_author=Y. Kamachi; citation_author=Y. Higashi; citation_author=H. Kondoh; citation_author=M. Uchikawa; citation_journal_title=Dev. Growth Differ.; citation_volume=58; citation_firstpage=205; citation_lastpage=214; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=SHROOM2 inhibits tumor metastasis through RhoA-ROCK pathway-dependent and -independent mechanisms in nasopharyngeal carcinoma; citation_author=J. Yuan; citation_author=L. Chen; citation_author=J. Xiao; citation_author=X.K. Qi; citation_author=J. Zhang; citation_author=X. Li; citation_author=Z. Wang; citation_author=Y.F. Lian; citation_author=T. Xiang; citation_author=Y. Zhang; citation_author=et al.; citation_journal_title=Cell Death Dis.; citation_volume=10; citation_firstpage=58; citation_publication_date=2019; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The human brain has undergone rapid expansion since humans diverged from other great apes, but the mechanism of this human-specific enlargement is still unknown. Here, we use cerebral organoids derived from human, gorilla, and chimpanzee cells to study developmental mechanisms driving evolutionary brain expansion. We find that neuroepithelial differentiation is a protracted process in apes, involving a previously unrecognized transition state characterized by a change in cell shape. Furthermore, we show that human organoids are larger due to a delay in this transition, associated with differences in interkinetic nuclear migration and cell cycle length. Comparative RNA sequencing (RNA-seq) reveals differences in expression dynamics of cell morphogenesis factors, including ZEB2, a known epithelial-mesenchymal transition regulator. We show that ZEB2 promotes neuroepithelial transition, and its manipulation and downstream signaling leads to acquisition of nonhuman ape architecture in the human context and vice versa, establishing an important role for neuroepithelial cell shape in human brain expansion.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7274&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7&amp;be=3118&amp;fe=6720&amp;dc=4528&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749589552482,%22n%22:0,%22f%22:2536,%22dn%22:2536,%22dne%22:2536,%22c%22:2536,%22s%22:2536,%22ce%22:2536,%22rq%22:2537,%22rp%22:3095,%22rpe%22:4032,%22dl%22:3100,%22di%22:4405,%22ds%22:4528,%22de%22:4757,%22dc%22:6720,%22l%22:6720,%22le%22:6869%7D,%22navigation%22:%7B%7D%7D&amp;fp=3632&amp;fcp=3632&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style>stream {
  position: relative;
  display: block;
}
</style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c210E {
  padding: 0.694em 0.576em 0.011em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00239-7"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-722973" aria-expanded="false" aria-controls="login-links-722973" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-722973" id="login-links--722973" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-722973" data-opens="#links-dialog-722973" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-722973" aria-modal="true"><button data-closes="#links-dialog-722973" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900239-7" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.02.050">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00239-7" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0009-2"><span>Volume 184</span>, <span>Issue 8</span></a></span><span class="meta-panel__pages"><span>p2084-2102.e19</span></span><span class="meta-panel__onlineDate">April 15, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=8&amp;pii=S0092-8674%2820%29X0009-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">An early cell shape transition drives evolutionary expansion of the human forebrain</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Silvia</span> <span property="familyName">Benito-Kwiecinski</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Silvia</span> <span property="familyName">Benito-Kwiecinski</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0821-3858" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0821-3858</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Benito-Kwiecinski/Silvia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">6</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Stefano L.</span> <span property="familyName">Giandomenico</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stefano L.</span> <span property="familyName">Giandomenico</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Giandomenico/Stefano+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">6</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Magdalena</span> <span property="familyName">Sutcliffe</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Magdalena</span> <span property="familyName">Sutcliffe</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Sutcliffe/Magdalena" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Erlend S.</span> <span property="familyName">Riis</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Erlend S.</span> <span property="familyName">Riis</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9656-4123" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9656-4123</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Wilberforce Road, Cambridge CB3 0WA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Riis/Erlend+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Paula</span> <span property="familyName">Freire-Pritchett</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Paula</span> <span property="familyName">Freire-Pritchett</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7090-3016" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7090-3016</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Freire-Pritchett/Paula" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Iva</span> <span property="familyName">Kelava</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Iva</span> <span property="familyName">Kelava</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Kelava/Iva" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Stephanie</span> <span property="familyName">Wunderlich</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephanie</span> <span property="familyName">Wunderlich</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Leibniz Research Laboratories for Biotechnology and Artificial Organs (LEBAO), REBIRTH-Research Center for Translational and Regenerative Medicine, Hannover Medical School, 30625 Hannover, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Biomedical Research in Endstage and Obstructive Lung Disease (BREATH), Member of the German Center for Lung Research (DZL), Hannover Medical School, 30625 Hannover, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Wunderlich/Stephanie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Ulrich</span> <span property="familyName">Martin</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ulrich</span> <span property="familyName">Martin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Leibniz Research Laboratories for Biotechnology and Artificial Organs (LEBAO), REBIRTH-Research Center for Translational and Regenerative Medicine, Hannover Medical School, 30625 Hannover, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">Biomedical Research in Endstage and Obstructive Lung Disease (BREATH), Member of the German Center for Lung Research (DZL), Hannover Medical School, 30625 Hannover, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Martin/Ulrich" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Gregory A.</span> <span property="familyName">Wray</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gregory A.</span> <span property="familyName">Wray</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Duke University, Biological Sciences Building, 124 Science Drive, Durham, NC 27708, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wray/Gregory+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Kate</span> <span property="familyName">McDole</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kate</span> <span property="familyName">McDole</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/McDole/Kate" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Madeline A.</span> <span property="familyName">Lancaster</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Madeline A.</span> <span property="familyName">Lancaster</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:madeline.lancaster@mrc-lmb.cam.ac.uk" property="email" title="Send email to Madeline A. Lancaster">madeline.lancaster@mrc-lmb.cam.ac.uk</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Lancaster/Madeline+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">7</a></sup> <a id="eadd0010" href="mailto:madeline.lancaster@mrc-lmb.cam.ac.uk" property="email" title="Send email to Madeline A. Lancaster">madeline.lancaster@mrc-lmb.cam.ac.uk</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">MRC Laboratory of Molecular Biology, Cambridge Biomedical Campus, Francis Crick Avenue, Cambridge CB2 0QH, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Wilberforce Road, Cambridge CB3 0WA, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Leibniz Research Laboratories for Biotechnology and Artificial Organs (LEBAO), REBIRTH-Research Center for Translational and Regenerative Medicine, Hannover Medical School, 30625 Hannover, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Biomedical Research in Endstage and Obstructive Lung Disease (BREATH), Member of the German Center for Lung Research (DZL), Hannover Medical School, 30625 Hannover, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Biology, Duke University, Biological Sciences Building, 124 Science Drive, Durham, NC 27708, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">7</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received July 6, 2020</span>; <span>Revised December 10, 2020</span>; <span>Accepted February 22, 2021</span>; <span>Published online March 24, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.02.050" property="sameAs" target="_blank">10.1016/j.cell.2021.02.050 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421002397" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 MRC Laboratory of Molecular Biology. Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution (CC BY 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.02.050" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0009-2"><img src="/cms/asset/cc130920-993e-4c33-91e0-6ebab7265780/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 8"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900239-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (2)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900239-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.02.050">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900239-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.02.050&amp;pii=S0092-8674%2821%2900239-7" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900239-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.02.050">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.02.050" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421002397&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421002397&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=8&amp;pii=S0092-8674%2820%29X0009-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00365-2" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00234-8" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental information (2)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Human brain organoids are expanded relative to nonhuman apes prior to neurogenesis</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Ape neural progenitors go through a newly identified transition morphotype state</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Delayed morphological transition with shorter cell cycles underlie human expansion</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">ZEB2 is as an evolutionary regulator of this transition</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The human brain has undergone rapid expansion since humans diverged from other great apes, but the mechanism of this human-specific enlargement is still unknown. Here, we use cerebral organoids derived from human, gorilla, and chimpanzee cells to study developmental mechanisms driving evolutionary brain expansion. We find that neuroepithelial differentiation is a protracted process in apes, involving a previously unrecognized transition state characterized by a change in cell shape. Furthermore, we show that human organoids are larger due to a delay in this transition, associated with differences in interkinetic nuclear migration and cell cycle length. Comparative RNA sequencing (RNA-seq) reveals differences in expression dynamics of cell morphogenesis factors, including ZEB2, a known epithelial-mesenchymal transition regulator. We show that ZEB2 promotes neuroepithelial transition, and its manipulation and downstream signaling leads to acquisition of nonhuman ape architecture in the human context and vice versa, establishing an important role for neuroepithelial cell shape in human brain expansion.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/2b71edbe-7fd1-4342-bdfc-8289422eebd4/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.02.050/asset/14928b4f-0dca-4bc9-983b-87346f06a4b9/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/2b71edbe-7fd1-4342-bdfc-8289422eebd4/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;brain&quot;" alt="Search on this keyword" rel="nofollow">brain</a></li><li><a href="/action/doSearch?AllField=&quot;evolution&quot;" alt="Search on this keyword" rel="nofollow">evolution</a></li><li><a href="/action/doSearch?AllField=&quot;cell+shape&quot;" alt="Search on this keyword" rel="nofollow">cell shape</a></li><li><a href="/action/doSearch?AllField=&quot;organoids&quot;" alt="Search on this keyword" rel="nofollow">organoids</a></li><li><a href="/action/doSearch?AllField=&quot;brain+expansion&quot;" alt="Search on this keyword" rel="nofollow">brain expansion</a></li><li><a href="/action/doSearch?AllField=&quot;neuroepithelium&quot;" alt="Search on this keyword" rel="nofollow">neuroepithelium</a></li><li><a href="/action/doSearch?AllField=&quot;neural+stem+cells&quot;" alt="Search on this keyword" rel="nofollow">neural stem cells</a></li><li><a href="/action/doSearch?AllField=&quot;ZEB2&quot;" alt="Search on this keyword" rel="nofollow">ZEB2</a></li><li><a href="/action/doSearch?AllField=&quot;gorilla&quot;" alt="Search on this keyword" rel="nofollow">gorilla</a></li><li><a href="/action/doSearch?AllField=&quot;chimpanzee&quot;" alt="Search on this keyword" rel="nofollow">chimpanzee</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">One of the most distinctive features of humans as a species is an enlarged brain. The human brain exhibits a 1,000-fold increase in total neuron number compared with mice (<span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Herculano-Houzel et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Herculano-Houzel, S. ∙ Mota, B. ∙ Lent, R.</div><div><strong>Cellular scaling rules for rodent brains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:12138-12143</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0604911103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33747046064" target="_blank" aria-label="Open Scopus in new tab">Scopus (352)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16880386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0604911103&amp;pmid=16880386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and is the largest of all primates, roughly 3-fold larger than that of our closest living relatives, chimpanzees and gorillas (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Herculano-Houzel, 2012</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Herculano-Houzel, S.</div><div><strong>The remarkable, yet not extraordinary, human brain as a scaled-up primate brain and its associated cost</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:10661-10668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1201895109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84866172357" target="_blank" aria-label="Open Scopus in new tab">Scopus (419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1201895109&amp;pmid=22723358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A number of studies have examined differences in brain development between human, mouse, and other distantly related model organisms (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Bae et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bae, B.I. ∙ Jayaraman, D. ∙ Walsh, C.A.</div><div><strong>Genetic changes shaping the human brain</strong></div><div><em>Dev. Cell.</em> 2015; <strong>32</strong>:423-434</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.devcel.2015.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.devcel.2015.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84923224807" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25710529/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2015.01.035&amp;pmid=25710529" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Lui et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Lui, J.H. ∙ Hansen, D.V. ∙ Kriegstein, A.R.</div><div><strong>Development and evolution of the human neocortex</strong></div><div><em>Cell.</em> 2011; <strong>146</strong>:18-36</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-79959924122" target="_blank" aria-label="Open Scopus in new tab">Scopus (1009)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21729779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2011.06.030&amp;pmid=21729779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Sousa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Sousa, A.M.M. ∙ Meyer, K.A. ∙ Santpere, G. ...</div><div><strong>Evolution of the Human Nervous System Function, Structure, and Development</strong></div><div><em>Cell.</em> 2017; <strong>170</strong>:226-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85023612934" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28708995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.06.036&amp;pmid=28708995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), highlighting divergence in neural progenitor behavior (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Florio et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Florio, M. ∙ Albert, M. ∙ Taverna, E. ...</div><div><strong>Human-specific gene ARHGAP11B promotes basal progenitor amplification and neocortex expansion</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1465-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa1975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84928704517" target="_blank" aria-label="Open Scopus in new tab">Scopus (460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25721503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa1975&amp;pmid=25721503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), neurogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">O’Neill et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>O’Neill, A.C. ∙ Kyrousi, C. ∙ Klaus, J. ...</div><div><strong>A Primate-Specific Isoform of PLEKHG6 Regulates Neurogenesis and Neuronal Migration</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:2729-2741.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.celrep.2018.11.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.celrep.2018.11.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85057030381" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30517861/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.11.029&amp;pmid=30517861" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and cytoarchitecture (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Dehay et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Dehay, C. ∙ Kennedy, H. ∙ Kosik, K.S.</div><div><strong>The outer subventricular zone and primate-specific cortical complexification</strong></div><div><em>Neuron.</em> 2015; <strong>85</strong>:683-694</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2014.12.060&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2014.12.060&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84925443036" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25695268/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2014.12.060&amp;pmid=25695268" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Fietz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Fietz, S.A. ∙ Kelava, I. ∙ Vogt, J. ...</div><div><strong>OSVZ progenitors of human and ferret neocortex are epithelial-like and expand by integrin signaling</strong></div><div><em>Nat. Neurosci.</em> 2010; <strong>13</strong>:690-699</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2553" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77952867780" target="_blank" aria-label="Open Scopus in new tab">Scopus (650)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436478/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2553&amp;pmid=20436478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Hansen et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Hansen, D.V. ∙ Lui, J.H. ∙ Parker, P.R.L. ...</div><div><strong>Neurogenic radial glia in the outer subventricular zone of human neocortex</strong></div><div><em>Nature.</em> 2010; <strong>464</strong>:554-561</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08845" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77950076985" target="_blank" aria-label="Open Scopus in new tab">Scopus (1019)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20154730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08845&amp;pmid=20154730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although much has been learned about the mechanisms governing evolutionary divergence of primates from rodents, human-specific changes compared to other apes are less well understood.</div><div id="p0035" role="paragraph">Comparative studies have led to a number of important insights. Early in development, before the onset of neurogenesis, the human forebrain is already larger than that of mouse and macaque, indicating differences in expansion of precursor cells called neuroepithelial (NE) cells (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Rakic, 2007</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Rakic, P.</div><div><strong>The radial edifice of cortical architecture: from neuronal silhouettes to genetic engineering</strong></div><div><em>Brain Res. Brain Res. Rev.</em> 2007; <strong>55</strong>:204-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainresrev.2007.02.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-35048888932" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17467805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainresrev.2007.02.010&amp;pmid=17467805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, it has long been hypothesized that changes in these cells could lead to an expansion of the neocortical primordium (<span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Rakic, 1988</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Rakic, P.</div><div><strong>Specification of cerebral cortical areas</strong></div><div><em>Science.</em> 1988; <strong>241</strong>:170-176</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.3291116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0023785808" target="_blank" aria-label="Open Scopus in new tab">Scopus (2535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3291116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.3291116&amp;pmid=3291116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" aria-label="Reference 1995" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">1995</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Rakic, P.</div><div><strong>A small step for the cell, a giant leap for mankind: a hypothesis of neocortical expansion during evolution</strong></div><div><em>Trends Neurosci.</em> 1995; <strong>18</strong>:383-388</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2F0166-2236%2895%2993934-P&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2F0166-2236%2895%2993934-P&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0029132983" target="_blank" aria-label="Open Scopus in new tab">Scopus (878)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7482803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0166-2236%2895%2993934-P&amp;pmid=7482803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Comparison of adult brain morphology has also revealed non-uniform thickening of cortical layers in primates, especially apes (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Charvet et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Charvet, C.J. ∙ Stimpson, C.D. ∙ Kim, Y.D. ...</div><div><strong>Gradients in cytoarchitectural landscapes of the isocortex: Diprotodont marsupials in comparison to eutherian mammals</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2017; <strong>525</strong>:1811-1826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.24160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85014996412" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28001295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.24160&amp;pmid=28001295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">DeFelipe et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>DeFelipe, J. ∙ Alonso-Nanclares, L. ∙ Arellano, J.I.</div><div><strong>Microstructure of the neocortex: comparative aspects</strong></div><div><em>J.&nbsp;Neurocytol.</em> 2002; <strong>31</strong>:299-316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1024130211265" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0038372036" target="_blank" aria-label="Open Scopus in new tab">Scopus (514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12815249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1024130211265&amp;pmid=12815249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Hutsler et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hutsler, J.J. ∙ Lee, D.G. ∙ Porter, K.K.</div><div><strong>Comparative analysis of cortical layering and supragranular layer enlargement in rodent carnivore and primate species</strong></div><div><em>Brain Res.</em> 2005; <strong>1052</strong>:71-81</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainres.2005.06.015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-23744472001" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16018988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainres.2005.06.015&amp;pmid=16018988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Nowakowski et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Nowakowski, T.J. ∙ Pollen, A.A. ∙ Sandoval-Espinosa, C. ...</div><div><strong>Transformation of the Radial Glia Scaffold Demarcates Two Stages of Human Cerebral Cortex Development</strong></div><div><em>Neuron.</em> 2016; <strong>91</strong>:1219-1227</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2016.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2016.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84991263060" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27657449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2016.09.005&amp;pmid=27657449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Rockel et&nbsp;al., 1980</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Rockel, A.J. ∙ Hiorns, R.W. ∙ Powell, T.P.S.</div><div><strong>The basic uniformity in structure of the neocortex</strong></div><div><em>Brain.</em> 1980; <strong>103</strong>:221-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/brain/103.2.221" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0018948139" target="_blank" aria-label="Open Scopus in new tab">Scopus (671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6772266/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbrain%2F103.2.221&amp;pmid=6772266" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Sousa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Sousa, A.M.M. ∙ Meyer, K.A. ∙ Santpere, G. ...</div><div><strong>Evolution of the Human Nervous System Function, Structure, and Development</strong></div><div><em>Cell.</em> 2017; <strong>170</strong>:226-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85023612934" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28708995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.06.036&amp;pmid=28708995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Given that cortical layers are generated successively over time, this points to expansion events in humans and other apes after the onset of neurogenesis. However, comparisons of the human brain to that of other apes reveal a general increase in size without an expansion in particular layers of the cerebral cortex (<span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Donahue et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Donahue, C.J. ∙ Glasser, M.F. ∙ Preuss, T.M. ...</div><div><strong>Quantitative assessment of prefrontal cortex in humans relative to nonhuman primates</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E5183-E5192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1721653115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047898223" target="_blank" aria-label="Open Scopus in new tab">Scopus (162)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29739891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1721653115&amp;pmid=29739891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">de Sousa et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>de Sousa, A.A. ∙ Sherwood, C.C. ∙ Schleicher, A. ...</div><div><strong>Comparative cytoarchitectural analyses of striate and extrastriate areas in hominoids</strong></div><div><em>Cereb. Cortex.</em> 2010; <strong>20</strong>:966-981</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/cercor/bhp158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77949351275" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19776344/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcercor%2Fbhp158&amp;pmid=19776344" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Hutsler et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hutsler, J.J. ∙ Lee, D.G. ∙ Porter, K.K.</div><div><strong>Comparative analysis of cortical layering and supragranular layer enlargement in rodent carnivore and primate species</strong></div><div><em>Brain Res.</em> 2005; <strong>1052</strong>:71-81</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainres.2005.06.015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-23744472001" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16018988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainres.2005.06.015&amp;pmid=16018988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), suggesting that while primate expansion may have been governed by both early and late changes, human-specific differences likely take place prior to the generation of cortical layer neurons. However, without being able to examine and manipulate early progenitor cell behavior in apes and humans, these observations have remained correlative.</div><div id="p0040" role="paragraph">Most of our understanding of the principles governing mammalian brain development comes from studying the mouse, where at ∼E10, cells rapidly undergo a switch from NE cells to radial glia (RG), triggering the onset of neurogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Kriegstein and Alvarez-Buylla, 2009</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Kriegstein, A. ∙ Alvarez-Buylla, A.</div><div><strong>The glial nature of embryonic and adult neural stem cells</strong></div><div><em>Annu. Rev. Neurosci.</em> 2009; <strong>32</strong>:149-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.051508.135600" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-67651036549" target="_blank" aria-label="Open Scopus in new tab">Scopus (1820)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19555289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.051508.135600&amp;pmid=19555289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Before this transition, NE cells exhibit a columnar morphology and divide in a symmetric proliferative manner. This proliferation results in an exponential expansion leading to a ballooning out of the neocortex and enlargement of the ventricles in humans (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Bayer et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Bayer, S.A. ∙ Altman, J. ∙ Russo, R.J. ...</div><div><strong>Timetables of neurogenesis in the human brain based on experimentally determined patterns in the rat</strong></div><div><em>Neurotoxicology.</em> 1993; <strong>14</strong>:83-144</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8361683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=8361683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). NE cells are strongly epithelial in character and held together near their apical surface by cell-cell junctions, namely tight junction components (i.e., OCLN) and adherens junction components (i.e., CDH1) (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Aaku-Saraste et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aaku-Saraste, E. ∙ Hellwig, A. ∙ Huttner, W.B.</div><div><strong>Loss of occludin and functional tight junctions, but not ZO-1, during neural tube closure--remodeling of the neuroepithelium prior to neurogenesis</strong></div><div><em>Dev. Biol.</em> 1996; <strong>180</strong>:664-679</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1996.0336" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0030589622" target="_blank" aria-label="Open Scopus in new tab">Scopus (221)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8954735/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1996.0336&amp;pmid=8954735" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Rogers et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Rogers, C.D. ∙ Sorrells, L.K. ∙ Bronner, M.E.</div><div><strong>A catenin-dependent balance between N-cadherin and E-cadherin controls neuroectodermal cell fate choices</strong></div><div><em>Mech. Dev.</em> 2018; <strong>152</strong>:44-56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mod.2018.07.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85049778697" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30009960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mod.2018.07.003&amp;pmid=30009960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In rodents and other model vertebrates, it is well-documented that the transition to neurogenic RG cells involves a loss of some epithelial features, a thinning and elongation of the cell, the onset of expression of certain glial markers, and a switch to asymmetric cell division with one daughter remaining a RG, and the other becoming more differentiated and leaving the ventricular zone (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Götz and Huttner, 2005</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Götz, M. ∙ Huttner, W.B.</div><div><strong>The cell biology of neurogenesis</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2005; <strong>6</strong>:777-788</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm1739" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-24344510750" target="_blank" aria-label="Open Scopus in new tab">Scopus (1671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16314867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm1739&amp;pmid=16314867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This switch in cell fate thus leads to a change in expansion from tangential to radial, with additional cells being added to the more basal layers, rather than along the apical surface of the ventricles. Although this NE-to-RG conversion has been studied in mice, where it occurs in a matter of hours, it remains entirely unknown how this process occurs in humans and other apes.</div><div id="p0045" role="paragraph">The ability to generate brain organoids from induced pluripotent stem cells (iPSCs) has enabled researchers to study a variety of neurodevelopmental processes that were previously inaccessible (<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Lancaster et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lancaster, M.A. ∙ Renner, M. ∙ Martin, C.A. ...</div><div><strong>Cerebral organoids model human brain development and microcephaly</strong></div><div><em>Nature.</em> 2013; <strong>501</strong>:373-379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12517" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84884414984" target="_blank" aria-label="Open Scopus in new tab">Scopus (3474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23995685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12517&amp;pmid=23995685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Kadoshima et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Kadoshima, T. ∙ Sakaguchi, H. ∙ Nakano, T. ...</div><div><strong>Self-organization of axial polarity, inside-out layer pattern, and species-specific progenitor dynamics in human ES cell-derived neocortex</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:20284-20289</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1315710110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84890282623" target="_blank" aria-label="Open Scopus in new tab">Scopus (692)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24277810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1315710110&amp;pmid=24277810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Paşca et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Paşca, A.M. ∙ Sloan, S.A. ∙ Clarke, L.E. ...</div><div><strong>Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:671-678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84934442315" target="_blank" aria-label="Open Scopus in new tab">Scopus (1049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26005811/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3415&amp;pmid=26005811" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Quadrato et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Quadrato, G. ∙ Nguyen, T. ∙ Macosko, E.Z. ...</div><div><strong>Cell diversity and network dynamics in photosensitive human brain organoids</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:48-53</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85018435810" target="_blank" aria-label="Open Scopus in new tab">Scopus (813)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28445462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22047&amp;pmid=28445462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recently, a number of studies have started tackling evolutionary differences across primates, highlighting human-specific patterns of gene expression (<span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Kanton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Mora-Bermúdez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Mora-Bermúdez, F. ∙ Badsha, F. ∙ Kanton, S. ...</div><div><strong>Differences and similarities between human and chimpanzee neural progenitors during cerebral cortex development</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e18683</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.18683" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84995486924" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27669147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.18683&amp;pmid=27669147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Pollen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Pollen, A.A. ∙ Bhaduri, A. ∙ Andrews, M.G. ...</div><div><strong>Establishing Cerebral Organoids as Models of Human-Specific Brain Evolution</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:743-756.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2019.01.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2019.01.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85060531675" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30735633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.01.017&amp;pmid=30735633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Human organoids appear to develop and mature at a slower pace, with neurogenesis commencing later (<span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Otani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Otani, T. ∙ Marchetto, M.C. ∙ Gage, F.H. ...</div><div><strong>2D and 3D Stem Cell Models of Primate Cortical Development Identify Species-Specific Differences in Progenitor Behavior Contributing to Brain Size</strong></div><div><em>Cell Stem Cell.</em> 2016; <strong>18</strong>:467-480</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2016.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2016.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84962090418" target="_blank" aria-label="Open Scopus in new tab">Scopus (238)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27049876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.03.003&amp;pmid=27049876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), RG cells displaying a longer mitotic phase (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Mora-Bermúdez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Mora-Bermúdez, F. ∙ Badsha, F. ∙ Kanton, S. ...</div><div><strong>Differences and similarities between human and chimpanzee neural progenitors during cerebral cortex development</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e18683</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.18683" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84995486924" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27669147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.18683&amp;pmid=27669147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and neurons showing a decreased expression of genes related to neuronal maturation (<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Kanton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, genetic comparisons have identified a number of human specific loci capable of regulating RG expansion and neurogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Fiddes et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Fiddes, I.T. ∙ Lodewijk, G.A. ∙ Mooring, M. ...</div><div><strong>Human-Specific NOTCH2NL Genes Affect Notch Signaling and Cortical Neurogenesis</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:1356-1369.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.051&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.051&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047424138" target="_blank" aria-label="Open Scopus in new tab">Scopus (349)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29856954/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.03.051&amp;pmid=29856954" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Suzuki et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Suzuki, I.K. ∙ Gacquer, D. ∙ Heurck, R. ...</div><div><strong>Human-Specific NOTCH2NL Genes Expand Cortical Neurogenesis through Delta/Notch Regulation</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:1370-1384.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.067&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.067&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047337419" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29856955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.03.067&amp;pmid=29856955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Boyd et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Boyd, J.L. ∙ Skove, S.L. ∙ Rouanet, J.P. ...</div><div><strong>Human-chimpanzee differences in a FZD8 enhancer alter cell-cycle dynamics in the developing neocortex</strong></div><div><em>Curr. Biol.</em> 2015; <strong>25</strong>:772-779</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84926420296" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25702574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2015.01.041&amp;pmid=25702574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although these studies point to important differences in terms of RG behaviors, all these comparisons have been performed at stages where neurogenesis is already underway. Given the neuroanatomical comparisons that point to an early expansion and a universal increase in neurons, it would be highly informative to examine early human and ape progenitors at a stage preceding neurogenesis. Furthermore, extending comparative analysis to include other great apes beyond chimpanzee would help identify true human-specific cellular adaptations (rather than chimpanzee-specific differences) involved in brain size determination.</div><div id="p0050" role="paragraph">Here, we examined developing human, chimpanzee, and gorilla brain organoids. We found that during early NE expansion, the NE to RG transition in human and ape is a gradual process, taking place over the course of several days, unlike the rapid differentiation of NE to RG cells described in mouse, and it involves an intermediate cell morphology state we named transitioning NE (tNE) cells. During this transition, cell shape changes occur before the change in cell identity and the onset of neurogenesis. Furthermore, human NE cells are delayed relative to ape in making this transition and proliferate more. To identify the molecular mechanism underlying these differences, we performed time-resolved sequencing analysis and identified differential expression dynamics of the zinc-finger transcription factor <i>ZEB2</i>. Through <i>ZEB2</i> loss- and gain-of-function experiments, we show that this gene is a driver of the NE to RG cell transition. Finally, manipulation of <i>ZEB2</i> expression and its downstream signaling is able to force a human-like phenotype in the ape context and vice versa. Thus, our data suggest that in humans, a delayed onset of <i>ZEB2</i> expression extends the NE stage, contributing to human-specific neocortical expansion.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Human telencephalic organoids show differences in tissue architecture prior to neurogenesis</h3><div id="p0055" role="paragraph">To investigate evolutionary differences in brain development between apes, we generated cerebral organoids from human, gorilla, and chimpanzee cell lines, including a new gorilla cell line (<a id="crosref0295" href="#figs1">Figure&nbsp;S1</a>A). We optimized the protocol so that an identical method could be used for all species (<a id="crosref0300" href="#fig1">Figure&nbsp;1</a>A) resulting in highly comparable telencephalic tissues (<a id="crosref0305" href="#figs1">Figure&nbsp;S1</a>B). This was vital to be able to attribute any differences to species-specific divergence and not protocol variations. We did, however, notice that prior to neural induction, chimpanzee embryoid bodies (EBs) progressed faster than human and gorilla EBs, potentially reflecting the shorter gestational period of chimpanzees (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Ardito, 1976</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Ardito, G.</div><div><strong>Check-list of the data on the gestation length of primates</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 1976; <strong>5</strong>:213-222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0047-2484(76)90023-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0017253494" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0047-2484%2876%2990023-3" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the difference in timing may also be a result of cell line differences and further study is needed. Nonetheless, in order to obtain comparable tissues of the proper identity (<a id="crosref0315" href="#figs1">Figure&nbsp;S1</a>C), timing was optimized and chimpanzee organoids were transferred to neural induction 2&nbsp;days earlier (<a id="crosref0320" href="#fig1">Figure&nbsp;1</a>A). All steps subsequent to germ layer determination were identical across the species.</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/cef042d7-7ff0-40b7-9036-e61eed0ca18e/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.02.050/asset/10a924e9-9944-4197-9975-506afa504d7b/main.assets/gr1.jpg" height="803" width="649" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/cef042d7-7ff0-40b7-9036-e61eed0ca18e/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Human telencephalic organoids are larger with extended apical lumens</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Schematic of the timeline for generating brain organoids from human, gorilla, and chimpanzee stem cells. Colors represent changes in media and stages of the protocol.&nbsp;+MG represents Matrigel embedding. Note chimpanzee organoids have an EB stage that is 2&nbsp;days shorter than human and gorilla.</div><div id="fspara0020" role="paragraph">(B) Bright field images of human (H9, top panels), gorilla (G1, middle panels), and chimpanzee (Chmp, bottom panels) organoids at days 3, 5, and 10. Black arrows indicate neuroepithelial buds, which appear more elongated in the human beginning at day 5. Scale bar, 200&nbsp;μm.</div><div id="fspara0025" role="paragraph">(C) Quantification of bright field images at day 10 show human (H9) neural tissue is enlarged relative to gorilla (G1) and chimpanzee (Chmp). Left: area occupied by individual organoids. Right: visible perimeter of individual neuroepithelial buds. Mean organoid area: H9&nbsp;= 168,327&nbsp;μm<sup>2</sup>; G1&nbsp;= 84,876&nbsp;μm<sup>2</sup>; Chmp&nbsp;= 81,086&nbsp;μm<sup>2</sup>. Mean neuroepithelial bud perimeter: H9&nbsp;= 322&nbsp;μm; G1&nbsp;= 237&nbsp;μm; Chmp&nbsp;= 204&nbsp;μm. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Kruskal-Wallis and post hoc Dunn’s multiple comparisons test, n (H9)&nbsp;= 22 organoids and 144 neuroepithelial buds from 6 independent batches, n (G1)&nbsp;= 23 organoids and 148 neuroepithelial buds from 5 independent batches, n (Chmp)&nbsp;= 13 organoids and 107 neuroepithelial buds from 3 independent batches, error bars are SD.</div><div id="fspara0030" role="paragraph">(D and E) Representative immunofluorescence images of the center of whole mount human (H9), gorilla (G1), and chimpanzee (Chmp) organoids with staining for apical marker ZO1 and neural progenitor marker SOX2 at day 3 (D) and day 5 (E). Note the appearance of less rounded ZO1 positive apical lumens (arrowheads) in human organoids relative to nonhuman ape organoids at day 5. DAPI is in blue. Red background signal outside the organoid comes from nonspecific uneven staining of surrounding Matrigel. Scale bar, 100&nbsp;μm.</div><div id="fspara0035" role="paragraph">(F) 3D MATLAB reconstructions of apical lumens of day 5 organoids showing a representative example used for quantifications in (G). Values on the axes are in μm. Luminal surface area of reconstructed examples: human (H9)&nbsp;= 63,394&nbsp;μm<sup>2</sup>; gorilla (G1)&nbsp;= 15,146&nbsp;μm<sup>2</sup>; chimpanzee (Chmp)&nbsp;= 19,730&nbsp;μm<sup>2</sup>.</div><div id="fspara0040" role="paragraph">(G) Quantification of the surface area of the largest apical lumen per day 5 organoid reveals significantly expanded luminal surface areas in human versus nonhuman apes. Mean luminal surface area: human (H9)&nbsp;= 51,243&nbsp;μm<sup>2</sup>; gorilla (G1)&nbsp;= 15,437&nbsp;μm<sup>2</sup>; chimpanzee (Chmp)&nbsp;= 19,632&nbsp;μm<sup>2</sup>. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;= 0.0012, <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, one-way ANOVA and post hoc Tukey’s multiple comparisons test, n (H9 and G1)&nbsp;= 11 organoids from 5 independent batches, n (Chmp)&nbsp;= 8 organoids from 2 independent batches, error bars are SD.</div><div id="fspara0045" role="paragraph">See also <a id="crosref1500" href="#figs1">Figure&nbsp;S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/a3bbc4fa-6fb6-4032-80ac-ce7543fa47cf/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.02.050/asset/953c9099-e07f-4aed-b9a4-5af71378f4c0/main.assets/figs1.jpg" height="615" width="699" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/a3bbc4fa-6fb6-4032-80ac-ce7543fa47cf/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Human and ape stem cells and organoids are highly comparable in terms of identity and morphology, related to <a id="crosref1585" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0375" role="paragraph">A. RT-PCR shows expression of pluripotency markers (<i>OCT3/4</i>, <i>NANOG</i>, <i>SOX2</i>) and <i>GAPDH</i> loading control in IMR-90, G1 and G2 cell lines. –ve is the water negative control. B. 5-week organoids stained for neural progenitor marker SOX2 (red), dorsal telencephalic/intermediate progenitor marker TBR2 (gray), neuronal markers TUJ1 (human) and HuCD (gorilla, chimpanzee) in yellow, and DAPI (blue) showing human (H9) derived organoids become larger in overall size than gorilla (G1) and chimpanzee (Chmp) organoids. Scale bar: 1mm. C. Immunofluorescent staining of representative 5-week ape organoids for neural progenitor marker SOX2, intermediate progenitor marker TBR2 and DAPI show cortical lobules of similar thickness and relative proportions of cell types. Scale bar: 100&nbsp;μm. D. Quantification of the area occupied by individual organoids (left) and the perimeter of visible neuroepithelial buds (right) from brightfield images of day 10 organoids generated from all cell lines tested: human (H9, IMR-90), chimpanzee (Chmp), and gorilla (G1, G2), revealing human organoids grow consistently larger than nonhuman ape organoids. Mean organoid area: IMR-90&nbsp;= 175,708&nbsp;μm<sup>2</sup>, G2&nbsp;= 106,589&nbsp;μm<sup>2</sup>; H9, G1, Chmp as reported in <a id="crosref1590" href="#fig1">Figure&nbsp;1</a>C. Mean neuroepithelial bud perimeter: IMR-90&nbsp;= 307&nbsp;μm, G2&nbsp;= 211&nbsp;μm; H9, G1, Chmp as reported in <a id="crosref1595" href="#fig1">Figure&nbsp;1</a>C. <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Kruskal-Wallis and post hoc Dunn’s multiple comparisons test, n (IMR-90)&nbsp;= 15 organoids and 105 neuroepithelial buds from 5 independent batches, n (G2)&nbsp;= 15 organoids and 94 neuroepithelial buds from 5 independent batches, n (H9, G1 and Chmp)&nbsp;= reported in <a id="crosref1600" href="#fig1">Figure&nbsp;1</a>C. E. Representative brightfield images of pluripotent stem cells and organoids taken throughout the differentiation protocol. Note the more rounded neuroepithelium observed in day 5 organoids generated from nonhuman ape (Chmp, G1 and G2) cells versus day 5 organoids generated from human (H9, IMR-90) cells. Scale bar: 1&nbsp;mm. F. Representative immunofluorescence images of the center of whole mount human (IMR-90) and gorilla (G2) day 5 organoids with staining for ZO1 and SOX2 showing polarized neural progenitor cells organized around rounded (gorilla) and more convoluted (human) ZO1 positive apical lumens. Scale bar: 100&nbsp;μm. G. Quantification of the surface area of the largest apical lumen per day 5 organoid showing calculations for organoids derived from all cell lines tested. The data reveals the same trend of more expanded buds in human versus non-human ape. Mean luminal surface area: IMR-90&nbsp;= 78,463&nbsp;μm<sup>2</sup>; G2&nbsp;= 27,426&nbsp;μm<sup>2</sup>; H9, G1, Chmp as reported in <a id="crosref1605" href="#fig1">Figure&nbsp;1</a>F.</div></div></div></div></figcaption></figure></div><div id="p0060" role="paragraph">Human organoids were consistently larger than gorilla and&nbsp;chimpanzee organoids at a stage when telencephalic neuroepithelial buds had formed and were undergoing expansion (<a id="crosref0325" href="#fig1">Figures 1</a>B, 1C, and <a id="crosref0330" href="#figs1">S1</a>D). We examined several time points during neuroepithelial expansion (<a id="crosref0335" href="#figs1">Figure&nbsp;S1</a>E) revealing similar size (<a id="crosref0340" href="#fig1">Figure&nbsp;1</a>B) and tissue architecture (<a id="crosref0345" href="#fig1">Figure&nbsp;1</a>D) across species at 3&nbsp;days after neural induction. At day 5, however, gorilla and chimpanzee organoids presented more rounded circular neuroepithelial buds, while human organoids showed more elongated neuroepithelial buds (<a id="crosref0350" href="#fig1">Figure&nbsp;1</a>B). Staining for ZO1, a tight junction protein that marks the apical surface of polarized epithelia (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Ando-Akatsuka et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Ando-Akatsuka, Y. ∙ Yonemura, S. ∙ Itoh, M. ...</div><div><strong>Differential behavior of E-cadherin and occludin in their colocalization with ZO-1 during the establishment of epithelial cell polarity</strong></div><div><em>J.&nbsp;Cell. Physiol.</em> 1999; <strong>179</strong>:115-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-4652(199905)179:2%3C115::AID-JCP1%3E3.0.CO;2-T" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0032941962" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10199550/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-4652%28199905%29179%3A2%253C115%3A%3AAID-JCP1%253E3.0.CO%3B2-T&amp;pmid=10199550" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Medelnik et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Medelnik, J.P. ∙ Roensch, K. ∙ Okawa, S. ...</div><div><strong>Signaling-Dependent Control of Apical Membrane Size and Self-Renewal in Rosette-Stage Human Neuroepithelial Stem Cells</strong></div><div><em>Stem Cell Reports.</em> 2018; <strong>10</strong>:1751-1765</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85046875468" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.04.018&amp;pmid=29779899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), revealed smaller, more rounded apical lumens in nonhuman ape organoids (<a id="crosref0365" href="#fig1">Figures 1</a>E and <a id="crosref0370" href="#figs1">S1</a>F). Using a custom image analysis pipeline, apical lumens were segmented and reconstructed in 3D, revealing species-specific differences in luminal shape and size (<a id="crosref0375" href="#fig1">Figure&nbsp;1</a>F). Quantification of the largest lumen per organoid revealed that human organoids were capable of forming larger lumens than those of the other apes (<a id="crosref0380" href="#fig1">Figures 1</a>G and <a id="crosref0385" href="#figs1">S1</a>G).</div><div id="p0065" role="paragraph">The most likely explanation for expanded lumens would be a delay in the switch from symmetrically expanding NE cells to neurogenic RG cells. We therefore tested whether at this stage ape organoids had already switched to generating neurons by staining for neurogenic TBR2<sup>+</sup> intermediate progenitors and DCX<sup>+</sup> newborn neurons. In human and gorilla organoids, neither cell type was present at day 5, only becoming clearly evident between days 10 and 15 (<a id="crosref0390" href="#figs2">Figure&nbsp;S2</a>A). This suggests that a faster neurogenic switch in ape compared to human does not explain the difference in size of cortical tissues.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/11d2409a-12ce-4dc2-b5a8-7ecc45c8dde7/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.02.050/asset/d6fd4987-f4bf-40ef-a033-ee420d61d0b8/main.assets/figs2.jpg" height="650" width="753" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/11d2409a-12ce-4dc2-b5a8-7ecc45c8dde7/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Ape NE cells undergo cell shape transition before the onset of neurogenesis, related to <a id="crosref1610" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0385" role="paragraph">A. Representative immunofluorescence images through the center of whole mount organoids derived from human (H9) and gorilla (G1) at days 5, 10 and 15 post-neural induction with staining for intermediate progenitor cells (TBR2), newborn neurons (DCX), ZO1 and DAPI, showing that neurogenesis has not started by day 5, is commencing at day 10 and is underway by day 15. Scale bar: 100&nbsp;μm. B. Representative immunofluorescence images through whole mount organoids showing the morphology of neural progenitor cells revealed by sparse labeling with viral GFP with staining for ZO1, SOX2 and DAPI for human (H9) and gorilla (G1) organoids at days 8 and 10. Note in both species the thin, elongated tNE cell morphologies on day 8 and RG cell morphologies on day 10 with stereotypical elongated narrowed apical and basal processes. Scale bar: 20&nbsp;μm.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>The NE to RG switch involves a transitioning NE cell morphotype that is delayed in human</h3><div id="p0070" role="paragraph">Another possible explanation for the difference in tissue morphology is a difference in cell morphology. Sparse viral labeling with GFP revealed progenitors with a wide, columnar shape characteristic of NE cells at day 3 (<a id="crosref0395" href="#fig2">Figure&nbsp;2</a>A). At day 5, however, when the difference in tissue architecture first appears, human NE cells were still wide and columnar in shape, whereas chimpanzee and gorilla NE cells exhibited more constricted apical processes (<a id="crosref0400" href="#fig2">Figure&nbsp;2</a>A). To quantify these differences, we measured the apical surface area of individual progenitor cells using ZO1 staining to delineate apical edges at the lumen (<a id="crosref0405" href="#fig2">Figure&nbsp;2</a>B). At day 5, the apical surface of nonhuman ape NE cells was significantly more constricted compared to human (<a id="crosref0410" href="#fig2">Figure&nbsp;2</a>C). These findings point to species-specific differences in cell shape and reveal an apically constricted transition state, herein referred to as transitioning NE (tNE), that occurs prior to the change in cellular identity and neurogenesis characteristic of RG cells.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/e9312a0c-1df2-4d54-a89e-abfcf8354f95/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.02.050/asset/81daa671-c338-44af-a164-5376bf94778e/main.assets/gr2.jpg" height="665" width="752" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/e9312a0c-1df2-4d54-a89e-abfcf8354f95/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Human NE cells exhibit species-specific differences in cell shape</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0055" role="paragraph">(A) Representative immunofluorescence images through whole mount organoids showing the morphology of neural progenitor cells (SOX2<sup>+</sup>), polarized around apical (ZO1<sup>+</sup>) lumens, revealed by sparse labeling with viral GFP in human (H9, IMR-90), gorilla (G1, G2), and chimpanzee (Chmp) organoids. Day 3 cells (top panels) are columnar and exhibit typical NE morphology. Day 5 (bottom panels) human cells still appear columnar, whereas gorilla and chimpanzee cells show a thinning of apical processes (arrows). DAPI is in blue. Red background signal outside the organoid comes from nonspecific uneven staining of surrounding Matrigel. Scale bar, 20&nbsp;μm.</div><div id="fspara0060" role="paragraph">(B) Immunofluorescent staining for ZO1 on the surface of apical lumens showing the apical surface areas of individual progenitor cells at day 5 in human (H9, IMR-90), gorilla (G1, G2), and chimpanzee (Chmp) organoids. Perimeters of some individual progenitor cells are delineated in white. Scale bar, 10&nbsp;μm.</div><div id="fspara0065" role="paragraph">(C) Quantification of the apical surface area of individual neural progenitor cells of day 5 organoids show significantly smaller apical surface sizes of gorilla (G1, G2) and chimpanzee (Chmp) progenitor cells compared to human (H9, IMR-90). Measurements were performed on delineated ZO1 cell perimeters as shown in (B). Mean apical surface area/cell: H9&nbsp;= 9.39&nbsp;μm<sup>2</sup>, IMR-90&nbsp;= 8.53&nbsp;μm<sup>2</sup>, G1&nbsp;= 4.48&nbsp;μm<sup>2</sup>, G2&nbsp;= 4.54&nbsp;μm<sup>2</sup>, and Chmp&nbsp;= 3.55&nbsp;μm<sup>2</sup>. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Kruskal-Wallis and post hoc Dunn’s multiple comparisons test, n (H9)&nbsp;= 341 cells from 8 organoids from 2 independent batches, n (IMR-90)&nbsp;= 121 cells from 4 organoids of 1 batch, n (G1)&nbsp;= 321 cells from 9 organoids from 2 independent batches, n (G2)&nbsp;= 172 cells from 5 organoids of 1 batch, and n (Chmp)&nbsp;= 277 cells from 6 organoids and 2 independent batches. Boxplots show median with whiskers representing min-max values; data points on the boxplots represent individual cells.</div><div id="fspara0070" role="paragraph">(D) Representative immunofluorescence images through whole mount human (H9) and gorilla (G1) organoids with superimposed individual segmented cells (white) showing the 3D morphology of individual GFP<sup>+</sup> progenitors. Note the thinning of apical processes observed in gorilla at day 5 that becomes pronounced in both species by day 10. Scale bar, 10&nbsp;μm.</div><div id="fspara0075" role="paragraph">(E) Quantification of the volume normalized to surface area of the apical processes of human (H9) and gorilla (G1) neural progenitor cells on day 3 (left), day 5 (middle), and day 10 (right), showing significantly reduced apical volumes in gorilla cells relative to human at day 5. The apical processes of segmented cells directly below the cell body were used for quantification. Mean apical process volume:surface area ratio: human day 3&nbsp;= 1.21, gorilla day 3&nbsp;= 1.31, human day 5&nbsp;= 1.23, gorilla day 5&nbsp;= 0.76, human day 10&nbsp;= 0.66, and gorilla day 10&nbsp;= 0.65. Mann-Whitney U, <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, two-tailed, n (day 3 human)&nbsp;= 10 cells, n (day 3 gorilla)&nbsp;= 8 cells, n (day 5 human)&nbsp;= 14 cells, n (day 5 gorilla)&nbsp;= 16 cells, n (day 10 human)&nbsp;= 8 cells, and n (day 10 gorilla)&nbsp;= 8 cells. Error bars are SD.</div><div id="fspara0080" role="paragraph">See also <a id="crosref1505" href="#figs2">Figure&nbsp;S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">To further quantify cell shape, we turned to higher resolution spinning-disk confocal imaging with segmentation of individual cells. Because our findings were consistent within species and across nonhuman ape cell lines, we focused on a comparison of organoids from one representative human and one representative ape cell line. Although NE cells of the two species exhibited similar morphologies at day 3, gorilla NE cells at day 5 exhibited decreased apical process volume compared with human as a result of their highly constricted morphology (<a id="crosref0415" href="#fig2">Figures 2</a>D and 2E). Human cells instead exhibited this more elongated, narrow shape at day 8 (<a id="crosref0420" href="#figs2">Figure&nbsp;S2</a>B), whereas by day 10, both human and gorilla NE cells exhibited a similarly elongated and constricted morphology typical of RG cells (<a id="crosref0425" href="#fig2">Figures 2</a>D, 2E, and <a id="crosref0430" href="#figs2">S2</a>B).</div><div id="p0080" role="paragraph">We next performed a temporal assessment of progenitor cell apical surface area as labeled by ZO1. At day 3, the average apical surface of cells was large and indistinguishable between human and gorilla. By day 10, both human and gorilla exhibited complete transition with apical surface area decreasing by 7-fold (<a id="crosref0435" href="#fig3">Figures 3</a>A and 3B). However, the transition was delayed in human, where apical surface area at day 8 more closely matched that of gorilla at day 5. These findings reveal a later switch to apically constricted tNE morphologies in human compared with nonhuman ape organoids.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/648d2f5a-8625-4dc2-b486-44f6dcc265d2/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.02.050/asset/14600da9-32af-4d18-848b-071ed811ea75/main.assets/gr3.jpg" height="942" width="660" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/648d2f5a-8625-4dc2-b486-44f6dcc265d2/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Delayed human NE transition is associated with a shorter cell cycle</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0090" role="paragraph">(A) Immunofluorescent staining for ZO1 on the surface of apical lumens of human (H9) and gorilla (G1) organoids revealing apical surface areas of individual neural progenitor cells at days 3, 5, 8, and 10. Perimeters of some individual cells of day 5 organoids are delineated in white. Scale bar, 10&nbsp;μm.</div><div id="fspara0095" role="paragraph">(B) Quantification of the surface area of individual human (H9) and gorilla (G1) NPCs between day 3 and 10 showing a gradual reduction in apical surface area over time in both species by 7-fold. Note gorilla cells are more constricted than human during the transitioning period from days 5 to 8. Measurements were performed on delineated ZO1 cell perimeters as demonstrated in (A). Mean apical surface area/cell: human day 3&nbsp;= 13.82&nbsp;μm<sup>2</sup>, gorilla day 3&nbsp;= 14.80&nbsp;μm<sup>2</sup>, human day 8&nbsp;= 3.86&nbsp;μm<sup>2</sup>, gorilla day 8&nbsp;= 2.46&nbsp;μm<sup>2</sup>, human day 10&nbsp;= 1.92&nbsp;μm<sup>2</sup>, gorilla day 10&nbsp;= 2.13&nbsp;μm<sup>2</sup>, and human and gorilla day 5 are reported in <a id="crosref1510" href="#fig2">Figure&nbsp;2</a>C. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, per time point Mann-Whitney U, two-tailed, n (day 3 human)&nbsp;= 164 cells from 8 organoids and 2 batches, n (day 3 gorilla)&nbsp;= 176 cells from 8 organoids and 2 batches, n (day 8 human)&nbsp;= 171 cells from 4 organoids of 1 batch, n (day 8 gorilla)&nbsp;= 55 cells from 2 organoids of 1 batch, n (day 10 human)&nbsp;= 68 cells from 3 organoids of 1 batch, n (day 10 gorilla)&nbsp;= 74 cells from 4 organoids of 1 batch, and n (day 5)&nbsp;= reported in <a id="crosref1515" href="#fig2">Figure&nbsp;2</a>C.</div><div id="fspara0100" role="paragraph">(C) Representative immunofluorescence images showing the position of nuclei (DAPI, blue) of neural progenitor cells relative to the apical surface (ZO1, red) in neuroepithelial buds of day 5 human (H9) and gorilla (G1) organoids. Dashed lines in white and yellow represent apical and basal surfaces of the neuroepithelial bud respectively. Some nuclear distances from the apical surface are shown as dashed lines in cyan, and as quantified in (D) and (E). Note the more basal distribution of nuclei in gorilla tissue. Scale bar, 10&nbsp;μm.</div><div id="fspara0105" role="paragraph">(D and E) Quantification of the position of nuclei of human (H9) and gorilla (G1) neural progenitor cells relative to the apical surface as a percentage of apicobasal neuroepithelial (NE) thickness. Measurements were performed on images as shown in (C)</div><div id="fspara0110" role="paragraph">(D) Quantification of positions of individual cells showing gorilla progenitors located more basally.</div><div id="fspara0115" role="paragraph">(E) Mean nuclear position per neuroepithelial bud showing gorilla cells located significantly more basal. Mean nuclear position: human&nbsp;= 59.34%, gorilla&nbsp;= 67.14%. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Student’s t test, unpaired, two-tailed, n (human)&nbsp;= 335 nuclei from 9 organoids and 3 independent batches, and n (gorilla)&nbsp;= 307 nuclei from 9 organoids and 3 independent batches. Error bars are SD.</div><div id="fspara0120" role="paragraph">(F) Still frames of live imaging of neural progenitor cells sparsely labeled with GFP (grayscale) in human (H9) (<a id="crosref1520" href="#mmc4">Video S2</a>) and gorilla (G1) (<a id="crosref1525" href="#mmc5">Video S3</a>) organoids covering an entire cell cycle between day 3 and 4.5. Note the presence of neural progenitor cells with columnar NE morphology in both species prior to the first division (00:00, reference time point) and loss of basal process (white arrowheads) during cell mitosis (blue arrowheads). Note daughter cells (yellow arrowheads) show thicker apical processes (magenta arrowheads) in human relative to the more constricted morphology of gorilla apical processes. Time shown in hours:minutes.</div><div id="fspara0125" role="paragraph">(G) Quantification of cell cycle duration of human (H9) and gorilla (G1) neural progenitor cells imaged between days 3 and 5 showing significantly longer cell cycles in gorilla. Mean cell cycle length: human&nbsp;= 18.83 h, gorilla&nbsp;= 22.10. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Mann-Whitney U, two-tailed, n (human)&nbsp;= 30 cells from 5 independent batches, n (gorilla)&nbsp;= 29 cells from 5 independent batches. Error bars are SD.</div><div id="fspara0130" role="paragraph">(H) Growth curve modeling of the predicted effect that differences in cell cycle length, as measured in (G), would have on human and gorilla progenitor numbers (bold lines) and on neuron numbers (dashed lines) with a 1.9-fold expected increase in human for both.</div><div id="fspara0135" role="paragraph">(I) Schematic summarizing the morphological changes in neural progenitor cells. Progenitor cells of apes undergo a gradual transition from NE to tNE to neurogenic RG cells. Nonhuman ape cells show tNE morphologies (blue background) earlier than human cells, which show shorter cell cycles leading to increased neuron numbers.</div><div id="fspara0140" role="paragraph">See also <a id="crosref1530" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref1535" href="#mmc2">Videos S1</a>, <a id="crosref1540" href="#mmc6">S4</a>, and <a id="crosref1545" href="#mmc7">S5</a>.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Cell shape is closely linked to cell cycle through interkinetic nuclear migration, a behavior characteristic of NE and RG cells in which the nucleus moves along the apicobasal axis in time with the cell cycle. We investigated nuclear positioning and observed more frequent apical nuclei in human NE at day 5 (<a id="crosref0440" href="#fig3">Figures 3</a>C–3E), suggesting a difference in interkinetic nuclear migration and pointing to potential cell cycle differences. To explore this possibility, we performed live imaging of whole organoids over the course of several days, using SiMView adaptive light sheet microscopy (<a id="crosref0445" href="#mmc3">Video S1</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">McDole et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>McDole, K. ∙ Guignard, L. ∙ Amat, F. ...</div><div><strong>In Toto Imaging and Reconstruction of Post-Implantation Mouse Development at the Single-Cell Level</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:859-876.e33</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85055072359" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30318151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.09.031&amp;pmid=30318151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Subramanian et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Subramanian, L. ∙ Bershteyn, M. ∙ Paredes, M.F. ...</div><div><strong>Dynamic behaviour of human neuroepithelial cells in the developing forebrain</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14167" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85011672129" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28139695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14167&amp;pmid=28139695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observed a transient loss of the basal process during NE cell division in both human and gorilla. Following the division, the apical processes of human NE daughter cells appeared thicker than those of gorilla (<a id="crosref0460" href="#fig3">Figures 3</a>F and <a id="crosref0465" href="#figs3">S3</a>; <a id="crosref0470" href="#mmc4">Videos S2</a>, <a id="crosref0475" href="#mmc5">S3</a>, <a id="crosref0480" href="#mmc6">S4</a>, and <a id="crosref0485" href="#mmc7">S5</a>), consistent with our findings above. We also measured cell cycle lengths, revealing a shorter cell cycle in human NE cells (<a id="crosref0490" href="#fig3">Figure&nbsp;3</a>G). Mathematical growth curve modeling of the effect of this 3-h cell cycle difference predicted a progenitor and neuron number increase in human of 1.9-fold (<a id="crosref0495" href="#fig3">Figure&nbsp;3</a>H). These findings together suggest changes in cell shape are coupled to the cell cycle, and a delay in this transition in human leads to a larger founder progenitor pool and therefore an expected increase in neuron number (<a id="crosref0500" href="#fig3">Figure&nbsp;3</a>I).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/295e8b06-95a0-4504-a5e8-beafcf8bb17c/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.02.050/asset/f35c64f2-0c13-47ef-9e62-527144f4f0f6/main.assets/figs3.jpg" height="353" width="752" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/295e8b06-95a0-4504-a5e8-beafcf8bb17c/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Live imaging of human and gorilla cerebral organoids, related to <a id="crosref1615" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0395" role="paragraph">Representative still frames of live imaging of neural progenitor cells sparsely labeled with GFP (greyscale) in human (H9) (<a id="crosref1620" href="#mmc6">Video S4</a>) and gorilla (G1) (<a id="crosref1625" href="#mmc7">Video S5</a>) organoids covering an entire cell cycle between day 4 and 5, showing a significantly longer cell cycle in gorilla, calculated as the period between cell divisions (blue arrowheads). Parent and daughter cells (yellow arrowheads) show more constricted apical processes (magenta arrowheads) in gorilla relative to the apical process of human progenitors which thickens into a NE morphology in the frames following the first division. Note the loss of the basal process (white arrowhead) during mitosis. Time shown in hours:minutes, 00:00 marks the first division and reference time point</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph"><div id="mmc3" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc3" data-type="mp4"><span id="video_stream_mmc3_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzYzRmNWY4MThiNTc1ZmFlM2UxN2UxZjY1MWM1Yzc3NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.nhbIgRdKhvvIwF3sv6IPfqH41XnKZ0ka3jScGBS5pBNlt805UAesIL3whkLFrYtilFBBqVEQSyKEkrZq_F0YLP4e9EsCEbUtZr02cvWk1pl-_N4W0RV-leLqkdlS_1wjtiKuZqmwYZsWv2z5HRW2Ac1i5L-D3qTkPl6b3a7MJAwSvp9K1bA8Vtx5N72zRXlaF7bePjm4xUfxLow-6pcNppFsfOjklauVOB0ijlBZLwNSBOaLi3qKS6RBaKpdFOuBMkahkOGoQruHJxynFah6X3o9XODIdd3LIo9Jszk5KpULRTUlrru_nq3IEr0vqX4EUZVqgYHucr90dN3PG1ceDw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzYzRmNWY4MThiNTc1ZmFlM2UxN2UxZjY1MWM1Yzc3NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.nhbIgRdKhvvIwF3sv6IPfqH41XnKZ0ka3jScGBS5pBNlt805UAesIL3whkLFrYtilFBBqVEQSyKEkrZq_F0YLP4e9EsCEbUtZr02cvWk1pl-_N4W0RV-leLqkdlS_1wjtiKuZqmwYZsWv2z5HRW2Ac1i5L-D3qTkPl6b3a7MJAwSvp9K1bA8Vtx5N72zRXlaF7bePjm4xUfxLow-6pcNppFsfOjklauVOB0ijlBZLwNSBOaLi3qKS6RBaKpdFOuBMkahkOGoQruHJxynFah6X3o9XODIdd3LIo9Jszk5KpULRTUlrru_nq3IEr0vqX4EUZVqgYHucr90dN3PG1ceDw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc3" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 39.9101%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIzYzRmNWY4MThiNTc1ZmFlM2UxN2UxZjY1MWM1Yzc3NyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.nhbIgRdKhvvIwF3sv6IPfqH41XnKZ0ka3jScGBS5pBNlt805UAesIL3whkLFrYtilFBBqVEQSyKEkrZq_F0YLP4e9EsCEbUtZr02cvWk1pl-_N4W0RV-leLqkdlS_1wjtiKuZqmwYZsWv2z5HRW2Ac1i5L-D3qTkPl6b3a7MJAwSvp9K1bA8Vtx5N72zRXlaF7bePjm4xUfxLow-6pcNppFsfOjklauVOB0ijlBZLwNSBOaLi3qKS6RBaKpdFOuBMkahkOGoQruHJxynFah6X3o9XODIdd3LIo9Jszk5KpULRTUlrru_nq3IEr0vqX4EUZVqgYHucr90dN3PG1ceDw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.02.050/attachment/b6623388-1750-40b3-bf9b-480a51c844d0/mmc3.mp4" download="mmc3.mp4" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Video (11.43 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Video S1. Side-by-side movie of SiMView imaged human and gorilla organoids, related to Figure&nbsp;3</div><div id="espara0010" role="paragraph">Live imaging using SiMView adaptive light sheet microscopy of progenitor cells sparsely labeled with GFP in a human cerebral organoid (left) and a gorilla cerebral organoid (right) beginning on day 3. Images were acquired every 5&nbsp;minutes. Note the appearance of labeled radially organized neuroepithelial cells beginning at approximately 6 hours of imaging and the progressive thinning of the apical processes, particularly in gorilla which becomes more apparently different at approximately 40 hours. Time shown in day:hours:minutes. Scale bar: 25&nbsp;μm.</div></div></div></div><div id="p0095" role="paragraph"><div id="mmc4" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc4" data-type="mp4"><span id="video_stream_mmc4_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlN2RjZWQ5MTY2OTY4NDNiNzJjMzcyM2E1MmE0NDkwYiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.REnr1Hk3nJEVa_X34Tt7zLs3syRpDFMJdtS79JAitAV46gefLY9GtH9rqthcET6s5FycfgEIO716rJFGhatVUEzPjG0vjZG4ZTJD2D6aMWt6SSCkyyD5fKm5oliz0qDN8t-_A6s92u_xDo-zYCHMntfmLDuJRKK5wKjgIvyYlC5txCTdofF2Gr7cEIFb6pXhyCWGAWwX7URnuM_MYVJEmgt3D4z6kRmC8SCqN3AYxXPL9B6jqorfaI2FXvZIJMCmpEaEF9XE09f_niYZUpxQrwuWwhLuXnAm2UsGowzmLsEfDBhh0TVV-oFGggetRRE916FjoXNfZ0ohRW8Ycd5u7g</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlN2RjZWQ5MTY2OTY4NDNiNzJjMzcyM2E1MmE0NDkwYiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.REnr1Hk3nJEVa_X34Tt7zLs3syRpDFMJdtS79JAitAV46gefLY9GtH9rqthcET6s5FycfgEIO716rJFGhatVUEzPjG0vjZG4ZTJD2D6aMWt6SSCkyyD5fKm5oliz0qDN8t-_A6s92u_xDo-zYCHMntfmLDuJRKK5wKjgIvyYlC5txCTdofF2Gr7cEIFb6pXhyCWGAWwX7URnuM_MYVJEmgt3D4z6kRmC8SCqN3AYxXPL9B6jqorfaI2FXvZIJMCmpEaEF9XE09f_niYZUpxQrwuWwhLuXnAm2UsGowzmLsEfDBhh0TVV-oFGggetRRE916FjoXNfZ0ohRW8Ycd5u7g" class="cloudflare-stream-player ready tracked" id="video_stream_mmc4" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJlN2RjZWQ5MTY2OTY4NDNiNzJjMzcyM2E1MmE0NDkwYiIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.REnr1Hk3nJEVa_X34Tt7zLs3syRpDFMJdtS79JAitAV46gefLY9GtH9rqthcET6s5FycfgEIO716rJFGhatVUEzPjG0vjZG4ZTJD2D6aMWt6SSCkyyD5fKm5oliz0qDN8t-_A6s92u_xDo-zYCHMntfmLDuJRKK5wKjgIvyYlC5txCTdofF2Gr7cEIFb6pXhyCWGAWwX7URnuM_MYVJEmgt3D4z6kRmC8SCqN3AYxXPL9B6jqorfaI2FXvZIJMCmpEaEF9XE09f_niYZUpxQrwuWwhLuXnAm2UsGowzmLsEfDBhh0TVV-oFGggetRRE916FjoXNfZ0ohRW8Ycd5u7g" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.02.050/attachment/7c09d3a1-cbeb-48d9-a394-427923919b26/mmc4.mp4" download="mmc4.mp4" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Video (452.98 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Video S2. Human cerebral organoid movie beginning on day 3, related to Figure&nbsp;3</div><div id="espara0015" role="paragraph">Live imaging using SiMView of progenitor cells sparsely labeled with GFP in a human cerebral organoid beginning on day 3. Images were acquired every 5&nbsp;minutes. Time shown in hours:minutes. Scale bar: 20&nbsp;μm.</div></div></div></div><div id="p0100" role="paragraph"><div id="mmc5" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc5" data-type="mp4"><span id="video_stream_mmc5_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI1ZWU2YTViZDBmMDhiNjNkMzc4Njg5ZDkxYzVlMTJkMyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.nPtDgWvOClHzMB3lvJtllBJIDyDkUgJrlGBCqs-NuKtLpnCM9ygBDG8Va2K4ZPQRhqYZILawyjS1Vf5VnGgXLYkfCVgFuDpjh40lwDNZ2ncpaYjPWM1RCGMuqtyh1OIs40Ystdeus0_fChJP9A8mqqwBYlSbSpkVP45NugteyqulHhi0MVeNSB8cjlCPXYQ4MqNfuQIp5cEABTBOe5wtv8ZmWlWP8C_D-KCljE15ppih4bw-ZQgGY7Y7n5R-sNcGKq7Oei46NYjYOJd6lmNZkAsYWPJ5oSFSjmZ88EAGyaOP5NAy9JIS7AhbpJ4lYwD_ZckOzZtlk0wLKSPmZNdQpQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI1ZWU2YTViZDBmMDhiNjNkMzc4Njg5ZDkxYzVlMTJkMyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.nPtDgWvOClHzMB3lvJtllBJIDyDkUgJrlGBCqs-NuKtLpnCM9ygBDG8Va2K4ZPQRhqYZILawyjS1Vf5VnGgXLYkfCVgFuDpjh40lwDNZ2ncpaYjPWM1RCGMuqtyh1OIs40Ystdeus0_fChJP9A8mqqwBYlSbSpkVP45NugteyqulHhi0MVeNSB8cjlCPXYQ4MqNfuQIp5cEABTBOe5wtv8ZmWlWP8C_D-KCljE15ppih4bw-ZQgGY7Y7n5R-sNcGKq7Oei46NYjYOJd6lmNZkAsYWPJ5oSFSjmZ88EAGyaOP5NAy9JIS7AhbpJ4lYwD_ZckOzZtlk0wLKSPmZNdQpQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc5" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI1ZWU2YTViZDBmMDhiNjNkMzc4Njg5ZDkxYzVlMTJkMyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.nPtDgWvOClHzMB3lvJtllBJIDyDkUgJrlGBCqs-NuKtLpnCM9ygBDG8Va2K4ZPQRhqYZILawyjS1Vf5VnGgXLYkfCVgFuDpjh40lwDNZ2ncpaYjPWM1RCGMuqtyh1OIs40Ystdeus0_fChJP9A8mqqwBYlSbSpkVP45NugteyqulHhi0MVeNSB8cjlCPXYQ4MqNfuQIp5cEABTBOe5wtv8ZmWlWP8C_D-KCljE15ppih4bw-ZQgGY7Y7n5R-sNcGKq7Oei46NYjYOJd6lmNZkAsYWPJ5oSFSjmZ88EAGyaOP5NAy9JIS7AhbpJ4lYwD_ZckOzZtlk0wLKSPmZNdQpQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.02.050/attachment/40b16440-36ae-4c9e-9374-7d3c1db9c397/mmc5.mp4" download="mmc5.mp4" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Video (385.31 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Video S3. Gorilla cerebral organoid movie beginning on day 3, related to Figure&nbsp;3</div><div id="espara0020" role="paragraph">Live imaging using SiMView of progenitor cells sparsely labeled with GFP in a gorilla cerebral organoid beginning on day 3. Images were acquired every 5&nbsp;minutes. Time shown in hours:minutes. Scale bar: 20&nbsp;μm.</div></div></div></div><div id="p0105" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI4ODAyMDBjYzgwNGI3ZTM0OTgwYWQxOGIyOTZlZWU4MSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.XUibO5VelUvZ0hqKCJtYklwT92slpfWs80mWsD2F8NJjmKA9hEwoILdJFc5axfhXmBOvI5vELnKiIR7sRvMS2LXhiqrzQLy8WaW0C8NP3_UH4TSuV2qhoRp0JuUAUWfl1Hq6Lra88VvSFBK5YJ2PD2VvcrhxU5XBBaPokIWW0edqh4gfb2fHrqj_gqOau7gKEWMyo_1idtJ51clSwGbgCJqAOQBvmqIoiYU8ql3Y8z4x5eFsscOFWPd3-AmzZZ8VaPTW0z-YeOwBBlo9aC8vD-lYxSn6wEQfhVIUygxD7aK0bx80A3Vay23lGS4EGDAUEmIK22ibhKJOJaG3wdamqw</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI4ODAyMDBjYzgwNGI3ZTM0OTgwYWQxOGIyOTZlZWU4MSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.XUibO5VelUvZ0hqKCJtYklwT92slpfWs80mWsD2F8NJjmKA9hEwoILdJFc5axfhXmBOvI5vELnKiIR7sRvMS2LXhiqrzQLy8WaW0C8NP3_UH4TSuV2qhoRp0JuUAUWfl1Hq6Lra88VvSFBK5YJ2PD2VvcrhxU5XBBaPokIWW0edqh4gfb2fHrqj_gqOau7gKEWMyo_1idtJ51clSwGbgCJqAOQBvmqIoiYU8ql3Y8z4x5eFsscOFWPd3-AmzZZ8VaPTW0z-YeOwBBlo9aC8vD-lYxSn6wEQfhVIUygxD7aK0bx80A3Vay23lGS4EGDAUEmIK22ibhKJOJaG3wdamqw" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiI4ODAyMDBjYzgwNGI3ZTM0OTgwYWQxOGIyOTZlZWU4MSIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.XUibO5VelUvZ0hqKCJtYklwT92slpfWs80mWsD2F8NJjmKA9hEwoILdJFc5axfhXmBOvI5vELnKiIR7sRvMS2LXhiqrzQLy8WaW0C8NP3_UH4TSuV2qhoRp0JuUAUWfl1Hq6Lra88VvSFBK5YJ2PD2VvcrhxU5XBBaPokIWW0edqh4gfb2fHrqj_gqOau7gKEWMyo_1idtJ51clSwGbgCJqAOQBvmqIoiYU8ql3Y8z4x5eFsscOFWPd3-AmzZZ8VaPTW0z-YeOwBBlo9aC8vD-lYxSn6wEQfhVIUygxD7aK0bx80A3Vay23lGS4EGDAUEmIK22ibhKJOJaG3wdamqw" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.02.050/attachment/9c25b646-ec13-4d3b-aadd-37650e2b9790/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (325.56 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S4. Human cerebral organoid movie beginning on day 4, related to Figure&nbsp;3</div><div id="espara0025" role="paragraph">Live imaging using SiMView of progenitor cells sparsely labeled with GFP in a human cerebral organoid beginning on day 4. Images were acquired every 5&nbsp;minutes. Time shown in hours:minutes. Scale bar: 20&nbsp;μm.</div></div></div></div><div id="p0110" role="paragraph"><div id="mmc7" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc7" data-type="mp4"><span id="video_stream_mmc7_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjZGU1ZmE3NzAxOTVmYzIxNWRmMTFlYzkzYTFkZGRlNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.ef1NX1UzEgutoAbNu5oFlD4LeGU0Wv6bgUMX1M7I3_uu8T_J-A7QDiyZjBuJnymVP9f_E0IUsEfdJuRJJSSmQ0wyoFJ3XwKn00xB8Bb-FWwtGtWl2jcm8snh3eGvt9Ame6cL7ESuXgPDLBwyZ-yro_BsCuzPjfIw4Ri7LllNdsyah5T3QNzaq_sAKuqRofln3_UtwEh5040gHlG_oh28_tnb1L1uGbzjgM9y5pnlyyTCgqHSMquEp5ASdXb8gbUFHxkKCn7ERwTWHelaFIHBhdFJQUu8NImYCHiBykJ3b06S6_iSbkqhlt0z6YBBat0Yo3vRMb2PnPx7_oPUNK-MnQ</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjZGU1ZmE3NzAxOTVmYzIxNWRmMTFlYzkzYTFkZGRlNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.ef1NX1UzEgutoAbNu5oFlD4LeGU0Wv6bgUMX1M7I3_uu8T_J-A7QDiyZjBuJnymVP9f_E0IUsEfdJuRJJSSmQ0wyoFJ3XwKn00xB8Bb-FWwtGtWl2jcm8snh3eGvt9Ame6cL7ESuXgPDLBwyZ-yro_BsCuzPjfIw4Ri7LllNdsyah5T3QNzaq_sAKuqRofln3_UtwEh5040gHlG_oh28_tnb1L1uGbzjgM9y5pnlyyTCgqHSMquEp5ASdXb8gbUFHxkKCn7ERwTWHelaFIHBhdFJQUu8NImYCHiBykJ3b06S6_iSbkqhlt0z6YBBat0Yo3vRMb2PnPx7_oPUNK-MnQ" class="cloudflare-stream-player ready tracked" id="video_stream_mmc7" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 100%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjZGU1ZmE3NzAxOTVmYzIxNWRmMTFlYzkzYTFkZGRlNCIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjExMTU2fQ.ef1NX1UzEgutoAbNu5oFlD4LeGU0Wv6bgUMX1M7I3_uu8T_J-A7QDiyZjBuJnymVP9f_E0IUsEfdJuRJJSSmQ0wyoFJ3XwKn00xB8Bb-FWwtGtWl2jcm8snh3eGvt9Ame6cL7ESuXgPDLBwyZ-yro_BsCuzPjfIw4Ri7LllNdsyah5T3QNzaq_sAKuqRofln3_UtwEh5040gHlG_oh28_tnb1L1uGbzjgM9y5pnlyyTCgqHSMquEp5ASdXb8gbUFHxkKCn7ERwTWHelaFIHBhdFJQUu8NImYCHiBykJ3b06S6_iSbkqhlt0z6YBBat0Yo3vRMb2PnPx7_oPUNK-MnQ" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2021.02.050/attachment/42d3dae2-1ae1-417e-901e-f93d677a3dd0/mmc7.mp4" download="mmc7.mp4" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Video (908.46 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Video S5. Gorilla cerebral organoid movie beginning on day 4, related to Figure&nbsp;3</div><div id="espara0030" role="paragraph">Live imaging using SiMView of progenitor cells sparsely labeled with GFP in a gorilla cerebral organoid beginning on day 4. Images were acquired every 5&nbsp;minutes. Time shown in hours:minutes. Scale bar: 20&nbsp;μm.</div></div></div></div></section><section id="sec-2-3"><h3>RNA sequencing analysis captures dynamics of gene expression across multiple early time points</h3><div id="p0115" role="paragraph">To identify what factors might be controlling these changes in cell shape, we examined the transcriptome profile of human and gorilla organoids. Although there are a number of published cerebral organoid single-cell RNA sequencing (RNA-seq) datasets, including from human and chimpanzee (<span class="dropBlock reference-citations"><a id="body-ref-sref48-3" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-3" aria-controls="sref48-3">Kanton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-3" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-3" aria-controls="sref65-3">Mora-Bermúdez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref65-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Mora-Bermúdez, F. ∙ Badsha, F. ∙ Kanton, S. ...</div><div><strong>Differences and similarities between human and chimpanzee neural progenitors during cerebral cortex development</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e18683</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.18683" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84995486924" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27669147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.18683&amp;pmid=27669147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Pollen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Pollen, A.A. ∙ Bhaduri, A. ∙ Andrews, M.G. ...</div><div><strong>Establishing Cerebral Organoids as Models of Human-Specific Brain Evolution</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:743-756.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2019.01.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2019.01.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85060531675" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30735633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.01.017&amp;pmid=30735633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), those collected to date do not capture early NE to RG transition stages with a high enough temporal resolution. Therefore, we decided to perform bulk RNA-seq of a large number of organoids across many early time points and replicates to identify high confidence differentially expressed genes. In total, we performed RNA-seq of 42 samples of ∼9,000 organoids, collected from day 0, corresponding to pre-neurulation tissue, up until day 25, corresponding to fully committed neurogenesis (<a id="crosref0520" href="#fig4">Figure&nbsp;4</a>A). To compare expression of genes shared between both species, transcripts per millions (TPMs) were calculated using a filtered list of annotated genes for which an orthologous gene was present in both species (<a id="crosref0525" href="#figs4">Figure&nbsp;S4</a>A). Principal component analysis (PCA) of the samples showed biological replicates grouping together, highlighting the robustness and reproducibility of the protocol (<a id="crosref0530" href="#figs4">Figure&nbsp;S4</a>B). It also showed samples separated by species along PC1, followed by grouping by time along PC2 and PC3. PC1 was likely driven by technical artifacts arising from read mapping and annotation differences coming from the reference genomes of the two species. Therefore, to normalize within each species and at the same time focus on dynamics rather than absolute levels, the expression of each gene was normalized by <i>Z</i>-scaling across time.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/14b72e5f-1fc7-472d-9ec8-217f6acc48ae/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.02.050/asset/1f529a2e-cedd-43e7-bfb3-6b62e6559670/main.assets/gr4.jpg" height="547" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/14b72e5f-1fc7-472d-9ec8-217f6acc48ae/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Ape organoids exhibit comparable developmental molecular trajectories</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0150" role="paragraph">(A) Schematic of the timeline for human and gorilla brain organoids with RNA-seq collection time points shown in red. 3 biological replicates of organoids derived from human (H9) and gorilla (G1) cells were collected at each of the 7 time points.</div><div id="fspara0155" role="paragraph">(B) PCA biplot of PC1 versus PC2 performed on <i>Z</i>-scaled log2-transformed TPMs of the 3,000 most variable genes. Samples are color-coded by time point and species. Note samples separating primarily by time point with a slight separation between samples of different species at days 5, 10, and 25, highlighted by ellipses (black ellipses for human and fuchsia for gorilla).</div><div id="fspara0160" role="paragraph">(C) Heatmap with hierarchical clustering based on shared expression pattern (<i>Z</i> scores of log2-transformed TPMs of the 3,000 most variable genes) between samples. The dendrogram shows samples clustering by time point.</div><div id="fspara0165" role="paragraph">(D) Pearson’s correlation map using <i>Z</i>-scaled log2-transformed TPMs of all genes. Darker blue depicts stronger positive correlation between samples and darker red a stronger negative correlation. Red boxes highlight the correlation between species at matched time points. Note lower correlation between species at days 5 and 10. Pearson’s correlation coefficient: r&nbsp;= 0.66 (day 0), 0.63 (day 2), 0.66 (day 3), 0.52 (day 5), 0.40 (day 10), 0.62 (day 15), and 0.66 (day 25).</div><div id="fspara0170" role="paragraph">(E) Temporal expression pattern (<i>Z</i>-scaled) of characteristic developmental markers show predicted expression dynamics. Non-neural ectoderm markers (<i>GATA3</i>, <i>CDH1</i>) are lost rapidly, followed by a gain in neural progenitor markers (<i>CDH2</i>, <i>VIM</i>), and a later increase in intermediate progenitor (<i>TBR2</i>) and early-born neuron (<i>TBR1</i>) markers. Shaded error bar is SD.</div><div id="fspara0175" role="paragraph">(F) GO term enrichment analysis on the top 300 genes driving species variance at days 5, 10, and 25. Shown are the 8 most significant (p&nbsp;&lt; 0.05) enrichments for GO categories molecular function (GO:MF), cellular compartment (GO:CC), and biological process (GO:BP).</div><div id="fspara0180" role="paragraph">See also <a id="crosref1550" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref1555" href="#mmc2">Data S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/0df149ab-8125-4849-9269-c88036453706/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.02.050/asset/a7b740bd-69e8-4713-8458-fd38a26be30e/main.assets/figs4.jpg" height="436" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/0df149ab-8125-4849-9269-c88036453706/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">RNA-seq data analysis pipeline and normalization, related to <a id="crosref1630" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0405" role="paragraph">A. Workflow summarizing the RNaseq analysis pipeline (see methods). B. PCA performed using log2-transformed TPMs. Samples are color-coded by time point and species. Graph of PC1 versus PC2 (left) shows biological replicates grouping together, samples separating by species along PC1 and separating by time point along PC2. Plotting PC2 versus PC3 (right) shows samples separating by time point and not species. C. Pearson’s correlation of all samples using z-scaled log2-transformed TPMs of all genes. Darker blue depicts stronger positive correlation between samples and darker red a stronger negative correlation. Data shows the strongest correlation between biological replicates within a species followed by between species time point-matched samples. Correlation between species is lowest at day 5 and 10. D. Temporal expression pattern (z-scaled) of genes related to synaptic formation and maturation (<i>GRIA2</i>, <i>GRIK2</i>, <i>SNAP25</i>, <i>SNAP91</i>). Shaded error bar is SD.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">PCA of normalized data showed samples of the same time point across species grouped together (<a id="crosref0535" href="#fig4">Figure&nbsp;4</a>B). Hierarchical clustering on mean <i>Z</i> scores (<a id="crosref0540" href="#fig4">Figure&nbsp;4</a>C) and Pearson’s correlation analysis (<a id="crosref0545" href="#fig4">Figures 4</a>D and <a id="crosref0550" href="#figs4">S4</a>C) also showed grouping by time rather than species. A panel of genes with characteristic developmental roles showed expected patterns of expression in both species (<a id="crosref0555" href="#fig4">Figure&nbsp;4</a>E). These data further confirm the proper identity and developmental trajectory of organoids from both species and demonstrate they are highly comparable.</div><div id="p0125" role="paragraph">Although organoids collected at the same time point appeared highly correlated across species, the PCA biplot revealed a degree of separation between species at days 5, 10, and 25 (<a id="crosref0560" href="#fig4">Figure&nbsp;4</a>B). We performed Gene Ontology (GO) term enrichment analysis on the top 300 genes driving the variance at these time points (<a id="crosref0565" href="#mmc2">Data S1</a>). GO terms at all time points were generally associated with nervous system development, with earlier day 5 time point terms primarily covering organ development and morphogenesis, and day 25 terms associated with neuronal compartments such as dendrites and synapses, as well as neurogenesis (<a id="crosref0570" href="#fig4">Figure&nbsp;4</a>F). We further examined some of the genes driving the variance at day 25 and saw a steeper rate of increase in genes associated with synaptic formation and maturation in gorilla (<a id="crosref0575" href="#figs4">Figure&nbsp;S4</a>D). This is consistent with previous studies (<span class="dropBlock reference-citations"><a id="body-ref-sref48-4" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-4" aria-controls="sref48-4">Kanton et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Liu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Liu, X. ∙ Somel, M. ∙ Tang, L. ...</div><div><strong>Extension of cortical synaptic development distinguishes humans from chimpanzees and macaques</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:611-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.127324.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84859541449" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22300767/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.127324.111&amp;pmid=22300767" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Marchetto et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Marchetto, M.C. ∙ Hrvoj-Mihic, B. ∙ Kerman, B.E. ...</div><div><strong>Species-specific maturation profiles of human, chimpanzee and bonobo neural cells</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e37527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.37527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85061147585" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30730291/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.37527&amp;pmid=30730291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) suggesting that human neurons mature at a slower rate.</div></section><section id="sec-2-4"><h3>Species-specific differences in gene expression and biological processes related to cell morphogenesis during NE to RG transition</h3><div id="p0130" role="paragraph">Pearson’s correlation coefficient was also lower at day 5 and day 10 than any of the other time points (<a id="crosref0595" href="#fig4">Figure&nbsp;4</a>D), coinciding with our observed differences in cell shape. We next focused on gene expression changes arising between day 3 and day 10, which would cover the time taken in both species for progenitors to transition from NE through tNE to RG morphologies. We analyzed abundantly expressed genes (TPM &gt;10) with reproducible expression patterns between biological replicates (squared difference &lt;6) and those with more dynamic expression patterns over the 3 time points (fold change &gt;1.5 between any two time points in at least one species). This resulted in a list of 2,905 genes that were then subjected to analysis by TCseq (<span class="dropBlock reference-citations"><a id="body-ref-sref107-1" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-1" aria-controls="sref107-1">Wu and Gu, 2020</a><div class="dropBlock__holder" data-db-target-of="sref107-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Wu, M. ∙ Gu, L.</div><div><strong>TCseq: Time course sequencing data analysis. R package version 1.12.0</strong></div><div>2020</div><div><a id="intref0305" href="https://www.bioconductor.org/packages/release/bioc/html/TCseq.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/TCseq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.WuL.GuTCseq%3A+Time+course+sequencing+data+analysis.+R+package+version+1.12.02020https%3A%2F%2Fwww.bioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FTCseq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to cluster genes with similar temporal patterns. Focusing on 3 time points meant that temporal patterns could be represented by and separated into 10 clusters with distinct shapes (<a id="crosref0605" href="#fig5">Figure&nbsp;5</a>A). During TCseq analysis, biological replicates were kept separate in order to further filter for genes with reliable cluster assignment (i.e., genes where ≥2 replicates were assigned to the same cluster), yielding a list of 2,342 genes per species (<a id="crosref0610" href="#mmc2">Data S1</a>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/24cca89b-b4c2-4822-8c36-fe88c3dd9b6a/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.02.050/asset/da87986f-5113-4031-a526-4342429b3f68/main.assets/gr5.jpg" height="882" width="749" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/24cca89b-b4c2-4822-8c36-fe88c3dd9b6a/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">The human neuroepithelium exhibits differential temporal dynamics of morphogenesis genes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0190" role="paragraph">(A) Clustering genes by temporal expression dynamics shows species differences in GO:BP term enrichment. Columns from left to right: far left, TCseq clusters with genes in each cluster plotted with their temporal expression (z-scaled) and color-coded by membership value (degree to which data points of a gene belong to the cluster, pink represents high membership values). The 10 clusters are ordered (top to bottom) based on similarity in expression pattern. Middle left: representative GO:BP term from shared (purple), human-exclusive (black), or gorilla-exclusive (fuchsia) terms for each cluster. Middle right: histograms show the number of enriched (p&nbsp;&lt; 0.05) GO:BP terms found in both species (purple), exclusively in human (black) or gorilla (fuchsia) per cluster. Axis range: 0–8 (cluster 2,5,7,8); 0–15 (cluster 9,10); 0–20 (cluster 1); 0–25 (cluster 3); 0–50 (cluster 4); 0–80 (cluster 6). Far right: weighted arc network graph visualizing interspecies differences in the enrichment/membership of specific GO:BP terms per cluster. The bases of the arc are aligned to both a human (black) and a gorilla (fuchsia) bar from the histogram in the adjacent panel, highlighting the species-specific shifts in expression patterns associated with specific GO:BP terms. Weight/thickness of the arc is dictated by the number of GO:BP terms enriched in a species-exclusive manner “moving” between clusters in the defined pattern.</div><div id="fspara0195" role="paragraph">(B) Mean temporal expression pattern (z-scaled) of genes in clusters enriched for “cell morphogenesis”-related GO:BP terms (human clusters 1, 9, 10, and gorilla cluster 3).</div><div id="fspara0200" role="paragraph">(C) Temporal expression pattern (<i>Z</i>-scaled) of <i>SHROOM3</i>, a gene involved in cell morphogenesis and apical constriction.</div><div id="fspara0205" role="paragraph">(D) Immunofluorescent staining of day 5 organoids for SHROOM3 showing strong apical expression in gorilla (G1) neuroepithelium, but not human (H9) at this time point. Scale bar, 40&nbsp;μm.</div><div id="fspara0210" role="paragraph">(E) Immunofluorescent staining of day 5 organoids for OCLN showing expression spread along the apicobasal length of human (IMR-90) progenitor cells (white arrowheads) but more limited apically (yellow arrowheads) in gorilla (G1) progenitor cells. DAPI is shown in blue. Scale bar, 100&nbsp;μm.</div><div id="fspara0215" role="paragraph">(F) Venn diagram summarizing search for cell morphogenesis-related transcription factors with species-specific expression patterns.</div><div id="fspara0220" role="paragraph">(G) Mean temporal expression pattern (<i>Z</i>-scaled) of <i>ZEB2</i>, showing peak expression earlier in gorilla (G1) than human (H9) organoids. Shaded error bars are SD.</div><div id="fspara0225" role="paragraph">See also <a id="crosref1560" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref1565" href="#mmc2">Data S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0135" role="paragraph">To assign biological functions to the identified temporal patterns, we performed GO term enrichment analysis on each TCseq cluster, keeping human and gorilla genes separate (<a id="crosref0615" href="#mmc2">Data S1</a>). We focused on GO:BP terms found in both species and examined whether each term was found in the same cluster for both species, or different clusters which would indicate a species-specific change in temporal dynamics for that biological function. This revealed a number of GO:BP terms associated with clusters common to both species (<a id="crosref0620" href="#fig5">Figure&nbsp;5</a>A), for example, GO:BP terms related to “neurogenesis” and “RNA processing” exhibited identical expression dynamics in both species.</div><div id="p0140" role="paragraph">GO:BP terms with differential expression dynamics between the two species represented particularly interesting biological functions. To visualize the major shifts in temporal expression patterns, we generated a weighted network graph of GO:BP terms that were uniquely present for one species in a given cluster, but present in a different cluster for the other species (<a id="crosref0625" href="#fig5">Figure&nbsp;5</a>A). The largest shift of GO:BP terms was between human cluster 4 and gorilla cluster 6, where terms such as “ribonucleoprotein complex assembly” were seen to move from a pattern showing a more gradual decrease in expression in human, to one dropping sharply after day 3 in gorilla (<a id="crosref0630" href="#figs5">Figure&nbsp;S5</a>A). These terms were linked to the umbrella term, “RNA processing,” and point to interesting changes in alternative splicing, previously highlighted as a key driver of evolutionary differences across primates (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Bush et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bush, S.J. ∙ Chen, L. ∙ Tovar-Corona, J.M. ...</div><div><strong>Alternative splicing and the evolution of phenotypic novelty</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20150474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2015.0474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85007137817" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27994117/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2015.0474&amp;pmid=27994117" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Another interesting shift in GO terms was “cell cycle” moving from a cluster where expression levels were similar between day 3 and 5 in human (cluster 8), to a cluster where expression levels had already started declining by day 5 in gorilla (cluster 4) (<a id="crosref0640" href="#figs5">Figure&nbsp;S5</a>B). This finding fits well with the observed differences in cell cycle length (<a id="crosref0645" href="#fig3">Figure&nbsp;3</a>G).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/1092023f-b269-42db-9ea8-b816b1622186/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.02.050/asset/3258c6e8-d973-4e22-a4b6-395e7646b70f/main.assets/figs5.jpg" height="580" width="744" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/1092023f-b269-42db-9ea8-b816b1622186/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Expression patterns of key factors with differential temporal dynamics, related to <a id="crosref1635" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0415" role="paragraph">A, B. Mean temporal expression pattern (z-scaled) of genes in clusters enriched for: A. ‘ribonucleoprotein complex assembly’-related GO:BP terms (human cluster 4, gorilla cluster 6) B. ‘cell cycle’-related GO:BP terms (human cluster 8, gorilla cluster 4). C. Immunofluorescent staining of day 10 organoids for SHROOM3 shows strong apical expression in neuroepithelium of both species (H9, G1). Scale bar: 40&nbsp;μm. D, E. Immunofluorescent staining for F-actin (Phalloidin) and DAPI of human (H9) and gorilla (G1) organoids on day 5 (D) and day 10 (E) showing weaker apical accumulation in human versus gorilla at day 5 with strong apical accumulation observed in both species by day 10. Scale bar: 40&nbsp;μm. F. Mean temporal expression (z-scaled) of EMT-related genes enriched for WP term ‘Epithelial to mesenchymal transition in colorectal cancer’ <i>MAPK12</i>, <i>MEF2D</i>, <i>PIK3R1</i> and <i>MAPK11</i>, robustly changing pattern between species in the same way enrichment of ‘cell morphogenesis’-related terms changes pattern between species (<a id="crosref1640" href="#fig5">Figure&nbsp;5</a>B). G, H. Immunofluorescent staining of organoids (H9, G1) for OCLN at G. day 3, showing expression along the apicobasal length of progenitor cells in both species H. day 10, showing lowered expression limited apically in both species. Scale bar: 40&nbsp;μm.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">The shift that we found most relevant was in GO terms such as “cell morphogenesis” and “cellular component morphogenesis,” which were present in cluster 3 of gorilla but in clusters 1, 9, and 10 of human (<a id="crosref0650" href="#fig5">Figure&nbsp;5</a>A), because differences in cell morphogenesis were precisely what we had observed. These terms were associated with genes whose expression increased more rapidly in gorilla than in human (<a id="crosref0655" href="#fig5">Figure&nbsp;5</a>B), matching the more rapid transition in cell shape observed in nonhuman apes. In particular, <i>SHROOM3</i> (<a id="crosref0660" href="#fig5">Figure&nbsp;5</a>C) immediately stood out due to its role in actin-mediated apical constriction (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Haigo et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Haigo, S.L. ∙ Hildebrand, J.D. ∙ Harland, R.M. ...</div><div><strong>Shroom induces apical constriction and is required for hingepoint formation during neural tube closure</strong></div><div><em>Curr. Biol.</em> 2003; <strong>13</strong>:2125-2137</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2003.11.054&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2003.11.054&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0346403360" target="_blank" aria-label="Open Scopus in new tab">Scopus (273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14680628/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2003.11.054&amp;pmid=14680628" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Hildebrand and Soriano, 1999</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Hildebrand, J.D. ∙ Soriano, P.</div><div><strong>Shroom, a PDZ domain-containing actin-binding protein, is required for neural tube morphogenesis in mice</strong></div><div><em>Cell.</em> 1999; <strong>99</strong>:485-497</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2FS0092-8674%2800%2981537-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2FS0092-8674%2800%2981537-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0033601076" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10589677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2981537-8&amp;pmid=10589677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Lee et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Lee, C. ∙ Scherr, H.M. ∙ Wallingford, J.B.</div><div><strong>Shroom family proteins regulate γ-tubulin distribution and microtubule architecture during epithelial cell shape change</strong></div><div><em>Development.</em> 2007; <strong>134</strong>:1431-1441</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02828" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34248165460" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17329357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02828&amp;pmid=17329357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Immunostaining for SHROOM3 at day 5 confirmed the differences detected by RNA-seq, with strong SHROOM3 localization visible along the apical surface in gorilla, whereas apical surfaces of human progenitors showed minimal to no staining (<a id="crosref0680" href="#fig5">Figure&nbsp;5</a>D). By day 10, strong SHROOM3 staining was seen at the apical surface in both species (<a id="crosref0685" href="#figs5">Figure&nbsp;S5</a>C). Phalloidin staining also revealed more extensive accumulation of actin at the apical surface of day 5 gorilla tNE cells (<a id="crosref0690" href="#figs5">Figure&nbsp;S5</a>D), whereas such accumulation appeared later in human (<a id="crosref0695" href="#figs5">Figure&nbsp;S5</a>E). These results point to SHROOM3 and actin in apical constriction in the context of NE transition.</div><div id="p0150" role="paragraph">A closer inspection of the genes contributing to the GO:BP term shift that displayed earlier onset of expression in gorilla, revealed a group of genes involved in epithelial-to-mesenchymal transition (EMT) (<a id="crosref0700" href="#figs5">Figure&nbsp;S5</a>F). This was particularly interesting because the process of corticogenesis, from NE cells to neurons, is characterized by a progressive and gradual loss of epithelial features, reminiscent of EMT (<span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Singh and Solecki, 2015</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Singh, S. ∙ Solecki, D.J.</div><div><strong>Polarity transitions during neurogenesis and germinal zone exit in the developing central nervous system</strong></div><div><em>Front. Cell. Neurosci.</em> 2015; <strong>9</strong>:62</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2015.00062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84939615626" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25852469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2015.00062&amp;pmid=25852469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One such epithelial feature that is lost is the presence of the tight junction protein OCLN (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Aaku-Saraste et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aaku-Saraste, E. ∙ Hellwig, A. ∙ Huttner, W.B.</div><div><strong>Loss of occludin and functional tight junctions, but not ZO-1, during neural tube closure--remodeling of the neuroepithelium prior to neurogenesis</strong></div><div><em>Dev. Biol.</em> 1996; <strong>180</strong>:664-679</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1996.0336" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0030589622" target="_blank" aria-label="Open Scopus in new tab">Scopus (221)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8954735/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1996.0336&amp;pmid=8954735" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Staining for OCLN revealed a difference in its distribution over time, going from extended staining along the entire apico-basal axis of individual cells at day 3 (<a id="crosref0715" href="#figs5">Figure&nbsp;S5</a>G) to a limited distribution apically at day 10 (<a id="crosref0720" href="#figs5">Figure&nbsp;S5</a>H). This change in its distribution was delayed in human, with day 5 organoids exhibiting a similar pattern to day 3, whereas gorilla OCLN was already more apically restricted (<a id="crosref0725" href="#fig5">Figure&nbsp;5</a>E).</div><div id="p0155" role="paragraph">To identify upstream regulators of this process, we queried the set of genes with robust differential temporal dynamics (<a id="crosref0730" href="#mmc2">Data S1</a>) that belonged to the “cell morphogenesis” GO:BP term for genes predicted to be transcription factors, revealing a set of 8 genes (<a id="crosref0735" href="#fig5">Figure&nbsp;5</a>F). Of these, <i>ZEB2</i> stood out due to its well-described role as an EMT master regulator (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Peinado et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Peinado, H. ∙ Olmeda, D. ∙ Cano, A.</div><div><strong>Snail, Zeb and bHLH factors in tumour progression: an alliance against the epithelial phenotype?</strong></div><div><em>Nat. Rev. Cancer.</em> 2007; <strong>7</strong>:415-428</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrc2131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34249289041" target="_blank" aria-label="Open Scopus in new tab">Scopus (2685)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17508028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrc2131&amp;pmid=17508028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Stemmler et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Stemmler, M.P. ∙ Eccles, R.L. ∙ Brabletz, S. ...</div><div><strong>Non-redundant functions of EMT transcription factors</strong></div><div><em>Nat. Cell Biol.</em> 2019; <strong>21</strong>:102-112</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-018-0196-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85059497859" target="_blank" aria-label="Open Scopus in new tab">Scopus (352)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30602760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-018-0196-y&amp;pmid=30602760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, several comparative genomics analyses have pulled out <i>ZEB2</i> as a human gene under selective pressure and exhibiting various human accelerated regions (HARs) (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Erwin et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Erwin, G.D. ∙ Oksenberg, N. ∙ Truty, R.M. ...</div><div><strong>Integrating diverse datasets improves developmental enhancer prediction</strong></div><div><em>PLoS Comput. Biol.</em> 2014; <strong>10</strong>:e1003677</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003677" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84903388630" target="_blank" aria-label="Open Scopus in new tab">Scopus (119)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24967590/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003677&amp;pmid=24967590" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Lindblad-Toh et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Lindblad-Toh, K. ∙ Garber, M. ∙ Zuk, O. ..., Broad Institute Sequencing Platform and Whole Genome Assembly Team, Baylor College of Medicine Human Genome Sequencing Center Sequencing Team, Genome Institute at Washington University</div><div><strong>A high-resolution map of human evolutionary constraint using 29 mammals</strong></div><div><em>Nature.</em> 2011; <strong>478</strong>:476-482</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10530" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-80054973803" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21993624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10530&amp;pmid=21993624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Pollard et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ King, B. ...</div><div><strong>Forces shaping the fastest evolving regions in the human genome</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e168</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33750428182" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17040131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020168&amp;pmid=17040131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>ZEB2</i>, which showed a shift from cluster 1 in human to 2 in gorilla, therefore stood out as a potential regulator of cell shape in this context. The expression dynamics of <i>ZEB2</i> revealed an earlier peak of expression in gorilla compared with human (<a id="crosref0765" href="#fig5">Figure&nbsp;5</a>G), matching the expected expression pattern of a gene that could be initiating the tNE shape transition.</div></section><section id="sec-2-5"><h3><i>ZEB2</i> is expressed during NE to RG cell transition and this process is delayed in <i>ZEB2</i> heterozygous loss-of-function organoids</h3><div id="p0160" role="paragraph">Further examination of the full time course of <i>ZEB2</i> expression and its overall levels in the RNA-seq data (<a id="crosref0770" href="#fig6">Figure&nbsp;6</a>A) revealed <i>ZEB2</i> mRNA levels peaking at day 5 in gorilla, whereas in human <i>ZEB2</i> peaked at day 10. We next examined ZEB2 protein and its downstream targets in human and gorilla organoids by western blot, revealing an earlier onset of expression of ZEB2 in gorilla compared to human (<a id="crosref0775" href="#fig6">Figure&nbsp;6</a>B). Consistent with a precocious EMT-like transition in gorilla, the epithelial cell adhesion proteins CDH1 and EpCAM showed earlier downregulation, while Vimentin, a marker of both RG and EMT, showed earlier upregulation in gorilla compared to human (<a id="crosref0780" href="#fig6">Figure&nbsp;6</a>B).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/6d5b08d4-30dc-4a86-bc6e-335052ef2680/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.02.050/asset/c5860e62-dc86-4622-a6c7-01512d3fa9e0/main.assets/gr6.jpg" height="941" width="713" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/6d5b08d4-30dc-4a86-bc6e-335052ef2680/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Decreased ZEB2 leads to expanded NE with delayed transition</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(A) Mean temporal expression pattern (log normalized transcripts per million) of <i>ZEB2</i> across the entire time series, showing peak expression earlier in gorilla (G1) than human (H9) organoids. Shaded error bars are SD.</div><div id="fspara0240" role="paragraph">(B) Western blot expression time course from PSCs to day 25 human (H9) and gorilla (G1) organoids reveals a premature onset and higher levels of ZEB2 protein expression in gorilla compared to human. This is accompanied by a premature expression of the radial glial marker vimentin, and premature downregulation of the epithelial markers CDH1 and EpCAM in gorilla relative to human. Bottom panel shows quantification of ZEB2 relative to GAPDH (AU, arbitrary units).</div><div id="fspara0245" role="paragraph">(C) Immunofluorescent stain for ZEB2 and DAPI in human (H9) and gorilla (G1) organoids at days 3, 5, and 10 showing neuroepithelial buds with nuclear expression (yellow arrows). Note the interspecies difference at day 3 where gorilla organoids already display nuclear expression compared to a weaker stain in most human cells. Insets show higher magnification of the boxed regions. Scale bar, 40&nbsp;μm.</div><div id="fspara0250" role="paragraph">(D) Immunofluorescence image of a day 25 human (H9) organoid showing a mutually exclusively pattern of expression between ZEB2 and the committed radial glia marker, BLBP. Scale bar, 100&nbsp;μm.</div><div id="fspara0255" role="paragraph">(E) Western blot of H9 wild-type (WT) and <i>ZEB2</i><sup>+/−</sup> organoids at day 16 for ZEB2, the tight-junction protein OCLN, the junction components CDH1 and CDH2, the dorsal telencephalic marker EMX1, and the loading control β-actin. The blots show a sizeable increase in CDH1 and OCLN and a decrease in CDH2, whereas EMX1 levels, and thus dorsal telencephalic identity, appears to be largely unaffected.</div><div id="fspara0260" role="paragraph">(F) Representative bright field images of day 12 WT and <i>ZEB2</i><sup>+/−</sup>. Insets show higher magnification of the boxed regions, dashed yellow are representative neuroepithelial bud perimeters quantified in (G), dashed turquois are representative neuroepithelial bud thicknesses quantified in (H). Scale bar, 500&nbsp;μm.</div><div id="fspara0265" role="paragraph">(G) Quantification of neuroepithelial bud perimeters of WT (n&nbsp;= 106) and <i>ZEB2</i><sup>+/−</sup> (n&nbsp;= 116) organoid buds from 27 WT and 28 <i>ZEB2</i><sup>+/−</sup> organoids at day 17, Mann-Whitney U test, two-tailed (<sup>∗∗∗</sup>p&nbsp;= 0.0001) from 3 organoid batches.</div><div id="fspara0270" role="paragraph">(H) Quantification of neuroepithelial bud thickness of WT (n&nbsp;= 80) and <i>ZEB2</i><sup>+/−</sup> (n&nbsp;= 119) organoid buds from 26 WT and 31 <i>ZEB2</i><sup>+/−</sup> organoids at day 12, Mann-Whitney U test, two-tailed (<sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001) from 3 organoid batches.</div><div id="fspara0275" role="paragraph">(I) Quantification of neuroepithelial bud perimeters of two <i>ZEB2</i><sup>+/−</sup>;<i>iZEB2</i> colonies treated with and without doxycycline. Colony 1: − Dox (n&nbsp;= 108 buds from 25 organoids),&nbsp;+ Dox (n&nbsp;= 70 buds from 18 organoids). Colony 2: − Dox (n&nbsp;= 62 buds from 17 organoids),&nbsp;+ Dox (n&nbsp;= 84 buds from 19 organoids) across 3 organoid batches. Mann-Whitney U tests, two-tailed (<sup>∗∗∗</sup>p&nbsp;= 0.0003 <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001).</div><div id="fspara0280" role="paragraph">(J) Immunofluorescence images of day 15 WT and <i>ZEB2</i><sup>+/−</sup> organoids showing increased OCLN immunostaining (yellow arrowheads) along the apico-(ZO1) basal (dashed line) axis of progenitor cells and reduced numbers of TBR2<sup>+</sup> cells in <i>ZEB2</i><sup>+/−</sup> organoids compared to WT. Scale bar, 100&nbsp;μm.</div><div id="fspara0285" role="paragraph">See also <a id="crosref1570" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">Immunofluorescence staining for ZEB2 across various stages of organoid development revealed specific nuclear expression appearing earlier in gorilla (<a id="crosref0785" href="#fig6">Figure&nbsp;6</a>C). Expression was gradually lost from progenitors as neurogenesis began with a switch at later stages to postmitotic neurons (<a id="crosref0790" href="#figs6">Figures S6</a>A–S6C). In organoids that were still completing the transition to neurogenic states, we observed a mutually exclusive pattern of expression between ZEB2 and markers of committed RG identity, such as BLBP (<a id="crosref0795" href="#fig6">Figure&nbsp;6</a>D) and GLAST (<a id="crosref0800" href="#figs6">Figure&nbsp;S6</a>D). These data suggest that ZEB2 expression increases in NE cells during their transition to RG cells, but is then lost in fully transitioned RG cells.</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/3c614255-ef1d-406a-9933-0ac3f794d48a/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.02.050/asset/c3ae15bf-b6c6-4df0-868c-0c8d2410da4b/main.assets/figs6.jpg" height="941" width="709" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/3c614255-ef1d-406a-9933-0ac3f794d48a/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">ZEB2 expression and targeting for loss of function, related to <a id="crosref1645" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0425" role="paragraph">A. Representative immunofluorescence image showing ZEB2 expression in SOX2+ progenitor cells in day 10 human (H9) organoid. Scale bar: 50&nbsp;μm. B. Representative immunofluorescence image showing a salt-and-pepper pattern of ZEB2 expression in the ventricular zone at day 25, after the onset of neurogenesis in human (H9) organoid. DCX (Doublecortin) stains newly born neurons. Scale bar: 100&nbsp;μm. C. Representative immunofluorescence image of a mature day 60 human (H9) organoid revealing ZEB2 expression in CTIP2+ neurons and absence of ZEB2 staining in the ventricular zone. Scale bar: 100&nbsp;μm. D. Representative immunofluorescence image of a day 25 human (H9) organoid showing a mutually exclusive pattern of expression between ZEB2 and the radial glia marker protein GLAST. Scale bar: 100&nbsp;μm. E. Schematic representation of the CRISPR-Cas9n editing strategy, where the first coding exon of the <i>ZEB2</i> gene (exon 2, NCBI ref sequence NM_014795.4:182-323) was targeted by two nickases (dashed lines) and screening was performed by assaying the drop-off frequency of a HEX-labeled probe, binding to one of the nick sites, relative to a FAM-labeled reference probe binding away from the disrupted region. The exon is marked in orange while introns are marked in purple. F. Example ddPCR 2D scatter-plots of a negative control sample (HEK293 cells), showing only a FAM-HEX double positive (red) and an empty droplet cluster (black) and a positive control sample (HEK293 cells expressing WT Cas9 and <i>ZEB2</i> guides), showing a FAM-only cluster (blue) in the upper-left quadrant of the 2D plot corresponding to edited alleles. ddPCR 2D scatter-plot of the H9 <i>ZEB2</i><sup><i>+/−</i></sup> hESC edited line showing a 1:1 ratio between the WT and edited allele. (G). Representative chromatograms of the <i>ZEB2</i> alleles in the H9 <i>ZEB2</i><sup><i>+/−</i></sup> hESC cells. The CRISPR-Cas9 target region was PCR amplified with a high-fidelity polymerase, the PCR product was blunt-end cloned into the pJET1.2 vector and following purification, plasmids from different colonies carrying the insert were sequenced. Sequencing reveals that the edited allele harbors a 23&nbsp;bp deletion. H. DNA-PAGE analysis of a short PCR amplicon spanning the CRISPR-Cas9 <i>ZEB2</i> target site in WT H9 and H9 <i>ZEB2</i><sup><i>+/−</i></sup> hESCs. The gel reveals the presence of two bands, corresponding to the WT and the edited allele in H9 <i>ZEB2</i><sup><i>+/−</i></sup> hESCs. I. Representative images of karyotype analysis on 20 G-banded metaphase spreads from the H9 <i>ZEB2</i><sup><i>+/−</i></sup> hESCs used to generate the stock. The cell line displays normal karyotype. J. RT-PCR analysis for expression of <i>ZEB2,</i> the key pluripotency markers <i>SOX2, NANOG, OCT4</i> and <i>DPPA5</i> and the loading control <i>GAPDH</i>. PCR shows that upon a ~50% reduction in <i>ZEB2</i> mRNA levels the mutant stem cells retain expression of pluripotency markers at comparable levels to WT H9 hESCs. WT and <i>ZEB2</i><sup>+/−</sup> were run on the same gel but not adjacent to each other, the dashed line indicates where the gel was spliced. K. Full length western blot for ZEB2 in WT and <i>ZEB2</i><sup><i>+/−</i></sup> organoids at day 15 – loading control was GAPDH L. Box and whiskers plot reporting the quantifications of the number of TBR2+ cells per unit area (TBR2+ cells/mm<sup>2</sup>) in day 16 WT and <i>ZEB2</i><sup><i>+/−</i></sup> organoids. Quantifications were performed by manual counting on n&nbsp;= 52 WT and n&nbsp;= 68 <i>ZEB2</i><sup><i>+/−</i></sup> ventricles corresponding to 12 organoids from 2 distinct batches. A two-tailed Mann-Whitney U test was used for statistical comparison (<sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001). M. Representative immunofluorescence images of day 55 WT and <i>ZEB2</i><sup><i>+/−</i></sup> cerebral organoid buds used for quantifications shown in N. Scale bar: 200&nbsp;μm. N. Box and whiskers plot reporting the quantifications of the number of TBR2+ cells per unit area (TBR2+ cells/mm<sup>2</sup>) in day 55 WT and <i>ZEB2</i><sup><i>+/−</i></sup> organoids. Quantifications were performed using an automated cell segmentation pipeline on n&nbsp;= 17 WT and n&nbsp;= 17 <i>ZEB2</i><sup><i>+/−</i></sup> organoid regions from 3 distinct batches. A two-tailed Mann-Whitney U test was used for statistical comparison (ns, p&nbsp;= 0.1139). O. Plasmid maps of the CRISPR homology-directed repair (HDR) templates used to target the AAVS1 safe-harbor locus in H9 hESC cells – top is the CAG-lox-STOP-lox-ZEB2-GFP-Flag inducible expression construct and bottom is the construct encoding CRE recombinase under the control of a tetracycline responsive promoter and the reverse tetracycline transactivator (rtTA) driven by the CAG promoter. P. UCSC Genome Browser view of the AAVS1 locus and CRISPR-Cas9 targeting strategy of intron 1 of <i>PPP1R12C</i>. The promoter-less splice-acceptor (SA), T2A peptide-linked “gene trap” is such that expression of the promoter-less selection cassette is driven by the endogenous <i>PPP1R12C</i> gene, thus effectively eliminating false-positive background arising from random integration. The panel reports the PCR genotyping strategy – upon successful targeting of the AAVS1 locus, while amplicon 1 is lost due to the size increase following insert integration, amplicons 2 and 3 are gained - see <a id="crosref0589" href="#figs7">Figure S7</a>A. Q. PCR gel showing successful genotyping of the two rescue clones used for the experiments shown. R. Representative brightfield images of day 15 <i>ZEB2</i><sup><i>+/−</i></sup><i>; iZEB2</i> cerebral organoids treated with and without doxycycline. Scale bar: 100&nbsp;μm. S. Representative immunofluorescence images of <i>ZEB2</i><sup><i>+/−</i></sup><i>; iZEB2</i> treated with and without doxycycline stained for GFP, TBR2 and DAPI. Scale bar: 100&nbsp;μm T. Box and whiskers plot reporting the quantifications done using an automated cell segmentation pipeline of the number of TBR2+ cells per unit area (TBR2+ cells/mm<sup>2</sup>) in day 15 <i>ZEB2</i><sup><i>+/−</i></sup><i>; iZEB2</i> organoids - colony 1: -Dox (n&nbsp;= 17 organoid regions),&nbsp;+Dox (n&nbsp;= 16 organoid regions); colony 2: -Dox (n&nbsp;= 13 organoid regions),&nbsp;+Dox (n&nbsp;= 13 organoid regions) from three independent batches. Mann-Whitney U tests, two-tailed (<sup>∗∗</sup>p&nbsp;&lt; 0.01).</div></div></div></div></figcaption></figure></div><div id="p0170" role="paragraph">To test whether ZEB2 was required for the NE-tNE-RG transition, we generated <i>ZEB2</i> mutant human embryonic stem cells (ESCs). CRISPR targeting resulted in recovery of only heterozygous loss-of-function cells (<a id="crosref0805" href="#figs6">Figures S6</a>E–S6H), suggesting it may be necessary for pluripotency or survival. Heterozygous loss-of-function mutations in <i>ZEB2</i> have been described to cause Mowat-Wilson syndrome, a complex disorder that manifests as an array of brain developmental defects with variable penetrance (<span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Mowat et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Mowat, D.R. ∙ Wilson, M.J. ∙ Goossens, M.</div><div><strong>Mowat-Wilson syndrome</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2003; <strong>40</strong>:305-310</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.40.5.305" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0038577214" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12746390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.40.5.305&amp;pmid=12746390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore reasoned that <i>ZEB2</i><sup>+/−</sup>organoids may be informative, both from an evolutionary point of view, and in understanding this human condition. Cytological and RT-PCR analyses confirmed the cells were healthy and expressed key pluripotency marker genes to the same levels as the parental control (<a id="crosref0815" href="#figs6">Figures S6</a>I and S6J) despite showing a reduction in <i>ZEB2</i> mRNA levels (<a id="crosref0820" href="#figs6">Figure&nbsp;S6</a>J). Western blot analysis of <i>ZEB2</i><sup>+/−</sup> organoids confirmed a reduction in ZEB2 protein compared to control (<a id="crosref0825" href="#figs6">Figure&nbsp;S6</a>K) and revealed expected effects on junctional proteins CDH1, CDH2, and OCLN (<a id="crosref0830" href="#fig6">Figure&nbsp;6</a>E).</div><div id="p0175" role="paragraph">We next tested whether the <i>ZEB2</i><sup>+/−</sup> mutation led to changes in tissue architecture consistent with a role in NE to RG cell transition. Bright field imaging revealed more prominent and extended neuroepithelial buds (<a id="crosref0835" href="#fig6">Figure&nbsp;6</a>F) with overall larger sizes (<a id="crosref0840" href="#fig6">Figure&nbsp;6</a>G) and reduced thickness (<a id="crosref0845" href="#fig6">Figure&nbsp;6</a>H) consistent with defective NE to RG transition. Furthermore, <i>ZEB2</i><sup>+/−</sup> organoids showed significantly decreased numbers of TBR2<sup>+</sup> intermediate progenitor cells at early stages (<a id="crosref0850" href="#figs6">Figure&nbsp;S6</a>L) that returned to normal at later stages (<a id="crosref0855" href="#figs6">Figures S6</a>M and S6N), suggesting a potential delay in neurogenesis that would be consistent with a delayed transition to neurogenic RG. We tested the specificity of this phenotype by introducing a doxycycline inducible overexpression construct for <i>ZEB2-GFP</i> (<a id="crosref0860" href="#figs6">Figure&nbsp;S6</a>O) into <i>ZEB2</i><sup>+/−</sup> cells and selecting clones with proper integration of the expression cassette (<a id="crosref0865" href="#figs6">Figures S6</a>P and S6Q). <i>iZEB2</i> clones of <i>ZEB2</i><sup>+/−</sup> cells (<i>ZEB2</i><sup>+/−</sup>; <i>iZEB2</i>) displayed a rescued phenotype upon doxycycline induction, both in terms of neuroepithelial bud size (<a id="crosref0870" href="#fig6">Figures 6</a>I and <a id="crosref0875" href="#figs6">S6</a>R) and TBR2<sup>+</sup> cell numbers (<a id="crosref0880" href="#figs6">Figures S6</a>S and S6T).</div><div id="p0180" role="paragraph">Staining of early neurogenic organoids showed that although control organoids had redistributed the tight junction epithelial protein OCLN near the apical surface of cells, <i>ZEB2</i><sup>+/−</sup> mutant organoids retained marked OCLN staining along the entire apico-basal axis of individual progenitor cells (<a id="crosref0885" href="#fig6">Figure&nbsp;6</a>J). Together, these findings reveal a specific effect of decreased ZEB2 levels on tight junctions and cell shape of transitioning NE cells that leads to an enlarged neuroepithelium.</div></section><section id="sec-2-6"><h3>Manipulation of <i>ZEB2</i> expression and downstream signaling leads to interspecies phenocopy</h3><div id="p0185" role="paragraph">We next tested whether changing <i>ZEB2</i> expression dynamics to match that of the gorilla would be sufficient to trigger an earlier transition to tNE cells in human organoids. We generated an otherwise wild-type human cell line harboring the inducible <i>ZEB2-GFP</i> described above (<a id="crosref0890" href="#figs7">Figure&nbsp;S7</a>A). We observed robust and tight expression upon induction, both by western blot and immunofluorescent staining (<a id="crosref0895" href="#figs7">Figures S7</a>B and S7C). Following doxycycline treatment in stem cells, we observed a reduction in EpCAM and CDH1 staining accompanied by an increase in VIM and CDH2 staining (<a id="crosref0900" href="#figs7">Figures S7</a>D and S7E) relative to the uninduced control.</div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/f1d4285a-3489-4902-ae85-b0c1453424fe/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.02.050/asset/628ca97d-3458-4822-bfe8-01de8fd25ca4/main.assets/figs7.jpg" height="941" width="661" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/f1d4285a-3489-4902-ae85-b0c1453424fe/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Modulation of ZEB2 and SMAD signaling in human and gorilla cells, related to <a id="crosref1650" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0435" role="paragraph">A. PCR gel showing successful genotyping of the Hum<sup><i>iZEB2</i></sup> colony used for all experiments shown based on the PCR genotyping strategy outlined in <a id="crosref1655" href="#figs6">Figure&nbsp;S6</a>P. The asterisks mark unspecific bands. B. Transgene induction in Hum<sup><i>iZEB2</i></sup> cells treated with and without doxycycline for 6&nbsp;days and assayed by western blot for ZEB2, GFP and β-actin. C. Immunofluorescence images of 6-day induced and uninduced Hum<sup><i>iZEB2</i></sup> cells stained for ZEB2 and DAPI, showing that doxycyline induction results in ZEB2 expression and nuclear translocation, without adverse effects on its localization due to tagging with GFP. Scale bar: 20&nbsp;μm D. Immunofluorescence images of 6-day induced and uninduced Hum<sup><i>iZEB2</i></sup> cells stained for DAPI, CDH1 and CDH2. The data reveal a reduction in CDH1 expression and an increase in CDH2 expression following induction. Scale bar: 50&nbsp;μm. E. Immunofluorescence images of 6-day induced and uninduced Hum<sup><i>iZEB2</i></sup> cells stained for GFP, Vimentin and EpCAM. The data reveal a reduction in EpCAM expression and an increase in Vimentin expression following expression of ZEB2-GFP. Scale bar: 50&nbsp;μm. F. Brightfield images of induced (+ Dox) and uninduced (- Dox) Hum<sup><i>iZEB2</i></sup> organoids and gorilla (G1) organoids at days 3, 5 and 10, showing indistinguishable tissue architecture between organoids at day 3, while day 5 and 10 organoids show neuroepithelial buds that are smaller and more rounded in gorilla and <i>ZEB2</i> induced (+ Dox) versus uninduced (- Dox) organoids. Scale bar: 200&nbsp;μm. G. Western blot of day 5 WT and <i>ZEB2</i><sup><i>+/−</i></sup> organoids revealing both decreased ZEB2 and SHROOM3 levels in <i>ZEB2</i><sup><i>+/−</i></sup>&nbsp;organoids compared to WT control β-Actin was used as loading control. H. Representative bright-field images of uninduced and induced Hum<sup><i>iZEB2</i></sup> and gorilla neuroepithelial buds at day 5, used for quantification in <a id="crosref1660" href="#fig7">Figure&nbsp;7</a>H. Scale bar: 100&nbsp;μm. I. Immunofluorescent staining for ZO1, SOX2, DCX and DAPI showing normal tissue morphology and onset of neurogenesis (DCX+ neurons) in uninduced (- Dox) and induced (+ Dox) Hum<sup><i>iZEB2</i></sup> organoids at day 10. Scale bar: 100&nbsp;μm. J. Western blot for ZEB2, CDH1, and CDH2, with β-Actin as loading control, of WT and <i>ZEB2</i><sup><i>+/−</i></sup> organoids treated with dual-SMAD inhibitors, or treated with vehicle, for 10&nbsp;days and assayed at day 12. Note the rescued levels of junctional components CDH1 and CDH2. K. Morphological assessment of WT and <i>ZEB2</i><sup><i>+/−</i></sup> organoids treated with dual-SMAD inhibitors, or treated with vehicle, for 10&nbsp;days and assayed at day 12 by brightfield imaging (left panels) and hematoxylin-eosin staining (right panels). Note the elongated neuroepithelial buds (arrows) in mutant organoids that appear rescued (arrowheads) upon SMAD inhibition. Scale bars: 1&nbsp;mm (left panels), 50&nbsp;μm (right panels).</div></div></div></div></figcaption></figure></div><div id="p0190" role="paragraph">Organoids from these inducible <i>ZEB2</i> human (Hum<sup><i>iZEB2</i></sup>) cells were treated with doxycycline to induce <i>ZEB2</i> expression at an earlier stage (<a id="crosref0905" href="#fig7">Figure&nbsp;7</a>A). Examination of tissue architecture at day 3 revealed indistinguishable morphology between uninduced Hum<sup><i>iZEB2</i></sup>, induced Hum<sup><i>iZEB2</i></sup>, and gorilla organoids (<a id="crosref0910" href="#figs7">Figure&nbsp;S7</a>F). By day 5, however, neuroepithelial buds generated in induced Hum<sup><i>iZEB2</i></sup> organoids were more similar to gorilla in shape, appearing smaller and more rounded, whereas uninduced Hum<sup><i>iZEB2</i></sup> organoids exhibited buds with more elongated shapes typical of human organoids (<a id="crosref0915" href="#fig7">Figures 7</a>B and <a id="crosref0920" href="#figs7">S7</a>F). This change in morphology was also accompanied by earlier expression and localization of SHROOM3 (<a id="crosref0925" href="#fig7">Figures 7</a>A and <a id="crosref0930" href="#figs7">S7</a>G), the actin regulator identified to exhibit precocious expression in gorilla (<a id="crosref0935" href="#fig5">Figure&nbsp;5</a>D).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.02.050/asset/4c15e73a-cadf-45e0-8a6c-b04a43317edc/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.02.050/asset/18f58f27-3d28-4256-baaf-927b8ce8ba9a/main.assets/gr7.jpg" height="943" width="709" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/4c15e73a-cadf-45e0-8a6c-b04a43317edc/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">ZEB2-driven junctional remodeling and apical constriction dictate species-specific timing of NE transition</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0295" role="paragraph">(A) Immunofluorescent staining of uninduced (− Dox) and induced (+ Dox) Hum<sup><i>iZEB2</i></sup> organoids for GFP and SHROOM3. Note the expression of ZEB2-GFP and apical accumulation of SHROOM3 in induced organoids. Scale bar, 50&nbsp;μm.</div><div id="fspara0300" role="paragraph">(B) Representative bright field images of day 5 Hum<sup><i>iZEB2</i></sup> and gorilla organoids. Induced (+ Dox) Hum<sup><i>iZEB2</i></sup> organoids show smaller neuroepithelial buds (arrowheads) that are more round in shape, similar to gorilla (G1), while uninduced (− Dox) show more elongated structures typical of human. Scale bar, 200&nbsp;μm.</div><div id="fspara0305" role="paragraph">(C) Immunofluorescence images through day 5 whole mount Hum<sup><i>iZEB2</i></sup> uninduced (− Dox), induced (+ Dox) and gorilla (G1) organoids stained for GFP, ZO1, and SOX2. Sparse labeling with viral GFP shows <i>ZEB2</i> induction triggers the constriction of apical processes (arrows) in progenitor cells, similar to gorilla at day 5. Scale bar, 50&nbsp;μm.</div><div id="fspara0310" role="paragraph">(D) Representative immunofluorescence images through whole mount day 5 uninduced (− Dox), induced (+ Dox) Hum<sup><i>iZEB2</i></sup> and gorilla (G1) organoids with superimposed individual segmented GFP+ progenitor cells (white) showing their 3D morphology. Note the thinning of apical processes observed upon <i>ZEB2</i> induction. Scale bar, 10&nbsp;μm.</div><div id="fspara0315" role="paragraph">(E) Immunofluorescent staining for ZO1 on the surface of apical lumens showing the apical surface areas of individual progenitor cells in day 5 Hum<sup><i>iZEB2</i></sup> uninduced (− Dox), induced (+ Dox) and gorilla (G1) organoids. Perimeters of some individual progenitor cells of day 5 organoids are delineated in white. Scale bar, 10&nbsp;μm.</div><div id="fspara0320" role="paragraph">(F) Quantification of the volume as normalized to surface area of the apical processes of induced (+ Dox) versus uninduced (− Dox) Hum<sup><i>iZEB2</i></sup> neural progenitor cells on day 5. The apical processes of segmented cells directly below the cell body were used for quantification. Gorilla day 5 measurements from <a id="crosref1575" href="#fig2">Figure&nbsp;2</a>E are included for comparison. Mean apical process volume:surface area ratio: Hum<sup><i>iZEB2</i></sup> − Dox&nbsp;= 1.11; Hum<sup><i>iZEB2</i></sup>&nbsp;+ Dox&nbsp;= 0.76. <sup>∗∗</sup>p&nbsp;&lt; 0.01, Mann-Whitney U, two-tailed, n (− Dox and&nbsp;+ Dox)&nbsp;= 9 cells. Error bars are SD.</div><div id="fspara0325" role="paragraph">(G) Quantification of the surface area of individual delineated ZO1 cell perimeters as shown in (E). Gorilla measurements from two cell lines (G1, G2) combined are shown for comparison. Mean apical surface area/cell: Hum<sup><i>iZEB2</i></sup> − Dox&nbsp;= 9.62&nbsp;μm<sup>2</sup>, Hum<sup><i>iZEB2</i></sup>&nbsp;+ Dox&nbsp;= 3.08&nbsp;μm<sup>2</sup>, and gorilla (G1,G2)&nbsp;= 4.50&nbsp;μm<sup>2</sup>. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Mann-Whitney U, two-tailed, n (− Dox)&nbsp;= 180 cells from 8 organoids, n (+ Dox)&nbsp;= 199 cells from 8 organoids, both from 2 independent batches, box and whisker plots show median with min-max values, data points represent individual cells.</div><div id="fspara0330" role="paragraph">(H) Quantification of perimeters of neuroepithelial buds from bright field images at days 5 and 10, and overall organoid size at day 10. Gorilla (G1,G2) measurements were combined and included for comparison. Day 5 mean neuroepithelial bud perimeter: Hum<sup><i>iZEB2</i></sup> − Dox&nbsp;= 272&nbsp;μm, Hum<sup><i>iZEB2</i></sup>&nbsp;+ Dox&nbsp;= 237&nbsp;μm, and gorilla (G1,G2)&nbsp;= 232&nbsp;μm. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Mann-Whitney U, two-tailed, n (− Dox)&nbsp;= 142 neuroepithelial buds from 41 organoids from 3 independent batches; n (+Dox)&nbsp;= 195 neuroepithelial buds from 38 organoids from 3 independent batches; n (G1,G2)&nbsp;= 555 neuroepithelial buds from 114 organoids from 16 independent batches. Day 10 mean neuroepithelial bud perimeter: Hum<sup><i>iZEB2</i></sup> − Dox&nbsp;= 300&nbsp;μm, Hum<sup><i>iZEB2</i></sup>&nbsp;+ Dox&nbsp;= 198&nbsp;μm, and gorilla (G1,G2)&nbsp;= 227&nbsp;μm. Day 10 mean organoid area: Hum<sup><i>iZEB2</i></sup> − Dox&nbsp;= 201,434&nbsp;μm<sup>2</sup>, Hum<sup><i>iZEB2</i></sup>&nbsp;+ Dox&nbsp;= 132,325&nbsp;μm<sup>2</sup>, and gorilla (G1,G2)&nbsp;= 93,447&nbsp;μm<sup>2</sup>. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Mann-Whitney U, two-tailed, n (− Dox day 10)&nbsp;= 15 organoids and 106 neuroepithelial buds from 3 independent batches, n (+ Dox day 10)&nbsp;= 15 organoids and 149 neuroepithelial buds from 3 independent batches, error bars are SD.</div><div id="fspara0335" role="paragraph">(I) Representative immunofluorescence images showing the effect of BMP4 on the morphology of neural progenitor cells revealed by sparse viral labeling with GFP on day 5 untreated (− BMP4) and treated (+ BMP4) gorilla (G1) organoids with staining for GFP, SOX2, and DAPI. Arrows indicate the apical process. Scale bar, 40&nbsp;μm.</div><div id="fspara0340" role="paragraph">(J) Immunofluorescent staining for ZO1 showing apical surface areas of individual progenitor cells from BMP4-treated (+ BMP4) and untreated (− BMP4) gorilla (G1) organoids at day 5. Perimeters of some individual progenitor cells are delineated in white. Scale bar, 10&nbsp;μm.</div><div id="fspara0345" role="paragraph">(K) Quantification of individual delineated ZO1 cell perimeters as shown in (J). Mean apical surface area/cell: gorilla − BMP4&nbsp;= 2.72&nbsp;μm<sup>2</sup> and gorilla&nbsp;+ BMP4&nbsp;= 4.13&nbsp;μm<sup>2</sup>. <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Mann-Whitney U, two-tailed, n (− BMP4)&nbsp;= 301 cells from 8 organoids from 2 independent batches, n (+ BMP4)&nbsp;= 326 cells from 8 organoids from 2 independent batches, box and whisker plots are median with min-max values, data points represent individual cells.</div><div id="fspara0350" role="paragraph">(L) Immunofluorescence images of human (H9) and gorilla (G1) day 5 organoids untreated (− LPA) and treated (+ LPA) with staining for OCLN, ZO1, and DAPI. LPA treatment in gorilla results in increased OCLN distribution along the apicobasal axis of cells (arrowheads) and expanded apical surfaces of cells (ZO1, bottom panel). Scale bar, 40&nbsp;μm (upper panels), 10&nbsp;μm (bottom panels).</div><div id="fspara0355" role="paragraph">(M) Quantification of individual delineated ZO1 cell perimeters as shown in (L). Mean apical surface area/cell: human − LPA&nbsp;= 5.36&nbsp;μm<sup>2</sup>, human&nbsp;+ LPA&nbsp;= 5.25&nbsp;μm<sup>2</sup>, gorilla − LPA&nbsp;= 2.44&nbsp;μm<sup>2</sup>, and gorilla&nbsp;+ LPA&nbsp;= 4.73&nbsp;μm<sup>2</sup>. <sup>∗</sup>p&nbsp;&lt; 0.05 <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001, Kruskal-Wallis and post hoc Dunn’s multiple comparisons test, n (human − LPA)&nbsp;= 146 cells from 3 organoids of 1 batch, n (human&nbsp;+ LPA)&nbsp;= 200 cells from 3 organoids of 1 batch, n (gorilla − LPA)&nbsp;= 375 cells from 7 organoids and 2 independent batches, n (gorilla&nbsp;+ LPA)&nbsp;= 457 cells from 10 organoids and 2 independent batches, box and whisker plots show median with min-max values, data points represent individual cells.</div><div id="fspara0360" role="paragraph">(N) Schematic summarizing the morphological changes that occur in neural progenitor cells as they transition from NE to tNE cells (purple background). ZEB2 is highlighted as a driver, which acts through BMP-responsive SMADs to downregulate epithelial features, notably tight-junction proteins (TJs, green), and involves apical constriction through rearrangements in the actin cytoskeleton (actin, magenta).</div><div id="fspara0365" role="paragraph">See also <a id="crosref1580" href="#figs7">Figure&nbsp;S7</a>.</div></div></div></div></figcaption></figure></div><div id="p0195" role="paragraph">Sparse viral labeling with GFP revealed that NE cells of induced Hum<sup><i>iZEB2</i></sup> organoids were less columnar and more tNE-like in shape than uninduced cells, with a thinning of apical and basal processes that was highly similar to gorilla tNE cells at the same time point (<a id="crosref0940" href="#fig7">Figure&nbsp;7</a>C). Changes in cell shape and apical constriction were quantified by cell segmentation (<a id="crosref0945" href="#fig7">Figure&nbsp;7</a>D) and ZO1 staining of the apical surface (<a id="crosref0950" href="#fig7">Figure&nbsp;7</a>E), revealing a remarkably similar apical constriction in human <i>ZEB2</i> overexpressing organoids to gorilla organoids (<a id="crosref0955" href="#fig7">Figures 7</a>F and 7G). We further tested the effect on tissue size by measuring neuroepithelial bud perimeter and organoid area (<a id="crosref0960" href="#fig7">Figures 7</a>H and <a id="crosref0965" href="#figs7">S7</a>H), revealing a phenotype reminiscent of the nonhuman ape. Staining of later stages revealed normal neurogenesis (<a id="crosref0970" href="#figs7">Figure&nbsp;S7</a>I), demonstrating that these effects seemed to specifically affect the timing and not the identity or fate of progenitors. These findings suggest that premature expression of <i>ZEB2</i> in human organoids is sufficient to recapitulate the precocious tNE cell shape change seen in gorilla organoids.</div><div id="p0200" role="paragraph">ZEB2, also known as SMAD interacting protein 1 (SIP1), is a known regulator of SMAD signaling that leads to downstream effects on cell-cell junction proteins including OCLN and CDH1 and 2 (<span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Vandewalle et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Vandewalle, C. ∙ Comijn, J. ∙ De Craene, B. ...</div><div><strong>SIP1/ZEB2 induces EMT by repressing genes of different epithelial cell-cell junctions</strong></div><div><em>Nucleic Acids Res.</em> 2005; <strong>33</strong>:6566-6578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gki965" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-29144485712" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16314317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgki965&amp;pmid=16314317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore tested whether SMAD signaling may also be responsible for the effects of ZEB2 in this context by performing rescue experiments with SMAD inhibitors in the context of <i>ZEB2</i><sup>+/−</sup> mutation. We observed a rescue of the effects of ZEB2 on expression of junction protein CDH1 (<a id="crosref0980" href="#figs7">Figure&nbsp;S7</a>J) as well as rescued tissue architecture (<a id="crosref0985" href="#figs7">Figure&nbsp;S7</a>K), suggesting SMAD signaling may be responsible for the EMT effects of ZEB2 during the NE to RG transition.</div><div id="p0205" role="paragraph">We next took advantage of this information to see whether gorilla organoids could be encouraged to take on a more human-like phenotype by manipulating SMAD signaling. We focused on activation of BMP-responsive SMADs, because this pathway has previously been shown to inhibit EMT (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Amarnath and Agarwala, 2017</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Amarnath, S. ∙ Agarwala, S.</div><div><strong>Cell-cycle-dependent TGFβ-BMP antagonism regulates neural tube closure by modulating tight junctions</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2017; <strong>130</strong>:119-131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.179192" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85010223698" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27034139/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.179192&amp;pmid=27034139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and ZEB2 can inhibit BMP-responsive SMADs in other neuronal contexts (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Menuchin-Lasowski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Menuchin-Lasowski, Y. ∙ Dagan, B. ∙ Conidi, A. ...</div><div><strong>Zeb2 regulates the balance between retinal interneurons and Müller glia by inhibition of BMP-Smad signaling</strong></div><div><em>Dev. Biol.</em> 2020; <strong>468</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2020.09.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85091666282" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32950463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2020.09.006&amp;pmid=32950463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Application of BMP4 on gorilla organoids when they would normally begin transitioning to tNE cell morphologies led to the appearance of cells with more dilated apical processes (<a id="crosref1000" href="#fig7">Figure&nbsp;7</a>I) and enlarged apical contacts (<a id="crosref1005" href="#fig7">Figures 7</a>J and 7K), reminiscent of human NE cells. Thus, by activating downstream SMADs, we were able to override the EMT processes that would normally promote acquisition of tNE morphology at this stage.</div><div id="p0210" role="paragraph">Finally, because we observed a species-specific change in actin-associated apical constriction and SHROOM3 expression that was also responsive to ZEB2 (<a id="crosref1010" href="#fig7">Figures 7</a>A and <a id="crosref1015" href="#figs7">S7</a>G), we used an orthogonal approach to manipulate apical constriction through the use of lysophosphatidic acid (LPA), a molecule previously shown to enlarge apical contacts of NE cells <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Medelnik et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Medelnik, J.P. ∙ Roensch, K. ∙ Okawa, S. ...</div><div><strong>Signaling-Dependent Control of Apical Membrane Size and Self-Renewal in Rosette-Stage Human Neuroepithelial Stem Cells</strong></div><div><em>Stem Cell Reports.</em> 2018; <strong>10</strong>:1751-1765</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85046875468" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.04.018&amp;pmid=29779899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Application of LPA on gorilla organoids at the onset of NE transition led to morphologies more closely mimicking the human phenotype (<a id="crosref1025" href="#fig7">Figure&nbsp;7</a>L) with enlarged apical contacts (<a id="crosref1030" href="#fig7">Figure&nbsp;7</a>M) and more extended OCLN staining along the apicobasal axis (<a id="crosref1035" href="#fig7">Figure&nbsp;7</a>L). Together, these findings point to a close interaction between cell-cell junctions and apical constriction, both of which are regulated by ZEB2 to drive the change in cell shape that characterizes this newly described transition state (<a id="crosref1040" href="#fig7">Figure&nbsp;7</a>N).</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0215" role="paragraph">These results provide insight into how NE cells transition into neurogenic RG cells, highlighting an intermediate tNE cell stage, hitherto unidentified in the mammalian neocortex. Interestingly, a transitioning NE cell type has been described in <i>Drosophila</i> optic primordia with features analogous to those we have observed in human and ape organoids (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Orihara-Ono et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Orihara-Ono, M. ∙ Toriya, M. ∙ Nakao, K. ...</div><div><strong>Downregulation of Notch mediates the seamless transition of individual Drosophila neuroepithelial progenitors into optic medullar neuroblasts during prolonged G1</strong></div><div><em>Dev. Biol.</em> 2011; <strong>351</strong>:163-175</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2010.12.044" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-79551681632" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21215740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2010.12.044&amp;pmid=21215740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), raising the intriguing possibility that such a transition may be evolutionarily conserved. Furthermore, the temporal progression of events and RNA-seq reveal a shift in cell morphology preceding other changes associated with RG identity, namely expression of glial markers and production of neurons. Previous findings have similarly revealed an important role for cell morphology during neurogenesis, but in the context of proliferating basal progenitors (<span class="dropBlock reference-citations"><a id="body-ref-optQB60jIAXuw" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optQB60jIAXuw" aria-controls="optQB60jIAXuw">Kalebic et al., 2019</a><div class="dropBlock__holder" data-db-target-of="optQB60jIAXuw"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Kalebic, N. ∙ Gilardi, C. ∙ Stepien, B. ...</div><div><strong>Neocortical Expansion Due to Increased Proliferation of Basal Progenitors Is Linked to Changes in Their Morphology</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:535-550</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2019.02.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2019.02.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85063715304" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30905618/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.02.017&amp;pmid=30905618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, cell shape may be a key initiator in general during these developmental transitions.</div><div id="p0220" role="paragraph">The protracted shift to the tNE morphotype in humans, coupled with a shorter cell cycle, is particularly intriguing with regard to human brain evolution, as it may explain the linear scaling of the human brain compared with apes (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Herculano-Houzel, 2009</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Herculano-Houzel, S.</div><div><strong>The human brain in numbers: a linearly scaled-up primate brain</strong></div><div><em>Front. Hum. Neurosci.</em> 2009; <strong>3</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/neuro.09.031.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84890924627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1054)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19915731/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Fneuro.09.031.2009&amp;pmid=19915731" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An evolutionary modification at such an early developmental stage would be expected to affect all subsequent steps of neurogenesis and lead to a proportional increase in all neuron types. Indeed, mathematical modeling reveals an expected increase in neurons, and staining for neurons even at early stages reveals increased numbers in human organoids (<a id="crosref1055" href="#figs2">Figure&nbsp;S2</a>A). Furthermore, cytoarchitecturally, this would be expected to lead not to a thickened cortical gray matter, but rather to a more expanded brain in the tangential dimension, which is precisely what is seen in comparative neuroanatomy to other apes (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Donahue et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Donahue, C.J. ∙ Glasser, M.F. ∙ Preuss, T.M. ...</div><div><strong>Quantitative assessment of prefrontal cortex in humans relative to nonhuman primates</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E5183-E5192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1721653115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047898223" target="_blank" aria-label="Open Scopus in new tab">Scopus (162)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29739891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1721653115&amp;pmid=29739891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-3" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-3" aria-controls="sref46-3">Hutsler et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref46-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Hutsler, J.J. ∙ Lee, D.G. ∙ Porter, K.K.</div><div><strong>Comparative analysis of cortical layering and supragranular layer enlargement in rodent carnivore and primate species</strong></div><div><em>Brain Res.</em> 2005; <strong>1052</strong>:71-81</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainres.2005.06.015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-23744472001" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16018988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainres.2005.06.015&amp;pmid=16018988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0225" role="paragraph">Together, these findings reveal a mechanism for evolutionary human brain expansion, and highlight ZEB2 as a key regulator. ZEB2 was previously shown to be involved in early germ layer specification (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Chng et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Chng, Z. ∙ Teo, A. ∙ Pedersen, R.A. ...</div><div><strong>SIP1 mediates cell-fate decisions between neuroectoderm and mesendoderm in human pluripotent stem cells</strong></div><div><em>Cell Stem Cell.</em> 2010; <strong>6</strong>:59-70</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2009.11.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2009.11.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-73049085665" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20074535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2009.11.015&amp;pmid=20074535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Eisaki et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Eisaki, A. ∙ Kuroda, H. ∙ Fukui, A. ...</div><div><strong>XSIP1, a member of two-handed zinc finger proteins, induced anterior neural markers in Xenopus laevis animal cap</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2000; <strong>271</strong>:151-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/bbrc.2000.2545" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0034728782" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10777695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fbbrc.2000.2545&amp;pmid=10777695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Stryjewska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Stryjewska, A. ∙ Dries, R. ∙ Pieters, T. ...</div><div><strong>Zeb2 regulates cell fate at the exit from epiblast state in mouse embryonic stem cells</strong></div><div><em>Stem Cells.</em> 2017; <strong>35</strong>:611-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/stem.2521" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85002194595" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27739137/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fstem.2521&amp;pmid=27739137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), neural tube morphogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Miyoshi et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Miyoshi, T. ∙ Maruhashi, M. ∙ Van De Putte, T. ...</div><div><strong>Complementary expression pattern of Zfhx1 genes Sip1 and deltaEF1 in the mouse embryo and their genetic interaction revealed by compound mutants</strong></div><div><em>Dev. Dyn.</em> 2006; <strong>235</strong>:1941-1952</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20799" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33745166261" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16598713/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20799&amp;pmid=16598713" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and neural crest differentiation (<span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Yasumi et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Yasumi, T. ∙ Inoue, M. ∙ Maruhashi, M. ...</div><div><strong>Regulation of trunk neural crest delamination by δEF1 and Sip1 in the chicken embryo</strong></div><div><em>Dev. Growth Differ.</em> 2016; <strong>58</strong>:205-214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/dgd.12256" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84958917944" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26691438/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fdgd.12256&amp;pmid=26691438" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the cortex specifically, ZEB2 has been shown to regulate the timing of cortical laminar fate determination (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Epifanova et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Epifanova, E. ∙ Babaev, A. ∙ Newman, A.G. ...</div><div><strong>Role of Zeb2/Sip1 in neuronal development</strong></div><div><em>Brain Res.</em> 2019; <strong>1705</strong>:24-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainres.2018.09.034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85054511567" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30266271/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainres.2018.09.034&amp;pmid=30266271" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Parthasarathy et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Parthasarathy, S. ∙ Srivatsa, S. ∙ Nityanandam, A. ...</div><div><strong>Ntf3 acts downstream of Sip1 in cortical postmitotic neurons to control progenitor cell fate through feedback signaling</strong></div><div><em>Development.</em> 2014; <strong>141</strong>:3324-3330</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.114173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84906215257" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25085976/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.114173&amp;pmid=25085976" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Seuntjens et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Seuntjens, E. ∙ Nityanandam, A. ∙ Miquelajauregui, A. ...</div><div><strong>Sip1 regulates sequential fate decisions by feedback signaling from postmitotic neurons to progenitors</strong></div><div><em>Nat. Neurosci.</em> 2009; <strong>12</strong>:1373-1380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2409" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-74949109986" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19838179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2409&amp;pmid=19838179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as well as neocortical axon outgrowth (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Srivatsa et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Srivatsa, S. ∙ Parthasarathy, S. ∙ Molnár, Z. ...</div><div><strong>Sip1 downstream Effector ninein controls neocortical axonal growth, ipsilateral branching, and microtubule growth and stability</strong></div><div><em>Neuron.</em> 2015; <strong>85</strong>:998-1012</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2015.01.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2015.01.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84924263870" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25741725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.01.018&amp;pmid=25741725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>ZEB2</i>’s previous identification as the causative gene in Mowat-Wilson syndrome (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Mowat et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Mowat, D.R. ∙ Wilson, M.J. ∙ Goossens, M.</div><div><strong>Mowat-Wilson syndrome</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2003; <strong>40</strong>:305-310</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.40.5.305" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0038577214" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12746390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.40.5.305&amp;pmid=12746390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) further highlights its importance in human neurodevelopment. Interestingly, patients with Mowat-Wilson syndrome also exhibit ventriculomegaly (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Garavelli et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Garavelli, L. ∙ Ivanovski, I. ∙ Caraffi, S.G. ...</div><div><strong>Neuroimaging findings in Mowat-Wilson syndrome: a study of 54 patients</strong></div><div><em>Genet. Med.</em> 2017; <strong>19</strong>:691-700</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1038%2Fgim.2016.176&amp;cf=fulltext&amp;site=gim-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1038%2Fgim.2016.176&amp;cf=pdf&amp;site=gim-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85020205876" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27831545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fgim.2016.176&amp;pmid=27831545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), or enlarged ventricles, a phenotype paralleled by the enlarged neuroepithelial buds seen in <i>ZEB2</i><sup>+/−</sup> organoids.</div><div id="p0230" role="paragraph">ZEB2 is also well-known as a key regulator of EMT (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Fardi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Fardi, M. ∙ Alivand, M. ∙ Baradaran, B. ...</div><div><strong>The crucial role of ZEB2: From development to epithelial-to-mesenchymal transition and cancer complexity</strong></div><div><em>J.&nbsp;Cell. Physiol.</em> 2019; <strong>234</strong>:14783-14799</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcp.28277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85061776287" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30773635/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcp.28277&amp;pmid=30773635" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In many respects, the NE-tNE-RG transition resembles a partial EMT process with a progressive change in cell morphology, apical constriction, and a reduction in cell-cell junctions (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Pastushenko and Blanpain, 2019</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Pastushenko, I. ∙ Blanpain, C.</div><div><strong>EMT Transition States during Tumor Progression and Metastasis</strong></div><div><em>Trends Cell Biol.</em> 2019; <strong>29</strong>:212-226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.tcb.2018.12.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.tcb.2018.12.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85059015645" target="_blank" aria-label="Open Scopus in new tab">Scopus (1765)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30594349/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tcb.2018.12.001&amp;pmid=30594349" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, apical constriction is the first morphological change associated with EMT in several tissues (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Gracia et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Gracia, M. ∙ Theis, S. ∙ Proag, A. ...</div><div><strong>Mechanical impact of epithelial-mesenchymal transition on epithelial morphogenesis in Drosophila</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2951</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10720-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85068462870" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31273212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10720-0&amp;pmid=31273212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Saunders and McClay, 2014</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Saunders, L.R. ∙ McClay, D.R.</div><div><strong>Sub-circuits of a gene regulatory network control a developmental epithelial-mesenchymal transition</strong></div><div><em>Development.</em> 2014; <strong>141</strong>:1503-1513</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.101436" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896302113" target="_blank" aria-label="Open Scopus in new tab">Scopus (86)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24598159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.101436&amp;pmid=24598159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Williams et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Williams, M. ∙ Burdsal, C. ∙ Periasamy, A. ...</div><div><strong>Mouse primitive streak forms in&nbsp;situ by initiation of epithelial to mesenchymal transition without migration of a cell population</strong></div><div><em>Dev. Dyn.</em> 2012; <strong>241</strong>:270-283</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.23711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84856221662" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22170865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.23711&amp;pmid=22170865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and changes in epithelial cell-cell junctions are linked to cytoskeletal changes (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Morris and Machesky, 2015</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Morris, H.T. ∙ Machesky, L.M.</div><div><strong>Actin cytoskeletal control during epithelial to mesenchymal transition: focus on the pancreas and intestinal tract</strong></div><div><em>Br. J. Cancer.</em> 2015; <strong>112</strong>:613-620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/bjc.2014.658" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84923587504" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25611303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fbjc.2014.658&amp;pmid=25611303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with several reports having shown cross-regulation between ZEB2 and the actin cytoskeleton (<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Wiles et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Wiles, E.T. ∙ Bell, R. ∙ Thomas, D. ...</div><div><strong>ZEB2 Represses the Epithelial Phenotype and Facilitates Metastasis in Ewing Sarcoma</strong></div><div><em>Genes Cancer.</em> 2013; <strong>4</strong>:486-500</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/1947601913506115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84891542120" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24386509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F1947601913506115&amp;pmid=24386509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110" aria-controls="sref110">Yuan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref110"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Yuan, J. ∙ Chen, L. ∙ Xiao, J. ...</div><div><strong>SHROOM2 inhibits tumor metastasis through RhoA-ROCK pathway-dependent and -independent mechanisms in nasopharyngeal carcinoma</strong></div><div><em>Cell Death Dis.</em> 2019; <strong>10</strong>:58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41419-019-1325-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85060554590" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30683844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41419-019-1325-7&amp;pmid=30683844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our findings add to this and support a link between ZEB2 and cytoskeletal targets with SHROOM3 as a key player (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Chu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Chu, C.W. ∙ Gerstenzang, E. ∙ Ossipova, O. ...</div><div><strong>Lulu regulates Shroom-induced apical constriction during neural tube closure</strong></div><div><em>PLoS ONE.</em> 2013; <strong>8</strong>:e81854</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0081854" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896739280" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24282618/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0081854&amp;pmid=24282618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0235" role="paragraph">Overall, our findings suggest that a relatively simple evolutionary change in cell shape, regulated by the fundamental process of EMT, can have major consequences in brain evolution.</div><section id="sec-3-1"><h3>Limitations of study</h3><div id="p0240" role="paragraph">This study was performed using neural organoids, and like all models, such organoids will never be a perfect representation of the actual brain <i>in&nbsp;vivo</i>. However, because it is impossible to perform experiments on living human or ape embryos, organoids represent an important tool for both comparative studies and functional interrogation. Nonetheless, further examination using <i>in&nbsp;vivo</i> approaches, such as transgenic mice, or even non-invasive imaging of ape embryos would be highly informative in the future.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0250" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>mouse anti-β-actin</td><td>Abcam</td><td>Cat# ab8226, RRID:<a id="intref0010" href="https://antibodyregistry.org/AB_306371" target="_blank">AB_306371</a></td></tr><tr><td>mouse anti-ZEB2</td><td>Origene</td><td>Cat# TA802113, RRID:<a id="intref0015" href="https://antibodyregistry.org/AB_2616296" target="_blank">AB_2616296</a></td></tr><tr><td>mouse anti-CDH2</td><td>BD Biosciences</td><td>Cat# 610920, RRID:<a id="intref0020" href="https://antibodyregistry.org/AB_2077527" target="_blank">AB_2077527</a></td></tr><tr><td>mouse anti-CDH1</td><td>BD Biosciences</td><td>Cat# 610181, RRID:<a id="intref0025" href="https://antibodyregistry.org/AB_397580" target="_blank">AB_397580</a></td></tr><tr><td>rabbit anti-SOX2</td><td>Abcam</td><td>Cat# ab97959, RRID:<a id="intref0030" href="https://antibodyregistry.org/AB_2341193" target="_blank">AB_2341193</a></td></tr><tr><td>sheep anti-TBR2</td><td>R&amp;D Systems</td><td>Cat# AF6166, RRID:<a id="intref0035" href="https://antibodyregistry.org/AB_10569705" target="_blank">AB_10569705</a></td></tr><tr><td>chicken anti-GFP</td><td>Thermo Fisher Scientific</td><td>Cat# A10262, RRID:<a id="intref0040" href="https://antibodyregistry.org/AB_2534023" target="_blank">AB_2534023</a></td></tr><tr><td>rabbit anti-OCLN</td><td>Abcam</td><td>Cat# ab31721, RRID:<a id="intref0045" href="https://antibodyregistry.org/AB_881773" target="_blank">AB_881773</a></td></tr><tr><td>rabbit anti-GLAST</td><td>Abcam</td><td>Cat# ab416, RRID:<a id="intref0050" href="https://antibodyregistry.org/AB_304334" target="_blank">AB_304334</a></td></tr><tr><td>rat anti-CTIP2</td><td>Abcam</td><td>Cat# ab18465, RRID:<a id="intref0055" href="https://antibodyregistry.org/AB_2064130" target="_blank">AB_2064130</a></td></tr><tr><td>mouse anti-HuC/D</td><td>Life Technologies</td><td>Cat# A-21271, RRID:<a id="intref0060" href="https://antibodyregistry.org/AB_221448" target="_blank">AB_221448</a></td></tr><tr><td>rabbit anti-EMX1</td><td>Atlas Antibodies</td><td>Cat# HPA006421, RRID:<a id="intref0065" href="https://antibodyregistry.org/AB_1078739" target="_blank">AB_1078739</a></td></tr><tr><td>rabbit anti-EMX1</td><td>Origene</td><td>TA325087</td></tr><tr><td>rabbit anti-BLBP</td><td>Abcam</td><td>Cat# ab32423, RRID:<a id="intref0070" href="https://antibodyregistry.org/AB_880078" target="_blank">AB_880078</a></td></tr><tr><td>goat anti-DCX</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-8067, RRID:<a id="intref0075" href="https://antibodyregistry.org/AB_2088491" target="_blank">AB_2088491</a></td></tr><tr><td>mouse anti-ZO1</td><td>BD Biosciences</td><td>Cat# 610966, RRID:<a id="intref0080" href="https://antibodyregistry.org/AB_398279" target="_blank">AB_398279</a></td></tr><tr><td>rabbit anti-GFP</td><td>Abcam</td><td>Cat# ab290, RRID:<a id="intref0085" href="https://antibodyregistry.org/AB_303395" target="_blank">AB_303395</a></td></tr><tr><td>rabbit anti-EpCAM</td><td>Abcam</td><td>Cat# ab71916, RRID:<a id="intref0090" href="https://antibodyregistry.org/AB_1603782" target="_blank">AB_1603782</a></td></tr><tr><td>mouse anti-Vimentin (V9)</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-6260, RRID:<a id="intref0095" href="https://antibodyregistry.org/AB_628437" target="_blank">AB_628437</a></td></tr><tr><td>rabbit anti-PAX6</td><td>Abcam</td><td>Cat# ab195045, RRID:<a id="intref0100" href="https://antibodyregistry.org/AB_2750924" target="_blank">AB_2750924</a></td></tr><tr><td>rabbit anti-SHROOM3</td><td>Atlas Antibodies</td><td>Cat# HPA047784, RRID:<a id="intref0105" href="https://antibodyregistry.org/AB_2680155" target="_blank">AB_2680155</a></td></tr><tr><td>mouse anti-GAPDH</td><td>Abcam</td><td>Cat# ab8245, RRID: <a id="interref315" href="https://antibodyregistry.org/AB_2107448" target="_blank">AB_2107448</a></td></tr><tr><td>mouse anti-TUJ1</td><td>Biolegend</td><td>Cat# 801213, RRID: <a id="interref310" href="https://antibodyregistry.org/AB_2728521" target="_blank">AB_2728521</a></td></tr><tr><td>HRP-linked goat anti-rabbit</td><td>Dako</td><td>Cat# P0448, RRID:<a id="intref0110" href="https://antibodyregistry.org/AB_2617138" target="_blank">AB_2617138</a></td></tr><tr><td>HRP-linked rabbit anti-mouse</td><td>Dako</td><td>Cat# P0161, RRID:<a id="intref0115" href="https://antibodyregistry.org/AB_2687969" target="_blank">AB_2687969</a></td></tr><tr><td>Alexa Fluor 594 Phalloidin</td><td>Thermo Fisher Scientific</td><td>Cat# A12381, RRID:<a id="intref0120" href="https://antibodyregistry.org/AB_2315633" target="_blank">AB_2315633</a></td></tr><tr><td>AlexaFluor 488, donkey anti-chicken IgY (IgG) (H+L)</td><td>Jackson ImmunoResearch Labs</td><td>Cat# 703-545-155, RRID:<a id="intref0125" href="https://antibodyregistry.org/AB_2340375" target="_blank">AB_2340375</a></td></tr><tr><td>AlexaFluor 488, 568, 647 donkey anti-rabbit IgG (H+L)</td><td>Life Technologies</td><td>Cat#A32790, RRID: <a id="interref320" href="https://antibodyregistry.org/AB_2762833" target="_blank">AB_2762833</a>; A10042, RRID: <a id="interref325" href="https://antibodyregistry.org/AB_2534017" target="_blank">AB_2534017</a>; A31573, RRID: <a id="interref330" href="https://antibodyregistry.org/AB_2536183" target="_blank">AB_2536183</a></td></tr><tr><td>AlexaFluor 488, 568, 647 donkey anti-mouse IgG (H+L)</td><td>Life Technologies</td><td>Cat#A21202, RRID: <a id="interref335" href="https://antibodyregistry.org/AB_141607" target="_blank">AB_141607</a>; A10037, RRID: <a id="interref340" href="https://antibodyregistry.org/AB_2534013" target="_blank">AB_2534013</a>; A31571, RRID: <a id="interref345" href="https://antibodyregistry.org/AB_162542" target="_blank">AB_162542</a></td></tr><tr><td>AlexaFluor 568, 647 donkey anti-sheep IgG (H+L)</td><td>Life Technologies</td><td>Cat#A21099, RRID: <a id="interref350" href="https://antibodyregistry.org/AB_2535753" target="_blank">AB_2535753</a>; A21448, RRID: <a id="interref355" href="https://antibodyregistry.org/AB_2535865" target="_blank">AB_2535865</a></td></tr><tr class="rowsep"><td>AlexaFluor 488, 568, 647 donkey anti-goat IgG (H+L)</td><td>Life Technologies</td><td>Cat#A11055, RRID: <a id="interref360" href="https://antibodyregistry.org/AB_2534102" target="_blank">AB_2534102</a>; A11057, RRID: <a id="interref365" href="https://antibodyregistry.org/AB_2534104" target="_blank">AB_2534104</a>; A21447, RRID: <a id="interref370" href="https://antibodyregistry.org/AB_2535864" target="_blank">AB_2535864</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr class="rowsep"><td>CytoTune emGFP Sendai fluorescence reporter</td><td>Thermo Fisher Scientific</td><td>Cat#A16519</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Matrigel</td><td>Corning</td><td>Cat#354234</td></tr><tr><td>StemFlex culture media</td><td>Thermo Fisher Scientific</td><td>Cat#A3349401</td></tr><tr><td>Lymphoprep</td><td>Stem Cell Technologies</td><td>Cat#07801</td></tr><tr><td>Ascorbic acid</td><td>Sigma</td><td>Cat#A4403</td></tr><tr><td>SCF</td><td>Miltenyi</td><td>Cat#130-096-692</td></tr><tr><td>IL-3</td><td>Invitrogen</td><td>Cat#PHC0035</td></tr><tr><td>EPO</td><td>R&amp;D systems</td><td>Cat#287-TC-500</td></tr><tr><td>IGF-1</td><td>Miltenyi</td><td>Cat#130-093-885</td></tr><tr><td>Dexamethasone</td><td>Sigma</td><td>Cat# D8893</td></tr><tr><td>iPS Boost Supplements II</td><td>Millipore</td><td>Cat#SCM094</td></tr><tr><td>FBS</td><td>Thermo Fisher Scientific</td><td>Cat#10270106</td></tr><tr><td>TrypLE</td><td>Thermo Fisher Scientific</td><td>Cat#12563029</td></tr><tr><td>Rock Inhibitor Y27632</td><td>Merck</td><td>Cat#688000-5</td></tr><tr><td>Accutase</td><td>Sigma</td><td>Cat#A6964</td></tr><tr><td>DMEM F-12</td><td>Thermo Fisher Scientific</td><td>Cat#11330032</td></tr><tr><td>Neurobasal Medium</td><td>Invitrogen</td><td>Cat#21103049</td></tr><tr><td>MEM-NEEA</td><td>Sigma</td><td>Cat#M7145</td></tr><tr><td>GlutaMAX</td><td>Thermo Fisher Scientific</td><td>Cat#35050038</td></tr><tr><td>Penicillin-Streptomycin</td><td>Sigma</td><td>Cat#P0781</td></tr><tr><td>N2</td><td>Thermo Fisher Scientific</td><td>Cat#17502048</td></tr><tr><td>B27</td><td>Thermo Fisher Scientific</td><td>Cat#17504044</td></tr><tr><td>B27&nbsp;minus vitamin A</td><td>Thermo Fisher Scientific</td><td>Cat#12587010</td></tr><tr><td>β-Mercaptoethanol</td><td>Life Technologies</td><td>Cat#31350-010</td></tr><tr><td>Knockout Serum Replacement</td><td>Thermo Fisher Scientific</td><td>Cat#10828028</td></tr><tr><td>bFGF (FGF2)</td><td>Peprotech</td><td>Cat#100-18B</td></tr><tr><td>Insulin</td><td>Sigma</td><td>Cat#I9278</td></tr><tr><td>Heparin</td><td>Sigma</td><td>Cat#H3149</td></tr><tr><td>CHIR99021</td><td>Tocris</td><td>Cat#4423</td></tr><tr><td>LPA</td><td>Tocris</td><td>Cat#325465-93-8</td></tr><tr><td>BMP4</td><td>R&amp;D Systems</td><td>Cat#314-BP</td></tr><tr><td>Noggin</td><td>R&amp;D Systems</td><td>Cat#6057-NG</td></tr><tr><td>SB-431542</td><td>R&amp;D Systems</td><td>Cat#1614</td></tr><tr><td>Halt Protease inhibitor cocktail</td><td>Thermo Fisher Scientific</td><td>Cat#78430</td></tr><tr><td>PhosSTOP</td><td>Sigma</td><td>Cat#4906845001</td></tr><tr><td>DAPI</td><td>Life Technologies</td><td>Cat#D1306</td></tr><tr><td>Prolong Diamond Antifade mountant</td><td>Invitrogen</td><td>Cat#P36965</td></tr><tr><td>2-methylbutane</td><td>Sigma</td><td>Cat#M32631</td></tr><tr><td>TRIZOL reagent</td><td>Thermo Fisher Scientific</td><td>Cat#15596026</td></tr><tr class="rowsep"><td>DAPI</td><td>Life Technologies</td><td>Cat#D1306</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Cerebral Organoid kit</td><td>Stem Cell Technologies</td><td>Cat#08570, 08571</td></tr><tr><td>Direct-zol-96 RNA kit</td><td>Zymo Research</td><td>Cat#R2055</td></tr><tr><td>NEBNext Ultra II Directional RNA Library Prep Kit for Illumina</td><td>New England Biolabs</td><td>Cat#E7760</td></tr><tr><td>NEBNext Multiplex Oligos for Illumina</td><td>New England Biolabs</td><td>Cat#E7600</td></tr><tr><td>Qubit dsDNA HS Assay Kit</td><td>Thermo Fisher Scientific</td><td>Cat#Q32851</td></tr><tr><td>Quick Start Bradford Dye Reagent</td><td>Bio-Rad</td><td>Cat#5000205</td></tr><tr><td>Amersham Hybond PVDF blotting membrane</td><td>GE Healthcare</td><td>Cat#10600023</td></tr><tr><td>ECL Prime enhanced chemoluminescent detection reagent</td><td>GE Healthcare</td><td>Cat#RPN2232</td></tr><tr><td>SuperSignal West Femto chemoluminescent substrate</td><td>Thermo Fisher Scientific</td><td>Cat#34094</td></tr><tr><td>Human Pluripotent Stem Cell Assessment Primer Pair Panel</td><td>R&amp;D Systems</td><td>Cat#SC012</td></tr><tr><td>GoTaq Green Master Mix</td><td>Promega</td><td>Cat#9PIM712</td></tr><tr><td>Q5 High Fidelity 2X Master Mix</td><td>New England Biolabs</td><td>Cat#M0492S</td></tr><tr><td>ddPCR Supermix for Probes</td><td>Bio-Rad</td><td>Cat#1863024</td></tr><tr><td>SuperScript III first-strand synthesis supermix</td><td>Thermo Fisher Scientific</td><td>Cat#18080400</td></tr><tr><td>Droplet Generation Oil for Probes</td><td>Bio-Rad</td><td>Cat# 1863005</td></tr><tr><td>Human Stem Cell Nucleofector Kit 1 (Lonza, VPH-5012</td><td>Lonza</td><td>Cat#VPH-5012</td></tr><tr class="rowsep"><td>RevitaCell</td><td>Thermo Fisher Scientific</td><td>Cat#A2644501</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Raw and analyzed RNA-sequencing data</td><td>This paper</td><td>NCBI GEO: <a id="intref0130" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE153076" target="_blank">GSE153076</a></td></tr><tr><td>Human reference genome NCBI build 38, GRCh38</td><td>Genome Reference Consortium</td><td><a id="intref0135" href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/" target="_blank">https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/</a></td></tr><tr class="rowsep"><td>Gorilla reference genome, Kamilah_GGO_v0</td><td>University of Washington</td><td><a id="intref0140" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_008122165.1/" target="_blank">https://www.ncbi.nlm.nih.gov/assembly/GCF_008122165.1/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Human: H9 ES</td><td>WiCell</td><td>WA09</td></tr><tr><td>Human: IMR-90 iPS</td><td>WiCell</td><td>iPS(IMR90)-4</td></tr><tr><td>Chimpanzee: Chmp iPS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Gallego Romero et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gallego Romero, I. ∙ Pavlovic, B.J. ∙ Hernando-Herraez, I. ...</div><div><strong>A panel of induced pluripotent stem cells from chimpanzees: a resource for comparative functional genomics</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26102527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07103&amp;pmid=26102527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>C3651</td></tr><tr><td>Gorilla: G1 iPS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref108-1" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-1" aria-controls="sref108-1">Wunderlich et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref108-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Wunderlich, S. ∙ Kircher, M. ∙ Vieth, B. ...</div><div><strong>Primate iPS cells as tools for evolutionary analyses</strong></div><div><em>Stem Cell Res. (Amst.).</em> 2014; <strong>12</strong>:622-629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2014.02.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896051735" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24631741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2014.02.001&amp;pmid=24631741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>goiPSC clone 1</td></tr><tr><td>Gorilla: G2 iPS</td><td>This paper</td><td>N/A</td></tr><tr><td>Human: H9 <i>ZEB2</i><sup><i>+/−</i></sup> ES</td><td>This paper</td><td>N/A</td></tr><tr><td>Human: H9 <i>ZEB2</i><sup><i>+/−</i></sup><i>; iZEB2</i> ES</td><td>This paper</td><td>N/A</td></tr><tr><td>Human: Hum<sup><i>iZEB2</i></sup> (H9<sup><i>iZEB2</i></sup><i>)</i> ES</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>Mouse: MEF feeders</td><td>GIBCO</td><td>Cat#A34181</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>See <a id="crosref1230" href="#mmc1">Table S1</a></td><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>AAVS1-Neo-TRE-CMV-Cre-rtTA</td><td>This paper</td><td>RRID: Addgene_165457</td></tr><tr><td>AAVS1-Puro-CAG-fl-STOP-fl-ZEB2-GFP-Flag-WPRE-poly(A)</td><td>This paper</td><td>RRID: Addgene_165456</td></tr><tr><td>pT2-CAG-fGFP</td><td>Addgene</td><td>RRID: Addgene_108714</td></tr><tr><td>Puro-Cas9 donor</td><td>Addgene</td><td>RRID: Addgene_58409</td></tr><tr><td>pAAV-Neo_CAG-Cas9</td><td>Addgene</td><td>RRID: Addgene_86698</td></tr><tr><td>AAVS1-Neo-M2rtTA</td><td>Addgene</td><td>RRID: Addgene_60843</td></tr><tr><td>pCAG-Cre</td><td>Addgene</td><td>RRID: Addgene_13775</td></tr><tr><td>pSpCas9(BB)-2A-GFP (PX458)</td><td>Addgene</td><td>RRID: Addgene_48138</td></tr><tr><td>pENTRY221-ZEB2 ORF</td><td>Thermo Fisher Scientific</td><td>Ultimate ORF clone number: IOH53645</td></tr><tr class="rowsep"><td>pSpCas9n (BB) (PX460)</td><td>Addgene</td><td>RRID: Addgene_48873</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Fiji</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (39955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0145" href="https://imagej.net/Fiji" target="_blank">https://imagej.net/Fiji</a></td></tr><tr><td>Prism v8.4.2</td><td>GraphPad</td><td><a id="intref0150" href="https://www.graphpad.com/" target="_blank">https://www.graphpad.com/</a></td></tr><tr><td>R v3.5.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">R Core Team, 2018</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2018</div><div><a id="intref0300" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2018R+Foundation+for+Statistical+ComputingVienna%2C+Austriahttps%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="http://www.R-project.org/" target="_blank">http://www.R-project.org/</a></td></tr><tr><td>MATLAB R2019a</td><td>Mathworks</td><td><a id="intref0160" href="https://www.mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a></td></tr><tr><td>Imaris v9.2</td><td>Oxford Instruments</td><td><a id="intref0165" href="https://imaris.oxinst.com/" target="_blank">https://imaris.oxinst.com/</a></td></tr><tr><td>MaMuT v0.27.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Wolff et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Wolff, C. ∙ Tinevez, J.Y. ∙ Pietzsch, T. ...</div><div><strong>Multi-view light-sheet imaging and tracking with the MaMuT software reveals the cell lineage of a direct developing arthropod limb</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e34410</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.34410" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85053006324" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29595475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.34410&amp;pmid=29595475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0170" href="https://imagej.net/MaMuT" target="_blank">https://imagej.net/MaMuT</a></td></tr><tr><td>PRAGUI</td><td>MRC LMB</td><td><a id="intref0175" href="https://github.com/lmb-seq/PRAGUI" target="_blank">https://github.com/lmb-seq/PRAGUI</a></td></tr><tr><td>Trim Galore! v0.6.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Krueger, 2012</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Krueger, F.</div><div><strong>Trim Galore! v0.4.0</strong></div><div>2012</div><div><a id="intref0295" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+Galore%21+v0.4.02012https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0180" href="https://github.com/FelixKrueger/TrimGalore/releases" target="_blank">https://github.com/FelixKrueger/TrimGalore/releases</a></td></tr><tr><td>Cutadapt v2.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Martin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnet. J.</em> 2011; <strong>17</strong>:10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.14806%2Fej.17.1.200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0185" href="https://cutadapt.readthedocs.io/en/stable/" target="_blank">https://cutadapt.readthedocs.io/en/stable/</a></td></tr><tr><td>FASTQC v0.11.5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrews, S.</div><div><strong>FastQC: a quality control tool for high throughput sequence data</strong></div><div>2010</div><div><a id="intref0290" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+a+quality+control+tool+for+high+throughput+sequence+data2010https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0190" href="https://github.com/s-andrews/FastQC" target="_blank">https://github.com/s-andrews/FastQC</a></td></tr><tr><td>HISAT2 v2.0.0-beta</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">Kim et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kim, D. ∙ Langmead, B. ∙ Salzberg, S.L.</div><div><strong>HISAT: a fast spliced aligner with low memory requirements</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:357-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84926519013" target="_blank" aria-label="Open Scopus in new tab">Scopus (13753)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3317&amp;pmid=25751142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0195" href="http://daehwankimlab.github.io/hisat2/" target="_blank">http://daehwankimlab.github.io/hisat2/</a></td></tr><tr><td>HTSeq v0.11.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Anders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Anders, S. ∙ Pyl, P.T. ∙ Huber, W.</div><div><strong>HTSeq--a Python framework to work with high-throughput sequencing data</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:166-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84928987900" target="_blank" aria-label="Open Scopus in new tab">Scopus (13500)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu638&amp;pmid=25260700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0200" href="https://htseq.readthedocs.io/en/master/" target="_blank">https://htseq.readthedocs.io/en/master/</a></td></tr><tr><td>g:Profiler</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Reimand et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Reimand, J. ∙ Kull, M. ∙ Peterson, H. ...</div><div><strong>g:Profiler--a web-based toolset for functional profiling of gene lists from large-scale experiments</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:W193-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkm226" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34547571030" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17478515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkm226&amp;pmid=17478515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0205" href="https://biit.cs.ut.ee/gprofiler/gost" target="_blank">https://biit.cs.ut.ee/gprofiler/gost</a></td></tr><tr><td>TCseq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref107-2" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-2" aria-controls="sref107-2">Wu and Gu, 2020</a><div class="dropBlock__holder" data-db-target-of="sref107-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Wu, M. ∙ Gu, L.</div><div><strong>TCseq: Time course sequencing data analysis. R package version 1.12.0</strong></div><div>2020</div><div><a id="intref0305" href="https://www.bioconductor.org/packages/release/bioc/html/TCseq.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/TCseq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.WuL.GuTCseq%3A+Time+course+sequencing+data+analysis.+R+package+version+1.12.02020https%3A%2F%2Fwww.bioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FTCseq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0210" href="https://rdrr.io/bioc/TCseq/f/inst/doc/TCseq.pdf" target="_blank">https://rdrr.io/bioc/TCseq/f/inst/doc/TCseq.pdf</a></td></tr><tr><td>TBR2+ cell counter</td><td>This paper</td><td><a id="intref0215" href="https://github.com/StefanoLG92/FIJI-macros" target="_blank">https://github.com/StefanoLG92/FIJI-macros</a></td></tr><tr><td>Organoid lumen segmentation</td><td>This paper</td><td><a id="intref0220" href="https://github.com/esriis/organoid-lumen-segmentation" target="_blank">https://github.com/esriis/organoid-lumen-segmentation</a></td></tr><tr><td>QuantaSoft Software</td><td>Bio-Rad</td><td><a id="intref0225" href="https://www.bio-rad.com/" target="_blank">https://www.bio-rad.com/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900239-7" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0255" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Madeline Lancaster (<a id="intref0230" href="mailto:madeline.lancaster@mrc-lmb.cam.ac.uk">madeline.lancaster@mrc-lmb.cam.ac.uk</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0260" role="paragraph">All cell lines, except Gorilla gorC1 iPS cells, were previously published or available from commercial sources. gorC1 cells are available from the Lead Contact subject to institutional MTA and CITES regulations. Newly generated plasmid constructs AAVS1-Neo-TRE-CMV-Cre-rtTA and AAVS1-Puro-CAG-fl-STOP-fl-ZEB2-GFP-Flag-WPRE-poly(A) are available through Addgene (165457 and 165456, respectively).</div></section><section id="sec-4-2-3"><h4>Data and code availability</h4><div id="p0265" role="paragraph">The accession number for the RNaseq data reported in this paper is NCBI GEO: <a id="intref0235" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE153076" target="_blank">GSE153076</a>. The custom code used for image analysis and lumen segmentation is publicly available at <a id="intref0240" href="https://github.com/esriis/organoid-lumen-segmentation" target="_blank">https://github.com/esriis/organoid-lumen-segmentation</a>. The custom code used for segmentation and counting of TBR2+ cells is publicly available at (<a id="intref0245" href="https://github.com/StefanoLG92/FIJI-macros" target="_blank">https://github.com/StefanoLG92/FIJI-macros</a>).</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Cell lines</h4><div id="p0270" role="paragraph">One human ESC line (H9), one human iPSC line (iPS(IMR90)-4, shortened to IMR-90 in this study), one chimpanzee iPSC line (C3651, shortened to Chmp) and two gorilla iPSC lines (goiPSC clone 1 and gorC1, shortened to G1 and G2 respectively) were used in this study. All cell lines were female. H9 (WA09) and iPS(IMR90)-4 were purchased from WiCell. C3651 was a gift from Yoav Gilad (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Gallego Romero et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gallego Romero, I. ∙ Pavlovic, B.J. ∙ Hernando-Herraez, I. ...</div><div><strong>A panel of induced pluripotent stem cells from chimpanzees: a resource for comparative functional genomics</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26102527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07103&amp;pmid=26102527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). goiPSC clone 1 was previously published (<span class="dropBlock reference-citations"><a id="body-ref-sref108-2" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-2" aria-controls="sref108-2">Wunderlich et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref108-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Wunderlich, S. ∙ Kircher, M. ∙ Vieth, B. ...</div><div><strong>Primate iPS cells as tools for evolutionary analyses</strong></div><div><em>Stem Cell Res. (Amst.).</em> 2014; <strong>12</strong>:622-629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2014.02.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896051735" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24631741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2014.02.001&amp;pmid=24631741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We generated gorC1 using peripheral blood mononuclear cells (PBMCs) isolated from leftover blood collected from a female western lowland gorilla during a routine check-up at Twycross zoo. Briefly, PBMCs were isolated using Lymphoprep (Stem Cell Technologies, 07801), followed by expansion for 9&nbsp;days in Expansion Media composed of Stemspan H3000 (Stem Cell Technologies, 09850) supplemented with 50&nbsp;μg/ml ascorbic acid (Sigma, A4403), 50&nbsp;ng/ml SCF (Miltenyi, 130-096-692), 10&nbsp;ng/ml IL-3 (Invitrogen, PHC0035), 2&nbsp;U/ml EPO (R&amp;D Systems, 287-TC-500), 40&nbsp;ng/ml IGF-1 (Miltenyi, 130-093-885), 1&nbsp;μM Dexamethasone (Sigma, D8893-1MG). On day 10, expanded PBMCs were transduced with non-integrating Sendai reprogramming virus (Cytotune 2.0, Thermo Fisher, A16517) according to manufacturer’s instructions added to the Expansion Media. The next day, the media was changed and two days later the cells were collected and plated on gamma irradiated MEF feeders (GIBCO, A34181) in human ES media (DMEM/F12 (Invitrogen, 11330-032) with 20% knockout serum replacement (Invitrogen, 10828-028), 1:100 Glutamax (Invitrogen 35050-038), 1:100 NEAA (Sigma, M7145), 3.5&nbsp;μl/500ml 2-mercaptoethanol, 20&nbsp;ng/ml bFGF (Peprotech, 100-18B)) supplemented with ascorbic acid, SCF, IL-3, EPO, IGF-1, and Dexamethasone at the above concentrations along with iPS Boost Supplements II (Millipore, SCM094). Two days later (5&nbsp;days after induction), media was changed to human ES media without PBMC supplements but with Boost Supplements. Cells were maintained in hES&nbsp;+ Boost until colonies began to form (10&nbsp;days after induction). The small colonies were then split off MEFs 15&nbsp;days after induction using collagenase and a cell lifter and replated on Matrigel (Corning) coated dishes in StemFlex (Thermo Fisher, A334901) for further selection and propagation. Human ESCs used in this project were approved for use in this project by the UK Stem Cell Bank Steering Committee and iPSCs and ESCs were approved by an ERC ethics committee and are registered on the Human Pluripotent Stem Cell Registry (hpscreg.eu). Nonhuman primate cells were approved by the Animal Welfare and Ethical Review Body (AWERB) of the MRC-LMB. RT-PCR (refer to the <i>‘</i>PCR analysis’ section of the methods) confirmed pluripotent gene expression (<a id="crosref1295" href="#figs1">Figure&nbsp;S1</a>F). All cell lines were maintained in StemFlex on Matrigel coated plates. Cells were passaged every 3-4&nbsp;days using 0.7&nbsp;mM EDTA. HEK293 cells were a gift from Dr Harvey McMahon and were cultured in high-glucose DMEM supplemented with GlutaMAX (Thermo Fisher, 31966047) and 10% FBS (Thermo Fisher, 10270106) and passaged once a week using TrypLE (Thermo Fisher, 12563029).</div><div id="p0275" role="paragraph">Commercial cell lines (H9 and iPS(IMR90)-4) were authenticated by the provider. Nonhuman primate cells were not authenticated, but were established by the authors, apart from C3651.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Plasmid Constructs</h4><div id="p0280" role="paragraph">All oligos used for cloning are listed in <a id="crosref1300" href="#mmc1">Table S1</a>. All PCR amplification steps were done using Q5 polymerase (NEB, M049L) and TOP10 chemically competent <i>E.&nbsp;coli</i> (Thermo Fisher, C404010) were used for transformation. In order to generate the construct pT2-CAG-fl-STOP-fl-fGFP, a fl-STOP-fl cassette was ligated into the pT2-CAG-fGFP (Addgene plasmid #108714) at EcoRI and Ascl restriction sites. The construct AAVS1-Puro-CAG-fl-STOP-fl-Cas9 was generated as follows; the backbone was amplified from the Puro-Cas9 donor plasmid (a gift from Dr Danwei Huangfu, Addgene plasmid #58409) (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">González et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>González, F. ∙ Zhu, Z. ∙ Shi, Z.D. ...</div><div><strong>An iCRISPR platform for rapid, multiplexable, and inducible genome editing in human pluripotent stem cells</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>15</strong>:215-226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2014.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2014.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84905905141" target="_blank" aria-label="Open Scopus in new tab">Scopus (368)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24931489/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2014.05.018&amp;pmid=24931489" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with primers AAVS1_CAG_fl_STOP_fl_F &amp; R to introduce MluI (NEB, R3198S) and KpnI (NEB, R3142S) restriction sites and blunt-end ligated to the CAG-fl-STOP-fl cassette, excised from the construct pT2-CAG-fl-STOP-fl-fGFP with restriction enzymes AflI (NEB, R0520S) and AscI (NEB, R0558S) and blunted with the Quick Blunting Kit (NEB, E1201). The resulting intermediate plasmid was then digested with MluI and KpnI and ligated to the cassette containing Cas9 and the bGH-poly(A) sequence amplified from pAAV-Neo_CAG-Cas9 (a gift from Dr Ludovic Vallier, Addgene plasmid #86698) (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Bertero et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bertero, A. ∙ Pawlowski, M. ∙ Ortmann, D. ...</div><div><strong>Optimized inducible shRNA and CRISPR/Cas9 platforms for in&nbsp;vitro studies of human development using hPSCs</strong></div><div><em>Development.</em> 2016; <strong>143</strong>:4405-4418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.138081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85002083552" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.138081&amp;pmid=27899508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with primers Cas9_β_globin_pA_F &amp; R. All ligations to generate this construct were done with the Quick Ligation Kit (NEB, M2200S) and the final construct was sequence validated. The construct AAVS1-Neo-TRE-CMV-CRE-rtTA was generated as follows; the tight TRE promoter was amplified from the Puro-Cas9 donor plasmid (Addgene plasmid #58409) with primers TRE_F &amp;R and inserted into the EcoRV restriction site of AAVS1-Neo-M2rtTA (a gift from Dr Rudolf Jaenisch, Addgene plasmid #60843) (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">DeKelver et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>DeKelver, R.C. ∙ Choi, V.M. ∙ Moehle, E.A. ...</div><div><strong>Functional genomics, proteomics, and regulatory DNA analysis in isogenic settings using zinc finger nuclease-driven transgenesis into a safe harbor locus in the human genome</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1133-1142</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.106773.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77955157281" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20508142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.106773.110&amp;pmid=20508142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to produce the intermediate construct AAVS1-Neo-M2rtTA-TRE. A fragment containing CMV-2TO-MCS-bGH-poly(A) was PCR amplified from pcDNA4/TO (Thermo Fisher, V102020) using primers CMV_bGH_F and _R, and inserted into the EcoRV restriction site of AAVS1-Neo-M2rtTA. The fragment comprised between the SalI and AleI restriction sites was replaced with the Cre recombinase cDNA amplified from pCAG-Cre (a gift from Dr Connie Cepko, Addgene plasmid #13775) (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Matsuda and Cepko, 2007</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Matsuda, T. ∙ Cepko, C.L.</div><div><strong>Controlled expression of transgenes introduced by in&nbsp;vivo electroporation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:1027-1032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0610155104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33846470670" target="_blank" aria-label="Open Scopus in new tab">Scopus (505)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17209010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0610155104&amp;pmid=17209010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using primers Cre_SalI_F and Cre_KpnI_R. From the resulting plasmid a fragment containing CRE-bGH-poly(A) was excised with restriction enzymes SalI (NEB, R3138S) and AleI (NEB, R0634) and inserted into AAVS1-Neo-M2rtTA-TRE digested with the same restriction enzymes. All ligations to generate this construct were done with the Quick Ligation Kit (NEB, M2200S). The final construct was sequence validated. The PX458-AAVS1 construct used to target the AAVS1 locus was generated by cloning the AAVS1_sgRNA sequence into pSpCas9(BB)-2A-GFP (PX458) (a gift from Dr Fang Zhang, Addgene plasmid #48138) as previously described by <span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Ran et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (7783)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. In order to generate the construct for inducible <i>ZEB2</i> expression from the AAVS1 site the coding sequence of the human <i>ZEB2</i> transcript variant 1 (NM_014795.3) was purchased as an ultimate ORF clone in the pENTRY221 (Thermo Scientific, #IOH53645). The <i>ZEB2</i> ORF was amplified with primers ZEB2_IndOE_F &amp; _R and tagged with GFP-Flag by Gibson assembly into the pT2-CAG-MCS-2A-Puro plasmid linearized by restriction digestion with EcoRI (NEB, R3101S) and AgeI (NEB, R3552S). The GFP fragment was amplified and C-terminally Flag-tagged using primers GFP_IndOE_F &amp; R and the construct pT2-CAG-fGFP (Addgene plasmid #108714) as template. While the original design aimed to fuse ZEB2 to GFP via a 4x(GGGGS)-linker motif, Gibson assembly resulted in motif duplication and produced a 6x(GGGGS)-linker between the two coding sequences. The resulting plasmid, pT2-CAG-ZEB2-GFP-Flag-2A-Puro, was used as template in a PCR reaction with primers ZEB2_GFP_Flag_IndOE_F &amp; _R. The resulting fragment encoding ZEB2-GFP-Flag was then digested with restriction enzymes AgeI (NEB, R3552S) and KpnI (NEB, R3142S) and ligated into the CAGs-MCS-Enhanced Episomal Vector (EEV) (System Biosciences, EEV600A-1) linearized with AgeI and KpnI, using T4 ligase. Next, using primers ZEB2_AAVS1 IndOE_F &amp;_R a fragment containing ZEB2-GFP-Flag-WPRE-poly(A) and overhangs containing MluI and FseI restriction sites at the 5′ and 3′ end, respectively, was generated. The construct AAVS1-Puro-CAG-fl-STOP-fl-Cas9 was linearized with MluI (NEB, R3198S) and FseI (NEB, R0588S) to remove the Cas9 coding sequence, which was then replaced with the fragment encoding ZEB2-GFP-Flag-WPRE-poly(A) by ligation with T4 ligase. The <i>ZEB2</i> ORF was verified after every cloning step by sequencing using primers ZEB2_Seq_1-9. For simplicity, this overexpressing construct is referred to as ZEB2-GFP throughout the text. For CRISPR-Cas9 knockout of <i>ZEB2</i> in Human ESCs guide RNAs were designed using the online tool from the lab of Feng Zhang and the sequences are listed in <a id="crosref1330" href="#mmc1">Table S1</a> as ZEB2<i>_</i>sgRNA_1 &amp; _2. Cloning of the guides was performed as outlined by <span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Ran et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (7783)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; briefly, the sense and antisense strand oligos for ZEB2_sgRNA_1 &amp; 2 were annealed and phosphorylated, and the duplexes were cloned into pSpCas9n (BB) (PX460) and pSpCas9(BB)-2A-GFP (PX458) (a gift from Dr Fang Zhang, Addgene plasmid #48873 and #48138, respectively) by BbsI (NEB,R3539S) digestion and ligation with T4 ligase. Colonies were sequence validated using the U6_F primer.</div></section><section id="sec-4-4-2"><h4>Transgenic cell lines</h4><div id="p0285" role="paragraph">For establishment of the H9 <i>ZEB2</i><sup><i>+/−</i></sup> line, plasmids pSpCas9n(BB)-<i>ZEB2</i>-Guide-A &amp;-B (0.5&nbsp;μg of each plasmid) were electroporated into 1x10<sup>6</sup> H9 cells using the Human Stem Cell Nucleofector Kit 1 (Lonza, VPH-5012). Following electroporation, cells were grown in one well of a 24-well plate, reduced to a single-cell suspension and seeded into a 96-well plate at a density ranging between 1000 – 20 c/w in mTesR™1 supplemented with 1&nbsp;nM ROCK inhibitor (BD Biosciences, 562822). Alternatively, cells were seeded at a density of 0.5 c/w in StemFlex supplemented with RevitaCell supplement (Thermo Fisher, A2644501). Once the cells reached ∼80% confluence the 96-well plate was split to two replica plates, one used for screening by ddPCR and the other used for further expansion. Mutant screening relied on a droplet digital PCR (ddPCR) drop-off assay (<span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Findlay et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Findlay, S.D. ∙ Vincent, K.M. ∙ Berman, J.R. ...</div><div><strong>A digital pcr-based method for efficient and highly specific screening of genome edited cells</strong></div><div><em>PLoS ONE.</em> 2016; <strong>11</strong>:e0153901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0153901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84977260192" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27089539/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0153901&amp;pmid=27089539" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Screening was performed as described in the <i>‘</i>Digital droplet PCR (ddPCR)’ section of the methods. The mutant colonies were characterized by sequencing following cloning of the edit-sites into the pJET1.2 vector (using primers ZEB2_Cas9screening_F &amp; _R), PCR followed by DNA-PAGE (using primers ZEB2_DNAPAGE_ F &amp; _R), karyotype analyses (outsourced to Cell Guidance Systems) and RT-PCR of pluripotency markers (details in the ‘PCR analysis’ section of the methods).</div><div id="p0290" role="paragraph">For establishment of the H9 doxycycline-inducible ZEB2 line (shortened to Hum<sup><i>iZEB2</i></sup>) and the <i>ZEB2</i><sup><i>+/−</i></sup><i>; iZEB2</i> rescue line, plasmids AAVS1-Neo-TRE-CMV-Cre-rtTA (2&nbsp;μg), AAVS1-Puro-CAG-fl-STOP-fl-ZEB2-GFP-Flag-WPRE-poly(A) (2&nbsp;μg) and PX458-AAVS1 (3&nbsp;μg) were electroporated into 1.3x10<sup>6</sup> H9 cells or H9 <i>ZEB2</i><sup><i>+/−</i></sup> cells using the Human Stem Cell Nucleofector Kit 1 (Lonza, VPH-5012). Following electroporation the cells were seeded across 3-wells of a 6 well plate coated with Matrigel in StemFlex supplemented with RevitaCell supplement (Thermo Fisher, A2644501). After 3&nbsp;days from electroporation the cells were selected with StemFlex supplemented with G418 (25&nbsp;μg/ml) and Puromycin (0.5&nbsp;μg/ml). After approximately two weeks in selection medium, single colonies were picked and screened using three primers pair combinations; AAVS1_F &amp; _R (AMP_1 in <a id="crosref1345" href="#figs6">Figure&nbsp;S6</a>), AAVS1_F &amp; Puro_R (AMP_2 in <a id="crosref1350" href="#figs6">Figure&nbsp;S6</a>), AAVS1_F &amp; Neo_R (AMP_3 in <a id="crosref1355" href="#figs6">Figure&nbsp;S6</a>), listed in <a id="crosref1360" href="#mmc1">Table S1</a>. Individual colonies of the Hum<sup><i>iZEB2</i></sup> were tested for <i>ZEB2-GFP</i> transgene induction by western blot and immunofluorescence following treatment with StemFlex supplemented with 1.7&nbsp;μg/ml Doxycyline for 6&nbsp;days.</div></section><section id="sec-4-4-3"><h4>Cerebral organoid generation</h4><div id="p0295" role="paragraph">Cerebral organoids with telencephalic identity from human and ape cells (all female) were generated using a previously described modified protocol (<span class="dropBlock reference-citations"><a id="body-ref-optEWDeCX5gW6" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optEWDeCX5gW6" aria-controls="optEWDeCX5gW6">Lancaster et al., 2017</a><div class="dropBlock__holder" data-db-target-of="optEWDeCX5gW6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Lancaster, M.A. ∙ Corsini, N.S. ∙ Wolfinger, S. ...</div><div><strong>Guided self-organization and cortical plate formation in human brain organoids</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:659-666</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85020668535" target="_blank" aria-label="Open Scopus in new tab">Scopus (531)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28562594/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3906&amp;pmid=28562594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to generate EBs with increased surface area:volume either through the use of fibrous microscaffolds or fewer cell numbers to generate smaller EBs. For the smaller EB approach, Aggrewell 800 plates (StemCell Technologies, 34815) containing 300 microwells, each 800&nbsp;μm in size, per well were used to reproducibly generate organoids in multiples of 300. Following previously described directions for EB generation in Aggrewell plates (StemCell Technologies Document #DX21397), 6&nbsp;× 10<sup>5</sup> cells in 1.5&nbsp;mL media were plated per well, i.e., 2,000 cells per EB. STEMdiff Cerebral Organoid Kit (StemCell Technologies, 08570) was used for organoid culture, with the addition of 1:1000 Fungizone and 1:100 Penicillin-streptomycin to avoid contamination risk associated with repeated plate usage. The timing of the protocol (StemCell Technologies Document #DX21849) was followed, with the exception of chimpanzee EBs being moved to neural induction media 2&nbsp;days earlier than described. Organoids were fed every 2&nbsp;days during the Aggrewell period prior to Matrigel embedding, using an electronic pipette set to the slowest speed in order to avoid flushing EBs out of microwells. Matrigel embedding was based on a previously described method (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Qian et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Qian, X. ∙ Jacob, F. ∙ Song, M.M. ...</div><div><strong>Generation of human brain region-specific organoids using a miniaturized spinning bioreactor</strong></div><div><em>Nat. Protoc.</em> 2018; <strong>13</strong>:565-580</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2017.152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047385267" target="_blank" aria-label="Open Scopus in new tab">Scopus (331)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29470464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2017.152&amp;pmid=29470464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with minor modifications. Briefly, organoids were removed from their well, resuspended in 300&nbsp;μL Expansion media and split into 3 tubes where 150&nbsp;μL Matrigel was added resulting in a 3:2 Matrigel dilution. The contents of each tube were then spread into a well of an ultra-low attachment 24-well plate (Corning, 7524). After a 30-minute incubation at 37°C, 1&nbsp;mL expansion media was added per well. The fibrous microscaffold approach and in-house media was used to generate <i>ZEB2</i><sup><i>+/−</i></sup>, <i>ZEB2</i><sup><i>+/−</i></sup><i>; iZEB2</i>, and control. All comparisons between samples and treatments were performed on organoids generated using identical protocols.</div><div id="p0300" role="paragraph">To achieve sparse labeling of neural progenitor cells, 5&nbsp;μL CytoTune emGFP Sendai fluorescence reporter (Thermo Fisher, A16519) was added to Aggrewells when the organoids were switched to neural induction medium. For ZEB2 overexpression, organoids generated from the H9 doxycycline-inducible ZEB2 (Hum<sup><i>iZEB2</i></sup>) cell line were treated with 1.7&nbsp;μg/ml doxycycline (Merck, PHR1145) during the first 5&nbsp;days of the protocol. For ZEB2 rescue, <i>ZEB2</i><sup><i>+/−</i></sup> <i>;iZEB2</i> organoids were treated with 1.7&nbsp;μg/ml doxycycline from day 2 until day 7. For the pharmacological rescue of <i>ZEB2</i><sup><i>+/−</i></sup> organoids, starting from day 2, after embedding, <i>ZEB2</i><sup><i>+/−</i></sup> and WT control organoids were treated with 0.5&nbsp;μg/ml Noggin (R&amp;D, 6057-NG) and 10&nbsp;μM SB-431542 (R&amp;D, 1614) for 10&nbsp;days and samples were collected for biochemical and histological analysis on day 12. For treatment with LPA, 3&nbsp;μM LPA (Tocris, 325465-93-8) was added to the media of aggrewell organoids 1&nbsp;day after Matrigel-embedding (3&nbsp;days post neural induction, using the STEMdiff Cerebral Organoid Kit). Media was changed 2x daily, as previous studies have reported that LPA activity in culture medium decreases after 10 hours (<span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Medelnik et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Medelnik, J.P. ∙ Roensch, K. ∙ Okawa, S. ...</div><div><strong>Signaling-Dependent Control of Apical Membrane Size and Self-Renewal in Rosette-Stage Human Neuroepithelial Stem Cells</strong></div><div><em>Stem Cell Reports.</em> 2018; <strong>10</strong>:1751-1765</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85046875468" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.04.018&amp;pmid=29779899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and organoids were fixed for analysis after 48 hours of treatment (5&nbsp;days post neural induction). For treatment with BMP4, 500ng/ml BMP4 (R&amp;D, 314-BP) was added to the media of aggrewell organoids 1&nbsp;day post neural induction and media was changed daily until organoids were fixed for analysis on day 5 post neural induction.</div></section><section id="sec-4-4-4"><h4>Histological and immunohistochemical analysis</h4><div id="p0305" role="paragraph">Organoids were fixed in 4% PFA either overnight at 4°C or at room temperature for 20-60&nbsp;min, and washed in PBS (3x10&nbsp;min). Samples for cryostat processing were incubated overnight in 30% sucrose in 0.2&nbsp;M PB (21.8 g/l Na<sub>2</sub>HPO<sub>4</sub>, 6.4 g/l NaH<sub>2</sub>PO4 in dH<sub>2</sub>O), embedded in gelatin (7.5% gelatin, 10% sucrose in 0.2&nbsp;M PB), plunge frozen in 2-methylbutane (Sigma-Aldrich, M32631) at ∼-40°C for 1-2&nbsp;minutes and cryosectioned at a thickness of 20&nbsp;μm. Both sectioned and whole mount samples were stained as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Lancaster et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lancaster, M.A. ∙ Renner, M. ∙ Martin, C.A. ...</div><div><strong>Cerebral organoids model human brain development and microcephaly</strong></div><div><em>Nature.</em> 2013; <strong>501</strong>:373-379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12517" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84884414984" target="_blank" aria-label="Open Scopus in new tab">Scopus (3474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23995685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12517&amp;pmid=23995685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For histological analysis, sections were stained for hematoxylin and eosin (Carl Roth, 9194.1) followed by dehydration in ethanol and xylene and mounting in permount mounting media.</div></section><section id="sec-4-4-5"><h4>Imaging and image analysis</h4><div id="p0310" role="paragraph">Confocal images of fixed cryosectioned samples were obtained with a Zeiss LSM 780 or 880 confocal microscope. Images of fixed whole mount samples were obtained with a Zeiss LSM 780 confocal microscope or a Nikon CSU-W1 spinning disk microscope. Whole mount organoids were imaged at 8&nbsp;μm intervals on Zeiss LSM 780 while the speed of the Nikon CSU-W1 system allowed for higher resolution imaging at 0.5&nbsp;μm intervals. To calculate the apical luminal surface area of organoids, folders containing Zeiss LSM 780 Z stacks of ZO1 stained organoids as an image stack were carried over to MATLAB (Mathworks). The largest lumen per organoids was segmented using a tailor-made code that delineates the perimeter of the lumen in each Z stack by detecting fluorescent intensity and then combines the delineated perimeters into a 3D structure from which the surface area is calculated. This code is publicly available (<a id="intref0255" href="https://github.com/esriis/organoid-lumen-segmentation" target="_blank">https://github.com/esriis/organoid-lumen-segmentation</a>). The apical surface area of individual cells was measured from cryosections of organoids where cells were selected based on SOX2+ staining and sectioning angle being at the apical surface of progenitors. ZO1 stain was used as a marker of apical cell perimeters and Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (39955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to manually delineate perimeters and perform apical surface area measurements. Because of differences in tissue shrinkage during PFA fixation and handling, comparisons were performed with controls fixed and stained at the same time and treated identically. TBR2 positive cells (<a id="crosref1385" href="#figs6">Figure&nbsp;S6</a>T) were counted using a FIJI plug-in generated by Johannes Schindelin. For quantification of TBR2+ cell numbers per unit area in ZEB2-rescue and day 55 organoids, a custom Fiji macro was used to count the number of TBR2+ cells, this was then divided by the corresponding area. This macro is publicly available (<a id="intref0260" href="https://github.com/StefanoLG92/FIJI-macros" target="_blank">https://github.com/StefanoLG92/FIJI-macros</a>). Distance of nuclei as a percentage of VZ length was performed in Fiji on cryosections of organoids stained with DAPI and ZO1, where ZO1 staining indicated the sectioning angle was perpendicular to the apical surface of neuroepithelial tissue. The distance from the ZO1+ apical surface to the center of nuclei was measured and reported as a percentage of the total apico-basal thickness of the neuroepithelial tissue. Total thickness was measured from ZO1+ apical surface to the basal surface as determined by background low-level fluorescence from the tissue (average of 3 total thickness measurements). To segment and quantify apical processes of individual neural progenitor cells, the surfaces option in Imaris (Oxford Instruments, v9.2) was used to manually segment virally-labeled GFP+ cells that were clearly separated from neighboring labeled cells using images of whole mount organoids obtained on a Nikon CSU-W1 spinning disk microscope. To capture differences in apical process thickness, apical process volumes were derived by cutting segmented cells apical to the cell body (‘cut surface’ in Imaris) and obtaining the volume of the isolated apical processes in Imaris. Because volume can be influenced by topology (i.e., curvature, length), in ways that do not necessarily reflect thickness (diameter), we normalized volume measurements to surface area to account for these topological effects.</div><div id="p0315" role="paragraph">For quantification of organoid size and neuroepithelial bud perimeter, brightfield images of live organoids were taken on an EVOS (Thermo Fisher, AMF4300). Organoid area was measured by manually delineating the perimeter of individual organoids in Fiji and measuring the area within these structures. Neuroepithelial bud perimeters were obtained by manually delineating the visible basal surface of neuroepithelial buds projecting from the main organoid mass and measuring their lengths.</div></section><section id="sec-4-4-6"><h4>Immunoblotting</h4><div id="p0320" role="paragraph">Cell and organoid samples were washed twice in ice-cold PBS, pelleted by centrifugation (500 g, 3&nbsp;min) and lysed with modified-RIPA (mRIPA: 1% Triton-X, 0.1% SDS, 150&nbsp;mM NaCl, 50&nbsp;mM Tris pH 7.4, 2&nbsp;mM EDTA, 12&nbsp;mM sodium deoxycholate) supplemented immediately prior to lysis with protease (Thermo Fisher, 78430) and phosphatase (Sigma-Aldrich, 4906845001) inhibitors. The protein concentration of the samples was measured using the Quick Start Bradford Dye Reagent (Bio-Rad, 5000205). Between 3-20&nbsp;μg of total protein per sample were resolved by SDS-PAGE (4%–20% gels) and transferred to Amersham Hybond P 0.45 PVDF blotting membranes (GE Healthcare, 10600023). Membranes were blocked overnight at 4°C in 5% milk or 5% BSA in PBST. Specific blocking conditions were optimized for each antibody during the initial validation stages. Primary antibodies were incubated overnight at 4°C. HRP-linked goat anti-rabbit (Dako, P0448, 1:3000) and rabbit anti-mouse (Dako. P0161, 1:3000) secondary antibodies were incubated for ∼1&nbsp;hr at room temperature. The blots were developed using ECL Prime enhanced chemoluminescent detection reagent (GE Healthcare, RPN2232) or alternatively SuperSignal West Femto chemoluminescent substrate (Thermo Fisher, 34094) and X-ray films (Photon Imaging Systems Ltd, FM024) and developer or a Gel Doc XR<sup>+</sup> system.</div></section><section id="sec-4-4-7"><h4>Antibodies</h4><div id="p0325" role="paragraph">Primary antibodies used for protein detection, with their corresponding dilutions for immunofluorescence (IF), western blotting (WB) and WB blocking conditions were as follows: mouse anti-β-actin (Abcam, 8226, WB 1:2000 in BSA), mouse anti-ZEB2 (Origene, TA802113, IF 1:150, WB 1:2000 in milk), sheep anti-TBR2 (R&amp;D Systems, AF6166, IF 1:200), mouse anti-CDH2 (BD Biosciences, 610920, IF 1:500, WB 1:1000 in milk), mouse anti-CDH1 (BD Biosciences, 610181, IF 1:500, 1:1000 in milk), rabbit anti-OCLN (Abcam, ab31721, IF 1:200, WB 1:1000 in milk), rabbit anti-EMX1 (ATLAS Antibodies, HPA006421, IF 1:100), rabbit anti-EMX1 (Origene, TA325087, WB 1:1000 in BSA), rabbit anti-BLBP (Abcam, ab32423, IF 1:200), rabbit anti-GLAST (Abcam, ab416, IF 1:200), goat anti-DCX (N-19) (Santa Cruz, sc-8067, IF 1:300), rat anti-CTIP2 (Abcam, ab18465, IF 1:200), mouse anti-HuC/D (Life Technologies, A21271, IF 1:200), mouse anti-ZO1 (BD Biosciences, 610966, IF 1:300), chicken anti-GFP (Thermo Fisher, A10262, IF 1:500), rabbit anti-GFP (Abcam, ab290, WB 1:1000 in milk), rabbit anti-EpCAM (Abcam, ab71916, IF 1:300, WB 1:1000 in milk), mouse anti-Vimentin (V9) (Santa Cruz, sc-6260, IF 1:200, WB 1:1000 in BSA) rabbit anti-PAX6 (Abcam, ab195045, IF 1:200), rabbit anti-SOX2 (Abcam, ab97959, IF 1:200), rabbit anti-SHROOM3 (ATLAS Antibodies, HPA047784, IF 1:200, WB 1:1000), mouse anti-GAPDH (Abcam, ab8245, WB 1:2000 in milk), mouse anti-TUJ1 (Biolegend, 801213, IF 1:500). Alexa Fluor 594 Phalloidin (Thermo Fisher, A12380, 1:400) was used to detect F-actin. Alexafluor 405, 488, 568 and 647 secondary antibodies (Thermo Fisher) were used for detection of primary antibodies in IF.</div></section><section id="sec-4-4-8"><h4>PCR analysis</h4><div id="p0330" role="paragraph">For applications aimed at amplicon size comparison GoTaq Green Master Mix (Promega, 9PIM712) was used according to the manufacturer’s guidelines. For molecular cloning or any other application requiring high sequence fidelity Q5 High Fidelity 2X Master Mix (NEB, M0492S) was used. PCR analysis of pluripotency markers was done using the Human Pluripotent Stem Cell Assessment Primer Pair Panel (R&amp;D Systems, SC012) and amplification was done using GoTaq Green. Novex TBE 10% gels (Thermo Fisher, EC6275BOX) were used for DNA-PAGE analysis of <i>ZEB2</i><sup>+/−</sup> mutant gDNA. The primers used were ZEB2_DNAPAGE_F &amp; R (<a id="crosref1390" href="#mmc1">Table S1</a>), GoTaq green was used for amplification and 1X SYBR Gold Nuclei Acid Stain (Thermo Scientific, S11494) was used for detection. Samples were prepared as outlined in the PAGE gels technical sheet.</div></section><section id="sec-4-4-9"><h4>Droplet digital PCR (ddPCR)</h4><div id="p0335" role="paragraph">In order to detect CRISPR mutants and perform enrichment by sib-selection a TaqMan-based ddPCR drop-off assay was designed (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Findlay et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Findlay, S.D. ∙ Vincent, K.M. ∙ Berman, J.R. ...</div><div><strong>A digital pcr-based method for efficient and highly specific screening of genome edited cells</strong></div><div><em>PLoS ONE.</em> 2016; <strong>11</strong>:e0153901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0153901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84977260192" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27089539/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0153901&amp;pmid=27089539" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An amplicon of 198&nbsp;bp overlapping the edited genomic region (GRCh38/hg38 chr2:144,517,275-144,517,351) was produced using primers ZEB2_Cas9screening_F &amp; R. The assay was designed so as to have a 5′-HEX-labeled 3′-BHQ1 probe binding to a nick site (i.e., ZEB2_drop-off_probe) and a 5′-FAM-labeled 3′-BHQ1 probe binding to both edited and WT amplicons (i.e., ZEB2_reference_probe). The specific sequence of primers and probes used in the assay are listed in <a id="crosref1400" href="#mmc1">Table S1</a>. The assay was performed using the ddPCR Supermix for Probes (Bio-Rad, 1863024) as described in the product’s technical bulletin. Briefly, reaction mixes (20&nbsp;μl/reaction) were prepared as follows: 100&nbsp;nM primers, 200&nbsp;nM probes, 10&nbsp;U MseI (NEB, R0525S), 1x ddPCR supermix for probes and 50-300&nbsp;ng of genomic DNA (gDNA). The reactions were loaded into DG8 cartridges (Bio-Rad, 1864008) with droplet generation oil for probes (Bio-Rad, 1863005), the cartridge was then fitted with the DG8 gaskets (Bio-Rad, 1863009) and run on the QX200 droplet generator (Bio-Rad, 10031907). The droplet-oil emulsions were transferred to a ddPCR-compatible 96-well plate that was sealed with the PX1 PCR Plate Sealer (Bio-Rad) and the PCR reaction was run on a C1000 touch thermal cycler following the PCR protocol detailed in the technical bulletin. After thermal cycling, data were acquired on the QX200 Droplet Reader (Bio-Rad) using the QuantaSoft Software (Bio-Rad). Negative and positive control samples for the assay were gDNA extracts from HEK293 cells and HEK293 cells transfected with plasmids pSpCas9(BB)-2A-GFP-<i>ZEB2-</i>guide-A &amp; -B, respectively.</div></section><section id="sec-4-4-10"><h4>RNA-seq sample preparation and sequencing</h4><div id="p0340" role="paragraph">Organoids generated from 3 replicate batches of H9 and G1 cell lines were collected at 7 time points for RNA-seq analysis: 0, 2, 3, 5, 10, 15 and 25&nbsp;days post neural induction. ∼300 organoids per replicate were collected at the time points ranging between day 0 and 5, ∼150 organoids at day 10, ∼100 organoids at day 15, and ∼50 organoids at day 20. Organoids were collected in Eppendorf tubes, washed with PBS, flash-frozen in liquid nitrogen and stored at −80°C.</div><div id="p0345" role="paragraph">RNA from all 42 samples was isolated in parallel with the Direct-zol-96 RNA kit (Zymo Research, R2055), following the product’s manual. cDNA libraries were generated according to NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (NEB E7760). cDNA was amplified for 12 cycles and barcoded with NEBNext Multiplex Oligos for Illumina (NEB E7600). Libraries were tested for quality using the 2100 Bioanalyzer system (Agilent) and concentrations were measured using Qubit dsDNA HS Assay Kit (Thermo Fisher, Q32851) on Qubit 3 Fluorometer (Thermo Fisher). Samples were pooled together to 15&nbsp;nM concentration and sequenced with single-end 50 base mode on 3 lanes of an Illumina HiSeq 4000 instrument.</div></section><section id="sec-4-4-11"><h4>RNA-seq data analysis</h4><div id="p0350" role="paragraph">RNA-seq reads were processed using the PRAGUI pipeline (<a id="intref0265" href="https://github.com/lmb-seq/PRAGUI" target="_blank">https://github.com/lmb-seq/PRAGUI</a>). Reads were trimmed using Trim Galore! version 0.6.3_dev (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Krueger, 2012</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Krueger, F.</div><div><strong>Trim Galore! v0.4.0</strong></div><div>2012</div><div><a id="intref0295" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+Galore%21+v0.4.02012https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0270" href="https://github.com/FelixKrueger/TrimGalore" target="_blank">https://github.com/FelixKrueger/TrimGalore</a>) with Cutadapt version 2.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Martin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnet. J.</em> 2011; <strong>17</strong>:10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.14806%2Fej.17.1.200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Default parameters were used. Read quality was analyzed by FASTQC version v0.11.5 (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Andrews, S.</div><div><strong>FastQC: a quality control tool for high throughput sequence data</strong></div><div>2010</div><div><a id="intref0290" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+a+quality+control+tool+for+high+throughput+sequence+data2010https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Human and gorilla reads were aligned to GRCh38 and Kamilah_GGO_v0 reference genomes respectively using HISAT2 v2.0.0-beta (<span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">Kim et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kim, D. ∙ Langmead, B. ∙ Salzberg, S.L.</div><div><strong>HISAT: a fast spliced aligner with low memory requirements</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:357-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84926519013" target="_blank" aria-label="Open Scopus in new tab">Scopus (13753)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3317&amp;pmid=25751142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GTF gene annotation files for both of the reference genomes were downloaded and filtered for a list of 16,763 genes that are annotated with identical gene names in both species. Reads were assigned to genes in the custom GTF files and counted using HTSeq version 0.11.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Anders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Anders, S. ∙ Pyl, P.T. ∙ Huber, W.</div><div><strong>HTSeq--a Python framework to work with high-throughput sequencing data</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:166-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84928987900" target="_blank" aria-label="Open Scopus in new tab">Scopus (13500)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu638&amp;pmid=25260700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Read counts were then normalized to transcripts per million (TPM) within each sample and log2 transformed (log2(TPM+1)). Unbiased clustering of the samples with principal component analysis was performed using the 3000 most variable genes. Log2 transformed TPM values were converted to z-scores, (x<sub>i</sub> – μ)/σ, where x<sub>i</sub> is the TPM at a given time point, μ is the mean TPM of the gene across all time points and σ is the standard deviation of the TPM across all time points. Pearson correlation was performed on z-scaled mean log2 transformed TPM and on all replicates z-scaled. Hierarchical clustering was done on z-scaled means of the 3000 most variable genes of mean log2 transformed TPMs. GO term enrichment analysis was performed using the online g:Profiler software (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Reimand et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Reimand, J. ∙ Kull, M. ∙ Peterson, H. ...</div><div><strong>g:Profiler--a web-based toolset for functional profiling of gene lists from large-scale experiments</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:W193-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkm226" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34547571030" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17478515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkm226&amp;pmid=17478515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0275" href="https://biit.cs.ut.ee/gprofiler/gost" target="_blank">https://biit.cs.ut.ee/gprofiler/gost</a>).</div></section><section id="sec-4-4-12"><h4>Differential temporal expression analysis</h4><div id="p0355" role="paragraph">RNA-seq analysis was performed using R v3.5.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">R Core Team, 2018</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2018</div><div><a id="intref0300" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2018R+Foundation+for+Statistical+ComputingVienna%2C+Austriahttps%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To assess temporal changes in gene expression between day 3, 5 and 10, the list of 16,763 genes was first filtered based on the following criteria: 1) genes were expressed at &gt; 10 TPM in at least one time point in all replicates of both species, 2) genes displayed a fold-change of &gt; 1.5 TPM between any two time points in at least one species. This resulted in a list of 3,526 genes. Log2 normalized TPMs were then z-scaled across the three time points. Next, the gene list was further filtered in order to remove genes with variable expression patterns across replicates by removing genes with a squared difference &gt; 6 in either species. The squared difference value was obtained by calculating the squared difference of z-scores per time point between all three replicates and then taking the sum of this number for all time points. This list of 2,905 genes was used for time course sequencing data analysis using the TCseq package (<span class="dropBlock reference-citations"><a id="body-ref-sref107-3" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-3" aria-controls="sref107-3">Wu and Gu, 2020</a><div class="dropBlock__holder" data-db-target-of="sref107-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Wu, M. ∙ Gu, L.</div><div><strong>TCseq: Time course sequencing data analysis. R package version 1.12.0</strong></div><div>2020</div><div><a id="intref0305" href="https://www.bioconductor.org/packages/release/bioc/html/TCseq.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/TCseq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.WuL.GuTCseq%3A+Time+course+sequencing+data+analysis.+R+package+version+1.12.02020https%3A%2F%2Fwww.bioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FTCseq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) available online (<a id="intref0280" href="https://rdrr.io/bioc/TCseq/f/inst/doc/TCseq.pdf" target="_blank">https://rdrr.io/bioc/TCseq/f/inst/doc/TCseq.pdf</a>). Replicates were kept separate for the analysis, meaning that the input was 17,430 unique patterns of gene expression representing 2,905 genes in 2 species and 3 replicates per species. The TCSeq analysis was run using the <i>timeclust</i> function with the settings <i>algo&nbsp;= ‘cm’, k&nbsp;= 10</i>, resulting in the unsupervised soft clustering of gene expression patterns into 10 clusters with similar z-scaled temporal patterns. Replicates of 563 genes were found to be in 3 different clusters in at least one species and were removed from downstream analysis resulting in 2,342 genes (<a id="crosref1665" href="#mmc2">Data S1</a>). 22.5% of genes, 527 genes, were found to never be in the same cluster between species showing a robustly different expression pattern (<a id="crosref1670" href="#mmc2">Data S1</a>). Genes were assigned to the cluster where 2 or all of the replicates were found per species. 59% of genes were assigned to different clusters between species. GO term enrichment analysis was performed on the genes present in each of the 10 clusters generated by TCseq per species, resulting in a total of 159 GO:BP terms found in both species, of which 85 were found to be moving between species (Data S1). GO term analysis on the 527 genes moving robustly between species revealed 67 were linked to enriched cell morphogenesis-related terms (“cell morphogenesis,” “cell part morphogenesis,” “cellular component morphogenesis”). These genes were intersected with a list of 1,639 confirmed transcription factors (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Lambert et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Lambert, S.A. ∙ Jolma, A. ∙ Campitelli, L.F. ...</div><div><strong>The Human Transcription Factors</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:650-665</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85041527619" target="_blank" aria-label="Open Scopus in new tab">Scopus (1603)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29425488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.01.029&amp;pmid=29425488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), revealing 8 transcription factors related to cell morphogenesis (<a id="crosref1465" href="#fig5">Figure&nbsp;5</a>F).</div></section><section id="sec-4-4-13"><h4>Live imaging and cell cycle analysis</h4><div id="p0360" role="paragraph">For live imaging, a SiMView light sheet microscope was primarily used in addition to an Andor Revolution spinning disk microscope. Whole organoids were imaged on a SiMView light-sheet microscope with environmental controls as described in <span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">McDole et&nbsp;al. (2018</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>McDole, K. ∙ Guignard, L. ∙ Amat, F. ...</div><div><strong>In Toto Imaging and Reconstruction of Post-Implantation Mouse Development at the Single-Cell Level</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:859-876.e33</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85055072359" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30318151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.09.031&amp;pmid=30318151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, organoids were embedded in Matrigel inside of a glass capillary or placed on a cylindrical PEEK platform and either extruded into or placed inside a Teflon FEP tube (Zeus, Inc.) filled with expansion media. This was then placed inside the imaging chamber, which was kept at 37°C and 5% CO2 in atmosphere using an Okolab Bold line environmental system. Culture media was exchanged for fresh media every 24 hours. Image stacks were acquired using a 488&nbsp;nm laser every 5&nbsp;minutes with 2.031&nbsp;μm z-spacing using two Nikon 16x 0.8 NA water-dipping objectives for detection and two Special Optics 54-12.5-31 @ 450-1000nm 6.45x 0.2NA illumination objectives. Images were collected by two Hamamatsu Orca Fusion cameras. Acquired images were then processed according to previously described methods (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Amat et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Amat, F. ∙ Höckendorf, B. ∙ Wan, Y. ...</div><div><strong>Efficient processing and analysis of large-scale light-sheet microscopy data</strong></div><div><em>Nat. Protoc.</em> 2015; <strong>10</strong>:1679-1696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2015.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84944386016" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26426501/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2015.111&amp;pmid=26426501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">McDole et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>McDole, K. ∙ Guignard, L. ∙ Amat, F. ...</div><div><strong>In Toto Imaging and Reconstruction of Post-Implantation Mouse Development at the Single-Cell Level</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:859-876.e33</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85055072359" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30318151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.09.031&amp;pmid=30318151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For analysis of cell cycle duration, individual cells were tracked manually using the MaMuT plugin in Fiji (<span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Wolff et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Wolff, C. ∙ Tinevez, J.Y. ∙ Pietzsch, T. ...</div><div><strong>Multi-view light-sheet imaging and tracking with the MaMuT software reveals the cell lineage of a direct developing arthropod limb</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e34410</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.34410" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85053006324" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29595475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.34410&amp;pmid=29595475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the cell-cycle calculated as the time period between two divisions. For imaging on the Andor Revolution spinning disk microscope, organoids were placed on an imaging dish (2BScientific, 6160-30) and were held in place by a drop of Matrigel that was allowed to polymerize at 37°C before addition of expansion media. Single cells were selected as the focal point and a images were acquired at 4% laser power every 20&nbsp;minutes and stacked at 2&nbsp;μm intervals in an incubation chamber set to 37°C and 5% CO<sub>2</sub> between day 3 and 5 post neural induction.</div></section><section id="sec-4-4-14"><h4>Growth curve modeling</h4><div id="p0365" role="paragraph">Mathematical modeling of progenitor growth used a standard growth curve equation:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p Subscript t Baseline equals p Subscript t minus 1 Baseline 2 Superscript f" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 (2 0 1) 3 (14 (9 4 (8 5 6 7)) 13 (12 10 11)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="15" data-semantic-children="2,14" data-semantic-content="3" data-semantic-owns="2 3 14" data-semantic-speech="p Subscript t Baseline equals p Subscript t minus 1 Baseline 2 Superscript f"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="15" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="15" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="9,12" data-semantic-content="13" data-semantic-parent="15" data-semantic-owns="9 13 12"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="4,8" data-semantic-parent="14" data-semantic-owns="4 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="9" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="14" data-semantic-owns="10 11"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="12"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p Subscript t" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="p Subscript t"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the number of progenitors at time <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="t"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> in days, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="f"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the number of divisions in 1&nbsp;day. This equation was applied to calculate the number of progenitors in human and gorilla using arbitrary time length and starting cell number. There were certain assumptions governing the shape of the curves, but these assumptions were applied equally to both the human and gorilla. These assumptions were: 1. Divisions were symmetric expansive for the first half of the time period, with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="f"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> being higher early and lower at later time points. 2. During the second half of the time period, divisions were increasingly asymmetric neurogenic, where proportions of symmetric to asymmetric were taken from <i>in&nbsp;vivo</i> measurements (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Noctor et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Noctor, S.C. ∙ Martínez-Cerdeño, V. ∙ Kriegstein, A.R.</div><div><strong>Distinct behaviors of neural stem and progenitor cells underlie cortical neurogenesis</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2008; <strong>508</strong>:28-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.21669" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-42149188826" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18288691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.21669&amp;pmid=18288691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0370" role="paragraph">Because of the second assumption, during the second half of the time period, the growth curve was modified as follows:<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p Subscript t Baseline equals left parenthesis p Subscript t minus 1 Baseline r Subscript a Baseline right parenthesis plus left parenthesis p Subscript t minus 1 Baseline r Subscript s Baseline 2 Superscript f Baseline right parenthesis" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="(38 (2 0 1) 3 (37 (17 4 (15 (10 5 (9 6 7 8)) 14 (13 11 12)) 16) 18 (36 19 (34 (25 20 (24 21 22 23)) 32 (28 26 27) 33 (31 29 30)) 35)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="38" data-semantic-children="2,37" data-semantic-content="3" data-semantic-owns="2 3 37" data-semantic-speech="p Subscript t Baseline equals left parenthesis p Subscript t minus 1 Baseline r Subscript a Baseline right parenthesis plus left parenthesis p Subscript t minus 1 Baseline r Subscript s Baseline 2 Superscript f Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="38" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="38" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="37" data-semantic-children="17,36" data-semantic-content="18" data-semantic-parent="38" data-semantic-owns="17 18 36"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="15" data-semantic-content="4,16" data-semantic-parent="37" data-semantic-owns="4 15 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-children="10,13" data-semantic-content="14" data-semantic-parent="17" data-semantic-owns="10 14 13"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="5,9" data-semantic-parent="15" data-semantic-owns="5 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="9" data-semantic-children="6,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="14" data-semantic-parent="15" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="15" data-semantic-owns="11 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="18" data-semantic-parent="37" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="36" data-semantic-children="34" data-semantic-content="19,35" data-semantic-parent="37" data-semantic-owns="19 34 35"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="19" data-semantic-parent="36" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="34" data-semantic-children="25,28,31" data-semantic-content="32,33" data-semantic-parent="36" data-semantic-owns="25 32 28 33 31"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="20,24" data-semantic-parent="34" data-semantic-owns="20 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="25"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-children="21,23" data-semantic-content="22" data-semantic-parent="25" data-semantic-owns="21 22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="24"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-parent="24" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="34" data-semantic-owns="26 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="34" data-semantic-owns="29 30"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.363em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="35" data-semantic-parent="36" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript a" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript a"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the ratio of divisions that are asymmetric at time <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="t"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="r Subscript s" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="r Subscript s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the ratio of divisions that are symmetric at time <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="t"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0375" role="paragraph">Number of neurons was calculated from the progenitor number at each time point according to the following equation:<div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n Subscript t Baseline equals n Subscript t minus 1 Baseline plus left parenthesis p Subscript t minus 1 Baseline r Subscript a Baseline right parenthesis" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="(26 (2 0 1) 3 (25 (9 4 (8 5 6 7)) 10 (24 11 (22 (17 12 (16 13 14 15)) 21 (20 18 19)) 23)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="26" data-semantic-children="2,25" data-semantic-content="3" data-semantic-owns="2 3 25" data-semantic-speech="n Subscript t Baseline equals n Subscript t minus 1 Baseline plus left parenthesis p Subscript t minus 1 Baseline r Subscript a Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="26" data-semantic-owns="0 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="26" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="25" data-semantic-children="9,24" data-semantic-content="10" data-semantic-parent="26" data-semantic-owns="9 10 24"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="4,8" data-semantic-parent="25" data-semantic-owns="4 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="9" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="10" data-semantic-parent="25" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="24" data-semantic-children="22" data-semantic-content="11,23" data-semantic-parent="25" data-semantic-owns="11 22 23"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="24" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="17,20" data-semantic-content="21" data-semantic-parent="24" data-semantic-owns="17 21 20"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="12,16" data-semantic-parent="22" data-semantic-owns="12 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="17"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="16" data-semantic-children="13,15" data-semantic-content="14" data-semantic-parent="17" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="16"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-parent="16" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="22" data-semantic-owns="18 19"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="20"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="23" data-semantic-parent="24" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n Subscript t" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="n Subscript t"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the number of neurons at time <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="t"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> in days, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p Subscript t minus 1" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="p Subscript t minus 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="4" data-semantic-children="1,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the number of progenitors at <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t minus 1" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="t minus 1"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> as calculated above.</div><div id="p0380" role="paragraph">The only difference between human and gorilla models was an increased <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="f"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> in human for a period of 5&nbsp;days during the symmetric expansive phase, which was the length of time in which human and gorilla exhibited differences in their cell morphology (day 3 to day 8, <a id="crosref1495" href="#fig3">Figure&nbsp;3</a>B). During this time period, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript h u m a n" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 0 (10 1 6 2 7 3 8 4 9 5))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="0,10" data-semantic-owns="0 10" data-semantic-speech="f Subscript h u m a n"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="11"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="10" data-semantic-children="1,2,3,4,5" data-semantic-content="6,7,8,9" data-semantic-parent="11" data-semantic-owns="1 6 2 7 3 8 4 9 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="10"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="10"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="10"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="10"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>was equal to 1.274, calculated from the measured average cell cycle length of 18.831 hours, while <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f Subscript g o r i l l a" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 0 (14 1 8 2 9 3 10 4 11 5 12 6 13 7))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="15" data-semantic-children="0,14" data-semantic-owns="0 14" data-semantic-speech="f Subscript g o r i l l a"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="15"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="1,2,3,4,5,6,7" data-semantic-content="8,9,10,11,12,13" data-semantic-parent="15" data-semantic-owns="1 8 2 9 3 10 4 11 5 12 6 13 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="14"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="14"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="14"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="14"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="14"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="14"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>was equal to 1.086, calculated from the measured average cell cycle length of 22.099.</div><div id="p0385" role="paragraph">Overall, the model revealed an increase in human of 1.9 fold in both <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p Subscript t" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="p Subscript t"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="n Subscript t" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="n Subscript t"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, which was consistent and independent of the arbitrary values otherwise used to generate the growth curves.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0390" role="paragraph">All biological experiments were performed with multiple independent biological samples as detailed in the figure legends. Numbers of samples are detailed in figure legends for each quantification performed, including numbers of cells, organoids, and batches for each cell line. Statistical test details can be found in the figure legends. For comparisons across multiple samples, multiple comparisons corrections were applied as detailed in figure legends. Statistics were performed and graphs were generated in Prism v8.4.2 (Graphpad).</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0395" role="paragraph">The authors would like to thank members of the Lancaster lab for helpful feedback and discussions. We also thank the Light Microscopy facility of the MRC Laboratory of Molecular Biology and M. Kellner, A. Kalinka, G. Ghattaoraya, A. Crisp, and T. Stevens for help with bioinformatics. The authors thank Twycross zoo, in particular Zak Shovell and Clare Ellis, for sample collection. We also thank Johannes Schindelin for the FIJI plug-in used for cell counting and Mara Müller for help in writing the macro for automated quantification of TBR2+ cells. This work was supported by the Medical Research Council (MC_UP_1201/9) and the European Research Council (ERC STG 757710). S.B.-K. is supported by a CRUK Cancer Centre PhD Studentship. E.S.R. is supported by Cantab Capital Institute for Mathematics of Information (CCIMI) and London Mathematical Society (LSM).</div><section id="sec-5"><h3>Author contributions</h3><div id="p0400" role="paragraph">S.B-K. designed and conducted experiments, performed bioinformatics analysis, analyzed data, and wrote the manuscript. S.L.G. designed and conducted experiments, analyzed data, and wrote the manuscript. M.S. optimized and cultured organoids, conducted experiments, and prepared reagents. E.S.R. wrote and optimized scripts for image analysis and quantification. P.F.-P. performed bioinformatics analysis. I.K. prepared ape cells for reprogramming. S.W., U.M., and G.W. provided cell lines and intellectual input. K.M. performed light sheet imaging of organoids and analyzed data. M.A.L. designed and supervised the project, analyzed data, and wrote the manuscript.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0405" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (2)</h2><div id="p0415" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.02.050/attachment/2055f421-8374-4696-8b06-4f317fbc2892/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (102.73 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Table of oligonucleotides, related to STAR Methods</div><div id="espara0035" role="paragraph">Sequence and application of oligonucleotide primers.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.02.050/attachment/3511789b-1be7-4337-baeb-632f8a7f9c7b/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (337.13 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Data S1. Data related to interspecies comparisons from RNA-seq data analysis and GO term enrichment analysis, related to Figures 4 and 5</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Aaku-Saraste, E. ∙ Hellwig, A. ∙ Huttner, W.B.</div><div><strong>Loss of occludin and functional tight junctions, but not ZO-1, during neural tube closure--remodeling of the neuroepithelium prior to neurogenesis</strong></div><div><em>Dev. Biol.</em> 1996; <strong>180</strong>:664-679</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/dbio.1996.0336" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0030589622" target="_blank" aria-label="Open Scopus in new tab">Scopus (221)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8954735/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fdbio.1996.0336&amp;pmid=8954735" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Amarnath, S. ∙ Agarwala, S.</div><div><strong>Cell-cycle-dependent TGFβ-BMP antagonism regulates neural tube closure by modulating tight junctions</strong></div><div><em>J.&nbsp;Cell Sci.</em> 2017; <strong>130</strong>:119-131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/jcs.179192" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85010223698" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27034139/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fjcs.179192&amp;pmid=27034139" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Amat, F. ∙ Höckendorf, B. ∙ Wan, Y. ...</div><div><strong>Efficient processing and analysis of large-scale light-sheet microscopy data</strong></div><div><em>Nat. Protoc.</em> 2015; <strong>10</strong>:1679-1696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2015.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84944386016" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26426501/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2015.111&amp;pmid=26426501" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Anders, S. ∙ Pyl, P.T. ∙ Huber, W.</div><div><strong>HTSeq--a Python framework to work with high-throughput sequencing data</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:166-169</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu638" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84928987900" target="_blank" aria-label="Open Scopus in new tab">Scopus (13500)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu638&amp;pmid=25260700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Ando-Akatsuka, Y. ∙ Yonemura, S. ∙ Itoh, M. ...</div><div><strong>Differential behavior of E-cadherin and occludin in their colocalization with ZO-1 during the establishment of epithelial cell polarity</strong></div><div><em>J.&nbsp;Cell. Physiol.</em> 1999; <strong>179</strong>:115-125</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-4652(199905)179:2%3C115::AID-JCP1%3E3.0.CO;2-T" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0032941962" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10199550/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-4652%28199905%29179%3A2%253C115%3A%3AAID-JCP1%253E3.0.CO%3B2-T&amp;pmid=10199550" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Andrews, S.</div><div><strong>FastQC: a quality control tool for high throughput sequence data</strong></div><div>2010</div><div><a id="intref0290" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+a+quality+control+tool+for+high+throughput+sequence+data2010https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Ardito, G.</div><div><strong>Check-list of the data on the gestation length of primates</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 1976; <strong>5</strong>:213-222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0047-2484(76)90023-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0017253494" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0047-2484%2876%2990023-3" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Bae, B.I. ∙ Jayaraman, D. ∙ Walsh, C.A.</div><div><strong>Genetic changes shaping the human brain</strong></div><div><em>Dev. Cell.</em> 2015; <strong>32</strong>:423-434</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.devcel.2015.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.devcel.2015.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84923224807" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25710529/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2015.01.035&amp;pmid=25710529" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Bayer, S.A. ∙ Altman, J. ∙ Russo, R.J. ...</div><div><strong>Timetables of neurogenesis in the human brain based on experimentally determined patterns in the rat</strong></div><div><em>Neurotoxicology.</em> 1993; <strong>14</strong>:83-144</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8361683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=8361683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Bertero, A. ∙ Pawlowski, M. ∙ Ortmann, D. ...</div><div><strong>Optimized inducible shRNA and CRISPR/Cas9 platforms for in&nbsp;vitro studies of human development using hPSCs</strong></div><div><em>Development.</em> 2016; <strong>143</strong>:4405-4418</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.138081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85002083552" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.138081&amp;pmid=27899508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Boyd, J.L. ∙ Skove, S.L. ∙ Rouanet, J.P. ...</div><div><strong>Human-chimpanzee differences in a FZD8 enhancer alter cell-cycle dynamics in the developing neocortex</strong></div><div><em>Curr. Biol.</em> 2015; <strong>25</strong>:772-779</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84926420296" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25702574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2015.01.041&amp;pmid=25702574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Bush, S.J. ∙ Chen, L. ∙ Tovar-Corona, J.M. ...</div><div><strong>Alternative splicing and the evolution of phenotypic novelty</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2017; <strong>372</strong>:20150474</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2015.0474" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85007137817" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27994117/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2015.0474&amp;pmid=27994117" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">13.</a></div><div>Charvet, C.J. ∙ Stimpson, C.D. ∙ Kim, Y.D. ...</div><div><strong>Gradients in cytoarchitectural landscapes of the isocortex: Diprotodont marsupials in comparison to eutherian mammals</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2017; <strong>525</strong>:1811-1826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.24160" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85014996412" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28001295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.24160&amp;pmid=28001295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">14.</a></div><div>Chng, Z. ∙ Teo, A. ∙ Pedersen, R.A. ...</div><div><strong>SIP1 mediates cell-fate decisions between neuroectoderm and mesendoderm in human pluripotent stem cells</strong></div><div><em>Cell Stem Cell.</em> 2010; <strong>6</strong>:59-70</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2009.11.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2009.11.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-73049085665" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20074535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2009.11.015&amp;pmid=20074535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">15.</a></div><div>Chu, C.W. ∙ Gerstenzang, E. ∙ Ossipova, O. ...</div><div><strong>Lulu regulates Shroom-induced apical constriction during neural tube closure</strong></div><div><em>PLoS ONE.</em> 2013; <strong>8</strong>:e81854</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0081854" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896739280" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24282618/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0081854&amp;pmid=24282618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">16.</a></div><div>de Sousa, A.A. ∙ Sherwood, C.C. ∙ Schleicher, A. ...</div><div><strong>Comparative cytoarchitectural analyses of striate and extrastriate areas in hominoids</strong></div><div><em>Cereb. Cortex.</em> 2010; <strong>20</strong>:966-981</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/cercor/bhp158" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77949351275" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19776344/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcercor%2Fbhp158&amp;pmid=19776344" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">17.</a></div><div>DeFelipe, J. ∙ Alonso-Nanclares, L. ∙ Arellano, J.I.</div><div><strong>Microstructure of the neocortex: comparative aspects</strong></div><div><em>J.&nbsp;Neurocytol.</em> 2002; <strong>31</strong>:299-316</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1023/A:1024130211265" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0038372036" target="_blank" aria-label="Open Scopus in new tab">Scopus (514)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12815249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1023%2FA%3A1024130211265&amp;pmid=12815249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">18.</a></div><div>Dehay, C. ∙ Kennedy, H. ∙ Kosik, K.S.</div><div><strong>The outer subventricular zone and primate-specific cortical complexification</strong></div><div><em>Neuron.</em> 2015; <strong>85</strong>:683-694</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2014.12.060&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2014.12.060&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84925443036" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25695268/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2014.12.060&amp;pmid=25695268" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">19.</a></div><div>DeKelver, R.C. ∙ Choi, V.M. ∙ Moehle, E.A. ...</div><div><strong>Functional genomics, proteomics, and regulatory DNA analysis in isogenic settings using zinc finger nuclease-driven transgenesis into a safe harbor locus in the human genome</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1133-1142</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.106773.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77955157281" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20508142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.106773.110&amp;pmid=20508142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">20.</a></div><div>Donahue, C.J. ∙ Glasser, M.F. ∙ Preuss, T.M. ...</div><div><strong>Quantitative assessment of prefrontal cortex in humans relative to nonhuman primates</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E5183-E5192</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1721653115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047898223" target="_blank" aria-label="Open Scopus in new tab">Scopus (162)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29739891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1721653115&amp;pmid=29739891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">21.</a></div><div>Eisaki, A. ∙ Kuroda, H. ∙ Fukui, A. ...</div><div><strong>XSIP1, a member of two-handed zinc finger proteins, induced anterior neural markers in Xenopus laevis animal cap</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2000; <strong>271</strong>:151-157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/bbrc.2000.2545" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0034728782" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10777695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fbbrc.2000.2545&amp;pmid=10777695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">22.</a></div><div>Epifanova, E. ∙ Babaev, A. ∙ Newman, A.G. ...</div><div><strong>Role of Zeb2/Sip1 in neuronal development</strong></div><div><em>Brain Res.</em> 2019; <strong>1705</strong>:24-31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainres.2018.09.034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85054511567" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30266271/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainres.2018.09.034&amp;pmid=30266271" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">23.</a></div><div>Erwin, G.D. ∙ Oksenberg, N. ∙ Truty, R.M. ...</div><div><strong>Integrating diverse datasets improves developmental enhancer prediction</strong></div><div><em>PLoS Comput. Biol.</em> 2014; <strong>10</strong>:e1003677</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1003677" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84903388630" target="_blank" aria-label="Open Scopus in new tab">Scopus (119)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24967590/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1003677&amp;pmid=24967590" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">24.</a></div><div>Fardi, M. ∙ Alivand, M. ∙ Baradaran, B. ...</div><div><strong>The crucial role of ZEB2: From development to epithelial-to-mesenchymal transition and cancer complexity</strong></div><div><em>J.&nbsp;Cell. Physiol.</em> 2019; <strong>234</strong>:14783-14799</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcp.28277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85061776287" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30773635/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcp.28277&amp;pmid=30773635" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">25.</a></div><div>Fiddes, I.T. ∙ Lodewijk, G.A. ∙ Mooring, M. ...</div><div><strong>Human-Specific NOTCH2NL Genes Affect Notch Signaling and Cortical Neurogenesis</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:1356-1369.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.051&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.051&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047424138" target="_blank" aria-label="Open Scopus in new tab">Scopus (349)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29856954/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.03.051&amp;pmid=29856954" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">26.</a></div><div>Fietz, S.A. ∙ Kelava, I. ∙ Vogt, J. ...</div><div><strong>OSVZ progenitors of human and ferret neocortex are epithelial-like and expand by integrin signaling</strong></div><div><em>Nat. Neurosci.</em> 2010; <strong>13</strong>:690-699</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2553" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77952867780" target="_blank" aria-label="Open Scopus in new tab">Scopus (650)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436478/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2553&amp;pmid=20436478" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">27.</a></div><div>Findlay, S.D. ∙ Vincent, K.M. ∙ Berman, J.R. ...</div><div><strong>A digital pcr-based method for efficient and highly specific screening of genome edited cells</strong></div><div><em>PLoS ONE.</em> 2016; <strong>11</strong>:e0153901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0153901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84977260192" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27089539/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0153901&amp;pmid=27089539" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">28.</a></div><div>Florio, M. ∙ Albert, M. ∙ Taverna, E. ...</div><div><strong>Human-specific gene ARHGAP11B promotes basal progenitor amplification and neocortex expansion</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:1465-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa1975" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84928704517" target="_blank" aria-label="Open Scopus in new tab">Scopus (460)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25721503/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa1975&amp;pmid=25721503" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">29.</a></div><div>Gallego Romero, I. ∙ Pavlovic, B.J. ∙ Hernando-Herraez, I. ...</div><div><strong>A panel of induced pluripotent stem cells from chimpanzees: a resource for comparative functional genomics</strong></div><div><em>eLife.</em> 2015; <strong>4</strong>:e07103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.07103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26102527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.07103&amp;pmid=26102527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">30.</a></div><div>Garavelli, L. ∙ Ivanovski, I. ∙ Caraffi, S.G. ...</div><div><strong>Neuroimaging findings in Mowat-Wilson syndrome: a study of 54 patients</strong></div><div><em>Genet. Med.</em> 2017; <strong>19</strong>:691-700</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1038%2Fgim.2016.176&amp;cf=fulltext&amp;site=gim-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1038%2Fgim.2016.176&amp;cf=pdf&amp;site=gim-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85020205876" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27831545/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fgim.2016.176&amp;pmid=27831545" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">31.</a></div><div>González, F. ∙ Zhu, Z. ∙ Shi, Z.D. ...</div><div><strong>An iCRISPR platform for rapid, multiplexable, and inducible genome editing in human pluripotent stem cells</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>15</strong>:215-226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2014.05.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2014.05.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84905905141" target="_blank" aria-label="Open Scopus in new tab">Scopus (368)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24931489/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2014.05.018&amp;pmid=24931489" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">32.</a></div><div>Götz, M. ∙ Huttner, W.B.</div><div><strong>The cell biology of neurogenesis</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2005; <strong>6</strong>:777-788</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm1739" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-24344510750" target="_blank" aria-label="Open Scopus in new tab">Scopus (1671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16314867/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm1739&amp;pmid=16314867" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">33.</a></div><div>Gracia, M. ∙ Theis, S. ∙ Proag, A. ...</div><div><strong>Mechanical impact of epithelial-mesenchymal transition on epithelial morphogenesis in Drosophila</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2951</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10720-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85068462870" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31273212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10720-0&amp;pmid=31273212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">34.</a></div><div>Haigo, S.L. ∙ Hildebrand, J.D. ∙ Harland, R.M. ...</div><div><strong>Shroom induces apical constriction and is required for hingepoint formation during neural tube closure</strong></div><div><em>Curr. Biol.</em> 2003; <strong>13</strong>:2125-2137</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2003.11.054&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cub.2003.11.054&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0346403360" target="_blank" aria-label="Open Scopus in new tab">Scopus (273)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14680628/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2003.11.054&amp;pmid=14680628" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">35.</a></div><div>Hansen, D.V. ∙ Lui, J.H. ∙ Parker, P.R.L. ...</div><div><strong>Neurogenic radial glia in the outer subventricular zone of human neocortex</strong></div><div><em>Nature.</em> 2010; <strong>464</strong>:554-561</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature08845" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-77950076985" target="_blank" aria-label="Open Scopus in new tab">Scopus (1019)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20154730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature08845&amp;pmid=20154730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">36.</a></div><div>Herculano-Houzel, S.</div><div><strong>The human brain in numbers: a linearly scaled-up primate brain</strong></div><div><em>Front. Hum. Neurosci.</em> 2009; <strong>3</strong>:31</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/neuro.09.031.2009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84890924627" target="_blank" aria-label="Open Scopus in new tab">Scopus (1054)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19915731/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Fneuro.09.031.2009&amp;pmid=19915731" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">37.</a></div><div>Herculano-Houzel, S.</div><div><strong>The remarkable, yet not extraordinary, human brain as a scaled-up primate brain and its associated cost</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:10661-10668</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1201895109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84866172357" target="_blank" aria-label="Open Scopus in new tab">Scopus (419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22723358/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1201895109&amp;pmid=22723358" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">38.</a></div><div>Herculano-Houzel, S. ∙ Mota, B. ∙ Lent, R.</div><div><strong>Cellular scaling rules for rodent brains</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2006; <strong>103</strong>:12138-12143</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0604911103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33747046064" target="_blank" aria-label="Open Scopus in new tab">Scopus (352)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16880386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0604911103&amp;pmid=16880386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">39.</a></div><div>Hildebrand, J.D. ∙ Soriano, P.</div><div><strong>Shroom, a PDZ domain-containing actin-binding protein, is required for neural tube morphogenesis in mice</strong></div><div><em>Cell.</em> 1999; <strong>99</strong>:485-497</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2FS0092-8674%2800%2981537-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2FS0092-8674%2800%2981537-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0033601076" target="_blank" aria-label="Open Scopus in new tab">Scopus (299)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10589677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2981537-8&amp;pmid=10589677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">40.</a></div><div>Hutsler, J.J. ∙ Lee, D.G. ∙ Porter, K.K.</div><div><strong>Comparative analysis of cortical layering and supragranular layer enlargement in rodent carnivore and primate species</strong></div><div><em>Brain Res.</em> 2005; <strong>1052</strong>:71-81</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainres.2005.06.015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-23744472001" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16018988/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainres.2005.06.015&amp;pmid=16018988" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">41.</a></div><div>Kadoshima, T. ∙ Sakaguchi, H. ∙ Nakano, T. ...</div><div><strong>Self-organization of axial polarity, inside-out layer pattern, and species-specific progenitor dynamics in human ES cell-derived neocortex</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:20284-20289</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1315710110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84890282623" target="_blank" aria-label="Open Scopus in new tab">Scopus (692)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24277810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1315710110&amp;pmid=24277810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="optQB60jIAXuw" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optQB60jIAXuw" title="View in article">42.</a></div><div>Kalebic, N. ∙ Gilardi, C. ∙ Stepien, B. ...</div><div><strong>Neocortical Expansion Due to Increased Proliferation of Basal Progenitors Is Linked to Changes in Their Morphology</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:535-550</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2019.02.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2019.02.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85063715304" target="_blank" aria-label="Open Scopus in new tab">Scopus (109)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30905618/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.02.017&amp;pmid=30905618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">43.</a></div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (405)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">44.</a></div><div>Kim, D. ∙ Langmead, B. ∙ Salzberg, S.L.</div><div><strong>HISAT: a fast spliced aligner with low memory requirements</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:357-360</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84926519013" target="_blank" aria-label="Open Scopus in new tab">Scopus (13753)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3317&amp;pmid=25751142" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">45.</a></div><div>Kriegstein, A. ∙ Alvarez-Buylla, A.</div><div><strong>The glial nature of embryonic and adult neural stem cells</strong></div><div><em>Annu. Rev. Neurosci.</em> 2009; <strong>32</strong>:149-184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev.neuro.051508.135600" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-67651036549" target="_blank" aria-label="Open Scopus in new tab">Scopus (1820)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19555289/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev.neuro.051508.135600&amp;pmid=19555289" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">46.</a></div><div>Krueger, F.</div><div><strong>Trim Galore! v0.4.0</strong></div><div>2012</div><div><a id="intref0295" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+Galore%21+v0.4.02012https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">47.</a></div><div>Lambert, S.A. ∙ Jolma, A. ∙ Campitelli, L.F. ...</div><div><strong>The Human Transcription Factors</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:650-665</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.01.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85041527619" target="_blank" aria-label="Open Scopus in new tab">Scopus (1603)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29425488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.01.029&amp;pmid=29425488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="optEWDeCX5gW6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optEWDeCX5gW6" title="View in article">48.</a></div><div>Lancaster, M.A. ∙ Corsini, N.S. ∙ Wolfinger, S. ...</div><div><strong>Guided self-organization and cortical plate formation in human brain organoids</strong></div><div><em>Nat. Biotechnol.</em> 2017; <strong>35</strong>:659-666</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3906" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85020668535" target="_blank" aria-label="Open Scopus in new tab">Scopus (531)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28562594/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3906&amp;pmid=28562594" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">49.</a></div><div>Lancaster, M.A. ∙ Renner, M. ∙ Martin, C.A. ...</div><div><strong>Cerebral organoids model human brain development and microcephaly</strong></div><div><em>Nature.</em> 2013; <strong>501</strong>:373-379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12517" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84884414984" target="_blank" aria-label="Open Scopus in new tab">Scopus (3474)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23995685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12517&amp;pmid=23995685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">50.</a></div><div>Lee, C. ∙ Scherr, H.M. ∙ Wallingford, J.B.</div><div><strong>Shroom family proteins regulate γ-tubulin distribution and microtubule architecture during epithelial cell shape change</strong></div><div><em>Development.</em> 2007; <strong>134</strong>:1431-1441</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02828" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34248165460" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17329357/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02828&amp;pmid=17329357" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">51.</a></div><div>Lindblad-Toh, K. ∙ Garber, M. ∙ Zuk, O. ..., Broad Institute Sequencing Platform and Whole Genome Assembly Team, Baylor College of Medicine Human Genome Sequencing Center Sequencing Team, Genome Institute at Washington University</div><div><strong>A high-resolution map of human evolutionary constraint using 29 mammals</strong></div><div><em>Nature.</em> 2011; <strong>478</strong>:476-482</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10530" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-80054973803" target="_blank" aria-label="Open Scopus in new tab">Scopus (825)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21993624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10530&amp;pmid=21993624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">52.</a></div><div>Liu, X. ∙ Somel, M. ∙ Tang, L. ...</div><div><strong>Extension of cortical synaptic development distinguishes humans from chimpanzees and macaques</strong></div><div><em>Genome Res.</em> 2012; <strong>22</strong>:611-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.127324.111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84859541449" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22300767/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.127324.111&amp;pmid=22300767" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">53.</a></div><div>Lui, J.H. ∙ Hansen, D.V. ∙ Kriegstein, A.R.</div><div><strong>Development and evolution of the human neocortex</strong></div><div><em>Cell.</em> 2011; <strong>146</strong>:18-36</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2011.06.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-79959924122" target="_blank" aria-label="Open Scopus in new tab">Scopus (1009)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21729779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2011.06.030&amp;pmid=21729779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">54.</a></div><div>Marchetto, M.C. ∙ Hrvoj-Mihic, B. ∙ Kerman, B.E. ...</div><div><strong>Species-specific maturation profiles of human, chimpanzee and bonobo neural cells</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e37527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.37527" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85061147585" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30730291/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.37527&amp;pmid=30730291" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">55.</a></div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnet. J.</em> 2011; <strong>17</strong>:10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.14806%2Fej.17.1.200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">56.</a></div><div>Matsuda, T. ∙ Cepko, C.L.</div><div><strong>Controlled expression of transgenes introduced by in&nbsp;vivo electroporation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:1027-1032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0610155104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33846470670" target="_blank" aria-label="Open Scopus in new tab">Scopus (505)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17209010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0610155104&amp;pmid=17209010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">57.</a></div><div>McDole, K. ∙ Guignard, L. ∙ Amat, F. ...</div><div><strong>In Toto Imaging and Reconstruction of Post-Implantation Mouse Development at the Single-Cell Level</strong></div><div><em>Cell.</em> 2018; <strong>175</strong>:859-876.e33</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.09.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85055072359" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30318151/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.09.031&amp;pmid=30318151" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">58.</a></div><div>Medelnik, J.P. ∙ Roensch, K. ∙ Okawa, S. ...</div><div><strong>Signaling-Dependent Control of Apical Membrane Size and Self-Renewal in Rosette-Stage Human Neuroepithelial Stem Cells</strong></div><div><em>Stem Cell Reports.</em> 2018; <strong>10</strong>:1751-1765</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stemcr.2018.04.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85046875468" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29779899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.04.018&amp;pmid=29779899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">59.</a></div><div>Menuchin-Lasowski, Y. ∙ Dagan, B. ∙ Conidi, A. ...</div><div><strong>Zeb2 regulates the balance between retinal interneurons and Müller glia by inhibition of BMP-Smad signaling</strong></div><div><em>Dev. Biol.</em> 2020; <strong>468</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2020.09.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85091666282" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32950463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2020.09.006&amp;pmid=32950463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">60.</a></div><div>Miyoshi, T. ∙ Maruhashi, M. ∙ Van De Putte, T. ...</div><div><strong>Complementary expression pattern of Zfhx1 genes Sip1 and deltaEF1 in the mouse embryo and their genetic interaction revealed by compound mutants</strong></div><div><em>Dev. Dyn.</em> 2006; <strong>235</strong>:1941-1952</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20799" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33745166261" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16598713/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20799&amp;pmid=16598713" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">61.</a></div><div>Mora-Bermúdez, F. ∙ Badsha, F. ∙ Kanton, S. ...</div><div><strong>Differences and similarities between human and chimpanzee neural progenitors during cerebral cortex development</strong></div><div><em>eLife.</em> 2016; <strong>5</strong>:e18683</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.18683" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84995486924" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27669147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.18683&amp;pmid=27669147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">62.</a></div><div>Morris, H.T. ∙ Machesky, L.M.</div><div><strong>Actin cytoskeletal control during epithelial to mesenchymal transition: focus on the pancreas and intestinal tract</strong></div><div><em>Br. J. Cancer.</em> 2015; <strong>112</strong>:613-620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/bjc.2014.658" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84923587504" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25611303/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fbjc.2014.658&amp;pmid=25611303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">63.</a></div><div>Mowat, D.R. ∙ Wilson, M.J. ∙ Goossens, M.</div><div><strong>Mowat-Wilson syndrome</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2003; <strong>40</strong>:305-310</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.40.5.305" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0038577214" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12746390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.40.5.305&amp;pmid=12746390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">64.</a></div><div>Noctor, S.C. ∙ Martínez-Cerdeño, V. ∙ Kriegstein, A.R.</div><div><strong>Distinct behaviors of neural stem and progenitor cells underlie cortical neurogenesis</strong></div><div><em>J.&nbsp;Comp. Neurol.</em> 2008; <strong>508</strong>:28-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cne.21669" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-42149188826" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18288691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcne.21669&amp;pmid=18288691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">65.</a></div><div>Nowakowski, T.J. ∙ Pollen, A.A. ∙ Sandoval-Espinosa, C. ...</div><div><strong>Transformation of the Radial Glia Scaffold Demarcates Two Stages of Human Cerebral Cortex Development</strong></div><div><em>Neuron.</em> 2016; <strong>91</strong>:1219-1227</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2016.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2016.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84991263060" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27657449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2016.09.005&amp;pmid=27657449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">66.</a></div><div>O’Neill, A.C. ∙ Kyrousi, C. ∙ Klaus, J. ...</div><div><strong>A Primate-Specific Isoform of PLEKHG6 Regulates Neurogenesis and Neuronal Migration</strong></div><div><em>Cell Rep.</em> 2018; <strong>25</strong>:2729-2741.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.celrep.2018.11.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.celrep.2018.11.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85057030381" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30517861/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.11.029&amp;pmid=30517861" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">67.</a></div><div>Orihara-Ono, M. ∙ Toriya, M. ∙ Nakao, K. ...</div><div><strong>Downregulation of Notch mediates the seamless transition of individual Drosophila neuroepithelial progenitors into optic medullar neuroblasts during prolonged G1</strong></div><div><em>Dev. Biol.</em> 2011; <strong>351</strong>:163-175</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2010.12.044" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-79551681632" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21215740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2010.12.044&amp;pmid=21215740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">68.</a></div><div>Otani, T. ∙ Marchetto, M.C. ∙ Gage, F.H. ...</div><div><strong>2D and 3D Stem Cell Models of Primate Cortical Development Identify Species-Specific Differences in Progenitor Behavior Contributing to Brain Size</strong></div><div><em>Cell Stem Cell.</em> 2016; <strong>18</strong>:467-480</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2016.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.stem.2016.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84962090418" target="_blank" aria-label="Open Scopus in new tab">Scopus (238)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27049876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.03.003&amp;pmid=27049876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">69.</a></div><div>Parthasarathy, S. ∙ Srivatsa, S. ∙ Nityanandam, A. ...</div><div><strong>Ntf3 acts downstream of Sip1 in cortical postmitotic neurons to control progenitor cell fate through feedback signaling</strong></div><div><em>Development.</em> 2014; <strong>141</strong>:3324-3330</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.114173" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84906215257" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25085976/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.114173&amp;pmid=25085976" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">70.</a></div><div>Paşca, A.M. ∙ Sloan, S.A. ∙ Clarke, L.E. ...</div><div><strong>Functional cortical neurons and astrocytes from human pluripotent stem cells in 3D culture</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:671-678</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3415" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84934442315" target="_blank" aria-label="Open Scopus in new tab">Scopus (1049)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26005811/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3415&amp;pmid=26005811" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">71.</a></div><div>Pastushenko, I. ∙ Blanpain, C.</div><div><strong>EMT Transition States during Tumor Progression and Metastasis</strong></div><div><em>Trends Cell Biol.</em> 2019; <strong>29</strong>:212-226</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.tcb.2018.12.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.tcb.2018.12.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85059015645" target="_blank" aria-label="Open Scopus in new tab">Scopus (1765)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30594349/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.tcb.2018.12.001&amp;pmid=30594349" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">72.</a></div><div>Peinado, H. ∙ Olmeda, D. ∙ Cano, A.</div><div><strong>Snail, Zeb and bHLH factors in tumour progression: an alliance against the epithelial phenotype?</strong></div><div><em>Nat. Rev. Cancer.</em> 2007; <strong>7</strong>:415-428</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrc2131" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34249289041" target="_blank" aria-label="Open Scopus in new tab">Scopus (2685)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17508028/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrc2131&amp;pmid=17508028" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">73.</a></div><div>Pollard, K.S. ∙ Salama, S.R. ∙ King, B. ...</div><div><strong>Forces shaping the fastest evolving regions in the human genome</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e168</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-33750428182" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17040131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020168&amp;pmid=17040131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">74.</a></div><div>Pollen, A.A. ∙ Bhaduri, A. ∙ Andrews, M.G. ...</div><div><strong>Establishing Cerebral Organoids as Models of Human-Specific Brain Evolution</strong></div><div><em>Cell.</em> 2019; <strong>176</strong>:743-756.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2019.01.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2019.01.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85060531675" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30735633/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.01.017&amp;pmid=30735633" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">75.</a></div><div>Qian, X. ∙ Jacob, F. ∙ Song, M.M. ...</div><div><strong>Generation of human brain region-specific organoids using a miniaturized spinning bioreactor</strong></div><div><em>Nat. Protoc.</em> 2018; <strong>13</strong>:565-580</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2017.152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047385267" target="_blank" aria-label="Open Scopus in new tab">Scopus (331)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29470464/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2017.152&amp;pmid=29470464" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">76.</a></div><div>Quadrato, G. ∙ Nguyen, T. ∙ Macosko, E.Z. ...</div><div><strong>Cell diversity and network dynamics in photosensitive human brain organoids</strong></div><div><em>Nature.</em> 2017; <strong>545</strong>:48-53</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85018435810" target="_blank" aria-label="Open Scopus in new tab">Scopus (813)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28445462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22047&amp;pmid=28445462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">77.</a></div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing</strong></div><div>R Foundation for Statistical Computing, Vienna, Austria, 2018</div><div><a id="intref0300" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing2018R+Foundation+for+Statistical+ComputingVienna%2C+Austriahttps%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">78.</a></div><div>Rakic, P.</div><div><strong>Specification of cerebral cortical areas</strong></div><div><em>Science.</em> 1988; <strong>241</strong>:170-176</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.3291116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0023785808" target="_blank" aria-label="Open Scopus in new tab">Scopus (2535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3291116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.3291116&amp;pmid=3291116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">79.</a></div><div>Rakic, P.</div><div><strong>A small step for the cell, a giant leap for mankind: a hypothesis of neocortical expansion during evolution</strong></div><div><em>Trends Neurosci.</em> 1995; <strong>18</strong>:383-388</div></div><div class="external-links"><div class="core-xlink-abstract"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2F0166-2236%2895%2993934-P&amp;cf=abstract&amp;site=cell-site">Abstract</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2F0166-2236%2895%2993934-P&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0029132983" target="_blank" aria-label="Open Scopus in new tab">Scopus (878)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7482803/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0166-2236%2895%2993934-P&amp;pmid=7482803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">80.</a></div><div>Rakic, P.</div><div><strong>The radial edifice of cortical architecture: from neuronal silhouettes to genetic engineering</strong></div><div><em>Brain Res. Brain Res. Rev.</em> 2007; <strong>55</strong>:204-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.brainresrev.2007.02.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-35048888932" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17467805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.brainresrev.2007.02.010&amp;pmid=17467805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">81.</a></div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (7783)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">82.</a></div><div>Reimand, J. ∙ Kull, M. ∙ Peterson, H. ...</div><div><strong>g:Profiler--a web-based toolset for functional profiling of gene lists from large-scale experiments</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:W193-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkm226" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-34547571030" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17478515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkm226&amp;pmid=17478515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">83.</a></div><div>Rockel, A.J. ∙ Hiorns, R.W. ∙ Powell, T.P.S.</div><div><strong>The basic uniformity in structure of the neocortex</strong></div><div><em>Brain.</em> 1980; <strong>103</strong>:221-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/brain/103.2.221" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-0018948139" target="_blank" aria-label="Open Scopus in new tab">Scopus (671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6772266/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbrain%2F103.2.221&amp;pmid=6772266" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">84.</a></div><div>Rogers, C.D. ∙ Sorrells, L.K. ∙ Bronner, M.E.</div><div><strong>A catenin-dependent balance between N-cadherin and E-cadherin controls neuroectodermal cell fate choices</strong></div><div><em>Mech. Dev.</em> 2018; <strong>152</strong>:44-56</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.mod.2018.07.003" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85049778697" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30009960/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.mod.2018.07.003&amp;pmid=30009960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">85.</a></div><div>Saunders, L.R. ∙ McClay, D.R.</div><div><strong>Sub-circuits of a gene regulatory network control a developmental epithelial-mesenchymal transition</strong></div><div><em>Development.</em> 2014; <strong>141</strong>:1503-1513</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.101436" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896302113" target="_blank" aria-label="Open Scopus in new tab">Scopus (86)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24598159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.101436&amp;pmid=24598159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">86.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (39955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">87.</a></div><div>Seuntjens, E. ∙ Nityanandam, A. ∙ Miquelajauregui, A. ...</div><div><strong>Sip1 regulates sequential fate decisions by feedback signaling from postmitotic neurons to progenitors</strong></div><div><em>Nat. Neurosci.</em> 2009; <strong>12</strong>:1373-1380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.2409" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-74949109986" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19838179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.2409&amp;pmid=19838179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">88.</a></div><div>Singh, S. ∙ Solecki, D.J.</div><div><strong>Polarity transitions during neurogenesis and germinal zone exit in the developing central nervous system</strong></div><div><em>Front. Cell. Neurosci.</em> 2015; <strong>9</strong>:62</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fncel.2015.00062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84939615626" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25852469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffncel.2015.00062&amp;pmid=25852469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">89.</a></div><div>Sousa, A.M.M. ∙ Meyer, K.A. ∙ Santpere, G. ...</div><div><strong>Evolution of the Human Nervous System Function, Structure, and Development</strong></div><div><em>Cell.</em> 2017; <strong>170</strong>:226-247</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2017.06.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85023612934" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28708995/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.06.036&amp;pmid=28708995" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">90.</a></div><div>Srivatsa, S. ∙ Parthasarathy, S. ∙ Molnár, Z. ...</div><div><strong>Sip1 downstream Effector ninein controls neocortical axonal growth, ipsilateral branching, and microtubule growth and stability</strong></div><div><em>Neuron.</em> 2015; <strong>85</strong>:998-1012</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2015.01.018&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.neuron.2015.01.018&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84924263870" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25741725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.01.018&amp;pmid=25741725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">91.</a></div><div>Stemmler, M.P. ∙ Eccles, R.L. ∙ Brabletz, S. ...</div><div><strong>Non-redundant functions of EMT transcription factors</strong></div><div><em>Nat. Cell Biol.</em> 2019; <strong>21</strong>:102-112</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-018-0196-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85059497859" target="_blank" aria-label="Open Scopus in new tab">Scopus (352)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30602760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-018-0196-y&amp;pmid=30602760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">92.</a></div><div>Stryjewska, A. ∙ Dries, R. ∙ Pieters, T. ...</div><div><strong>Zeb2 regulates cell fate at the exit from epiblast state in mouse embryonic stem cells</strong></div><div><em>Stem Cells.</em> 2017; <strong>35</strong>:611-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/stem.2521" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85002194595" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27739137/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fstem.2521&amp;pmid=27739137" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">93.</a></div><div>Subramanian, L. ∙ Bershteyn, M. ∙ Paredes, M.F. ...</div><div><strong>Dynamic behaviour of human neuroepithelial cells in the developing forebrain</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14167" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85011672129" target="_blank" aria-label="Open Scopus in new tab">Scopus (60)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28139695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14167&amp;pmid=28139695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">94.</a></div><div>Suzuki, I.K. ∙ Gacquer, D. ∙ Heurck, R. ...</div><div><strong>Human-Specific NOTCH2NL Genes Expand Cortical Neurogenesis through Delta/Notch Regulation</strong></div><div><em>Cell.</em> 2018; <strong>173</strong>:1370-1384.e16</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.067&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=10.1016%2Fj.cell.2018.03.067&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85047337419" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29856955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.03.067&amp;pmid=29856955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">95.</a></div><div>Vandewalle, C. ∙ Comijn, J. ∙ De Craene, B. ...</div><div><strong>SIP1/ZEB2 induces EMT by repressing genes of different epithelial cell-cell junctions</strong></div><div><em>Nucleic Acids Res.</em> 2005; <strong>33</strong>:6566-6578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gki965" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-29144485712" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16314317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgki965&amp;pmid=16314317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">96.</a></div><div>Wiles, E.T. ∙ Bell, R. ∙ Thomas, D. ...</div><div><strong>ZEB2 Represses the Epithelial Phenotype and Facilitates Metastasis in Ewing Sarcoma</strong></div><div><em>Genes Cancer.</em> 2013; <strong>4</strong>:486-500</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/1947601913506115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84891542120" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24386509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F1947601913506115&amp;pmid=24386509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">97.</a></div><div>Williams, M. ∙ Burdsal, C. ∙ Periasamy, A. ...</div><div><strong>Mouse primitive streak forms in&nbsp;situ by initiation of epithelial to mesenchymal transition without migration of a cell population</strong></div><div><em>Dev. Dyn.</em> 2012; <strong>241</strong>:270-283</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.23711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84856221662" target="_blank" aria-label="Open Scopus in new tab">Scopus (96)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22170865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.23711&amp;pmid=22170865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106-1" title="View in article">98.</a></div><div>Wolff, C. ∙ Tinevez, J.Y. ∙ Pietzsch, T. ...</div><div><strong>Multi-view light-sheet imaging and tracking with the MaMuT software reveals the cell lineage of a direct developing arthropod limb</strong></div><div><em>eLife.</em> 2018; <strong>7</strong>:e34410</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.34410" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85053006324" target="_blank" aria-label="Open Scopus in new tab">Scopus (102)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29595475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.34410&amp;pmid=29595475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107-1" title="View in article">99.</a></div><div>Wu, M. ∙ Gu, L.</div><div><strong>TCseq: Time course sequencing data analysis. R package version 1.12.0</strong></div><div>2020</div><div><a id="intref0305" href="https://www.bioconductor.org/packages/release/bioc/html/TCseq.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/TCseq.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.WuL.GuTCseq%3A+Time+course+sequencing+data+analysis.+R+package+version+1.12.02020https%3A%2F%2Fwww.bioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FTCseq.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108-1" title="View in article">100.</a></div><div>Wunderlich, S. ∙ Kircher, M. ∙ Vieth, B. ...</div><div><strong>Primate iPS cells as tools for evolutionary analyses</strong></div><div><em>Stem Cell Res. (Amst.).</em> 2014; <strong>12</strong>:622-629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2014.02.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84896051735" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24631741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2014.02.001&amp;pmid=24631741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">101.</a></div><div>Yasumi, T. ∙ Inoue, M. ∙ Maruhashi, M. ...</div><div><strong>Regulation of trunk neural crest delamination by δEF1 and Sip1 in the chicken embryo</strong></div><div><em>Dev. Growth Differ.</em> 2016; <strong>58</strong>:205-214</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/dgd.12256" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-84958917944" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26691438/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fdgd.12256&amp;pmid=26691438" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110" title="View in article">102.</a></div><div>Yuan, J. ∙ Chen, L. ∙ Xiao, J. ...</div><div><strong>SHROOM2 inhibits tumor metastasis through RhoA-ROCK pathway-dependent and -independent mechanisms in nasopharyngeal carcinoma</strong></div><div><em>Cell Death Dis.</em> 2019; <strong>10</strong>:58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41419-019-1325-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.02.050&amp;key=2-s2.0-85060554590" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30683844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41419-019-1325-7&amp;pmid=30683844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/10a924e9-9944-4197-9975-506afa504d7b/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/953c9099-e07f-4aed-b9a4-5af71378f4c0/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/d6fd4987-f4bf-40ef-a033-ee420d61d0b8/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/81daa671-c338-44af-a164-5376bf94778e/main.assets/gr2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/14600da9-32af-4d18-848b-071ed811ea75/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/f35c64f2-0c13-47ef-9e62-527144f4f0f6/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/1f529a2e-cedd-43e7-bfb3-6b62e6559670/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/a7b740bd-69e8-4713-8458-fd38a26be30e/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/da87986f-5113-4031-a526-4342429b3f68/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/3258c6e8-d973-4e22-a4b6-395e7646b70f/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/c5860e62-dc86-4622-a6c7-01512d3fa9e0/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/c3ae15bf-b6c6-4df0-868c-0c8d2410da4b/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/628ca97d-3458-4822-bfe8-01de8fd25ca4/main.assets/figs7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/asset/18f58f27-3d28-4256-baaf-927b8ce8ba9a/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867421002397%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00092%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">162</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">447</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">31</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">259</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.02.050&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (2)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/attachment/2055f421-8374-4696-8b06-4f317fbc2892/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (102.73 KB)</span></a><div class="core-description">Table S1. Table of oligonucleotides, related to STAR Methods</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.02.050/attachment/3511789b-1be7-4337-baeb-632f8a7f9c7b/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (337.13 KB)</span></a><div class="core-description">Data S1. Data related to interspecies comparisons from RNA-seq data analysis and GO term enrichment analysis, related to Figures 4 and 5</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421002397" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.02.050","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">An early cell shape transition drives evolutionary expansion of the human forebrain</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.02.050/asset/10a924e9-9944-4197-9975-506afa504d7b/main.assets/gr1.jpg" height="803" width="649" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/cef042d7-7ff0-40b7-9036-e61eed0ca18e/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.02.050/asset/953c9099-e07f-4aed-b9a4-5af71378f4c0/main.assets/figs1.jpg" height="615" width="699" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/a3bbc4fa-6fb6-4032-80ac-ce7543fa47cf/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.02.050/asset/d6fd4987-f4bf-40ef-a033-ee420d61d0b8/main.assets/figs2.jpg" height="650" width="753" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/11d2409a-12ce-4dc2-b5a8-7ecc45c8dde7/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.02.050/asset/81daa671-c338-44af-a164-5376bf94778e/main.assets/gr2.jpg" height="665" width="752" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/e9312a0c-1df2-4d54-a89e-abfcf8354f95/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.02.050/asset/14600da9-32af-4d18-848b-071ed811ea75/main.assets/gr3.jpg" height="942" width="660" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/648d2f5a-8625-4dc2-b486-44f6dcc265d2/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.02.050/asset/f35c64f2-0c13-47ef-9e62-527144f4f0f6/main.assets/figs3.jpg" height="353" width="752" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/295e8b06-95a0-4504-a5e8-beafcf8bb17c/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.02.050/asset/1f529a2e-cedd-43e7-bfb3-6b62e6559670/main.assets/gr4.jpg" height="547" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/14b72e5f-1fc7-472d-9ec8-217f6acc48ae/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.02.050/asset/a7b740bd-69e8-4713-8458-fd38a26be30e/main.assets/figs4.jpg" height="436" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/0df149ab-8125-4849-9269-c88036453706/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.02.050/asset/da87986f-5113-4031-a526-4342429b3f68/main.assets/gr5.jpg" height="882" width="749" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/24cca89b-b4c2-4822-8c36-fe88c3dd9b6a/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.02.050/asset/3258c6e8-d973-4e22-a4b6-395e7646b70f/main.assets/figs5.jpg" height="580" width="744" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/1092023f-b269-42db-9ea8-b816b1622186/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.02.050/asset/c5860e62-dc86-4622-a6c7-01512d3fa9e0/main.assets/gr6.jpg" height="941" width="713" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/6d5b08d4-30dc-4a86-bc6e-335052ef2680/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.02.050/asset/c3ae15bf-b6c6-4df0-868c-0c8d2410da4b/main.assets/figs6.jpg" height="941" width="709" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/3c614255-ef1d-406a-9933-0ac3f794d48a/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.02.050/asset/628ca97d-3458-4822-bfe8-01de8fd25ca4/main.assets/figs7.jpg" height="941" width="661" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/f1d4285a-3489-4902-ae85-b0c1453424fe/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.02.050/asset/18f58f27-3d28-4256-baaf-927b8ce8ba9a/main.assets/gr7.jpg" height="943" width="709" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/4c15e73a-cadf-45e0-8a6c-b04a43317edc/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.02.050/asset/10a924e9-9944-4197-9975-506afa504d7b/main.assets/gr1.jpg" height="803" width="649" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/cef042d7-7ff0-40b7-9036-e61eed0ca18e/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.02.050/asset/953c9099-e07f-4aed-b9a4-5af71378f4c0/main.assets/figs1.jpg" height="615" width="699" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/a3bbc4fa-6fb6-4032-80ac-ce7543fa47cf/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.02.050/asset/d6fd4987-f4bf-40ef-a033-ee420d61d0b8/main.assets/figs2.jpg" height="650" width="753" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/11d2409a-12ce-4dc2-b5a8-7ecc45c8dde7/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.02.050/asset/81daa671-c338-44af-a164-5376bf94778e/main.assets/gr2.jpg" height="665" width="752" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/e9312a0c-1df2-4d54-a89e-abfcf8354f95/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.02.050/asset/14600da9-32af-4d18-848b-071ed811ea75/main.assets/gr3.jpg" height="942" width="660" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/648d2f5a-8625-4dc2-b486-44f6dcc265d2/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.02.050/asset/f35c64f2-0c13-47ef-9e62-527144f4f0f6/main.assets/figs3.jpg" height="353" width="752" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/295e8b06-95a0-4504-a5e8-beafcf8bb17c/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.02.050/asset/1f529a2e-cedd-43e7-bfb3-6b62e6559670/main.assets/gr4.jpg" height="547" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/14b72e5f-1fc7-472d-9ec8-217f6acc48ae/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.02.050/asset/a7b740bd-69e8-4713-8458-fd38a26be30e/main.assets/figs4.jpg" height="436" width="753" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/0df149ab-8125-4849-9269-c88036453706/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.02.050/asset/da87986f-5113-4031-a526-4342429b3f68/main.assets/gr5.jpg" height="882" width="749" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/24cca89b-b4c2-4822-8c36-fe88c3dd9b6a/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.02.050/asset/3258c6e8-d973-4e22-a4b6-395e7646b70f/main.assets/figs5.jpg" height="580" width="744" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/1092023f-b269-42db-9ea8-b816b1622186/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.02.050/asset/c5860e62-dc86-4622-a6c7-01512d3fa9e0/main.assets/gr6.jpg" height="941" width="713" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/6d5b08d4-30dc-4a86-bc6e-335052ef2680/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.02.050/asset/c3ae15bf-b6c6-4df0-868c-0c8d2410da4b/main.assets/figs6.jpg" height="941" width="709" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/3c614255-ef1d-406a-9933-0ac3f794d48a/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.02.050/asset/628ca97d-3458-4822-bfe8-01de8fd25ca4/main.assets/figs7.jpg" height="941" width="661" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/f1d4285a-3489-4902-ae85-b0c1453424fe/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.02.050/asset/18f58f27-3d28-4256-baaf-927b8ce8ba9a/main.assets/gr7.jpg" height="943" width="709" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.02.050/asset/4c15e73a-cadf-45e0-8a6c-b04a43317edc/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00239-7';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749589556259","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"8","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421002397","coverDate":"202104","type":"XOCS-JOURNAL:SCOPE-FULL","title":"An early cell shape transition drives evolutionary expansion of the human forebrain","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbd45ea954428c',t:'MTc0OTU4OTU1NS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbd45ea954428c&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.02.050&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>