<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="d27bfa98-4019-4b27-998f-f94f047edd0e" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867420X00067;article:article:pii\:S0092867421000805;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00080-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/08af3be4-d98a-4a9a-b194-25b6764957ee/fx1.jpg">
    


        <meta property="og:description" content="Epidemiological, clinical, molecular, and structural characterization of the N439K
mutation in the SARS-CoV-2 spike receptor binding motif demonstrates that it results
in similar viral fitness compared to wild-type while conferring resistance against
some neutralizing monoclonal antibodies and reducing the activity of some polyclonal
antibody responses.">
    











        <meta name="twitter:title" content="Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/08af3be4-d98a-4a9a-b194-25b6764957ee/fx1.jpg">
    


        <meta name="twitter:description" content="Epidemiological, clinical, molecular, and structural characterization of the N439K
mutation in the SARS-CoV-2 spike receptor binding motif demonstrates that it results
in similar viral fitness compared to wild-type while conferring resistance against
some neutralizing monoclonal antibodies and reducing the activity of some polyclonal
antibody responses.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00080-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity"> <meta name="citation_date" content="2021/03/04"> <meta name="citation_online_date" content="2021/01/28"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="5"> <meta name="citation_firstpage" content="1171"> <meta name="citation_lastpage" content="1187.e20"> <meta name="citation_pmid" content="33621484"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421000805"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421000805"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421000805"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.01.037"> <meta name="citation_pmid" content="33621484"> <meta name="citation_author" content="Emma C. Thomson"><meta name="citation_author_orcid" content="0000-0003-1482-0889"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author_institution" content="Department of Clinical Research, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK"><meta name="citation_author" content="Laura E. Rosen"><meta name="citation_author_orcid" content="0000-0002-8030-0219"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="James G. Shepherd"><meta name="citation_author_orcid" content="0000-0003-3915-048X"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Roberto Spreafico"><meta name="citation_author_orcid" content="0000-0001-8282-7658"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Ana da Silva Filipe"><meta name="citation_author_orcid" content="0000-0002-9442-2903"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Jason A. Wojcechowskyj"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Chris Davis"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Luca Piccoli"><meta name="citation_author_orcid" content="0000-0002-1085-6502"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="David J. Pascall"><meta name="citation_author_orcid" content="0000-0002-7543-0860"><meta name="citation_author_institution" content="Institute of Biodiversity, Animal Health and Comparative Medicine, Boyd Orr Centre for Population and Ecosystem Health, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Josh Dillen"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Spyros Lytras"><meta name="citation_author_orcid" content="0000-0003-4202-6682"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Nadine Czudnochowski"><meta name="citation_author_orcid" content="0000-0002-3146-4110"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Rajiv Shah"><meta name="citation_author_orcid" content="0000-0002-2827-5108"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Marcel Meury"><meta name="citation_author_orcid" content="0000-0002-8718-2374"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Natasha Jesudason"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Anna De Marco"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Kathy Li"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Jessica Bassi"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Aine O’Toole"><meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK"><meta name="citation_author" content="Dora Pinto"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Rachel M. Colquhoun"><meta name="citation_author_orcid" content="0000-0002-5577-9897"><meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK"><meta name="citation_author" content="Katja Culap"><meta name="citation_author_orcid" content="0000-0002-0956-0018"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Ben Jackson"><meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK"><meta name="citation_author" content="Fabrizia Zatta"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Andrew Rambaut"><meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK"><meta name="citation_author" content="Stefano Jaconi"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Vattipally B. Sreenu"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Jay Nix"><meta name="citation_author_institution" content="Molecular Biology Consortium, Advanced Light Source, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA"><meta name="citation_author" content="Ivy Zhang"><meta name="citation_author_institution" content="Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"><meta name="citation_author_institution" content="Tri-Institutional PhD Program in Computational Biology and Medicine, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA"><meta name="citation_author" content="Ruth F. Jarrett"><meta name="citation_author_orcid" content="0000-0003-0390-3478"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="William G. Glass"><meta name="citation_author_institution" content="Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"><meta name="citation_author" content="Martina Beltramello"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Kyriaki Nomikou"><meta name="citation_author_orcid" content="0000-0002-7013-1853"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Matteo Pizzuto"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Lily Tong"><meta name="citation_author_orcid" content="0000-0003-3882-3339"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Elisabetta Cameroni"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Tristan I. Croll"><meta name="citation_author_institution" content="Cambridge Institute for Medical Research, Department of Haematology, University of Cambridge, Cambridge CB2 0XY, UK"><meta name="citation_author" content="Natasha Johnson"><meta name="citation_author_orcid" content="0000-0002-4246-6082"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Julia Di Iulio"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Arthur Wickenhagen"><meta name="citation_author_orcid" content="0000-0001-5470-1505"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Alessandro Ceschi"><meta name="citation_author_orcid" content="0000-0002-4308-0405"><meta name="citation_author_institution" content="Faculty of Biomedical Sciences, Università della Svizzera italiana, 6900 Lugano, Switzerland"><meta name="citation_author_institution" content="Division of Clinical Pharmacology and Toxicology, Institute of Pharmacological Sciences of Southern Switzerland, Ente Ospedaliero Cantonale, 6900 Lugano, Switzerland"><meta name="citation_author_institution" content="Department of Clinical Pharmacology and Toxicology, University Hospital Zurich, 8091 Zurich, Switzerland"><meta name="citation_author" content="Aoife M. Harbison"><meta name="citation_author_institution" content="Department of Chemistry and Hamilton Institute, Maynooth University, Maynooth, Ireland"><meta name="citation_author" content="Daniel Mair"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Paolo Ferrari"><meta name="citation_author_orcid" content="0000-0002-9619-3104"><meta name="citation_author_institution" content="Department of Nephrology, Ospedale Civico Lugano, Ente Ospedaliero Cantonale, 6900 Lugano, Switzerland"><meta name="citation_author_institution" content="Prince of Wales Hospital Clinical School, University of New South Wales, Sydney, NSW 2052, Australia"><meta name="citation_author" content="Katherine Smollett"><meta name="citation_author_orcid" content="0000-0001-7961-7377"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Federica Sallusto"><meta name="citation_author_institution" content="Institute for Research in Biomedicine, Università della Svizzera italiana, 6500 Bellinzona, Switzerland"><meta name="citation_author_institution" content="ETH Institute of Microbiology, ETH Zurich, 8093 Zürich, Switzerland"><meta name="citation_author" content="Stephen Carmichael"><meta name="citation_author_orcid" content="0000-0003-4704-4185"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Christian Garzoni"><meta name="citation_author_institution" content="Clinic of Internal Medicine and Infectious Diseases, Clinica Luganese Moncucco, 6900 Lugano, Switzerland"><meta name="citation_author" content="Jenna Nichols"><meta name="citation_author_orcid" content="0000-0003-0093-7170"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Massimo Galli"><meta name="citation_author_institution" content="III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy"><meta name="citation_author" content="Joseph Hughes"><meta name="citation_author_orcid" content="0000-0003-2556-2563"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Agostino Riva"><meta name="citation_author_institution" content="III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy"><meta name="citation_author" content="Antonia Ho"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Marco Schiuma"><meta name="citation_author_institution" content="III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy"><meta name="citation_author" content="Malcolm G. Semple"><meta name="citation_author_orcid" content="0000-0001-9700-0418"><meta name="citation_author_institution" content="NIHR Health Protection Research Unit in Emerging and Zoonotic Infections, Institute of Infection, Veterinary and Ecological Sciences, Faculty of Health and Life Sciences, University of Liverpool, Liverpool L69 7BE, UK"><meta name="citation_author_institution" content="Respiratory Medicine, Alder Hey Children’s Hospital, Liverpool L12 2AP, UK"><meta name="citation_author" content="Peter J.M. Openshaw"><meta name="citation_author_orcid" content="0000-0002-7220-2555"><meta name="citation_author_institution" content="National Heart and Lung Institute, Imperial College London, London SW3 6LY, UK"><meta name="citation_author" content="Elisa Fadda"><meta name="citation_author_institution" content="Department of Chemistry and Hamilton Institute, Maynooth University, Maynooth, Ireland"><meta name="citation_author" content="J. Kenneth Baillie"><meta name="citation_author_institution" content="The Roslin Institute, University of Edinburgh, Edinburgh EH25 9RG, UK"><meta name="citation_author_institution" content="Intensive Care Unit, Royal Infirmary Edinburgh, Edinburgh EH16 4SA, UK"><meta name="citation_author" content="John D. Chodera"><meta name="citation_author_institution" content="Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"><meta name="citation_author" content="Suzannah J. Rihn"><meta name="citation_author_orcid" content="0000-0001-9495-4056"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Samantha J. Lycett"><meta name="citation_author_institution" content="The Roslin Institute, University of Edinburgh, Edinburgh EH25 9RG, UK"><meta name="citation_author" content="Herbert W. Virgin"><meta name="citation_author_orcid" content="0000-0001-8580-7628"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author_institution" content="Washington University School of Medicine, Saint Louis, MO 63110, USA"><meta name="citation_author" content="Amalio Telenti"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_author" content="Davide Corti"><meta name="citation_author_orcid" content="0000-0002-5797-1364"><meta name="citation_author_institution" content="Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland"><meta name="citation_author" content="David L. Robertson"><meta name="citation_author_email" content="david.l.robertson@glasgow.ac.uk"><meta name="citation_author_orcid" content="0000-0001-6338-0221"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK"><meta name="citation_author" content="Gyorgy Snell"><meta name="citation_author_email" content="gsnell@vir.bio"><meta name="citation_author_orcid" content="0000-0003-1475-659X"><meta name="citation_author_institution" content="Vir Biotechnology, San Francisco, CA 94158, USA"><meta name="citation_keywords" content="SARS-CoV-2; COVID-19; mutation; N439K; variant; Spike; receptor binding motif; monoclonal antibody escape; protein structure"><meta name="citation_reference" content="citation_title=Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies; citation_author=A. Baum; citation_author=B.O. Fulton; citation_author=E. Wloga; citation_author=R. Copin; citation_author=K.E. Pascal; citation_author=V. Russo; citation_author=S. Giordano; citation_author=K. Lanza; citation_author=N. Negron; citation_author=M. Ni; citation_author=et al.; citation_journal_title=Science; citation_volume=369; citation_firstpage=1014; citation_lastpage=1018; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic; citation_author=M.F. Boni; citation_author=P. Lemey; citation_author=X. Jiang; citation_author=T.T. Lam; citation_author=B.W. Perry; citation_author=T.A. Castoe; citation_author=A. Rambaut; citation_author=D.L. Robertson; citation_journal_title=Nat. Microbiol.; citation_volume=5; citation_firstpage=1408; citation_lastpage=1417; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Advanced bayesian multilevel modeling with the R package brms; citation_author=P.-C. Bürkner; citation_journal_title=R J.; citation_volume=10; citation_firstpage=395; citation_lastpage=411; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Beyond Shielding: The Roles of Glycans in the SARS-CoV-2 Spike Protein; citation_author=L. Casalino; citation_author=Z. Gaieb; citation_author=J.A. Goldsmith; citation_author=C.K. Hjorth; citation_author=A.C. Dommer; citation_author=A.M. Harbison; citation_author=C.A. Fogarty; citation_author=E.P. Barros; citation_author=B.C. Taylor; citation_author=J.S. McLellan; citation_author=et al.; citation_journal_title=ACS Cent. Sci.; citation_volume=6; citation_firstpage=1722; citation_lastpage=1734; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=AMBER 2017; citation_author=D.A. Case; citation_author=D.S. Cerutti; citation_author=T.E. Cheatham; citation_author=T.A. Darden; citation_publication_date=2017; citation_publisher=University of California, San Francisco; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 Neutralizing Antibody LY-CoV555 in Outpatients with Covid-19; citation_author=P. Chen; citation_author=A. Nirula; citation_author=B. Heller; citation_author=R.L. Gottlieb; citation_author=J. Boscia; citation_author=J. Morris; citation_author=G. Huhn; citation_author=J. Cardona; citation_author=B. Mocherla; citation_author=V. Stosor; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=384; citation_firstpage=229; citation_lastpage=237; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host; citation_author=B. Choi; citation_author=M.C. Choudhary; citation_author=J. Regan; citation_author=J.A. Sparks; citation_author=R.F. Padera; citation_author=X. Qiu; citation_author=I.H. Solomon; citation_author=H.H. Kuo; citation_author=J. Boucau; citation_author=K. Bowman; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=383; citation_firstpage=2291; citation_lastpage=2293; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The SARS-CoV-2 Spike protein has a broad tropism for mammalian ACE2 proteins; citation_author=C. Conceicao; citation_author=N. Thakur; citation_author=S. Human; citation_author=J.T. Kelly; citation_author=L. Logan; citation_author=D. Bialy; citation_author=S. Bhat; citation_author=P. Stevenson-Leggett; citation_author=A.K. Zagrajek; citation_author=P. Hollinghurst; citation_author=et al.; citation_journal_title=PLoS Biol.; citation_volume=18; citation_firstpage=e3001016; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A neutralizing antibody selected from plasma cells that binds to group 1 and group 2 influenza A hemagglutinins; citation_author=D. Corti; citation_author=J. Voss; citation_author=S.J. Gamblin; citation_author=G. Codoni; citation_author=A. Macagno; citation_author=D. Jarrossay; citation_author=S.G. Vachieri; citation_author=D. Pinna; citation_author=A. Minola; citation_author=F. Vanzetta; citation_author=et al.; citation_journal_title=Science; citation_volume=333; citation_firstpage=850; citation_lastpage=856; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=An integrated national scale SARS-CoV-2 genomic surveillance network; citation_author=COVID-19 Genomics UK (COG-UK) Consortium; citation_journal_title=Lancet Microbe; citation_volume=1; citation_firstpage=e99; citation_lastpage=e100; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps; citation_author=T.I. Croll; citation_journal_title=Acta Crystallogr. D Struct. Biol.; citation_volume=74; citation_firstpage=519; citation_lastpage=530; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genomic epidemiology reveals multiple introductions of SARS-CoV-2 from mainland Europe into Scotland; citation_author=A. da Silva Filipe; citation_author=J.G. Shepherd; citation_author=T. Williams; citation_author=J. Hughes; citation_author=E. Aranday-Cortes; citation_author=P. Asamaphan; citation_author=S. Ashraf; citation_author=C. Balcazar; citation_author=K. Brunker; citation_author=A. Campbell; citation_author=COVID-19 Genomics UK (COG-UK) Consortium; citation_author=et al.; citation_journal_title=Nat. Microbiol.; citation_volume=6; citation_firstpage=112; citation_lastpage=122; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Particle mesh Ewald: An N log(N) method for Ewald sums in large systems; citation_author=T. Darden; citation_author=D. York; citation_author=L. Pedersen; citation_journal_title=J.&nbsp;Chem. Physiol.; citation_volume=98; citation_firstpage=10089; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Issues with SARS-CoV-2 sequencing data; citation_author=N. De Maio; citation_author=C. Walker; citation_author=R. Borges; citation_author=L. Weilguny; citation_author=G. Slodkowicz; citation_author=N. Goldman; citation_journal_title=nCoV-2019 Genomic Epidemiology; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants; citation_author=B. Dearlove; citation_author=E. Lewitus; citation_author=H. Bai; citation_author=Y. Li; citation_author=D.B. Reeves; citation_author=M.G. Joyce; citation_author=P.T. Scott; citation_author=M.F. Amare; citation_author=S. Vasan; citation_author=N.L. Michael; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=23652; citation_lastpage=23662; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=How single mutations affect viral escape from broad and narrow antibodies to H1 influenza hemagglutinin; citation_author=M.B. Doud; citation_author=J.M. Lee; citation_author=J.D. Bloom; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=1386; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=OpenMM 7: Rapid development of high performance algorithms for molecular dynamics; citation_author=P. Eastman; citation_author=J. Swails; citation_author=J.D. Chodera; citation_author=R.T. McGibbon; citation_author=Y. Zhao; citation_author=K.A. Beauchamp; citation_author=L.P. Wang; citation_author=A.C. Simmonett; citation_author=M.P. Harrigan; citation_author=C.D. Stern; citation_author=et al.; citation_journal_title=PLoS Comput. Biol.; citation_volume=13; citation_firstpage=e1005659; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Features and development of Coot; citation_author=P. Emsley; citation_author=B. Lohkamp; citation_author=W.G. Scott; citation_author=K. Cowtan; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=66; citation_firstpage=486; citation_lastpage=501; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Safety and immunogenicity of the ChAdOx1 nCoV-19 vaccine against SARS-CoV-2: a preliminary report of a phase 1/2, single-blind, randomised controlled trial; citation_author=P.M. Folegatti; citation_author=K.J. Ewer; citation_author=P.K. Aley; citation_author=B. Angus; citation_author=S. Becker; citation_author=S. Belij-Rammerstorfer; citation_author=D. Bellamy; citation_author=S. Bibi; citation_author=M. Bittaye; citation_author=E.A. Clutterbuck; citation_author=Oxford COVID Vaccine Trial Group; citation_author=et al.; citation_journal_title=Lancet; citation_volume=396; citation_firstpage=467; citation_lastpage=478; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Evolution of Antibody Immunity to SARS-CoV-2; citation_author=C. Gaebler; citation_author=Z. Wang; citation_author=J.C.C. Lorenzi; citation_author=F. Muecksch; citation_author=S. Finkin; citation_author=M. Tokuyama; citation_author=M. Ladinsky; citation_author=A. Cho; citation_author=M. Jankovic; citation_author=D. Schaefer-Babajew; citation_author=et al.; citation_journal_title=Nature; citation_doi=10.1038/s41586-021-03207-w; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Broad and potent activity against SARS-like viruses by an engineered human monoclonal antibody; citation_author=C. Garrett Rappazzo; citation_author=L.V. Tse; citation_author=C.I. Kaku; citation_author=D. Wrapp; citation_author=M. Sakharkar; citation_author=D. Huang; citation_author=L.M. Deveau; citation_author=T.J. Yockachonis; citation_author=A.S. Herbert; citation_author=M.B. Battles; citation_author=et al.; citation_journal_title=Science; citation_doi=10.1126/science.abf4830; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Retroviral vectors pseudotyped with severe acute respiratory syndrome coronavirus S protein; citation_author=T. Giroglou; citation_author=J. Cinatl; citation_author=H. Rabenau; citation_author=C. Drosten; citation_author=H. Schwalbe; citation_author=H.W. Doerr; citation_author=D. von Laer; citation_journal_title=J.&nbsp;Virol.; citation_volume=78; citation_firstpage=9007; citation_lastpage=9015; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=MDAnalysis: A Python Package for the Rapid Analysis of Molecular Dynamics Simulations; citation_author=R.J. Gowers; citation_author=M. Linke; citation_author=J. Barnoud; citation_author=T.J.E. Reddy; citation_journal_title=Proceedings of the 15th Python in Science Conference; citation_firstpage=98; citation_lastpage=105; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Comprehensive mapping of mutations to the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies; citation_author=A.J. Greaney; citation_author=A.N. Loes; citation_author=K.H.D. Crawford; citation_author=T.N. Starr; citation_author=K.D. Malone; citation_author=Y. Helen; citation_author=H.Y. Chu; citation_author=J.D. Bloom; citation_journal_title=Cell Host &amp; Microbe; citation_doi=10.1016/j.chom.2021.02.003; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Complete mapping of mutations to the SARS-CoV-2 spike receptor-binding domain that escape antibody recognition; citation_author=A.J. Greaney; citation_author=T.N. Starr; citation_author=P. Gilchuk; citation_author=S.J. Zost; citation_author=E. Binshtein; citation_author=A.N. Loes; citation_author=S.K. Hilton; citation_author=J. Huddleston; citation_author=R. Eguia; citation_author=K.H.D. Crawford; citation_author=et al.; citation_journal_title=Cell Host &amp; Microbe; citation_doi=10.1016/j.chom.2020.11.007; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Dates and Times Made Easy with lubridate; citation_author=G. Grolemund; citation_author=H. Wickham; citation_journal_title=J.&nbsp;Stat. Softw.; citation_volume=40.3; citation_firstpage=1; citation_lastpage=25; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar; citation_author=N.D. Grubaugh; citation_author=K. Gangavarapu; citation_author=J. Quick; citation_author=N.L. Matteson; citation_author=J.G. De Jesus; citation_author=B.J. Main; citation_author=A.L. Tan; citation_author=L.M. Paul; citation_author=D.E. Brackney; citation_author=S. Grewal; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=8; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail; citation_author=J. Hansen; citation_author=A. Baum; citation_author=K.E. Pascal; citation_author=V. Russo; citation_author=S. Giordano; citation_author=E. Wloga; citation_author=B.O. Fulton; citation_author=Y. Yan; citation_author=K. Koon; citation_author=K. Patel; citation_author=et al.; citation_journal_title=Science; citation_volume=369; citation_firstpage=1010; citation_lastpage=1014; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Sequence-to-structure dependence of isolated IgG Fc complex biantennary N-glycans: a molecular dynamics study; citation_author=A.M. Harbison; citation_author=L.P. Brosnan; citation_author=K. Fenlon; citation_author=E. Fadda; citation_journal_title=Glycobiology; citation_volume=29; citation_firstpage=94; citation_lastpage=103; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 D614G variant exhibits efficient replication ex&nbsp;vivo and transmission in&nbsp;vivo; citation_author=Y.J. Hou; citation_author=S. Chiba; citation_author=P. Halfmann; citation_author=C. Ehre; citation_author=M. Kuroda; citation_author=K.H. Dinnon; citation_author=S.R. Leist; citation_author=A. Schäfer; citation_author=N. Nakajima; citation_author=K. Takahashi; citation_author=et al.; citation_journal_title=Science; citation_volume=370; citation_firstpage=1464; citation_lastpage=1468; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=D614G mutation of SARS-CoV-2 spike protein enhances viral infectivity; citation_author=J. Hu; citation_author=C.-L. He; citation_author=Q.-Z. Gao; citation_author=G.-J. Zhang; citation_author=X.-X. Cao; citation_author=Q.-X. Long; citation_author=H.-J. Deng; citation_author=L.-Y. Huang; citation_author=J. Chen; citation_author=K. Wang; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.06.20.161323; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=An mRNA Vaccine against SARS-CoV-2 - Preliminary Report; citation_author=L.A. Jackson; citation_author=E.J. Anderson; citation_author=N.G. Rouphael; citation_author=P.C. Roberts; citation_author=M. Makhene; citation_author=R.N. Coler; citation_author=M.P. McCullough; citation_author=J.D. Chappell; citation_author=M.R. Denison; citation_author=L.J. Stevens; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=383; citation_firstpage=1920; citation_lastpage=1931; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Neutralizing Antibodies against SARS-CoV-2 and Other Human Coronaviruses; citation_author=S. Jiang; citation_author=C. Hillyer; citation_author=L. Du; citation_journal_title=Trends Immunol.; citation_volume=41; citation_firstpage=355; citation_lastpage=359; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=LY-CoV555, a rapidly isolated potent neutralizing antibody, provides protection in a non-human primate model of SARS-CoV-2 infection; citation_author=B.E. Jones; citation_author=P.L. Brown-Augsburger; citation_author=K.S. Corbett; citation_author=K. Westendorf; citation_author=J. Davies; citation_author=T.P. Cujec; citation_author=C.M. Wiethoff; citation_author=J.L. Blackbourne; citation_author=B.A. Heinz; citation_author=D. Foster; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.09.30.318972; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Comparison of simple potential functions for simulating liquid water; citation_author=W.L. Jorgensen; citation_author=J. Chandrasekhar; citation_author=J.D. Madura; citation_author=R.W. Impey; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=79; citation_firstpage=926; citation_lastpage=935; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=Xds; citation_author=W. Kabsch; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=66; citation_firstpage=125; citation_lastpage=132; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Phase 1-2 Trial of a SARS-CoV-2 Recombinant Spike Protein Nanoparticle Vaccine; citation_author=C. Keech; citation_author=G. Albert; citation_author=I. Cho; citation_author=A. Robertson; citation_author=P. Reed; citation_author=S. Neal; citation_author=J.S. Plested; citation_author=M. Zhu; citation_author=S. Cloney-Clark; citation_author=H. Zhou; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=383; citation_firstpage=2320; citation_lastpage=2332; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The influenza virus hemagglutinin head evolves faster than the stalk domain; citation_author=E. Kirkpatrick; citation_author=X. Qiu; citation_author=P.C. Wilson; citation_author=J. Bahl; citation_author=F. Krammer; citation_journal_title=Sci. Rep.; citation_volume=8; citation_firstpage=10432; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=GLYCAM06: a generalizable biomolecular force field. Carbohydrates; citation_author=K.N. Kirschner; citation_author=A.B. Yongye; citation_author=S.M. Tschampel; citation_author=J. González-Outeiriño; citation_author=C.R. Daniels; citation_author=B.L. Foley; citation_author=R.J. Woods; citation_journal_title=J.&nbsp;Comput. Chem.; citation_volume=29; citation_firstpage=622; citation_lastpage=655; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Jupyter Notebooks – a publishing format for reproducible computational workflows; citation_inbook_title=Positioning and Power in Academic Publishing: Players, Agents and Agendas; citation_author=T. Kluyver; citation_author=B. Ragan-Kelley; citation_author=F. Pérez; citation_author=B. Granger; citation_author=M. Bussonnier; citation_author=J. Frederic; citation_author=K. Kelley; citation_author=J. Hamrick; citation_author=J. Grout; citation_author=S. Corlay; citation_author=et al.; citation_firstpage=87; citation_lastpage=90; citation_publication_date=2016; citation_publisher=IOS Press; "><meta name="citation_reference" content="citation_title=Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus; citation_author=B. Korber; citation_author=W.M. Fischer; citation_author=S. Gnanakaran; citation_author=H. Yoon; citation_author=J. Theiler; citation_author=W. Abfalterer; citation_author=N. Hengartner; citation_author=E.E. Giorgi; citation_author=T. Bhattacharya; citation_author=B. Foley; citation_author=Sheffield COVID-19 Genomics Group; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=812; citation_lastpage=827.e19; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor; citation_author=J. Lan; citation_author=J. Ge; citation_author=J. Yu; citation_author=S. Shan; citation_author=H. Zhou; citation_author=S. Fan; citation_author=Q. Zhang; citation_author=X. Shi; citation_author=Q. Wang; citation_author=L. Zhang; citation_author=X. Wang; citation_journal_title=Nature; citation_volume=581; citation_firstpage=215; citation_lastpage=220; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Robust and efficient configurational molecular sampling via Langevin dynamics; citation_author=B. Leimkuhler; citation_author=C. Matthews; citation_journal_title=J.&nbsp;Chem. Phys.; citation_volume=138; citation_firstpage=174102; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM; citation_author=H. Li; citation_journal_title=arXiv; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Minimap2: pairwise alignment for nucleotide sequences; citation_author=H. Li; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=3094; citation_lastpage=3100; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Structure of SARS coronavirus spike receptor-binding domain complexed with receptor; citation_author=F. Li; citation_author=W. Li; citation_author=M. Farzan; citation_author=S.C. Harrison; citation_journal_title=Science; citation_volume=309; citation_firstpage=1864; citation_lastpage=1868; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Systematic Parameterization of Monovalent Ions Employing the Nonbonded Model; citation_author=P. Li; citation_author=L.F. Song; citation_author=K.M. Merz; citation_journal_title=J.&nbsp;Chem. Theory Comput.; citation_volume=11; citation_firstpage=1645; citation_lastpage=1657; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity; citation_author=Q. Li; citation_author=J. Wu; citation_author=J. Nie; citation_author=L. Zhang; citation_author=H. Hao; citation_author=S. Liu; citation_author=C. Zhao; citation_author=Q. Zhang; citation_author=H. Liu; citation_author=L. Nie; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=1284; citation_lastpage=1294.e9; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Emergence of SARS-CoV-2 through recombination and strong purifying selection; citation_author=X. Li; citation_author=E.E. Giorgi; citation_author=M.H. Marichannegowda; citation_author=B. Foley; citation_author=C. Xiao; citation_author=X.P. Kong; citation_author=Y. Chen; citation_author=S. Gnanakaran; citation_author=B. Korber; citation_author=F. Gao; citation_journal_title=Sci. Adv.; citation_volume=6; citation_firstpage=eabb9153; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Transmission dynamics and evolutionary history of 2019-nCoV; citation_author=X. Li; citation_author=W. Wang; citation_author=X. Zhao; citation_author=J. Zai; citation_author=Q. Zhao; citation_author=Y. Li; citation_author=A. Chaillon; citation_journal_title=J.&nbsp;Med. Virol.; citation_volume=92; citation_firstpage=501; citation_lastpage=511; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Simulation-efficient shortest probability intervals; citation_author=Y. Liu; citation_author=A. Gelman; citation_author=T. Zheng; citation_journal_title=Stat. Comput.; citation_volume=25; citation_firstpage=809; citation_lastpage=819; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A clade of SARS-CoV-2 viruses associated with lower viral loads in patient upper airways; citation_author=R. Lorenzo-Redondo; citation_author=H.H. Nam; citation_author=S.C. Roberts; citation_author=L.M. Simons; citation_author=L.J. Jennings; citation_author=C. Qi; citation_author=C.J. Achenbach; citation_author=A.R. Hauser; citation_author=M.G. Ison; citation_author=J.F. Hultquist; citation_author=E.A. Ozer; citation_journal_title=EBioMedicine; citation_volume=62; citation_firstpage=103112; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Epidemic waves of COVID-19 in Scotland: a genomic perspective on the impact of the introduction and relaxation of lockdown on SARS-CoV-2; citation_author=S.J. Lycett; citation_author=J. Hughes; citation_author=M.P. McHugh; citation_author=A. da Silva Felipe; citation_author=R. Dewar; citation_author=L. Lu; citation_author=T. Doherty; citation_author=A. Shepherd; citation_author=R. Inward; citation_author=G. Rossi; citation_author=et al.; citation_journal_title=medRxiv; citation_doi=10.1101/2021.01.08.20248677; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB; citation_author=J.A. Maier; citation_author=C. Martinez; citation_author=K. Kasavajhala; citation_author=L. Wickstrom; citation_author=K.E. Hauser; citation_author=C. Simmerling; citation_journal_title=J.&nbsp;Chem. Theory Comput.; citation_volume=11; citation_firstpage=3696; citation_lastpage=3713; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Recurrent deletions in the SARS-CoV-2 spike glycoprotein drive antibody escape; citation_author=K.R. McCarthy; citation_author=L.J. Rennick; citation_author=S. Nambulli; citation_author=L.R. Robinson-McCarthy; citation_author=W.G. Bain; citation_author=G. Haidar; citation_author=P. Duprex; citation_journal_title=Science; citation_doi=10.1126/science.abf6950; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Phaser crystallographic software; citation_author=A.L. McCoy; citation_author=R.W. Grosse-Kunstleve; citation_author=P.D. Adams; citation_author=M.D. Winn; citation_author=L.C. Storoni; citation_author=R.J. Read; citation_journal_title=J. Appl. Crystallogr.; citation_volume=40; citation_firstpage=658; citation_lastpage=674; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories; citation_author=R.T. McGibbon; citation_author=K.A. Beauchamp; citation_author=M.P. Harrigan; citation_author=C. Klein; citation_author=J.M. Swails; citation_author=C.X. Hernández; citation_author=C.R. Schwantes; citation_author=L.P. Wang; citation_author=T.J. Lane; citation_author=V.S. Pande; citation_journal_title=Biophys. J.; citation_volume=109; citation_firstpage=1528; citation_lastpage=1532; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Rapid implementation of SARS-CoV-2 sequencing to investigate cases of health-care associated COVID-19: a prospective genomic surveillance study; citation_author=L.W. Meredith; citation_author=W.L. Hamilton; citation_author=B. Warne; citation_author=C.J. Houldcroft; citation_author=M. Hosmillo; citation_author=A.S. Jahun; citation_author=M.D. Curran; citation_author=S. Parmar; citation_author=L.G. Caller; citation_author=S.L. Caddy; citation_author=et al.; citation_journal_title=Lancet Infect. Dis.; citation_volume=20; citation_firstpage=P1263; citation_lastpage=P1271; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=MDAnalysis: a toolkit for the analysis of molecular dynamics simulations; citation_author=N. Michaud-Agrawal; citation_author=E.J. Denning; citation_author=T.B. Woolf; citation_author=O. Beckstein; citation_journal_title=J.&nbsp;Comput. Chem.; citation_volume=32; citation_firstpage=2319; citation_lastpage=2327; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era; citation_author=B.Q. Minh; citation_author=H.A. Schmidt; citation_author=O. Chernomor; citation_author=D. Schrempf; citation_author=M.D. Woodhams; citation_author=A. von Haeseler; citation_author=R. Lanfear; citation_journal_title=Mol. Biol. Evol.; citation_volume=37; citation_firstpage=1530; citation_lastpage=1534; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Viral genomes reveal patterns of the SARS-CoV-2 outbreak in Washington State; citation_author=N.F. Mueller; citation_author=C. Wagner; citation_author=C.D. Frazar; citation_author=P. Roychoudhury; citation_author=J. Lee; citation_author=L.H. Moncla; citation_author=B. Pelle; citation_author=M. Richardson; citation_author=E. Ryke; citation_author=H. Xie; citation_author=et al.; citation_journal_title=medRxiv; citation_doi=10.1101/2020.09.30.20204230; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=REFMAC5 for the refinement of macromolecular crystal structures; citation_author=G.N. Murshudov; citation_author=P. Skubák; citation_author=A.A. Lebedev; citation_author=N.S. Pannu; citation_author=R.A. Steiner; citation_author=R.A. Nicholls; citation_author=M.D. Winn; citation_author=F. Long; citation_author=A.A. Vagin; citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_volume=67; citation_firstpage=355; citation_lastpage=367; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans; citation_author=B.B. Oude Munnink; citation_author=R.S. Sikkema; citation_author=D.F. Nieuwenhuijse; citation_author=R.J. Molenaar; citation_author=E. Munger; citation_author=R. Molenkamp; citation_author=A. van der Spek; citation_author=P. Tolsma; citation_author=A. Rietveld; citation_author=M. Brouwer; citation_author=et al.; citation_journal_title=Science; citation_volume=371; citation_firstpage=172; citation_lastpage=177; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R; citation_author=E. Paradis; citation_author=K. Schliep; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=526; citation_lastpage=528; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=IPython: A System for Interactive Scientific Computing; citation_author=F. Perez; citation_author=B.E. Granger; citation_journal_title=Comput. Sci. Eng.; citation_volume=9; citation_firstpage=21; citation_lastpage=29; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=UCSF ChimeraX: Structure visualization for researchers, educators, and developers; citation_author=E.F. Pettersen; citation_author=T.D. Goddard; citation_author=C.C. Huang; citation_author=E.C. Meng; citation_author=G.S. Couch; citation_author=T.I. Croll; citation_author=J.H. Morris; citation_author=T.E. Ferrin; citation_journal_title=Protein Sci.; citation_volume=30; citation_firstpage=70; citation_lastpage=82; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology; citation_author=L. Piccoli; citation_author=Y.J. Park; citation_author=M.A. Tortorici; citation_author=N. Czudnochowski; citation_author=A.C. Walls; citation_author=M. Beltramello; citation_author=C. Silacci-Fregni; citation_author=D. Pinto; citation_author=L.E. Rosen; citation_author=J.E. Bowen; citation_author=et al.; citation_journal_title=Cell; citation_volume=183; citation_firstpage=1024; citation_lastpage=1042.e21; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody; citation_author=D. Pinto; citation_author=Y.J. Park; citation_author=M. Beltramello; citation_author=A.C. Walls; citation_author=M.A. Tortorici; citation_author=S. Bianchi; citation_author=S. Jaconi; citation_author=K. Culap; citation_author=F. Zatta; citation_author=A. De Marco; citation_author=et al.; citation_journal_title=Nature; citation_volume=583; citation_firstpage=290; citation_lastpage=295; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology; citation_author=A. Rambaut; citation_author=E.C. Holmes; citation_author=Á. O’Toole; citation_author=V. Hill; citation_author=J.T. McCrone; citation_author=C. Ruis; citation_author=L. du Plessis; citation_author=O.G. Pybus; citation_journal_title=Nat. Microbiol.; citation_volume=5; citation_firstpage=1403; citation_lastpage=1407; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A Haploid Genetic Screen Identifies Heparan Sulfate Proteoglycans Supporting Rift Valley Fever Virus Infection; citation_author=A.M. Riblett; citation_author=V.A. Blomen; citation_author=L.T. Jae; citation_author=L.A. Altamura; citation_author=R.W. Doms; citation_author=T.R. Brummelkamp; citation_author=J.A. Wojcechowskyj; citation_journal_title=J.&nbsp;Virol.; citation_volume=90; citation_firstpage=1414; citation_lastpage=1423; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Convergent antibody responses to SARS-CoV-2 in convalescent individuals; citation_author=D.F. Robbiani; citation_author=C. Gaebler; citation_author=F. Muecksch; citation_author=J.C.C. Lorenzi; citation_author=Z. Wang; citation_author=A. Cho; citation_author=M. Agudelo; citation_author=C.O. Barnes; citation_author=A. Gazumyan; citation_author=S. Finkin; citation_author=et al.; citation_journal_title=Nature; citation_volume=584; citation_firstpage=437; citation_lastpage=442; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Coronavirus RNA Proofreading: Molecular Basis and Therapeutic Targeting; citation_author=F. Robson; citation_author=K.S. Khan; citation_author=T.K. Le; citation_author=C. Paris; citation_author=S. Demirbag; citation_author=P. Barfuss; citation_author=P. Rocchi; citation_author=W.L. Ng; citation_journal_title=Mol. Cell; citation_volume=79; citation_firstpage=710; citation_lastpage=727; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=pdb-tools: a swiss army knife for molecular structures; citation_author=J.P.G.L.M. Rodrigues; citation_author=J.M.C. Teixeira; citation_author=M. Trellet; citation_author=A.M.J.J. Bonvin; citation_journal_title=F1000Res.; citation_volume=7; citation_firstpage=1961; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Isolation of potent SARS-CoV-2 neutralizing antibodies and protection from disease in a small animal model; citation_author=T.F. Rogers; citation_author=F. Zhao; citation_author=D. Huang; citation_author=N. Beutler; citation_author=A. Burns; citation_author=W.T. He; citation_author=O. Limbo; citation_author=C. Smith; citation_author=G. Song; citation_author=J. Woehl; citation_author=et al.; citation_journal_title=Science; citation_volume=369; citation_firstpage=956; citation_lastpage=963; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Comprehensive characterization of N- and O- glycosylation of SARS-CoV-2 human receptor angiotensin converting enzyme 2; citation_author=A. Shajahan; citation_author=S. Archer-Hartmann; citation_author=N.T. Supekar; citation_author=A.S. Gleinich; citation_author=C. Heiss; citation_author=P. Azadi; citation_journal_title=Glycobiology; citation_doi=10.1093/glycob/cwaa101; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Structural basis of receptor recognition by SARS-CoV-2; citation_author=J. Shang; citation_author=G. Ye; citation_author=K. Shi; citation_author=Y. Wan; citation_author=C. Luo; citation_author=H. Aihara; citation_author=Q. Geng; citation_author=A. Auerbach; citation_author=F. Li; citation_journal_title=Nature; citation_volume=581; citation_firstpage=221; citation_lastpage=224; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=COMPUTING: Screen Savers of the World Unite!; citation_author=M. Shirts; citation_author=V.S. Pande; citation_journal_title=Science; citation_volume=290; citation_firstpage=1903; citation_lastpage=1904; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Suptavumab for the Prevention of Medically Attended Respiratory Syncytial Virus Infection in Preterm Infants; citation_author=E.A.F. Simões; citation_author=E. Forleo-Neto; citation_author=G.P. Geba; citation_author=M. Kamal; citation_author=F. Yang; citation_author=H. Cicirello; citation_author=M.R. Houghton; citation_author=R. Rideman; citation_author=Q. Zhao; citation_author=S.L. Benvin; citation_author=et al.; citation_journal_title=Clin. Infect. Dis.; citation_firstpage=ciaa951; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Prospective mapping of viral mutations that escape antibodies used to treat COVID-19; citation_author=T.N. Starr; citation_author=A.J. Greaney; citation_author=A. Addetia; citation_author=W.W. Hannon; citation_author=M.C. Choudhary; citation_author=A.S. Dingens; citation_author=J.Z. Li; citation_author=J.D. Bloom; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.11.30.405472; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding; citation_author=T.N. Starr; citation_author=A.J. Greaney; citation_author=S.K. Hilton; citation_author=D. Ellis; citation_author=K.H.D. Crawford; citation_author=A.S. Dingens; citation_author=M.J. Navarro; citation_author=J.E. Bowen; citation_author=M.A. Tortorici; citation_author=A.C. Walls; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=1295; citation_lastpage=1310.e20; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A system for functional analysis of Ebola virus glycoprotein; citation_author=A. Takada; citation_author=C. Robison; citation_author=H. Goto; citation_author=A. Sanchez; citation_author=K.G. Murti; citation_author=M.A. Whitt; citation_author=Y. Kawaoka; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=94; citation_firstpage=14764; citation_lastpage=14769; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa; citation_author=H. Tegally; citation_author=E. Wilkinson; citation_author=M. Giovanetti; citation_author=A. Iranzadeh; citation_author=V. Fonseca; citation_author=J. Giandhari; citation_author=D. Doolabh; citation_author=S. Pillay; citation_author=E.J. San; citation_author=N. Msomi; citation_author=et al.; citation_journal_title=medRxiv; citation_doi=10.1101/2020.12.21.20248640; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms; citation_author=M.A. Tortorici; citation_author=M. Beltramello; citation_author=F.A. Lempp; citation_author=D. Pinto; citation_author=H.V. Dang; citation_author=L.E. Rosen; citation_author=M. McCallum; citation_author=J. Bowen; citation_author=A. Minola; citation_author=S. Jaconi; citation_author=et al.; citation_journal_title=Science; citation_volume=370; citation_firstpage=950; citation_lastpage=957; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Scalable relaxed clock phylogenetic dating; citation_author=E.M. Volz; citation_author=S.D.W. Frost; citation_journal_title=Virus Evol.; citation_volume=3; citation_firstpage=vex025; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity; citation_author=E.M. Volz; citation_author=V. Hill; citation_author=J.T. McCrone; citation_author=A. Price; citation_author=D. Jorgensen; citation_author=A. O’Toole; citation_author=A. Southgate; citation_author=R. Johnson; citation_author=B. Jackson; citation_author=F.F. Nascimento; citation_author=et al.; citation_journal_title=Cell; citation_doi=10.1016/j.cell.2020.11.020; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking epidemiological and genetic data; citation_author=E. Volz; citation_author=S. Mishra; citation_author=M. Chand; citation_author=J.C. Barrett; citation_author=R. Johnson; citation_author=L. Geidelberg; citation_author=W.R. Hinsley; citation_author=D.J. Laydon; citation_author=G. Dabrera; citation_author=A. O’Toole; citation_author=et al.; citation_journal_title=medRxiv; citation_doi=10.1101/2020.12.30.20249034; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein; citation_author=A.C. Walls; citation_author=Y.J. Park; citation_author=M.A. Tortorici; citation_author=A. Wall; citation_author=A.T. McGuire; citation_author=D. Veesler; citation_journal_title=Cell; citation_volume=181; citation_firstpage=281; citation_lastpage=292.e6; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Site-specific glycan analysis of the SARS-CoV-2 spike; citation_author=Y. Watanabe; citation_author=J.D. Allen; citation_author=D. Wrapp; citation_author=J.S. McLellan; citation_author=M. Crispin; citation_journal_title=Science; citation_volume=369; citation_firstpage=330; citation_lastpage=333; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Broad neutralization of SARS-related viruses by human monoclonal antibodies; citation_author=A.Z. Wec; citation_author=D. Wrapp; citation_author=A.S. Herbert; citation_author=D.P. Maurer; citation_author=D. Haslwanter; citation_author=M. Sakharkar; citation_author=R.K. Jangra; citation_author=M.E. Dieterle; citation_author=A. Lilov; citation_author=D. Huang; citation_author=et al.; citation_journal_title=Science; citation_volume=369; citation_firstpage=731; citation_lastpage=736; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants; citation_author=Y. Weisblum; citation_author=F. Schmidt; citation_author=F. Zhang; citation_author=J. DaSilva; citation_author=D. Poston; citation_author=J.C. Lorenzi; citation_author=F. Muecksch; citation_author=M. Rutkowska; citation_author=H.H. Hoffmann; citation_author=E. Michailidis; citation_author=et al.; citation_journal_title=eLife; citation_volume=9; citation_firstpage=e61312; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=D614G Spike Mutation Increases SARS CoV-2 Susceptibility to Neutralization; citation_author=D. Weissman; citation_author=M.G. Alameh; citation_author=T. de Silva; citation_author=P. Collini; citation_author=H. Hornsby; citation_author=R. Brown; citation_author=C.C. LaBranche; citation_author=R.J. Edwards; citation_author=L. Sutherland; citation_author=S. Santra; citation_author=et al.; citation_journal_title=Cell Host Microbe; citation_volume=29; citation_firstpage=23; citation_lastpage=31.e4; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Structural Basis for Potent Neutralization of Betacoronaviruses by Single-Domain Camelid Antibodies; citation_author=D. Wrapp; citation_author=D. De Vlieger; citation_author=K.S. Corbett; citation_author=G.M. Torres; citation_author=N. Wang; citation_author=W. Van Breedam; citation_author=K. Roose; citation_author=L. van Schie; citation_author=M. Hoffmann; citation_author=S. Pöhlmann; citation_author=VIB-CMB COVID-19 Response Team; citation_author=et al.; citation_journal_title=Cell; citation_volume=181; citation_firstpage=1004; citation_lastpage=1015.e15; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation; citation_author=D. Wrapp; citation_author=N. Wang; citation_author=K.S. Corbett; citation_author=J.A. Goldsmith; citation_author=C.L. Hsieh; citation_author=O. Abiona; citation_author=B.S. Graham; citation_author=J.S. McLellan; citation_journal_title=Science; citation_volume=367; citation_firstpage=1260; citation_lastpage=1263; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A new coronavirus associated with human respiratory disease in China; citation_author=F. Wu; citation_author=S. Zhao; citation_author=B. Yu; citation_author=Y.M. Chen; citation_author=W. Wang; citation_author=Z.G. Song; citation_author=Y. Hu; citation_author=Z.W. Tao; citation_author=J.H. Tian; citation_author=Y.Y. Pei; citation_author=et al.; citation_journal_title=Nature; citation_volume=579; citation_firstpage=265; citation_lastpage=269; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Substantial underestimation of SARS-CoV-2 infection in the United States; citation_author=S.L. Wu; citation_author=A.N. Mertens; citation_author=Y.S. Crider; citation_author=A. Nguyen; citation_author=N.N. Pokpongkiat; citation_author=S. Djajadi; citation_author=A. Seth; citation_author=M.S. Hsiang; citation_author=J.M. Colford; citation_author=A. Reingold; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=4507; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant; citation_author=L. Yurkovetskiy; citation_author=X. Wang; citation_author=K.E. Pascal; citation_author=C. Tomkins-Tinch; citation_author=T.P. Nyalile; citation_author=Y. Wang; citation_author=A. Baum; citation_author=W.E. Diehl; citation_author=A. Dauphin; citation_author=C. Carbone; citation_author=et al.; citation_journal_title=Cell; citation_volume=183; citation_firstpage=739; citation_lastpage=751.e8; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity; citation_author=L. Zhang; citation_author=C.B. Jackson; citation_author=H. Mou; citation_author=A. Ojha; citation_author=H. Peng; citation_author=B.D. Quinlan; citation_author=E.S. Rangarajan; citation_author=A. Pan; citation_author=A. Vanderheiden; citation_author=M.S. Suthar; citation_author=et al.; citation_journal_title=Nature Communications; citation_doi=10.1038/s41467-020-19808-4; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Virus-Receptor Interactions of Glycosylated SARS-CoV-2 Spike and Human ACE2 Receptor; citation_author=P. Zhao; citation_author=J.L. Praissman; citation_author=O.C. Grant; citation_author=Y. Cai; citation_author=T. Xiao; citation_author=K.E. Rosenbalm; citation_author=K. Aoki; citation_author=B.P. Kellman; citation_author=R. Bridger; citation_author=D.H. Barouch; citation_author=et al.; citation_journal_title=Cell Host Microbe; citation_volume=28; citation_firstpage=586; citation_lastpage=601.e6; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome; citation_author=M.I. Zimmerman; citation_author=J.R. Porter; citation_author=M.D. Ward; citation_author=S. Singh; citation_author=N. Vithani; citation_author=A. Meller; citation_author=U.L. Mallimadugula; citation_author=C.E. Kuhn; citation_author=J.H. Borowsky; citation_author=R.P. Wiewiora; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.06.27.175430; citation_publication_date=2020; "><meta name="citation_abstract" content="<h2>Summary</h2><p>SARS-CoV-2 can mutate and evade immunity, with consequences for efficacy of emerging vaccines and antibody therapeutics. Here, we demonstrate that the immunodominant SARS-CoV-2 spike (S) receptor binding motif (RBM) is a highly variable region of S and provide epidemiological, clinical, and molecular characterization of a prevalent, sentinel RBM mutation, N439K. We demonstrate N439K S protein has enhanced binding affinity to the hACE2 receptor, and N439K viruses have similar <i>in&nbsp;vitro</i> replication fitness and cause infections with similar clinical outcomes as compared to wild type. We show the N439K mutation confers resistance against several neutralizing monoclonal antibodies, including one authorized for emergency use by the US Food and Drug Administration (FDA), and reduces the activity of some polyclonal sera from persons recovered from infection. Immune evasion mutations that maintain virulence and fitness such as N439K can emerge within SARS-CoV-2 S, highlighting the need for ongoing molecular surveillance to guide development and usage of vaccines and therapeutics.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=12784&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00080-5&amp;be=10922&amp;fe=12739&amp;dc=12364&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749589356312,%22n%22:0,%22f%22:9105,%22dn%22:9106,%22dne%22:9106,%22c%22:9106,%22s%22:9106,%22ce%22:9106,%22rq%22:9107,%22rp%22:10376,%22rpe%22:12196,%22dl%22:10381,%22di%22:12308,%22ds%22:12364,%22de%22:12379,%22dc%22:12738,%22l%22:12738,%22le%22:12769%7D,%22navigation%22:%7B%7D%7D&amp;fp=11572&amp;fcp=11572&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00080-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00080-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-271767" aria-expanded="false" aria-controls="login-links-271767" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-271767" id="login-links--271767" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-271767" data-opens="#links-dialog-271767" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-271767" aria-modal="true"><button data-closes="#links-dialog-271767" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900080-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.01.037">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail long-title" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00080-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0006-7"><span>Volume 184</span>, <span>Issue 5</span></a></span><span class="meta-panel__pages"><span>p1171-1187.e20</span></span><span class="meta-panel__onlineDate">March 04, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=5&amp;pii=S0092-8674%2820%29X0006-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Emma C.</span> <span property="familyName">Thomson</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Emma C.</span> <span property="familyName">Thomson</span> <a class="orcid-id" href="https://orcid.org/0000-0003-1482-0889" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-1482-0889</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">29</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Clinical Research, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Thomson/Emma+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">29</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Laura E.</span> <span property="familyName">Rosen</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Laura E.</span> <span property="familyName">Rosen</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8030-0219" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8030-0219</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">29</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rosen/Laura+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">29</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">James G.</span> <span property="familyName">Shepherd</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James G.</span> <span property="familyName">Shepherd</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3915-048X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3915-048X</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">29</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Shepherd/James+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">29</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Roberto</span> <span property="familyName">Spreafico</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Roberto</span> <span property="familyName">Spreafico</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8282-7658" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8282-7658</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">29</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Spreafico/Roberto" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">29</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Ana</span> <span property="familyName">da Silva Filipe</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ana</span> <span property="familyName">da Silva Filipe</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9442-2903" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9442-2903</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/da+Silva+Filipe/Ana" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Jason A.</span> <span property="familyName">Wojcechowskyj</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jason A.</span> <span property="familyName">Wojcechowskyj</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wojcechowskyj/Jason+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Chris</span> <span property="familyName">Davis</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Chris</span> <span property="familyName">Davis</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Davis/Chris" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Luca</span> <span property="familyName">Piccoli</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Luca</span> <span property="familyName">Piccoli</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1085-6502" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1085-6502</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Piccoli/Luca" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">David J.</span> <span property="familyName">Pascall</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David J.</span> <span property="familyName">Pascall</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7543-0860" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7543-0860</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Biodiversity, Animal Health and Comparative Medicine, Boyd Orr Centre for Population and Ecosystem Health, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Pascall/David+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Josh</span> <span property="familyName">Dillen</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Josh</span> <span property="familyName">Dillen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dillen/Josh" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Spyros</span> <span property="familyName">Lytras</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Spyros</span> <span property="familyName">Lytras</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4202-6682" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4202-6682</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Lytras/Spyros" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Nadine</span> <span property="familyName">Czudnochowski</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nadine</span> <span property="familyName">Czudnochowski</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3146-4110" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3146-4110</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Czudnochowski/Nadine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Rajiv</span> <span property="familyName">Shah</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rajiv</span> <span property="familyName">Shah</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2827-5108" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2827-5108</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Shah/Rajiv" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Marcel</span> <span property="familyName">Meury</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marcel</span> <span property="familyName">Meury</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8718-2374" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8718-2374</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Meury/Marcel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Natasha</span> <span property="familyName">Jesudason</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Natasha</span> <span property="familyName">Jesudason</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Jesudason/Natasha" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Anna</span> <span property="familyName">De Marco</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anna</span> <span property="familyName">De Marco</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/De+Marco/Anna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Kathy</span> <span property="familyName">Li</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kathy</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Kathy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Jessica</span> <span property="familyName">Bassi</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jessica</span> <span property="familyName">Bassi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Bassi/Jessica" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Aine</span> <span property="familyName">O’Toole</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aine</span> <span property="familyName">O’Toole</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK</span></div></div><div class="core-author-link"><a href="/authored-by/O%E2%80%99Toole/Aine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Dora</span> <span property="familyName">Pinto</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dora</span> <span property="familyName">Pinto</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Pinto/Dora" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Rachel M.</span> <span property="familyName">Colquhoun</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rachel M.</span> <span property="familyName">Colquhoun</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5577-9897" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5577-9897</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Colquhoun/Rachel+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Katja</span> <span property="familyName">Culap</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katja</span> <span property="familyName">Culap</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0956-0018" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0956-0018</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Culap/Katja" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Ben</span> <span property="familyName">Jackson</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ben</span> <span property="familyName">Jackson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Jackson/Ben" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Fabrizia</span> <span property="familyName">Zatta</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fabrizia</span> <span property="familyName">Zatta</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Zatta/Fabrizia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Andrew</span> <span property="familyName">Rambaut</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Andrew</span> <span property="familyName">Rambaut</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Rambaut/Andrew" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Stefano</span> <span property="familyName">Jaconi</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stefano</span> <span property="familyName">Jaconi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Jaconi/Stefano" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Vattipally B.</span> <span property="familyName">Sreenu</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Vattipally B.</span> <span property="familyName">Sreenu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Sreenu/Vattipally+B" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Jay</span> <span property="familyName">Nix</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jay</span> <span property="familyName">Nix</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Molecular Biology Consortium, Advanced Light Source, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nix/Jay" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au29" aria-controls="axel_author_au29"><span property="givenName">Ivy</span> <span property="familyName">Zhang</span></a><div id="au29" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ivy</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Tri-Institutional PhD Program in Computational Biology and Medicine, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Ivy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au30" aria-controls="axel_author_au30"><span property="givenName">Ruth F.</span> <span property="familyName">Jarrett</span></a><div id="au30" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ruth F.</span> <span property="familyName">Jarrett</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0390-3478" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0390-3478</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Jarrett/Ruth+F" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au31" aria-controls="axel_author_au31"><span property="givenName">William G.</span> <span property="familyName">Glass</span></a><div id="au31" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">William G.</span> <span property="familyName">Glass</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Glass/William+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au32" aria-controls="axel_author_au32"><span property="givenName">Martina</span> <span property="familyName">Beltramello</span></a><div id="au32" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Martina</span> <span property="familyName">Beltramello</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Beltramello/Martina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au33" aria-controls="axel_author_au33"><span property="givenName">Kyriaki</span> <span property="familyName">Nomikou</span></a><div id="au33" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kyriaki</span> <span property="familyName">Nomikou</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7013-1853" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7013-1853</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Nomikou/Kyriaki" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au34" aria-controls="axel_author_au34"><span property="givenName">Matteo</span> <span property="familyName">Pizzuto</span></a><div id="au34" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matteo</span> <span property="familyName">Pizzuto</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Pizzuto/Matteo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au35" aria-controls="axel_author_au35"><span property="givenName">Lily</span> <span property="familyName">Tong</span></a><div id="au35" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lily</span> <span property="familyName">Tong</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3882-3339" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3882-3339</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Tong/Lily" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au36" aria-controls="axel_author_au36"><span property="givenName">Elisabetta</span> <span property="familyName">Cameroni</span></a><div id="au36" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elisabetta</span> <span property="familyName">Cameroni</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Cameroni/Elisabetta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au37" aria-controls="axel_author_au37"><span property="givenName">Tristan I.</span> <span property="familyName">Croll</span></a><div id="au37" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tristan I.</span> <span property="familyName">Croll</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Cambridge Institute for Medical Research, Department of Haematology, University of Cambridge, Cambridge CB2 0XY, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Croll/Tristan+I" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au38" aria-controls="axel_author_au38"><span property="givenName">Natasha</span> <span property="familyName">Johnson</span></a><div id="au38" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Natasha</span> <span property="familyName">Johnson</span> <a class="orcid-id" href="https://orcid.org/0000-0002-4246-6082" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-4246-6082</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Johnson/Natasha" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au39" aria-controls="axel_author_au39"><span property="givenName">Julia</span> <span property="familyName">Di Iulio</span></a><div id="au39" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Julia</span> <span property="familyName">Di Iulio</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Di+Iulio/Julia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au40" aria-controls="axel_author_au40"><span property="givenName">Arthur</span> <span property="familyName">Wickenhagen</span></a><div id="au40" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Arthur</span> <span property="familyName">Wickenhagen</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5470-1505" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5470-1505</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Wickenhagen/Arthur" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au41" aria-controls="axel_author_au41"><span property="givenName">Alessandro</span> <span property="familyName">Ceschi</span></a><div id="au41" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alessandro</span> <span property="familyName">Ceschi</span> <a class="orcid-id" href="https://orcid.org/0000-0002-4308-0405" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-4308-0405</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Faculty of Biomedical Sciences, Università della Svizzera italiana, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Clinical Pharmacology and Toxicology, Institute of Pharmacological Sciences of Southern Switzerland, Ente Ospedaliero Cantonale, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Clinical Pharmacology and Toxicology, University Hospital Zurich, 8091 Zurich, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Ceschi/Alessandro" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au42" aria-controls="axel_author_au42"><span property="givenName">Aoife M.</span> <span property="familyName">Harbison</span></a><div id="au42" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aoife M.</span> <span property="familyName">Harbison</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Hamilton Institute, Maynooth University, Maynooth, Ireland</span></div></div><div class="core-author-link"><a href="/authored-by/Harbison/Aoife+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au43" aria-controls="axel_author_au43"><span property="givenName">Daniel</span> <span property="familyName">Mair</span></a><div id="au43" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Daniel</span> <span property="familyName">Mair</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Mair/Daniel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au44" aria-controls="axel_author_au44"><span property="givenName">Paolo</span> <span property="familyName">Ferrari</span></a><div id="au44" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Paolo</span> <span property="familyName">Ferrari</span> <a class="orcid-id" href="https://orcid.org/0000-0002-9619-3104" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-9619-3104</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Nephrology, Ospedale Civico Lugano, Ente Ospedaliero Cantonale, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span property="name">Prince of Wales Hospital Clinical School, University of New South Wales, Sydney, NSW 2052, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Ferrari/Paolo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au45" aria-controls="axel_author_au45"><span property="givenName">Katherine</span> <span property="familyName">Smollett</span></a><div id="au45" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katherine</span> <span property="familyName">Smollett</span> <a class="orcid-id" href="https://orcid.org/0000-0001-7961-7377" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-7961-7377</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Smollett/Katherine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au46" aria-controls="axel_author_au46"><span property="givenName">Federica</span> <span property="familyName">Sallusto</span></a><div id="au46" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Federica</span> <span property="familyName">Sallusto</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Research in Biomedicine, Università della Svizzera italiana, 6500 Bellinzona, Switzerland</span></div><div property="affiliation" typeof="Organization"><span property="name">ETH Institute of Microbiology, ETH Zurich, 8093 Zürich, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Sallusto/Federica" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">17</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">18</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au47" aria-controls="axel_author_au47"><span property="givenName">Stephen</span> <span property="familyName">Carmichael</span></a><div id="au47" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephen</span> <span property="familyName">Carmichael</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4704-4185" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4704-4185</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Carmichael/Stephen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au48" aria-controls="axel_author_au48"><span property="givenName">Christian</span> <span property="familyName">Garzoni</span></a><div id="au48" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christian</span> <span property="familyName">Garzoni</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinic of Internal Medicine and Infectious Diseases, Clinica Luganese Moncucco, 6900 Lugano, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Garzoni/Christian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au49" aria-controls="axel_author_au49"><span property="givenName">Jenna</span> <span property="familyName">Nichols</span></a><div id="au49" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jenna</span> <span property="familyName">Nichols</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0093-7170" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0093-7170</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Nichols/Jenna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au50" aria-controls="axel_author_au50"><span property="givenName">Massimo</span> <span property="familyName">Galli</span></a><div id="au50" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Massimo</span> <span property="familyName">Galli</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy</span></div></div><div class="core-author-link"><a href="/authored-by/Galli/Massimo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au51" aria-controls="axel_author_au51"><span property="givenName">Joseph</span> <span property="familyName">Hughes</span></a><div id="au51" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joseph</span> <span property="familyName">Hughes</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2556-2563" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2556-2563</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Hughes/Joseph" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au52" aria-controls="axel_author_au52"><span property="givenName">Agostino</span> <span property="familyName">Riva</span></a><div id="au52" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Agostino</span> <span property="familyName">Riva</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy</span></div></div><div class="core-author-link"><a href="/authored-by/Riva/Agostino" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au53" aria-controls="axel_author_au53"><span property="givenName">Antonia</span> <span property="familyName">Ho</span></a><div id="au53" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Antonia</span> <span property="familyName">Ho</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Ho/Antonia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au54" aria-controls="axel_author_au54"><span property="givenName">Marco</span> <span property="familyName">Schiuma</span></a><div id="au54" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marco</span> <span property="familyName">Schiuma</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy</span></div></div><div class="core-author-link"><a href="/authored-by/Schiuma/Marco" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au55" aria-controls="axel_author_au55"><span property="givenName">Malcolm G.</span> <span property="familyName">Semple</span></a><div id="au55" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Malcolm G.</span> <span property="familyName">Semple</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9700-0418" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9700-0418</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">NIHR Health Protection Research Unit in Emerging and Zoonotic Infections, Institute of Infection, Veterinary and Ecological Sciences, Faculty of Health and Life Sciences, University of Liverpool, Liverpool L69 7BE, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Respiratory Medicine, Alder Hey Children’s Hospital, Liverpool L12 2AP, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Semple/Malcolm+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">21</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">22</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au56" aria-controls="axel_author_au56"><span property="givenName">Peter J.M.</span> <span property="familyName">Openshaw</span></a><div id="au56" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Peter J.M.</span> <span property="familyName">Openshaw</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7220-2555" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7220-2555</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">National Heart and Lung Institute, Imperial College London, London SW3 6LY, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Openshaw/Peter+J+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">23</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au57" aria-controls="axel_author_au57"><span property="givenName">Elisa</span> <span property="familyName">Fadda</span></a><div id="au57" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elisa</span> <span property="familyName">Fadda</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Chemistry and Hamilton Institute, Maynooth University, Maynooth, Ireland</span></div></div><div class="core-author-link"><a href="/authored-by/Fadda/Elisa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au58" aria-controls="axel_author_au58"><span property="givenName">J. Kenneth</span> <span property="familyName">Baillie</span></a><div id="au58" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">J. Kenneth</span> <span property="familyName">Baillie</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute, University of Edinburgh, Edinburgh EH25 9RG, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Intensive Care Unit, Royal Infirmary Edinburgh, Edinburgh EH16 4SA, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Baillie/J+Kenneth" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">24</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">25</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au59" aria-controls="axel_author_au59"><span property="givenName">John D.</span> <span property="familyName">Chodera</span></a><div id="au59" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">John D.</span> <span property="familyName">Chodera</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chodera/John+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_collab0010" aria-controls="axel_author_collab0010"><span property="name">The ISARIC4C Investigators</span></a><div id="collab0010" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_collab0010"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="name">The ISARIC4C Investigators</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">ISARIC4C Investigators</span></div></div><div class="core-author-link"><a href="/authored-by/group/The+ISARIC4C+Investigators" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">26</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_collab0015" aria-controls="axel_author_collab0015"><span property="name">the COVID-19 Genomics UK (COG-UK) Consortium</span></a><div id="collab0015" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_collab0015"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="name">the COVID-19 Genomics UK (COG-UK) Consortium</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">https://www.cogconsortium.uk</span></div></div><div class="core-author-link"><a href="/authored-by/group/the+COVID-19+Genomics+UK+%28COG-UK%29+Consortium" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">27</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au60" aria-controls="axel_author_au60"><span property="givenName">Suzannah J.</span> <span property="familyName">Rihn</span></a><div id="au60" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Suzannah J.</span> <span property="familyName">Rihn</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9495-4056" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9495-4056</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Rihn/Suzannah+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au61" aria-controls="axel_author_au61"><span property="givenName">Samantha J.</span> <span property="familyName">Lycett</span></a><div id="au61" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Samantha J.</span> <span property="familyName">Lycett</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Roslin Institute, University of Edinburgh, Edinburgh EH25 9RG, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Lycett/Samantha+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">24</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au62" aria-controls="axel_author_au62"><span property="givenName">Herbert W.</span> <span property="familyName">Virgin</span></a><div id="au62" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Herbert W.</span> <span property="familyName">Virgin</span> <a class="orcid-id" href="https://orcid.org/0000-0001-8580-7628" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-8580-7628</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Washington University School of Medicine, Saint Louis, MO 63110, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Virgin/Herbert+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">28</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au63" aria-controls="axel_author_au63"><span property="givenName">Amalio</span> <span property="familyName">Telenti</span></a><div id="au63" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Amalio</span> <span property="familyName">Telenti</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Telenti/Amalio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au64" aria-controls="axel_author_au64"><span property="givenName">Davide</span> <span property="familyName">Corti</span></a><div id="au64" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Davide</span> <span property="familyName">Corti</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5797-1364" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5797-1364</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Corti/Davide" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au65" aria-controls="axel_author_au65"><span property="givenName">David L.</span> <span property="familyName">Robertson</span></a><div id="au65" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David L.</span> <span property="familyName">Robertson</span> <a class="orcid-id" href="https://orcid.org/0000-0001-6338-0221" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-6338-0221</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:david.l.robertson@glasgow.ac.uk" property="email" title="Send email to David L. Robertson">david.l.robertson@glasgow.ac.uk</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Robertson/David+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup> <a id="eadd0010" href="mailto:david.l.robertson@glasgow.ac.uk" property="email" title="Send email to David L. Robertson">david.l.robertson@glasgow.ac.uk</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au66" aria-controls="axel_author_au66"><span property="givenName">Gyorgy</span> <span property="familyName">Snell</span></a><div id="au66" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gyorgy</span> <span property="familyName">Snell</span> <a class="orcid-id" href="https://orcid.org/0000-0003-1475-659X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-1475-659X</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:gsnell@vir.bio" property="email" title="Send email to Gyorgy Snell">gsnell@vir.bio</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Snell/Gyorgy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">30</a></sup> <a id="eadd0015" href="mailto:gsnell@vir.bio" property="email" title="Send email to Gyorgy Snell">gsnell@vir.bio</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Glasgow G61 1QH, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Clinical Research, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Vir Biotechnology, San Francisco, CA 94158, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Humabs Biomed SA, a subsidiary of Vir Biotechnology, 6500 Bellinzona, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Institute of Biodiversity, Animal Health and Comparative Medicine, Boyd Orr Centre for Population and Ecosystem Health, University of Glasgow, Glasgow G61 1QH, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh EH9 3FL, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Molecular Biology Consortium, Advanced Light Source, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Tri-Institutional PhD Program in Computational Biology and Medicine, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Cambridge Institute for Medical Research, Department of Haematology, University of Cambridge, Cambridge CB2 0XY, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Faculty of Biomedical Sciences, Università della Svizzera italiana, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Division of Clinical Pharmacology and Toxicology, Institute of Pharmacological Sciences of Southern Switzerland, Ente Ospedaliero Cantonale, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">Department of Clinical Pharmacology and Toxicology, University Hospital Zurich, 8091 Zurich, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Department of Chemistry and Hamilton Institute, Maynooth University, Maynooth, Ireland</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">Department of Nephrology, Ospedale Civico Lugano, Ente Ospedaliero Cantonale, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Prince of Wales Hospital Clinical School, University of New South Wales, Sydney, NSW 2052, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">17</span><span id="aff17" property="name">Institute for Research in Biomedicine, Università della Svizzera italiana, 6500 Bellinzona, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">18</span><span id="aff18" property="name">ETH Institute of Microbiology, ETH Zurich, 8093 Zürich, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">19</span><span id="aff19" property="name">Clinic of Internal Medicine and Infectious Diseases, Clinica Luganese Moncucco, 6900 Lugano, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">20</span><span id="aff20" property="name">III Division of Infectious Diseases, ASST Fatebenefratelli Sacco, Luigi Sacco Hospital, 20157 Milan, Italy</span></div><div property="affiliation" typeof="Organization"><span class="label">21</span><span id="aff21" property="name">NIHR Health Protection Research Unit in Emerging and Zoonotic Infections, Institute of Infection, Veterinary and Ecological Sciences, Faculty of Health and Life Sciences, University of Liverpool, Liverpool L69 7BE, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">22</span><span id="aff22" property="name">Respiratory Medicine, Alder Hey Children’s Hospital, Liverpool L12 2AP, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">23</span><span id="aff23" property="name">National Heart and Lung Institute, Imperial College London, London SW3 6LY, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">24</span><span id="aff24" property="name">The Roslin Institute, University of Edinburgh, Edinburgh EH25 9RG, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">25</span><span id="aff25" property="name">Intensive Care Unit, Royal Infirmary Edinburgh, Edinburgh EH16 4SA, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">26</span><span id="aff26" property="name">ISARIC4C Investigators</span></div><div property="affiliation" typeof="Organization"><span class="label">27</span><span id="aff27" property="name">https://www.cogconsortium.uk</span></div><div property="affiliation" typeof="Organization"><span class="label">28</span><span id="aff28" property="name">Washington University School of Medicine, Saint Louis, MO 63110, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">29</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">30</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received October 30, 2020</span>; <span>Revised December 12, 2020</span>; <span>Accepted January 22, 2021</span>; <span>Published online January 28, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.01.037" property="sameAs" target="_blank">10.1016/j.cell.2021.01.037 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421000805" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 The Authors. Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution (CC BY 4.0)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.01.037" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0006-7"><img src="/cms/asset/30f73b63-b48b-4f99-8afd-18334d02293c/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 5"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900080-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★methods</a></li><li><a href="#sec-5">Consortia</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (4)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900080-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.01.037">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900080-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.01.037&amp;pii=S0092-8674%2821%2900080-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900080-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.01.037">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.01.037" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421000805&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421000805&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=5&amp;pii=S0092-8674%2820%29X0006-7" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00013-1" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00078-7" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★methods</a></li><li><a href="#sec-5" aria-current="false">Consortia</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental information (4)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">The receptor-binding motif (RBM) is a highly variable region of SARS-CoV-2 spike</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">RBM mutation N439K has emerged independently in multiple lineages</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">N439K increases spike affinity for hACE2; viral fitness and disease are unchanged</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">N439K confers resistance to several mAbs and escapes some polyclonal responses</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">SARS-CoV-2 can mutate and evade immunity, with consequences for efficacy of emerging vaccines and antibody therapeutics. Here, we demonstrate that the immunodominant SARS-CoV-2 spike (S) receptor binding motif (RBM) is a highly variable region of S and provide epidemiological, clinical, and molecular characterization of a prevalent, sentinel RBM mutation, N439K. We demonstrate N439K S protein has enhanced binding affinity to the hACE2 receptor, and N439K viruses have similar <i>in&nbsp;vitro</i> replication fitness and cause infections with similar clinical outcomes as compared to wild type. We show the N439K mutation confers resistance against several neutralizing monoclonal antibodies, including one authorized for emergency use by the US Food and Drug Administration (FDA), and reduces the activity of some polyclonal sera from persons recovered from infection. Immune evasion mutations that maintain virulence and fitness such as N439K can emerge within SARS-CoV-2 S, highlighting the need for ongoing molecular surveillance to guide development and usage of vaccines and therapeutics.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/e64b2b53-9d11-413d-bd9e-0913f4c17173/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.01.037/asset/08af3be4-d98a-4a9a-b194-25b6764957ee/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/e64b2b53-9d11-413d-bd9e-0913f4c17173/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;SARS-CoV-2&quot;" alt="Search on this keyword" rel="nofollow">SARS-CoV-2</a></li><li><a href="/action/doSearch?AllField=&quot;COVID-19&quot;" alt="Search on this keyword" rel="nofollow">COVID-19</a></li><li><a href="/action/doSearch?AllField=&quot;mutation&quot;" alt="Search on this keyword" rel="nofollow">mutation</a></li><li><a href="/action/doSearch?AllField=&quot;N439K&quot;" alt="Search on this keyword" rel="nofollow">N439K</a></li><li><a href="/action/doSearch?AllField=&quot;variant&quot;" alt="Search on this keyword" rel="nofollow">variant</a></li><li><a href="/action/doSearch?AllField=&quot;Spike&quot;" alt="Search on this keyword" rel="nofollow">Spike</a></li><li><a href="/action/doSearch?AllField=&quot;receptor+binding+motif&quot;" alt="Search on this keyword" rel="nofollow">receptor binding motif</a></li><li><a href="/action/doSearch?AllField=&quot;monoclonal+antibody+escape&quot;" alt="Search on this keyword" rel="nofollow">monoclonal antibody escape</a></li><li><a href="/action/doSearch?AllField=&quot;protein+structure&quot;" alt="Search on this keyword" rel="nofollow">protein structure</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">SARS-CoV-2, the cause of COVID-19, emerged in late 2019 and expanded globally, resulting in over 82 million confirmed cases as of the end of 2020. Molecular epidemiology studies across the world have generated over 330,000 viral genomic sequences, shared with unprecedented speed via the GISAID Initiative (<a id="intref0010" href="https://gisaid.org" target="_blank">https://gisaid.org</a>). These data are essential for monitoring virus transmission and spread (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Meredith et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Meredith, L.W. ∙ Hamilton, W.L. ∙ Warne, B. ...</div><div><strong>Rapid implementation of SARS-CoV-2 sequencing to investigate cases of health-care associated COVID-19: a prospective genomic surveillance study</strong></div><div><em>Lancet Infect. Dis.</em> 2020; <strong>20</strong>:P1263-P1271</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS1473-3099%2820%2930562-4&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS1473-3099%2820%2930562-4&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089295953" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32679081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2820%2930562-4&amp;pmid=32679081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Of special interest is the evolution of the SARS-CoV-2 surface protein, spike (S), which is responsible for viral entry via its interaction with the human angiotensin-converting enzyme 2 (hACE2) receptor on host cells. The S protein is the target of neutralizing antibodies generated by infection (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Jiang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Jiang, S. ∙ Hillyer, C. ∙ Du, L.</div><div><strong>Neutralizing Antibodies against SARS-CoV-2 and Other Human Coronaviruses</strong></div><div><em>Trends Immunol.</em> 2020; <strong>41</strong>:355-359</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.it.2020.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.it.2020.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85082822449" target="_blank" aria-label="Open Scopus in new tab">Scopus (618)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32249063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2020.03.007&amp;pmid=32249063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or vaccination (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Folegatti et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Folegatti, P.M. ∙ Ewer, K.J. ∙ Aley, P.K. ..., Oxford COVID Vaccine Trial Group</div><div><strong>Safety and immunogenicity of the ChAdOx1 nCoV-19 vaccine against SARS-CoV-2: a preliminary report of a phase 1/2, single-blind, randomised controlled trial</strong></div><div><em>Lancet.</em> 2020; <strong>396</strong>:467-478</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS0140-6736%2820%2931604-4&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS0140-6736%2820%2931604-4&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088146774" target="_blank" aria-label="Open Scopus in new tab">Scopus (1772)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32702298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2820%2931604-4&amp;pmid=32702298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Jackson et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Jackson, L.A. ∙ Anderson, E.J. ∙ Rouphael, N.G. ...</div><div><strong>An mRNA Vaccine against SARS-CoV-2 - Preliminary Report</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2020; <strong>383</strong>:1920-1931</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2022483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096120857" target="_blank" aria-label="Open Scopus in new tab">Scopus (2300)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32663912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2022483&amp;pmid=32663912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Keech et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Keech, C. ∙ Albert, G. ∙ Cho, I. ...</div><div><strong>Phase 1-2 Trial of a SARS-CoV-2 Recombinant Spike Protein Nanoparticle Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2020; <strong>383</strong>:2320-2332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2026920" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091689534" target="_blank" aria-label="Open Scopus in new tab">Scopus (839)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32877576/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2026920&amp;pmid=32877576" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as well as monoclonal antibody (mAb) drugs currently in clinical trials and/or approved for Emergency Use Authorization (EUA) by the US Food and Drug Administration (FDA) (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Chen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Chen, P. ∙ Nirula, A. ∙ Heller, B. ...</div><div><strong>SARS-CoV-2 Neutralizing Antibody LY-CoV555 in Outpatients with Covid-19</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:229-237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2029849" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098855067" target="_blank" aria-label="Open Scopus in new tab">Scopus (939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33113295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2029849&amp;pmid=33113295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Hansen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Hansen, J. ∙ Baum, A. ∙ Pascal, K.E. ...</div><div><strong>Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1010-1014</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089787231" target="_blank" aria-label="Open Scopus in new tab">Scopus (833)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0827&amp;pmid=32540901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Jones et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Jones, B.E. ∙ Brown-Augsburger, P.L. ∙ Corbett, K.S. ...</div><div><strong>LY-CoV555, a rapidly isolated potent neutralizing antibody, provides protection in a non-human primate model of SARS-CoV-2 infection</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.09.30.318972" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85102722540" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.09.30.318972" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Pinto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0035" role="paragraph">A SARS-CoV-2&nbsp;S amino acid change, D614G, is now dominant in most places around the globe (<span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Korber et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827.e19</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Studies <i>in&nbsp;vitro</i> indicate that this mutation confers greater infectivity while molecular epidemiology correlates it with an increase in transmissibility with no evidence to date for increased virulence (<span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Hou et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Hou, Y.J. ∙ Chiba, S. ∙ Halfmann, P. ...</div><div><strong>SARS-CoV-2 D614G variant exhibits efficient replication ex&nbsp;vivo and transmission in&nbsp;vivo</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:1464-1468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe8499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098742011" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33184236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe8499&amp;pmid=33184236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Hu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Hu, J. ∙ He, C.-L. ∙ Gao, Q.-Z. ...</div><div><strong>D614G mutation of SARS-CoV-2 spike protein enhances viral infectivity</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.06.20.161323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098900825" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.06.20.161323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Korber et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827.e19</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Zhang et al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Zhang, L. ∙ Jackson, C.B. ∙ Mou, H. ...</div><div><strong>SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity</strong></div><div><em>Nature Communications.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19808-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096795416" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19808-4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Amino acid 614 is located outside the receptor binding domain (RBD) of S, the domain targeted by 90% of neutralizing antibody activity in serum of SARS-CoV-2 survivors (<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Initial studies suggest that D614G viruses exhibit increased sensitivity to neutralizing antibodies, likely due to the effect of the mutation on the molecular dynamics of the S protein (<span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Hou et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Hou, Y.J. ∙ Chiba, S. ∙ Halfmann, P. ...</div><div><strong>SARS-CoV-2 D614G variant exhibits efficient replication ex&nbsp;vivo and transmission in&nbsp;vivo</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:1464-1468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe8499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098742011" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33184236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe8499&amp;pmid=33184236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Weissman et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Weissman, D. ∙ Alameh, M.G. ∙ de Silva, T. ...</div><div><strong>D614G Spike Mutation Increases SARS CoV-2 Susceptibility to Neutralization</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:23-31.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097884640" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33306985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.012&amp;pmid=33306985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Yurkovetskiy et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Yurkovetskiy, L. ∙ Wang, X. ∙ Pascal, K.E. ...</div><div><strong>Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:739-751.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.032&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.032&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091680714" target="_blank" aria-label="Open Scopus in new tab">Scopus (677)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.032&amp;pmid=32991842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, this now dominant variant is unlikely to jeopardize natural or vaccine-derived antibody-mediated immunity generated in response to D614&nbsp;S protein.</div><div id="p0040" role="paragraph">The low numbers of novel mutations reaching high frequency in sequenced SARS-CoV-2 genomes relates to the moderate intrinsic error rate of SARS-CoV-2 RNA replication (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Li et&nbsp;al., 2020c</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Li, X. ∙ Wang, W. ∙ Zhao, X. ...</div><div><strong>Transmission dynamics and evolutionary history of 2019-nCoV</strong></div><div><em>J.&nbsp;Med. Virol.</em> 2020; <strong>92</strong>:501-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jmv.25701" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85079436157" target="_blank" aria-label="Open Scopus in new tab">Scopus (226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32027035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjmv.25701&amp;pmid=32027035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Robson et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Robson, F. ∙ Khan, K.S. ∙ Le, T.K. ...</div><div><strong>Coronavirus RNA Proofreading: Molecular Basis and Therapeutic Targeting</strong></div><div><em>Mol. Cell.</em> 2020; <strong>79</strong>:710-727</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.molcel.2020.07.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.molcel.2020.07.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089858704" target="_blank" aria-label="Open Scopus in new tab">Scopus (241)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32853546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2020.07.027&amp;pmid=32853546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Nevertheless, the increasing number of infected individuals and the large reservoir of hosts susceptible to infection increase the likelihood that novel variants that impact vaccine and therapeutic development will emerge and spread by chance. Moreover, the full impact of immune selection, which can drive variant selection, has not yet influenced the pandemic, because herd immunity has not been attained. As population immunity increases and vaccines are deployed at scale, this will very likely change. The potential for circulating viral variants to derail promising vaccine or antibody-based prophylactics or treatments, even in the absence of selective pressure from the drug or vaccine, is demonstrated by the failure of a phase III clinical trial of a mAb targeting the respiratory syncytial virus (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Simões et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Simões, E.A.F. ∙ Forleo-Neto, E. ∙ Geba, G.P. ...</div><div><strong>Suptavumab for the Prevention of Medically Attended Respiratory Syncytial Virus Infection in Preterm Infants</strong></div><div><em>Clin. Infect. Dis.</em> 2020; ciaa951</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/cid/ciaa951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85122546142" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32897368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcid%2Fciaa951&amp;pmid=32897368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the need for new influenza vaccines on a yearly basis. It is therefore critical to understand whether and how SARS-CoV-2 may evolve to evade antibody-dependent immunity.</div><div id="p0045" role="paragraph">Here, we examine the immunodominant SARS-CoV-2 receptor binding motif (RBM), the primary target of the neutralizing Ab response within the RBD (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and find it to be a highly variable region of the S protein in circulating viruses. To understand the implications of this structural plasticity, which could allow the RBD to accommodate amino acids changes that&nbsp;could contribute to immune evasion, we defined the clinical and epidemiological impact, molecular features, and immune response to the RBM mutation N439K. This amino acid replacement has arisen independently multiple times, and in two cases formed lineages of more than 500 sequences. As of January 6, 2021, it was observed in 34 countries and was the second most commonly observed RBD mutation worldwide, and the sixth most common S mutation. We find that the N439K mutation results in enhanced RBD affinity for hACE2, it is associated with a similar clinical spectrum of disease and slightly higher viral loads <i>in&nbsp;vivo</i> compared to viruses with the wild-type (WT) N439 residue, and it results in immune escape from polyclonal sera from&nbsp;a proportion of recovered individuals and some neutralizing mAbs.</div><div id="p0050" role="paragraph">N439K provides a sentinel example of immune escape, indicating that RBM variants must be evaluated when considering vaccines and the therapeutic or prophylactic use of mAbs. Long-term control of the pandemic with vaccines will require systematic monitoring of immune escape variants and may require new vaccine preparations that address the variants circulating globally.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>The RBM is a variable region of the SARS-CoV-2&nbsp;S protein</h3><div id="p0055" role="paragraph">Competing pressures influence the evolution of the S RBM. First, the RBM mediates viral entry (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Shang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Shang, J. ∙ Ye, G. ∙ Shi, K. ...</div><div><strong>Structural basis of receptor recognition by SARS-CoV-2</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:221-224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2179-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85083343318" target="_blank" aria-label="Open Scopus in new tab">Scopus (2572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32225175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2179-y&amp;pmid=32225175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Walls et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Walls, A.C. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:281-292.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.02.058&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.02.058&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081283286" target="_blank" aria-label="Open Scopus in new tab">Scopus (5950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32155444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.02.058&amp;pmid=32155444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Wrapp et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Wrapp, D. ∙ Wang, N. ∙ Corbett, K.S. ...</div><div><strong>Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:1260-1263</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb2507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081904519" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32075877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb2507&amp;pmid=32075877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and therefore must maintain sufficient affinity to engage the entry receptor hACE2. Second, it is a major target of neutralizing antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref65-3" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-3" aria-controls="sref65-3">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Robbiani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Robbiani, D.F. ∙ Gaebler, C. ∙ Muecksch, F. ...</div><div><strong>Convergent antibody responses to SARS-CoV-2 in convalescent individuals</strong></div><div><em>Nature.</em> 2020; <strong>584</strong>:437-442</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2456-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85086589703" target="_blank" aria-label="Open Scopus in new tab">Scopus (1262)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32555388/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2456-9&amp;pmid=32555388" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Rogers et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Rogers, T.F. ∙ Zhao, F. ∙ Huang, D. ...</div><div><strong>Isolation of potent SARS-CoV-2 neutralizing antibodies and protection from disease in a small animal model</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:956-963</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc7520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087691308" target="_blank" aria-label="Open Scopus in new tab">Scopus (891)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc7520&amp;pmid=32540903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Wec et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Wec, A.Z. ∙ Wrapp, D. ∙ Herbert, A.S. ...</div><div><strong>Broad neutralization of SARS-related viruses by human monoclonal antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:731-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc7424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087044294" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc7424&amp;pmid=32540900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and so would be a primary location for the emergence of immune escape mutations. We set out to understand these competing pressures by&nbsp;evaluating the landscape of RBM sequence divergence observed in circulating SARS-CoV-2 variants and in other viruses of the <i>Sarbecovirus</i> lineage.</div><div id="p0060" role="paragraph">We used re-refined published X-ray structures of SARS-CoV and SARS-CoV-2 RBD:hACE2 complexes (<span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Lan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Lan, J. ∙ Ge, J. ∙ Yu, J. ...</div><div><strong>Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:215-220</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2180-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85083295918" target="_blank" aria-label="Open Scopus in new tab">Scopus (4081)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32225176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2180-5&amp;pmid=32225176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Li et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Li, F. ∙ Li, W. ∙ Farzan, M. ...</div><div><strong>Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</strong></div><div><em>Science.</em> 2005; <strong>309</strong>:1864-1868</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1116480" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-24944498409" target="_blank" aria-label="Open Scopus in new tab">Scopus (1524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16166518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1116480&amp;pmid=16166518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to define the RBM residues using a 6&nbsp;Å distance cutoff (<a id="crosref0590" href="#fig1">Figures 1</a>A–1D and <a id="crosref0595" href="#figs1">S1</a>A, 2). We evaluated SARS-CoV-2 genomic sequences deposited in GISAID as of November 30th, 2020 and observed a high number of variants occurring in the RBM. To understand how the variability of the RBM compares to the variability of the entire RBD and the whole S protein, we evaluated well-defined S protein domains: within S1, the N-terminal domain (NTD) and the RBD (further split into RBM and non-RBM), and the S2 domain. Analysis of entropy, which estimates sequence variability at a given position in a protein alignment, identified the RBM as a highly variable region of the RBD and of the entire S protein (<a id="crosref0600" href="#fig1">Figures 1</a>B–1D), with a median entropy within the top 10% of equivalently sized sets of randomly sampled residues (<a id="crosref0605" href="#fig1">Figure&nbsp;1</a>B). This result is confirmed by an analysis of sequence variability that is not weighted by total counts of&nbsp;each variant, thereby capturing the diversity of circulating variants&nbsp;with mitigated bias toward oversampled variants (<a id="crosref0610" href="#figs1">Figure&nbsp;S1</a>A).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/a5cc9bf7-4dec-4e00-9eb1-3ed5c8a46eae/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.01.037/asset/afd18e18-f3ee-464d-b3e8-2c0841f11e14/main.assets/gr1.jpg" height="463" width="504" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/a5cc9bf7-4dec-4e00-9eb1-3ed5c8a46eae/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">The RBM exhibits significant natural diversity in circulating SARS-CoV-2 viruses</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">SARS-CoV-2 variants (retrieved from CoV-GLUE) are based on 209,239 high-quality sequences downloaded from GISAID on November 30, 2020.</div><div id="fspara0020" role="paragraph">(A) Structure of the SARS-CoV-2 RBD-hACE2 complex (PDB: <a id="intref0395" href="http://firstglance.jmol.org/fg.htm?mol6M0J" target="_blank">6M0J</a>) highlighting the RBM (blue) and residue N439 (yellow).</div><div id="fspara0025" role="paragraph">(B) Thirty-four residues (the size of the RBM) were randomly sampled without replacement 50,000 times from the mature S protein (excluding the RBM). Median entropies were computed for each draw. The resulting 50,000 median entropies were used to build the entropy distribution of residues other than the RBM. The top 10% medians are highlighted in red. The median entropy of RBM residues was compared with the non-RBM entropy distribution to determine the variability of the RBM relative to non-RBM residues. To allow for a fair comparison, sampling was performed without enforcing residue contiguity, as the RBM is not contiguous in sequence space. Therefore, in any given sample, residues are unlikely to share any functional relationship.</div><div id="fspara0030" role="paragraph">(C) Per-residue entropies of the mature S protein were smoothed by plotting medians of a 25-aa center-aligned sliding window. Smoothing allows visualizing local peaks of variability. The RBM residues and the NTD, RBD, and S2 domains are highlighted. Due to the non-contiguous nature of the RBM in sequence space, the sliding window median at RBM residues is diluted by neighboring non-RBM residues.</div><div id="fspara0035" role="paragraph">(D) Boxplot of per-residue entropies in four S domains (or full mature S protein). The lower and upper hinges correspond to the first and third quartiles. The lower/upper whiskers extend from the hinge to the smallest/largest value no further than 1.5 times the inter-quartile range. Outliers beyond the end of the whiskers are not plotted but are retained for statistical testing. Pairwise comparisons by Mann-Whitney U tests. p value thresholds are 0.05 (<sup>∗</sup>), 0.01 (<sup>∗∗</sup>) and 0.001 (<sup>∗∗∗</sup>); ns, not significant.</div><div id="fspara0040" role="paragraph">See also <a id="crosref1775" href="#figs1">Figures S1</a> and <a id="crosref1780" href="#figs2">S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/b4f0584f-5b8a-4222-80c1-179e5663f17a/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.01.037/asset/be2cc464-9d0b-46d8-9589-1fb1994776d5/main.assets/figs1.jpg" height="649" width="654" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/b4f0584f-5b8a-4222-80c1-179e5663f17a/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">High RBM variability in deposited SARS-CoV-2 sequences is consistent with a dynamic RBD:hACE2 binding interface, related to <a id="crosref1870" href="#fig1">Figures 1</a> and <a id="crosref1875" href="#fig2">2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0200" role="paragraph">(A) Number of observed variants in four S domains (or full mature S protein) normalized by the total number of residues in each domain, where the number of observed isolates required to call a variant is varied along the x axis.</div><div id="fspara0205" role="paragraph">(B) Distributions of distances observed for RBD (gray):hACE2 (gold) residue pairs: K417-D30, E484-K31, Q493-K31, Q493-E35, G496bb-K353, G502bb-K353bb, Y449-Q42, Y449-D38, K31-E35 (bb&nbsp;= backbone interaction). RBD:ACE2 residue pairs were chosen based on RBM residues with high binding energies as determined by the binding energy % column (green) in <a id="crosref1880" href="#fig2">Figure&nbsp;2</a>. Distances were computed every 2.5&nbsp;ns from 118.7&nbsp;μs of molecular dynamics simulation data. Dashed lines indicate a distance of 3.5&nbsp;Å and the percentage of distances below and above 3.5&nbsp;Å are annotated to the left and right of the lines, respectively.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">To understand constraints on RBM variability, we evaluated the published deep mutational scanning (DMS) dataset of the RBD (<span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Starr et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and compared it to sequences of circulating viruses. The DMS data define the effect of each possible single amino acid change on both expression of the RBD and its capacity to bind hACE2. For each position in the RBM, we compared the DMS results for all amino acid replacements at that position versus only changes that have been observed in circulating SARS-CoV-2 variants (<a id="crosref0620" href="#fig2">Figure&nbsp;2</a>). A subset of residues shows the largest loss of hACE2 binding on mutation (top ∼1/3 of RBM residues in <a id="crosref0625" href="#fig2">Figure&nbsp;2</a>) and, as would be expected, few natural occurrences of mutations at these residues have been observed to be circulating. However, for the majority of the RBM (bottom ∼2/3 of RBM residues in <a id="crosref0630" href="#fig2">Figure&nbsp;2</a>), variation in circulating virus sequences confirms the tolerance to mutation predicted by the DMS data.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/54f1aa54-66e7-4a69-a03f-ddd988480e30/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.01.037/asset/56f7486a-7c31-4876-9cf6-0cd1868b7766/main.assets/gr2.jpg" height="688" width="374" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/54f1aa54-66e7-4a69-a03f-ddd988480e30/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">RBM functional constraints compared to RBM natural diversity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0050" role="paragraph">Each residue in the RBM is annotated by several metrics, depicted as a heatmap. DMS scores: outlined in black boxes (center) are summaries of hACE2 binding and RBD expression deep mutational scanning (DMS) experimental results (<span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Starr et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). DMS score is the binding or expression fold change of a variant over WT on a log10 scale (red indicating improvement and blue indicating loss as compared to WT). In the “mutagenesis” columns, DMS results are given for each residue as either the minimum (most disruptive variant) or the average score across all possible variants of a residue, except for the reference residue and the stop codon. In the “observed variants” columns, minimum and average scores are computed only across variants that have been observed in GISAID (same set of sequences as used for <a id="crosref1790" href="#fig1">Figure&nbsp;1</a>). When no natural variants have been observed, cells are gray. Data were sorted on the leftmost DMS column. Frequency: each RBM position is annotated with the frequency of non-reference amino acids in deposited sequences (darker red indicating higher frequency; at least 1 supporting sequence per 25,000 deposited sequences is required to call a variant). The number of countries in which variants have been observed is also annotated (darker purple indicating more countries). Binding energy: a re-refined SARS-CoV-2 RBD:hACE2 complex X-ray structure (PDB: 6M0J) was used to determine the approximate, decomposed binding free energy associated with each RBM residue. Results for each RBM residue are expressed as a percentage of the total binding interface interaction energy (darker green indicating stronger contribution to the binding energy).</div><div id="fspara0055" role="paragraph">See also <a id="crosref1795" href="#figs1">Figures S1</a> and <a id="crosref1800" href="#figs2">S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0070" role="paragraph">To further assess the ability of the RBM to accommodate mutations without disrupting hACE2 binding, we examined the structural dynamics and energetics of the RBM:hACE2 binding interface. We performed an approximate, residue-level decomposition of binding free energy based on the RBD:hACE2 complex X-ray structure (green in <a id="crosref0635" href="#fig2">Figure&nbsp;2</a>) as well as molecular dynamics simulations of the complex, resulting in ∼118&nbsp;μs of aggregate simulation data (<a id="crosref0640" href="#figs1">Figure&nbsp;S1</a>B). Consistent with expectation, the two residues with the highest variant frequency (S477 and N439) contribute weakly to the binding energy (<a id="crosref0645" href="#fig2">Figure&nbsp;2</a>). Surprisingly, the two RBM residues with the strongest interactions with hACE2 based on the X-ray structure (K417 and E484, dark green in <a id="crosref0650" href="#fig2">Figure&nbsp;2</a>) were not highly conserved (variant % in red, <a id="crosref0655" href="#fig2">Figure&nbsp;2</a>), with ∼10-fold more variants for E484. This could be explained by results from the molecular dynamics simulation: K417 formed close interactions with hACE2 70% of the simulation time, while E484 only 3% of the time (<a id="crosref0660" href="#figs1">Figure&nbsp;S1</a>B). The low percent for E484 is also consistent with the non-conservative amino acid replacements observed for circulating variants (e.g., the most common E484 substitution is currently E484K), with a positively charged lysine substituting for the negatively charged glutamate. Overall, these results demonstrate that the RBM has a high degree of structural plasticity whereby it is able to accommodate amino acid changes without disrupting hACE2 binding.</div><div id="p0075" role="paragraph">Evolutionary analysis of the <i>Sarbecovirus</i> subgenus provides further support for RBM plasticity (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Boni et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Boni, M.F. ∙ Lemey, P. ∙ Jiang, X. ...</div><div><strong>Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1408-1417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0771-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088964983" target="_blank" aria-label="Open Scopus in new tab">Scopus (599)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32724171/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0771-4&amp;pmid=32724171" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Li et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Li, X. ∙ Giorgi, E.E. ∙ Marichannegowda, M.H. ...</div><div><strong>Emergence of SARS-CoV-2 through recombination and strong purifying selection</strong></div><div><em>Sci. Adv.</em> 2020; <strong>6</strong>:eabb9153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.abb9153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088461737" target="_blank" aria-label="Open Scopus in new tab">Scopus (241)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32937441/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.abb9153&amp;pmid=32937441" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Rambaut et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Rambaut, A. ∙ Holmes, E.C. ∙ O’Toole, Á. ...</div><div><strong>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0770-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087930366" target="_blank" aria-label="Open Scopus in new tab">Scopus (1576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32669681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0770-5&amp;pmid=32669681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The SARS-CoV RBM is highly divergent from the SARS-CoV-2 RBM (<a id="crosref0680" href="#figs2">Figures S2</a>A and S2B) while maintaining hACE2 binding affinity. Additionally, there are many sequence changes in the RBM across a panel of related coronaviruses from animal isolates (<a id="crosref0685" href="#figs2">Figures S2</a>A and S2B; <a id="crosref0690" href="#mmc1">Table S1</a>). To determine the ability of members of the <i>Sarbecovirus</i> lineage to bind hACE2, we produced nine recombinant RBD proteins corresponding to seven animal isolates, SARS-CoV-2, and SARS-CoV, and evaluated their binding to recombinant hACE2 (<a id="crosref0695" href="#figs2">Figure&nbsp;S2</a>C). We found that three of the RBDs from animal isolates showed strong affinity for hACE2: GD Pangolin, which has a highly similar RBM to SARS-CoV-2, GX Pangolin, which has a more divergent RBM, and Bat CoV WIV1 which is highly divergent (<a id="crosref0700" href="#figs2">Figures S2</a>A and S2B). This further indicates that the RBM is structurally plastic, retaining binding with hACE2 as a receptor despite changes to sequence. Given this plasticity, we next considered whether an RBM variant can lead to immune evasion while retaining virulence.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/34eb3058-a236-4656-8a40-dd28c2e03fda/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.01.037/asset/ce87eaed-bb97-4133-a3ed-bec54d1ef16f/main.assets/figs2.jpg" height="954" width="711" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/34eb3058-a236-4656-8a40-dd28c2e03fda/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">RBDs from bat and pangolin <i>Sarbecovirus</i> isolates bind to hACE2 despite RBM divergence, related to <a id="crosref1885" href="#fig1">Figures 1</a> and <a id="crosref1890" href="#fig2">2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0215" role="paragraph">(A) Top – Percent identity to SARS-CoV-2 using a sliding window size of 30 amino acids for seven related <i>Sarbecoviruses</i> (see figure key, <sup>∗</sup>: viruses which bind to hACE2) across the RBD region of the Spike protein. Bottom – Site-specific entropy plot across the RBD protein alignment of SARS-CoV-2 and 68 related viruses (<a id="crosref1895" href="#mmc1">Table S1</a>). Sites constituting the RBM are annotated in blue; the x axis refers to absolute positions in the SARS-CoV-2 Spike protein sequence. Right – boxplot of site-specific entropy values for the RBM sites (blue) and the full RBD (gray).</div><div id="fspara0220" role="paragraph">(B) Sequence alignment (left) and identity for RBM and RBD (right) to SARS-CoV-2 of the RBD sequences showing binding to hACE2. RBM residues indicated by blue boxes.</div><div id="fspara0225" role="paragraph">(C) Binding of hACE2 to human, pangolin, and bat <i>Sarbecovirus</i> RBDs by BLI. Bat CoV RaTG13, Bat CoVs ZC45, BtKY72 and BGR2008 have also been tested and did not bind hACE2.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Phylogenetic analysis of the prevalent SARS-CoV-2 RBM mutation N439K</h3><div id="p0080" role="paragraph">N439K is a prevalent RBM mutation (the second most common mutation in the RBD through the end of 2020) which was first sampled in March 2020 in Scotland from lineage B.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Rambaut et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Rambaut, A. ∙ Holmes, E.C. ∙ O’Toole, Á. ...</div><div><strong>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0770-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087930366" target="_blank" aria-label="Open Scopus in new tab">Scopus (1576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32669681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0770-5&amp;pmid=32669681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on the background of D614G. Using phylogenetic analysis, we determined that the earliest reported N439K sequences represented a single SARS-CoV-2 lineage (<a id="crosref0710" href="#fig3">Figure&nbsp;3</a>A) that increased in frequency to 542 sequences in Scotland by June 20, 2020 (∼10% of the available Scottish viral genome sequences for this time period). Subsequently, numbers of N439K and all other variants decreased in Scotland concurrent with control of the pandemic after initiation of stringent public health measures, with this specific N439K lineage (designated here as lineage i) not being detected since June 2020 (<a id="crosref0715" href="#fig3">Figures 3</a>B and 3C). However, the N439K mutation appears in &gt;6,000 additional sequences in the GISAID database as of January 6, 2021. Our analysis demonstrates that the majority of these sequences represent a second, independent lineage (designated lineage ii) which was first sampled in Romania on May 13, 2020, then Norway on June 23, 2020, and is now detected to be circulating in 32 countries (<a id="crosref0720" href="#fig3">Figures 3</a>A–3C). N439K lineages i and ii have recently received the lineage designations B.1.141 and B.1.258, respectively (<span class="dropBlock reference-citations"><a id="body-ref-sref67-3" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-3" aria-controls="sref67-3">Rambaut et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref67-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Rambaut, A. ∙ Holmes, E.C. ∙ O’Toole, Á. ...</div><div><strong>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0770-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087930366" target="_blank" aria-label="Open Scopus in new tab">Scopus (1576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32669681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0770-5&amp;pmid=32669681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We also observe at least seven instances of the N439K mutation that have arisen independently of these two large lineages, including again in the United States in at least four linked infections, and in Brazil and Nigeria where no lineage ii/B.1.258 has been observed, resulting in a total of 34 countries where N439K has been detected to date (<a id="crosref0730" href="#fig3">Figures 3</a>A and 3B).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/0e81e35a-e259-47eb-854c-493805cfd243/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.01.037/asset/b4611dfe-5ecf-4a04-91d3-c121f913aeda/main.assets/gr3.jpg" height="424" width="504" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/0e81e35a-e259-47eb-854c-493805cfd243/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">The N439K RBM mutation has arisen independently multiple times, twice forming significant lineages</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">(A) Phylogenetic tree (de-duplicated and down-sampled) showing the relationship among representative global SARS-CoV-2 variants, with N439K variants highlighted in color. Two significant N439K lineages, one in Scotland (&gt;500 sequences, blue circles) and one in 32 countries (&gt;6,000 sequences, yellow circles) were detected as of January 6, 2021. The N439K mutation has also emerged independently on at least seven occasions (red circles show four of these) bringing the total country count to 34. Vertical bars indicate global lineage, the presence of N439K (same colors as tree), D614G (orange) or D614N (dark gray). The scale bar corresponds to a single nucleotide polymorphism (SNP).</div><div id="fspara0070" role="paragraph">(B) Frequency of N439K variants relative to sampling time and their geographical area of occurrence (see key): Africa (Morocco, Nigeria), Americas (Brazil, USA), Asia (Japan, Singapore, South Korea), the European countries Denmark, England, Republic of Ireland and Scotland and other European countries (Belgium, Bosnia-Herzegovina, Croatia, Czech Republic, Faroe Islands, Finland, France, Germany, Hungary, Italy, Luxembourg, Netherlands, Northern Ireland, Norway, Poland, Romania, Slovakia, Sweden, Switzerland, Wales), and Oceania (Australia, New Zealand). The prominent light gray bars correspond to other European countries. See <a id="crosref1805" href="#mmc1">Table S2</a> for total numbers for each country.</div><div id="fspara0075" role="paragraph">(C) Frequency of the two N439K lineages (same colors as A) over time relative to all sequences for that country (gray) and their normalized contributions (lower panels) in Scotland, England, Republic of Ireland, and Denmark.</div><div id="fspara0080" role="paragraph">See also <a id="crosref1810" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Sequence counts are heavily influenced by sampling frequency, which varies widely between countries, and N439K as a percentage of total sequences appears low: as of January 6, 2021, there have been 6,868 N439K observations in GISAID, 2% out of ∼290,000 SARS-CoV-2 genome sequences for the 34 countries where this mutation has been detected (<a id="crosref0735" href="#mmc1">Table S2</a>). Nevertheless, when comparing the percentage of N439K sequences over time in countries with sufficient data, the proportion can be significant: ∼10% in Scotland from March to June 2020 and ∼10% in Denmark from August to December 2020, both countries with high sequencing rates, and ∼13% in Ireland from July to December 2020, where regional coverage is reasonable, but the sequencing rate is lower (<a id="crosref0740" href="#fig3">Figure&nbsp;3</a>C). Importantly, on the scale of a pandemic, small proportions correspond to large numbers of infections. If the proportion of N439K sequences in each country predicts what proportion of its confirmed infections are associated with N439K variants, then N439K variants correspond to ∼764,000 of the confirmed SARS-CoV-2 infections as of January 6, 2021 (<a id="crosref0745" href="#mmc1">Table S2</a>). If detected cases represent 5%–33% of true infections, as has been estimated for the United States (<span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Wu et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Wu, S.L. ∙ Mertens, A.N. ∙ Crider, Y.S. ...</div><div><strong>Substantial underestimation of SARS-CoV-2 infection in the United States</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:4507</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-18272-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85090393211" target="_blank" aria-label="Open Scopus in new tab">Scopus (239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32908126/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-18272-4&amp;pmid=32908126" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), then a very rough approximation of the actual cumulative number of N439K-associated infections would be in the range of 2–15 million.</div><div id="p0090" role="paragraph">Overall, the spread of N439K to at least 34 countries is concerning, as is its repeated independent emergence. At the nucleotide level, all N439K variants to date have arisen from the same mutation: a C-to-A transversion in the third codon position. Interestingly, 4,209 of sequences in lineage ii/B.1.258 also carry the S 69-70 deletion that has occurred independently multiple times in the pandemic and most notably with the Y453F amino acid replacement associated with mink infections (<span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Oude Munnink et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Oude Munnink, B.B. ∙ Sikkema, R.S. ∙ Nieuwenhuijse, D.F. ...</div><div><strong>Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans</strong></div><div><em>Science.</em> 2021; <strong>371</strong>:172-177</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe5901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099167478" target="_blank" aria-label="Open Scopus in new tab">Scopus (684)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33172935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe5901&amp;pmid=33172935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In both cases the 69-70 deletion mutation has arisen subsequent to the RBM mutation and then been retained in all subsequent variants. This deletion has also been recently reported to provide an escape for NTD-specific neutralizing antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">McCarthy et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>McCarthy, K.R. ∙ Rennick, L.J. ∙ Nambulli, S. ...</div><div><strong>Recurrent deletions in the SARS-CoV-2 spike glycoprotein drive antibody escape</strong></div><div><em>Science.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf6950" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101278912" target="_blank" aria-label="Open Scopus in new tab">Scopus (346)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33536258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf6950&amp;pmid=33536258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Very recently, this deletion has also been observed to co-occur with another RBM mutation, N501Y (<span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Volz et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ...</div><div><strong>Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking epidemiological and genetic data</strong></div><div><em>medRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.30.20249034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85120008977" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.30.20249034&amp;pmid=34981069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0095" role="paragraph">Because there is concern that mutations with high prevalence may have increased virus transmissibility, we next evaluated whether any difference could be detected in the rate of spread of the N439K lineages as compared to other lineages. Because Scotland has a high sampling frequency for its population size (<a id="crosref0770" href="#mmc1">Table S2</a>), it is possible to calculate a growth rate for N439K lineage i based on a comparison with other Scottish lineages (see <a id="crosref0775" href="#sec-4">STAR methods</a> and <a id="intref0015" href="http://sars2.cvr.gla.ac.uk/RiseFallScotCOVID/" target="_blank">http://sars2.cvr.gla.ac.uk/RiseFallScotCOVID/</a>). We find that while the N439K/D614G lineage is one of the largest to emerge in Scotland, its growth rate is&nbsp;similar to the median N439/D614 or N439/D614G WT growth rates, with no evidence&nbsp;for a faster growth conferred by the N439K mutation (<a id="crosref0780" href="#figs3">Figure&nbsp;S3</a>A).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/b317abb8-7ebd-4ff2-97e2-0b1ab476f3e3/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.01.037/asset/129ea2e7-5114-452a-a75b-2dea0391b399/main.assets/figs3.jpg" height="516" width="504" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/b317abb8-7ebd-4ff2-97e2-0b1ab476f3e3/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Virological and clinical results stratified by positions 439 and 614, related to <a id="crosref1900" href="#fig3">Figures 3</a> and <a id="crosref1905" href="#fig5">5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(A) Phylodynamic analysis showing lineage growth rates relative to sampling times for UK lineages in Scotland. Data used for analysis were sampled between Feb 28, 2020 and Aug 18, 2020 (see <a id="crosref1910" href="#sec-4">STAR methods</a> and <a id="intref0400" href="http://sars2.cvr.gla.ac.uk/RiseFallScotCOVID/" target="_blank">http://sars2.cvr.gla.ac.uk/RiseFallScotCOVID/</a>). The Scottish N439K lineage i (which co-occurs with D614G) is indicated in black along with whether wild-type N439 lineages are D614 (red) or D614G (blue). The inset shows a boxplot for the distributions of these genotypes.&nbsp;Note, only the growth rates between −50 and 50 are plotted.</div><div id="fspara0240" role="paragraph">(B) Comparison of clinical severity between D614/N439, D614G/N439 and D614G/N439K genotypes by patient age group for 1591 patients whose diagnostic samples were sequenced. Ordinal scale scored by oxygen requirement: 1. No respiratory support, 2: Supplemental oxygen, 3: Invasive or non-invasive ventilation or oxygen delivery by high flow nasal cannulae, 4: Death.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3"><h3>N439K RBD forms a new interaction with hACE2 and has enhanced hACE2 affinity</h3><div id="p0100" role="paragraph">In addition to its frequency and repeated emergence, the N439K mutation stood out from other circulating RBM mutations as having a plausible mechanism for maintenance of viral fitness. The equivalent position to N439K in the SARS-CoV RBM is also a positively charged amino acid (R426), which forms a salt bridge with hACE2 (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Li et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Li, F. ∙ Li, W. ∙ Farzan, M. ...</div><div><strong>Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</strong></div><div><em>Science.</em> 2005; <strong>309</strong>:1864-1868</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1116480" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-24944498409" target="_blank" aria-label="Open Scopus in new tab">Scopus (1524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16166518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1116480&amp;pmid=16166518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0790" href="#fig4">Figure&nbsp;4</a>A). We therefore hypothesized that the N439K SARS-CoV-2 variant may form a similar salt bridge at the RBD-hACE2 interface (RBD N439K:hACE2 E329) (<a id="crosref0795" href="#fig4">Figure&nbsp;4</a>B). We determined the X-ray structure of the N439K RBD in complex with hACE2 at 2.8&nbsp;Å resolution and observed that this new interaction does indeed form (<a id="crosref0800" href="#fig4">Figure&nbsp;4</a>C; <a id="crosref0805" href="#mmc1">Table S3</a>). Because salt bridges can be strong non-covalent bonds, and therefore the N439K mutation plausibly adds a strong interaction at the binding interface, we hypothesized that the N439K variant has enhanced binding for hACE2.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/d7e5c7f5-02b7-4f84-86fb-dbc8d8dce027/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.01.037/asset/8f3370c3-66c1-4c5d-a1aa-e5d2f15403c4/main.assets/gr4.jpg" height="454" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/d7e5c7f5-02b7-4f84-86fb-dbc8d8dce027/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">N439K creates a new RBD:hACE2 salt bridge and enhances RBD:hACE2 affinity</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0090" role="paragraph">(A–C) X-ray structures of the SARS-CoV (A), SARS-CoV-2 WT (B), and SARS-CoV-2 N439K (C) RBD in complex with hACE2 (based on 2AJF, 6M0J, and current work, respectively). Select interface residues are shown as sticks. hACE2 is shown in orange and RBD in gray. The inset in (C) shows the 2Fo-Fc electron density contoured at 1σ for the K439-E329 salt bridge.</div><div id="fspara0095" role="paragraph">(D) Binding affinity of RBD and Spike variants for hACE2 measured by surface plasmon resonance. Monomeric hACE2 is injected successively at 11, 33, 100, and 300&nbsp;nM onto surface-captured spike extracellular domain (ECD) or RBD; alternately, RBD is injected successively at 3.1, 12.5, and 50&nbsp;nM onto surface-captured hACE2. All spike ECD contain the D614G mutation. Bar graph: affinity measurements (averages of 3–4 replicates) expressed as a fold change relative to WT binding within each experiment format, where &gt;1 indicates improved binding (smaller K<sub>D</sub>) relative to WT. WT K<sub>D</sub> values measured as: 95 ± 1.6&nbsp;nM (Spike surface), 63 ± 1.0&nbsp;nM (RBD surface), 19 ± 3.3&nbsp;nM (hACE2 surface); errors are SEM.</div><div id="fspara0100" role="paragraph">See also <a id="crosref1815" href="#mmc1">Table S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">To test this hypothesis, we used surface plasmon resonance (SPR) to evaluate binding of recombinant N439K S or RBD protein to recombinant hACE2. We also evaluated the N439R and K417V variants, each of which are found in SARS-CoV at these positions, and the latter of which would remove a salt bridge at the RBD:hACE2 interface. Across multiple assay formats, we found that the N439K and N439R variants exhibited an ∼2-fold enhanced binding affinity for hACE2 as compared to the original N439 variant (termed herein WT) (<a id="crosref0810" href="#fig4">Figure&nbsp;4</a>D). The magnitude of this enhancement was paralleled by an ∼2-fold loss of binding affinity for the K417V variant relative to WT. Our data are in line with the DMS results (<span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Starr et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which show a 2-fold loss of binding for K417V and no change for N439K/R, as the two assays are inherently different and the DMS data are much higher-throughput but lower sensitivity. We also tested the effect of the N439K/R and K417V mutations in combination. These double mutants swap one salt bridge at the hACE2 binding interface at RBD position 417 for one at position 439; we found they had an hACE2 affinity similar to the WT (<a id="crosref0820" href="#fig4">Figure&nbsp;4</a>D).</div><div id="p0110" role="paragraph">Overall, these data indicate that acquisition of the N439K mutation enhances hACE2 binding, which could have implications <i>in&nbsp;vivo</i> in the context of infection and transmission. At a minimum, we found no evidence for any decreased success of N439K lineage i relative to other lineages present in Scotland at the same time (<a id="crosref0825" href="#figs3">Figure&nbsp;S3</a>A). The enhanced affinity could compensate for other mutations that would otherwise decrease binding (e.g., K417V), further highlighting the plasticity of the RBM and the need for surveillance.</div></section><section id="sec-2-4"><h3>N439K SARS-CoV-2 maintains fitness and virulence</h3><div id="p0115" role="paragraph">The enhanced hACE2 affinity conferred by the N439K mutation, its geographical emergence as independent lineages, as well as its prevalence among circulating viral isolates is consistent with no effect on viral fitness. We set out to directly examine N439K impact on viral fitness by evaluating clinical data and outcomes associated with virus carrying the N439K mutation versus WT N439, as well as by direct <i>in&nbsp;vitro</i> viral growth and competition. Clinical data including age, gender, date of diagnosis, hospitalization status, and mortality were collected prospectively, and sequencing was carried out in real time, as part of the Scottish strategy for COVID-19 surveillance.</div><div id="p0120" role="paragraph">We used qPCR to evaluate viral load (as measured by cycle threshold [Ct]) in 1,918 Scottish patients whose positive samples had been sequenced (<a id="crosref0830" href="#fig5">Figures 5</a>A and 5B). Variants were either N439K/D614G (n&nbsp;= 406), N439/D614G (n&nbsp;= 978), or ancestral (N439/D614) (n&nbsp;= 534). Our analysis found strong evidence that the N439K/D614G genotype was associated with marginally lower Ct than the N439/D614G genotype, even after controlling for confounders: age, sex, viral co-ancestry, and epidemic stage (mean Ct value difference between N439K/D614G and N439/D614G: −0.65, 95% confidence interval [CI]: −1.22, −0.07) (<a id="crosref0835" href="#fig5">Figure&nbsp;5</a>B; <a id="crosref0840" href="#mmc1">Table S4</a>). Assuming the PCR was 95% efficient, then a mean Ct difference of 0.65 would represent an RNA copy number increase of 1.54-fold in N439K/D614G relative to N439/D614G. Because Ct measurements were from multiple locations in Scotland, a sub-analysis of viral load using RNA standards was carried out with available samples. This analysis showed a near-complete correlation with Ct values (<a id="crosref0845" href="#fig5">Figure&nbsp;5</a>B). D614G has previously been associated with higher viral loads/lower Ct values (<span class="dropBlock reference-citations"><a id="body-ref-sref40-3" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-3" aria-controls="sref40-3">Korber et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref40-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827.e19</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Lorenzo-Redondo et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Lorenzo-Redondo, R. ∙ Nam, H.H. ∙ Roberts, S.C. ...</div><div><strong>A clade of SARS-CoV-2 viruses associated with lower viral loads in patient upper airways</strong></div><div><em>EBioMedicine.</em> 2020; <strong>62</strong>:103112</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.ebiom.2020.103112&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.ebiom.2020.103112&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85095946935" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33186810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ebiom.2020.103112&amp;pmid=33186810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Mueller et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Mueller, N.F. ∙ Wagner, C. ∙ Frazar, C.D. ...</div><div><strong>Viral genomes reveal patterns of the SARS-CoV-2 outbreak in Washington State</strong></div><div><em>medRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.09.30.20204230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099556918" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.09.30.20204230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); although our data suggest a similar trend in a naive analysis, when controlling for confounders (given above), we could not detect this effect (<a id="crosref0870" href="#mmc1">Table S4</a>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/df089412-f4ac-48e0-aa82-331802faec2d/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.01.037/asset/1708c5e8-5f97-4988-86d7-5f65a8fcc1fb/main.assets/gr5.jpg" height="592" width="504" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/df089412-f4ac-48e0-aa82-331802faec2d/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Clinical outcomes and virological evaluation of N439K lineage i indicate maintenance of fitness relative to WT virus</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">(A) Epidemiological growth of the N439/D614, N439/D614G, or N439K/D614G virus in the National Health Service (NHS) Greater Glasgow and Clyde (GGC) Health Board area relative to sampling time in epidemiological (epi) weeks (top) and their relative contributions (bottom) for 1,918 patients whose diagnostic samples were sequenced.</div><div id="fspara0115" role="paragraph">(B) Top: real-time PCR data for N439/D614, N439/D614G, and N439K/D614G groups, same patient population as in (A). The N439K genotype was associated with marginally lower Ct values than the N439 genotype (posterior mean Ct value difference between N439K/D614G and N439/D614G: −0.65, 95% CI: −1.22, −0.07). Bottom: correlation between Ct and quantitative viral load.</div><div id="fspara0120" role="paragraph">(C) Severity of disease within NHS GGC for a subset of 1,591 patients. Ordinal scale scored by requirement for supplementary oxygen: (1) no respiratory support, (2) supplemental oxygen, (3) invasive or non-invasive ventilation or oxygen delivered by high-flow nasal cannula, and (4) death. Ordinal regression analysis indicated that the N439K viral genotype was associated with similar clinical outcomes compared to the N439 genotype (posterior mean of N439K/D614G genotype effect: 0.06, 95% CI: −1.21, 1.33).</div><div id="fspara0125" role="paragraph">(D) Growth curves for GLA1 (N439/D614G) or GLA2 (N439K/D614G) virus isolates in Vero E6&nbsp;cells with ACE2 and TMPRSS2 overexpression (+TMPRSS2&nbsp;+ACE2), ACE2 overexpression (+ACE2), or no overexpression. Error bars are SD from three replicates.</div><div id="fspara0130" role="paragraph">(E) Competition of GLA1 and GLA2 virus isolates for growth in Vero E6 cells with ACE2 and TMPRSS2 overexpression (+TMPRSS2&nbsp;+ACE2), ACE2 overexpression (+ACE2), or no overexpression, after inoculation at a matched MOI. Quantification of each virus was performed by tracking the frequency of N439K within the spike gene using metagenomic NGS. Error bars are SD from three replicates.</div><div id="fspara0135" role="paragraph">See also <a id="crosref1820" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref1825" href="#mmc1">Tables S4–S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">Clinical outcomes were also obtained for a subset of these patients (n&nbsp;= 1,591), who were scored for severity of disease based on oxygen requirement: (1) no respiratory support, (2) supplemental oxygen, (3) invasive or non-invasive ventilation or high flow nasal cannula, or (4) death (<a id="crosref0875" href="#fig5">Figures 5</a>C and <a id="crosref0880" href="#figs3">S3</a>B). The requirement for oxygen therapy or ventilation was collected retrospectively. Variant counts for the clinical outcome analysis were double mutant (N439K/D614G, n&nbsp;= 399), D614G mutants (with N439 WT, n&nbsp;= 735), or ancestral genotype (N439/D614, n&nbsp;= 457). Our ordinal regression indicated that the N439K/D614G viral genotype was associated with similar clinical outcomes compared to D614G or ancestral genotypes (posterior mean of N439K/D614G genotype effect: 0.06, 95% CI: −1.21, 1.33) (<a id="crosref0885" href="#mmc1">Table S5</a>). All other results from the severity analysis were qualitatively similar to a previous analysis of the D614G mutation (<span class="dropBlock reference-citations"><a id="body-ref-sref83-3" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-3" aria-controls="sref83-3">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These clinical outcome data indicate that the N439K virus is neither linked to an attenuated phenotype nor linked to increased severity.</div><div id="p0130" role="paragraph">We next experimentally tested growth of two representative SARS-CoV-2 isolates, GLA1 (N439) and GLA2 (N439K), both with the D614G background (<a id="crosref0895" href="#mmc1">Table S6</a>). Culture was carried out for 72&nbsp;h in Vero E6 cells with either hACE2 and TMPRSS2 overexpression, hACE2 overexpression, or no overexpression. There was no significant difference between the growth of these isolates after inoculation at multiplicities of infection (MOIs) of 0.005 and 0.01. The N439K variant replicated slightly faster initially after inoculation (<a id="crosref0900" href="#fig5">Figure&nbsp;5</a>D). These experimental data indicate that the N439K mutation does not exhibit positive or negative effects on viral growth. To further assess fitness for replication in cultured cells, we carried out a cross-competition assay using inoculation of cells at a matched MOI followed by quantitation of N439 and N439K by metagenomic sequencing over time (<a id="crosref0905" href="#fig5">Figure&nbsp;5</a>E). N439K demonstrated similar fitness as the WT N439 variant, with a slight fitness advantage for N439K in cells expressing TMPRSS2. Collectively, these results indicate that the N439K mutation results in viral fitness that is similar or possibly slightly improved relative to the WT N439 virus. These results may relate to the improved hACE2 affinity measured for the N439K RBD in the SPR binding assays, or could relate to additional mechanisms, such as changes to S density on the viral particle surface or changes to the conformational dynamics of the S protein.</div></section><section id="sec-2-5"><h3>The N439K mutation promotes evasion of antibody-mediated immunity</h3><div id="p0135" role="paragraph">Having established that the N439K mutation has no detectable effect on virus replication, we sought to test whether it promotes evasion of antibody-mediated immunity by evaluating recognition of N439K RBD by mAbs and by polyclonal immune serum from 442 recovered individuals, including six donors who were infected by the SARS-CoV-2 N439K variant. 6.8% of the tested sera showed a &gt;2-fold reduction in binding to N439K RBD as compared to WT (<a id="crosref0910" href="#fig6">Figures 6</a>A, 6B, and <a id="crosref0915" href="#figs4">S4</a>; <a id="crosref0920" href="#mmc2">Data S1</a>). In some individuals, the &gt;2-fold reduction diminished the RBD ED<sub>50</sub> response below 30 (<a id="crosref0925" href="#fig6">Figure&nbsp;6</a>A; <a id="crosref0930" href="#mmc2">Data S1</a>), a threshold previously determined to be a cutoff for specific binding (<span class="dropBlock reference-citations"><a id="body-ref-sref65-4" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-4" aria-controls="sref65-4">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, the response to the RBD can be significantly influenced by the N439K mutation in a number of individuals infected by WT SARS-CoV-2. The majority of serum samples for which there was a loss of binding were those that had overall lower Ab titers against WT RBD. The sera from the six individuals known to have recovered from infection with SARS-CoV-2 N439K virus all showed &lt;2-fold change in binding levels to WT RBD as compared to N439K RBD (<a id="crosref0940" href="#fig6">Figures 6</a>A, 6B, and <a id="crosref0945" href="#figs4">S4</a>). This may reflect a true variant-specific response or that differential binding could not be measured due to the limited number of samples analyzed.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/8a54a9c7-b5ab-4e49-b159-3e567e934581/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.01.037/asset/951595be-6cb6-43a0-8864-71160a934fbf/main.assets/gr6.jpg" height="666" width="654" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/8a54a9c7-b5ab-4e49-b159-3e567e934581/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">RBM variants exhibit escape from monoclonal antibodies and sera binding</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0145" role="paragraph">(A and B) Binding of serum and plasma samples from 442 SARS-CoV-2 infected individuals against WT and N439K RBD plotted as (A) ELISA ED<sub>50</sub> for each RBD (cut-off for positive binding to WT set at 30) and (B) fold change relative to WT. Data shown are the average of two independent replicates (source data given in <a id="crosref1830" href="#mmc2">Data S1</a>). Blue dots indicate sera with at least 2-fold loss of binding to the N439K RBD variant as compared to WT in both replicates. Purple dots indicate sera from individuals infected with SARS-CoV-2 N439K variant.</div><div id="fspara0150" role="paragraph">(C and D) Binding of 140 mAbs from SARS-CoV-2 infected individuals and four clinical-stage or EUA-approved mAbs against WT, N439K, K417V, and N439K/K417V RBD, plotted as (C) ELISA AUC for each RBD and (D) fold change relative to WT. Data shown are the average of two independent replicates (source data given in <a id="crosref1835" href="#mmc2">Data S1</a>). For all, the colored dots indicate mAbs demonstrating at least 2-fold loss of binding to the variant RBD as compared to WT (counted if the average of both replicates is at least 2-fold and each individual replicate is at least 1.7-fold).</div><div id="fspara0155" role="paragraph">(E) Kinetics of binding to RBD variants by Octet of six representative mAbs (representative of n&nbsp;= 2 independent experiments).</div><div id="fspara0160" role="paragraph">(F) Distribution of the 144 mAbs based on binding to RBD variants (expressed as fold-change over WT) and hACE2 competition (expressed as the mAb concentration blocking 80% of hACE2 binding, BC80, also indicated as a blue gradient; source data in <a id="crosref1840" href="#mmc2">Data S1</a>). Higher BC80 values (lighter blue) correspond to less hACE2 competition, with mAbs indicated at the top of the panels (white) showing no competition at all.</div><div id="fspara0165" role="paragraph">See also <a id="crosref0011" href="#figs4">Figures S4</a>, <a id="crosref0012" href="#figs5">S5</a>, and <a id="crosref1940" href="#figs6">S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/29c83851-1e53-434f-9914-766fb64d430f/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.01.037/asset/5e900c8c-6443-4a4e-a5a6-2ce174b976ec/main.assets/figs4.jpg" height="855" width="770" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/29c83851-1e53-434f-9914-766fb64d430f/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Sera ELISA results, related to <a id="crosref1915" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0250" role="paragraph">ELISA binding of the 33 human sera with a &gt; 2-fold reduction of binding to RBD N439K (A) and of the 6 sera of individuals infected with SARS-CoV-2 N439K viruses (B) to RBD WT (gray), N439K (blue), K417V (yellow) and N439K/K417V (red). Representative of n&nbsp;= 2 independent experiments.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">To understand our results at the level of individual antibodies, we evaluated a panel of 140 mAbs isolated from individuals recovered from SARS-CoV-2 infection early in the pandemic (likely N439 WT virus), which are a representative sample of the RBD-targeting mAbs generated after infection (<span class="dropBlock reference-citations"><a id="body-ref-sref65-5" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-5" aria-controls="sref65-5">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Tortorici et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Tortorici, M.A. ∙ Beltramello, M. ∙ Lempp, F.A. ...</div><div><strong>Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:950-957</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe3354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096508515" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32972994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe3354&amp;pmid=32972994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We also evaluated mAbs REGN10933, REGN10987, LY-CoV555, and S309 (the parent of VIR-7831) which are either clinical stage or approved for EUA (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Baum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baum, A. ∙ Fulton, B.O. ∙ Wloga, E. ...</div><div><strong>Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1014-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087939157" target="_blank" aria-label="Open Scopus in new tab">Scopus (877)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0831&amp;pmid=32540904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Chen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Chen, P. ∙ Nirula, A. ∙ Heller, B. ...</div><div><strong>SARS-CoV-2 Neutralizing Antibody LY-CoV555 in Outpatients with Covid-19</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:229-237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2029849" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098855067" target="_blank" aria-label="Open Scopus in new tab">Scopus (939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33113295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2029849&amp;pmid=33113295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Hansen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Hansen, J. ∙ Baum, A. ∙ Pascal, K.E. ...</div><div><strong>Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1010-1014</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089787231" target="_blank" aria-label="Open Scopus in new tab">Scopus (833)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0827&amp;pmid=32540901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Pinto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). 16.7% of these mAbs demonstrated a &gt;2-fold reduction of RBD binding in response to the N439K mutation (<a id="crosref0980" href="#fig6">Figures 6</a>C, 6D, and <a id="crosref0985" href="#figs5">S5</a>; <a id="crosref0990" href="#mmc2">Data S1</a>). For comparison, we also evaluated the K417V and N439K/K417V mutations. A similar percentage, 9.7% for K417V and 14.6% for N439K/K417V, lost &gt;2-fold binding to these variants (<a id="crosref0995" href="#fig6">Figures 6</a>C, 6D, and <a id="crosref1000" href="#figs5">S5</a>; <a id="crosref1005" href="#mmc2">Data S1</a>). Of note, some mAbs demonstrated a larger loss of binding to the double mutant as compared to either single mutant (<a id="crosref1010" href="#fig6">Figures 6</a>C, 6D; and <a id="crosref1015" href="#figs5">S5</a>; <a id="crosref1020" href="#mmc2">Data S1</a>). The reduced binding of mAbs to these RBD mutants was also confirmed by bio-layer interferometry analysis (<a id="crosref1025" href="#fig6">Figures 6</a>E and <a id="crosref1030" href="#figs6">S6</a>). The mAb panel was evaluated by RBD-binding competition experiments with hACE2 as well as with three structurally characterized antibodies defining distinct epitopes on the RBD: S304/site II, S309/site IV, and S2H14/site I, the latter significantly overlapping with the RBM (<span class="dropBlock reference-citations"><a id="body-ref-sref65-6" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-6" aria-controls="sref65-6">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The majority of the panel were site I, hACE2-blocking mAbs; the mAbs with sensitivity to N439K were enriched for site I mAbs with moderate or weak/no hACE2 blockade, consistent with the positioning of N439K at the edge of the RBM (<a id="crosref1040" href="#fig1">Figures 1</a>A and <a id="crosref1045" href="#fig6">6</a>F; <a id="crosref1050" href="#mmc2">Data S1</a>).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/9c757d14-877c-4e8b-be4d-d29d8d05a51d/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.01.037/asset/5c4e191f-28c9-418e-bcc2-b1446148b052/main.assets/figs5.jpg" height="918" width="641" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/9c757d14-877c-4e8b-be4d-d29d8d05a51d/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">mAb ELISA results, related to <a id="crosref1920" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0260" role="paragraph">ELISA binding of 80 out of the 144 mAbs to RBD WT (gray), N439K (blue), K417V (yellow) and N439K/K417V (red). AUC used for quantification is highlighted between dotted lines. Representative of n&nbsp;= 2 independent experiments. See <a id="crosref1925" href="#mmc2">Data S1</a> for results of all 144 mAbs.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/fed283ea-2a65-475c-bfe4-75c2c8d71627/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.01.037/asset/29852d9e-08e7-4f1f-b36e-75de13d51410/main.assets/figs6.jpg" height="629" width="504" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/fed283ea-2a65-475c-bfe4-75c2c8d71627/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">mAb BLI results, related to <a id="crosref1930" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0270" role="paragraph">Binding of 13 selected mAbs to RBD WT (gray), N439K (blue), K417V (yellow) and N439K/K417V (red) as measured by BLI. Representative of n&nbsp;= 2 independent experiments.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">To define the potential biological importance of these mutations for evasion of antibody-mediated neutralization, we tested mAbs against pseudoviruses expressing S variants N439K, K417V, and N439K/K417V (<a id="crosref1055" href="#fig7">Figures 7</a>A–7C and <a id="crosref1060" href="#figs7">S7</a>; <a id="crosref1065" href="#mmc2">Data S1</a>). Neutralization of pseudoviruses containing these mutations was significantly diminished for certain mAbs, including some that are currently in use in patients under EAU. As predicted by its non-RBM epitope (<span class="dropBlock reference-citations"><a id="body-ref-sref66-3" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-3" aria-controls="sref66-3">Pinto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), S309 was capable of neutralizing each of these variants. We also evaluated a cross-reactive camelid nanobody, VHH-72, which has enhanced potency for SARS-CoV as compared to SARS-CoV-2, predicted to be partially due to a contact with R426 in SARS-CoV RBD, the same position as 439 in SARS-CoV-2 RBD (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Wrapp et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Wrapp, D. ∙ De Vlieger, D. ∙ Corbett, K.S. ..., VIB-CMB COVID-19 Response Team</div><div><strong>Structural Basis for Potent Neutralization of Betacoronaviruses by Single-Domain Camelid Antibodies</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1004-1015.e15</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084038496" target="_blank" aria-label="Open Scopus in new tab">Scopus (389)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32375025/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.031&amp;pmid=32375025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with this prediction, VHH-72 showed enhanced potency against N439K SARS-CoV-2 pseudovirus compared to WT N439 (<a id="crosref1080" href="#fig7">Figures 7</a>A and 7C), highlighting the possibility that a single mutation can impact antibody efficacy positively as well as negatively. Sensitivity of a few neutralizing mAbs to mutations at positions 417 and 439 have also been reported in other studies (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Baum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baum, A. ∙ Fulton, B.O. ∙ Wloga, E. ...</div><div><strong>Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1014-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087939157" target="_blank" aria-label="Open Scopus in new tab">Scopus (877)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0831&amp;pmid=32540904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Gaebler et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gaebler, C. ∙ Wang, Z. ∙ Lorenzi, J.C.C. ...</div><div><strong>Evolution of Antibody Immunity to SARS-CoV-2</strong></div><div><em>Nature.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03207-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85100142212" target="_blank" aria-label="Open Scopus in new tab">Scopus (1073)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33461210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03207-w&amp;pmid=33461210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Greaney et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete mapping of mutations to the SARS-CoV-2 spike receptor-binding domain that escape antibody recognition</strong></div><div><em>Cell Host &amp; Microbe.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Li et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Li, Q. ∙ Wu, J. ∙ Nie, J. ...</div><div><strong>The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1284-1294.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.07.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.07.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088997441" target="_blank" aria-label="Open Scopus in new tab">Scopus (1084)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32730807/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.07.012&amp;pmid=32730807" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Starr et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Addetia, A. ...</div><div><strong>Prospective mapping of viral mutations that escape antibodies used to treat COVID-19</strong></div><div><em>bioRxiv.</em> 2020;</div><div><a id="interref0010" href="https://science.sciencemag.org/content/early/2021/01/22/science.abf9302" target="_blank">https://science.sciencemag.org/content/early/2021/01/22/science.abf9302</a></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.11.30.405472" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099854711" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.11.30.405472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Weisblum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Weisblum, Y. ∙ Schmidt, F. ∙ Zhang, F. ...</div><div><strong>Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e61312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.61312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096589275" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33112236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.61312&amp;pmid=33112236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), although combinations of mutations have typically not been evaluated. Overall, our results demonstrate that mutations compatible with equivalent viral fitness to WT can result in immune evasion from both monoclonal and polyclonal antibody responses.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/95fab88f-b33e-4ba2-8c13-56d07f95552c/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.01.037/asset/ecfd0e9b-6332-4917-a762-90e89612bfa9/main.assets/gr7.jpg" height="652" width="504" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/95fab88f-b33e-4ba2-8c13-56d07f95552c/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Neutralization of four RBM variants by a panel of antibodies and a nanobody</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(A) Neutralization of four VSV-pseudovirus variants by six of the mAbs tested. Data shown are representative of n&nbsp;= 3 biological replicates, bars&nbsp;= SD of technical duplicate (<a id="crosref1845" href="#mmc2">Data S1</a>).</div><div id="fspara0180" role="paragraph">(B) Correlation of ELISA-binding fold change and neutralization fold change for each variant relative to WT.</div><div id="fspara0185" role="paragraph">(C) Top: neutralization IC<sub>50</sub> of the D614G virus determined as the geometric mean of three biological replicates. Bottom: neutralization results for all mAbs tested, expressed as a fold-change relative to D614G (all variants are in the background of D614G) (<a id="crosref1850" href="#mmc2">Data S1</a>). The individual values of the three replicates are shown as open circles, their geometric mean as colored bars and the geometric SD as error bars. Each antibody is annotated according to its hACE2 competition (as shown in <a id="crosref1855" href="#fig6">Figure&nbsp;6</a>F) as well as its epitope (site I, II, or IV) (<a id="crosref1860" href="#mmc2">Data S1</a>). Gray boxes with a slash indicate not tested for hACE2 competition or epitope analysis.</div><div id="fspara0190" role="paragraph">See also <a id="crosref1865" href="#figs7">Figure&nbsp;S7</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.01.037/asset/1cb403eb-c387-4470-898e-eeca83af8dae/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.01.037/asset/18eaf6c8-c9f2-408c-ac1c-93d32b917262/main.assets/figs7.jpg" height="938" width="704" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/1cb403eb-c387-4470-898e-eeca83af8dae/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">VSV pseudovirus neutralization curves of all mAbs tested, related to <a id="crosref1935" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0280" role="paragraph">Representative of n&nbsp;= 3 biological replicates, bars&nbsp;= SD of technical duplicate.</div></div></div></div></figcaption></figure></div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0150" role="paragraph">Here, we describe an example of a circulating RBM mutation, N439K, which can evade antibody-mediated immunity without losing fitness relative to WT. The success of variants with the N439K mutation is evidenced by their repeated emergence by convergent evolution on at least nine occasions, spread to 34 countries as of January 2021, significant representation in sampled genome sequences (indicative of high infection rates), the fact that the N439K RBD retains a high-affinity interaction with the hACE2 receptor, and efficient replication of N439K virus in cultured cells. Additionally, we observed no evidence for change in disease severity in a large cohort of individuals infected with N439K virus as compared to WT N439 virus, although we acknowledge some limitations in the data collection, including variations in testing guidelines and availability of testing during the course of the study (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">da Silva Filipe et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>da Silva Filipe, A. ∙ Shepherd, J.G. ∙ Williams, T. ..., COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>Genomic epidemiology reveals multiple introductions of SARS-CoV-2 from mainland Europe into Scotland</strong></div><div><em>Nat. Microbiol.</em> 2021; <strong>6</strong>:112-122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-00838-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097954045" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33349681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-00838-z&amp;pmid=33349681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0155" role="paragraph">The success of the N439K mutation is consistent with our findings that the RBM is a highly variable region of S. It demonstrates the ability of SARS-CoV-2 to accommodate mutations at the RBM while retaining efficient hACE2 binding. This ability could have emerged by chance or in response to immune pressure from neutralizing Ab responses in viral hosts. There is precedent for the most immunogenic region of a viral surface protein to be highly divergent despite harboring the receptor binding site; for example, the immunogenic globular head domain of the influenza virus hemagglutinin surface protein, which contains the sialic acid receptor binding site, evolves faster than the stalk region (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Doud et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Doud, M.B. ∙ Lee, J.M. ∙ Bloom, J.D.</div><div><strong>How single mutations affect viral escape from broad and narrow antibodies to H1 influenza hemagglutinin</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-03665-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85045389473" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29643370/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-03665-3&amp;pmid=29643370" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Kirkpatrick et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Kirkpatrick, E. ∙ Qiu, X. ∙ Wilson, P.C. ...</div><div><strong>The influenza virus hemagglutinin head evolves faster than the stalk domain</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:10432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-28706-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85049794119" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29992986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-28706-1&amp;pmid=29992986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The ability to readily accommodate mutations in the RBM indicates a high likelihood that potentially immune-evading SARS-CoV-2 variants will continue to emerge, with implications for reinfection, vaccines, and both monoclonal and polyclonal antibody therapeutics.</div><div id="p0160" role="paragraph">A few other circulating RBM mutations have become prominent since N439K first emerged. S477N appeared in the sequence databases in March 2020 but did not become the most prevalent RBD mutation until the summer (as of January 2021, it has &gt;19,000 counts). Consistent with the high prevalence, position 477 is the RBM position where mutations are predicted to be the most well-tolerated with respect to hACE2 binding (<a id="crosref1130" href="#fig2">Figure&nbsp;2</a>). Studies across multiple mAb panels have not found this mutation to be conferring resistance (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Gaebler et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gaebler, C. ∙ Wang, Z. ∙ Lorenzi, J.C.C. ...</div><div><strong>Evolution of Antibody Immunity to SARS-CoV-2</strong></div><div><em>Nature.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03207-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85100142212" target="_blank" aria-label="Open Scopus in new tab">Scopus (1073)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33461210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03207-w&amp;pmid=33461210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Greaney et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete mapping of mutations to the SARS-CoV-2 spike receptor-binding domain that escape antibody recognition</strong></div><div><em>Cell Host &amp; Microbe.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Tortorici et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Tortorici, M.A. ∙ Beltramello, M. ∙ Lempp, F.A. ...</div><div><strong>Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:950-957</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe3354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096508515" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32972994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe3354&amp;pmid=32972994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Weisblum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Weisblum, Y. ∙ Schmidt, F. ∙ Zhang, F. ...</div><div><strong>Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e61312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.61312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096589275" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33112236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.61312&amp;pmid=33112236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In contrast, mutations at position 484 in the RBM have been reported to confer resistance to many mAbs across multiple studies (<span class="dropBlock reference-citations"><a id="body-ref-sref1-3" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-3" aria-controls="sref1-3">Baum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref1-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baum, A. ∙ Fulton, B.O. ∙ Wloga, E. ...</div><div><strong>Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1014-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087939157" target="_blank" aria-label="Open Scopus in new tab">Scopus (877)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0831&amp;pmid=32540904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19-3" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-3" aria-controls="sref19-3">Gaebler et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref19-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gaebler, C. ∙ Wang, Z. ∙ Lorenzi, J.C.C. ...</div><div><strong>Evolution of Antibody Immunity to SARS-CoV-2</strong></div><div><em>Nature.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03207-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85100142212" target="_blank" aria-label="Open Scopus in new tab">Scopus (1073)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33461210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03207-w&amp;pmid=33461210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23-3" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-3" aria-controls="sref23-3">Greaney et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref23-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete mapping of mutations to the SARS-CoV-2 spike receptor-binding domain that escape antibody recognition</strong></div><div><em>Cell Host &amp; Microbe.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-3" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-3" aria-controls="sref81-3">Tortorici et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref81-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Tortorici, M.A. ∙ Beltramello, M. ∙ Lempp, F.A. ...</div><div><strong>Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:950-957</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe3354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096508515" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32972994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe3354&amp;pmid=32972994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88-3" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-3" aria-controls="sref88-3">Weisblum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref88-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Weisblum, Y. ∙ Schmidt, F. ∙ Zhang, F. ...</div><div><strong>Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e61312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.61312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096589275" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33112236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.61312&amp;pmid=33112236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and also appeared in a persistent (&gt;150&nbsp;days) infection in an immunocompromised individual who did not respond to treatment with a two-mAb cocktail (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Choi et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Choi, B. ∙ Choudhary, M.C. ∙ Regan, J. ...</div><div><strong>Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2020; <strong>383</strong>:2291-2293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2031364" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097210304" target="_blank" aria-label="Open Scopus in new tab">Scopus (755)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33176080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2031364&amp;pmid=33176080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The variant count at position 484 has been steadily increasing (&gt;500 as of January 2021), and the possibility for no fitness consequences for these variants is notable (<a id="crosref1185" href="#fig2">Figures 2</a> and <a id="crosref1190" href="#figs1">S1</a>B) (<span class="dropBlock reference-citations"><a id="body-ref-sref78-4" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-4" aria-controls="sref78-4">Starr et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref78-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as is a recent study identifying this position as particularly important for escape from polyclonal serum antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Greaney et al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations to the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host &amp; Microbe.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the appearance of this mutation in a new, fast-growing viral lineage (<span class="dropBlock reference-citations"><a id="body-ref-sref80-1" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-1" aria-controls="sref80-1">Tegally et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref80-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Tegally, H. ∙ Wilkinson, E. ∙ Giovanetti, M. ...</div><div><strong>Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa</strong></div><div><em>medRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.21.20248640" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099895681" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.21.20248640" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The Y453F mutation has become noteworthy recently for its association with virus circulating in mink farms and its transmission back to humans (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Oude Munnink et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Oude Munnink, B.B. ∙ Sikkema, R.S. ∙ Nieuwenhuijse, D.F. ...</div><div><strong>Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans</strong></div><div><em>Science.</em> 2021; <strong>371</strong>:172-177</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe5901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099167478" target="_blank" aria-label="Open Scopus in new tab">Scopus (684)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33172935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe5901&amp;pmid=33172935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the DMS measurement indicating it confers significantly increased hACE2 binding (<span class="dropBlock reference-citations"><a id="body-ref-sref78-5" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-5" aria-controls="sref78-5">Starr et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref78-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To date, we know of only one example of published immune escape documented for Y453F (<span class="dropBlock reference-citations"><a id="body-ref-sref1-4" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-4" aria-controls="sref1-4">Baum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref1-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baum, A. ∙ Fulton, B.O. ∙ Wloga, E. ...</div><div><strong>Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1014-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087939157" target="_blank" aria-label="Open Scopus in new tab">Scopus (877)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0831&amp;pmid=32540904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but more examples may arise as this new mutation is investigated further. Last, the N501Y mutation has gained notice in the final weeks of 2020 for its association with high rates of infection (<span class="dropBlock reference-citations"><a id="body-ref-sref80-2" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-2" aria-controls="sref80-2">Tegally et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref80-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Tegally, H. ∙ Wilkinson, E. ∙ Giovanetti, M. ...</div><div><strong>Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa</strong></div><div><em>medRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.21.20248640" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099895681" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.21.20248640" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Volz et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ...</div><div><strong>Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking epidemiological and genetic data</strong></div><div><em>medRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.30.20249034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85120008977" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.30.20249034&amp;pmid=34981069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), although further research is needed to determine the impact on immune escape.</div><div id="p0165" role="paragraph">SARS-CoV-2 appears to be evolving relatively slowly consistent with its low mutation rate, the highly susceptible human population, and its generalist nature (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Conceicao et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Conceicao, C. ∙ Thakur, N. ∙ Human, S. ...</div><div><strong>The SARS-CoV-2 Spike protein has a broad tropism for mammalian ACE2 proteins</strong></div><div><em>PLoS Biol.</em> 2020; <strong>18</strong>:e3001016</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098929344" target="_blank" aria-label="Open Scopus in new tab">Scopus (131)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33347434/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001016&amp;pmid=33347434" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and, at present, evidence indicates it will be controllable by vaccines based on early SARS-CoV-2 genome sequences (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Dearlove et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Dearlove, B. ∙ Lewitus, E. ∙ Bai, H. ...</div><div><strong>A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:23652-23662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2008281117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091566837" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32868447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2008281117&amp;pmid=32868447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additionally, for the majority of our tested sera and mAbs, a single amino acid change in the RBM was not sufficient to confer resistance. Nevertheless, our data indicate that individuals with a mild antibody response to vaccination or first infection could be at risk from a virus carrying a mutation in the RBM. Furthermore, considering the high level of structural plasticity of the RBM demonstrated in the present study, there could be many combinations of RBM mutations, including some requiring compensatory changes, that are compatible with high viral fitness. Some of these combinations will contribute to efficient immune escape. For example, our data show that N439K can compensate for a mutation (K417V) that otherwise decreases receptor binding affinity (<a id="crosref1245" href="#fig4">Figure&nbsp;4</a>D) and that several mAbs were more sensitive to these mutations in combination versus individually (<a id="crosref1250" href="#fig6">Figure&nbsp;6</a>D; <a id="crosref1255" href="#mmc2">Data S1</a>). This particular combination of mutations is plausibly compatible with maintained viral fitness as it parallels SARS-CoV RBM:hACE2 interactions (salt bridge at SARS-CoV RBD position R426 and no salt bridge at V404) (<a id="crosref1260" href="#fig4">Figure&nbsp;4</a>A). Current SARS-CoV-2 mutations have arisen in the absence of pressure from significant population immunity. However, as immunity to the WT virus becomes more widespread, immune escape mutations can be expected to increasingly circulate. In the final weeks of 2020, SARS-CoV-2 variants carrying multiple mutations in the S protein, in both the RBM and Domain A, have been observed (<span class="dropBlock reference-citations"><a id="body-ref-sref84-3" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-3" aria-controls="sref84-3">Volz et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref84-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ...</div><div><strong>Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking epidemiological and genetic data</strong></div><div><em>medRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.30.20249034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85120008977" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.30.20249034&amp;pmid=34981069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) including one variant carrying three simultaneous RBM mutations (K417N, E484K, and N501Y) (<span class="dropBlock reference-citations"><a id="body-ref-sref80-3" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-3" aria-controls="sref80-3">Tegally et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref80-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Tegally, H. ∙ Wilkinson, E. ∙ Giovanetti, M. ...</div><div><strong>Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa</strong></div><div><em>medRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.21.20248640" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099895681" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.21.20248640" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Accumulation of multiple changes may increase the risk of immune escape from vaccines that are based on early SARS-CoV-2 sequences.</div><div id="p0170" role="paragraph">Mutations in the RBM will also impact the prophylactic or therapeutic use of mAbs. In our profile of immune escape from the N439K variant, we observed resistance to a mAb which is part of a two-mAb cocktail that recently received EAU. The promise of using cocktails of mAbs is that they should significantly lower the likelihood of drug-induced selection of resistant viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref1-5" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-5" aria-controls="sref1-5">Baum et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref1-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Baum, A. ∙ Fulton, B.O. ∙ Wloga, E. ...</div><div><strong>Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1014-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087939157" target="_blank" aria-label="Open Scopus in new tab">Scopus (877)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0831&amp;pmid=32540904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, if circulating viral variants already carry resistant mutations to one antibody in the cocktail, this could cause the cocktail to be reduced to a monotherapy. Additionally, we observed that two mutations together (N439K/K417V) conferred resistance <i>in&nbsp;vitro</i> to the two-mAb cocktail (<a id="crosref1280" href="#fig7">Figure&nbsp;7</a>C).</div><div id="p0175" role="paragraph">Two approaches will be critical for minimizing the impact of mAb escape mutations. One is to develop mAbs with epitopes that are highly resistant to viral escape. This may include epitopes outside of the RBM and/or epitopes that are cross-reactive across SARS-CoV and SARS-CoV-2, indicating conserved epitopes with a low tolerance for mutation (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Garrett Rappazzo et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Garrett Rappazzo, C. ∙ Tse, L.V. ∙ Kaku, C.I. ...</div><div><strong>Broad and potent activity against SARS-like viruses by an engineered human monoclonal antibody</strong></div><div><em>Science.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf4830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101189212" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33495307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf4830&amp;pmid=33495307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-4" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-4" aria-controls="sref66-4">Pinto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Wec et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Wec, A.Z. ∙ Wrapp, D. ∙ Herbert, A.S. ...</div><div><strong>Broad neutralization of SARS-related viruses by human monoclonal antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:731-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc7424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087044294" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc7424&amp;pmid=32540900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Wrapp et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Wrapp, D. ∙ De Vlieger, D. ∙ Corbett, K.S. ..., VIB-CMB COVID-19 Response Team</div><div><strong>Structural Basis for Potent Neutralization of Betacoronaviruses by Single-Domain Camelid Antibodies</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1004-1015.e15</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084038496" target="_blank" aria-label="Open Scopus in new tab">Scopus (389)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32375025/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.031&amp;pmid=32375025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A comparison of epitopes of RBM-targeting mAbs with the most conserved regions of the RBM (<a id="crosref1305" href="#fig2">Figure&nbsp;2</a>) may also identify RBM mAbs with a higher barrier to escape. The second will be to screen patients, likely at the population level, for the presence of potential resistance variants prior to drug administration. The availability of multiple different mAb therapeutics in the clinic could provide the opportunity to tailor the choice of therapeutics to local circulating variants.</div><div id="p0180" role="paragraph">In general, given that access to therapeutic mAbs is expanding, and as more people develop immune responses to the WT virus via infection or vaccination, monitoring the evolution of SARS-CoV-2 for escape mutants will be critical. Although we only report on evasion of antibody-mediated immunity here, it would be surprising if similar changes are not observed that confer evasion of T&nbsp;cell immunity and innate immunity.</div><section id="sec-3-1"><h3>Limitations of study</h3><div id="p0185" role="paragraph">This study presents the finding that the RBM is a highly variable region of the SARS-CoV-2&nbsp;S protein, and we provide a thorough characterization of the N439K RBM amino acid replacement, and the ability of this mutation to confer immune evasion without attenuating (or enhancing) fitness or disease. When this study was initiated in June 2020, the general consensus was that the slow rate of evolution of SARS-CoV-2 would result in no immediate threat to vaccines or therapies. N439K was the first RBM amino acid replacement, relative to the ancestral SARS-CoV-2 variant used in vaccine preparations, to increase to high frequency and so can be viewed as a sentinel mutation for SARS-CoV-2 antigenic drift. Since initial submission to Cell in late October 2020, the emergence of multiple highly transmissible variants carrying other RBM mutations of significance has brought the study of immune evasion variants to the forefront of SARS-CoV-2 research. Future retrospective studies will confirm whether these new RBM mutations fall into the same category as N439K: mutations that do not attenuate viral fitness or disease but cause immune evasion. Our results on the plasticity of the RBM also anticipated the emergence of the RBM mutations present in the SARS-CoV-2 variants of concern.</div></section></section><section id="sec-4"><h2>STAR★methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0190" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>S304, S309 IgG and Fab fragments</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref66-5" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-5" aria-controls="sref66-5">Pinto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB: <a id="intref0020" href="http://firstglance.jmol.org/fg.htm?mol7JX3" target="_blank">7JX3</a></td></tr><tr><td>S2H13, S2H14, S2A4, S2X35 IgG</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref65-7" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-7" aria-controls="sref65-7">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB: <a id="intref0025" href="http://firstglance.jmol.org/fg.htm?mol7JV2" target="_blank">7JV2</a>, <a id="intref0030" href="http://firstglance.jmol.org/fg.htm?mol7JXC" target="_blank">7JXC</a>, <a id="intref0035" href="http://firstglance.jmol.org/fg.htm?mol7JXD" target="_blank">7JXD</a>, <a id="intref0040" href="http://firstglance.jmol.org/fg.htm?mol7JXE" target="_blank">7JXE</a></td></tr><tr><td>S2E12, S2M11 IgG</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref81-4" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-4" aria-controls="sref81-4">Tortorici et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref81-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Tortorici, M.A. ∙ Beltramello, M. ∙ Lempp, F.A. ...</div><div><strong>Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:950-957</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe3354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096508515" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32972994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe3354&amp;pmid=32972994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB: <a id="intref0045" href="http://firstglance.jmol.org/fg.htm?mol7K3Q" target="_blank">7K3Q</a>, <a id="intref0050" href="http://firstglance.jmol.org/fg.htm?mol7K43" target="_blank">7K43</a></td></tr><tr><td>LY-CoV555 IgG</td><td>Eli Lilly and Company</td><td>N/A</td></tr><tr><td>REGN10933, REGN10987 IgG</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-3" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-3" aria-controls="sref27-3">Hansen et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Hansen, J. ∙ Baum, A. ∙ Pascal, K.E. ...</div><div><strong>Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1010-1014</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089787231" target="_blank" aria-label="Open Scopus in new tab">Scopus (833)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0827&amp;pmid=32540901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB: <a id="intref0055" href="http://firstglance.jmol.org/fg.htm?mol6XDG" target="_blank">6XDG</a></td></tr><tr><td>VHH-72</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref90-3" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-3" aria-controls="sref90-3">Wrapp et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref90-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Wrapp, D. ∙ De Vlieger, D. ∙ Corbett, K.S. ..., VIB-CMB COVID-19 Response Team</div><div><strong>Structural Basis for Potent Neutralization of Betacoronaviruses by Single-Domain Camelid Antibodies</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1004-1015.e15</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084038496" target="_blank" aria-label="Open Scopus in new tab">Scopus (389)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32375025/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.031&amp;pmid=32375025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PDB: <a id="intref0060" href="http://firstglance.jmol.org/fg.htm?mol6WAQ" target="_blank">6WAQ</a></td></tr><tr><td>Panel of Human IgG</td><td>This study</td><td>N/A</td></tr><tr><td>Goat Anti-Human IgG-AP</td><td>Southern Biotech</td><td>Cat. No. 2040-04; RRID:<a id="interref405" href="https://antibodyregistry.org/AB_2795643" target="_blank">AB_2795643</a></td></tr><tr><td>Goat F(ab’)2 Anti-Mouse IgG(H+L), Human ads-AP</td><td>Southern Biotech</td><td>Cat. No. 1030-04; RRID:<a id="interref410" href="https://antibodyregistry.org/AB_2794293" target="_blank">AB_2794293</a></td></tr><tr class="rowsep"><td>Anti-Avi-tag Antibody, pAb, Rabbit</td><td>GenScript</td><td>Cat. No. A00674; RRID:<a id="interref415" href="https://antibodyregistry.org/AB_915553" target="_blank">AB_915553</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial strains and pseudotype viruses</b></td></tr><tr><td>VSV-G-glycoprotein-pseudotyped virus</td><td>Kerafast</td><td>Cat. No. EH1020-PM</td></tr><tr class="rowsep"><td>VSV-SARS-CoV-2 S-glycoprotein-pseudotyped virus</td><td>This study</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr><td>Serum and plasma of SARS-CoV-2 infected individuals</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref65-8" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-8" aria-controls="sref65-8">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Serum from SARS-CoV-2 N439K infected individuals</td><td>ISARIC4C <a id="intref0065" href="https://isaric4c.net/" target="_blank">https://isaric4c.net/</a></td><td>N/A</td></tr><tr><td>Nasopharyngeal swabs from SARS-CoV-2 infected individuals</td><td>West of Scotland Specialist Virology Centre</td><td>N/A</td></tr><tr class="rowsep"><td>Sputum from SARS-CoV-2 infected individuals</td><td>West of Scotland Specialist Virology Centre</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals and recombinant proteins</b></td></tr><tr><td>PEI MAX</td><td>Polysciences</td><td>Cat. No. POL24765-1</td></tr><tr><td>TransIT-Lenti</td><td>Mirus</td><td>Cat. No. 6600</td></tr><tr><td>4-Nitrophenyl phosphate disodium salt hexahydrate (pNPP)</td><td>Sigma-Aldrich</td><td>Cat. No. N2765-100TAB</td></tr><tr><td>Blocker Casein (1%) in PBS</td><td>Thermo Fisher Scientific</td><td>Cat. No. 37528</td></tr><tr><td>Tween 20</td><td>Sigma Aldrich</td><td>Cat. No. 93773</td></tr><tr><td>Bovine Serum Albumine</td><td>Sigma</td><td>Cat. No. 3059</td></tr><tr><td>hACE2, mFc tag</td><td>ATUM</td><td>N/A</td></tr><tr><td>BioLock - Biotin Blocking Solution</td><td>IBA GmbH</td><td>2-0205-050</td></tr><tr><td>PNGase F</td><td>New England Biolabs</td><td>P0704L</td></tr><tr><td>EndoH</td><td>New England Biolabs</td><td>P0702L</td></tr><tr><td>Thrombin</td><td>Sigma-Aldrich</td><td>T1063-250UN</td></tr><tr><td>RBD mouse Fc-tagged</td><td>Sino Biological</td><td>Cat. No. 40592-V05H</td></tr><tr class="rowsep"><td>Streptavidin-alkaline phosphatase conjugated</td><td>Jackson ImmunoResearch</td><td>Cat. No. 016-050-084</td></tr><tr class="rowsep"><td colspan="3"><b>Cell lines</b></td></tr><tr><td>ExpiCHO-S</td><td>Thermo Fisher Scientific</td><td>Cat. No. A29127</td></tr><tr><td>Expi293F</td><td>Thermo Fisher Scientific</td><td>Cat. No. A14527</td></tr><tr><td>Lenti-X 293T cells</td><td>Takara</td><td>Cat. No. 632180</td></tr><tr class="rowsep"><td>Vero E6 cells</td><td>ATCC</td><td>Cat. No. CRL-1586</td></tr><tr class="rowsep"><td colspan="3"><b>Commercial assays, kits, and products</b></td></tr><tr><td>HiTrap Protein A columns (HiTrap Mab select PrismA)</td><td>Cytiva</td><td>Cat. No. 17-5498-54P</td></tr><tr><td>Strep-Tactin XT Superflow high capacity cartridge</td><td>IBA GmbH</td><td>Cat. No. 2-4026-001</td></tr><tr><td>HisTALON Superflow Cartridges, 5&nbsp;mL</td><td>Takara Bio</td><td>Cat. No. 635683</td></tr><tr><td>HisTALON Superflow Cartridges, 1&nbsp;mL</td><td>Takara Bio</td><td>Cat. No. 635650</td></tr><tr><td>Superdex 200 Increase 10/300 GL</td><td>Cytiva</td><td>Cat. No. 28-9909-44</td></tr><tr><td>Superose 6 Increase 10/300 GL</td><td>Cytiva</td><td>Cat. No. 29-0915-96</td></tr><tr><td>StrepTrap HP column, 1&nbsp;mL</td><td>Cytiva</td><td>Cat. No. 28-9075-46</td></tr><tr><td>ExpiFectamine 293 Transfection Kit</td><td>Thermo Fisher Scientific</td><td>Cat. No. A14524</td></tr><tr><td>ExpiFectamine CHO Transfection Kit</td><td>Thermo Fisher Scientific</td><td>Cat. No. A29129</td></tr><tr><td>ExpiFectamine CHO Transfection Kit</td><td>Thermo Fisher Scientific</td><td>Cat. No. A29130</td></tr><tr><td>HiPrep 26/10 desalting columns</td><td>Cytiva</td><td>Cat. No. 17-5087-02</td></tr><tr><td>CaptureSelect C-tag Affinity Matrix</td><td>Thermo Fisher Scientific</td><td>Cat. No. 2943072010</td></tr><tr><td>Zeba Spin Desalting columns, 7 K MWCO, 5&nbsp;ml</td><td>Thermo Fisher Scientific</td><td>Cat. No. 89892</td></tr><tr><td>Bio-Glo</td><td>Promega</td><td>Cat. No. G7940</td></tr><tr><td>Biosensor Protein A</td><td>FortéBio</td><td>Cat. No. 733-2137</td></tr><tr><td>MiSeq Reagent v2 500 cycle kit</td><td>Illumina</td><td>Cat. No. MS-102-2003</td></tr><tr><td>NextSeq 500/550 High Output Kit v2.5 (300 Cycles)</td><td>Illumina</td><td>Cat. No. 20024908</td></tr><tr><td>DMEM GlutaMAX</td><td>Thermo Fisher Scientific</td><td>Cat. No. 10566016</td></tr><tr><td>Fetal Bovine Serum (FBS)</td><td>Thermo Fisher Scientific</td><td>Cat. No. A4766801</td></tr><tr><td>DNaseI</td><td>Thermo Fisher Scientific</td><td>Cat. No. AM2222</td></tr><tr><td>Agencourt RNA Clean AMPure XP Beads</td><td>Beckman Coulter</td><td>Cat. No. A63987</td></tr><tr><td>Qubit dsDNA HS Kit</td><td>Thermo Fisher Scientific</td><td>Cat No. Q32854</td></tr><tr><td>SuperScript III</td><td>Thermo Fisher Scientific</td><td>Cat No. 18080044</td></tr><tr><td>NEBNext Ultra II Non-Directional RNA Second Strand Synthesis Module</td><td>New England Biolabs</td><td>Cat. No. E6111L</td></tr><tr><td>Kapa HyperPrep kit</td><td>Roche</td><td>Cat. No. KK8504</td></tr><tr><td>Kapa LTP Library Preparation Kit for Illumina Platforms</td><td>Roche</td><td>Cat. No. KK8232</td></tr><tr><td>NEBNext Multiplex Oligos for Illumina 96&nbsp;Unique Dual Index Primer Pairs</td><td>New England Biolabs</td><td>Cat. No. E6442S</td></tr><tr><td>High Sensitivity D5000 Screentape</td><td>Agilent</td><td>Cat. No. 5067-5592</td></tr><tr><td>NEB Luna Universal Probe One-Step RT-qPCR Kit</td><td>New England Biolabs</td><td>Cat No. E3006E</td></tr><tr><td>2019-nCoV_N1 assay RT-qPCR assay</td><td>Integrated DNA Technologies</td><td>Cat No. 10006713</td></tr><tr><td>HBS-N buffer</td><td>Cytiva</td><td>Cat. No. BR100369</td></tr><tr><td>HBS-EP+ buffer</td><td>Cytiva</td><td>Cat. No. BR100669</td></tr><tr><td>Series S Sensor Chip CM5</td><td>Cytiva</td><td>Cat. No. BR100530</td></tr><tr><td>Series S Sensor Chip C1</td><td>Cytiva</td><td>Cat. No. BR100535</td></tr><tr><td>Mouse antibody capture kit</td><td>Cytiva</td><td>Cat. No. BR100838</td></tr><tr><td>Twin-Strep-Tag Capture Kit</td><td>IBA GmbH</td><td>Cat. No. 2-4370-000</td></tr><tr><td>EZ-Link NHS-PEG Solid-Phase Biotinylation Kit - Mini-Spin Columns</td><td>Thermo Fisher Scientific</td><td>Cat. No. 21450</td></tr><tr><td>Spectraplate-384 with high protein binding</td><td>Perkin Elmer</td><td>Cat. No. CUSG83093</td></tr><tr class="rowsep"><td>Nunc-Immuno plates - 96-well plate, MaxiSorp</td><td>Sigma-Aldrich Chemie GmbH</td><td>Cat. No. M9410-1CS</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>SARS-CoV-2 RBD N439K/ hACE2/ S304 Fab/ S309 Fab X-ray structure</td><td>This study</td><td>PDB: <a id="intref0070" href="http://firstglance.jmol.org/fg.htm?mol7L0N" target="_blank">7L0N</a></td></tr><tr class="rowsep"><td>SARS-CoV-2 RBD/hACE2 Molecular Dynamics trajectories</td><td>This study</td><td><a id="intref0075" href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2" target="_blank">https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2</a></td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Human antibody expression vectors (IgG1)</td><td>This study</td><td>N/A</td></tr><tr><td>Plasmids encoding SARS-CoV-2 2P S ectodomain variants (amino acids 14-1211) with C-terminal AviTag-8xHis-C-tag</td><td>This study (ATUM Bio)</td><td>GenBank: <a id="intref0080" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_045512.2" target="_blank">NC_045512.2</a></td></tr><tr><td>Plasmids encoding SARS-CoV-2 RBD WT or variants (amino acids 328-531) with C-terminal thrombin cleavage site-TwinStrep-8xHis-tag</td><td>This study</td><td>GenBank: <a id="intref0085" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_045512.2" target="_blank">NC_045512.2</a></td></tr><tr><td>Plasmid encoding SARS-CoV-2 RBD WT (amino acids 328-531) with N-terminal ‘ETGT’ and C-terminal GS linker-Strep-8xHis-tag</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref65-9" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-9" aria-controls="sref65-9">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GenBank: <a id="intref0090" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_045512.2" target="_blank">NC_045512.2</a></td></tr><tr><td>Plasmid encoding SARS-CoV-2 RBD N439K (amino acids 328-531) with N-terminal ‘ETGT’ and C-terminal 8xHis-tag</td><td>This study</td><td>GenBank: <a id="intref0095" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC_045512.2" target="_blank">NC_045512.2</a></td></tr><tr><td>Plasmids encoding Sarbecovirus RBDs with C-terminal thrombin cleavage site-TwinStrep-8xHis-tag</td><td>This study</td><td>See Data S1</td></tr><tr><td>Plasmid encoding human ACE2 receptor (amino acids 19-615) plus C-terminal thrombin cleavage site-TwinStrep-10xHis-GGG-tag</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref65-10" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-10" aria-controls="sref65-10">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>UniProt: <a id="intref0100" href="https://www.uniprot.org/uniprot/Q9BYF1" target="_blank">Q9BYF1</a></td></tr><tr class="rowsep"><td>Plasmids encoding SARS-CoV-2 D19 Spike variants</td><td>This study</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Prism</td><td>GraphPad</td><td>Version 8</td></tr><tr><td>UNICORN</td><td>Cytiva</td><td>Versions 7.3 and 7.5</td></tr><tr><td>Biacore T200 Evaluation Software</td><td>Cytiva</td><td>Version 3.1</td></tr><tr><td>SDS software</td><td>Thermo Fisher Scientific</td><td>Version 2.3</td></tr><tr><td>Refmac5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Murshudov et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Murshudov, G.N. ∙ Skubák, P. ∙ Lebedev, A.A. ...</div><div><strong>REFMAC5 for the refinement of macromolecular crystal structures</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2011; <strong>67</strong>:355-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444911001314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-79953763877" target="_blank" aria-label="Open Scopus in new tab">Scopus (6601)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21460454/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444911001314&amp;pmid=21460454" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 5.8.0258</td></tr><tr><td>Coot</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (19617)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 0.9</td></tr><tr><td>XDS</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Kabsch, 2010</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Kabsch, W.</div><div><strong>Xds</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:125-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444909047337" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-76449099287" target="_blank" aria-label="Open Scopus in new tab">Scopus (12648)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444909047337&amp;pmid=20124692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>MOE</td><td>Chemical Computing Group</td><td>Version 2019.0102</td></tr><tr><td>BioEdit</td><td>Tom Hall <a id="intref0105" href="http://bioedit.software.informer.com/7.0" target="_blank">http://bioedit.software.informer.com/7.0</a></td><td>Version 7.0.5.3</td></tr><tr><td>R: A Language and Environment for Statistical Computing</td><td>R Foundation for Statistical Computing</td><td>Version 4.0.3</td></tr><tr><td>Skygrowth</td><td><a id="intref0110" href="https://github.com/mrc-ide/skygrowth" target="_blank">https://github.com/mrc-ide/skygrowth</a></td><td>N/A</td></tr><tr><td>SPIn</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Liu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Liu, Y. ∙ Gelman, A. ∙ Zheng, T.</div><div><strong>Simulation-efficient shortest probability intervals</strong></div><div><em>Stat. Comput.</em> 2015; <strong>25</strong>:809-819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11222-015-9563-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84932196966" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11222-015-9563-8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.1</td></tr><tr><td>IQ-TREE 2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Minh et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Minh, B.Q. ∙ Schmidt, H.A. ∙ Chernomor, O. ...</div><div><strong>IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:1530-1534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msaa015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084101286" target="_blank" aria-label="Open Scopus in new tab">Scopus (4768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsaa015&amp;pmid=32011700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 2.0.6</td></tr><tr><td>lubridate</td><td><a id="intref0115" href="https://github.com/tidyverse/lubridate" target="_blank">https://github.com/tidyverse/lubridate</a></td><td>Version 1.7.4</td></tr><tr><td>ape</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Paradis and Schliep, 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Paradis, E. ∙ Schliep, K.</div><div><strong>ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:526-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty633" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85056898420" target="_blank" aria-label="Open Scopus in new tab">Scopus (4152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30016406/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty633&amp;pmid=30016406" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 5.3</td></tr><tr><td>brms</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Bürkner, 2018</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bürkner, P.-C.</div><div><strong>Advanced bayesian multilevel modeling with the R package brms</strong></div><div><em>R J.</em> 2018; <strong>10</strong>:395-411</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.32614/RJ-2018-017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.32614%2FRJ-2018-017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 2.13.5</td></tr><tr><td>drc</td><td><a id="intref0120" href="https://cran.r-project.org/web/packages/drc/drc.pdf" target="_blank">https://cran.r-project.org/web/packages/drc/drc.pdf</a></td><td>Version 3.0-1</td></tr><tr><td>entropy</td><td><a id="intref0125" href="https://cran.r-project.org/web/packages/entropy/" target="_blank">https://cran.r-project.org/web/packages/entropy/</a></td><td>Version 1.2.1</td></tr><tr><td>RcppRoll</td><td><a id="intref0130" href="https://cran.r-project.org/web/packages/RcppRoll/index.html" target="_blank">https://cran.r-project.org/web/packages/RcppRoll/index.html</a></td><td>Version 0.3.0</td></tr><tr><td>MinKNOW</td><td>Oxford Nanopore technologies</td><td>Version 19.12.6</td></tr><tr><td>Porechop</td><td><a id="intref0135" href="https://github.com/rrwick/Porechop" target="_blank">https://github.com/rrwick/Porechop</a></td><td>Version 0.2.4</td></tr><tr><td>Guppy basecaller</td><td>Oxford Nanopore technologies</td><td>Version 3.2.10</td></tr><tr><td>Nanopolish</td><td><a id="intref0140" href="https://github.com/jts/nanopolish" target="_blank">https://github.com/jts/nanopolish</a></td><td>Version 0.11.3</td></tr><tr><td>trim_galore</td><td><a id="intref0145" href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a></td><td>Version 0.6.5</td></tr><tr><td>BWA</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Li, 2013</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Li, H.</div><div><strong>Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</strong></div><div><em>arXiv.</em> 2013;</div><div>arXiv:1303.3997</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.LiAligning+sequence+reads%2C+clone+sequences+and+assembly+contigs+with+BWA-MEMarXiv2013arXiv%3A1303.3997" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 0.7.5</td></tr><tr><td>iVar</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">Grubaugh et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Grubaugh, N.D. ∙ Gangavarapu, K. ∙ Quick, J. ...</div><div><strong>An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1618-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85059769331" target="_blank" aria-label="Open Scopus in new tab">Scopus (488)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30621750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1618-7&amp;pmid=30621750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.2.2</td></tr><tr><td>Minimap2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Li, 2018</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Li, H.</div><div><strong>Minimap2: pairwise alignment for nucleotide sequences</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:3094-3100</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85054132627" target="_blank" aria-label="Open Scopus in new tab">Scopus (5152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty191&amp;pmid=29750242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 2.17</td></tr><tr><td>Baltic Python library</td><td><a id="intref0150" href="https://github.com/evogytis/baltic" target="_blank">https://github.com/evogytis/baltic</a></td><td>N/A</td></tr><tr><td>Artic sequencing bioinformatic pipeline</td><td>Artic network <a id="intref0155" href="https://artic.network/ncov-2019" target="_blank">https://artic.network/ncov-2019</a></td><td>N/A</td></tr><tr><td>Miniconda</td><td>Anaconda <a id="intref0160" href="http://www.anaconda.com" target="_blank">http://www.anaconda.com</a></td><td>Anaconda Version 2-2.4.0 Miniconda Version 4.9.0</td></tr><tr><td>Folding@home</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Shirts and Pande, 2000</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Shirts, M. ∙ Pande, V.S.</div><div><strong>COMPUTING: Screen Savers of the World Unite!</strong></div><div><em>Science.</em> 2000; <strong>290</strong>:1903-1904</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.290.5498.1903" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0034623787" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17742054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.290.5498.1903&amp;pmid=17742054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Zimmerman et al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Zimmerman, M.I. ∙ Porter, J.R. ∙ Ward, M.D. ...</div><div><strong>SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.06.27.175430" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098837589" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.06.27.175430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>IPython</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Perez and Granger, 2007</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Perez, F. ∙ Granger, B.E.</div><div><strong>IPython: A System for Interactive Scientific Computing</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:21-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/MCSE.2007.53" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-34247481878" target="_blank" aria-label="Open Scopus in new tab">Scopus (2954)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FMCSE.2007.53" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 7.14.0</td></tr><tr><td>Jupyter Notebook</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Kluyver et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Kluyver, T. ∙ Ragan-Kelley, B. ∙ Pérez, F. ...</div><div><strong>Jupyter Notebooks – a publishing format for reproducible computational workflows</strong></div><div><div>Loizides, F. ∙ Scmidt, B. (Editors)</div></div><div><strong>Positioning and Power in Academic Publishing: Players, Agents and Agendas</strong></div><div>IOS Press, 2016; 87-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3233/978-1-61499-649-1-87" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087152625" target="_blank" aria-label="Open Scopus in new tab">Scopus (2684)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3233%2F978-1-61499-649-1-87" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 6.1.5</td></tr><tr><td>MDAnalysis</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Michaud-Agrawal et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Michaud-Agrawal, N. ∙ Denning, E.J. ∙ Woolf, T.B. ...</div><div><strong>MDAnalysis: a toolkit for the analysis of molecular dynamics simulations</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2011; <strong>32</strong>:2319-2327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.21787" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-79958185452" target="_blank" aria-label="Open Scopus in new tab">Scopus (1850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21500218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.21787&amp;pmid=21500218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Gowers et al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Gowers, R.J. ∙ Linke, M. ∙ Barnoud, J. ...</div><div><strong>MDAnalysis: A Python Package for the Rapid Analysis of Molecular Dynamics Simulations</strong></div><div><em>Proceedings of the 15th Python in Science Conference.</em> 2016; 98-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.25080/Majora-629e541a-00e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.25080%2FMajora-629e541a-00e" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.0.0</td></tr><tr><td>NumPy</td><td><a id="intref0165" href="https://numpy.org" target="_blank">https://numpy.org</a></td><td>Version 1.19.1</td></tr><tr><td>OpenMM</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Eastman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Eastman, P. ∙ Swails, J. ∙ Chodera, J.D. ...</div><div><strong>OpenMM 7: Rapid development of high performance algorithms for molecular dynamics</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85026639054" target="_blank" aria-label="Open Scopus in new tab">Scopus (1317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28746339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005659&amp;pmid=28746339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 7.4.2</td></tr><tr><td>OpenMMTools</td><td><a id="intref0170" href="https://github.com/choderalab/openmmtools" target="_blank">https://github.com/choderalab/openmmtools</a></td><td>Version 0.20.0</td></tr><tr><td>PyMOL</td><td>Schrödinger</td><td>Version 2.3.2</td></tr><tr><td>ISOLDE</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Croll, 2018</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Croll, T.I.</div><div><strong>ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318002425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85048328149" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872003/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318002425&amp;pmid=29872003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.0.1</td></tr><tr><td>ChimeraX</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Pettersen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF ChimeraX: Structure visualization for researchers, educators, and developers</strong></div><div><em>Protein Sci.</em> 2021; <strong>30</strong>:70-82</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85093816376" target="_blank" aria-label="Open Scopus in new tab">Scopus (3298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32881101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3943&amp;pmid=32881101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.0</td></tr><tr><td>AmberTools</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Case et al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Case, D.A. ∙ Cerutti, D.S. ∙ Cheatham, III, T.E. ...</div><div><strong>AMBER 2017</strong></div><div>University of California, San Francisco, 2017</div><div><a id="interref0420" href="https://ambermd.org/doc12/Amber17.pdf" target="_blank">https://ambermd.org/doc12/Amber17.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.A.CaseD.S.CeruttiT.E.CheathamIIIT.A.DardenAMBER+20172017University+of+California%2C+San+Franciscohttps%3A%2F%2Fambermd.org%2Fdoc12%2FAmber17.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 17.0</td></tr><tr><td>pdb-tools</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Rodrigues et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Rodrigues, J.P.G.L.M. ∙ Teixeira, J.M.C. ∙ Trellet, M. ...</div><div><strong>pdb-tools: a swiss army knife for molecular structures</strong></div><div><em>F1000Res.</em> 2018; <strong>7</strong>:1961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.17456.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85060775176" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30705752/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.17456.1&amp;pmid=30705752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 2.0.5</td></tr><tr><td>MDTraj</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">McGibbon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>McGibbon, R.T. ∙ Beauchamp, K.A. ∙ Harrigan, M.P. ...</div><div><strong>MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</strong></div><div><em>Biophys. J.</em> 2015; <strong>109</strong>:1528-1532</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.bpj.2015.08.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.bpj.2015.08.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84944750932" target="_blank" aria-label="Open Scopus in new tab">Scopus (1229)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26488642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2015.08.015&amp;pmid=26488642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Version 1.9.4</td></tr><tr><td>Pandas</td><td><a id="intref0175" href="https://conference.scipy.org/proceedings/scipy2010/pdfs/mckinney.pdf" target="_blank">https://conference.scipy.org/proceedings/scipy2010/pdfs/mckinney.pdf</a></td><td>Version 1.0.5</td></tr><tr><td>Custom code, molecular dynamics set up and processing</td><td>This paper</td><td><a id="intref0180" href="https://github.com/choderalab/rbd-ace2-contact-analysis" target="_blank">https://github.com/choderalab/rbd-ace2-contact-analysis</a></td></tr><tr class="rowsep"><td>Custom code, evaluation of clinical samples</td><td>This paper</td><td><a id="intref0185" href="https://github.com/dpascall/SARS-CoV-2-mutation-analysis" target="_blank">https://github.com/dpascall/SARS-CoV-2-mutation-analysis</a></td></tr><tr class="rowsep"><td colspan="3"><b>Instruments</b></td></tr><tr><td>ÄKTA Xpress FPLC</td><td>Cytiva</td><td>N/A</td></tr><tr><td>ÄKTA Pure 25</td><td>Cytiva</td><td>N/A</td></tr><tr><td>Synergy H1 Hybrid Multi-Mode plate reader</td><td>Biotek</td><td>N/A</td></tr><tr><td>EL406 washer/dispenser BSL2 M</td><td>Biotek</td><td>N/A</td></tr><tr><td>Biacore T200</td><td>Cytiva</td><td>N/A</td></tr><tr><td>Octet Red96</td><td>Pall FortéBio</td><td>N/A</td></tr><tr><td>7500 Fast Real-Time PCR System</td><td>Applied Biosystems</td><td>N/A</td></tr><tr><td>Illumina MiSeq</td><td>Illumina</td><td>SY-410-1003</td></tr><tr><td>Illumina’sNextSeq550</td><td>Illumina</td><td>SY-415-1002</td></tr><tr><td>Flowcell R9.4.1</td><td>Oxford Nanopore technologies</td><td>FLO-MIN106D</td></tr><tr><td>Envision multimode plate reader</td><td>PerkinElmer</td><td>2105</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900080-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0195" role="paragraph">Further information and requests for resources and reagents should be directed to the Lead Contact, Gyorgy Snell (<a id="intref0190" href="mailto:gsnell@vir.bio">gsnell@vir.bio</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0200" role="paragraph">Materials generated in this study will be made available on request, but we may require a completed materials transfer agreement.</div></section><section id="sec-4-2-3"><h4>Data and code availability</h4><div id="p0205" role="paragraph">Datasets generated during this study are included in the article or are available from the corresponding authors on request. The&nbsp;X-ray structure data and model has been deposited with accession code PDB: 7L0N. The code used to set up, run, and&nbsp;analyze the molecular dynamics simulations is available at: <a id="intref0195" href="https://github.com/choderalab/rbd-ace2-contact-analysis" target="_blank">https://github.com/choderalab/rbd-ace2-contact-analysis</a>. Raw and processed molecular dynamics trajectory data are available at the MolSSI COVID-19 Molecular Structure and Therapeutics&nbsp;Hub: <a id="intref0200" href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2" target="_blank">https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2</a>. Code for evaluation of clinical samples is available from GitHub: <a id="intref0205" href="https://github.com/dpascall/SARS-CoV-2-mutation-analysis" target="_blank">https://github.com/dpascall/SARS-CoV-2-mutation-analysis</a>.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Cell lines</h4><div id="p0210" role="paragraph">Cell lines were obtained from ATCC (Vero E6) or Thermo Fisher Scientific (Expi293F, ExpiCHO-S). Expi293F and ExpiCHO-S cells were maintained in Expi293 Expression Medium and ExpiCHO- Expression Medium (Thermo Fisher Scientific), respectively.</div></section><section id="sec-4-3-2"><h4>Sample donors</h4><div id="p0215" role="paragraph">Samples from 442 SARS-CoV-2 infected individuals were obtained from the Ticino healthcare workers cohort (Switzerland), described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref65-11" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-11" aria-controls="sref65-11">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and under study protocols approved by the local Institutional Review Board (Canton Ticino Ethics Committee, Switzerland). All donors provided written informed consent for the use of blood and blood components (such as PBMCs, sera or plasma). In the Ticino region of Switzerland and during the time period of collection (February-March 2020) no N439K SARS-CoV-2 isolates were reported.</div><div id="p0220" role="paragraph">Samples from six N439K variant infected individuals were obtained from the ISARIC4C consortium (<a id="intref0210" href="https://isaric4c.net/" target="_blank">https://isaric4c.net/</a>). Ethical approval was given by the South Central-Oxford C Research Ethics Committee in England (reference 13/SC/0149), and by the Scotland A Research Ethics Committee (reference 20/SS/0028). The study was registered at <a id="intref0215" href="https://www.isrctn.com/ISRCTN66726260" target="_blank">https://www.isrctn.com/ISRCTN66726260</a>.</div><div id="p0225" role="paragraph">Residual nucleic acid extracts derived from the nose-throat swabs of 1918 SARS-CoV-2 positive individuals whose diagnostic samples were submitted to the West of Scotland specialist virology center between 3<sup>rd</sup> March and 30<sup>th</sup> June 2020 were sequenced as part of the COG-UK consortium under study protocols approved by the relevant national biorepositories (16/WS/0207NHS and 10/S1402/33) (<a id="intref0220" href="mailto:consortiumcontact@cogconsortium.uk">consortiumcontact@cogconsortium.uk</a>, 2020).</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Structural analysis</h4><div id="p0230" role="paragraph">RBM residues were determined based on the RBD:hACE2 complex crystal structures 2AJF for SARS-CoV (<span class="dropBlock reference-citations"><a id="body-ref-sref46-3" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-3" aria-controls="sref46-3">Li et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref46-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Li, F. ∙ Li, W. ∙ Farzan, M. ...</div><div><strong>Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</strong></div><div><em>Science.</em> 2005; <strong>309</strong>:1864-1868</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1116480" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-24944498409" target="_blank" aria-label="Open Scopus in new tab">Scopus (1524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16166518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1116480&amp;pmid=16166518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and 6M0J for SARS-CoV-2 (<span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Lan et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Lan, J. ∙ Ge, J. ∙ Yu, J. ...</div><div><strong>Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:215-220</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2180-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85083295918" target="_blank" aria-label="Open Scopus in new tab">Scopus (4081)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32225176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2180-5&amp;pmid=32225176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The 2AJF structure was obtained from the PDB-REDO server (<a id="intref0225" href="https://pdb-redo.eu" target="_blank">https://pdb-redo.eu</a>) and was subsequently prepared in the molecular modeling software MOE (v2019.0102, <a id="intref0230" href="https://www.chemcomp.com" target="_blank">https://www.chemcomp.com</a>) using the structure preparation, protonation and energy minimization steps with default settings. RBD residues within 6.0&nbsp;Å distance of any hACE2 atoms (determined using MOE) were determined for each of the two copies of the complex in the asymmetric unit, and then were combined to obtain the definition of the RBM used in this work (<a id="crosref1475" href="#fig2">Figure&nbsp;2</a>). 6M0J was obtained from the Coronavirus Structural Task Force server (<a id="intref0235" href="https://github.com/thorn-lab/coronavirus_structural_task_force" target="_blank">https://github.com/thorn-lab/coronavirus_structural_task_force</a>) and was further refined (using Refmac5 v5.8.0258), manually fitted (using Coot v0.9) and prepared (using MOE, as described above) in multiple iterative cycles. The final structure was analyzed for RBD-hACE2 contact residues with a 6.0&nbsp;Å cutoff to obtain the RBM (using MOE). The final list of RBM residues (<a id="crosref1480" href="#fig1">Figure&nbsp;1</a>C) was arrived at by combining the SARS-CoV and SARS-CoV-2 results.</div><div id="p0235" role="paragraph">Using MOE, the pairwise binding energy (the sum of van der Waals, ionic, and hydrogen-bond interactions) between each residue in SARS-CoV-2 RBD and each residue in hACE2, and the total binding energy for all interactions, was determined at cutoff distances 3.0&nbsp;Å, 3.5&nbsp;Å, 4.0&nbsp;Å, 4.5&nbsp;Å, 5.0&nbsp;Å, 5.5&nbsp;Å, 6.0&nbsp;Å, 6.5&nbsp;Å and 7.0&nbsp;Å. The percentage of the total binding energy for each interacting RBD residue was calculated for each distance cutoff and was then averaged over all cutoffs. The resulting values are shown in green in <a id="crosref1485" href="#fig1">Figure&nbsp;1</a>C.</div></section><section id="sec-4-4-2"><h4>RBM variability across SARS-CoV-2 sequences</h4><div id="p0240" role="paragraph">Using CoV-GLUE-reported variants (<a id="intref0240" href="http://cov-glue.cvr.gla.ac.uk/" target="_blank">http://cov-glue.cvr.gla.ac.uk/</a>, downloaded from GISAID on November 30<sup>th</sup> 2020, n&nbsp;= 209,239) the Shannon’s entropy (natural log units, as implemented in the R package entropy) was computed at each residue of the mature (excluding signal peptide) spike protein. Then, entropy was aggregated by domain, in sliding windows (using the R package RcppRoll), or in bins of randomly sampled residues, as detailed in figure legends. Due to the non-normal distribution of variant frequencies, the median rather than the mean was used as the aggregation metric.</div><div id="p0245" role="paragraph">As an alternative to entropy, we also quantitated variability by counting the number of variants passing an increasing threshold of supporting sequences. Unlike entropy, this metric only uses variant frequency for thresholding a digital (presence/absence) variant call, hence it is less affected by sampling/deposition bias.</div></section><section id="sec-4-4-3"><h4>Evaluation of deep mutational scanning (DMS) data</h4><div id="p0250" role="paragraph">The DMS dataset was retrieved from <span class="dropBlock reference-citations"><a id="body-ref-sref78-6" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-6" aria-controls="sref78-6">Starr et&nbsp;al. (2020b)</a><div class="dropBlock__holder" data-db-target-of="sref78-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Variant-level DMS scores were aggregated by residue by taking the minimum (most disruptive variant) or the average score across all variants of that residue, except for the reference amino acid and stop codons. Alternatively, minimum and average scores were computed only across variants that have been observed as naturally occurring. Data were represented as a heatmap annotated with: frequency of variants from CoV-GLUE (at least 1 supporting sequence per 25,000 deposited sequences was required to call a variant); number of countries in which a variant was observed; and percentage of total binding energy computed from an X-ray crystal structure (<i>cf.</i> structural analysis methods section).</div></section><section id="sec-4-4-4"><h4>Molecular dynamics simulations</h4><section id="sec-4-4-4-1"><h5>Structure preparation</h5><div id="p0255" role="paragraph">The RBD:hACE2 complex was constructed from individual RBD (PDB: <a id="intref0245" href="http://firstglance.jmol.org/fg.htm?mol6m0j" target="_blank">6m0j</a>, Chain E) and hACE2 (PDB: <a id="intref0250" href="http://firstglance.jmol.org/fg.htm?mol1r42" target="_blank">1r42</a>, Chain A) monomers aligned to the full RBD:hACE2 structure (PDB: <a id="intref0255" href="http://firstglance.jmol.org/fg.htm?mol6m0j" target="_blank">6m0j</a>). The 1r42 structure was used for hACE2 because 1) 1r42 is higher resolution (2.20&nbsp;Å, whereas 6m0j is 2.45&nbsp;Å) and 2) the electron density map of 1r42 clearly reveals N-acetylglucosamine (NAG) orientation at each glycosylated asparagine residue, providing a reliable building block on which to construct more complex glycan structures. These complex glycans were constructed at each NAG due to earlier work suggesting their role in mediating RBD:hACE2 binding (<span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Zhao et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Zhao, P. ∙ Praissman, J.L. ∙ Grant, O.C. ...</div><div><strong>Virus-Receptor Interactions of Glycosylated SARS-CoV-2 Spike and Human ACE2 Receptor</strong></div><div><em>Cell Host Microbe.</em> 2020; <strong>28</strong>:586-601.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.08.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.08.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85090566044" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.08.004&amp;pmid=32841605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as it is in the spike proteins’ intrinsic RBD dynamics (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Casalino et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Casalino, L. ∙ Gaieb, Z. ∙ Goldsmith, J.A. ...</div><div><strong>Beyond Shielding: The Roles of Glycans in the SARS-CoV-2 Spike Protein</strong></div><div><em>ACS Cent. Sci.</em> 2020; <strong>6</strong>:1722-1734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acscentsci.0c01056" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85092393131" target="_blank" aria-label="Open Scopus in new tab">Scopus (562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33140034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facscentsci.0c01056&amp;pmid=33140034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0260" role="paragraph">In order to start from the most reliable structural models, we obtained 6m0j and 1r42 from the Coronavirus Structural Taskforce (CST) database, which contains refined structural models based on careful examination of the electron density. In the RBD of the refined 6m0j structure, amino acid rotamers and peptide bonds were flipped to increase Ramachandran favorability, decrease rotamer outliers,&nbsp;reduce clashes, and improve fit to density. A more detailed summary of the 6m0j refinement details is available at: <a id="intref0260" href="https://github.com/thorn-lab/coronavirus_structural_task_force/blob/master/pdb/surface_glycoprotein/SARS-CoV-2/6m0j/isolde/notes.txt" target="_blank">https://github.com/thorn-lab/coronavirus_structural_task_force/blob/master/pdb/surface_glycoprotein/SARS-CoV-2/6m0j/isolde/notes.txt</a>. The 1r42 refined structure differs from the PDB-deposited structure in that it includes the missing C-terminal domain of hACE2 (copied from the 6m17 PDB structure). A more detailed summary of the 1r42 refinement details is available at: <a id="intref0265" href="https://github.com/thorn-lab/coronavirus_structural_task_force/blob/master/pdb/human_interaction_partners/ACE2/1r42/isolde/notes.txt" target="_blank">https://github.com/thorn-lab/coronavirus_structural_task_force/blob/master/pdb/human_interaction_partners/ACE2/1r42/isolde/notes.txt</a>.</div><div id="p0265" role="paragraph">The resulting RBD and hACE2 monomers were then aligned in PyMOL 2.3.2 (Schrödinger, LLC) to the CST 6m0j structure to create an initial RBD:hACE2 complex. The overall root-mean-square deviation (RMSD) was 0.426&nbsp;Å and the interface RMSD was 0.405&nbsp;Å, where RMSD was computed for all atoms and the interface residues were defined as all residues within 4&nbsp;Å of the other binding partner.</div><div id="p0270" role="paragraph">Next, the full glycosylation patterns for hACE2 and RBD glycans were determined from <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Shajahan et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Shajahan, A. ∙ Archer-Hartmann, S. ∙ Supekar, N.T. ...</div><div><strong>Comprehensive characterization of N- and O- glycosylation of SARS-CoV-2 human receptor angiotensin converting enzyme 2</strong></div><div><em>Glycobiology.</em> 2020;</div><div>Published online October 29, 2020</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/glycob/cwaa101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85106069987" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32363391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fglycob%2Fcwaa101&amp;pmid=32363391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and <span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Watanabe et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Watanabe, Y. ∙ Allen, J.D. ∙ Wrapp, D. ...</div><div><strong>Site-specific glycan analysis of the SARS-CoV-2 spike</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:330-333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb9983" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085763940" target="_blank" aria-label="Open Scopus in new tab">Scopus (1020)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32366695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb9983&amp;pmid=32366695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. For the constructed RBD:hACE2 complex, these included sites: N53, N90, N103, N322, N432, N546, and N690 on hACE2 and N343 on the RBD. The glycan structures used for each site (FA2, FA26G1, FA2, FA2, FA2G2, A2, FA2, FA2G2, respectively) correspond to the most stable conformers obtained from multi microsecond MD simulations of cumulative sampling (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Harbison et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Harbison, A.M. ∙ Brosnan, L.P. ∙ Fenlon, K. ...</div><div><strong>Sequence-to-structure dependence of isolated IgG Fc complex biantennary N-glycans: a molecular dynamics study</strong></div><div><em>Glycobiology.</em> 2019; <strong>29</strong>:94-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/glycob/cwy097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85058598461" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30325416/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fglycob%2Fcwy097&amp;pmid=30325416" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Base NAG residues at the reducing end of each glycan structure were aligned to the corresponding NAG stub in the&nbsp;RBD:hACE2 model in PyMOL 2.3.2 (Schrödinger, LLC) and any resulting clashes were refined in ISOLDE (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Croll, 2018</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Croll, T.I.</div><div><strong>ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318002425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85048328149" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872003/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318002425&amp;pmid=29872003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Full details&nbsp;of the glycosylation patterns / structures used and full workflow are available at: <a id="intref0270" href="https://github.com/choderalab/rbd-ace2-contact-analysis" target="_blank">https://github.com/choderalab/rbd-ace2-contact-analysis</a>.</div></section></section><section id="sec-4-4-5"><h4>System solvation and parametrization</h4><div id="p0275" role="paragraph">The refined glycosylated RBD:hACE2 complex was prepared for simulation using the AmberTools17 tleap suite (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Case et al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Case, D.A. ∙ Cerutti, D.S. ∙ Cheatham, III, T.E. ...</div><div><strong>AMBER 2017</strong></div><div>University of California, San Francisco, 2017</div><div><a id="interref0420" href="https://ambermd.org/doc12/Amber17.pdf" target="_blank">https://ambermd.org/doc12/Amber17.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.A.CaseD.S.CeruttiT.E.CheathamIIIT.A.DardenAMBER+20172017University+of+California%2C+San+Franciscohttps%3A%2F%2Fambermd.org%2Fdoc12%2FAmber17.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All relevant disulfide bridges were specified as well as covalent connectivity within each glycan structure. The glycosylated protein was parameterized with the Amber ff14SB (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Maier et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Maier, J.A. ∙ Martinez, C. ∙ Kasavajhala, K. ...</div><div><strong>ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</strong></div><div><em>J.&nbsp;Chem. Theory Comput.</em> 2015; <strong>11</strong>:3696-3713</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jctc.5b00255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84938930908" target="_blank" aria-label="Open Scopus in new tab">Scopus (6593)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26574453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jctc.5b00255&amp;pmid=26574453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and GLYCAM_06j-1 (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Kirschner et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Kirschner, K.N. ∙ Yongye, A.B. ∙ Tschampel, S.M. ...</div><div><strong>GLYCAM06: a generalizable biomolecular force field. Carbohydrates</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2008; <strong>29</strong>:622-655</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.20820" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-39049147617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1708)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17849372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.20820&amp;pmid=17849372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) force fields. The system was solvated using the TIP3P rigid water model (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Jorgensen et&nbsp;al., 1983</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Jorgensen, W.L. ∙ Chandrasekhar, J. ∙ Madura, J.D. ...</div><div><strong>Comparison of simple potential functions for simulating liquid water</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1983; <strong>79</strong>:926-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.445869" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0004016501" target="_blank" aria-label="Open Scopus in new tab">Scopus (32859)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.445869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in a cubic box with 1.5&nbsp;nm solvent padding on all sides. The solvated system was then minimally neutralized with 0.15&nbsp;M NaCl using the Li/Merz ion parameters of monovalent ions for the TIP3P water model (12-6 normal usage set) (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Li et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Li, P. ∙ Song, L.F. ∙ Merz, Jr., K.M.</div><div><strong>Systematic Parameterization of Monovalent Ions Employing the Nonbonded Model</strong></div><div><em>J.&nbsp;Chem. Theory Comput.</em> 2015; <strong>11</strong>:1645-1657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ct500918t" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84927941776" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26574374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fct500918t&amp;pmid=26574374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Full details and tleap scripts can be found at: <a id="intref0275" href="https://github.com/choderalab/rbd-ace2-contact-analysis" target="_blank">https://github.com/choderalab/rbd-ace2-contact-analysis</a>.</div></section><section id="sec-4-4-6"><h4>System equilibration</h4><div id="p0280" role="paragraph">The system was energy-minimized with an energy tolerance of 10&nbsp;kJ mol<sup>−1</sup>and equilibrated using the OpenMM 7.4.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Eastman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Eastman, P. ∙ Swails, J. ∙ Chodera, J.D. ...</div><div><strong>OpenMM 7: Rapid development of high performance algorithms for molecular dynamics</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85026639054" target="_blank" aria-label="Open Scopus in new tab">Scopus (1317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28746339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005659&amp;pmid=28746339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) Langevin integrator for 300&nbsp;ns in the NPT (p&nbsp;= 1 atm, T&nbsp;= 310 K) ensemble with a timestep of 4.0 femtoseconds, a collision rate of 1.0 picoseconds <sup>-1</sup>, and a constraint tolerance of 1&nbsp;× 10<sup>−5</sup>. Hydrogen atom masses were set to 4.0 amu by transferring mass from connected heavy atoms, bonds to hydrogen were constrained, and center of mass motion was not removed. Pressure was controlled by a molecular-scaling Monte Carlo barostat with an update interval of 25 steps. Non-bonded interactions were treated with the Particle Mesh Ewald method (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Darden et&nbsp;al., 1993</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Darden, T. ∙ York, D. ∙ Pedersen, L.</div><div><strong>Particle mesh Ewald: An N log(N) method for Ewald sums in large systems</strong></div><div><em>J.&nbsp;Chem. Physiol.</em> 1993; <strong>98</strong>:10089</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.464397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-33846823909" target="_blank" aria-label="Open Scopus in new tab">Scopus (23398)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.464397" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using a real-space cutoff of 1.0&nbsp;nm and the OpenMM (<span class="dropBlock reference-citations"><a id="body-ref-sref16-3" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-3" aria-controls="sref16-3">Eastman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref16-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Eastman, P. ∙ Swails, J. ∙ Chodera, J.D. ...</div><div><strong>OpenMM 7: Rapid development of high performance algorithms for molecular dynamics</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85026639054" target="_blank" aria-label="Open Scopus in new tab">Scopus (1317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28746339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005659&amp;pmid=28746339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) default relative error tolerance of 0.0005, with grid spacing selected automatically. For improved stability, the structure was then equilibrated using the OpenMMTools 0.20.0 BAOAB Langevin integrator (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Leimkuhler and Matthews, 2013</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Leimkuhler, B. ∙ Matthews, C.</div><div><strong>Robust and efficient configurational molecular sampling via Langevin dynamics</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2013; <strong>138</strong>:174102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.4802990" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84877787427" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23656109/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.4802990&amp;pmid=23656109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for 10&nbsp;ns using all of the same simulation parameters described above. This simulation was subsequently packaged to seed for production simulation on Folding@home (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Shirts and Pande, 2000</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Shirts, M. ∙ Pande, V.S.</div><div><strong>COMPUTING: Screen Savers of the World Unite!</strong></div><div><em>Science.</em> 2000; <strong>290</strong>:1903-1904</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.290.5498.1903" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0034623787" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17742054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.290.5498.1903&amp;pmid=17742054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Zimmerman et al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Zimmerman, M.I. ∙ Porter, J.R. ∙ Ward, M.D. ...</div><div><strong>SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.06.27.175430" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098837589" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.06.27.175430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Default parameters were used unless noted otherwise. Further details of the equilibration protocol are available at: <a id="intref0280" href="https://github.com/choderalab/rbd-ace2-contact-analysis" target="_blank">https://github.com/choderalab/rbd-ace2-contact-analysis</a></div></section><section id="sec-4-4-7"><h4>Folding@home simulations</h4><div id="p0285" role="paragraph">The equilibrated structure was then used to initiate parallel distributed MD simulations on Folding@home (<span class="dropBlock reference-citations"><a id="body-ref-sref75-3" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-3" aria-controls="sref75-3">Shirts and Pande, 2000</a><div class="dropBlock__holder" data-db-target-of="sref75-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Shirts, M. ∙ Pande, V.S.</div><div><strong>COMPUTING: Screen Savers of the World Unite!</strong></div><div><em>Science.</em> 2000; <strong>290</strong>:1903-1904</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.290.5498.1903" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0034623787" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17742054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.290.5498.1903&amp;pmid=17742054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref97-3" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-3" aria-controls="sref97-3">Zimmerman et al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref97-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Zimmerman, M.I. ∙ Porter, J.R. ∙ Ward, M.D. ...</div><div><strong>SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.06.27.175430" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098837589" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.06.27.175430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Simulations were run with OpenMM 7.4.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref16-4" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-4" aria-controls="sref16-4">Eastman et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref16-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Eastman, P. ∙ Swails, J. ∙ Chodera, J.D. ...</div><div><strong>OpenMM 7: Rapid development of high performance algorithms for molecular dynamics</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85026639054" target="_blank" aria-label="Open Scopus in new tab">Scopus (1317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28746339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005659&amp;pmid=28746339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Folding@home core22 0.0.13). Production simulations used the same Langevin integrator as the NPT equilibration described above. In total, 2000 independent MD simulations were generated on Folding@home. Conformational snapshots (frames) were stored at an interval of 0.5&nbsp;ns/frame for subsequent analysis. The resulting final dataset contained 2000 trajectories, 183.8&nbsp;μs of aggregate simulation time, and 367610 frames. This amount of simulation time corresponds to approximately 13.7 GPU-years on an NVIDIA GeForce GTX 1080Ti. This trajectory dataset with solvent is available at the MolSSI COVID-19 Molecular Structure and Therapeutics Hub: <a id="intref0285" href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2" target="_blank">https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2</a>.</div></section><section id="sec-4-4-8"><h4>Simulation analysis</h4><div id="p0290" role="paragraph">The longest 1000 trajectories were chosen for analysis, ranging from 90&nbsp;ns to 230&nbsp;ns in length, which represent an aggregate simulation time of 118.7&nbsp;μs. Each frame in a trajectory was aligned using MDAnalysis (<span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Gowers et al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Gowers, R.J. ∙ Linke, M. ∙ Barnoud, J. ...</div><div><strong>MDAnalysis: A Python Package for the Rapid Analysis of Molecular Dynamics Simulations</strong></div><div><em>Proceedings of the 15th Python in Science Conference.</em> 2016; 98-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.25080/Majora-629e541a-00e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.25080%2FMajora-629e541a-00e" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Michaud-Agrawal et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Michaud-Agrawal, N. ∙ Denning, E.J. ∙ Woolf, T.B. ...</div><div><strong>MDAnalysis: a toolkit for the analysis of molecular dynamics simulations</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2011; <strong>32</strong>:2319-2327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.21787" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-79958185452" target="_blank" aria-label="Open Scopus in new tab">Scopus (1850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21500218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.21787&amp;pmid=21500218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to the equilibrated structure. This was to ensure no crossing of periodic boundary conditions during calculation of distances between residue pairs. The distances between residue pairs (K417-D30, E484-K31, Q493-E35, Q493-K31, G496-K353, G502-K353, Y449-D38, Y449-Q42, K31-E35) were calculated once every 5 frames (2.5&nbsp;ns) using MDAnalysis after discarding the first 100 frames (50&nbsp;ns) of each trajectory to ensure relaxation away from the initial seed conformation. Distance was defined as the minimum distance between sidechain heavy atoms for a given residue pair. Further details of the analysis pipeline are available at: <a id="intref0290" href="https://github.com/choderalab/rbd-ace2-contact-analysis" target="_blank">https://github.com/choderalab/rbd-ace2-contact-analysis</a></div></section><section id="sec-4-4-9"><h4>RBM variability across Sarbecoviruses</h4><div id="p0295" role="paragraph">A pairwise comparison of <i>Sarbecovirus</i> RBD sequences (see <a id="crosref1605" href="#mmc1">Table S1</a>) to SARS-CoV-2 RBD was performed by calculating percent identity over a window size of 30 amino acids at each RBD position. For the site-specific entropy plot across the RBD alignment of SARS-CoV-2 and 68 related viruses, entropy for each position <i>l</i> (H(<i>l</i>)) was calculated using Shannon’s entropy formula with a natural log as implemented in Bioedit (H(<i>l</i>)&nbsp;= -Σf(<i>a,l</i>)ln(f(<i>a,l</i>)); f(<i>a,l</i>) being the frequency of amino acid <i>a</i> at position <i>l</i>).</div></section><section id="sec-4-4-10"><h4>Recombinant glycoprotein production</h4><div id="p0300" role="paragraph">Prefusion-stabilized SARS-CoV-2 spike protein variants (residues 14-1211), containing the 2P and Furin cleavage site mutations (<span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Walls et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Walls, A.C. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:281-292.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.02.058&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.02.058&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081283286" target="_blank" aria-label="Open Scopus in new tab">Scopus (5950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32155444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.02.058&amp;pmid=32155444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with a mu-phosphatase signal peptide and a C-terminal Avi-8xHis-C-tag (ATUM Bio) were expressed in Expi293F cells at 37°C and 8% CO<sub>2</sub>. Transfections were performed using the ExpiFectamine 293 Transfection Kit (Thermo Fisher Scientific). Cell culture supernatant was collected after four days and purified over a 5&nbsp;mL C-tag affinity matrix (Thermo Fisher Scientific). Elution fractions were concentrated and injected on a Superose 6 Increase 10/300 GL column (Cytiva) with 1x PBS pH 7.4 as running buffer.</div><div id="p0305" role="paragraph">SARS-CoV-2 RBD WT (with N-terminal signal peptide and C-terminal thrombin cleavage site-TwinStrep-8xHis-tag) and variants were expressed in Expi293F cells at 37°C and 8% CO<sub>2</sub>. Transfections were performed using the ExpiFectamine 293 Transfection Kit (Thermo Fisher Scientific). Cell culture supernatant was collected three days after transfection and supplemented with 10x PBS to a final concentration of 2.5x PBS (342.5&nbsp;mM NaCl, 6.75&nbsp;mM KCl and 29.75&nbsp;mM phosphates), or 3.2x for RBD N439R. SARS-CoV-2 RBDs were purified using 1 or 5&nbsp;mL HisTALON superflow cartridges (Takara Bio) and subsequently buffer exchanged into 1x HBS-N buffer (Cytiva) or PBS using Zeba Spin Desalting or HiPrep 26/10 desalting columns.</div><div id="p0310" role="paragraph">RBDs from other sarbecoviruses and SARS-CoV-2 RBD WT (with N-terminal signal peptide and ‘ETGT’, and C-terminal GS linker-Strep-8xHis-tag) were expressed in Expi293F cells at 37°C and 8% CO<sub>2</sub>. Cells were transfected using PEI MAX (Polysciences) at a DNA:PEI ratio of 1:3.75. Transfected cells were supplemented three days after transfection with 3 g/L glucose (Bioconcept) and 5 g/L soy hydrolysate (Sigma-Aldrich Chemie GmbH). Cell culture supernatant (423&nbsp;mL) was collected seven days after transfection and supplemented with 47&nbsp;mL 10x binding buffer (1&nbsp;M Tris-HCl, 1.5&nbsp;M NaCl, 20&nbsp;mM EDTA, pH 8.0) and 25&nbsp;mL BioLock (IBA GmbH) and incubated on ice for 30&nbsp;min. Proteins were purified using a 5&nbsp;mL Strep-Tactin XT Superflow high capacity cartridge (IBA GmbH) followed by buffer exchange to PBS using HiPrep 26/10 desalting columns (Cytiva).</div><div id="p0315" role="paragraph">For S binding measurements, recombinant ACE2 (residues 19-615 from Uniprot <a id="intref0295" href="https://www.uniprot.org/uniprot/Q9BYF1" target="_blank">Q9BYF1</a> with a C-terminal thrombin cleavage site-TwinStrep-10xHis-GGG-tag, and N-terminal signal peptide) was expressed in Expi293F cells at 37°C and 8% CO<sub>2</sub>. Transfections were performed using the ExpiFectamine 293 Transfection Kit (Thermo Fisher Scientific). Cell culture supernatant was collected seven or eight days after transfection and supplemented to a final concentration of 80&nbsp;mM Tris-HCl pH 8.0, 100&nbsp;mM NaCl, and then incubated with BioLock (IBA GmbH) solution. After filtration through a 0.22&nbsp;μm filter, ACE2 was purified using a 1&nbsp;mL StrepTrap HP column (Cytiva) followed by isolation of the monomeric ACE2 by size exclusion chromatography using a Superdex 200 Increase 10/300 GL column (Cytiva) pre-equilibrated in PBS.</div><div id="p0320" role="paragraph">For crystallography, the same hACE2 construct as above was expressed in ExpiCHO-S cells at 37°C and 8% CO<sub>2</sub> with kifunensine added to 10&nbsp;μM. Transfections were performed using the ExpiFectamine CHO transfection kit (Thermo Fisher Scientific). Cell culture supernatant was collected six days after transfection and supplemented to a final concentration of 80&nbsp;mM Tris-HCl pH 8.0, 100&nbsp;mM NaCl, and then incubated with BioLock (IBA GmbH) solution for one hour. hACE2 was purified using a 1&nbsp;mL StrepTrap HP column (Cytiva).</div><div id="p0325" role="paragraph">For SPR binding measurements with surface-captured RBD, recombinant hACE2 (residues 19-615 from Uniprot <a id="intref0300" href="https://www.uniprot.org/uniprot/Q9BYF1" target="_blank">Q9BYF1</a> with a C-terminal AviTag-10xHis-GGG-tag, and N-terminal signal peptide) was expressed in HEK293.sus using standard methods (ATUM Bio). Protein was purified via Ni Sepharose resin followed by isolation of the monomeric ACE2 by size exclusion chromatography using a Superdex 200 Increase 10/300 GL column pre-equilibrated with PBS.</div><div id="p0330" role="paragraph">For binding measurements with surface-captured hACE2, recombinant hACE2 (residues 18-615 with a C-terminal GS-IgG2a-Mm-Fc tag, and N-terminal signal peptide) was stably transfected in CHO-K1 GS knock-down cell line (ATUM Bio). Protein was purified via protein A and buffer exchanged into PBS.</div></section><section id="sec-4-4-11"><h4>Crystallization, data collection, structure determination, and analysis</h4><div id="p0335" role="paragraph">The SARS-CoV-2 RBD N439K-hACE2 complex was formed together with two Fab fragments (S304 and S309) to aid in crystallization. Prior to forming the SARS-CoV-2 RBD N439K-ACE2-S304-S309 complex, recombinant hACE2 protein was digested using EndoH (New England Biolabs) and thrombin (Sigma-Aldrich). Recombinant SARS-CoV-2 RBD N439K was digested with PNGase F (New England Biolabs) and thrombin (Sigma-Aldrich). RBD was mixed with a 1.3-fold molar excess of deglycosylated hACE2, S304 Fab, and S309 Fab. The complex was purified on a Superdex 200 10/300 GL column pre-equilibrated with 20&nbsp;mM Tris-HCl pH 7.5, 150&nbsp;mM NaCl. Crystals of the SARS-CoV-2 RBD N439K-hACE2-S304-S309 complex were obtained at 20°C by sitting drop vapor diffusion. A total of 200 nL of the complex at 6&nbsp;mg/mL were mixed with 200 nL mother liquor solution containing 0.1&nbsp;M ammonium sulfate, 20% v/v ethylene glycol, 10% w/v PEG 8000, and 0.1&nbsp;M bicine/tris pH 8.5.</div><div id="p0340" role="paragraph">Data were collected at the Molecular Biology Consortium beamline 4.2.2 at the Advanced Light Source synchrotron facility in Berkeley, CA. Datasets from two crystals were individually processed and then merged with the XDS software package (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Kabsch, 2010</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Kabsch, W.</div><div><strong>Xds</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:125-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444909047337" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-76449099287" target="_blank" aria-label="Open Scopus in new tab">Scopus (12648)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444909047337&amp;pmid=20124692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for a final dataset of 2.78&nbsp;Å in space group P2<sub>1</sub>. The RBD N439K-hACE2-S304-S309 complex structure was solved by molecular replacement using phaser (<span class="dropBlock reference-citations"><a id="body-ref-opt0p615ydFV9" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opt0p615ydFV9" aria-controls="opt0p615ydFV9">McCoy et al., 2007</a><div class="dropBlock__holder" data-db-target-of="opt0p615ydFV9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>McCoy, A.L. ∙ Grosse-Kunstleve, R.W. ∙ Adams, P.D. ...</div><div><strong>Phaser crystallographic software</strong></div><div><em>J. Appl. Crystallogr.</em> 2007; <strong>40</strong>:658-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889807021206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-34447508216" target="_blank" aria-label="Open Scopus in new tab">Scopus (15801)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889807021206&amp;pmid=19461840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) from starting models consisting of RBD-S304-S309 (PDB: <a id="intref0305" href="http://firstglance.jmol.org/fg.htm?mol7JX3" target="_blank">7JX3</a>) and hACE2 (PDB: <a id="intref0310" href="http://firstglance.jmol.org/fg.htm?mol6m0j" target="_blank">6m0j</a>). Several subsequent rounds of model building and refinement were performed using Coot (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Emsley et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (19617)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), ISOLDE (<span class="dropBlock reference-citations"><a id="body-ref-sref11-3" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-3" aria-controls="sref11-3">Croll, 2018</a><div class="dropBlock__holder" data-db-target-of="sref11-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Croll, T.I.</div><div><strong>ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318002425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85048328149" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872003/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318002425&amp;pmid=29872003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Refmac5 (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Murshudov et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Murshudov, G.N. ∙ Skubák, P. ∙ Lebedev, A.A. ...</div><div><strong>REFMAC5 for the refinement of macromolecular crystal structures</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2011; <strong>67</strong>:355-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444911001314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-79953763877" target="_blank" aria-label="Open Scopus in new tab">Scopus (6601)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21460454/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444911001314&amp;pmid=21460454" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and MOE (<a id="intref0315" href="http://www.chemcomp.com" target="_blank">https://www.chemcomp.com</a>), to arrive at a final model for the quarternary complex.</div></section><section id="sec-4-4-12"><h4>Binding measurements using surface plasmon resonance (SPR)</h4><div id="p0345" role="paragraph">SPR binding measurements were performed using a Biacore T200 instrument. S protein was surface captured via anti-AviTag pAb (Genscript) covalently immobilized on a CM5 chip, RBD protein was surface captured via StrepTactin XT (Twin-Strep-Tag Capture Kit, IBA GmbH) covalently immobilized on a CM5 chip, and ACE2-mFc was surface captured via covalent immobilization of the Cytiva Mouse antibody capture kit on a C1 chip. Running buffer was HBS-EP+ pH 7.4 (Cytiva) and all measurements were performed at 25°C. All experiments were performed as single-cycle kinetics, with a 3-fold dilution series of monomeric hACE2 starting from 300&nbsp;nM, each concentration injected for 180 s, or a 3-fold dilution series of RBD starting from 50&nbsp;nM, each concentration injected for 240 s. All data were double reference-subtracted and fit to a binding model using Biacore Evaluation software. For one representative replicate, capture levels were normalized to WT for visualization. Binding data with hACE2 as analyte were fit to a 1:1 binding model. Binding data with RBD as analyte were fit to a Heterogeneous Ligand binding model, due to an artifactual kinetic phase with very slow dissociation that arises when RBD is an analyte; the lower affinity of the two K<sub>D</sub>s reported by the fit is reported as the K<sub>D</sub> of the RBD-ACE2 interaction (the two reported K<sub>D</sub>s are separated by at least two orders of magnitude for all fits). The measured K<sub>D</sub> for hACE2 binding to S is likely influenced by conformational dynamics of the RBDs in the context of the prefusion S trimer. Reported K<sub>D</sub>s are an average of 3-4 replicates measured on at least two separate days, with error given as SEM.</div><div id="p0350" role="paragraph">Differences between the SPR assay and the published DMS binding assay (<span class="dropBlock reference-citations"><a id="body-ref-sref78-7" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-7" aria-controls="sref78-7">Starr et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref78-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) include using targeted measurements of purified proteins expressed in mammalian cells versus yeast surface display, as well as the use of dimeric hACE2 in the DMS experiment, which incorporates avidity effects into the RBD-hACE2 binding measurements that can mask modest changes in binding affinity.</div></section><section id="sec-4-4-13"><h4>hACE2 binding measurements using bio-layer interferometry (BLI)</h4><div id="p0355" role="paragraph">Binding measurements of Sarbecovirus RBDs to hACE2 were performed by diluting RBDs to 8&nbsp;μg/ml in kinetic buffer (PBS supplemented with 0.05% BSA) and immobilization on Anti-Penta-His Biosensors of an Octet RED96 system (FortéBio). RBD-coated biosensors were incubated for 5 min with a solution containing 5, 1 or 0.2 μg/ml of hACE2. A dissociation step was then performed by incubating the biosensors for 10 min in kinetic buffer. The change in material bound to the biosensors caused a shift in the interference pattern that was recorded in real time and plotted using GraphPad Prism 8 software.</div></section><section id="sec-4-4-14"><h4>Epidemiological and genome surveillance</h4><div id="p0360" role="paragraph">A national sequencing collaboration formed at the start of the epidemic in the UK, CoG-UK consortium (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">COVID-19 Genomics UK (COG-UK) Consortium, 2020</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>An integrated national scale SARS-CoV-2 genomic surveillance network</strong></div><div><em>Lancet Microbe.</em> 2020; <strong>1</strong>:e99-e100</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS2666-5247%2820%2930054-9&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS2666-5247%2820%2930054-9&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85100239304" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32835336/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS2666-5247%2820%2930054-9&amp;pmid=32835336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) has facilitated the tracking of SARS-CoV-2 sequences across Scotland since the start of the outbreak in February 2020 and real-time monitoring of genetic changes in the Spike gene that might be associated with changes in virulence or transmissibility. Sequencing was carried out using an amplicon-based protocol in real-time at a rate of up to 300 genomes per week. 50% of samples were selected as surveillance samples, representing Scottish health boards proportionately based on population size, while 50% were selected to allow intervention with local issues such as nosocomial infection in hospitals and nursing homes. The N439K mutation was noted to become increasingly prevalent during April 2020. This was noted to be particularly common in the Greater Glasgow &amp; Clyde NHS health board region but spread to adjacent Scottish health boards also.</div><div id="p0365" role="paragraph">Sequencing libraries were prepared according to the ARTIC nCoV-2019 described in detail at <a id="intref0320" href="https://artic.network/ncov-2019" target="_blank">https://artic.network/ncov-2019</a>. Briefly, PCR amplicons were generated using the nCoV-2019 PrimalSeq sequencing primers using 25-35 cycles of amplification. Generated amplicons were used to prepare either Oxford Nanopore or Illumina sequencing libraries. Oxford Nanopore libraries were prepared as described in the link above and sequenced in a flow cell R9.4.1 (Oxford Nanopore Technologies, Part Number FLO-MIN106D), using MinKNOW version 19.12.6. Raw FAST5 files were basecalled using Guppy version 3.2.10 in high accuracy mode with a minimum quality score of 7. Reads were size filtered, demultiplexed and trimmed with Porechop (<a id="intref0325" href="https://github.com/rrwick/Porechop" target="_blank">https://github.com/rrwick/Porechop</a>), and mapped against reference strain Wuhan-Hu-1 (MN908947). Variants were called using Nanopolish 0.11.3 and accepted if they had a log-likelihood score of greater than 200 and minimum read coverage of 20. For Illumina sequencing, amplicons were used to prepare libraries using the Kapa HyperPrep kit (Roche, Part Number KK8504) and further processed as described in the competition assay sequencing method. Sequencing was carried out on Illumina’s MiSeq system (Illumina, Part Number SY-410-1003) using a MiSeq Reagent v2 500 cycle kit (Illumina, Part Number MS-102-2003). Reads were trimmed with trim_galore (<a id="intref0330" href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>) and mapped with BWA (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (31658)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to the Wuhan-Hu-1 (MN908947) reference sequence, followed by primer trimming and consensus calling with iVar (<span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">Grubaugh et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Grubaugh, N.D. ∙ Gangavarapu, K. ∙ Quick, J. ...</div><div><strong>An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1618-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85059769331" target="_blank" aria-label="Open Scopus in new tab">Scopus (488)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30621750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1618-7&amp;pmid=30621750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and a minimum read coverage of 10.</div></section><section id="sec-4-4-15"><h4>Phylogenetic and phylodynamic analysis</h4><div id="p0370" role="paragraph">UK sequences were obtained from the COG-UK consortium (<a id="intref0335" href="https://www.cogconsortium.uk" target="_blank">https://www.cogconsortium.uk</a>) and global sequences from the GISAID Initiative (<a id="intref0340" href="https://www.gisaid.org" target="_blank">https://www.gisaid.org</a>) on November 23 2020. The sequences were mapped using minimap2 and padded against the Wuhan/WH04/2020 reference. The sequences were downsampled with weights that normalize sequence count per epiweek, maximize the number of countries and lineages represented, and enriching for sequences with the N439K mutation. A maximum-likelihood phylogenetic tree was constructed using IQ-TREE with the the following parameters: -czb -blmin 0.0000000001 -m HKY–runs 5 and all other parameters set to default. The tree was visualized with custom python code using the baltic library (<a id="intref0345" href="https://github.com/evogytis/baltic" target="_blank">https://github.com/evogytis/baltic</a>).</div><div id="p0375" role="paragraph">For the phylodynamic analysis, Scottish “introduction” lineages with ten or more sequences were identified (<span class="dropBlock reference-citations"><a id="body-ref-opte7fwGm6cg0" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opte7fwGm6cg0" aria-controls="opte7fwGm6cg0">Lycett et al., 2021</a><div class="dropBlock__holder" data-db-target-of="opte7fwGm6cg0"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Lycett, S.J. ∙ Hughes, J. ∙ McHugh, M.P. ...</div><div><strong>Epidemic waves of COVID-19 in Scotland: a genomic perspective on the impact of the introduction and relaxation of lockdown on SARS-CoV-2</strong></div><div><em>medRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.08.20248677" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101208596" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.08.20248677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the skygrowth package in R was used to estimate the effective population size over time (using up to ten time intervals), and the growth rates of the lineages within Scotland (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Volz and Frost, 2017</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Volz, E.M. ∙ Frost, S.D.W.</div><div><strong>Scalable relaxed clock phylogenetic dating</strong></div><div><em>Virus Evol.</em> 2017; <strong>3</strong>:vex025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ve/vex025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fve%2Fvex025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The data used for analysis were sampled between Feb 28, 2020 and Aug 18, 2020. Growth estimates were calculated for the intervals in between the time points that Ne is estimated over, from the TMRCA onward. Lineages with less than ten sequences in total, less than 50% and/or less than five Scottish sequences were excluded.</div></section><section id="sec-4-4-16"><h4>Evaluation of clinical samples</h4><div id="p0380" role="paragraph">Clinical samples submitted to the West of Scotland Specialist Virology Centre for SARS-CoV-2 diagnostic rt-PCR testing were selected for sequencing as part of the COVID-19&nbsp;UK Genomics UK Consortium (COG-UK) project, resulting in 1918 whole genome sequences originating from the NHS Greater Glasgow and Clyde Health Board region. Sequences were linked to electronic patient records and basic metadata including sample date, age, sex, admission to hospital and mortality at 28&nbsp;days post diagnosis extracted. The electronic patient records of a subset of 1591 patients underwent full case-note review and clinical severity was recorded based on a 4-level ordinal scale: 1. no requirement for respiratory support, 2. treatment with supplemental oxygen via facemask or low-flow nasal cannulae, 3. intubation and ventilation, non-invasive ventilation or oxygen delivery by high flow nasal cannulae devices, 4. death within the 28&nbsp;days following diagnosis. We modified the WHO ordinal scale to these 4 points as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref83-4" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-4" aria-controls="sref83-4">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to avoid using hospitalisation as a criterion of severity because 1) many patients in nursing homes had severe infection but were not admitted to hospital, and 2) early in the outbreak, all cases were hospitalised irrespective of the severity of their infection.</div><div id="p0385" role="paragraph">These data had previously been analyzed to test for an effect of the D614G mutation on the severity of disease (<span class="dropBlock reference-citations"><a id="body-ref-sref83-5" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-5" aria-controls="sref83-5">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); we extend that analysis here using the same methodology to test for an effect of the N439K mutation. Additionally, we perform a new analysis using a model with the same structure to test for an effect of both the D614G mutation and the D614G/N439K mutation combination on the viral load of infected patients, as measured by cycle threshold (Ct) value. Ct values were generated in different locations, on different platforms and then collected centrally.</div><div id="p0390" role="paragraph">In both cases we cannot estimate the marginal effect of the N439K mutation, as we only have the mutation on the 614G genetic background, so the individual effect of N439K cannot be separated from any potential epistatic interactions between the mutations.</div><div id="p0395" role="paragraph">Briefly, the structure of the model used previously (<span class="dropBlock reference-citations"><a id="body-ref-sref83-6" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-6" aria-controls="sref83-6">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and in the present study is a phylogenetic generalized additive model with mutation being the primary predictor of interest. The model controls for biological sex, age and the number of days since the first reported case in the dataset, with the latter two being included as penalised splines with a maximum of 30 knots. If the patient was part of a cluster of cases, this was included as a random effect, with individuals not part of clusters being assigned their own levels. Correlations driven by the rest of the genome are controlled for by a phylogenetic random effect using a correlation matrix generated under a Brownian motion assumption from a phylogeny estimated in IQ-TREE 2 v. 2.0.6 (<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Minh et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Minh, B.Q. ∙ Schmidt, H.A. ∙ Chernomor, O. ...</div><div><strong>IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:1530-1534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msaa015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084101286" target="_blank" aria-label="Open Scopus in new tab">Scopus (4768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsaa015&amp;pmid=32011700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using a HKY&nbsp;+ Γ model, masking the positions recommended by (<span class="dropBlock reference-citations"><a id="body-ref-opthayqePlwZZ" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opthayqePlwZZ" aria-controls="opthayqePlwZZ">De Maio et al. 2020</a><div class="dropBlock__holder" data-db-target-of="opthayqePlwZZ"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>De Maio, N. ∙ Walker, C. ∙ Borges, R. ...</div><div><strong>Issues with SARS-CoV-2 sequencing data</strong></div><div><em>nCoV-2019 Genomic Epidemiology.</em> 2020;</div><div><a id="interref0425" href="https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473" target="_blank">https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.De+MaioC.WalkerR.BorgesL.WeilgunyG.SlodkowiczN.GoldmanIssues+with+SARS-CoV-2+sequencing+datanCoV-2019+Genomic+Epidemiology2020https%3A%2F%2Fvirological.org%2Ft%2Fissues-with-sars-cov-2-sequencing-data%2F473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as of 22/7/2020 (<a id="intref0350" href="https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473/13" target="_blank">https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473/13</a>), rooted on the first sequenced SARS-CoV-2 genome (<span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Wu et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Wu, F. ∙ Zhao, S. ∙ Yu, B. ...</div><div><strong>A new coronavirus associated with human respiratory disease in China</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:265-269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2008-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081626097" target="_blank" aria-label="Open Scopus in new tab">Scopus (7705)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32015508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2008-3&amp;pmid=32015508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The priors for the severity model were those used in the previous analysis of this data. The priors for the model of the viral load were a student-t (mean&nbsp;= 20, scale&nbsp;= 10, degrees of freedom&nbsp;= 3) prior on the model intercept, a Gaussian (mean&nbsp;= 0, standard deviation&nbsp;= 10) prior over the fixed effects, and an exponential (lambda&nbsp;= 0.1) prior over the random effect, penalised spline and residual standard deviations.</div><div id="p0400" role="paragraph">There are two key structural differences between the model used previously (<span class="dropBlock reference-citations"><a id="body-ref-sref83-7" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-7" aria-controls="sref83-7">Volz et al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the model used here. First, mutation is a three level rather than two level factor (D614/N439, D614G/N439 and D614G/N439K) with the ancestral D614/N439 being the reference level. Second, as we are now interested in two mutations, we estimated the phylogeny used to control for the effect of&nbsp;the rest of the genome excluding both the nucleotide position underlying the D614G mutation and the nucleotide position underlying the N439K mutation (in addition to the sites from De Maio et&nbsp;al. mentioned above).</div><div id="p0405" role="paragraph">The severity model used a cumulative error structure while the model on the CT values used a Gaussian error structure. In both cases, the models were estimated in brms v. 2.13.5 (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Bürkner, 2018</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Bürkner, P.-C.</div><div><strong>Advanced bayesian multilevel modeling with the R package brms</strong></div><div><em>R J.</em> 2018; <strong>10</strong>:395-411</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.32614/RJ-2018-017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.32614%2FRJ-2018-017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The presented models had no divergent transitions, Rhat values less than 1.01, and appropriate bulk and tail effective sample sizes for all parameters. Shortest probability intervals were calculated using the R package SPIn v. 1.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Liu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Liu, Y. ∙ Gelman, A. ∙ Zheng, T.</div><div><strong>Simulation-efficient shortest probability intervals</strong></div><div><em>Stat. Comput.</em> 2015; <strong>25</strong>:809-819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11222-015-9563-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84932196966" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11222-015-9563-8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Trees were manipulated using ape v. 5.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Paradis and Schliep, 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Paradis, E. ∙ Schliep, K.</div><div><strong>ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:526-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty633" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85056898420" target="_blank" aria-label="Open Scopus in new tab">Scopus (4152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30016406/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty633&amp;pmid=30016406" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and dates were manipulated using lubridate v. 1.7.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Grolemund and Wickham, 2011</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Grolemund, G. ∙ Wickham, H.</div><div><strong>Dates and Times Made Easy with lubridate</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2011; <strong>40.3</strong>:1-25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.GrolemundH.WickhamDates+and+Times+Made+Easy+with+lubridateJ.%C2%A0Stat.+Softw.40.32011125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Analysis code is available at <a id="intref0355" href="https://github.com/dpascall/SARS-CoV-2-mutation-analysis" target="_blank">https://github.com/dpascall/SARS-CoV-2-mutation-analysis</a>.</div></section><section id="sec-4-4-17"><h4>qPCR of clinical samples</h4><div id="p0410" role="paragraph">All samples were tested in duplicate using the 2019-nCoV_N1 assay RT-qPCR assay (<a id="intref0360" href="https://www.fda.gov/media/134922/download" target="_blank">https://www.fda.gov/media/134922/download</a>). Ready-mixed primers and probe were obtained from IDT (Leuven, Belgium). PCR was carried out using NEB Luna Universal Probe One-Step RT-qPCR Kit (New England Biolabs, Herts, UK), primers and probe at 500&nbsp;nM and 127.5&nbsp;nM, respectively, and 5&nbsp;μL of RNA sample in a final volume of 20&nbsp;μL. No template negative controls were included after every seventh sample. Six ten-fold dilutions of SARS-CoV-2 RNA standards were tested in duplicate in each assay; standards were calibrated using a plasmid containing the N sequence that had been quantified using droplet digital PCR. Thermal cycling was performed on an Applied Biosystems 7500 Fast PCR instrument running SDS software v2.3 (Thermo Fisher Scientific) under the following conditions: 55°C for 10&nbsp;minutes and 95°C for 1&nbsp;minute followed by 45 cycles of 95°C for 10&nbsp;s and 58°C for 1&nbsp;minute. Assays were repeated if the reaction efficiency was&nbsp;&lt; 90% or the R2 value of the standard curve was ≤ 0.998. Where possible, testing of samples was repeated if the %CV of the duplicates was&nbsp;&lt; 10%.</div></section><section id="sec-4-4-18"><h4>Viral growth curve</h4><div id="p0415" role="paragraph">Vero E6-hACE2 cells (Vero E6 cells induced to overexpress hACE2) either with or without TMPRSS2 overexpression (S.J.R., S.B., and A.W., unpublished data) were seeded in a 12-well plate and inoculated with an MOI of 0.01 with either the GLA1 (N439/D614G) or GLA2 (N439K/D614G) virus isolates for 1 hr before washing the cells three times in PBS and replacing with DMEM supplemented with 2% FBS. 100&nbsp;uL of media was removed at each time point, RNA was extracted using the RNAdvance Blood kit (Beckman Coulter), and the presence of SARS-CoV-2 determined using 2019-nCOV-N1 assays (IDT) with an NEB Luna Universal Probe One-Step RT-qPCR Kit. A standard curve was used to determine the copy number present per mL of cell culture media. 100 uL of the fresh media was also tested for the presence of virus, which was undetectable in all wells. Experiment was performed in triplicate, each with an independent preparation of virus inoculum.</div></section><section id="sec-4-4-19"><h4>Competition assay</h4><div id="p0420" role="paragraph">Three T25 flasks were seeded with Vero E6-hACE2 or Vero E6-hACE2-TMPRSS2 and inoculated with either single viruses or both GLA1 and GLA2 virus strains at an MOI of 0.01 for 1&nbsp;hr. The flasks were washed three times with PBS, with 100 uL of the final wash being retained to determine the presence of free virus, before adding 5&nbsp;mL of fresh DMEM, supplemented with 2% FBS. At 24, 48, and 72&nbsp;hr, 500 uL of media was removed, which was replaced with 500 uL fresh media. 300 uL was used for RNA extraction using the RNAdvance Blood kit (Beckman Coulter) and NGS analysis of the frequencies of the specific positions within the spike protein. The single virus inoculations showed no alternations in the frequency of the amino acid positions and the final wash showing no free virus in the supernatant. We used an unbiased metagenomic NGS sequencing pipeline to quantify variation across the whole viral genome on the Illumina NGS Next Seq platform. Briefly, extracted nucleic acid was incubated with DNaseI (Thermo Fisher Scientific, Part Number AM2222), cleaned with RNA Ampure beads (Agencourt RNA Clean AMPure XP Beads, Beckman Coulter, Cat. No. A63987) followed by cDNA synthesis using SuperScript III (Thermo Fisher Scientific, Part Number 18080044) and NEBNext Ultra II Non-Directional RNA Second Strand Synthesis Module (New England Biolabs, Part Number E6111L). Samples were further processed using the Kapa LTP Library Preparation Kit for Illumina Platforms (Roche, Part Number KK8232) and indexed with the NEBNext Multiplex Oligos for Illumina 96&nbsp;Unique Dual Index Primer Pairs (New England Biolabs, Part Number E6442S). Libraries were characterized utilizing using the Qubit dsDNA HS Kit (Thermo Fisher Scientific, Cat No. Q32854) and the sequenced on Illumina’s NextSeq 550 System (Illumina, Part Number SY-415-1002), using a NextSeq 500/550 High Output Kit v2.5 (300 Cycles) (Illumina, Cat. No. 20024908) generating aproximately 10 million pairs of reads per sample. Experiment was performed in triplicate, each with an independent preparation of virus inoculum.</div></section><section id="sec-4-4-20"><h4>Ab discovery and recombinant expression</h4><div id="p0425" role="paragraph">Human mAbs were isolated from plasma cells or memory B cells of SARS-CoV or SARS-CoV-2 immune donors, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Corti et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Corti, D. ∙ Voss, J. ∙ Gamblin, S.J. ...</div><div><strong>A neutralizing antibody selected from plasma cells that binds to group 1 and group 2 influenza A hemagglutinins</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:850-856</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1205669" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-80051670323" target="_blank" aria-label="Open Scopus in new tab">Scopus (965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21798894/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1205669&amp;pmid=21798894" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-6" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-6" aria-controls="sref66-6">Pinto et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-5" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-5" aria-controls="sref81-5">Tortorici et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref81-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Tortorici, M.A. ∙ Beltramello, M. ∙ Lempp, F.A. ...</div><div><strong>Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:950-957</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe3354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096508515" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32972994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe3354&amp;pmid=32972994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recombinant antibodies were expressed in ExpiCHO cells at 37°C and 8% CO<sub>2</sub>. Cells were transfected using ExpiFectamine. Transfected cells were supplemented 1&nbsp;day after transfection with ExpiCHO Feed and ExpiFectamine CHO Enhancer. Cell culture supernatant was collected eight days after transfection and filtered through a 0.2&nbsp;μm filter. Recombinant antibodies were affinity purified on an ÄKTA xpress FPLC device using 5&nbsp;mL HiTrap MabSelect PrismA columns followed by buffer exchange to Histidine buffer (20&nbsp;mM Histidine, 8% sucrose, pH 6) using HiPrep 26/10 desalting columns.</div></section><section id="sec-4-4-21"><h4>Enzyme-linked immunosorbent assay (ELISA)</h4><div id="p0430" role="paragraph">A total of 144 human monoclonal antibodies or 442 human sera were tested for binding to RBD WT and mutants, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref65-12" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-12" aria-controls="sref65-12">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Spectraplate-384 plates with high protein binding treatment (custom made from Perkin Elmer) were coated overnight at 4 °C with 0.1 μg/mL (for mAbs) or 5 ug/mL (for sera) SARS-CoV-2 RBD WT, N439K, K417V or N439K/K417V in phosphate-buffered saline (PBS), pH 7.2. Plates were subsequently blocked with Blocker Casein 1% supplemented with 0.05% Tween 20 (Sigma-Aldrich) for 1&nbsp;h at room temperature. The coated plates were incubated with serial dilutions of the monoclonal antibodies or of the sera for 1 h at room temperature. The plates were then washed with PBS containing 0.05% Tween-20 (PBS-T), and alkaline phosphatase-goat anti-human IgG (Southern Biotech) was added and incubated for 1&nbsp;h at room temperature. After 3 washing steps with PBS-T, p-NitroPhenyl Phosphate (pNPP, Sigma-Aldrich) substrate was added and incubated for 30&nbsp;min at room temperature. The absorbance of 405 nm was measured by a microplate reader (Synergy H1 Hybrid Multi-Mode plate reader, Biotek). For mAbs, fitting was performed using the drc R package with a 4-parameter logistic (4PL) model, yielding dose-response curves from which the area under the curve (AUC) between 5 and 500&nbsp;ng/mL was computed. The AUC allows to capture, in a single metric, shifts in two parameters of the 4PL model: EC<sub>50</sub> and upper asymptote. For sera, fitting was performed using GraphPrism 8 with a 4PL model from which ED<sub>50</sub> was calculated. A cutoff of 30 for serum ED<sub>50</sub> was set based on previously published data (<span class="dropBlock reference-citations"><a id="body-ref-sref65-13" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-13" aria-controls="sref65-13">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A difference in reactivity to RBD mutants compared to WT was considered when a minimum 2-fold-variation in the AUC or ED<sub>50</sub> WT/mutant ratios was observed in two independent experiments (for mAbs, fold variation as low as 1.7 was accepted provided that the average of two experiments was greater or equal to 2-fold).</div></section><section id="sec-4-4-22"><h4>Antibody binding measurements using bio-layer interferometry (BLI)</h4><div id="p0435" role="paragraph">BLI binding measurement was performed on a selection of human monoclonal antibodies tested by ELISA (16/144). Antibodies were diluted to 2.7&nbsp;μg/mL in kinetic buffer (PBS supplemented with 0.05% BSA) and immobilized on Protein A Biosensors of an Octet RED96 system (FortéBio). Antibody-coated biosensors were incubated for 5 min with a solution containing 5 μg /mL of SARS-CoV2 RBD WT, N439K, K417V or N439/K417V in kinetic buffer. A dissociation step was then performed by incubating the biosensors for 5 min in kinetic buffer. Change in molecules bound to the biosensors caused a shift in the interference pattern that was recorded in real time and plotted using GraphPad Prism 8 software.</div></section><section id="sec-4-4-23"><h4>Blockade of RBD binding to ACE2</h4><div id="p0440" role="paragraph">Blockade of WT RBD binding to hACE2 was performed, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref65-14" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-14" aria-controls="sref65-14">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unlabeled mAbs were serially diluted, mixed with RBD mouse Fc-tagged antigen (Sino Biological, final concentration 20&nbsp;ng/mL) and incubated for 30&nbsp;min at 37°C. The mix was added for 30&nbsp;min to ELISA 96-well plates (Corning) pre-coated overnight at 4°C with 2&nbsp;μg/mL hACE2 in PBS. Plates were washed (EL406 washer/dispenser BSL2 M, Biotek) and RBD binding was revealed using a secondary goat anti-mouse IgG (Southern Biotech). After washing, pNPP substrate was added and plates were read at 405&nbsp;nm (Synergy H1 Hybrid Multi-Mode plate reader, Biotek). The percentage of inhibition was calculated as follow: (1−(OD sample−OD neg ctr)/(OD pos ctr−OD neg ctr)) × 100.</div></section><section id="sec-4-4-24"><h4>RBD epitope mapping (blockade of binding assay)</h4><div id="p0445" role="paragraph">RBD epitope mapping of the 144 mAbs was performed through blockade of binding (BOB) assay as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref65-15" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-15" aria-controls="sref65-15">Piccoli et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref65-15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Human mAbs binding to RBD site I (S2H14), site II (S304) and site IV (S309) were biotinylated using the EZ-Link NHS-PEO solid phase biotinylation kit (Pierce). Labeled mAbs were tested for binding to RBD by ELISA and the optimal concentration of each mAb to achieve 80% maximal binding was determined. Unlabeled mAbs were serially diluted and added to ELISA 96-well plates (Corning) pre-coated overnight at 4°C with 1&nbsp;μg/mL of RBD mouse Fc-tagged antigen (Sino Biological) in PBS. After 30&nbsp;min, biotinylated anti-RBD mAbs were added at the concentration achieving 80% maximal binding and the mixture was incubated at room temperature for 20&nbsp;min. Plates were washed (EL406 washer/dispenser BSL2 M, Biotek) and antibody binding was revealed using alkaline phosphatase-conjugated streptavidin (Jackson ImmunoResearch). After washing, pNPP substrate (Sigma-Aldrich) was added and plates were read at 405&nbsp;nm (Synergy H1 Hybrid Multi-Mode plate reader, Biotek). The percentage of inhibition was calculated as follow: (1−(OD sample−OD neg ctr)/ (OD pos ctr−OD neg ctr)) × 100.</div></section><section id="sec-4-4-25"><h4>VSV pseudovirus generation</h4><div id="p0450" role="paragraph">Replication defective VSV pseudovirus (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Takada et&nbsp;al., 1997</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Takada, A. ∙ Robison, C. ∙ Goto, H. ...</div><div><strong>A system for functional analysis of Ebola virus glycoprotein</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:14764-14769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.26.14764" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0031449064" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9405687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.26.14764&amp;pmid=9405687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) expressing SARS-CoV-2 spike protein were generated as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Riblett et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Riblett, A.M. ∙ Blomen, V.A. ∙ Jae, L.T. ...</div><div><strong>A Haploid Genetic Screen Identifies Heparan Sulfate Proteoglycans Supporting Rift Valley Fever Virus Infection</strong></div><div><em>J.&nbsp;Virol.</em> 2015; <strong>90</strong>:1414-1423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02055-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84958074109" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26581979/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02055-15&amp;pmid=26581979" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with some modifications. Plasmids encoding SARS-CoV-2 spike variants were generated by site-directed mutagenesis of the wild-type plasmid, pcDNA3.1(+)-spike-D19 (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Giroglou et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Giroglou, T. ∙ Cinatl, Jr., J. ∙ Rabenau, H. ...</div><div><strong>Retroviral vectors pseudotyped with severe acute respiratory syndrome coronavirus S protein</strong></div><div><em>J.&nbsp;Virol.</em> 2004; <strong>78</strong>:9007-9015</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.78.17.9007-9015.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-4143132180" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15308697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.78.17.9007-9015.2004&amp;pmid=15308697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lenti-X 293T cells (Takara, 632180) were seeded in 10-cm dishes at a density of 1e5 cells/cm<sup>2</sup> and the following day transfected with 5&nbsp;μg of spike expression plasmid with TransIT-Lenti (Mirus, 6600) according to the manufacturer’s instructions. One day post-transfection, cells were infected with VSV-luc (VSV-G) (Kerafast, EH1020-PM) for 1 h, rinsed three times with PBS, then incubated for an additional 24&nbsp;h in complete media at 37°C. The cell supernatant was clarified by centrifugation, filtered (0.45&nbsp;μm), aliquoted, and frozen at −80°C.</div></section><section id="sec-4-4-26"><h4>Pseudovirus neutralization</h4><div id="p0455" role="paragraph">Vero E6 cells (ATCC CRL-1586) were grown in DMEM supplemented with 10% FBS and seeded into clear bottom white 96 well plates (Costar, 3903) at a density of 2e4 cells per well. The next day, mAbs were serially diluted in pre-warmed complete media, mixed at a 1:1 ratio with pseudovirus and incubated for 1&nbsp;h at 37°C in round bottom polypropylene plates. Media from cells was aspirated and 50&nbsp;μL of virus-mAb complexes were added to cells and then incubated for 1&nbsp;h at 37°C. An additional 100&nbsp;μL of prewarmed complete media was then added on top of complexes and cells incubated for an additional 16-24 h. Conditions were tested in duplicate wells on each plate and at least six wells per plate contained uninfected, untreated cells (mock) and infected, untreated cells (‘no mAb control’). Virus-mAb-containing media was then aspirated from cells and 100&nbsp;uL of a 1:4 dilution of Bio-glo (Promega, G7940) in PBS was added to cells. Plates were incubated for 10&nbsp;min at room temperature and then were analyzed on the Envision plate reader (PerkinElmer). Relative light units (RLUs) for infected wells were subtracted by the average of RLU values for the mock wells (background subtraction) and then normalized to the average of background subtracted “no mAb control” RLU values within each plate. Percent neutralization was calculated by subtracting from 1 the normalized mAb infection condition. Data were analyzed and visualized with Prism (Version 8.4.3). IC<sub>50</sub> and IC80 values were calculated from the interpolated value from the log(inhibitor) versus response – variable slope (four parameters) nonlinear regression with an upper constraint of&nbsp;&lt; 100. Each neutralization experiment was conducted on three independent days, i.e., biological replicates, where each biological replicate contains a technical duplicate. IC<sub>50</sub> values across biological replicates are presented as geometric mean ± geometric standard deviation. The loss or gain of neutralization potency across spike variants was calculated by dividing the variant IC<sub>50</sub> by the parental (D614G) IC<sub>50</sub> within each biological replicate, and then visualized as geometric mean ± geometric standard deviation.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0460" role="paragraph">Quantification and statistical analyses were performed using GraphPad Prism (v8), R, and Biacore T200 Evaluation software, as described in the <a id="crosref1760" href="#sec-4-4">Method details</a>.</div></section></section><section id="sec-5"><h2>Consortia</h2><div id="p0465" role="paragraph">The members of the COG-UK consortia are Thomas R. Connor, Nicholas J. Loman, Samuel C. Robson, Tanya Golubchik, M. Estee Torok, William L. Hamilton, David Bonsall, Ali R. Awan, Sally Corden, Ian Goodfellow, Darren L. Smith, Martin D. Curran, Surendra Parmar, James G. Shepherd, Matthew D. Parker, Catherine Moore, Derek J. Fairley, Matthew W. Loose, Joanne Watkins, Matthew Bull, Sam Nicholls, David M. Aanensen, Sharon Glaysher, Matthew Bashton, Nicole Pacchiarini, Anthony P. Underwood, Thushan I. de Silva, Dennis Wang, Monique Andersson, Anoop J. Chauhan, Mariateresa de Cesare, Catherine Ludden, Tabitha W. Mahungu, Rebecca Dewar, Martin P. McHugh, Natasha G. Jesudason, Kathy K. Li, Rajiv N. Shah, Yusri Taha, Kate E. Templeton, Simon Cottrell, Justin O’Grady, Andrew Rambaut, Colin P. Smith, Matthew T.G. Holden, Emma C. Thomson, Samuel Moses, Meera Chand, Chrystala Constantinidou, Alistair C. Darby, Julian A. Hiscox, Steve Paterson, Meera Unnikrishnan, Andrew J. Page, Erik M. Volz, Charlotte J. Houldcroft, Aminu S. Jahun, James P. McKenna, Luke W. Meredith, Andrew Nelson, Sarojini Pandey, Gregory R. Young, Anna Price, Sara Rey, Sunando Roy, Ben Temperton, Matthew Wyles, Stefan Rooke, Sharif Shaaban, Helen Adams, Yann Bourgeois, Katie F. Loveson, Áine O’Toole, Richard Stark, Ewan M. Harrison, David Heyburn, Sharon J. Peacock, David Buck, Michaela John, Dorota Jamrozy, Joshua Quick, Rahul Batra, Katherine L. Bellis, Beth Blane, Sophia T. Girgis, Angie Green, Anita Justice, Mark Kristiansen, Rachel J. Williams, Radoslaw Poplawski, Garry P. Scarlett, John A. Todd, Christophe Fraser, Judith Breuer, Sergi Castellano, Stephen L. Michell, Dimitris Gramatopoulos, Jonathan Edgeworth, Gemma L. Kay, Ana da Silva Filipe, Aaron R. Jeffries, Sascha Ott, Oliver Pybus, David L. Robertson, David A. Simpson, Chris Williams, Cressida Auckland, John Boyes, Samir Dervisevic, Sian Ellard, Sonia Goncalves, Emma J. Meader, Peter Muir, Husam Osman, Reenesh Prakash, Venkat Sivaprakasam, Ian B. Vipond, Jane A.H. Masoli, Nabil-Fareed Alikhan, Matthew Carlile, Noel Craine, Sam T. Haldenby, Nadine Holmes, Ronan A. Lyons, Christopher Moore, Malorie Perry, Ben Warne, Thomas Williams, Lisa Berry, Andrew Bosworth, Julianne Rose Brown, Sharon Campbell, Anna Casey, Gemma Clark, Jennifer Collins, Alison Cox, Thomas Davis, Gary Eltringham, Cariad Evans, Clive Graham, Fenella Halstead, Kathryn Ann Harris, Christopher Holmes, Stephanie Hutchings, Miren Iturriza-Gomara, Kate Johnson, Katie Jones, Alexander J. Keeley, Bridget A. Knight, Cherian Koshy, Steven Liggett, Hannah Lowe, Anita O. Lucaci, Jessica Lynch, Patrick C McClure, Nathan Moore, Matilde Mori, David G. Partridge, Pinglawathee Madona, Hannah M. Pymont, Paul Anthony Randell, Mohammad Raza, Felicity Ryan, Robert Shaw, Tim J. Sloan, Emma Swindells, Alexander Adams, Hibo Asad, Alec Birchley, Tony Thomas Brooks, Giselda Bucca, Ethan Butcher, Sarah L. Caddy, Laura G. Caller, Yasmin Chaudhry, Jason Coombes, Michelle Cronin, Patricia L. Dyal, Johnathan M. Evans, Laia Fina, Bree Gatica-Wilcox, Iliana Georgana, Lauren Gilbert, Lee Graham, Danielle C. Groves, Grant Hall, Ember Hilvers, Myra Hosmillo, Hannah Jones, Sophie Jones, Fahad A. Khokhar, Sara Kumziene-Summerhayes, George MacIntyre-Cockett, Rocio T. Martinez Nunez, Caoimhe McKerr, Claire McMurray, Richard Myers, Yasmin Nicole Panchbhaya, Malte L. Pinckert, Amy Plimmer, Joanne Stockton, Sarah Taylor, Alicia Thornton, Amy Trebes, Alexander J. Trotter, Helena Jane Tutill, Charlotte A. Williams, Anna Yakovleva, Wen C. Yew, Mohammad T. Alam, Laura Baxter, Olivia Boyd, Fabricia F. Nascimento, Timothy M. Freeman, Lily Geidelberg, Joseph Hughes, David Jorgensen, Benjamin B. Lindsey, Richard J. Orton, Manon Ragonnet-Cronin, Joel Southgate, Sreenu Vattipally, Igor Starinskij, Joshua B. Singer, Khalil Abudahab, Leonardo de Oliveira Martins, Thanh Le-Viet, Mirko Menegazzo, Ben E.W. Taylor, Corin A. Yeats, Sophie Palmer, Carol M. Churcher, Alisha Davies, Elen De Lacy, Fatima Downing, Sue Edward, Nikki Smith, Frances Bolt, Alex Alderton, Matt Berriman, Ian G. Charles, Nicholas Cortes, Tanya Curran, John Danesh, Sahar Eldirdiri, Ngozi Elumogo, Andrew Hattersley, Alison Holmes, Robin Howe, Rachel Jones, Anita Kenyon, Robert A. Kingsley, Dominic Kwiatkowski, Cordelia Langford, Jenifer Mason, Alison E. Mather, Lizzie Meadows, Sian Morgan, James Price, Trevor I. Robinson, Giri Shankar, John Wain, Mark A. Webber, Declan T. Bradley, Michael R. Chapman, Derrick Crooke, David Eyre, Martyn Guest, Huw Gulliver, Sarah Hoosdally, Christine Kitchen, Ian Merrick, Siddharth Mookerjee, Robert Munn, Timothy Peto, Will Potter, Dheeraj K Sethi, Wendy Smith, Luke B. Snell, Rachael Stanley, Claire Stuart, Elizabeth Wastenge, Erwan Acheson, Safiah Afifi, Elias Allara, Roberto Amato, Adrienn Angyal, Elihu Aranday-Cortes, Cristina Ariani, Jordan Ashworth, Stephen Attwood, Alp Aydin, David J. Baker, Carlos E. Balcazar, Angela Beckett, Robert Beer, Gilberto Betancor, Emma Betteridge, David Bibby, Daniel Bradshaw, Catherine Bresner, Hannah E. Bridgewater, Alice Broos, Rebecca Brown, Paul E. Brown, Kirstyn Brunker, Stephen N. Carmichael, Jeffrey K.J. Cheng, Rachel Colquhoun, Gavin Dabrera, Johnny Debebe, Eleanor Drury, Louis du Plessis, Richard Eccles, Nicholas Ellaby, Audrey Farbos, Ben Farr, Jacqueline Findlay, Chloe L. Fisher, Leysa Marie Forrest, Sarah Francois, Lucy R. Frost, William Fuller, Eileen Gallagher, Michael D. Gallagher, Matthew Gemmell, Rachel A.J. Gilroy, Scott Goodwin, Luke R. Green, Richard Gregory, Natalie Groves, James W. Harrison, Hassan Hartman, Andrew R. Hesketh, Verity Hill, Jonathan Hubb, Margaret Hughes, David K. Jackson, Ben Jackson, Keith James, Natasha Johnson, Ian Johnston, Jon-Paul Keatley, Moritz Kraemer, Angie Lackenby, Mara Lawniczak, David Lee, Rich Livett, Stephanie Lo, Daniel Mair, Joshua Maksimovic, Nikos Manesis, Robin Manley, Carmen Manso, Angela Marchbank, Inigo Martincorena, Tamyo Mbisa, Kathryn McCluggage, J.T. McCrone, Shahjahan Miah, Michelle L. Michelsen, Mari Morgan, Gaia Nebbia, Charlotte Nelson, Jenna Nichols, Paola Niola, Kyriaki Nomikou, Steve Palmer, Naomi Park, Yasmin A. Parr, Paul J. Parsons, Vineet Patel, Minal Patel, Clare Pearson, Steven Platt, Christoph Puethe, Mike Quail, Jayna Raghwani, Lucille Rainbow, Shavanthi Rajatileka, Mary Ramsay, Paola C. Resende Silva, Steven Rudder, Chris Ruis, Christine M. Sambles, Fei Sang, Ulf Schaefer, Emily Scher, Carol Scott, Lesley Shirley, Adrian W. Signell, John Sillitoe, Christen Smith, Katherine L. Smollett, Karla Spellman, Thomas D. Stanton, David J. Studholme, Grace Taylor-Joyce, Ana P. Tedim, Thomas Thompson, Nicholas M. Thomson, Scott Thurston, Lily Tong, Gerry Tonkin-Hill, Rachel M. Tucker, Edith E. Vamos, Tetyana Vasylyeva, Joanna Warwick-Dugdale, Danni Weldon, Mark Whitehead, David Williams, Kathleen A. Williamson, Harry D. Wilson, Trudy Workman, Muhammad Yasir, Xiaoyu Yu, Alex Zarebski, Evelien M. Adriaenssens, Shazaad S.Y. Ahmad, Adela Alcolea-Medina, John Allan, Patawee Asamaphan, Laura Atkinson, Paul Baker, Jonathan Ball, Edward Barton, Mathew A. Beale, Charlotte Beaver, Andrew Beggs, Andrew Bell, Duncan J Berger, Louise Berry, Claire M. Bewshea, Kelly Bicknell, Paul Bird, Chloe Bishop, Tim Boswell, Cassie Breen, Sarah K. Buddenborg, Shirelle Burton-Fanning, Vicki Chalker, Joseph G. Chappell, Themoula Charalampous, Claire Cormie, Nick Cortes, Lindsay J. Coupland, Angela Cowell, Rose K. Davidson, Joana Dias, Maria Diaz, Thomas Dibling, Matthew J. Dorman, Nichola Duckworth, Scott Elliott, Sarah Essex, Karlie Fallon, Theresa Feltwell, Vicki M Fleming, Sally Forrest, Luke Foulser, Maria V. Garcia-Casado, Artemis Gavriil, Ryan P. George, Laura Gifford, Harmeet K. Gill, Jane Greenaway, Luke Griffith, Ana Victoria Gutierrez, Antony D. Hale, Tanzina Haque, Katherine L. Harper, Ian Harrison, Judith Heaney, Thomas Helmer, Ellen E. Higginson, Richard Hopes, Hannah C. Howson-Wells, Adam D. Hunter, Robert Impey, Dianne Irish-Tavares, David A. Jackson, Kathryn A. Jackson, Amelia Joseph, Leanne Kane, Sally Kay, Leanne M. Kermack, Manjinder Khakh, Stephen P. Kidd, Anastasia Kolyva, Jack C.D. Lee, Laura Letchford, Nick Levene, Lisa J. Levett, Michelle M. Lister, Allyson Lloyd, Joshua Loh, Louissa R. Macfarlane-Smith, Nicholas W. Machin, Mailis Maes, Samantha McGuigan, Liz McMinn, Lamia Mestek-Boukhibar, Zoltan Molnar, Lynn Monaghan, Catrin Moore, Plamena Naydenova, Alexandra S. Neaverson, Rachel Nelson, Marc O. Niebel, Elaine O’Toole, Debra Padgett, Gaurang Patel, Brendan A.I. Payne, Liam Prestwood, Veena Raviprakash, Nicola Reynolds, Alex Richter, Esther Robinson, Hazel A. Rogers, Aileen Rowan, Garren Scott, Divya Shah, Nicola Sheriff, Graciela Sluga, Emily Souster, Michael Spencer-Chapman, Sushmita Sridhar, Tracey Swingler, Julian Tang, Graham P. Taylor, Theocharis Tsoleridis, Lance Turtle, Sarah Walsh, Michelle Wantoch, Joanne Watts, Sheila Waugh, Sam Weeks, Rebecca Williams, Iona Willingham, Emma L. Wise, Victoria Wright, Sarah Wyllie, Jamie Young, Amy Gaskin, Will Rowe, Igor Siveroni, and Robert Johnson. See <a id="crosref1765" href="#mmc4">Data S3</a> for affiliation information. The members of the ISARIC4C consortia are Consortium Lead Investigator: J. Kenneth Baillie; Chief Investigator: Malcolm G. Semple; Co-Lead Investigator: Peter J.M. Openshaw; ISARIC Clinical Coordinator: Gail Carson; Co-Investigators: Beatrice Alex, Benjamin Bach, Wendy S. Barclay, Debby Bogaert, Meera Chand, Graham S. Cooke, Annemarie B. Docherty, Jake Dunning, Ana da Silva Filipe, Tom Fletcher, Christopher A. Green, Ewen M. Harrison, Julian A. Hiscox, Antonia Ying Wai Ho, Peter W. Horby, Samreen Ijaz, Saye Khoo, Paul Klenerman, Andrew Law, Wei Shen Lim, Alexander J. Mentzer, Laura Merson, Alison M. Meynert, Mahdad Noursadeghi, Shona C. Moore, Massimo Palmarini, William A. Paxton, Georgios Pollakis, Nicholas Price, Andrew Rambaut, David L. Robertson, Clark D. Russell, Vanessa Sancho-Shimizu, Janet T. Scott, Thushan de Silva, Louise Sigfrid, Tom Solomon, Shiranee Sriskandan, David Stuart, Charlotte Summers, Richard S. Tedder, Emma C. Thomson, A.A. Roger Thompson, Ryan S. Thwaites, Lance C.W. Turtle, and Maria Zambon; Project Managers: Hayley Hardwick, Chloe Donohue, Ruth Lyons, Fiona Griffiths, and Wilna Oosthuyzen; Data Analysts: Lisa Norman, Riinu Pius, Tom M. Drake, Cameron J. Fairfield, Stephen Knight, Kenneth A. Mclean, Derek Murphy, and Catherine A. Shaw; Data and Information System Managers: Jo Dalton, James Lee, Daniel Plotkin, Michelle Girvan, Egle Saviciute, Stephanie Roberts, Janet Harrison, Laura Marsh, Marie Connor, Sophie Halpin, Clare Jackson, and Carrol Gamble; Data Integration and Presentation: Gary Leeming, Andrew Law, Murray Wham, Sara Clohisey, Ross Hendry, and James Scott-Brown; Material Management: William Greenhalf, Victoria Shaw, and Sarah McDonald; Patient Engagement: Seán Keating; Outbreak Laboratory Staff and Volunteers: Katie A. Ahmed, Jane A. Armstrong, Milton Ashworth, Innocent G. Asiimwe, Siddharth Bakshi, Samantha L. Barlow, Laura Booth, Benjamin Brennan, Katie Bullock, Benjamin W.A. Catterall, Jordan J. Clark, Emily A. Clarke, Sarah Cole, Louise Cooper, Helen Cox, Christopher Davis, Oslem Dincarslan, Chris Dunn, Philip Dyer, Angela Elliott, Anthony Evans, Lorna Finch, Lewis W.S. Fisher, Terry Foster, Isabel Garcia-Dorival, Willliam Greenhalf, Philip Gunning, Catherine Hartley, Antonia Ho, Rebecca L. Jensen, Christopher B. Jones, Trevor R. Jones, Shadia Khandaker, Katharine King, Robyn T. Kiy, Chrysa Koukorava, Annette Lake, Suzannah Lant, Diane Latawiec, L. Lavelle-Langham, Daniella Lefteri, Lauren Lett, Lucia A. Livoti, Maria Mancini, Sarah McDonald, Laurence McEvoy, John McLauchlan, Soeren Metelmann, Nahida S. Miah, Joanna Middleton, Joyce Mitchell, Shona C. Moore, Ellen G. Murphy, Rebekah Penrice-Randal, Jack Pilgrim, Tessa Prince, Will Reynolds, P. Matthew Ridley, Debby Sales, Victoria E. Shaw, Rebecca K. Shears, Benjamin Small, Krishanthi S. Subramaniam, Agnieska Szemiel, Aislynn Taggart, Jolanta Tanianis-Hughes, Jordan Thomas, Erwan Trochu, Libby van Tonder, Eve Wilcock, and J. Eunice Zhang; Local Principal Investigators: Kayode Adeniji, Daniel Agranoff, Ken Agwuh, Dhiraj Ail, Ana Alegria, Brian Angus, Abdul Ashish, Dougal Atkinson, Shahedal Bari, Gavin Barlow, Stella Barnass, Nicholas Barrett, Christopher Bassford, David Baxter, Michael Beadsworth, Jolanta Bernatoniene, John Berridge, Nicola Best, Pieter Bothma, David Brealey, Robin Brittain-Long, Naomi Bulteel, Tom Burden, Andrew Burtenshaw, Vikki Caruth, David Chadwick, Duncan Chambler, Nigel Chee, Jenny Child, Srikanth Chukkambotla, Tom Clark, Paul Collini, Catherine Cosgrove, Jason Cupitt, Maria-Teresa Cutino-Moguel, Paul Dark, Chris Dawson, Samir Dervisevic, Phil Donnison, Sam Douthwaite, Ingrid DuRand, Ahilanadan Dushianthan, Tristan Dyer, Cariad Evans, Chi Eziefula, Chrisopher Fegan, Adam Finn, Duncan Fullerton, Sanjeev Garg, Sanjeev Garg, Atul Garg, Effrossyni Gkrania-Klotsas, Jo Godden, Arthur Goldsmith, Clive Graham, Elaine Hardy, Stuart Hartshorn, Daniel Harvey, Peter Havalda, Daniel B. Hawcutt, Maria Hobrok, Luke Hodgson, Anil Hormis, Michael Jacobs, Susan Jain, Paul Jennings, Agilan Kaliappan, Vidya Kasipandian, Stephen Kegg, Michael Kelsey, Jason Kendall, Caroline Kerrison, Ian Kerslake, Oliver Koch, Gouri Koduri, George Koshy, Shondipon Laha, Steven Laird, Susan Larkin, Tamas Leiner, Patrick Lillie, James Limb, Vanessa Linnett, Jeff Little, Michael MacMahon, Emily MacNaughton, Ravish Mankregod, Huw Masson, Elijah Matovu, Katherine McCullough, Ruth McEwen, Manjula Meda, Gary Mills, Jane Minton, Mariyam Mirfenderesky, Kavya Mohandas, Quen Mok, James Moon, Elinoor Moore, Patrick Morgan, Craig Morris, Katherine Mortimore, Samuel Moses, Mbiye Mpenge, Rohinton Mulla, Michael Murphy, Megan Nagel, Thapas Nagarajan, Mark Nelson, Igor Otahal, Mark Pais, Selva Panchatsharam, Hassan Paraiso, Brij Patel, Natalie Pattison, Justin Pepperell, Mark Peters, Mandeep Phull, Stefania Pintus, Jagtur Singh Pooni, Frank Post, David Price, Rachel Prout, Nikolas Rae, Henrik Reschreiter, Tim Reynolds, Neil Richardson, Mark Roberts, Devender Roberts, Alistair Rose, Guy Rousseau, Brendan Ryan, Taranprit Saluja, Aarti Shah, Prad Shanmuga, Anil Sharma, Anna Shawcross, Jeremy Sizer, Manu Shankar-Hari, Richard Smith, Catherine Snelson, Nick Spittle, Nikki Staines, Tom Stambach, Richard Stewart, Pradeep Subudhi, Tamas Szakmany, Kate Tatham, Jo Thomas, Chris Thompson, Robert Thompson, Ascanio Tridente, Darell Tupper-Carey, Mary Twagira, Andrew Ustianowski, Nick Vallotton, Lisa Vincent-Smith, Shico Visuvanathan, Alan Vuylsteke, Sam Waddy, Rachel Wake, Andrew Walden, Ingeborg Welters, Tony Whitehouse, Paul Whittaker, Ashley Whittington, Meme Wijesinghe, Martin Williams, Lawrence Wilson, Sarah Wilson, Stephen Winchester, Martin Wiselka, Adam Wolverson, Daniel G. Wooton, Andrew Workman, Bryan Yates, and Peter Young.</div></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0470" role="paragraph">We thank all Scottish NHS virology laboratories who provided samples for sequencing and Scott Arkison for HPC maintenance. We thank Chiara Silacci-Fregni from Humabs BioMed, Sandra Jovic, Blanca Fernandez Rodriguez, and Federico Mele, from the Institute for Research in Biomedicine in Bellinzona, and Tatiana Terrot from Ente Ospedaliero Cantonale in Lugano for the help in collecting serum samples. We thank Cindy Ng for help with protein production, Jesse Bloom for helpful comments, and Siro Bianchi for help with the graphical abstract. We gratefully acknowledge the authors originating and submitting laboratories of the sequences from GISAID (<a id="intref0365" href="https://www.gisaid.org" target="_blank">https://www.gisaid.org</a>). Our genome sequence acknowledgments can be found in <a id="crosref1770" href="#mmc3">Data S2</a>. Molecular Biology Consortium beamline 4.2.2 of the Advanced Light Source, a DOE Office of Science User Facility under Contract DE-AC02-05CH11231, is supported in part by the ALS-ENABLE program funded by the NIH, National Institute of General Medical Sciences (P30 GM124169-01). The ISARIC WHO CCP-UK study protocol is available at <a id="intref0370" href="http://isaric4c.net/protocols" target="_blank">http://isaric4c.net/protocols</a>; study registry <a id="intref0375" href="https://www.isrctn.com/ISRCTN66726260" target="_blank">https://www.isrctn.com/ISRCTN66726260</a>. This work uses data provided by patients and collected by the NHS as part of their care and support #DataSavesLives. We are grateful to the 2,648 frontline NHS clinical and research staff and volunteer medical students who collected the data in challenging circumstances and the generosity of the participants and their families for their individual contributions in these difficult times. We also acknowledge the support of Jeremy J. Farrar and Nahoko Shindo. We are grateful to Gregory R. Bowman at Washington University in St. Louis, Joseph Coffland at Cauldron Development, Peter K. Eastman at Stanford University for developing and maintaining the Folding@home infrastructure, Amazon Web Services for critical Folding@home infrastructure support, and the many Folding@home volunteers who contributed their computational resources to this project (FAH Project 17311). We are also grateful to Rafal P. Wiewiora at Memorial Sloan Kettering Cancer Center and Redesign Science and Sukrit Singh at Washington University in St. Louis for their guidance on setting up and managing simulations on Folding@home. For funding, we thank the&nbsp;Medical Research Council (MRC) (MC UU 1201412), Wellcome Trust (Collaborators Award 206298/Z/17/Z-ARTIC Network), and Chief Scientist Office (Project COV/EDI/20/11). COG-UK is supported by funding from the MRC part of UK Research &amp; Innovation (UKRI), the NIH Research (NIHR) and Genome Research Limited, operating as the Wellcome Sanger Institute. We also thank NIHR (CO-CIN-01), MRC (MC_PC_19059), the NIHR Health Protection Research Unit (HPRU) in Emerging and Zoonotic Infections at University of Liverpool in partnership with Public Health England (PHE) and in collaboration with Liverpool School of Tropical Medicine and the University of Oxford (200907), NIHR HPRU in Respiratory Infections at Imperial College London with PHE (200927), Wellcome Trust and Department for International Development (DID; 215091/Z/18/Z), the Bill and Melinda Gates Foundation (OPP1209135), Liverpool Experimental Cancer Medicine Centre (C18616/A25153), NIHR Biomedical Research Centre at Imperial College London (IS-BRC-1215-20013), EU Platform for European Preparedness Against (Re-)emerging Epidemics (PREPARE; FP7 project 602525), and NIHR Clinical Research Network for providing infrastructure support for this research. P.J.M.O. is supported by an NIHR senior investigator award (201385). The views expressed are those of the authors and not necessarily those of the Department of Health and Social Care, DID, NIHR, MRC, Wellcome Trust, or PHE. J.D.C. acknowledges support from NIH (P30 CA008748, R01 GM121505, and R01 GM132386), NSF (CHI-1904822), and the Sloan Kettering Institute. F.S. is supported by the Henry Krenter Foundation. F.S. and the Institute for Research in Biomedicine are supported by the Helmut Horten Foundation. I.Z. acknowledges support from the Tri-Institutional PhD Program in Computational Biology and Medicine, Vir Biotechnology, the Molecular Sciences Software Institute, and the Sloan Kettering Institute. W.G.G. acknowledges support from Bayer and the Sloan Kettering Institute. T.I.C acknowledges support from Randy Read’s Wellcome Trust (209407/Z/17/Z).</div><section id="sec-6"><h3>Author contributions</h3><div id="p0475" role="paragraph">Conceived research, E.C.T., R. Spreafico, H.W.V., A.T., D.C., D.L.R., and G.S.; designed experiments, E.C.T., A.T., D.C., and G.S.; donors’ recruitment and sample collection for serological analysis and mAbs isolation, A.C., P.F., F.S., C.G., M.G., A. Riva, A.H., M.S., M.G.S., P.J.M.O., and J.K.B.; isolation of mAbs, D.P., K.C., F.Z., M.B., M.P., and E.C.; expression and purification of proteins: J.D., N.C., M.M., and S.J.; crystallization, X-ray structure determination and refinement, J.D., N.C., J. Nix, T.I.C., and G.S.; preparation and analysis of molecular dynamics simulations, I.Z., W.G.G., T.I.C., A.M.H., E.F., and J.D.C.; binding and neutralization assays, L.E.R., J.A.W., L.P., A.D.M., J.B., and S.J.; NGS sequencing and analysis, E.C.T., A.S.F., J.H., V.B.S., K.N., L.T., N. Johnson, D.M., K.S., S.C., and J. Nichols; phylogenetic and epidemiological analysis, S.L., J.H., R.M.C., B.J., A. Rambaut, A.O., S.J.L., and D.L.R.; cross-competition and growth assays of primary isolates, A.S.F., C.D., A.W., and S.J.R.; collected and analyzed clinical data, E.C.T., J.G.S., D.J.P., R. Shah, N. Jesudason, K.L., M.G.S., P.J.M.O., and J.K.B.; performed and analyzed real-time qPCR assays, R. Shah, N. Jesudason, and R.F.J.; analyzed data, E.C.T., L.E.R., J.G.S., R. Spreafico, J.A.W., L.P., D.J.P., A. Rambaut, J. Nix, J.D.I., S.J.L., D.L.R., and G.S.; wrote the manuscript, E.C.T., L.E.R., H.W.V., A.T., D.C., D.L.R., and G.S.; edited the manuscript, R. Spreafico, L.P., A.S.F., J.A.W., D.J.P., J.D., N.C., E.C., J.G.S., I.Z., W.G.G., and J.D.C.; supervised the project, E.C.T., D.L.R., G.S.</div></section><section id="sec-7"><h3>Declaration of interests</h3><div id="p0480" role="paragraph">L.E.R., R. Spreafico, J.A.W., L.P., J.D., N.C., M.M., A.D.M., J.B., D.P., K.C., F.Z., S.J., M.B., M.P., E.C., J.D.I., H.W.V., A.T., D.C., and G.S. are or were employees of Vir Biotechnology and may hold shares in Vir Biotechnology. C.G. is an external scientific advisor for Humabs BioMed SA. J. Nix and T.I.C. are consultants with Vir Biotechnology. M.G.S. declares interest in Integrum Scientific, Greensboro, NC, outside the scope of this work. J.D.C. is a current member of the Scientific Advisory Board of OpenEye Scientific Software and is a scientific consultant to Foresite Labs. The Chodera laboratory (I.Z., W.G.G., and J.D.C.) receives or has received funding from multiple sources, including the NIH, the National Science Foundation, the Parker Institute for Cancer Immunotherapy, Relay Therapeutics, Entasis Therapeutics, Silicon Therapeutics, EMD Serono (Merck KGaA), AstraZeneca, Vir Biotechnology, XtalPi, the Molecular Sciences Software Institute, the Starr Cancer Consortium, the Open Force Field Consortium, Cycle for Survival, a Louis V. Gerstner Young Investigator Award, and the Sloan Kettering Institute. A complete funding history for the Chodera lab can be found at <a id="intref0380" href="https://www.choderalab.org/funding" target="_blank">https://www.choderalab.org/funding</a>. The other authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (4)</h2><div id="p0490" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.037/attachment/2ce26ffb-d8e7-4e7b-b157-042b566515ab/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (754.32 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Document S1. Tables S1–S6</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.037/attachment/9be8102f-1280-4ca6-9416-74bfb8832572/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (2.09 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Data S1. Binding, competition, and neutralization data for mAbs and sera, related to Figures 6 and 7</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.037/attachment/adc4499f-62fe-4303-971a-b1493daaf200/mmc3.pdf" download="mmc3.pdf" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">PDF (4.00 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Data S2. GISAID acknowledgment table</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="document"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.01.037/attachment/efb7a2ae-5e72-4408-adb6-7c97d8387b32/mmc4.docx" download="mmc4.docx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Document (42.07 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Data S3. COG-UK consortium names and affiliations</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Baum, A. ∙ Fulton, B.O. ∙ Wloga, E. ...</div><div><strong>Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1014-1018</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0831" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087939157" target="_blank" aria-label="Open Scopus in new tab">Scopus (877)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0831&amp;pmid=32540904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Boni, M.F. ∙ Lemey, P. ∙ Jiang, X. ...</div><div><strong>Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1408-1417</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0771-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088964983" target="_blank" aria-label="Open Scopus in new tab">Scopus (599)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32724171/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0771-4&amp;pmid=32724171" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Bürkner, P.-C.</div><div><strong>Advanced bayesian multilevel modeling with the R package brms</strong></div><div><em>R J.</em> 2018; <strong>10</strong>:395-411</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.32614/RJ-2018-017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.32614%2FRJ-2018-017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Casalino, L. ∙ Gaieb, Z. ∙ Goldsmith, J.A. ...</div><div><strong>Beyond Shielding: The Roles of Glycans in the SARS-CoV-2 Spike Protein</strong></div><div><em>ACS Cent. Sci.</em> 2020; <strong>6</strong>:1722-1734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acscentsci.0c01056" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85092393131" target="_blank" aria-label="Open Scopus in new tab">Scopus (562)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33140034/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facscentsci.0c01056&amp;pmid=33140034" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Case, D.A. ∙ Cerutti, D.S. ∙ Cheatham, III, T.E. ...</div><div><strong>AMBER 2017</strong></div><div>University of California, San Francisco, 2017</div><div><a id="interref0420" href="https://ambermd.org/doc12/Amber17.pdf" target="_blank">https://ambermd.org/doc12/Amber17.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.A.CaseD.S.CeruttiT.E.CheathamIIIT.A.DardenAMBER+20172017University+of+California%2C+San+Franciscohttps%3A%2F%2Fambermd.org%2Fdoc12%2FAmber17.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Chen, P. ∙ Nirula, A. ∙ Heller, B. ...</div><div><strong>SARS-CoV-2 Neutralizing Antibody LY-CoV555 in Outpatients with Covid-19</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:229-237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2029849" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098855067" target="_blank" aria-label="Open Scopus in new tab">Scopus (939)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33113295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2029849&amp;pmid=33113295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Choi, B. ∙ Choudhary, M.C. ∙ Regan, J. ...</div><div><strong>Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2020; <strong>383</strong>:2291-2293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2031364" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097210304" target="_blank" aria-label="Open Scopus in new tab">Scopus (755)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33176080/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2031364&amp;pmid=33176080" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Conceicao, C. ∙ Thakur, N. ∙ Human, S. ...</div><div><strong>The SARS-CoV-2 Spike protein has a broad tropism for mammalian ACE2 proteins</strong></div><div><em>PLoS Biol.</em> 2020; <strong>18</strong>:e3001016</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098929344" target="_blank" aria-label="Open Scopus in new tab">Scopus (131)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33347434/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001016&amp;pmid=33347434" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Corti, D. ∙ Voss, J. ∙ Gamblin, S.J. ...</div><div><strong>A neutralizing antibody selected from plasma cells that binds to group 1 and group 2 influenza A hemagglutinins</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:850-856</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1205669" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-80051670323" target="_blank" aria-label="Open Scopus in new tab">Scopus (965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21798894/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1205669&amp;pmid=21798894" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>An integrated national scale SARS-CoV-2 genomic surveillance network</strong></div><div><em>Lancet Microbe.</em> 2020; <strong>1</strong>:e99-e100</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS2666-5247%2820%2930054-9&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS2666-5247%2820%2930054-9&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85100239304" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32835336/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS2666-5247%2820%2930054-9&amp;pmid=32835336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Croll, T.I.</div><div><strong>ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps</strong></div><div><em>Acta Crystallogr. D Struct. Biol.</em> 2018; <strong>74</strong>:519-530</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S2059798318002425" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85048328149" target="_blank" aria-label="Open Scopus in new tab">Scopus (694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29872003/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS2059798318002425&amp;pmid=29872003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>da Silva Filipe, A. ∙ Shepherd, J.G. ∙ Williams, T. ..., COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>Genomic epidemiology reveals multiple introductions of SARS-CoV-2 from mainland Europe into Scotland</strong></div><div><em>Nat. Microbiol.</em> 2021; <strong>6</strong>:112-122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-00838-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097954045" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33349681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-00838-z&amp;pmid=33349681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Darden, T. ∙ York, D. ∙ Pedersen, L.</div><div><strong>Particle mesh Ewald: An N log(N) method for Ewald sums in large systems</strong></div><div><em>J.&nbsp;Chem. Physiol.</em> 1993; <strong>98</strong>:10089</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.464397" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-33846823909" target="_blank" aria-label="Open Scopus in new tab">Scopus (23398)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.464397" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="opthayqePlwZZ" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opthayqePlwZZ" title="View in article">14.</a></div><div>De Maio, N. ∙ Walker, C. ∙ Borges, R. ...</div><div><strong>Issues with SARS-CoV-2 sequencing data</strong></div><div><em>nCoV-2019 Genomic Epidemiology.</em> 2020;</div><div><a id="interref0425" href="https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473" target="_blank">https://virological.org/t/issues-with-sars-cov-2-sequencing-data/473</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.De+MaioC.WalkerR.BorgesL.WeilgunyG.SlodkowiczN.GoldmanIssues+with+SARS-CoV-2+sequencing+datanCoV-2019+Genomic+Epidemiology2020https%3A%2F%2Fvirological.org%2Ft%2Fissues-with-sars-cov-2-sequencing-data%2F473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">15.</a></div><div>Dearlove, B. ∙ Lewitus, E. ∙ Bai, H. ...</div><div><strong>A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:23652-23662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2008281117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091566837" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32868447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2008281117&amp;pmid=32868447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">16.</a></div><div>Doud, M.B. ∙ Lee, J.M. ∙ Bloom, J.D.</div><div><strong>How single mutations affect viral escape from broad and narrow antibodies to H1 influenza hemagglutinin</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:1386</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-03665-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85045389473" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29643370/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-03665-3&amp;pmid=29643370" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">17.</a></div><div>Eastman, P. ∙ Swails, J. ∙ Chodera, J.D. ...</div><div><strong>OpenMM 7: Rapid development of high performance algorithms for molecular dynamics</strong></div><div><em>PLoS Comput. Biol.</em> 2017; <strong>13</strong>:e1005659</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1005659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85026639054" target="_blank" aria-label="Open Scopus in new tab">Scopus (1317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28746339/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1005659&amp;pmid=28746339" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">18.</a></div><div>Emsley, P. ∙ Lohkamp, B. ∙ Scott, W.G. ...</div><div><strong>Features and development of Coot</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:486-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444910007493" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-77949535720" target="_blank" aria-label="Open Scopus in new tab">Scopus (19617)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444910007493&amp;pmid=20383002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">19.</a></div><div>Folegatti, P.M. ∙ Ewer, K.J. ∙ Aley, P.K. ..., Oxford COVID Vaccine Trial Group</div><div><strong>Safety and immunogenicity of the ChAdOx1 nCoV-19 vaccine against SARS-CoV-2: a preliminary report of a phase 1/2, single-blind, randomised controlled trial</strong></div><div><em>Lancet.</em> 2020; <strong>396</strong>:467-478</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS0140-6736%2820%2931604-4&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS0140-6736%2820%2931604-4&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088146774" target="_blank" aria-label="Open Scopus in new tab">Scopus (1772)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32702298/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2820%2931604-4&amp;pmid=32702298" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">20.</a></div><div>Gaebler, C. ∙ Wang, Z. ∙ Lorenzi, J.C.C. ...</div><div><strong>Evolution of Antibody Immunity to SARS-CoV-2</strong></div><div><em>Nature.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03207-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85100142212" target="_blank" aria-label="Open Scopus in new tab">Scopus (1073)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33461210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03207-w&amp;pmid=33461210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">21.</a></div><div>Garrett Rappazzo, C. ∙ Tse, L.V. ∙ Kaku, C.I. ...</div><div><strong>Broad and potent activity against SARS-like viruses by an engineered human monoclonal antibody</strong></div><div><em>Science.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf4830" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101189212" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33495307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf4830&amp;pmid=33495307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">22.</a></div><div>Giroglou, T. ∙ Cinatl, Jr., J. ∙ Rabenau, H. ...</div><div><strong>Retroviral vectors pseudotyped with severe acute respiratory syndrome coronavirus S protein</strong></div><div><em>J.&nbsp;Virol.</em> 2004; <strong>78</strong>:9007-9015</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.78.17.9007-9015.2004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-4143132180" target="_blank" aria-label="Open Scopus in new tab">Scopus (122)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15308697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.78.17.9007-9015.2004&amp;pmid=15308697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">23.</a></div><div>Gowers, R.J. ∙ Linke, M. ∙ Barnoud, J. ...</div><div><strong>MDAnalysis: A Python Package for the Rapid Analysis of Molecular Dynamics Simulations</strong></div><div><em>Proceedings of the 15th Python in Science Conference.</em> 2016; 98-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.25080/Majora-629e541a-00e" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.25080%2FMajora-629e541a-00e" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations to the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host &amp; Microbe.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">25.</a></div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete mapping of mutations to the SARS-CoV-2 spike receptor-binding domain that escape antibody recognition</strong></div><div><em>Cell Host &amp; Microbe.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">26.</a></div><div>Grolemund, G. ∙ Wickham, H.</div><div><strong>Dates and Times Made Easy with lubridate</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2011; <strong>40.3</strong>:1-25</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.GrolemundH.WickhamDates+and+Times+Made+Easy+with+lubridateJ.%C2%A0Stat.+Softw.40.32011125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">27.</a></div><div>Grubaugh, N.D. ∙ Gangavarapu, K. ∙ Quick, J. ...</div><div><strong>An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:8</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-018-1618-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85059769331" target="_blank" aria-label="Open Scopus in new tab">Scopus (488)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30621750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-018-1618-7&amp;pmid=30621750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">28.</a></div><div>Hansen, J. ∙ Baum, A. ∙ Pascal, K.E. ...</div><div><strong>Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:1010-1014</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0827" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089787231" target="_blank" aria-label="Open Scopus in new tab">Scopus (833)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0827&amp;pmid=32540901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">29.</a></div><div>Harbison, A.M. ∙ Brosnan, L.P. ∙ Fenlon, K. ...</div><div><strong>Sequence-to-structure dependence of isolated IgG Fc complex biantennary N-glycans: a molecular dynamics study</strong></div><div><em>Glycobiology.</em> 2019; <strong>29</strong>:94-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/glycob/cwy097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85058598461" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30325416/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fglycob%2Fcwy097&amp;pmid=30325416" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">30.</a></div><div>Hou, Y.J. ∙ Chiba, S. ∙ Halfmann, P. ...</div><div><strong>SARS-CoV-2 D614G variant exhibits efficient replication ex&nbsp;vivo and transmission in&nbsp;vivo</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:1464-1468</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe8499" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098742011" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33184236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe8499&amp;pmid=33184236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">31.</a></div><div>Hu, J. ∙ He, C.-L. ∙ Gao, Q.-Z. ...</div><div><strong>D614G mutation of SARS-CoV-2 spike protein enhances viral infectivity</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.06.20.161323" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098900825" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.06.20.161323" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">32.</a></div><div>Jackson, L.A. ∙ Anderson, E.J. ∙ Rouphael, N.G. ...</div><div><strong>An mRNA Vaccine against SARS-CoV-2 - Preliminary Report</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2020; <strong>383</strong>:1920-1931</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2022483" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096120857" target="_blank" aria-label="Open Scopus in new tab">Scopus (2300)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32663912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2022483&amp;pmid=32663912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">33.</a></div><div>Jiang, S. ∙ Hillyer, C. ∙ Du, L.</div><div><strong>Neutralizing Antibodies against SARS-CoV-2 and Other Human Coronaviruses</strong></div><div><em>Trends Immunol.</em> 2020; <strong>41</strong>:355-359</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.it.2020.03.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.it.2020.03.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85082822449" target="_blank" aria-label="Open Scopus in new tab">Scopus (618)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32249063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2020.03.007&amp;pmid=32249063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">34.</a></div><div>Jones, B.E. ∙ Brown-Augsburger, P.L. ∙ Corbett, K.S. ...</div><div><strong>LY-CoV555, a rapidly isolated potent neutralizing antibody, provides protection in a non-human primate model of SARS-CoV-2 infection</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.09.30.318972" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85102722540" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.09.30.318972" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">35.</a></div><div>Jorgensen, W.L. ∙ Chandrasekhar, J. ∙ Madura, J.D. ...</div><div><strong>Comparison of simple potential functions for simulating liquid water</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 1983; <strong>79</strong>:926-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.445869" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0004016501" target="_blank" aria-label="Open Scopus in new tab">Scopus (32859)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.445869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">36.</a></div><div>Kabsch, W.</div><div><strong>Xds</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2010; <strong>66</strong>:125-132</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444909047337" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-76449099287" target="_blank" aria-label="Open Scopus in new tab">Scopus (12648)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444909047337&amp;pmid=20124692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">37.</a></div><div>Keech, C. ∙ Albert, G. ∙ Cho, I. ...</div><div><strong>Phase 1-2 Trial of a SARS-CoV-2 Recombinant Spike Protein Nanoparticle Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2020; <strong>383</strong>:2320-2332</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2026920" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091689534" target="_blank" aria-label="Open Scopus in new tab">Scopus (839)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32877576/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2026920&amp;pmid=32877576" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">38.</a></div><div>Kirkpatrick, E. ∙ Qiu, X. ∙ Wilson, P.C. ...</div><div><strong>The influenza virus hemagglutinin head evolves faster than the stalk domain</strong></div><div><em>Sci. Rep.</em> 2018; <strong>8</strong>:10432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-018-28706-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85049794119" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29992986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-018-28706-1&amp;pmid=29992986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">39.</a></div><div>Kirschner, K.N. ∙ Yongye, A.B. ∙ Tschampel, S.M. ...</div><div><strong>GLYCAM06: a generalizable biomolecular force field. Carbohydrates</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2008; <strong>29</strong>:622-655</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.20820" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-39049147617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1708)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17849372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.20820&amp;pmid=17849372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">40.</a></div><div>Kluyver, T. ∙ Ragan-Kelley, B. ∙ Pérez, F. ...</div><div><strong>Jupyter Notebooks – a publishing format for reproducible computational workflows</strong></div><div><div>Loizides, F. ∙ Scmidt, B. (Editors)</div></div><div><strong>Positioning and Power in Academic Publishing: Players, Agents and Agendas</strong></div><div>IOS Press, 2016; 87-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3233/978-1-61499-649-1-87" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087152625" target="_blank" aria-label="Open Scopus in new tab">Scopus (2684)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3233%2F978-1-61499-649-1-87" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">41.</a></div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827.e19</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">42.</a></div><div>Lan, J. ∙ Ge, J. ∙ Yu, J. ...</div><div><strong>Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:215-220</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2180-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85083295918" target="_blank" aria-label="Open Scopus in new tab">Scopus (4081)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32225176/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2180-5&amp;pmid=32225176" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">43.</a></div><div>Leimkuhler, B. ∙ Matthews, C.</div><div><strong>Robust and efficient configurational molecular sampling via Langevin dynamics</strong></div><div><em>J.&nbsp;Chem. Phys.</em> 2013; <strong>138</strong>:174102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1063/1.4802990" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84877787427" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23656109/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1063%2F1.4802990&amp;pmid=23656109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">44.</a></div><div>Li, H.</div><div><strong>Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</strong></div><div><em>arXiv.</em> 2013;</div><div>arXiv:1303.3997</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.LiAligning+sequence+reads%2C+clone+sequences+and+assembly+contigs+with+BWA-MEMarXiv2013arXiv%3A1303.3997" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">45.</a></div><div>Li, H.</div><div><strong>Minimap2: pairwise alignment for nucleotide sequences</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:3094-3100</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85054132627" target="_blank" aria-label="Open Scopus in new tab">Scopus (5152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty191&amp;pmid=29750242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">46.</a></div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (31658)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">47.</a></div><div>Li, F. ∙ Li, W. ∙ Farzan, M. ...</div><div><strong>Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</strong></div><div><em>Science.</em> 2005; <strong>309</strong>:1864-1868</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1116480" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-24944498409" target="_blank" aria-label="Open Scopus in new tab">Scopus (1524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16166518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1116480&amp;pmid=16166518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">48.</a></div><div>Li, P. ∙ Song, L.F. ∙ Merz, Jr., K.M.</div><div><strong>Systematic Parameterization of Monovalent Ions Employing the Nonbonded Model</strong></div><div><em>J.&nbsp;Chem. Theory Comput.</em> 2015; <strong>11</strong>:1645-1657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/ct500918t" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84927941776" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26574374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Fct500918t&amp;pmid=26574374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">49.</a></div><div>Li, Q. ∙ Wu, J. ∙ Nie, J. ...</div><div><strong>The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1284-1294.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.07.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.07.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088997441" target="_blank" aria-label="Open Scopus in new tab">Scopus (1084)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32730807/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.07.012&amp;pmid=32730807" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">50.</a></div><div>Li, X. ∙ Giorgi, E.E. ∙ Marichannegowda, M.H. ...</div><div><strong>Emergence of SARS-CoV-2 through recombination and strong purifying selection</strong></div><div><em>Sci. Adv.</em> 2020; <strong>6</strong>:eabb9153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.abb9153" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85088461737" target="_blank" aria-label="Open Scopus in new tab">Scopus (241)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32937441/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.abb9153&amp;pmid=32937441" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">51.</a></div><div>Li, X. ∙ Wang, W. ∙ Zhao, X. ...</div><div><strong>Transmission dynamics and evolutionary history of 2019-nCoV</strong></div><div><em>J.&nbsp;Med. Virol.</em> 2020; <strong>92</strong>:501-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jmv.25701" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85079436157" target="_blank" aria-label="Open Scopus in new tab">Scopus (226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32027035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjmv.25701&amp;pmid=32027035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">52.</a></div><div>Liu, Y. ∙ Gelman, A. ∙ Zheng, T.</div><div><strong>Simulation-efficient shortest probability intervals</strong></div><div><em>Stat. Comput.</em> 2015; <strong>25</strong>:809-819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11222-015-9563-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84932196966" target="_blank" aria-label="Open Scopus in new tab">Scopus (13)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11222-015-9563-8" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">53.</a></div><div>Lorenzo-Redondo, R. ∙ Nam, H.H. ∙ Roberts, S.C. ...</div><div><strong>A clade of SARS-CoV-2 viruses associated with lower viral loads in patient upper airways</strong></div><div><em>EBioMedicine.</em> 2020; <strong>62</strong>:103112</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.ebiom.2020.103112&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.ebiom.2020.103112&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85095946935" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33186810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ebiom.2020.103112&amp;pmid=33186810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="opte7fwGm6cg0" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opte7fwGm6cg0" title="View in article">54.</a></div><div>Lycett, S.J. ∙ Hughes, J. ∙ McHugh, M.P. ...</div><div><strong>Epidemic waves of COVID-19 in Scotland: a genomic perspective on the impact of the introduction and relaxation of lockdown on SARS-CoV-2</strong></div><div><em>medRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.08.20248677" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101208596" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.08.20248677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">55.</a></div><div>Maier, J.A. ∙ Martinez, C. ∙ Kasavajhala, K. ...</div><div><strong>ff14SB: Improving the Accuracy of Protein Side Chain and Backbone Parameters from ff99SB</strong></div><div><em>J.&nbsp;Chem. Theory Comput.</em> 2015; <strong>11</strong>:3696-3713</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1021/acs.jctc.5b00255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84938930908" target="_blank" aria-label="Open Scopus in new tab">Scopus (6593)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26574453/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1021%2Facs.jctc.5b00255&amp;pmid=26574453" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">56.</a></div><div>McCarthy, K.R. ∙ Rennick, L.J. ∙ Nambulli, S. ...</div><div><strong>Recurrent deletions in the SARS-CoV-2 spike glycoprotein drive antibody escape</strong></div><div><em>Science.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf6950" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85101278912" target="_blank" aria-label="Open Scopus in new tab">Scopus (346)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33536258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf6950&amp;pmid=33536258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="opt0p615ydFV9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opt0p615ydFV9" title="View in article">57.</a></div><div>McCoy, A.L. ∙ Grosse-Kunstleve, R.W. ∙ Adams, P.D. ...</div><div><strong>Phaser crystallographic software</strong></div><div><em>J. Appl. Crystallogr.</em> 2007; <strong>40</strong>:658-674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0021889807021206" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-34447508216" target="_blank" aria-label="Open Scopus in new tab">Scopus (15801)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0021889807021206&amp;pmid=19461840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">58.</a></div><div>McGibbon, R.T. ∙ Beauchamp, K.A. ∙ Harrigan, M.P. ...</div><div><strong>MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</strong></div><div><em>Biophys. J.</em> 2015; <strong>109</strong>:1528-1532</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.bpj.2015.08.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.bpj.2015.08.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84944750932" target="_blank" aria-label="Open Scopus in new tab">Scopus (1229)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26488642/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bpj.2015.08.015&amp;pmid=26488642" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">59.</a></div><div>Meredith, L.W. ∙ Hamilton, W.L. ∙ Warne, B. ...</div><div><strong>Rapid implementation of SARS-CoV-2 sequencing to investigate cases of health-care associated COVID-19: a prospective genomic surveillance study</strong></div><div><em>Lancet Infect. Dis.</em> 2020; <strong>20</strong>:P1263-P1271</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS1473-3099%2820%2930562-4&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2FS1473-3099%2820%2930562-4&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089295953" target="_blank" aria-label="Open Scopus in new tab">Scopus (264)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32679081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2820%2930562-4&amp;pmid=32679081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">60.</a></div><div>Michaud-Agrawal, N. ∙ Denning, E.J. ∙ Woolf, T.B. ...</div><div><strong>MDAnalysis: a toolkit for the analysis of molecular dynamics simulations</strong></div><div><em>J.&nbsp;Comput. Chem.</em> 2011; <strong>32</strong>:2319-2327</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jcc.21787" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-79958185452" target="_blank" aria-label="Open Scopus in new tab">Scopus (1850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21500218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjcc.21787&amp;pmid=21500218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">61.</a></div><div>Minh, B.Q. ∙ Schmidt, H.A. ∙ Chernomor, O. ...</div><div><strong>IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:1530-1534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msaa015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084101286" target="_blank" aria-label="Open Scopus in new tab">Scopus (4768)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsaa015&amp;pmid=32011700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">62.</a></div><div>Mueller, N.F. ∙ Wagner, C. ∙ Frazar, C.D. ...</div><div><strong>Viral genomes reveal patterns of the SARS-CoV-2 outbreak in Washington State</strong></div><div><em>medRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.09.30.20204230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099556918" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.09.30.20204230" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">63.</a></div><div>Murshudov, G.N. ∙ Skubák, P. ∙ Lebedev, A.A. ...</div><div><strong>REFMAC5 for the refinement of macromolecular crystal structures</strong></div><div><em>Acta Crystallogr. D Biol. Crystallogr.</em> 2011; <strong>67</strong>:355-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1107/S0907444911001314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-79953763877" target="_blank" aria-label="Open Scopus in new tab">Scopus (6601)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21460454/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1107%2FS0907444911001314&amp;pmid=21460454" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">64.</a></div><div>Oude Munnink, B.B. ∙ Sikkema, R.S. ∙ Nieuwenhuijse, D.F. ...</div><div><strong>Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans</strong></div><div><em>Science.</em> 2021; <strong>371</strong>:172-177</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe5901" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099167478" target="_blank" aria-label="Open Scopus in new tab">Scopus (684)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33172935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe5901&amp;pmid=33172935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">65.</a></div><div>Paradis, E. ∙ Schliep, K.</div><div><strong>ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:526-528</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty633" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85056898420" target="_blank" aria-label="Open Scopus in new tab">Scopus (4152)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30016406/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty633&amp;pmid=30016406" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">66.</a></div><div>Perez, F. ∙ Granger, B.E.</div><div><strong>IPython: A System for Interactive Scientific Computing</strong></div><div><em>Comput. Sci. Eng.</em> 2007; <strong>9</strong>:21-29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/MCSE.2007.53" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-34247481878" target="_blank" aria-label="Open Scopus in new tab">Scopus (2954)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FMCSE.2007.53" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">67.</a></div><div>Pettersen, E.F. ∙ Goddard, T.D. ∙ Huang, C.C. ...</div><div><strong>UCSF ChimeraX: Structure visualization for researchers, educators, and developers</strong></div><div><em>Protein Sci.</em> 2021; <strong>30</strong>:70-82</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/pro.3943" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85093816376" target="_blank" aria-label="Open Scopus in new tab">Scopus (3298)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32881101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fpro.3943&amp;pmid=32881101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">68.</a></div><div>Piccoli, L. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Mapping Neutralizing and Immunodominant Sites on the SARS-CoV-2 Spike Receptor-Binding Domain by Structure-Guided High-Resolution Serology</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:1024-1042.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091685968" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.037&amp;pmid=32991844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">69.</a></div><div>Pinto, D. ∙ Park, Y.J. ∙ Beltramello, M. ...</div><div><strong>Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</strong></div><div><em>Nature.</em> 2020; <strong>583</strong>:290-295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2349-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085000117" target="_blank" aria-label="Open Scopus in new tab">Scopus (1209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32422645/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2349-y&amp;pmid=32422645" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">70.</a></div><div>Rambaut, A. ∙ Holmes, E.C. ∙ O’Toole, Á. ...</div><div><strong>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0770-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087930366" target="_blank" aria-label="Open Scopus in new tab">Scopus (1576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32669681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0770-5&amp;pmid=32669681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">71.</a></div><div>Riblett, A.M. ∙ Blomen, V.A. ∙ Jae, L.T. ...</div><div><strong>A Haploid Genetic Screen Identifies Heparan Sulfate Proteoglycans Supporting Rift Valley Fever Virus Infection</strong></div><div><em>J.&nbsp;Virol.</em> 2015; <strong>90</strong>:1414-1423</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02055-15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-84958074109" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26581979/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02055-15&amp;pmid=26581979" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">72.</a></div><div>Robbiani, D.F. ∙ Gaebler, C. ∙ Muecksch, F. ...</div><div><strong>Convergent antibody responses to SARS-CoV-2 in convalescent individuals</strong></div><div><em>Nature.</em> 2020; <strong>584</strong>:437-442</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2456-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85086589703" target="_blank" aria-label="Open Scopus in new tab">Scopus (1262)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32555388/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2456-9&amp;pmid=32555388" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">73.</a></div><div>Robson, F. ∙ Khan, K.S. ∙ Le, T.K. ...</div><div><strong>Coronavirus RNA Proofreading: Molecular Basis and Therapeutic Targeting</strong></div><div><em>Mol. Cell.</em> 2020; <strong>79</strong>:710-727</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.molcel.2020.07.027&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.molcel.2020.07.027&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089858704" target="_blank" aria-label="Open Scopus in new tab">Scopus (241)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32853546/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2020.07.027&amp;pmid=32853546" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">74.</a></div><div>Rodrigues, J.P.G.L.M. ∙ Teixeira, J.M.C. ∙ Trellet, M. ...</div><div><strong>pdb-tools: a swiss army knife for molecular structures</strong></div><div><em>F1000Res.</em> 2018; <strong>7</strong>:1961</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.17456.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85060775176" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30705752/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.17456.1&amp;pmid=30705752" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">75.</a></div><div>Rogers, T.F. ∙ Zhao, F. ∙ Huang, D. ...</div><div><strong>Isolation of potent SARS-CoV-2 neutralizing antibodies and protection from disease in a small animal model</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:956-963</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc7520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087691308" target="_blank" aria-label="Open Scopus in new tab">Scopus (891)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc7520&amp;pmid=32540903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">76.</a></div><div>Shajahan, A. ∙ Archer-Hartmann, S. ∙ Supekar, N.T. ...</div><div><strong>Comprehensive characterization of N- and O- glycosylation of SARS-CoV-2 human receptor angiotensin converting enzyme 2</strong></div><div><em>Glycobiology.</em> 2020;</div><div>Published online October 29, 2020</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/glycob/cwaa101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85106069987" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32363391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fglycob%2Fcwaa101&amp;pmid=32363391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">77.</a></div><div>Shang, J. ∙ Ye, G. ∙ Shi, K. ...</div><div><strong>Structural basis of receptor recognition by SARS-CoV-2</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:221-224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2179-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85083343318" target="_blank" aria-label="Open Scopus in new tab">Scopus (2572)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32225175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2179-y&amp;pmid=32225175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">78.</a></div><div>Shirts, M. ∙ Pande, V.S.</div><div><strong>COMPUTING: Screen Savers of the World Unite!</strong></div><div><em>Science.</em> 2000; <strong>290</strong>:1903-1904</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.290.5498.1903" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0034623787" target="_blank" aria-label="Open Scopus in new tab">Scopus (438)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17742054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.290.5498.1903&amp;pmid=17742054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">79.</a></div><div>Simões, E.A.F. ∙ Forleo-Neto, E. ∙ Geba, G.P. ...</div><div><strong>Suptavumab for the Prevention of Medically Attended Respiratory Syncytial Virus Infection in Preterm Infants</strong></div><div><em>Clin. Infect. Dis.</em> 2020; ciaa951</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/cid/ciaa951" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85122546142" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32897368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fcid%2Fciaa951&amp;pmid=32897368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">80.</a></div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Addetia, A. ...</div><div><strong>Prospective mapping of viral mutations that escape antibodies used to treat COVID-19</strong></div><div><em>bioRxiv.</em> 2020;</div><div><a id="interref0010" href="https://science.sciencemag.org/content/early/2021/01/22/science.abf9302" target="_blank">https://science.sciencemag.org/content/early/2021/01/22/science.abf9302</a></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.11.30.405472" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099854711" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.11.30.405472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">81.</a></div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">82.</a></div><div>Takada, A. ∙ Robison, C. ∙ Goto, H. ...</div><div><strong>A system for functional analysis of Ebola virus glycoprotein</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1997; <strong>94</strong>:14764-14769</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.94.26.14764" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-0031449064" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9405687/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.94.26.14764&amp;pmid=9405687" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80-1" title="View in article">83.</a></div><div>Tegally, H. ∙ Wilkinson, E. ∙ Giovanetti, M. ...</div><div><strong>Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa</strong></div><div><em>medRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.21.20248640" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85099895681" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.21.20248640" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">84.</a></div><div>Tortorici, M.A. ∙ Beltramello, M. ∙ Lempp, F.A. ...</div><div><strong>Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:950-957</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abe3354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096508515" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32972994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abe3354&amp;pmid=32972994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">85.</a></div><div>Volz, E.M. ∙ Frost, S.D.W.</div><div><strong>Scalable relaxed clock phylogenetic dating</strong></div><div><em>Virus Evol.</em> 2017; <strong>3</strong>:vex025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ve/vex025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fve%2Fvex025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">86.</a></div><div>Volz, E.M. ∙ Hill, V. ∙ McCrone, J.T. ...</div><div><strong>Evaluating the effects of SARS-CoV-2 Spike mutation D614G on transmissibility and pathogenicity</strong></div><div><em>Cell.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.11.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098925617" target="_blank" aria-label="Open Scopus in new tab">Scopus (663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.11.020&amp;pmid=33275900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84-1" title="View in article">87.</a></div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ...</div><div><strong>Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking epidemiological and genetic data</strong></div><div><em>medRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.30.20249034" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85120008977" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.30.20249034&amp;pmid=34981069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">88.</a></div><div>Walls, A.C. ∙ Park, Y.J. ∙ Tortorici, M.A. ...</div><div><strong>Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:281-292.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.02.058&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.02.058&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081283286" target="_blank" aria-label="Open Scopus in new tab">Scopus (5950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32155444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.02.058&amp;pmid=32155444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">89.</a></div><div>Watanabe, Y. ∙ Allen, J.D. ∙ Wrapp, D. ...</div><div><strong>Site-specific glycan analysis of the SARS-CoV-2 spike</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:330-333</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb9983" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85085763940" target="_blank" aria-label="Open Scopus in new tab">Scopus (1020)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32366695/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb9983&amp;pmid=32366695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">90.</a></div><div>Wec, A.Z. ∙ Wrapp, D. ∙ Herbert, A.S. ...</div><div><strong>Broad neutralization of SARS-related viruses by human monoclonal antibodies</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:731-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc7424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85087044294" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32540900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc7424&amp;pmid=32540900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">91.</a></div><div>Weisblum, Y. ∙ Schmidt, F. ∙ Zhang, F. ...</div><div><strong>Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants</strong></div><div><em>eLife.</em> 2020; <strong>9</strong>:e61312</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.61312" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096589275" target="_blank" aria-label="Open Scopus in new tab">Scopus (887)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33112236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.61312&amp;pmid=33112236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">92.</a></div><div>Weissman, D. ∙ Alameh, M.G. ∙ de Silva, T. ...</div><div><strong>D614G Spike Mutation Increases SARS CoV-2 Susceptibility to Neutralization</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:23-31.e4</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.11.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85097884640" target="_blank" aria-label="Open Scopus in new tab">Scopus (246)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33306985/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.012&amp;pmid=33306985" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">93.</a></div><div>Wrapp, D. ∙ De Vlieger, D. ∙ Corbett, K.S. ..., VIB-CMB COVID-19 Response Team</div><div><strong>Structural Basis for Potent Neutralization of Betacoronaviruses by Single-Domain Camelid Antibodies</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1004-1015.e15</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.04.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85084038496" target="_blank" aria-label="Open Scopus in new tab">Scopus (389)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32375025/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.031&amp;pmid=32375025" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">94.</a></div><div>Wrapp, D. ∙ Wang, N. ∙ Corbett, K.S. ...</div><div><strong>Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</strong></div><div><em>Science.</em> 2020; <strong>367</strong>:1260-1263</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb2507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081904519" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32075877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb2507&amp;pmid=32075877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">95.</a></div><div>Wu, F. ∙ Zhao, S. ∙ Yu, B. ...</div><div><strong>A new coronavirus associated with human respiratory disease in China</strong></div><div><em>Nature.</em> 2020; <strong>579</strong>:265-269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2008-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85081626097" target="_blank" aria-label="Open Scopus in new tab">Scopus (7705)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32015508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2008-3&amp;pmid=32015508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">96.</a></div><div>Wu, S.L. ∙ Mertens, A.N. ∙ Crider, Y.S. ...</div><div><strong>Substantial underestimation of SARS-CoV-2 infection in the United States</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:4507</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-18272-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85090393211" target="_blank" aria-label="Open Scopus in new tab">Scopus (239)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32908126/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-18272-4&amp;pmid=32908126" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">97.</a></div><div>Yurkovetskiy, L. ∙ Wang, X. ∙ Pascal, K.E. ...</div><div><strong>Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant</strong></div><div><em>Cell.</em> 2020; <strong>183</strong>:739-751.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.032&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.cell.2020.09.032&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85091680714" target="_blank" aria-label="Open Scopus in new tab">Scopus (677)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32991842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.09.032&amp;pmid=32991842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">98.</a></div><div>Zhang, L. ∙ Jackson, C.B. ∙ Mou, H. ...</div><div><strong>SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity</strong></div><div><em>Nature Communications.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19808-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85096795416" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19808-4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">99.</a></div><div>Zhao, P. ∙ Praissman, J.L. ∙ Grant, O.C. ...</div><div><strong>Virus-Receptor Interactions of Glycosylated SARS-CoV-2 Spike and Human ACE2 Receptor</strong></div><div><em>Cell Host Microbe.</em> 2020; <strong>28</strong>:586-601.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.08.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=10.1016%2Fj.chom.2020.08.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85090566044" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.08.004&amp;pmid=32841605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">100.</a></div><div>Zimmerman, M.I. ∙ Porter, J.R. ∙ Ward, M.D. ...</div><div><strong>SARS-CoV-2 Simulations Go Exascale to Capture Spike Opening and Reveal Cryptic Pockets Across the Proteome</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.06.27.175430" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.01.037&amp;key=2-s2.0-85098837589" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.06.27.175430" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/afd18e18-f3ee-464d-b3e8-2c0841f11e14/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/be2cc464-9d0b-46d8-9589-1fb1994776d5/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/56f7486a-7c31-4876-9cf6-0cd1868b7766/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/ce87eaed-bb97-4133-a3ed-bec54d1ef16f/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/b4611dfe-5ecf-4a04-91d3-c121f913aeda/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/129ea2e7-5114-452a-a75b-2dea0391b399/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/8f3370c3-66c1-4c5d-a1aa-e5d2f15403c4/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/1708c5e8-5f97-4988-86d7-5f65a8fcc1fb/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/951595be-6cb6-43a0-8864-71160a934fbf/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/5e900c8c-6443-4a4e-a5a6-2ce174b976ec/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/5c4e191f-28c9-418e-bcc2-b1446148b052/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/29852d9e-08e7-4f1f-b36e-75de13d51410/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/ecfd0e9b-6332-4917-a762-90e89612bfa9/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/asset/18eaf6c8-c9f2-408c-ac1c-93d32b917262/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00067%3Barticle%3Aarticle%3Apii%5C%3AS0092867421000805%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">544</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">653</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">13</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">46</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.01.037&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (4)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/attachment/2ce26ffb-d8e7-4e7b-b157-042b566515ab/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (754.32 KB)</span></a><div class="core-description">Document S1. Tables S1–S6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/attachment/9be8102f-1280-4ca6-9416-74bfb8832572/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (2.09 MB)</span></a><div class="core-description">Data S1. Binding, competition, and neutralization data for mAbs and sera, related to Figures 6 and 7</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/attachment/adc4499f-62fe-4303-971a-b1493daaf200/mmc3.pdf" download="mmc3.pdf"><span id="mmc3-link">PDF (4.00 MB)</span></a><div class="core-description">Data S2. GISAID acknowledgment table</div></div></div><div class="core-supplementary-material" data-category="document"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.01.037/attachment/efb7a2ae-5e72-4408-adb6-7c97d8387b32/mmc4.docx" download="mmc4.docx"><span id="mmc4-link">Document (42.07 KB)</span></a><div class="core-description">Data S3. COG-UK consortium names and affiliations</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421000805" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.01.037","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.01.037/asset/afd18e18-f3ee-464d-b3e8-2c0841f11e14/main.assets/gr1.jpg" height="463" width="504" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/a5cc9bf7-4dec-4e00-9eb1-3ed5c8a46eae/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.01.037/asset/be2cc464-9d0b-46d8-9589-1fb1994776d5/main.assets/figs1.jpg" height="649" width="654" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/b4f0584f-5b8a-4222-80c1-179e5663f17a/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.01.037/asset/56f7486a-7c31-4876-9cf6-0cd1868b7766/main.assets/gr2.jpg" height="688" width="374" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/54f1aa54-66e7-4a69-a03f-ddd988480e30/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.01.037/asset/ce87eaed-bb97-4133-a3ed-bec54d1ef16f/main.assets/figs2.jpg" height="954" width="711" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/34eb3058-a236-4656-8a40-dd28c2e03fda/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.01.037/asset/b4611dfe-5ecf-4a04-91d3-c121f913aeda/main.assets/gr3.jpg" height="424" width="504" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/0e81e35a-e259-47eb-854c-493805cfd243/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.01.037/asset/129ea2e7-5114-452a-a75b-2dea0391b399/main.assets/figs3.jpg" height="516" width="504" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/b317abb8-7ebd-4ff2-97e2-0b1ab476f3e3/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.01.037/asset/8f3370c3-66c1-4c5d-a1aa-e5d2f15403c4/main.assets/gr4.jpg" height="454" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/d7e5c7f5-02b7-4f84-86fb-dbc8d8dce027/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.01.037/asset/1708c5e8-5f97-4988-86d7-5f65a8fcc1fb/main.assets/gr5.jpg" height="592" width="504" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/df089412-f4ac-48e0-aa82-331802faec2d/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.01.037/asset/951595be-6cb6-43a0-8864-71160a934fbf/main.assets/gr6.jpg" height="666" width="654" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/8a54a9c7-b5ab-4e49-b159-3e567e934581/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.01.037/asset/5e900c8c-6443-4a4e-a5a6-2ce174b976ec/main.assets/figs4.jpg" height="855" width="770" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/29c83851-1e53-434f-9914-766fb64d430f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.01.037/asset/5c4e191f-28c9-418e-bcc2-b1446148b052/main.assets/figs5.jpg" height="918" width="641" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/9c757d14-877c-4e8b-be4d-d29d8d05a51d/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.01.037/asset/29852d9e-08e7-4f1f-b36e-75de13d51410/main.assets/figs6.jpg" height="629" width="504" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/fed283ea-2a65-475c-bfe4-75c2c8d71627/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.01.037/asset/ecfd0e9b-6332-4917-a762-90e89612bfa9/main.assets/gr7.jpg" height="652" width="504" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/95fab88f-b33e-4ba2-8c13-56d07f95552c/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.01.037/asset/18eaf6c8-c9f2-408c-ac1c-93d32b917262/main.assets/figs7.jpg" height="938" width="704" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/1cb403eb-c387-4470-898e-eeca83af8dae/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.01.037/asset/afd18e18-f3ee-464d-b3e8-2c0841f11e14/main.assets/gr1.jpg" height="463" width="504" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/a5cc9bf7-4dec-4e00-9eb1-3ed5c8a46eae/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.01.037/asset/be2cc464-9d0b-46d8-9589-1fb1994776d5/main.assets/figs1.jpg" height="649" width="654" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/b4f0584f-5b8a-4222-80c1-179e5663f17a/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.01.037/asset/56f7486a-7c31-4876-9cf6-0cd1868b7766/main.assets/gr2.jpg" height="688" width="374" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/54f1aa54-66e7-4a69-a03f-ddd988480e30/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.01.037/asset/ce87eaed-bb97-4133-a3ed-bec54d1ef16f/main.assets/figs2.jpg" height="954" width="711" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/34eb3058-a236-4656-8a40-dd28c2e03fda/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.01.037/asset/b4611dfe-5ecf-4a04-91d3-c121f913aeda/main.assets/gr3.jpg" height="424" width="504" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/0e81e35a-e259-47eb-854c-493805cfd243/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.01.037/asset/129ea2e7-5114-452a-a75b-2dea0391b399/main.assets/figs3.jpg" height="516" width="504" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/b317abb8-7ebd-4ff2-97e2-0b1ab476f3e3/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.01.037/asset/8f3370c3-66c1-4c5d-a1aa-e5d2f15403c4/main.assets/gr4.jpg" height="454" width="506" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/d7e5c7f5-02b7-4f84-86fb-dbc8d8dce027/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.01.037/asset/1708c5e8-5f97-4988-86d7-5f65a8fcc1fb/main.assets/gr5.jpg" height="592" width="504" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/df089412-f4ac-48e0-aa82-331802faec2d/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.01.037/asset/951595be-6cb6-43a0-8864-71160a934fbf/main.assets/gr6.jpg" height="666" width="654" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/8a54a9c7-b5ab-4e49-b159-3e567e934581/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.01.037/asset/5e900c8c-6443-4a4e-a5a6-2ce174b976ec/main.assets/figs4.jpg" height="855" width="770" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/29c83851-1e53-434f-9914-766fb64d430f/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.01.037/asset/5c4e191f-28c9-418e-bcc2-b1446148b052/main.assets/figs5.jpg" height="918" width="641" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/9c757d14-877c-4e8b-be4d-d29d8d05a51d/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.01.037/asset/29852d9e-08e7-4f1f-b36e-75de13d51410/main.assets/figs6.jpg" height="629" width="504" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/fed283ea-2a65-475c-bfe4-75c2c8d71627/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.01.037/asset/ecfd0e9b-6332-4917-a762-90e89612bfa9/main.assets/gr7.jpg" height="652" width="504" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/95fab88f-b33e-4ba2-8c13-56d07f95552c/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.01.037/asset/18eaf6c8-c9f2-408c-ac1c-93d32b917262/main.assets/figs7.jpg" height="938" width="704" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.01.037/asset/1cb403eb-c387-4470-898e-eeca83af8dae/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00080-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00080-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749589367322","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"5","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421000805","coverDate":"202103","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Circulating SARSCoV2 spike N439K variants maintain fitness while evading antibodymediated immunity","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbcfbda82427fd',t:'MTc0OTU4OTM2Ni4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbcfbda82427fd&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.01.037&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>