<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="98de9b6a-985e-425a-ac13-04292f14453b" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;article:article:pii\:S009286742200842X;requestedJournal:journal:cell;issue:issue:pii\:S0092867421X00189;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="The intrinsic and extrinsic effects of TET proteins during gastrulation">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00842-X">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/9af7c20c-69bd-433f-905b-97e6fa22656f/fx1.jpg">
    


        <meta property="og:description" content="Single-embryo, single-cell temporal models of embryos lacking Tet contribution, either
partially or fully, clarify the cell-intrinsic effects of the TET machinery from its
subsequent tissue-level ramifications. TET-mediated demethylation alters gene expression
in a lineage- and time-specific fashion, but such alterations can be overcome in the
presence of inter-cellular signals from neighboring cells.">
    











        <meta name="twitter:title" content="The intrinsic and extrinsic effects of TET proteins during gastrulation">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/9af7c20c-69bd-433f-905b-97e6fa22656f/fx1.jpg">
    


        <meta name="twitter:description" content="Single-embryo, single-cell temporal models of embryos lacking Tet contribution, either
partially or fully, clarify the cell-intrinsic effects of the TET machinery from its
subsequent tissue-level ramifications. TET-mediated demethylation alters gene expression
in a lineage- and time-specific fashion, but such alterations can be overcome in the
presence of inter-cellular signals from neighboring cells.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00842-X">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>The intrinsic and extrinsic effects of TET proteins during gastrulation: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="The intrinsic and extrinsic effects of TET proteins during gastrulation"> <meta name="citation_date" content="2022/08/18"> <meta name="citation_online_date" content="2022/07/30"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="17"> <meta name="citation_firstpage" content="3169"> <meta name="citation_lastpage" content="3185.e20"> <meta name="citation_pmid" content="35908548"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S009286742200842X"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S009286742200842X"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S009286742200842X"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.06.049"> <meta name="citation_pmid" content="35908548"> <meta name="citation_author" content="Saifeng Cheng"><meta name="citation_author_orcid" content="0000-0003-2712-3521"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Markus Mittnenzweig"><meta name="citation_author_orcid" content="0000-0002-8502-1702"><meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Yoav Mayshar"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Aviezer Lifshitz"><meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Marko Dunjić"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Yoach Rais"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Raz Ben-Yair"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Stephanie Gehrs"><meta name="citation_author_institution" content="Division of Vascular Oncology and Metastasis, German Cancer Research Center (DKFZ), Heidelberg, Germany"><meta name="citation_author_institution" content="European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany"><meta name="citation_author" content="Elad Chomsky"><meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Zohar Mukamel"><meta name="citation_author_orcid" content="0000-0002-1623-9564"><meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Hernan Rubinstein"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Katharina Schlereth"><meta name="citation_author_orcid" content="0000-0002-2109-7222"><meta name="citation_author_institution" content="Division of Vascular Oncology and Metastasis, German Cancer Research Center (DKFZ), Heidelberg, Germany"><meta name="citation_author_institution" content="European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany"><meta name="citation_author" content="Netta Reines"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Ayelet-Hashahar Orenbuch"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Amos Tanay"><meta name="citation_author_email" content="amos.tanay@weizmann.ac.il"><meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_author" content="Yonatan Stelzer"><meta name="citation_author_email" content="yonatan.stelzer@weizmann.ac.il"><meta name="citation_author_orcid" content="0000-0001-9207-1479"><meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel"><meta name="citation_keywords" content="single-cell genomics; developmental biology; mouse gastrulation; stem cells; cell fate decisions; DNA demethylation; epigenetics; genome editing"><meta name="citation_reference" content="citation_title=Ex utero mouse embryogenesis from pre-gastrulation to late organogenesis; citation_author=A. Aguilera-Castrejon; citation_author=B. Oldak; citation_author=T. Shani; citation_author=N. Ghanem; citation_author=C. Itzkovich; citation_author=S. Slomovich; citation_author=S. Tarazi; citation_author=J. Bayerl; citation_author=V. Chugaeva; citation_author=M. Ayyash; citation_author=et al.; citation_journal_title=Nature; citation_volume=593; citation_doi=10.1038/s41586-021-03416-3; citation_firstpage=119; citation_lastpage=124; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Multi-omics profiling of mouse gastrulation at single-cell resolution; citation_author=R. Argelaguet; citation_author=S.J. Clark; citation_author=H. Mohammed; citation_author=L.C. Stapel; citation_author=C. Krueger; citation_author=C.-A. Kapourani; citation_author=I. Imaz-Rosshandler; citation_author=T. Lohoff; citation_author=Y. Xiang; citation_author=C.W. Hanna; citation_author=et al.; citation_journal_title=Nature; citation_volume=576; citation_doi=10.1038/s41586-019-1825-8; citation_firstpage=487; citation_lastpage=491; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Making a commitment: cell lineage allocation and axis patterning in the early mouse embryo; citation_author=S.J. Arnold; citation_author=E.J. Robertson; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=10; citation_doi=10.1038/nrm2618; citation_firstpage=91; citation_lastpage=103; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=MetaCell: analysis of single-cell RNA-seq data using K-nn graph partitions; citation_author=Y. Baran; citation_author=A. Bercovich; citation_author=A. Sebe-Pedros; citation_author=Y. Lubling; citation_author=A. Giladi; citation_author=E. Chomsky; citation_author=Z. Meir; citation_author=M. Hoichman; citation_author=A. Lifshitz; citation_author=A. Tanay; citation_journal_title=Genome Biol.; citation_volume=20; citation_doi=10.1186/s13059-019-1812-2; citation_firstpage=206; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Cell proliferation in mammalian gastrulation: the ventral node and notochord are relatively quiescent; citation_author=D. Bellomo; citation_author=A. Lander; citation_author=I. Harragan; citation_author=N.A. Brown; citation_journal_title=Dev. Dyn.; citation_volume=205; citation_firstpage=471; citation_lastpage=485; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Molecular recording of mammalian embryogenesis; citation_author=M.M. Chan; citation_author=Z.D. Smith; citation_author=S. Grosswendt; citation_author=H. Kretzmer; citation_author=T.M. Norman; citation_author=B. Adamson; citation_author=M. Jost; citation_author=J.J. Quinn; citation_author=D. Yang; citation_author=M.G. Jones; citation_author=et al.; citation_journal_title=Nature; citation_volume=570; citation_doi=10.1038/s41586-019-1184-5; citation_firstpage=77; citation_lastpage=82; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The Vg1-related protein Gdf3 acts in a Nodal signaling pathway in the pre-gastrulation mouse embryo; citation_author=C. Chen; citation_author=S.M. Ware; citation_author=A. Sato; citation_author=D.E. Houston-Hawkins; citation_author=R. Habas; citation_author=M.M. Matzuk; citation_author=M.M. Shen; citation_author=C.W. Brown; citation_journal_title=Development; citation_volume=133; citation_doi=10.1242/dev.02210; citation_firstpage=319; citation_lastpage=329; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=TET-mediated DNA demethylation controls gastrulation by regulating Lefty–Nodal signalling; citation_author=H.-Q. Dai; citation_author=B.-A. Wang; citation_author=L. Yang; citation_author=J.-J. Chen; citation_author=G.-C. Zhu; citation_author=M.-L. Sun; citation_author=H. Ge; citation_author=R. Wang; citation_author=D.L. Chapman; citation_author=F. Tang; citation_author=et al.; citation_journal_title=Nature; citation_volume=538; citation_doi=10.1038/nature20095; citation_firstpage=528; citation_lastpage=532; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Loss of tet enzymes compromises proper differentiation of embryonic stem cells; citation_author=M.M. Dawlaty; citation_author=A. Breiling; citation_author=T. Le; citation_author=M.I. Barrasa; citation_author=G. Raddatz; citation_author=Q. Gao; citation_author=B.E. Powell; citation_author=A.W. Cheng; citation_author=K.F. Faull; citation_author=F. Lyko; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=29; citation_doi=10.1016/j.devcel.2014.03.003; citation_firstpage=102; citation_lastpage=111; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Combined deficiency of Tet1 and Tet2 causes epigenetic abnormalities but is compatible with postnatal development; citation_author=M.M. Dawlaty; citation_author=A. Breiling; citation_author=T. Le; citation_author=G. Raddatz; citation_author=M.I. Barrasa; citation_author=A.W. Cheng; citation_author=Q. Gao; citation_author=B.E. Powell; citation_author=Z. Li; citation_author=M. Xu; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=24; citation_doi=10.1016/j.devcel.2012.12.015; citation_firstpage=310; citation_lastpage=323; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Tet1 is dispensable for maintaining pluripotency and its loss is compatible with embryonic and postnatal development; citation_author=M.M. Dawlaty; citation_author=K. Ganz; citation_author=B.E. Powell; citation_author=Y.-C. Hu; citation_author=S. Markoulaki; citation_author=A.W. Cheng; citation_author=Q. Gao; citation_author=J. Kim; citation_author=S.-W. Choi; citation_author=D.C. Page; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=9; citation_doi=10.1016/j.stem.2011.07.010; citation_firstpage=166; citation_lastpage=175; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=QSER1 protects DNA methylation valleys from de novo methylation; citation_author=G. Dixon; citation_author=H. Pan; citation_author=D. Yang; citation_author=B.P. Rosen; citation_author=T. Jashari; citation_author=N. Verma; citation_author=J. Pulecio; citation_author=I. Caspi; citation_author=K. Lee; citation_author=S. Stransky; citation_author=et al.; citation_journal_title=Science; citation_volume=372; citation_doi=10.1126/science.abd0875; citation_firstpage=eabd0875; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions; citation_author=J.R. Dixon; citation_author=S. Selvaraj; citation_author=F. Yue; citation_author=A. Kim; citation_author=Y. Li; citation_author=Y. Shen; citation_author=M. Hu; citation_author=J.S. Liu; citation_author=B. Ren; citation_journal_title=Nature; citation_volume=485; citation_doi=10.1038/nature11082; citation_firstpage=376; citation_lastpage=380; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=A genome-scale map of DNA methylation turnover identifies site-specific dependencies of DNMT and TET activity; citation_author=P.A. Ginno; citation_author=D. Gaidatzis; citation_author=A. Feldmann; citation_author=L. Hoerner; citation_author=D. Imanci; citation_author=L. Burger; citation_author=F. Zilbermann; citation_author=A.H.F.M. Peters; citation_author=F. Edenhofer; citation_author=S.A. Smallwood; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_doi=10.1038/s41467-020-16354-x; citation_firstpage=2680; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Epigenetic regulator function through mouse gastrulation; citation_author=S. Grosswendt; citation_author=H. Kretzmer; citation_author=Z.D. Smith; citation_author=A.S. Kumar; citation_author=S. Hetzel; citation_author=L. Wittler; citation_author=S. Klages; citation_author=B. Timmermann; citation_author=S. Mukherji; citation_author=A. Meissner; citation_journal_title=Nature; citation_volume=584; citation_doi=10.1038/s41586-020-2552-x; citation_firstpage=102; citation_lastpage=108; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The role of Tet3 DNA dioxygenase in epigenetic reprogramming by oocytes; citation_author=T.-P. Gu; citation_author=F. Guo; citation_author=H. Yang; citation_author=H.-P. Wu; citation_author=G.-F. Xu; citation_author=W. Liu; citation_author=Z.-G. Xie; citation_author=L. Shi; citation_author=X. He; citation_author=S.G. Jin; citation_author=et al.; citation_journal_title=Nature; citation_volume=477; citation_doi=10.1038/nature10443; citation_firstpage=606; citation_lastpage=610; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Germline DNA demethylation dynamics and imprint erasure Through 5-hydroxymethylcytosine; citation_author=J.A. Hackett; citation_author=R. SenGupta; citation_author=J.J. Zylicz; citation_author=K. Murakami; citation_author=C. Lee; citation_author=T.A. Down; citation_author=M.A. Surani; citation_journal_title=Science; citation_volume=339; citation_doi=10.1126/science.1229277; citation_firstpage=448; citation_lastpage=452; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Mapping the mouse cell atlas by microwell-seq; citation_author=X. Han; citation_author=R. Wang; citation_author=Y. Zhou; citation_author=L. Fei; citation_author=H. Sun; citation_author=S. Lai; citation_author=A. Saadatpour; citation_author=Z. Zhou; citation_author=H. Chen; citation_author=F. Ye; citation_author=et al.; citation_journal_title=Cell; citation_volume=172; citation_doi=10.1016/j.cell.2018.02.001; citation_firstpage=1091; citation_lastpage=1107.e17; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Tet-mediated formation of 5-Carboxylcytosine and its excision by TDG in mammalian DNA; citation_author=Y.-F. He; citation_author=B.-Z. Li; citation_author=Z. Li; citation_author=P. Liu; citation_author=Y. Wang; citation_author=Q. Tang; citation_author=J. Ding; citation_author=Y. Jia; citation_author=Z. Chen; citation_author=L. Li; citation_author=et al.; citation_journal_title=Science; citation_volume=333; citation_doi=10.1126/science.1210944; citation_firstpage=1303; citation_lastpage=1307; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=5-mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation; citation_author=G.C. Hon; citation_author=C.-X. Song; citation_author=T. Du; citation_author=F. Jin; citation_author=S. Selvaraj; citation_author=A.Y. Lee; citation_author=C.A. Yen; citation_author=Z. Ye; citation_author=S.-Q. Mao; citation_author=B.-A. Wang; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=56; citation_doi=10.1016/j.molcel.2014.08.026; citation_firstpage=286; citation_lastpage=297; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A PCR-based method for RNA probes and applications in neuroscience; citation_author=R. Hua; citation_author=S. Yu; citation_author=M. Liu; citation_author=H. Li; citation_journal_title=Front. Neurosci.; citation_volume=12; citation_doi=10.3389/fnins.2018.00266; citation_firstpage=266; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification; citation_author=S. Ito; citation_author=A.C. D’Alessio; citation_author=O.V. Taranova; citation_author=K. Hong; citation_author=L.C. Sowers; citation_author=Y. Zhang; citation_journal_title=Nature; citation_volume=466; citation_doi=10.1038/nature09303; citation_firstpage=1129; citation_lastpage=1133; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Tet proteins can convert 5-methylcytosine to 5-Formylcytosine and 5-Carboxylcytosine; citation_author=S. Ito; citation_author=L. Shen; citation_author=Q. Dai; citation_author=S.C. Wu; citation_author=L.B. Collins; citation_author=J.A. Swenberg; citation_author=C. He; citation_author=Y. Zhang; citation_journal_title=Science; citation_volume=333; citation_doi=10.1126/science.1210597; citation_firstpage=1300; citation_lastpage=1303; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=DNA methylation disruption reshapes the hematopoietic differentiation landscape; citation_author=F. Izzo; citation_author=S.C. Lee; citation_author=A. Poran; citation_author=R. Chaligne; citation_author=F. Gaiti; citation_author=B. Gross; citation_author=R.R. Murali; citation_author=S.D. Deochand; citation_author=C. Ang; citation_author=P.W. Jones; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_doi=10.1038/s41588-020-0595-4; citation_firstpage=378; citation_lastpage=387; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=DNA-binding specificities of human transcription factors; citation_author=A. Jolma; citation_author=J. Yan; citation_author=T. Whitington; citation_author=J. Toivonen; citation_author=K.R. Nitta; citation_author=P. Rastas; citation_author=E. Morgunova; citation_author=M. Enge; citation_author=M. Taipale; citation_author=G. Wei; citation_author=et al.; citation_journal_title=Cell; citation_volume=152; citation_doi=10.1016/j.cell.2012.12.009; citation_firstpage=327; citation_lastpage=339; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Simultaneous deletion of the methylcytosine oxidases Tet1 and Tet3 increases transcriptome variability in early embryogenesis; citation_author=J. Kang; citation_author=M. Lienhard; citation_author=W.A. Pastor; citation_author=A. Chawla; citation_author=M. Novotny; citation_author=A. Tsagaratou; citation_author=R.S. Lasken; citation_author=E.C. Thompson; citation_author=M.A. Surani; citation_author=S.B. Koralov; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_doi=10.1073/pnas.1510510112; citation_firstpage=E4236; citation_lastpage=E4245; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Adeno-associated virus effectively mediates conditional gene modification in the brain; citation_author=B.K. Kaspar; citation_author=B. Vissel; citation_author=T. Bengoechea; citation_author=S. Crone; citation_author=L. Randolph-Moore; citation_author=R. Muller; citation_author=E.P. Brandon; citation_author=D. Schaffer; citation_author=I.M. Verma; citation_author=K.-F. Lee; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=99; citation_doi=10.1073/pnas.042678699; citation_firstpage=2320; citation_lastpage=2325; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing; citation_author=H. Keren-Shaul; citation_author=E. Kenigsberg; citation_author=D.A. Jaitin; citation_author=E. David; citation_author=F. Paul; citation_author=A. Tanay; citation_author=I. Amit; citation_journal_title=Nat. Protoc.; citation_volume=14; citation_doi=10.1038/s41596-019-0164-4; citation_firstpage=1841; citation_lastpage=1862; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Lineage-specific functions of TET1 in the postimplantation mouse embryo; citation_author=R. Khoueiry; citation_author=A. Sohni; citation_author=B. Thienpont; citation_author=X. Luo; citation_author=J.V. Velde; citation_author=M. Bartoccetti; citation_author=B. Boeckx; citation_author=A. Zwijsen; citation_author=A. Rao; citation_author=D. Lambrechts; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_doi=10.1038/ng.3868; citation_firstpage=1061; citation_lastpage=1072; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Ten-Eleven-Translocation 2 (TET2) negatively regulates homeostasis and differentiation of hematopoietic stem cells in mice; citation_author=M. Ko; citation_author=H.S. Bandukwala; citation_author=J. An; citation_author=E.D. Lamperti; citation_author=E.C. Thompson; citation_author=R. Hastie; citation_author=A. Tsangaratou; citation_author=K. Rajewsky; citation_author=S.B. Koralov; citation_author=A. Rao; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=108; citation_doi=10.1073/pnas.1112317108; citation_firstpage=14566; citation_lastpage=14571; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Tet1 and Tet2 regulate 5-hydroxymethylcytosine production and cell lineage specification in mouse embryonic stem cells; citation_author=K.P. Koh; citation_author=A. Yabuuchi; citation_author=S. Rao; citation_author=Y. Huang; citation_author=K. Cunniff; citation_author=J. Nardone; citation_author=A. Laiho; citation_author=M. Tahiliani; citation_author=C.A. Sommer; citation_author=G. Mostoslavsky; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=8; citation_doi=10.1016/j.stem.2011.01.008; citation_firstpage=200; citation_lastpage=213; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Controlled gene expression in primary Lgr5 organoid cultures; citation_author=B.-K. Koo; citation_author=D.E. Stange; citation_author=T. Sato; citation_author=W. Karthaus; citation_author=H.F. Farin; citation_author=M. Huch; citation_author=J.H. van Es; citation_author=H. Clevers; citation_journal_title=Nat. Methods; citation_volume=9; citation_doi=10.1038/nmeth.1802; citation_firstpage=81; citation_lastpage=83; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=RNA velocity of single cells; citation_author=G. La Manno; citation_author=R. Soldatov; citation_author=A. Zeisel; citation_author=E. Braun; citation_author=H. Hochgerner; citation_author=V. Petukhov; citation_author=K. Lidschreiber; citation_author=M.E. Kastriti; citation_author=P. Lönnerberg; citation_author=A. Furlan; citation_author=et al.; citation_journal_title=Nature; citation_volume=560; citation_doi=10.1038/s41586-018-0414-6; citation_firstpage=494; citation_lastpage=498; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_doi=10.1038/nmeth.1923; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Tet proteins influence the balance between neuroectodermal and mesodermal fate choice by inhibiting Wnt signaling; citation_author=X. Li; citation_author=X. Yue; citation_author=W.A. Pastor; citation_author=L. Lin; citation_author=R. Georges; citation_author=L. Chavez; citation_author=S.M. Evans; citation_author=A. Rao; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_doi=10.1073/pnas.1617802113; citation_firstpage=E8267; citation_lastpage=E8276; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Deletion of Tet2 in mice leads to dysregulated hematopoietic stem cells and subsequent development of myeloid malignancies; citation_author=Z. Li; citation_author=X. Cai; citation_author=C.-L. Cai; citation_author=J. Wang; citation_author=W. Zhang; citation_author=B.E. Petersen; citation_author=F.-C. Yang; citation_author=M. Xu; citation_journal_title=Blood; citation_volume=118; citation_doi=10.1182/blood-2010-12-325241; citation_firstpage=4509; citation_lastpage=4518; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Editing DNA methylation in the mammalian genome; citation_author=X.S. Liu; citation_author=H. Wu; citation_author=X. Ji; citation_author=Y. Stelzer; citation_author=X. Wu; citation_author=S. Czauderna; citation_author=J. Shu; citation_author=D. Dadon; citation_author=R.A. Young; citation_author=R. Jaenisch; citation_journal_title=Cell; citation_volume=167; citation_doi=10.1016/j.cell.2016.08.056; citation_firstpage=233; citation_lastpage=247.e17; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Paradoxical association of TET loss of function with genome-wide DNA hypomethylation; citation_author=I.F. López-Moyado; citation_author=A. Tsagaratou; citation_author=H. Yuita; citation_author=H. Seo; citation_author=B. Delatte; citation_author=S. Heinz; citation_author=C. Benner; citation_author=A. Rao; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_doi=10.1073/pnas.1903059116; citation_firstpage=16933; citation_lastpage=16942; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Developmental pluripotency-associated 4 (DPPA4) localized in active chromatin inhibits mouse embryonic stem cell differentiation into a primitive ectoderm lineage; citation_author=H. Masaki; citation_author=T. Nishida; citation_author=S. Kitajima; citation_author=K. Asahina; citation_author=H. Teraoka; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=282; citation_doi=10.1074/jbc.M703245200; citation_firstpage=33034; citation_lastpage=33042; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Single-cell analysis of clonal maintenance of transcriptional and epigenetic states in cancer cells; citation_author=Z. Meir; citation_author=Z. Mukamel; citation_author=E. Chomsky; citation_author=A. Lifshitz; citation_author=A. Tanay; citation_journal_title=Nat. Genet.; citation_volume=52; citation_doi=10.1038/s41588-020-0645-y; citation_firstpage=709; citation_lastpage=718; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A single-embryo, single-cell time-resolved model for mouse gastrulation; citation_author=M. Mittnenzweig; citation_author=Y. Mayshar; citation_author=S. Cheng; citation_author=R. Ben-Yair; citation_author=R. Hadas; citation_author=Y. Rais; citation_author=E. Chomsky; citation_author=N. Reines; citation_author=A. Uzonyi; citation_author=L. Lumerman; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_doi=10.1016/j.cell.2021.04.004; citation_firstpage=2825; citation_lastpage=2842.e22; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Single-cell landscape of transcriptional heterogeneity and cell fate decisions during mouse early gastrulation; citation_author=H. Mohammed; citation_author=I. Hernando-Herraez; citation_author=A. Savino; citation_author=A. Scialdone; citation_author=I. Macaulay; citation_author=C. Mulas; citation_author=T. Chandra; citation_author=T. Voet; citation_author=W. Dean; citation_author=J. Nichols; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=20; citation_doi=10.1016/j.celrep.2017.07.009; citation_firstpage=1215; citation_lastpage=1228; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Tet2 loss leads to increased hematopoietic stem cell self-renewal and myeloid transformation; citation_author=K. Moran-Crusio; citation_author=L. Reavie; citation_author=A. Shih; citation_author=O. Abdel-Wahab; citation_author=D. Ndiaye-Lobry; citation_author=C. Lobry; citation_author=M.E. Figueroa; citation_author=A. Vasanthakumar; citation_author=J. Patel; citation_author=X. Zhao; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=20; citation_doi=10.1016/j.ccr.2011.06.001; citation_firstpage=11; citation_lastpage=24; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Transition states and cell fate decisions in epigenetic landscapes; citation_author=N. Moris; citation_author=C. Pina; citation_author=A.M. Arias; citation_journal_title=Nat. Rev. Genet.; citation_volume=17; citation_doi=10.1038/nrg.2016.98; citation_firstpage=693; citation_lastpage=703; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Cell-cycle dynamics of chromosomal organization at single-cell resolution; citation_author=T. Nagano; citation_author=Y. Lubling; citation_author=C. Várnai; citation_author=C. Dudley; citation_author=W. Leung; citation_author=Y. Baran; citation_author=N. Mendelson Cohen; citation_author=S. Wingett; citation_author=P. Fraser; citation_author=A. Tanay; citation_journal_title=Nature; citation_volume=547; citation_doi=10.1038/nature23001; citation_firstpage=61; citation_lastpage=67; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Embryonic stem cells alone are able to support fetal development in the mouse; citation_author=A. Nagy; citation_author=E. Gócza; citation_author=E.M. Diaz; citation_author=V.R. Prideaux; citation_author=E. Iványi; citation_author=M. Markkula; citation_author=J. Rossant; citation_journal_title=Development; citation_volume=110; citation_firstpage=815; citation_lastpage=821; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Derivation of completely cell culture-derived mice from early-passage embryonic stem cells; citation_author=A. Nagy; citation_author=J. Rossant; citation_author=R. Nagy; citation_author=W. Abramow-Newerly; citation_author=J.C. Roder; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=90; citation_doi=10.1073/pnas.90.18.8424; citation_firstpage=8424; citation_lastpage=8428; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=The emergent landscape of the mouse gut endoderm at single-cell resolution; citation_author=S. Nowotschin; citation_author=M. Setty; citation_author=Y.-Y. Kuo; citation_author=V. Liu; citation_author=V. Garg; citation_author=R. Sharma; citation_author=C.S. Simon; citation_author=N. Saiz; citation_author=R. Gardner; citation_author=S.C. Boutet; citation_author=et al.; citation_journal_title=Nature; citation_volume=569; citation_doi=10.1038/s41586-019-1127-1; citation_firstpage=361; citation_lastpage=367; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing; citation_author=J.K. Nuñez; citation_author=J. Chen; citation_author=G.C. Pommier; citation_author=J.Z. Cogan; citation_author=J.M. Replogle; citation_author=C. Adriaens; citation_author=G.N. Ramadoss; citation_author=Q. Shi; citation_author=K.L. Hung; citation_author=A.J. Samelson; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_doi=10.1016/j.cell.2021.03.025; citation_firstpage=2503; citation_lastpage=2519.e17; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=UTF1, a novel transcriptional coactivator expressed in pluripotent embryonic stem cells and extra-embryonic cells; citation_author=A. Okuda; citation_author=A. Fukushima; citation_author=M. Nishimoto; citation_author=A. Orimo; citation_author=T. Yamagishi; citation_author=Y. Nabeshima; citation_author=M. Kuro-o; citation_author=Y. Nabeshima; citation_author=K. Boon; citation_author=M. Keaveney; citation_author=et al.; citation_journal_title=EMBO J.; citation_volume=17; citation_doi=10.1093/emboj/17.7.2019; citation_firstpage=2019; citation_lastpage=2032; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Ability of the hydrophobic FGF and basic TAT peptides to promote cellular uptake of recombinant Cre recombinase: A tool for efficient genetic engineering of mammalian genomes; citation_author=M. Peitz; citation_author=K. Pfannkuche; citation_author=K. Rajewsky; citation_author=F. Edenhofer; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=99; citation_doi=10.1073/pnas.032068699; citation_firstpage=4489; citation_lastpage=4494; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Molecular architecture of lineage allocation and tissue organization in early mouse embryo; citation_author=G. Peng; citation_author=S. Suo; citation_author=G. Cui; citation_author=F. Yu; citation_author=R. Wang; citation_author=J. Chen; citation_author=S. Chen; citation_author=Z. Liu; citation_author=G. Chen; citation_author=Y. Qian; citation_author=et al.; citation_journal_title=Nature; citation_volume=572; citation_doi=10.1038/s41586-019-1469-8; citation_firstpage=528; citation_lastpage=532; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=An optimized procedure for whole-mount in&nbsp;situ hybridization on mouse embryos and embryoid bodies; citation_author=D. Piette; citation_author=M. Hendrickx; citation_author=E. Willems; citation_author=C.R. Kemp; citation_author=L. Leyns; citation_journal_title=Nat. Protoc.; citation_volume=3; citation_doi=10.1038/nprot.2008.103; citation_firstpage=1194; citation_lastpage=1201; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=A single-cell molecular map of mouse gastrulation and early organogenesis; citation_author=B. Pijuan-Sala; citation_author=J.A. Griffiths; citation_author=C. Guibentif; citation_author=T.W. Hiscock; citation_author=W. Jawaid; citation_author=F.J. Calero-Nieto; citation_author=C. Mulas; citation_author=X. Ibarra-Soria; citation_author=R.C.V. Tyser; citation_author=D.L.L. Ho; citation_author=et al.; citation_journal_title=Nature; citation_volume=566; citation_doi=10.1038/s41586-019-0933-9; citation_firstpage=490; citation_lastpage=495; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Single-cell chromatin accessibility maps reveal regulatory programs driving early mouse organogenesis; citation_author=B. Pijuan-Sala; citation_author=N.K. Wilson; citation_author=J. Xia; citation_author=X. Hou; citation_author=R.L. Hannah; citation_author=S. Kinston; citation_author=F.J. Calero-Nieto; citation_author=O. Poirion; citation_author=S. Preissl; citation_author=F. Liu; citation_author=et al.; citation_journal_title=Nat. Cell Biol.; citation_volume=22; citation_doi=10.1038/s41556-020-0489-9; citation_firstpage=487; citation_lastpage=497; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Topologically associating domains are stable units of replication-timing regulation; citation_author=B.D. Pope; citation_author=T. Ryba; citation_author=V. Dileep; citation_author=F. Yue; citation_author=W. Wu; citation_author=O. Denas; citation_author=D.L. Vera; citation_author=Y. Wang; citation_author=R.S. Hansen; citation_author=T.K. Canfield; citation_author=et al.; citation_journal_title=Nature; citation_volume=515; citation_doi=10.1038/nature13986; citation_firstpage=402; citation_lastpage=405; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Deterministic direct reprogramming of somatic cells to pluripotency; citation_author=Y. Rais; citation_author=A. Zviran; citation_author=S. Geula; citation_author=O. Gafni; citation_author=E. Chomsky; citation_author=S. Viukov; citation_author=A.A. Mansour; citation_author=I. Caspi; citation_author=V. Krupalnik; citation_author=M. Zerbib; citation_author=et al.; citation_journal_title=Nature; citation_volume=502; citation_doi=10.1038/nature12587; citation_firstpage=65; citation_lastpage=70; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=MesP1: a novel basic helix-loop-helix protein expressed in the nascent mesodermal cells during mouse gastrulation; citation_author=Y. Saga; citation_author=N. Hata; citation_author=S. Kobayashi; citation_author=T. Magnuson; citation_author=M.F. Seldin; citation_author=M.M. Taketo; citation_journal_title=Development; citation_volume=122; citation_firstpage=2769; citation_lastpage=2778; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=NIH Image to ImageJ: 25 years of image analysis; citation_author=C.A. Schneider; citation_author=W.S. Rasband; citation_author=K.W. Eliceiri; citation_journal_title=Nat. Methods; citation_volume=9; citation_doi=10.1038/nmeth.2089; citation_firstpage=671; citation_lastpage=675; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Resolving early mesoderm diversification through single-cell expression profiling; citation_author=A. Scialdone; citation_author=Y. Tanaka; citation_author=W. Jawaid; citation_author=V. Moignard; citation_author=N.K. Wilson; citation_author=I.C. Macaulay; citation_author=J.C. Marioni; citation_author=B. Göttgens; citation_journal_title=Nature; citation_volume=535; citation_doi=10.1038/nature18633; citation_firstpage=289; citation_lastpage=293; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Epigenetic restriction of extraembryonic lineages mirrors the somatic transition to cancer; citation_author=Z.D. Smith; citation_author=J. Shi; citation_author=H. Gu; citation_author=J. Donaghey; citation_author=K. Clement; citation_author=D. Cacchiarelli; citation_author=A. Gnirke; citation_author=F. Michor; citation_author=A. Meissner; citation_journal_title=Nature; citation_volume=549; citation_doi=10.1038/nature23891; citation_firstpage=543; citation_lastpage=547; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs; citation_author=Y. Song; citation_author=P.R. van den Berg; citation_author=S. Markoulaki; citation_author=F. Soldner; citation_author=A. Dall’Agnese; citation_author=J.E. Henninger; citation_author=J. Drotar; citation_author=N. Rosenau; citation_author=M.A. Cohen; citation_author=R.A. Young; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=75; citation_doi=10.1016/j.molcel.2019.06.045; citation_firstpage=905; citation_lastpage=920.e6; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Embryo-scale, single-cell spatial transcriptomics; citation_author=S.R. Srivatsan; citation_author=M.C. Regier; citation_author=E. Barkan; citation_author=J.M. Franks; citation_author=J.S. Packer; citation_author=P. Grosjean; citation_author=M. Duran; citation_author=S. Saxton; citation_author=J.J. Ladd; citation_author=M. Spielmann; citation_author=et al.; citation_journal_title=Science; citation_volume=373; citation_doi=10.1126/science.abb9536; citation_firstpage=111; citation_lastpage=117; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Tracing dynamic changes of DNA methylation at single-cell resolution; citation_author=Y. Stelzer; citation_author=C.S. Shivalila; citation_author=F. Soldner; citation_author=S. Markoulaki; citation_author=R. Jaenisch; citation_journal_title=Cell; citation_volume=163; citation_doi=10.1016/j.cell.2015.08.046; citation_firstpage=218; citation_lastpage=229; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1; citation_author=M. Tahiliani; citation_author=K.P. Koh; citation_author=Y. Shen; citation_author=W.A. Pastor; citation_author=H. Bandukwala; citation_author=Y. Brudno; citation_author=S. Agarwal; citation_author=L.M. Iyer; citation_author=D.R. Liu; citation_author=L. Aravind; citation_author=et al.; citation_journal_title=Science; citation_volume=324; citation_doi=10.1126/science.1170116; citation_firstpage=930; citation_lastpage=935; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Mouse gastrulation: the formation of a mammalian body plan; citation_author=P.P.L. Tam; citation_author=R.R. Behringer; citation_journal_title=Mech. Dev.; citation_volume=68; citation_firstpage=3; citation_lastpage=25; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Gene function in mouse embryogenesis: get set for gastrulation; citation_author=P.P.L. Tam; citation_author=D.A.F. Loebel; citation_journal_title=Nat. Rev. Genet.; citation_volume=8; citation_doi=10.1038/nrg2084; citation_firstpage=368; citation_lastpage=381; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Single-cell and spatial transcriptomics reveal somitogenesis in gastruloids; citation_author=S.C. van den Brink; citation_author=A. Alemany; citation_author=V. van Batenburg; citation_author=N. Moris; citation_author=M. Blotenburg; citation_author=J. Vivié; citation_author=P. Baillie-Johnson; citation_author=J. Nichols; citation_author=K.F. Sonnen; citation_author=A. Martinez Arias; citation_author=et al.; citation_journal_title=Nature; citation_volume=582; citation_doi=10.1038/s41586-020-2024-3; citation_firstpage=405; citation_lastpage=409; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Stage-specific roles for Tet1 and Tet2 in DNA demethylation in primordial germ cells; citation_author=J.J. Vincent; citation_author=Y. Huang; citation_author=P.-Y. Chen; citation_author=S. Feng; citation_author=J.H. Calvopiña; citation_author=K. Nee; citation_author=S.A. Lee; citation_author=T. Le; citation_author=A.J. Yoon; citation_author=K. Faull; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=12; citation_doi=10.1016/j.stem.2013.01.016; citation_firstpage=470; citation_lastpage=478; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Imprecise DNMT1 activity coupled with neighbor-guided correction enables robust yet flexible epigenetic inheritance; citation_author=Q. Wang; citation_author=G. Yu; citation_author=X. Ming; citation_author=W. Xia; citation_author=X. Xu; citation_author=Y. Zhang; citation_author=W. Zhang; citation_author=Y. Li; citation_author=C. Huang; citation_author=H. Xie; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_doi=10.1038/s41588-020-0661-y; citation_firstpage=828; citation_lastpage=839; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=TET1 and hydroxymethylcytosine in transcription and DNA methylation fidelity; citation_author=K. Williams; citation_author=J. Christensen; citation_author=M.T. Pedersen; citation_author=J.V. Johansen; citation_author=P.A.C. Cloos; citation_author=J. Rappsilber; citation_author=K. Helin; citation_journal_title=Nature; citation_volume=473; citation_doi=10.1038/nature10066; citation_firstpage=343; citation_lastpage=348; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Correction of a genetic disease in mouse via use of CRISPR-Cas9; citation_author=Y. Wu; citation_author=D. Liang; citation_author=Y. Wang; citation_author=M. Bai; citation_author=W. Tang; citation_author=S. Bao; citation_author=Z. Yan; citation_author=D. Li; citation_author=J. Li; citation_journal_title=Cell Stem Cell; citation_volume=13; citation_doi=10.1016/j.stem.2013.10.016; citation_firstpage=659; citation_lastpage=662; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Epigenomic analysis of gastrulation identifies a unique chromatin state for primed pluripotency; citation_author=Y. Xiang; citation_author=Y. Zhang; citation_author=Q. Xu; citation_author=C. Zhou; citation_author=B. Liu; citation_author=Z. Du; citation_author=K. Zhang; citation_author=B. Zhang; citation_author=X. Wang; citation_author=S. Gayen; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=52; citation_doi=10.1038/s41588-019-0545-1; citation_firstpage=95; citation_lastpage=105; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Visualizing the node and notochordal plate in gastrulating mouse embryos using scanning electron microscopy and whole mount immunofluorescence; citation_author=C. Xiao; citation_author=F. Nitsche; citation_author=H. Bazzi; citation_journal_title=J.&nbsp;Vis. Exp.; citation_volume=141; citation_doi=10.3791/58321; citation_firstpage=e58321; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Tet1 controls meiosis by regulating meiotic gene expression; citation_author=S. Yamaguchi; citation_author=K. Hong; citation_author=R. Liu; citation_author=L. Shen; citation_author=A. Inoue; citation_author=D. Diep; citation_author=K. Zhang; citation_author=Y. Zhang; citation_journal_title=Nature; citation_volume=492; citation_doi=10.1038/nature11709; citation_firstpage=443; citation_lastpage=447; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Role of Tet1 in erasure of genomic imprinting; citation_author=S. Yamaguchi; citation_author=L. Shen; citation_author=Y. Liu; citation_author=D. Sendler; citation_author=Y. Zhang; citation_journal_title=Nature; citation_volume=504; citation_doi=10.1038/nature12805; citation_firstpage=460; citation_lastpage=464; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=DNMT3A and TET2 compete and cooperate to repress lineage-specific transcription factors in hematopoietic stem cells; citation_author=X. Zhang; citation_author=J. Su; citation_author=M. Jeong; citation_author=M. Ko; citation_author=Y. Huang; citation_author=H.J. Park; citation_author=A. Guzman; citation_author=Y. Lei; citation_author=Y.-H. Huang; citation_author=A. Rao; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=48; citation_doi=10.1038/ng.3610; citation_firstpage=1014; citation_lastpage=1023; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Combined loss of Tet1 and Tet2 promotes B cell, but not myeloid malignancies, in mice; citation_author=Z. Zhao; citation_author=L. Chen; citation_author=M.M. Dawlaty; citation_author=F. Pan; citation_author=O. Weeks; citation_author=Y. Zhou; citation_author=Z. Cao; citation_author=H. Shi; citation_author=J. Wang; citation_author=L. Lin; citation_author=et al.; citation_journal_title=Cell Rep; citation_volume=13; citation_doi=10.1016/j.celrep.2015.10.037; citation_firstpage=1692; citation_lastpage=1704; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Deterministic somatic cell reprogramming involves continuous transcriptional changes governed by Myc and epigenetic-driven modules; citation_author=A. Zviran; citation_author=N. Mor; citation_author=Y. Rais; citation_author=H. Gingold; citation_author=S. Peles; citation_author=E. Chomsky; citation_author=S. Viukov; citation_author=J.D. Buenrostro; citation_author=R. Scognamiglio; citation_author=L. Weinberger; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=24; citation_doi=10.1016/j.stem.2018.11.014; citation_firstpage=328; citation_lastpage=341.e9; citation_publication_date=2019; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Mice deficient for all ten-eleven translocation (TET) genes exhibit early gastrulation lethality. However, separating cause and effect in such embryonic failure is challenging. To isolate cell-autonomous effects of TET loss, we used temporal single-cell atlases from embryos with partial or complete mutant contributions. Strikingly, when developing within a wild-type embryo, <i>Tet</i>-mutant cells retain near-complete differentiation potential, whereas embryos solely comprising mutant cells are defective in epiblast to ectoderm transition with degenerated mesoderm potential. We map de-repressions of early epiblast factors (e.g., <i>Dppa4</i> and <i>Gdf3</i>) and failure to activate multiple signaling from nascent mesoderm (Lefty, FGF, and Notch) as likely cell-intrinsic drivers of TET loss phenotypes. We further suggest loss of enhancer demethylation as the underlying mechanism. Collectively, our work demonstrates an unbiased approach for defining intrinsic and extrinsic embryonic gene function based on temporal differentiation atlases and disentangles the intracellular effects of the demethylation machinery from its broader tissue-level ramifications.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7604&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00842-X&amp;be=2960&amp;fe=6651&amp;dc=6106&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749594225463,%22n%22:0,%22f%22:2330,%22dn%22:2331,%22dne%22:2331,%22c%22:2331,%22s%22:2331,%22ce%22:2331,%22rq%22:2332,%22rp%22:2944,%22rpe%22:3748,%22dl%22:2950,%22di%22:4114,%22ds%22:6106,%22de%22:6298,%22dc%22:6651,%22l%22:6651,%22le%22:6753%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c67 {
  padding: 0.453em 0.5em 0.206em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c35 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D716 {
  padding: 0.431em 0.406em 0.011em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D448 {
  padding: 0.683em 0.76em 0.022em 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c38 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c37 {
  padding: 0.676em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c6D {
  padding: 0.442em 0.833em 0 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c36 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c7C {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c1D43E {
  padding: 0.683em 0.889em 0 0;
}

mjx-c.mjx-c1D442 {
  padding: 0.705em 0.763em 0.022em 0;
}

mjx-c.mjx-c1D43B {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c3F5 {
  padding: 0.431em 0.421em 0.011em 0;
}

mjx-c.mjx-c22C5 {
  padding: 0.303em 0.278em 0 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00842-X"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00842-X"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-86749" aria-expanded="false" aria-controls="login-links-86749" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-86749" id="login-links--86749" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-86749" data-opens="#links-dialog-86749" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-86749" aria-modal="true"><button data-closes="#links-dialog-86749" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900842-X" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.06.049">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00842-X" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0018-9"><span>Volume 185</span>, <span>Issue 17</span></a></span><span class="meta-panel__pages"><span>p3169-3185.e20</span></span><span class="meta-panel__onlineDate">August 18, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--open"><span>Open access</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=17&amp;pii=S0092-8674%2821%29X0018-9" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">The intrinsic and extrinsic effects of TET proteins during gastrulation</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Saifeng</span> <span property="familyName">Cheng</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Saifeng</span> <span property="familyName">Cheng</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2712-3521" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2712-3521</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Cheng/Saifeng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">5</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Markus</span> <span property="familyName">Mittnenzweig</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Markus</span> <span property="familyName">Mittnenzweig</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8502-1702" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8502-1702</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Mittnenzweig/Markus" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">5</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Yoav</span> <span property="familyName">Mayshar</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yoav</span> <span property="familyName">Mayshar</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Mayshar/Yoav" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Aviezer</span> <span property="familyName">Lifshitz</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aviezer</span> <span property="familyName">Lifshitz</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Lifshitz/Aviezer" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Marko</span> <span property="familyName">Dunjić</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marko</span> <span property="familyName">Dunjić</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Dunji%C4%87/Marko" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Yoach</span> <span property="familyName">Rais</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yoach</span> <span property="familyName">Rais</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Rais/Yoach" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Raz</span> <span property="familyName">Ben-Yair</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Raz</span> <span property="familyName">Ben-Yair</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Ben-Yair/Raz" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Stephanie</span> <span property="familyName">Gehrs</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephanie</span> <span property="familyName">Gehrs</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Vascular Oncology and Metastasis, German Cancer Research Center (DKFZ), Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Gehrs/Stephanie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Elad</span> <span property="familyName">Chomsky</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elad</span> <span property="familyName">Chomsky</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Chomsky/Elad" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Zohar</span> <span property="familyName">Mukamel</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Zohar</span> <span property="familyName">Mukamel</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1623-9564" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1623-9564</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Mukamel/Zohar" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Hernan</span> <span property="familyName">Rubinstein</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Hernan</span> <span property="familyName">Rubinstein</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Rubinstein/Hernan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Katharina</span> <span property="familyName">Schlereth</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Katharina</span> <span property="familyName">Schlereth</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2109-7222" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2109-7222</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Vascular Oncology and Metastasis, German Cancer Research Center (DKFZ), Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span property="name">European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Schlereth/Katharina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Netta</span> <span property="familyName">Reines</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Netta</span> <span property="familyName">Reines</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Reines/Netta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Ayelet-Hashahar</span> <span property="familyName">Orenbuch</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ayelet-Hashahar</span> <span property="familyName">Orenbuch</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Orenbuch/Ayelet-Hashahar" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Amos</span> <span property="familyName">Tanay</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Amos</span> <span property="familyName">Tanay</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:amos.tanay@weizmann.ac.il" property="email" title="Send email to Amos Tanay">amos.tanay@weizmann.ac.il</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Tanay/Amos" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup> <a id="eadd0010" href="mailto:amos.tanay@weizmann.ac.il" property="email" title="Send email to Amos Tanay">amos.tanay@weizmann.ac.il</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Yonatan</span> <span property="familyName">Stelzer</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yonatan</span> <span property="familyName">Stelzer</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9207-1479" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9207-1479</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:yonatan.stelzer@weizmann.ac.il" property="email" title="Send email to Yonatan Stelzer">yonatan.stelzer@weizmann.ac.il</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div></div><div class="core-author-link"><a href="/authored-by/Stelzer/Yonatan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">6</a></sup> <a id="eadd0015" href="mailto:yonatan.stelzer@weizmann.ac.il" property="email" title="Send email to Yonatan Stelzer">yonatan.stelzer@weizmann.ac.il</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Computer Science and Applied Mathematics and Department of Molecular Cell Biology, Weizmann Institute of Science, 7610001 Rehovot, Israel</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Division of Vascular Oncology and Metastasis, German Cancer Research Center (DKFZ), Heidelberg, Germany</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">European Center for Angioscience (ECAS), Medical Faculty Mannheim, Heidelberg University, Heidelberg, Germany</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">5</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">6</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received October 14, 2021</span>; <span>Revised April 18, 2022</span>; <span>Accepted June 25, 2022</span>; <span>Published online July 30, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.06.049" property="sameAs" target="_blank">10.1016/j.cell.2022.06.049 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S009286742200842X" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 The Author(s). Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">Creative Commons Attribution – NonCommercial (CC BY-NC)</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.06.049" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0018-9"><img src="/cms/asset/342a07ae-009d-4965-9822-9cd0774cf64a/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 17"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900842-X" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (3)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900842-X" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.06.049">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900842-X" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.06.049&amp;pii=S0092-8674%2822%2900842-X" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900842-X" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.06.049">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.06.049" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S009286742200842X&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S009286742200842X&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=17&amp;pii=S0092-8674%2821%29X0018-9" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00797-8" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00789-9" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (3)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Chimeras with full or partial <i>Tet</i> deficiency are mapped over the course of gastrulation</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph"><i>Tet</i>-TKO cells disrupt signaling, leading to skewed whole-embryo mutant gastrulation</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph"><i>Tet</i>-TKO cells retain near-complete differentiation potential in a chimera context</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Loss of TET leads to pervasive hypermethylation and mildly perturbed gene expression</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Mice deficient for all ten-eleven translocation (TET) genes exhibit early gastrulation lethality. However, separating cause and effect in such embryonic failure is challenging. To isolate cell-autonomous effects of TET loss, we used temporal single-cell atlases from embryos with partial or complete mutant contributions. Strikingly, when developing within a wild-type embryo, <i>Tet</i>-mutant cells retain near-complete differentiation potential, whereas embryos solely comprising mutant cells are defective in epiblast to ectoderm transition with degenerated mesoderm potential. We map de-repressions of early epiblast factors (e.g., <i>Dppa4</i> and <i>Gdf3</i>) and failure to activate multiple signaling from nascent mesoderm (Lefty, FGF, and Notch) as likely cell-intrinsic drivers of TET loss phenotypes. We further suggest loss of enhancer demethylation as the underlying mechanism. Collectively, our work demonstrates an unbiased approach for defining intrinsic and extrinsic embryonic gene function based on temporal differentiation atlases and disentangles the intracellular effects of the demethylation machinery from its broader tissue-level ramifications.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/123d97e9-a5de-4c17-93d8-de59ac587882/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.06.049/asset/9af7c20c-69bd-433f-905b-97e6fa22656f/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/123d97e9-a5de-4c17-93d8-de59ac587882/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;single-cell+genomics&quot;" alt="Search on this keyword" rel="nofollow">single-cell genomics</a></li><li><a href="/action/doSearch?AllField=&quot;developmental+biology&quot;" alt="Search on this keyword" rel="nofollow">developmental biology</a></li><li><a href="/action/doSearch?AllField=&quot;mouse+gastrulation&quot;" alt="Search on this keyword" rel="nofollow">mouse gastrulation</a></li><li><a href="/action/doSearch?AllField=&quot;stem+cells&quot;" alt="Search on this keyword" rel="nofollow">stem cells</a></li><li><a href="/action/doSearch?AllField=&quot;cell+fate+decisions&quot;" alt="Search on this keyword" rel="nofollow">cell fate decisions</a></li><li><a href="/action/doSearch?AllField=&quot;DNA+demethylation&quot;" alt="Search on this keyword" rel="nofollow">DNA demethylation</a></li><li><a href="/action/doSearch?AllField=&quot;epigenetics&quot;" alt="Search on this keyword" rel="nofollow">epigenetics</a></li><li><a href="/action/doSearch?AllField=&quot;genome+editing&quot;" alt="Search on this keyword" rel="nofollow">genome editing</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0040" role="paragraph">Gastrulation is a pivotal step for the formation of the mammalian body plan (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Tam and Behringer, 1997</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Tam, P.P.L. ∙ Behringer, R.R.</div><div><strong>Mouse gastrulation: the formation of a mammalian body plan</strong></div><div><em>Mech. Dev.</em> 1997; <strong>68</strong>:3-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(97)00123-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0031464087" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9431800/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2897%2900123-8&amp;pmid=9431800" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and as such, it epitomizes the emergence of organismal structure from highly interactive ensembles of individual cells (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Moris et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Moris, N. ∙ Pina, C. ∙ Arias, A.M.</div><div><strong>Transition states and cell fate decisions in epigenetic landscapes</strong></div><div><em>Nat. Rev. Genet.</em> 2016; <strong>17</strong>:693-703</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2016.98" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84991585454" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27616569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg.2016.98&amp;pmid=27616569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At the cellular level, gastrulation involves the rapid expansion of the embryos’ cell state repertoire by the conversion of pluripotent epiblast cells through transcriptional, epigenetic, and functional diversifications. However, from a perspective of the entire embryo, cellular trajectories are shaped by continuously reacting to intercellular signals that, in turn, induce new differentiation programs and trigger secretion of additional signals in a dynamic fashion (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Arnold and Robertson, 2009</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Arnold, S.J. ∙ Robertson, E.J.</div><div><strong>Making a commitment: cell lineage allocation and axis patterning in the early mouse embryo</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2009; <strong>10</strong>:91-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm2618" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-58849097959" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19129791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm2618&amp;pmid=19129791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Tam and Loebel, 2007</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Tam, P.P.L. ∙ Loebel, D.A.F.</div><div><strong>Gene function in mouse embryogenesis: get set for gastrulation</strong></div><div><em>Nat. Rev. Genet.</em> 2007; <strong>8</strong>:368-381</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg2084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-34247226529" target="_blank" aria-label="Open Scopus in new tab">Scopus (463)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17387317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg2084&amp;pmid=17387317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Rapid developments in single-cell technologies are now transforming our ability to elucidate gastrulation at single-cell resolution (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Argelaguet et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Argelaguet, R. ∙ Clark, S.J. ∙ Mohammed, H. ...</div><div><strong>Multi-omics profiling of mouse gastrulation at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:487-491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1825-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85076568486" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1825-8&amp;pmid=31827285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">van den Brink et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>van den Brink, S.C. ∙ Alemany, A. ∙ van Batenburg, V. ...</div><div><strong>Single-cell and spatial transcriptomics reveal somitogenesis in gastruloids</strong></div><div><em>Nature.</em> 2020; <strong>582</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2024-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85079785374" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32076263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2024-3&amp;pmid=32076263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Chan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Chan, M.M. ∙ Smith, Z.D. ∙ Grosswendt, S. ...</div><div><strong>Molecular recording of mammalian embryogenesis</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:77-82</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1184-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85065790772" target="_blank" aria-label="Open Scopus in new tab">Scopus (204)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31086336/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1184-5&amp;pmid=31086336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Grosswendt et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Grosswendt, S. ∙ Kretzmer, H. ∙ Smith, Z.D. ...</div><div><strong>Epigenetic regulator function through mouse gastrulation</strong></div><div><em>Nature.</em> 2020; <strong>584</strong>:102-108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2552-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85088787000" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32728215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2552-x&amp;pmid=32728215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Han et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Han, X. ∙ Wang, R. ∙ Zhou, Y. ...</div><div><strong>Mapping the mouse cell atlas by microwell-seq</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:1091-1107.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2018.02.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2018.02.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85042366842" target="_blank" aria-label="Open Scopus in new tab">Scopus (901)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29474909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.02.001&amp;pmid=29474909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">La Manno et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>La Manno, G. ∙ Soldatov, R. ∙ Zeisel, A. ...</div><div><strong>RNA velocity of single cells</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:494-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0414-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85052109231" target="_blank" aria-label="Open Scopus in new tab">Scopus (1950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0414-6&amp;pmid=30089906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Mohammed et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Mohammed, H. ∙ Hernando-Herraez, I. ∙ Savino, A. ...</div><div><strong>Single-cell landscape of transcriptional heterogeneity and cell fate decisions during mouse early gastrulation</strong></div><div><em>Cell Rep.</em> 2017; <strong>20</strong>:1215-1228</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2017.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2017.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85026887626" target="_blank" aria-label="Open Scopus in new tab">Scopus (198)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28768204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.07.009&amp;pmid=28768204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Nowotschin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Nowotschin, S. ∙ Setty, M. ∙ Kuo, Y.-Y. ...</div><div><strong>The emergent landscape of the mouse gut endoderm at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:361-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1127-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85065227179" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30959515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1127-1&amp;pmid=30959515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Scialdone et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Scialdone, A. ∙ Tanaka, Y. ∙ Jawaid, W. ...</div><div><strong>Resolving early mesoderm diversification through single-cell expression profiling</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18633" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84978761773" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27383781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18633&amp;pmid=27383781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and within a context of a detailed temporal model (<span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Peng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Peng, G. ∙ Suo, S. ∙ Cui, G. ...</div><div><strong>Molecular architecture of lineage allocation and tissue organization in early mouse embryo</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1469-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85070308555" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31391582/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1469-8&amp;pmid=31391582" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Pijuan-Sala et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Pijuan-Sala, B. ∙ Griffiths, J.A. ∙ Guibentif, C. ...</div><div><strong>A single-cell molecular map of mouse gastrulation and early organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:490-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0933-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85062300354" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0933-9&amp;pmid=30787436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Srivatsan et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Srivatsan, S.R. ∙ Regier, M.C. ∙ Barkan, E. ...</div><div><strong>Embryo-scale, single-cell spatial transcriptomics</strong></div><div><em>Science.</em> 2021; <strong>373</strong>:111-117</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb9536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85109116319" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34210887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb9536&amp;pmid=34210887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Nevertheless, using such models to understand the mechanisms regulating differentiation and cell fate acquisition is extremely challenging, to a large extent, due to the constant interplay between direct intracellular effects and indirect intercellular signals. When perturbing a gene or a system and monitoring the impact on gastrulation, it is becoming essential to deconvolute the potential intracellular effects on different temporal stages and to decouple it from effects arising through perturbation of proper signaling from other lineages. Understanding direct and indirect gene function becomes particularly challenging when considering broad and pleiotropic regulatory mechanisms that function in multiple lineages. A key family of such mechanisms, whose function is indeed poorly understood in the embryo, entails the pathways controlling the build-up and maturation of lineage-specific DNA methylation landscapes.</div><div id="p0045" role="paragraph">The ten-eleven translocation (TET) family dioxygenases comprise three genes (<i>Tet1</i>-<i>3</i>) that can catalyze the oxidation of 5-methylcytosine (5mC) to 5-hydroxymethylcytosine (5hmC), which may lead to demethylation (<span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">He et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>He, Y.-F. ∙ Li, B.-Z. ∙ Li, Z. ...</div><div><strong>Tet-mediated formation of 5-Carboxylcytosine and its excision by TDG in mammalian DNA</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:1303-1307</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1210944" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052495940" target="_blank" aria-label="Open Scopus in new tab">Scopus (2220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21817016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1210944&amp;pmid=21817016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Ito et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Ito, S. ∙ D’Alessio, A.C. ∙ Taranova, O.V. ...</div><div><strong>Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification</strong></div><div><em>Nature.</em> 2010; <strong>466</strong>:1129-1133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-77956189495" target="_blank" aria-label="Open Scopus in new tab">Scopus (2083)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20639862/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09303&amp;pmid=20639862" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" aria-label="Reference 2011" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">2011</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Ito, S. ∙ Shen, L. ∙ Dai, Q. ...</div><div><strong>Tet proteins can convert 5-methylcytosine to 5-Formylcytosine and 5-Carboxylcytosine</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:1300-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1210597" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052461558" target="_blank" aria-label="Open Scopus in new tab">Scopus (2734)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21778364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1210597&amp;pmid=21778364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Tahiliani et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Tahiliani, M. ∙ Koh, K.P. ∙ Shen, Y. ...</div><div><strong>Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1</strong></div><div><em>Science.</em> 2009; <strong>324</strong>:930-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1170116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-66149146320" target="_blank" aria-label="Open Scopus in new tab">Scopus (4619)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19372391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1170116&amp;pmid=19372391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Single and double disruptions of <i>Tet</i> genes were shown to exert effects during mouse development, with notable examples documented in preimplantation embryos (<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Gu et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Gu, T.-P. ∙ Guo, F. ∙ Yang, H. ...</div><div><strong>The role of Tet3 DNA dioxygenase in epigenetic reprogramming by oocytes</strong></div><div><em>Nature.</em> 2011; <strong>477</strong>:606-610</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10443" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80053348585" target="_blank" aria-label="Open Scopus in new tab">Scopus (935)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21892189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10443&amp;pmid=21892189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Ito et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Ito, S. ∙ D’Alessio, A.C. ∙ Taranova, O.V. ...</div><div><strong>Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification</strong></div><div><em>Nature.</em> 2010; <strong>466</strong>:1129-1133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-77956189495" target="_blank" aria-label="Open Scopus in new tab">Scopus (2083)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20639862/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09303&amp;pmid=20639862" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Kang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Kang, J. ∙ Lienhard, M. ∙ Pastor, W.A. ...</div><div><strong>Simultaneous deletion of the methylcytosine oxidases Tet1 and Tet3 increases transcriptome variability in early embryogenesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4236-E4245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1510510112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84938719137" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26199412/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1510510112&amp;pmid=26199412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), differentiation of pluripotent (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Dawlaty et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Dawlaty, M.M. ∙ Ganz, K. ∙ Powell, B.E. ...</div><div><strong>Tet1 is dispensable for maintaining pluripotency and its loss is compatible with embryonic and postnatal development</strong></div><div><em>Cell Stem Cell.</em> 2011; <strong>9</strong>:166-175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.07.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.07.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79961139741" target="_blank" aria-label="Open Scopus in new tab">Scopus (428)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21816367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2011.07.010&amp;pmid=21816367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" aria-label="Reference 2013" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">2013</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Combined deficiency of Tet1 and Tet2 causes epigenetic abnormalities but is compatible with postnatal development</strong></div><div><em>Dev. Cell.</em> 2013; <strong>24</strong>:310-323</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2012.12.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2012.12.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84873707539" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23352810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2012.12.015&amp;pmid=23352810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Khoueiry et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Khoueiry, R. ∙ Sohni, A. ∙ Thienpont, B. ...</div><div><strong>Lineage-specific functions of TET1 in the postimplantation mouse embryo</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1061-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3868" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85021769555" target="_blank" aria-label="Open Scopus in new tab">Scopus (81)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28504700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3868&amp;pmid=28504700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Koh et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Koh, K.P. ∙ Yabuuchi, A. ∙ Rao, S. ...</div><div><strong>Tet1 and Tet2 regulate 5-hydroxymethylcytosine production and cell lineage specification in mouse embryonic stem cells</strong></div><div><em>Cell Stem Cell.</em> 2011; <strong>8</strong>:200-213</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.01.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.01.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79551587102" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21295276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2011.01.008&amp;pmid=21295276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Li et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Li, X. ∙ Yue, X. ∙ Pastor, W.A. ...</div><div><strong>Tet proteins influence the balance between neuroectodermal and mesodermal fate choice by inhibiting Wnt signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E8267-E8276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1617802113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85006421153" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27930333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1617802113&amp;pmid=27930333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and multipotent stem cells (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Izzo et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Izzo, F. ∙ Lee, S.C. ∙ Poran, A. ...</div><div><strong>DNA methylation disruption reshapes the hematopoietic differentiation landscape</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:378-387</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0595-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85083041344" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32203468/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0595-4&amp;pmid=32203468" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Ko et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Ko, M. ∙ Bandukwala, H.S. ∙ An, J. ...</div><div><strong>Ten-Eleven-Translocation 2 (TET2) negatively regulates homeostasis and differentiation of hematopoietic stem cells in mice</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:14566-14571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1112317108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052284526" target="_blank" aria-label="Open Scopus in new tab">Scopus (476)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21873190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1112317108&amp;pmid=21873190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Li et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Li, Z. ∙ Cai, X. ∙ Cai, C.-L. ...</div><div><strong>Deletion of Tet2 in mice leads to dysregulated hematopoietic stem cells and subsequent development of myeloid malignancies</strong></div><div><em>Blood.</em> 2011; <strong>118</strong>:4509-4518</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2010-12-325241" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052285127" target="_blank" aria-label="Open Scopus in new tab">Scopus (543)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21803851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2010-12-325241&amp;pmid=21803851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Moran-Crusio et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Moran-Crusio, K. ∙ Reavie, L. ∙ Shih, A. ...</div><div><strong>Tet2 loss leads to increased hematopoietic stem cell self-renewal and myeloid transformation</strong></div><div><em>Cancer Cell.</em> 2011; <strong>20</strong>:11-24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.ccr.2011.06.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.ccr.2011.06.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79960064353" target="_blank" aria-label="Open Scopus in new tab">Scopus (1058)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21723200/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2011.06.001&amp;pmid=21723200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Zhang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Zhang, X. ∙ Su, J. ∙ Jeong, M. ...</div><div><strong>DNMT3A and TET2 compete and cooperate to repress lineage-specific transcription factors in hematopoietic stem cells</strong></div><div><em>Nat. Genet.</em> 2016; <strong>48</strong>:1014-1023</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3610" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84978680214" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27428748/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3610&amp;pmid=27428748" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Zhao et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Zhao, Z. ∙ Chen, L. ∙ Dawlaty, M.M. ...</div><div><strong>Combined loss of Tet1 and Tet2 promotes B cell, but not myeloid malignancies, in mice</strong></div><div><em>Cell Rep.</em> 2015; <strong>13</strong>:1692-1704</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2015.10.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2015.10.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84952874773" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26586431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2015.10.037&amp;pmid=26586431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and germ cell specification and function (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Gu et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Gu, T.-P. ∙ Guo, F. ∙ Yang, H. ...</div><div><strong>The role of Tet3 DNA dioxygenase in epigenetic reprogramming by oocytes</strong></div><div><em>Nature.</em> 2011; <strong>477</strong>:606-610</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10443" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80053348585" target="_blank" aria-label="Open Scopus in new tab">Scopus (935)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21892189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10443&amp;pmid=21892189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Hackett et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Hackett, J.A. ∙ SenGupta, R. ∙ Zylicz, J.J. ...</div><div><strong>Germline DNA demethylation dynamics and imprint erasure Through 5-hydroxymethylcytosine</strong></div><div><em>Science.</em> 2013; <strong>339</strong>:448-452</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1229277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84872770694" target="_blank" aria-label="Open Scopus in new tab">Scopus (636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23223451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1229277&amp;pmid=23223451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Vincent et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Vincent, J.J. ∙ Huang, Y. ∙ Chen, P.-Y. ...</div><div><strong>Stage-specific roles for Tet1 and Tet2 in DNA demethylation in primordial germ cells</strong></div><div><em>Cell Stem Cell.</em> 2013; <strong>12</strong>:470-478</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.01.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.01.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84875949201" target="_blank" aria-label="Open Scopus in new tab">Scopus (147)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.01.016&amp;pmid=23415914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Yamaguchi et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Yamaguchi, S. ∙ Hong, K. ∙ Liu, R. ...</div><div><strong>Tet1 controls meiosis by regulating meiotic gene expression</strong></div><div><em>Nature.</em> 2012; <strong>492</strong>:443-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11709" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84871438065" target="_blank" aria-label="Open Scopus in new tab">Scopus (243)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23151479/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11709&amp;pmid=23151479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" aria-label="Reference 2013" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">2013</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Yamaguchi, S. ∙ Shen, L. ∙ Liu, Y. ...</div><div><strong>Role of Tet1 in erasure of genomic imprinting</strong></div><div><em>Nature.</em> 2013; <strong>504</strong>:460-464</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84890568163" target="_blank" aria-label="Open Scopus in new tab">Scopus (202)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24291790/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12805&amp;pmid=24291790" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unlike the more nuanced phenotypes associated with partial disruption of this pathway, <i>Tet</i> triple knockout (<i>Tet</i>-TKO) resulted in early embryonic lethality with post-implantation embryos exhibiting marked perturbations in Lefty-Nodal and Wnt signaling pathways (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Dai, H.-Q. ∙ Wang, B.-A. ∙ Yang, L. ...</div><div><strong>TET-mediated DNA demethylation controls gastrulation by regulating Lefty–Nodal signalling</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84994051865" target="_blank" aria-label="Open Scopus in new tab">Scopus (156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20095&amp;pmid=27760115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Li et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Li, X. ∙ Yue, X. ∙ Pastor, W.A. ...</div><div><strong>Tet proteins influence the balance between neuroectodermal and mesodermal fate choice by inhibiting Wnt signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E8267-E8276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1617802113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85006421153" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27930333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1617802113&amp;pmid=27930333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At the intracellular level, multi-omic analysis of single <i>Tet</i>-TKO cells suggested that lineage-specific enhancers with TET-dependent reduction in methylation are linked to the regulation of mesoderm differentiation <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Argelaguet et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Argelaguet, R. ∙ Clark, S.J. ∙ Mohammed, H. ...</div><div><strong>Multi-omics profiling of mouse gastrulation at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:487-491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1825-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85076568486" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1825-8&amp;pmid=31827285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Together, these studies implicated <i>Tet</i> genes in the regulation of multiple lineages and developmental stages during gastrulation. The catastrophic failure of gastrulation in these mutants highlights the need for an experimental framework that allows examining the primary function of <i>Tet</i> genes at the cellular level while circumventing secondary effects that <i>Tet</i> perturbation may exert by modifying the embryonic niche.</div><div id="p0050" role="paragraph">Here, we utilized a chimeric embryo platform in which <i>Tet</i>-deficient and control mouse embryonic stem cells (mESCs) were either injected into tetraploid (4N) or diploid (2N) blastocysts and allowed to develop <i>in utero</i>. In 4N complemented embryos, the resulting embryonic compartment solely comprises the injected-mESCs derivatives (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Nagy et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Nagy, A. ∙ Gócza, E. ∙ Diaz, E.M. ...</div><div><strong>Embryonic stem cells alone are able to support fetal development in the mouse</strong></div><div><em>Development.</em> 1990; <strong>110</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.110.3.815" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2088722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.110.3.815&amp;pmid=2088722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" aria-label="Reference 1993" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">1993</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Nagy, A. ∙ Rossant, J. ∙ Nagy, R. ...</div><div><strong>Derivation of completely cell culture-derived mice from early-passage embryonic stem cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1993; <strong>90</strong>:8424-8428</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.90.18.8424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0027321548" target="_blank" aria-label="Open Scopus in new tab">Scopus (2011)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8378314/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.90.18.8424&amp;pmid=8378314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (hereinafter denoted as whole-embryo chimera), whereas in chimeras obtained using 2N host blastocysts, the embryonic compartment contains both wild-type (WT) and injected-cell derivatives (hereinafter denoted as mixed chimera, see <a id="crosref0350" href="#fig1">Figure&nbsp;1</a>A). We then performed a combined analysis of timed chimeric embryos in the context of a precise single-embryo/single-cell temporal gastrulation atlas. Although whole-embryo mutants capture the combined cell-intrinsic and -extrinsic consequences following <i>Tet</i> loss, mixed chimera embryos allow isolating cell-intrinsic effects simultaneously in multiple lineages. This allows for natural separation of the impact of TET on intracellular gene expression programs, from the broader embryo-wide phenotypes that emerge once the delicate balance between interacting cellular lineages in the embryo is disrupted (<a id="crosref0355" href="#fig1">Figure&nbsp;1</a>A).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/a608eaf2-6444-45e6-bb5e-600439f2082f/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.049/asset/27232823-6e22-4543-86a3-a72f1bb9f3ff/main.assets/gr1.jpg" height="940" width="691" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/a608eaf2-6444-45e6-bb5e-600439f2082f/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Whole-embryo <i>Tet</i> mutants display morphological and molecular gastrulation defects</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Graphic view of the experimental design. Fluorescently tagged mESCs were injected into 4N or 2N blastocysts to generate whole-embryo or mixed chimeric embryos, which were subsequently index-sorted for scRNA-seq. For each embryo, the transcriptome was compared to a reference WT gastrulation atlas to see if injected cells introduce cell state, composition, and differentiation rate (temporal) changes.</div><div id="fspara0020" role="paragraph">(B) Representative images of E7.5–E9.5 <i>Tet</i>-TKO whole-embryo mutants generated by injection of <i>Tet</i>-TKO mESCs into 4N blastocysts. Dashed lines depict embryo structure. Arrowheads show aberrant accumulation of cells inside amniotic cavity. A, anterior; P, posterior; Al, allantois; Em, embryonic tissues. Scale bars, 100&nbsp;μm.</div><div id="fspara0025" role="paragraph">(C) 2D-projection of transcriptome profiles of 7,480 control and 9,793 <i>Tet</i>-TKO cells from E7.5 to E8.5 whole-embryo chimeras onto the WT atlas. Major lineages of the WT atlas are highlighted on enlarged subpanels.</div><div id="fspara0030" role="paragraph">(D) Cell-type composition per embryo. Embryos (represented by columns) are arranged according to their inferred E<sub>t</sub> on the x axis.</div><div id="fspara0035" role="paragraph">(E) Fraction of major lineages per embryo. Black and red dots represent control and <i>Tet</i>-TKO whole-embryo chimeras, respectively. Black line represents the moving average frequency of WT atlas embryos for each lineage, and the shaded gray area represents two moving standard deviations around the mean (window size&nbsp;= 9). Two-sided Wilcoxon-Mann-Whitney rank sum test was used to compare cell-type frequencies of the 11 control and 6 <i>Tet</i>-TKO embryos older than E<sub>t</sub>&nbsp;= 7.5.</div><div id="fspara0040" role="paragraph">(F) Distribution of E<sub>t</sub> between 16 control and 15 <i>Tet</i>-TKO whole-embryo chimeras sampled at E7.5. Wilcoxon-Mann-Whitney test, two-tailed. Data are represented as mean ± SD.</div><div id="fspara0045" role="paragraph">(G) Variance and mean single-cell time distributions of <i>Tet</i>-TKO and control whole-embryo chimeras. Black line represents the moving average variance of WT atlas embryos, and the shaded gray area represents two moving standard deviations around the mean (window size&nbsp;= 17).</div><div id="fspara0050" role="paragraph">See also <a id="crosref1290" href="#figs1">Figures&nbsp;S1</a> and <a id="crosref1295" href="#figs2">S2</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Early gastrulation defects in 4N blastocysts injected with <i>Tet</i>-TKO cells</h3><div id="p0055" role="paragraph">To map the impact of complete loss of the TET machinery on gastrulation, we generated fluorescently tagged <i>Tet</i>-TKO mESCs lines alongside corresponding controls. All <i>Tet</i>-TKO lines were validated for loss of function of all three TET proteins and a global decrease in 5hmC levels (<a id="crosref0360" href="#figs1">Figures&nbsp;S1</a>A–S1F). mCherry-tagged <i>Tet</i>-TKO (TKO1 and TKO2) and GFP-tagged control mESCs (Ctrl1, Ctrl2, and Ctrl3) were separately injected into 4N blastocysts, and embryos were dissected at embryonic day (E) 7.5. Consistent with the previously observed phenotype for <i>Tet</i>-TKO germline KO (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Dai, H.-Q. ∙ Wang, B.-A. ∙ Yang, L. ...</div><div><strong>TET-mediated DNA demethylation controls gastrulation by regulating Lefty–Nodal signalling</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84994051865" target="_blank" aria-label="Open Scopus in new tab">Scopus (156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20095&amp;pmid=27760115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Li et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Li, X. ∙ Yue, X. ∙ Pastor, W.A. ...</div><div><strong>Tet proteins influence the balance between neuroectodermal and mesodermal fate choice by inhibiting Wnt signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E8267-E8276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1617802113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85006421153" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27930333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1617802113&amp;pmid=27930333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), 4N embryos complemented with TKO mESCs (whole-embryo mutants) displayed overt growth retardation and aberrant accumulation of cells inside the amniotic cavity (<a id="crosref0375" href="#fig1">Figures&nbsp;1</a>B, <a id="crosref0380" href="#figs1">S1</a>G, and S1H). <i>Tet</i>-TKO embryos recovered from later time points (E8.0–E9.5) demonstrated a persistent delay in development, together with abnormal morphology characterized by a small and underdeveloped embryonic compartment and excessive overgrowth of extraembryonic mesoderm tissues (<a id="crosref0385" href="#fig1">Figures&nbsp;1</a>B, <a id="crosref0390" href="#figs1">S1</a>I, and S2E).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/e56e1776-856c-4a59-bd39-376a65db6940/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.049/asset/8cd67a8a-9f00-4334-801b-bffa145f80fd/main.assets/figs1.jpg" height="895" width="761" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/e56e1776-856c-4a59-bd39-376a65db6940/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Generation of fluorescently labeled <i>Tet</i>-TKO mESCs clones and embryos, related to <a id="crosref1340" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0275" role="paragraph">(A) Targeting strategy using CRISPR/Cas9-mediated genome editing and 3 gRNAs against <i>Tet</i> genes for generation of <i>Tet</i> KO mESC clone. Exons and conserved domains were shown in black and green/blue boxes, introns marked in lines.</div><div id="fspara0280" role="paragraph">(B) Screening for clones with reduced 5hmC intensity after gene targeting using dot blot. <i>Tet</i>-TKO3 and Ctrl3 mESCs lines were marked in rectangles.</div><div id="fspara0285" role="paragraph">(C) DNA sequencing validation for <i>Tet</i>-TKO3 mESCs clone. Protospacer adjacent motif (PAM) sequences are marked in red. Intron sequences are shown in lower-case letters.</div><div id="fspara0290" role="paragraph">(D) Western blot validation for <i>Tet</i>-TKO3 mESCs clone. Following confirmation for lack of expression of TET2 protein, TKO3 clone was further validated for the lack of TET1 and TET3 protein expression.</div><div id="fspara0295" role="paragraph">(E) Generation of <i>Tet</i>-TKO4 mESCs clone. <i>Tet</i> triple floxed mESCs were derived from blastocysts followed by Cre recombinase treatment (<a id="crosref1345" href="#sec-4">STAR Methods</a>). Validation of the deleted alleles was performed by PCR. <i>f</i>/<i>f</i>, homozygous floxed allele, <i>Δ</i>/<i>Δ</i>, homozygous deleted allele. Sizes of each allele were indicated on the right.</div><div id="fspara0300" role="paragraph">(F) Flow cytometric analysis of reporter activity for day 5 (Ctrl3 and <i>Tet</i>-TKO3) and day 8 (Ctrl1, Ctrl2, <i>Tet</i>-TKO1, and <i>Tet</i>-TKO2) embryoid bodies.</div><div id="fspara0305" role="paragraph">(G) Embryo size (extraembryonic mesodermal part excluded, log<sub>2</sub> of μm<sup>2</sup>) comparisons between E7.5 control and <i>Tet</i>-TKO whole-embryo chimeras. Wilcoxon-Mann-Whitney test, two-tailed. Data are represented as mean ± SD.</div><div id="fspara0310" role="paragraph">(H) Representative images of E7.5 control whole-embryo chimeras. Scale bars, 100&nbsp;μm.</div><div id="fspara0315" role="paragraph">(I) Representative images of E8.5 <i>Tet</i>-TKO3 and <i>Tet</i>-TKO4 whole-embryo chimeras. Scale bars, 100&nbsp;μm.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Massively perturbed cell-type composition in <i>Tet</i>-TKO whole-embryo mutants</h3><div id="p0060" role="paragraph">To characterize the cellular and molecular changes associated with these morphological phenotypes, we performed single-cell RNA sequencing (scRNA-seq) on 30 individual <i>Tet</i>-TKO and 18 control embryos spanning E7.5–E8.5 (<a id="crosref0395" href="#figs2">Figures&nbsp;S2</a>A and S2B). We compared the resulting single-cell profiles to a reference WT temporal gastrulation atlas (<span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), systematically searching for three classes of mutant effects (see <a id="crosref0405" href="#fig1">Figure&nbsp;1</a>A): (1) appearance of new transcriptional states resulting from gross perturbation to existing ones (denoted as class I or <i>state</i> perturbations), (2) redistribution of the ensemble of transcriptional states per embryo, up to the elimination of certain states from embryos (class II or <i>compositional</i> perturbations), and (3) changes in differentiation rate or changes in synchronicity between cell states over time (class III or <i>temporal</i> perturbations). Analysis of single cells from whole-embryo <i>Tet</i>-TKO mutants suggested that class I <i>state</i> perturbations are generally mild, such that no fundamentally abnormal cell state was observed (<a id="crosref0410" href="#figs2">Figure&nbsp;S2</a>C). We could assess compositional and temporal perturbations based on high fidelity mapping of <i>Tet</i>-TKO cell states over the WT atlas. Class II <i>compositional</i> perturbation analysis showed a massive redistribution of cell states in mutant embryos (<a id="crosref0415" href="#fig1">Figures&nbsp;1</a>C–1E). Specifically, <i>Tet</i>-TKO mutants initiated gastrulation with early endoderm and mesoderm differentiation but were largely devoid of ectoderm and mature embryonic mesoderm lineages (<a id="crosref0420" href="#fig1">Figure&nbsp;1</a>E). Mesoderm cell states showed perturbations in anterior-posterior patterning with depletion of rostral mesodermal lineages (<a id="crosref0425" href="#fig1">Figure&nbsp;1</a>D; <a id="crosref0430" href="#mmc1">Table&nbsp;S1</a>). Interestingly, early specified posterior mesoderm cell types, such as hematoendothelial and extraembryonic mesodermal lineages, were over-represented in the mutants (<a id="crosref0435" href="#fig1">Figures&nbsp;1</a>E and <a id="crosref0440" href="#figs2">S2</a>D). This is in contrast to similarly derived and analyzed embryos generated by injection of control mESCs that mapped to all embryonic lineages. <i>In situ</i> hybridization of marker genes for node/notochord (<i>Noto</i>), caudal mesoderm (<i>Cdx1</i>), and rostral mesoderm (<i>Twist1</i>) in E8.5 <i>Tet</i>-TKO embryos further confirmed the compositional aberration of <i>Tet</i>-TKO mutants (<a id="crosref0445" href="#figs2">Figure&nbsp;S2</a>E).</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/47fc4a49-4c0a-4f46-bdc6-9b451ecc15d2/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.049/asset/f4ac48f8-82de-42f0-845a-ec2b8f786218/main.assets/figs2.jpg" height="941" width="654" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/47fc4a49-4c0a-4f46-bdc6-9b451ecc15d2/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">scRNA-seq profiling of <i>Tet</i>-TKO whole-embryo chimeras, related to <a id="crosref1350" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0325" role="paragraph">(A) Distribution of unique reads per cell in control and <i>Tet</i>-TKO whole-embryo chimeras. Cells with number of UMIs&nbsp;&lt;&nbsp;2,000 or&nbsp;&gt;&nbsp;12,000 were excluded in subsequent analysis.</div><div id="fspara0330" role="paragraph">(B) Number of cells for individual control and <i>Tet</i>-TKO whole-embryo chimeras alongside their inferred E<sub>t</sub>.</div><div id="fspara0335" role="paragraph">(C) Gene expression in <i>Tet</i>-TKO and control whole-embryo chimeras compared with WT for a selection of cell types. The comparison was performed between query cells from whole-embryo chimeras and their projected WT cells (<a id="crosref1355" href="#sec-4">STAR Methods</a>). The dashed line indicates a 2-fold change in expression. Genes with over 50% fold change were labeled in black.</div><div id="fspara0340" role="paragraph">(D) Frequency distribution of cell types in <i>Tet</i>-TKO, control whole-embryo chimeras and WT embryos spanning E<sub>t</sub>7.75–E<sub>t</sub>8.1 as calculated in <a id="crosref1360" href="#fig2">Figure&nbsp;2</a>F. Wilcoxon-Mann-Whitney rank sum test, two-tailed. ns, not significant; <sup>∗</sup>, q value&nbsp;&lt;&nbsp;0.05 (Benjamini-Hochberg procedure).</div><div id="fspara0345" role="paragraph">(E) Whole-mount <i>in situ</i> hybridization analysis of <i>Noto</i>, <i>Cdx1</i>, and <i>Twist1</i> expression in E8.5 <i>Tet</i>-TKO whole-embryo mutant and time-matched WT embryos. The number of embryos analyzed for each probe in WT and <i>Tet</i>-TKO embryos, respectively, is <i>Noto</i> (8, 2), <i>Cdx1</i> (4, 3), and <i>Twist1</i> (4, 3). Scale bars, 100&nbsp;μm.</div><div id="fspara0350" role="paragraph">(F) Time distribution of sampled cells from <i>Tet</i>-TKO whole-embryo chimeras compared with those of WT embryos with matched transcriptional time.</div><div id="fspara0355" role="paragraph">(G) Genotyping of <i>Tet</i> triple floxed embryos following AAV-mediated Cre treatment. For each <i>Tet</i> gene, the absence of floxed allele and the presence of excised allele were tested. Note that the WT band is likely due to maternal blood contamination from the DNA sampling (see <a id="crosref1365" href="#sec-4">STAR Methods</a>).&nbsp;+/+, WT allele. Sizes of each allele were indicated on the right.</div><div id="fspara0360" role="paragraph">(H) Phase-contrast images of E7.5 <i>Tet-</i>TKO embryos recovered post Cre recombinase treatment. Scale bars, 100&nbsp;μm.</div><div id="fspara0365" role="paragraph">(I) Cell-type composition as calculated in <a id="crosref1370" href="#fig1">Figure&nbsp;1</a>D for each <i>Tet</i>-TKO embryo recovered post Cre recombinase treatment. E<sub>t</sub> is indicated in parentheses.</div><div id="fspara0370" role="paragraph">(J) Fraction of major lineages per embryo.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">Analysis of the estimated transcriptional time (E<sub>t</sub>) distribution in mutant and control embryos also quantified class III (<i>temporal</i>) perturbations linked with <i>Tet</i> inactivation (<a id="crosref0450" href="#sec-4">STAR Methods</a>). First, the embryonic time estimation of <i>Tet</i>-TKO cells based on the WT atlas showed a marked delay compared with controls (median E<sub>t</sub>7.1, compared with E<sub>t</sub>7.6 in controls) for embryos sampled at the same time (E7.5) (<a id="crosref0455" href="#fig1">Figure&nbsp;1</a>F). Second, <i>Tet</i>-TKO embryos showed high variance in their cell timing composition (<a id="crosref0460" href="#fig1">Figure&nbsp;1</a>G) and included subpopulations transcriptionally matching more differentiated cell states (mainly extraembryonic lineages), together with cells matching much earlier states (i.e., epiblast; <a id="crosref0465" href="#figs2">Figure&nbsp;S2</a>F). Very similar compositional and temporal effects were observed when analyzing embryos in which the TET system was targeted at both the embryonic and extraembryonic compartments (<a id="crosref0470" href="#figs2">Figures&nbsp;S2</a>G–S2J; <a id="crosref0475" href="#sec-4">STAR Methods</a>). Together, both morphological and single-cell analyses show that inactivation of all three TET enzymes in the entire embryo leads to growth retardation and gastrulation defects characterized by anterior-posterior patterning deficiencies. Interestingly, such defects were not traceable to new aberrant cell states or intrinsic cell state perturbations but rather represented disruption in the balance and timing of multiple differentiation processes.</div></section><section id="sec-2-3"><h3><i>Tet</i>-TKO differentiation defects are rescued by a normal embryonic niche</h3><div id="p0070" role="paragraph">To begin separating the embryo-wide effects from cell-autonomous consequences of TET machinery loss during gastrulation, we injected labeled <i>Tet</i>-TKO mESCs together with control mESCs or separately into normal 2N blastocysts and profiled chimeric embryos by scRNA-seq. Both control and mutant cells were successfully detected in embryos spanning E7.0–E9.5, although <i>Tet</i>-TKO cells generally showed significantly lower levels of chimerism (<a id="crosref0480" href="#fig2">Figures&nbsp;2</a>A and 2B). We noted that some chimeric embryos with &gt;15% contribution of <i>Tet</i>-TKO cells showed an accumulation of cells inside the amniotic cavity, in a manner similar to whole-embryo mutants. We also observed a bias of mutant cells toward the posterior part and base of allantois in several embryos (<a id="crosref0485" href="#figs3">Figure&nbsp;S3</a>A). For analysis using scRNA-seq, we considered embryos with a discernible contribution of <i>Tet</i>-TKO cells (roughly over 1%) and used index sorting to distinguish host cells from injected cells from each embryo. Overall, we processed 44 individually tagged E7.5 embryos and derived a total of 7,008 mutant, 4,334 control, and 4,834 host cells. Composition and temporal analyses of individual embryos based on either injected control or host cells showed high concordance between them, such that in subsequent analyses, these cells were considered together (<a id="crosref0490" href="#figs3">Figures&nbsp;S3</a>B–S3D). We then applied the three-tier analysis framework to detect <i>state</i>, <i>composition</i>, and <i>temporal</i> aberrations in mutant and control cell populations. Similar to the whole-embryo KO data, class I <i>state</i> aberrations were generally mild, with an overall good recapitulation of transcriptional states in both mutant and control populations compared with the reference atlas (<a id="crosref0495" href="#figs3">Figure&nbsp;S3</a>E). In contrast to the whole-embryo KO, <i>Tet</i>-TKO cells differentiated alongside WT cells showed extensive contribution to almost all embryonic cell lineages, as evidenced by the presence of mature mesodermal and ectodermal lineages that were largely missing from whole-embryo mutants (<a id="crosref0500" href="#fig2">Figure&nbsp;2</a>C). Moreover, E<sub>t</sub> calculated for each embryo using either <i>Tet</i>-TKO or host/control cells was highly correlated, suggesting a high degree of synchronization between host and mutant cells within each embryo (<a id="crosref0505" href="#fig2">Figure&nbsp;2</a>D). Interestingly, we did not observe a correlation between the estimated degree of chimerism in each embryo and the intensity of the effect on <i>Tet</i>-TKO temporal and compositional distributions (<a id="crosref0510" href="#figs3">Figures&nbsp;S3</a>F and S3G). This suggests that host niche signals can robustly mitigate the developmental catastrophe observed in <i>Tet</i>-TKO whole-embryo mutants (given a host contribution of &gt;40%), prompting us to analyze more fine-grained differentiation fates imbalance.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/f724d68e-5b28-4271-ab18-c7f8042e2064/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.049/asset/cb8b6ecd-9a64-4b5a-96bd-8afe4ff7d0fa/main.assets/gr2.jpg" height="938" width="760" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/f724d68e-5b28-4271-ab18-c7f8042e2064/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Differentiation capacity of <i>Tet</i>-TKO cells in mixed chimera embryos</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0060" role="paragraph">(A) Representative images of E7.0–E9.5 control and <i>Tet</i>-TKO mixed chimera embryos generated by injection of respective mESCs into 2N blastocysts. Dashed lines depict embryo structure. HF, head fold; NF, neural fold; H, head; S, somite. Scale bars, 100&nbsp;μm.</div><div id="fspara0065" role="paragraph">(B) Flow cytometric analysis for degree of chimerism per embryo. Number of embryos for each genotype is indicated in parentheses. Wilcoxon-Mann-Whitney test, two-tailed. Data are represented as mean ± SD.</div><div id="fspara0070" role="paragraph">(C) 2D-projection of transcriptome profiles onto the WT atlas of host-/control-derived cells and <i>Tet</i>-TKO-derived cells obtained from E7.5 mixed chimera embryos. Single cells are colored by projected atlas cell type.</div><div id="fspara0075" role="paragraph">(D) Comparison of E<sub>t</sub> calculated for each mixed chimera embryo using either <i>Tet</i>-TKO or host-/control-derived cells.</div><div id="fspara0080" role="paragraph">(E) Cell-type composition per embryo as in <a id="crosref1300" href="#fig1">Figure&nbsp;1</a>D. Embryos (represented by columns) are placed along the x axis according to their inferred E<sub>t</sub> calculated by their host/control cells. Fraction of cell types contributed by host/control and by <i>Tet</i>-TKO-derived cells for the same embryo are shown.</div><div id="fspara0085" role="paragraph">(F–H) Frequency of indicated cell types contributed by different groups of cells (TKO, Host/Ctrl, WT) in mixed (n&nbsp;= 12), whole-embryo chimeras (n<sub>TKO</sub>&nbsp;= 6, n<sub>Ctrl</sub>&nbsp;= 6) and WT embryos (n&nbsp;= 29) spanning E<sub>t</sub>7.75–E<sub>t</sub>8.1. Medians of frequencies were compared using a Wilcoxon-Mann-Whitney rank sum test after downsampling of each embryo to 100 (mixed chimera) and 250 cells (whole-embryo chimera). q values were calculated from p values according to the Benjamini-Hochberg procedure. ns, not significant; <sup>∗</sup>, q value&nbsp;&lt;&nbsp;0.05.</div><div id="fspara0090" role="paragraph">See also <a id="crosref1305" href="#figs3">Figure&nbsp;S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/7ac14406-02b2-4830-9085-079abf9baac5/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.049/asset/7004aef9-525b-4316-990b-d6d6d143723d/main.assets/figs3.jpg" height="941" width="707" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/7ac14406-02b2-4830-9085-079abf9baac5/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text"><i>Tet</i>-TKO cells differentiation in the context of mixed chimeras, related to <a id="crosref1375" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0380" role="paragraph">(A) Distribution pattern of <i>Tet</i>-TKO cells in mixed chimeric embryos. <i>Tet</i>-TKO cells show accumulation in the amniotic cavity (arrowheads), and a bias toward the posterior side and base of the allantois. Mixed chimeras were generated by separately injecting three independent <i>Tet</i>-TKO lines. Scale bars, 100&nbsp;μm.</div><div id="fspara0385" role="paragraph">(B) Cell-type composition per embryo. Embryos (represented by columns) are placed along the x axis according to their inferred E<sub>t</sub> calculated by their host cells. Fraction of cell types contributed by host- and by control-derived cells for the same embryo are shown.</div><div id="fspara0390" role="paragraph">(C) Fraction of major lineages per embryo. Black and blue dots represent host- and control-derived cells per chimera, respectively. Black line represents the moving average frequency of WT atlas embryos for indicated lineage, and the shaded gray area represents two moving standard deviations around the mean (window size&nbsp;= 9). Two-sided Wilcoxon-Mann-Whitney rank sum test was used to compare frequencies between host- and control-derived cell types.</div><div id="fspara0395" role="paragraph">(D) Comparison of E<sub>t</sub> calculated for each mixed chimera embryo using host-derived cells, and using control-derived cells.</div><div id="fspara0400" role="paragraph">(E) Gene expression in <i>Tet</i>-TKO and host-/control-derived cells compared with WT for selected cell types. The comparison was performed between query cells from mixed chimeras and their projected WT metacells (<a id="crosref1380" href="#sec-4">STAR Methods</a>). Dashed line indicates a 2-fold change in expression. Genes with over 50% fold change were labeled in black dots.</div><div id="fspara0405" role="paragraph">(F) Distribution of <i>Tet</i>-TKO-derived cell percentage, E<sub>t</sub>, and embryonic time for each mixed chimera. Embryos were ranked by the proportion of <i>Tet</i>-TKO contribution on x axis. Embryonic time was defined by detection of vaginal plugs (designated as E0.5 for am10:00) and served to control for potential effects imposed by sampling times.</div><div id="fspara0410" role="paragraph">(G) Cell-fraction comparison between chimeras with different <i>Tet</i>-TKO contributions. Mixed chimera embryos with E<sub>t</sub>&nbsp;&gt;&nbsp;7.5 were split into groups with low (&lt;0.1) and high (&gt;0.1) contributions of TKO cells. Each dot represents a mixed chimeric embryo, with TKO cells shown in red and control/host cells colored in black. ns, not significant.</div><div id="fspara0415" role="paragraph">(H) Frequency distribution of indicated lineages in mixed chimeric embryos for host/control and <i>Tet</i>-TKO derived cells and WT embryos spanning E<sub>t</sub>7.5–E<sub>t</sub>8.1. Wilcoxon-Mann-Whitney rank sum test, two-tailed. ns, not significant; <sup>∗</sup>, q value&nbsp;&lt;&nbsp;0.05 (Benjamini-Hochberg procedure).</div><div id="fspara0420" role="paragraph">(I) Frequency distribution as calculated in <a id="crosref1385" href="#fig2">Figure&nbsp;2</a>F. Medians of frequencies were compared after downsampling of each embryo to 100 cells. Wilcoxon-Mann-Whitney rank sum test, two-tailed. ns, not significant; <sup>∗</sup>, q value&nbsp;&lt;&nbsp;0.05 (Benjamini-Hochberg procedure).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-4"><h3>Differentiation imbalance in <i>Tet</i>-TKO whole-mutant and mixed chimera embryos</h3><div id="p0075" role="paragraph">Comparison of cell-type frequencies between whole-embryo and mixed chimeric embryos provided us with a sensitive tool for identifying differentiation biases of mutant cells with and without a WT embryonic niche (<a id="crosref0515" href="#fig1">Figures&nbsp;1</a>D and <a id="crosref0520" href="#fig2">2</a>E). While ectoderm and embryonic mesoderm cell populations could not be properly established in whole-embryo mutants, they appear with normal frequencies in mixed chimera embryos (<a id="crosref0525" href="#fig2">Figures&nbsp;2</a>F, <a id="crosref0530" href="#figs3">S3</a>H, and S3I; <a id="crosref0535" href="#mmc1">Table&nbsp;S1</a>). Despite most differentiation programs that were rescued by the host, comparative analysis revealed node/notochord cells to be under-represented also in a mixed chimera setting, suggesting a potential direct contribution of the TET machinery to the regulation of this lineage (<a id="crosref0540" href="#fig2">Figure&nbsp;2</a>G). In a seemingly paradoxical manner, we observed an unexpected elimination of embryonic blood populations in mixed chimeric embryos, despite their robust appearance in whole-embryo KOs (<a id="crosref0545" href="#fig2">Figure&nbsp;2</a>H). Notably, based on previous temporal modeling, specification of blood was shown to occur at late-streak embryos (E<sub>t</sub>7.1), from the highly transient primitive streak populations (<span class="dropBlock reference-citations"><a id="body-ref-sref41-3" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-3" aria-controls="sref41-3">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The tight window for commitment toward this lineage may imply that <i>Tet</i>-TKO cells are outcompeted from this differentiation niche when developing alongside host or control cells. In summary, despite the loss of <i>Tet</i> genes, <i>Tet</i>-TKO cells are intrinsically capable of differentiation into most embryonic cell types. This impact of losing TET activity on gastrulating embryos is highly dependent on the lineage and temporal context, as well as on the existence of supporting signals from WT or mutant cells, and possibly also on competition between cells over restricted differentiation niches.</div></section><section id="sec-2-5"><h3>Cell-autonomous effects of <i>Tet</i>-TKO on epiblast differentiation</h3><div id="p0080" role="paragraph">To identify the intracellular origins of the <i>Tet</i>-linked differentiation defects in whole-embryo mutants, we sought to focus on the earliest effects of <i>Tet</i> loss in the epiblast before any additional indirect perturbations can accumulate. Bulk comparison of <i>Tet</i>-TKO epiblast cells with WT identified little or no changes in the overall transcriptional states (<a id="crosref0555" href="#figs2">Figures&nbsp;S2</a>C and S3E). This however does not preclude that the severe phenotypes emerging in whole-embryo <i>Tet</i>-TKO mutants could initiate from the propagation of smaller changes in the expression of specific <i>Tet</i>-TKO epiblast genes. To test if <i>Tet</i>-TKO cells are running an impaired epiblast program, we computed an “epiblast module score” (<a id="crosref0560" href="#sec-4">STAR Methods</a>), summing up the expression from genes most correlated with <i>Utf1</i>, a master pluripotency transcription factor persistently expressed in the epiblast over time (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Okuda et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Okuda, A. ∙ Fukushima, A. ∙ Nishimoto, M. ...</div><div><strong>UTF1, a novel transcriptional coactivator expressed in pluripotent embryonic stem cells and extra-embryonic cells</strong></div><div><em>EMBO J.</em> 1998; <strong>17</strong>:2019-2032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/17.7.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0032055125" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9524124/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2F17.7.2019&amp;pmid=9524124" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0570" href="#figs4">Figure&nbsp;S4</a>A). Overall, a similar distribution of this score was observed for mutant and control cells, demonstrating that cells were able to maintain the core epiblast signature in the absence of <i>Tet</i> expression (<a id="crosref0575" href="#fig3">Figure&nbsp;3</a>A). This prompted us to compute differential gene expression in a more refined manner, aiming to identify subtle changes within seemingly similar epiblast populations. To control for time-dependent gene expression changes within the epiblast program, we projected each cell onto its most similar WT atlas metacell, using this as a reference to compare both control and mutant cells (<a id="crosref0580" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref0585" href="#sec-4">STAR Methods</a>). In contrast to controls, <i>Tet</i>-TKO lines (in both whole-embryo and mixed chimera contexts) consistently up- and down-regulated multiple genes, indicating that although the epiblast program is generally conserved in mutants, perturbation of specific sub-programs may underlie later, more pronounced, phenotypes.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/75206a90-365b-4381-ac1e-887550ea3b3b/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.049/asset/feb40f6c-f328-4228-bb65-b05fd56ddac2/main.assets/gr3.jpg" height="940" width="661" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/75206a90-365b-4381-ac1e-887550ea3b3b/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Cell-autonomous and non-cell-autonomous effects of <i>Tet</i>-TKO during epiblast and early nascent mesoderm differentiation</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0100" role="paragraph">(A) Density plot of aggregated single-cell expression of epiblast-specific genes among epiblast cells of indicated genotypes from mixed chimera (2N) and whole-embryo control/mutants (4N). x axis shows absolute expression (log<sub>2</sub> of relative unique molecular identifier [UMI] frequency). Different lines of controls and mutants were pooled separately.</div><div id="fspara0105" role="paragraph">(B) Relative gene expression (log<sub>2</sub> of fold change) of <i>Tet</i>-TKO and control epiblast cells compared with projected WT metacells.</div><div id="fspara0110" role="paragraph">(C) Absolute gene expression (log<sub>2</sub> of UMI frequency) in epiblast cells differentiated in 4N (triangle) and 2N (circle) embryos. Black line shows moving average expression for each gene in epiblast cells from WT embryos. Small black dots represent individual embryos of the WT model, and shaded area represents two moving standard deviations around the mean (window size&nbsp;= 13). Wilcoxon-Mann-Whitney rank sum test, two-tailed. Number of 2N, 4N, and WT embryos: 15, 24, and 118.</div><div id="fspara0115" role="paragraph">(D) Density plot of aggregated single-cell expression of early nascent mesoderm genes (as described in A).</div><div id="fspara0120" role="paragraph">(E) Relative gene expression (log<sub>2</sub> of fold change) of <i>Tet</i>-TKO and control early nascent mesoderm cells compared with projected WT metacells.</div><div id="fspara0125" role="paragraph">(F and G) Absolute gene expression (log<sub>2</sub> of UMI frequency) in early nascent mesoderm cells differentiated in 4N and 2N embryos. Wilcoxon-Mann-Whitney rank sum test, two-tailed. Number of 2N, 4N, and WT embryos: 18, 27, and 54.</div><div id="fspara0130" role="paragraph">See also <a id="crosref1310" href="#figs4">Figure&nbsp;S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/06bc28d7-67d2-4217-9c05-ba5262172949/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.049/asset/ac654048-6763-4dee-a690-071d63aee9cd/main.assets/figs4.jpg" height="839" width="762" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/06bc28d7-67d2-4217-9c05-ba5262172949/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text"><i>Tet</i>-TKO effects on epiblast and early nascent mesoderm differentiation, related to <a id="crosref1390" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0430" role="paragraph">(A) Correlation matrix heatmap depicting WT expression of epiblast module genes (upper heatmap), and relative gene expression (log<sub>2</sub> of fold change) for epiblast module genes in <i>Tet</i>-TKO and control epiblast cells compared with projected WT metacells (lower heatmap).</div><div id="fspara0435" role="paragraph">(B) Expression of <i>Pou3f</i> and <i>Sox11</i> in epiblast cells of individual whole-embryo (4N, triangle) and mixed chimera embryos (2N, circle) over time. Wilcoxon-Mann-Whitney rank sum test, two-tailed.</div><div id="fspara0440" role="paragraph">(C) Expression of <i>Tdgf1</i> and <i>Nodal</i> in epiblast cells of individual whole-embryo and mixed chimera embryos over time (E<sub>t</sub>). Wilcoxon-Mann-Whitney rank sum test, two-tailed.</div><div id="fspara0445" role="paragraph">(D) Correlation matrix heatmap, and relative gene expression for nascent mesoderm as shown in <a id="crosref1395" href="#figs5">Figure&nbsp;S5</a>A.</div><div id="fspara0450" role="paragraph">(E and F) Expression of <i>Fgf3</i>, <i>Jag1, Cfc1</i>, <i>Pcdh8</i>, <i>Pcdh19</i> (E), <i>Hand2</i>, and <i>Pitx2</i> (F)&nbsp;in early nascent mesoderm cells from whole-embryo and mixed chimera embryos over time. Wilcoxon-Mann-Whitney rank sum test, two-tailed.</div></div></div></div></figcaption></figure></div><div id="p0085" role="paragraph">Several key factors (e.g., <i>Pou3f1, Id3/2</i>, <i>Sox2</i>, <i>Sox11</i>, and <i>Gdf1</i>) with strong expression in the epiblast were found to be consistently down-regulated in <i>Tet</i>-TKO epiblast cells (<a id="crosref0590" href="#fig3">Figures&nbsp;3</a>B and <a id="crosref0595" href="#figs4">S4</a>B), although only moderately (median log<sub>2</sub> fold changes from −0.23 to −1.1). More notably, <i>Tet</i>-TKO cells failed to repress genes previously implicated in early epiblast differentiation and promotion of mesoderm and endoderm specification, such as <i>Dppa4</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Masaki et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Masaki, H. ∙ Nishida, T. ∙ Kitajima, S. ...</div><div><strong>Developmental pluripotency-associated 4 (DPPA4) localized in active chromatin inhibits mouse embryonic stem cell differentiation into a primitive ectoderm lineage</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:33034-33042</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1074%2Fjbc.M703245200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1074%2Fjbc.M703245200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-36349037021" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17855347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M703245200&amp;pmid=17855347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (log<sub>2</sub> fold change 1.0–3.0, interquartile range [IQR]) and <i>Gdf3</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Chen et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Chen, C. ∙ Ware, S.M. ∙ Sato, A. ...</div><div><strong>The Vg1-related protein Gdf3 acts in a Nodal signaling pathway in the pre-gastrulation mouse embryo</strong></div><div><em>Development.</em> 2006; <strong>133</strong>:319-329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-32244439642" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16368929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02210&amp;pmid=16368929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (log<sub>2</sub> fold change 1.5–2.5, IQR). For these genes, the canonical repression with time in the epiblast was shown to be greatly impaired (<a id="crosref0610" href="#fig3">Figure&nbsp;3</a>C), even in mixed chimera embryos that generally supported WT signaling and near-normal differentiation of <i>Tet</i>-TKO cells. This analysis highlights the potency of our assay to robustly detect intracellular TET effects on genes with multiple&nbsp;levels of controls (temporal, compositional, and multiple cell lines).</div></section><section id="sec-2-6"><h3>Perturbed signaling from <i>Tet</i>-TKO early nascent mesoderm cells</h3><div id="p0090" role="paragraph">Although the composition of mesoderm in whole-embryo mutants was found to be biased compared with mixed chimeras and WT, for those cells that were annotated as early nascent mesoderm (NM), the expression of the core NM gene module, represented by genes correlated to <i>Mesp1</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Saga et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Saga, Y. ∙ Hata, N. ∙ Kobayashi, S. ...</div><div><strong>MesP1: a novel basic helix-loop-helix protein expressed in the nascent mesodermal cells during mouse gastrulation</strong></div><div><em>Development.</em> 1996; <strong>122</strong>:2769-2778</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.122.9.2769" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8787751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.122.9.2769&amp;pmid=8787751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), was conserved (<a id="crosref0620" href="#fig3">Figures&nbsp;3</a>D and <a id="crosref0625" href="#figs4">S4</a>D). This enabled us to screen for intracellular <i>Tet</i>-TKO effects in the earliest multipotent mesoderm progenitor state. Similar to epiblast cells, we validated the consistency of control and host cells with the projected atlas states (<a id="crosref0630" href="#fig3">Figure&nbsp;3</a>E). In contrast, all mutant lines in both whole-embryo chimera and mixed chimera embryos showed significant perturbation of multiple genes. Notably, this included <i>Lefty2</i>, for which we observed failure to induce expression (log<sub>2</sub>-fold changes from −2.7 to −0.8 IQR in whole-mutant embryos, −3.1 to −1.6 IQR in mixed mutant embryos; <a id="crosref0635" href="#fig3">Figure&nbsp;3</a>F). Perturbed Lefty-Nodal signaling was previously suggested to drive <i>Tet</i>-TKO developmental arrest (<span class="dropBlock reference-citations"><a id="body-ref-sref8-3" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-3" aria-controls="sref8-3">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref8-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Dai, H.-Q. ∙ Wang, B.-A. ∙ Yang, L. ...</div><div><strong>TET-mediated DNA demethylation controls gastrulation by regulating Lefty–Nodal signalling</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84994051865" target="_blank" aria-label="Open Scopus in new tab">Scopus (156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20095&amp;pmid=27760115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our data suggest that the origin of this effect is not initiated by intrinsic aberrant Nodal signaling in the epiblast (<a id="crosref0645" href="#figs4">Figure&nbsp;S4</a>C) but is instead rooted in the failure to induce <i>Lefty2</i> in the NM. This may contribute to the skewed differentiation toward an embryonic mesodermal program, at the expense of epiblast differentiation toward definitive ectoderm. <i>Tet</i>-TKO effects on gastrulation signaling from the mesoderm involve, however, multiple other pathways. The data show that <i>Tet</i>-TKO early NM fails to properly induce the FGF signaling molecules <i>Fgf3</i> and <i>Fgf15</i>, Fgf and Ras-Raf-MAPK signaling inhibitor <i>Spry4</i>, Notch signaling factors <i>Dll1</i> and <i>Jag1</i>, Nodal co-receptor <i>Cfc1/Cryptic</i>, and cell adhesion molecules <i>Pcdh8</i> and <i>Pcdh19</i> that interact with Wnt pathway and apoptotic cascades (<a id="crosref0650" href="#fig3">Figures&nbsp;3</a>E, 3G, and <a id="crosref0655" href="#figs4">S4</a>E). Additional <i>Tet</i>-TKO intrinsic transcriptional perturbation involves induction (or de-repression) of genes, including the TFs <i>Hand2</i> and <i>Pitx2</i>, linked with extraembryonic mesoderm cells (ExM), which are indeed over-abundant in mutant embryos (<a id="crosref0660" href="#figs4">Figure&nbsp;S4</a>F). In summary, <i>Tet</i>-TKO cells are capable of establishing an early NM program, but this state is severely impaired in its signaling capacity. Failure to generate normal signaling involves perturbations in the Lefty-Nodal signaling pathway and multiple additional signaling axes, which, together with the direct effects of TET on early epiblast genes, may explain ectoderm depletion and alteration in mesoderm differentiation for whole-embryo mutants.</div></section><section id="sec-2-7"><h3>Double <i>Tet</i> knockouts establish all embryonic lineages, given WT host context</h3><div id="p0095" role="paragraph">To better understand intracellular differentiation effects and link them with specific <i>Tet</i> genes, we generated sets of GFP-tagged combinatorial <i>Tet</i> double-KO (DKO) mESCs and isogenic controls, hereinafter referred to as DKO1/2, DKO1/3, and DKO2/3 (<a id="crosref0665" href="#figs5">Figures&nbsp;S5</a>A and S5B; <a id="crosref0670" href="#sec-4">STAR Methods</a>). Overall, at the time of dissection (∼E8.0), chimeric embryos generated by injecting mutant cells into 2N blastocysts displayed normal morphology with high levels of chimerism associated with all three genotypes (<a id="crosref0675" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref0680" href="#figs5">S5</a>C). Interestingly, we observed aberrant accumulation of cells inside the amniotic cavity in a total 16 of 40 DKO1/2, 11 of 46 DKO1/3 chimeras, but in none of 33 DKO2/3 embryos (from three independent experiments). This suggests that <i>Tet1</i> may directly regulate early gastrulation phenotypes in <i>Tet</i>-TKO cells (<a id="crosref0685" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref0690" href="#figs5">S5</a>D). We further identified high levels of concordance when determining embryos’ transcriptional time by either <i>Tet</i>-DKO or host cells. This demonstrated a lack of class III <i>temporal</i> aberrations and reproducible synchronization of mutant and control cells, with some potential developmental delay observed for DKO1/2 mutant cells (<a id="crosref0695" href="#figs5">Figure&nbsp;S5</a>E). scRNA-seq analysis of mutant and host cells showed an overall robust contribution of all DKO genotypes to nearly all cell types expected in the examined embryos (<a id="crosref0700" href="#fig4">Figures&nbsp;4</a>B and <a id="crosref0705" href="#figs5">S5</a>F; <a id="crosref0710" href="#mmc1">Table&nbsp;S1</a>). Type II <i>compositional</i> perturbations analysis showed that similar to <i>Tet</i>-TKO chimeras, DKO1/2 and DKO1/3 mutants generated almost no blood lineages compared with host cells, whereas DKO2/3 mutants did populate blood lineages, albeit with less efficiency. This, again, could be possibly due to the lack of competitiveness compared with host cells in populating this lineage, since DKO1/2 live pups can be born (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Dawlaty et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Combined deficiency of Tet1 and Tet2 causes epigenetic abnormalities but is compatible with postnatal development</strong></div><div><em>Dev. Cell.</em> 2013; <strong>24</strong>:310-323</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2012.12.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2012.12.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84873707539" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23352810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2012.12.015&amp;pmid=23352810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, DKO1/2 and DKO1/3 mutants were also relatively under-represented in endoderm lineages compared with the host cells, such as primitive foregut. Finally, similar to <i>Tet</i>-TKO chimeras, all three DKO mutants exhibited adequate contribution to ExM lineages (<a id="crosref0720" href="#fig4">Figures&nbsp;4</a>B and <a id="crosref0725" href="#figs5">S5</a>G; <a id="crosref0730" href="#mmc1">Table&nbsp;S1</a>). In conclusion, this analysis showed that DKO cells are capable of establishing most transcriptional states when differentiated in a chimeric context, with some significant compositional biases that motivate further in-depth analysis of the underlying transcriptional perturbations within each state.</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/558dac20-94e6-4346-aa4e-817c999e1d8e/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.049/asset/b15f54dc-0655-492a-ab7d-4e591ae734b9/main.assets/figs5.jpg" height="941" width="674" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/558dac20-94e6-4346-aa4e-817c999e1d8e/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Morphological and molecular perturbations in <i>Tet</i>-DKO mixed chimera embryos, related to <a id="crosref1400" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0460" role="paragraph">(A) DNA sequencing validation for DKO clones after genome editing. PAM sequences are marked in red.</div><div id="fspara0465" role="paragraph">(B) Western blot validation of <i>Tet</i>-DKO mESCs clones. DKO1/2#10, DKO1/2#25, DKO1/3#3, and DKO2/3#5 were selected for subsequent chimera assays.</div><div id="fspara0470" role="paragraph">(C) Flow cytometric analysis for degree of chimerism for each embryo. Number of embryos shown for each genotype. Data are represented as mean ± SD.</div><div id="fspara0475" role="paragraph">(D) Representative images of E7.5 DKO1/2 and DKO1/3 mixed chimeric embryos. Arrowheads show the accumulation of cells inside the amniotic cavity. Embryos orientated with posterior to the right (anterior side faces up for the last DKO1/3 embryo). Scale bars, 100&nbsp;μm.</div><div id="fspara0480" role="paragraph">(E) Comparison of E<sub>t</sub> calculated for each DKO mixed chimera embryo using both DKO- and host-derived cells.</div><div id="fspara0485" role="paragraph">(F) 2D-projection of transcriptome profiles onto the WT atlas (gray background) of knockout-derived cells and host-derived cells obtained from DKO1/2, DKO1/3, and DKO2/3 mixed chimera embryos, respectively. Single cells are colored by projected atlas cell type.</div><div id="fspara0490" role="paragraph">(G) Contribution of DKO1/2, DKO1/3, and DKO2/3 mutant cells to blood, foregut, non-embryonic mesoderm, and node/notochord as calculated in <a id="crosref1405" href="#fig2">Figure&nbsp;2</a>F. Medians of frequencies were compared after downsampling of each embryo to 90 cells. Wilcoxon-Mann-Whitney rank sum test, two-tailed. ns, not significant; <sup>∗</sup>, q value&nbsp;&lt;&nbsp;0.05 (Benjamini-Hochberg procedure).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-8"><h3><i>Tet</i> knockouts perturb transcriptional states quantitatively</h3><div id="p0100" role="paragraph">Comparing transcriptional states between WT, <i>Tet</i>-TKO, and different <i>Tet</i>-DKO genotypes showed a high degree of conservation in the embryonic mesoderm, endoderm, and ectoderm cell lineages (<a id="crosref0735" href="#fig4">Figure&nbsp;4</a>C). Following up on the quantitative analysis in the epiblast and NM states, we conducted a refined search for quantitative differences in more advanced mutant and WT transcriptional programs. We aggregated cells representing 11 differentiated cell states, separately from each of the genotypes (<a id="crosref0740" href="#sec-4">STAR Methods</a>). As demonstrated in <a id="crosref0745" href="#fig4">Figure&nbsp;4</a>D for surface ectoderm, we observed overall high agreement in quantitative gene expression between mutant and host states, with few genes having more than 2-fold differential expression. This strongly supports the notion that for the majority of affected loci, the TET system acts to fine-tune gene expression quantitatively, rather than instructively regulating it. Nevertheless, estimation of the overall transcriptional deviation between host and mutant genotypes showed an intriguing hierarchy in which TKO cell types are consistently most strongly affected, followed by DKO1/2 and DKO1/3. Interestingly, among the mutants, DKO2/3 predominantly manifested the least transcriptional deviation compared with the WT program (<a id="crosref0750" href="#fig4">Figure&nbsp;4</a>E). Taken together, individual <i>Tet</i> genes appear to be largely compensatory for each other across advanced cell types. At the same time, the TET system (with TET1 being more prominent in that respect) is shown to have a global quantitative impact on the regulation of a large number of genes across multiple lineages.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/b51cf184-797b-4178-86fa-1e290db072db/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.049/asset/288d8c90-53fc-4834-ab6e-916291acf84f/main.assets/gr4.jpg" height="812" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/b51cf184-797b-4178-86fa-1e290db072db/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Quantitative effects of <i>Tet</i> genes on transcriptomes of advanced cell types</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) Representative images of E7.5 <i>Tet</i>-DKO mixed chimera embryos generated by injection of GFP-labeled mESCs into 2N blastocysts. Arrowheads showing cells protruding into the cavity in early-stage DKO1/2 and DKO1/3 chimeric embryos. Scale bars, 100&nbsp;μm.</div><div id="fspara0145" role="paragraph">(B) Cell-type composition per embryo as calculated in <a id="crosref1315" href="#fig2">Figure&nbsp;2</a>E. Fraction of cell types contributed by host and by <i>Tet</i>-DKO-derived cells for the same embryo are shown.</div><div id="fspara0150" role="paragraph">(C) Transcriptional similarity (correlation of gene expression) of advanced cell types between host, <i>Tet</i>-TKO, DKO1/2, DKO1/3, and DKO2/3 cells from mixed chimeras and WT atlas.</div><div id="fspara0155" role="paragraph">(D) Scatter plot showing gene expression of surface ectoderm in lines of control and mutants compared with WT. Dashed lines indicate a 2-fold change.</div><div id="fspara0160" role="paragraph">(E) A summary chart showing average differential gene expression (log<sub>2</sub> of fold change) in different cell types, each compared with the corresponding WT profile (<a id="crosref1320" href="#sec-4">STAR Methods</a>). Number of embryos: 13 DKO1/2, 8 DKO1/3, 6 DKO2/3, 37 <i>Tet</i>-TKO, 16 Ctrl, and 45 Host embryos.</div><div id="fspara0165" role="paragraph">See also <a id="crosref1325" href="#figs5">Figure&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-9"><h3>Loss of the TET machinery is linked with massive embryonic hypermethylation</h3><div id="p0105" role="paragraph">To map the impact of <i>Tet</i> deficiency on embryonic DNA methylation while focusing on intrinsic effects, we injected a mixture of TKO and control cells to 2N blastocysts, generating chimeric embryos that were harvested at E8.5 when substantial differentiation is already established. We then sorted apart <i>Tet</i>-TKO mutant and control cells for analysis using post-bisulfite adaptor tagging (PBAT) (<a id="crosref0755" href="#fig5">Figure&nbsp;5</a>A). Importantly, this experimental design ensured analysis of only embryonic cells, excluding the potentially confounding extraembryonic ectoderm, while controlling for temporal effects (since control and mutant cells were collected from the same embryos). Analysis of over 10M CpGs (see <a id="crosref0760" href="#sec-4">STAR Methods</a>) showed a dramatic increase in methylation in TKO cells in the majority of partially methylated CpGs in the genome, but not for fully protected loci (<a id="crosref0765" href="#fig5">Figure&nbsp;5</a>B; close to zero methylation in both control and <i>Tet</i>-TKO). Importantly, although analysis of pooled <i>Tet</i>-TKO cells potentially limits the ability to understand methylation perturbation in specific lineages or cell types, the dominant and pervasive <i>Tet</i>-TKO hypermethylation effect observed suggests it is unlikely to come from perturbed methylation profile of specific cell types.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/138a9dec-8112-4930-ba09-b52d35372770/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.049/asset/ebac1e04-366c-4b52-9fad-8c2da1b5f204/main.assets/gr5.jpg" height="592" width="761" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/138a9dec-8112-4930-ba09-b52d35372770/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Charting DNA methylation landscape in E8.5 <i>Tet</i>-TKO and control cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0175" role="paragraph">(A) Schematic of PBAT experiment.</div><div id="fspara0180" role="paragraph">(B) Smoothed scatter plot between DNA methylation levels of individual CpGs genome-wide. Density of data points ranges from blue (low) to yellow (intermediate) and red (high).</div><div id="fspara0185" role="paragraph">(C) DNA methylation at TADs (n&nbsp;= 2,461), binned according to the methylation level in control cells. The middle line indicates the median; box limits represent quartiles; and whiskers are 1.5× the interquartile range.</div><div id="fspara0190" role="paragraph">(D) DNA methylation distribution for early and late replicating loci in <i>Tet</i>-TKO and control.</div><div id="fspara0195" role="paragraph">(E) DNA methylation distribution for H3K4me3 (left, n&nbsp;= 953) and H3K27me3 (right, n&nbsp;= 2,587) marked loci in <i>Tet</i>-TKO and control. Bivalent loci (left, n&nbsp;= 668; right, n&nbsp;= 2,298) are marked in red.</div><div id="fspara0200" role="paragraph">(F) DNA methylation distribution for CTCF-bound sites marked (left, n&nbsp;= 3,276) or unmarked (right, n&nbsp;= 17,860) by H3K4me3 modification in <i>Tet</i>-TKO and control.</div><div id="fspara0205" role="paragraph">(G) DNA methylation distribution for exons (left, n&nbsp;= 47,734) and promoters unmarked by H3K4me3 modification (right, n&nbsp;= 3,381) in <i>Tet</i>-TKO and control.</div><div id="fspara0210" role="paragraph">(H) Smoothed scatter plot between DNA methylation levels of putative enhancers in <i>Tet</i>-TKO and control (n&nbsp;= 12,720).</div><div id="fspara0215" role="paragraph">(I) Distribution of DNA methylation around the center of putative enhancers in <i>Tet</i>-TKO and control, separated into three plots according to control methylation levels. The middle line indicates the median; box limits represent quartiles; and whiskers are 1.5× the interquartile range. Number of loci: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m Subscript upper C t r l" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="m Subscript upper C t r l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;&lt;&nbsp;0.3 (n&nbsp;= 12,016), 0.3&nbsp;&lt;&nbsp;<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m Subscript upper C t r l" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="m Subscript upper C t r l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;&lt;&nbsp;0.7 (n&nbsp;= 9,971), 0.7&nbsp;&lt;&nbsp;<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m Subscript upper C t r l" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="m Subscript upper C t r l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;&lt;&nbsp;1 (n&nbsp;= 7,280).</div><div id="fspara0220" role="paragraph">See also <a id="crosref1330" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">To characterize the effects of <i>Tet</i>-TKO on methylation, we analyzed methylation in different epigenomic contexts. First, we computed mean methylation in broad genomic domains (defined using topologically associated domains [TADs]) (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Dixon et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (4600)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while first eliminating all CpGs linked with any putative functional or epigenomic role. This allowed for analysis of the basal (or <i>background</i>) methylation levels over TADs with different control methylation levels (<a id="crosref0775" href="#fig5">Figure&nbsp;5</a>C). The data showed that the variation in background methylation between TADs was greatly diminished in <i>Tet</i>-TKO cells. Remarkably, when stratifying domains by their estimated time of replication (<span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Nagano et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Nagano, T. ∙ Lubling, Y. ∙ Várnai, C. ...</div><div><strong>Cell-cycle dynamics of chromosomal organization at single-cell resolution</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:61-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85022195504" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28682332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23001&amp;pmid=28682332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observed that lower background methylation is linked with early replicating TADs in control and a reciprocal effect in <i>Tet</i>-TKO (<a id="crosref0785" href="#fig5">Figure&nbsp;5</a>D). We note that our analysis infers higher methylation levels in the embryo compared with some previously published WT data. This is likely due to the elimination of extraembryonic ectoderm from the analysis, which represents a more hypomethylated (<span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Smith et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Smith, Z.D. ∙ Shi, J. ∙ Gu, H. ...</div><div><strong>Epigenetic restriction of extraembryonic lineages mirrors the somatic transition to cancer</strong></div><div><em>Nature.</em> 2017; <strong>549</strong>:543-547</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23891" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85032626002" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28959968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23891&amp;pmid=28959968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) cell population that can affect the estimation of average methylation when not excluded (<a id="crosref0795" href="#figs6">Figure&nbsp;S6</a>A). These data support a role for widespread TET-mediated demethylation in early replicating TADs, which is lost upon <i>Tet</i> KO. Such demethylation may rely on the enhanced accessibility of early replicating domains as part of the chromosomal A-compartment (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">López-Moyado et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>López-Moyado, I.F. ∙ Tsagaratou, A. ∙ Yuita, H. ...</div><div><strong>Paradoxical association of TET loss of function with genome-wide DNA hypomethylation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:16933-16942</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1903059116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071273156" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1903059116&amp;pmid=31371502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Pope et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Pope, B.D. ∙ Ryba, T. ∙ Dileep, V. ...</div><div><strong>Topologically associating domains are stable units of replication-timing regulation</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:402-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13986" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84911478490" target="_blank" aria-label="Open Scopus in new tab">Scopus (614)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409831/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13986&amp;pmid=25409831" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/e434c13b-5558-497f-b964-caa444b64c79/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.049/asset/1d6607e8-f368-42ec-9059-0395756cea3c/main.assets/figs6.jpg" height="942" width="734" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/e434c13b-5558-497f-b964-caa444b64c79/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Supporting information, related to <a id="crosref1410" href="#fig5">Figures&nbsp;5</a> and <a id="crosref1415" href="#fig6">6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0500" role="paragraph">(A) DNA methylation at TADs, binned according to the methylation level in control cells (n&nbsp;= 4,690). The middle line indicates the median; box limits represent quartiles; and whiskers are 1.5× the interquartile range.</div><div id="fspara0505" role="paragraph">(B) H3K4me3 and H3K27me3 modification levels for loci with control methylation lower than 0.25 (n&nbsp;= 1,766,303; n for control&nbsp;= 15,379,400).</div><div id="fspara0510" role="paragraph">(C) DNA methylation distribution for exons and promoters unmarked by H3K4me3 in <i>Tet</i>-TKO and published WT (<a id="crosref1420" href="#sec-4">STAR Methods</a>).</div><div id="fspara0515" role="paragraph">(D) Distribution of H3K4me3 density for embryonically low (n&nbsp;= 11,758), middle (n&nbsp;= 6,214), and highly (n&nbsp;= 5,040) expressed genes.</div><div id="fspara0520" role="paragraph">(E) Distribution of promoter DNA methylation in <i>Tet</i>-TKO and control for non-H3K4me3 promoter related genes. Plotted for embryonically low (n&nbsp;= 3,428), middle (n&nbsp;= 254), and highly (n&nbsp;= 158) expressed genes.</div><div id="fspara0525" role="paragraph">(F) H3K4me1 and H3K27ac modification for loci that exhibit a local reduction in methylation (methylation dips, n&nbsp;= 25,134). 51,113,471 50&nbsp;bp sized genomic bins are used as control.</div><div id="fspara0530" role="paragraph">(G) Smoothed scatter plot showing DNA methylation levels of CpGs (n&nbsp;= 34,185) associated with putative enhancers in Tet-TKO and WT.</div><div id="fspara0535" role="paragraph">(H) DNA methylation levels for a selection of TF binding motif in enhancer regions identified in <i>Tet</i>-TKO and control. Enhancer regions without the motifs were used as controls. The middle line indicates the median; box limits represent quartiles; and whiskers are 1.5× the interquartile range.</div><div id="fspara0540" role="paragraph">(I) Heatmap of gene expression (log<sub>2</sub> of relative UMI frequency) of marker genes defining crown cells, pit cells, notochord, and foregut in the WT atlas. Metacells were extracted from the WT atlas.</div><div id="fspara0545" role="paragraph">(J) DNA sequencing validation for <i>Tet3</i> KO mESCs clone. PAM sequences are marked in red. Intron sequences are shown in lower-case letters.</div><div id="fspara0550" role="paragraph">(K) Node/notochord development in <i>Tet3</i> whole-embryo KO chimeras. Phase-contrast images for three KO embryos are shown in the first column with node area pointed by triangles. z stack images of node/notochord stained by DAPI (gray) and anti-FOXA2 (red) are shown on the right. Pseudo color is used for a better contrast. Dashed line outlines the node/notochord structure. n, node. Scare bars, 100&nbsp;μm.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-10"><h3>Partial <i>Tet</i>-TKO hypermethylation at H3K4me3/H3K27me3-marked loci and nearly complete hypermethylation at putative enhancers</h3><div id="p0115" role="paragraph">Within a background of very high methylation (&gt;0.9 average), hotspots of low methylation (control methylation &lt;0.25) were associated with H3K4me3 or H3K27me3 marking and, in some cases, bivalent marking (<a id="crosref0810" href="#figs6">Figure&nbsp;S6</a>B). While increase in <i>Tet</i>-TKO methylation in those regions was observed for almost all loci, it represented an incomplete process, where out of the loci with control methylation &lt;0.25, only 5% showed <i>Tet</i>-TKO methylation &gt;0.5, and none showed methylation &gt;0.8 (<a id="crosref0815" href="#fig5">Figure&nbsp;5</a>E). Loci linked with CCCTC-binding factor (CTCF) occupancy (either in promoter/H3K4me3 or out of such context) showed a similar effect (<a id="crosref0820" href="#fig5">Figure&nbsp;5</a>F). In contrast, loci within exons or promoters lacking H3K4me3 markup showed remarkably extensive hypermethylation in <i>Tet</i>-TKO cells (<a id="crosref0825" href="#fig5">Figures&nbsp;5</a>G and <a id="crosref0830" href="#figs6">S6</a>C; <a id="crosref0835" href="#mmc2">Table&nbsp;S2</a>). We note that since the vast majority (96%) of embryonically expressed genes (in any lineage) are enriched for H3K4me3 (or bivalent H3K4me3/H3K27me3 markup) (<a id="crosref0840" href="#figs6">Figure&nbsp;S6</a>D), hypermethylation at non-H3K4me3 promoters was generally independent of gene expression (<a id="crosref0845" href="#figs6">Figure&nbsp;S6</a>E). This showed that loci normally protected from de novo methylation (i.e., gene promoters associated with H3K4me3) preserved some of this protection in <i>Tet</i>-TKO cells compared with normally unprotected loci that gained near-complete methylation. Therefore, the TET machinery contributes to, but not solely responsible for, the lack of methylation associated with developmentally regulated loci (such as those targeted by Trithorax/Polycomb).</div><div id="p0120" role="paragraph">Next, we analyzed <i>Tet</i>-TKO methylation distribution at distal elements that showed partial methylation in an independent E8.5 whole-genome bisulfite sequencing (WGBS) dataset (<a id="crosref0850" href="#sec-4">STAR Methods</a>). Such elements are strongly correlated with putative enhancer marks in differentiating lineages (<a id="crosref0855" href="#figs6">Figure&nbsp;S6</a>F). A subset of these loci that was fully protected from <i>de novo</i> methylation in WT and controls remained largely hypomethylated in mutant cells (<i>Tet</i>-TKO methylation &lt;0.2 for 60% of the enhancer loci with control methylation &lt;0.1; <a id="crosref0860" href="#fig5">Figures&nbsp;5</a>H and <a id="crosref0865" href="#figs6">S6</a>G). However, remarkably, almost all other putative enhancers acquired extremely high methylation levels in <i>Tet</i>-TKO cells (median mutant methylation 0.75 for loci with control methylation between 0.2 and 0.6). Methylation distribution analysis in these regions (<a id="crosref0870" href="#fig5">Figure&nbsp;5</a>I) showed that enhancers with low methylation in the control (&lt;0.3) preserved a characteristic hypomethylation trend in mutant cells (average minimal control 0.16, compared with 0.41 in <i>Tet</i>-TKO, p ≪ 0.001), whereas in partially methylated enhancers, protection from <i>de novo</i> methylation was reduced by over 50% (average control 0.5, and <i>Tet</i>-TKO 0.72, p ≪ 0.001). Such pervasive and consistent increase in enhancer methylation was not restricted to specific lineages, as supported by the lack of strong correlation between <i>Tet</i>-TKO methylation and known mesoderm, ectoderm, and endoderm TF binding sites (<a id="crosref0875" href="#figs6">Figure&nbsp;S6</a>H; <a id="crosref0880" href="#mmc3">Table&nbsp;S3</a>). Together, these data suggest a role for TET-mediated protection from <i>de novo</i> methylation at enhancers, where hypomethylation is not restricted to loci that are active in specific lineages or harboring specific co-factors. Although we cannot directly link such methylation changes with gene regulation, the data are consistent with the pervasive quantitative transcriptional effect we observed for <i>Tet</i> disrupted cells.</div></section><section id="sec-2-11"><h3>Elevated <i>Tet3</i> expression in the embryonic node/notochord lineages</h3><div id="p0125" role="paragraph">As shown above, triple and double <i>Tet</i> KO cells were represented in nearly all embryonic lineages when differentiated alongside host cells in chimera assays. However, a notable exception to this trend was the reduction in representation of node/notochord cell states among <i>Tet</i>-TKO cells. Morphological examination of <i>Tet</i>-TKO cells in mixed chimeric embryos showed the absence of <i>Tet</i>-TKO cells from the distal part of the embryo (<a id="crosref0885" href="#fig6">Figure&nbsp;6</a>A). Further imaging analysis by co-staining with FOXA2 antibody showed little or no contribution of mutant cells to the node and notochord in E8.0 embryos (<a id="crosref0890" href="#fig6">Figure&nbsp;6</a>B). Closer examination of node/notochord annotated metacells in the WT atlas identified transcriptional states corresponding to three structural components: notochord, pit cells, and crown cells (<a id="crosref0895" href="#fig6">Figures&nbsp;6</a>C and <a id="crosref0900" href="#figs6">S6</a>I). Inferred differentiation kinetics toward these states, compared with control endoderm metacells (foregut), demonstrated remarkable transcriptional dynamics associated with <i>Tet</i> genes and <i>de novo</i> methylation genes (<i>Dnmts</i>) (<a id="crosref0905" href="#fig6">Figure&nbsp;6</a>D; <a id="crosref0910" href="#sec-4">STAR Methods</a>). Most notably, we observed induction of <i>Tet3</i> expression and reduction of <i>Tet1</i>, <i>Dnmt3a</i>, and <i>Dnmt3b</i> beginning at E<sub>t</sub>7.5 when the node state is initially specified. The housekeeping methylation maintenance machinery (<i>Dnmt1</i> and <i>Uhrf1</i>) showed stable expression in the gut, notochord, and crown cells fates, but a specific decline in pit cells. This coincides with an overall dramatic reduction in cell-cycle rate in these cells (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Bellomo et&nbsp;al., 1996</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bellomo, D. ∙ Lander, A. ∙ Harragan, I. ...</div><div><strong>Cell proliferation in mammalian gastrulation: the ventral node and notochord are relatively quiescent</strong></div><div><em>Dev. Dyn.</em> 1996; <strong>205</strong>:471-485</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-0177(199604)205:4%3C471::AID-AJA10%3E3.0.CO;2-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0029873599" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8901057/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-0177%28199604%29205%3A4%253C471%3A%3AAID-AJA10%253E3.0.CO%3B2-4&amp;pmid=8901057" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-4" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-4" aria-controls="sref41-4">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Pijuan-Sala et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Pijuan-Sala, B. ∙ Wilson, N.K. ∙ Xia, J. ...</div><div><strong>Single-cell chromatin accessibility maps reveal regulatory programs driving early mouse organogenesis</strong></div><div><em>Nat. Cell Biol.</em> 2020; <strong>22</strong>:487-497</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-020-0489-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85082943514" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32231307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-020-0489-9&amp;pmid=32231307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We estimated the proliferation rate through the expression of S-phase and M-phase genes (<a id="crosref0930" href="#sec-4">STAR Methods</a>), indicating the pit cells to be almost completely arrested (91% showing low cell-cycle gene expression), the crown cells to be largely arrested (53% with low cell-cycle gene expression), and the notochord to maintain replication but at a much slower rate than other embryonic lineages (<a id="crosref0935" href="#fig6">Figure&nbsp;6</a>E; 32% compared with 97% over the entire embryo). Interestingly, all three DKO lines can give rise to notochord cells (<a id="crosref0940" href="#figs5">Figure&nbsp;S5</a>G), and node/notochord structure seems to stay largely intact when knocking out <i>Tet3</i> alone from the embryo proper (<a id="crosref0945" href="#figs6">Figures&nbsp;S6</a>J and S6K), suggesting a compensatory role for <i>Tet3</i> by the other <i>Tet</i> family members during notochord specification.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.06.049/asset/16829617-bcaf-4f0f-a6a7-3b06d36bce96/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.049/asset/193b5ebf-ad5f-44e0-90d9-d91edf044e63/main.assets/gr6.jpg" height="881" width="753" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/16829617-bcaf-4f0f-a6a7-3b06d36bce96/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Analysis of <i>Tet</i> effects in the node/notochord lineage</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0230" role="paragraph">(A) Representative images showing little or no contribution of <i>Tet</i>-TKO cells to the node/notochord in a head-fold stage chimeric embryo, generated by co-injection of control (green) and <i>Tet</i>-TKO (red) mESCs into 2N blastocysts. High-magnification images of the node area are shown on the right. n, node. Scale bars, 100&nbsp;μm.</div><div id="fspara0235" role="paragraph">(B) Representative z stack images of E8.0 chimeric embryo generated by co-injection of control (green) and <i>Tet</i>-TKO (red) mESCs and stained with DAPI (blue), and anti-FOXA2 (purple). Node/notochord structure is outlined by a dashed line. Scale bars, 100&nbsp;μm.</div><div id="fspara0240" role="paragraph">(C) A ventral view of node/notochord structure and cell types comprising it. R, right; L, left; V, ventral; D, dorsal.</div><div id="fspara0245" role="paragraph">(D) Expression kinetics for indicated genes, shown using absolute expression level (log<sub>2</sub> of UMI frequency) along trajectories leading to crown cells, pit cells, notochord, and foregut.</div><div id="fspara0250" role="paragraph">(E) Cell-cycle score for crown cells, pit cells, and notochord cells compared with cells from all other embryonic cell types of the WT atlas. Cells below the threshold of 0.5 are defined as having low cell-cycle expression.</div><div id="fspara0255" role="paragraph">(F) Representative images of head-fold stage wild-type embryo sections, stained with DAPI (blue), anti-FOXA2 (red), and anti-5hmC (green). Scare bars, 100&nbsp;μm.</div><div id="fspara0260" role="paragraph">(G) Quantification of 5hmC intensity in notochord FOXA2<sup>+</sup> cells, compared with FOXA2<sup>−</sup> mesoderm cells adjacent to the notochord, in five head-fold to 2–3 somite stage wild-type embryos. Wilcoxon-Mann-Whitney test, two-tailed. Error bars denote SD.</div><div id="fspara0265" role="paragraph">See also <a id="crosref1335" href="#figs6">Figure&nbsp;S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">Consistent with the unique expression pattern of <i>Tet3</i>, staining of late head-fold stage embryos identified relatively higher 5hmC intensity with node/notochord cells (marked by FOXA2) but not in mesoderm cells in its lateral proximal vicinity (<a id="crosref0950" href="#fig6">Figures&nbsp;6</a>F and 6G). We speculate that high 5hmC levels in this developing niche may be linked to a shift in the balance between passive demethylation through DNA replication and active demethylation mediated by TET activity. The potential involvement of TETs (and 5hmC levels) in direct regulation of the notochord program remains to be described functionally. However, the combined evidence of specific <i>Tet3</i> expression, elevated 5hmC levels, reduced proliferation, and altered differentiation of <i>Tet</i>-TKO cells toward the notochord are highly suggestive of potential regulatory function.</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0135" role="paragraph">Here, we systematically dissected the intrinsic and indirect impact of complete or partial TET enzyme deficiency during gastrulation by analyzing single cells derived from precisely timed chimeric embryos. At the phenomenological level, whole-embryo <i>Tet</i>-TKO mutants largely recapitulated germline KO phenotypes (<span class="dropBlock reference-citations"><a id="body-ref-sref8-4" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-4" aria-controls="sref8-4">Dai et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref8-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Dai, H.-Q. ∙ Wang, B.-A. ∙ Yang, L. ...</div><div><strong>TET-mediated DNA demethylation controls gastrulation by regulating Lefty–Nodal signalling</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84994051865" target="_blank" aria-label="Open Scopus in new tab">Scopus (156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20095&amp;pmid=27760115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-4" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-4" aria-controls="sref35-4">Li et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref35-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Li, X. ∙ Yue, X. ∙ Pastor, W.A. ...</div><div><strong>Tet proteins influence the balance between neuroectodermal and mesodermal fate choice by inhibiting Wnt signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E8267-E8276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1617802113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85006421153" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27930333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1617802113&amp;pmid=27930333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with delayed and abnormal gastrulation characterized by the failure to form mesoderm and ectoderm lineages, excessive differentiation toward the extraembryonic mesoderm, and aberrant accumulation of cells inside the embryonic cavity. However, these severe developmental phenotypes are rescued almost completely in mixed-mutant chimeras, when host cells provide a normal developmental niche to the mutant cells and support their robust contribution to almost all embryonic lineages. To define the initial embryonic function of the TET machinery at the cellular level, we combined analysis of three layers of single-cell datasets: (1) a temporal gastrulation atlas representing WT programs, (2) transcriptional maps of mutant cells developing in a WT host, and (3) whole-mutant embryos. Complementing previous results from genetic <i>Tet</i> ablation, we show that the loss of <i>Tet</i> expression in the entire embryo initially leads to failure to repress the early epiblast program, resulting in delayed epiblast maturation. However, continuous impaired differentiation is likely the consequence of a cell-intrinsic failure to induce NM signaling (Lefty, FGF, and Notch) that indirectly affects the balance and synchronization of epiblast conversion to ectoderm and specification of the anterior-posterior mesodermal axis.</div><div id="p0140" role="paragraph">The results from the chimera analyses show how the interpretation of the developmental function of a gene or pathway can be extremely misleading when relying solely on a broad phenotypic assay. A notable example is the absence of <i>Tet</i>-TKO cells from the blood lineage when differentiated alongside host cells. While this can be misinterpreted as an altered potential of the injected cells, we propose that the tight specification window toward this lineage renders host cells more favorable to occupy this niche. Indeed, in support of this notion, <i>Tet</i>-TKO cells retain the potential to differentiate to blood in whole-embryo mutants, and control mESCs showed stochastic and limited contribution to this lineage. In contrast to these context-specific effects, we show that the node/notochord lineage is consistently affected by loss of the TET machinery, demonstrating that <i>Tet</i> genes do have the potential to intrinsically regulate key lineages during gastrulation. Collectively, our work outlines an experimental and methodological framework to map the intracellular consequences of embryonic gene manipulation and disentangle them from indirect non-cell-autonomous effects propagating between lineages.</div><div id="p0145" role="paragraph">Analyzing combinations of triple and double <i>Tet</i> KO cells differentiated in mixed chimeric embryos showed a balanced contribution of mutant cells to almost all lineages. Nevertheless, quantitative analysis highlighted a mild yet pervasive perturbation of the regulation of many genes in DKO and TKO mutants. This suggested that much of the impact of the TET machinery consists in the quantitative modulation of gene expression across the various gastrulation lineages. Only some of these effects (e.g., the regulation of signaling in the mesoderm) escalate toward major downstream aberrations. Analysis of DNA methylation in mutant cells acquired from chimeric embryos showed intense hypermethylation of the majority of putative gene regulatory elements. Nevertheless, gene promoters associated with H3K4me3 or bivalent marks maintained most of their protection from DNA methylation. This implicates <i>Tet</i> genes in the maintenance of hypomethylation across distal regulatory elements, which may stabilize precise transcriptional programs in differentiating lineages. Future studies, using a better temporal resolution of methylation turnover (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Ginno et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Ginno, P.A. ∙ Gaidatzis, D. ∙ Feldmann, A. ...</div><div><strong>A genome-scale map of DNA methylation turnover identifies site-specific dependencies of DNMT and TET activity</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2680</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16354-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85085693059" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32471981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16354-x&amp;pmid=32471981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Hon et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Hon, G.C. ∙ Song, C.-X. ∙ Du, T. ...</div><div><strong>5-mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation</strong></div><div><em>Mol. Cell.</em> 2014; <strong>56</strong>:286-297</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2014.08.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2014.08.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84908204470" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25263596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2014.08.026&amp;pmid=25263596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Wang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Wang, Q. ∙ Yu, G. ∙ Ming, X. ...</div><div><strong>Imprecise DNMT1 activity coupled with neighbor-guided correction enables robust yet flexible epigenetic inheritance</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:828-839</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0661-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85088264348" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32690947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0661-y&amp;pmid=32690947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Williams et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Williams, K. ∙ Christensen, J. ∙ Pedersen, M.T. ...</div><div><strong>TET1 and hydroxymethylcytosine in transcription and DNA methylation fidelity</strong></div><div><em>Nature.</em> 2011; <strong>473</strong>:343-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79956302047" target="_blank" aria-label="Open Scopus in new tab">Scopus (846)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21490601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10066&amp;pmid=21490601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and precise locus-specific perturbations (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Liu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Liu, X.S. ∙ Wu, H. ∙ Ji, X. ...</div><div><strong>Editing DNA methylation in the mammalian genome</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:233-247.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2016.08.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2016.08.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84988569121" target="_blank" aria-label="Open Scopus in new tab">Scopus (854)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27662091/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.056&amp;pmid=27662091" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Nuñez et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Nuñez, J.K. ∙ Chen, J. ∙ Pommier, G.C. ...</div><div><strong>Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2503-2519.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.03.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.03.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85105829180" target="_blank" aria-label="Open Scopus in new tab">Scopus (304)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33838111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.025&amp;pmid=33838111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with controlled readout (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Dixon et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Dixon, G. ∙ Pan, H. ∙ Yang, D. ...</div><div><strong>QSER1 protects DNA methylation valleys from de novo methylation</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:eabd0875</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0875" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85104166307" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33833093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0875&amp;pmid=33833093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Song, Y. ∙ van den Berg, P.R. ∙ Markoulaki, S. ...</div><div><strong>Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:905-920.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071542699" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31422875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.045&amp;pmid=31422875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Stelzer et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Stelzer, Y. ∙ Shivalila, C.S. ∙ Soldner, F. ...</div><div><strong>Tracing dynamic changes of DNA methylation at single-cell resolution</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:218-229</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2015.08.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2015.08.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84948451913" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26406378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.08.046&amp;pmid=26406378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), will help to further clarify the involvement of TET machinery in gene regulation.</div><div id="p0150" role="paragraph">The native genome-wide function of <i>Tet</i> genes as regulators of DNA demethylation across the entire genome accentuates the challenge of characterizing the function of epigenetic factors in the embryo. Indeed, alongside the TETs, other epigenetic factors ensure low methylation levels in active promoters during gastrulation, likely enabling the progression of <i>Tet</i>-deficient cells toward advanced cell lineages with only mild aberrations. Such factors can interact with essentially any gene, but extrapolating from the current findings regarding <i>Tet</i>, their impact could be quantitative and cell-type-dependent. Ultimately, combining findings regarding TET function together with other epigenetic machineries (DNMTs and Polycomb) should lead to a better understanding of how multiple lineages emerge from pluripotent and multipotent progenitors through synchronized epigenetic and transcriptional differentiation.</div><section id="sec-3-1"><h3>Limitations of the study</h3><div id="p0155" role="paragraph">Cell signaling orchestrates cell function through intra- and extra-cellular information, and it is highly heterogeneous in cell populations. We have identified multiple perturbed signaling pathways as intrinsic effects of <i>Tet</i>-TKO. However, how cell differentiation trajectories are hierarchically affected by those integrated disruptions remains to be addressed. Another key unresolved issue is how a perturbed spatial structure in mutants affects the process. Current methylation analysis is done on bulk and lacks resolution, and we cannot build the causal relationship between DNA methylation and gene expression in a cell-type-specific manner. Genetic and epigenetic alterations are widely documented consequences of cell culturing. Our mutants and control mESCs also show various imprinting abnormalities that should be considered, albeit their impact is likely to be manifested in later developmental stages. Finally, owing to&nbsp;the nature of the chimera assay, the contribution of mESCs to each mixed chimeras can vary in a wide range, posing a potential dosage effect that might lead to cell-extrinsic impacts caused by <i>Tet</i>-TKO cells in mixed chimeras.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0160" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>5-Hydroxymethylcytosine antibody (pAb)</td><td>ActiveMotif</td><td>Cat#39069; RRID: <a id="intref0025" href="https://antibodyregistry.org/AB_10013602" target="_blank">AB_10013602</a></td></tr><tr><td>5-Hydroxymethylcytosine monoclonal antibody HMC/4D9]</td><td>Epigentek</td><td>Cat#A-1018</td></tr><tr><td>Anti-TET1 antibody - C-terminal</td><td>abcam</td><td>Cat#ab191698; RRID: <a id="intref0030" href="https://antibodyregistry.org/AB_2858250" target="_blank">AB_2858250</a></td></tr><tr><td>Anti-TET2 antibody</td><td>abcam</td><td>ab124297; RRID: <a id="intref0035" href="https://antibodyregistry.org/AB_2722695" target="_blank">AB_2722695</a></td></tr><tr><td>Anti-TET3 antibody [C3], C-term</td><td>GeneTex</td><td>Cat#GTX121453; RRID: <a id="intref0040" href="https://antibodyregistry.org/AB_10723106" target="_blank">AB_10723106</a></td></tr><tr><td>Recombinant Anti-FOXA2 antibody [EPR4466]</td><td>abcam</td><td>Cat#ab108422; RRID: <a id="intref0045" href="https://antibodyregistry.org/AB_11157157" target="_blank">AB_11157157</a></td></tr><tr><td>HNF-3β/FOXA2 Antibody (M-20)</td><td>Santa Cruz Biotechnology</td><td>Cat#sc-6554; RRID: <a id="intref0050" href="https://antibodyregistry.org/AB_2262810" target="_blank">AB_2262810</a></td></tr><tr><td>Recombinant Anti-GAPDH antibody [EPR16891]</td><td>abcam</td><td>Cat#ab181602; RRID: <a id="intref0055" href="https://antibodyregistry.org/AB_2630358" target="_blank">AB_2630358</a></td></tr><tr><td>Anti-alpha Tubulin Antibody, nontyrosinated</td><td>Millipore</td><td>Cat#ABT170</td></tr><tr><td>Anti-GFP antibody</td><td>abcam</td><td>Cat#ab13970; RRID: <a id="intref0060" href="https://antibodyregistry.org/AB_300798" target="_blank">AB_300798</a></td></tr><tr><td>Anti-mCherry</td><td>SICGEN</td><td>Cat#AB0040-200; RRID: <a id="intref0065" href="https://antibodyregistry.org/AB_2333093" target="_blank">AB_2333093</a></td></tr><tr><td>Alexa Fluor 647-AffiniPure Donkey Anti-Rabbit IgG (H+L)</td><td>Jackson ImmunoResearch</td><td>Cat#711-605-152; RRID: <a id="intref0070" href="https://antibodyregistry.org/AB_2492288" target="_blank">AB_2492288</a></td></tr><tr><td>Cy2-AffiniPure Donkey Anti-Chicken IgY (IgG) (H+L)</td><td>Jackson ImmunoResearch</td><td>Cat#703-225-155; RRID: <a id="intref0075" href="https://antibodyregistry.org/AB_2340370" target="_blank">AB_2340370</a></td></tr><tr><td>Alexa Fluor® 594 AffiniPure Goat Anti-Rabbit IgG (H+L)</td><td>Jackson ImmunoResearch</td><td>Cat#111-585-144; RRID: <a id="intref0080" href="https://antibodyregistry.org/AB_2307325" target="_blank">AB_2307325</a></td></tr><tr><td>Alexa Fluor 488-AffiniPure Fab Fragment Goat Anti-Mouse IgG (H+L)</td><td>Jackson ImmunoResearch</td><td>Cat#115-547-003; RRID: <a id="intref0085" href="https://antibodyregistry.org/AB_2338869" target="_blank">AB_2338869</a></td></tr><tr><td>Donkey anti-Goat IgG-Heavy and Light Chain Antibody DyLight® 550 Conjugated</td><td>Bethyl</td><td>Cat#A50-101D3; RRID: <a id="intref0090" href="https://antibodyregistry.org/AB_10630313" target="_blank">AB_10630313</a></td></tr><tr><td>Goat anti-Rabbit IgG (H+L) Secondary Antibody, HRP</td><td>Invitrogen</td><td>Cat# 31460; RRID: <a id="intref0095" href="https://antibodyregistry.org/AB_228341" target="_blank">AB_228341</a></td></tr><tr class="rowsep"><td>Goat anti-Mouse IgG (H+L) Secondary Antibody, HRP</td><td>Invitrogen</td><td>Cat#31430; RRID: <a id="intref0100" href="https://antibodyregistry.org/AB_228307" target="_blank">AB_228307</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and virus strains</b></td></tr><tr class="rowsep"><td>AAV-Cre-GFP</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>PD0325901</td><td>Sigma</td><td>PZ0162</td></tr><tr><td>CHIR99021</td><td>Sigma</td><td>SML1046</td></tr><tr><td>RNase A</td><td>QIAGEN</td><td>19101</td></tr><tr><td>TransIT-X2 Transfection Reagent</td><td>Mirus Bio</td><td>MIR6003</td></tr><tr class="rowsep"><td>Cryonase cold-active nuclease</td><td>TaKaRa</td><td>2670A</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Pierce BCA protein assay kit</td><td>Thermo Fisher Scientific</td><td>23227</td></tr><tr><td>SuperSignal™ West Pico PLUS Chemiluminescent Substrate</td><td>Thermo Fisher Scientific</td><td>34579</td></tr><tr><td>HiScribe™ T7 High Yield RNA Synthesis Kit</td><td>New England Biolabs</td><td>E2040S</td></tr><tr><td>Quick-DNA™ Miniprep Plus Kit</td><td>Zymo Research</td><td>D4068</td></tr><tr class="rowsep"><td>EZ DNA Methylation-Direct Kit</td><td>Zymo Research</td><td>D5020</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>scRNA-seq and PBAT</td><td>This paper</td><td>GEO: <a id="intref0105" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205917" target="_blank">GSE205917</a></td></tr><tr class="rowsep"><td>Original western blot images</td><td>This paper</td><td>Mendeley Data: <a id="interref0010" href="https://doi.org/10.17632/7j64ch46gz.1" target="_blank">https://doi.org/10.17632/7j64ch46gz.1</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Cell lines</b></td></tr><tr><td>Mouse: V6.5 mESCs</td><td>Jaenisch lab, MIT</td><td>RRID:CVCL_C865</td></tr><tr><td>Mouse: <i>Tet</i>-TKO1 mESCs</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Dawlaty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Loss of tet enzymes compromises proper differentiation of embryonic stem cells</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:102-111</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84898482006" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24735881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.003&amp;pmid=24735881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Mouse: <i>Tet</i>-TKO2 mESCs</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Dawlaty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Loss of tet enzymes compromises proper differentiation of embryonic stem cells</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:102-111</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84898482006" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24735881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.003&amp;pmid=24735881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Mouse: <i>Tet</i>-TKO3 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: <i>Tet</i>-TKO4 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: Ctrl1 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: Ctrl2 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: Ctrl3 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: DKO12#10 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: DKO12#25 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: DKO13#3 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: DKO23#5 mESCs</td><td>This paper</td><td>N/A</td></tr><tr><td>Mouse: <i>Tet3</i> KO#7 mESCs</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td>Human: HEK293T cells</td><td>ATCC</td><td>RRID:CRL-3216</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>Mouse: C57BL/6JRccHsd</td><td>Envigo</td><td>RRID:<br>MGI:<a id="intref0110" href="https://www.informatics.jax.org/accession/MGI:6151402" target="_blank">6151402</a></td></tr><tr><td>Mouse: Hsd:ICR(CD-1)</td><td>Envigo</td><td>RRID:<br>MGI:<a id="intref0115" href="https://www.informatics.jax.org/accession/MGI:5649797" target="_blank">5649797</a></td></tr><tr><td>Mouse: B6D2F1</td><td>Envigo</td><td>RRID:<br>MGI:<a id="intref0120" href="https://www.informatics.jax.org/accession/MGI:5651959" target="_blank">5651959</a></td></tr><tr class="rowsep"><td>Mouse: Tet1/2/3 triple floxed</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Zviran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Zviran, A. ∙ Mor, N. ∙ Rais, Y. ...</div><div><strong>Deterministic somatic cell reprogramming involves continuous transcriptional changes governed by Myc and epigenetic-driven modules</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:328-341.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2018.11.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2018.11.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85060996758" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30554962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2018.11.014&amp;pmid=30554962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td><i>Tet1</i> gRNA:<br>ACCACGTCTACTGCAGTCCA</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Song, Y. ∙ van den Berg, P.R. ∙ Markoulaki, S. ...</div><div><strong>Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:905-920.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071542699" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31422875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.045&amp;pmid=31422875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td><i>Tet2</i> gRNA:<br>TTCTGAAAAGGACAGGTACC</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Song, Y. ∙ van den Berg, P.R. ∙ Markoulaki, S. ...</div><div><strong>Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:905-920.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071542699" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31422875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.045&amp;pmid=31422875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td><i>Tet3</i> gRNA:<br>AAAAGCGGGGCTTCTTAAGG</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-4" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-4" aria-controls="sref62-4">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Song, Y. ∙ van den Berg, P.R. ∙ Markoulaki, S. ...</div><div><strong>Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:905-920.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071542699" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31422875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.045&amp;pmid=31422875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td><i>Noto in&nbsp;situ</i> forward primer<br>CCTGATTCACAAGTCTCCCGT</td><td>This paper</td><td>N/A</td></tr><tr><td><i>Noto in&nbsp;situ</i> reverse primer<br>GAAATTAATACGACTCACTAT<br>AGGGCCCATTTCAGAGGGCCTAGAC</td><td>This paper</td><td>N/A</td></tr><tr><td><i>Cdx1 in&nbsp;situ</i> forward primer<br>GGCCACCCCTAGGAAGTCTA</td><td>This paper</td><td>N/A</td></tr><tr><td><i>Cdx1 in&nbsp;situ</i> reverse primer<br>GAAATTAATACGACTCACTATA<br>GGGTGCTAGGGCAGGTGAAAGTG</td><td>This paper</td><td>N/A</td></tr><tr><td><i>Twist1 in&nbsp;situ</i> forward primer<br>AGGCCGGAGACCTAGATGTC</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td><i>Twist1 in&nbsp;situ</i> reverse primer<br>GAAATTAATACGACTCACTATAGGG<br>TGAATGCATTTAGACACCGGA</td><td>This paper</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>Cas9 targeting plasmid: px330</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">Wu et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Wu, Y. ∙ Liang, D. ∙ Wang, Y. ...</div><div><strong>Correction of a genetic disease in mouse via use of CRISPR-Cas9</strong></div><div><em>Cell Stem Cell.</em> 2013; <strong>13</strong>:659-662</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.10.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.10.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84890050551" target="_blank" aria-label="Open Scopus in new tab">Scopus (494)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24315440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.10.016&amp;pmid=24315440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid: #98750</td></tr><tr><td>HTNC expression plasmid: pTriEx-HTNC</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Peitz et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Peitz, M. ∙ Pfannkuche, K. ∙ Rajewsky, K. ...</div><div><strong>Ability of the hydrophobic FGF and basic TAT peptides to promote cellular uptake of recombinant Cre recombinase: A tool for efficient genetic engineering of mammalian genomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:4489-4494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.032068699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0037007121" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11904364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.032068699&amp;pmid=11904364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid:<br>#13763</td></tr><tr><td>Ctrl3 florescent labeling plasmid: loxp-dsRed-loxp-EGFP</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Koo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Koo, B.-K. ∙ Stange, D.E. ∙ Sato, T. ...</div><div><strong>Controlled gene expression in primary Lgr5 organoid cultures</strong></div><div><em>Nat. Methods.</em> 2011; <strong>9</strong>:81-83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1802" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84856490464" target="_blank" aria-label="Open Scopus in new tab">Scopus (249)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22138822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1802&amp;pmid=22138822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid:<br>#32702</td></tr><tr><td>TKO1 and TKO2 florescent labeling plasmid: pBRY-nuclear mCherry-IRES-PURO</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Rais et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Rais, Y. ∙ Zviran, A. ∙ Geula, S. ...</div><div><strong>Deterministic direct reprogramming of somatic cells to pluripotency</strong></div><div><em>Nature.</em> 2013; <strong>502</strong>:65-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12587" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84885619736" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24048479/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12587&amp;pmid=24048479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid:<br>#52409</td></tr><tr><td>AAV helper plasmid: pAdDeltaF6</td><td>Addgene</td><td>Addgene plasmid<br>#112867</td></tr><tr><td>AAV packing plasmid expressing Rep/Cap genes: pAAV2/1</td><td>Addgene</td><td>Addgene plasmid:<br>#112862</td></tr><tr class="rowsep"><td>AAV encoding GFP/Cre recombinase fusion protein: AAV-GFP/Cre</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Kaspar et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Kaspar, B.K. ∙ Vissel, B. ∙ Bengoechea, T. ...</div><div><strong>Adeno-associated virus effectively mediates conditional gene modification in the brain</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:2320-2325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.042678699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0037133198" target="_blank" aria-label="Open Scopus in new tab">Scopus (164)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11842206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.042678699&amp;pmid=11842206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>Addgene plasmid:<br>#49056</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Metacell</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Baran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Baran, Y. ∙ Bercovich, A. ∙ Sebe-Pedros, A. ...</div><div><strong>MetaCell: analysis of single-cell RNA-seq data using K-nn graph partitions</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1812-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85073149395" target="_blank" aria-label="Open Scopus in new tab">Scopus (167)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31604482/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1812-2&amp;pmid=31604482" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PMID: <a id="intref0125" href="https://pubmed.ncbi.nlm.nih.gov/31604482/" target="_blank">https://pubmed.ncbi.nlm.nih.gov/31604482/</a></td></tr><tr><td>MARS-seq pipe mapping/UMI pipeline</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Keren-Shaul et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Keren-Shaul, H. ∙ Kenigsberg, E. ∙ Jaitin, D.A. ...</div><div><strong>MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:1841-1862</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0164-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85066030980" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31101904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0164-4&amp;pmid=31101904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>PMID: <a id="intref0130" href="https://pubmed.ncbi.nlm.nih.gov/31101904/" target="_blank">https://pubmed.ncbi.nlm.nih.gov/31101904/</a></td></tr><tr><td>Original code</td><td>This paper</td><td><a id="intref0135" href="https://tanaylab.github.io/tet-gastrulation/" target="_blank">https://tanaylab.github.io/tet-gastrulation/</a>; <a id="interref0020" href="https://doi.org/10.5281/zenodo.6720248" target="_blank">https://doi.org/10.5281/zenodo.6720248</a></td></tr><tr><td>ImageJ</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Schneider et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (43944)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a></td></tr><tr><td>Prism</td><td>GraphPad</td><td><a id="intref0145" href="https://www.graphpad.com/" target="_blank">https://www.graphpad.com/</a></td></tr><tr><td>FlowJo</td><td>FlowJo</td><td><a id="intref0150" href="https://www.flowjo.com" target="_blank">https://www.flowjo.com</a></td></tr><tr><td>Bowtie2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (33760)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td>Bissli2</td><td><a id="intref0160" href="https://github.com/tanaylab/bissli2" target="_blank">https://github.com/tanaylab/bissli2</a></td><td><a id="intref0165" href="https://github.com/tanaylab/bissli2" target="_blank">https://github.com/tanaylab/bissli2</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900842-X" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0165" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact: Yonatan Stelzer (<a id="intref0170" href="mailto:yonatan.stelzer@weizmann.ac.il">yonatan.stelzer@weizmann.ac.il</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0170" role="paragraph">Plasmids and cell lines generated in this study will be made available by the <a id="crosref1085" href="#sec-4-2-1">lead contact</a> upon request.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Culture of mESCs</h4><div id="p0175" role="paragraph">All mouse embryonic stem cells (mESCs) were cultured on irradiated mouse embryonic fibroblasts (MEFs) with standard medium: 500&nbsp;ml DMEM (gibco, 41965-039), 20% fetal bovine serum (Biological Industries, 04-001-1A), 10&nbsp;mg recombinant leukemia inhibitory factor (LIF, homemade), 0.1&nbsp;mM beta-mercaptoethanol (gibco, 31350-010), penicillin/streptomycin (Biological Industries, 03-031-1B), 1&nbsp;mM L-glutamine (Biological Industries, 03-020-1B), and 1% nonessential amino acids (Biological Industries, 01-340-1B). For chimera assays, mESCs were in standard medium for 2-3&nbsp;days and replaced with 2i+LIF medium containing additional 1&nbsp;mM PD0325901 (Sigma, PZ0162) and 3&nbsp;mM CHIR99021 (Sigma, SML1046) for another growth of 10-12 h prior to injection.</div></section><section id="sec-4-3-2"><h4>Generation of knockout mESCs</h4><div id="p0180" role="paragraph">Control cell lines were generated by targeting to the <i>Hipp11</i> locus with core chicken β-actin promoter driving EGFP (Ctrl1 and Ctrl2) or loxp-dsRed-loxp-EGFP cassette (Ctrl3, plasmid from Addgene, #32702) in V6.5 mESCs (Jaenisch lab, CVCL_C865). Cre recombinase treatment will excise dsRed and turn on the expression of EGFP for Ctrl3 cell line. Trial blastocyst injections were performed using control cell lines for robust contribution of chimerism.</div><div id="p0185" role="paragraph">On top of Ctrl3 cell line, CRISPR/Cas9-mediated genome editing was adopted to generate <i>Tet</i> KO mESCs. sgRNA sequences (<span class="dropBlock reference-citations"><a id="body-ref-sref62-5" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-5" aria-controls="sref62-5">Song et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref62-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Song, Y. ∙ van den Berg, P.R. ∙ Markoulaki, S. ...</div><div><strong>Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:905-920.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071542699" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31422875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.045&amp;pmid=31422875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for each <i>Tet</i> are cloned into px330 under U6 promoter. px330-BFP-sgRNA vectors were transfected into pre-plated Ctrl3 cells using TransIT-X2 Transfection Reagent (Mirus Bio, MIR6003) according to the provider’s instructions. Cells were sorted for BFP 48 h post-transfection. Single clones were picked up for further validation. Two and three sgRNAs were co-transfected to get <i>Tet</i> double and triple KO mESCs, separately. Western blot, dot blot, and next generation sequencing (NGS) methods were used to validate KO clones. <i>Tet</i>-TKO clone (TKO3), <i>Tet3</i> KO clone (<i>Tet3</i> KO#7), and <i>Tet</i>-DKO clones (DKO1/2#10, DKO1/2#25, DKO1/3#3, and DKO2/3#5) were generated through this system.</div><div id="p0190" role="paragraph">TKO1 and TKO2 have been described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref9-3" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-3" aria-controls="sref9-3">Dawlaty et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref9-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Loss of tet enzymes compromises proper differentiation of embryonic stem cells</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:102-111</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84898482006" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24735881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.003&amp;pmid=24735881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<i>Tet</i>-TKO1 refers to TKO#26, and <i>Tet</i>-TKO2 refers to TKO#29). For stable labeling, plasmid pBRY-nuclear mCherry-IRES-PURO (Addgene, #52409) linearized through <i>ScaI</i> was electroplated into mESCs under the following settings: voltage: 500, capacitance: 25, Resistance: ∞, and cuvette: 4 (BioRad, Gene Pulser Xcell). Puromycin was added 24 h post-electroporation. Clones survived from puromycin selection were picked up for another few passages in mECS medium, and further differentiated into embryoid bodies (EBs) to check the salability of reporter signal using flow cell cytometry.</div><div id="p0195" role="paragraph"><i>Tet</i>-TKO4 was derived from mouse blastocysts. Specially, Tet1/2/3 floxed mice (<span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Zviran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Zviran, A. ∙ Mor, N. ∙ Rais, Y. ...</div><div><strong>Deterministic somatic cell reprogramming involves continuous transcriptional changes governed by Myc and epigenetic-driven modules</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:328-341.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2018.11.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2018.11.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85060996758" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30554962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2018.11.014&amp;pmid=30554962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were crossed to derive Tet1/2/3 floxed mESC clones from inner cell mass. Male clone was chosen for Cre recombinase protein treatment to take out the floxed alleles. After validation of the deleted allele for all <i>Tet</i>&nbsp;alleles by PCR, the clone was further labeled with mCherry by electroporation as described above.</div></section><section id="sec-4-3-3"><h4>Chimera and Tetraploid complementary Assay</h4><div id="p0200" role="paragraph">All animal procedures were approved by the Institutional Animal Care and Use Committee and were performed in strict adherence to Weizmann Institute guidelines. Mice were monitored for health and activity and were given ad libitum access to water and standard mouse chow with 12 h light/dark cycles.</div><div id="p0205" role="paragraph">To generate chimera embryos, a mix of <i>Tet</i> KO and/or control mESCs were injected into diploid or tetraploid B6D2F x B6D2F1 E3.5 blastocysts and surgically implanted into E2.5 postcoitum pseudo-pregnant Hsd:ICR (CD-1) females following standard procedures. Embryos were harvested and dissected in ice-cold 1×PBS, followed by imaging in a drop of DMEM medium supplemented with 10% foetal bovine serum under a Nikon Eclipse Ti2 inverted microscope. Embryos positive for both <i>Tet</i> KO and control mESCs or <i>Tet</i> KO single positive were chosen for single-cell suspension preparation and kept in sorting buffer (1×PBS with 0.5% BSA) on ice until flow cell cytometry analyses. Mixed chimeric embryos with either too high or two low (sparse) contribution of injected cells (based on the florescent signal) were excluded from index-sorting, given that they won’t provide enough number of control/host or knockout cells for single embryo resolution.</div></section><section id="sec-4-3-4"><h4>AAV-mediated Cre recombinase delivery to <i>Tet</i> triple floxed zygotes</h4><div id="p0210" role="paragraph">Zygotes from Tet1/2/3 floxed mice were treated by acidic Tryode’s solution (Merck, T1788-100ML), rinsed in M2 medium and transferred to potassium simplex optimized medium (KSOM) for incubation in a tissue culture incubator containing 5% CO<sub>2</sub> and 5% O<sub>2</sub>. Drops containing 35&nbsp;μl KSOM medium were placed in 35&nbsp;mm plates under mineral oil. 6e9 vg AAV-Cre-GFP viral were added to each drop, followed by a 15&nbsp;min incubation at 37 °C in the same tissue culture incubator. Subsequently, zygotes were transferred into the viral drop and cultured for 24&nbsp;h. Qualified 2-cell embryos were rinsed in M2 medium and transferred into the oviduct of E0.5 pseudo-pregnant ICR females. Recipient females were allowed to carry the embryos and sacrificed at E7.5. After imaging of each embryo, DNA from extraembryonic parts (mostly trophoblast cells, ectoplacental cone, and parietal yolk sac. Note that those parts have maternal blood contamination and will yield WT alleles when genotyping) were extracted for validation of Cre recombinase activity. The corresponding embryonic part was individually dissociated into single cells and subjected to FACS sorting into pre-barcoded 384-well plates. PCR verified embryos were chosen for further scRNA-seq analysis.</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Dot blot for 5hmC level detection</h4><div id="p0215" role="paragraph">To screen for potential knockout clones after CRISPR/Cas9-mediated gene editing, genomic DNA was extracted from each single clone after 5-6 passages post-transfection. A total of 50-100&nbsp;ng genomic DNA was blotted onto nitrocellulose membranes with Bio-Dot microfiltration apparatus. Afterward, the membrane was air-dried and cross-linked on each side with a UV-agarose gel box. 5% non-fat milk in PBST was used to block the membrane, followed by primary and secondary antibody incubation. Antibodies used for this assay: anti-5hmC (ActiveMotif, 39069; Epigentek, A-1018), Goat anti-Rabbit IgG (H+L), HRP (Invitrogen, 31460), Goat anti-Mouse IgG (H+L), HRP (Invitrogen, 31430). Signal was developed with SuperSignal™ West Pico PLUS Chemiluminescent Substrate (Thermo, 34579), and images were taken using ChemiDoc™ MP Imaging System (BioRad).</div></section><section id="sec-4-4-2"><h4>Western blot for TET protein detection</h4><div id="p0220" role="paragraph">For detection of TET1 and TET2, mESCs cultured in standard mESCs medium were harvested after MEF depletion. For TET3, 8-10&nbsp;day EBs were collected after differentiation in mESCs medium (supplemented with 20% serum, without 2i and LiF) in a petri dish. All the pellets were lysis in lysis buffer (150&nbsp;mM sodium chloride, 1% triton x-100, 50&nbsp;mM Tris HCl pH8.0 with freshly added proteinase inhibitor) for 30&nbsp;min on ice. The supernatant was reserved for total protein quantification using Pierce BCA protein assay kit (Thermo Fisher Scientific, 23227). A total of 10-20&nbsp;μg protein were loaded for SDS-PAGE analysis. Primary antibodies used in the assay: anti-Tet1 (Abcam, ab191698), anti-Tet2 (Abcam, ab124297), anti-Tet3 (GeneTex, GTX121453), anti-GAPDH (Abcam, ab181602). anti-alpha Tubulin (Millipore, ABT170). Secondary antibodies used: Goat anti-Rabbit IgG (H+L), HRP (Invitrogen, 31460), and Goat anti-Mouse IgG (H+L), HRP (Invitrogen, 31430). Signal was developed with SuperSignal™ West Pico PLUS Chemiluminescent Substrate (Thermo, 34580), and images were taken using ChemiDoc™ MP Imaging System (BioRad).</div></section><section id="sec-4-4-3"><h4>NGS-based validation of CRISPR/Cas9-mediated gene editing</h4><div id="p0225" role="paragraph">For further validation of <i>Tet</i> knockout mESCs at DNA sequence level, clones with reduced 5hmC levels from dot-blot assay were chosen as candidates. Genomic DNA was extracted for each clone, and primers flanking the PAM sequence of each target gene were used for the first round PCR amplification, yielding a size around ∼100bp. The sequences of the primers were F: 5’-ACACTCTTTCCCTACACGACGCTCTTCCGATCT-gene specific forward primer-3’, R: 5’-GTGACTGGAGTTCAGACGTGT GCTCTTCCGATCT-gene specific reverse primer-3’. Illumina i5 and i7 index primer pairs were chosen in the second round of PCR, which will confer each amplicon with a unique index. Indexed amplicons were pooled, purified, and sequenced under Illumina NextSeq 500 or NovaSeq 6000 platform. Reads were demultiplexed with dual barcodes and mapped against targeted genes.</div></section><section id="sec-4-4-4"><h4>Whole-mount Immunofluorescence</h4><div id="p0230" role="paragraph">To detect control and <i>Tet</i>-TKO mESCs contribution to node/notochord, headfold to 2-4 somite stage mixed chimera embryos were harvested and fixed with 4% EM grade paraformaldehyde in PBS at 4 °C overnight. The staining procedure was performed as described (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Aguilera-Castrejon et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aguilera-Castrejon, A. ∙ Oldak, B. ∙ Shani, T. ...</div><div><strong>Ex utero mouse embryogenesis from pre-gastrulation to late organogenesis</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:119-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03416-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85102676173" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33731940/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03416-3&amp;pmid=33731940" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To detect 5hmC level, embryos were treated in 0.1&nbsp;μg/μl RNase A (QIAGEN, 19101) in PBS at 37 °C for 30&nbsp;min after permeabilization. Afterward, embryos were washed in PBS for 5&nbsp;min three times, incubated in 4N HCl for 15&nbsp;min at room temperature, followed by 10&nbsp;min neutralization in 100&nbsp;mM Tris pH 8.0. Then, embryos were rinsed three times for 5&nbsp;min each in PBS, blocked in 3% BSA for 1 h, and stained for primary and secondary antibodies. Primary antibodies used: anti-FOXA2 (Abcam, ab108422; Santa Cruz Biotechnology, sc-6554), anti-GFP (Abacm, ab13970), anti-mCherry (SICGEN, AB0040-200), anti-5hmC (Epigentek, A-1018). Secondary antibodies used: Donkey Anti-Rabbit IgG (IgG) (H+L), Alex 647 (Jackson ImmunoResearch, 711-605-152), Donkey Anti-Chicken Cy2 (Jackson ImmunoResearch, 703-225-155), Donkey anti-Goat DyLight 550 Conjugated (Bethyl, A50-101D3) for images from <a id="crosref1110" href="#fig6">Figure&nbsp;6</a>B. Goat Anti-Rabbit Alexa 594 (Jackson, 111-585-144), Goat Anti-Mouse Alex 488 (Jackson, 115-547-003) for images from <a id="crosref1115" href="#fig6">Figure&nbsp;6</a>F.</div></section><section id="sec-4-4-5"><h4>Confocal microscopy and 5hmC quantification</h4><div id="p0235" role="paragraph">After whole-mount immunofluorescence, embryos were unfolded with forceps from the lateral side, placed onto a pre-treated glass slide with their ventral side (node/notochord) facing up, mounted with a drop of mounting medium (Thermo Scientific Shandon, 9990402), then sealed with a coverslip (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Xiao et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Xiao, C. ∙ Nitsche, F. ∙ Bazzi, H.</div><div><strong>Visualizing the node and notochordal plate in gastrulating mouse embryos using scanning electron microscopy and whole mount immunofluorescence</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2018; <strong>141</strong>:e58321</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3791/58321" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85057071924" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3791%2F58321" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Z-stack images were acquired with a Zeiss LSM 880 inverted confocal microscope (Carl Zeiss), using a Plan-Apochromat 10X/0.45 objective for node/notochord 5hmC imaging for embryo E4 (embryo from <a id="crosref1125" href="#fig6">Figure&nbsp;6</a>G), a Plan-Apochromat 20X/0.8 objective for embryos in <a id="crosref1130" href="#fig6">Figure&nbsp;6</a>B and embryo E1, E2, E3, and E5 in <a id="crosref1135" href="#fig6">Figure&nbsp;6</a>G. Z-stacks with a thickness that ranged approximately 15-22&nbsp;μm made up of 16-25 images were collected. To visualize the contribution of control and <i>Tet</i>-TKO mESCs into node/notochord in mixed chimera embryos, 3D-projection of all optical sections was performed for each channel using ImageJ software. To quantify 5hmC intensity, a single optical section of z-stacks image that covers node/notochord (FOXA2<sup>+</sup> horseshoe-shaped morphology) and its lateral proximal mesoderm territory (FOXA2<sup>-</sup> and DAPI<sup>+</sup>) were chosen for each embryo to measure 5hmC and DAPI signal. Normalized signal (5hmC/DAPI) was used as an indication of the 5hmC level.</div></section><section id="sec-4-4-6"><h4>Whole-mount RNA <i>in situ</i> hybridization</h4><div id="p0240" role="paragraph">Whole-mount <i>in situ</i> hybridization was performed as described (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Piette et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Piette, D. ∙ Hendrickx, M. ∙ Willems, E. ...</div><div><strong>An optimized procedure for whole-mount in&nbsp;situ hybridization on mouse embryos and embryoid bodies</strong></div><div><em>Nat. Protoc.</em> 2008; <strong>3</strong>:1194-1201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2008.103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-50249113849" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18600225/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2008.103&amp;pmid=18600225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For validation of missing cell types in <i>Tet</i>-TKO 4N chimera embryos, <i>Noto</i>, <i>Cdx1</i>, and <i>Twist1</i> were chosen as probes. A PCR-based method for RNA probes was used for labeling those probes with digoxigenin (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Hua et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Hua, R. ∙ Yu, S. ∙ Liu, M. ...</div><div><strong>A PCR-based method for RNA probes and applications in neuroscience</strong></div><div><em>Front. Neurosci.</em> 2018; <strong>12</strong>:266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fnins.2018.00266" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85046622006" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29770110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffnins.2018.00266&amp;pmid=29770110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Primers used for each probe are: <i>Noto</i>-F: 5’-CCTGATTCACAAGTCTCCCGT-3’, <i>Noto</i>-R: 5’-GAAATTAATACGACTCACTATAGGGCCCATTTCAGAGGGCCTAGAC-3’; <i>Cdx1</i>-F: 5’-GGCCAC CCCTAGGAAGTCTA-3’; <i>Cdx1</i>-R: 5’-GAAATTAATACGACTCACTATAGGGTGCTAGGGCAGGTGAAAGTG-3’; <i>Twist1</i>-F: 5’-AGGCCGGAGACCTAGATGTC-3’; <i>Twist1</i>-R: 5’-GAAATTAATACGACTCACTATAGGGTGAATGCATTTAGACACCGGA-3’. Anti-sense mRNA probe was transcribed with HiScribe™ T7 High Yield RNA Synthesis Kit (New England Biolabs, E2040S) <i>in&nbsp;vitro</i>, and purified with 4M LiCl precipitation.</div></section><section id="sec-4-4-7"><h4>Small scale AAV-Cre-GFP viral vector production</h4><div id="p0250" role="paragraph">One well of a 6-well tissue culture plate, containing 2X10<sup>5</sup> human embryonic kidney (HEK) 293T cells (ATCC; RRID:CRL-3216) was transfected using homemade polyethylenimine with following reaction: 1200&nbsp;ng pAdDeltaF6 (A gift from James M, Wilson, Addgene, #112867), 400&nbsp;ng pAAV2/1 plasmid (A gift from James M. Wilson, Addgene, #112862), and 400&nbsp;ng AAV-GFP/Cre plasmid (Addgene, #49056) in a total of 25&nbsp;μl Opti-MEM™ (Gibco, 31985-070), with additional 8&nbsp;μl mg/ml polyethylenimine solution. 72&nbsp;h after transfection, supernatant was collected into a 1.5&nbsp;ml Eppendorf tube and applied to cryonase cold-active nuclease (TaKaRa, 2670A, 1:1000 ratio) treatment at 37°C for 1 h. Cell debris was spun down at 2000&nbsp;g for 5 min at 15°C, and the supernatant was purified and concentrated to 100&nbsp;μl using the Amicon Ultra-0.5 Centrifugal Filter Unit (Merck, UFC510008). The titer of AAV-Cre-GFP was titrated by counting the genomes of GFP with real-time PCR. The titer was found to be 5e12 vg/ml.</div></section><section id="sec-4-4-8"><h4>Massively Parallel Single-Cell RNA-seq (MARS-seq) library preparation</h4><div id="p0255" role="paragraph">Single-cell cDNA libraries were prepared using the MARS-Seq method as described (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Keren-Shaul et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Keren-Shaul, H. ∙ Kenigsberg, E. ∙ Jaitin, D.A. ...</div><div><strong>MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:1841-1862</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0164-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85066030980" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31101904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0164-4&amp;pmid=31101904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, single cells were sorted into 384-well cell capture plates containing 2&nbsp;μl of lysis solution and barcoded poly (T)&nbsp;reverse-transcription (RT) primers. 0.5&nbsp;μl RT buffer was added to each well by MANTIS microfluidic liquid handler (FORMULATRIX) to get complementary DNA (cDNA), followed by 1&nbsp;μl Exonuclease I mixture treatment without heat inactivation. Afterward, all liquid from each 384-well plate were pooled into VBLOCK200 reservoir (Clickbio) by centrifuge, thus denoted as one library. The library was converted to double-stranded DNA, transcribed using T7 RNA polymerase, and fragmented. Following this, a unique plate adaptor was added to each library to be sequenced together, and RNA::ssDNA (plate adaptor) ligation was performed. The purified ligates were applied to a second RT step with a primer complementary to the ligated adaptor using AffinityScripts RT enzyme, and the library is completed by cDNA enrichment through PCR amplification. Library concentration and profile were evaluated before sequencing.</div></section><section id="sec-4-4-9"><h4>Whole genome bisulfate sequencing by post-bisulfite adaptor tagging (PBAT)</h4><div id="p0260" role="paragraph">For bisulfite sequencing, <i>Tet</i>-TKO1, <i>Tet</i>-TKO2, Ctrl1, and Ctrl2 derived cells were FACS-sorted from E8.5 chimera embryos respectively. Genomic DNA was extracted using the Quick-DNA™ Miniprep Plus Kit (Zymo Research, D4068), and library preparation was performed with the protocol as in <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Meir et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Meir, Z. ∙ Mukamel, Z. ∙ Chomsky, E. ...</div><div><strong>Single-cell analysis of clonal maintenance of transcriptional and epigenetic states in cancer cells</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:709-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0645-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85087038708" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32601473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0645-y&amp;pmid=32601473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Briefly, genomic DNA was treated with EZ DNA Methylation-Direct Kit (Zymo Research, D5020) according to the manufacturer’s instruction. The concerted DNA was applied to the first strand synthesis using PBAT oligo_1 with the following sequence format: 5’-ACACTCTTTCCCTACACGACGCTCTTCCGATCT-8bp index-9bp random sequence-3’, followed by Klenow fragment treatment and heat inactivation. Afterward, the DNA was treated by Exonuclease I and shrimp alkaline phosphatase. Second strand synthesis was performed using PBAT oligo_2 with the following sequence format: 5’-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT-8bp index-9bp random sequence-3’. The product was again treated by Klenow fragment and purified. A final PCR was performed using a universal primer F: 5’-AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT-3’, and indexed primers (R)&nbsp;with the following format: 5’-CAAGCAGAAGACGGCATACGAGAT-6&nbsp;bp index- GTGACTGGAGTTCAGACGTGTG-3’. The final product was clean and sequenced using NovaSeq 6000.</div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><section id="sec-4-5-1"><h4>MARS-seq data processing</h4><div id="p0265" role="paragraph">MARS-seq libraries were sequenced using NextSeq 500 or NovaSeq 6000. Reads were processed following the MARS-seq2.0 protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref28-3" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-3" aria-controls="sref28-3">Keren-Shaul et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref28-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Keren-Shaul, H. ∙ Kenigsberg, E. ∙ Jaitin, D.A. ...</div><div><strong>MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:1841-1862</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0164-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85066030980" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31101904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0164-4&amp;pmid=31101904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, reads from Read 1 were mapped to the mm9 genome using bowtie2. Demultiplexing of the reads and construction of the single-cell UMI matrix was based on the well barcodes (7 base pairs) and UMI barcodes (8 base pairs) from Read 2. Registration of embryo identity per well during sorting allowed to map each single cell to the embryo it came from. For chimera embryos, GFP and mCherry intensity were extracted for each cell, using indexed sorting files, and a threshold was set to distinguish host and mESCs-derived cells. The threshold was further validated by comparing it to the GFP/mCherry intensity of all annotated extraembryonic ectoderm and extraembryonic endoderm cells (that are purely host-derived). Overall, we processed 79,872 wells.</div></section><section id="sec-4-5-2"><h4>Metacell (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Baran et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Baran, Y. ∙ Bercovich, A. ∙ Sebe-Pedros, A. ...</div><div><strong>MetaCell: analysis of single-cell RNA-seq data using K-nn graph partitions</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1812-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85073149395" target="_blank" aria-label="Open Scopus in new tab">Scopus (167)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31604482/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1812-2&amp;pmid=31604482" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) analysis of MARS-seq data</h4><div id="p0270" role="paragraph">Mitochondrial transcripts were removed from all cells in the UMI matrix. We only retained cells with at least 2000&nbsp;UMIs and not more than 12000&nbsp;UMIs for further analysis, resulting in 57,356 filtered cells in total. After removing extraembryonic ectoderm and endoderm cells and combining the dataset with the WT gastrulation atlas from <span class="dropBlock reference-citations"><a id="body-ref-sref41-5" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-5" aria-controls="sref41-5">Mittnenzweig et&nbsp;al. (2021)</a><div class="dropBlock__holder" data-db-target-of="sref41-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, we generated an initial list of feature genes by filtering genes passing a standard threshold on minimal coverage and on variance over mean across the whole dataset (T_vm&nbsp;= 0.1). We clustered these genes into 80 gene modules and manually selected modules that were enriched for cell-cycle, ribosomal or stress-associated genes. This resulted in a list of 173 genes which were excluded for constructing balanced single-cell similarity graphs (Knn&nbsp;= 100), but retained otherwise. Absolute gene expression <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="e Subscript g comma m" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="e Subscript g comma m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="4" data-semantic-children="1,2,3" data-semantic-content="2" data-semantic-parent="5" data-semantic-owns="1 2 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for metacell <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="m"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and gene <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="g"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is defined as the relative frequency of UMIs from that gene normalized by the total number of UMIs per metacell. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l e Subscript g comma m Baseline equals log Subscript 2 Baseline left parenthesis e Subscript g comma m Baseline plus 10 Superscript negative 5 Baseline right parenthesis" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(32 (31 0 30 (6 1 (5 2 3 4))) 7 (29 (11 8 10) 28 (27 12 (25 (18 13 (17 14 15 16)) 19 (24 20 (23 21 22))) 26)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="32" data-semantic-children="31,29" data-semantic-content="7" data-semantic-owns="31 7 29" data-semantic-speech="l e Subscript g comma m Baseline equals log Subscript 2 Baseline left parenthesis e Subscript g comma m Baseline plus 10 Superscript negative 5 Baseline right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="31" data-semantic-children="0,6" data-semantic-content="30" data-semantic-parent="32" data-semantic-owns="0 30 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="31"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="1,5" data-semantic-parent="31" data-semantic-owns="1 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="5" data-semantic-children="2,3,4" data-semantic-content="3" data-semantic-parent="6" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="32" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="29" data-semantic-children="11,27" data-semantic-content="28,8" data-semantic-parent="32" data-semantic-owns="11 28 27"><mjx-msub data-semantic-type="subscript" data-semantic-role="prefix function" data-semantic-id="11" data-semantic-children="8,10" data-semantic-parent="29" data-semantic-owns="8 10"><mjx-mrow><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="8" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow><mjx-script style="vertical-align: -0.241em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="27" data-semantic-children="25" data-semantic-content="12,26" data-semantic-parent="29" data-semantic-owns="12 25 26"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="27" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="25" data-semantic-children="18,24" data-semantic-content="19" data-semantic-parent="27" data-semantic-owns="18 19 24"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="13,17" data-semantic-parent="25" data-semantic-owns="13 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="18"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="17" data-semantic-children="14,15,16" data-semantic-content="15" data-semantic-parent="18" data-semantic-owns="14 15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="17"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="15" data-semantic-parent="17" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="19" data-semantic-parent="25" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="24" data-semantic-children="20,23" data-semantic-parent="25" data-semantic-owns="20 23"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="24"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-children="22" data-semantic-content="21" data-semantic-parent="24" data-semantic-owns="21 22"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="21" data-semantic-parent="23" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="26" data-semantic-parent="27" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> denotes the regularized, logarithmic, absolute expression.</div></section><section id="sec-4-5-3"><h4>Cell type annotation of cells in whole-embryo or mixed-chimera</h4><div id="p0275" role="paragraph">Single cells from chimera embryos were annotated with cell types from the WT atlas as previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref41-6" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-6" aria-controls="sref41-6">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): For each genotype (TKO/DKO/Ctrl) and experimental condition (whole-embryo or mixed-chimera), we constructed a common metacell balanced K-nn graph consisting of 33.889 cells from the wildtype atlas (<i>atlas cells</i>) and the corresponding number of unannotated query cells. We computed for each query cell the empirical distribution of WT cell types among its 50 nearest neighbors. We repeated this computation also for the atlas cells, rationalizing that when cell types show uneven frequencies in the atlas, one should control for annotation drift toward the more common types. We therefore annotated each query cell not with the cell type that represents the majority of its atlas neighbors, but with the cell type of the atlas cells that maximized correlation with the nearest neighbors cell type distribution. Naturally in most cases the two definitions coincided.</div></section><section id="sec-4-5-4"><h4>Transcriptional time of mixed and whole-body chimera embryos</h4><div id="p0280" role="paragraph">For each chimera embryo, we inferred a best-matching WT transcriptional time in the way as was done previously for the Foxc1/2 DKO chimera embryos (<span class="dropBlock reference-citations"><a id="body-ref-sref41-7" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-7" aria-controls="sref41-7">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, we construct a common single-cell balanced KNN graph consisting of cells from the 153 WT embryos and cells from the given group of chimeras (containing either injected <i>Tet</i>-TKO, -DKO or control cells). Each non-WT cell is labeled with the transcriptional rank of its nearest WT neighbor (ranging from 1 to 153). The collective time of a query group of non-WT cells (host or injected cells of a chimera embryo) is computed in the following way: we resample time ranks for each WT cell by using the time rank of its nearest neighbor WT cell instead. After computing the pairwise correlation between the time distribution of the query group and the resampled WT time distributions per embryo, we use the time of the best correlated WT embryo as the collective time of the query group.</div></section><section id="sec-4-5-5"><h4>Projection of single cells on WT gastrulation atlas</h4><div id="p0285" role="paragraph">For single cell projection on the temporal atlas (<a id="crosref1185" href="#fig1">Figures&nbsp;1</a>C and <a id="crosref1190" href="#fig2">2</a>C) we associated each query cell with a metacell in the atlas. This was done using direct calculation of the correlations between<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="log Subscript 2 Baseline left parenthesis 1 plus u Subscript g Baseline right parenthesis" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (3 1 2) 13 (12 10 (9 4 5 (8 6 7)) 11))" breakable="true" class="MM-N" aria-hidden="true"><mjx-break size="0"></mjx-break><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="3,12" data-semantic-content="13" data-semantic-owns="3 13 12" data-semantic-speech="log Subscript 2 Baseline left parenthesis 1 plus u Subscript g Baseline right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="14" data-semantic-owns="1 2"><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mtext><mjx-script style="vertical-align: -0.241em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="9" data-semantic-content="10,11" data-semantic-parent="14" data-semantic-owns="10 9 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="9" data-semantic-children="4,8" data-semantic-content="5" data-semantic-parent="12" data-semantic-owns="4 5 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="5" data-semantic-parent="9" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of the cell UMI vector <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="u Subscript g" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="u Subscript g"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-math></mjx-container> and the atlas metacells’ expression vectors <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="log Subscript 2 Baseline left parenthesis e Subscript g comma m Baseline plus epsilon right parenthesis" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="16" data-semantic-children="2,14" data-semantic-content="15,0" data-semantic-owns="2 15 14" data-semantic-structure="(16 (2 0 1) 15 (14 12 (11 (8 3 (7 4 5 6)) 9 10) 13))" data-semantic-speech="log Subscript 2 Baseline left parenthesis e Subscript g comma m Baseline plus epsilon right parenthesis" class="MM-N" aria-hidden="true"><mjx-msub data-semantic-type="subscript" data-semantic-role="prefix function" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="16" data-semantic-owns="0 1"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.241em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="11" data-semantic-content="12,13" data-semantic-parent="16" data-semantic-owns="12 11 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="11" data-semantic-children="8,10" data-semantic-content="9" data-semantic-parent="14" data-semantic-owns="8 9 10"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="11" data-semantic-owns="3 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="4 5 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="9" data-semantic-parent="11" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D716">𝜖</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>. To that end we were using the 665 feature genes used for the metacell construction in the WT gastrulation atlas (<span class="dropBlock reference-citations"><a id="body-ref-sref41-8" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-8" aria-controls="sref41-8">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each query cell was matched with its best-correlated atlas metacell. For 2D projections we add gaussian noise to the metacell coordinate of the projected metacell of each cell.</div></section><section id="sec-4-5-6"><h4>Epiblast and nascent mesoderm module scores</h4><div id="p0290" role="paragraph">For each group of genes, the corresponding gene module score per cell is defined as the aggregated relative frequency of UMIs from that group of genes. The epiblast module consists of all genes that are highly correlated with Utf1 expression, i.e. all genes <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="g"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> for which <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c o r left parenthesis l e Subscript g comma m Baseline comma l e Subscript upper U t f Baseline 1 comma m Baseline right parenthesis greater than 0.87 period" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="(42 (41 (40 0 38 1 39 (37 2 36 (32 3 (30 (27 4 26 (10 5 (9 6 7 8))) 11 (29 12 28 (25 13 (24 (23 14 20 15 21 16 22 17) 18 19)))) 31))) 33 34) 35)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="42" data-semantic-children="41,35" data-semantic-content="35" data-semantic-owns="41 35" data-semantic-speech="c o r left parenthesis l e Subscript g comma m Baseline comma l e Subscript upper U t f Baseline 1 comma m Baseline right parenthesis greater than 0.87 period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="41" data-semantic-children="40,34" data-semantic-content="33" data-semantic-parent="42" data-semantic-owns="40 33 34"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="40" data-semantic-children="0,1,37" data-semantic-content="38,39" data-semantic-parent="41" data-semantic-owns="0 38 1 39 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="40"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="38" data-semantic-parent="40" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="40"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-parent="40" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="37" data-semantic-children="2,32" data-semantic-content="36,2" data-semantic-parent="40" data-semantic-owns="2 36 32"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="37" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="32" data-semantic-children="30" data-semantic-content="3,31" data-semantic-parent="37" data-semantic-owns="3 30 31"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="30" data-semantic-children="27,11,29" data-semantic-content="11" data-semantic-parent="32" data-semantic-owns="27 11 29"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-children="4,10" data-semantic-content="26" data-semantic-parent="30" data-semantic-owns="4 26 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="27"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="5,9" data-semantic-parent="27" data-semantic-owns="5 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="6,7,8" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="30" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-children="12,25" data-semantic-content="28" data-semantic-parent="30" data-semantic-owns="12 28 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="29"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="25" data-semantic-children="13,24" data-semantic-parent="29" data-semantic-owns="13 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="25"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="24" data-semantic-children="23,18,19" data-semantic-content="18" data-semantic-parent="25" data-semantic-owns="23 18 19"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="23" data-semantic-children="14,15,16,17" data-semantic-content="20,21,22" data-semantic-parent="24" data-semantic-owns="14 20 15 21 16 22 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="23"><mjx-c class="mjx-c1D448">𝑈</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="23"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="23"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="23" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="23"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="18" data-semantic-parent="24" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="24"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="31" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="33" data-semantic-parent="41" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="float" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="41"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.676em;">0</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.676em;">.</mjx-c><mjx-c class="mjx-c38" noic="true" style="padding-top: 0.676em;">8</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="35" data-semantic-parent="42" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> Those are <i>Dtx1</i>, <i>Enpp3</i>, <i>Foxd3</i>, <i>L1td1</i>, <i>Mal2</i>, <i>Mkrn1</i>, <i>Pdzd4</i>, <i>Pim2</i>, <i>Pipox</i>, <i>Pou3f1</i>, <i>Slc35d3</i>, <i>Slc35f2</i>, <i>Slc7a3</i>, <i>Ugt8a</i>, <i>Utf1</i>. Because of its very high expression relative to all other genes of the epiblast module, Dnmt3b was removed from the final list of epiblast genes. The early nascent mesoderm gene module consists of genes differentially expressed in the metacells MC#449 to MC#456 (<span class="dropBlock reference-citations"><a id="body-ref-sref41-9" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-9" aria-controls="sref41-9">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) from the WT atlas, which are the early nascent mesoderm metacells closest to the primitive streak. More precisely, let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l f c Subscript g Superscript m" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 7 1 8 (6 (5 2 3) 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="0,1,6" data-semantic-content="7,8" data-semantic-owns="0 7 1 8 6" data-semantic-speech="l f c Subscript g Superscript m"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="9"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-parent="9" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> denote the log fold change of a gene <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="g"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> in a metacell <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="m"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> relative to the median among all the other metacells. The early nascent mesoderm score is comprised of genes for which <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m e a n left parenthesis l f c Subscript g Superscript m Baseline right parenthesis greater than 1.7" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="(25 (24 0 21 1 22 2 23 (20 3 19 (16 4 (14 5 12 6 13 (11 (10 7 8) 9)) 15))) 17 18)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="25" data-semantic-children="24,18" data-semantic-content="17" data-semantic-owns="24 17 18" data-semantic-speech="m e a n left parenthesis l f c Subscript g Superscript m Baseline right parenthesis greater than 1.7"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="0,1,2,20" data-semantic-content="21,22,23" data-semantic-parent="25" data-semantic-owns="0 21 1 22 2 23 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="24"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="24"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="24"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="20" data-semantic-children="3,16" data-semantic-content="19,3" data-semantic-parent="24" data-semantic-owns="3 19 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="20" data-semantic-operator="appl"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="16" data-semantic-children="14" data-semantic-content="4,15" data-semantic-parent="20" data-semantic-owns="4 14 15"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="16" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="5,6,11" data-semantic-content="12,13" data-semantic-parent="16" data-semantic-owns="5 12 6 13 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="14"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="14"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="7,8,9" data-semantic-parent="14" data-semantic-collapsed="(11 (10 7 8) 9)" data-semantic-owns="7 8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c6D">m</mjx-c></mjx-mtext><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c67">g</mjx-c></mjx-mtext></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="15" data-semantic-parent="16" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="17" data-semantic-parent="25" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="float" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="25"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.676em;">1</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.676em;">.</mjx-c><mjx-c class="mjx-c37" style="padding-top: 0.676em;">7</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> among the early nascent mesoderm metacells MC#449 to MC#456 from the WT atlas and which are not part of the epiblast gene module. The final list consisted of the 15 genes <i>Apln</i>, <i>Cyp26a1</i>, <i>Eomes</i>, <i>Evx1</i>, <i>Fgf8</i>, <i>Frzb</i>, <i>Fst</i>, <i>Gas1</i>, <i>Mesp1</i>, <i>Mixl1</i>, <i>Snai1</i>, <i>Sp5</i>, <i>T</i>, <i>Tdgf1</i> and <i>Wnt3</i> (<a id="crosref1205" href="#figs5">Figure&nbsp;S5</a>D).</div></section><section id="sec-4-5-7"><h4>Differential expression analysis in epiblast and early nascent mesoderm</h4><div id="p0295" role="paragraph">To analyze differential gene expression among the various genotypes in epiblast and early nascent mesoderm cells, we filtered all genes that a) satisfied a threshold of minimal expression in at least one of the clones, i.e. <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="log Subscript 2 Baseline left parenthesis e Subscript g comma c l Baseline plus 10 Superscript negative 5 Baseline right parenthesis greater than negative 14" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="(29 (27 (3 0 2) 26 (22 4 (20 (13 5 (12 6 7 (11 8 10 9))) 14 (19 15 (18 16 17))) 21)) 23 (28 24 25))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="29" data-semantic-children="27,28" data-semantic-content="23" data-semantic-owns="27 23 28" data-semantic-speech="log Subscript 2 Baseline left parenthesis e Subscript g comma c l Baseline plus 10 Superscript negative 5 Baseline right parenthesis greater than negative 14"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="27" data-semantic-children="3,22" data-semantic-content="26,0" data-semantic-parent="29" data-semantic-owns="3 26 22"><mjx-msub data-semantic-type="subscript" data-semantic-role="prefix function" data-semantic-id="3" data-semantic-children="0,2" data-semantic-parent="27" data-semantic-owns="0 2"><mjx-mrow><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow><mjx-script style="vertical-align: -0.241em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="22" data-semantic-children="20" data-semantic-content="4,21" data-semantic-parent="27" data-semantic-owns="4 20 21"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="22" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="20" data-semantic-children="13,19" data-semantic-content="14" data-semantic-parent="22" data-semantic-owns="13 14 19"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="5,12" data-semantic-parent="20" data-semantic-owns="5 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="13"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="12" data-semantic-children="6,7,11" data-semantic-content="7" data-semantic-parent="13" data-semantic-owns="6 7 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="12"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="12" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="11" data-semantic-children="8,9" data-semantic-content="10" data-semantic-parent="12" data-semantic-owns="8 10 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="20" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="19" data-semantic-children="15,18" data-semantic-parent="20" data-semantic-owns="15 18"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="19"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-children="17" data-semantic-content="16" data-semantic-parent="19" data-semantic-owns="16 17"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="16" data-semantic-parent="18" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="21" data-semantic-parent="22" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="23" data-semantic-parent="29" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-children="25" data-semantic-content="24" data-semantic-parent="29" data-semantic-owns="24 25"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-parent="28" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="28"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.677em;">1</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for any clone <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c l" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 2 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-owns="0 2 1" data-semantic-speech="c l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and b) had a sufficiently large differential expression relative to the matched wildtype expression, i.e. all genes <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="g"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> for which <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l f Subscript g Superscript c l Baseline greater than log Subscript 2 Baseline left parenthesis 1.6 right parenthesis" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="(22 (21 0 20 (8 (7 1 2) (6 3 5 4))) 9 (19 (13 10 12) 18 (17 14 15 16)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="22" data-semantic-children="21,19" data-semantic-content="9" data-semantic-owns="21 9 19" data-semantic-speech="l f Subscript g Superscript c l Baseline greater than log Subscript 2 Baseline left parenthesis 1.6 right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="21" data-semantic-children="0,8" data-semantic-content="20" data-semantic-parent="22" data-semantic-owns="0 20 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="21"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="21" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="1,2,6" data-semantic-parent="21" data-semantic-collapsed="(8 (7 1 2) 6)" data-semantic-owns="1 2 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="3 5 4" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="9" data-semantic-parent="22" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-children="13,17" data-semantic-content="18,10" data-semantic-parent="22" data-semantic-owns="13 18 17"><mjx-msub data-semantic-type="subscript" data-semantic-role="prefix function" data-semantic-id="13" data-semantic-children="10,12" data-semantic-parent="19" data-semantic-owns="10 12"><mjx-mrow><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="10" data-semantic-parent="13" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow><mjx-script style="vertical-align: -0.241em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="15" data-semantic-content="14,16" data-semantic-parent="19" data-semantic-owns="14 15 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="14" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="float" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.666em;">.</mjx-c><mjx-c class="mjx-c36" style="padding-top: 0.666em;">6</mjx-c></mjx-mn><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for at least two clones <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="c l" tabindex="0" ctxtmenu_counter="20" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 2 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-owns="0 2 1" data-semantic-speech="c l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. Here the log<sub>2</sub> fold change <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="l f Subscript g Superscript c l" tabindex="0" ctxtmenu_counter="21" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 9 (8 (7 1 2) (6 3 5 4)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="10" data-semantic-children="0,8" data-semantic-content="9" data-semantic-owns="0 9 8" data-semantic-speech="l f Subscript g Superscript c l"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="1,2,6" data-semantic-parent="10" data-semantic-collapsed="(8 (7 1 2) 6)" data-semantic-owns="1 2 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.247em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="3,4" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="3 5 4" style="margin-left: 0.163em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> is defined relative to the WT atlas bulk expression in the respective cell type (epiblast and early nascent mesoderm). This resulted in 337 filtered genes for early nascent mesoderm and 297 for epiblast. We additionally removed cell-cycle and stress-related genes that were also removed during metacell construction, Y chromosome genes and the imprinted genes <i>Igf2</i>, <i>H19</i>, <i>Grb10</i> and <i>Peg5</i>. For the final list of genes shown in the heatmaps in <a id="crosref1210" href="#fig3">Figures&nbsp;3</a>B and 3E, we only included genes that had higher differential expression in <i>Tet</i>-TKO relative to host and control (i.e., <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartAbsoluteValue l f Subscript g comma upper T upper K upper O Baseline minus l f Subscript g comma upper H o s t Baseline EndAbsoluteValue plus StartAbsoluteValue l f Subscript g comma upper T upper K upper O Baseline minus l f Subscript g comma upper C t r l Baseline EndAbsoluteValue greater than log Subscript 2 Baseline left parenthesis 1.5 right parenthesis" tabindex="0" ctxtmenu_counter="22" style="font-size: 130%;"><mjx-math data-semantic-structure="(83 (82 (34 0 (32 (29 1 28 (12 2 (11 3 4 (10 5 8 6 9 7)))) 13 (31 14 30 (27 15 (26 16 17 (25 18 22 19 23 20 24 21))))) 33) 35 (70 36 (68 (65 37 64 (48 38 (47 39 40 (46 41 44 42 45 43)))) 49 (67 50 66 (63 51 (62 52 53 (61 54 58 55 59 56 60 57))))) 69)) 71 (81 (75 72 74) 80 (79 76 77 78)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="83" data-semantic-children="82,81" data-semantic-content="71" data-semantic-owns="82 71 81" data-semantic-speech="StartAbsoluteValue l f Subscript g comma upper T upper K upper O Baseline minus l f Subscript g comma upper H o s t Baseline EndAbsoluteValue plus StartAbsoluteValue l f Subscript g comma upper T upper K upper O Baseline minus l f Subscript g comma upper C t r l Baseline EndAbsoluteValue greater than log Subscript 2 Baseline left parenthesis 1.5 right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="82" data-semantic-children="34,70" data-semantic-content="35" data-semantic-parent="83" data-semantic-owns="34 35 70"><mjx-mrow data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="34" data-semantic-children="32" data-semantic-content="0,33" data-semantic-parent="82" data-semantic-owns="0 32 33"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="0" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="32" data-semantic-children="29,31" data-semantic-content="13" data-semantic-parent="34" data-semantic-owns="29 13 31"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-children="1,12" data-semantic-content="28" data-semantic-parent="32" data-semantic-owns="1 28 12"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="29"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="12" data-semantic-children="2,11" data-semantic-parent="29" data-semantic-owns="2 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="12"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="11" data-semantic-children="3,4,10" data-semantic-content="4" data-semantic-parent="12" data-semantic-owns="3 4 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="4" data-semantic-parent="11" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="10" data-semantic-children="5,6,7" data-semantic-content="8,9" data-semantic-parent="11" data-semantic-owns="5 8 6 9 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="10"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="10"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="10"><mjx-c class="mjx-c1D442">𝑂</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-parent="32" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="31" data-semantic-children="14,27" data-semantic-content="30" data-semantic-parent="32" data-semantic-owns="14 30 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="31"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="27" data-semantic-children="15,26" data-semantic-parent="31" data-semantic-owns="15 26"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="27"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="26" data-semantic-children="16,17,25" data-semantic-content="17" data-semantic-parent="27" data-semantic-owns="16 17 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="26"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="17" data-semantic-parent="26" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="25" data-semantic-children="18,19,20,21" data-semantic-content="22,23,24" data-semantic-parent="26" data-semantic-owns="18 22 19 23 20 24 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="25"><mjx-c class="mjx-c1D43B">𝐻</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="25"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="25"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="25"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="33" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="35" data-semantic-parent="82" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="fenced" data-semantic-role="neutral" data-semantic-id="70" data-semantic-children="68" data-semantic-content="36,69" data-semantic-parent="82" data-semantic-owns="36 68 69"><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="36" data-semantic-parent="70" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="68" data-semantic-children="65,67" data-semantic-content="49" data-semantic-parent="70" data-semantic-owns="65 49 67"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="65" data-semantic-children="37,48" data-semantic-content="64" data-semantic-parent="68" data-semantic-owns="37 64 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="65"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="64" data-semantic-parent="65" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="48" data-semantic-children="38,47" data-semantic-parent="65" data-semantic-owns="38 47"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="48"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="47" data-semantic-children="39,40,46" data-semantic-content="40" data-semantic-parent="48" data-semantic-owns="39 40 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="47"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="40" data-semantic-parent="47" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="46" data-semantic-children="41,42,43" data-semantic-content="44,45" data-semantic-parent="47" data-semantic-owns="41 44 42 45 43"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="46"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="44" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="46"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="46"><mjx-c class="mjx-c1D442">𝑂</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="49" data-semantic-parent="68" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="67" data-semantic-children="50,63" data-semantic-content="66" data-semantic-parent="68" data-semantic-owns="50 66 63"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="67"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="66" data-semantic-parent="67" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="63" data-semantic-children="51,62" data-semantic-parent="67" data-semantic-owns="51 62"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="63"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.062em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="62" data-semantic-children="52,53,61" data-semantic-content="53" data-semantic-parent="63" data-semantic-owns="52 53 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="62"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="53" data-semantic-parent="62" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="61" data-semantic-children="54,55,56,57" data-semantic-content="58,59,60" data-semantic-parent="62" data-semantic-owns="54 58 55 59 56 60 57"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="54" data-semantic-parent="61"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="61"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="59" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="61"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="60" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="61"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="neutral" data-semantic-id="69" data-semantic-parent="70" data-semantic-operator="fenced"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="71" data-semantic-parent="83" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-children="75,79" data-semantic-content="80,72" data-semantic-parent="83" data-semantic-owns="75 80 79"><mjx-msub data-semantic-type="subscript" data-semantic-role="prefix function" data-semantic-id="75" data-semantic-children="72,74" data-semantic-parent="81" data-semantic-owns="72 74"><mjx-mrow><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="72" data-semantic-parent="75" data-semantic-operator="appl"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mi><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow><mjx-script style="vertical-align: -0.241em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="75"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="80" data-semantic-parent="81" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="79" data-semantic-children="77" data-semantic-content="76,78" data-semantic-parent="81" data-semantic-owns="76 77 78"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="76" data-semantic-parent="79" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="float" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="79"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.666em;">.</mjx-c><mjx-c class="mjx-c35" style="padding-top: 0.666em;">5</mjx-c></mjx-mn><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="78" data-semantic-parent="79" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>).</div></section><section id="sec-4-5-8"><h4>Mean differential expression among transcriptional states from mixed chimera embryos</h4><div id="p0300" role="paragraph">For each clone and cell type, we computed bulk gene expression profiles after down-sampling to the same number of cells (50 cells) and UMIs (190.000) per condition. We then filtered genes that a) had a minimal expression of at least <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="10 Superscript negative 4" tabindex="0" ctxtmenu_counter="23" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msup data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="10 Superscript negative 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-math></mjx-container> in any of the bulk profiles and b) had a minimal log<sub>2</sub> fold change of at least 1.6 (regularization <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="epsilon equals 6 dot 10 Superscript negative 5" tabindex="0" ctxtmenu_counter="24" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 1 (9 2 3 (8 4 (7 5 6))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="10" data-semantic-children="0,9" data-semantic-content="1" data-semantic-owns="0 1 9" data-semantic-speech="epsilon equals 6 dot 10 Superscript negative 5"><mjx-mtext data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="0" data-semantic-parent="10"><mjx-c class="mjx-c3F5">ϵ</mjx-c></mjx-mtext><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="10" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="3" data-semantic-parent="10" data-semantic-owns="2 3 8"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="9"><mjx-c class="mjx-c36">6</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="infixop,⋅"><mjx-c class="mjx-c22C5">⋅</mjx-c></mjx-mo><mjx-msup space="3" data-semantic-type="superscript" data-semantic-role="integer" data-semantic-id="8" data-semantic-children="4,7" data-semantic-parent="9" data-semantic-owns="4 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-script style="vertical-align: 0.393em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-children="6" data-semantic-content="5" data-semantic-parent="8" data-semantic-owns="5 6"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c35">5</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container>) between any two bulk profiles. From this putative group of feature genes, we removed cell-cycle and stress-related genes and added cell-type-specific genes for each included cell type (for this purpose, a cell-type-specific gene was defined as being on average at least twofold enriched in the metacells from this cell type relative to the global average). Using this group of genes, we computed for each clone and cell type the mean absolute log<sub>2</sub> fold change between the bulk profile and the corresponding WT bulk profile from the same cell type. <a id="crosref1215" href="#fig4">Figure&nbsp;4</a>E shows this quantity for all cell types that have at least one clone from each KO genotype, i.e. from <i>Tet</i>-TKO, DKO1/2, DKO1/3, and DKO2/3 cells.</div></section><section id="sec-4-5-9"><h4>Cell cycle score of single cells</h4><div id="p0305" role="paragraph">For each single cell, we computed fold changes of expression of mitosis phase related genes (<i>Mki67, Cenpf, Top2a, Smc4;SMC4, Ube2c, Ccnb1, Cdk1, Arl6ip1, Ankrd11, Hmmr;IHABP, Cenpa;Cenp-a, Tpx2, Aurka, Kif4, Kif2c, Bub1b, Ccna2, Kif23, Kif20a, Sgol2, Smc2, Kif11, Cdca2, Incenp, Cenpe</i>) as well as synthesis phase related genes (<i>Pcna, Rrm2, Mcm5, Mcm6, Mcm4, Ung, Mcm7, Mcm2, Uhrf1, Orc6, Tipin</i>) relative to the mean among all cells (after normalizing expression in each cell by the total number of UMIs per cell). The cell cycle score of a single cell is the average fold change of mitosis and synthesis phase scores. High values correspond to cycling cells and low values to non-cycling cells. Single-cell score distributions are shown in <a id="crosref1220" href="#fig6">Figure&nbsp;6</a>E.</div></section><section id="sec-4-5-10"><h4>Trajectories of node/notochord lineages</h4><div id="p0310" role="paragraph">Trajectories in WT atlas for pit cells (MC #367), crown cells (MC #108), notochord (MC #366) and foregut (MCs #377,382,385) were computed by back propagation of the flow passing through each metacell at the final time point (<span class="dropBlock reference-citations"><a id="body-ref-sref41-10" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-10" aria-controls="sref41-10">Mittnenzweig et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-5-11"><h4>Processing of PBAT data</h4><div id="p0315" role="paragraph">Raw PBAT sequencing reads were stripped of indices and random oligos, and were then mapped to the mm9 reference genome using an in-house script that is based on Bowtie2 (<a id="intref0175" href="https://github.com/tanaylab/bissli2" target="_blank">https://github.com/tanaylab/bissli2</a>) with the following parameters “--maxins 1000 –ga”. In cases where the two reads ends were not aligned concordantly, the reads were discarded. Reads that were mapped to the same genome coordinates where marked as duplicates and were used only once. Reads with mapping quality (MAPQ) below 30 or reads that had more than 3 non-converted C’s in non-CpG content (CHH) were discarded. Individual CpG methylation level was then called for each read, discarding bases with base quality &lt;20.</div></section><section id="sec-4-5-12"><h4>Histone modifications reference data processing</h4><div id="p0320" role="paragraph">We downloaded H3K4me3 and H3K27me3 tracks representing mouse embryonic tissues from ENCODE-phase3 (<a id="intref0180" href="https://www.encodeproject.org/" target="_blank">https://www.encodeproject.org/</a>) (limb, liver, heart, midbrain, hindbrain, forebrain, E10-E11.5), and H3K4me3 data from E7.5 mouse embryo (<span class="dropBlock reference-citations"><a id="body-ref-sref73-1" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-1" aria-controls="sref73-1">Xiang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref73-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Xiang, Y. ∙ Zhang, Y. ∙ Xu, Q. ...</div><div><strong>Epigenomic analysis of gastrulation identifies a unique chromatin state for primed pluripotency</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:95-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0545-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85076571979" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31844322/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0545-1&amp;pmid=31844322" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We merged replicates and transformed coverage statistics into percentiles, using -log<sub>2</sub>(1- coverage percentile) to identify loci enriched with each mark. A locus was defined as enriched if the percentile score was more than 7 in any of the above tracks. We identified such 53722 H3K4me3 and 330371 H3K27me3 intervals and extracted methylation calls in each locus. ATAC peaks were taken from (<span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Xiang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Xiang, Y. ∙ Zhang, Y. ∙ Xu, Q. ...</div><div><strong>Epigenomic analysis of gastrulation identifies a unique chromatin state for primed pluripotency</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:95-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0545-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85076571979" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31844322/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0545-1&amp;pmid=31844322" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), defined as -log<sub>2</sub>(1-coverage percentile)&nbsp;&gt;&nbsp;7 in one of the four sorted subpopulations (ectoderm, endoderm, mesoderm, VE).</div></section><section id="sec-4-5-13"><h4>Defining putative enhancers from hypo-methylation hotspots</h4><div id="p0325" role="paragraph">We used E8.5 embryos WGBS data (Z.M., unpublished data; GEO: <a id="intref0185" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE199806" target="_blank">GSE199806</a>) to screen for 300bp long loci that are covered by at least 50 methylation calls and have a reduction of at least 40% in methylation from the average methylation of their flanking regions - 500bp from each side, starting 50bp from the locus edge. We identified 61944 such loci and extracted average methylation from 12720 with coverage &gt;= 10 in either control or <i>Tet</i>-TKO. Such methylation dips were validated to overlap extensively with other data defining putative distal regulatory elements (<a id="crosref1240" href="#figs6">Figure&nbsp;S6</a>F)</div></section><section id="sec-4-5-14"><h4>Comparison of <i>Tet</i>-TKO and control methylation</h4><div id="p0330" role="paragraph">To compare <i>Tet</i>-TKO and control methylation (<a id="crosref1245" href="#fig5">Figure&nbsp;5</a>B) we extracted average methylation of 10837835 CpGs smoothing over 200bp from each side of the CpG and requiring a coverage of at least 20 methylation calls over the smoothed window. We used a subset of these CpGs that were on CTCF peaks (see below) for <a id="crosref1250" href="#fig5">Figure&nbsp;5</a>F.</div><div id="p0335" role="paragraph">When analyzing methylation differences in correlation with histone modifications (<a id="crosref1255" href="#fig5">Figure&nbsp;5</a>E), exons or promoters (<a id="crosref1260" href="#fig5">Figure&nbsp;5</a>G), we extracted all the methylation calls inside each locus without smoothing, requiring at least 20 methylation calls per locus. Promoters were defined as 250bp downstream and 50bp upstream to TSS.</div></section><section id="sec-4-5-15"><h4>Background methylation in TADs</h4><div id="p0340" role="paragraph">We used single cell Hi-C data from a pool of ES cells taken from published datasets (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Nagano et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Nagano, T. ∙ Lubling, Y. ∙ Várnai, C. ...</div><div><strong>Cell-cycle dynamics of chromosomal organization at single-cell resolution</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:61-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85022195504" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28682332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23001&amp;pmid=28682332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to define 2461 TADs. We summarized background methylation in each TAD by summing up all methylation calls excluding regions that are on a H3K4me1 peak, H3K27me3 peak, ATAC peak or putative (E8.5 methylation based) enhancer.</div><div id="p0345" role="paragraph">To define time of replication (<a id="crosref1270" href="#fig5">Figure&nbsp;5</a>D) we used Encode Repli-seq mESCs data (D3, ENCFF001JUO). Loci with values above zero were considered “late” and loci below zero were considered “early”.</div></section><section id="sec-4-5-16"><h4>Motif analysis</h4><div id="p0350" role="paragraph">To screen for enrichment of known TF binding sites, we extracted PWM scores (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Jolma et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Jolma, A. ∙ Yan, J. ∙ Whitington, T. ...</div><div><strong>DNA-binding specificities of human transcription factors</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:327-339</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2012.12.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2012.12.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84872540994" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23332764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2012.12.009&amp;pmid=23332764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for each of the 12720 putative enhancer regions. We then counted for each motif the number of loci with PWM score that was in the top 1% percentile genomically, and calculated the average methylation in these loci and the rest of the putative enhancers, as can be seen in <a id="crosref1280" href="#figs6">Figure&nbsp;S6</a>H.</div><div id="p0355" role="paragraph">CTCF peaks (<a id="crosref1285" href="#fig5">Figure&nbsp;5</a>F) where defined as CpGs with the top 0.1% of CTCF motif PWM energy from the HOMER (<a id="intref0190" href="http://homer.ucsd.edu/homer/motif/" target="_blank">http://homer.ucsd.edu/homer/motif/</a>) motifs catalog.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0360" role="paragraph">We thank the Tanay and Stelzer group members for discussion and advice. Y.S. is the incumbent of the Louis and Ida Rich Career Development Chair and is supported by European Research Council (ERC_StG 852865), Moross Integrated Cancer Center, the Israel Cancer Research Fund (ICRF), Helen and Martin Kimmel Stem Cell Institute, Yeda-Sela Center, ISF (1610/18), the Minerva Foundation, Human Frontier Science Program (CDA00023/2019-C), Schwartz/Reisman Collaborative Science Program, Hadar Impact Fund, Lord Sieff of Brimpton Memorial Fund, Maurice and Vivienne Wohl Biology Endowment, Janet and Steven Anixter, JoAnne Silva, and Lester and Edward Anixter Family. S.C. was supported by the EMBO long-term fellowship (ALTF 268-2018). M.M. was a postdoctoral fellow of the Minerva Stiftung and is supported by the Walter Benjamin program of the German Research Foundation (DFG). A.T. is supported by the European Research Council (ERC CoG scAssembly), the EU BRAINTIME project, the Israel Science Foundation, and the Chen-Zuckerberg Foundation. This research was further supported by Israeli Council for Higher Education (CHE) Data Science program and by a grant from Madame Olga Klein-Astracha. Research in the Y.S. and A.T. labs is supported by Barry and Janet Lang.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0365" role="paragraph">S.C., M.M., A.T., and Y.S. conceived and designed the experiment and performed data analysis and its interpretation. Y.M. and S.G. assisted in the collection of single cells from individual embryos, and N.R. prepared scRNA-seq libraries. A.L., M.D., and E.C. assisted with scRNA-seq data analyses. S.C. and Z.M. prepared PBAT libraries, and A.L. analyzed DNA methylation data. Y.R. assisted with the generation of control cell lines and AAV production. R.B.-Y. helped in immunostaining and microscopy. H.R. assisted in graphic design. A.-H.O. and Y.M. performed chimera injections. S.C., M.M., A.L., A.T., and Y.S. wrote the manuscript with input from all the authors.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0370" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (3)</h2><div id="p0380" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.049/attachment/03b48e99-f0de-4d9d-93dc-2b640ecc9aec/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (45.13 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table&nbsp;S1. Relative cell-type composition for mixed and whole-embryo chimera with transcriptional age &gt;7.75, related to Figures&nbsp;1, 2, and 4</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.049/attachment/1558c7fa-9dda-4657-a53f-a64b13f89f68/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (80.84 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table&nbsp;S2. List of exons and H3K4me3 unmarked promoters that are hypomethylated in Tet-TKO cells, related to Figure&nbsp;5</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.06.049/attachment/c9e1f813-f868-4ca4-8dc0-72d8b3f70b18/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (26.00 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table&nbsp;S3. Transcription factor binding site enrichment analysis for hypermethylated enhancers in Tet-TKO cells, related to Figure&nbsp;5</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">All raw and processed data is available at GEO: <a id="intref0010" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205917" target="_blank">GSE205917</a>.</div><div id="p0035" role="paragraph">Code has been deposited at: <a id="intref0015" href="https://tanaylab.github.io/tet-gastrulation/" target="_blank">https://tanaylab.github.io/tet-gastrulation/</a>, <a id="intref0020" href="https://doi.org/10.5281/zenodo.6720248" target="_blank">https://doi.org/10.5281/zenodo.6720248</a>.</div><div id="p0035a" role="paragraph">Original western blot images have been deposited at Mendeley Data: <a id="interref0015" href="https://doi.org/10.17632/7j64ch46gz.1" target="_blank">https://doi.org/10.17632/7j64ch46gz.1</a>.</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Aguilera-Castrejon, A. ∙ Oldak, B. ∙ Shani, T. ...</div><div><strong>Ex utero mouse embryogenesis from pre-gastrulation to late organogenesis</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:119-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03416-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85102676173" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33731940/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03416-3&amp;pmid=33731940" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Argelaguet, R. ∙ Clark, S.J. ∙ Mohammed, H. ...</div><div><strong>Multi-omics profiling of mouse gastrulation at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>576</strong>:487-491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1825-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85076568486" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31827285/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1825-8&amp;pmid=31827285" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Arnold, S.J. ∙ Robertson, E.J.</div><div><strong>Making a commitment: cell lineage allocation and axis patterning in the early mouse embryo</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2009; <strong>10</strong>:91-103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm2618" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-58849097959" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19129791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm2618&amp;pmid=19129791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Baran, Y. ∙ Bercovich, A. ∙ Sebe-Pedros, A. ...</div><div><strong>MetaCell: analysis of single-cell RNA-seq data using K-nn graph partitions</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1812-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85073149395" target="_blank" aria-label="Open Scopus in new tab">Scopus (167)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31604482/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1812-2&amp;pmid=31604482" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bellomo, D. ∙ Lander, A. ∙ Harragan, I. ...</div><div><strong>Cell proliferation in mammalian gastrulation: the ventral node and notochord are relatively quiescent</strong></div><div><em>Dev. Dyn.</em> 1996; <strong>205</strong>:471-485</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-0177(199604)205:4%3C471::AID-AJA10%3E3.0.CO;2-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0029873599" target="_blank" aria-label="Open Scopus in new tab">Scopus (107)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8901057/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-0177%28199604%29205%3A4%253C471%3A%3AAID-AJA10%253E3.0.CO%3B2-4&amp;pmid=8901057" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Chan, M.M. ∙ Smith, Z.D. ∙ Grosswendt, S. ...</div><div><strong>Molecular recording of mammalian embryogenesis</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:77-82</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1184-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85065790772" target="_blank" aria-label="Open Scopus in new tab">Scopus (204)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31086336/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1184-5&amp;pmid=31086336" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Chen, C. ∙ Ware, S.M. ∙ Sato, A. ...</div><div><strong>The Vg1-related protein Gdf3 acts in a Nodal signaling pathway in the pre-gastrulation mouse embryo</strong></div><div><em>Development.</em> 2006; <strong>133</strong>:319-329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-32244439642" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16368929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02210&amp;pmid=16368929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Dai, H.-Q. ∙ Wang, B.-A. ∙ Yang, L. ...</div><div><strong>TET-mediated DNA demethylation controls gastrulation by regulating Lefty–Nodal signalling</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature20095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84994051865" target="_blank" aria-label="Open Scopus in new tab">Scopus (156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27760115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature20095&amp;pmid=27760115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Loss of tet enzymes compromises proper differentiation of embryonic stem cells</strong></div><div><em>Dev. Cell.</em> 2014; <strong>29</strong>:102-111</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2014.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84898482006" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24735881/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2014.03.003&amp;pmid=24735881" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Dawlaty, M.M. ∙ Breiling, A. ∙ Le, T. ...</div><div><strong>Combined deficiency of Tet1 and Tet2 causes epigenetic abnormalities but is compatible with postnatal development</strong></div><div><em>Dev. Cell.</em> 2013; <strong>24</strong>:310-323</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2012.12.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.devcel.2012.12.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84873707539" target="_blank" aria-label="Open Scopus in new tab">Scopus (362)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23352810/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2012.12.015&amp;pmid=23352810" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Dawlaty, M.M. ∙ Ganz, K. ∙ Powell, B.E. ...</div><div><strong>Tet1 is dispensable for maintaining pluripotency and its loss is compatible with embryonic and postnatal development</strong></div><div><em>Cell Stem Cell.</em> 2011; <strong>9</strong>:166-175</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.07.010&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.07.010&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79961139741" target="_blank" aria-label="Open Scopus in new tab">Scopus (428)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21816367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2011.07.010&amp;pmid=21816367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Dixon, G. ∙ Pan, H. ∙ Yang, D. ...</div><div><strong>QSER1 protects DNA methylation valleys from de novo methylation</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:eabd0875</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abd0875" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85104166307" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33833093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abd0875&amp;pmid=33833093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Dixon, J.R. ∙ Selvaraj, S. ∙ Yue, F. ...</div><div><strong>Topological domains in mammalian genomes identified by analysis of chromatin interactions</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:376-380</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11082" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84861095603" target="_blank" aria-label="Open Scopus in new tab">Scopus (4600)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11082&amp;pmid=22495300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Ginno, P.A. ∙ Gaidatzis, D. ∙ Feldmann, A. ...</div><div><strong>A genome-scale map of DNA methylation turnover identifies site-specific dependencies of DNMT and TET activity</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2680</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16354-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85085693059" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32471981/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16354-x&amp;pmid=32471981" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Grosswendt, S. ∙ Kretzmer, H. ∙ Smith, Z.D. ...</div><div><strong>Epigenetic regulator function through mouse gastrulation</strong></div><div><em>Nature.</em> 2020; <strong>584</strong>:102-108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2552-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85088787000" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32728215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2552-x&amp;pmid=32728215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Gu, T.-P. ∙ Guo, F. ∙ Yang, H. ...</div><div><strong>The role of Tet3 DNA dioxygenase in epigenetic reprogramming by oocytes</strong></div><div><em>Nature.</em> 2011; <strong>477</strong>:606-610</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10443" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80053348585" target="_blank" aria-label="Open Scopus in new tab">Scopus (935)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21892189/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10443&amp;pmid=21892189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Hackett, J.A. ∙ SenGupta, R. ∙ Zylicz, J.J. ...</div><div><strong>Germline DNA demethylation dynamics and imprint erasure Through 5-hydroxymethylcytosine</strong></div><div><em>Science.</em> 2013; <strong>339</strong>:448-452</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1229277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84872770694" target="_blank" aria-label="Open Scopus in new tab">Scopus (636)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23223451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1229277&amp;pmid=23223451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Han, X. ∙ Wang, R. ∙ Zhou, Y. ...</div><div><strong>Mapping the mouse cell atlas by microwell-seq</strong></div><div><em>Cell.</em> 2018; <strong>172</strong>:1091-1107.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2018.02.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2018.02.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85042366842" target="_blank" aria-label="Open Scopus in new tab">Scopus (901)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29474909/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2018.02.001&amp;pmid=29474909" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>He, Y.-F. ∙ Li, B.-Z. ∙ Li, Z. ...</div><div><strong>Tet-mediated formation of 5-Carboxylcytosine and its excision by TDG in mammalian DNA</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:1303-1307</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1210944" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052495940" target="_blank" aria-label="Open Scopus in new tab">Scopus (2220)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21817016/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1210944&amp;pmid=21817016" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Hon, G.C. ∙ Song, C.-X. ∙ Du, T. ...</div><div><strong>5-mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation</strong></div><div><em>Mol. Cell.</em> 2014; <strong>56</strong>:286-297</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2014.08.026&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2014.08.026&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84908204470" target="_blank" aria-label="Open Scopus in new tab">Scopus (259)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25263596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2014.08.026&amp;pmid=25263596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Hua, R. ∙ Yu, S. ∙ Liu, M. ...</div><div><strong>A PCR-based method for RNA probes and applications in neuroscience</strong></div><div><em>Front. Neurosci.</em> 2018; <strong>12</strong>:266</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fnins.2018.00266" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85046622006" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29770110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffnins.2018.00266&amp;pmid=29770110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Ito, S. ∙ D’Alessio, A.C. ∙ Taranova, O.V. ...</div><div><strong>Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification</strong></div><div><em>Nature.</em> 2010; <strong>466</strong>:1129-1133</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-77956189495" target="_blank" aria-label="Open Scopus in new tab">Scopus (2083)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20639862/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09303&amp;pmid=20639862" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Ito, S. ∙ Shen, L. ∙ Dai, Q. ...</div><div><strong>Tet proteins can convert 5-methylcytosine to 5-Formylcytosine and 5-Carboxylcytosine</strong></div><div><em>Science.</em> 2011; <strong>333</strong>:1300-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1210597" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052461558" target="_blank" aria-label="Open Scopus in new tab">Scopus (2734)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21778364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1210597&amp;pmid=21778364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Izzo, F. ∙ Lee, S.C. ∙ Poran, A. ...</div><div><strong>DNA methylation disruption reshapes the hematopoietic differentiation landscape</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:378-387</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0595-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85083041344" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32203468/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0595-4&amp;pmid=32203468" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Jolma, A. ∙ Yan, J. ∙ Whitington, T. ...</div><div><strong>DNA-binding specificities of human transcription factors</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:327-339</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2012.12.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2012.12.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84872540994" target="_blank" aria-label="Open Scopus in new tab">Scopus (882)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23332764/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2012.12.009&amp;pmid=23332764" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Kang, J. ∙ Lienhard, M. ∙ Pastor, W.A. ...</div><div><strong>Simultaneous deletion of the methylcytosine oxidases Tet1 and Tet3 increases transcriptome variability in early embryogenesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E4236-E4245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1510510112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84938719137" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26199412/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1510510112&amp;pmid=26199412" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Kaspar, B.K. ∙ Vissel, B. ∙ Bengoechea, T. ...</div><div><strong>Adeno-associated virus effectively mediates conditional gene modification in the brain</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:2320-2325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.042678699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0037133198" target="_blank" aria-label="Open Scopus in new tab">Scopus (164)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11842206/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.042678699&amp;pmid=11842206" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Keren-Shaul, H. ∙ Kenigsberg, E. ∙ Jaitin, D.A. ...</div><div><strong>MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:1841-1862</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0164-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85066030980" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31101904/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0164-4&amp;pmid=31101904" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Khoueiry, R. ∙ Sohni, A. ∙ Thienpont, B. ...</div><div><strong>Lineage-specific functions of TET1 in the postimplantation mouse embryo</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1061-1072</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3868" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85021769555" target="_blank" aria-label="Open Scopus in new tab">Scopus (81)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28504700/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3868&amp;pmid=28504700" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Ko, M. ∙ Bandukwala, H.S. ∙ An, J. ...</div><div><strong>Ten-Eleven-Translocation 2 (TET2) negatively regulates homeostasis and differentiation of hematopoietic stem cells in mice</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2011; <strong>108</strong>:14566-14571</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1112317108" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052284526" target="_blank" aria-label="Open Scopus in new tab">Scopus (476)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21873190/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1112317108&amp;pmid=21873190" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Koh, K.P. ∙ Yabuuchi, A. ∙ Rao, S. ...</div><div><strong>Tet1 and Tet2 regulate 5-hydroxymethylcytosine production and cell lineage specification in mouse embryonic stem cells</strong></div><div><em>Cell Stem Cell.</em> 2011; <strong>8</strong>:200-213</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.01.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2011.01.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79551587102" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21295276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2011.01.008&amp;pmid=21295276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Koo, B.-K. ∙ Stange, D.E. ∙ Sato, T. ...</div><div><strong>Controlled gene expression in primary Lgr5 organoid cultures</strong></div><div><em>Nat. Methods.</em> 2011; <strong>9</strong>:81-83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1802" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84856490464" target="_blank" aria-label="Open Scopus in new tab">Scopus (249)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22138822/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1802&amp;pmid=22138822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>La Manno, G. ∙ Soldatov, R. ∙ Zeisel, A. ...</div><div><strong>RNA velocity of single cells</strong></div><div><em>Nature.</em> 2018; <strong>560</strong>:494-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-018-0414-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85052109231" target="_blank" aria-label="Open Scopus in new tab">Scopus (1950)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-018-0414-6&amp;pmid=30089906" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (33760)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Li, X. ∙ Yue, X. ∙ Pastor, W.A. ...</div><div><strong>Tet proteins influence the balance between neuroectodermal and mesodermal fate choice by inhibiting Wnt signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:E8267-E8276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1617802113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85006421153" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27930333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1617802113&amp;pmid=27930333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Li, Z. ∙ Cai, X. ∙ Cai, C.-L. ...</div><div><strong>Deletion of Tet2 in mice leads to dysregulated hematopoietic stem cells and subsequent development of myeloid malignancies</strong></div><div><em>Blood.</em> 2011; <strong>118</strong>:4509-4518</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2010-12-325241" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-80052285127" target="_blank" aria-label="Open Scopus in new tab">Scopus (543)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21803851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2010-12-325241&amp;pmid=21803851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Liu, X.S. ∙ Wu, H. ∙ Ji, X. ...</div><div><strong>Editing DNA methylation in the mammalian genome</strong></div><div><em>Cell.</em> 2016; <strong>167</strong>:233-247.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2016.08.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2016.08.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84988569121" target="_blank" aria-label="Open Scopus in new tab">Scopus (854)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27662091/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.08.056&amp;pmid=27662091" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>López-Moyado, I.F. ∙ Tsagaratou, A. ∙ Yuita, H. ...</div><div><strong>Paradoxical association of TET loss of function with genome-wide DNA hypomethylation</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:16933-16942</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1903059116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071273156" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31371502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1903059116&amp;pmid=31371502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Masaki, H. ∙ Nishida, T. ∙ Kitajima, S. ...</div><div><strong>Developmental pluripotency-associated 4 (DPPA4) localized in active chromatin inhibits mouse embryonic stem cell differentiation into a primitive ectoderm lineage</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2007; <strong>282</strong>:33034-33042</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1074%2Fjbc.M703245200&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1074%2Fjbc.M703245200&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-36349037021" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17855347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M703245200&amp;pmid=17855347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Meir, Z. ∙ Mukamel, Z. ∙ Chomsky, E. ...</div><div><strong>Single-cell analysis of clonal maintenance of transcriptional and epigenetic states in cancer cells</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:709-718</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0645-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85087038708" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32601473/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0645-y&amp;pmid=32601473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Mittnenzweig, M. ∙ Mayshar, Y. ∙ Cheng, S. ...</div><div><strong>A single-embryo, single-cell time-resolved model for mouse gastrulation</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2825-2842.e22</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.04.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85107319480" target="_blank" aria-label="Open Scopus in new tab">Scopus (92)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33932341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.04.004&amp;pmid=33932341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Mohammed, H. ∙ Hernando-Herraez, I. ∙ Savino, A. ...</div><div><strong>Single-cell landscape of transcriptional heterogeneity and cell fate decisions during mouse early gastrulation</strong></div><div><em>Cell Rep.</em> 2017; <strong>20</strong>:1215-1228</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2017.07.009&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2017.07.009&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85026887626" target="_blank" aria-label="Open Scopus in new tab">Scopus (198)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28768204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2017.07.009&amp;pmid=28768204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Moran-Crusio, K. ∙ Reavie, L. ∙ Shih, A. ...</div><div><strong>Tet2 loss leads to increased hematopoietic stem cell self-renewal and myeloid transformation</strong></div><div><em>Cancer Cell.</em> 2011; <strong>20</strong>:11-24</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.ccr.2011.06.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.ccr.2011.06.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79960064353" target="_blank" aria-label="Open Scopus in new tab">Scopus (1058)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21723200/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ccr.2011.06.001&amp;pmid=21723200" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Moris, N. ∙ Pina, C. ∙ Arias, A.M.</div><div><strong>Transition states and cell fate decisions in epigenetic landscapes</strong></div><div><em>Nat. Rev. Genet.</em> 2016; <strong>17</strong>:693-703</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg.2016.98" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84991585454" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27616569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg.2016.98&amp;pmid=27616569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">45.</a></div><div>Nagano, T. ∙ Lubling, Y. ∙ Várnai, C. ...</div><div><strong>Cell-cycle dynamics of chromosomal organization at single-cell resolution</strong></div><div><em>Nature.</em> 2017; <strong>547</strong>:61-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85022195504" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28682332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23001&amp;pmid=28682332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Nagy, A. ∙ Gócza, E. ∙ Diaz, E.M. ...</div><div><strong>Embryonic stem cells alone are able to support fetal development in the mouse</strong></div><div><em>Development.</em> 1990; <strong>110</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.110.3.815" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2088722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.110.3.815&amp;pmid=2088722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Nagy, A. ∙ Rossant, J. ∙ Nagy, R. ...</div><div><strong>Derivation of completely cell culture-derived mice from early-passage embryonic stem cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 1993; <strong>90</strong>:8424-8428</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.90.18.8424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0027321548" target="_blank" aria-label="Open Scopus in new tab">Scopus (2011)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8378314/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.90.18.8424&amp;pmid=8378314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Nowotschin, S. ∙ Setty, M. ∙ Kuo, Y.-Y. ...</div><div><strong>The emergent landscape of the mouse gut endoderm at single-cell resolution</strong></div><div><em>Nature.</em> 2019; <strong>569</strong>:361-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1127-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85065227179" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30959515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1127-1&amp;pmid=30959515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Nuñez, J.K. ∙ Chen, J. ∙ Pommier, G.C. ...</div><div><strong>Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2503-2519.e17</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.03.025&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2021.03.025&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85105829180" target="_blank" aria-label="Open Scopus in new tab">Scopus (304)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33838111/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.025&amp;pmid=33838111" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Okuda, A. ∙ Fukushima, A. ∙ Nishimoto, M. ...</div><div><strong>UTF1, a novel transcriptional coactivator expressed in pluripotent embryonic stem cells and extra-embryonic cells</strong></div><div><em>EMBO J.</em> 1998; <strong>17</strong>:2019-2032</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/emboj/17.7.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0032055125" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9524124/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Femboj%2F17.7.2019&amp;pmid=9524124" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Peitz, M. ∙ Pfannkuche, K. ∙ Rajewsky, K. ...</div><div><strong>Ability of the hydrophobic FGF and basic TAT peptides to promote cellular uptake of recombinant Cre recombinase: A tool for efficient genetic engineering of mammalian genomes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:4489-4494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.032068699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0037007121" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11904364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.032068699&amp;pmid=11904364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Peng, G. ∙ Suo, S. ∙ Cui, G. ...</div><div><strong>Molecular architecture of lineage allocation and tissue organization in early mouse embryo</strong></div><div><em>Nature.</em> 2019; <strong>572</strong>:528-532</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1469-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85070308555" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31391582/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1469-8&amp;pmid=31391582" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">53.</a></div><div>Piette, D. ∙ Hendrickx, M. ∙ Willems, E. ...</div><div><strong>An optimized procedure for whole-mount in&nbsp;situ hybridization on mouse embryos and embryoid bodies</strong></div><div><em>Nat. Protoc.</em> 2008; <strong>3</strong>:1194-1201</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2008.103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-50249113849" target="_blank" aria-label="Open Scopus in new tab">Scopus (65)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18600225/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2008.103&amp;pmid=18600225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">54.</a></div><div>Pijuan-Sala, B. ∙ Griffiths, J.A. ∙ Guibentif, C. ...</div><div><strong>A single-cell molecular map of mouse gastrulation and early organogenesis</strong></div><div><em>Nature.</em> 2019; <strong>566</strong>:490-495</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-0933-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85062300354" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30787436/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-0933-9&amp;pmid=30787436" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Pijuan-Sala, B. ∙ Wilson, N.K. ∙ Xia, J. ...</div><div><strong>Single-cell chromatin accessibility maps reveal regulatory programs driving early mouse organogenesis</strong></div><div><em>Nat. Cell Biol.</em> 2020; <strong>22</strong>:487-497</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41556-020-0489-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85082943514" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32231307/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41556-020-0489-9&amp;pmid=32231307" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Pope, B.D. ∙ Ryba, T. ∙ Dileep, V. ...</div><div><strong>Topologically associating domains are stable units of replication-timing regulation</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:402-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13986" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84911478490" target="_blank" aria-label="Open Scopus in new tab">Scopus (614)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409831/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13986&amp;pmid=25409831" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Rais, Y. ∙ Zviran, A. ∙ Geula, S. ...</div><div><strong>Deterministic direct reprogramming of somatic cells to pluripotency</strong></div><div><em>Nature.</em> 2013; <strong>502</strong>:65-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12587" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84885619736" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24048479/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12587&amp;pmid=24048479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Saga, Y. ∙ Hata, N. ∙ Kobayashi, S. ...</div><div><strong>MesP1: a novel basic helix-loop-helix protein expressed in the nascent mesodermal cells during mouse gastrulation</strong></div><div><em>Development.</em> 1996; <strong>122</strong>:2769-2778</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.122.9.2769" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/8787751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.122.9.2769&amp;pmid=8787751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>Schneider, C.A. ∙ Rasband, W.S. ∙ Eliceiri, K.W.</div><div><strong>NIH Image to ImageJ: 25 years of image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:671-675</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84863205849" target="_blank" aria-label="Open Scopus in new tab">Scopus (43944)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2089&amp;pmid=22930834" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">60.</a></div><div>Scialdone, A. ∙ Tanaka, Y. ∙ Jawaid, W. ...</div><div><strong>Resolving early mesoderm diversification through single-cell expression profiling</strong></div><div><em>Nature.</em> 2016; <strong>535</strong>:289-293</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18633" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84978761773" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27383781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18633&amp;pmid=27383781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>Smith, Z.D. ∙ Shi, J. ∙ Gu, H. ...</div><div><strong>Epigenetic restriction of extraembryonic lineages mirrors the somatic transition to cancer</strong></div><div><em>Nature.</em> 2017; <strong>549</strong>:543-547</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature23891" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85032626002" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28959968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature23891&amp;pmid=28959968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Song, Y. ∙ van den Berg, P.R. ∙ Markoulaki, S. ...</div><div><strong>Dynamic enhancer DNA methylation as basis for transcriptional and cellular heterogeneity of ESCs</strong></div><div><em>Mol. Cell.</em> 2019; <strong>75</strong>:905-920.e6</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.molcel.2019.06.045&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85071542699" target="_blank" aria-label="Open Scopus in new tab">Scopus (64)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31422875/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2019.06.045&amp;pmid=31422875" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Srivatsan, S.R. ∙ Regier, M.C. ∙ Barkan, E. ...</div><div><strong>Embryo-scale, single-cell spatial transcriptomics</strong></div><div><em>Science.</em> 2021; <strong>373</strong>:111-117</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abb9536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85109116319" target="_blank" aria-label="Open Scopus in new tab">Scopus (136)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34210887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abb9536&amp;pmid=34210887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">64.</a></div><div>Stelzer, Y. ∙ Shivalila, C.S. ∙ Soldner, F. ...</div><div><strong>Tracing dynamic changes of DNA methylation at single-cell resolution</strong></div><div><em>Cell.</em> 2015; <strong>163</strong>:218-229</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2015.08.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.cell.2015.08.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84948451913" target="_blank" aria-label="Open Scopus in new tab">Scopus (104)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26406378/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.08.046&amp;pmid=26406378" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Tahiliani, M. ∙ Koh, K.P. ∙ Shen, Y. ...</div><div><strong>Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1</strong></div><div><em>Science.</em> 2009; <strong>324</strong>:930-935</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1170116" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-66149146320" target="_blank" aria-label="Open Scopus in new tab">Scopus (4619)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19372391/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1170116&amp;pmid=19372391" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Tam, P.P.L. ∙ Behringer, R.R.</div><div><strong>Mouse gastrulation: the formation of a mammalian body plan</strong></div><div><em>Mech. Dev.</em> 1997; <strong>68</strong>:3-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0925-4773(97)00123-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-0031464087" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9431800/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0925-4773%2897%2900123-8&amp;pmid=9431800" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Tam, P.P.L. ∙ Loebel, D.A.F.</div><div><strong>Gene function in mouse embryogenesis: get set for gastrulation</strong></div><div><em>Nat. Rev. Genet.</em> 2007; <strong>8</strong>:368-381</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg2084" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-34247226529" target="_blank" aria-label="Open Scopus in new tab">Scopus (463)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17387317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg2084&amp;pmid=17387317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>van den Brink, S.C. ∙ Alemany, A. ∙ van Batenburg, V. ...</div><div><strong>Single-cell and spatial transcriptomics reveal somitogenesis in gastruloids</strong></div><div><em>Nature.</em> 2020; <strong>582</strong>:405-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2024-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85079785374" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32076263/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2024-3&amp;pmid=32076263" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">69.</a></div><div>Vincent, J.J. ∙ Huang, Y. ∙ Chen, P.-Y. ...</div><div><strong>Stage-specific roles for Tet1 and Tet2 in DNA demethylation in primordial germ cells</strong></div><div><em>Cell Stem Cell.</em> 2013; <strong>12</strong>:470-478</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.01.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.01.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84875949201" target="_blank" aria-label="Open Scopus in new tab">Scopus (147)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.01.016&amp;pmid=23415914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Wang, Q. ∙ Yu, G. ∙ Ming, X. ...</div><div><strong>Imprecise DNMT1 activity coupled with neighbor-guided correction enables robust yet flexible epigenetic inheritance</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:828-839</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-020-0661-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85088264348" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32690947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-020-0661-y&amp;pmid=32690947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">71.</a></div><div>Williams, K. ∙ Christensen, J. ∙ Pedersen, M.T. ...</div><div><strong>TET1 and hydroxymethylcytosine in transcription and DNA methylation fidelity</strong></div><div><em>Nature.</em> 2011; <strong>473</strong>:343-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-79956302047" target="_blank" aria-label="Open Scopus in new tab">Scopus (846)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21490601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10066&amp;pmid=21490601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">72.</a></div><div>Wu, Y. ∙ Liang, D. ∙ Wang, Y. ...</div><div><strong>Correction of a genetic disease in mouse via use of CRISPR-Cas9</strong></div><div><em>Cell Stem Cell.</em> 2013; <strong>13</strong>:659-662</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.10.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2013.10.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84890050551" target="_blank" aria-label="Open Scopus in new tab">Scopus (494)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24315440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.10.016&amp;pmid=24315440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73-1" title="View in article">73.</a></div><div>Xiang, Y. ∙ Zhang, Y. ∙ Xu, Q. ...</div><div><strong>Epigenomic analysis of gastrulation identifies a unique chromatin state for primed pluripotency</strong></div><div><em>Nat. Genet.</em> 2020; <strong>52</strong>:95-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0545-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85076571979" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31844322/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0545-1&amp;pmid=31844322" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">74.</a></div><div>Xiao, C. ∙ Nitsche, F. ∙ Bazzi, H.</div><div><strong>Visualizing the node and notochordal plate in gastrulating mouse embryos using scanning electron microscopy and whole mount immunofluorescence</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2018; <strong>141</strong>:e58321</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3791/58321" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85057071924" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3791%2F58321" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">75.</a></div><div>Yamaguchi, S. ∙ Hong, K. ∙ Liu, R. ...</div><div><strong>Tet1 controls meiosis by regulating meiotic gene expression</strong></div><div><em>Nature.</em> 2012; <strong>492</strong>:443-447</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11709" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84871438065" target="_blank" aria-label="Open Scopus in new tab">Scopus (243)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23151479/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11709&amp;pmid=23151479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">76.</a></div><div>Yamaguchi, S. ∙ Shen, L. ∙ Liu, Y. ...</div><div><strong>Role of Tet1 in erasure of genomic imprinting</strong></div><div><em>Nature.</em> 2013; <strong>504</strong>:460-464</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12805" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84890568163" target="_blank" aria-label="Open Scopus in new tab">Scopus (202)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24291790/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12805&amp;pmid=24291790" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">77.</a></div><div>Zhang, X. ∙ Su, J. ∙ Jeong, M. ...</div><div><strong>DNMT3A and TET2 compete and cooperate to repress lineage-specific transcription factors in hematopoietic stem cells</strong></div><div><em>Nat. Genet.</em> 2016; <strong>48</strong>:1014-1023</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3610" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84978680214" target="_blank" aria-label="Open Scopus in new tab">Scopus (187)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27428748/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3610&amp;pmid=27428748" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">78.</a></div><div>Zhao, Z. ∙ Chen, L. ∙ Dawlaty, M.M. ...</div><div><strong>Combined loss of Tet1 and Tet2 promotes B cell, but not myeloid malignancies, in mice</strong></div><div><em>Cell Rep.</em> 2015; <strong>13</strong>:1692-1704</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2015.10.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.celrep.2015.10.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-84952874773" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26586431/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2015.10.037&amp;pmid=26586431" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">79.</a></div><div>Zviran, A. ∙ Mor, N. ∙ Rais, Y. ...</div><div><strong>Deterministic somatic cell reprogramming involves continuous transcriptional changes governed by Myc and epigenetic-driven modules</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:328-341.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2018.11.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=10.1016%2Fj.stem.2018.11.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.06.049&amp;key=2-s2.0-85060996758" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30554962/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2018.11.014&amp;pmid=30554962" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (12)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/27232823-6e22-4543-86a3-a72f1bb9f3ff/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/8cd67a8a-9f00-4334-801b-bffa145f80fd/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/f4ac48f8-82de-42f0-845a-ec2b8f786218/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/cb8b6ecd-9a64-4b5a-96bd-8afe4ff7d0fa/main.assets/gr2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/7004aef9-525b-4316-990b-d6d6d143723d/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/feb40f6c-f328-4228-bb65-b05fd56ddac2/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/ac654048-6763-4dee-a690-071d63aee9cd/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/b15f54dc-0655-492a-ab7d-4e591ae734b9/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/288d8c90-53fc-4834-ab6e-916291acf84f/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/ebac1e04-366c-4b52-9fad-8c2da1b5f204/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/1d6607e8-f368-42ec-9059-0395756cea3c/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/asset/193b5ebf-ad5f-44e0-90d9-d91edf044e63/main.assets/gr6.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3Barticle%3Aarticle%3Apii%5C%3AS009286742200842X%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00189%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (3)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/attachment/03b48e99-f0de-4d9d-93dc-2b640ecc9aec/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (45.13 KB)</span></a><div class="core-description">Table&nbsp;S1. Relative cell-type composition for mixed and whole-embryo chimera with transcriptional age &gt;7.75, related to Figures&nbsp;1, 2, and 4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/attachment/1558c7fa-9dda-4657-a53f-a64b13f89f68/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (80.84 KB)</span></a><div class="core-description">Table&nbsp;S2. List of exons and H3K4me3 unmarked promoters that are hypomethylated in Tet-TKO cells, related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.06.049/attachment/c9e1f813-f868-4ca4-8dc0-72d8b3f70b18/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (26.00 KB)</span></a><div class="core-description">Table&nbsp;S3. Transcription factor binding site enrichment analysis for hypermethylated enhancers in Tet-TKO cells, related to Figure&nbsp;5</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S009286742200842X" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.06.049","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">The intrinsic and extrinsic effects of TET proteins during gastrulation</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.049/asset/27232823-6e22-4543-86a3-a72f1bb9f3ff/main.assets/gr1.jpg" height="940" width="691" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/a608eaf2-6444-45e6-bb5e-600439f2082f/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.049/asset/8cd67a8a-9f00-4334-801b-bffa145f80fd/main.assets/figs1.jpg" height="895" width="761" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/e56e1776-856c-4a59-bd39-376a65db6940/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.049/asset/f4ac48f8-82de-42f0-845a-ec2b8f786218/main.assets/figs2.jpg" height="941" width="654" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/47fc4a49-4c0a-4f46-bdc6-9b451ecc15d2/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.049/asset/cb8b6ecd-9a64-4b5a-96bd-8afe4ff7d0fa/main.assets/gr2.jpg" height="938" width="760" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/f724d68e-5b28-4271-ab18-c7f8042e2064/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.049/asset/7004aef9-525b-4316-990b-d6d6d143723d/main.assets/figs3.jpg" height="941" width="707" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/7ac14406-02b2-4830-9085-079abf9baac5/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.049/asset/feb40f6c-f328-4228-bb65-b05fd56ddac2/main.assets/gr3.jpg" height="940" width="661" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/75206a90-365b-4381-ac1e-887550ea3b3b/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.049/asset/ac654048-6763-4dee-a690-071d63aee9cd/main.assets/figs4.jpg" height="839" width="762" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/06bc28d7-67d2-4217-9c05-ba5262172949/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.049/asset/b15f54dc-0655-492a-ab7d-4e591ae734b9/main.assets/figs5.jpg" height="941" width="674" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/558dac20-94e6-4346-aa4e-817c999e1d8e/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.049/asset/288d8c90-53fc-4834-ab6e-916291acf84f/main.assets/gr4.jpg" height="812" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/b51cf184-797b-4178-86fa-1e290db072db/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.049/asset/ebac1e04-366c-4b52-9fad-8c2da1b5f204/main.assets/gr5.jpg" height="592" width="761" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/138a9dec-8112-4930-ba09-b52d35372770/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.049/asset/1d6607e8-f368-42ec-9059-0395756cea3c/main.assets/figs6.jpg" height="942" width="734" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/e434c13b-5558-497f-b964-caa444b64c79/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.049/asset/193b5ebf-ad5f-44e0-90d9-d91edf044e63/main.assets/gr6.jpg" height="881" width="753" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/16829617-bcaf-4f0f-a6a7-3b06d36bce96/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.06.049/asset/27232823-6e22-4543-86a3-a72f1bb9f3ff/main.assets/gr1.jpg" height="940" width="691" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/a608eaf2-6444-45e6-bb5e-600439f2082f/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.06.049/asset/8cd67a8a-9f00-4334-801b-bffa145f80fd/main.assets/figs1.jpg" height="895" width="761" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/e56e1776-856c-4a59-bd39-376a65db6940/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.06.049/asset/f4ac48f8-82de-42f0-845a-ec2b8f786218/main.assets/figs2.jpg" height="941" width="654" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/47fc4a49-4c0a-4f46-bdc6-9b451ecc15d2/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.06.049/asset/cb8b6ecd-9a64-4b5a-96bd-8afe4ff7d0fa/main.assets/gr2.jpg" height="938" width="760" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/f724d68e-5b28-4271-ab18-c7f8042e2064/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.06.049/asset/7004aef9-525b-4316-990b-d6d6d143723d/main.assets/figs3.jpg" height="941" width="707" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/7ac14406-02b2-4830-9085-079abf9baac5/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.06.049/asset/feb40f6c-f328-4228-bb65-b05fd56ddac2/main.assets/gr3.jpg" height="940" width="661" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/75206a90-365b-4381-ac1e-887550ea3b3b/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.06.049/asset/ac654048-6763-4dee-a690-071d63aee9cd/main.assets/figs4.jpg" height="839" width="762" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/06bc28d7-67d2-4217-9c05-ba5262172949/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.06.049/asset/b15f54dc-0655-492a-ab7d-4e591ae734b9/main.assets/figs5.jpg" height="941" width="674" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/558dac20-94e6-4346-aa4e-817c999e1d8e/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.06.049/asset/288d8c90-53fc-4834-ab6e-916291acf84f/main.assets/gr4.jpg" height="812" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/b51cf184-797b-4178-86fa-1e290db072db/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.06.049/asset/ebac1e04-366c-4b52-9fad-8c2da1b5f204/main.assets/gr5.jpg" height="592" width="761" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/138a9dec-8112-4930-ba09-b52d35372770/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.06.049/asset/1d6607e8-f368-42ec-9059-0395756cea3c/main.assets/figs6.jpg" height="942" width="734" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/e434c13b-5558-497f-b964-caa444b64c79/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.06.049/asset/193b5ebf-ad5f-44e0-90d9-d91edf044e63/main.assets/gr6.jpg" height="881" width="753" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.06.049/asset/16829617-bcaf-4f0f-a6a7-3b06d36bce96/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00842-X';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00842-X';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749594229080","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"185","issueNumber":"17","viewState":"fulltext","format":"MIME-HTML","id":"pii:S009286742200842X","coverDate":"202208","type":"XOCS-JOURNAL:SCOPE-FULL","title":"The intrinsic and extrinsic effects of TET proteins during gastrulation","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc46740a894293',t:'MTc0OTU5NDIyOC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc46740a894293&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.06.049&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>