<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="6da32839-2fd6-4557-8d77-7c3c7ec8ac17" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867421010503;issue:issue:pii\:S0092867420X00213;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="The emergence and ongoing convergent evolution of the SARS-CoV-2 N501Y lineages">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01050-3">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/89854977-b07c-4f39-812b-23a1ee593e87/fx1.jpg">
    


        <meta property="og:description" content="An analysis of synonymous and non-synonymous mutations in SARS-CoV-2 genomes since
the inception of the COVID-19 pandemic provides insights into the emergence of a convergent
mutational signature in the 501Y lineage (alpha, beta, and gamma variants) that is
also likely present in other lineages that impacts host immune recognition.">
    











        <meta name="twitter:title" content="The emergence and ongoing convergent evolution of the SARS-CoV-2 N501Y lineages">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/89854977-b07c-4f39-812b-23a1ee593e87/fx1.jpg">
    


        <meta name="twitter:description" content="An analysis of synonymous and non-synonymous mutations in SARS-CoV-2 genomes since
the inception of the COVID-19 pandemic provides insights into the emergence of a convergent
mutational signature in the 501Y lineage (alpha, beta, and gamma variants) that is
also likely present in other lineages that impacts host immune recognition.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01050-3">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>The emergence and ongoing convergent evolution of the SARS-CoV-2 N501Y lineages: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="The emergence and ongoing convergent evolution of the SARS-CoV-2 N501Y lineages"> <meta name="citation_date" content="2021/09/30"> <meta name="citation_online_date" content="2021/09/06"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="20"> <meta name="citation_firstpage" content="5189"> <meta name="citation_lastpage" content="5200.e7"> <meta name="citation_pmid" content="34537136"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421010503"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421010503"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421010503"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.09.003"> <meta name="citation_pmid" content="34537136"> <meta name="citation_author" content="Darren P. Martin"><meta name="citation_author_email" content="darrenpatrickmartin@gmail.com"><meta name="citation_author_institution" content="Institute of Infectious Diseases and Molecular Medicine, Division Of Computational Biology, Department of Integrative Biomedical Sciences, University of Cape Town, Cape Town 7701, South Africa"><meta name="citation_author" content="Steven Weaver"><meta name="citation_author_orcid" content="0000-0002-6931-7191"><meta name="citation_author_institution" content="Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA"><meta name="citation_author" content="Houriiyah Tegally"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="James Emmanuel San"><meta name="citation_author_orcid" content="0000-0002-5736-664X"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Stephen D. Shank"><meta name="citation_author_institution" content="Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA"><meta name="citation_author" content="Eduan Wilkinson"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Alexander G. Lucaci"><meta name="citation_author_institution" content="Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA"><meta name="citation_author" content="Jennifer Giandhari"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Sureshnee Naidoo"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Yeshnee Pillay"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Lavanya Singh"><meta name="citation_author_orcid" content="0000-0002-1726-4454"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Richard J. Lessells"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author" content="Ravindra K. Gupta"><meta name="citation_author_institution" content="Clinical Microbiology, University of Cambridge, Cambridge CB2 1TN, UK"><meta name="citation_author_institution" content="Africa Health Research Institute, KwaZulu-Natal 4013, South Africa"><meta name="citation_author" content="Joel O. Wertheim"><meta name="citation_author_institution" content="Department of Medicine, University of California San Diego, La Jolla, CA 92093, USA"><meta name="citation_author" content="Anton Nekturenko"><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University, State College, PA 16802, USA"><meta name="citation_author" content="Ben Murrell"><meta name="citation_author_institution" content="Department of Microbiology, Tumor and Cell Biology, Karolinska Institutet, Stockholm 141 83, Sweden"><meta name="citation_author" content="Gordon W. Harkins"><meta name="citation_author_institution" content="South African Medical Research Council Capacity Development Unit, South African National Bioinformatics Institute, University of the Western Cape, Bellville 7635, South Africa"><meta name="citation_author" content="Philippe Lemey"><meta name="citation_author_institution" content="Department of Microbiology, Immunology and Transplantation, Rega Institute, KU Leuven, Leuven 3000, Belgium"><meta name="citation_author" content="Oscar A. MacLean"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, Glasgow 12 8QQ, Scotland, UK"><meta name="citation_author" content="David L. Robertson"><meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, Glasgow 12 8QQ, Scotland, UK"><meta name="citation_author" content="Tulio de Oliveira"><meta name="citation_author_email" content="tuliodna@gmail.com"><meta name="citation_author_orcid" content="0000-0002-3027-5254"><meta name="citation_author_institution" content="KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa"><meta name="citation_author_institution" content="Department of Global Health, University of Washington, Seattle, WA 98195-4550, USA"><meta name="citation_author" content="Sergei L. Kosakovsky Pond"><meta name="citation_author_email" content="spond@temple.edu"><meta name="citation_author_institution" content="Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA"><meta name="citation_keywords" content="COVID 19; evolutionary adaptation; positive selection; directional selection; recurrent mutations; convergent mutations; diversifying selection; immune evasion; transmission advantage; lineage-defining mutations"><meta name="citation_reference" content="citation_title=A tale of two variants: Spread of SARS-CoV-2 variants Alpha in Geneva, Switzerland, and Beta in South Africa; citation_author=C.L. Althaus; citation_author=S. Baggio; citation_author=M.L. Reichmuth; citation_author=E.B. Hodcroft; citation_author=J. Riou; citation_author=R.A. Neher; citation_author=F. Jacquerioz; citation_author=H. Spechbach; citation_author=J. Salamun; citation_author=P. Vetter; citation_author=et al.; citation_journal_title=MedRxiv; citation_doi=10.1101/2021.02.23.21252268; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A Novel SARS-CoV-2 Variant of Concern, B.1.526, Identified in New York; citation_author=M.K. Annavajhala; citation_author=H. Mohri; citation_author=J.E. Zucker; citation_author=Z. Sheng; citation_author=P. Wang; citation_author=A. Gomez-Simmonds; citation_author=D.D. Ho; citation_author=A.-C. Uhlemann; citation_journal_title=MedRxiv; citation_doi=10.1101/2021.02.23.21252259; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles; citation_author=K.M. Campbell; citation_author=G. Steiner; citation_author=D.K. Wells; citation_author=A. Ribas; citation_author=A. Kalbasi; citation_journal_title=BioRxiv; citation_doi=10.1101/2020.03.30.016931; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Escape of SARS-CoV-2 501Y.V2 from neutralization by convalescent plasma; citation_author=S. Cele; citation_author=I. Gazy; citation_author=L. Jackson; citation_author=S.-H. Hwa; citation_author=H. Tegally; citation_author=G. Lustig; citation_author=J. Giandhari; citation_author=S. Pillay; citation_author=E. Wilkinson; citation_author=Y. Naidoo; citation_author=Network for Genomic Surveillance in South Africa; citation_author=COMMIT-KZN Team; citation_author=et al.; citation_journal_title=Nature; citation_volume=593; citation_firstpage=142; citation_lastpage=146; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Sensitivity of SARS-CoV-2 B.1.1.7 to mRNA vaccine-elicited antibodies; citation_author=D.A. Collier; citation_author=A. De Marco; citation_author=I.A.T.M. Ferreira; citation_author=B. Meng; citation_author=R.P. Datir; citation_author=A.C. Walls; citation_author=S.A. Kemp; citation_author=J. Bassi; citation_author=D. Pinto; citation_author=C. Silacci-Fregni; citation_author=CITIID-NIHR BioResource COVID-19 Collaboration; citation_author=COVID-19 Genomics UK (COG-UK) Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=593; citation_firstpage=136; citation_lastpage=141; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Coronavirus nsp6 proteins generate autophagosomes from the endoplasmic reticulum via an omegasome intermediate; citation_author=E.M. Cottam; citation_author=H.J. Maier; citation_author=M. Manifava; citation_author=L.C. Vaux; citation_author=P. Chandra-Schoenfelder; citation_author=W. Gerner; citation_author=P. Britton; citation_author=N.T. Ktistakis; citation_author=T. Wileman; citation_journal_title=Autophagy; citation_volume=7; citation_firstpage=1335; citation_lastpage=1347; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A novel variant of&nbsp;interest of SARS-CoV-2 with multiple spike mutations is identified from travel surveillance in Africa; citation_author=T. de Oliveira; citation_author=S. Lutucuta; citation_author=J. Nkengasong; citation_author=J. Morais; citation_author=J. Paula Paixao; citation_author=Z. Neto; citation_author=P. Afonso; citation_author=J. Miranda; citation_author=K. David; citation_author=L. Ingles; citation_author=et al.; citation_journal_title=MedRxiv; citation_doi=10.1101/2021.02.23.21252268; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants; citation_author=B. Dearlove; citation_author=E. Lewitus; citation_author=H. Bai; citation_author=Y. Li; citation_author=D.B. Reeves; citation_author=M.G. Joyce; citation_author=P.T. Scott; citation_author=M.F. Amare; citation_author=S. Vasan; citation_author=N.L. Michael; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=23652; citation_lastpage=23662; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Travel-driven emergence and spread of SARS-CoV-2 lineage B.1.620 with multiple VOC-like mutations and deletions in Europe; citation_author=G. Dudas; citation_author=S.L. Hong; citation_author=B.I. Potter; citation_author=S. Calvignac-Spencer; citation_author=F.S. Niatou-Singa; citation_author=T.B. Tombolomako; citation_author=T. Fuh-Neba; citation_author=U. Vickos; citation_author=M. Ulrich; citation_author=F.H. Leendertz; citation_author=et al.; citation_journal_title=MedRxiv; citation_doi=10.1101/2021.05.04.21256637v1; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Data, disease and diplomacy: GISAID’s innovative contribution to global health; citation_author=S. Elbe; citation_author=G. Buckland-Merrett; citation_journal_title=Glob. Chall.; citation_volume=1; citation_firstpage=33; citation_lastpage=46; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil; citation_author=N.R. Faria; citation_author=T.A. Mellan; citation_author=C. Whittaker; citation_author=I.M. Claro; citation_author=D.D.S. Candido; citation_author=S. Mishra; citation_author=M.A.E. Crispim; citation_author=F.C.S. Sales; citation_author=I. Hawryluk; citation_author=J.T. McCrone; citation_author=et al.; citation_journal_title=Science; citation_volume=372; citation_firstpage=815; citation_lastpage=821; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Multiple SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity; citation_author=W.F. Garcia-Beltran; citation_author=E.C. Lam; citation_author=K. St Denis; citation_author=A.D. Nitido; citation_author=Z.H. Garcia; citation_author=B.M. Hauser; citation_author=J. Feldman; citation_author=M.N. Pavlovic; citation_author=D.J. Gregory; citation_author=M.C. Poznansky; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=2372; citation_lastpage=2383; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Spike protein mutations in novel SARS-CoV-2 ‘variants of concern’ commonly occur&nbsp;in or near indels; citation_author=R.F. Garry; citation_author=K.G. Andersen; citation_author=W.R. Gallaher; citation_author=T.T.-Y. Lam; citation_author=K. Gangaparapu; citation_author=A.A. Latif; citation_author=B.J. Beddingfield; citation_author=A. Rambaut; citation_author=E.C. Holmes; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Potentially adaptive SARS-CoV-2 mutations discovered with novel spatiotemporal and explainable AI models; citation_author=M.R. Garvin; citation_author=E. T Prates; citation_author=M. Pavicic; citation_author=P. Jones; citation_author=B.K. Amos; citation_author=A. Geiger; citation_author=M.B. Shah; citation_author=J. Streich; citation_author=J.G. Felipe Machado Gazolla; citation_author=D. Kainer; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=21; citation_firstpage=304; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies; citation_author=A.J. Greaney; citation_author=A.N. Loes; citation_author=K.H.D. Crawford; citation_author=T.N. Starr; citation_author=K.D. Malone; citation_author=H.Y. Chu; citation_author=J.D. Bloom; citation_journal_title=Cell Host Microbe; citation_volume=29; citation_firstpage=463; citation_lastpage=476; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition; citation_author=A.J. Greaney; citation_author=T.N. Starr; citation_author=P. Gilchuk; citation_author=S.J. Zost; citation_author=E. Binshtein; citation_author=A.N. Loes; citation_author=S.K. Hilton; citation_author=J. Huddleston; citation_author=R. Eguia; citation_author=K.H.D. Crawford; citation_author=et al.; citation_journal_title=Cell Host Microbe; citation_volume=29; citation_firstpage=44; citation_lastpage=57; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Interferon Resistance of Emerging SARS-CoV-2 Variants; citation_author=K. Guo; citation_author=B.S. Barrett; citation_author=K.L. Mickens; citation_author=K.J. Hasenkrug; citation_author=M.L. Santiago; citation_journal_title=BioRxiv; citation_doi=10.1101/2021.03.20.436257; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 variants B.1.351 and P.1 escape from neutralizing antibodies; citation_author=M. Hoffmann; citation_author=P. Arora; citation_author=R. Groß; citation_author=A. Seidel; citation_author=B.F. Hörnich; citation_author=A.S. Hahn; citation_author=N. Krüger; citation_author=L. Graichen; citation_author=H. Hofmann-Winkler; citation_author=A. Kempf; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=2384; citation_lastpage=2393; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Non-parametric analysis of fatal outcomes associated with&nbsp;B1.1.7; citation_author=P. Horby; citation_author=C. Huntley; citation_author=N. Davies; citation_author=J. Edmunds; citation_author=N. Ferguson; citation_author=G. Medley; citation_author=C. Semple; citation_publication_date=2021; citation_publisher=Imperial College London; "><meta name="citation_reference" content="citation_title=Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis; citation_author=B.A. Johnson; citation_author=X. Xie; citation_author=A.L. Bailey; citation_author=B. Kalveram; citation_author=K.G. Lokugamage; citation_author=A. Muruato; citation_author=J. Zou; citation_author=X. Zhang; citation_author=T. Juelich; citation_author=J.K. Smith; citation_author=et al.; citation_journal_title=Nature; citation_volume=591; citation_firstpage=293; citation_lastpage=299; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform; citation_author=K. Katoh; citation_author=K. Misawa; citation_author=K. Kuma; citation_author=T. Miyata; citation_journal_title=Nucleic Acids Res.; citation_volume=30; citation_firstpage=3059; citation_lastpage=3066; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus; citation_author=B. Korber; citation_author=W.M. Fischer; citation_author=S. Gnanakaran; citation_author=H. Yoon; citation_author=J. Theiler; citation_author=W. Abfalterer; citation_author=N. Hengartner; citation_author=E.E. Giorgi; citation_author=T. Bhattacharya; citation_author=B. Foley; citation_author=Sheffield COVID-19 Genomics Group; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=812; citation_lastpage=827; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Not so different after all: a comparison of methods for detecting amino acid sites under selection; citation_author=S.L. Kosakovsky Pond; citation_author=S.D.W. Frost; citation_journal_title=Mol. Biol. Evol.; citation_volume=22; citation_firstpage=1208; citation_lastpage=1222; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=HIV-TRACE (TRAnsmission Cluster Engine): a Tool for Large Scale Molecular Epidemiology of HIV-1 and Other Rapidly Evolving Pathogens; citation_author=S.L. Kosakovsky Pond; citation_author=S. Weaver; citation_author=A.J. Leigh Brown; citation_author=J.O. Wertheim; citation_journal_title=Mol. Biol. Evol.; citation_volume=35; citation_firstpage=1812; citation_lastpage=1819; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies; citation_author=S.L. Kosakovsky Pond; citation_author=A.F.Y. Poon; citation_author=R. Velazquez; citation_author=S. Weaver; citation_author=N.L. Hepler; citation_author=B. Murrell; citation_author=S.D. Shank; citation_author=B.R. Magalis; citation_author=D. Bouvier; citation_author=A. Nekrutenko; citation_author=et al.; citation_journal_title=Mol. Biol. Evol.; citation_volume=37; citation_firstpage=295; citation_lastpage=299; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference; citation_author=A.M. Kozlov; citation_author=D. Darriba; citation_author=T. Flouri; citation_author=B. Morel; citation_author=A. Stamatakis; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=4453; citation_lastpage=4455; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Attenuated SARS-CoV-2 variants with deletions at the S1/S2 junction; citation_author=S.-Y. Lau; citation_author=P. Wang; citation_author=B.W.-Y. Mok; citation_author=A.J. Zhang; citation_author=H. Chu; citation_author=A.C.-Y. Lee; citation_author=S. Deng; citation_author=P. Chen; citation_author=K.-H. Chan; citation_author=W. Song; citation_author=et al.; citation_journal_title=Emerg. Microbes Infect.; citation_volume=9; citation_firstpage=837; citation_lastpage=842; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Activation and evasion of type I interferon responses by SARS-CoV-2; citation_author=X. Lei; citation_author=X. Dong; citation_author=R. Ma; citation_author=W. Wang; citation_author=X. Xiao; citation_author=Z. Tian; citation_author=C. Wang; citation_author=Y. Wang; citation_author=L. Li; citation_author=L. Ren; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=3810; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Functional evaluation of proteolytic activation for the SARS-CoV-2 variant B.1.1.7: role of the P681H mutation; citation_author=B. Lubinski; citation_author=T. Tang; citation_author=S. Daniel; citation_author=J.A. Jaimes; citation_author=G.R. Whittaker; citation_journal_title=BioRxiv; citation_doi=10.1101/2021.04.06.438731; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=RASCL: Rapid assessment of SARS-COV-2 clades enabled through molecular sequence analysis and its application to B.1.617.1 and B.1.617.2.; citation_author=A.G. Lucaci; citation_author=J.D. Zehr; citation_author=S.D. Shank; citation_author=D. Bouvier; citation_author=H. Mei; citation_author=A. Nekrutenko; citation_author=S.L. Kosakovsky Pond; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen; citation_author=O.A. MacLean; citation_author=S. Lytras; citation_author=S. Weaver; citation_author=J.B. Singer; citation_author=M.F. Boni; citation_author=P. Lemey; citation_author=S.L. Kosakovsky Pond; citation_author=D.L. Robertson; citation_journal_title=PLoS Biol.; citation_volume=19; citation_firstpage=e3001115; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2; citation_author=M. McCallum; citation_author=A. De Marco; citation_author=F.A. Lempp; citation_author=M.A. Tortorici; citation_author=D. Pinto; citation_author=A.C. Walls; citation_author=M. Beltramello; citation_author=A. Chen; citation_author=Z. Liu; citation_author=F. Zatta; citation_author=et al.; citation_journal_title=Cell; citation_volume=184; citation_firstpage=2332; citation_lastpage=2347; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 Orf6 hijacks Nup98 to block STAT nuclear import and antagonize interferon signaling; citation_author=L. Miorin; citation_author=T. Kehrer; citation_author=M.T. Sanchez-Aparicio; citation_author=K. Zhang; citation_author=P. Cohen; citation_author=R.S. Patel; citation_author=A. Cupic; citation_author=T. Makio; citation_author=M. Mei; citation_author=E. Moreno; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=28344; citation_lastpage=28354; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Detecting individual sites subject to episodic diversifying selection; citation_author=B. Murrell; citation_author=J.O. Wertheim; citation_author=S. Moola; citation_author=T. Weighill; citation_author=K. Scheffler; citation_author=S.L. Kosakovsky Pond; citation_journal_title=PLoS Genet.; citation_volume=8; citation_firstpage=e1002764; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=FUBAR: a fast, unconstrained bayesian approximation for inferring selection; citation_author=B. Murrell; citation_author=S. Moola; citation_author=A. Mabona; citation_author=T. Weighill; citation_author=D. Sheward; citation_author=S.L. Kosakovsky Pond; citation_author=K. Scheffler; citation_journal_title=Mol. Biol. Evol.; citation_volume=30; citation_firstpage=1196; citation_lastpage=1205; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Gene-wide identification of episodic selection; citation_author=B. Murrell; citation_author=S. Weaver; citation_author=M.D. Smith; citation_author=J.O. Wertheim; citation_author=S. Murrell; citation_author=A. Aylward; citation_author=K. Eren; citation_author=T. Pollner; citation_author=D.P. Martin; citation_author=D.M. Smith; citation_author=et al.; citation_journal_title=Mol. Biol. Evol.; citation_volume=32; citation_firstpage=1365; citation_lastpage=1371; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Molecular dynamic simulation reveals E484K mutation enhances spike RBD-ACE2 affinity and the combination of E484K, K417N and N501Y mutations (501Y.V2 variant) induces conformational change greater than N501Y mutant alone, potentially resulting in an escape mutant; citation_author=G. Nelson; citation_author=O. Buzko; citation_author=P. Spilman; citation_author=K. Niazi; citation_author=S. Rabizadeh; citation_author=P. Soon-Shiong; citation_journal_title=BioRxiv; citation_doi=10.1101/2021.01.13.426558; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Genomic mutations and changes in protein secondary structure and solvent accessibility of SARS-CoV-2 (COVID-19 virus); citation_author=T.T. Nguyen; citation_author=P.N. Pathirana; citation_author=T. Nguyen; citation_author=Q.V.H. Nguyen; citation_author=A. Bhatti; citation_author=D.C. Nguyen; citation_author=D.T. Nguyen; citation_author=N.D. Nguyen; citation_author=D. Creighton; citation_author=M. Abdelrazek; citation_journal_title=Sci. Rep.; citation_volume=11; citation_firstpage=3487; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=HIV-specific probabilistic models of protein evolution; citation_author=D.C. Nickle; citation_author=L. Heath; citation_author=M.A. Jensen; citation_author=P.B. Gilbert; citation_author=J.I. Mullins; citation_author=S.L. Kosakovsky Pond; citation_journal_title=PLoS ONE; citation_volume=2; citation_firstpage=e503; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 one year on: evidence for ongoing viral adaptation; citation_author=T.P. Peacock; citation_author=R. Penrice-Randal; citation_author=J.A. Hiscox; citation_author=W.S. Barclay; citation_journal_title=J.&nbsp;Gen. Virol.; citation_volume=102; citation_firstpage=001584; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Estimates of severity and transmissibility of novel SARS-CoV-2 variant 501Y.V2 in South Africa | CMMID Repository; citation_author=C.A. Pearson; citation_author=T.W. Russell; citation_author=N. Davies; citation_author=A.J. Kucharski; citation_journal_title=MedRxiv; citation_doi=10.1101/2020.12.24.20248822; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Spike mutation D614G alters SARS-CoV-2 fitness; citation_author=J.A. Plante; citation_author=Y. Liu; citation_author=J. Liu; citation_author=H. Xia; citation_author=B.A. Johnson; citation_author=K.G. Lokugamage; citation_author=X. Zhang; citation_author=A.E. Muruato; citation_author=J. Zou; citation_author=C.R. Fontes-Garfias; citation_author=et al.; citation_journal_title=Nature; citation_volume=592; citation_firstpage=116; citation_lastpage=121; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Adaptation to different human populations by HIV-1 revealed by codon-based analyses; citation_author=S.L.K. Pond; citation_author=S.D.W. Frost; citation_author=Z. Grossman; citation_author=M.B. Gravenor; citation_author=D.D. Richman; citation_author=A.J.L. Brown; citation_journal_title=PLoS Comput. Biol.; citation_volume=2; citation_firstpage=e62; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=An evolutionary-network model reveals stratified interactions in the V3 loop of the HIV-1 envelope; citation_author=A.F.Y. Poon; citation_author=F.I. Lewis; citation_author=S.L.K. Pond; citation_author=S.D.W. Frost; citation_journal_title=PLoS Comput. Biol.; citation_volume=3; citation_firstpage=e231; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Investigation of novel SARS-CoV-2 variant. Variant of Concern; citation_author=Public Health England; citation_publication_date=2020; citation_publisher=Public Health England; "><meta name="citation_reference" content="citation_title=Phylogenetic evidence for deleterious mutation load in RNA viruses and its contribution to viral evolution; citation_author=O.G. Pybus; citation_author=A. Rambaut; citation_author=R. Belshaw; citation_author=R.P. Freckleton; citation_author=A.J. Drummond; citation_author=E.C. Holmes; citation_journal_title=Mol. Biol. Evol.; citation_volume=24; citation_firstpage=845; citation_lastpage=852; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations; citation_author=A. Rambaut; citation_author=N. Loman; citation_author=O. Pybus; citation_author=W. Barclay; citation_author=J. Barrett; citation_author=A. Carabelli; citation_author=T. Connor; citation_author=T. Peacock; citation_author=D.L. Robertson; citation_author=E. Volz; citation_author=et al.; citation_publication_date=2020; citation_publisher=Virological; "><meta name="citation_reference" content="citation_title=A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology; citation_author=A. Rambaut; citation_author=E.C. Holmes; citation_author=Á. O’Toole; citation_author=V. Hill; citation_author=J.T. McCrone; citation_author=C. Ruis; citation_author=L. du Plessis; citation_author=O.G. Pybus; citation_journal_title=Nat. Microbiol.; citation_volume=5; citation_firstpage=1403; citation_lastpage=1407; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=National and International Dimensions of Human Immunodeficiency Virus-1 Sequence Clusters in a Northern California Clinical Cohort; citation_author=S.-Y. Rhee; citation_author=B.R. Magalis; citation_author=L. Hurley; citation_author=M.J. Silverberg; citation_author=J.L. Marcus; citation_author=S. Slome; citation_author=S.L. Kosakovsky Pond; citation_author=R.W. Shafer; citation_journal_title=Open Forum Infect. Dis.; citation_volume=6; citation_firstpage=ofz135; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=On the validity of evolutionary models with site-specific parameters; citation_author=K. Scheffler; citation_author=B. Murrell; citation_author=S.L. Kosakovsky Pond; citation_journal_title=PLoS ONE; citation_volume=9; citation_firstpage=e94534; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Efficacy of NVX-CoV2373 Covid-19 Vaccine against the B.1.351 Variant; citation_author=V. Shinde; citation_author=S. Bhikha; citation_author=Z. Hoosain; citation_author=M. Archary; citation_author=Q. Bhorat; citation_author=L. Fairlie; citation_author=U. Lalloo; citation_author=M.S.L. Masilela; citation_author=D. Moodley; citation_author=S. Hanley; citation_author=2019nCoV-501 Study Group; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=384; citation_firstpage=1899; citation_lastpage=1909; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Rapid neighbour-joining; citation_author=M. Simonsen; citation_author=T. Mailund; citation_author=C.N.S. Pedersen; citation_firstpage=113; citation_lastpage=122; citation_publication_date=2008; citation_publisher=Springer Berlin Heidelberg; "><meta name="citation_reference" content="citation_title=Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding; citation_author=T.N. Starr; citation_author=A.J. Greaney; citation_author=S.K. Hilton; citation_author=D. Ellis; citation_author=K.H.D. Crawford; citation_author=A.S. Dingens; citation_author=M.J. Navarro; citation_author=J.E. Bowen; citation_author=M.A. Tortorici; citation_author=A.C. Walls; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=1295; citation_lastpage=1310.e20; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Prospective mapping of viral mutations that escape antibodies used to treat COVID-19; citation_author=T.N. Starr; citation_author=A.J. Greaney; citation_author=A. Addetia; citation_author=W.W. Hannon; citation_author=M.C. Choudhary; citation_author=A.S. Dingens; citation_author=J.Z. Li; citation_author=J.D. Bloom; citation_journal_title=Science; citation_volume=371; citation_firstpage=850; citation_lastpage=854; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of an emergent SARS-CoV-2 variant characterized by multiple spike protein mutations detected from the Central Visayas Region of the Philippines; citation_author=F.A. Tablizo; citation_author=K.M. Kim; citation_author=C.M. Lapid; citation_author=M.J.R. Castro; citation_author=M.S.L. Yangzon; citation_author=B.A. Maralit; citation_author=M.E.C. Ayes; citation_author=E.M. Cutiongco-de la Paz; citation_author=A.R. De Guzman; citation_author=J.M.C. Yap; citation_author=et al.; citation_journal_title=MedRxiv; citation_doi=10.1101/2021.03.03.21252812; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Detection of a SARS-CoV-2 variant of concern in South Africa; citation_author=H. Tegally; citation_author=E. Wilkinson; citation_author=M. Giovanetti; citation_author=A. Iranzadeh; citation_author=V. Fonseca; citation_author=J. Giandhari; citation_author=D. Doolabh; citation_author=S. Pillay; citation_author=E.J. San; citation_author=N. Msomi; citation_author=et al.; citation_journal_title=Nature; citation_volume=592; citation_firstpage=438; citation_lastpage=443; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Evolution of enhanced innate immune evasion by the SARS-CoV-2 B.1.1.7&nbsp;UK variant; citation_author=L.G. Thorne; citation_author=M. Bouhaddou; citation_author=A.-K. Reuschl; citation_author=L. Zuliani-Alvarez; citation_author=B. Polacco; citation_author=A. Pelin; citation_author=J. Batra; citation_author=M.V.X. Whelan; citation_author=M. Ummadi; citation_author=A. Rojc; citation_author=et al.; citation_journal_title=BioRxiv; citation_doi=10.1101/2021.06.06.446826; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Assessing transmissibility of SARS-CoV-2 lineage B.1.1.7 in England; citation_author=E. Volz; citation_author=S. Mishra; citation_author=M. Chand; citation_author=J.C. Barrett; citation_author=R. Johnson; citation_author=L. Geidelberg; citation_author=W.R. Hinsley; citation_author=D.J. Laydon; citation_author=G. Dabrera; citation_author=Á. O’Toole; citation_author=COVID-19 Genomics UK (COG-UK) consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=593; citation_firstpage=266; citation_lastpage=269; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Antibody resistance of SARS-CoV-2 variants B.1.351 and B.1.1.7; citation_author=P. Wang; citation_author=M.S. Nair; citation_author=L. Liu; citation_author=S. Iketani; citation_author=Y. Luo; citation_author=Y. Guo; citation_author=M. Wang; citation_author=J. Yu; citation_author=B. Zhang; citation_author=P.D. Kwong; citation_author=et al.; citation_journal_title=Nature; citation_volume=593; citation_firstpage=130; citation_lastpage=135; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants; citation_author=Z. Wang; citation_author=F. Schmidt; citation_author=Y. Weisblum; citation_author=F. Muecksch; citation_author=C.O. Barnes; citation_author=S. Finkin; citation_author=D. Schaefer-Babajew; citation_author=M. Cipolla; citation_author=C. Gaebler; citation_author=J.A. Lieberman; citation_author=et al.; citation_journal_title=Nature; citation_volume=592; citation_firstpage=616; citation_lastpage=622; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma; citation_author=C.K. Wibmer; citation_author=F. Ayres; citation_author=T. Hermanus; citation_author=M. Madzivhandila; citation_author=P. Kgagudi; citation_author=B. Oosthuysen; citation_author=B.E. Lambson; citation_author=T. de Oliveira; citation_author=M. Vermeulen; citation_author=K. van der Berg; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=27; citation_firstpage=622; citation_lastpage=625; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine; citation_author=K. Wu; citation_author=A.P. Werner; citation_author=M. Koch; citation_author=A. Choi; citation_author=E. Narayanan; citation_author=G.B.E. Stewart-Jones; citation_author=T. Colpitts; citation_author=H. Bennett; citation_author=S. Boyoglu-Barnum; citation_author=W. Shi; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=384; citation_firstpage=1468; citation_lastpage=1470; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Evasion of Type I Interferon by SARS-CoV-2; citation_author=H. Xia; citation_author=Z. Cao; citation_author=X. Xie; citation_author=X. Zhang; citation_author=J.Y.-C. Chen; citation_author=H. Wang; citation_author=V.D. Menachery; citation_author=R. Rajsbaum; citation_author=P.-Y. Shi; citation_journal_title=Cell Rep.; citation_volume=33; citation_firstpage=108234; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Structural and functional ramifications of antigenic drift in recent SARS-CoV-2 variants; citation_author=M. Yuan; citation_author=D. Huang; citation_author=C.D. Lee; citation_author=N.C. Wu; citation_author=A.M. Jackson; citation_author=X. Zhu; citation_author=H. Liu; citation_author=L. Peng; citation_author=M.J. van Gils; citation_author=R.W. Sanders; citation_author=et al.; citation_journal_title=Science; citation_volume=373; citation_firstpage=818; citation_lastpage=823; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 RBD in&nbsp;vitro evolution follows contagious mutation spread, yet generates an able infection inhibitor; citation_author=J. Zahradnik; citation_author=S. Marciano; citation_author=M. Shemesh; citation_author=E. Zoler; citation_author=J. Chiaravalli; citation_author=B. Meyer; citation_author=O. Dym; citation_author=N. Elad; citation_author=G. Schreiber; citation_journal_title=BioRxiv; citation_doi=10.1101/2021.01.06.425392; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity; citation_author=L. Zhang; citation_author=C.B. Jackson; citation_author=H. Mou; citation_author=A. Ojha; citation_author=H. Peng; citation_author=B.D. Quinlan; citation_author=E.S. Rangarajan; citation_author=A. Pan; citation_author=A. Vanderheiden; citation_author=M.S. Suthar; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=6013; citation_publication_date=2020; "><meta name="citation_abstract" content="<h2>Summary</h2><p>The independent emergence late in 2020 of the B.1.1.7, B.1.351, and P.1 lineages of SARS-CoV-2 prompted renewed concerns about the evolutionary capacity of this virus to overcome public health interventions and rising population immunity. Here, by examining patterns of synonymous and non-synonymous mutations that have accumulated in SARS-CoV-2 genomes since the pandemic began, we find that the emergence of these three &quot;501Y lineages&quot; coincided with a major global shift in the selective forces acting on various SARS-CoV-2 genes. Following their emergence, the adaptive evolution of 501Y lineage viruses has involved repeated selectively favored convergent mutations at 35 genome sites, mutations we refer to as the 501Y meta-signature. The ongoing convergence of viruses in many other lineages on this meta-signature suggests that it includes multiple mutation combinations capable of promoting the persistence of diverse SARS-CoV-2 lineages in the face of mounting host immune recognition.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6359&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01050-3&amp;be=3572&amp;fe=6264&amp;dc=6028&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749591597044,%22n%22:0,%22f%22:2943,%22dn%22:2944,%22dne%22:2944,%22c%22:2944,%22s%22:2944,%22ce%22:2944,%22rq%22:2944,%22rp%22:3557,%22rpe%22:4123,%22dl%22:3562,%22di%22:4326,%22ds%22:6028,%22de%22:6038,%22dc%22:6263,%22l%22:6263,%22le%22:6301%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01050-3"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01050-3"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-735893" aria-expanded="false" aria-controls="login-links-735893" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-735893" id="login-links--735893" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-735893" data-opens="#links-dialog-735893" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-735893" aria-modal="true"><button data-closes="#links-dialog-735893" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901050-3" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.09.003">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01050-3" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="full" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0021-3"><span>Volume 184</span>, <span>Issue 20</span></a></span><span class="meta-panel__pages"><span>p5189-5200.e7</span></span><span class="meta-panel__onlineDate">September 30, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=20&amp;pii=S0092-8674%2820%29X0021-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">The emergence and ongoing convergent evolution of the SARS-CoV-2 N501Y lineages</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Darren P.</span> <span property="familyName">Martin</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Darren P.</span> <span property="familyName">Martin</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:darrenpatrickmartin@gmail.com" property="email" title="Send email to Darren P. Martin">darrenpatrickmartin@gmail.com</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Infectious Diseases and Molecular Medicine, Division Of Computational Biology, Department of Integrative Biomedical Sciences, University of Cape Town, Cape Town 7701, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Martin/Darren+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup> <a id="eadd0010" href="mailto:darrenpatrickmartin@gmail.com" property="email" title="Send email to Darren P. Martin">darrenpatrickmartin@gmail.com</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Steven</span> <span property="familyName">Weaver</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Steven</span> <span property="familyName">Weaver</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6931-7191" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6931-7191</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Weaver/Steven" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Houriiyah</span> <span property="familyName">Tegally</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Houriiyah</span> <span property="familyName">Tegally</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Tegally/Houriiyah" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">James Emmanuel</span> <span property="familyName">San</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James Emmanuel</span> <span property="familyName">San</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5736-664X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5736-664X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/San/James+Emmanuel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Stephen D.</span> <span property="familyName">Shank</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephen D.</span> <span property="familyName">Shank</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Shank/Stephen+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Eduan</span> <span property="familyName">Wilkinson</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Eduan</span> <span property="familyName">Wilkinson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Wilkinson/Eduan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Alexander G.</span> <span property="familyName">Lucaci</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alexander G.</span> <span property="familyName">Lucaci</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lucaci/Alexander+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Jennifer</span> <span property="familyName">Giandhari</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jennifer</span> <span property="familyName">Giandhari</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Giandhari/Jennifer" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Sureshnee</span> <span property="familyName">Naidoo</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sureshnee</span> <span property="familyName">Naidoo</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Naidoo/Sureshnee" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Yeshnee</span> <span property="familyName">Pillay</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yeshnee</span> <span property="familyName">Pillay</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Pillay/Yeshnee" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Lavanya</span> <span property="familyName">Singh</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lavanya</span> <span property="familyName">Singh</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1726-4454" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1726-4454</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Singh/Lavanya" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Richard J.</span> <span property="familyName">Lessells</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Richard J.</span> <span property="familyName">Lessells</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Lessells/Richard+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_collab0010" aria-controls="axel_author_collab0010"><span property="name">NGS-SA</span></a><div id="collab0010" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_collab0010"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="name">NGS-SA</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn1"><div id="ntpara0010" role="paragraph">A full list of consortium names and affiliations are in Appendix 1</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">http://www.krisp.org.za/ngs-sa/ngs-sa_network_for_genomic_surveillance_south_africa/</span></div></div><div class="core-author-link"><a href="/authored-by/group/NGS-SA" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_collab0015" aria-controls="axel_author_collab0015"><span property="name">COVID-19 Genomics UK (COG-UK)</span></a><div id="collab0015" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_collab0015"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="name">COVID-19 Genomics UK (COG-UK)</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn2"><div id="ntpara0015" role="paragraph">A full list of consortium names and affiliations are in Appendix 2</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">https://www.cogconsortium.uk</span></div></div><div class="core-author-link"><a href="/authored-by/group/COVID-19+Genomics+UK+%28COG-UK%29" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Ravindra K.</span> <span property="familyName">Gupta</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ravindra K.</span> <span property="familyName">Gupta</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Clinical Microbiology, University of Cambridge, Cambridge CB2 1TN, UK</span></div><div property="affiliation" typeof="Organization"><span property="name">Africa Health Research Institute, KwaZulu-Natal 4013, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Gupta/Ravindra+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Joel O.</span> <span property="familyName">Wertheim</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joel O.</span> <span property="familyName">Wertheim</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, University of California San Diego, La Jolla, CA 92093, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wertheim/Joel+O" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Anton</span> <span property="familyName">Nekturenko</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anton</span> <span property="familyName">Nekturenko</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry and Molecular Biology, The Pennsylvania State University, State College, PA 16802, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nekturenko/Anton" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Ben</span> <span property="familyName">Murrell</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ben</span> <span property="familyName">Murrell</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology, Tumor and Cell Biology, Karolinska Institutet, Stockholm 141 83, Sweden</span></div></div><div class="core-author-link"><a href="/authored-by/Murrell/Ben" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Gordon W.</span> <span property="familyName">Harkins</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gordon W.</span> <span property="familyName">Harkins</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">South African Medical Research Council Capacity Development Unit, South African National Bioinformatics Institute, University of the Western Cape, Bellville 7635, South Africa</span></div></div><div class="core-author-link"><a href="/authored-by/Harkins/Gordon+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Philippe</span> <span property="familyName">Lemey</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Philippe</span> <span property="familyName">Lemey</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology, Immunology and Transplantation, Rega Institute, KU Leuven, Leuven 3000, Belgium</span></div></div><div class="core-author-link"><a href="/authored-by/Lemey/Philippe" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Oscar A.</span> <span property="familyName">MacLean</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Oscar A.</span> <span property="familyName">MacLean</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, Glasgow 12 8QQ, Scotland, UK</span></div></div><div class="core-author-link"><a href="/authored-by/MacLean/Oscar+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">David L.</span> <span property="familyName">Robertson</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David L.</span> <span property="familyName">Robertson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MRC-University of Glasgow Centre for Virus Research, Glasgow 12 8QQ, Scotland, UK</span></div></div><div class="core-author-link"><a href="/authored-by/Robertson/David+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Tulio</span> <span property="familyName">de Oliveira</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tulio</span> <span property="familyName">de Oliveira</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3027-5254" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3027-5254</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:tuliodna@gmail.com" property="email" title="Send email to Tulio de Oliveira">tuliodna@gmail.com</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Global Health, University of Washington, Seattle, WA 98195-4550, USA</span></div></div><div class="core-author-link"><a href="/authored-by/de+Oliveira/Tulio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a role="doc-noteref">17</a></sup> <a id="eadd0015" href="mailto:tuliodna@gmail.com" property="email" title="Send email to Tulio de Oliveira">tuliodna@gmail.com</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Sergei L.</span> <span property="familyName">Kosakovsky Pond</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sergei L.</span> <span property="familyName">Kosakovsky Pond</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:spond@temple.edu" property="email" title="Send email to Sergei L. Kosakovsky Pond">spond@temple.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kosakovsky+Pond/Sergei+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup> <a id="eadd0020" href="mailto:spond@temple.edu" property="email" title="Send email to Sergei L. Kosakovsky Pond">spond@temple.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Institute of Infectious Diseases and Molecular Medicine, Division Of Computational Biology, Department of Integrative Biomedical Sciences, University of Cape Town, Cape Town 7701, South Africa</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Institute for Genomics and Evolutionary Medicine, Department of Biology, Temple University, Philadelphia, PA 19122, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">KwaZulu-Natal Research Innovation and Sequencing Platform, School of Laboratory Medicine &amp; Medical Sciences, University of KwaZulu- Natal, Durban 4001, South Africa</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">http://www.krisp.org.za/ngs-sa/ngs-sa_network_for_genomic_surveillance_south_africa/</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">https://www.cogconsortium.uk</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Clinical Microbiology, University of Cambridge, Cambridge CB2 1TN, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Africa Health Research Institute, KwaZulu-Natal 4013, South Africa</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Department of Medicine, University of California San Diego, La Jolla, CA 92093, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Department of Biochemistry and Molecular Biology, The Pennsylvania State University, State College, PA 16802, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Microbiology, Tumor and Cell Biology, Karolinska Institutet, Stockholm 141 83, Sweden</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">South African Medical Research Council Capacity Development Unit, South African National Bioinformatics Institute, University of the Western Cape, Bellville 7635, South Africa</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Department of Microbiology, Immunology and Transplantation, Rega Institute, KU Leuven, Leuven 3000, Belgium</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">MRC-University of Glasgow Centre for Virus Research, Glasgow 12 8QQ, Scotland, UK</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Department of Global Health, University of Washington, Seattle, WA 98195-4550, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">15</div><div id="fn1"><div id="ntpara0010" role="paragraph">A full list of consortium names and affiliations are in Appendix 1</div></div></div><div role="doc-footnote" data-has="label"><div class="label">16</div><div id="fn2"><div id="ntpara0015" role="paragraph">A full list of consortium names and affiliations are in Appendix 2</div></div></div><div role="doc-footnote" data-has="label"><div class="label">17</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received February 24, 2021</span>; <span>Revised July 5, 2021</span>; <span>Accepted September 1, 2021</span>; <span>Published online September 6, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.09.003" property="sameAs" target="_blank">10.1016/j.cell.2021.09.003 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421010503" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Published by Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.09.003" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0021-3"><img src="/cms/asset/00daceeb-44c0-4398-a4fc-e14099a3d1c7/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 20"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901050-3" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results and discussion</a></li><li><a href="#sec-3">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901050-3" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.09.003">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901050-3" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.09.003&amp;pii=S0092-8674%2821%2901050-3" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901050-3" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.09.003">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.09.003" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421010503&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421010503&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=20&amp;pii=S0092-8674%2820%29X0021-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00984-3" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01012-6" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results and discussion</a></li><li><a href="#sec-3">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Detected a major global shift in the SARS-CoV-2 selective landscape in late 2020</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Identified ongoing convergent evolution between the alpha, beta, and gamma lineages</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Defined the mutational meta-signature upon which these lineages are converging</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">The independent emergence late in 2020 of the B.1.1.7, B.1.351, and P.1 lineages of SARS-CoV-2 prompted renewed concerns about the evolutionary capacity of this virus to overcome public health interventions and rising population immunity. Here, by examining patterns of synonymous and non-synonymous mutations that have accumulated in SARS-CoV-2 genomes since the pandemic began, we find that the emergence of these three “501Y lineages” coincided with a major global shift in the selective forces acting on various SARS-CoV-2 genes. Following their emergence, the adaptive evolution of 501Y lineage viruses has involved repeated selectively favored convergent mutations at 35 genome sites, mutations we refer to as the 501Y meta-signature. The ongoing convergence of viruses in many other lineages on this meta-signature suggests that it includes multiple mutation combinations capable of promoting the persistence of diverse SARS-CoV-2 lineages in the face of mounting host immune recognition.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/aa3cbe1a-90b6-469f-a65d-1fe6374ac4a2/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.09.003/asset/89854977-b07c-4f39-812b-23a1ee593e87/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/aa3cbe1a-90b6-469f-a65d-1fe6374ac4a2/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;COVID+19&quot;" alt="Search on this keyword" rel="nofollow">COVID 19</a></li><li><a href="/action/doSearch?AllField=&quot;evolutionary+adaptation&quot;" alt="Search on this keyword" rel="nofollow">evolutionary adaptation</a></li><li><a href="/action/doSearch?AllField=&quot;positive+selection&quot;" alt="Search on this keyword" rel="nofollow">positive selection</a></li><li><a href="/action/doSearch?AllField=&quot;directional+selection&quot;" alt="Search on this keyword" rel="nofollow">directional selection</a></li><li><a href="/action/doSearch?AllField=&quot;recurrent+mutations&quot;" alt="Search on this keyword" rel="nofollow">recurrent mutations</a></li><li><a href="/action/doSearch?AllField=&quot;convergent+mutations&quot;" alt="Search on this keyword" rel="nofollow">convergent mutations</a></li><li><a href="/action/doSearch?AllField=&quot;diversifying+selection&quot;" alt="Search on this keyword" rel="nofollow">diversifying selection</a></li><li><a href="/action/doSearch?AllField=&quot;immune+evasion&quot;" alt="Search on this keyword" rel="nofollow">immune evasion</a></li><li><a href="/action/doSearch?AllField=&quot;transmission+advantage&quot;" alt="Search on this keyword" rel="nofollow">transmission advantage</a></li><li><a href="/action/doSearch?AllField=&quot;lineage-defining+mutations&quot;" alt="Search on this keyword" rel="nofollow">lineage-defining mutations</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0045" role="paragraph">In the first 11&nbsp;months of the SARS-CoV-2 pandemic (December 2019 – October 2020), the evolution of the virus worldwide was in the context of a highly susceptible new host population (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Dearlove et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Dearlove, B. ∙ Lewitus, E. ∙ Bai, H. ...</div><div><strong>A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:23652-23662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2008281117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85091566837" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32868447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2008281117&amp;pmid=32868447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31-1" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-1" aria-controls="sref31-1">MacLean et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>MacLean, O.A. ∙ Lytras, S. ∙ Weaver, S. ...</div><div><strong>Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen</strong></div><div><em>PLoS Biol.</em> 2021; <strong>19</strong>:e3001115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102654960" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33711012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001115&amp;pmid=33711012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Other than the early identification of the D614G substitution in the viral Spike protein (<span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Korber et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-1" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-1" aria-controls="sref42-1">Plante et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Plante, J.A. ∙ Liu, Y. ∙ Liu, J. ...</div><div><strong>Spike mutation D614G alters SARS-CoV-2 fitness</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:116-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2895-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85093821865" target="_blank" aria-label="Open Scopus in new tab">Scopus (992)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33106671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2895-3&amp;pmid=33106671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Zhang, L. ∙ Jackson, C.B. ∙ Mou, H. ...</div><div><strong>SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:6013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19808-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85096795416" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33243994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19808-4&amp;pmid=33243994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and P323L in the viral RNA-dependent RNA polymerase protein (<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Garvin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Garvin, M.R. ∙ T Prates, E. ∙ Pavicic, M. ...</div><div><strong>Potentially adaptive SARS-CoV-2 mutations discovered with novel spatiotemporal and explainable AI models</strong></div><div><em>Genome Biol.</em> 2020; <strong>21</strong>:304</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-020-02191-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85098009930" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33357233/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-020-02191-0&amp;pmid=33357233" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), both of which may have increased viral transmissibility without impacting pathogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Peacock et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Peacock, T.P. ∙ Penrice-Randal, R. ∙ Hiscox, J.A. ...</div><div><strong>SARS-CoV-2 one year on: evidence for ongoing viral adaptation</strong></div><div><em>J.&nbsp;Gen. Virol.</em> 2021; <strong>102</strong>:001584</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/jgv.0.001584" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85104384538" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33855951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2Fjgv.0.001584&amp;pmid=33855951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), few mutations were epidemiologically significant and the evolutionary dynamics of the virus were predominantly characterized by a mutational pattern of slow and selectively neutral random genetic drift (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Dearlove et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Dearlove, B. ∙ Lewitus, E. ∙ Bai, H. ...</div><div><strong>A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:23652-23662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2008281117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85091566837" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32868447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2008281117&amp;pmid=32868447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31-2" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-2" aria-controls="sref31-2">MacLean et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>MacLean, O.A. ∙ Lytras, S. ∙ Weaver, S. ...</div><div><strong>Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen</strong></div><div><em>PLoS Biol.</em> 2021; <strong>19</strong>:e3001115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102654960" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33711012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001115&amp;pmid=33711012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This behavior is consistent with exponential growth in a population of naive susceptible hosts that do not exert significant selective pressures on the pathogen prior to transmission events (<span class="dropBlock reference-citations"><a id="body-ref-sref31-3" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-3" aria-controls="sref31-3">MacLean et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>MacLean, O.A. ∙ Lytras, S. ∙ Weaver, S. ...</div><div><strong>Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen</strong></div><div><em>PLoS Biol.</em> 2021; <strong>19</strong>:e3001115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102654960" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33711012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001115&amp;pmid=33711012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Past pandemics and long-term evolutionary dynamics of RNA viruses attest to the fact that such an evolutionary “lull” rarely lasts. Indeed, in late 2020, three relatively divergent SARS-CoV-2 lineages emerged in rapid succession: (1) alpha, B.1.1.7 or 501Y.V1 which will hereafter be referred to as V1 (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Rambaut et&nbsp;al., 2020a</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Rambaut, A. ∙ Loman, N. ∙ Pybus, O. ...</div><div><strong>Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations</strong></div><div>Virological, 2020</div><div><a id="interref0010" href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563" target="_blank">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.RambautN.LomanO.PybusW.BarclayJ.BarrettA.CarabelliT.ConnorT.PeacockD.L.RobertsonE.VolzPreliminary+genomic+characterisation+of+an+emergent+SARS-CoV-2+lineage+in+the+UK+defined+by+a+novel+set+of+spike+mutations2020Virologicalhttps%3A%2F%2Fvirological.org%2Ft%2Fpreliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations%2F563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), (2) beta, B.1.351 or 501Y.V2 which will hereafter be referred to as V2 (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Tegally et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Tegally, H. ∙ Wilkinson, E. ∙ Giovanetti, M. ...</div><div><strong>Detection of a SARS-CoV-2 variant of concern in South Africa</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:438-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03402-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102321294" target="_blank" aria-label="Open Scopus in new tab">Scopus (1057)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33690265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03402-9&amp;pmid=33690265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and (3) gamma, P.1 or 501Y.V3, which will hereafter be referred to as V3 (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Faria et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Faria, N.R. ∙ Mellan, T.A. ∙ Whittaker, C. ...</div><div><strong>Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh2644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105876512" target="_blank" aria-label="Open Scopus in new tab">Scopus (883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33853970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh2644&amp;pmid=33853970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0050" role="paragraph">Viruses in each of the three lineages (which will hereafter be collectively referred to as 501Y lineages) have multiple signature (or lineage defining) deletions and amino acid substitutions (<a id="crosref0245" href="#fig1">Figure&nbsp;1</a>), many of which impact key domains of the Spike protein: the primary target of both infection and vaccine-induced immune responses. While many distinct Spike mutations had been observed prior to the 501Y lineages, all circulating SARS-CoV-2 lineages were defined by small numbers of mutations. All of the 501Y lineages also have significantly altered phenotypes: increased human ACE2 receptor affinity (V1, V2, and V3) (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Nelson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Nelson, G. ∙ Buzko, O. ∙ Spilman, P. ...</div><div><strong>Molecular dynamic simulation reveals E484K mutation enhances spike RBD-ACE2 affinity and the combination of E484K, K417N and N501Y mutations (501Y.V2 variant) induces conformational change greater than N501Y mutant alone, potentially resulting in an escape mutant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.13.426558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100737104" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34790982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.13.426558&amp;pmid=34790982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Starr et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Zahradnik et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Zahradnik, J. ∙ Marciano, S. ∙ Shemesh, M. ...</div><div><strong>SARS-CoV-2 RBD in&nbsp;vitro evolution follows contagious mutation spread, yet generates an able infection inhibitor</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.06.425392" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100750192" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.06.425392&amp;pmid=34981049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), increased transmissibility (V1, V2, and V3) (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Althaus et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Althaus, C.L. ∙ Baggio, S. ∙ Reichmuth, M.L. ...</div><div><strong>A tale of two variants: Spread of SARS-CoV-2 variants Alpha in Geneva, Switzerland, and Beta in South Africa</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252268" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33269368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252268&amp;pmid=33269368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Faria et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Faria, N.R. ∙ Mellan, T.A. ∙ Whittaker, C. ...</div><div><strong>Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh2644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105876512" target="_blank" aria-label="Open Scopus in new tab">Scopus (883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33853970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh2644&amp;pmid=33853970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Lubinski et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Lubinski, B. ∙ Tang, T. ∙ Daniel, S. ...</div><div><strong>Functional evaluation of proteolytic activation for the SARS-CoV-2 variant B.1.1.7: role of the P681H mutation</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.04.06.438731" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85121939608" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33851153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.04.06.438731&amp;pmid=33851153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Pearson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Pearson, C.A. ∙ Russell, T.W. ∙ Davies, N. ...</div><div><strong>Estimates of severity and transmissibility of novel SARS-CoV-2 variant 501Y.V2 in South Africa | CMMID Repository</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.24.20248822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105024215" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.24.20248822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Public Health England, 2020</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Public Health England</div><div><strong>Investigation of novel SARS-CoV-2 variant. Variant of Concern</strong></div><div>Public Health England, 2020</div><div><a id="interref0015" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959438/Technical_Briefing_VOC_SH_NJL2_SH2.pdf" target="_blank">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959438/Technical_Briefing_VOC_SH_NJL2_SH2.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Public+Health+EnglandInvestigation+of+novel+SARS-CoV-2+variant.+Variant+of+Concern2020Public+Health+Englandhttps%3A%2F%2Fassets.publishing.service.gov.uk%2Fgovernment%2Fuploads%2Fsystem%2Fuploads%2Fattachment_data%2Ffile%2F959438%2FTechnical_Briefing_VOC_SH_NJL2_SH2.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Volz et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ..., COVID-19 Genomics UK (COG-UK) consortium</div><div><strong>Assessing transmissibility of SARS-CoV-2 lineage B.1.1.7 in England</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:266-269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03470-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103190451" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33767447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03470-x&amp;pmid=33767447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), substantially increased capacity to overcome prior infection and/or vaccination-induced immunity (V2 and V3) (<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Cele et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Cele, S. ∙ Gazy, I. ∙ Jackson, L. ..., Network for Genomic Surveillance in South Africa, COMMIT-KZN Team</div><div><strong>Escape of SARS-CoV-2 501Y.V2 from neutralization by convalescent plasma</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:142-146</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03471-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103385503" target="_blank" aria-label="Open Scopus in new tab">Scopus (393)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33780970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03471-w&amp;pmid=33780970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Garcia-Beltran et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Garcia-Beltran, W.F. ∙ Lam, E.C. ∙ St Denis, K. ...</div><div><strong>Multiple SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2372-2383</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102763305" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33743213/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.013&amp;pmid=33743213" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Hoffmann et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Hoffmann, M. ∙ Arora, P. ∙ Groß, R. ...</div><div><strong>SARS-CoV-2 variants B.1.351 and P.1 escape from neutralizing antibodies</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2384-2393</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103496270" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33794143/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.036&amp;pmid=33794143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Shinde et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Shinde, V. ∙ Bhikha, S. ∙ Hoosain, Z. ..., 2019nCoV-501 Study Group</div><div><strong>Efficacy of NVX-CoV2373 Covid-19 Vaccine against the B.1.351 Variant</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1899-1909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2103055" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85106456686" target="_blank" aria-label="Open Scopus in new tab">Scopus (419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33951374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2103055&amp;pmid=33951374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Wibmer et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Wibmer, C.K. ∙ Ayres, F. ∙ Hermanus, T. ...</div><div><strong>SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:622-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01285-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102044511" target="_blank" aria-label="Open Scopus in new tab">Scopus (709)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33654292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01285-x&amp;pmid=33654292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Wu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Wu, K. ∙ Werner, A.P. ∙ Koch, M. ...</div><div><strong>Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1468-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2102179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103371216" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33730471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2102179&amp;pmid=33730471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and associations with increased virulence (V1 and V3) (<span class="dropBlock reference-citations"><a id="body-ref-sref11-3" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-3" aria-controls="sref11-3">Faria et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Faria, N.R. ∙ Mellan, T.A. ∙ Whittaker, C. ...</div><div><strong>Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh2644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105876512" target="_blank" aria-label="Open Scopus in new tab">Scopus (883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33853970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh2644&amp;pmid=33853970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Horby et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Horby, P. ∙ Huntley, C. ∙ Davies, N. ...</div><div><strong>Non-parametric analysis of fatal outcomes associated with&nbsp;B1.1.7</strong></div><div>Imperial College London, 2021</div><div><a id="intref0175" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/961037/NERVTAG_note_on_B.1.1.7_severity_for_SAGE_77__1_.pdf" target="_blank">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/961037/NERVTAG_note_on_B.1.1.7_severity_for_SAGE_77__1_.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.HorbyC.HuntleyN.DaviesJ.EdmundsN.FergusonG.MedleyC.SempleNon-parametric+analysis+of+fatal+outcomes+associated+with%C2%A0B1.1.72021Imperial+College+Londonhttps%3A%2F%2Fassets.publishing.service.gov.uk%2Fgovernment%2Fuploads%2Fsystem%2Fuploads%2Fattachment_data%2Ffile%2F961037%2FNERVTAG_note_on_B.1.1.7_severity_for_SAGE_77__1_.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Why did the heavily mutated 501Y lineages all arise on different continents at almost the same time? Was it due to an intrinsic change in the capacity of SARS-CoV-2 to adapt, or was it a shift in the host selective environment extrinsic to the virus?</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/9106dcb3-8472-4943-8a2b-1992f14d2fef/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.09.003/asset/84c1be4e-16d1-4e74-97f4-c5b86aba7688/main.assets/gr1.jpg" height="352" width="506" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/9106dcb3-8472-4943-8a2b-1992f14d2fef/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure 1</span> <span class="figure__title__text">SARS-CoV-2 genome map indicating the locations and encoded amino acid changes of what we considered here to be signature mutations of V1, V2, and V3 sequences</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">Genes represented with light-blue blocks encode non-structural proteins and genes in orange encode structural proteins: S encodes the Spike protein, E the envelope protein, M the matrix protein, and N the nucleocapsid protein. Within the S-gene, the receptor binding domain (RBD) is indicated by a darker shade, and the furin cleavage site is indicated by a dotted vertical line.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">Evidence that natural selection has played a pivotal role in the emergence of V1, V2, and V3 can be found in the remarkable patterns of independently evolved convergent mutations that have arisen within the members of these lineages (<a id="crosref0335" href="#fig1">Figure&nbsp;1</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Peacock et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Peacock, T.P. ∙ Penrice-Randal, R. ∙ Hiscox, J.A. ...</div><div><strong>SARS-CoV-2 one year on: evidence for ongoing viral adaptation</strong></div><div><em>J.&nbsp;Gen. Virol.</em> 2021; <strong>102</strong>:001584</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/jgv.0.001584" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85104384538" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33855951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2Fjgv.0.001584&amp;pmid=33855951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One of the most striking of these parallel changes is a nine-nucleotide deletion between genome coordinates 11288 and 11296 (here and hereafter all nucleotide and amino acid coordinates refer to&nbsp;the GenBank reference genome NC_045512). This deletion is within the portion of ORF1ab that encodes non-structural protein 6 (nsp6): a component of the SARS-CoV-2 membrane-tethered replication complex that likely influences the formation and maturation of autophagosomes (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Cottam et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Cottam, E.M. ∙ Maier, H.J. ∙ Manifava, M. ...</div><div><strong>Coronavirus nsp6 proteins generate autophagosomes from the endoplasmic reticulum via an omegasome intermediate</strong></div><div><em>Autophagy.</em> 2011; <strong>7</strong>:1335-1347</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/auto.7.11.16642" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-80655124392" target="_blank" aria-label="Open Scopus in new tab">Scopus (205)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21799305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Fauto.7.11.16642&amp;pmid=21799305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and decreases the effectiveness of host innate antiviral defenses by reducing the responsiveness of infected cells to, and antagonizing the production of, type I interferons (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Lei et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Lei, X. ∙ Dong, X. ∙ Ma, R. ...</div><div><strong>Activation and evasion of type I interferon responses by SARS-CoV-2</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:3810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-17665-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85088826828" target="_blank" aria-label="Open Scopus in new tab">Scopus (675)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32733001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-17665-9&amp;pmid=32733001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Miorin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Miorin, L. ∙ Kehrer, T. ∙ Sanchez-Aparicio, M.T. ...</div><div><strong>SARS-CoV-2 Orf6 hijacks Nup98 to block STAT nuclear import and antagonize interferon signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:28344-28354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2016650117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85096080108" target="_blank" aria-label="Open Scopus in new tab">Scopus (328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33097660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2016650117&amp;pmid=33097660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Xia et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Xia, H. ∙ Cao, Z. ∙ Xie, X. ...</div><div><strong>Evasion of Type I Interferon by SARS-CoV-2</strong></div><div><em>Cell Rep.</em> 2020; <strong>33</strong>:108234</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.celrep.2020.108234&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.celrep.2020.108234&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85091687040" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32979938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2020.108234&amp;pmid=32979938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Relative to some of the earliest characterized SARS-CoV-2 A and B variants, V1 and V2 viruses have demonstrably less sensitivity to type I interferons (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Guo et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Guo, K. ∙ Barrett, B.S. ∙ Mickens, K.L. ...</div><div><strong>Interferon Resistance of Emerging SARS-CoV-2 Variants</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.03.20.436257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105395922" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.03.20.436257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Thorne et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Thorne, L.G. ∙ Bouhaddou, M. ∙ Reuschl, A.-K. ...</div><div><strong>Evolution of enhanced innate immune evasion by the SARS-CoV-2 B.1.1.7&nbsp;UK variant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.06.06.446826" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85110367314" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.06.06.446826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and V1 displays greater antagonism of type I interferon-mediated immune activation during the early stages of infection in cultured lung epithelial cells (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Thorne et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Thorne, L.G. ∙ Bouhaddou, M. ∙ Reuschl, A.-K. ...</div><div><strong>Evolution of enhanced innate immune evasion by the SARS-CoV-2 B.1.1.7&nbsp;UK variant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.06.06.446826" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85110367314" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.06.06.446826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, it remains unknown whether these characteristics of V1 and V2 viruses are in any way attributable to their shared 11288–11296 deletion. Independently evolved instances of this deletion have been repeatedly found prior to the emergence of the 501Y lineages and identical independently evolved deletions are also found together with other 501Y lineage signature mutations in the SARS-CoV-2 lineages, B.1.620 (<span class="dropBlock reference-citations"><a id="body-ref-sref9-1" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-1" aria-controls="sref9-1">Dudas et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref9-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Dudas, G. ∙ Hong, S.L. ∙ Potter, B.I. ...</div><div><strong>Travel-driven emergence and spread of SARS-CoV-2 lineage B.1.620 with multiple VOC-like mutations and deletions in Europe</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.05.04.21256637v1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.05.04.21256637v1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), B.1.1.318, B.1.525 (<a id="intref0015" href="https://github.com/cov-lineages/pango-designation/issues/4" target="_blank">https://github.com/cov-lineages/pango-designation/issues/4</a>), and B.1.526 (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Annavajhala et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Annavajhala, M.K. ∙ Mohri, H. ∙ Zucker, J.E. ...</div><div><strong>A Novel SARS-CoV-2 Variant of Concern, B.1.526, Identified in New York</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85116172155" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33655278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252259&amp;pmid=33655278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This degree of convergent evolution implies that, in the context of the B.1.620, B.1.1.318, B.1.525, B.1.526, and the 501Y lineages at least, the 11288–11296 deletion is likely highly adaptive.</div><div id="p0060" role="paragraph">Additionally, there are four convergent spike gene mutations that are each shared between members of different 501Y lineages. Almost all the spike genes of sequences in these lineages carry the N501Y mutation at a key receptor binding domain (RBD) site that increases the affinity of the Spike protein for human ACE2 receptors by ∼2.1- to 3.5-fold (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Starr et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Yuan et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Yuan, M. ∙ Huang, D. ∙ Lee, C.D. ...</div><div><strong>Structural and functional ramifications of antigenic drift in recent SARS-CoV-2 variants</strong></div><div><em>Science.</em> 2021; <strong>373</strong>:818-823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh1139" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85112522453" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34016740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh1139&amp;pmid=34016740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Zahradnik et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Zahradnik, J. ∙ Marciano, S. ∙ Shemesh, M. ...</div><div><strong>SARS-CoV-2 RBD in&nbsp;vitro evolution follows contagious mutation spread, yet generates an able infection inhibitor</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.06.425392" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100750192" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.06.425392&amp;pmid=34981049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The vast majority of V2 and V3 variants and ∼0.3% of more recent samples of V1 variants also have a Spike E484K mutation. Whereas in the presence of 501N, 484K has a modest positive impact on ACE2 binding (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Starr et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), when present with 501Y, these mutations together synergistically increase ACE2-RBD binding affinity ∼12.7-fold (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Nelson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Nelson, G. ∙ Buzko, O. ∙ Spilman, P. ...</div><div><strong>Molecular dynamic simulation reveals E484K mutation enhances spike RBD-ACE2 affinity and the combination of E484K, K417N and N501Y mutations (501Y.V2 variant) induces conformational change greater than N501Y mutant alone, potentially resulting in an escape mutant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.13.426558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100737104" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34790982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.13.426558&amp;pmid=34790982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-3" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-3" aria-controls="sref65-3">Zahradnik et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref65-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Zahradnik, J. ∙ Marciano, S. ∙ Shemesh, M. ...</div><div><strong>SARS-CoV-2 RBD in&nbsp;vitro evolution follows contagious mutation spread, yet generates an able infection inhibitor</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.06.425392" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100750192" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.06.425392&amp;pmid=34981049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Crucially, E484K and other mutations at S/484 also frequently confer protection from neutralization by both convalescent sera (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Greaney et&nbsp;al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:463-476</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003&amp;pmid=33592168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), vaccine-elicited antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Collier et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Collier, D.A. ∙ De Marco, A. ∙ Ferreira, I.A.T.M. ..., CITIID-NIHR BioResource COVID-19 Collaboration, COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>Sensitivity of SARS-CoV-2 B.1.1.7 to mRNA vaccine-elicited antibodies</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:136-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03412-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102509491" target="_blank" aria-label="Open Scopus in new tab">Scopus (497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33706364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03412-7&amp;pmid=33706364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Wang et&nbsp;al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Wang, P. ∙ Nair, M.S. ∙ Liu, L. ...</div><div><strong>Antibody resistance of SARS-CoV-2 variants B.1.351 and B.1.1.7</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:130-135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03398-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102173598" target="_blank" aria-label="Open Scopus in new tab">Scopus (1469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33684923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03398-2&amp;pmid=33684923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Wu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Wu, K. ∙ Werner, A.P. ∙ Koch, M. ...</div><div><strong>Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1468-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2102179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103371216" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33730471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2102179&amp;pmid=33730471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and some monoclonal antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Greaney et&nbsp;al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:463-476</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003&amp;pmid=33592168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Starr et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Addetia, A. ...</div><div><strong>Prospective mapping of viral mutations that escape antibodies used to treat COVID-19</strong></div><div><em>Science.</em> 2021; <strong>371</strong>:850-854</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf9302" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046439" target="_blank" aria-label="Open Scopus in new tab">Scopus (497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33495308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf9302&amp;pmid=33495308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Wang et&nbsp;al., 2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There is therefore increasing evidence that viruses carrying the E484K mutation (with or without 501Y) will be able to more frequently infect both previously infected and vaccinated individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Collier et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Collier, D.A. ∙ De Marco, A. ∙ Ferreira, I.A.T.M. ..., CITIID-NIHR BioResource COVID-19 Collaboration, COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>Sensitivity of SARS-CoV-2 B.1.1.7 to mRNA vaccine-elicited antibodies</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:136-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03412-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102509491" target="_blank" aria-label="Open Scopus in new tab">Scopus (497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33706364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03412-7&amp;pmid=33706364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Wang et&nbsp;al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Wang, P. ∙ Nair, M.S. ∙ Liu, L. ...</div><div><strong>Antibody resistance of SARS-CoV-2 variants B.1.351 and B.1.1.7</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:130-135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03398-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102173598" target="_blank" aria-label="Open Scopus in new tab">Scopus (1469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33684923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03398-2&amp;pmid=33684923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Wu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Wu, K. ∙ Werner, A.P. ∙ Koch, M. ...</div><div><strong>Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1468-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2102179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103371216" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33730471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2102179&amp;pmid=33730471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0065" role="paragraph">A third RBD site that is mutated in both V2 and V3 is S/417. Whereas V2 sequences generally carry a K417N mutation, V3 sequences carry a K417T mutation. Although both the K417N and K417T mutations can reduce the affinity of Spike for ACE2, in conjunction with the N501Y and E484K mutations, ACE2 binding is restored to that of wild-type Spike (<span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Yuan et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Yuan, M. ∙ Huang, D. ∙ Lee, C.D. ...</div><div><strong>Structural and functional ramifications of antigenic drift in recent SARS-CoV-2 variants</strong></div><div><em>Science.</em> 2021; <strong>373</strong>:818-823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh1139" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85112522453" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34016740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh1139&amp;pmid=34016740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). K417N and K417T also both have moderately positive impacts on Spike expression (<span class="dropBlock reference-citations"><a id="body-ref-sref53-4" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-4" aria-controls="sref53-4">Starr et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and these and other mutations at S/417 provide modest protection from neutralization by some convalescent sera (<span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">Greaney et&nbsp;al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:463-476</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003&amp;pmid=33592168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-4" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-4" aria-controls="sref60-4">Wang et&nbsp;al., 2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), vaccine-induced antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref60-5" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-5" aria-controls="sref60-5">Wang et&nbsp;al., 2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and some neutralizing monoclonal antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Starr et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Addetia, A. ...</div><div><strong>Prospective mapping of viral mutations that escape antibodies used to treat COVID-19</strong></div><div><em>Science.</em> 2021; <strong>371</strong>:850-854</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf9302" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046439" target="_blank" aria-label="Open Scopus in new tab">Scopus (497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33495308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf9302&amp;pmid=33495308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-6" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-6" aria-controls="sref60-6">Wang et&nbsp;al., 2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Wibmer et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Wibmer, C.K. ∙ Ayres, F. ∙ Hermanus, T. ...</div><div><strong>SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:622-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01285-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102044511" target="_blank" aria-label="Open Scopus in new tab">Scopus (709)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33654292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01285-x&amp;pmid=33654292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0070" role="paragraph">A fourth spike gene mutation that is shared by ∼48% of V2 sequences and by all V3 sequences is L18F. This amino acid change is predicted to have a modest impact on the structure of Spike (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Nguyen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Nguyen, T.T. ∙ Pathirana, P.N. ∙ Nguyen, T. ...</div><div><strong>Genomic mutations and changes in protein secondary structure and solvent accessibility of SARS-CoV-2 (COVID-19 virus)</strong></div><div><em>Sci. Rep.</em> 2021; <strong>11</strong>:3487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-021-83105-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101468226" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33568759/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-021-83105-3&amp;pmid=33568759" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and also protects from some neutralizing monoclonal antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">McCallum et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>McCallum, M. ∙ De Marco, A. ∙ Lempp, F.A. ...</div><div><strong>N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2332-2347</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103091737" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33761326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.028&amp;pmid=33761326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Viruses carrying the L18F mutation increased in prevalence from the start of the pandemic and now account for ∼10% of sampled SARS-CoV-2 sequences.</div><div id="p0075" role="paragraph">These five convergent mutations in different rapidly spreading SARS-CoV-2 lineages serve as compelling evidence that they each, either alone or in combination, provide some significant fitness advantage. The individual and collective fitness impacts of the other signature mutations in V1, V2, and V3 remain unclear. A key way to infer the fitness impacts of these mutations is to examine patterns of synonymous and non-synonymous substitutions at the codon sites where the mutations occurred (<span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Kosakovsky Pond and Frost, 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Kosakovsky Pond, S.L. ∙ Frost, S.D.W.</div><div><strong>Not so different after all: a comparison of methods for detecting amino acid sites under selection</strong></div><div><em>Mol. Biol. Evol.</em> 2005; <strong>22</strong>:1208-1222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msi105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-17744396121" target="_blank" aria-label="Open Scopus in new tab">Scopus (1654)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15703242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsi105&amp;pmid=15703242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Specifically, it is expected that the most biologically important of these mutations will have occurred at codon sites that display substitution patterns across the wider SARS-CoV-2 phylogeny that are dominated by non-synonymous mutations (i.e., mutations that alter encoded amino acid sequences), patterns that are indicative of positive selection.</div><div id="p0080" role="paragraph">Here, using a suite of phylogenetics-based natural selection analysis techniques, we examine patterns of positive selection within the protein coding sequences of viruses in the V1, V2, and V3 lineages which, together with mutation frequency changes over time, we use to identify the specific mutations that are at present most likely contributing to the increased adaptation of these lineages. We find that the emergence of the 501Y lineages coincided with a marked global change in positive selection signals, indicative of a general shift in the selective environment within which SARS-CoV-2 is evolving. Against this backdrop, the 501Y lineages all display evidence of substantial ongoing adaptation involving, in many cases, mutations at positively selected genome sites that both converge on mutations seen in other 501Y lineages and are rapidly rising in frequency in different lineages. This pattern suggests that viruses in all three lineages are presently still climbing very similar adaptive peaks, and, therefore, that viruses in all three lineages are likely in the process of converging on a similar adaptive endpoint.</div></section><section id="sec-2"><h2>Results and discussion</h2><section id="sec-2-1"><h3>In late 2020, there was a detectable shift in the selective pressures acting on circulating SARS-CoV-2 variants</h3><div id="p0085" role="paragraph">Analyses of positive selection on SARS-CoV-2 genomes undertaken prior to the emergence of 501Y lineages revealed mutational patterns dominated by neutral evolution (<span class="dropBlock reference-citations"><a id="body-ref-sref31-4" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-4" aria-controls="sref31-4">MacLean et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>MacLean, O.A. ∙ Lytras, S. ∙ Weaver, S. ...</div><div><strong>Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen</strong></div><div><em>PLoS Biol.</em> 2021; <strong>19</strong>:e3001115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102654960" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33711012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001115&amp;pmid=33711012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). There were, however, indications that some sites in the genome had experienced episodes of positive selection (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Garvin et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Garvin, M.R. ∙ T Prates, E. ∙ Pavicic, M. ...</div><div><strong>Potentially adaptive SARS-CoV-2 mutations discovered with novel spatiotemporal and explainable AI models</strong></div><div><em>Genome Biol.</em> 2020; <strong>21</strong>:304</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-020-02191-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85098009930" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33357233/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-020-02191-0&amp;pmid=33357233" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Korber et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref42-2" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42-2" aria-controls="sref42-2">Plante et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref42-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Plante, J.A. ∙ Liu, Y. ∙ Liu, J. ...</div><div><strong>Spike mutation D614G alters SARS-CoV-2 fitness</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:116-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2895-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85093821865" target="_blank" aria-label="Open Scopus in new tab">Scopus (992)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33106671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2895-3&amp;pmid=33106671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Zhang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Zhang, L. ∙ Jackson, C.B. ∙ Mou, H. ...</div><div><strong>SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:6013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19808-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85096795416" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33243994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19808-4&amp;pmid=33243994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Through regular analyses of global GISAID data (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Elbe and Buckland-Merrett, 2017</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Elbe, S. ∙ Buckland-Merrett, G.</div><div><strong>Data, disease and diplomacy: GISAID’s innovative contribution to global health</strong></div><div><em>Glob. Chall.</em> 2017; <strong>1</strong>:33-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gch2.1018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85163179799" target="_blank" aria-label="Open Scopus in new tab">Scopus (1137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31565258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgch2.1018&amp;pmid=31565258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) starting in March 2020, we tracked the extent and location of positive and negative selective pressures on SARS-CoV-2 genomes (<a id="crosref0565" href="#fig2">Figure&nbsp;2</a>). The power of these analyses to detect evidence of selection acting on individual codon sites progressively increased over time with rising numbers of sampled genome sequences and sequence diversification.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/d9bf1a74-cc6d-4e6e-8772-bd72d7ec678e/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.09.003/asset/8d3803a1-2c32-47b1-9172-94988bb29b42/main.assets/gr2.jpg" height="404" width="504" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/d9bf1a74-cc6d-4e6e-8772-bd72d7ec678e/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure 2</span> <span class="figure__title__text">Signals of positive and negative selection at individual codon sites that were detectable with the FEL method at different times between March 2020 and April 2021 applied to sequences sampled over 90-day intervals</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0025" role="paragraph">The plotted dates show the end of 90-day periods.</div><div id="fspara0030" role="paragraph">(A and B) The gene-by-gene/per Kb/per unit tree length density of codons that are detectably evolving under positive/negative selection between March 2020 and February 2021. Whereas genes for which the maximum observed density of positively/negatively selected sites was reached in February 2021 are shown with thicker lines, genes with associated trees that have a total length shorter than 0.5 subs/site for a given time period are not shown. A version of (A) with all genes displayed separately is given in <a id="crosref1045" href="#figs1">Figure S1</a>.</div><div id="fspara0035" role="paragraph">(C) Signals of positive selection detected at 37 V1, V2, and V3 signature mutation sites between March 2020 and April 2021. Also included for reference are sites previously detected to be evolving under positive selection such as S/614, the site of the D614G mutation that is present in all three of the 501Y lineages, S/5, and RdRp/P323L (ORF1b/314). Circles indicate the statistical significance of the FEL test with red indicating positive selection and blue indicating negative selection. The vertical line indicates December 1, 2020, the approximate date when the importance of the V1 and V2 lineages were first noticed.</div><div id="fspara0040" role="paragraph">See also <a id="crosref1050" href="#figs1">Figure S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">Even accounting for this expected increased power of detection, it is evident that a significant shift in selective pressures occurred ∼11&nbsp;months after SARS-CoV-2 cases were first reported in Wuhan City in December 2019. Specifically, during November 2020 this change in selection pressures manifested in substantial increases in the numbers of SARS-CoV-2 codon sites that were detectably evolving under both positive and negative selection. This increase accelerated through February 2021, with sites found to be evolving under diversifying positive selection in several genomic regions (p ≤ 0.01 with the FEL selection detection method [<span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Kosakovsky Pond and Frost, 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Kosakovsky Pond, S.L. ∙ Frost, S.D.W.</div><div><strong>Not so different after all: a comparison of methods for detecting amino acid sites under selection</strong></div><div><em>Mol. Biol. Evol.</em> 2005; <strong>22</strong>:1208-1222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msi105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-17744396121" target="_blank" aria-label="Open Scopus in new tab">Scopus (1654)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15703242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsi105&amp;pmid=15703242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]) rapidly increasing in density for several key genes including S, nsp2, and nsp6 (<a id="crosref0575" href="#fig2">Figure&nbsp;2</a>A; <a id="crosref0580" href="#figs1">Figure&nbsp;S1</a>). This increase cannot be fully explained by increased sampling at later time points, as our estimates of positively selected site densities are corrected to account for variations in phylogenetic signal (i.e., they are fractions of the total length of internal tree branches [<span class="dropBlock reference-citations"><a id="body-ref-sref31-5" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31-5" aria-controls="sref31-5">MacLean et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>MacLean, O.A. ∙ Lytras, S. ∙ Weaver, S. ...</div><div><strong>Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen</strong></div><div><em>PLoS Biol.</em> 2021; <strong>19</strong>:e3001115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102654960" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33711012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001115&amp;pmid=33711012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]). This sudden increase in the density of sites that were detectably evolving under positive selection coincided with epidemic surges in multiple parts of the world in both hemispheres, many of which were driven by the emerging V1, V2, and V3 lineages.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/6b174e55-79e0-4395-9fef-fe00a47d7553/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.09.003/asset/9d01e26e-d77e-4038-9cfc-d2593f50652a/main.assets/figs1.jpg" height="862" width="504" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/6b174e55-79e0-4395-9fef-fe00a47d7553/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure S1</span> <span class="figure__title__text">Signals of positive selection at individual codon sites that were detectable with the FEL method at different times between March 2020 and February 2021 applied to sequences sampled over 90-day intervals, related to <a id="crosref1060" href="#fig2">Figure 2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0085" role="paragraph">The plotted date shows the end of the 90-day period. Genes with associated trees that have a total length shorter than 0.5 subs/site for a given time-period are not shown. Note that for the ORF3a and the N gene the interpretation of positive selection signals is complicated by the fact that each of these genes encompasses multiple smaller genes that are expressed in different reading frames. This is because synonymous substitutions in the ORF3a and N reading frames will be non-synonymous substitutions in the reading frames of the smaller genes that they encompass (and vice versa): a situation that is expected to inflate positive selection signals.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">Among the 37 signature mutation sites in V1, V2, and V3 (<a id="crosref0590" href="#fig1">Figure&nbsp;1</a>), 14 were detectably evolving under positive selection in November 2020, whereas this number increased to 22 by January 2021 and 30 by April 2021 (<a id="crosref0595" href="#fig2">Figure&nbsp;2</a>C). The only signature mutation shared between any of the three 501Y lineages that was detectably evolving under positive selection before November 2020 was S/18, which was detected for the first time in August 2020.</div><div id="p0100" role="paragraph">Our regular tracking of positively selected SARS-CoV-2 codon sites prior to November 2020 therefore yielded no clear indications that non-synonymous substitutions at the crucial RBD sites, S/417, S/484, and S/501 (the other key convergent signature mutation sites in the 501Y lineages), provided SARS-CoV-2 with any substantial fitness advantages prior to November 2020. Instead, the sporadic weak selection signals that these analyses yielded between July and November were of adaptive amino acid substitutions in the Spike N-terminal domain (S/18, and the V3 signature sites, S/26 and S/138), near the furin cleavage site (the V3 signature site, S/655, and the V1 signature site S/681), and in the C-terminal domain (the V1 signature site, S/1118, and the V3 signature site, S/1176). Conversely, for much of the latter half of 2020 the relatively strong and consistently detected selection signals at the V1 signature site, N/3, and the V2 signature sites, ORF1ab/265 (nsp2 codon 85) and ORF3a/57, clearly indicated that some substitutions at these sites were likely adaptative.</div><div id="p0105" role="paragraph">Taken together, these patterns of detectable selection suggest that the adaptive value of signature 501Y lineage RBD mutations may have only manifested after a selective shift that occurred shortly before November 2020.</div></section><section id="sec-2-2"><h3>Signals of selection within the V1, V2, and V3 lineages up until March 2021</h3><div id="p0110" role="paragraph">We initially restricted our selection analyses to only consider mutations arising within V1, V2, and V3 lineage sequences sampled before April 2021 to assess the adaptive processes at play within these lineages during and immediately after the perceived shift in&nbsp;the SARS-CoV-2 selective landscape in late 2020. We were specifically interested in identifying positive selection signals at individual codon sites that reflected these “early” selective processes.</div><div id="p0115" role="paragraph">We collected all sequences assigned to B.1.1.7 (V1), B.1.351 (V2), and P1 (V3) PANGO lineages (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Rambaut et&nbsp;al., 2020b</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Rambaut, A. ∙ Holmes, E.C. ∙ O’Toole, Á. ...</div><div><strong>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0770-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85087930366" target="_blank" aria-label="Open Scopus in new tab">Scopus (1575)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32669681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0770-5&amp;pmid=32669681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in GISAID (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Elbe and Buckland-Merrett, 2017</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Elbe, S. ∙ Buckland-Merrett, G.</div><div><strong>Data, disease and diplomacy: GISAID’s innovative contribution to global health</strong></div><div><em>Glob. Chall.</em> 2017; <strong>1</strong>:33-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gch2.1018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85163179799" target="_blank" aria-label="Open Scopus in new tab">Scopus (1137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31565258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgch2.1018&amp;pmid=31565258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as of April 20, 2021 and tested these sequences for evidence of positive selection at individual codon sites using MEME (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Murrell et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Murrell, B. ∙ Wertheim, J.O. ∙ Moola, S. ...</div><div><strong>Detecting individual sites subject to episodic diversifying selection</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1002764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1002764" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84864605220" target="_blank" aria-label="Open Scopus in new tab">Scopus (1148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22807683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1002764&amp;pmid=22807683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and FEL (<span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Pond et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Pond, S.L.K. ∙ Frost, S.D.W. ∙ Grossman, Z. ...</div><div><strong>Adaptation to different human populations by HIV-1 revealed by codon-based analyses</strong></div><div><em>PLoS Comput. Biol.</em> 2006; <strong>2</strong>:e62</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0020062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-33745611774" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16789820/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0020062&amp;pmid=16789820" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Both of these methods were restricted to analyzing mutations that mapped to internal branches of V1, V2, and V3 phylogenetic trees: i.e., mutations that almost certainly would have only arisen before mid-March 2021.</div><div id="p0120" role="paragraph">These analyses revealed evidence of positive selection at 151 individual codon sites (at p&nbsp;&lt; 0.05) across all lineages including 80 in V1, 41 in V2, and 37 in V3 (<a id="crosref0620" href="#mmc1">Table S1</a>). This is indicative of substantial adaptation of V1, V2, and V3 sequences between the time of their emergence and March 2021.</div></section><section id="sec-2-3"><h3>Signals of ongoing mutational convergence at signature mutation sites</h3><div id="p0125" role="paragraph">Notable among the lineage-specific positive selection signals were 22/151 at lineage defining mutation sites: 8/11 of the V1 signature sites, 4/14 of the V2 ones, and 13/17 of the V3 ones (<a id="crosref0625" href="#fig3">Figure&nbsp;3</a> and see underlined codon-site numbers in <a id="crosref0630" href="#fig1">Figure&nbsp;1</a>). Given that (1) each lineage was defined by the signature mutations along the phylogenetic tree branch basal to its clade, and (2) that these basal branches were included in the lineage-specific selection analyses, these selection analysis results were biased in favor of detecting the signature mutations as evolving under positive selection. We therefore used the selection results for signature mutation sites only to identify the signature mutations that, relative to the background reference sequences, were evolving under the strongest degrees of positive selection during or before March 2021.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/35b0da7d-9394-4342-b5b4-eae613ca086f/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.09.003/asset/bded973d-9b4e-4f44-91dc-7dff4ac6f93e/main.assets/gr3.jpg" height="760" width="373" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/35b0da7d-9394-4342-b5b4-eae613ca086f/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure 3</span> <span class="figure__title__text">Genome sites where signature and convergent mutations occur within the 501Y lineage sequences</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0050" role="paragraph">Sites detectably evolving under positive selection along internal branches (MEME p ≤ 0.05) of the V1, V2, and V3 phylogenies are indicated with red icons. We restricted our analysis to data collected up to April 2021 to focus on interpreting predictive positive selection signals arising from mutations occurring before March 2021, which could then be corroborated by examining mutation frequency data from later months. Labels within the colored blocks&nbsp;indicate amino acid substitutions with block colors indicating model-based predictions of the probable evolutionary viability of the observed amino acid substitutions based on the numbers of times these substitutions have been observed in related coronaviruses that infect other host species. The absence of color indicates unprecedented substitutions, red indicates highly unusual substitutions, and green indicates common substitutions seen at homologous sites in non-SARS-CoV-2 coronaviruses. ORF8 signals have been excluded.</div><div id="fspara0055" role="paragraph">See also <a id="crosref1055" href="#figs2">Figure S2</a>.</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">Most noteworthy of the 22 signature mutation sites that displayed the strongest evidence of lineage-specific positive selection are codons S/18, S/80, S/417, S/501, S/655, and S/681 in that all are either suspected or known to harbor mutations with potentially significant fitness impacts (<span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Garry et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Garry, R.F. ∙ Andersen, K.G. ∙ Gallaher, W.R. ...</div><div><strong>Spike protein mutations in novel SARS-CoV-2 ‘variants of concern’ commonly occur&nbsp;in or near indels</strong></div><div>2021</div><div><a id="intref0170" href="https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605" target="_blank">https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.F.GarryK.G.AndersenW.R.GallaherT.T.-Y.LamK.GangaparapuA.A.LatifB.J.BeddingfieldA.RambautE.C.HolmesSpike+protein+mutations+in+novel+SARS-CoV-2+%E2%80%98variants+of+concern%E2%80%99+commonly+occur%C2%A0in+or+near+indels2021https%3A%2F%2Fvirological.org%2Ft%2Fspike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels%2F605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Greaney et&nbsp;al., 2021b</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:44-57</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33259788/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007&amp;pmid=33259788" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref15-4" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-4" aria-controls="sref15-4">2021a</a><div class="dropBlock__holder" data-db-target-of="sref15-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:463-476</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003&amp;pmid=33592168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Lubinski et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Lubinski, B. ∙ Tang, T. ∙ Daniel, S. ...</div><div><strong>Functional evaluation of proteolytic activation for the SARS-CoV-2 variant B.1.1.7: role of the P681H mutation</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.04.06.438731" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85121939608" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33851153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.04.06.438731&amp;pmid=33851153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">McCallum et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>McCallum, M. ∙ De Marco, A. ∙ Lempp, F.A. ...</div><div><strong>N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2332-2347</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103091737" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33761326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.028&amp;pmid=33761326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref53-5" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-5" aria-controls="sref53-5">Starr et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59-3" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-3" aria-controls="sref59-3">Wang et&nbsp;al., 2021a</a><div class="dropBlock__holder" data-db-target-of="sref59-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Wang, P. ∙ Nair, M.S. ∙ Liu, L. ...</div><div><strong>Antibody resistance of SARS-CoV-2 variants B.1.351 and B.1.1.7</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:130-135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03398-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102173598" target="_blank" aria-label="Open Scopus in new tab">Scopus (1469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33684923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03398-2&amp;pmid=33684923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref60-7" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-7" aria-controls="sref60-7">2021b</a><div class="dropBlock__holder" data-db-target-of="sref60-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65-4" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-4" aria-controls="sref65-4">Zahradnik et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref65-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Zahradnik, J. ∙ Marciano, S. ∙ Shemesh, M. ...</div><div><strong>SARS-CoV-2 RBD in&nbsp;vitro evolution follows contagious mutation spread, yet generates an able infection inhibitor</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.06.425392" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100750192" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.06.425392&amp;pmid=34981049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0135" role="paragraph">Before March 2021, there were particularly interesting mutational dynamics at codon S/18 in the V1 lineage. Whereas the L18F mutation is effectively fixed in all currently sampled V3 lineage sequences, it occurred (and persisted in descendent variants) at least twice in the V1 lineage and at least four times in the V2 lineage. S/18 falls within multiple different predicted CTL epitopes (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Campbell et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Campbell, K.M. ∙ Steiner, G. ∙ Wells, D.K. ...</div><div><strong>Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles</strong></div><div><em>BioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.30.016931" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85098906862" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.30.016931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the L18F mutation is known to reduce viral sensitivity to some neutralizing monoclonal antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref32-3" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-3" aria-controls="sref32-3">McCallum et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>McCallum, M. ∙ De Marco, A. ∙ Lempp, F.A. ...</div><div><strong>N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2332-2347</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103091737" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33761326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.028&amp;pmid=33761326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An F at residue S/18 is also observed in 10% of other known Sarbecoviruses and the L18F mutation was the 28<sup>th</sup> most common in sampled SARS-CoV-2 genomes on June 4, 2021. Having occurred independently numerous times since the start of the pandemic, S/18 has also been detectably evolving under positive selection in the global SARS-CoV-2 genome dataset since August 2020 (<a id="crosref0690" href="#fig2">Figure&nbsp;2</a>C).</div><div id="p0140" role="paragraph">Similar convergence patterns to those observed at S/18 could be seen at 17 other signature mutation sites that, before March 2021, were detectably evolving under positive selection in either the global (<a id="crosref0695" href="#fig2">Figure&nbsp;2</a>C) or lineage-specific (<a id="crosref0700" href="#fig3">Figure&nbsp;3</a>) datasets: ORF1a/265, ORF1a/1188, S/26, S/138, S/215, S/417, S/484, S/501, S/655, S/681, S/701, S/716, S/1027, S/1176, ORF3a/57, N/205, and N/235.</div><div id="p0145" role="paragraph">Of these, S/655, S/681, S/701, and S/716 are noteworthy in that they fall within 30 residues of the biologically important Spike protein furin cleavage site (S/680 to S/689). Whereas some V2 and V3 sequences had, by March 2021, independently acquired the signature V1 mutations, P681H and T716I, some V1 and V2 sequences had independently acquired the V3 signature mutation, H655Y, and some V1 sequences had independently acquired the V2 signature mutation, A701V. Additionally, whereas a convergent A701V mutation is also found in the B.1.526 and S/E484K carrying lineage that was first identified in New York (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Annavajhala et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Annavajhala, M.K. ∙ Mohri, H. ∙ Zucker, J.E. ...</div><div><strong>A Novel SARS-CoV-2 Variant of Concern, B.1.526, Identified in New York</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85116172155" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33655278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252259&amp;pmid=33655278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), P681H is found in the S/E484K and S/N501Y carrying P.3 lineage first identified in the Philippines (<span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Tablizo et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Tablizo, F.A. ∙ Kim, K.M. ∙ Lapid, C.M. ...</div><div><strong>Genome sequencing and analysis of an emergent SARS-CoV-2 variant characterized by multiple spike protein mutations detected from the Central Visayas Region of the Philippines</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.03.03.21252812" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85104474905" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.03.03.21252812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and both S/H655Y and S/P681H are found in the highly mutated S/E484K carrying A.VOI.V2 lineage first identified in Tanzanian travelers (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">de Oliveira et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>de Oliveira, T. ∙ Lutucuta, S. ∙ Nkengasong, J. ...</div><div><strong>A novel variant of&nbsp;interest of SARS-CoV-2 with multiple spike mutations is identified from travel surveillance in Africa</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252268" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85114081907" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252268" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0150" role="paragraph">Any of H655Y, P681H, A701V, or T716I might directly impact the efficiency of viral entry into host cells (<span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Garry et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Garry, R.F. ∙ Andersen, K.G. ∙ Gallaher, W.R. ...</div><div><strong>Spike protein mutations in novel SARS-CoV-2 ‘variants of concern’ commonly occur&nbsp;in or near indels</strong></div><div>2021</div><div><a id="intref0170" href="https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605" target="_blank">https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.F.GarryK.G.AndersenW.R.GallaherT.T.-Y.LamK.GangaparapuA.A.LatifB.J.BeddingfieldA.RambautE.C.HolmesSpike+protein+mutations+in+novel+SARS-CoV-2+%E2%80%98variants+of+concern%E2%80%99+commonly+occur%C2%A0in+or+near+indels2021https%3A%2F%2Fvirological.org%2Ft%2Fspike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels%2F605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SARS-CoV-2 variants with deletions of the furin cleavage site have reduced pathogenicity (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Johnson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Johnson, B.A. ∙ Xie, X. ∙ Bailey, A.L. ...</div><div><strong>Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis</strong></div><div><em>Nature.</em> 2021; <strong>591</strong>:293-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03237-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85099867614" target="_blank" aria-label="Open Scopus in new tab">Scopus (409)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33494095/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03237-4&amp;pmid=33494095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Lau et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Lau, S.-Y. ∙ Wang, P. ∙ Mok, B.W.-Y. ...</div><div><strong>Attenuated SARS-CoV-2 variants with deletions at the S1/S2 junction</strong></div><div><em>Emerg. Microbes Infect.</em> 2020; <strong>9</strong>:837-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/22221751.2020.1756700" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85083865808" target="_blank" aria-label="Open Scopus in new tab">Scopus (205)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32301390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F22221751.2020.1756700&amp;pmid=32301390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the P681H mutation—which falls within this site—likely increases the efficiency of furin cleavage by replacing a less favorable uncharged amino acid with a more favorable positively charged basic one (<span class="dropBlock reference-citations"><a id="body-ref-sref13-3" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-3" aria-controls="sref13-3">Garry et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref13-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Garry, R.F. ∙ Andersen, K.G. ∙ Gallaher, W.R. ...</div><div><strong>Spike protein mutations in novel SARS-CoV-2 ‘variants of concern’ commonly occur&nbsp;in or near indels</strong></div><div>2021</div><div><a id="intref0170" href="https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605" target="_blank">https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.F.GarryK.G.AndersenW.R.GallaherT.T.-Y.LamK.GangaparapuA.A.LatifB.J.BeddingfieldA.RambautE.C.HolmesSpike+protein+mutations+in+novel+SARS-CoV-2+%E2%80%98variants+of+concern%E2%80%99+commonly+occur%C2%A0in+or+near+indels2021https%3A%2F%2Fvirological.org%2Ft%2Fspike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels%2F605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29-3" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-3" aria-controls="sref29-3">Lubinski et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref29-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Lubinski, B. ∙ Tang, T. ∙ Daniel, S. ...</div><div><strong>Functional evaluation of proteolytic activation for the SARS-CoV-2 variant B.1.1.7: role of the P681H mutation</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.04.06.438731" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85121939608" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33851153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.04.06.438731&amp;pmid=33851153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Whereas sites S/655 and S/681 are also detectably evolving under positive selection in at least one of the lineage specific datasets, S/655, S/681, A/701, and S/716 are all detectably evolving under positive selection in the March and April 2021 global SARS-CoV-2 datasets; important additional indicators that are consistent with the H655Y, P681H, A701V, and T716I mutations being adaptive.</div></section><section id="sec-2-4"><h3>Non-convergent mutations at signature mutation sites might still be evolutionarily convergent</h3><div id="p0155" role="paragraph">In addition to the 17 signature mutation sites displaying evidence of both convergent mutations between the V1, V2, and V3 lineages, and positive selection in the global and/or lineage-specific datasets, four signature mutation sites with lineage-specific signals of positive selection (S/20, S/138, S/215, and S/570; <a id="crosref0745" href="#fig1">Figure&nbsp;1</a>) display evidence of predominantly divergent mutations (at the amino acid replacement level), where the same site is mutated as in another lineage, but to a different amino acid (<a id="crosref0750" href="#fig3">Figure&nbsp;3</a>). A fifth site, S/80, displays evidence of both convergent and divergent mutations. All but S/20 are also detectably evolving under positive selection in the global SARS-CoV-2 dataset (<a id="crosref0755" href="#fig2">Figure&nbsp;2</a>).</div><div id="p0160" role="paragraph">The diverging mutations at these five sites might also be contributing to the overall patterns of evolutionary convergence between the lineages; just via different routes. Four of these sites (S/20, S/80, S/138, and S/215) fall within a portion of the Spike N-terminal domain that is an “antigenic supersite” targeted by multiple monoclonal and infection-induced neutralizing antibodies (<span class="dropBlock reference-citations"><a id="body-ref-sref32-4" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-4" aria-controls="sref32-4">McCallum et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref32-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>McCallum, M. ∙ De Marco, A. ∙ Lempp, F.A. ...</div><div><strong>N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2332-2347</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103091737" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33761326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.028&amp;pmid=33761326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It is therefore plausible that these sites are evolving under immunity-driven diversifying selective pressures. In this regard, while mutations at S/20, S/80, S/138, S/215, and S/570 in different lineages do not predominantly converge on the same encoded amino acid states, they could nevertheless still be convergent on similar fitness objectives (immune escape or compensation for the fitness costs of other mutations): such as is likely the case with the also not strictly convergent V2 K417N and V3 K417T signature mutations (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Greaney et&nbsp;al., 2021b</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:44-57</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33259788/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007&amp;pmid=33259788" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref15-5" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-5" aria-controls="sref15-5">2021a</a><div class="dropBlock__holder" data-db-target-of="sref15-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:463-476</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003&amp;pmid=33592168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Nelson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Nelson, G. ∙ Buzko, O. ∙ Spilman, P. ...</div><div><strong>Molecular dynamic simulation reveals E484K mutation enhances spike RBD-ACE2 affinity and the combination of E484K, K417N and N501Y mutations (501Y.V2 variant) induces conformational change greater than N501Y mutant alone, potentially resulting in an escape mutant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.13.426558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100737104" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34790982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.13.426558&amp;pmid=34790982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-5"><h3>Positive selection may be driving further convergence at non-signature mutation sites</h3><div id="p0165" role="paragraph">In addition to lineage-specific signals of positive selection being detected at 22 of the signature mutation sites that characterize each of V1, V2, and V3 (<a id="crosref0780" href="#fig1">Figure&nbsp;1</a>), such signals were also detected in lineage-specific datasets at 129 non-signature mutation sites (<a id="crosref0785" href="#mmc1">Table S1</a>). As with the positively selected signature mutation sites, these selection signals are based on mutations that map to internal V1, V2, and V3 tree branches and likely reflect selective processes operating before mid-March 2021.</div><div id="p0170" role="paragraph">To test whether positive selection acting at these 129 codon sites might have favored convergent amino acid changes across the three lineages, we examined mutations occurring at these sites for evidence of convergence between two or more of the lineages. This revealed the occurrence of convergent mutations between sequences in different lineages at 28/129 (21.7%) of these sites, including ten in ORF1a, seven in the N-gene, five in the S-gene (<a id="crosref0790" href="#fig3">Figure&nbsp;3</a>), three in ORF3A, and three in ORF1b (<a id="crosref0795" href="#fig3">Figure&nbsp;3</a>).</div><div id="p0175" role="paragraph">The lineage-specific positive selection signals detected at these 28 sites reflect repeated convergent non-synonymous mutations within each lineage that likely increase the fitness of the genomes in which they occur. Accordingly, 18/28 of the codons where these inter- and intra-lineage convergent mutations occur were also detectably evolving under positive selection within the April global SARS-CoV-2 dataset (FEL p&nbsp;&lt; 0.05; <a id="crosref0800" href="#figs2">Figure&nbsp;S2</a>). This concordance between the lineage-specific and global selection signals is strong evidence that an appreciable proportion of the convergent non-signature site mutations are broadly adaptive (as opposed to being only epistatically adaptive in the context of 501Y lineage virus genomes).</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/0fa1abe5-8ffb-4277-8306-96812eebb712/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.09.003/asset/cccaaf2f-8948-4720-83cc-46de9c476cf9/main.assets/figs2.jpg" height="823" width="373" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/0fa1abe5-8ffb-4277-8306-96812eebb712/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure S2</span> <span class="figure__title__text">Global selection trends at sites in 501Y lineage viruses that are either signature mutations or that, on April 20, 2021, displayed evidence of both lineage-specific positive selection on internal tree branches and mutational convergence between viruses in different 501Y lineages, related to <a id="crosref1065" href="#fig3">Figure 3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0095" role="paragraph">Red dots indicate positive selection and blue dots indicate negative selection in the global SARS-CoV-2 dataset. The sizes of the dots indicate the strength of the positive/negative selection signals. Selection signals indicate those detected when considering only the sequences sampled in the preceding three months.</div></div></div></div></figcaption></figure></div><div id="p0180" role="paragraph">The degree of overlap between non-signature mutation sites detectably evolving under selection in the V1, V2, and V3 lineages and displaying evidence of inter-lineage convergent mutations (<a id="crosref0805" href="#fig3">Figure&nbsp;3</a>; <a id="crosref0810" href="#mmc1">Table S1</a>) cannot be adequately explained by chance alone. Restricting ourselves only to variable sites that are shared between lineages (i.e., just those sites where it was possible to detect selection and/or convergent mutations) and the numbers of selected sites in each lineage, we conducted a permutation test for the numbers of detectable convergent, positively selected non-signature site mutations between each pair of lineages. Overlap by chance can be rejected for V1/V2 (p&nbsp;&lt; 0.001), V1/V3 (p&nbsp;= 0.002), and V2/V3 (p&nbsp;= 0.002). This pattern supports the hypothesis that all three lineages are at present accumulating convergent non-signature site mutations that are contributing to their ascent of the same fitness peak.</div></section><section id="sec-2-6"><h3>Changes in mutation frequencies since mid-March 2021 corroborate the inferred fitness advantages of convergent mutations at positively selected genome sites</h3><div id="p0185" role="paragraph">Although it is clear that in the regions of the world where the prevalence of 501Y lineage viruses have increased since December 2020, these viruses have had substantial fitness advantages over the SARS-CoV-2 variants that preceded them, it remains unclear what the precise biological advantages were. The two most likely, non-exclusive, reasons for their increased fitness are (1) that they were better at infecting people that had been previously infected (V2 and V3) (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Cele et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Cele, S. ∙ Gazy, I. ∙ Jackson, L. ..., Network for Genomic Surveillance in South Africa, COMMIT-KZN Team</div><div><strong>Escape of SARS-CoV-2 501Y.V2 from neutralization by convalescent plasma</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:142-146</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03471-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103385503" target="_blank" aria-label="Open Scopus in new tab">Scopus (393)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33780970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03471-w&amp;pmid=33780970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Garcia-Beltran et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Garcia-Beltran, W.F. ∙ Lam, E.C. ∙ St Denis, K. ...</div><div><strong>Multiple SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2372-2383</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102763305" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33743213/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.013&amp;pmid=33743213" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">Wibmer et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Wibmer, C.K. ∙ Ayres, F. ∙ Hermanus, T. ...</div><div><strong>SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:622-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01285-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102044511" target="_blank" aria-label="Open Scopus in new tab">Scopus (709)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33654292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01285-x&amp;pmid=33654292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62-4" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-4" aria-controls="sref62-4">Wu et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref62-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Wu, K. ∙ Werner, A.P. ∙ Koch, M. ...</div><div><strong>Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1468-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2102179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103371216" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33730471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2102179&amp;pmid=33730471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and/or (2) that they were more transmissible (V1, V2, and V3) (<span class="dropBlock reference-citations"><a id="body-ref-sref11-4" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-4" aria-controls="sref11-4">Faria et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref11-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Faria, N.R. ∙ Mellan, T.A. ∙ Whittaker, C. ...</div><div><strong>Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh2644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105876512" target="_blank" aria-label="Open Scopus in new tab">Scopus (883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33853970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh2644&amp;pmid=33853970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Pearson et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Pearson, C.A. ∙ Russell, T.W. ∙ Davies, N. ...</div><div><strong>Estimates of severity and transmissibility of novel SARS-CoV-2 variant 501Y.V2 in South Africa | CMMID Repository</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.24.20248822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105024215" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.24.20248822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Volz et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ..., COVID-19 Genomics UK (COG-UK) consortium</div><div><strong>Assessing transmissibility of SARS-CoV-2 lineage B.1.1.7 in England</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:266-269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03470-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103190451" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33767447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03470-x&amp;pmid=33767447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0190" role="paragraph">While it is likely that all the convergent mutations that are detectable at positively selected sites (<a id="crosref0850" href="#fig3">Figure&nbsp;3</a>) impact SARS-CoV-2 transmissibility and/or immune escape in at least some specific situations, it remains unclear what the relative fitness impacts of particular mutations at these sites are. It is, however, expected that population-wide frequencies of newly arising mutations that directly contribute to increased fitness should, at least initially, increase at a rate that is proportional to the magnitude of their fitness contribution. We therefore tested for changes in the frequencies of inter-lineage convergent mutations at the 28 non-signature mutation sites represented in <a id="crosref0855" href="#fig3">Figure&nbsp;3</a>, and the full complement of 37 signature substitution mutation sites found in the 501Y lineage viruses (<a id="crosref0860" href="#fig1">Figure&nbsp;1</a>). Specifically, this involved partitioning V1, V2, and V3 sequences deposited in GISAID by June 1 into “early” and “late” datasets that respectively contained sequences sampled before and after March 15, 2021: a date by which all of the internal branch mutations that yielded the detectable positive selection signals in our lineage-specific selection analysis datasets (i.e., those represented in <a id="crosref0865" href="#fig3">Figure&nbsp;3</a> and <a id="crosref0870" href="#mmc1">Table S1</a>) would have already arisen. The frequencies of mutations evident at the 37 signature and 28 non-signature mutation sites were then compared between each of the V1, V2, and V3 early and late dataset pairs.</div><div id="p0195" role="paragraph">Over 2-fold increases in frequency between March 15, and June 1, 2021 were detected in at least one of the three 501Y lineages for at least one of the observed mutations at 29/65 of the analyzed genome sites (these increases are all individually statistically significant in 2×2 contingency tables using the conservative Bonferroni multiple testing correction). Among these 29 sites are 20 where mutations in one (at 15 sites) or two (at five sites) of the 501Y lineages first converged on a signature mutation that characterizes a different 501Y lineage and then proceeded to double in frequency between March 15 and June 1, 2021 (<a id="crosref0875" href="#mmc2">Table S2</a>), an indication that the convergence mutations at these 20 sites may have each provided a fitness advantage. Based on the observed degree of frequency increases, mutations such as ORF1a/1708D (corresponding to nsp3/890D with 15.8 and &gt;12.0-fold increases in V2 and V3, respectively), S/26S (&gt;13-fold increase in V2), S/716I (3.7 and &gt;13.5-fold increases in V2 and V3, respectively), S/1027I (&gt;44-fold increase in V2), S/1118H (4.0 and &gt;20-fold increases in V2 and V3, respectively), S/1176F (19.5-fold increase in V2), and ORF3/171L (11.9-fold increase in V3) are the signature mutations that, in addition to the ORF1a/3675-3677Del, S/18F, S/417N/T E484K, and S/501Y mutations, are likely to have the greatest positive impact on the fitness of the 501Y lineage viruses within which they occur.</div><div id="p0200" role="paragraph">Similarly, among the nine positively selected sites where non-signature mutations both converge between viruses in two or more of the 501Y lineages and then more than double in frequency between March 15 and June 1, 2021, ORF1b/1522I (corresponding to Heicase/590I with 1.9-, 12.7-, and 4.8-fold increases in V1, V2, and V3, respectively), S/98F (2.5-, 5.3-, and &gt;6.0-fold increases in V1, V2, and V3, respectively), and E71T/R (respectively 5.8- and &gt;10-fold increases in V1) are likely the most fitness-enhancing mutations.</div><div id="p0205" role="paragraph">In total, 20/47 of the analyzed convergent mutations that were suggested by our global and lineage-specific positive selection analyses to have contributed to the fitness of 501Y lineage viruses prior to March 2021 more than doubled in frequency in at least one of the lineages between 15 March and June 1, 2021. The June 2021 dataset revealed a further nine previously undetected convergent mutations at 501Y lineage signature sites (ORF1a/1001I, ORF1a/1655N, ORF1a/1708D, ORF1b/970, S/215, S/1118, S/1176F, E/71L, and N/205; <a id="crosref0880" href="#mmc2">Table S2</a>) that were associated with positive selection signals in the global dataset (<a id="crosref0885" href="#fig4">Figure&nbsp;4</a>) and which also more than doubled in frequency between March 15 and June 2021. Of all the 501Y lineage mutations that have so far been observed, the convergent mutations at these 29 sites have the strongest corroborating evidence supporting their individual and/or collective contributions to the ongoing adaptation of 501Y lineage viruses during the present phase of the pandemic.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/550316fb-7cb5-4496-9281-c1a582adfe45/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.09.003/asset/f9f6027d-ecfa-4105-b9e1-0d488e25cfb6/main.assets/gr4.jpg" height="511" width="654" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/550316fb-7cb5-4496-9281-c1a582adfe45/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure 4</span> <span class="figure__title__text">Selection signals evident in the global data at the subset of sites identified by the positive selection, convergence, and mutation frequency change analyses as likely contributing to the fitness of the 501Y lineage viruses: a subset of sites and their associated amino acid states that we refer to as the 501Y lineage meta-signature</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">The strengths of detected selection signals (with the IFEL method) are indicated by the sizes of the red dots. Selection tests were performed on sequence data collected within the preceding three months (i.e., red spots plotted in April reflect the analysis of sequences sampled between January 1 and April 1). The vertical bar indicates December 1, 2020. The global frequencies of the represented mutations are indicated in gray. These frequencies are strongly biased by, and therefore track in many instances, the rapid rise of V1 viruses in the UK, Europe and North America: the regions of the world responsible for &gt;90% of all SARS-CoV-2 genome sequencing between January and June 2001.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-7"><h3>Where is the evolution of the 501Y lineages headed?</h3><div id="p0210" role="paragraph">Regardless of how exactly each of these 29 convergent mutations impact the fitness of 501Y lineage viruses, it is apparent that the evolution of these viruses will likely involve further selection-driven mutational convergence at these sites both between viruses within individual lineages, and between viruses in the different lineages. Based on our selection analyses, the convergence patterns that we have so far detected, and the rises in frequencies between March 15 and June 1, 2021 of 501Y viruses carrying particular convergent mutations, we can propose a “meta-signature” for the most adaptive amino acid states at 35 sites within 501Y lineage genomes (<a id="crosref0890" href="#mmc3">Table S3</a>; <a id="crosref0895" href="#fig4">Figure&nbsp;4</a>). In addition to the 29 convergent mutations that displayed frequency increases between March 15 and June 1, 2021, the meta-signature includes deletion mutations at ORF1a/3675–3677, S/69–70, S/144, and S/241–243 (which, while displaying convergence between the different 501Y lineages, were not amenable to selection analyses) and the convergent signature substitutions L18F, K417N/K, and N501Y (which were already at high frequencies in multiple 501Y lineages by March 15, 2021).</div><div id="p0215" role="paragraph">Before March 2021 most V1, V2, and V3 viruses respectively carried 10, 13, and 11 of the mutations within this meta-signature. By June 1, 2021, 17 different V1 variants (represented by 53 sequenced genomes) matched 13 of the meta-signature sites, two different V2 variants (represented by 20 sequenced genomes) matched 16 of the sites, and one V3 variant (represented by 4 sequences) matched 14 of the sites: i.e., in all three lineages viruses were present that had taken three additional mutational steps that converged on the meta-signature.</div><div id="p0220" role="paragraph">Given that 19/35 of the meta-signature sites are in Spike, these patterns can be best illustrated by examining convergence on the meta-signature Spike sites within the different 501Y lineages since October 2020 (<a id="crosref0900" href="#fig5">Figure&nbsp;5</a>). Whereas the prototype V1, V2, and V3 genomes, respectively, carried only six, six, and nine of the 19 Spike mutations in the meta-signature, by June 1, 2021 V1, V2, and V3 variants had arisen that, respectively, carried 10, 10, and 11 of these mutations (<a id="crosref0905" href="#fig5">Figure&nbsp;5</a>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.09.003/asset/7d5ca85f-575e-40f1-99e0-f44f4ca33a8b/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.09.003/asset/49f42151-9b71-48ce-99f0-8b00f1824501/main.assets/gr5.jpg" height="356" width="769" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/7d5ca85f-575e-40f1-99e0-f44f4ca33a8b/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure 5</span> <span class="figure__title__text">Weekly changes in the counts of sequences displaying multiple convergence mutations at the 19 sites in Spike predicted by our analyses to provide 501Y lineage viruses with selective advantages</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0075" role="paragraph">This 501Y lineage meta-signature includes the following mutations: 18F, 26R/L/S, 69-70Del, 98F, 138HY, 144Del, 215G/H/V/Y, 241-243Del, 417N/T, 484K, 501Y, 655Y, 681L/R/H, 701V, 716I, 1027I, 1118H, 1176F, 1264L. The matches plots (top row) indicate the numbers of sequenced V1, V2, and V3 genomes carrying a given number of matching mutations at sites on this list: archetypical V1, V2, and V3 sequences, respectively, have Spike sequences with six, six, and nine matches. The signature sequence plots (bottom row) indicate the counts of particular V1, V2, and V3 Spike sequence haplotypes with the highest numbers of matches and indicate the subsets of matching mutations in these haplotype sequences. The signature lists included together with these plots indicate the subset of mutations at the 19 convergence list sites that are present in the different Spike haplotype sequences represented in the plots. “.” symbols indicate the absence of a convergence list mutation, “-” symbols indicate the occurrence of convergence list deletion mutations and letters indicate the presence of convergence list amino acid substitutions.</div></div></div></div></figcaption></figure></div><div id="p0225" role="paragraph">Beyond the three 501Y lineages, we compared degrees of convergence on the 501Y meta-signature of all SARS-CoV-2 genome sequences in GISAID on June 1, 2021 that were classified as belonging to lineages designated by either Public Health England or the US CDC as variants of concern (VOCs), variants of interest (VOI) or variants under investigation (VUI; <a id="crosref0910" href="#mmc3">Table S3</a>). All of these lineages had viruses assigned to them that matched at least one of the meta-signature mutations. The lineages containing sequences with the most matches were B.1.526 and B.1.621 (both with modal matches&nbsp;= 5 and maximum&nbsp;= 7), suggesting that they too are possibly scaling the same fitness peak as the 501Y lineage viruses.</div><div id="p0230" role="paragraph">Other prominent VOC, VOI, and VUI lineages, however, had very few matches. For example, the best-matched sequences within the B.1.617.2, P.2, and R.1 lineages each had only three matches to the meta-signature (with modal matches in each being one, two, and one, respectively), implying that viruses in these lineages are likely scaling a different fitness peak to the one that the 501Y lineage viruses are on. While in the B.1.617.1 lineage (the sister lineage to B.1.617.2), most sequences only contain one match to the meta-signature, there are some sequences in this lineage that match it at six sites. This suggests that at least some sub-lineages within B.617 are climbing the same fitness peak as the 501Y lineage viruses.</div><div id="p0235" role="paragraph">The non-501Y lineage SARS-CoV-2 isolates that most closely match the meta-signature are found within B.1.620, a lineage first detected in Lithuania (but likely originating in Central Africa) that is presently not considered a VOI, VOC, or VUI (<span class="dropBlock reference-citations"><a id="body-ref-sref9-2" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9-2" aria-controls="sref9-2">Dudas et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref9-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Dudas, G. ∙ Hong, S.L. ∙ Potter, B.I. ...</div><div><strong>Travel-driven emergence and spread of SARS-CoV-2 lineage B.1.620 with multiple VOC-like mutations and deletions in Europe</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.05.04.21256637v1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.05.04.21256637v1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Whereas the modal number of meta-signature matches for members of B.1.620 is eight (3675–3677Del in ORF1a; 26S, 69–70Del, 241–243Del, 484K, 681H, 1027I, 1118H in Spike), some sequences within the lineage have ten matches, suggesting both that the members of this lineage are on the same fitness peak as the 501Y lineage viruses, and that they too are discovering predictable paths to its summit.</div><div id="p0240" role="paragraph">We therefore anticipate that the culmination of the currently ongoing evolutionary convergence of 501Y lineage viruses will yield a succession of variants possessing increasing subsets of 501Y lineage meta-signature mutations. The most important issue is not whether we correctly predict the emergence of a variant carrying mutations at every one of the 35 meta-signature sites. It is rather that the convergent mutations that are continuing to arise, both in members of the 501Y lineages and those of lineages such as B.1.620, B1.621, and B1.526, imply that all these viruses are presently on, and are actively scaling, the same broad peak in the fitness landscape. Whatever SARS-CoV-2 variants eventually summit that peak could be a considerably bigger problem for us than any we currently know, in that they might have any combinations of increased transmissibility, altered virulence and/or increased capacity to escape population immunity.</div><div id="p0245" role="paragraph">Although only time can test the accuracy of this prediction, it should also be possible using <i>in&nbsp;vitro</i> evolution to infer some amino acid sequence features at the adaptive summit of this fitness peak. An obvious, albeit potentially controversial, approach would be to use replicated, laboratory infections of either synthesized or sampled live viruses carrying complements of mutations that are representative of the current standing diversity within the V1, V2, and V3 lineages. In the presence of mixed sera from multiple previously infected and/or vaccinated individuals these infections would create the appropriate conditions both for genetic recombination to occur, and for selection to rapidly sort multiple recombination-generated combinations of input immune evasion, cell entry, and replication impacting mutations. Although the chimeras that ultimately dominate these <i>in&nbsp;vitro</i> infections will doubtlessly be cell-culture optimized (as opposed to transmission between, and replication within, humans optimized), they should nevertheless carry many combinations of mutations that will be relevant to the continuing pandemic and that should include some of the most concerning mutation combinations that might arise before the pandemic concludes: perhaps particularly so when 501Y lineage viruses start frequently recombining with each other. Even if the concerning combinations that are discovered are only triplets or quartets of mutations, these would still be invaluable hints at what we should start looking for when it comes to trawling the rapidly growing pool of SARS-CoV-2 genomic surveillance data for potential vaccine escape mutants and other potentially problematic variants.</div></section><section id="sec-2-8"><h3>Limitations of the study</h3><div id="p0250" role="paragraph">Selection analyses employed here are not well suited to detecting certain types of selection (e.g., directional selection), for which other specialized techniques can be used (e.g., the DEPS method). Despite our stringent filtering, some of the selection signals detected may have been false positives attributable to sequencing errors, undetected genetic recombination and/or inaccurate phylogenetic inference. Similarly, given the relatively low divergence of SARS-COV-2 genomes, lack of power to detect positive selection (i.e., enough synonymous and non-synonymous substitutions occurring at individual codon sites) was, and will remain, a persistent issue with detecting positive selection in SARS-CoV-2 sequences. Further, all of our comparative analyses were subject to temporal and spatial sampling biases, and country-to-country heterogeneity in time lags between when viruses were sampled, and their sequences became publicly accessible.</div></section></section><section id="sec-3"><h2>STAR★Methods</h2><section id="sec-3-1"><h3>Key resources table</h3><div id="p0255" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Reagent or resource</th><th>Source</th><th>Identifier</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>Raw sequence data</td><td>GISAID</td><td>Cannot be shared due to GISAID licensing rules. GISAID attribution table with accession numbers provided in <a id="crosref0920" href="#mmc5">Table S5</a>.</td></tr><tr><td>SARS-CoV-2 genome reference</td><td>NCBI</td><td>NC_045512</td></tr><tr><td>Sequence variation information</td><td>This paper</td><td><a id="intref0020" href="https://doi.org/10.5281/zenodo.5149112">https://doi.org/10.5281/zenodo.5149112</a></td></tr><tr class="rowsep"><td>Analyzed codon sequence alignments and summary files for V1,V2,V3 and other clade analyses</td><td>This paper</td><td><a id="intref0025" href="https://github.com/spond/SARS-CoV-2-clade-analysis" target="_blank">https://github.com/spond/SARS-CoV-2-clade-analysis</a></td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>Sequence data processing workflows and pipelines</td><td>COVID19 Galaxy project</td><td><a id="intref0030" href="https://github.com/veg/SARS-CoV-2" target="_blank">https://github.com/veg/SARS-CoV-2</a></td></tr><tr><td>bealign</td><td>BioExt Package</td><td><a id="intref0035" href="https://github.com/veg/bioext" target="_blank">https://github.com/veg/bioext</a></td></tr><tr><td>RapidNJ</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref52-1" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-1" aria-controls="sref52-1">Simonsen et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref52-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Simonsen, M. ∙ Mailund, T. ∙ Pedersen, C.N.S.</div><div><strong>Rapid neighbour-joining</strong></div><div>Springer Berlin Heidelberg,<div>Crandall, K.A. ∙ Lagergren, J. (Editors)</div>in: Algorithms in Bioinformatics. 2008; 113-122</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SimonsenT.MailundC.N.S.PedersenRapid+neighbour-joiningK.A.CrandallJ.LagergrenAlgorithms+in+Bioinformatics2008Springer+Berlin+Heidelberg113122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://birc.au.dk/software/rapidnj/" target="_blank">https://birc.au.dk/software/rapidnj/</a></td></tr><tr><td>HyPhy v2.5.31</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Kosakovsky Pond et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Kosakovsky Pond, S.L. ∙ Poon, A.F.Y. ∙ Velazquez, R. ...</div><div><strong>HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:295-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msz197" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85078566726" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31504749/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsz197&amp;pmid=31504749" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="http://www.hyphy.org/" target="_blank">http://www.hyphy.org/</a></td></tr><tr><td>RASCL</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Lucaci et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Lucaci, A.G. ∙ Zehr, J.D. ∙ Shank, S.D. ...</div><div><strong>RASCL: Rapid assessment of SARS-COV-2 clades enabled through molecular sequence analysis and its application to B.1.617.1 and B.1.617.2.</strong></div><div>2021</div><div><a id="intref0180" href="https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709" target="_blank">https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.G.LucaciJ.D.ZehrS.D.ShankD.BouvierH.MeiA.NekrutenkoS.L.Kosakovsky+PondRASCL%3A+Rapid+assessment+of+SARS-COV-2+clades+enabled+through+molecular+sequence+analysis+and+its+application+to+B.1.617.1+and+B.1.617.2.2021https%3A%2F%2Fvirological.org%2Ft%2Frascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2%2F709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="https://github.com/veg/SARS-CoV-2/tree/compact/clades" target="_blank">https://github.com/veg/SARS-CoV-2/tree/compact/clades</a></td></tr><tr><td>tn93-cluster</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Kosakovsky Pond et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Kosakovsky Pond, S.L. ∙ Weaver, S. ∙ Leigh Brown, A.J. ...</div><div><strong>HIV-TRACE (TRAnsmission Cluster Engine): a Tool for Large Scale Molecular Epidemiology of HIV-1 and Other Rapidly Evolving Pathogens</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:1812-1819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msy016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85051869178" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29401317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsy016&amp;pmid=29401317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://github.com/veg/tn93" target="_blank">https://github.com/veg/tn93</a></td></tr><tr><td>raxml-ng</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">Kozlov et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Kozlov, A.M. ∙ Darriba, D. ∙ Flouri, T. ...</div><div><strong>RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:4453-4455</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz305" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85074309434" target="_blank" aria-label="Open Scopus in new tab">Scopus (1861)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31070718/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz305&amp;pmid=31070718" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://github.com/amkozlov/raxml-ng" target="_blank">https://github.com/amkozlov/raxml-ng</a></td></tr><tr class="rowsep"><td>MAFFT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Katoh et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Katoh, K. ∙ Misawa, K. ∙ Kuma, K. ...</div><div><strong>MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</strong></div><div><em>Nucleic Acids Res.</em> 2002; <strong>30</strong>:3059-3066</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkf436" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-0037100671" target="_blank" aria-label="Open Scopus in new tab">Scopus (11062)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkf436&amp;pmid=12136088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://mafft.cbrc.jp/alignment/software/" target="_blank">https://mafft.cbrc.jp/alignment/software/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>ObservableHQ notebook detailing N501Y lineage-specific selection results (other clades included as well)</td><td>This paper</td><td><a id="intref0070" href="https://observablehq.com/@spond/n501y-clades@3752" target="_blank">https://observablehq.com/@spond/n501y-clades@3752</a></td></tr><tr><td>ObservableHQ notebook for calculating changes in mutation frequencies, relative frequency ratios, and selection profiles in SARS-CoV-2 sequences</td><td>This paper</td><td><a id="intref0075" href="https://observablehq.com/@spond/sc2-selection-trends" target="_blank">https://observablehq.com/@spond/sc2-selection-trends</a></td></tr><tr><td>ObservableHQ notebook for calculating selection strengths and profiles at individual sites using sliding window analysis.</td><td>This paper</td><td><a id="intref0080" href="https://observablehq.com/@spond/sc2-temporal-selection-trends" target="_blank">https://observablehq.com/@spond/sc2-temporal-selection-trends</a></td></tr><tr><td>ObservableHQ notebook for displaying a combined view of sites under selection in N501Y clades</td><td>This paper</td><td><a id="intref0085" href="https://observablehq.com/@spond/n501y-sites" target="_blank">https://observablehq.com/@spond/n501y-sites</a></td></tr><tr><td>ObservableHQ notebook for displaying a combined view of sites which may be experiencing convergence in N501Y clades</td><td>This paper</td><td><a id="intref0090" href="https://observablehq.com/@spond/clade-convergence" target="_blank">https://observablehq.com/@spond/clade-convergence</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901050-3" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-3-2"><h3>Resource availability</h3><section id="sec-3-2-1"><h4>Lead contact</h4><div id="p0260" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Tulio de Oliviera (<a id="intref0095" href="mailto:tuliodna@gmail.com">tuliodna@gmail.com</a>).</div></section><section id="sec-3-2-2"><h4>Materials availability</h4><div id="p0265" role="paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec-3-3"><h3>Method details</h3><section id="sec-3-3-1"><h4>Global SARS-CoV-2 dataset preparation</h4><div id="p0270" role="paragraph">Since genes/peptides are the targets of selection, unless specified otherwise here and hereafter, all analyses were performed on single genes (e.g., the spike gene) or peptide encoding gene segments (e.g., nsp3). We developed an open-source bioinformatics workflow to handle large volumes of sequencing data (&gt; 1,000,000 sequences) in a systematic and scalable manner (<a id="intref0100" href="https://covid19.galaxyproject.org/" target="_blank">https://covid19.galaxyproject.org/</a>). Until SARS-CoV-2 emerged in 2019, analyses of natural selection using a few thousand viral sequences would be considered “large scale” (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Murrell et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Murrell, B. ∙ Moola, S. ∙ Mabona, A. ...</div><div><strong>FUBAR: a fast, unconstrained bayesian approximation for inferring selection</strong></div><div><em>Mol. Biol. Evol.</em> 2013; <strong>30</strong>:1196-1205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/mst030" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84876522258" target="_blank" aria-label="Open Scopus in new tab">Scopus (844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23420840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmst030&amp;pmid=23420840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our approach represents a substantial technical advance over the previous state of the art. Because we were also interested in temporal trends in selective pressures, we partitioned all the sequences into three-month intervals based on the date of sampling, and analyzing sliding temporal windows, starting on the 1st of each month from March 2020 to May 2021:<div id="olist0010" role="list"><div id="o0010" role="listitem"><div class="label">1</div><div class="content"><div id="p0275" role="paragraph">We downloaded and curated GISAID sequence data, removed sequences that contained too many ambiguous or unresolved nucleotides, and identified all unique haplotypes for each of the 23 genomic regions that were then analyzed individually (<i>3C, E, endornase, exonuclease, helicase, leader, M, methyltransferase, N, nsp2, nsp3, nsp4, nsp6, nsp7, nsp8, nsp9, nsp10, ORF3a, ORF6, ORF7a, ORF8, RdRp, S</i>).</div></div></div><div id="o0015" role="listitem"><div class="label">2</div><div class="content"><div id="p0280" role="paragraph">We translated each unique haplotype into an amino-acid sequence via a procedure that allows correction of out-of-frame sequencing errors (while not common, there are several thousand sequences in GISAID which have these errors), following which these translated sequences were mapped to the NCBI SARS-CoV-2 genome reference using the <i>bealign</i> tool from the BioExt package (github.com/veg/bioext) using the scoring matrix developed for rapidly evolving RNA viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Nickle et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Nickle, D.C. ∙ Heath, L. ∙ Jensen, M.A. ...</div><div><strong>HIV-specific probabilistic models of protein evolution</strong></div><div><em>PLoS ONE.</em> 2007; <strong>2</strong>:e503</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0000503" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-39849090643" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17551583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0000503&amp;pmid=17551583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We did not keep track of insertions relative to the reference genome (this is common practice in the field, since there are no widely circulating strains with evidence of insertions). Therefore, mapping to the reference sequences of individual gene and gene segments generated multiple sequence alignments that were suitable for downstream analyses. These data were directly used for tabulating mutation frequencies and tracking haplotypes with mutations that matched specific mutation signatures (for example, see <a id="intref0105" href="https://observablehq.com/@spond/spike-trends" target="_blank">https://observablehq.com/@spond/spike-trends</a>)</div></div></div><div id="o0020" role="listitem"><div class="label">3</div><div class="content"><div id="p0285" role="paragraph">Direct comparative analyses of tens or hundreds of thousands of haplotypes is technically very challenging but is not necessary with respect to extracting the majority of the available selection signal. This is due to two factors. First, much of the variation in individual genomes is either artifactual (sequencing or assembly errors), or not biologically informative (occurs only in a few strains). Second, analyses in such settings need to account for a well-known feature of viral evolution (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Poon et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Poon, A.F.Y. ∙ Lewis, F.I. ∙ Pond, S.L.K. ...</div><div><strong>An evolutionary-network model reveals stratified interactions in the V3 loop of the HIV-1 envelope</strong></div><div><em>PLoS Comput. Biol.</em> 2007; <strong>3</strong>:e231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0030231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-36948998606" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18039027/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0030231&amp;pmid=18039027" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) where terminal branches include “dead-end” mutation events within individual hosts which, although maladaptive or deleterious at the population level (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Pybus et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Pybus, O.G. ∙ Rambaut, A. ∙ Belshaw, R. ...</div><div><strong>Phylogenetic evidence for deleterious mutation load in RNA viruses and its contribution to viral evolution</strong></div><div><em>Mol. Biol. Evol.</em> 2007; <strong>24</strong>:845-852</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msm001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-33847649474" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17218639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsm001&amp;pmid=17218639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), have not been “seen” by natural selection. Mutations that map to internal tree branches on the other hand are far less likely to be severely maladaptive since they must include at least one transmission event. We therefore implemented three layers of compression to reduce the numbers of sequence haplotypes that needed to be subjected to comparative analyses.</div></div></div><div id="o0025" role="listitem"><div class="label">1</div><div class="content"><div id="p0290" role="paragraph">We did not retain copies of identical sequences. Instead, all identical sequences were represented by a single haplotype. This was because comparative phylogenetic analyses of evolutionary rates do not gain information from the inclusion of identical sequences.</div></div></div><div id="o0030" role="listitem"><div class="label">2</div><div class="content"><div id="p0295" role="paragraph">We filtered putative sequencing errors and “problematic” sequences. A mutation that occurs in X out of N total sequences (counting all sequences, not just the unique ones) was considered to be an “error” if the binomial probability of observing X or more error mutations at a site was sufficiently high (in our case p &gt; 0.999) assuming a sequencing error rate of 1:10,000. For example, if N&nbsp;= 500,000, then X would be 29. This means that unless a mutation occurred in 30/500,000 or more individual sequences, it would have been treated as an error and replaced with a phylogenetically uninformative gap character (i.e., “-”). One exception to this rule occurred when two rare mutations <b>a</b> and <b>b</b> that would have been filtered out if considered in isolation, occurred together in more than one sequence (i.e., they represented rare linked mutations); we retained such mutations because the probability of coincidental doublets occurring by chance is quadratically small. Additionally, assuming that the distribution of differences from reference sequences across all sequences had a mean, M, and standard deviation, D, we further removed all sequences that had more than M&nbsp;+ 5D mutations. These sequences were deemed to be “unusually” mutated and potentially the product of sequencing device contamination, extensive sequencing errors, and/or real/artifactual sequencing assembly-associated recombination.</div></div></div><div id="o0035" role="listitem"><div class="label">3</div><div class="content"><div id="p0300" role="paragraph">We grouped the remaining filtered haplotypes into clusters based on complete linkage using TN93 distances for computing pairwise sequence similarities (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Rhee et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Rhee, S.-Y. ∙ Magalis, B.R. ∙ Hurley, L. ...</div><div><strong>National and International Dimensions of Human Immunodeficiency Virus-1 Sequence Clusters in a Northern California Clinical Cohort</strong></div><div><em>Open Forum Infect. Dis.</em> 2019; <strong>6</strong>:ofz135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ofid/ofz135" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85066411962" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31041344/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fofid%2Fofz135&amp;pmid=31041344" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); sequences were placed in a cluster if and only if all of the pairwise distances between them were ≤ d, where <b>d</b> is a gene-specific threshold, e.g., d&nbsp;= 0.001 for S. All the sequences belonging to the same cluster were represented by a single “median” sequence from the cluster.</div></div></div><div id="o0040" role="listitem"><div class="label">4</div><div class="content"><div id="p0305" role="paragraph">For example, consider Spike sequences sampled between 01 March 2021 and 31 May 2021. 534,345 sequences passed the initial step 1 filter. Following the step 3a filter, these were reduced to 63,559 unique haplotypes. Following error correction and haplotype recompression in step 3b (where ‘-’ characters were introduced to reflect corrected putative sequencing errors and where ‘-’ matched any resolved characters) 41,103 haplotypes remained. In step 3c these were then compressed down to 5,147 clusters, each yielding a single representative haplotype sequence that was used for downstream selection analyses.</div></div></div><div id="o0045" role="listitem"><div class="label">4</div><div class="content"><div id="p0310" role="paragraph">We next reconstructed phylogenetic trees for the remaining haplotype sequences using RapidNJ (<span class="dropBlock reference-citations"><a id="body-ref-sref52-2" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52-2" aria-controls="sref52-2">Simonsen et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref52-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Simonsen, M. ∙ Mailund, T. ∙ Pedersen, C.N.S.</div><div><strong>Rapid neighbour-joining</strong></div><div>Springer Berlin Heidelberg,<div>Crandall, K.A. ∙ Lagergren, J. (Editors)</div>in: Algorithms in Bioinformatics. 2008; 113-122</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SimonsenT.MailundC.N.S.PedersenRapid+neighbour-joiningK.A.CrandallJ.LagergrenAlgorithms+in+Bioinformatics2008Springer+Berlin+Heidelberg113122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div><div id="o0050" role="listitem"><div class="label">5</div><div class="content"><div id="p0315" role="paragraph">We used HyPhy v2.5.31 (<a id="intref0110" href="http://www.hyphy.org/" target="_blank">http://www.hyphy.org/</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Kosakovsky Pond et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Kosakovsky Pond, S.L. ∙ Poon, A.F.Y. ∙ Velazquez, R. ...</div><div><strong>HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:295-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msz197" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85078566726" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31504749/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsz197&amp;pmid=31504749" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to perform a series of selection analyses. This version of HyPhy includes many optimizations that were introduced specifically to deal with large SARS-CoV-2 datasets; since March 2020, targeted optimizations for trees with ≥ 1,000 leaves allowed HyPhy to process 10-25 times as many sequences as earlier versions. We performed SLAC (for substitution mapping (<span class="dropBlock reference-citations"><a id="body-ref-sref23-3" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-3" aria-controls="sref23-3">Kosakovsky Pond and Frost, 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Kosakovsky Pond, S.L. ∙ Frost, S.D.W.</div><div><strong>Not so different after all: a comparison of methods for detecting amino acid sites under selection</strong></div><div><em>Mol. Biol. Evol.</em> 2005; <strong>22</strong>:1208-1222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msi105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-17744396121" target="_blank" aria-label="Open Scopus in new tab">Scopus (1654)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15703242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsi105&amp;pmid=15703242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)), FEL (for pervasive positive diversifying and negative selection detection (<span class="dropBlock reference-citations"><a id="body-ref-sref23-4" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-4" aria-controls="sref23-4">Kosakovsky Pond and Frost, 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Kosakovsky Pond, S.L. ∙ Frost, S.D.W.</div><div><strong>Not so different after all: a comparison of methods for detecting amino acid sites under selection</strong></div><div><em>Mol. Biol. Evol.</em> 2005; <strong>22</strong>:1208-1222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msi105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-17744396121" target="_blank" aria-label="Open Scopus in new tab">Scopus (1654)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15703242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsi105&amp;pmid=15703242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)), and MEME (for episodic positive diversifying selection detection (<span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Murrell et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Murrell, B. ∙ Weaver, S. ∙ Smith, M.D. ...</div><div><strong>Gene-wide identification of episodic selection</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:1365-1371</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msv035" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84929648750" target="_blank" aria-label="Open Scopus in new tab">Scopus (323)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25701167/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsv035&amp;pmid=25701167" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) analyses that were restricted to considering only mutations mapping to internal branches of inferred trees. These analyses reported p values and inferred dS and dN rates and ratios for individual codon sites.</div></div></div></div></div></section><section id="sec-3-3-2"><h4>Lineage-specific dataset preparation</h4><div id="p0320" role="paragraph">We used the RASCL tool (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Lucaci et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Lucaci, A.G. ∙ Zehr, J.D. ∙ Shank, S.D. ...</div><div><strong>RASCL: Rapid assessment of SARS-COV-2 clades enabled through molecular sequence analysis and its application to B.1.617.1 and B.1.617.2.</strong></div><div>2021</div><div><a id="intref0180" href="https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709" target="_blank">https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.G.LucaciJ.D.ZehrS.D.ShankD.BouvierH.MeiA.NekrutenkoS.L.Kosakovsky+PondRASCL%3A+Rapid+assessment+of+SARS-COV-2+clades+enabled+through+molecular+sequence+analysis+and+its+application+to+B.1.617.1+and+B.1.617.2.2021https%3A%2F%2Fvirological.org%2Ft%2Frascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2%2F709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to perform a more detailed analysis of downsampled V1, V2, and V3 gene and gene-segment datasets. For a given gene or gene segment we aligned all sequences from an individual lineage (i.e., V1, V2 or V3) and reference sequences (GISAID unique haplotypes in the corresponding gene/peptide encoding gene segment) to the GenBank reference genome protein sequence for that gene/peptide encoding gene segment using <i>bealign</i> with the HIV-BETWEEN-F scoring matrix which is optimized for low-diversity viral sequences.</div><div id="p0325" role="paragraph">Because the codon-based selection analyses that we performed gain no power from including identical sequences, and minimal power from including sequences that are essentially identical, we filtered the V1, V2, V3, and reference (GISAID) sequences using pairwise genetic distances complete linkage clustering with the <i>tn93-cluster</i> tool (<a id="intref0115" href="https://github.com/veg/tn93" target="_blank">https://github.com/veg/tn93</a>; (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Kosakovsky Pond et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Kosakovsky Pond, S.L. ∙ Weaver, S. ∙ Leigh Brown, A.J. ...</div><div><strong>HIV-TRACE (TRAnsmission Cluster Engine): a Tool for Large Scale Molecular Epidemiology of HIV-1 and Other Rapidly Evolving Pathogens</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:1812-1819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msy016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85051869178" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29401317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsy016&amp;pmid=29401317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)). All groups of sequences that were within <b>D</b> genetic distance (determined using a Tamura-Nei 93 nucleotide substitution model) of every other sequence in the group were represented by a single randomly chosen sequence in the group. We set <b>D</b> at <i>0.0001</i> for lineage-specific sequence sets, and at <i>0.0015</i> for GISAID reference (or “background”) sequence sets. We restricted the reference sequence set to sequences sampled before 15 October 2020 since we were specifically interested in, on a lineage-by-lineage basis, disentangling the impacts of selective processes operating before this date from those operating thereafter. This date approximately marks what appears to have been a major shift in the selective environment within which SARS-CoV-2 is evolving.</div><div id="p0330" role="paragraph">We inferred a maximum likelihood tree from the combined sequence dataset with <i>raxml-ng (</i><span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2"><i>Kozlov et&nbsp;al., 2019</i></a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Kozlov, A.M. ∙ Darriba, D. ∙ Flouri, T. ...</div><div><strong>RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:4453-4455</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz305" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85074309434" target="_blank" aria-label="Open Scopus in new tab">Scopus (1861)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31070718/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz305&amp;pmid=31070718" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><i>)</i> using default settings (GTR+G nucleotide substitution model and 20 starting trees). We partitioned internal branches in the resulting tree into two non-overlapping sets used for testing via encoded annotations made to the Newick tree. Because of low phylogenetic resolution in some of the genes/peptide encoding segments, not all analyses were possible for all segments/genes. In particular this is true when lineage V1, V2 or V3 sequences were not monophyletic in a specific gene/segment, and no internal branches could be labeled as belonging to the foreground lineage.</div></section><section id="sec-3-3-3"><h4>Lineage-specific selection analyses</h4><div id="p0335" role="paragraph">We used HyPhy v2.5.31 (<a id="intref0120" href="http://www.hyphy.org/" target="_blank">http://www.hyphy.org/</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref25-3" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-3" aria-controls="sref25-3">Kosakovsky Pond et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref25-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Kosakovsky Pond, S.L. ∙ Poon, A.F.Y. ∙ Velazquez, R. ...</div><div><strong>HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:295-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msz197" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85078566726" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31504749/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsz197&amp;pmid=31504749" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to perform a series of selection analyses. As with the analysis of global datasets we considered only internal-branch mutations in the lineage-specific selection analyses</div><div id="p0340" role="paragraph">We performed codon-site-level tests for episodic diversifying (MEME) (<span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Murrell et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Murrell, B. ∙ Weaver, S. ∙ Smith, M.D. ...</div><div><strong>Gene-wide identification of episodic selection</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:1365-1371</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msv035" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84929648750" target="_blank" aria-label="Open Scopus in new tab">Scopus (323)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25701167/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsv035&amp;pmid=25701167" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and pervasive positive or negative selection (FEL) (<span class="dropBlock reference-citations"><a id="body-ref-sref23-5" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-5" aria-controls="sref23-5">Kosakovsky Pond and Frost, 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Kosakovsky Pond, S.L. ∙ Frost, S.D.W.</div><div><strong>Not so different after all: a comparison of methods for detecting amino acid sites under selection</strong></div><div><em>Mol. Biol. Evol.</em> 2005; <strong>22</strong>:1208-1222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msi105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-17744396121" target="_blank" aria-label="Open Scopus in new tab">Scopus (1654)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15703242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsi105&amp;pmid=15703242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on the internal branches of the V1,V2 or V3 clade datasets containing sequences deposited in GISAID by 20 April 2021, to infer the selective dynamics at individual codon-sites across the different SARS-CoV2 genes. Analyses were run with default settings using–branches Internal command line flags to restrict dN/dS testing to internal branches only. We only considered as significant those selection signals detected at codon-sites that did not contain nucleotides expressed in multiple frames.</div><div id="p0345" role="paragraph">We performed model-based predictions of codons expected to arise in SARS-CoV-2 based on the evolution of related coronaviruses in other host species was determined using the PRIME method <a id="intref0125" href="http://hyphy.org/w/index.php/PRIME" target="_blank">http://hyphy.org/w/index.php/PRIME</a> with default settings.</div><div id="p0350" role="paragraph">We combined the results of all these analyses using a Python script and visualized them using several open source libraries in an ObservableHQ notebook (<a id="intref0130" href="https://observablehq.com/@spond/n501y-clades" target="_blank">https://observablehq.com/@spond/n501y-clades</a>).</div></section><section id="sec-3-3-4"><h4>Calculating mutation frequency changes</h4><div id="p0355" role="paragraph">To calculate the relative ratio of a given mutation frequency (M) for a given time period, we utilized the curated variant dataset with metadata based on GISAID sequences (see additional resources). We used the GISAID Pangolin annotation to extract sequences assigned to the V1, V2 or V3 lineages discarding all sequences for which sampling dates were not recorded. Given a cutoff date D, we binned all sequences into those collected prior to D, and at or after D. We next generated a 2×2 table (before/after D, with/without mutation M), performed a chi-square test, and computed the relative ratio for RR(M) as the ratio of mutation frequencies after&nbsp;/ before the cutoff date. P values from the test were adjusted using a conservative Bonferroni multiple testing correction, using the total number of distinct mutations in a given gene as the number of tests. These calculations were carried out in a web-browser inside an interactive notebook (<a id="intref0135" href="https://observablehq.com/@spond/sc2-selection-trends" target="_blank">https://observablehq.com/@spond/sc2-selection-trends</a>)</div></section><section id="sec-3-3-5"><h4>Identification of potentially adaptive convergent mutations</h4><div id="p0360" role="paragraph">We produced a list of 501Y lineage signature mutation sites at which mutations arising in viruses of any one of the 501Y lineages during or before March 2021 converged on the signature mutation states of viruses in another 501Y lineage. To this list of sites we added a list of non-signature mutation sites at which convergent mutations occurring before March 2021 were observed between two or more different 501Y lineages that incurred enough convergent non-synonymous mutations along internal tree branches (i.e., excluding mutations mapping to terminal tree branches) to trigger positive selection signals with associated MEME or FEL p values&nbsp;&lt; 0.05. We then tested this combined “convergence list” for evidence of its constituent convergent mutations having more than doubled in frequency within V1, V2 or V3 sequences sampled between 15 March and 01 June 2021 relative to frequencies seen within these lineages before that time. We then repeated this test with 501Y lineage signature mutation sites at which convergent mutations had not been detected prior to March 2021. Finally, all convergent mutations analyzed in these tests that (1) more than doubled in frequency within individual 501Y lineages between 15 March 2021 and 01 June 2021, and (2) occurred at signature/non-signature mutation sites displaying significant signals of positive selection in either the global SARS-CoV-2 sequence dataset (FEL p value&nbsp;&lt; 0.05) or any one of the lineage-specific datasets (MEME/FEL p values&nbsp;&lt; 0.05) were identified as the mutations most likely to positively impact the fitness of the 501Y lineage viruses. This list of mutations was merged with the list of deletion mutations that characterize the different 501Y lineages and the three cardinal 501Y lineage signature mutations, L18F, K417N/K and N501Y which, due to already high frequencies in multiple 501Y lineages could not have doubled in frequency in more than a single lineage between 15 March and 01 June 2021. This final mutation list is what we called the 501Y meta-signature.</div></section></section><section id="sec-3-4"><h3>Quantification and statistical analysis</h3><div id="p0365" role="paragraph">Comparative sequence analyses were conducted in HyPhy v2.5.31 as described in the method details section. Detection of individual sites subject to various selective forces were based on published approaches described in the original publications (cited in the methods details section) with statistical significance based on likelihood ratio tests with asymptotic test statistic distributions; p&nbsp;values used are reported in the text. Effective sample sizes for these tests are difficult to quantify, but the number of unique sequences and total tree branch lengths are both positively correlated with reduction in sampling variance of estimates (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Scheffler et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Scheffler, K. ∙ Murrell, B. ∙ Kosakovsky Pond, S.L.</div><div><strong>On the validity of evolutionary models with site-specific parameters</strong></div><div><em>PLoS ONE.</em> 2014; <strong>9</strong>:e94534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0094534" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84899509429" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24722425/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0094534&amp;pmid=24722425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Numbers of haplotypes and sequence divergence are available in the corresponding data repositories in the key resources table.</div></section><section id="sec-3-5"><h3>Additional resources</h3><div id="p0370" role="paragraph">ObservableHQ notebook detailing N501Y lineage-specific selection results (other clades included as well): <a id="intref0140" href="https://observablehq.com/@spond/n501y-clades@3752" target="_blank">https://observablehq.com/@spond/n501y-clades@3752</a></div><div id="p0375" role="paragraph">ObservableHQ notebook for calculating changes in mutation frequencies, relative frequency ratios, and selection profiles in SARS-CoV-2 sequences: <a id="intref0145" href="https://observablehq.com/@spond/sc2-selection-trends" target="_blank">https://observablehq.com/@spond/sc2-selection-trends</a></div><div id="p0380" role="paragraph">ObservableHQ notebook for calculating selection strengths and profiles at individual sites using sliding window analysis: <a id="intref0150" href="https://observablehq.com/@spond/sc2-temporal-selection-trends" target="_blank">https://observablehq.com/@spond/sc2-temporal-selection-trends</a></div><div id="p0385" role="paragraph">ObservableHQ notebook for displaying a combined view of sites under selection in N501Y clades: <a id="intref0155" href="https://observablehq.com/@spond/n501y-sites" target="_blank">https://observablehq.com/@spond/n501y-sites</a></div><div id="p0390" role="paragraph">ObservableHQ notebook for displaying a combined view of sites which may be experiencing convergence in N501Y clades: <a id="intref0160" href="https://observablehq.com/@spond/clade-convergence" target="_blank">https://observablehq.com/@spond/clade-convergence</a></div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0395" role="paragraph">We gratefully acknowledge all of the authors from the originating laboratories responsible for obtaining the specimens and the submitting laboratories where genetic sequence data were generated and shared via the GISAID Initiative, on which this research is based (<a id="crosref1040" href="#mmc5">Table S5</a>). We thank Carolyn Williamson, Robert Wilkinson, Valarie Mizrahe, Jonathan Blackburn, and Nicola Mulder for reading and commenting on the manuscript. The Wellcome Trust funded D.P.M. (222574/Z/21/Z), D.L.R. (220977/Z/20/Z), P.L. (2206298/Z/17/Z), and O.A.M. (206369/Z/17/Z). The US National Institutes of Health funded S.L.K.P. (R01 AI134384 and AI140970), J.O.W. (R01 AI13599), G.W.H. (1U01Al152151-01), and J.E.S. and H.T. (via H3ABioNet U24HG006941). The US National Science Foundation funded S.L.K.P. (RAPID 2027196 NSF/DBI,BIO), The UK Medical Research Council funded D.L.R. (MC_UU_1201412) and COG-UK (via UK Research &amp; Innovation, the National Institute of Health Research, and Genome Research Limited operating as the Wellcome Sanger Institute). The European Research Council funded P.L. (725422-ReservoirDOCS and 874850-MOOD). The South African Medical Research Council funded NGS-SA via the Strategic Health Innovation Partnerships Unit of the South African Medical Research Council and Department of Science and Innovation.</div><section id="sec-4"><h3>Author contributions</h3><div id="p0400" role="paragraph">Conceptualization, D.P.M., T.d.O., and S.L.K.P.; methodology, D.P.M., B.M., S.W., A.G.L., A.N., and S.L.K.P.; software programming, S.W., A.N., A.G.L., and S.L.K.P.; validation, O.A.M. and D.P.M.; formal analysis; D.P.M., S.L.K., S.D.S., and O.A.M.; investigation, D.P.M., J.G., S.N., Y.P., L.S., H.T., E.J.S., and E.W.; resources, NGS-SA and COG-UK; data curation, A.N., S.L.K.P., S.W., and A.G.L.; writing – original draft preparation, D.P.M., S.L.K.P., O.A.M., D.L.R., R.K.G., J.O.M., T.d.O., P.L., R.J.L., and G.W.H.; writing – review &amp; editing preparation, D.P.M., S.L.K.P., O.A.M., D.L.R., R.K.G., J.O.M., T.d.O., P.L., G.W.H., and R.J.L.; visualization preparation, S.W., S.D.S., S.L.K.P., and D.P.M., supervision, T.d.O. and S.L.K.P.; project administration, S.L.K.P., A.N., and T.d.O.; funding acquisition, S.L.K.P., A.N., and T.d.O.</div></section><section id="sec-5"><h3>Declaration of interests</h3><div id="p0405" role="paragraph">J.O.W. has been funded by Gilead Sciences, LLC (completed) and the CDC (ongoing) via grants and contracts to his institution unrelated to this research.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (7)</h2><div id="p0415" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/51242a28-447e-4cf9-b1ed-dbec19a52995/mmc1.zip" download="mmc1.zip" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Archive (2.93 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Sites evolving under positive selection in 501Y lineage-specific datasets, related to Figure&nbsp;3</div><div id="espara0010" role="paragraph">Also indicated are the alternative encoded amino acids at these sites with subscripts associated with the encoded amino acid states indicating numbers of analyzed sequences encoding those amino acids. Black diamonds indicate encoded amino acid states in lineages that are different to the reference strain, red squares indicate significant evidence (IFEL p value&nbsp;&lt; 0.05) of pervasive positive selection, red circles indicate significant evidence (MEME p value&nbsp;&lt; 0.05) of episodic selection with numbers indicating the numbers of specific tree branches where this is evident, and green squares indicate significant evidence of pervasive negative selection (IFEL p value&nbsp;&lt; 0.05)</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/c12e46b2-e6cd-48db-9a3e-e16a4d0c5d21/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (31.17 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. Changes in mutation frequencies at signature and/or convergent mutation sites within 501Y lineage viruses between March 15, and June 1, 2021 that are detectably evolving under positive selection in individual lineages and/or in the global SARS-CoV-2 dataset, related to Figure&nbsp;4</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/5a84e664-92f6-483e-8c43-ca282c5311d9/mmc3.pdf" download="mmc3.pdf" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">PDF (25.24 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Mutations comprising the 501Y lineage meta-signature, related to Figures 4 and 5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/8356713d-7abb-476d-a547-799b9a40c18c/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (13.08 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Sequences within different SARS-COV-2 lineages of concern, lineages of interest, and lineages under investigation that have most converged on the 501Y lineage meta-signature, related to Figure&nbsp;5</div><div id="espara0015" role="paragraph">Indicated in green are mutations that converge on the 501Y lineage meta-signature which are not signature/lineage-defining mutations in the lineages from which the sequences were drawn</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/06ad78f2-32c6-44ff-b221-0dcfaafcd444/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (11.72 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. GISAID acknowledgments and SARS-CoV-2 sequence accession numbers, related to the Acknowledgments</div></div></div><div class="core-supplementary-material" data-category="document"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/e368c2e3-a74e-4f5b-b407-a7a315fc2562/mmc6.docx" download="mmc6.docx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Document (19.55 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Appendix 1. NGS-SA authors</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.09.003/attachment/146d6194-686d-4739-8104-2a94101f4988/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (36.21 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Appendix 2. COG-UK authors</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0025" role="paragraph">Nucleotide sequence data were obtained from GISAID (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Elbe and Buckland-Merrett, 2017</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Elbe, S. ∙ Buckland-Merrett, G.</div><div><strong>Data, disease and diplomacy: GISAID’s innovative contribution to global health</strong></div><div><em>Glob. Chall.</em> 2017; <strong>1</strong>:33-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gch2.1018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85163179799" target="_blank" aria-label="Open Scopus in new tab">Scopus (1137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31565258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgch2.1018&amp;pmid=31565258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) on 01June 2021 but are only available under the data access criteria specified by GISAID. Accession numbers are listed in <a id="crosref0175" href="#mmc5">Table S5</a>. To register with GISAID for access to these sequences, register to do so at <a id="intref0010" href="https://www.gisaid.org/registration/register/" target="_blank">https://www.gisaid.org/registration/register/</a>.</div><div id="p0030" role="paragraph">All analyzed multiple sequence alignments containing compressed de-identified gene or gene segment sequences have been deposited at Zenodo and are publicly available as of the date of publication. DOIs are listed in the key resources table.</div><div id="p0035" role="paragraph">All original code has been deposited at Zenodo and is publicly available as of the date of publication. DOIs are listed in the key resources table.</div><div id="p0040" role="paragraph">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Althaus, C.L. ∙ Baggio, S. ∙ Reichmuth, M.L. ...</div><div><strong>A tale of two variants: Spread of SARS-CoV-2 variants Alpha in Geneva, Switzerland, and Beta in South Africa</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252268" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33269368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252268&amp;pmid=33269368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Annavajhala, M.K. ∙ Mohri, H. ∙ Zucker, J.E. ...</div><div><strong>A Novel SARS-CoV-2 Variant of Concern, B.1.526, Identified in New York</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85116172155" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33655278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252259&amp;pmid=33655278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Campbell, K.M. ∙ Steiner, G. ∙ Wells, D.K. ...</div><div><strong>Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles</strong></div><div><em>BioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.30.016931" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85098906862" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.30.016931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Cele, S. ∙ Gazy, I. ∙ Jackson, L. ..., Network for Genomic Surveillance in South Africa, COMMIT-KZN Team</div><div><strong>Escape of SARS-CoV-2 501Y.V2 from neutralization by convalescent plasma</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:142-146</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03471-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103385503" target="_blank" aria-label="Open Scopus in new tab">Scopus (393)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33780970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03471-w&amp;pmid=33780970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Collier, D.A. ∙ De Marco, A. ∙ Ferreira, I.A.T.M. ..., CITIID-NIHR BioResource COVID-19 Collaboration, COVID-19 Genomics UK (COG-UK) Consortium</div><div><strong>Sensitivity of SARS-CoV-2 B.1.1.7 to mRNA vaccine-elicited antibodies</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:136-141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03412-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102509491" target="_blank" aria-label="Open Scopus in new tab">Scopus (497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33706364/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03412-7&amp;pmid=33706364" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Cottam, E.M. ∙ Maier, H.J. ∙ Manifava, M. ...</div><div><strong>Coronavirus nsp6 proteins generate autophagosomes from the endoplasmic reticulum via an omegasome intermediate</strong></div><div><em>Autophagy.</em> 2011; <strong>7</strong>:1335-1347</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/auto.7.11.16642" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-80655124392" target="_blank" aria-label="Open Scopus in new tab">Scopus (205)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21799305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Fauto.7.11.16642&amp;pmid=21799305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>de Oliveira, T. ∙ Lutucuta, S. ∙ Nkengasong, J. ...</div><div><strong>A novel variant of&nbsp;interest of SARS-CoV-2 with multiple spike mutations is identified from travel surveillance in Africa</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.02.23.21252268" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85114081907" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.02.23.21252268" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Dearlove, B. ∙ Lewitus, E. ∙ Bai, H. ...</div><div><strong>A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:23652-23662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2008281117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85091566837" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32868447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2008281117&amp;pmid=32868447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9-1" title="View in article">9.</a></div><div>Dudas, G. ∙ Hong, S.L. ∙ Potter, B.I. ...</div><div><strong>Travel-driven emergence and spread of SARS-CoV-2 lineage B.1.620 with multiple VOC-like mutations and deletions in Europe</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.05.04.21256637v1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.05.04.21256637v1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Elbe, S. ∙ Buckland-Merrett, G.</div><div><strong>Data, disease and diplomacy: GISAID’s innovative contribution to global health</strong></div><div><em>Glob. Chall.</em> 2017; <strong>1</strong>:33-46</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/gch2.1018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85163179799" target="_blank" aria-label="Open Scopus in new tab">Scopus (1137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31565258/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fgch2.1018&amp;pmid=31565258" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Faria, N.R. ∙ Mellan, T.A. ∙ Whittaker, C. ...</div><div><strong>Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil</strong></div><div><em>Science.</em> 2021; <strong>372</strong>:815-821</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh2644" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105876512" target="_blank" aria-label="Open Scopus in new tab">Scopus (883)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33853970/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh2644&amp;pmid=33853970" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Garcia-Beltran, W.F. ∙ Lam, E.C. ∙ St Denis, K. ...</div><div><strong>Multiple SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2372-2383</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102763305" target="_blank" aria-label="Open Scopus in new tab">Scopus (881)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33743213/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.013&amp;pmid=33743213" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">13.</a></div><div>Garry, R.F. ∙ Andersen, K.G. ∙ Gallaher, W.R. ...</div><div><strong>Spike protein mutations in novel SARS-CoV-2 ‘variants of concern’ commonly occur&nbsp;in or near indels</strong></div><div>2021</div><div><a id="intref0170" href="https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605" target="_blank">https://virological.org/t/spike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels/605</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.F.GarryK.G.AndersenW.R.GallaherT.T.-Y.LamK.GangaparapuA.A.LatifB.J.BeddingfieldA.RambautE.C.HolmesSpike+protein+mutations+in+novel+SARS-CoV-2+%E2%80%98variants+of+concern%E2%80%99+commonly+occur%C2%A0in+or+near+indels2021https%3A%2F%2Fvirological.org%2Ft%2Fspike-protein-mutations-in-novel-sars-cov-2-variants-of-concern-commonly-occur-in-or-near-indels%2F605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Garvin, M.R. ∙ T Prates, E. ∙ Pavicic, M. ...</div><div><strong>Potentially adaptive SARS-CoV-2 mutations discovered with novel spatiotemporal and explainable AI models</strong></div><div><em>Genome Biol.</em> 2020; <strong>21</strong>:304</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-020-02191-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85098009930" target="_blank" aria-label="Open Scopus in new tab">Scopus (47)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33357233/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-020-02191-0&amp;pmid=33357233" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Greaney, A.J. ∙ Loes, A.N. ∙ Crawford, K.H.D. ...</div><div><strong>Comprehensive mapping of mutations in the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human plasma antibodies</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:463-476</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2021.02.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046943" target="_blank" aria-label="Open Scopus in new tab">Scopus (679)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33592168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2021.02.003&amp;pmid=33592168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Greaney, A.J. ∙ Starr, T.N. ∙ Gilchuk, P. ...</div><div><strong>Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition</strong></div><div><em>Cell Host Microbe.</em> 2021; <strong>29</strong>:44-57</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.chom.2020.11.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85097068093" target="_blank" aria-label="Open Scopus in new tab">Scopus (644)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33259788/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.chom.2020.11.007&amp;pmid=33259788" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Guo, K. ∙ Barrett, B.S. ∙ Mickens, K.L. ...</div><div><strong>Interferon Resistance of Emerging SARS-CoV-2 Variants</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.03.20.436257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105395922" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.03.20.436257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Hoffmann, M. ∙ Arora, P. ∙ Groß, R. ...</div><div><strong>SARS-CoV-2 variants B.1.351 and P.1 escape from neutralizing antibodies</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2384-2393</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.036&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.036&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103496270" target="_blank" aria-label="Open Scopus in new tab">Scopus (651)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33794143/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.036&amp;pmid=33794143" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Horby, P. ∙ Huntley, C. ∙ Davies, N. ...</div><div><strong>Non-parametric analysis of fatal outcomes associated with&nbsp;B1.1.7</strong></div><div>Imperial College London, 2021</div><div><a id="intref0175" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/961037/NERVTAG_note_on_B.1.1.7_severity_for_SAGE_77__1_.pdf" target="_blank">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/961037/NERVTAG_note_on_B.1.1.7_severity_for_SAGE_77__1_.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.HorbyC.HuntleyN.DaviesJ.EdmundsN.FergusonG.MedleyC.SempleNon-parametric+analysis+of+fatal+outcomes+associated+with%C2%A0B1.1.72021Imperial+College+Londonhttps%3A%2F%2Fassets.publishing.service.gov.uk%2Fgovernment%2Fuploads%2Fsystem%2Fuploads%2Fattachment_data%2Ffile%2F961037%2FNERVTAG_note_on_B.1.1.7_severity_for_SAGE_77__1_.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Johnson, B.A. ∙ Xie, X. ∙ Bailey, A.L. ...</div><div><strong>Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis</strong></div><div><em>Nature.</em> 2021; <strong>591</strong>:293-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03237-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85099867614" target="_blank" aria-label="Open Scopus in new tab">Scopus (409)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33494095/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03237-4&amp;pmid=33494095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Katoh, K. ∙ Misawa, K. ∙ Kuma, K. ...</div><div><strong>MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</strong></div><div><em>Nucleic Acids Res.</em> 2002; <strong>30</strong>:3059-3066</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkf436" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-0037100671" target="_blank" aria-label="Open Scopus in new tab">Scopus (11062)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkf436&amp;pmid=12136088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">22.</a></div><div>Korber, B. ∙ Fischer, W.M. ∙ Gnanakaran, S. ..., Sheffield COVID-19 Genomics Group</div><div><strong>Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:812-827</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.06.043&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85087676352" target="_blank" aria-label="Open Scopus in new tab">Scopus (2714)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32697968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.06.043&amp;pmid=32697968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">23.</a></div><div>Kosakovsky Pond, S.L. ∙ Frost, S.D.W.</div><div><strong>Not so different after all: a comparison of methods for detecting amino acid sites under selection</strong></div><div><em>Mol. Biol. Evol.</em> 2005; <strong>22</strong>:1208-1222</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msi105" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-17744396121" target="_blank" aria-label="Open Scopus in new tab">Scopus (1654)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15703242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsi105&amp;pmid=15703242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">24.</a></div><div>Kosakovsky Pond, S.L. ∙ Weaver, S. ∙ Leigh Brown, A.J. ...</div><div><strong>HIV-TRACE (TRAnsmission Cluster Engine): a Tool for Large Scale Molecular Epidemiology of HIV-1 and Other Rapidly Evolving Pathogens</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:1812-1819</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msy016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85051869178" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29401317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsy016&amp;pmid=29401317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">25.</a></div><div>Kosakovsky Pond, S.L. ∙ Poon, A.F.Y. ∙ Velazquez, R. ...</div><div><strong>HyPhy 2.5-A Customizable Platform for Evolutionary Hypothesis Testing Using Phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2020; <strong>37</strong>:295-299</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msz197" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85078566726" target="_blank" aria-label="Open Scopus in new tab">Scopus (234)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31504749/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsz197&amp;pmid=31504749" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">26.</a></div><div>Kozlov, A.M. ∙ Darriba, D. ∙ Flouri, T. ...</div><div><strong>RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:4453-4455</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btz305" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85074309434" target="_blank" aria-label="Open Scopus in new tab">Scopus (1861)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31070718/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtz305&amp;pmid=31070718" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Lau, S.-Y. ∙ Wang, P. ∙ Mok, B.W.-Y. ...</div><div><strong>Attenuated SARS-CoV-2 variants with deletions at the S1/S2 junction</strong></div><div><em>Emerg. Microbes Infect.</em> 2020; <strong>9</strong>:837-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/22221751.2020.1756700" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85083865808" target="_blank" aria-label="Open Scopus in new tab">Scopus (205)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32301390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F22221751.2020.1756700&amp;pmid=32301390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Lei, X. ∙ Dong, X. ∙ Ma, R. ...</div><div><strong>Activation and evasion of type I interferon responses by SARS-CoV-2</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:3810</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-17665-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85088826828" target="_blank" aria-label="Open Scopus in new tab">Scopus (675)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32733001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-17665-9&amp;pmid=32733001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Lubinski, B. ∙ Tang, T. ∙ Daniel, S. ...</div><div><strong>Functional evaluation of proteolytic activation for the SARS-CoV-2 variant B.1.1.7: role of the P681H mutation</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.04.06.438731" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85121939608" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33851153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.04.06.438731&amp;pmid=33851153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">30.</a></div><div>Lucaci, A.G. ∙ Zehr, J.D. ∙ Shank, S.D. ...</div><div><strong>RASCL: Rapid assessment of SARS-COV-2 clades enabled through molecular sequence analysis and its application to B.1.617.1 and B.1.617.2.</strong></div><div>2021</div><div><a id="intref0180" href="https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709" target="_blank">https://virological.org/t/rascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2/709</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.G.LucaciJ.D.ZehrS.D.ShankD.BouvierH.MeiA.NekrutenkoS.L.Kosakovsky+PondRASCL%3A+Rapid+assessment+of+SARS-COV-2+clades+enabled+through+molecular+sequence+analysis+and+its+application+to+B.1.617.1+and+B.1.617.2.2021https%3A%2F%2Fvirological.org%2Ft%2Frascl-rapid-assessment-of-sars-cov-2-clades-enabled-through-molecular-sequence-analysis-and-its-application-to-b-1-617-1-and-b-1-617-2%2F709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31-1" title="View in article">31.</a></div><div>MacLean, O.A. ∙ Lytras, S. ∙ Weaver, S. ...</div><div><strong>Natural selection in the evolution of SARS-CoV-2 in bats created a generalist virus and highly capable human pathogen</strong></div><div><em>PLoS Biol.</em> 2021; <strong>19</strong>:e3001115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.3001115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102654960" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33711012/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.3001115&amp;pmid=33711012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>McCallum, M. ∙ De Marco, A. ∙ Lempp, F.A. ...</div><div><strong>N-terminal domain antigenic mapping reveals a site of vulnerability for SARS-CoV-2</strong></div><div><em>Cell.</em> 2021; <strong>184</strong>:2332-2347</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2021.03.028&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103091737" target="_blank" aria-label="Open Scopus in new tab">Scopus (530)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33761326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2021.03.028&amp;pmid=33761326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Miorin, L. ∙ Kehrer, T. ∙ Sanchez-Aparicio, M.T. ...</div><div><strong>SARS-CoV-2 Orf6 hijacks Nup98 to block STAT nuclear import and antagonize interferon signaling</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2020; <strong>117</strong>:28344-28354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2016650117" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85096080108" target="_blank" aria-label="Open Scopus in new tab">Scopus (328)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33097660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2016650117&amp;pmid=33097660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Murrell, B. ∙ Wertheim, J.O. ∙ Moola, S. ...</div><div><strong>Detecting individual sites subject to episodic diversifying selection</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1002764</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1002764" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84864605220" target="_blank" aria-label="Open Scopus in new tab">Scopus (1148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22807683/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1002764&amp;pmid=22807683" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Murrell, B. ∙ Moola, S. ∙ Mabona, A. ...</div><div><strong>FUBAR: a fast, unconstrained bayesian approximation for inferring selection</strong></div><div><em>Mol. Biol. Evol.</em> 2013; <strong>30</strong>:1196-1205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/mst030" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84876522258" target="_blank" aria-label="Open Scopus in new tab">Scopus (844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23420840/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmst030&amp;pmid=23420840" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Murrell, B. ∙ Weaver, S. ∙ Smith, M.D. ...</div><div><strong>Gene-wide identification of episodic selection</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:1365-1371</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msv035" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84929648750" target="_blank" aria-label="Open Scopus in new tab">Scopus (323)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25701167/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsv035&amp;pmid=25701167" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">37.</a></div><div>Nelson, G. ∙ Buzko, O. ∙ Spilman, P. ...</div><div><strong>Molecular dynamic simulation reveals E484K mutation enhances spike RBD-ACE2 affinity and the combination of E484K, K417N and N501Y mutations (501Y.V2 variant) induces conformational change greater than N501Y mutant alone, potentially resulting in an escape mutant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.13.426558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100737104" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34790982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.13.426558&amp;pmid=34790982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">38.</a></div><div>Nguyen, T.T. ∙ Pathirana, P.N. ∙ Nguyen, T. ...</div><div><strong>Genomic mutations and changes in protein secondary structure and solvent accessibility of SARS-CoV-2 (COVID-19 virus)</strong></div><div><em>Sci. Rep.</em> 2021; <strong>11</strong>:3487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-021-83105-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101468226" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33568759/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-021-83105-3&amp;pmid=33568759" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Nickle, D.C. ∙ Heath, L. ∙ Jensen, M.A. ...</div><div><strong>HIV-specific probabilistic models of protein evolution</strong></div><div><em>PLoS ONE.</em> 2007; <strong>2</strong>:e503</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0000503" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-39849090643" target="_blank" aria-label="Open Scopus in new tab">Scopus (88)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17551583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0000503&amp;pmid=17551583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">40.</a></div><div>Peacock, T.P. ∙ Penrice-Randal, R. ∙ Hiscox, J.A. ...</div><div><strong>SARS-CoV-2 one year on: evidence for ongoing viral adaptation</strong></div><div><em>J.&nbsp;Gen. Virol.</em> 2021; <strong>102</strong>:001584</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1099/jgv.0.001584" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85104384538" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33855951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1099%2Fjgv.0.001584&amp;pmid=33855951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Pearson, C.A. ∙ Russell, T.W. ∙ Davies, N. ...</div><div><strong>Estimates of severity and transmissibility of novel SARS-CoV-2 variant 501Y.V2 in South Africa | CMMID Repository</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.12.24.20248822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85105024215" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.12.24.20248822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42-1" title="View in article">42.</a></div><div>Plante, J.A. ∙ Liu, Y. ∙ Liu, J. ...</div><div><strong>Spike mutation D614G alters SARS-CoV-2 fitness</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:116-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2895-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85093821865" target="_blank" aria-label="Open Scopus in new tab">Scopus (992)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33106671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2895-3&amp;pmid=33106671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Pond, S.L.K. ∙ Frost, S.D.W. ∙ Grossman, Z. ...</div><div><strong>Adaptation to different human populations by HIV-1 revealed by codon-based analyses</strong></div><div><em>PLoS Comput. Biol.</em> 2006; <strong>2</strong>:e62</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0020062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-33745611774" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16789820/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0020062&amp;pmid=16789820" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Poon, A.F.Y. ∙ Lewis, F.I. ∙ Pond, S.L.K. ...</div><div><strong>An evolutionary-network model reveals stratified interactions in the V3 loop of the HIV-1 envelope</strong></div><div><em>PLoS Comput. Biol.</em> 2007; <strong>3</strong>:e231</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.0030231" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-36948998606" target="_blank" aria-label="Open Scopus in new tab">Scopus (87)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18039027/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.0030231&amp;pmid=18039027" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Public Health England</div><div><strong>Investigation of novel SARS-CoV-2 variant. Variant of Concern</strong></div><div>Public Health England, 2020</div><div><a id="interref0015" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959438/Technical_Briefing_VOC_SH_NJL2_SH2.pdf" target="_blank">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959438/Technical_Briefing_VOC_SH_NJL2_SH2.pdf</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Public+Health+EnglandInvestigation+of+novel+SARS-CoV-2+variant.+Variant+of+Concern2020Public+Health+Englandhttps%3A%2F%2Fassets.publishing.service.gov.uk%2Fgovernment%2Fuploads%2Fsystem%2Fuploads%2Fattachment_data%2Ffile%2F959438%2FTechnical_Briefing_VOC_SH_NJL2_SH2.pdf" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Pybus, O.G. ∙ Rambaut, A. ∙ Belshaw, R. ...</div><div><strong>Phylogenetic evidence for deleterious mutation load in RNA viruses and its contribution to viral evolution</strong></div><div><em>Mol. Biol. Evol.</em> 2007; <strong>24</strong>:845-852</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msm001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-33847649474" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17218639/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsm001&amp;pmid=17218639" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Rambaut, A. ∙ Loman, N. ∙ Pybus, O. ...</div><div><strong>Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations</strong></div><div>Virological, 2020</div><div><a id="interref0010" href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563" target="_blank">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.RambautN.LomanO.PybusW.BarclayJ.BarrettA.CarabelliT.ConnorT.PeacockD.L.RobertsonE.VolzPreliminary+genomic+characterisation+of+an+emergent+SARS-CoV-2+lineage+in+the+UK+defined+by+a+novel+set+of+spike+mutations2020Virologicalhttps%3A%2F%2Fvirological.org%2Ft%2Fpreliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations%2F563" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Rambaut, A. ∙ Holmes, E.C. ∙ O’Toole, Á. ...</div><div><strong>A dynamic nomenclature proposal for SARS-CoV-2 lineages to assist genomic epidemiology</strong></div><div><em>Nat. Microbiol.</em> 2020; <strong>5</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41564-020-0770-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85087930366" target="_blank" aria-label="Open Scopus in new tab">Scopus (1575)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32669681/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41564-020-0770-5&amp;pmid=32669681" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Rhee, S.-Y. ∙ Magalis, B.R. ∙ Hurley, L. ...</div><div><strong>National and International Dimensions of Human Immunodeficiency Virus-1 Sequence Clusters in a Northern California Clinical Cohort</strong></div><div><em>Open Forum Infect. Dis.</em> 2019; <strong>6</strong>:ofz135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/ofid/ofz135" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85066411962" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31041344/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fofid%2Fofz135&amp;pmid=31041344" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">50.</a></div><div>Scheffler, K. ∙ Murrell, B. ∙ Kosakovsky Pond, S.L.</div><div><strong>On the validity of evolutionary models with site-specific parameters</strong></div><div><em>PLoS ONE.</em> 2014; <strong>9</strong>:e94534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0094534" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-84899509429" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24722425/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0094534&amp;pmid=24722425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Shinde, V. ∙ Bhikha, S. ∙ Hoosain, Z. ..., 2019nCoV-501 Study Group</div><div><strong>Efficacy of NVX-CoV2373 Covid-19 Vaccine against the B.1.351 Variant</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1899-1909</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa2103055" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85106456686" target="_blank" aria-label="Open Scopus in new tab">Scopus (419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33951374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa2103055&amp;pmid=33951374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52-1" title="View in article">52.</a></div><div>Simonsen, M. ∙ Mailund, T. ∙ Pedersen, C.N.S.</div><div><strong>Rapid neighbour-joining</strong></div><div>Springer Berlin Heidelberg,<div>Crandall, K.A. ∙ Lagergren, J. (Editors)</div>in: Algorithms in Bioinformatics. 2008; 113-122</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.SimonsenT.MailundC.N.S.PedersenRapid+neighbour-joiningK.A.CrandallJ.LagergrenAlgorithms+in+Bioinformatics2008Springer+Berlin+Heidelberg113122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Hilton, S.K. ...</div><div><strong>Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</strong></div><div><em>Cell.</em> 2020; <strong>182</strong>:1295-1310.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.cell.2020.08.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85089695617" target="_blank" aria-label="Open Scopus in new tab">Scopus (1188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32841599/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.08.012&amp;pmid=32841599" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Starr, T.N. ∙ Greaney, A.J. ∙ Addetia, A. ...</div><div><strong>Prospective mapping of viral mutations that escape antibodies used to treat COVID-19</strong></div><div><em>Science.</em> 2021; <strong>371</strong>:850-854</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abf9302" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85101046439" target="_blank" aria-label="Open Scopus in new tab">Scopus (497)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33495308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abf9302&amp;pmid=33495308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Tablizo, F.A. ∙ Kim, K.M. ∙ Lapid, C.M. ...</div><div><strong>Genome sequencing and analysis of an emergent SARS-CoV-2 variant characterized by multiple spike protein mutations detected from the Central Visayas Region of the Philippines</strong></div><div><em>MedRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.03.03.21252812" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85104474905" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.03.03.21252812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Tegally, H. ∙ Wilkinson, E. ∙ Giovanetti, M. ...</div><div><strong>Detection of a SARS-CoV-2 variant of concern in South Africa</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:438-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03402-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102321294" target="_blank" aria-label="Open Scopus in new tab">Scopus (1057)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33690265/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03402-9&amp;pmid=33690265" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">57.</a></div><div>Thorne, L.G. ∙ Bouhaddou, M. ∙ Reuschl, A.-K. ...</div><div><strong>Evolution of enhanced innate immune evasion by the SARS-CoV-2 B.1.1.7&nbsp;UK variant</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.06.06.446826" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85110367314" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.06.06.446826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">58.</a></div><div>Volz, E. ∙ Mishra, S. ∙ Chand, M. ..., COVID-19 Genomics UK (COG-UK) consortium</div><div><strong>Assessing transmissibility of SARS-CoV-2 lineage B.1.1.7 in England</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:266-269</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03470-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103190451" target="_blank" aria-label="Open Scopus in new tab">Scopus (704)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33767447/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03470-x&amp;pmid=33767447" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Wang, P. ∙ Nair, M.S. ∙ Liu, L. ...</div><div><strong>Antibody resistance of SARS-CoV-2 variants B.1.351 and B.1.1.7</strong></div><div><em>Nature.</em> 2021; <strong>593</strong>:130-135</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03398-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102173598" target="_blank" aria-label="Open Scopus in new tab">Scopus (1469)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33684923/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03398-2&amp;pmid=33684923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>Wang, Z. ∙ Schmidt, F. ∙ Weisblum, Y. ...</div><div><strong>mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants</strong></div><div><em>Nature.</em> 2021; <strong>592</strong>:616-622</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03324-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100998586" target="_blank" aria-label="Open Scopus in new tab">Scopus (911)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33567448/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03324-6&amp;pmid=33567448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">61.</a></div><div>Wibmer, C.K. ∙ Ayres, F. ∙ Hermanus, T. ...</div><div><strong>SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma</strong></div><div><em>Nat. Med.</em> 2021; <strong>27</strong>:622-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41591-021-01285-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85102044511" target="_blank" aria-label="Open Scopus in new tab">Scopus (709)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33654292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41591-021-01285-x&amp;pmid=33654292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Wu, K. ∙ Werner, A.P. ∙ Koch, M. ...</div><div><strong>Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2021; <strong>384</strong>:1468-1470</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMc2102179" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85103371216" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33730471/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMc2102179&amp;pmid=33730471" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Xia, H. ∙ Cao, Z. ∙ Xie, X. ...</div><div><strong>Evasion of Type I Interferon by SARS-CoV-2</strong></div><div><em>Cell Rep.</em> 2020; <strong>33</strong>:108234</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.celrep.2020.108234&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=10.1016%2Fj.celrep.2020.108234&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85091687040" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32979938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2020.108234&amp;pmid=32979938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">64.</a></div><div>Yuan, M. ∙ Huang, D. ∙ Lee, C.D. ...</div><div><strong>Structural and functional ramifications of antigenic drift in recent SARS-CoV-2 variants</strong></div><div><em>Science.</em> 2021; <strong>373</strong>:818-823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abh1139" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85112522453" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34016740/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abh1139&amp;pmid=34016740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">65.</a></div><div>Zahradnik, J. ∙ Marciano, S. ∙ Shemesh, M. ...</div><div><strong>SARS-CoV-2 RBD in&nbsp;vitro evolution follows contagious mutation spread, yet generates an able infection inhibitor</strong></div><div><em>BioRxiv.</em> 2021;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2021.01.06.425392" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85100750192" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34981049/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2021.01.06.425392&amp;pmid=34981049" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">66.</a></div><div>Zhang, L. ∙ Jackson, C.B. ∙ Mou, H. ...</div><div><strong>SARS-CoV-2 spike-protein D614G mutation increases virion spike density and infectivity</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:6013</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-19808-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.09.003&amp;key=2-s2.0-85096795416" target="_blank" aria-label="Open Scopus in new tab">Scopus (613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33243994/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-19808-4&amp;pmid=33243994" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (7)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure 1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/84c1be4e-16d1-4e74-97f4-c5b86aba7688/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure 2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/8d3803a1-2c32-47b1-9172-94988bb29b42/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/9d01e26e-d77e-4038-9cfc-d2593f50652a/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure 3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/bded973d-9b4e-4f44-91dc-7dff4ac6f93e/main.assets/gr3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/cccaaf2f-8948-4720-83cc-46de9c476cf9/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure 4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/f9f6027d-ecfa-4105-b9e1-0d488e25cfb6/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure 5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/asset/49f42151-9b71-48ce-99f0-8b00f1824501/main.assets/gr5.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867421010503%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00213%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (7)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="archive"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/51242a28-447e-4cf9-b1ed-dbec19a52995/mmc1.zip" download="mmc1.zip"><span id="mmc1-link">Archive (2.93 MB)</span></a><div class="core-description">Table S1. Sites evolving under positive selection in 501Y lineage-specific datasets, related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/c12e46b2-e6cd-48db-9a3e-e16a4d0c5d21/mmc2.pdf" download="mmc2.pdf"><span id="mmc2-link">PDF (31.17 KB)</span></a><div class="core-description">Table S2. Changes in mutation frequencies at signature and/or convergent mutation sites within 501Y lineage viruses between March 15, and June 1, 2021 that are detectably evolving under positive selection in individual lineages and/or in the global SARS-CoV-2 dataset, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/5a84e664-92f6-483e-8c43-ca282c5311d9/mmc3.pdf" download="mmc3.pdf"><span id="mmc3-link">PDF (25.24 KB)</span></a><div class="core-description">Table S3. Mutations comprising the 501Y lineage meta-signature, related to Figures 4 and 5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/8356713d-7abb-476d-a547-799b9a40c18c/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (13.08 KB)</span></a><div class="core-description">Table S4. Sequences within different SARS-COV-2 lineages of concern, lineages of interest, and lineages under investigation that have most converged on the 501Y lineage meta-signature, related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/06ad78f2-32c6-44ff-b221-0dcfaafcd444/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (11.72 MB)</span></a><div class="core-description">Table S5. GISAID acknowledgments and SARS-CoV-2 sequence accession numbers, related to the Acknowledgments</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="document"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/e368c2e3-a74e-4f5b-b407-a7a315fc2562/mmc6.docx" download="mmc6.docx"><span id="mmc6-link">Document (19.55 KB)</span></a><div class="core-description">Appendix 1. NGS-SA authors</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.09.003/attachment/146d6194-686d-4739-8104-2a94101f4988/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (36.21 KB)</span></a><div class="core-description">Appendix 2. COG-UK authors</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421010503" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(22)03938-X" target="_blank" title="The SARS-CoV-2 Delta (B.1.617.2) variant with spike N501Y mutation in the shadow of Omicron emergence">The SARS-CoV-2 Delta (B.1.617.2) variant with spike N501Y mutation in the shadow of Omicron emergence</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Salimović-Bešić et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">December 24, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(24)00621-9" target="_blank" title="Ongoing evolution of SARS-CoV-2 drives escape from mRNA vaccine-induced humoral immunity">Ongoing evolution of SARS-CoV-2 drives escape from mRNA vaccine-induced humoral immunity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Roederer et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">December 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(23)00915-7" target="_blank" title="SARS-CoV-2 variants evolve convergent strategies to remodel the host response">SARS-CoV-2 variants evolve convergent strategies to remodel the host response</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bouhaddou et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">September 21, 2023</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/microbiology/fulltext/S0966-842X(22)00291-8" target="_blank" title="On the evolution of SARS-CoV-2 and the emergence of variants of concern">On the evolution of SARS-CoV-2 and the emergence of variants of concern</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gkikas Magiorkinis</li></ul></div><div class="linked-article__details"><span><i>Trends in Microbiology</i></span><span id="item_date">October 23, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01466-9" target="_blank" title="Pressure to evade cell-autonomous innate sensing reveals interplay between mitophagy, IFN signaling, and SARS-CoV-2 evolution">Pressure to evade cell-autonomous innate sensing reveals interplay between mitophagy, IFN signaling, and SARS-CoV-2 evolution</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lee et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">December 20, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(23)00035-6" target="_blank" title="Accelerated SARS-CoV-2 intrahost evolution leading to distinct genotypes during chronic infection">Accelerated SARS-CoV-2 intrahost evolution leading to distinct genotypes during chronic infection</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chaguza et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">January 26, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/immunology/fulltext/S1471-4906(25)00076-6" target="_blank" title="Deciphering SARS-CoV-2 evolution under antibody immune pressure">Deciphering SARS-CoV-2 evolution under antibody immune pressure</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jian et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Immunology</i></span><span id="item_date">March 27, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/plant-communications/fulltext/S2590-3462(25)00020-3" target="_blank" title="Convergent evolution in angiosperms adapted to cold climates">Convergent evolution in angiosperms adapted to cold climates</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Plant Communications</i></span><span id="item_date">January 22, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-host-microbe/fulltext/S1931-3128(22)00315-8" target="_blank" title="Emergence and widespread circulation of a recombinant SARS-CoV-2 lineage in North America">Emergence and widespread circulation of a recombinant SARS-CoV-2 lineage in North America</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Gutierrez et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Host &amp; Microbe</i></span><span id="item_date">June 19, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/immunity/fulltext/S1074-7613(25)00133-5" target="_blank" title="Immunological drivers of zoonotic virus emergence, evolution, and endemicity">Immunological drivers of zoonotic virus emergence, evolution, and endemicity</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Purushotham et al.</li></ul></div><div class="linked-article__details"><span><i>Immunity</i></span><span id="item_date">March 31, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)11363-1" target="_blank" title="Wastewater multiplex PCR amplicon sequencing revealed community transmission of SARS-CoV-2 lineages during the outbreak of infection in Chinese Mainland">Wastewater multiplex PCR amplicon sequencing revealed community transmission of SARS-CoV-2 lineages during the outbreak of infection in Chinese Mainland</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(25)00914-4" target="_blank" title="Wastewater surveillance of open drains for mapping the trajectory and succession of SARS-CoV-2 lineages in 23 cities of Maharashtra state (India) during June 2022 to May 2023">Wastewater surveillance of open drains for mapping the trajectory and succession of SARS-CoV-2 lineages in 23 cities of Maharashtra state (India) during June 2022 to May 2023</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Matra et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">February 6, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-plant/fulltext/S1674-2052(24)00394-0" target="_blank" title="Genomic, transcriptomic, and metabolomic analyses reveal convergent evolution of oxime biosynthesis in Darwin’s orchid">Genomic, transcriptomic, and metabolomic analyses reveal convergent evolution of oxime biosynthesis in Darwin’s orchid</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Jiang et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Plant</i></span><span id="item_date">December 18, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)09644-5" target="_blank" title="The emergence and successful elimination of SARS-CoV-2 dominant strains with increasing epidemic potential in Taiwan’s 2021 outbreak">The emergence and successful elimination of SARS-CoV-2 dominant strains with increasing epidemic potential in Taiwan’s 2021 outbreak</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 20, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(23)01456-0" target="_blank" title="Evolution of the SARS-CoV-2 Omicron spike">Evolution of the SARS-CoV-2 Omicron spike</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Parsons et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">November 17, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)08170-2" target="_blank" title="APOBEC3-related mutations in the spike protein-encoding region facilitate SARS-CoV-2 evolution">APOBEC3-related mutations in the spike protein-encoding region facilitate SARS-CoV-2 evolution</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Shen et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)06188-1" target="_blank" title="SARS-CoV-2 genomic characterization and evolution in China">SARS-CoV-2 genomic characterization and evolution in China</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 12, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)01954-0" target="_blank" title="Drug susceptibility and the potential for drug-resistant SARS-CoV-2 emergence in immunocompromised animals">Drug susceptibility and the potential for drug-resistant SARS-CoV-2 emergence in immunocompromised animals</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kiso et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">August 16, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)10396-9" target="_blank" title="Genomic analysis of three SARS-CoV-2 waves in west Sumatra: Evolutionary dynamics and variant clustering">Genomic analysis of three SARS-CoV-2 waves in west Sumatra: Evolutionary dynamics and variant clustering</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Linosefa et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">July 9, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/structure/fulltext/S0969-2126(24)00323-X" target="_blank" title="Structural insights into the convergent evolution of sulfoxide synthase EgtB-IV, an ergothioneine-biosynthetic homolog of ovothiol synthase OvoA">Structural insights into the convergent evolution of sulfoxide synthase EgtB-IV, an ergothioneine-biosynthetic homolog of ovothiol synthase OvoA</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ireland et al.</li></ul></div><div class="linked-article__details"><span><i>Structure</i></span><span id="item_date">August 30, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00353-8" target="_blank" title="The recency and geographical origins of the bat viruses ancestral to SARS-CoV and SARS-CoV-2">The recency and geographical origins of the bat viruses ancestral to SARS-CoV and SARS-CoV-2</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pekar et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">May 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(24)01475-2" target="_blank" title="Molecular basis of convergent evolution of ACE2 receptor utilization among HKU5 coronaviruses">Molecular basis of convergent evolution of ACE2 receptor utilization among HKU5 coronaviruses</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Park et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">February 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)09564-6" target="_blank" title="Wastewater surveillance in the COVID-19 post-emergency pandemic period: A promising approach to monitor and predict SARS-CoV-2 surges and evolution">Wastewater surveillance in the COVID-19 post-emergency pandemic period: A promising approach to monitor and predict SARS-CoV-2 surges and evolution</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Sanchez Jimenez et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 16, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-cell/fulltext/S1097-2765(24)00535-5" target="_blank" title="SARS-CoV-2 Omicron XBB lineage spike structures, conformations, antigenicity, and receptor recognition">SARS-CoV-2 Omicron XBB lineage spike structures, conformations, antigenicity, and receptor recognition</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Cell</i></span><span id="item_date">July 25, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/immunology/fulltext/S1471-4906(24)00119-4" target="_blank" title="SARS-CoV-2 humoral immunity in people living with HIV-1">SARS-CoV-2 humoral immunity in people living with HIV-1</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Motsoeneng et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Immunology</i></span><span id="item_date">June 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(23)01076-0" target="_blank" title="Population immunity predicts evolutionary trajectories of SARS-CoV-2">Population immunity predicts evolutionary trajectories of SARS-CoV-2</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Meijers et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">October 23, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00712-6" target="_blank" title="Determination of resilience of a panel of broadly neutralizing mAbs to emerging variants of SARS-CoV-2 generated using reverse genetics">Determination of resilience of a panel of broadly neutralizing mAbs to emerging variants of SARS-CoV-2 generated using reverse genetics</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Afzal et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">April 16, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(21)00889-8" target="_blank" title="Emergence of an early SARS-CoV-2 epidemic in the United States">Emergence of an early SARS-CoV-2 epidemic in the United States</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zeller et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">July 26, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)00068-8" target="_blank" title="Evolutionary analysis identifies a Golgi pathway and correlates lineage-specific factors with endomembrane organelle emergence in apicomplexans">Evolutionary analysis identifies a Golgi pathway and correlates lineage-specific factors with endomembrane organelle emergence in apicomplexans</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Klinger et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 15, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/molecular-therapy-family/methods/fulltext/S2329-0501(25)00054-3" target="_blank" title="Engineered ACE2 decoy in dry powder form for inhalation: A novel therapy for SARS-CoV-2 variants">Engineered ACE2 decoy in dry powder form for inhalation: A novel therapy for SARS-CoV-2 variants</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ito et al.</li></ul></div><div class="linked-article__details"><span><i>Molecular Therapy Methods &amp; Clinical Development</i></span><span id="item_date">March 31, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)00014-9" target="_blank" title="Convergent evolution of water-conducting cells in Marchantia recruited the ZHOUPI gene promoting cell wall reinforcement and programmed cell death">Convergent evolution of water-conducting cells in Marchantia recruited the ZHOUPI gene promoting cell wall reinforcement and programmed cell death</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lu et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">January 30, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(21)01298-0" target="_blank" title="Endonuclease-based genotyping of the RBM as a method to track the emergence or evolution of SARS-CoV-2 variants">Endonuclease-based genotyping of the RBM as a method to track the emergence or evolution of SARS-CoV-2 variants</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Lopez et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">October 20, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)00537-2" target="_blank" title="Convergence and molecular evolution of floral fragrance after independent transitions to self-fertilization">Convergence and molecular evolution of floral fragrance after independent transitions to self-fertilization</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Woźniak et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">May 21, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(22)00849-2" target="_blank" title="Emergence of immune escape at dominant SARS-CoV-2 killer T&nbsp;cell epitope">Emergence of immune escape at dominant SARS-CoV-2 killer T&nbsp;cell epitope</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Dolton et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">July 13, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(24)01491-8" target="_blank" title="Nonhuman primate antigenic cartography of SARS-CoV-2">Nonhuman primate antigenic cartography of SARS-CoV-2</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Rössler et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">January 3, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00274-8" target="_blank" title="Insights into convergent evolution of cosexuality in liverworts from the Marchantia quadrata genome">Insights into convergent evolution of cosexuality in liverworts from the Marchantia quadrata genome</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Potente et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">April 2, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)00758-2" target="_blank" title="Long COVID across SARS-CoV-2 variants, lineages, and sublineages">Long COVID across SARS-CoV-2 variants, lineages, and sublineages</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Padilla et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">March 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(25)00133-6" target="_blank" title="Mutational pressure promotes release of public CD8+ T&nbsp;cell epitopes by proteasome from SARS-CoV-2 RBD of Omicron and its current lineages">Mutational pressure promotes release of public CD8+ T&nbsp;cell epitopes by proteasome from SARS-CoV-2 RBD of Omicron and its current lineages</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Kudriaeva et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 22, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)09260-0" target="_blank" title="Assessing the performance of LumiraDx™ SARS-CoV-2 Ag test in detecting Omicron lineages: 2022–2023 study">Assessing the performance of LumiraDx™ SARS-CoV-2 Ag test in detecting Omicron lineages: 2022–2023 study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Cocchio et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)02331-3" target="_blank" title="Evolving antibody evasion and receptor affinity of the Omicron BA.2.75 sublineage of SARS-CoV-2">Evolving antibody evasion and receptor affinity of the Omicron BA.2.75 sublineage of SARS-CoV-2</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">October 18, 2023</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.09.003","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">The emergence and ongoing convergent evolution of the SARS-CoV-2 N501Y lineages</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.09.003/asset/84c1be4e-16d1-4e74-97f4-c5b86aba7688/main.assets/gr1.jpg" height="352" width="506" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/9106dcb3-8472-4943-8a2b-1992f14d2fef/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.09.003/asset/8d3803a1-2c32-47b1-9172-94988bb29b42/main.assets/gr2.jpg" height="404" width="504" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/d9bf1a74-cc6d-4e6e-8772-bd72d7ec678e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.09.003/asset/9d01e26e-d77e-4038-9cfc-d2593f50652a/main.assets/figs1.jpg" height="862" width="504" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/6b174e55-79e0-4395-9fef-fe00a47d7553/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.09.003/asset/bded973d-9b4e-4f44-91dc-7dff4ac6f93e/main.assets/gr3.jpg" height="760" width="373" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/35b0da7d-9394-4342-b5b4-eae613ca086f/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.09.003/asset/cccaaf2f-8948-4720-83cc-46de9c476cf9/main.assets/figs2.jpg" height="823" width="373" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/0fa1abe5-8ffb-4277-8306-96812eebb712/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.09.003/asset/f9f6027d-ecfa-4105-b9e1-0d488e25cfb6/main.assets/gr4.jpg" height="511" width="654" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/550316fb-7cb5-4496-9281-c1a582adfe45/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.09.003/asset/49f42151-9b71-48ce-99f0-8b00f1824501/main.assets/gr5.jpg" height="356" width="769" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/7d5ca85f-575e-40f1-99e0-f44f4ca33a8b/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure 1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.09.003/asset/84c1be4e-16d1-4e74-97f4-c5b86aba7688/main.assets/gr1.jpg" height="352" width="506" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/9106dcb3-8472-4943-8a2b-1992f14d2fef/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.09.003/asset/8d3803a1-2c32-47b1-9172-94988bb29b42/main.assets/gr2.jpg" height="404" width="504" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/d9bf1a74-cc6d-4e6e-8772-bd72d7ec678e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.09.003/asset/9d01e26e-d77e-4038-9cfc-d2593f50652a/main.assets/figs1.jpg" height="862" width="504" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/6b174e55-79e0-4395-9fef-fe00a47d7553/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.09.003/asset/bded973d-9b4e-4f44-91dc-7dff4ac6f93e/main.assets/gr3.jpg" height="760" width="373" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/35b0da7d-9394-4342-b5b4-eae613ca086f/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.09.003/asset/cccaaf2f-8948-4720-83cc-46de9c476cf9/main.assets/figs2.jpg" height="823" width="373" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/0fa1abe5-8ffb-4277-8306-96812eebb712/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.09.003/asset/f9f6027d-ecfa-4105-b9e1-0d488e25cfb6/main.assets/gr4.jpg" height="511" width="654" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/550316fb-7cb5-4496-9281-c1a582adfe45/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.09.003/asset/49f42151-9b71-48ce-99f0-8b00f1824501/main.assets/gr5.jpg" height="356" width="769" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.09.003/asset/7d5ca85f-575e-40f1-99e0-f44f4ca33a8b/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01050-3';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01050-3';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749591601222","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"20","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421010503","coverDate":"202109","type":"XOCS-JOURNAL:SCOPE-FULL","title":"The emergence and ongoing convergent evolution of the SARSCoV2 N501Y lineages","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc064d0d1752eb',t:'MTc0OTU5MTYwMC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc064d0d1752eb&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.09.003&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>