<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="adca6ead-4e29-4bce-88f3-cb5b623899dd" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;article:article:pii\:S0092867420302348;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text;issue:issue:pii\:S0092867419X00094">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)30234-8">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/b1fa9844-fe2d-4264-bb31-09e29eb9fc83/fx1.jpg">
    


        <meta property="og:description" content="Epigenetic profiling of chondrocytes reveals selection and constraint on knee regulatory
elements including those overlapping osteoarthritis risk variants, including a functionally
verified common enhancer variant sufficient to affect knee shape in mouse.">
    











        <meta name="twitter:title" content="Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/b1fa9844-fe2d-4264-bb31-09e29eb9fc83/fx1.jpg">
    


        <meta name="twitter:description" content="Epigenetic profiling of chondrocytes reveals selection and constraint on knee regulatory
elements including those overlapping osteoarthritis risk variants, including a functionally
verified common enhancer variant sufficient to affect knee shape in mouse.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)30234-8">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk"> <meta name="citation_date" content="2020/04/16"> <meta name="citation_online_date" content="2020/03/26"> <meta name="citation_volume" content="181"> <meta name="citation_issue" content="2"> <meta name="citation_firstpage" content="362"> <meta name="citation_lastpage" content="381.e28"> <meta name="citation_pmid" content="32220312"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420302348"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420302348"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420302348"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.02.057"> <meta name="citation_pmid" content="32220312"> <meta name="citation_author" content="Daniel Richard"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Zun Liu"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Jiaxue Cao"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author_institution" content="Farm Animal Genetic Resources Exploration and Innovation Key Laboratory of Sichuan Province, Sichuan Agricultural University, Chengdu 611130, China"><meta name="citation_author" content="Ata M. Kiapour"><meta name="citation_author_institution" content="Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Jessica Willen"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Siddharth Yarlagadda"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Evelyn Jagoda"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Vijaya B. Kolachalama"><meta name="citation_author_institution" content="Department of Medicine, Boston University School of Medicine, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Whitaker Cardiovascular Institute, Boston University School of Medicine, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Hariri Institute for Computing and Computational Science and Engineering, Boston University, Boston, MA 02115, USA"><meta name="citation_author" content="Jakob T. Sieker"><meta name="citation_author_institution" content="Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Pathology and Laboratory Medicine, Tufts Medical Center, Boston, MA 02111, USA"><meta name="citation_author" content="Gary H. Chang"><meta name="citation_author_institution" content="Department of Medicine, Boston University School of Medicine, Boston, MA 02115, USA"><meta name="citation_author" content="Pushpanathan Muthuirulan"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Mariel Young"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Anand Masson"><meta name="citation_author_institution" content="McCaig Institute for Bone and Joint Health, University of Calgary, Calgary, AB T2N 1N4, Canada"><meta name="citation_author" content="Johannes Konrad"><meta name="citation_author_institution" content="Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Shayan Hosseinzadeh"><meta name="citation_author_institution" content="Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="David E. Maridas"><meta name="citation_author_institution" content="Developmental Biology, Harvard School of Dental Medicine, Boston, MA 02115, USA"><meta name="citation_author" content="Vicki Rosen"><meta name="citation_author_institution" content="Developmental Biology, Harvard School of Dental Medicine, Boston, MA 02115, USA"><meta name="citation_author" content="Roman Krawetz"><meta name="citation_author_institution" content="McCaig Institute for Bone and Joint Health, University of Calgary, Calgary, AB T2N 1N4, Canada"><meta name="citation_author" content="Neil Roach"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author" content="Terence D. Capellini"><meta name="citation_author_email" content="tcapellini@fas.harvard.edu"><meta name="citation_author_orcid" content="0000-0003-3842-8478"><meta name="citation_author_institution" content="Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA"><meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA"><meta name="citation_keywords" content="osteoarthritis; knee; gene regulation; ATAC-seq; human evolution; natural selection; genetic drift; GDF5; mouse model; chondrocyte"><meta name="citation_reference" content="citation_title=Histochemical studies on the histogenesis of the knee joint and superior tibio-fibular joint in human foetuses; citation_author=H. Andersen; citation_journal_title=Acta Anat. (Basel); citation_volume=46; citation_firstpage=279; citation_lastpage=303; citation_publication_date=1961; "><meta name="citation_reference" content="citation_title=Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study; citation_author=E. Zeggini; citation_author=K. Panoutsopoulou; citation_author=L. Southam; citation_author=N.W. Rayner; citation_author=A.G. Day-Williams; citation_author=M.C. Lopes; citation_author=V. Boraska; citation_author=T. Esko; citation_author=arcOGEN Consortium; citation_author=arcOGEN Collaborators; citation_author=et al.; citation_journal_title=Lancet; citation_volume=380; citation_firstpage=815; citation_lastpage=823; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=A global reference for human genetic variation; citation_author=A. Auton; citation_author=L.D. Brooks; citation_author=R.M. Durbin; citation_author=E.P. Garrison; citation_author=H.M. Kang; citation_author=J.O. Korbel; citation_author=J.L. Marchini; citation_author=S. McCarthy; citation_author=G.A. McVean; citation_author=G.R. Abecasis; citation_author=1000 Genomes Project Consortium; citation_journal_title=Nature; citation_volume=526; citation_firstpage=68; citation_lastpage=74; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The relationship between three-dimensional knee MRI bone shape and total knee replacement-a case control study: data from the Osteoarthritis Initiative; citation_author=A.J. Barr; citation_author=B. Dube; citation_author=E.M.A. Hensor; citation_author=S.R. Kingsbury; citation_author=G. Peat; citation_author=M.A. Bowes; citation_author=L.D. Sharples; citation_author=P.G. Conaghan; citation_journal_title=Rheumatology (Oxford); citation_volume=55; citation_firstpage=1585; citation_lastpage=1593; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Haploview: analysis and visualization of LD and haplotype maps; citation_author=J.C. Barrett; citation_author=B. Fry; citation_author=J. Maller; citation_author=M.J. Daly; citation_journal_title=Bioinformatics; citation_volume=21; citation_firstpage=263; citation_lastpage=265; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=A novel insertion mutation in the cartilage-derived morphogenetic protein-1 (CDMP1) gene underlies Grebe-type chondrodysplasia in a consanguineous Pakistani family; citation_author=S. Basit; citation_author=S.K.-H. Naqvi; citation_author=N. Wasif; citation_author=G. Ali; citation_author=M. Ansar; citation_author=W. Ahmad; citation_journal_title=BMC Med. Genet.; citation_volume=9; citation_firstpage=102; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Regulation and Role of TGFβ Signaling Pathway in Aging and Osteoarthritis Joints; citation_author=C. Baugé; citation_author=N. Girard; citation_author=E. Lhuissier; citation_author=C. Bazille; citation_author=K. Boumediene; citation_journal_title=Aging Dis.; citation_volume=5; citation_firstpage=394; citation_lastpage=405; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; citation_author=Y. Benjamini; citation_author=Y. Hochberg; citation_journal_title=J.&nbsp;R. Stat. Soc. B; citation_volume=57; citation_firstpage=289; citation_lastpage=300; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Modern-day environmental factors in the pathogenesis of osteoarthritis; citation_author=F. Berenbaum; citation_author=I.J. Wallace; citation_author=D.E. Lieberman; citation_author=D.T. Felson; citation_journal_title=Nat. Rev. Rheumatol.; citation_volume=14; citation_firstpage=674; citation_lastpage=681; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast-evolving noncoding sequences in the human genome; citation_author=C.P. Bird; citation_author=B.E. Stranger; citation_author=M. Liu; citation_author=D.J. Thomas; citation_author=C.E. Ingle; citation_author=C. Beazley; citation_author=W. Miller; citation_author=M.E. Hurles; citation_author=E.T. Dermitzakis; citation_journal_title=Genome Biol.; citation_volume=8; citation_firstpage=R118; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Mouse Genome Database (MGD)-2017: community knowledge resource for the laboratory mouse; citation_author=J.A. Blake; citation_author=J.T. Eppig; citation_author=J.A. Kadin; citation_author=J.E. Richardson; citation_author=C.L. Smith; citation_author=C.J. Bult; citation_author=the Mouse Genome Database Group; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D723; citation_lastpage=D729; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Human-chimpanzee differences in a FZD8 enhancer alter cell-cycle dynamics in the developing neocortex; citation_author=J.L. Boyd; citation_author=S.L. Skove; citation_author=J.P. Rouanet; citation_author=L.-J. Pilaz; citation_author=T. Bepler; citation_author=R. Gordân; citation_author=G.A. Wray; citation_author=D.L. Silver; citation_journal_title=Curr. Biol.; citation_volume=25; citation_firstpage=772; citation_lastpage=779; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Endurance running and the evolution of Homo; citation_author=D.M. Bramble; citation_author=D.E. Lieberman; citation_journal_title=Nature; citation_volume=432; citation_firstpage=345; citation_lastpage=352; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=A One-Penny Imputed Genome from Next-Generation Reference Panels; citation_author=B.L. Browning; citation_author=Y. Zhou; citation_author=S.R. Browning; citation_journal_title=Am. J. Hum. Genet.; citation_volume=103; citation_firstpage=338; citation_lastpage=348; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position; citation_author=J.D. Buenrostro; citation_author=P.G. Giresi; citation_author=L.C. Zaba; citation_author=H.Y. Chang; citation_author=W.J. Greenleaf; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=1213; citation_lastpage=1218; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=ATAC-seq: a method for assaying chromatin accessibility genome-wide; citation_author=J.D. Buenrostro; citation_author=B. Wu; citation_author=H.Y. Chang; citation_author=W.J. Greenleaf; citation_journal_title=Curr. Protoc. Mol. Biol.; citation_volume=109; citation_firstpage=1; citation_lastpage=9; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A genome-wide screen for noncoding elements important in primate evolution; citation_author=E.C. Bush; citation_author=B.T. Lahn; citation_journal_title=BMC Evol. Biol.; citation_volume=8; citation_firstpage=17; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Accelerating functional gene discovery in osteoarthritis; citation_author=N.C. Butterfield; citation_author=K.F. Curry; citation_author=J. Steinberg; citation_author=H. Dewhurst; citation_author=D. Komla-Ebri; citation_author=N.S. Mannan; citation_author=A.-T. Adoum; citation_author=V.D. Leitch; citation_author=J.G. Logan; citation_author=J.A. Waung; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/836221; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Pbx1/Pbx2 requirement for distal limb patterning is mediated by the hierarchical control of Hox gene spatial distribution and Shh expression; citation_author=T.D. Capellini; citation_author=G. Di Giacomo; citation_author=V. Salsi; citation_author=A. Brendolan; citation_author=E. Ferretti; citation_author=D. Srivastava; citation_author=V. Zappavigna; citation_author=L. Selleri; citation_journal_title=Development; citation_volume=133; citation_firstpage=2263; citation_lastpage=2273; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk; citation_author=T.D. Capellini; citation_author=H. Chen; citation_author=J. Cao; citation_author=A.C. Doxey; citation_author=A.M. Kiapour; citation_author=M. Schoor; citation_author=D.M. Kingsley; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1202; citation_lastpage=1210; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Many human accelerated regions are developmental enhancers; citation_author=J.A. Capra; citation_author=G.D. Erwin; citation_author=G. McKinsey; citation_author=J.L.R. Rubenstein; citation_author=K.S. Pollard; citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_volume=368; citation_firstpage=20130025; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Evolutionary Perspectives on Human Senescence; citation_author=B.A. Carnes; citation_author=S.J. Olshansky; citation_journal_title=Popul. Dev. Rev.; citation_volume=19; citation_firstpage=793; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=A novel variant in GLIS3 is associated with osteoarthritis; citation_author=E. Casalone; citation_author=I. Tachmazidou; citation_author=E. Zengini; citation_author=K. Hatzikotoulas; citation_author=S. Hackinger; citation_author=D. Suveges; citation_author=J. Steinberg; citation_author=N.W. Rayner; citation_author=J.M. Wilkinson; citation_author=K. Panoutsopoulou; citation_author=E. Zeggini; citation_author=arcOGEN Consortium; citation_journal_title=Ann. Rheum. Dis.; citation_volume=77; citation_firstpage=620; citation_lastpage=623; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genome-wide association and functional studies identify the DOT1L gene to be involved in cartilage thickness and hip osteoarthritis; citation_author=M.C. Castaño-Betancourt; citation_author=F. Cailotto; citation_author=H.J. Kerkhof; citation_author=F.M. Cornelis; citation_author=S.A. Doherty; citation_author=D.J. Hart; citation_author=A. Hofman; citation_author=F.P. Luyten; citation_author=R.A. Maciewicz; citation_author=M. Mangino; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=8218; citation_lastpage=8223; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Novel genetic variants for cartilage thickness and hip osteoarthritis; citation_author=M.C. Castaño-Betancourt; citation_author=D.S. Evans; citation_author=Y.F. Ramos; citation_author=C.G. Boer; citation_author=S. Metrustry; citation_author=Y. Liu; citation_author=W. den Hollander; citation_author=J. van Rooij; citation_author=V.B. Kraus; citation_author=M.S. Yau; citation_author=et al.; citation_journal_title=PLoS Genet.; citation_volume=12; citation_firstpage=e1006260; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a Pitx1 enhancer; citation_author=Y.F. Chan; citation_author=M.E. Marks; citation_author=F.C. Jones; citation_author=G. Villarreal; citation_author=M.D. Shapiro; citation_author=S.D. Brady; citation_author=A.M. Southwick; citation_author=D.M. Absher; citation_author=J. Grimwood; citation_author=J. Schmutz; citation_author=et al.; citation_journal_title=Science; citation_volume=327; citation_firstpage=302; citation_lastpage=305; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets; citation_author=C.C. Chang; citation_author=C.C. Chow; citation_author=L.C. Tellier; citation_author=S. Vattikuti; citation_author=S.M. Purcell; citation_author=J.J. Lee; citation_journal_title=Gigascience; citation_volume=4; citation_firstpage=7; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Magnetic resonance imaging-based topographical differences between control and recurrent patellofemoral instability patients; citation_author=M.D. Charles; citation_author=S. Haloman; citation_author=L. Chen; citation_author=S.R. Ward; citation_author=D. Fithian; citation_author=R. Afra; citation_journal_title=Am. J. Sports Med.; citation_volume=41; citation_firstpage=374; citation_lastpage=384; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Heads, shoulders, elbows, knees, and toes: modular Gdf5 enhancers control different joints in the vertebrate skeleton; citation_author=H. Chen; citation_author=T.D. Capellini; citation_author=M. Schoor; citation_author=D.P. Mortlock; citation_author=A.H. Reddi; citation_author=D.M. Kingsley; citation_journal_title=PLoS Genet.; citation_volume=12; citation_firstpage=e1006454; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Three-dimensional in&nbsp;vitro effects of compression and time in culture on aggregate modulus and on gene expression and protein content of collagen type II in murine chondrocytes; citation_author=K. Chokalingam; citation_author=S. Hunter; citation_author=C. Gooch; citation_author=C. Frede; citation_author=J. Florer; citation_author=R. Wenstrup; citation_author=D. Butler; citation_journal_title=Tissue Eng. Part A; citation_volume=15; citation_firstpage=2807; citation_lastpage=2816; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=motifbreakR: an R/Bioconductor package for predicting variant effects at transcription factor binding sites; citation_author=S.G. Coetzee; citation_author=G.A. Coetzee; citation_author=D.J. Hazelett; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=3847; citation_lastpage=3849; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The evolution of lineage-specific regulatory activities in the human embryonic limb; citation_author=J. Cotney; citation_author=J. Leng; citation_author=J. Yin; citation_author=S.K. Reilly; citation_author=L.E. DeMare; citation_author=D. Emera; citation_author=A.E. Ayoub; citation_author=P. Rakic; citation_author=J.P. Noonan; citation_journal_title=Cell; citation_volume=154; citation_firstpage=185; citation_lastpage=196; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Distinct PAR/IQGAP expression patterns during murine development: implications for thrombin-associated cytoskeletal reorganization; citation_author=L.D. Cupit; citation_author=V.A. Schmidt; citation_author=F. Miller; citation_author=W.F. Bahou; citation_journal_title=Mamm. Genome; citation_volume=15; citation_firstpage=618; citation_lastpage=629; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=The variant call format and VCFtools; citation_author=P. Danecek; citation_author=A. Auton; citation_author=G. Abecasis; citation_author=C.A. Albers; citation_author=E. Banks; citation_author=M.A. DePristo; citation_author=R.E. Handsaker; citation_author=G. Lunter; citation_author=G.T. Marth; citation_author=S.T. Sherry; citation_author=1000 Genomes Project Analysis Group; citation_author=et al.; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=2156; citation_lastpage=2158; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The Descent of Man and Selection in Relation to Sex; citation_author=C. Darwin; citation_publication_date=1888; citation_publisher=Murray; "><meta name="citation_reference" content="citation_title=Rapid, automated imaging of mouse articular cartilage by microCT for early detection of osteoarthritis and finite element modelling of joint mechanics; citation_author=P. Das Neves Borges; citation_author=A.E. Forte; citation_author=T.L. Vincent; citation_author=D. Dini; citation_author=M. Marenzana; citation_journal_title=Osteoarthritis Cartilage; citation_volume=22; citation_firstpage=1419; citation_lastpage=1428; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The Encyclopedia of DNA elements (ENCODE): data portal update; citation_author=C.A. Davis; citation_author=B.C. Hitz; citation_author=C.A. Sloan; citation_author=E.T. Chan; citation_author=J.M. Davidson; citation_author=I. Gabdank; citation_author=J.A. Hilton; citation_author=K. Jain; citation_author=U.K. Baymuradov; citation_author=A.K. Narayanan; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D794; citation_lastpage=D801; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Identifying a high fraction of the human genome to be under selective constraint using GERP++; citation_author=E.V. Davydov; citation_author=D.L. Goode; citation_author=M. Sirota; citation_author=G.M. Cooper; citation_author=A. Sidow; citation_author=S. Batzoglou; citation_journal_title=PLoS Comput. Biol.; citation_volume=6; citation_firstpage=e1001025; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A variant in MCF2L is associated with osteoarthritis; citation_author=A.G. Day-Williams; citation_author=L. Southam; citation_author=K. Panoutsopoulou; citation_author=N.W. Rayner; citation_author=T. Esko; citation_author=K. Estrada; citation_author=H.T. Helgadottir; citation_author=A. Hofman; citation_author=T. Ingvarsson; citation_author=H. Jonsson; citation_author=arcOGEN Consortium; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=89; citation_firstpage=446; citation_lastpage=450; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Genesis and morphogenesis of limb synovial joints and articular cartilage; citation_author=R.S. Decker; citation_author=E. Koyama; citation_author=M. Pacifici; citation_journal_title=Matrix Biol.; citation_volume=39; citation_firstpage=5; citation_lastpage=10; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=fitdistrplus: An R Package for Fitting Distributions; citation_author=M.L. Delignette-Muller; citation_author=C. Dutang; citation_journal_title=J.&nbsp;Stat. Softw.; citation_volume=64; citation_firstpage=1; citation_lastpage=34; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The genomic landscape of cohesin-associated chromatin interactions; citation_author=L.E. DeMare; citation_author=J. Leng; citation_author=J. Cotney; citation_author=S.K. Reilly; citation_author=J. Yin; citation_author=R. Sarro; citation_author=J.P. Noonan; citation_journal_title=Genome Res.; citation_volume=23; citation_firstpage=1224; citation_lastpage=1234; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=High-throughput discovery of novel developmental phenotypes; citation_author=M.E. Dickinson; citation_author=A.M. Flenniken; citation_author=X. Ji; citation_author=L. Teboul; citation_author=M.D. Wong; citation_author=J.K. White; citation_author=T.F. Meehan; citation_author=W.J. Weninger; citation_author=H. Westerberg; citation_author=H. Adissu; citation_author=International Mouse Phenotyping Consortium; citation_author=Jackson Laboratory; citation_author=Infrastructure Nationale PHENOMIN, Institut Clinique de la Souris (ICS); citation_author=Charles River Laboratories; citation_author=MRC Harwell; citation_author=Toronto Centre for Phenogenomics; citation_author=Wellcome Trust Sanger Institute; citation_author=RIKEN BioResource Center; citation_author=et al.; citation_journal_title=Nature; citation_volume=537; citation_firstpage=508; citation_lastpage=514; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A high-resolution anatomical atlas of the transcriptome in the mouse embryo; citation_author=G. Diez-Roux; citation_author=S. Banfi; citation_author=M. Sultan; citation_author=L. Geffers; citation_author=S. Anand; citation_author=D. Rozado; citation_author=A. Magen; citation_author=E. Canidio; citation_author=M. Pagani; citation_author=I. Peluso; citation_author=et al.; citation_journal_title=PLoS Biol.; citation_volume=9; citation_firstpage=e1000582; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=An extensive 3′ regulatory region controls expression of Bmp5 in specific anatomical structures of the mouse embryo; citation_author=R.J. DiLeone; citation_author=L.B. Russell; citation_author=D.M. Kingsley; citation_journal_title=Genetics; citation_volume=148; citation_firstpage=401; citation_lastpage=408; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Proposal for a nomenclature for magnetic resonance imaging based measures of articular cartilage in osteoarthritis; citation_author=F. Eckstein; citation_author=G. Ateshian; citation_author=R. Burgkart; citation_author=D. Burstein; citation_author=F. Cicuttini; citation_author=B. Dardzinski; citation_author=M. Gray; citation_author=T.M. Link; citation_author=S. Majumdar; citation_author=T. Mosher; citation_author=et al.; citation_journal_title=Osteoarthritis Cartilage; citation_volume=14; citation_firstpage=974; citation_lastpage=983; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=An integrated encyclopedia of DNA elements in the human genome; citation_author=ENCODE Project Consortium; citation_journal_title=Nature; citation_volume=489; citation_firstpage=57; citation_lastpage=74; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Meta-analysis of genome-wide association studies confirms a susceptibility locus for knee osteoarthritis on chromosome 7q22; citation_author=E. Evangelou; citation_author=A.M. Valdes; citation_author=H.J. Kerkhof; citation_author=U. Styrkarsdottir; citation_author=Y. Zhu; citation_author=I. Meulenbelt; citation_author=R.J. Lories; citation_author=F.B. Karassa; citation_author=P. Tylzanowski; citation_author=S.D. Bos; citation_author=arcOGEN Consortium; citation_author=Translation Research in Europe Applied Technologies for Osteoarthritis (TreatOA); citation_author=et al.; citation_journal_title=Ann. Rheum. Dis.; citation_volume=70; citation_firstpage=349; citation_lastpage=355; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A meta-analysis of genome-wide association studies identifies novel variants associated with osteoarthritis of the hip; citation_author=E. Evangelou; citation_author=H.J. Kerkhof; citation_author=U. Styrkarsdottir; citation_author=E.E. Ntzani; citation_author=S.D. Bos; citation_author=T. Esko; citation_author=D.S. Evans; citation_author=S. Metrustry; citation_author=K. Panoutsopoulou; citation_author=Y.F. Ramos; citation_author=arcOGEN Consortium; citation_author=et al.; citation_journal_title=Ann. Rheum. Dis.; citation_volume=73; citation_firstpage=2130; citation_lastpage=2136; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Obesity and knee osteoarthritis. The Framingham Study; citation_author=D.T. Felson; citation_author=J.J. Anderson; citation_author=A. Naimark; citation_author=A.M. Walker; citation_author=R.F. Meenan; citation_journal_title=Ann. Intern. Med.; citation_volume=109; citation_firstpage=18; citation_lastpage=24; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=On detecting incomplete soft or hard selective sweeps using haplotype structure; citation_author=A. Ferrer-Admetlla; citation_author=M. Liang; citation_author=T. Korneliussen; citation_author=R. Nielsen; citation_journal_title=Mol. Biol. Evol.; citation_volume=31; citation_firstpage=1275; citation_lastpage=1291; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Molecular phenotyping of human chondrocyte cell lines T/C-28a2, T/C-28a4, and C-28/I2; citation_author=F. Finger; citation_author=C. Schörle; citation_author=A. Zien; citation_author=P. Gebhard; citation_author=M.B. Goldring; citation_author=T. Aigner; citation_journal_title=Arthritis Rheum.; citation_volume=48; citation_firstpage=3395; citation_lastpage=3403; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Partitioning heritability by functional annotation using genome-wide association summary statistics; citation_author=H.K. Finucane; citation_author=B. Bulik-Sullivan; citation_author=A. Gusev; citation_author=G. Trynka; citation_author=Y. Reshef; citation_author=P.-R. Loh; citation_author=V. Anttila; citation_author=H. Xu; citation_author=C. Zang; citation_author=K. Farh; citation_author=ReproGen Consortium; citation_author=Schizophrenia Working Group of the Psychiatric Genomics Consortium; citation_author=RACI Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=47; citation_firstpage=1228; citation_lastpage=1235; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Segmenting articular cartilage automatically using a voxel classification approach; citation_author=J. Folkesson; citation_author=E.B. Dam; citation_author=O.F. Olsen; citation_author=P.C. Pettersen; citation_author=C. Christiansen; citation_journal_title=IEEE Trans. Med. Imaging; citation_volume=26; citation_firstpage=106; citation_lastpage=115; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=An R Companion to Applied Regression; citation_author=J. Fox; citation_author=S. Weisberg; citation_publication_date=2019; citation_publisher=SAGE Publications; "><meta name="citation_reference" content="citation_title=Evolution of the hominin knee and ankle; citation_author=M.A. Frelat; citation_author=C.N. Shaw; citation_author=S. Sukhdeo; citation_author=J.-J. Hublin; citation_author=S. Benazzi; citation_author=T.M. Ryan; citation_journal_title=J.&nbsp;Hum. Evol.; citation_volume=108; citation_firstpage=147; citation_lastpage=160; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The structure of haplotype blocks in the human genome; citation_author=S.B. Gabriel; citation_author=S.F. Schaffner; citation_author=H. Nguyen; citation_author=J.M. Moore; citation_author=J. Roy; citation_author=B. Blumenstiel; citation_author=J. Higgins; citation_author=M. DeFelice; citation_author=A. Lochner; citation_author=M. Faggart; citation_author=et al.; citation_journal_title=Science; citation_volume=296; citation_firstpage=2225; citation_lastpage=2229; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=The Role of Bmp2 in the Maturation and Maintenance of the Murine Knee Joint; citation_author=L.W. Gamer; citation_author=S. Pregizer; citation_author=J. Gamer; citation_author=M. Feigenson; citation_author=A. Ionescu; citation_author=Q. Li; citation_author=L. Han; citation_author=V. Rosen; citation_journal_title=J.&nbsp;Bone Miner. Res.; citation_volume=33; citation_firstpage=1708; citation_lastpage=1717; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The early development of the knee joint in staged human embryos; citation_author=E. Gardner; citation_author=R. O’Rahilly; citation_journal_title=J.&nbsp;Anat.; citation_volume=102; citation_firstpage=289; citation_lastpage=299; citation_publication_date=1968; "><meta name="citation_reference" content="citation_title=Linkage disequilibrium-dependent architecture of human complex traits shows action of negative selection; citation_author=S. Gazal; citation_author=H.K. Finucane; citation_author=N.A. Furlotte; citation_author=P.-R. Loh; citation_author=P.F. Palamara; citation_author=X. Liu; citation_author=A. Schoech; citation_author=B. Bulik-Sullivan; citation_author=B.M. Neale; citation_author=A. Gusev; citation_author=A.L. Price; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1421; citation_lastpage=1427; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Deep conservation of wrist and digit enhancers in fish; citation_author=A.R. Gehrke; citation_author=I. Schneider; citation_author=E. de la Calle-Mustienes; citation_author=J.J. Tena; citation_author=C. Gomez-Marin; citation_author=M. Chandran; citation_author=T. Nakamura; citation_author=I. Braasch; citation_author=J.H. Postlethwait; citation_author=J.L. Gómez-Skarmeta; citation_author=N.H. Shubin; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=803; citation_lastpage=808; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests; citation_author=B. Gel; citation_author=A. Díez-Villanueva; citation_author=E. Serra; citation_author=M. Buschbeck; citation_author=M.A. Peinado; citation_author=R. Malinverni; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=289; citation_lastpage=291; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Assessing identity, redundancy and confounds in Gene Ontology annotations over time; citation_author=J. Gillis; citation_author=P. Pavlidis; citation_journal_title=Bioinformatics; citation_volume=29; citation_doi=10.1093/bioinformatics/bts727; citation_firstpage=476; citation_lastpage=482; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Comprehensive identification and analysis of human accelerated regulatory DNA; citation_author=R.M. Gittelman; citation_author=E. Hun; citation_author=F. Ay; citation_author=J. Madeoy; citation_author=L. Pennacchio; citation_author=W.S. Noble; citation_author=R.D. Hawkins; citation_author=J.M. Akey; citation_journal_title=Genome Res.; citation_volume=25; citation_firstpage=1245; citation_lastpage=1255; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The OARSI histopathology initiative - recommendations for histological assessments of osteoarthritis in the mouse; citation_author=S.S. Glasson; citation_author=M.G. Chambers; citation_author=W.B. Van Den Berg; citation_author=C.B. Little; citation_journal_title=Osteoarthritis Cartilage; citation_volume=18; citation_number=Suppl&nbsp;3; citation_firstpage=S17; citation_lastpage=S23; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Recent Regulatory Changes Shaped Human Facial and Vocal Anatomy; citation_author=D. Gokhman; citation_author=L. Agranat-Tamir; citation_author=G. Housman; citation_author=M. Nissim-Rafinia; citation_author=M. Nieves-Colón; citation_author=H. Gu; citation_author=M. Ferrando-Bernal; citation_author=P. Gelabert; citation_author=I. Lipende; citation_author=R. Bontrop; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/106955; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Prenatal development of the human knee and superior tibiofibular joints; citation_author=D.J. Gray; citation_author=E. Gardner; citation_journal_title=Am. J. Anat.; citation_volume=86; citation_firstpage=235; citation_lastpage=287; citation_publication_date=1950; "><meta name="citation_reference" content="citation_title=Identifying recent adaptations in large-scale genomic data; citation_author=S.R. Grossman; citation_author=K.G. Andersen; citation_author=I. Shlyakhter; citation_author=S. Tabrizi; citation_author=S. Winnicki; citation_author=A. Yen; citation_author=D.J. Park; citation_author=D. Griesemer; citation_author=E.K. Karlsson; citation_author=S.H. Wong; citation_author=1000 Genomes Project; citation_author=et al.; citation_journal_title=Cell; citation_volume=152; citation_firstpage=703; citation_lastpage=713; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Complex heatmaps reveal patterns and correlations in multidimensional genomic data; citation_author=Z. Gu; citation_author=R. Eils; citation_author=M. Schlesner; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=2847; citation_lastpage=2849; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Shaping skeletal growth by modular regulatory elements in the Bmp5 gene; citation_author=C. Guenther; citation_author=L. Pantalena-Filho; citation_author=D.M. Kingsley; citation_journal_title=PLoS Genet.; citation_volume=4; citation_firstpage=e1000308; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Epigenetic profiling of growth plate chondrocytes sheds insight into regulatory genetic variation influencing height; citation_author=M. Guo; citation_author=Z. Liu; citation_author=J. Willen; citation_author=C.P. Shaw; citation_author=D. Richard; citation_author=E. Jagoda; citation_author=A.C. Doxey; citation_author=J. Hirschhorn; citation_author=T.D. Capellini; citation_journal_title=eLife; citation_volume=6; citation_firstpage=e29329; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The geometry of the tibial plateau and its influence on the biomechanics of the tibiofemoral joint; citation_author=J. Hashemi; citation_author=N. Chandrashekar; citation_author=B. Gill; citation_author=B.D. Beynnon; citation_author=J.R. Slauterbeck; citation_author=R.C. Schutt; citation_author=H. Mansouri; citation_author=E. Dabezies; citation_journal_title=J.&nbsp;Bone Joint Surg. Am.; citation_volume=90; citation_firstpage=2724; citation_lastpage=2734; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The role of retinoic acid in hepatic lipid homeostasis defined by genomic binding and transcriptome profiling; citation_author=Y. He; citation_author=L. Gong; citation_author=Y. Fang; citation_author=Q. Zhan; citation_author=H.X. Liu; citation_author=Y. Lu; citation_author=G.L. Guo; citation_author=L. Lehman-McKeeman; citation_author=J. Fang; citation_author=Y.J.Y. Wan; citation_journal_title=BMC Genomics; citation_volume=14; citation_firstpage=575; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The biology of IQGAP proteins: beyond the cytoskeleton; citation_author=A.C. Hedman; citation_author=J.M. Smith; citation_author=D.B. Sacks; citation_journal_title=EMBO Rep.; citation_volume=16; citation_firstpage=427; citation_lastpage=446; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities; citation_author=S. Heinz; citation_author=C. Benner; citation_author=N. Spann; citation_author=E. Bertolino; citation_author=Y.C. Lin; citation_author=P. Laslo; citation_author=J.X. Cheng; citation_author=C. Murre; citation_author=H. Singh; citation_author=C.K. Glass; citation_journal_title=Mol. Cell; citation_volume=38; citation_firstpage=576; citation_lastpage=589; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Investigating the role of CCCTC-binding factor in osteoarthritis pathogenesis; citation_author=A. Hijazi; citation_journal_title=Osteoarthritis Cartilage; citation_volume=26; citation_firstpage=S154; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Potential etiologic and functional implications of genome-wide association loci for human diseases and traits; citation_author=L.A. Hindorff; citation_author=P. Sethupathy; citation_author=H.A. Junkins; citation_author=E.M. Ramos; citation_author=J.P. Mehta; citation_author=F.S. Collins; citation_author=T.A. Manolio; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=106; citation_firstpage=9362; citation_lastpage=9367; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Leveraging molecular quantitative trait loci to understand the genetic architecture of diseases and complex traits; citation_author=F. Hormozdiari; citation_author=S. Gazal; citation_author=B. van de Geijn; citation_author=H.K. Finucane; citation_author=C.J.-T. Ju; citation_author=P.-R. Loh; citation_author=A. Schoech; citation_author=Y. Reshef; citation_author=X. Liu; citation_author=L. O’Connor; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=1041; citation_lastpage=1047; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Homeostatic mechanisms in articular cartilage and role of inflammation in osteoarthritis; citation_author=X. Houard; citation_author=M.B. Goldring; citation_author=F. Berenbaum; citation_journal_title=Curr. Rheumatol. Rep.; citation_volume=15; citation_firstpage=375; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Assessment of the radii of the medial and lateral femoral condyles in varus and valgus knees with osteoarthritis; citation_author=S.M. Howell; citation_author=S.J. Howell; citation_author=M.L. Hull; citation_journal_title=J.&nbsp;Bone Joint Surg. Am.; citation_volume=92; citation_firstpage=98; citation_lastpage=104; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=PHAST and RPHAST: phylogenetic analysis with space/time models; citation_author=M.J. Hubisz; citation_author=K.S. Pollard; citation_author=A. Siepel; citation_journal_title=Brief. Bioinform.; citation_volume=12; citation_firstpage=41; citation_lastpage=51; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=UniPROBE, update 2015: new tools and content for the online database of protein-binding microarray data on protein-DNA interactions; citation_author=M.A. Hume; citation_author=L.A. Barrera; citation_author=S.S. Gisselbrecht; citation_author=M.L. Bulyk; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D117; citation_lastpage=D122; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Osteoarthritis; citation_author=D.J. Hunter; citation_author=S. Bierma-Zeinstra; citation_journal_title=Lancet; citation_volume=393; citation_firstpage=1745; citation_lastpage=1759; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Longitudinal validation of periarticular bone area and 3D shape as biomarkers for knee OA progression? Data from the FNIH OA Biomarkers Consortium; citation_author=D. Hunter; citation_author=M. Nevitt; citation_author=J. Lynch; citation_author=V.B. Kraus; citation_author=J.N. Katz; citation_author=J.E. Collins; citation_author=M. Bowes; citation_author=A. Guermazi; citation_author=F.W. Roemer; citation_author=E. Losina; citation_author=FNIH OA Biomarkers Consortium; citation_journal_title=Ann. Rheum. Dis.; citation_volume=75; citation_firstpage=1607; citation_lastpage=1614; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Evolving New Skeletal Traits by cis-Regulatory Changes in Bone Morphogenetic Proteins; citation_author=V.B. Indjeian; citation_author=G.A. Kingman; citation_author=F.C. Jones; citation_author=C.A. Guenther; citation_author=J. Grimwood; citation_author=J. Schmutz; citation_author=R.M. Myers; citation_author=D.M. Kingsley; citation_journal_title=Cell; citation_volume=164; citation_firstpage=45; citation_lastpage=56; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Pitx1 broadly associates with limb enhancers and is enriched on hindlimb cis-regulatory elements; citation_author=C.R. Infante; citation_author=S. Park; citation_author=A.G. Mihala; citation_author=D.M. Kingsley; citation_author=D.B. Menke; citation_journal_title=Dev. Biol.; citation_volume=374; citation_firstpage=234; citation_lastpage=244; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title={pscl}: Classes and Methods for {R} Developed in the Political Science Computational Laboratory; citation_author=S. Jackman; citation_publication_date=2017; citation_publisher=R package version 1.5.2. United States Studies Centre, University of Sydney; "><meta name="citation_reference" content="citation_title=Disentangling Immediate Adaptive Introgression from Selection on Standing Introgressed Variation in Humans; citation_author=E. Jagoda; citation_author=D.J. Lawson; citation_author=J.D. Wall; citation_author=D. Lambert; citation_author=C. Muller; citation_author=M. Westaway; citation_author=M. Leavesley; citation_author=T.D. Capellini; citation_author=M. Mirazón Lahr; citation_author=P. Gerbault; citation_author=et al.; citation_journal_title=Mol. Biol. Evol.; citation_volume=35; citation_firstpage=623; citation_lastpage=630; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Relative joint size and hominoid locomotor adaptations with implications for the evolution of hominid bipedalism; citation_author=W.L. Jungers; citation_journal_title=J.&nbsp;Hum. Evol.; citation_volume=17; citation_firstpage=247; citation_lastpage=265; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=Employing molecular genetics of chondrodysplasias to inform the study of osteoarthritis; citation_author=P. Kannu; citation_author=J.F. Bateman; citation_author=D. Belluoccio; citation_author=A.J. Fosang; citation_author=R. Savarirayan; citation_journal_title=Arthritis Rheum.; citation_volume=60; citation_firstpage=325; citation_lastpage=334; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Organoid single-cell genomic atlas uncovers human-specific features of brain development; citation_author=S. Kanton; citation_author=M.J. Boyle; citation_author=Z. He; citation_author=M. Santel; citation_author=A. Weigert; citation_author=F. Sanchís-Calleja; citation_author=P. Guijarro; citation_author=L. Sidow; citation_author=J.S. Fleck; citation_author=D. Han; citation_author=et al.; citation_journal_title=Nature; citation_volume=574; citation_firstpage=418; citation_lastpage=422; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The UCSC Table Browser data retrieval tool; citation_author=D. Karolchik; citation_author=A.S. Hinrichs; citation_author=T.S. Furey; citation_author=K.M. Roskin; citation_author=C.W. Sugnet; citation_author=D. Haussler; citation_author=W.J. Kent; citation_journal_title=Nucleic Acids Res.; citation_volume=32; citation_firstpage=D493; citation_lastpage=D496; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=The UCSC Genome Browser database: 2014 update; citation_author=D. Karolchik; citation_author=G.P. Barber; citation_author=J. Casper; citation_author=H. Clawson; citation_author=M.S. Cline; citation_author=M. Diekhans; citation_author=T.R. Dreszer; citation_author=P.A. Fujita; citation_author=L. Guruvadoo; citation_author=M. Haeussler; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=D764; citation_lastpage=D770; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Factoextra: extract and visualize the results of multivariate data analyses; citation_author=A. Kassambara; citation_author=F. Mundt; citation_publication_date=2017; citation_publisher=R package version 1.0. 4; "><meta name="citation_reference" content="citation_title=The human genome browser at UCSC; citation_author=W.J. Kent; citation_author=C.W. Sugnet; citation_author=T.S. Furey; citation_author=K.M. Roskin; citation_author=T.H. Pringle; citation_author=A.M. Zahler; citation_author=D. Haussler; citation_journal_title=Genome Res.; citation_volume=12; citation_firstpage=996; citation_lastpage=1006; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=A genome-wide association study identifies an osteoarthritis susceptibility locus on chromosome 7q22; citation_author=H.J. Kerkhof; citation_author=R.J. Lories; citation_author=I. Meulenbelt; citation_author=I. Jonsdottir; citation_author=A.M. Valdes; citation_author=P. Arp; citation_author=T. Ingvarsson; citation_author=M. Jhamai; citation_author=H. Jonsson; citation_author=L. Stolk; citation_author=et al.; citation_journal_title=Arthritis Rheum.; citation_volume=62; citation_firstpage=499; citation_lastpage=510; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Evolution at two levels in humans and chimpanzees; citation_author=M.-C. King; citation_author=A.C. Wilson; citation_journal_title=Science; citation_volume=188; citation_firstpage=107; citation_lastpage=116; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Functional testing of thousands of osteoarthritis-associated variants for regulatory activity; citation_author=J.C. Klein; citation_author=A. Keith; citation_author=S.J. Rice; citation_author=C. Shepherd; citation_author=V. Agarwal; citation_author=J. Loughlin; citation_author=J. Shendure; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=2434; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Proteoglycan production by immortalized human chondrocyte cell lines cultured under conditions that promote expression of the differentiated phenotype; citation_author=R. Kokenyesi; citation_author=L. Tan; citation_author=J.R. Robbins; citation_author=M.B. Goldring; citation_journal_title=Arch. Biochem. Biophys.; citation_volume=383; citation_firstpage=79; citation_lastpage=90; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Repair of double-strand breaks induced by CRISPR-Cas9 leads to large deletions and complex rearrangements; citation_author=M. Kosicki; citation_author=K. Tomberg; citation_author=A. Bradley; citation_journal_title=Nat. Biotechnol.; citation_volume=36; citation_firstpage=765; citation_lastpage=771; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Tabix: fast retrieval of sequence features from generic TAB-delimited files; citation_author=H. Li; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=718; citation_lastpage=719; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data; citation_author=H. Li; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=2987; citation_lastpage=2993; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Measuring reproducibility of high-throughput experiments; citation_author=Q. Li; citation_author=J.B. Brown; citation_author=H. Huang; citation_author=P.J. Bickel; citation_journal_title=Ann. Appl. Stat.; citation_volume=5; citation_firstpage=1752; citation_lastpage=1779; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Transcriptional control of chondrocyte specification and differentiation; citation_author=C.-F. Liu; citation_author=W.E. Samsa; citation_author=G. Zhou; citation_author=V. Lefebvre; citation_journal_title=Semin. Cell Dev. Biol.; citation_volume=62; citation_firstpage=34; citation_lastpage=49; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genetic Determinants of Radiographic Knee Osteoarthritis in African Americans; citation_author=Y. Liu; citation_author=M.S. Yau; citation_author=L.M. Yerges-Armstrong; citation_author=D.J. Duggan; citation_author=J.B. Renner; citation_author=M.C. Hochberg; citation_author=B.D. Mitchell; citation_author=R.D. Jackson; citation_author=J.M. Jordan; citation_journal_title=J.&nbsp;Rheumatol.; citation_volume=44; citation_firstpage=1652; citation_lastpage=1658; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genetic contribution to osteoarthritis development: current state of evidence; citation_author=J. Loughlin; citation_journal_title=Curr. Opin. Rheumatol.; citation_volume=27; citation_firstpage=284; citation_lastpage=288; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The natural history of human gait and posture. Part 3. The knee; citation_author=C.O. Lovejoy; citation_journal_title=Gait Posture; citation_volume=25; citation_firstpage=325; citation_lastpage=341; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Comparative Pathology of Aging Great Apes: Bonobos, Chimpanzees, Gorillas, and Orangutans; citation_author=L.J. Lowenstine; citation_author=R. McManamon; citation_author=K.A. Terio; citation_journal_title=Vet. Pathol.; citation_volume=53; citation_firstpage=250; citation_lastpage=276; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=BMPs, TGFβ, and border security at the interzone; citation_author=K.M. Lyons; citation_author=V. Rosen; citation_journal_title=Curr. Top. Dev. Biol.; citation_volume=133; citation_firstpage=153; citation_lastpage=170; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Single base pair change in the long-range Sonic hedgehog limb-specific enhancer is a genetic basis for preaxial polydactyly; citation_author=S.A. Maas; citation_author=J.F. Fallon; citation_journal_title=Dev. Dyn.; citation_volume=232; citation_firstpage=345; citation_lastpage=348; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Reproductive status and sex show strong effects on knee OA in a baboon model; citation_author=T.E. Macrini; citation_author=H.B. Coan; citation_author=S.M. Levine; citation_author=T. Lerma; citation_author=C.D. Saks; citation_author=D.J. Araujo; citation_author=T.L. Bredbenner; citation_author=R.D. Coutts; citation_author=D.P. Nicolella; citation_author=L.M. Havill; citation_journal_title=Osteoarthritis Cartilage; citation_volume=21; citation_firstpage=839; citation_lastpage=848; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=cluster: Cluster Analysis Basics and Extensions. R Packages Version 1; citation_author=M. Maechler; citation_author=P. Rousseeuw; citation_author=A. Struyf; citation_author=M. Hubert; citation_author=K. Hornik; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles; citation_author=A. Mathelier; citation_author=O. Fornes; citation_author=D.J. Arenillas; citation_author=C.Y. Chen; citation_author=G. Denay; citation_author=J. Lee; citation_author=W. Shi; citation_author=C. Shyr; citation_author=G. Tan; citation_author=R. Worsley-Hunt; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=D1; citation_firstpage=D110; citation_lastpage=D115; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=GREAT improves functional interpretation of cis-regulatory regions; citation_author=C.Y. McLean; citation_author=D. Bristor; citation_author=M. Hiller; citation_author=S.L. Clarke; citation_author=B.T. Schaar; citation_author=C.B. Lowe; citation_author=A.M. Wenger; citation_author=G. Bejerano; citation_journal_title=Nat. Biotechnol.; citation_volume=28; citation_firstpage=495; citation_lastpage=501; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Human-specific loss of regulatory DNA and the evolution of human-specific traits; citation_author=C.Y. McLean; citation_author=P.L. Reno; citation_author=A.A. Pollen; citation_author=A.I. Bassan; citation_author=T.D. Capellini; citation_author=C. Guenther; citation_author=V.B. Indjeian; citation_author=X. Lim; citation_author=D.B. Menke; citation_author=B.T. Schaar; citation_author=et al.; citation_journal_title=Nature; citation_volume=471; citation_firstpage=216; citation_lastpage=219; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Development of the human knee joint; citation_author=J.A. Mérida-Velasco; citation_author=I. Sánchez-Montesinos; citation_author=J. Espín-Ferra; citation_author=J.F. Rodríguez-Vázquez; citation_author=J.R. Mérida-Velasco; citation_author=J. Jiménez-Collado; citation_journal_title=Anat. Rec.; citation_volume=248; citation_firstpage=269; citation_lastpage=278; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=The RUNX2 cistrome in osteoblasts: characterization, down-regulation following differentiation, and relationship to gene expression; citation_author=M.B. Meyer; citation_author=N.A. Benkusky; citation_author=J.W. Pike; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=289; citation_firstpage=16016; citation_lastpage=16031; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis; citation_author=Y. Miyamoto; citation_author=A. Mabuchi; citation_author=D. Shi; citation_author=T. Kubo; citation_author=Y. Takatori; citation_author=S. Saito; citation_author=M. Fujioka; citation_author=A. Sudo; citation_author=A. Uchida; citation_author=S. Yamamoto; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=39; citation_firstpage=529; citation_lastpage=533; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Common variants in DVWA on chromosome 3p24.3 are associated with susceptibility to knee osteoarthritis; citation_author=Y. Miyamoto; citation_author=D. Shi; citation_author=M. Nakajima; citation_author=K. Ozaki; citation_author=A. Sudo; citation_author=A. Kotani; citation_author=A. Uchida; citation_author=T. Tanaka; citation_author=N. Fukui; citation_author=T. Tsunoda; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=40; citation_firstpage=994; citation_lastpage=998; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The mechanics of the knee joint in relation to normal walking; citation_author=J.B. Morrison; citation_journal_title=J.&nbsp;Biomech.; citation_volume=3; citation_firstpage=51; citation_lastpage=61; citation_publication_date=1970; "><meta name="citation_reference" content="citation_title=Advanced Analytical Models: over 800 Models and 300 Applications from the Basel II Accord to Wall Street and Beyond; citation_author=J. Mun; citation_publication_date=2008; citation_publisher=Wiley; "><meta name="citation_reference" content="citation_title=New sequence variants in HLA class II/III region associated with susceptibility to knee osteoarthritis identified by genome-wide association study; citation_author=M. Nakajima; citation_author=A. Takahashi; citation_author=I. Kou; citation_author=C. Rodriguez-Fontenla; citation_author=J.J. Gomez-Reino; citation_author=T. Furuichi; citation_author=J. Dai; citation_author=A. Sudo; citation_author=A. Uchida; citation_author=N. Fukui; citation_author=et al.; citation_journal_title=PLoS ONE; citation_volume=5; citation_firstpage=e9723; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Pitx1 directly modulates the core limb development program to implement hindlimb identity; citation_author=S. Nemec; citation_author=M. Luxey; citation_author=D. Jain; citation_author=A. Huang Sung; citation_author=T. Pastinen; citation_author=J. Drouin; citation_journal_title=Development; citation_volume=144; citation_firstpage=3325; citation_lastpage=3335; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Osteoarthritis: Bone as an imaging biomarker and treatment target in OA; citation_author=T. Neogi; citation_author=D.T. Felson; citation_journal_title=Nat. Rev. Rheumatol.; citation_volume=12; citation_firstpage=503; citation_lastpage=504; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Magnetic resonance imaging-based three-dimensional bone shape of the knee predicts onset of knee osteoarthritis: data from the osteoarthritis initiative; citation_author=T. Neogi; citation_author=M.A. Bowes; citation_author=J. Niu; citation_author=K.M. De Souza; citation_author=G.R. Vincent; citation_author=J. Goggins; citation_author=Y. Zhang; citation_author=D.T. Felson; citation_journal_title=Arthritis Rheum.; citation_volume=65; citation_firstpage=2048; citation_lastpage=2058; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The early prenatal development of the human knee joint; citation_author=R. O’Rahilly; citation_journal_title=J.&nbsp;Anat.; citation_volume=85; citation_firstpage=166; citation_lastpage=170; citation_publication_date=1951; "><meta name="citation_reference" content="citation_title=Distinct Transcriptional Programs Underlie Sox9 Regulation of the Mammalian Chondrocyte; citation_author=S. Ohba; citation_author=X. He; citation_author=H. Hojo; citation_author=A.P. McMahon; citation_journal_title=Cell Rep.; citation_volume=12; citation_firstpage=229; citation_lastpage=243; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Genomic analyses inform on migration events during the peopling of Eurasia; citation_author=L. Pagani; citation_author=D.J. Lawson; citation_author=E. Jagoda; citation_author=A. Mörseburg; citation_author=A. Eriksson; citation_author=M. Mitt; citation_author=F. Clemente; citation_author=G. Hudjashov; citation_author=M. DeGiorgio; citation_author=L. Saag; citation_author=et al.; citation_journal_title=Nature; citation_volume=538; citation_firstpage=238; citation_lastpage=242; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Insights into the genetic architecture of osteoarthritis from stage 1 of the arcOGEN study; citation_author=K. Panoutsopoulou; citation_author=L. Southam; citation_author=K.S. Elliott; citation_author=N. Wrayner; citation_author=G. Zhai; citation_author=C. Beazley; citation_author=G. Thorleifsson; citation_author=N.K. Arden; citation_author=A. Carr; citation_author=K. Chapman; citation_author=arcOGEN Consortium; citation_author=et al.; citation_journal_title=Ann. Rheum. Dis.; citation_volume=70; citation_firstpage=864; citation_lastpage=867; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=pegas: an R package for population genetics with an integrated-modular approach; citation_author=E. Paradis; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=419; citation_lastpage=420; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=New emerging role of pitx1 transcription factor in osteoarthritis pathogenesis; citation_author=C. Picard; citation_author=B. Azeddine; citation_author=F. Moldovan; citation_author=J. Martel-Pelletier; citation_author=A. Moreau; citation_journal_title=Clin. Orthop. Relat. Res.; citation_volume=462; citation_firstpage=59; citation_lastpage=66; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Body size and joint posture in primates; citation_author=J.D. Polk; citation_author=S.A. Williams; citation_author=J.V. Peterson; citation_journal_title=Am. J. Phys. Anthropol.; citation_volume=140; citation_firstpage=359; citation_lastpage=367; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Forces shaping the fastest evolving regions in the human genome; citation_author=K.S. Pollard; citation_author=S.R. Salama; citation_author=B. King; citation_author=A.D. Kern; citation_author=T. Dreszer; citation_author=S. Katzman; citation_author=A. Siepel; citation_author=J.S. Pedersen; citation_author=G. Bejerano; citation_author=R. Baertsch; citation_author=et al.; citation_journal_title=PLoS Genet.; citation_volume=2; citation_firstpage=e168; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=An RNA gene expressed during cortical development evolved rapidly in humans; citation_author=K.S. Pollard; citation_author=S.R. Salama; citation_author=N. Lambert; citation_author=M.-A. Lambot; citation_author=S. Coppens; citation_author=J.S. Pedersen; citation_author=S. Katzman; citation_author=B. King; citation_author=C. Onodera; citation_author=A. Siepel; citation_author=et al.; citation_journal_title=Nature; citation_volume=443; citation_firstpage=167; citation_lastpage=172; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Detection of nonneutral substitution rates on mammalian phylogenies; citation_author=K.S. Pollard; citation_author=M.J. Hubisz; citation_author=K.R. Rosenbloom; citation_author=A. Siepel; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=110; citation_lastpage=121; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Accelerated evolution of conserved noncoding sequences in humans; citation_author=S. Prabhakar; citation_author=J.P. Noonan; citation_author=S. Pääbo; citation_author=E.M. Rubin; citation_journal_title=Science; citation_volume=314; citation_firstpage=786; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Human-specific gain of function in a developmental enhancer; citation_author=S. Prabhakar; citation_author=A. Visel; citation_author=J.A. Akiyama; citation_author=M. Shoukry; citation_author=K.D. Lewis; citation_author=A. Holt; citation_author=I. Plajzer-Frick; citation_author=H. Morrison; citation_author=D.R. Fitzpatrick; citation_author=V. Afzal; citation_author=et al.; citation_journal_title=Science; citation_volume=321; citation_firstpage=1346; citation_lastpage=1350; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Great ape genetic diversity and population history; citation_author=J. Prado-Martinez; citation_author=P.H. Sudmant; citation_author=J.M. Kidd; citation_author=H. Li; citation_author=J.L. Kelley; citation_author=B. Lorente-Galdos; citation_author=K.R. Veeramah; citation_author=A.E. Woerner; citation_author=T.D. O’Connor; citation_author=G. Santpere; citation_author=et al.; citation_journal_title=Nature; citation_volume=499; citation_firstpage=471; citation_lastpage=475; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Impact of broad regulatory regions on Gdf5 expression and function in knee development and susceptibility to osteoarthritis; citation_author=S.K. Pregizer; citation_author=A.M. Kiapour; citation_author=M. Young; citation_author=H. Chen; citation_author=M. Schoor; citation_author=Z. Liu; citation_author=J. Cao; citation_author=V. Rosen; citation_author=T.D. Capellini; citation_journal_title=Ann. Rheum. Dis.; citation_volume=77; citation_firstpage=450; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features; citation_author=A.R. Quinlan; citation_author=I.M. Hall; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=841; citation_lastpage=842; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.; citation_author=R Development Core Team; citation_publication_date=2008; citation_publisher=R Foundation for Statistical Computing; "><meta name="citation_reference" content="citation_title=Testing for ancient selection using cross-population allele frequency differentiation; citation_author=F. Racimo; citation_journal_title=Genetics; citation_volume=202; citation_firstpage=733; citation_lastpage=750; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Meta-analysis identifies loci affecting levels of the potential osteoarthritis biomarkers sCOMP and uCTX-II with genome wide significance; citation_author=Y.F. Ramos; citation_author=S. Metrustry; citation_author=N. Arden; citation_author=A.C. Bay-Jensen; citation_author=M. Beekman; citation_author=A.J. de Craen; citation_author=L.A. Cupples; citation_author=T. Esko; citation_author=E. Evangelou; citation_author=D.T. Felson; citation_author=arcOGEN Consortium; citation_author=TreatOA Collaborators; citation_author=et al.; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=51; citation_firstpage=596; citation_lastpage=604; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Genome engineering using the CRISPR-Cas9 system; citation_author=F.A. Ran; citation_author=P.D. Hsu; citation_author=J. Wright; citation_author=V. Agarwala; citation_author=D.A. Scott; citation_author=F. Zhang; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=2281; citation_lastpage=2308; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Genome-wide inference of ancestral recombination graphs; citation_author=M.D. Rasmussen; citation_author=M.J. Hubisz; citation_author=I. Gronau; citation_author=A. Siepel; citation_journal_title=PLoS Genet.; citation_volume=10; citation_firstpage=e1004342; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Association between overweight and obesity and risk of clinically diagnosed knee, hip, and hand osteoarthritis: a population-based cohort study; citation_author=C. Reyes; citation_author=K.M. Leyland; citation_author=G. Peat; citation_author=C. Cooper; citation_author=N.K. Arden; citation_author=D. Prieto-Alhambra; citation_journal_title=Arthritis Rheumatol; citation_volume=68; citation_firstpage=1869; citation_lastpage=1875; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Benefits of massive weight loss on symptoms, systemic inflammation and cartilage turnover in obese patients with knee osteoarthritis; citation_author=P. Richette; citation_author=C. Poitou; citation_author=P. Garnero; citation_author=E. Vicaut; citation_author=J.-L. Bouillot; citation_author=J.-M. Lacorte; citation_author=A. Basdevant; citation_author=K. Clément; citation_author=T. Bardin; citation_author=X. Chevalier; citation_journal_title=Ann. Rheum. Dis.; citation_volume=70; citation_firstpage=139; citation_lastpage=144; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=BMP2 signaling in bone development and repair; citation_author=V. Rosen; citation_journal_title=Cytokine Growth Factor Rev.; citation_volume=20; citation_firstpage=475; citation_lastpage=480; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=BMP receptor signaling is required for postnatal maintenance of articular cartilage; citation_author=R.B. Rountree; citation_author=M. Schoor; citation_author=H. Chen; citation_author=M.E. Marks; citation_author=V. Harley; citation_author=Y. Mishina; citation_author=D.M. Kingsley; citation_journal_title=PLoS Biol.; citation_volume=2; citation_firstpage=e355; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=BMP signalling in skeletal development, disease and repair; citation_author=V.S. Salazar; citation_author=L.W. Gamer; citation_author=V. Rosen; citation_journal_title=Nat. Rev. Endocrinol.; citation_volume=12; citation_firstpage=203; citation_lastpage=221; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Effect of chronic chromolyn sodium therapy in a beautician with occupational asthma; citation_author=H.J. Schwartz; citation_journal_title=J.&nbsp;Occup. Med.; citation_volume=31; citation_firstpage=112; citation_lastpage=114; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Requirement for Pbx1 in skeletal patterning and programming chondrocyte proliferation and differentiation; citation_author=L. Selleri; citation_author=M.J. Depew; citation_author=Y. Jacobs; citation_author=S.K. Chanda; citation_author=K.Y. Tsang; citation_author=K.S.E. Cheah; citation_author=J.L.R. Rubenstein; citation_author=S. O’Gorman; citation_author=M.L. Cleary; citation_journal_title=Development; citation_volume=128; citation_firstpage=3543; citation_lastpage=3557; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=WISP3-dependent regulation of type II collagen and aggrecan production in chondrocytes; citation_author=M. Sen; citation_author=Y.-H. Cheng; citation_author=M.B. Goldring; citation_author=M.K. Lotz; citation_author=D.A. Carson; citation_journal_title=Arthritis Rheum.; citation_volume=50; citation_firstpage=488; citation_lastpage=497; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Multiple joint and skeletal patterning defects caused by single and double mutations in the mouse Gdf6 and Gdf5 genes; citation_author=S.H. Settle; citation_author=R.B. Rountree; citation_author=A. Sinha; citation_author=A. Thacker; citation_author=K. Higgins; citation_author=D.M. Kingsley; citation_journal_title=Dev. Biol.; citation_volume=254; citation_firstpage=116; citation_lastpage=130; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=MotifDb: An annotated collection of protein-DNA binding sequence motifs. R Package. Version 1; citation_author=P. Shannon; citation_author=M. Richards; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Kruppel-like factor 5 causes cartilage degradation through transactivation of matrix metalloproteinase 9; citation_author=Y. Shinoda; citation_author=N. Ogata; citation_author=A. Higashikawa; citation_author=I. Manabe; citation_author=T. Shindo; citation_author=T. Yamada; citation_author=F. Kugimiya; citation_author=T. Ikeda; citation_author=N. Kawamura; citation_author=Y. Kawasaki; citation_author=et al.; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=283; citation_firstpage=24682; citation_lastpage=24689; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Transcriptional enhancers: from properties to genome-wide predictions; citation_author=D. Shlyueva; citation_author=G. Stampfel; citation_author=A. Stark; citation_journal_title=Nat. Rev. Genet.; citation_volume=15; citation_firstpage=272; citation_lastpage=286; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The influence of the tibial slope and the size of the intercondylar notch on rupture of the anterior cruciate ligament; citation_author=B. Sonnery-Cottet; citation_author=P. Archbold; citation_author=T. Cucurulo; citation_author=J.-M. Fayard; citation_author=J. Bortolletto; citation_author=M. Thaunat; citation_author=T. Prost; citation_author=P. Chambat; citation_journal_title=J.&nbsp;Bone Joint Surg. Br.; citation_volume=93; citation_firstpage=1475; citation_lastpage=1478; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; citation_author=A. Stamatakis; citation_journal_title=Bioinformatics; citation_volume=22; citation_firstpage=2688; citation_lastpage=2690; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Severe osteoarthritis of the hand associates with common variants within the ALDH1A2 gene and with rare variants at 1p31; citation_author=U. Styrkarsdottir; citation_author=G. Thorleifsson; citation_author=H.T. Helgadottir; citation_author=N. Bomer; citation_author=S. Metrustry; citation_author=S. Bierma-Zeinstra; citation_author=A.M. Strijbosch; citation_author=E. Evangelou; citation_author=D. Hart; citation_author=M. Beekman; citation_author=TREAT-OA Consortium; citation_author=arcOGEN Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=498; citation_lastpage=502; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Whole-genome sequencing identifies rare genotypes in COMP and CHADL associated with high risk of hip osteoarthritis; citation_author=U. Styrkarsdottir; citation_author=H. Helgason; citation_author=A. Sigurdsson; citation_author=G.L. Norddahl; citation_author=A.B. Agustsdottir; citation_author=L.N. Reynard; citation_author=A. Villalvilla; citation_author=G.H. Halldorsson; citation_author=A. Jonasdottir; citation_author=A. Magnusdottir; citation_author=arcOGEN consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=801; citation_lastpage=805; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Meta-analysis of Icelandic and UK data sets identifies missense variants in SMO, IL11, COL11A1 and 13 more new loci associated with osteoarthritis; citation_author=U. Styrkarsdottir; citation_author=S.H. Lund; citation_author=G. Thorleifsson; citation_author=F. Zink; citation_author=O.A. Stefansson; citation_author=J.K. Sigurdsson; citation_author=K. Juliusson; citation_author=K. Bjarnadottir; citation_author=S. Sigurbjornsdottir; citation_author=S. Jonsson; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=1681; citation_lastpage=1687; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Identification of new therapeutic targets for osteoarthritis through genome-wide analyses of UK Biobank data; citation_author=I. Tachmazidou; citation_author=K. Hatzikotoulas; citation_author=L. Southam; citation_author=J. Esparza-Gordillo; citation_author=V. Haberland; citation_author=J. Zheng; citation_author=T. Johnson; citation_author=M. Koprulu; citation_author=E. Zengini; citation_author=J. Steinberg; citation_author=arcOGEN Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=230; citation_lastpage=236; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism; citation_author=F. Tajima; citation_journal_title=Genetics; citation_volume=123; citation_firstpage=585; citation_lastpage=595; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Ontogeny and phylogeny of femoro-tibial characters in humans and hominid fossils: functional influence and genetic determinism; citation_author=C. Tardieu; citation_journal_title=Am. J. Phys. Anthropol.; citation_volume=110; citation_firstpage=365; citation_lastpage=377; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=Development of the human hind limb and its importance for the evolution of bipedalism; citation_author=C. Tardieu; citation_journal_title=Evol. Anthropol.; citation_volume=19; citation_firstpage=174; citation_lastpage=186; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Reproducible genetic associations between candidate genes and clinical knee osteoarthritis in men and women; citation_author=A.M. Valdes; citation_author=M. Van Oene; citation_author=D.J. Hart; citation_author=G.L. Surdulescu; citation_author=J. Loughlin; citation_author=M. Doherty; citation_author=T.D. Spector; citation_journal_title=Arthritis Rheum.; citation_volume=54; citation_firstpage=533; citation_lastpage=539; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=The GDF5 rs143383 polymorphism is associated with osteoarthritis of the knee with genome-wide statistical significance; citation_author=A.M. Valdes; citation_author=E. Evangelou; citation_author=H.J.M. Kerkhof; citation_author=A. Tamm; citation_author=S.A. Doherty; citation_author=K. Kisand; citation_author=A. Tamm; citation_author=I. Kerna; citation_author=A. Uitterlinden; citation_author=A. Hofman; citation_author=et al.; citation_journal_title=Ann. Rheum. Dis.; citation_volume=70; citation_firstpage=873; citation_lastpage=875; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Comparing the human and chimpanzee genomes: searching for needles in a haystack; citation_author=A. Varki; citation_author=T.K. Altheide; citation_journal_title=Genome Res.; citation_volume=15; citation_firstpage=1746; citation_lastpage=1758; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=GenePaint.org: an atlas of gene expression patterns in the mouse embryo; citation_author=A. Visel; citation_author=C. Thaller; citation_author=G. Eichele; citation_journal_title=Nucleic Acids Res.; citation_volume=32; citation_firstpage=D552; citation_lastpage=D556; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Detecting natural selection in genomic data; citation_author=J.J. Vitti; citation_author=S.R. Grossman; citation_author=P.C. Sabeti; citation_journal_title=Annu. Rev. Genet.; citation_volume=47; citation_firstpage=97; citation_lastpage=120; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A map of recent positive selection in the human genome; citation_author=B.F. Voight; citation_author=S. Kudaravalli; citation_author=X. Wen; citation_author=J.K. Pritchard; citation_journal_title=PLoS Biol.; citation_volume=4; citation_firstpage=e72; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Bone Morphogenetic Protein (BMP) signaling in development and human diseases; citation_author=R.N. Wang; citation_author=J. Green; citation_author=Z. Wang; citation_author=Y. Deng; citation_author=M. Qiao; citation_author=M. Peabody; citation_author=Q. Zhang; citation_author=J. Ye; citation_author=Z. Yan; citation_author=S. Denduluri; citation_author=et al.; citation_journal_title=Genes Dis.; citation_volume=1; citation_firstpage=87; citation_lastpage=105; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=PITX1 promotes chondrogenesis and myogenesis in mouse hindlimbs through conserved regulatory targets; citation_author=J.S. Wang; citation_author=C.R. Infante; citation_author=S. Park; citation_author=D.B. Menke; citation_journal_title=Dev. Biol.; citation_volume=434; citation_firstpage=186; citation_lastpage=195; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants; citation_author=L.D. Ward; citation_author=M. Kellis; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=D930; citation_lastpage=D934; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Estimating F-statistics for the analysis of population structure; citation_author=B.S. Weir; citation_author=C.C. Cockerham; citation_journal_title=Evolution; citation_volume=38; citation_firstpage=1358; citation_lastpage=1370; citation_publication_date=1984; "><meta name="citation_reference" content="citation_title=Cis-regulatory elements: molecular mechanisms and evolutionary processes underlying divergence; citation_author=P.J. Wittkopp; citation_author=G. Kalay; citation_journal_title=Nat. Rev. Genet.; citation_volume=13; citation_firstpage=59; citation_lastpage=69; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=TGF-β and BMP signaling in osteoblast, skeletal development, and bone formation, homeostasis and disease; citation_author=M. Wu; citation_author=G. Chen; citation_author=Y.-P. Li; citation_journal_title=Bone Res.; citation_volume=4; citation_firstpage=16009; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Foxp2 regulates anatomical features that may be relevant for vocal behaviors and bipedal locomotion; citation_author=S. Xu; citation_author=P. Liu; citation_author=Y. Chen; citation_author=Y. Chen; citation_author=W. Zhang; citation_author=H. Zhao; citation_author=Y. Cao; citation_author=F. Wang; citation_author=N. Jiang; citation_author=S. Lin; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=8799; citation_lastpage=8804; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A unique stylopod patterning mechanism by Shox2-controlled osteogenesis; citation_author=W. Ye; citation_author=Y. Song; citation_author=Z. Huang; citation_author=M. Osterwalder; citation_author=A. Ljubojevic; citation_author=J. Xu; citation_author=B. Bobick; citation_author=S. Abassah-Oppong; citation_author=N. Ruan; citation_author=R. Shamby; citation_author=et al.; citation_journal_title=Development; citation_volume=143; citation_firstpage=2548; citation_lastpage=2560; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A comparative encyclopedia of DNA elements in the mouse genome; citation_author=F. Yue; citation_author=Y. Cheng; citation_author=A. Breschi; citation_author=J. Vierstra; citation_author=W. Wu; citation_author=T. Ryba; citation_author=R. Sandstrom; citation_author=Z. Ma; citation_author=C. Davis; citation_author=B.D. Pope; citation_author=Mouse ENCODE Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=515; citation_firstpage=355; citation_lastpage=364; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Homo sapiens-Specific Binding Site Variants within Brain Exclusive Enhancers Are Subject to Accelerated Divergence across Human Population; citation_author=R. Zehra; citation_author=A.A. Abbasi; citation_journal_title=Genome Biol. Evol.; citation_volume=10; citation_firstpage=956; citation_lastpage=966; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Regression Models for Count Data in {R}; citation_author=A. Zeileis; citation_author=C. Kleiber; citation_author=S. Jackman; citation_journal_title=J.&nbsp;Stat. Softw.; citation_volume=27; citation_firstpage=1; citation_lastpage=25; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis; citation_author=E. Zengini; citation_author=K. Hatzikotoulas; citation_author=I. Tachmazidou; citation_author=J. Steinberg; citation_author=F.P. Hartwig; citation_author=L. Southam; citation_author=S. Hackinger; citation_author=C.G. Boer; citation_author=U. Styrkarsdottir; citation_author=A. Gilly; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=549; citation_lastpage=558; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A genome-wide association study suggests that a locus within the ataxin 2 binding protein 1 gene is associated with hand osteoarthritis: the Treat-OA consortium; citation_author=G. Zhai; citation_author=J.B. van Meurs; citation_author=G. Livshits; citation_author=I. Meulenbelt; citation_author=A.M. Valdes; citation_author=N. Soranzo; citation_author=D. Hart; citation_author=F. Zhang; citation_author=B.S. Kato; citation_author=J.B. Richards; citation_author=et al.; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=46; citation_firstpage=614; citation_lastpage=616; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=TGF-β signal transduction pathways and osteoarthritis; citation_author=G. Zhai; citation_author=J. Doré; citation_author=P. Rahman; citation_journal_title=Rheumatol. Int.; citation_volume=35; citation_firstpage=1283; citation_lastpage=1292; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Non-coding genetic variants in human disease; citation_author=F. Zhang; citation_author=J.R. Lupski; citation_journal_title=Hum. Mol. Genet.; citation_volume=24; citation_number=R1; citation_firstpage=R102; citation_lastpage=R110; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y. Zhang; citation_author=T. Liu; citation_author=C.A. Meyer; citation_author=J. Eeckhoute; citation_author=D.S. Johnson; citation_author=B.E. Bernstein; citation_author=C. Nusbaum; citation_author=R.M. Myers; citation_author=M. Brown; citation_author=W. Li; citation_author=X.S. Liu; citation_journal_title=Genome Biol.; citation_volume=9; citation_firstpage=R137; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=CrossMap: a versatile tool for coordinate conversion between genome assemblies; citation_author=H. Zhao; citation_author=Z. Sun; citation_author=J. Wang; citation_author=H. Huang; citation_author=J.-P. Kocher; citation_author=L. Wang; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=1006; citation_lastpage=1007; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Foxp1/2/4 regulate endochondral ossification as a suppresser complex; citation_author=H. Zhao; citation_author=W. Zhou; citation_author=Z. Yao; citation_author=Y. Wan; citation_author=J. Cao; citation_author=L. Zhang; citation_author=J. Zhao; citation_author=H. Li; citation_author=R. Zhou; citation_author=B. Li; citation_author=et al.; citation_journal_title=Dev. Biol.; citation_volume=398; citation_firstpage=242; citation_lastpage=254; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis; citation_author=J. Zheng; citation_author=A.M. Erzurumluoglu; citation_author=B.L. Elsworth; citation_author=J.P. Kemp; citation_author=L. Howe; citation_author=P.C. Haycock; citation_author=G. Hemani; citation_author=K. Tansey; citation_author=C. Laurin; citation_author=B.S. Pourcain; citation_author=Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium; citation_author=et al.; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=272; citation_lastpage=279; citation_publication_date=2017; "><meta name="citation_abstract" content="<h2>Summary</h2><p>During human evolution, the knee adapted to the biomechanical demands of bipedalism by altering chondrocyte developmental programs. This adaptive process was likely not without deleterious consequences to health. Today, osteoarthritis occurs in 250 million people, with risk variants enriched in non-coding sequences near chondrocyte genes, loci that likely became optimized during knee evolution. We explore this relationship by epigenetically profiling joint chondrocytes, revealing ancient selection and recent constraint and drift on knee regulatory elements, which also overlap osteoarthritis variants that contribute to disease heritability by tending to modify constrained functional sequence. We propose a model whereby genetic violations to regulatory constraint, tolerated during knee development, lead to adult pathology. In support, we discover a causal enhancer variant (rs6060369) present in billions of people at a risk locus <i>(GDF5</i>-<i>UQCC1</i>), showing how it impacts mouse knee-shape and osteoarthritis. Overall, our methods link an evolutionarily novel aspect of human anatomy to its pathogenesis.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=7505&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)30234-8&amp;be=5592&amp;fe=7362&amp;dc=7109&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749586033676,%22n%22:0,%22f%22:3686,%22dn%22:3687,%22dne%22:3687,%22c%22:3687,%22s%22:3687,%22ce%22:3687,%22rq%22:3687,%22rp%22:4821,%22rpe%22:6927,%22dl%22:4826,%22di%22:7063,%22ds%22:7109,%22de%22:7126,%22dc%22:7361,%22l%22:7361,%22le%22:7396%7D,%22navigation%22:%7B%7D%7D&amp;fp=6180&amp;fcp=6180&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)30234-8"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)30234-8"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-802694" aria-expanded="false" aria-controls="login-links-802694" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-802694" id="login-links--802694" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-802694" data-opens="#links-dialog-802694" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-802694" aria-modal="true"><button data-closes="#links-dialog-802694" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2930234-8" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.02.057">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail long-title" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)30234-8" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0009-4"><span>Volume 181</span>, <span>Issue 2</span></a></span><span class="meta-panel__pages"><span>p362-381.e28</span></span><span class="meta-panel__onlineDate">April 16, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=2&amp;pii=S0092-8674%2819%29X0009-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Daniel</span> <span property="familyName">Richard</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Daniel</span> <span property="familyName">Richard</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Richard/Daniel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Zun</span> <span property="familyName">Liu</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Zun</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Zun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Jiaxue</span> <span property="familyName">Cao</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jiaxue</span> <span property="familyName">Cao</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Farm Animal Genetic Resources Exploration and Innovation Key Laboratory of Sichuan Province, Sichuan Agricultural University, Chengdu 611130, China</span></div></div><div class="core-author-link"><a href="/authored-by/Cao/Jiaxue" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Ata M.</span> <span property="familyName">Kiapour</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ata M.</span> <span property="familyName">Kiapour</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kiapour/Ata+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Jessica</span> <span property="familyName">Willen</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jessica</span> <span property="familyName">Willen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Willen/Jessica" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Siddharth</span> <span property="familyName">Yarlagadda</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Siddharth</span> <span property="familyName">Yarlagadda</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yarlagadda/Siddharth" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Evelyn</span> <span property="familyName">Jagoda</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Evelyn</span> <span property="familyName">Jagoda</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Jagoda/Evelyn" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Vijaya B.</span> <span property="familyName">Kolachalama</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Vijaya B.</span> <span property="familyName">Kolachalama</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Boston University School of Medicine, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Whitaker Cardiovascular Institute, Boston University School of Medicine, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Hariri Institute for Computing and Computational Science and Engineering, Boston University, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kolachalama/Vijaya+B" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Jakob T.</span> <span property="familyName">Sieker</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jakob T.</span> <span property="familyName">Sieker</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Pathology and Laboratory Medicine, Tufts Medical Center, Boston, MA 02111, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sieker/Jakob+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Gary H.</span> <span property="familyName">Chang</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gary H.</span> <span property="familyName">Chang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Boston University School of Medicine, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chang/Gary+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Pushpanathan</span> <span property="familyName">Muthuirulan</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Pushpanathan</span> <span property="familyName">Muthuirulan</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Muthuirulan/Pushpanathan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Mariel</span> <span property="familyName">Young</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mariel</span> <span property="familyName">Young</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Young/Mariel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Anand</span> <span property="familyName">Masson</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anand</span> <span property="familyName">Masson</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">McCaig Institute for Bone and Joint Health, University of Calgary, Calgary, AB T2N 1N4, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Masson/Anand" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Johannes</span> <span property="familyName">Konrad</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Johannes</span> <span property="familyName">Konrad</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Konrad/Johannes" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Shayan</span> <span property="familyName">Hosseinzadeh</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shayan</span> <span property="familyName">Hosseinzadeh</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Hosseinzadeh/Shayan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">David E.</span> <span property="familyName">Maridas</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David E.</span> <span property="familyName">Maridas</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Developmental Biology, Harvard School of Dental Medicine, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Maridas/David+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Vicki</span> <span property="familyName">Rosen</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Vicki</span> <span property="familyName">Rosen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Developmental Biology, Harvard School of Dental Medicine, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Rosen/Vicki" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Roman</span> <span property="familyName">Krawetz</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Roman</span> <span property="familyName">Krawetz</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">McCaig Institute for Bone and Joint Health, University of Calgary, Calgary, AB T2N 1N4, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Krawetz/Roman" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Neil</span> <span property="familyName">Roach</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Neil</span> <span property="familyName">Roach</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Roach/Neil" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Terence D.</span> <span property="familyName">Capellini</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Terence D.</span> <span property="familyName">Capellini</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3842-8478" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3842-8478</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:tcapellini@fas.harvard.edu" property="email" title="Send email to Terence D. Capellini">tcapellini@fas.harvard.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Capellini/Terence+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a role="doc-noteref">12</a></sup> <a id="eadd0010" href="mailto:tcapellini@fas.harvard.edu" property="email" title="Send email to Terence D. Capellini">tcapellini@fas.harvard.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Human Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Farm Animal Genetic Resources Exploration and Innovation Key Laboratory of Sichuan Province, Sichuan Agricultural University, Chengdu 611130, China</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Orthopaedic Surgery, Boston Children’s Hospital, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Medicine, Boston University School of Medicine, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Whitaker Cardiovascular Institute, Boston University School of Medicine, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Hariri Institute for Computing and Computational Science and Engineering, Boston University, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Pathology and Laboratory Medicine, Tufts Medical Center, Boston, MA 02111, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">McCaig Institute for Bone and Joint Health, University of Calgary, Calgary, AB T2N 1N4, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Developmental Biology, Harvard School of Dental Medicine, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received July 10, 2019</span>; <span>Revised December 10, 2019</span>; <span>Accepted February 26, 2020</span>; <span>Published online March 26, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.02.057" property="sameAs" target="_blank">10.1016/j.cell.2020.02.057 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420302348" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.02.057" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0009-4"><img src="/cms/asset/8e0cf1f8-74dd-4ff1-8cf9-225c94c2740a/cov200h.gif" alt="Cover Image - Cell, Volume 181, Issue 2"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2930234-8" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (9)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930234-8" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.02.057">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2930234-8" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.02.057&amp;pii=S0092-8674%2820%2930234-8" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2930234-8" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.02.057">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.02.057" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420302348&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420302348&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=181&amp;issue=2&amp;pii=S0092-8674%2819%29X0009-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30342-1" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30264-6" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (9)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Human/mouse long bone chondrocyte ATAC-seq reveals site-specific regulatory elements</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Ancient positive and purifying selection shape human knee regulation and morphology</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Modern human drift and pleiotropy violate regulatory constraint conferring OA risk</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Validation at <i>GDF5</i> reveals a single regulatory variant impacting both shape and OA</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">During human evolution, the knee adapted to the biomechanical demands of bipedalism by altering chondrocyte developmental programs. This adaptive process was likely not without deleterious consequences to health. Today, osteoarthritis occurs in 250 million people, with risk variants enriched in non-coding sequences near chondrocyte genes, loci that likely became optimized during knee evolution. We explore this relationship by epigenetically profiling joint chondrocytes, revealing ancient selection and recent constraint and drift on knee regulatory elements, which also overlap osteoarthritis variants that contribute to disease heritability by tending to modify constrained functional sequence. We propose a model whereby genetic violations to regulatory constraint, tolerated during knee development, lead to adult pathology. In support, we discover a causal enhancer variant (rs6060369) present in billions of people at a risk locus <i>(GDF5</i>-<i>UQCC1</i>), showing how it impacts mouse knee-shape and osteoarthritis. Overall, our methods link an evolutionarily novel aspect of human anatomy to its pathogenesis.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/f8d6c194-1b36-46bd-9681-bb02357f156a/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.02.057/asset/b1fa9844-fe2d-4264-bb31-09e29eb9fc83/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/f8d6c194-1b36-46bd-9681-bb02357f156a/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;osteoarthritis&quot;" alt="Search on this keyword" rel="nofollow">osteoarthritis</a></li><li><a href="/action/doSearch?AllField=&quot;knee&quot;" alt="Search on this keyword" rel="nofollow">knee</a></li><li><a href="/action/doSearch?AllField=&quot;gene+regulation&quot;" alt="Search on this keyword" rel="nofollow">gene regulation</a></li><li><a href="/action/doSearch?AllField=&quot;ATAC-seq&quot;" alt="Search on this keyword" rel="nofollow">ATAC-seq</a></li><li><a href="/action/doSearch?AllField=&quot;human+evolution&quot;" alt="Search on this keyword" rel="nofollow">human evolution</a></li><li><a href="/action/doSearch?AllField=&quot;natural+selection&quot;" alt="Search on this keyword" rel="nofollow">natural selection</a></li><li><a href="/action/doSearch?AllField=&quot;genetic+drift&quot;" alt="Search on this keyword" rel="nofollow">genetic drift</a></li><li><a href="/action/doSearch?AllField=&quot;GDF5&quot;" alt="Search on this keyword" rel="nofollow">GDF5</a></li><li><a href="/action/doSearch?AllField=&quot;mouse+model&quot;" alt="Search on this keyword" rel="nofollow">mouse model</a></li><li><a href="/action/doSearch?AllField=&quot;chondrocyte&quot;" alt="Search on this keyword" rel="nofollow">chondrocyte</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">A derived feature of humans and our hominin ancestors is a bipedal gait (<span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Darwin, 1888</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Darwin, C.</div><div><strong>The Descent of Man and Selection in Relation to Sex</strong></div><div>Murray, 1888</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.DarwinThe+Descent+of+Man+and+Selection+in+Relation+to+Sex1888Murray" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From the last common ancestor with chimpanzees, natural selection shaped the hominin hind limb to accommodate the biomechanical demands of bipedality, influencing human knee anatomy (<span class="dropBlock reference-citations"><a id="body-ref-sref127" href="#" role="doc-biblioref" data-xml-rid="bib94 bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127" aria-controls="sref127">Jungers, 1988; Morrison, 1970</a><div class="dropBlock__holder" data-db-target-of="sref127"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Jungers, W.L.</div><div><strong>Relative joint size and hominoid locomotor adaptations with implications for the evolution of hominid bipedalism</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 1988; <strong>17</strong>:247-265</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0047-2484(88)90056-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0047-2484%2888%2990056-5" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Morrison, J.B.</div><div><strong>The mechanics of the knee joint in relation to normal walking</strong></div><div><em>J.&nbsp;Biomech.</em> 1970; <strong>3</strong>:51-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0021-9290(70)90050-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0014698584" target="_blank" aria-label="Open Scopus in new tab">Scopus (764)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5521530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0021-9290%2870%2990050-3&amp;pmid=5521530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For example, the condyles of the distal femur expanded, dissipating the higher forces generated during bipedal walking/running. The proximal plateau and condyles of the tibia became more symmetrical, its epicondyles buttressed with bone mass, permitting more equal weight distribution and greater knee stabilization (<span class="dropBlock reference-citations"><a id="body-ref-sref114-1" href="#" role="doc-biblioref" data-xml-rid="bib13 bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-1" aria-controls="sref114-1">Bramble and Lieberman, 2004; Lovejoy, 2007</a><div class="dropBlock__holder" data-db-target-of="sref114-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Bramble, D.M. ∙ Lieberman, D.E.</div><div><strong>Endurance running and the evolution of Homo</strong></div><div><em>Nature.</em> 2004; <strong>432</strong>:345-352</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03052" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-9244222260" target="_blank" aria-label="Open Scopus in new tab">Scopus (892)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15549097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03052&amp;pmid=15549097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Lovejoy, C.O.</div><div><strong>The natural history of human gait and posture. Part 3. The knee</strong></div><div><em>Gait Posture.</em> 2007; <strong>25</strong>:325-341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gaitpost.2006.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33846904576" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16766186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gaitpost.2006.05.001&amp;pmid=16766186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These and other derived features were present by ∼1.6 million years ago in <i>Homo erectus</i>, and today, variation in human knees is reduced compared to earlier <i>Homo</i> and differs from patterns in African apes (<span class="dropBlock reference-citations"><a id="body-ref-sref177-1" href="#" role="doc-biblioref" data-xml-rid="bib58 bib114 bib177" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref177-1" aria-controls="sref177-1">Frelat et&nbsp;al., 2017; Lovejoy, 2007; Tardieu, 1999</a><div class="dropBlock__holder" data-db-target-of="sref177-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Frelat, M.A. ∙ Shaw, C.N. ∙ Sukhdeo, S. ...</div><div><strong>Evolution of the hominin knee and ankle</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 2017; <strong>108</strong>:147-160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jhevol.2017.03.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85019890304" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jhevol.2017.03.006&amp;pmid=28622927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Lovejoy, C.O.</div><div><strong>The natural history of human gait and posture. Part 3. The knee</strong></div><div><em>Gait Posture.</em> 2007; <strong>25</strong>:325-341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gaitpost.2006.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33846904576" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16766186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gaitpost.2006.05.001&amp;pmid=16766186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Tardieu, C.</div><div><strong>Ontogeny and phylogeny of femoro-tibial characters in humans and hominid fossils: functional influence and genetic determinism</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1999; <strong>110</strong>:365-377</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1096-8644(199911)110:3%3C365::AID-AJPA8%3E3.0.CO;2-T" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0032747147" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10516567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291096-8644%28199911%29110%3A3%253C365%3A%3AAID-AJPA8%253E3.0.CO%3B2-T&amp;pmid=10516567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0035" role="paragraph">Derived human traits have been shaped by selection and its targeting of early development (<span class="dropBlock reference-citations"><a id="body-ref-sref177-2" href="#" role="doc-biblioref" data-xml-rid="bib21 bib67 bib96 bib177" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref177-2" aria-controls="sref177-2">Capra et&nbsp;al., 2013; Gokhman et&nbsp;al., 2017; Kanton et&nbsp;al., 2019; Tardieu, 1999</a><div class="dropBlock__holder" data-db-target-of="sref177-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Capra, J.A. ∙ Erwin, G.D. ∙ McKinsey, G. ...</div><div><strong>Many human accelerated regions are developmental enhancers</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2013; <strong>368</strong>:20130025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84887496761" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24218637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0025&amp;pmid=24218637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Gokhman, D. ∙ Agranat-Tamir, L. ∙ Housman, G. ...</div><div><strong>Recent Regulatory Changes Shaped Human Facial and Vocal Anatomy</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/106955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F106955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Tardieu, C.</div><div><strong>Ontogeny and phylogeny of femoro-tibial characters in humans and hominid fossils: functional influence and genetic determinism</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1999; <strong>110</strong>:365-377</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1096-8644(199911)110:3%3C365::AID-AJPA8%3E3.0.CO;2-T" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0032747147" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10516567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291096-8644%28199911%29110%3A3%253C365%3A%3AAID-AJPA8%253E3.0.CO%3B2-T&amp;pmid=10516567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The development of knee features, many present by birth, is tied to the regulation of chondrocytes that prefigure joints (<span class="dropBlock reference-citations"><a id="body-ref-sref134" href="#" role="doc-biblioref" data-xml-rid="bib1 bib61 bib68 bib123 bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134" aria-controls="sref134">Andersen, 1961; Gardner and O’Rahilly, 1968; Gray and Gardner, 1950; Mérida-Velasco et&nbsp;al., 1997; O’Rahilly, 1951</a><div class="dropBlock__holder" data-db-target-of="sref134"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Andersen, H.</div><div><strong>Histochemical studies on the histogenesis of the knee joint and superior tibio-fibular joint in human foetuses</strong></div><div><em>Acta Anat. (Basel).</em> 1961; <strong>46</strong>:279-303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000141791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13861166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000141791&amp;pmid=13861166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Gardner, E. ∙ O’Rahilly, R.</div><div><strong>The early development of the knee joint in staged human embryos</strong></div><div><em>J.&nbsp;Anat.</em> 1968; <strong>102</strong>:289-299</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5643844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=5643844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Gray, D.J. ∙ Gardner, E.</div><div><strong>Prenatal development of the human knee and superior tibiofibular joints</strong></div><div><em>Am. J. Anat.</em> 1950; <strong>86</strong>:235-287</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/aja.1000860204" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15410671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Faja.1000860204&amp;pmid=15410671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Mérida-Velasco, J.A. ∙ Sánchez-Montesinos, I. ∙ Espín-Ferra, J. ...</div><div><strong>Development of the human knee joint</strong></div><div><em>Anat. Rec.</em> 1997; <strong>248</strong>:269-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-0185(199706)248:2%3C269::AID-AR14%3E3.0.CO;2-N" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0030974656" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9185993/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-0185%28199706%29248%3A2%253C269%3A%3AAID-AR14%253E3.0.CO%3B2-N&amp;pmid=9185993" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>O’Rahilly, R.</div><div><strong>The early prenatal development of the human knee joint</strong></div><div><em>J.&nbsp;Anat.</em> 1951; <strong>85</strong>:166-170</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14824030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=14824030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). During development, chondrocytes differentiate within the early knee to form femoral and tibial articular cartilage, while adjacent to the joint they arise to form condyle and plateau epiphyseal cartilages (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Decker et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Decker, R.S. ∙ Koyama, E. ∙ Pacifici, M.</div><div><strong>Genesis and morphogenesis of limb synovial joints and articular cartilage</strong></div><div><em>Matrix Biol.</em> 2014; <strong>39</strong>:5-10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.matbio.2014.08.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84908300461" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25172830/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.matbio.2014.08.006&amp;pmid=25172830" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These cartilages remain protected from ossification until postnatal life to respond to mechanical loading and hormonal signals during growth (<span class="dropBlock reference-citations"><a id="body-ref-sref178" href="#" role="doc-biblioref" data-xml-rid="bib178" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref178" aria-controls="sref178">Tardieu, 2010</a><div class="dropBlock__holder" data-db-target-of="sref178"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Tardieu, C.</div><div><strong>Development of the human hind limb and its importance for the evolution of bipedalism</strong></div><div><em>Evol. Anthropol.</em> 2010; <strong>19</strong>:174-186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/evan.20276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78049380999" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fevan.20276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Adaptive changes to these processes underlie anatomical features of the unique human knee. To this end, modifications to chondrocyte networks, in the form of regulatory changes, are the likely candidates as they typically are modular in effect (<span class="dropBlock reference-citations"><a id="body-ref-sref182" href="#" role="doc-biblioref" data-xml-rid="bib102 bib122 bib182" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref182" aria-controls="sref182">King and Wilson, 1975; McLean et&nbsp;al., 2011; Varki and Altheide, 2005</a><div class="dropBlock__holder" data-db-target-of="sref182"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>King, M.-C. ∙ Wilson, A.C.</div><div><strong>Evolution at two levels in humans and chimpanzees</strong></div><div><em>Science.</em> 1975; <strong>188</strong>:107-116</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1090005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1090005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1090005&amp;pmid=1090005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>McLean, C.Y. ∙ Reno, P.L. ∙ Pollen, A.A. ...</div><div><strong>Human-specific loss of regulatory DNA and the evolution of human-specific traits</strong></div><div><em>Nature.</em> 2011; <strong>471</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09774" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79952386187" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21390129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09774&amp;pmid=21390129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">170.</div><div>Varki, A. ∙ Altheide, T.K.</div><div><strong>Comparing the human and chimpanzee genomes: searching for needles in a haystack</strong></div><div><em>Genome Res.</em> 2005; <strong>15</strong>:1746-1758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.3737405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-28844466854" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16339373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.3737405&amp;pmid=16339373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we first investigate the evolutionary basis of human knee development by examining accessible/open chromatin sequences, regions often involved in gene regulation (<span class="dropBlock reference-citations"><a id="body-ref-sref167" href="#" role="doc-biblioref" data-xml-rid="bib167" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref167" aria-controls="sref167">Shlyueva et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref167"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">158.</div><div>Shlyueva, D. ∙ Stampfel, G. ∙ Stark, A.</div><div><strong>Transcriptional enhancers: from properties to genome-wide predictions</strong></div><div><em>Nat. Rev. Genet.</em> 2014; <strong>15</strong>:272-286</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg3682" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84899450857" target="_blank" aria-label="Open Scopus in new tab">Scopus (584)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24614317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg3682&amp;pmid=24614317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), via epigenomic profiling of long bone chondrocytes in human and mouse. We find evidence of the effects of ancient selection on knee-specific regulatory elements that likely underlie derived human morphology.</div><div id="p0040" role="paragraph">Natural selection undoubtedly shaped knee morphology but not without impacts to health. In the elderly today, there is a high prevalence of knee osteoarthritis (OA), a condition in which joint- and epiphyseal-derived tissues deteriorate. Knee OA risk includes non-genetic factors, such as joint mechanics (<span class="dropBlock reference-citations"><a id="body-ref-sref132-1" href="#" role="doc-biblioref" data-xml-rid="bib4 bib131 bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-1" aria-controls="sref132-1">Barr et&nbsp;al., 2016; Neogi and Felson, 2016; Neogi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref132-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Barr, A.J. ∙ Dube, B. ∙ Hensor, E.M.A. ...</div><div><strong>The relationship between three-dimensional knee MRI bone shape and total knee replacement-a case control study: data from the Osteoarthritis Initiative</strong></div><div><em>Rheumatology (Oxford).</em> 2016; <strong>55</strong>:1585-1593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/rheumatology/kew191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84995616558" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27185958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Frheumatology%2Fkew191&amp;pmid=27185958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Neogi, T. ∙ Felson, D.T.</div><div><strong>Osteoarthritis: Bone as an imaging biomarker and treatment target in OA</strong></div><div><em>Nat. Rev. Rheumatol.</em> 2016; <strong>12</strong>:503-504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrrheum.2016.113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84978174688" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27383914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrrheum.2016.113&amp;pmid=27383914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Neogi, T. ∙ Bowes, M.A. ∙ Niu, J. ...</div><div><strong>Magnetic resonance imaging-based three-dimensional bone shape of the knee predicts onset of knee osteoarthritis: data from the osteoarthritis initiative</strong></div><div><em>Arthritis Rheum.</em> 2013; <strong>65</strong>:2048-2058</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.37987" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881340844" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23650083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.37987&amp;pmid=23650083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), obesity (<span class="dropBlock reference-citations"><a id="body-ref-sref155" href="#" role="doc-biblioref" data-xml-rid="bib52 bib155" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref155" aria-controls="sref155">Felson et&nbsp;al., 1988; Reyes et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref155"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Felson, D.T. ∙ Anderson, J.J. ∙ Naimark, A. ...</div><div><strong>Obesity and knee osteoarthritis. The Framingham Study</strong></div><div><em>Ann. Intern. Med.</em> 1988; <strong>109</strong>:18-24</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7326/0003-4819-109-1-18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3377350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7326%2F0003-4819-109-1-18&amp;pmid=3377350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Reyes, C. ∙ Leyland, K.M. ∙ Peat, G. ...</div><div><strong>Association between overweight and obesity and risk of clinically diagnosed knee, hip, and hand osteoarthritis: a population-based cohort study</strong></div><div><em>Arthritis Rheumatol.</em> 2016; <strong>68</strong>:1869-1875</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.39707" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84979642437" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27059260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.39707&amp;pmid=27059260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), inflammation (<span class="dropBlock reference-citations"><a id="body-ref-sref156" href="#" role="doc-biblioref" data-xml-rid="bib84 bib156" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref156" aria-controls="sref156">Houard et&nbsp;al., 2013; Richette et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref156"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Houard, X. ∙ Goldring, M.B. ∙ Berenbaum, F.</div><div><strong>Homeostatic mechanisms in articular cartilage and role of inflammation in osteoarthritis</strong></div><div><em>Curr. Rheumatol. Rep.</em> 2013; <strong>15</strong>:375</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11926-013-0375-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84893782724" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24072604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11926-013-0375-6&amp;pmid=24072604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">148.</div><div>Richette, P. ∙ Poitou, C. ∙ Garnero, P. ...</div><div><strong>Benefits of massive weight loss on symptoms, systemic inflammation and cartilage turnover in obese patients with knee osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:139-144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.134015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78650676944" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20980288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.134015&amp;pmid=20980288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and longevity (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Berenbaum et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Berenbaum, F. ∙ Wallace, I.J. ∙ Lieberman, D.E. ...</div><div><strong>Modern-day environmental factors in the pathogenesis of osteoarthritis</strong></div><div><em>Nat. Rev. Rheumatol.</em> 2018; <strong>14</strong>:674-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41584-018-0073-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053529092" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30209413/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41584-018-0073-x&amp;pmid=30209413" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, knee OA risk is ∼40% heritable (<span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Loughlin, 2015</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Loughlin, J.</div><div><strong>Genetic contribution to osteoarthritis development: current state of evidence</strong></div><div><em>Curr. Opin. Rheumatol.</em> 2015; <strong>27</strong>:284-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/BOR.0000000000000171" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84926329234" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25775188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FBOR.0000000000000171&amp;pmid=25775188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a component that interacts with these modern conditions. Genome-wide association studies (GWAS) have revealed over 80 OA loci with ∼95% of risk variants present in non-coding sequences, enriched near genes involved in chondrocyte and bone development (<span class="dropBlock reference-citations"><a id="body-ref-sref199-1" href="#" role="doc-biblioref" data-xml-rid="bib103 bib173 bib175 bib199" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref199-1" aria-controls="sref199-1">Klein et&nbsp;al., 2019; Styrkarsdottir et&nbsp;al., 2018; Tachmazidou et&nbsp;al., 2019; Zengini et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref199-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Klein, J.C. ∙ Keith, A. ∙ Rice, S.J. ...</div><div><strong>Functional testing of thousands of osteoarthritis-associated variants for regulatory activity</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10439-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85066610023" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164647/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10439-y&amp;pmid=31164647" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Styrkarsdottir, U. ∙ Lund, S.H. ∙ Thorleifsson, G. ...</div><div><strong>Meta-analysis of Icelandic and UK data sets identifies missense variants in SMO, IL11, COL11A1 and 13 more new loci associated with osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1681-1687</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0247-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85055988857" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30374069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0247-0&amp;pmid=30374069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>Tachmazidou, I. ∙ Hatzikotoulas, K. ∙ Southam, L. ..., arcOGEN Consortium</div><div><strong>Identification of new therapeutic targets for osteoarthritis through genome-wide analyses of UK Biobank data</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:230-236</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0327-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85060491699" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30664745/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0327-1&amp;pmid=30664745" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">185.</div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These findings suggest a key role for knee chondrocyte regulatory elements in mediating OA risk. Here, we build on our evolutionary analyses and use chondrocyte chromatin datasets to study how regulatory variation, shaped by ancient selection but also more recent evolutionary forces (genetic drift and antagonistic pleiotropy), influence OA risk. We develop a model of how nucleotide changes, occurring within evolutionarily constrained developmental regulatory sequences, violate this constraint and underlie genetic OA risk, and find evidence using at-risk patient data and functional validation studies in human chondrocytes and the mouse model. Our methods link a derived aspect of human anatomy to its pathogenesis, an approach applicable to other features to explore disease links.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Epigenetic Profiling of Long Bone Chondrocytes</h3><div id="p0045" role="paragraph">We first used the assay for transposase-accessible chromatin with sequencing (ATAC-seq) (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to profile open chromatin regions (hereafter, “elements”) in chondrocytes from E15.5 mouse and stage-matched E59 human proximal/distal femur and tibia samples (<a id="crosref0190" href="#fig1">Figure&nbsp;1</a>A; <a id="crosref0195" href="#mmc1">Table S1</a>; <a id="crosref0200" href="#sec-4">STAR Methods</a>). For the mouse, we cataloged a “general knee set” comprising elements from the distal femur and proximal tibia. This set was filtered for shared proximal femur, distal tibia, and embryonic brain elements to make a “knee-specific set” (<a id="crosref0205" href="#mmc1">Table S1</a>), consisting of elements unique to the distal femur (“DF-specific”), proximal tibia (“PT-specific”), and both knee components (“knee-common [KC]-specific”). An identical approach was used on homologous humeri and radii, and generated “elbow-specific” sets show minimal overlap with knee-specific sets (<a id="crosref0210" href="#mmc2">Table S2</a>), highlighting each sets’ anatomical specificity. Comparing datasets to those using fluorescence-activated cell sorting (FACS) on <i>Col2a1</i>-eCFP-reporter mice, we found strong and significant overlap (<a id="crosref0215" href="#mmc2">Table S2</a>), indicating that gross dissection identifies chondrocyte cell populations typically marked by COL2A1. Given the lack of reliable FACs markers to isolate chondrocytes from procured human tissues, this helps ensure the reliability of dissection on rare human samples. We thus generated ATAC-seq sets on E59 human long bones using the same pipelines, and intersected human and mouse elements from equivalent tissues (e.g., distal-femur), finding a significant degree of overlap (<a id="crosref0220" href="#mmc1">Tables S1</a> and <a id="crosref0225" href="#mmc2">S2</a>). Overlapping mouse and human element sets show substantial one-to-one orthology (<a id="crosref0230" href="#mmc1">Table S1</a>). Although there is regulatory divergence between species (<span class="dropBlock reference-citations"><a id="body-ref-sref196" href="#" role="doc-biblioref" data-xml-rid="bib196" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref196" aria-controls="sref196">Yue et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref196"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">182.</div><div>Yue, F. ∙ Cheng, Y. ∙ Breschi, A. ..., Mouse ENCODE Consortium</div><div><strong>A comparative encyclopedia of DNA elements in the mouse genome</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:355-364</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13992" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84911462077" target="_blank" aria-label="Open Scopus in new tab">Scopus (718)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409824/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13992&amp;pmid=25409824" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), this overlap is sizable despite disparate collection sites/protocols and uncontrollable differences in time-points. Mouse and overlapping E59 human elements intersect with biological signals related to chondrocytes and skeletogenesis using GREAT (<span class="dropBlock reference-citations"><a id="body-ref-sref121-1" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-1" aria-controls="sref121-1">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref121-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (1686)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0245" href="#mmc1">Table S1</a>). We found significant overlap with regulatory activity in other mouse (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Guo et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Guo, M. ∙ Liu, Z. ∙ Willen, J. ...</div><div><strong>Epigenetic profiling of growth plate chondrocytes sheds insight into regulatory genetic variation influencing height</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e29329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.29329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040906101" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29205154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.29329&amp;pmid=29205154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and human E47 limb samples (<span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Cotney et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cotney, J. ∙ Leng, J. ∙ Yin, J. ...</div><div><strong>The evolution of lineage-specific regulatory activities in the human embryonic limb</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:185-196</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84879911829" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23827682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.05.056&amp;pmid=23827682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and bone marrow-derived chondrocytes (BMDCs) (<span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">ENCODE Project Consortium, 2012</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>ENCODE Project Consortium</div><div><strong>An integrated encyclopedia of DNA elements in the human genome</strong></div><div><em>Nature.</em> 2012; <strong>489</strong>:57-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11247" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865790047" target="_blank" aria-label="Open Scopus in new tab">Scopus (8320)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955616/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11247&amp;pmid=22955616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0265" href="#mmc2">Table S2</a>). We further observed enrichments for motifs similar to chondrogenesis-associated transcription factors (TFs), which significantly overlap with chromatin immunoprecipitation sequencing (ChIP-seq) datasets (<a id="crosref0270" href="#mmc2">Table S2</a>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/39fef6bf-4b87-4a1d-a8bc-f753fad2f1f3/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.02.057/asset/fd526227-1668-44e6-9757-ec0811af53e2/main.assets/gr1.jpg" height="699" width="750" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/39fef6bf-4b87-4a1d-a8bc-f753fad2f1f3/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Sequence Features of Chondrocyte Epigenetic Profiles</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Diagram of ATAC-seq tissues. Numbers indicate replicate-consolidated peak calls before filtering with brain data. Inset: genomic distribution of lifted-over DF (top) and PT (bottom) elements.</div><div id="fspara0020" role="paragraph">(B) Per-bp conservation scores for 20 primates (phyloP20ways) aggregated for ATAC-seq sets and compared via Wilcoxon rank-sum test.</div><div id="fspara0025" role="paragraph">(C) Overlaps of human acceleration regions and ATAC-seq sets. KC, knee-common; GM, GM12878; DF, distal femur-specific; PT, proximal tibia-specific; EB, embryonic brain. Overlaps shown relative to set size (per bp of sequence) for background (gray) and target (colored) sets.</div><div id="fspara0030" role="paragraph">(D) Relationship between human-chimp sequence identity (red line, right y axis) and phyloP20ways score averaged over an ATAC-seq peak (blue line, left y axis) for DF-specific elements. Dashed curve and horizontal lines indicate average phyloP20ways scores for random background region set. Shaded regions reflect low/high %ID regions used in GREAT. Significance codes: not significant (ns),&nbsp;&lt; 0.05 (<sup>∗</sup>),&nbsp;&lt; 0.01 (<sup>∗∗</sup>),&nbsp;&lt; 1e−5 (<sup>∗∗∗</sup>).</div><div id="fspara0035" role="paragraph">See also <a id="crosref2185" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref2190" href="#mmc1">Tables S1</a>, <a id="crosref2195" href="#mmc2">S2</a>, and <a id="crosref2200" href="#mmc4">S4</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Inter-species Sequence Variation in Element Sets</h3><div id="p0050" role="paragraph">To understand how accessible elements involved in human knee development have been modified evolutionarily, we broadly examined how primates with diverse locomotor repertoires (<span class="dropBlock reference-citations"><a id="body-ref-sref141" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141" aria-controls="sref141">Polk et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref141"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Polk, J.D. ∙ Williams, S.A. ∙ Peterson, J.V.</div><div><strong>Body size and joint posture in primates</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2009; <strong>140</strong>:359-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.21083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-70349149919" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19425104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.21083&amp;pmid=19425104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) modified these elements. We examined mouse and overlapping E59 human knee element sets for evidence of sequence evolution using phyloP (<span class="dropBlock reference-citations"><a id="body-ref-sref144-1" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144-1" aria-controls="sref144-1">Pollard et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref144-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Pollard, K.S. ∙ Hubisz, M.J. ∙ Rosenbloom, K.R. ...</div><div><strong>Detection of nonneutral substitution rates on mammalian phylogenies</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:110-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.097857.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-74949092081" target="_blank" aria-label="Open Scopus in new tab">Scopus (968)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19858363/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.097857.109&amp;pmid=19858363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a measure of nucleotide conservation/acceleration. Across primates, knee elements had elevated conservation, with strongest conservation toward each element’s center (<a id="crosref0285" href="#figs1">Figure&nbsp;S1</a>), likely due to conserved TF sites (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). When comparing DF-, PT-, and KC-specific sets, conservation was highest in KC-specific elements (i.e., those that are likely pleiotropic across the knee) (<a id="crosref0295" href="#fig1">Figure&nbsp;1</a>B; <a id="crosref0300" href="#mmc2">Table S2</a>). Conservation was reduced in elements unique to each bone’s end, especially for the proximal tibia. These patterns were also observed for overlapping E59 human elements (<a id="crosref0305" href="#mmc2">Table S2</a>).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/b7e0a7a9-129f-4d00-8590-4bad837b64b1/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.02.057/asset/ef8cf5f8-d12b-4722-8305-a1ef22267252/main.assets/figs1.jpg" height="503" width="752" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/b7e0a7a9-129f-4d00-8590-4bad837b64b1/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Related to <a id="crosref2290" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0260" role="paragraph">(A-C): phyloP scores (phyloP20ways) were averaged per-bp across all specific-knee ATAC-seq regions in a peak set and plotted in respective colors as a function of distance along a 1kb window centered on the peak middle; a random background distribution of conservation scores is indicated in gray (see <a id="crosref2295" href="#sec-4">STAR Methods</a>). (A) Specific Distal Femur (B) Specific Proximal Tibia (C) Specific Knee-common. (D-E): HAR enrichment gene loci screenshots. (D) A general distal-femur element overlaps a human-accelerated region (HAR) (<span class="dropBlock reference-citations"><a id="body-ref-sref145-1" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-1" aria-controls="sref145-1">Prabhakar et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref145-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Prabhakar, S. ∙ Noonan, J.P. ∙ Pääbo, S. ...</div><div><strong>Accelerated evolution of conserved noncoding sequences in humans</strong></div><div><em>Science.</em> 2006; <strong>314</strong>:786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1130738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750737061" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17082449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1130738&amp;pmid=17082449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) upstream of <i>BMP2</i>, a TGF-β family member with a well-established role in chondrogenic differentiation, that is expressed in the knee (<span class="dropBlock reference-citations"><a id="body-ref-sref157" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157" aria-controls="sref157">Rosen, 2009</a><div class="dropBlock__holder" data-db-target-of="sref157"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">149.</div><div>Rosen, V.</div><div><strong>BMP2 signaling in bone development and repair</strong></div><div><em>Cytokine Growth Factor Rev.</em> 2009; <strong>20</strong>:475-480</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.cytogfr.2009.10.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-70649094222" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19892583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cytogfr.2009.10.018&amp;pmid=19892583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and which been linked to osteoarthritis (<span class="dropBlock reference-citations"><a id="body-ref-sref179" href="#" role="doc-biblioref" data-xml-rid="bib60 bib179" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref179" aria-controls="sref179">Gamer et&nbsp;al., 2018; Valdes et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref179"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Gamer, L.W. ∙ Pregizer, S. ∙ Gamer, J. ...</div><div><strong>The Role of Bmp2 in the Maturation and Maintenance of the Murine Knee Joint</strong></div><div><em>J.&nbsp;Bone Miner. Res.</em> 2018; <strong>33</strong>:1708-1717</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jbmr.3441" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85052506820" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29665134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjbmr.3441&amp;pmid=29665134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">168.</div><div>Valdes, A.M. ∙ Van Oene, M. ∙ Hart, D.J. ...</div><div><strong>Reproducible genetic associations between candidate genes and clinical knee osteoarthritis in men and women</strong></div><div><em>Arthritis Rheum.</em> 2006; <strong>54</strong>:533-539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.21621" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-32444433647" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16453284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.21621&amp;pmid=16453284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ATAC-seq regions (“General DF”), ENCODE DNase hypersensitivity sites and phyloP100ways conservation tracks are shown on bottom. (E) A proximal-tibia-specific element overlaps an accelerated region (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Bird et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bird, C.P. ∙ Stranger, B.E. ∙ Liu, M. ...</div><div><strong>Fast-evolving noncoding sequences in the human genome</strong></div><div><em>Genome Biol.</em> 2007; <strong>8</strong>:R118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2007-8-6-r118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049157173" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17578567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2007-8-6-r118&amp;pmid=17578567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) intronic to <i>IQGAP1,</i> a protein complex scaffold (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Hedman et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Hedman, A.C. ∙ Smith, J.M. ∙ Sacks, D.B.</div><div><strong>The biology of IQGAP proteins: beyond the cytoskeleton</strong></div><div><em>EMBO Rep.</em> 2015; <strong>16</strong>:427-446</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embr.201439834" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84926252376" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembr.201439834&amp;pmid=25722290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) expressed in embryonic mouse cartilage (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Cupit et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Cupit, L.D. ∙ Schmidt, V.A. ∙ Miller, F. ...</div><div><strong>Distinct PAR/IQGAP expression patterns during murine development: implications for thrombin-associated cytoskeletal reorganization</strong></div><div><em>Mamm. Genome.</em> 2004; <strong>15</strong>:618-629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00335-004-2370-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-3943058758" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15457341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00335-004-2370-8&amp;pmid=15457341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) which impacts skeletal development, particularly in the tibia (<span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Dickinson et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Dickinson, M.E. ∙ Flenniken, A.M. ∙ Ji, X. ..., International Mouse Phenotyping Consortium, Jackson Laboratory, Infrastructure Nationale PHENOMIN, Institut Clinique de la Souris (ICS), Charles River Laboratories, MRC Harwell, Toronto Centre for Phenogenomics, Wellcome Trust Sanger Institute, RIKEN BioResource Center</div><div><strong>High-throughput discovery of novel developmental phenotypes</strong></div><div><em>Nature.</em> 2016; <strong>537</strong>:508-514</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84989204804" target="_blank" aria-label="Open Scopus in new tab">Scopus (309)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19356&amp;pmid=27626380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Format as in (D), with ATAC-seq track “PT” shown.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">We next examined signatures of evolution shaping the human knee. Knee sets were intersected with human sequences exhibiting accelerated nucleotide changes (i.e., human accelerated regions) relative to chimps and other apes, often indicative of positive selection (<span class="dropBlock reference-citations"><a id="body-ref-sref145-2" href="#" role="doc-biblioref" data-xml-rid="bib10 bib17 bib65 bib142 bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-2" aria-controls="sref145-2">Bird et&nbsp;al., 2007; Bush and Lahn, 2008; Gittelman et&nbsp;al., 2015; Pollard et&nbsp;al., 2006a; Prabhakar et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref145-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bird, C.P. ∙ Stranger, B.E. ∙ Liu, M. ...</div><div><strong>Fast-evolving noncoding sequences in the human genome</strong></div><div><em>Genome Biol.</em> 2007; <strong>8</strong>:R118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2007-8-6-r118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049157173" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17578567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2007-8-6-r118&amp;pmid=17578567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Bush, E.C. ∙ Lahn, B.T.</div><div><strong>A genome-wide screen for noncoding elements important in primate evolution</strong></div><div><em>BMC Evol. Biol.</em> 2008; <strong>8</strong>:17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-8-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049088628" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18215302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-8-17&amp;pmid=18215302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Gittelman, R.M. ∙ Hun, E. ∙ Ay, F. ...</div><div><strong>Comprehensive identification and analysis of human accelerated regulatory DNA</strong></div><div><em>Genome Res.</em> 2015; <strong>25</strong>:1245-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.192591.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84937476478" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.192591.115&amp;pmid=26104583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ King, B. ...</div><div><strong>Forces shaping the fastest evolving regions in the human genome</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e168</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-35648988566" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17040131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020168&amp;pmid=17040131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Prabhakar, S. ∙ Noonan, J.P. ∙ Pääbo, S. ...</div><div><strong>Accelerated evolution of conserved noncoding sequences in humans</strong></div><div><em>Science.</em> 2006; <strong>314</strong>:786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1130738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750737061" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17082449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1130738&amp;pmid=17082449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Knee-specific elements significantly overlapped these regions (<a id="crosref0310" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0315" href="#mmc4">Table S4</a>), largely recapitulated using overlapping E59 human elements, despite smaller set sizes. This suggests that ancient positive selection acted on knee elements. Mouse brain elements, mapped to their human orthologs, also enrich for overlaps, as reported for the human brain (<span class="dropBlock reference-citations"><a id="body-ref-sref143" href="#" role="doc-biblioref" data-xml-rid="bib142 bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143" aria-controls="sref143">Pollard et&nbsp;al., 2006a, 2006b</a><div class="dropBlock__holder" data-db-target-of="sref143"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ King, B. ...</div><div><strong>Forces shaping the fastest evolving regions in the human genome</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e168</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-35648988566" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17040131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020168&amp;pmid=17040131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ Lambert, N. ...</div><div><strong>An RNA gene expressed during cortical development evolved rapidly in humans</strong></div><div><em>Nature.</em> 2006; <strong>443</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33748675254" target="_blank" aria-label="Open Scopus in new tab">Scopus (626)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16915236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05113&amp;pmid=16915236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while enrichment was not seen for B-lymphocyte elements. When examining <i>en masse</i> accelerated sequences in knee-specific elements using GREAT, we identified enrichments for “ossification,” “skeletal phenotypes,” and “limb developmental expression” (<a id="crosref0320" href="#mmc4">Table S4</a>). Two example loci are shown in <a id="crosref0325" href="#figs1">Figure&nbsp;S1</a>.</div><div id="p0060" role="paragraph">We next considered sequence change between humans and chimps. Nucleotide similarity (%ID) positively tracked with cross-primate sequence conservation for each element set (<a id="crosref0330" href="#fig1">Figure&nbsp;1</a>D; <a id="crosref0335" href="#mmc2">Table S2</a>). To understand the biological significance of different levels of human-chimp %ID, we ran GREAT on conserved sequences (i.e., third quartile of %ID) from DF- and PT-specific sets (<a id="crosref0340" href="#mmc2">Table S2</a>) and saw enrichments for “regulation of ossification,” “skeletal system development,” and “chondrocyte differentiation.” Because divergence in morphology can result from sequence changes in existing regulatory switches (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Cotney et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cotney, J. ∙ Leng, J. ∙ Yin, J. ...</div><div><strong>The evolution of lineage-specific regulatory activities in the human embryonic limb</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:185-196</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84879911829" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23827682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.05.056&amp;pmid=23827682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we ran GREAT on first quartile %ID sequences, which tended to retain conservation in primates (<a id="crosref0350" href="#fig1">Figure&nbsp;1</a>D, left shaded area), and found enrichments for “mild short stature” and “negative regulation of ossification,” among other terms. Weaker/insignificant enrichments were seen in top/bottom 10% of sequences, with reduced strength due to smaller set sizes and the decreased power of GREAT to find enrichments (<a id="crosref0355" href="#mmc2">Table S2</a>).</div><div id="p0065" role="paragraph">Given that human-specific enhancer mutations can impact development (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Boyd et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Boyd, J.L. ∙ Skove, S.L. ∙ Rouanet, J.P. ...</div><div><strong>Human-chimpanzee differences in a FZD8 enhancer alter cell-cycle dynamics in the developing neocortex</strong></div><div><em>Curr. Biol.</em> 2015; <strong>25</strong>:772-779</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84926420296" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25702574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2015.01.041&amp;pmid=25702574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we considered the potential for base-pair alterations (i.e., chimp-human substitutions) in knee elements to alter sequence motifs, as shown for brain enhancers (<span class="dropBlock reference-citations"><a id="body-ref-sref197" href="#" role="doc-biblioref" data-xml-rid="bib197" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref197" aria-controls="sref197">Zehra and Abbasi, 2018</a><div class="dropBlock__holder" data-db-target-of="sref197"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">183.</div><div>Zehra, R. ∙ Abbasi, A.A.</div><div><strong>Homo sapiens-Specific Binding Site Variants within Brain Exclusive Enhancers Are Subject to Accelerated Divergence across Human Population</strong></div><div><em>Genome Biol. Evol.</em> 2018; <strong>10</strong>:956-966</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/gbe/evy052" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85054362503" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgbe%2Fevy052&amp;pmid=29608725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Identified motifs in knee sets (<a id="crosref0370" href="#mmc1">Table S3</a>) were combined with those of known chondrogenesis-related TFs (<span class="dropBlock reference-citations"><a id="body-ref-sref111-1" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-1" aria-controls="sref111-1">Liu et&nbsp;al., 2017a</a><div class="dropBlock__holder" data-db-target-of="sref111-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Liu, C.-F. ∙ Samsa, W.E. ∙ Zhou, G. ...</div><div><strong>Transcriptional control of chondrocyte specification and differentiation</strong></div><div><em>Semin. Cell Dev. Biol.</em> 2017; <strong>62</strong>:34-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.semcdb.2016.10.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85006055004" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27771362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.semcdb.2016.10.004&amp;pmid=27771362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and used to identify the biased occurrence of alterations predicted to modify certain motifs (<a id="crosref0375" href="#fig2">Figure&nbsp;2</a>A; <a id="crosref0380" href="#mmc3">Table S3</a>; <a id="crosref0385" href="#sec-4">STAR Methods</a>). Across knee-specific sets for several chondrocyte TF motifs there is a strong tendency for protection from modification (biased against possessing substitutions). For example, putative sequence motifs (i.e., similar to a known TF motif) for FOXP1/FOXP2, factors important for knee morphology (<span class="dropBlock reference-citations"><a id="body-ref-sref205" href="#" role="doc-biblioref" data-xml-rid="bib193 bib205" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref205" aria-controls="sref205">Xu et&nbsp;al., 2018; Zhao et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref205"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">180.</div><div>Xu, S. ∙ Liu, P. ∙ Chen, Y. ...</div><div><strong>Foxp2 regulates anatomical features that may be relevant for vocal behaviors and bipedal locomotion</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:8799-8804</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1721820115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053629492" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30104377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1721820115&amp;pmid=30104377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">191.</div><div>Zhao, H. ∙ Zhou, W. ∙ Yao, Z. ...</div><div><strong>Foxp1/2/4 regulate endochondral ossification as a suppresser complex</strong></div><div><em>Dev. Biol.</em> 2015; <strong>398</strong>:242-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2014.12.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84922711705" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25527076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2014.12.007&amp;pmid=25527076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), were biased against disruptive changes (<a id="crosref0390" href="#fig2">Figure&nbsp;2</a>A, left). Human FOXP1/FOXP2 ChIP-seq data also shows strong overlap with knee-specific elements (<a id="crosref0395" href="#mmc2">Table S2</a>). Conversely, we observed alterations biasing toward TF motifs (<a id="crosref0400" href="#mmc3">Table S3</a>). For example, predicted motifs for KLF5, involved in chondrogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref166" href="#" role="doc-biblioref" data-xml-rid="bib166" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref166" aria-controls="sref166">Shinoda et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref166"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">157.</div><div>Shinoda, Y. ∙ Ogata, N. ∙ Higashikawa, A. ...</div><div><strong>Kruppel-like factor 5 causes cartilage degradation through transactivation of matrix metalloproteinase 9</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2008; <strong>283</strong>:24682-24689</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M709857200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-54049154742" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18617520/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M709857200&amp;pmid=18617520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), displayed such biases in the DF-specific set (<a id="crosref0410" href="#fig2">Figure&nbsp;2</a>A, right), which also significantly overlap (p&nbsp;&lt; 0.05) KLF5 binding data (<a id="crosref0415" href="#mmc2">Tables S2</a> and <a id="crosref0420" href="#mmc3">S3</a>) (<a id="crosref0425" href="#fig2">Figure&nbsp;2</a>B for an example). We also observed in knee-specific sets a bias for alterations at predicted CTCF motifs, whose loss-of-function results in skeletal defects and spontaneous OA (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Hijazi, 2018</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Hijazi, A.</div><div><strong>Investigating the role of CCCTC-binding factor in osteoarthritis pathogenesis</strong></div><div><em>Osteoarthritis Cartilage.</em> 2018; <strong>26</strong>:S154</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2018.02.332&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2018.02.332&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29222056/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2018.02.332&amp;pmid=29222056" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Knee-specific sets also significantly overlap with CTCF binding data (<a id="crosref0435" href="#mmc2">Table S2</a>) (<a id="crosref0440" href="#fig2">Figure&nbsp;2</a>C for an example).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/26e99865-b0f5-4e6f-8b2a-d71427a405a9/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.02.057/asset/d24d3d98-6314-42c0-8c35-cd637ae08718/main.assets/gr2.jpg" height="939" width="482" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/26e99865-b0f5-4e6f-8b2a-d71427a405a9/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Sequence Modification of Putative Regulatory Elements</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0045" role="paragraph">(A) Histograms showing depletion and enrichment of human-chimp base pair alterations intersecting predicted FOXP1 and KLF5 TF motifs, respectively, in DF-specific elements relative to randomized sets (p &lt; 0.01). Counts of altered bases in background (gray) and element (red) sets shown as a fraction of total set size. See <a id="crosref2205" href="#mmc3">Table S3</a>.</div><div id="fspara0050" role="paragraph">(B) KLF5 motif example in a DF-specific element intronic to <i>PBX1</i>, a TF regulating HOX expression and endochondral ossification (<span class="dropBlock reference-citations"><a id="body-ref-sref162" href="#" role="doc-biblioref" data-xml-rid="bib19 bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref162" aria-controls="sref162">Capellini et&nbsp;al., 2006; Selleri et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref162"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Capellini, T.D. ∙ Di Giacomo, G. ∙ Salsi, V. ...</div><div><strong>Pbx1/Pbx2 requirement for distal limb patterning is mediated by the hierarchical control of Hox gene spatial distribution and Shh expression</strong></div><div><em>Development.</em> 2006; <strong>133</strong>:2263-2273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02395" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33745584051" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16672333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02395&amp;pmid=16672333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Selleri, L. ∙ Depew, M.J. ∙ Jacobs, Y. ...</div><div><strong>Requirement for Pbx1 in skeletal patterning and programming chondrocyte proliferation and differentiation</strong></div><div><em>Development.</em> 2001; <strong>128</strong>:3543-3557</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.128.18.3543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11566859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.128.18.3543&amp;pmid=11566859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Predicted disruption by a human-derived T/G nucleotide change shown. A region of acceleration in a knee element identified upstream of <i>PBX1</i> (<a id="crosref2210" href="#mmc3">Table S3</a>). Red shading indicates altered base relative to motif logo, blue indicates genomic position of sequence. ATAC-seq regions and phyloPways conservation tracks, UCSC sequence alignment shown.</div><div id="fspara0055" role="paragraph">(C) Example of a human-chimp alteration predicted to improve a CTCF motif within a PT-specific element intronic to <i>NCOR2</i>, involved in skeletal biology (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Blake et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Blake, J.A. ∙ Eppig, J.T. ∙ Kadin, J.A. ..., the Mouse Genome Database Group</div><div><strong>Mouse Genome Database (MGD)-2017: community knowledge resource for the laboratory mouse</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D723-D729</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1040" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85016041593" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1040&amp;pmid=27899570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Format similar to (B).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-3"><h3>Examination of Intra-species Sequence Diversity in Element Sets</h3><div id="p0070" role="paragraph">As selection also shapes within-species sequence variation (<span class="dropBlock reference-citations"><a id="body-ref-sref184" href="#" role="doc-biblioref" data-xml-rid="bib184" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref184" aria-controls="sref184">Vitti et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref184"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">172.</div><div>Vitti, J.J. ∙ Grossman, S.R. ∙ Sabeti, P.C.</div><div><strong>Detecting natural selection in genomic data</strong></div><div><em>Annu. Rev. Genet.</em> 2013; <strong>47</strong>:97-120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-genet-111212-133526" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888582669" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24274750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-genet-111212-133526&amp;pmid=24274750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we acquired variant data for humans (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), chimps and gorillas (<span class="dropBlock reference-citations"><a id="body-ref-sref147-1" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147-1" aria-controls="sref147-1">Prado-Martinez et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref147-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Prado-Martinez, J. ∙ Sudmant, P.H. ∙ Kidd, J.M. ...</div><div><strong>Great ape genetic diversity and population history</strong></div><div><em>Nature.</em> 2013; <strong>499</strong>:471-475</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881159598" target="_blank" aria-label="Open Scopus in new tab">Scopus (402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23823723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12228&amp;pmid=23823723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), examining whether mouse and overlapping E59 human element sets differ in patterns of diversity within and between species. Considering human variation we found that diversity in PT- and DF-specific elements is constrained compared to genomic backgrounds, as well as to promoter-TSS and intronic elements (<a id="crosref0460" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0465" href="#mmc5">Table S5</a>; <a id="crosref0470" href="#sec-4">STAR Methods</a>). Comparing knee-sets directly, we observed that DF- and PT-specific elements have significantly reduced diversity compared to KC-specific elements, with more marked constraint in PT relative to DF (<a id="crosref0475" href="#fig3">Figure&nbsp;3</a>B). These patterns between specific knee element sets were not observed for orthologous forelimb sets, nor unfiltered “general” knee sets (<a id="crosref0480" href="#fig3">Figures 3</a>B and <a id="crosref0485" href="#figs2">S2</a>; <a id="crosref0490" href="#mmc5">Tables S5</a> and <a id="crosref0495" href="#mmc6">S6</a>).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/e04b483d-720f-4d75-8757-202257505b66/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.02.057/asset/a5d4e21b-adc2-4f49-b380-e9a44db27165/main.assets/gr3.jpg" height="830" width="752" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/e04b483d-720f-4d75-8757-202257505b66/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Intra-species Variation in Regulatory Elements and Human Variation in Knee Morphology</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">(A) Counts of common human variants per bp of sequence for element sets compared to random region sets along with other genomic features; labels correspond to <a id="crosref2220" href="#mmc5">Table S5</a>.</div><div id="fspara0070" role="paragraph">(B) Common human variants intersecting elements in the elbow (left) and knee (right) specific sets were counted and compared across sets.</div><div id="fspara0075" role="paragraph">(C) Comparisons of chimp sequence diversity across knee-specific sets.</div><div id="fspara0080" role="paragraph">(D–F) Common variants in human, chimp, and gorilla intersecting a given ATAC-seq peak counted for all variable sequences in knee-specific sets, expressed as “SNPs per sequence”—mean/median values shown in dashed/bold lines, respectively.</div><div id="fspara0085" role="paragraph">(D) Distal femur-specific (DF).</div><div id="fspara0090" role="paragraph">(E) Proximal tibia-specific (PT).</div><div id="fspara0095" role="paragraph">(F) Knee-common-specific (KC).</div><div id="fspara0100" role="paragraph">(G) Measurements of human medial/lateral tibia and femur via MRI dataset. Volume of cartilage (left) and total area of subchondral bone (right) for medial/lateral portions of both bones were measured across all subjects. Individual points are plotted alongside an outlined density curve, quartiles indicated in dashed lines. Significance codes: not significant (ns),&nbsp;&lt; 0.05 (<sup>∗</sup>),&nbsp;&lt; 0.01 (<sup>∗∗</sup>),&nbsp;&lt; 1e−5 (<sup>∗∗∗</sup>).</div><div id="fspara0105" role="paragraph">See also <a id="crosref2225" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref2230" href="#mmc5">Tables S5</a> and <a id="crosref2235" href="#mmc6">S6</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/d8acb6b0-c9a7-4388-a2ec-2c4d26a8c130/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.02.057/asset/f192a26e-ddbf-4120-b778-edc444d7087e/main.assets/figs2.jpg" height="940" width="578" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/d8acb6b0-c9a7-4388-a2ec-2c4d26a8c130/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Related to <a id="crosref2330" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0270" role="paragraph">Intra-species sequence diversity in humans and apes. The number of intra-species variants intersecting a given ATAC-seq peak were counted for all variable sequences in a set within a given species and expressed as ‘SNPs per sequence’. Mean and median values of distributions are indicated in dashed and bold lines, respectively. <a id="crosref2335" href="#mmc5">Table S5</a> lists numerical values for each comparison. (A) General Knee Distal Femur, (B) General Knee Proximal Tibia, (C) General Knee Common, (D) General Elbow Distal Humerus, (E) General Elbow Proximal Radius, (F) General Elbow Common, (G) Elbow Distal Humerus-Specific, (H) Elbow Proximal Radius-Specific, (I) Elbow-Common-Specific. (J) Knee Distal Femur-Specific, overlapping human E59 Distal Femur data. (K) Knee Proximal Tibia-Specific, overlapping human E59 Proximal Tibia data. (L) Knee-Common-Specific, overlapping pooled human E59 Distal Femur and Proximal Tibia data. (M) Elbow Distal Humerus-Specific, overlapping human E59 Distal Humerus data. (N) Elbow Proximal Radius-Specific, overlapping human E59 Proximal Radius data. (O) Elbow-Common-Specific, overlapping pooled human E59 Distal Humerus and Proximal Radius data.&nbsp;(P) Counts of common chimpanzee variants (via the GADP dataset) per bp of sequence for element sets were compared to random region sets along with&nbsp;other genomic features for enrichment/depletion analysis; labels correspond to results in <a id="crosref2340" href="#mmc5">Table S5</a>. Significance codes: not significant (ns),&nbsp;&lt; 0.05 (<sup>∗</sup>),&nbsp;&lt; 0.01 (<sup>∗∗</sup>),&nbsp;&lt; 1e-5 (<sup>∗∗∗</sup>).</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Corresponding sets in chimps and gorillas were examined for similar patterns of diversity between DF-, PT-, and KC-specific elements (<a id="crosref0500" href="#fig3">Figure&nbsp;3</a>C). No significant differences were observed, similarly true for the forelimb and “general knee” sets (<a id="crosref0505" href="#mmc5">Table S5</a>). Comparing patterns of diversity between humans and African apes, we observed reduced variation within DF- and PT-specific elements in humans compared to corresponding chimp and gorilla elements (<a id="crosref0510" href="#fig3">Figures 3</a>D and 3E). We note that no significant differences in diversity levels in KC-specific elements between species were observed (<a id="crosref0515" href="#fig3">Figure&nbsp;3</a>F; <a id="crosref0520" href="#mmc5">Table S5</a>). These patterns were recapitulated in overlapping E59 human elements (<a id="crosref0525" href="#figs2">Figures S2</a>A–S2C and S2J–S2L; <a id="crosref0530" href="#mmc5">Tables S5</a> and <a id="crosref0535" href="#mmc6">S6</a>).</div><div id="p0080" role="paragraph">However, the human pattern we observed (<a id="crosref0540" href="#fig3">Figures 3</a>B–3F) could reflect a general increase in genetic diversity in apes, notably chimps (<span class="dropBlock reference-citations"><a id="body-ref-sref147-2" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147-2" aria-controls="sref147-2">Prado-Martinez et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref147-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Prado-Martinez, J. ∙ Sudmant, P.H. ∙ Kidd, J.M. ...</div><div><strong>Great ape genetic diversity and population history</strong></div><div><em>Nature.</em> 2013; <strong>499</strong>:471-475</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881159598" target="_blank" aria-label="Open Scopus in new tab">Scopus (402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23823723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12228&amp;pmid=23823723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore performed an analysis of sequence set constraint by generating chimp genomic backgrounds and functional annotations and comparing diversity with knee-element sets. We found elevated background diversity across the chimp genome, indicated by the left skew in the distribution. Regardless of skew, knee-specific sets in chimp behave markedly different; specifically, they each exhibit less constraint than promoter-TSS and intronic elements (<a id="crosref6550" href="#fig3">Figures 3</a>A and <a id="crosref0555" href="#figs2">S2</a>P; <a id="crosref0560" href="#mmc5">Table S5</a>). Overall, these data reveal a unique human pattern of constraint among DF- and PT-specific elements (i.e., reduction of genetic diversity relative to KC elements and functional annotations, not in African apes), suggesting earlier results were capturing, at least in part, these signals.</div></section><section id="sec-2-4"><h3>Examination of Human Knee Morphological Diversity</h3><div id="p0085" role="paragraph">Given these sequence constraint patterns, we sought a corresponding phenotypic signal in human knee morphology. Using morphometric data from OA Biomarkers Consortium Project (OABCP) MRI images, we confirmed reduced variation in all proximal tibia measurements compared to matched distal femur features (<a id="crosref0565" href="#fig3">Figure&nbsp;3</a>G; <a id="crosref0570" href="#mmc5">Table S5</a>). Intersecting OAI sequence data with knee elements, we found significantly less genetic variation in PT-specific compared to DT-specific elements (<a id="crosref0575" href="#mmc5">Table S5</a>), supporting a genotype-phenotype link between reduced knee regulatory sequence variation genome-wide and reductions in morphological diversity.</div></section><section id="sec-2-5"><h3>Examination of the Biological Impacts of Sequence Variation in Element Sets</h3><div id="p0090" role="paragraph">We next examined how genetic diversity in knee elements generally impacts phenotypic and regulatory variation within species. Human and chimp sequences in knee sets partitioned by variation were examined using GREAT. For the least variable (first quartile) human and chimp elements, similar enrichments were observed for terms such as “cartilage development” and “chondrocyte differentiation.” Conversely, the most variable (third quartile) sequences showed different enrichments for humans and chimps. The most variable human elements yielded enrichments for “collagen catabolism/metabolism,” “anchoring collagen,” and “osteoarthritis,” functional terms not identified with variable chimp elements (<a id="crosref0580" href="#mmc5">Table S5</a>).</div><div id="p0095" role="paragraph">These findings in variable human sequences lead us to examine the relationship between common variation in knee sets and OA loci identified via GWAS. We aggregated 95 lead variants across 83 OA GWAS loci (<a id="crosref0585" href="#mmc7">Table S7</a>) and found that such variants were enriched in knee-specific element sets, including E59 human elements (<a id="crosref0590" href="#fig4">Figure&nbsp;4</a>A; <a id="crosref0595" href="#mmc7">Table S7</a>). We saw no enrichment of variants in less-specific human E47 limb datasets, BMDCs, nor for the brain. These findings suggest a specific link between human variation in knee chondrocyte regulatory sequences and OA risk.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/ce5c71b8-f507-4271-aff1-6a2617d56b50/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.02.057/asset/4ad5fa4f-b9da-411d-9717-97ca0bfd4518/main.assets/gr4.jpg" height="923" width="752" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/ce5c71b8-f507-4271-aff1-6a2617d56b50/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Human Variation in Knee Elements and Its Impacts on OA Risk</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">(A) Enrichments for OA risk variants in general knee, knee-specific, human E59 hind limb, and brain region sets, along with human E47 and BMDC H3K27ac ChIP-seq regions. Calculated Z score enrichments over randomized sets shown as -log (p value); red line indicates significance threshold (p&nbsp;&lt; 0.05).</div><div id="fspara0120" role="paragraph">(B) Distribution of predicted motifs intersected by OA risk variants, counted per TF. Significantly enriched factors are indicated—FOXP1 significant following p&nbsp;value correction (p &lt; 0.05).</div><div id="fspara0125" role="paragraph">(C) Distribution of predicted motifs intersecting OA risk variants in knee elements for a set of chondrogenesis-related TFs.</div><div id="fspara0130" role="paragraph">(D) Overlaps of region sets and signals of recent selection calculated for general knee, knee-specific, brain, and human H3K27ac ChIP-seq region sets, along with OA risk-variants and the regions intersecting them. Clustered set overlaps also shown. Hypergeometric tests represented as -log (p value), with sign denoting enrichment/depletion; red lines indicate respective significance cut-offs (p&nbsp;&lt; 0.05).</div><div id="fspara0135" role="paragraph">(E) Top: formalized model for the role of evolutionary history in modern heritable OA risk. Ancient selection acting on ancestral sequence diversity in regulatory elements establishes a derived knee configuration, which is subsequently maintained through ancient purifying selection (i.e., functional constraint). More recently, genetic drift, in combination with antagonistic selection for other traits, increases the frequency of alternative alleles in functionally constrained sites. Bottom: the presence of moderate mutational load in highly constrained elements (e.g., prox. tibia elements), or high mutational load in less-constrained elements (e.g., dist. femur elements), stand to disrupt knee homeostasis and promote pathology risk, while low mutational loads (or low sequence constraint) are more tolerated (i.e., harbor lesser risk of pathology).</div><div id="fspara0140" role="paragraph">(F) The number of alternative alleles falling in chondrocyte regulatory elements counted per-individual for the 1KG3 population (blue) and the OAI patient cohort (red), shown as density distributions with mean values (dashed lines); significance bar indicates Student’s t test result (p&nbsp;&lt; 0.05).</div><div id="fspara0145" role="paragraph">See also <a id="crosref2240" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref2245" href="#mmc3">Tables S3</a>, <a id="crosref2250" href="#mmc7">S7</a>, and <a id="crosref2255" href="#mmc8">S8</a>.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">As GWAS variants are thought to alter gene expression via TF modifications (<span class="dropBlock reference-citations"><a id="body-ref-sref202" href="#" role="doc-biblioref" data-xml-rid="bib202" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref202" aria-controls="sref202">Zhang and Lupski, 2015</a><div class="dropBlock__holder" data-db-target-of="sref202"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">188.</div><div>Zhang, F. ∙ Lupski, J.R.</div><div><strong>Non-coding genetic variants in human disease</strong></div><div><em>Hum. Mol. Genet.</em> 2015; <strong>24</strong>:R102-R110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddv259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943752520" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26152199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddv259&amp;pmid=26152199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we examined if OA variants in knee elements modify sequence motifs (<a id="crosref0605" href="#mmc7">Table S7</a>). Although we observed that human-chimp substitutions in knee-specific elements bias against altering FOXP1/FOXP2 motifs, OA variants bias toward altering them (adj. p value&nbsp;&lt; 0.05) (<a id="crosref0610" href="#fig4">Figure&nbsp;4</a>B; <a id="crosref0615" href="#mmc7">Table S7</a>). We also observed these OA variants intersecting motifs for chondrogenesis-associated TFs (<a id="crosref0620" href="#fig4">Figure&nbsp;4</a>C; <a id="crosref0625" href="#mmc7">Table S7</a>); in particular, pooled knee- and PT-specific elements have variants tending to overlap predicted CTCF and KLF5 motifs (adj. p value&nbsp;&lt; 0.05). Furthermore, knee-specific elements capture a significant proportion of all OA variants intersecting these motifs (&gt;3-fold enrichment, adj. p value &lt;0.05, <a id="crosref0630" href="#mmc7">Table S7</a>). We examined if common variants in knee element sets (i.e., SNPs with minor allele frequency [MAF] ≥ 0.05) frequently modify chondrogenesis-related TF motifs but found a lack of significant bias (<a id="crosref0635" href="#mmc3">Table S3</a>), pointing to the specific enrichment of signals for OA GWAS variants. Nevertheless, at some functional OA-annotated loci (i.e., found via GREAT) common variants were predicted to alter relevant motifs in knee-specific elements. Two examples are shown in <a id="crosref0640" href="#figs3">Figures S3</a>A and S3B.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/5934fb0b-9456-4267-be08-77a1c32c40fd/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.02.057/asset/d49118e2-6689-4853-be1f-b446f31a8964/main.assets/figs3.jpg" height="940" width="553" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/5934fb0b-9456-4267-be08-77a1c32c40fd/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Related to <a id="crosref2345" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0280" role="paragraph">Examining modern human sequence variation in knee sets. (A-B): Motif alteration in putative regulatory elements near osteoarthritis-related genes. Red shading indicates altered base relative to motif logo, blue shading below indicates genomic position of sequence. ATAC-seq regions and phyloP100ways conservation tracks are shown on bottom. (A) Common human variant (rs265053: C/T) within an intron of <i>UNC5C</i>, improves a predicted CEBPB sequence motif downstream of <i>BMPR1B</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref201" href="#" role="doc-biblioref" data-xml-rid="bib7 bib201" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref201" aria-controls="sref201">Baugé et&nbsp;al., 2013; Zhai et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref201"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Baugé, C. ∙ Girard, N. ∙ Lhuissier, E. ...</div><div><strong>Regulation and Role of TGFβ Signaling Pathway in Aging and Osteoarthritis Joints</strong></div><div><em>Aging Dis.</em> 2013; <strong>5</strong>:394-405</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25489490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=25489490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">187.</div><div>Zhai, G. ∙ Doré, J. ∙ Rahman, P.</div><div><strong>TGF-β signal transduction pathways and osteoarthritis</strong></div><div><em>Rheumatol. Int.</em> 2015; <strong>35</strong>:1283-1292</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00296-015-3251-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84931569939" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25773660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00296-015-3251-z&amp;pmid=25773660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). (B) Common human variant (rs2280153: G/A) improves a predicted SOX9 sequence motif within the <i>WISP3</i> promoter (<span class="dropBlock reference-citations"><a id="body-ref-sref163" href="#" role="doc-biblioref" data-xml-rid="bib95 bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref163" aria-controls="sref163">Kannu et&nbsp;al., 2009; Sen et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref163"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Kannu, P. ∙ Bateman, J.F. ∙ Belluoccio, D. ...</div><div><strong>Employing molecular genetics of chondrodysplasias to inform the study of osteoarthritis</strong></div><div><em>Arthritis Rheum.</em> 2009; <strong>60</strong>:325-334</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.24251" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-59649092384" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19180483/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.24251&amp;pmid=19180483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">154.</div><div>Sen, M. ∙ Cheng, Y.-H. ∙ Goldring, M.B. ...</div><div><strong>WISP3-dependent regulation of type II collagen and aggrecan production in chondrocytes</strong></div><div><em>Arthritis Rheum.</em> 2004; <strong>50</strong>:488-497</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.20005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-1042290330" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14872491/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.20005&amp;pmid=14872491" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). (C) Common human variants within the indicated peak sets distributed across the first two PCs of an Fst-based PCA analysis. Two sub-groups based on k-means clustering are indicated in red (Cluster 1) and blue (Cluster 2). Variance explained by each PC is indicated on axes. (D) Representative correlation heatmap between PCs and population comparisons across all variants in the knee-specific pooled set. Pearson’s correlation between PCs and population pairs are clustered, with populations colored by continent: red - Asia, green - Europe, blue - Africa. (E, F) Calculated Fst for common variants across populations. Fst values for a single variant across multiple pairwise population comparisons are shown, with each row representing a single variant; the particular pairwise comparison is indicated below, color scheme as for (D). Red/blue scale for Fst values is indicated on far right. Both variants and pairwise comparisons are grouped by hierarchical clustering. (E) Cluster 1 variants within the knee-specific pooled set. (F) Cluster 2 variants within the knee-specific pooled set. (G-J) Visual genotype plots for selected populations in a given locus. SNP sequences for all individuals are arranged according to a maximum-likelihood tree. Red/Blue indicates ancestral-derived assignments, respectively. Solid orange lines indicate SNPs which are gapped in chimpanzee, solid green lines indicates individuals carrying alleles besides the ancestral/derived annotations. Vertical yellow dashed lines indicate proxy variants, vertical green dashed lines indicate lead variants, when present in indicated region. For plots with a distinguishable haplotype (G, H) the alternate haplotype is indicated with a thick green line. (G) Visual genotypes for selected population (Volga-Ural region of Russia) (<span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Capellini et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) within a ∼150kb haploblock in the <i>GDF5-UQCC1</i> locus. Distribution of genotypes indicates the presence of a high-frequency haploblock carrying the lead GWAS variant “A” at rs143383 (<span class="dropBlock reference-citations"><a id="body-ref-sref180" href="#" role="doc-biblioref" data-xml-rid="bib125 bib180" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref180" aria-controls="sref180">Miyamoto et&nbsp;al., 2007; Valdes et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref180"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Miyamoto, Y. ∙ Mabuchi, A. ∙ Shi, D. ...</div><div><strong>A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:529-533</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34047103363" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17384641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F2005&amp;pmid=17384641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">169.</div><div>Valdes, A.M. ∙ Evangelou, E. ∙ Kerkhof, H.J.M. ...</div><div><strong>The GDF5 rs143383 polymorphism is associated with osteoarthritis of the knee with genome-wide statistical significance</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:873-875</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.134155" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78650988509" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20870806/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.134155&amp;pmid=20870806" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and linked variants (rs4911178, rs6060369). (H) Visual genotypes for selected population (South-East Asia) within a ∼64kb haploblock in the <i>UNC5C-BMPR1B</i> locus; variant rs2626053 (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">arcOGEN Consortium et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>arcOGEN Consortium, arcOGEN Collaborators ∙ Zeggini, E. ...</div><div><strong>Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study</strong></div><div><em>Lancet.</em> 2012; <strong>380</strong>:815-823</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865595151" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22763110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2812%2960681-3&amp;pmid=22763110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) appears on a low-frequency haploblock. (I) Visual genotypes for selected population (South-Western Europe) in the <i>ENPP1/3</i> locus; given the lack of clearly-defined haploblock structure in the locus, a 200kb region centered on rs3850251 (<span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib2 bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">arcOGEN Consortium et&nbsp;al., 2012; Klein et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>arcOGEN Consortium, arcOGEN Collaborators ∙ Zeggini, E. ...</div><div><strong>Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study</strong></div><div><em>Lancet.</em> 2012; <strong>380</strong>:815-823</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865595151" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22763110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2812%2960681-3&amp;pmid=22763110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Klein, J.C. ∙ Keith, A. ∙ Rice, S.J. ...</div><div><strong>Functional testing of thousands of osteoarthritis-associated variants for regulatory activity</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10439-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85066610023" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164647/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10439-y&amp;pmid=31164647" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to generate this plot, with linked rs7744039, rs7773292, rs9493095 indicated. (J) Visual genotypes for selected population (Central Siberia) in the <i>LSM5</i> locus; given the lack of clearly-defined haploblock structure in the locus, a 200kb region centered on rs4141788, linked to rs7785659 (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">arcOGEN Consortium et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>arcOGEN Consortium, arcOGEN Collaborators ∙ Zeggini, E. ...</div><div><strong>Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study</strong></div><div><em>Lancet.</em> 2012; <strong>380</strong>:815-823</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865595151" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22763110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2812%2960681-3&amp;pmid=22763110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), was used to generate this plot.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">Given the polygenic nature of OA (<span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Hunter and Bierma-Zeinstra, 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Hunter, D.J. ∙ Bierma-Zeinstra, S.</div><div><strong>Osteoarthritis</strong></div><div><em>Lancet.</em> 2019; <strong>393</strong>:1745-1759</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2819%2930417-9&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2819%2930417-9&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31034380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2819%2930417-9&amp;pmid=31034380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), smaller effect-size variants in knee elements may cumulatively contribute to alter chondrocyte regulation, and ultimately risk heritability. We assessed the contributions that variants in knee elements, chondrocyte elements generally, and other genomic features have to OA risk heritability. Using linkage disequilibrium score regression (LDSC) (<span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Finucane et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Finucane, H.K. ∙ Bulik-Sullivan, B. ∙ Gusev, A. ..., ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, RACI Consortium</div><div><strong>Partitioning heritability by functional annotation using genome-wide association summary statistics</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1228-1235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000443086" target="_blank" aria-label="Open Scopus in new tab">Scopus (535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26414678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3404&amp;pmid=26414678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found orthologous chondrocyte elements pooled from long bones and human E59 hind limb elements captured variants explaining a significant proportion of OA heritability (adj. p&nbsp;&lt; 0.05, <a id="crosref0655" href="#mmc7">Table S7</a>), a feature not seen using brain or B-lymphocyte elements. Variants captured by more refined knee-specific element sets did not reach LDSC significance, owing to limits in ascertaining significant partitioned heritability from very small sets (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Finucane et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Finucane, H.K. ∙ Bulik-Sullivan, B. ∙ Gusev, A. ..., ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, RACI Consortium</div><div><strong>Partitioning heritability by functional annotation using genome-wide association summary statistics</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1228-1235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000443086" target="_blank" aria-label="Open Scopus in new tab">Scopus (535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26414678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3404&amp;pmid=26414678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interestingly, annotations pertaining to&nbsp;sequence conservation, i.e., Genomic Evolutionary Rate Profiling (GERP) score (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Davydov et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Davydov, E.V. ∙ Goode, D.L. ∙ Sirota, M. ...</div><div><strong>Identifying a high fraction of the human genome to be under selective constraint using GERP++</strong></div><div><em>PLoS Comput. Biol.</em> 2010; <strong>6</strong>:e1001025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1001025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78651237647" target="_blank" aria-label="Open Scopus in new tab">Scopus (720)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21152010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1001025&amp;pmid=21152010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), primate phastCons (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Hubisz et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Hubisz, M.J. ∙ Pollard, K.S. ∙ Siepel, A.</div><div><strong>PHAST and RPHAST: phylogenetic analysis with space/time models</strong></div><div><em>Brief. Bioinform.</em> 2011; <strong>12</strong>:41-51</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbq072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79551634894" target="_blank" aria-label="Open Scopus in new tab">Scopus (147)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21278375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbib%2Fbbq072&amp;pmid=21278375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and predicted allele age (<span class="dropBlock reference-citations"><a id="body-ref-sref154" href="#" role="doc-biblioref" data-xml-rid="bib154" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref154" aria-controls="sref154">Rasmussen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref154"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Rasmussen, M.D. ∙ Hubisz, M.J. ∙ Gronau, I. ...</div><div><strong>Genome-wide inference of ancestral recombination graphs</strong></div><div><em>PLoS Genet.</em> 2014; <strong>10</strong>:e1004342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1004342" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84901640930" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24831947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1004342&amp;pmid=24831947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) also associated with OA heritability.</div></section><section id="sec-2-6"><h3>Examination of Recent Evolutionary Forces</h3><div id="p0110" role="paragraph">We next wanted to determine whether more recent genetic drift and positive selection shaped human variant frequencies in knee-specific elements and influenced the observed disease associations. We considered the behavior of variants in knee elements, investigating pairwise Fst (<span class="dropBlock reference-citations"><a id="body-ref-sref189-1" href="#" role="doc-biblioref" data-xml-rid="bib189" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref189-1" aria-controls="sref189-1">Weir and Cockerham, 1984</a><div class="dropBlock__holder" data-db-target-of="sref189-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">177.</div><div>Weir, B.S. ∙ Cockerham, C.C.</div><div><strong>Estimating F-statistics for the analysis of population structure</strong></div><div><em>Evolution.</em> 1984; <strong>38</strong>:1358-1370</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2307/2408641" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28563791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2307%2F2408641&amp;pmid=28563791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) across 18 populations (<a id="crosref0685" href="#sec-4">STAR Methods</a>). Principal-component analysis (PCA) was used to visualize the separation of knee variants based on shared behaviors in population stratification (<a id="crosref0690" href="#figs3">Figure&nbsp;S3</a>C; <a id="crosref0695" href="#mmc8">Table S8</a>). The first two PCs represented population differences between Eurasia/Africa (PC1) and Europe/Asia (PC2) populations (<a id="crosref0700" href="#figs3">Figure&nbsp;S3</a>D), capturing the majority (&gt;60%) of observed variation. K-means clustering of variants based on Fst resulted in two groups per knee set (<a id="crosref0705" href="#mmc8">Table S8</a>; <a id="crosref0710" href="#sec-4">STAR Methods</a>): cluster 1 variants have a narrower Fst range across populations (<a id="crosref0715" href="#figs3">Figures S3</a>E and S3F), while cluster 2 trends toward increased Eurasia/Africa divergence. Using GREAT, we determined whether clusters have divergent functional enrichments (<a id="crosref0720" href="#mmc8">Table S8</a>). Cluster 2 variants were functionally associated with&nbsp;“skeletal development” and related chondrocyte terms, while cluster 1 variants tended to occur near genes with annotations such as “arthritis,” “arthralgia,” “osteoarthritis,” and “knee joint.”</div><div id="p0115" role="paragraph">We also assessed whether recent positive selection shaped the patterns of clustered variants, and OA variants in general. We found that only 18% of regions containing clustered knee elements (cluster 1) are known to have undergone positive selection in the last 30,000 years (<span class="dropBlock reference-citations"><a id="body-ref-sref137-1" href="#" role="doc-biblioref" data-xml-rid="bib93 bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-1" aria-controls="sref137-1">Jagoda et&nbsp;al., 2018; Pagani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref137-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Jagoda, E. ∙ Lawson, D.J. ∙ Wall, J.D. ...</div><div><strong>Disentangling Immediate Adaptive Introgression from Selection on Standing Introgressed Variation in Humans</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:623-630</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85046466019" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29220488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx314&amp;pmid=29220488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a significant depletion (1.25 fold-decrease, adjusted p&nbsp;value: 5.26e<sup>−11</sup>) (<a id="crosref0725" href="#fig4">Figure&nbsp;4</a>D; <a id="crosref0730" href="#mmc8">Table S8</a>). No cluster 2 set was enriched in selection windows; rather, they trended toward depletion (<a id="crosref0735" href="#fig4">Figure&nbsp;4</a>D). We then examined loci for which OA GWAS variants fell in knee elements and selection windows, identifying four (<a id="crosref0740" href="#mmc7">Tables S7</a> and <a id="crosref0745" href="#mmc8">S8</a>). For three loci (<i>UNC5C-BMPR1B</i>, <i>ENPP1/3</i>, <i>LSM5</i>) the putative OA risk variant occurred on the non-selected, recombined haplotype (<a id="crosref0750" href="#figs3">Figures S3</a>G–S3J). However, at <i>GDF5</i>, OA risk variants were found to reside on the positively selected haplotype (<span class="dropBlock reference-citations"><a id="body-ref-sref125-2" href="#" role="doc-biblioref" data-xml-rid="bib20 bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-2" aria-controls="sref125-2">Capellini et&nbsp;al., 2017; Miyamoto et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref125-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Miyamoto, Y. ∙ Mabuchi, A. ∙ Shi, D. ...</div><div><strong>A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:529-533</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34047103363" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17384641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F2005&amp;pmid=17384641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). OA GWAS variants, in general, were not enriched in selection regions (<a id="crosref0755" href="#fig4">Figure&nbsp;4</a>D), and by one test (<span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Grossman et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Grossman, S.R. ∙ Andersen, K.G. ∙ Shlyakhter, I. ..., 1000 Genomes Project</div><div><strong>Identifying recent adaptations in large-scale genomic data</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:703-713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874069538" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.035&amp;pmid=23415221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were strongly depleted (<a id="crosref0765" href="#mmc8">Table S8</a>). Last, TF motif analyses on clustered low- and high-Fst variants did not show strong biases for intersections, indicating a consistent lack of directionality in regulatory modification in line with genetic drift effects (<a id="crosref0770" href="#mmc3">Table S3</a>).</div></section><section id="sec-2-7"><h3>Formal Model of Chondrocyte Knee Developmental Regulation, Evolution, and OA Risk</h3><div id="p0120" role="paragraph">Our analyses revealed that DF- and PT-specific regulatory elements exhibit evidence in humans of ancient positive selection (<a id="crosref0775" href="#fig1">Figure&nbsp;1</a>C), followed by more recent sequence constraint not observed in African apes (<a id="crosref0780" href="#fig3">Figures 3</a>A–3E). Furthermore, the allele frequencies of human common variants in these elements, and OA risk variants in general, appear to have been shaped by genetic drift on the background of constraint (<a id="crosref0785" href="#mmc7">Tables S7</a> and <a id="crosref0790" href="#mmc8">S8</a>), rather than recent positive selection (<a id="crosref0795" href="#fig4">Figure&nbsp;4</a>D). These variants have consistent links to OA (<a id="crosref0800" href="#mmc8">Table S8</a>), likely by impacting TF motifs under constraint (<a id="crosref0805" href="#fig4">Figures 4</a>B and 4C), and contribute to overall heritability of risk (<a id="crosref0810" href="#mmc7">Table S7</a>). We propose a model in which violations to constraint in functional conserved sequences, tolerated during knee development, have pathological consequences later in life (<a id="crosref0815" href="#fig4">Figure&nbsp;4</a>E). Two evolutionary mechanisms may cause violation to constraint thereby increasing genetic risk for OA: drift and antagonistic selection. We provide genetic and functional evidence below addressing how both processes may drive heightened OA risk genome-wide and on a locus-specific level.</div></section><section id="sec-2-8"><h3>Functional Sequence Variation within At-Risk Individuals</h3><div id="p0125" role="paragraph">Because it is anticipated that drift should increase the frequency of violations underlying OA risk, patients should have higher loads of violating mutations than the general population. To test this, we took the OAI patient cohort, consisting of individuals suffering from, or identified as being at high risk of developing OA, in comparison to 1000Genomes populations. We considered variants falling in knee-specific elements and asked whether, across the set of variable sites, patients tended to possess more alternative alleles. We observed a significant increase in the average number of alternative alleles possessed by subsets of the OAI cohort (<a id="crosref0820" href="#fig4">Figure&nbsp;4</a>F), results replicated using another population control (<span class="dropBlock reference-citations"><a id="body-ref-sref137-2" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-2" aria-controls="sref137-2">Pagani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref137-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0830" href="#mmc7">Table S7</a>). To confirm that it is specifically sequence violation within constrained knee regulatory elements that is related to the heightened OA risk in this cohort, we compared the number of alternative alleles falling in B-lymphocyte elements as a control, observing different variant behaviors (<a id="crosref0835" href="#mmc7">Table S7</a>).</div></section><section id="sec-2-9"><h3>Functional Interrogation of an OA Risk Locus</h3><div id="p0130" role="paragraph">Another mechanism that may lead to constraint violations is antagonistic pleiotropy, in which positive selection increases a beneficial allele along with linked deleterious alleles. Earlier, we uncovered one such locus, <i>Growth Differentiation Factor Five</i> (<i>GDF5</i>). GDF5 is a BMP with quintessential roles in knee development across mammals (<span class="dropBlock reference-citations"><a id="body-ref-sref164" href="#" role="doc-biblioref" data-xml-rid="bib6 bib158 bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164" aria-controls="sref164">Basit et&nbsp;al., 2008; Rountree et&nbsp;al., 2004; Settle et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref164"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Basit, S. ∙ Naqvi, S.K.-H. ∙ Wasif, N. ...</div><div><strong>A novel insertion mutation in the cartilage-derived morphogenetic protein-1 (CDMP1) gene underlies Grebe-type chondrodysplasia in a consanguineous Pakistani family</strong></div><div><em>BMC Med. Genet.</em> 2008; <strong>9</strong>:102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2350-9-102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-58149327059" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19038017/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2350-9-102&amp;pmid=19038017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Rountree, R.B. ∙ Schoor, M. ∙ Chen, H. ...</div><div><strong>BMP receptor signaling is required for postnatal maintenance of articular cartilage</strong></div><div><em>PLoS Biol.</em> 2004; <strong>2</strong>:e355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0020355" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-13944283679" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15492776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0020355&amp;pmid=15492776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div><div>Settle, Jr., S.H. ∙ Rountree, R.B. ∙ Sinha, A. ...</div><div><strong>Multiple joint and skeletal patterning defects caused by single and double mutations in the mouse Gdf6 and Gdf5 genes</strong></div><div><em>Dev. Biol.</em> 2003; <strong>254</strong>:116-130</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0012-1606(02)00022-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0037330644" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12606286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0012-1606%2802%2900022-2&amp;pmid=12606286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and yet is the most reproducibly associated OA locus (<span class="dropBlock reference-citations"><a id="body-ref-sref199-2" href="#" role="doc-biblioref" data-xml-rid="bib125 bib199" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref199-2" aria-controls="sref199-2">Miyamoto et&nbsp;al., 2007; Zengini et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref199-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Miyamoto, Y. ∙ Mabuchi, A. ∙ Shi, D. ...</div><div><strong>A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:529-533</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34047103363" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17384641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F2005&amp;pmid=17384641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">185.</div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). At <i>GDF5</i>, selection on reduced height via a regulatory variant (rs4911178) in <i>GROW1</i>, a growth plate enhancer, likely increased frequencies of linked OA risk variants (<span class="dropBlock reference-citations"><a id="body-ref-sref20-3" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-3" aria-controls="sref20-3">Capellini et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As our model predicts high-frequency sequence violations contribute to OA risk, we explored this exemplar locus in-depth.</div><div id="p0135" role="paragraph">We intersected <i>GDF5</i> GWAS OA variants (<span class="dropBlock reference-citations"><a id="body-ref-sref199-3" href="#" role="doc-biblioref" data-xml-rid="bib199" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref199-3" aria-controls="sref199-3">Zengini et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref199-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">185.</div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with general E15.5 mouse/E59 human knee elements and found putatively causal variants, rs4911178 (G/A) in <i>GROW1</i> and rs6060369 (C/T) (<a id="crosref7850" href="#fig5">Figure&nbsp;5</a>A). Rs4911178 and <i>GROW1</i> do not impact knees when deleted in mice (<span class="dropBlock reference-citations"><a id="body-ref-sref20-4" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-4" aria-controls="sref20-4">Capellini et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), implicating rs6060369 as causative. Rs6060369 overlaps a knee element, <i>R4</i>, located in downstream sequence shown in rescue&nbsp;experiments to regulate expression and mediate knee morphology of <i>Gdf5</i> null mice (<span class="dropBlock reference-citations"><a id="body-ref-sref148-1" href="#" role="doc-biblioref" data-xml-rid="bib30 bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148-1" aria-controls="sref148-1">Chen et&nbsp;al., 2016; Pregizer et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref148-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Chen, H. ∙ Capellini, T.D. ∙ Schoor, M. ...</div><div><strong>Heads, shoulders, elbows, knees, and toes: modular Gdf5 enhancers control different joints in the vertebrate skeleton</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1006454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1006454" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000624431" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27902701/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1006454&amp;pmid=27902701" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">140.</div><div>Pregizer, S.K. ∙ Kiapour, A.M. ∙ Young, M. ...</div><div><strong>Impact of broad regulatory regions on Gdf5 expression and function in knee development and susceptibility to osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2018; <strong>77</strong>:450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2017-212475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85052526710" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29311146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2017-212475&amp;pmid=29311146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To characterize <i>R4</i> activity, we made stably expressing <i>R4 lacZ</i> reporter mice and found, at E14.5/15, expression in the early knee, including the condylar regions and notch (but not trochlea), and postnatally, expression restricted to femoral condyle and tibial articular chondrocytes and cruciate ligaments (<a id="crosref0860" href="#fig5">Figure&nbsp;5</a>B). Compiling human functional genomics data on E47 limbs, E59 distal femur chondrocytes, and BMDCs, we found <i>R4</i> activity at each time point (<a id="crosref0865" href="#fig5">Figure&nbsp;5</a>A), indicating a similar regulatory time course as in mouse.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/237f5184-6230-400c-ac77-aa5dd617273f/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.02.057/asset/3449439a-5be1-44fe-876e-427ae331f1de/main.assets/gr5.jpg" height="939" width="620" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/237f5184-6230-400c-ac77-aa5dd617273f/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Functional Characterization of the <i>GDF5</i> Locus and <i>R4</i> Enhancer in the Mouse</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">(A) UCSC Genome Browser view of human <i>GDF5</i> locus with intersections of OA variants, knee ATAC-seq regions (for human and mouse tissue), <i>GDF5</i> enhancers, and H3K27ac ChIP-seq signals from human embryonic limbs (E47) and BMDCs. Rs4911178 (left red line) and rs6060369 (right red line) overlap functional knee sequences.</div><div id="fspara0160" role="paragraph">(B) <i>R4</i>-driven <i>lacZ</i> expression in inferior-most distal femur (DF) and superior-most proximal tibia (PT) (left) tissues and in adult distal femora (right two images). AC, articular cartilage; FN, femoral notch; IC, inferior condyles; CL, cruciate ligaments; TG, trochlear groove. Scale bars, E14.5&nbsp;= 250&nbsp;μm; adult&nbsp;= 1&nbsp;mm.</div><div id="fspara0165" role="paragraph">(C) P30 knee anatomy in C57BL/6J <i>R4</i> null mice. #, same trends for medial condyle. <i>R4</i><sup>+/+</sup> (wild-type [WT]), <i>R4</i><sup>−/−</sup> homozygous (HOM).</div><div id="fspara0170" role="paragraph">(D) 1-year knee anatomy.</div><div id="fspara0175" role="paragraph">(E) OARSI scores on WT and HOM knees at P30 and 1 year. Triangles, heterotopic ossification.</div><div id="fspara0180" role="paragraph">(F) 3D renditions (top) and histology (bottom) of WT/HOM knees with minimum (blue), mean (green), and maximum (red) OARSI scores. Heterotopic ossification observed in HOM knees with highest OARSI scores and most cartilage damage. Scale bars, 50&nbsp;μm.</div><div id="fspara0185" role="paragraph">(G) X-ray 3D cartilage scanning of WT/HOM distal femur condyles (top) and proximal tibia platforms (bottom) at 1 year, showing OA lesions (white arrows). Scale bars, 1&nbsp;mm.</div><div id="fspara0190" role="paragraph">(H) Linear regressions between knee morphology and OARSI score at 1 year. In (C)–(E) colored dots correspond to specimens shown in (F). Significance code:&nbsp;&lt;0.05 (<sup>∗</sup>).</div><div id="fspara0195" role="paragraph">See also <a id="crosref2260" href="#figs4">Figures S4</a> and <a id="crosref2265" href="#figs5">S5</a> and <a id="crosref2270" href="#mmc9">Table S9</a>.</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">We then deleted <i>R4</i> in mice, and using allele-specific expression analyses (ASE) found that loss of one <i>R4</i> copy downregulated <i>Gdf5</i> expression in E15.5 distal femur chondrocytes (deletion-allele expression 69.6% ± 0.05% of control expression; n&nbsp;=&nbsp;4/genotype; p&nbsp;= 0.001). At P30, <i>R4</i> loss (<i>R4</i><sup>−/−</sup>) led to significant distal femur changes, specifically, smaller condyle curvature radius and notch sizes, among other changes (<a id="crosref0870" href="#fig5">Figures 5</a>C, <a id="crosref0875" href="#fig7">7</a>, and <a id="crosref0880" href="#figs4">S4</a>A–S4C; <a id="crosref0885" href="#mmc9">Table S9</a>). <i>R4</i><sup>+/−</sup> mice displayed a similar but non-significant trend (<a id="crosref0890" href="#mmc9">Table S9</a>). Despite changes, a normal articular cartilage and joint cavity formed in <i>R4</i><sup>−/−</sup> mice (<a id="crosref0895" href="#fig5">Figure&nbsp;5</a>F, left). By 1 year, <i>R4</i><sup>−/−</sup> mice had exacerbated defects in the same distal femur features (<a id="crosref0900" href="#fig5">Figure&nbsp;5</a>D, p&nbsp;&lt; 0.05) and new alterations to the proximal tibia (<a id="crosref0905" href="#mmc9">Table S9</a>). These changes are related to genetic variants and not aging as morphological comparisons occurred between genotypes at each time point separately. Strikingly, OA developed in <i>R4</i><sup>−/−</sup> mice as observed via histology, Osteoarthritis Research Society International (OARSI) score (Mann-Whitney test, p&nbsp;= 0.041), and cartilage imaging (<a id="crosref5910" href="#fig5">Figures 5</a>E–5G and <a id="crosref0915" href="#figs5">S5</a>). This included a loss of glycosaminoglycans throughout articular surfaces with minor fibrillations of isolated surfaces in most individuals and severe damage up to complete denudation of isolated joint surfaces in some individuals (<a id="crosref0920" href="#fig5">Figure&nbsp;5</a>F). Using X-ray 3D scanning, we observed OA lesions on the femoral condyle and proximal tibia (<a id="crosref0925" href="#fig5">Figure&nbsp;5</a>G). In the most effected mice, heterotopic ossification was observed, a phenotype that occurs in late human OA (<a id="crosref0930" href="#fig5">Figure&nbsp;5</a>F). Several morphologic features of <i>R4</i><sup>−/−</sup> knees (e.g., condyle curvature, notch size, etc.) were correlated with OA severity; features which have been shown to change during human OA progression (<span class="dropBlock reference-citations"><a id="body-ref-sref132-2" href="#" role="doc-biblioref" data-xml-rid="bib4 bib89 bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-2" aria-controls="sref132-2">Barr et&nbsp;al., 2016; Hunter et&nbsp;al., 2016; Neogi et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref132-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Barr, A.J. ∙ Dube, B. ∙ Hensor, E.M.A. ...</div><div><strong>The relationship between three-dimensional knee MRI bone shape and total knee replacement-a case control study: data from the Osteoarthritis Initiative</strong></div><div><em>Rheumatology (Oxford).</em> 2016; <strong>55</strong>:1585-1593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/rheumatology/kew191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84995616558" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27185958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Frheumatology%2Fkew191&amp;pmid=27185958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Hunter, D. ∙ Nevitt, M. ∙ Lynch, J. ..., FNIH OA Biomarkers Consortium</div><div><strong>Longitudinal validation of periarticular bone area and 3D shape as biomarkers for knee OA progression? Data from the FNIH OA Biomarkers Consortium</strong></div><div><em>Ann. Rheum. Dis.</em> 2016; <strong>75</strong>:1607-1614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2015-207602" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26483253/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2015-207602&amp;pmid=26483253" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Neogi, T. ∙ Bowes, M.A. ∙ Niu, J. ...</div><div><strong>Magnetic resonance imaging-based three-dimensional bone shape of the knee predicts onset of knee osteoarthritis: data from the osteoarthritis initiative</strong></div><div><em>Arthritis Rheum.</em> 2013; <strong>65</strong>:2048-2058</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.37987" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881340844" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23650083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.37987&amp;pmid=23650083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0935" href="#fig5">Figures 5</a>H and <a id="crosref0940" href="#figs4">S4</a>D). However, as these changes were also detected in <i>R4</i><sup>−/−</sup> mice at P30, such shape alterations preceded and may have caused OA.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/754fe31c-95bb-4506-93ed-8f70d604163d/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.02.057/asset/a6934eb5-5284-401f-a77a-03cc07336d7e/main.assets/figs4.jpg" height="939" width="584" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/754fe31c-95bb-4506-93ed-8f70d604163d/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Related to <a id="crosref2365" href="#fig5">Figures 5</a> and <a id="crosref2370" href="#fig7">7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0290" role="paragraph">Additional morphometric analyses on <i>R4</i> enhancer null mice. (A) Quantified anatomical indices of the femur and tibia used in this study. Femoral length (FL), bicondylar width of the femur (BCW), trochlear groove depth (TD), width of the lateral femoral condyle (LCW), width of the medial femoral condyle (MCW), intercondylar notch width (NW), trochlear groove sulcus angle (SA), width of the tibial plateau (TPW), height of the medial tibial spine (MTSH), height of the lateral tibial spine (LTSH), posterior slope of the tibial plateau (PSTP), and tibial length (TL). Curvature radius of the femoral condyle and the posterior slope of the tibial plateau were measured across both medial and lateral compartments. (B,C) Morphological defects in 1 year old <i>R4</i> enhancer null mice (HOM), compared to heterozygous mice (HET) and wild-type mice (WT). A number of measurements were carried out at 1 year of age (see (A)). This figure displays only those which revealed significant differences between control and <i>R4</i> enhancer null mice in distal femur (B) and proximal tibia (C) structures. See <a id="crosref2375" href="#mmc9">Table S9</a> for complete results of statistical comparisons for all measurements as well as significance values for each comparison. (D) Correlation between OARSI scores and morphometric measures of the distal femur and proximal tibia on <i>R4</i> enhancer mice. Correlation (R<sup>2</sup>) and p value results for Pearson’s correlation tests are indicated on each graph.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/574e7c7a-3e80-46a9-a7e2-3ca0cf05d324/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.02.057/asset/56b3e442-c772-43e4-a095-c7f66145053f/main.assets/figs5.jpg" height="940" width="598" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/574e7c7a-3e80-46a9-a7e2-3ca0cf05d324/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Related to <a id="crosref2380" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0300" role="paragraph">Additional histological sections on <i>R4</i> enhancer null mice at P30 and 1 year. (Left) Comparison of WT and HOM knee joints at P30, showing microCT renditions (top row), high-magnification images of sections of the unaffected knee joint in both genotypes as reported in <a id="crosref2385" href="#fig5">Figure&nbsp;5</a> (second row), and low-magnification images of the of same medial sections but at different planes (third and fourth rows). (Right) Comparison of WT and two HOM knee joints of differing phenotypic severity (HOM-1 mild osteoarthritis (green); HOM-2 severe osteoarthritis (red)) at 1 year, showing microCT renditions (top row), including an X-ray image of HOM-2 specimen with heterotopic ossification (top row, far right), high-magnification images showing effected medial and lateral sections (when effected) as reported in <a id="crosref2390" href="#fig5">Figure&nbsp;5</a> (second row), low-magnification images of the same medial and lateral sections of the joint but in two different planes (third and fourth rows), and total joint images for all three specimens (fifth row). Note, the loss of articular cartilage matrix in the lateral compartment and clustering and loss of cells in the cartilage of the medial compartment in HOM knees. Scale bars, 50&nbsp;μm (second row); 500&nbsp;μm (third/fourth rows); 250&nbsp;μm (fifth row).</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">We next explored <i>R4</i> function and its OA variant (rs6060369) in human and mouse chondrocytes <i>in&nbsp;vitro</i>. <i>R4</i> deletion in human T/C-28a2 articular chondrocytes caused downregulation of <i>GDF5</i> expression (p&nbsp;&lt; 0.005), but not nearby genes (<a id="crosref0945" href="#fig6">Figure&nbsp;6</a>A). Deletion of a 41&nbsp;bp subregion containing rs6060369 had a similar effect (p&nbsp;&lt; 0.05, <a id="crosref0950" href="#fig6">Figure&nbsp;6</a>B). Luciferase reporter assays in these cells for rs6060369 found that the risk allele “T” drove reduced expression (<a id="crosref0955" href="#fig6">Figure&nbsp;6</a>C, p&nbsp;= 0.000044; Fisher-combined p&nbsp;= 8.79E−09). We computationally predicted that Pituitary homeobox 1 (PITX1), a major TF in knee formation (<span class="dropBlock reference-citations"><a id="body-ref-sref130-1" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130-1" aria-controls="sref130-1">Nemec et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref130-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Nemec, S. ∙ Luxey, M. ∙ Jain, D. ...</div><div><strong><i>Pitx1</i> directly modulates the core limb development program to implement hindlimb identity</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:3325-3335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.154864" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85029782232" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28807899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.154864&amp;pmid=28807899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and OA factor (<span class="dropBlock reference-citations"><a id="body-ref-sref140-1" href="#" role="doc-biblioref" data-xml-rid="bib18 bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-1" aria-controls="sref140-1">Butterfield et&nbsp;al., 2019; Picard et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref140-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Butterfield, N.C. ∙ Curry, K.F. ∙ Steinberg, J. ...</div><div><strong>Accelerating functional gene discovery in osteoarthritis</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/836221" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F836221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Picard, C. ∙ Azeddine, B. ∙ Moldovan, F. ...</div><div><strong>New emerging role of pitx1 transcription factor in osteoarthritis pathogenesis</strong></div><div><em>Clin. Orthop. Relat. Res.</em> 2007; <strong>462</strong>:59-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/BLO.0b013e3180d09d9c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34548400346" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17549029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FBLO.0b013e3180d09d9c&amp;pmid=17549029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), binds to this variant position. Using ChIP on T/C-28a2 cells (<a id="crosref0965" href="#fig6">Figure&nbsp;6</a>D), E15.5 distal femur and proximal tibia cartilage, and PITX1 ChIP-seq data (<a id="crosref0970" href="#fig6">Figure&nbsp;6</a>E), we validated PITX1 binding at this position, indicating that it likely mediates the <i>cis</i>-acting effects of rs6060369 in <i>R4</i> on <i>GDF5</i> expression.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/a5de51ad-1152-4dc4-84d0-2457f53edd79/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.02.057/asset/da5bb7c7-8caf-4eaa-a435-260ff6762a13/main.assets/gr6.jpg" height="334" width="499" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/a5de51ad-1152-4dc4-84d0-2457f53edd79/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Functional Characterization of the <i>R4</i> Enhancer in Human and Mouse Chondrocytes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph">(A) Expression by qRT-PCR of <i>CEP250</i> (not significant [n.s.]), <i>GDF5</i> (p&nbsp;&lt; 0.005), and <i>UQCC1</i> (n.s.) in human T/C-28a2 chondrocytes lacking <i>R4</i>.</div><div id="fspara0210" role="paragraph">(B) Expression of <i>CEP250</i> (n.s.), <i>GDF5</i> (p&nbsp;&lt; 0.05), and <i>UQCC1</i> (n.s.) in cells lacking 41&nbsp;bp containing rs6060369 in <i>R4</i>.</div><div id="fspara0215" role="paragraph">(C) <i>In&nbsp;vitro</i> reporter analyses of <i>R4</i>-driven luciferase activity comparing constructs with the OA risk “T” or non-risk “C” variant in T/C-28a2 cells.</div><div id="fspara0220" role="paragraph">(D) PITX1 ChIP on a <i>R4</i> sub-region containing rs6060369 in T/C-28a2 cells showing input (left) and pull-down (right) in image.</div><div id="fspara0225" role="paragraph">(E) UCSC Genome Browser view of mouse <i>R4</i> corresponding to knee ATAC-seq region and PITX1 binding via ChIP-seq (<span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Infante et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Infante, C.R. ∙ Park, S. ∙ Mihala, A.G. ...</div><div><strong>Pitx1 broadly associates with limb enhancers and is enriched on hindlimb cis-regulatory elements</strong></div><div><em>Dev. Biol.</em> 2013; <strong>374</strong>:234-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.11.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84872376764" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23201014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.11.017&amp;pmid=23201014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div></div></figcaption></figure></div><div id="p0150" role="paragraph">We finally edited the rs6060369 orthologous position in C57BL/6J mice to contain the human risk “T” allele. ASE on mice with one <i>R4</i> “T” allele downregulated <i>Gdf5</i> expression in E15.5 distal femur chondrocytes (replacement “T” allele expression 82.3% ± 0.05% of control expression; n&nbsp;= 8/genotype; p&nbsp;=&nbsp;0.0005, <a id="crosref0975" href="#fig7">Figure&nbsp;7</a>A), revealing an endophenotype within an anatomically relevant context. Phenotyping P56 humanized mice revealed that several distal femur features, also dysmorphic in <i>R4</i><sup>−/−</sup> mice (e.g., notch width; lateral condyle sagittal curvature radius), display alterations in <i>R4</i><sup>rs6060369-T, rs6060369-T</sup> mice compared to controls, and in the same direction of effect (<a id="crosref0980" href="#fig7">Figures 7</a>B and 7C; <a id="crosref0985" href="#mmc9">Table S9</a>). While adjusted comparisons did not yield many significant differences, matched unadjusted analyses demonstrate marked differences in condyle curvature and width, and tibial spine size (<a id="crosref6990" href="#mmc9">Table S9</a>). These precise enhancer-mediated alterations are evident in heatmaps of morphological change between wild-type and <i>R4</i><sup>rs6060369-T, rs6060369-T</sup> limbs (<a id="crosref0995" href="#fig7">Figure&nbsp;7</a>B). As a subset of effected measures are correlated with OARSI score in <i>R4</i><sup>−/−</sup> mice (<a id="crosref1000" href="#fig5">Figure&nbsp;5</a>H), and as knee morphology and OA are complex polygenic phenotypes, these findings constitute evidence that the derived “T” at rs6060369 is a causal variant. Moreover, its location in a conserved TF binding site within a functional element that modifies knee development supports our model’s expectations that violations to constraint can confer elevated OA risk.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.02.057/asset/a692925a-4a72-4c86-b285-2217bc07813c/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2020.02.057/asset/25235b62-a8ee-4a11-b00a-b5859c2baf6a/main.assets/gr7.jpg" height="598" width="752" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/a692925a-4a72-4c86-b285-2217bc07813c/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Functional Characterization of the rs6060369 OA Risk Allele in Humanized Replacement Mice</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0235" role="paragraph">(A) ASE assays on E14.5 distal femur chondrocytes from C57BL/6J/129SVJ <i>R4</i><sup><i>rs6060369-T, rs6060369-</i>A</sup> replacement mice (right, p = 0.0005) and C57BL/6J/129SVJ <i>R4</i> heterozygous (<i>R4</i><sup>+/−</sup>) mice (left, p = 0.001).</div><div id="fspara0240" role="paragraph">(B) 3D comparative analysis indicating the locations of largest morphological differences between (left) WT <i>R4</i><sup><i>rs6060369-A, rs6060369-A</i></sup> and HOM <i>R4</i><sup><i>rs6060369-T,&nbsp;rs6060369-T</i></sup> hind limbs, as well as between (right) <i>R4</i><sup>+/+</sup> and <i>R4</i><sup>−/−</sup> hind limbs (zoom-in images focus on inferior distal femur [top], superior proximal tibia [bottom]). Areas with largest variations are highlighted in red (WT &gt; HOM) and dark blue (WT&nbsp;&lt; HOM), with minimal variation in green/yellow.</div><div id="fspara0245" role="paragraph">(C) μCT measurements of indicated features in base pair replacement and <i>R4</i> mice at postnatal days P30, P56, and 1 year.</div><div id="fspara0250" role="paragraph">See also <a id="crosref2280" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref2285" href="#mmc9">Table S9</a>.</div></div></div></div></figcaption></figure></div></section></section><section id="sec-3"><h2>Discussion</h2><section id="sec-3-1"><h3>The Evolutionary Developmental Regulation of the Human Knee</h3><div id="p0155" role="paragraph">To examine the developmental genetic changes involved in the evolution of the human knee, we profiled open chromatin regions from human and mouse developmental samples, comparing patterns of inter- and intra-species sequence evolution across hind- and forelimb elements. Open chromatin regions specific to distal femur and proximal tibia, reflecting site-specific regulatory elements involved in knee chondrogenesis, exhibited signals of the effects of ancient selection during primate and hominin evolution. These include reduced sequence conservation among primates relative to pleiotropic elements, enrichments for human accelerated regions, and significantly reduced diversity relative to pleiotropic elements and certain genome features (e.g., TSS and promoter elements) within humans but not chimps. These findings augment our limited understanding of prenatal knee development, pointing to a series of adaptive regulatory modifications to human knee formation.</div><div id="p0160" role="paragraph">Furthermore, we observed that proximal tibia elements display a stronger reduction in genetic diversity than distal femur, suggestive of greater purifying selection acting on this part of the knee to maintain its anatomical integrity. Importantly, we observed corresponding reductions in morphological variation in features of the proximal tibia relative to distal femur in the OAI dataset. Because we could only study KL&nbsp;= 1 patients (i.e., the lowest score depicting initial OA onset), we cannot rule out confounding issues regarding the time course of OA on different anatomical knee subdomains, although there is currently no consensus as to whether one particular bone end is the primary site of OA and morphological change. These findings on proximal tibia constraint in human genetic and morphological data, together with reduced sequence conservation across primates, suggest a more evolutionarily labile feature for locomotor adaptations which, when a more optimal configuration is achieved, becomes “locked-in” and is subject to negative selection pressures to maintain its morphology.</div><div id="p0165" role="paragraph">We were unable to examine gained human-specific regions, which could also show evidence of regulatory evolution in the human lineage. Similarly, the assumption of conserved chimp regulatory activity for overlapping elements present in mouse and human may not hold in all cases. Resolving these questions, however, would require chimp developmental samples, which is infeasible and unethical. As the majority of regions identified were shared across chondrocyte zones and uninformative on specific anatomical evolution and patterns, use of chondrocytes derived from chimp iPS cells, while providing a general “chondrocyte” signal, would not inform on anatomically specific open chromatin regions.</div><div id="p0170" role="paragraph">Nevertheless, given the expectation that changes to human knee development are mediated at the regulatory level, accelerated regions and more targeted nucleotide changes in existing knee-specific elements (i.e., substitutions modifying TF binding) likely participated in altering conserved regulatory networks. For example, biased occurrence of human-chimp changes intersecting motifs of chondrogenesis-associated factors, suggest that some network components were modified (KLF5 and CTCF) while ancient functional constraints precluded the exploitation of others (FOXP1/FOXP2). Once modified, these regulatory elements fell under new functional constraints to maintain the derived human knee, manifesting in reduced genetic diversity within modern humans at these sites relative to orthologous ape sequences and pleiotropically acting knee elements.</div></section><section id="sec-3-2"><h3>Evolutionary Insights on the Genetic Risk of OA and Causal Variant Discovery</h3><div id="p0175" role="paragraph">Despite evidence of ancient selection, human variants in knee elements show little evidence of recent positive selection (i.e., within the past 30,000 years), display weak population-level differentiation, exhibit no biases in their intersection of predicted TF motifs, and are enriched near genes that impact OA. These point to the predominant effects of genetic drift on functionally constrained regulatory elements in joint disease. To this end, we found that when regulatory constraint is violated (e.g., through the emergence of a variant disrupting a TF motif) there is greater likelihood for pathogenic consequences. Knee elements more variable in humans, but not chimps, are linked with OA gene annotations, knee-specific elements are enriched for OA GWAS variants, and notably, while predicted FOXP1/FOXP2 motifs are protected against human-chimp substitutions, they were also those likely to be disrupted by OA risk variants. We find it interesting that conditional excision of <i>FoxP2</i> using <i>Prx1-cre</i> mice leads to knee defects, changes in articular cartilage thickness, and OA (<span class="dropBlock reference-citations"><a id="body-ref-sref193-2" href="#" role="doc-biblioref" data-xml-rid="bib193" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref193-2" aria-controls="sref193-2">Xu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref193-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">180.</div><div>Xu, S. ∙ Liu, P. ∙ Chen, Y. ...</div><div><strong>Foxp2 regulates anatomical features that may be relevant for vocal behaviors and bipedal locomotion</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:8799-8804</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1721820115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053629492" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30104377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1721820115&amp;pmid=30104377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This suggests that numerous genetic perturbations to constrained FOXP binding sites might underlie human OA risk. Importantly, we also observed impacts at transforming growth factor β (TGF-β) and BMP loci, which not only exhibit signals of adaptive evolution but are involved in knee development (<span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Lyons and Rosen, 2019</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Lyons, K.M. ∙ Rosen, V.</div><div><strong>BMPs, TGFβ, and border security at the interzone</strong></div><div><em>Curr. Top. Dev. Biol.</em> 2019; <strong>133</strong>:153-170</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/bs.ctdb.2019.02.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85062191280" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30902251/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fbs.ctdb.2019.02.001&amp;pmid=30902251" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), OA (<span class="dropBlock reference-citations"><a id="body-ref-sref192" href="#" role="doc-biblioref" data-xml-rid="bib186 bib192" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref192" aria-controls="sref192">Wang et&nbsp;al., 2014; Wu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref192"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">174.</div><div>Wang, R.N. ∙ Green, J. ∙ Wang, Z. ...</div><div><strong>Bone Morphogenetic Protein (BMP) signaling in development and human diseases</strong></div><div><em>Genes Dis.</em> 2014; <strong>1</strong>:87-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gendis.2014.07.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25401122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gendis.2014.07.005&amp;pmid=25401122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">179.</div><div>Wu, M. ∙ Chen, G. ∙ Li, Y.-P.</div><div><strong>TGF-β and BMP signaling in osteoblast, skeletal development, and bone formation, homeostasis and disease</strong></div><div><em>Bone Res.</em> 2016; <strong>4</strong>:16009</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/boneres.2016.9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84979021935" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27563484/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fboneres.2016.9&amp;pmid=27563484" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and possess GWAS variants overlapping knee-specific elements (e.g., <i>BMPR1B</i> and <i>GDF5</i>).</div><div id="p0180" role="paragraph">We formulate a model of human knee evolution that impacts OA risk at the regulatory level (<a id="crosref1015" href="#fig4">Figure&nbsp;4</a>E). Normally, it would be expected that variants violating constraint in particular regulatory sequences (i.e., those integral to knee development, structure, and/or maintenance) would be selected against due to the fitness consequences of an inefficient bipedal gait. However, recently these violations may have become tolerated due to buffers (e.g., improved health, medical care, footwear, etc.). If so, their selective removal on a genome-wide level would become less efficient (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Carnes and Olshansky, 1993</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Carnes, B.A. ∙ Olshansky, S.J.</div><div><strong>Evolutionary Perspectives on Human Senescence</strong></div><div><em>Popul. Dev. Rev.</em> 1993; <strong>19</strong>:793</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2307/2938414" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2307%2F2938414" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), permitting retention and accumulation. Additionally, prehistorically tolerated variants in these elements (i.e., those that only mildly influenced OA risk in the past) may have become more potent risk effectors given complex interactions with modern factors, including obesity, inflammation, and activity changes (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Hunter and Bierma-Zeinstra, 2019</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Hunter, D.J. ∙ Bierma-Zeinstra, S.</div><div><strong>Osteoarthritis</strong></div><div><em>Lancet.</em> 2019; <strong>393</strong>:1745-1759</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2819%2930417-9&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2819%2930417-9&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31034380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2819%2930417-9&amp;pmid=31034380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In both contexts, given typical late-life onset of OA, the deleterious consequences of variants on morphology and joint homeostasis may emerge later, as joint stress compounds over decades, contributing to elevated disease risk. We observed that chondrocyte open-chromatin regions collectively capture a significant portion of OA heritability, and the OAI patient cohort exhibited a greater proportion of alternative alleles in knee elements specifically. These indicate an increased common variant load in constrained elements, consistent with our model (<a id="crosref1030" href="#fig4">Figure&nbsp;4</a>E).</div><div id="p0185" role="paragraph">However, it remains unclear how OA prevalence differs between primates. Studies on baboons reveal similar prevalence to humans (<span class="dropBlock reference-citations"><a id="body-ref-sref118" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118" aria-controls="sref118">Macrini et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref118"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Macrini, T.E. ∙ Coan, H.B. ∙ Levine, S.M. ...</div><div><strong>Reproductive status and sex show strong effects on knee OA in a baboon model</strong></div><div><em>Osteoarthritis Cartilage.</em> 2013; <strong>21</strong>:839-848</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2013.03.003&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2013.03.003&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84877111473" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23499674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2013.03.003&amp;pmid=23499674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), whereas in wild and captive apes OA prevalence is lower than humans and captive baboons (<span class="dropBlock reference-citations"><a id="body-ref-sref115-1" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-1" aria-controls="sref115-1">Lowenstine et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref115-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Lowenstine, L.J. ∙ McManamon, R. ∙ Terio, K.A.</div><div><strong>Comparative Pathology of Aging Great Apes: Bonobos, Chimpanzees, Gorillas, and Orangutans</strong></div><div><em>Vet. Pathol.</em> 2016; <strong>53</strong>:250-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985815612154" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84960119504" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26721908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985815612154&amp;pmid=26721908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This suggests that the nature of OA in humans is quite different compared to African apes. Understanding prevalence is also complicated, because wild catarrhine primates typically show high levels of past trauma, which can be associated with OA and obscure genetic effects (<span class="dropBlock reference-citations"><a id="body-ref-sref115-2" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-2" aria-controls="sref115-2">Lowenstine et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref115-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Lowenstine, L.J. ∙ McManamon, R. ∙ Terio, K.A.</div><div><strong>Comparative Pathology of Aging Great Apes: Bonobos, Chimpanzees, Gorillas, and Orangutans</strong></div><div><em>Vet. Pathol.</em> 2016; <strong>53</strong>:250-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985815612154" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84960119504" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26721908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985815612154&amp;pmid=26721908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We did find that primate sequence conservation and the predicted age of variants significantly associate with OA heritability. The latter has been suggested to represent negative selection acting on complex diseases, wherein recent risk variants have had less time to be removed (<span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Gazal et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Gazal, S. ∙ Finucane, H.K. ∙ Furlotte, N.A. ...</div><div><strong>Linkage disequilibrium-dependent architecture of human complex traits shows action of negative selection</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1421-1427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85030183053" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28892061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3954&amp;pmid=28892061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These findings support our model and suggest that part of human OA heritability may result from how genetic variants violate a phylogenetic sequence constraint retained in primates.</div><div id="p0190" role="paragraph">Our evolutionary model suggests that violations to constraint may arise not only via drift but also through antagonistic pleiotropy. Recent selection acting on other traits may oppose the functional constraints of a derived knee. Furthermore, recent selection may target the same loci used to sculpt the hominin knee because of their recurring importance to development, growth, and homeostasis of the musculoskeletal system (<span class="dropBlock reference-citations"><a id="body-ref-sref160" href="#" role="doc-biblioref" data-xml-rid="bib160 bib27 bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160" aria-controls="sref160">Salazar et&nbsp;al., 2016; Chan et&nbsp;al., 2010; Capellini et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref160"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Chan, Y.F. ∙ Marks, M.E. ∙ Jones, F.C. ...</div><div><strong>Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a Pitx1 enhancer</strong></div><div><em>Science.</em> 2010; <strong>327</strong>:302-305</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1182213" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-74549118236" target="_blank" aria-label="Open Scopus in new tab">Scopus (588)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20007865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1182213&amp;pmid=20007865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">151.</div><div>Salazar, V.S. ∙ Gamer, L.W. ∙ Rosen, V.</div><div><strong>BMP signalling in skeletal development, disease and repair</strong></div><div><em>Nat. Rev. Endocrinol.</em> 2016; <strong>12</strong>:203-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrendo.2016.12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84961215919" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26893264/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrendo.2016.12&amp;pmid=26893264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and underlying complex modular <i>cis</i>-regulatory systems (<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib30 bib47 bib71 bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Chen et&nbsp;al., 2016; DiLeone et&nbsp;al., 1998; Guenther et&nbsp;al., 2008; Indjeian et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Chen, H. ∙ Capellini, T.D. ∙ Schoor, M. ...</div><div><strong>Heads, shoulders, elbows, knees, and toes: modular Gdf5 enhancers control different joints in the vertebrate skeleton</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1006454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1006454" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000624431" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27902701/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1006454&amp;pmid=27902701" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>DiLeone, R.J. ∙ Russell, L.B. ∙ Kingsley, D.M.</div><div><strong>An extensive 3′ regulatory region controls expression of Bmp5 in specific anatomical structures of the mouse embryo</strong></div><div><em>Genetics.</em> 1998; <strong>148</strong>:401-408</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9475750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=9475750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Guenther, C. ∙ Pantalena-Filho, L. ∙ Kingsley, D.M.</div><div><strong>Shaping skeletal growth by modular regulatory elements in the Bmp5 gene</strong></div><div><em>PLoS Genet.</em> 2008; <strong>4</strong>:e1000308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1000308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-58149163279" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19096511/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1000308&amp;pmid=19096511" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Indjeian, V.B. ∙ Kingman, G.A. ∙ Jones, F.C. ...</div><div><strong>Evolving New Skeletal Traits by cis-Regulatory Changes in Bone Morphogenetic Proteins</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:45-56</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2015.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2015.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84954262110" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26774823/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.12.007&amp;pmid=26774823" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While we did not find enrichment of OA variants nor knee-specific regulatory elements in regions of recent positive selection, several loci were identified that fell in selection windows and may represent examples of pleiotropy-driven sequence violation. Here, we show that one OA risk variant (rs6060369), present on a haplotype under recent positive selection for height (<span class="dropBlock reference-citations"><a id="body-ref-sref20-6" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-6" aria-controls="sref20-6">Capellini et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), is located in a classic BMP locus involved in knee development, <i>GDF5</i>, and in a knee enhancer (<i>R4</i>) located using our ATAC-seq strategy. This enhancer and its variant (rs6060369) alter knee regulation <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> and contribute to OA susceptibility.</div><div id="p0195" role="paragraph">The <i>R4</i> enhancer is found in humans, mice, and anoles (<span class="dropBlock reference-citations"><a id="body-ref-sref187" href="#" role="doc-biblioref" data-xml-rid="bib187" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref187" aria-controls="sref187">Wang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref187"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">175.</div><div>Wang, J.S. ∙ Infante, C.R. ∙ Park, S. ...</div><div><strong>PITX1 promotes chondrogenesis and myogenesis in mouse hindlimbs through conserved regulatory targets</strong></div><div><em>Dev. Biol.</em> 2018; <strong>434</strong>:186-195</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2017.12.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85039043376" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29273440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2017.12.013&amp;pmid=29273440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and thus is part of a conserved BMP network in knee development. Enhancer loss in human chondrocytes downregulates <i>GDF5</i> expression, whereas its loss in mice (<i>R4</i><sup>−/−</sup>) downregulates <i>Gdf5</i> expression locally and alters knee shape at the specific anatomical locations of activity in prenatal and early postnatal development. Note, the localized knee defects seen in <i>R4</i><sup>−/−</sup> mice are only a subset of those plaguing <i>Gdf5</i> null knees (<span class="dropBlock reference-citations"><a id="body-ref-sref148-2" href="#" role="doc-biblioref" data-xml-rid="bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148-2" aria-controls="sref148-2">Pregizer et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref148-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">140.</div><div>Pregizer, S.K. ∙ Kiapour, A.M. ∙ Young, M. ...</div><div><strong>Impact of broad regulatory regions on Gdf5 expression and function in knee development and susceptibility to osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2018; <strong>77</strong>:450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2017-212475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85052526710" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29311146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2017-212475&amp;pmid=29311146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), revealing the modular effects that regulatory elements have on morphology and disease risk. While the <i>R4</i><sup>−/−</sup> knee is functional, these mice do not require any experimental insult (e.g., medial meniscus surgical destabilization) to recapitulate several features of human OA progression.</div><div id="p0200" role="paragraph">The human OA risk “T” allele at rs6060369 on the selected haplotype now exhibits frequencies between 40%–70% across Eurasia, making it present in billions of people, and confers a 1.3- to 1.8-fold increase in risk (<span class="dropBlock reference-citations"><a id="body-ref-sref199-4" href="#" role="doc-biblioref" data-xml-rid="bib125 bib199" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref199-4" aria-controls="sref199-4">Miyamoto et&nbsp;al., 2007; Zengini et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref199-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Miyamoto, Y. ∙ Mabuchi, A. ∙ Shi, D. ...</div><div><strong>A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:529-533</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34047103363" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17384641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F2005&amp;pmid=17384641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">185.</div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As it resides at a PITX1 binding site, whose binding occurs in human and mouse, this risk variant alters an evolutionary-constrained transcriptional complex governing <i>R4</i> function, which our model would predict to be causally linked to OA risk. Mice hetero/homozygous for the human risk “T” allele display reductions to <i>Gdf5</i> expression <i>in&nbsp;vivo</i> and present with alterations in some of the same structures and directions of effect as found in <i>R4</i> null mice. Yet, this variant modifies knee shape not so dramatically as to disrupt development or early postnatal locomotion. We suggest that by slightly altering knee shape in mice and humans, abnormal joint biomechanics and excessive cartilage wear and degeneration over time may lead to OA, especially when compounded with other risk factors. However, we cannot rule out other non-biomechanical influences for this variant on knee phenotypes. We believe both novel-engineered mouse models will provide effective and realistic tools for testing OA treatments in the context of genetic and non-genetic risk factors. Moreover, that the single base-pair change has such an observable effect on phenotype lends support for GWAS variant testing in the mouse, especially when it permits the proper 3D and physiological context to observe an effect. Overall, the model proposed here represents a generalized means by which hypotheses linking functional sequences, evolutionary history, and modern-day disease may be made. We present genetic and functional analyses, which seek to test such hypotheses in the context of heritable OA risk, and suggest an evolutionary framework in considering the molecular underpinnings of complex diseases as a useful avenue for scientific research.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0210" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr class="rowsep"><td>PITX1 (G-4) X antibody</td><td>Santa Cruz Biotechnology, NJ, USA</td><td>sc-271435; RRID: <a id="interref440" href="https://antibodyregistry.org/AB_10658969" target="_blank">AB_10658969</a></td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and Virus Strains</b></td></tr><tr class="rowsep"><td><i>E.&nbsp;coli DH10B</i></td><td>ThermoFisher Scientific</td><td>18297010</td></tr><tr class="rowsep"><td colspan="3"><b>Biological Samples</b></td></tr><tr><td>Human E59 embryonic sample</td><td>University of Washington Birth Defects Research Laboratory</td><td>H27242</td></tr><tr class="rowsep"><td>Human Knee MRI Image Data</td><td>OA Biomarkers Consortium Project for the National Institutes of Health (FNIH) Project</td><td><a id="intref0010" href="http://oai.epi-ucsf.org./datarelease/" target="_blank">http://oai.epi-ucsf.org./datarelease/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>Collagenase II</td><td>VWR</td><td>80056–222</td></tr><tr class="rowsep"><td>Nextera Tn5 transposase</td><td>Illumina</td><td>#FC-121-1030</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>SuperScript III First Strand cDNA Synthesis Reaction kit</td><td>Life Technologies</td><td>18090010</td></tr><tr><td>Pyrosequencing PSQ96 HS System</td><td>QIAGEN</td><td>PSQ H96A</td></tr><tr><td>Direct-zol RNA Miniprep kit</td><td>Zymo Research Corporation</td><td>R2072</td></tr><tr><td>Applied Biosystems Power SYBR master mix</td><td>Thermo Fisher Scientific</td><td>4368577</td></tr><tr><td>Lipofectamine 2000</td><td>Invitrogen</td><td>11668–019</td></tr><tr><td>Zymo-SpinTM ChIP Kit</td><td>Zymo Research Corporation</td><td>D5210</td></tr><tr><td>NEBNext High-Fidelity PCR Master Mix</td><td>NEB</td><td>M0541L</td></tr><tr class="rowsep"><td>KAPA Library Quantification Complete Kit</td><td>KAPA</td><td>KK4824</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>E15.5 Mouse Appendicular Skeleton ATAC-seq</td><td>This paper</td><td>GEO: <a id="interref445" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122877" target="_blank">GSE122877</a></td></tr><tr><td>E59 Human Appendicular Skeleton ATAC-seq</td><td>This paper</td><td>GEO: <a id="interref450" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122877" target="_blank">GSE122877</a></td></tr><tr><td>Mouse reference genome NCBI build GRCm38.p6</td><td>Genome Reference Consortium</td><td><a id="intref0015" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.26" target="_blank">https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.26</a></td></tr><tr><td>Human reference genome NCBI build 37, GRCh37</td><td>Genome Reference Consortium</td><td><a id="intref0020" href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/" target="_blank">https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/</a></td></tr><tr><td>BMDC ChIP-seq (H3K27ac)</td><td>Roadmap Epigenomics</td><td>GEO: <a id="interref455" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17312" target="_blank">GSE17312</a></td></tr><tr><td>Human E47 Limb Bud H3K27ac ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref33-3" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-3" aria-controls="sref33-3">Cotney et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref33-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cotney, J. ∙ Leng, J. ∙ Yin, J. ...</div><div><strong>The evolution of lineage-specific regulatory activities in the human embryonic limb</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:185-196</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84879911829" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23827682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.05.056&amp;pmid=23827682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref460" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1039552" target="_blank">GSM1039552</a> <a id="interref480" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1039553" target="_blank">GSM1039553</a></td></tr><tr><td>GM12878 ATAC-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref465" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE47753" target="_blank">GSE47753</a></td></tr><tr><td>Pitx1 ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Infante et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Infante, C.R. ∙ Park, S. ∙ Mihala, A.G. ...</div><div><strong>Pitx1 broadly associates with limb enhancers and is enriched on hindlimb cis-regulatory elements</strong></div><div><em>Dev. Biol.</em> 2013; <strong>374</strong>:234-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.11.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84872376764" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23201014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.11.017&amp;pmid=23201014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref470" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE41591" target="_blank">GSE41591</a></td></tr><tr><td>ENCODE TF ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">ENCODE Project Consortium, 2012</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>ENCODE Project Consortium</div><div><strong>An integrated encyclopedia of DNA elements in the human genome</strong></div><div><em>Nature.</em> 2012; <strong>489</strong>:57-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11247" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865790047" target="_blank" aria-label="Open Scopus in new tab">Scopus (8320)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955616/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11247&amp;pmid=22955616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>See <a id="crosref1090" href="#mmc2">Table S2</a>, Sheets 4-6 for accession codes</td></tr><tr><td>Mouse CTCF ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">DeMare et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>DeMare, L.E. ∙ Leng, J. ∙ Cotney, J. ...</div><div><strong>The genomic landscape of cohesin-associated chromatin interactions</strong></div><div><em>Genome Res.</em> 2013; <strong>23</strong>:1224-1234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.156570.113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881119091" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23704192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.156570.113&amp;pmid=23704192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref475" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE42237" target="_blank">GSE42237</a></td></tr><tr><td>Mouse Sox9 ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref136" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136" aria-controls="sref136">Ohba et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref136"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Ohba, S. ∙ He, X. ∙ Hojo, H. ...</div><div><strong>Distinct Transcriptional Programs Underlie Sox9 Regulation of the Mammalian Chondrocyte</strong></div><div><em>Cell Rep.</em> 2015; <strong>12</strong>:229-243</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.celrep.2015.06.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.celrep.2015.06.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84937521813" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26146088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2015.06.013&amp;pmid=26146088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref485" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69109" target="_blank">GSE69109</a></td></tr><tr><td>Mouse Runx2 ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref124" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124" aria-controls="sref124">Meyer et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref124"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Meyer, M.B. ∙ Benkusky, N.A. ∙ Pike, J.W.</div><div><strong>The RUNX2 cistrome in osteoblasts: characterization, down-regulation following differentiation, and relationship to gene expression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:16016-16031</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M114.552216" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84902207251" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24764292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M114.552216&amp;pmid=24764292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref490" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1027496" target="_blank">GSM1027496</a></td></tr><tr><td>Mouse RARb</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>He, Y. ∙ Gong, L. ∙ Fang, Y. ...</div><div><strong>The role of retinoic acid in hepatic lipid homeostasis defined by genomic binding and transcriptome profiling</strong></div><div><em>BMC Genomics.</em> 2013; <strong>14</strong>:575</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-14-575" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84883128543" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23981290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-14-575&amp;pmid=23981290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref495" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE53736" target="_blank">GSE53736</a>, <a id="interref500" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1299599" target="_blank">GSM1299599</a></td></tr><tr><td>Mouse Shox2 ChIP-seq</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref195" href="#" role="doc-biblioref" data-xml-rid="bib195" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref195" aria-controls="sref195">Ye et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref195"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">181.</div><div>Ye, W. ∙ Song, Y. ∙ Huang, Z. ...</div><div><strong>A unique stylopod patterning mechanism by Shox2-controlled osteogenesis</strong></div><div><em>Development.</em> 2016; <strong>143</strong>:2548-2560</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.138750" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84978700538" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27287812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.138750&amp;pmid=27287812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>GEO: <a id="interref505" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM2177161" target="_blank">GSM2177161</a></td></tr><tr><td>PhyloP20ways (hg38)</td><td>UCSC</td><td><a id="intref0025" href="http://hgdownload.cse.ucsc.edu/goldenpath/hg38/phyloP20way/" target="_blank">http://hgdownload.cse.ucsc.edu/goldenpath/hg38/phyloP20way/</a></td></tr><tr><td>PhyloP100ways (hg38)</td><td>UCSC</td><td><a id="intref0030" href="http://hgdownload.cse.ucsc.edu/goldenpath/hg38/phyloP100way/" target="_blank">http://hgdownload.cse.ucsc.edu/goldenpath/hg38/phyloP100way/</a></td></tr><tr><td>Multiz20way Alignment</td><td>UCSC</td><td><a id="intref0035" href="http://hgdownload.cse.ucsc.edu/goldenpath/hg38/multiz20way/" target="_blank">http://hgdownload.cse.ucsc.edu/goldenpath/hg38/multiz20way/</a></td></tr><tr><td>1000 Genomes Phase 3 Data</td><td>1000 Genomes Project Consortium</td><td><a id="intref0040" href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/" target="_blank">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/</a></td></tr><tr><td>Great Apes Diversity Project (GADP) Data</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref147-3" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147-3" aria-controls="sref147-3">Prado-Martinez et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref147-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Prado-Martinez, J. ∙ Sudmant, P.H. ∙ Kidd, J.M. ...</div><div><strong>Great ape genetic diversity and population history</strong></div><div><em>Nature.</em> 2013; <strong>499</strong>:471-475</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881159598" target="_blank" aria-label="Open Scopus in new tab">Scopus (402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23823723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12228&amp;pmid=23823723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="https://eichlerlab.gs.washington.edu/greatape/data/" target="_blank">https://eichlerlab.gs.washington.edu/greatape/data/</a></td></tr><tr><td>RepeatMasker (hg19)</td><td>UCSC Table Browser</td><td><a id="intref0050" href="https://genome.ucsc.edu/cgi-bin/hgTables" target="_blank">https://genome.ucsc.edu/cgi-bin/hgTables</a></td></tr><tr><td>UK Biobanks Summary Statistics – Self-reported OA</td><td><a id="intref0055" href="http://www.nealelab.is/uk-biobank/" target="_blank">http://www.nealelab.is/uk-biobank/</a></td><td><a id="intref0060" href="http://ldsc.broadinstitute.org/gwashare" target="_blank">http://ldsc.broadinstitute.org/gwashare</a>, <a id="intref0065" href="http://www.nealelab.is/uk-biobank/" target="_blank">http://www.nealelab.is/uk-biobank/</a></td></tr><tr><td>LDSC Baseline LD Model, 1KG3 reference allele frequencies, plink files and weights</td><td>Alkes Group</td><td><a id="intref0070" href="https://data.broadinstitute.org/alkesgroup/LDSCORE" target="_blank">https://data.broadinstitute.org/alkesgroup/LDSCORE</a></td></tr><tr><td>Estonian Biocenter Human Genome Diversity Panel (EGDP)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref137-3" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-3" aria-controls="sref137-3">Pagani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref137-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="https://evolbio.ut.ee/CGgenomes_VCF/" target="_blank">https://evolbio.ut.ee/CGgenomes_VCF/</a></td></tr><tr class="rowsep"><td>Human-Chimp Whole-Genome Alignment (hg38 – panTro4)</td><td>UCSC</td><td><a id="intref0080" href="http://hgdownload.cse.ucsc.edu/goldenPath/hg38/vsPanTro4/" target="_blank">http://hgdownload.cse.ucsc.edu/goldenPath/hg38/vsPanTro4/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Cell Lines</b></td></tr><tr><td>Human embryonic kidney (HEK293T)</td><td>ATCC</td><td>CRL-11268</td></tr><tr><td>Human T/C-28a2 chondrocytes</td><td>Mary Goldring</td><td>SCC042</td></tr><tr class="rowsep"><td>Mouse NIH 3T3</td><td>ATCC</td><td>CRL-1658</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Organisms/Strains</b></td></tr><tr><td><i>R4</i> enhancer lacZ line</td><td>Harvard Genome Modification Facility</td><td>PHC21</td></tr><tr><td><i>R4</i><sup><i>+/−</i></sup> enhancer null mouse line</td><td>Harvard Genome Modification Facility</td><td>R4(R37)</td></tr><tr><td><i>R4</i><sup><i>rs6060369-T</i>/+</sup> single allelic replacement mouse line</td><td>Applied StemCell</td><td>MC140</td></tr><tr><td>C57BL/6J, wildtype</td><td>Jackson Laboratories</td><td>000664</td></tr><tr><td>129X1SVJ, wildtype</td><td>Jackson Laboratories</td><td>000691</td></tr><tr class="rowsep"><td>FVB/NJ, wildtype</td><td>Jackson Laboratories</td><td>001800</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>See <a id="crosref2395" href="#mmc9">Table S9</a> for generated primers used in study</td><td>This Paper</td><td>N/A</td></tr><tr><td>See <a id="crosref2400" href="#mmc9">Table S9</a> for generated sgRNAs used in study</td><td>This Paper</td><td>N/A</td></tr><tr><td>sg1:ACAGGGTGGGAGCGCTCAAT</td><td>Applied StemCell</td><td>N/A</td></tr><tr><td>sg2: GGGTGGGAGCGCTCAATAGG</td><td>Applied StemCell</td><td>N/A</td></tr><tr><td>Chr2-Reg-III-PM.F CAGGATTCGATG<br>GCTGCTATTACTGAATG</td><td>Applied StemCell</td><td>N/A</td></tr><tr><td>Chr2-Reg-III-PM.R CACCTCAGCA<br>GAGAGCG</td><td>Applied StemCell</td><td>N/A</td></tr><tr class="rowsep"><td>Allele-specific Expression primers</td><td>EpigenDx</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>PX458 plasmid</td><td>Addgene</td><td>101731</td></tr><tr><td>pGL4.23 Firefly luciferase</td><td>Promega</td><td>E8411</td></tr><tr class="rowsep"><td>pGL4.74 Renilla luciferase</td><td>Promega</td><td>E6921</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>Bowtie version 2.3.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref107-1" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-1" aria-controls="sref107-1">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref107-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (15006)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td>Samtools version 1.5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref108-1" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-1" aria-controls="sref108-1">Li, 2011a</a><div class="dropBlock__holder" data-db-target-of="sref108-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Li, H.</div><div><strong>Tabix: fast retrieval of sequence features from generic TAB-delimited files</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:718-719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79951993896" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21208982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq671&amp;pmid=21208982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="http://samtools.sourceforge.net/" target="_blank">http://samtools.sourceforge.net/</a></td></tr><tr><td>MACS2 version 2.1.1.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref203-1" href="#" role="doc-biblioref" data-xml-rid="bib203" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref203-1" aria-controls="sref203-1">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref203-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">189.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (5513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="https://github.com/taoliu/MACS" target="_blank">https://github.com/taoliu/MACS</a></td></tr><tr><td>Irreproducible Discovery Rate</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">Li et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Li, Q. ∙ Brown, J.B. ∙ Huang, H. ...</div><div><strong>Measuring reproducibility of high-throughput experiments</strong></div><div><em>Ann. Appl. Stat.</em> 2011; <strong>5</strong>:1752-1779</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1214/11-AOAS466" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79955564639" target="_blank" aria-label="Open Scopus in new tab">Scopus (359)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1214%2F11-AOAS466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0100" href="https://www.encodeproject.org/software/idr/" target="_blank">https://www.encodeproject.org/software/idr/</a>, <a id="intref0105" href="https://codeload.github.com/nboley/idr/tar.gz/2.0.0" target="_blank">https://codeload.github.com/nboley/idr/tar.gz/2.0.0</a></td></tr><tr><td>Bedtools version 2.26.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref149-1" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149-1" aria-controls="sref149-1">Quinlan and Hall, 2010</a><div class="dropBlock__holder" data-db-target-of="sref149-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77951770756" target="_blank" aria-label="Open Scopus in new tab">Scopus (6963)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq033&amp;pmid=20110278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0110" href="https://bedtools.readthedocs.io/en/latest/" target="_blank">https://bedtools.readthedocs.io/en/latest/</a></td></tr><tr><td>HOMER version 4.10</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref79-1" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-1" aria-controls="sref79-1">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref79-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (3844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0115" href="http://homer.ucsd.edu/homer/" target="_blank">http://homer.ucsd.edu/homer/</a></td></tr><tr><td>R base version 3.4.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref150-1" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-1" aria-controls="sref150-1">R Development Core Team, 2008</a><div class="dropBlock__holder" data-db-target-of="sref150-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>R Development Core Team</div><div><strong>R: A Language and Environment for Statistical Computing.</strong></div><div>R Foundation for Statistical Computing, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Development+Core+TeamR%3A+A+Language+and+Environment+for+Statistical+Computing.2008R+Foundation+for+Statistical+Computing" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0120" href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a></td></tr><tr><td>regioneR version 1.8.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Gel et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Gel, B. ∙ Díez-Villanueva, A. ∙ Serra, E. ...</div><div><strong>regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:289-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv562" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84959886262" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26424858/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv562&amp;pmid=26424858" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0125" href="http://bioconductor.org/packages/release/bioc/html/regioneR.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/regioneR.html</a></td></tr><tr><td>GREAT version 3.0.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref121-2" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-2" aria-controls="sref121-2">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref121-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (1686)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0130" href="http://great.stanford.edu/public/html/" target="_blank">http://great.stanford.edu/public/html/</a></td></tr><tr><td>car version 3.0.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Fox and Weisberg, 2019</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Fox, J. ∙ Weisberg, S.</div><div><strong>An R Companion to Applied Regression</strong></div><div>SAGE Publications, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxS.WeisbergAn+R+Companion+to+Applied+RegressionThird+Edition2019SAGE+Publications" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="https://cran.r-project.org/web/packages/car/index.html" target="_blank">https://cran.r-project.org/web/packages/car/index.html</a></td></tr><tr><td>fitdistrplus version 1.0-14</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Delignette-Muller and Dutang, 2015</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Delignette-Muller, M.L. ∙ Dutang, C.</div><div><strong>fitdistrplus: An R Package for Fitting Distributions</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>64</strong>:1-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v064.i04" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84925685098" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v064.i04" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://cran.r-project.org/web/packages/fitdistrplus/index.html" target="_blank">https://cran.r-project.org/web/packages/fitdistrplus/index.html</a></td></tr><tr><td>Tabix version 1.7.24</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref108-2" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-2" aria-controls="sref108-2">Li, 2011a</a><div class="dropBlock__holder" data-db-target-of="sref108-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Li, H.</div><div><strong>Tabix: fast retrieval of sequence features from generic TAB-delimited files</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:718-719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79951993896" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21208982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq671&amp;pmid=21208982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0145" href="http://www.htslib.org/doc/tabix.html" target="_blank">http://www.htslib.org/doc/tabix.html</a></td></tr><tr><td>bcftools version 1.8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref108-3" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-3" aria-controls="sref108-3">Li, 2011a</a><div class="dropBlock__holder" data-db-target-of="sref108-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Li, H.</div><div><strong>Tabix: fast retrieval of sequence features from generic TAB-delimited files</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:718-719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79951993896" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21208982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq671&amp;pmid=21208982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0150" href="https://samtools.github.io/bcftools/bcftools.html" target="_blank">https://samtools.github.io/bcftools/bcftools.html</a></td></tr><tr><td>vcftools 0.1.15</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Danecek et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Danecek, P. ∙ Auton, A. ∙ Abecasis, G. ..., 1000 Genomes Project Analysis Group</div><div><strong>The variant call format and VCFtools</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2156-2158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr330" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79960405019" target="_blank" aria-label="Open Scopus in new tab">Scopus (3479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr330&amp;pmid=21653522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="https://vcftools.github.io/" target="_blank">https://vcftools.github.io/</a></td></tr><tr><td>ProxyFinder</td><td>Raychaudhuri Lab</td><td><a id="intref0160" href="https://github.com/immunogenomics/harmjan/tree/master/ProxyFinder" target="_blank">https://github.com/immunogenomics/harmjan/tree/master/ProxyFinder</a></td></tr><tr><td>LDSC</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55-3" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-3" aria-controls="sref55-3">Finucane et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Finucane, H.K. ∙ Bulik-Sullivan, B. ∙ Gusev, A. ..., ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, RACI Consortium</div><div><strong>Partitioning heritability by functional annotation using genome-wide association summary statistics</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1228-1235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000443086" target="_blank" aria-label="Open Scopus in new tab">Scopus (535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26414678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3404&amp;pmid=26414678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0165" href="https://github.com/bulik/ldsc" target="_blank">https://github.com/bulik/ldsc</a></td></tr><tr><td>Pegas version 0.11</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref139-1" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139-1" aria-controls="sref139-1">Paradis, 2010</a><div class="dropBlock__holder" data-db-target-of="sref139-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Paradis, E.</div><div><strong>pegas: an R package for population genetics with an integrated-modular approach</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:419-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp696" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77949527981" target="_blank" aria-label="Open Scopus in new tab">Scopus (757)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20080509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp696&amp;pmid=20080509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0170" href="https://cran.r-project.org/web/packages/pegas/index.html" target="_blank">https://cran.r-project.org/web/packages/pegas/index.html</a></td></tr><tr><td>ComplexHeatmap version 1.17.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Gu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M.</div><div><strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:2847-2849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw313" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84992187637" target="_blank" aria-label="Open Scopus in new tab">Scopus (664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw313&amp;pmid=27207943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0175" href="https://github.com/jokergoo/ComplexHeatmap" target="_blank">https://github.com/jokergoo/ComplexHeatmap</a></td></tr><tr><td>Factoextra version 1.0.5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref99-1" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-1" aria-controls="sref99-1">Kassambara and Mundt, 2017</a><div class="dropBlock__holder" data-db-target-of="sref99-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Kassambara, A. ∙ Mundt, F.</div><div><strong>Factoextra: extract and visualize the results of multivariate data analyses</strong></div><div>R package version 1.0. 4, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KassambaraF.MundtFactoextra%3A+extract+and+visualize+the+results+of+multivariate+data+analyses2017R+package+version+1.0.+4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0180" href="https://cran.r-project.org/web/packages/factoextra/index.html" target="_blank">https://cran.r-project.org/web/packages/factoextra/index.html</a></td></tr><tr><td>Cluster version 2.0.7-1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref119-1" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-1" aria-controls="sref119-1">Maechler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref119-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Maechler, M. ∙ Rousseeuw, P. ∙ Struyf, A. ...</div><div><strong>cluster: Cluster Analysis Basics and Extensions. R Packages Version 1</strong></div><div>2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MaechlerP.RousseeuwA.StruyfM.HubertK.Hornikcluster%3A+Cluster+Analysis+Basics+and+Extensions.+R+Packages+Version+12018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0185" href="https://cran.r-project.org/web/packages/cluster/index.html" target="_blank">https://cran.r-project.org/web/packages/cluster/index.html</a></td></tr><tr><td>Plink version 1.90</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Chang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Chang, C.C. ∙ Chow, C.C. ∙ Tellier, L.C. ...</div><div><strong>Second-generation PLINK: rising to the challenge of larger and richer datasets</strong></div><div><em>Gigascience.</em> 2015; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13742-015-0047-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84930213392" target="_blank" aria-label="Open Scopus in new tab">Scopus (1923)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13742-015-0047-8&amp;pmid=25722852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0190" href="http://www.cog-genomics.org/plink/1.9/" target="_blank">www.cog-genomics.org/plink/1.9/</a></td></tr><tr><td>Haploview version 4.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref5-1" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-1" aria-controls="sref5-1">Barrett et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref5-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Barrett, J.C. ∙ Fry, B. ∙ Maller, J. ...</div><div><strong>Haploview: analysis and visualization of LD and haplotype maps</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:263-265</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bth457" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-13444269543" target="_blank" aria-label="Open Scopus in new tab">Scopus (10961)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15297300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbth457&amp;pmid=15297300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0195" href="https://www.broadinstitute.org/haploview/haploview" target="_blank">https://www.broadinstitute.org/haploview/haploview</a></td></tr><tr><td>RaxML version 7.3.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref169-1" href="#" role="doc-biblioref" data-xml-rid="bib169" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref169-1" aria-controls="sref169-1">Stamatakis, 2006</a><div class="dropBlock__holder" data-db-target-of="sref169-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Stamatakis, A.</div><div><strong>RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</strong></div><div><em>Bioinformatics.</em> 2006; <strong>22</strong>:2688-2690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btl446" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750403801" target="_blank" aria-label="Open Scopus in new tab">Scopus (11499)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16928733/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtl446&amp;pmid=16928733" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0200" href="https://github.com/stamatak/standard-RAxML" target="_blank">https://github.com/stamatak/standard-RAxML</a></td></tr><tr><td>motifbreakR version 1.6.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Coetzee et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Coetzee, S.G. ∙ Coetzee, G.A. ∙ Hazelett, D.J.</div><div><strong>motifbreakR: an R/Bioconductor package for predicting variant effects at transcription factor binding sites</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3847-3849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84950293634" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26272984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv470&amp;pmid=26272984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0205" href="https://bioconductor.org/packages/release/bioc/html/motifbreakR.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/motifbreakR.html</a></td></tr><tr><td>MotifDb version 1.6.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref165-1" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref165-1" aria-controls="sref165-1">Shannon and Richards, 2014</a><div class="dropBlock__holder" data-db-target-of="sref165-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Shannon, P. ∙ Richards, M.</div><div><strong>MotifDb: An annotated collection of protein-DNA binding sequence motifs. R Package. Version 1</strong></div><div>2014</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.ShannonM.RichardsMotifDb%3A+An+annotated+collection+of+protein-DNA+binding+sequence+motifs.+R+Package.+Version+12014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0210" href="http://bioconductor.org/packages/release/bioc/html/MotifDb.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/MotifDb.html</a></td></tr><tr><td>Haploreg version 4.162</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref188-1" href="#" role="doc-biblioref" data-xml-rid="bib188" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref188-1" aria-controls="sref188-1">Ward and Kellis, 2012</a><div class="dropBlock__holder" data-db-target-of="sref188-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">176.</div><div>Ward, L.D. ∙ Kellis, M.</div><div><strong>HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D930-D934</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84858779229" target="_blank" aria-label="Open Scopus in new tab">Scopus (1288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22064851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr917&amp;pmid=22064851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0215" href="https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php" target="_blank">https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php</a></td></tr><tr><td>Uniprobe</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Hume et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Hume, M.A. ∙ Barrera, L.A. ∙ Gisselbrecht, S.S. ...</div><div><strong>UniPROBE, update 2015: new tools and content for the online database of protein-binding microarray data on protein-DNA interactions</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D117-D122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84922580691" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25378322/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1045&amp;pmid=25378322" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0220" href="http://the_brain.bwh.harvard.edu/uniprobe/about.php/" target="_blank">http://the_brain.bwh.harvard.edu/uniprobe/about.php/</a></td></tr><tr><td>Crossmap version 0.3.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref204-1" href="#" role="doc-biblioref" data-xml-rid="bib204" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref204-1" aria-controls="sref204-1">Zhao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref204-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">190.</div><div>Zhao, H. ∙ Sun, Z. ∙ Wang, J. ...</div><div><strong>CrossMap: a versatile tool for coordinate conversion between genome assemblies</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:1006-1007</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84897373336" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24351709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt730&amp;pmid=24351709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0225" href="http://crossmap.sourceforge.net/" target="_blank">http://crossmap.sourceforge.net/</a></td></tr><tr class="rowsep"><td>BEAGLE5</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Browning et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Browning, B.L. ∙ Zhou, Y. ∙ Browning, S.R.</div><div><strong>A One-Penny Imputed Genome from Next-Generation Reference Panels</strong></div><div><em>Am. J. Hum. Genet.</em> 2018; <strong>103</strong>:338-348</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2018.07.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2018.07.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85056238160" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2018.07.015&amp;pmid=30100085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0230" href="https://faculty.washington.edu/browning/beagle/beagle.html" target="_blank">https://faculty.washington.edu/browning/beagle/beagle.html</a></td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>Allele-specific Expression Assays</td><td>EpigenDx</td><td>N/A</td></tr><tr><td>MicroCT imaging</td><td>Center for Skeletal Research, MGH</td><td>N/A</td></tr><tr><td>Primer oligonucleotide synthesis</td><td>Integrated DNA Technologies, Inc</td><td>N/A</td></tr><tr><td>Human oligonucleotide synthesis</td><td>TwistBioscience</td><td>N/A</td></tr><tr><td>ATAC-sequencing</td><td>Harvard Bauer Core Facility</td><td>N/A</td></tr><tr><td>CRISPR Mi-seq sequencing</td><td>MGH CCIB DNA Core</td><td>N/A</td></tr><tr><td>Mouse R4 element targeting</td><td>Applied StemCell &amp; Harvard Genome Modification Facility</td><td>N/A</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2930234-8" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Lead Contact and Materials Availability</h3><div id="p0215" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Terence Capellini (<a id="intref0235" href="mailto:tcapellini@fas.harvard.edu">tcapellini@fas.harvard.edu</a>)</div><div id="p0220" role="paragraph">During the course of this study, three separate experimental mouse lines (<i>R4</i> enhancer <i>lacZ</i> line, <i>R4</i><sup><i>+/−</i></sup> enhancer null, and <i>R4</i><sup><i>rs6060369-T</i>/+</sup> single allelic replacement) were generated and are available from the Lead Contact after completion of a written Material Transfer Agreement between Harvard University and the requesting institution. Additionally, all ATAC-seq data raw sequencing fastq files and processed peak bed files have been deposited on NCBI GEO (GSE122877). The mouse lines and ATAC-seq datasets, described in this manuscript, are also outlined in the <a id="crosref1260" href="#sec-4-1">Key Resources Table</a>.</div></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><div id="p0225" role="paragraph"><span data-style="underline">Animals</span>: Three mouse lines were generated:<div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="content"><div id="p0230" role="paragraph">(1) The <i>R4</i> enhancer <i>lacZ</i> line contains the mouse <i>R4</i> enhancer cloned upstream of a minimal promoter and the <i>lacZ</i> reporter gene and was generated via transgenesis on the FVB/NJ <i>Mus musculus</i> background by the Harvard University Genome Modification Facility (HUGMF). Upon receipt of founder mice, the line was continually backcrossed on FVB/NJ for at least 4 generations. We followed standard breeding and husbandry protocols to maintain and expand the line; and both males and females were used in pre- and post-natal experimental studies examining expression at joint sites at E14.5, E15.5, P0, P56 and P180 (6&nbsp;months). All breeding, husbandry, euthanasia, and experimental protocols strictly followed IACUC-approved protocols (Capellini: 13-04-161-2) at Harvard University.</div></div></div><div id="u0035" role="listitem"><div class="content"><div id="p0235" role="paragraph">(2) The <i>R4</i><sup><i>+/−</i></sup> enhancer null mouse line contains a deletion of the <i>R4</i> enhancer and was generated on the C57BL/6J <i>Mus musculus</i> background by the Lead Contact and his laboratory. Upon receipt of founder mice from the HUGMF, each of three separate founder lines (constituting lines 3, 7, and 8) was continually backcrossed on C57BL/6J for at least 4 generations. We followed standard breeding and husbandry protocols to maintain and expand each line. Males from all three lines were used in post-natal experimental studies examining morphology and histology at P30 and P365 (1 year), with cohorts of different genotypes per sex being subjected to the above mentioned methods. All breeding, husbandry, euthanasia, and experimental protocols strictly followed IACUC-approved protocols (Capellini: 13-04-161-2) at Harvard University.</div></div></div><div id="u0040" role="listitem"><div class="content"><div id="p0240" role="paragraph">(3) The <i>R4</i><sup><i>rs6060369-T</i>/+</sup> single allelic replacement mouse line contains a single “T” allelic base-pair replacement of the orthologous human rs6060369 variant in the <i>R4</i> enhancer and was generated on the C57BL/6J <i>Mus musculus</i> background by Applied StemCell. Upon receipt of founder mice, the line was continually backcrossed on C57BL/6J for at least 4 generations. We followed standard breeding and husbandry protocols to maintain and expand the line; and both males and females were used in post-natal experimental studies examining morphology and histology at P56, with cohorts of different genotypes per sex being subjected to the above mentioned methods. All breeding, husbandry, euthanasia, and experimental protocols strictly followed IACUC-approved protocols (Capellini: 13-04-161-2) at Harvard University.</div></div></div></div></div><div id="p0245" role="paragraph">Cell lines: Human embryonic kidney (HEK293FT, female) cells were acquired from Dr. Pardis Sabeti (Harvard University and Broad Institute). T/C-28a2 human chondrocyte cells (female) were acquired from Dr. Li Zeng (Tufts University) courtesy of Dr. Mary Goldring (The Hospital for Special Surgery). Both cell lines were cultured at 5% CO<sub>2</sub> at 37°C in Dulbecco’s Modified Eagle’s Medium (DMEM), 10% fetal bovine serum (FBS), and 1% penicillin-streptomycin (P/S). Media was replaced every 2–3&nbsp;days and the cells were sub-cultured every 5&nbsp;days. After receipt of each cell line from source institution, cells were passaged and used in experimental assays without additional STR authentication or mycoplasma testing.</div><div id="p0250" role="paragraph">Human Samples: The human product of conception at gestational day (E) 59, female, was collected from late first-trimester termination through the Laboratory of Developmental Biology at the University of Washington in full compliance with the ethical guidelines of the National Institutes of Health and with the approval of the University of Washington Institutional Review Boards for the collection and distribution of human tissues for research, and Harvard University for the receipt and use of such materials. The Laboratory of Developmental Biology obtained written consent from all tissue donors. The University of Washington Birth Defects Research Laboratory was supported by NIH award number 5R24HD000836 from the Eunice Kennedy Shriver National Institute of Child Health and Human Development. Harvard University IRB determined this sample constitutes Non-Human Subjects Determination Status (Capellini: IRB16-1504). The Lead Contact (Capellini) received no federal funds (e.g., NIH) to acquire, receive, process, or utilize this sample. The human sample was briefly washed in HBSS and transported at 4°C during shipment. Upon arrival the sample was dissected under a light dissection microscope in identical fashion to all mouse samples reported above, and subjected to the ATAC-seq protocol described below, and following approved Harvard University IRB (Capellini: IRB16-1504) and COMS (Capellini: 18-103) protocols.</div><div id="p0255" role="paragraph">Bone and cartilage measurements of 600 knees were derived from the OA Biomarkers Consortium Project for the National Institutes of Health (FNIH) Project, which used the male and females patient knee MRI images from the Osteoarthritis Initiative (see <a id="intref0240" href="http://oai.epi-ucsf.org" target="_blank">http://oai.epi-ucsf.org</a> and <a id="intref0245" href="https://fnih.org/what-we-do/biomarkers-consortium/programs/osteoarthritis-project" target="_blank">https://fnih.org/what-we-do/biomarkers-consortium/programs/osteoarthritis-project</a>). The OAI is a public–private partnership between the NIH (contracts N01-AR-2-2258, N01-AR-22259, N01-AR-2-2260, N01-AR-2-2261, and N01-AR-2-2262) and private funding partners (Merck Research Laboratories, Novartis Pharmaceuticals, GlaxoSmithKline, and Pfizer, Inc.) and is conducted by the OAI Study Investigators. Private sector funding for the OAI is managed by the Foundation for the NIH. The OAI was also funded by the National Institute of Arthritis and Musculoskeletal and Skin Diseases (grant HHSN-268201000019C). Harvard University IRB determined the manner in which these sample will be utilized constitutes Non-Human Subjects Determination Status (Capellini: IRB19-0019).</div></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Animal Models</h4><div id="p0260" role="paragraph"><b><i>R4</i></b><sup><b><i>+/−</i></b></sup> <b>enhancer null line:</b> The following steps were used to generate <i>R4</i> enhancer null mice:<div id="ulist0020" role="list"><div id="u0045" role="listitem"><div class="content"><div id="p0265" role="paragraph">1. sgRNA guide design for targeting <i>R4</i></div></div></div></div></div><div id="p0270" role="paragraph">We used <a id="intref0250" href="http://zlab.bio/guide-design-resources" target="_blank">http://zlab.bio/guide-design-resources</a> to design sgRNAs, and ordered sgRNA oligo from Integrated DNA Technologies, Inc. The guide sequence information is provided in <a id="crosref1265" href="#mmc9">Table S9</a>.<div id="ulist0025" role="list"><div id="u0050" role="listitem"><div class="content"><div id="p0275" role="paragraph"><i>2.</i> sgRNAs cloning, cell transfection, and deletion of the <i>R4</i> element <i>in&nbsp;vitro</i>.</div></div></div></div></div><div id="p0280" role="paragraph">We cloned sgRNAs targeting <i>R4</i> into the PX458 plasmid following protocol (<span class="dropBlock reference-citations"><a id="body-ref-sref153-1" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-1" aria-controls="sref153-1">Ran et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref153-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (3965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (step 5-part B) and tested the efficiency of sgRNAs <i>in&nbsp;vitro</i> in mouse NIH 3T3 cells. Briefly, NIH 3T3 cells were maintained in DMEM (GIBCO) supplied with 10% Fetal Calf Serum (GIBCO) and 1% Pen/Strep (0.025%). Cells were passaged once every 3&nbsp;days. We seeded 0.3x10<sup>6</sup> cells per well of a 6-well plate at 1&nbsp;day prior to transfection. On the day of transfection, cell culture media was replaced with fresh complete media. Next, 1&nbsp;μg sgRNA1-PX458 and 1&nbsp;μg sgRNA2-PX458 were combined with 250&nbsp;μL Opti-MEM (Invitrogen); while 6&nbsp;μL Lipofectamine® 2000 (Thermo Fisher Scientific) and 250&nbsp;μL Opti-MEM (Invitrogen) were combined. Both reactions were incubated at room temperature for 5&nbsp;minutes, and then combined and incubated for 25&nbsp;minutes. We added the entire 500&nbsp;μL into each well, and placed the transfected cells at 37°C with 5% CO<sub>2</sub>. After 48 hours of transfection, we observed the plates under a GFP-microscope (Zeiss, Inc), noting strong GFP signals at ∼70% efficiency. Cells were then harvested, and DNA extracted by using E.Z.N.A Tissue DNA Kit according to the manufacture’s protocols. Next, we tested the efficiency of sgRNAs by using touch down PCR; PCR components included 10&nbsp;μL Taq PCR Master Mix (BioBasic), 1&nbsp;μL (5&nbsp;μM) Forward Primer, 1&nbsp;μL (5&nbsp;μM) Reverse Primer, and 1&nbsp;μL DDH20; Touch-down PCR cycling conditions were initial 94°C for 4&nbsp;minutes; 35 cycles of 94°C for 30 s, 58-52°C for 30&nbsp;s ramp down 1°C every 5 cycles, 72°C for 1&nbsp;minute; and 72°C for 10&nbsp;minutes; Primer information listed in <a id="crosref1275" href="#mmc9">Table S9</a>. PCR products were then run on a 1% agarose gel containing 0.5&nbsp;μg/mL of Ethidium bromide at 100V for 1 hour. The DNA bands visualized on the gel indicated the targeted, deleted region of 1.56kb encompassing the <i>R4</i> element. After gel purification using the E.Z.N.A Gel Extraction Kit, amplicons were sent to Eton Bioscience Inc for sequencing. Sequencing confirmed that the <i>R4</i> element was deleted <i>in&nbsp;vitro</i>.<div id="ulist0030" role="list"><div id="u0055" role="listitem"><div class="content"><div id="p0285" role="paragraph"><i>3.</i> Deletion of the <i>R4</i> element <i>in&nbsp;vivo</i></div></div></div></div></div><div id="p0290" role="paragraph">Following previously published protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref153-2" href="#" role="doc-biblioref" data-xml-rid="bib20 bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-2" aria-controls="sref153-2">Capellini et&nbsp;al., 2017; Ran et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref153-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (3965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) <i>in&nbsp;vitro</i> transcription of sgRNAs targeting <i>R4</i> was performed with products then sent to HUGMF for microinjection into wild-type C57BL/6J pronuclei. After, we recovered 4 F0 mice and in all cases deletion of 1.56kb region encompassing <i>R4</i> in founders was confirmed on DNA extracted from mouse tails using the strategy described above with PCR primers listed in <a id="crosref1280" href="#mmc9">Table S9</a>. These F0 founders were then crossed once again to wild-type C57BL/6J mice to confirm allelic transmission, upon which we recovered three separate functional <i>R4</i><sup><i>+/−</i></sup> F1 lines (constituting lines 3, 7, and&nbsp;8). All three lines were then backcrossed for 4 generations to C57BL/6J wild-type mice to purify each line and remove possible off-target effects. Additionally, given the finding that sgRNAs can cause local alterations (deletions/insertions/ rearrangements/substitutions) within a 5kb vicinity of the intended target site (<span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Kosicki et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Kosicki, M. ∙ Tomberg, K. ∙ Bradley, A.</div><div><strong>Repair of double-strand breaks induced by CRISPR-Cas9 leads to large deletions and complex rearrangements</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:765-771</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt0918-899c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053085220" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30010673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt0918-899c&amp;pmid=30010673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), homozygous null mice were screened using local primers targeting this interval followed by Sanger sequencing, resulting in no non-intended modifications.</div><div id="p0295" role="paragraph"><b><i>R4</i></b><sup><b><i>rs6060369-T</i>/+</b></sup> <b>single allelic line:</b> Initially, this variant position was examined across all sequenced strains of mice in order to find evidence of the “T” variant change. Interestingly, in C57BL/6J mice as well as all other 88 mouse strains reported in the MGI strain database, the “T” variant is absent, but instead these mice naturally harbor a different, fixed ancestral variant “A” compared to the primate ancestral “C” allele found in some humans. We next examined patterns of conservation across sequenced vertebrates to identify instances of the derived “T” allele as compared to the mouse “A” allele and human ancestral “C” allele. As the <i>R4</i> element is highly conserved through Coelacanth, 83 sequenced Sarcopterygii had detectable orthology at this element, with 43 species possessing the fixed mouse “A” variant, 36 species possessing the human ancestral “C” variant (including catarhine primates), 1 possessing a deletion of the element (cape elephant shrew in Afrotheria), and only 3 species [squirrel monkey (a single point mutation), brush tailed rat (a two base mutation), <i>X.&nbsp;tropicalis</i> (general poor sequence orthology)] possessing some form of the “T” variant. This indicated that in general the “T” variant is especially novel in humans compared to all sequenced catarhines including hominoids, and extremely rare across sequenced vertebrates. Therefore, we sought to replace the orthologous “A” variant, which appears to be tolerated evolutionarily across multiple Sarcopterygii and is fixed within laboratory mice, with the derived human risk allele “T.”</div><div id="p0300" role="paragraph">Targeting of the mouse ortholog of the human rs6060369 variant for replacement with the human risk variant at this locus was performed in collaboration with Applied StemCell. Specifically, the goal was to use CRISPR-Cas9 gene targeting to introduce a single base-pair change at chr2:155,863,176 in the C57BL/6J mouse strain, in which the ancestral “A” variant was substituted by the human risk “T.” To achieve this, a mixture containing <i>in&nbsp;vitro</i> transcribed active sgRNAs (sg1:ACAGGGTGGGAGCGCTCAAT and sg2: GGGTGGGAGCGCTCAATAGG), a single-stranded oligo deoxynucleotide (ssODN), and Cas9 protein was first microinjected into C57BL/6J embryos acquired from Jackson laboratory. Next, initial founder mice were screened for mutations in the region by extracting DNA from tail tissues and then PCR amplifying the target region using two primers (Chr2-Reg-III-PM.F CAGGATTCGATGGCTGCTATTACTGAATG and Chr2-Reg-III-PM.R CACCTCAGCAGAGAGCG) and the following conditions: All PCR amplifications were prepared in 25&nbsp;μL using MyTaqTM Red Mix (Bioline, Cat#, BIO-25044) and the amplifications were carried out using the following program: 95°C for 2&nbsp;minutes; 35 cycles of 95°C for 15 s, 60°C for 15 s, 72°C for 1+ minutes depending on amplicon size; and 72°C for 5&nbsp;minutes. Finally, next generation sequencing (NGS) libraries were prepared and subsequently sequenced using Illumina PCR genotyping to identify chr2:155,863,176A &gt; T point mutation.</div><div id="p0305" role="paragraph">After microinjecting greater than 300 embryos, and screening of five potential founders, we recovered one F0 mouse with the “T” replacement confirmed using NGS on DNA extracted from its tail. As assessed by sequencing, this F0 founder lacked any other artifactual alteration to the locus. This F0 was then crossed once again to wild-type C57BL/6J mice to confirm allelic transmission, upon which we recovered a number of transmitting F1 C57BL/6J <i>R4</i><sup><i>rs6060369-+, rs6060369-T</i></sup> males and females. These mice were then backcrossed for four generations to C57BL/6J wild-type mice to remove possible off-target effects. Additionally, given the finding that sgRNAs can cause local alterations (deletions/insertions/ rearrangements/substitutions) within a 5kb vicinity of the intended target site (<span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Kosicki et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Kosicki, M. ∙ Tomberg, K. ∙ Bradley, A.</div><div><strong>Repair of double-strand breaks induced by CRISPR-Cas9 leads to large deletions and complex rearrangements</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:765-771</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt0918-899c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053085220" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30010673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt0918-899c&amp;pmid=30010673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), homozygous replacement mice were screened using local primers targeting this interval followed by Sanger sequencing, resulting in no non-intended modifications. Subsequently, C57BL/6J <i>R4</i><sup><i>rs6060369-+, rs6060369-T</i></sup> mice were intercrossed to generate C57BL/6J <i>R4</i><sup><i>rs6060369-+, rs6060369-+</i></sup>, <i>R4</i><sup><i>rs6060369-+, rs6060369-T</i></sup>, and <i>R4</i><sup><i>rs6060369-T, rs6060369-T</i></sup> mice for all downstream functional experiments.</div></section><section id="sec-4-4-2"><h4>Phenotypic Assessment (of mouse lines)</h4><div id="p0310" role="paragraph"><b>MicroCT and morphometrics:</b> To quantify phenotypes in <i>R4</i><sup><i>+/+</i></sup>, <i>R4</i><sup><i>+/−</i></sup>, and <i>R4</i><sup><i>−/−</i></sup> mice, right femur and tibia of 15 male P30 mice (5 wild-type, 5 heterozygous, and 5 homozygous) and 35 male 1 year old mice (6 wild-type, 15 heterozygous and 14 homozygous) were scanned using high-resolution Micro-Computed Tomography (μCT40, SCANCO Medical AG, Brüttisellen, Switzerland) at the Center for Skeletal Research (CSR) at Massachusetts General Hospital, and in the laboratory of Vicki Rosen at the Harvard School of Dental Medicine. To quantify phenotypes in <i>R4</i><sup><i>rs6060369-+, rs6060369-+</i></sup>, <i>R4</i><sup><i>rs6060369-+, rs6060369-T</i></sup>, and <i>R4</i><sup><i>rs6060369-T, rs6060369-T</i></sup> mice, right femur and tibia of 20 male/female P56 (6-8 per genotype) were also scanned. Scan parameters were: 12&nbsp;μm<sup>3</sup> isotropic voxel size, 70 kVp peak X-ray tube intensity, 114 mA X-ray tube current, and 200&nbsp;ms integration time. While there is an effect of sex on knee osteoarthritis risk in humans, with females generally showing higher risk than males, at the <i>GDF5</i> locus there has been no identified sex-specific effect on osteoarthritis risk (<span class="dropBlock reference-citations"><a id="body-ref-sref199-5" href="#" role="doc-biblioref" data-xml-rid="bib125 bib199" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref199-5" aria-controls="sref199-5">Miyamoto et&nbsp;al., 2007; Zengini et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref199-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Miyamoto, Y. ∙ Mabuchi, A. ∙ Shi, D. ...</div><div><strong>A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:529-533</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34047103363" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17384641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F2005&amp;pmid=17384641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">185.</div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore chose to only include males in our study. Digital Imaging and Communications in Medicine (DICOM) images were exported for measurements of several femoral and tibial anatomical indices in Osirix MD v7.5 (Pixemo SARL, Bernex, Switzerland), using clinically established protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref168" href="#" role="doc-biblioref" data-xml-rid="bib29 bib75 bib85 bib161 bib168" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref168" aria-controls="sref168">Charles et&nbsp;al., 2013; Hashemi et&nbsp;al., 2008; Howell et&nbsp;al., 2010; Schwartz, 1989; Sonnery-Cottet et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref168"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Charles, M.D. ∙ Haloman, S. ∙ Chen, L. ...</div><div><strong>Magnetic resonance imaging-based topographical differences between control and recurrent patellofemoral instability patients</strong></div><div><em>Am. J. Sports Med.</em> 2013; <strong>41</strong>:374-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0363546512472441" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84873370967" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23371940/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0363546512472441&amp;pmid=23371940" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Hashemi, J. ∙ Chandrashekar, N. ∙ Gill, B. ...</div><div><strong>The geometry of the tibial plateau and its influence on the biomechanics of the tibiofemoral joint</strong></div><div><em>J.&nbsp;Bone Joint Surg. Am.</em> 2008; <strong>90</strong>:2724-2734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2106/JBJS.G.01358" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-57349117781" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19047719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2106%2FJBJS.G.01358&amp;pmid=19047719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Howell, S.M. ∙ Howell, S.J. ∙ Hull, M.L.</div><div><strong>Assessment of the radii of the medial and lateral femoral condyles in varus and valgus knees with osteoarthritis</strong></div><div><em>J.&nbsp;Bone Joint Surg. Am.</em> 2010; <strong>92</strong>:98-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2106/JBJS.H.01566" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-73949146227" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20048101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2106%2FJBJS.H.01566&amp;pmid=20048101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Schwartz, H.J.</div><div><strong>Effect of chronic chromolyn sodium therapy in a beautician with occupational asthma</strong></div><div><em>J.&nbsp;Occup. Med.</em> 1989; <strong>31</strong>:112-114</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2496209/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=2496209" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">159.</div><div>Sonnery-Cottet, B. ∙ Archbold, P. ∙ Cucurulo, T. ...</div><div><strong>The influence of the tibial slope and the size of the intercondylar notch on rupture of the anterior cruciate ligament</strong></div><div><em>J.&nbsp;Bone Joint Surg. Br.</em> 2011; <strong>93</strong>:1475-1478</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1302/0301-620X.93B11.26905" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-81155125190" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22058297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1302%2F0301-620X.93B11.26905&amp;pmid=22058297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1295" href="#figs5">Figure&nbsp;S5</a>A). Micro-CT images were also used to generate 3D models of each bones using bone segmentation process in a commercially available image processing software (Mimics v17.0, Materialise). The 3D models were then imported to 3-matic software package (v9.0, Materialise) and then co-registered together using a global n-point registration technique. The registered models from the wild-type and homozygous mice was then used to generate 3D heatmaps indicating the geometrical differences between the genotypes at P30 and at 1 year. The heatmaps were generated by calculating the distance between the corresponding points in co-registered models, where dark blue indicates the maximum deviation in the negative direction and red indicates the maximum deviation in the positive direction.</div><div id="p0315" role="paragraph"><b>3D X-ray microscopy (XRM)</b>: Hind limbs (n&nbsp;= 6 <i>R4</i><sup><i>+/+</i></sup> n&nbsp;= 10 <i>R4</i><sup><i>−/−</i></sup>) were fixed in 10% neutral buffered formalin (NBF) for 24h, then carefully disarticulated under a dissection microscope to expose the articular cartilage of the femur and tibia, and subsequently stored in 70% ethanol solution. Femora and tibiae were incubated in 1% phosphotungstic acid (PTA) contrast enhancer at room temperature for 12-18h before imaging (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Das Neves Borges et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Das Neves Borges, P. ∙ Forte, A.E. ∙ Vincent, T.L. ...</div><div><strong>Rapid, automated imaging of mouse articular cartilage by microCT for early detection of osteoarthritis and finite element modelling of joint mechanics</strong></div><div><em>Osteoarthritis Cartilage.</em> 2014; <strong>22</strong>:1419-1428</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2014.07.014&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2014.07.014&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84908179041" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25278053/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2014.07.014&amp;pmid=25278053" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Samples were attached and enclosed onto custom specimen chamber and imaged in a high-resolution X-ray microscopy (XRM) scanner (Zeiss, Xradia 520 Versa) employing low energy (40 kVp voltage, 3W power) X-ray source. XRM scans (2001 projections, 360o scan) were performed using a 4X objective, with resolution of 4.4&nbsp;μm/pixel. The 3D XRM datasets of the isolated femurs and tibia were imported into Amira software Version 6.0.1 (FEI, Portland, USA) for image reconstruction and visualization of cartilage injury between the HOM and WT mice.</div><div id="p0320" role="paragraph"><b>X-gal staining:</b> Whole-mount staining for β-galactosidase activity was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref20-8" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-8" aria-controls="sref20-8">Capellini et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref20-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gestational day (E) 14.5 <i>R4</i> enhancer <i>lacZ</i> positive embryos were hemisected and fixed in 4% paraformaldehyde (PFA) (Sigma, 158127) at 4°C, according to gestational-day guidelines. Fixed embryos were washed three times in wash buffer and stained for 16–24 hours in the dark with 1&nbsp;mg/ml X-gal (Sigma, B4252) in staining buffer at room temperature. Adult <i>R4</i> enhancer <i>lacZ</i> positive mice were skinned and eviscerated, and the limbs were removed. Each limb then had its most superficial muscles removed and its joint capsules gently pierced to permit solutions from entering into the joint cavity. Fixation and staining times were adjusted accordingly for these post-natal specimens. After staining, embryos and post-natal limbs were briefly rinsed in wash buffer and post-fixed in 4% PFA at 4°C for 5 hours. For sectioning, X-gal stained embryos were placed first in sucrose and then embedded in gelatin/sucrose solution and cryo-sectioned at 25&nbsp;μm. Sections were counterstained with Nuclear Fast Red (Vector labs, #H-3403).</div><div id="p0325" role="paragraph"><b>Assessment of osteoarthritis:</b> To assess the cartilage status in <i>R4</i> enhancer null mice, right hind limbs of 12 male P30 mice (3 wild-type and 9 homozygous) and right hind limbs of 20 male 1 year old mice (6 wild-type and 14 homozygous) were obtained for histological analysis. Given that we observed significant changes in shape at P30 and 1 year in <i>R4</i><sup><i>−/−</i></sup> (with only a trend of such effect in <i>R4</i><sup><i>+/−</i></sup> mice) we chose to only examine wild-type and <i>R4</i><sup><i>−/−</i></sup> mice for signs of osteoarthritis at P30 and 1 year. The also held true for <i>R4</i><sup><i>rs6060369+/−</i></sup> mice compared to <i>R4</i><sup><i>rs6060369</i></sup> homozygotes. After removal of the skin and excess muscle, all limbs were fixed in 10% neutral buffered formalin for 24 hours. The 1 year old mice were stored in 70% ethanol until completion of the micro-computed tomography. Note that each litter of mice consisted of all genotypes, and each litter was stored in 70% ethanol for the exact same duration. The limbs were then decalcified in 14% Ethylenediaminetetraacetic acid (EDTA) at pH 7.5 for 4 and 8&nbsp;days for P30 and 1 year old specimen, respectively. The specimens were then dehydrated by incubation in 70% ethanol, 95% ethanol, 100% ethanol, 1:1 ethanol-xylene-solution, 100% xylene (all at room temperature), and paraffin (at 60°C) (Sigma). The latter was exchanged once before embedding. Each incubation period was 12 hours for P30 and 1&nbsp;day for 1 year old specimens. The specimens were then bisected in a frontal plane and embedded. The formalin-fixed paraffin-embedded tissues were then cut into 6&nbsp;μm sections and a representative section from the center of each knee was selected for alcian blue staining at pH 2.5 with nuclear fast red counterstain. Alcian blue was selected for the evaluation of cartilage integrity in this study, as it is commonly used to stain skeletal preparations in developmental biology investigations (in combination with Alizarin Red S), and thus allows highlighting cartilaginous elements in a coherent color across all of our experiments. The P30 specimens were stained in one batch at Boston Children’s Hospital, while the 1 year old specimens were stained at Tufts Medical Center due to the fact that one of the co-authors (JS) moved institutions. All 1 year specimens were initially stained in one batch. If a specimen’s cartilage surface was not fully visualized due to artifactual folding of the cartilage on the slide, another adjacent section of the specimen was submitted for staining (batch 2). If required, a second additional section was submitted for staining (batch 3). For all three batches the identical staining protocol was utilized ad no difference in staining intensity was observed between the three batches. Embedding, sectioning, and staining were performed without knowledge of the genotype.</div><div id="p0330" role="paragraph">Alcian blue stained sections were then scored by one reader (JTS) who was blinded to the genotype, according to the Osteoarthritis Research Society International (OARSI) histopathology initiative recommendations for the histological assessment of osteoarthritis in the mouse (<span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Glasson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Glasson, S.S. ∙ Chambers, M.G. ∙ Van Den Berg, W.B. ...</div><div><strong>The OARSI histopathology initiative - recommendations for histological assessments of osteoarthritis in the mouse</strong></div><div><em>Osteoarthritis Cartilage.</em> 2010; <strong>18</strong>:S17-S23</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2010.05.025&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2010.05.025&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77956924858" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20864019/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2010.05.025&amp;pmid=20864019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The intra-observer reproducibility has been shown to be high for this scoring system (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Glasson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Glasson, S.S. ∙ Chambers, M.G. ∙ Van Den Berg, W.B. ...</div><div><strong>The OARSI histopathology initiative - recommendations for histological assessments of osteoarthritis in the mouse</strong></div><div><em>Osteoarthritis Cartilage.</em> 2010; <strong>18</strong>:S17-S23</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2010.05.025&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2010.05.025&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77956924858" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20864019/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2010.05.025&amp;pmid=20864019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Briefly, medial tibial plateau (MTP), medial femoral condyle (MFC), lateral tibial plateau (LTP), and lateral femoral condyle (LFC) were assigned a semiquantitative score ranging from 0 to 6, with 0 representing a normal articular cartilage and 6 representing vertical clefts/erosion to the calcified cartilage extending &gt; 75% of the articular surface. In addition, a score of 0.5 indicates the loss of alcian blue staining without structural changes. The sum score of all four joint surfaces was determined (referred to as OARSI Score in the manuscript) and selected as primary outcome. In addition, osteophytes were scored according to the OARSI recommendations, by assigning one semiquantitative score to the entire joint ranging from 0 to 3, with 0 indicating absence of osteophytes and 3 indicating severe osteophyte formation. Comparisons were only made between genotypes at single time points and not across time points.</div><div id="p0335" role="paragraph">One P30 limb (wild-type) disintegrated during processing and was omitted. A total of four individual joint surfaces were obscured despite re-staining and the respective scores were omitted from the sum OARSI Score (two wild-type and two homozygous).</div><div id="p0340" role="paragraph"><b>Allele-specific expression analyses (ASE):</b> Timed matings were established between C57BL/6J <i>R4</i><sup><i>+</i>/<i>−</i></sup> heterozygous mice and 129X1/SVJ <i>R4</i><sup><i>+/+</i></sup>wild-type mice. Likewise, timed matings were established between C57BL/6J <i>R4</i><sup><i>rs6060369-+, rs6060369-T</i></sup> heterozygous mice and 129X1/SvJ <i>R4</i> <sup><i>rs6060369-+, rs6060369-+</i></sup> wild-type mice. Pregnant females were sacrificed according to IACUC-approved protocols to acquire E15.5 embryos. Right and left hind limbs were stripped of all soft tissues, and distal femoral and proximal tibia chondrogenic tissues were dissected from each limb, with each tissue (e.g., left and right distal femur) placed in TRIzol reagent (15596-026, Ambion by Life Technologies) with a homogenizer bead, and then homogenized for 2&nbsp;minutes with a tissue homogenizer (QIAGEN). Samples were then stored at −80°C. RNA was then isolated with TRIzol reagent and a Direct-zol™ RNA Miniprep Kit (supplied with DNase I, Zymo) (For <i>R4</i> enhancer tests<i>: n&nbsp;=</i> 4 distal femoral biological replicates and; n&nbsp;= 4 proximal tibia biological replicates; for <i>R4</i> rs6060369 variant tests: n&nbsp;= 8 distal femur biological replicates; n&nbsp;= 8 proximal tibia biological replicates). Samples were then run on a Bioanalyzer to ensure RNA integrity numbers greater than 8. These RNA samples were then reverse transcribed with a SuperScript III First Strand cDNA Synthesis Reaction kit (18090010, Life Technologies) according to the manufacturer’s recommendations. Independently, tails from each embryo were used for <i>R4</i> enhancer and <i>R4</i> rs6060369 genotyping as described above.</div><div id="p0345" role="paragraph">cDNA samples were then sent to EpigenDx for allele-specific expression-assay design and execution. SNPs in the coding regions of <i>Gdf5</i> (rs27340038), <i>Cep250</i> (rs27339949), and <i>Uqcc1</i> (rs3684985) were identified by EpigenDx. Pyrosequencing for SNP genotyping (PSQ H96A, QIAGEN Pyrosequencing) is a real-time sequencing–based DNA analysis that quantitatively determines the genotypes of single or multiple mutations in a single reaction. Briefly, 1&nbsp;ng of sample cDNA was used for PCR amplification. PCR was performed with 10X PCR buffer (QIAGEN) with 3.0&nbsp;mM MgCl<sub>2</sub>, 200&nbsp;M each dNTP, 0.2&nbsp;μM each of the forward and reverse primers (available through EpigenDx), and 0.75&nbsp;U of HotStar DNA polymerase (QIAGEN) per 30&nbsp;μL reaction. The PCR cycling conditions were 94°C for 15&nbsp;minutes; 45 cycles of 94°C for 30 s, 60°C for 30 s, 72°C for 30 s; and 72°C for 5&nbsp;minutes. One of the PCR-primer pairs was biotinylated to convert the PCR product to single-stranded DNA-sequencing templates with streptavidin beads and the PyroMark Q96 Vacuum Workstation. 10&nbsp;μL of the PCR products was bound to streptavidin beads, and the single strand containing the biotinylated primer was isolated and combined with a specific sequencing primer (available through EpigenDx). The primed single-stranded DNA was sequenced with a Pyrosequencing PSQ96 HS System (QIAGEN Pyrosequencing) according to the manufacturer’s instructions (QIAGEN Pyrosequencing). The genotypes of each sample were analyzed with Q96 software AQ module (QIAGEN Pyrosequencing).</div><div id="p0350" role="paragraph"><b>qRT-PCR analysis (<i>GDF5, CEP250, UQCC1</i>):</b> Total RNA was extracted from T/C-28a2 cells (n&nbsp;= 3 biological replicates, with 4 technical replicates per experiment per condition) and prepared using the Trizol Reagent (Thermo Fisher Scientific, Springfield Township, New Jersey) and Direct-zol RNA Miniprep kit (ZYMO). Two micrograms of total RNA were used to synthesize first-strand cDNA using SuperScript III First-Strand Synthesis System (Thermo Fisher Scientific). qRT-PCR analysis was then performed with specific primers and Applied Biosystems Power SYBR master mix (Thermo Fisher Scientific) with <i>GAPDH</i> house-keeping gene as an internal control. Primers used for qRT-PCR are listed in <a id="crosref1320" href="#mmc9">Table S9</a>.</div></section><section id="sec-4-4-3"><h4>Cloning Methods</h4><div id="p0355" role="paragraph"><b><i>R4 lacZ</i> reporter line:</b> Cloning of constructs used to generate <i>R4 lacZ</i> stable-expressing reporter mouse lines is described above.</div><div id="p0360" role="paragraph"><b><i>R4</i></b><sup><b><i>+/−</i></b></sup> <b>enhancer null line:</b> Cloning of constructs used to generate CRISPR-Cas9 sgRNA for targeting <i>R4 in&nbsp;vitro</i> and <i>in&nbsp;vivo</i> is described above.</div><div id="p0365" role="paragraph"><b><i>R4</i></b><sup><b><i>rs6060369-T</i>/+</b></sup> <b>single allelic line:</b> Cloning of constructs used to generate CRISPR-Cas9 sgRNA for targeting rs6060369 ortholog <i>in&nbsp;vivo</i> was performed by Applied StemCell (see above).</div><div id="p0370" role="paragraph"><b>sgRNAs targeting <i>R4</i>/rs6060369 in cell lines:</b> All sgRNAs flanking the human <i>R4</i> regulatory element or a smaller 41&nbsp;bp region within <i>R4</i> containing the osteoarthritis-risk variant rs6060369 were designed using MIT CRISPR Tools (<a id="intref0255" href="http://zlab.bio/guide-design-resources" target="_blank">http://zlab.bio/guide-design-resources</a>), synthesized by Integrated DNA Technologies, Inc (Coralville, Iowa), and cloned into the PX458 vector following published protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref153-3" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-3" aria-controls="sref153-3">Ran et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref153-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (3965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The sequence of all sgRNAs along with their chromosomal locations (hg19) are listed in <a id="crosref1330" href="#mmc9">Table S9</a>.</div><div id="p0375" role="paragraph"><b>Transfections using rs6060369 enhancers:</b> To investigate the functional effects of the osteoarthritis-risk and non-risk variants at rs6060369, each human variant <i>R4</i> enhancer sequence (i.e., differing only at the non-risk “C” or risk “T” allele) was cloned into the pGL4.23 Firefly luciferase reporter construct, initially acquired from Promega (Madison, Wisconsin). A control pGL4.74 Renilla luciferase plasmid was also acquired from Promega. Primers were first designed to amplify the <i>R4</i> regulatory element from human synthesized sequences (TwistBioscience). These primers possessed KpnI/HindIII linker sequences for cloning into pGL4.23 vector. These primers are listed in <a id="crosref1335" href="#mmc9">Table S9</a>. The following PCR protocol was used to amplify each region of interest: 98°C for 30 s; 34 cycles at 98°C for 20 s, 60°C for 20 s, 72°C for 30 s; and a 5&nbsp;minute final extension at 72°C. Each amplicon was then ligated into a KpnI/HindIII digested pGL.4.23 Firefly luciferase vector. The digestion occurred in NEBuffer 2.1 (NEB B7202S) and ligation was performed using T4 DNA Ligase (NEB M0202S), following standard protocols (New England Biolabs, Beverly, Massachusetts). Insert-containing and empty (without regulatory insert) construct ligates were then transformed into <i>E.&nbsp;coli DH10B</i> cells. These cells were then streaked onto Luria Bertani (LB) agar plates containing ampicillin (50&nbsp;μg/ml), from which single colonies were picked, screened (using the same PCR protocol as above), and sequenced. Using the E.Z.N.A. Endo-Free Plasmid DNA Maxi Kit (Omega D6929-03), the colonies were purified. Each construct then underwent Sanger sequencing to confirm sequence identity and orientation of each insert as well as the entire pGL4.23 Firefly vector sequence.</div><div id="p0380" role="paragraph"><b>Cell lines and culture conditions:</b> Human embryonic kidney (HEK293FT) cells were acquired from Dr. Pardis Sabeti (Harvard University and Broad Institute). T/C-28a2 human chondrocyte cells were acquired from Dr. Li Zeng (Tufts University) courtesy of Dr. Mary Goldring (The Hospital for Special Surgery). Both cell lines were cultured at 5% CO<sub>2</sub> at 37°C in Dulbecco’s Modified Eagle’s Medium (DMEM), 10% fetal bovine serum (FBS), and 1% penicillin-streptomycin (P/S). Media was replaced every 2–3&nbsp;days and the cells were sub-cultured every 5&nbsp;days.</div></section><section id="sec-4-4-4"><h4>Human Cell Line Transfection</h4><div id="p0385" role="paragraph"><b>Transfection and targeting <i>R4</i> enhancer:</b> Guide RNAs, sg1 and sg2, flanking an ∼2kb region encompassing the <i>R4</i> element, and guide RNAs, sg3 and sg4, flanking a 41&nbsp;bp region containing osteoarthritis risk variant rs6060369 were initially tested for ability to induce efficient deletions of the human element in cultured HEK293FT cells (n&nbsp;= 2 biological replicates per assay). After two days of culture at 37°C, transfected HEK293FT cells were examined under a GFP-microscope (Zeiss) to verify successful transfection and GFP expression. DNA was then extracted using E.Z.N.A Tissue DNA Kit, and the <i>R4</i> regulatory element region was amplified using PCR with primers flanking the sgRNA locations (listed in <a id="crosref1340" href="#mmc9">Table S9</a>) using program above. PCR amplified products were isolated from 1% agarose gel (E.Z.N.A Gel Extraction Kit) and Sanger sequenced to verify deletions corresponding to the <i>R4</i> element and the 41&nbsp;bp region.</div><div id="p0390" role="paragraph">After confirmation that all sgRNAs worked efficiently in HEK293FT cells, we performed <i>in&nbsp;vitro</i> deletion of the <i>R4</i> enhancer element in the T/C-28a2 chondrocyte cell line (<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib54 bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Finger et&nbsp;al., 2003; Kokenyesi et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Finger, F. ∙ Schörle, C. ∙ Zien, A. ...</div><div><strong>Molecular phenotyping of human chondrocyte cell lines T/C-28a2, T/C-28a4, and C-28/I2</strong></div><div><em>Arthritis Rheum.</em> 2003; <strong>48</strong>:3395-3403</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.11341" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0346850618" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14673991/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.11341&amp;pmid=14673991" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Kokenyesi, R. ∙ Tan, L. ∙ Robbins, J.R. ...</div><div><strong>Proteoglycan production by immortalized human chondrocyte cell lines cultured under conditions that promote expression of the differentiated phenotype</strong></div><div><em>Arch. Biochem. Biophys.</em> 2000; <strong>383</strong>:79-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abbi.2000.2044" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0034326794" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11097179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabbi.2000.2044&amp;pmid=11097179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The same sgRNAs were used as above. T/C-28a2 cells were maintained in DMEM (GIBCO, Gaithersburg, Maryland) supplied with 10% FBS (GIBCO) and 1% Pen/Strep (0.025%), and seeded in a six-well plate 1&nbsp;day prior to transfection. After culturing at 37°C, we scanned cells under a GFP-microscope to verify successful transfection efficiency (i.e., &gt; 70% of cells were GFP positive) and GFP expression. DNA was then extracted using E.Z.N.A Tissue DNA Kit, and the <i>R4</i> regulatory element region was amplified using PCR primers flanking each sgRNA location (listed in <a id="crosref1345" href="#mmc9">Table S9</a>). PCR amplified products were isolated from 1% agarose gel (E.Z.N.A Gel Extraction Kit). Mi-Seq sequencing at the MGH CCIB DNA Core was used to verify successful targeting of the larger <i>R4</i> regulatory region with a modification efficiency of ∼10% and smaller 41&nbsp;bp <i>R4</i> regulatory region containing rs6060369 with a modification efficiency of ∼9%. RNA was also extracted from control and CRISPR-Cas9 targeted T/C-28a2 cells. RNA was DNase-treated and converted to cDNA for qPCR (see below). Three biological replicate experiments were performed per condition.</div><div id="p0395" role="paragraph"><b>Transfecting rs6060369 enhancer constructs:</b> Prior to transfection, T/C-28a2 cells were seeded in 96-well dishes at a density 3&nbsp;× 10<sup>4</sup> cells/well and cultured in DMEM with 10% FBS and 1% P/S for 24 hours. Total volume of media in each well was brought to 100&nbsp;μl. For T/C-28a2 transfection, to compare expression between human constructs containing the risk “T” and non-risk “C” allele at rs6060369, three independent transfection experiments (n&nbsp;= 3), each containing eight technical replicates (i.e., individual wells of a 96-well plate) per construct were performed. For T/C-28a2 cell transfection, Firefly luciferase reporter vector concentrations of 50&nbsp;ng, 100&nbsp;ng, 150&nbsp;ng, and 200&nbsp;ng were initially tested for efficient transfection. 100&nbsp;ng showed highest transfection efficiency and consistency and was used for all T/C-28a2 experiments. Cells were then transfected transiently with 5&nbsp;ng pGL4.74 Renilla luciferase vector and either 100&nbsp;ng of Firefly luciferase reporter vector or 100&nbsp;ng of empty pGL4.23 Firefly luciferase vector. The transfection reagent used in the experiment was Lipofectamine 2000 (Invitrogen 11668–019). The Lipofectamine:DNA ratio was 4:1 (as optimized in the lab and following manufacturer instructions); thus, there were 0.4&nbsp;μL of transfection reagent per 100&nbsp;ng of DNA. Forty-eight hours after the transfection, luciferase activity was measured on a SpectraMax L Microplate Reader (Molecular Devices; Cat# SpectraMax L Config), following the 96-well Dual-Glo Luciferase Assay System (Promega E2940) protocols. Plate reading conditions were as follows: 1&nbsp;minute dark adapt, 5&nbsp;s integration, and max range settings.</div></section><section id="sec-4-4-5"><h4>Chromatin Immunoprecipitation (ChIP) Assay</h4><div id="p0400" role="paragraph"><b>PITX1 ChIP in T/C-28a2 cells:</b> Cell line T/C-28a2 was cultured in DMEM containing 10% FBS and 1% PenStrep with 5% CO<sub>2</sub> at 37°C for 48 h. The cells were harvested at 48&nbsp;h by treatment with GIBCO Trypsin-EDTA (0.05%) solution (Life Technologies Corporation, NY, USA) and subsequently washed twice with 1X phosphate-buffered saline (PBS). ChIP-assay was performed using Zymo-SpinTM ChIP Kit (Zymo Research Corporation, USA). Briefly, the cells were counted using a hemocytometer and resuspended (5&nbsp;x10<sup>6</sup> cells) in 1ml of 1X PBS. For cross-linking proteins to DNA, formaldehyde was added to the cell suspension at a final concentration of 1% (v/v) and incubated for 7&nbsp;min at room temperature (RT) with gentle shaking. The cross-linking reaction was then stopped by addition of 0.125&nbsp;M glycine with gentle shaking for 5&nbsp;min at RT followed by centrifugation at 3,000 X g for 1&nbsp;min at 4°C. The cross-linked cell pellet was resuspended in 1&nbsp;mL of ice-cold 1X PBS containing a mixture of 1&nbsp;mM phenylmethylsulphonyl fluoride (PMSF) and 1X protease inhibitor cocktail (PIC). The cell pellet was then lysed with 500&nbsp;μL of ice-cold nuclei prep buffer and the cell lysate was sonicated in the chilled chromatin shearing buffer containing 1mM PMSF and 1X PIC using Covaris s220X Focused Ultrasonicator (Covaris, Inc. USA) to generate DNA fragments between 100 and 500&nbsp;bp. Once the cell debris was removed by centrifugation, the sheared chromatin in the supernatant was diluted in ChIP dilution buffer (1:10), and 1% of this sheared chromatin (as input DNA) was collected, purified, and subjected to genomic PCR with the primer sets described in <a id="crosref1350" href="#mmc9">Table S9</a>. The 100&nbsp;μL of sheared chromatin was incubated overnight at 4°C with PITX1 (G-4) X antibody (Santa Cruz Biotechnology, NJ, USA). Immune complexes were collected by magnetic separation using ZymoMag Protein A beads and the complexes were incubated with 6&nbsp;μL of 5M NaCl at 65°C followed by proteinase K digestion to reverse DNA-protein cross-linking. ChIP-DNA was then purified by Zymo-SpinTM IC Column as per manufacturer’s instruction. Four biological replicate ChIP-assays (n&nbsp;= 4) were performed.</div><div id="p0405" role="paragraph">The strand-specific PCR primers used for amplifying the 257&nbsp;bp region containing rs6060369 are indicated in <a id="crosref1355" href="#mmc9">Table S9</a>. PCR amplification was performed in a 50&nbsp;μL reaction mixture containing 2&nbsp;μL of input/ChIP DNA by addition of 0.02&nbsp;U/μl Q5® High-Fidelity DNA Polymerase. A hot start was performed at 98°C for 30 s; followed by 35 cycles at 98°C for 10 s, 55°C for 30 s, and 72°C for 30 s; with final extension at 72°C for 1&nbsp;min. The PCR product was separated on 1% agarose gel containing 0.05&nbsp;μg/mL of Ethidium Bromide (EtBr) and photographed using VWR photoimager Dual UV transilluminator system (VWR International, USA).</div><div id="p0410" role="paragraph"><b>Pitx1 ChIP <i>in&nbsp;vivo</i> in mouse:</b> PITX1 ChIP on the mouse <i>R4</i> element was performed as follows: Timed matings were performed between homozygous C57BL/6J <i>R4</i><sup><i>rs6060369-T,rs6060369-T</i></sup> and wild-type C57BL/6J <i>R4</i><sup><i>rs6060369-A,rs6060369-A</i></sup> mice. Pregnant females were sacrificed according to IACUC-approved protocols. Embryonic hindlimbs were dissected taking the distal femoral and proximal tibia at E15.5 and separately placed into a microcentrifuge tube containing 200&nbsp;μL of 5% FBS supplemented DMEM medium. The collected tissues were subjected to 1% collagenase, type 2 (Cat. No. LS004176, Worthington Biochemical Corp. NJ, USA) digestion for 2&nbsp;h at 37°C rocking, gentle mixing every 30&nbsp;minutes to generate single cell chondrocyte suspension. The dissociated single cells were then washed twice with 1X PBS and subsequently ChIP-assay was performed on these cells using the methods described above. Three biological replicate ChIP-assays (n&nbsp;= 3) were performed on each tissue. The strand-specific PCR primers used for amplifying the 273&nbsp;bp region containing the orthologous replacement rs6060369 base-position in the mouse genome are indicated in <a id="crosref1360" href="#mmc9">Table S9</a>. PCR amplification was performed in a 50&nbsp;μL reaction mixture containing 2&nbsp;μL of input/ChIP DNA by the addition of 0.02&nbsp;U/μl Q5® High-Fidelity DNA Polymerase. A hot start was performed (98°C for 30 s), followed by 35 cycles at 98°C for 10 s, 55°C for 30 s, and 72°C for 30&nbsp;s with final extension at 72°C for 1&nbsp;min. The PCR product was separated on 1% agarose gel containing 0.05&nbsp;μg/mL of Ethidium Bromide (EtBr) and photographed using VWR photoimager Dual UV transilluminator system (VWR International, USA).</div></section><section id="sec-4-4-6"><h4>ATAC-seq Data Collection and Analysis</h4><div id="p0415" role="paragraph">Raw sequencing fastq files and processed peak bed files have been deposited on NCBI GEO(GSE122877).</div><div id="p0420" role="paragraph"><b>Mouse samples:</b> FVB/NJ male and female mice were used to establish timed matings, and at E15.5 pregnant females were euthanized to acquire embryos. At this time point, chondrocytes are easily extracted from surrounding extracellular matrix for ATAC-seq with negligible effects on the epigenome (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Guo et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Guo, M. ∙ Liu, Z. ∙ Willen, J. ...</div><div><strong>Epigenetic profiling of growth plate chondrocytes sheds insight into regulatory genetic variation influencing height</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e29329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.29329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040906101" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29205154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.29329&amp;pmid=29205154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Embryos were dissected under a microscope in 1X PBS on ice and the proximal and distal portions of the right and left femur and tibia of the hind limb and right and left humerus and radius of the forelimb were stripped clear of soft tissues. Each proximal or distal cartilaginous end comprising of the articular chondrocytes, epiphyseal chondrocytes, and metaphyseal chondrocytes was then micro-dissected from the bony diaphysis (e.g., see <a id="crosref1370" href="#fig1">Figure&nbsp;1</a>A) and separately pooled from a single litter, consisting on average of eight animals. Two to three biological replicates were collected in line with previous ATAC-seq studies (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib63 bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Gehrke et&nbsp;al., 2015; Guo et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Gehrke, A.R. ∙ Schneider, I. ∙ de la Calle-Mustienes, E. ...</div><div><strong>Deep conservation of wrist and digit enhancers in fish</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:803-808</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1420208112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84921395685" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1420208112&amp;pmid=25535365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Guo, M. ∙ Liu, Z. ∙ Willen, J. ...</div><div><strong>Epigenetic profiling of growth plate chondrocytes sheds insight into regulatory genetic variation influencing height</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e29329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.29329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040906101" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29205154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.29329&amp;pmid=29205154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All samples were collected in micro-centrifuge Eppendorf tubes containing 200μL 5% FBS/DMEM. To generate a single-cell chondrocyte suspension, each pooled sample was then subjected to 1% Collagenase II (VWR 80056–222, Radnor, Pennsylvania) digestion for 2 hours at 37°C rocking, mixing every 30&nbsp;minutes. After placing on ice, samples were next filtered using a micro-centrifuge filter set-up by gently mashing the residual tissues through the filter followed by rinsing with 5% FBS/DMEM. Samples were then spun down at 500&nbsp;g at 4°C for 5&nbsp;minutes. All cell counting methods were performed using trypan blue and a hemocytometer and subsequent ATAC-seq steps were performed on those samples that had cell death rates well below 10%. On average we acquired 500,000–1,000,000 living cells per harvest. Next, cells were re-suspended in concentrations of 50,000 cells in 1x PBS. Cell samples then subjected to the ATAC-seq protocol as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib15 bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Buenrostro et&nbsp;al., 2013, 2015</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Buenrostro, J.D. ∙ Wu, B. ∙ Chang, H.Y. ...</div><div><strong>ATAC-seq: a method for assaying chromatin accessibility genome-wide</strong></div><div><em>Curr. Protoc. Mol. Biol.</em> 2015; <strong>109</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/0471142727.mb2129s109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84949212255" target="_blank" aria-label="Open Scopus in new tab">Scopus (441)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F0471142727.mb2129s109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), modifying the protocol by using 2&nbsp;μl of transposase per reaction. The transposase reaction product was then purified using the Omega MicroElute DNA Clean Up Kit following manufacturers protocols, eluted in 10&nbsp;μl of warmed ddH<sub>2</sub>0, and stored at −20°C. We also performed experiments on transgenic <i>Col2a1-ECFP</i> reporter mice (compliments of Dr. Cliff Tabin at Harvard Medical School) (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Chokalingam et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Chokalingam, K. ∙ Hunter, S. ∙ Gooch, C. ...</div><div><strong>Three-dimensional in&nbsp;vitro effects of compression and time in culture on aggregate modulus and on gene expression and protein content of collagen type II in murine chondrocytes</strong></div><div><em>Tissue Eng. Part A.</em> 2009; <strong>15</strong>:2807-2816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.tea.2008.0560" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-73349122898" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19231914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.tea.2008.0560&amp;pmid=19231914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which has an enhanced cyan fluorescent protein reporter under the control of the promoter of <i>Col2a1</i>. Identical to experiments on FVB/NJ mice, E15.5 proximal and distal skeletal elements of the femur were micro-dissected from pooled <i>Col2a1-ECFP+</i> littermates, then subjected to collagenase treatment to make single cell suspensions (as above), and finally fluorescently sorted for CFP at the Harvard University Bauer Core Facility (HUBCF) to acquire replicates of 50,000 purified <i>Col2a1-ECFP+</i> cells per bone-end for ATAC-seq (n&nbsp;= 4 Biological Replicates).</div><div id="p0425" role="paragraph">All samples were next subjected to PCR amplification and barcoding following (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib15 bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Buenrostro et&nbsp;al., 2013, 2015</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Buenrostro, J.D. ∙ Wu, B. ∙ Chang, H.Y. ...</div><div><strong>ATAC-seq: a method for assaying chromatin accessibility genome-wide</strong></div><div><em>Curr. Protoc. Mol. Biol.</em> 2015; <strong>109</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/0471142727.mb2129s109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84949212255" target="_blank" aria-label="Open Scopus in new tab">Scopus (441)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F0471142727.mb2129s109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Ten microliters of transposed DNA were then placed in a reaction containing NEBNext High-Fidelity PCR Master Mix, ddH<sub>2</sub>0, and primers. Following amplification, samples were transferred to new tubes and treated using the OMEGA Bead Purification Protocol following manufacturer’s instructions. The samples were eluted in 30&nbsp;μl of TE, nano-dropped, diluted to 5&nbsp;ng/μl and run on a Bioanalyzer. Prior to sequencing sample concentrations were determined using the KAPA Library Quantification Complete Kit (KK4824). Samples were then sent out to the Harvard University Bauer Core Facility for sequencing on one lane of the Illumina NextSeq 500. Sequencing yielded ∼400 million reads per lane and an average of 50 million per sample. Quality control statistics and primer information are presented in <a id="crosref1380" href="#mmc1">Table S1</a>.</div><div id="p0430" role="paragraph"><b>Human samples:</b> Human products of conception were collected from late first-trimester and early second-trimester terminations through the Laboratory of Developmental Biology in full compliance with the ethical guidelines of the National Institutes of Health and with the approval of the University of Washington Institutional Review Boards for the collection and distribution of human tissues for research, and Harvard University for the receipt and use of such materials. The Laboratory of Developmental Biology obtained written consent from all tissue donors. The University of Washington Birth Defects Research Laboratory was supported by NIH award number 5R24HD000836 from the Eunice Kennedy Shriver National Institute of Child Health and Human Development. The human sample was briefly washed in HBSS and transported at 4°C during shipment. One rare intact human developmental sample at E59&nbsp;days was acquired from the Laboratory of Developmental Biology. Upon arrival the sample was dissected under a light dissection microscope in identical fashion to all mouse samples reported above. All forelimb and hindlimb long bone elements were intact but then manually separated from one another, and each proximal or distal cartilaginous end corresponding to the cell populations mentioned above was then micro-dissected from the bony diaphysis. Samples were then processed accordingly using the same ATAC-seq pipeline as done for mouse samples. Proximal and distal bone end samples for each long bone (i.e., femur, tibia, humerus, radius) were then sent out for sequencing at the HUBCF running a pooled library for 3 rounds of sequencing on the Illumina NextSeq 500 in order to obtain at least 50 million reads per sample. Quality control statistics and primer information are presented in <a id="crosref1385" href="#mmc1">Table S1</a>.</div></section><section id="sec-4-4-7"><h4>Computational Methods</h4><div id="p0435" role="paragraph"><b>Initial ATAC-seq read processing:</b> Mouse: Sequence read quality was checked with FastQC and subsequently aligned to the mouse reference mm10 genome assembly with Bowtie2 v2.3.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref107-2" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107-2" aria-controls="sref107-2">Langmead and Salzberg, 2012</a><div class="dropBlock__holder" data-db-target-of="sref107-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (15006)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using default parameters for paired-end alignment. Reads were filtered for duplicates and subsequently used for peak calling using MACS2 (<span class="dropBlock reference-citations"><a id="body-ref-sref203-2" href="#" role="doc-biblioref" data-xml-rid="bib203" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref203-2" aria-controls="sref203-2">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref203-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">189.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (5513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) software (version 2.1.1.2), using the following flags for ‘callpeak’:–nomodel–shift −100–extsize 200. Peaks (referred to as “elements” in main text) reproducible across biological replicates were screened using an IDR threshold of&nbsp;&lt; 0.05, as defined by the IDR statistical test (<span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">Li et&nbsp;al., 2011b</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Li, Q. ∙ Brown, J.B. ∙ Huang, H. ...</div><div><strong>Measuring reproducibility of high-throughput experiments</strong></div><div><em>Ann. Appl. Stat.</em> 2011; <strong>5</strong>:1752-1779</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1214/11-AOAS466" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79955564639" target="_blank" aria-label="Open Scopus in new tab">Scopus (359)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1214%2F11-AOAS466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 2.0.3). Briefly, the IDR method looks for overlaps in peak calls across pairs of replicate samples through comparing ranked peak lists (using MACS2 q-value) to define a reproducibility score curve. All paired ranks are assigned a pointwise score based on this curve, subsequently sorted, and all peaks falling below an ‘irreproducible discovery rate’ (IDR) threshold of 0.05 are taken as our final reproducible peak set.</div><div id="p0440" role="paragraph">For initial comparisons with human ATAC-seq data (see below) called peaks were lifted-over from mm10 to hg19 using the UCSC ‘liftover’ utility (<span class="dropBlock reference-citations"><a id="body-ref-sref100" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100" aria-controls="sref100">Kent et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref100"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Kent, W.J. ∙ Sugnet, C.W. ∙ Furey, T.S. ...</div><div><strong>The human genome browser at UCSC</strong></div><div><em>Genome Res.</em> 2002; <strong>12</strong>:996-1006</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.229102.+Article+published+online+before+print+in+May+2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0036079158" target="_blank" aria-label="Open Scopus in new tab">Scopus (5344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12045153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002&amp;pmid=12045153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using the following flags: ‘-minMatch&nbsp;= 0.1 -bedPlus&nbsp;= 4’, with the relevant liftover chain file similarly obtained from UCSC.</div><div id="p0445" role="paragraph">Human: Sequenced reads across runs were pooled for each sample and subsequently aligned to the hg19 reference genome using bowtie2 (version 2.3.2) using default paired-end parameters. Duplicate reads were filtered and subsequently used for peak calling with MACS2 software (version 2.1.1.2), using the following flags for ‘callpeak’:–nomodel–shift −100–extsize 200.</div><div id="p0450" role="paragraph"><b>IDR replicate consistency checks:</b> Following the guidelines for high-quality, replicable ATAC-seq datasets as defined by the ENCODE Consortium (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Davis et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Davis, C.A. ∙ Hitz, B.C. ∙ Sloan, C.A. ...</div><div><strong>The Encyclopedia of DNA elements (ENCODE): data portal update</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D794-D801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040942752" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29126249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1081&amp;pmid=29126249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0260" href="https://www.encodeproject.org/atac-seq/" target="_blank">https://www.encodeproject.org/atac-seq/</a>), we performed additional quality checks to assess both within- and between-sample concordance. Briefly, each individual sample was randomly subset into two halves (‘pseudoreplicates’), which were subsequently compared using IDR (applying a 0.05 IDR threshold) to check for within-sample reproducibility in peak-calling. Subsequently, pseudoreplicates were pooled across samples (i.e., pseudo-rep1 from sampleA was pooled with pseudo-rep1 from sample B) and these pooled pseudo-samples were compared using IDR. The metric for internal replicate consistency is termed the ‘self-consistency ratio’, while the pseudoreplicate-compared (cross-sample) consistency is termed the ‘rescue ratio’. ENCODE guidelines indicate that high-quality samples should have values for these ratios less than two – this we observe for testing done on two distal-femur and proximal-tibia samples (see <a id="crosref1415" href="#mmc1">Table S1</a>, Sheet 10), confirming the reproducibility of our ATAC-seq datasets.</div><div id="p0455" role="paragraph"><b>Generation of element sets:</b> Mouse ATAC-seq element sets from both the grossly dissected proximal and distal ends of the femur, tibia, humerus, and radius were intersected with ATAC-seq data our group previously generated on mouse embryonic brain tissue (<span class="dropBlock reference-citations"><a id="body-ref-sref72-4" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-4" aria-controls="sref72-4">Guo et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref72-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Guo, M. ∙ Liu, Z. ∙ Willen, J. ...</div><div><strong>Epigenetic profiling of growth plate chondrocytes sheds insight into regulatory genetic variation influencing height</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e29329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.29329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040906101" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29205154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.29329&amp;pmid=29205154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using bedtools (<span class="dropBlock reference-citations"><a id="body-ref-sref149-2" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149-2" aria-controls="sref149-2">Quinlan and Hall, 2010</a><div class="dropBlock__holder" data-db-target-of="sref149-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77951770756" target="_blank" aria-label="Open Scopus in new tab">Scopus (6963)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq033&amp;pmid=20110278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 2.26.0) in order to remove non-specific open chromatin regions. Filtered element sets for the distal femur and proximal tibia were intersected to generate a “general knee set,” with shared elements merged using the ‘merge’ function in bedtools and default parameters. Additionally, distal-femur and proximal-tibia brain-filtered sets were filtered for proximal-femur and distal-tibia elements, respectively, with these further-refined sets subsequently intersected to generate a “knee-specific set,” with shared elements merged as before. Similarly, proximal femur-specific and distal tibia-specific elements were also generated. We followed this same strategy to generate a “general elbow set” as well as an “elbow-specific set,” the latter consisting of elbow-separated and common element sets. Like the hind limb, we also generated for the forelimb proximal humerus-specific and distal radius-specific elements. All processed element sets were subsequently lifted-over from mm10 to hg19 as described above. To confirm consistent one-to-one orthology behavior of these sets, elements were lifted-over from mm10 to hg19 (liftOver utility flags ‘-minMatch&nbsp;= 0.1 -bedPlus&nbsp;= 4’), then subsequently lifted-back to mm10 (same flags). The original mm10 sets were compared to those lifted-back (<a id="crosref1430" href="#mmc1">Table S1</a>, Sheet 30), indicating that, for the vast majority of sequences (&gt; 99%) one-to-one orthology in liftover was achieved. In order to account for variation in called peak sizes when performing several analyses (e.g., calculating human sequence diversity between sets and motif alteration analyses), peaks in these final sets were assigned a fixed size of 500&nbsp;bp (based on the frequency distribution of called peak sizes across all sets) and centered on the middle of a called peak.</div><div id="p0460" role="paragraph">The genomic distribution of these element sets was assessed with HOMER (<span class="dropBlock reference-citations"><a id="body-ref-sref79-2" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79-2" aria-controls="sref79-2">Heinz et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref79-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (3844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 4.10) ‘annotatePeaks.pl’, which assigns regions to the closest annotated TSS and subsequently categorizes based on their positioning relative to genes (e.g., intronic, exonic, intergenic, etc.). The distribution of assignments for lifted-over distal femur-specific and proximal tibia-specific elements is shown in <a id="crosref1440" href="#fig1">Figure&nbsp;1</a>A, with distributions for additional element sets shown in <a id="crosref1445" href="#mmc1">Table S1</a>, Sheet 31.</div><div id="p0465" role="paragraph">Overlaps between human and lifted-over mouse ATAC-seq peaks were assessed using bedtools (version 2.26.0) ‘intersect’ with default settings (see <a id="crosref1450" href="#mmc1">Table S1</a>, Sheet 29), overlapping the peak sets generated from brain-filtered mouse samples (e.g., distal femur, filtered for embryonic brain) as well as those knee-specific sets (distal femur-specific, proximal tibia-specific, knee-common-specific) made using the below filtering steps. Sets were matched for tissue-type (e.g., mouse and human distal femur), with resulting overlap sets used in subsequent analyses.</div><div id="p0470" role="paragraph"><b>Additional epigenetic dataset processing and intersections:</b> <i>Col2a-ECFP</i> ATAC-seq data was obtained and processed as described above and then intersected with ATAC-seq datasets acquired on grossly dissected long bone chondrocytes as described above. GM12878 ATAC-seq data was obtained from GEO datasets (GSE47753) (<span class="dropBlock reference-citations"><a id="body-ref-sref15-6" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-6" aria-controls="sref15-6">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as raw .fastq files (for 50K samples); reads were subsequently mapped to hg19 using the same ATAC-seq processing pipeline detailed above, with IDR replication performed for n&nbsp;= 4 replicates. Bone Marrow-Derived Chondrocytes (BMDC) ChIP-seq datasets were downloaded as .bed.gz reads from GEO datasets (Superset GSE17312), and used to call peaks with MACS2 (version 2.1.1.2), using the ‘–nomodel–shift 0–extsize X’ options, with extension size chosen using the ‘predictd’ utility from MACS2. BMDC replicates were subsequently consolidated using irreproducible discovery rate (IDR) (version 2.0.3) analysis with a cut-off of 0.05. Human embryonic limb bud H3K27ac data (<span class="dropBlock reference-citations"><a id="body-ref-sref33-4" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-4" aria-controls="sref33-4">Cotney et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref33-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cotney, J. ∙ Leng, J. ∙ Yin, J. ...</div><div><strong>The evolution of lineage-specific regulatory activities in the human embryonic limb</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:185-196</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84879911829" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23827682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.05.056&amp;pmid=23827682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was obtained from GEO datasets (GSM1039552, GSM1039553) as .bed.gz files, and used to call peaks with MACS2 (with IDR replicate-consolidation) as above for the BMDC data. ENCODE (<span class="dropBlock reference-citations"><a id="body-ref-sref49-3" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-3" aria-controls="sref49-3">ENCODE Project Consortium, 2012</a><div class="dropBlock__holder" data-db-target-of="sref49-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>ENCODE Project Consortium</div><div><strong>An integrated encyclopedia of DNA elements in the human genome</strong></div><div><em>Nature.</em> 2012; <strong>489</strong>:57-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11247" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865790047" target="_blank" aria-label="Open Scopus in new tab">Scopus (8320)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955616/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11247&amp;pmid=22955616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) TF ChIP-seq was obtained as processed .bed files (called peaks) (see <a id="crosref1470" href="#mmc2">Table S2</a>, Sheets 5-7 for accession codes) from the ENCODE website. Pitx1 ChIP-seq (<span class="dropBlock reference-citations"><a id="body-ref-sref91-3" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-3" aria-controls="sref91-3">Infante et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref91-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Infante, C.R. ∙ Park, S. ∙ Mihala, A.G. ...</div><div><strong>Pitx1 broadly associates with limb enhancers and is enriched on hindlimb cis-regulatory elements</strong></div><div><em>Dev. Biol.</em> 2013; <strong>374</strong>:234-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.11.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84872376764" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23201014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.11.017&amp;pmid=23201014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) data was obtained from GEO datasets (GSE41591) as called peaks in .bed format; replicated peaks were intersected and merged using bedtools (version 2.26.0). All datasets which are indicated as originating from mouse (See <a id="crosref1480" href="#mmc2">Table S2</a>) were lifted-over from their original mouse genome coordinates to hg19 using the liftOver utility as described above. E59 human bone-end chondrogenesis ATAC-seq datasets (see above) were matched to their equivalent mouse tissue (e.g., mouse distal femur and human distal femur) for overlaps. All datasets were tested for enriched overlap with knee elements using regioneR (<span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Gel et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Gel, B. ∙ Díez-Villanueva, A. ∙ Serra, E. ...</div><div><strong>regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:289-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv562" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84959886262" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26424858/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv562&amp;pmid=26424858" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.8.1) using the ‘permTest’ function, generating 1000 randomized region sets as a background using the ‘circularRandomizeRegions’ option and the ‘count.once’ flag, with all other options set to defaults. Significance was assessed at p&nbsp;&lt; 0.05 (<a id="crosref1490" href="#mmc2">Table S2</a>, Sheets 1-7).</div><div id="p0475" role="paragraph"><b>Inter-species conservation analyses:</b> To examine inter-species conservation, phyloP (<span class="dropBlock reference-citations"><a id="body-ref-sref144-2" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144-2" aria-controls="sref144-2">Pollard et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref144-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Pollard, K.S. ∙ Hubisz, M.J. ∙ Rosenbloom, K.R. ...</div><div><strong>Detection of nonneutral substitution rates on mammalian phylogenies</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:110-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.097857.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-74949092081" target="_blank" aria-label="Open Scopus in new tab">Scopus (968)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19858363/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.097857.109&amp;pmid=19858363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) scores for 20 primate species (phyloP20ways) were obtained from UCSC (<span class="dropBlock reference-citations"><a id="body-ref-sref98-1" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-1" aria-controls="sref98-1">Karolchik et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref98-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Karolchik, D. ∙ Barber, G.P. ∙ Casper, J. ...</div><div><strong>The UCSC Genome Browser database: 2014 update</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D764-D770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84891771466" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1168&amp;pmid=24270787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and used to extract per-bp conservation scores for all elements in the general knee, knee-specific, general elbow, and elbow-specific sets, as well as those knee-specific elements that overlapped human E59 ATAC-seq data. To make elements comparable for these analyses, region coordinates were fixed to a constant size of 500&nbsp;bp (see above). Given the potential for base-pair modifications to alter enhancer activity (<span class="dropBlock reference-citations"><a id="body-ref-sref146" href="#" role="doc-biblioref" data-xml-rid="bib117 bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146" aria-controls="sref146">Maas and Fallon, 2005; Prabhakar et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref146"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Maas, S.A. ∙ Fallon, J.F.</div><div><strong>Single base pair change in the long-range Sonic hedgehog limb-specific enhancer is a genetic basis for preaxial polydactyly</strong></div><div><em>Dev. Dyn.</em> 2005; <strong>232</strong>:345-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20254" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-12344299320" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15637698/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20254&amp;pmid=15637698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Prabhakar, S. ∙ Visel, A. ∙ Akiyama, J.A. ...</div><div><strong>Human-specific gain of function in a developmental enhancer</strong></div><div><em>Science.</em> 2008; <strong>321</strong>:1346-1350</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1159974" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-51149097366" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18772437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1159974&amp;pmid=18772437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), coupled with the expectation that selection forces likely do not act across the entire length of a multi-TFBS enhancer, instead modifying individual binding sites (<span class="dropBlock reference-citations"><a id="body-ref-sref191" href="#" role="doc-biblioref" data-xml-rid="bib191" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref191" aria-controls="sref191">Wittkopp and Kalay, 2011</a><div class="dropBlock__holder" data-db-target-of="sref191"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">178.</div><div>Wittkopp, P.J. ∙ Kalay, G.</div><div><strong>Cis-regulatory elements: molecular mechanisms and evolutionary processes underlying divergence</strong></div><div><em>Nat. Rev. Genet.</em> 2011; <strong>13</strong>:59-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg3095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-83855163414" target="_blank" aria-label="Open Scopus in new tab">Scopus (419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22143240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg3095&amp;pmid=22143240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), per-bp scores for each region were aggregated (within each set, averaged per-bp across all sequences in a set, for a total of 500 data-points) in order to compare pan-species sequence conservation across sets. These sets of points were subsequently used to perform a Wilcoxon Rank-Sum test as implemented in R (<span class="dropBlock reference-citations"><a id="body-ref-sref150-2" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-2" aria-controls="sref150-2">R Development Core Team, 2008</a><div class="dropBlock__holder" data-db-target-of="sref150-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>R Development Core Team</div><div><strong>R: A Language and Environment for Statistical Computing.</strong></div><div>R Foundation for Statistical Computing, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Development+Core+TeamR%3A+A+Language+and+Environment+for+Statistical+Computing.2008R+Foundation+for+Statistical+Computing" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 3.4.4) to test for the significance of differences in conservation, resulting one-sided p values were corrected for multiple testing using Benjamini-Hochberg (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Benjamini and Hochberg, 1995</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Benjamini, Y. ∙ Hochberg, Y.</div><div><strong>Controlling the false discovery rate: a practical and powerful approach to multiple testing</strong></div><div><em>J.&nbsp;R. Stat. Soc. B.</em> 1995; <strong>57</strong>:289-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) correction. Scores were also averaged over the length of elements (500&nbsp;bp, per-region in a set) and compared as for the per-bp results. Significance was determined at adjusted p&nbsp;&lt; 0.05; see <a id="crosref1520" href="#mmc2">Table S2</a>, Sheet 8 for comparison information.</div><div id="p0480" role="paragraph">For analysis of trends in phyloP score over the length of a peak (<a id="crosref1525" href="#figs1">Figure&nbsp;S1</a>), per-bp conservation scores were averaged per-bp across all ATAC-seq regions in a set and plotted as a function of distance along a 1kb window (centered in the middle of each ATAC-seq region). A randomly-generated set of 1kb regions (matched for set size) sampled genome-wide was also generated and average per-bp conservation scores calculated in order to provide a background set. For the analysis comparing primate conservation to human-chimpanzee divergence, aligned sequences for human and chimpanzee were extracted from a primate alignment (multiz20way) obtained from UCSC (<span class="dropBlock reference-citations"><a id="body-ref-sref98-2" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-2" aria-controls="sref98-2">Karolchik et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref98-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Karolchik, D. ∙ Barber, G.P. ∙ Casper, J. ...</div><div><strong>The UCSC Genome Browser database: 2014 update</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D764-D770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84891771466" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1168&amp;pmid=24270787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with sequence identity (%ID) for a given ATAC-seq region calculated as the number of nucleotide matches divided by total sequence length. These values were compared to per-bp phyloP20ways scores averaged over the length of the same region, and subsequently plotted in order of increasing %ID (as seen in <a id="crosref1535" href="#fig1">Figure&nbsp;1</a>F). Regions sorted by %ID were sliced into top and bottom 25%, as well as 10%, groups for each sequence set and were analyzed using&nbsp;GREAT (<span class="dropBlock reference-citations"><a id="body-ref-sref121-3" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-3" aria-controls="sref121-3">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref121-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (1686)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 3.0.0) (<a id="crosref1545" href="#mmc2">Table S2</a>, Sheets 9 - 20). Region coordinates (hg38), extracted sequences, calculated %ID, and averaged phyloP20ways score for each sequence set analyzed are available in <a id="crosref1550" href="#mmc2">Table S2</a>, Sheets 21-23.</div><div id="p0485" role="paragraph">Note about GREAT (<span class="dropBlock reference-citations"><a id="body-ref-sref121-4" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-4" aria-controls="sref121-4">McLean et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref121-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (1686)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): GREAT takes an input set of genomic regions along with a defined ontology of gene annotations; first, it defines regulatory domains for all genes genome-wide, then measures the fraction of the genome covered by the regulatory domains of genes associated with a particular annotation (e.g., ‘cartilage development’). These fractions are used as the expectation in a binomial test counting the number of input genomic regions falling within a given set of regulatory domains, which results in the reported significance of association between an input region set and a particular gene ontology term. GREAT also performs a more traditional gene-based hypergeometric test to test for significance of region set-ontology association. The program returns a set of enriched ontologies sorted by the joint rankings of FDR-corrected binomial and hypergeometric tests, as reported here in our supplemental tables. For this study, we chose terms in the GREAT output with relevance to chondrocyte biology – bone end-specific anatomy is not well annotated, particularly at the genetic level, and it is well known that there is a bias toward GO annotations for other phenotypes (like immune function), particularly for phenotypes involving better-studied genes which tend to have increased GO term representation (<span class="dropBlock reference-citations"><a id="body-ref-sref207" href="#" role="doc-biblioref" data-xml-rid="bib207" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref207" aria-controls="sref207">Gillis and Pavlidis, 2013</a><div class="dropBlock__holder" data-db-target-of="sref207"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Gillis, J. ∙ Pavlidis, P.</div><div><strong>Assessing identity, redundancy and confounds in Gene Ontology annotations over time</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:476-482</div><div>In press</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts727" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874298500" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23297035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts727&amp;pmid=23297035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0490" role="paragraph">A set of human sequences displaying evidence of nucleotide acceleration in a variety of different contexts was aggregated from multiple studies (<span class="dropBlock reference-citations"><a id="body-ref-sref145-3" href="#" role="doc-biblioref" data-xml-rid="bib10 bib17 bib65 bib142 bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-3" aria-controls="sref145-3">Bird et&nbsp;al., 2007; Bush and Lahn, 2008; Gittelman et&nbsp;al., 2015; Pollard et&nbsp;al., 2006a; Prabhakar et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref145-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bird, C.P. ∙ Stranger, B.E. ∙ Liu, M. ...</div><div><strong>Fast-evolving noncoding sequences in the human genome</strong></div><div><em>Genome Biol.</em> 2007; <strong>8</strong>:R118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2007-8-6-r118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049157173" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17578567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2007-8-6-r118&amp;pmid=17578567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Bush, E.C. ∙ Lahn, B.T.</div><div><strong>A genome-wide screen for noncoding elements important in primate evolution</strong></div><div><em>BMC Evol. Biol.</em> 2008; <strong>8</strong>:17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-8-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049088628" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18215302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-8-17&amp;pmid=18215302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Gittelman, R.M. ∙ Hun, E. ∙ Ay, F. ...</div><div><strong>Comprehensive identification and analysis of human accelerated regulatory DNA</strong></div><div><em>Genome Res.</em> 2015; <strong>25</strong>:1245-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.192591.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84937476478" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.192591.115&amp;pmid=26104583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Pollard, K.S. ∙ Salama, S.R. ∙ King, B. ...</div><div><strong>Forces shaping the fastest evolving regions in the human genome</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e168</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-35648988566" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17040131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020168&amp;pmid=17040131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Prabhakar, S. ∙ Noonan, J.P. ∙ Pääbo, S. ...</div><div><strong>Accelerated evolution of conserved noncoding sequences in humans</strong></div><div><em>Science.</em> 2006; <strong>314</strong>:786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1130738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750737061" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17082449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1130738&amp;pmid=17082449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and intersected with the general knee, knee-specific, general elbow, and elbow-specific sets (using original called-peak coordinates), knee-specific and elbow-specific elements that overlapped human E59 ATAC-seq data, along with ATAC-seq data obtained from embryonic brain tissue, using bedtools (version 2.26.0). In order to correct for both the variable number and size of peaks in different sets, intersections were calculated as intersections/bp of sequence in a given set. A background distribution was established by generating 10,000 random region sets consisting of 4015 (average set size for specific knee sequence sets) regions from across the genome with a constant length of 500&nbsp;bp using the bedtools ‘random’ function, and intersecting each set with accelerated regions. Intersections/bp values for both target and background sets were plotted as a histogram (<a id="crosref1560" href="#fig1">Figure&nbsp;1</a>C); the distribution of these values was assessed using the ‘qqnorm’ (R base) and ‘qqPlot’ (car version 3.0.3) (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Fox and Weisberg, 2019</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Fox, J. ∙ Weisberg, S.</div><div><strong>An R Companion to Applied Regression</strong></div><div>SAGE Publications, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxS.WeisbergAn+R+Companion+to+Applied+RegressionThird+Edition2019SAGE+Publications" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) functions – these suggested substantial deviations from a normal distribution. Accordingly, the ‘fitdistrplus’ package (version 1.0-14) (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Delignette-Muller and Dutang, 2015</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Delignette-Muller, M.L. ∙ Dutang, C.</div><div><strong>fitdistrplus: An R Package for Fitting Distributions</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>64</strong>:1-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v064.i04" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84925685098" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v064.i04" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to determine an appropriate distribution to fit the data. Given the limitations in fitting a number of distributions (e.g., gamma, beta, log-normal) for positive values background sets containing no accelerated region overlaps were removed for curve fitting (n&nbsp;= 9427 after filtering). The ‘descdist’ function was initially used to assess curve behavior; subsequently goodness-of-fit statistics (from the ‘gofstat’ function) for gamma, beta, exponential, and log-normal distributions were compared, with the beta-distribution subsequently selected (this choice also being appropriate given the fractional nature of the datapoints) (<span class="dropBlock reference-citations"><a id="body-ref-sref128" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128" aria-controls="sref128">Mun, 2008</a><div class="dropBlock__holder" data-db-target-of="sref128"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Mun, J.</div><div><strong>Advanced Analytical Models: over 800 Models and 300 Applications from the Basel II Accord to Wall Street and Beyond</strong></div><div>Wiley, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.MunAdvanced+Analytical+Models%3A+over+800+Models+and+300+Applications+from+the+Basel+II+Accord+to+Wall+Street+and+Beyond2008Wiley" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Beta distribution parameters (‘shape1’ and ‘shape2′ in the R implementation of ‘pbeta’) were fit using a bootstrap method (‘bootdist’ from ‘fitdistrplus’), with the median parameter estimates from 1000 samples used to define the distribution for significance testing of target set intersections/bp values with ‘pbeta’ (upper-tail p values). Results were subsequently adjusted using BH correction (<a id="crosref1580" href="#mmc4">Table S4</a>, Sheet 14).</div><div id="p0495" role="paragraph">Elements intersecting regions of acceleration were associated with the closest annotated TSS with HOMER (version 4.10) ‘annotatePeaks.pl’ (<a id="crosref1585" href="#mmc4">Table S4</a>, Sheets 2 – 13). All intersecting hindlimb elements were aggregated and used with GREAT to identify genome-wide signals (<a id="crosref1590" href="#mmc4">Table S4</a>, Sheet 15).</div><div id="p0500" role="paragraph"><b>Intra-species conservation analysis:</b> Variation data from the 1000 Genomes Project phase 3 (1KGP) (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (n&nbsp;= 2504 individuals) in .v<i>cf.</i>gz format was obtained and intersected with the general knee, knee-specific, general elbow, and elbow-specific ATAC-seq region sets, as well as those knee-specific elements that overlapped human E59 ATAC-seq data (elements fixed to a size of 500&nbsp;bp) using tabix (<span class="dropBlock reference-citations"><a id="body-ref-sref108-4" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-4" aria-controls="sref108-4">Li, 2011a</a><div class="dropBlock__holder" data-db-target-of="sref108-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Li, H.</div><div><strong>Tabix: fast retrieval of sequence features from generic TAB-delimited files</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:718-719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79951993896" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21208982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq671&amp;pmid=21208982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.7.24) to obtain variants occurring within these putative regulatory regions. Sequence data from chimpanzee (n&nbsp;= 25) and gorilla (n&nbsp;= 31) was similarly obtained from the Great Ape Genome Diversity Project (GADP) (<span class="dropBlock reference-citations"><a id="body-ref-sref147-4" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147-4" aria-controls="sref147-4">Prado-Martinez et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref147-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Prado-Martinez, J. ∙ Sudmant, P.H. ∙ Kidd, J.M. ...</div><div><strong>Great ape genetic diversity and population history</strong></div><div><em>Nature.</em> 2013; <strong>499</strong>:471-475</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881159598" target="_blank" aria-label="Open Scopus in new tab">Scopus (402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23823723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12228&amp;pmid=23823723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Peak sets were lifted-over from hg19 to hg18 for use with the GADP datasets with the UCSC ‘liftover’ utility and relevant liftover chain file. Resulting subset VCF files were converted to tab format with the following Unix command, using bcftools (version 1.8) (<span class="dropBlock reference-citations"><a id="body-ref-sref109-1" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-1" aria-controls="sref109-1">Li, 2011b</a><div class="dropBlock__holder" data-db-target-of="sref109-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Li, H.</div><div><strong>A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2987-2993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr509" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-80054915847" target="_blank" aria-label="Open Scopus in new tab">Scopus (1639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21903627/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr509&amp;pmid=21903627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>):</div><div id="p0505" role="paragraph">bcftools query -f ‘%CHROM\t%POS\t%ID\t%REF\t%ALT[\t%SAMPLE&nbsp;= %TGT]\n’ -o out.v<i>cf.</i> in.v<i>cf.</i></div><div id="p0510" role="paragraph">Variant data for all datasets were down-sampled to n&nbsp;= 25 (with replacement, 5 resamples for gorilla and 200 re-samples for the human set) in order to match sample size for all comparisons based on the least-sampled species (chimp), using a custom R script.</div><div id="p0515" role="paragraph">Variants were defined using a minor allele frequency (MAF) threshold of &gt;&nbsp;= 0.05 for all datasets, filtering tab-formatted files using a custom Python script. Counts data was defined as the number of variants intersecting a given element and averaged over resampled variant sets (see below). Counts data for human, chimp, and gorilla were then compared within a given element set (e.g., proximal tibia-specific set) to compare intra-species diversity of putative regulatory regions. Hurdle modeling was used to test for significant differences in both total number of sequences containing variants (hurdle) as well as degree of variation between species (counts); this was implemented using the ‘hurdle’ function from the pscl (<span class="dropBlock reference-citations"><a id="body-ref-sref198" href="#" role="doc-biblioref" data-xml-rid="bib92 bib198" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref198" aria-controls="sref198">Jackman, 2017; Zeileis et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref198"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Jackman, S.</div><div><strong>{pscl}: Classes and Methods for {R} Developed in the Political Science Computational Laboratory</strong></div><div>R package version 1.5.2. United States Studies Centre, University of Sydney, 2017</div><div><a id="intref0435" href="https://github.com/atahk/pscl/" target="_blank">https://github.com/atahk/pscl/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Jackman%7Bpscl%7D%3A+Classes+and+Methods+for+%7BR%7D+Developed+in+the+Political+Science+Computational+Laboratory2017R+package+version+1.5.2.+United+States+Studies+Centre%2C+University+of+Sydneyhttps%3A%2F%2Fgithub.com%2Fatahk%2Fpscl%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">184.</div><div>Zeileis, A. ∙ Kleiber, C. ∙ Jackman, S.</div><div><strong>Regression Models for Count Data in {R}</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2008; <strong>27</strong>:1-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v027.i08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v027.i08" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) package in R (version 1.5.2). A binomial model was applied for the initial hurdle/zero-counts step, with the subsequent counts modeling done using a negative binomial regression model. Tukey post hoc testing was performed using the emmeans package in R (version 1.2.1) for both hurdle/zero-counts and counts models, with significance assessed at adjusted p value&nbsp;&lt; 0.05 (<a id="crosref1615" href="#mmc5">Table S5</a>, Sheets 1-26). Additionally, element sets were compared to one another (e.g., distal femur-specific versus proximal tibia-specific) within a given species using the same methods. Variant per-sequence counts were visualized as boxplots using ggplot2 (version 2.2.1) with logged values (<a id="crosref1620" href="#fig3">Figures 3</a>B–3F and <a id="crosref1625" href="#figs2">S2</a>).</div><div id="p0520" role="paragraph">In order to check for consistency across sub-samples when taking averaged values for the above analyses, hurdle modeling was performed using human counts data calculated from the first and third quartiles of counts for a particular sequence across the sub-sampled groups to confirm that statistical relationships between species (for a given element set) and within species (comparing element sets) were reproduced. Additionally, subsampling was re-performed using a sub-sample size of n&nbsp;= 100 (for 200 resamples) to confirm that statistical relationships are robust to sub-sample size of the human set. These results are similarly presented in <a id="crosref1630" href="#mmc5">Table S5</a>. Sequences within each region set were ranked according to human SNP/bp value averaged over sub-samples of 1KG3 data and sliced into top and bottom 25% most/least-variable (respectively) sections. Slices were subsequently analyzed with GREAT (<a id="crosref1635" href="#mmc5">Table S5</a>, Sheets 31-42). Similar slicing was done for chimp SNP/bp values to generate equivalent most/least-variable sections (<a id="crosref1640" href="#mmc5">Table S5</a>, Sheets 43-54).</div><div id="p0525" role="paragraph">In order to look at sequence constraint of putative regulatory regions within humans the set of knee-specific elements (i.e., distal femur-specific, proximal tibia-specific, and knee-common-specific sets) were pooled and a set of 1000 randomly-generated region sets, consisting of 8491 sequences (size of the pooled set) of 500&nbsp;bp was generated using the bedtools ‘random’ function. These sets were subsequently pooled, sorted, and merged using bedtools, with the resulting bed file used to extract variants from the 1KG3 set with tabix (version 1.7.24). The pooled set of knee-specific elements was also used to extract variants from the 1KG3 set. Additionally, several genomic features were extracted from the HOMER set of genomic annotations (hg19) provided with the program, including the following region sets: intronic, promoter-TSS, TTS, and exon. Further, regions from RepeatMasker were also obtained from the UCSC Table Browser (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Karolchik et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Karolchik, D. ∙ Hinrichs, A.S. ∙ Furey, T.S. ...</div><div><strong>The UCSC Table Browser data retrieval tool</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:D493-D496</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkh103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14681465/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkh103&amp;pmid=14681465" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These additional sets were also used to extract variants from the 1KG3 set. The resulting files were filtered for duplicate variants and subsequently MAF &gt;&nbsp;= 0.05 with bcftools (<span class="dropBlock reference-citations"><a id="body-ref-sref109-2" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-2" aria-controls="sref109-2">Li, 2011b</a><div class="dropBlock__holder" data-db-target-of="sref109-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Li, H.</div><div><strong>A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2987-2993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr509" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-80054915847" target="_blank" aria-label="Open Scopus in new tab">Scopus (1639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21903627/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr509&amp;pmid=21903627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.8). Variants falling within particular elements in the random background, target (e.g., knee-specific), and genomic annotation sets were then extracted using tabix. The number of variants extracted for each set was counted using vcftools (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Danecek et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Danecek, P. ∙ Auton, A. ∙ Abecasis, G. ..., 1000 Genomes Project Analysis Group</div><div><strong>The variant call format and VCFtools</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2156-2158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr330" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79960405019" target="_blank" aria-label="Open Scopus in new tab">Scopus (3479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr330&amp;pmid=21653522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 0.1.15) ‘–counts2–stdout’ arguments. Variant counts were then adjusted to account for the number of bp within a given set. The background distribution of these values was investigated using the ‘qqnorm’ (R base) and ‘qqPlot’ (car package) functions to look for visible deviations from normality, for which no obvious deviations were observed. Values were standardized and statistical significance was assessed using a CDF of the standard normal distribution as implemented in the ‘pnorm’ function in R (version 3.4.4). P values for significant deviations from the background distribution were corrected for the number of sets (n&nbsp;= 10) tested using a BH correction. Significance was defined as adjusted p&nbsp;&lt; 0.05 (<a id="crosref1660" href="#mmc5">Table S5</a>, Sheet 27).</div><div id="p0530" role="paragraph">A similar analysis of sequence constraint of putative regulatory regions within chimpanzees was also performed. Knee-specific elements were pooled and lifted-over to hg18 using the ‘liftOver’ utility; a set of 1000 randomly-generated region sets, consisting of 8431 sequences (size of lifted-over pooled set) of 500bp was generated using the bedtools ‘random’ function. Randomized sequence sets were subsequently pooled, sorted, and merged using bedtools, with the resulting bed file used to extract variants from the GADP set with tabix (version 1.7.24). Additionally, several genomic features were extracted from a downloaded set of chimpanzee HOMER genomic annotations (panTro4), including the following region sets: intronic, promoter-TSS, TTS, and exon. These additional sets were lifted-over to hg18 (flags as indicated above) and used to extract variants from the GADP set. The resulting files were filtered for duplicate variants and subsequently MAF &gt;&nbsp;= 0.05 with bcftools (<span class="dropBlock reference-citations"><a id="body-ref-sref109-3" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-3" aria-controls="sref109-3">Li, 2011b</a><div class="dropBlock__holder" data-db-target-of="sref109-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Li, H.</div><div><strong>A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2987-2993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr509" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-80054915847" target="_blank" aria-label="Open Scopus in new tab">Scopus (1639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21903627/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr509&amp;pmid=21903627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.8). Variants falling within particular elements in the random background, target (e.g., knee-specific), and genomic annotation sets were then extracted using tabix. The number of variants extracted for each set was counted using vcftools (<span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Danecek et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Danecek, P. ∙ Auton, A. ∙ Abecasis, G. ..., 1000 Genomes Project Analysis Group</div><div><strong>The variant call format and VCFtools</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2156-2158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr330" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79960405019" target="_blank" aria-label="Open Scopus in new tab">Scopus (3479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr330&amp;pmid=21653522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 0.1.15) ‘–counts2–stdout’ arguments. Variant counts were then adjusted to account for the number of bp within a given set. The background distribution of these values was investigated using the ‘qqnorm’ (R base) and ‘qqPlot’ (car package) functions to look for visible deviations from normality, for which an obvious left-skew was observed (<a id="crosref1675" href="#figs2">Figure&nbsp;S2</a>P). For comparison with the above human analysis, background values were standardized and statistical significance was assessed using a CDF of the standard normal distribution as implemented in the ‘pnorm’ function in R (version 3.4.4). P values for significant deviations from the background distribution were corrected for the number of sets (n&nbsp;= 10) tested using a BH correction. Significance was defined as adjusted p&nbsp;&lt; 0.05. See <a id="crosref1680" href="#mmc5">Table S5</a>, Sheet 28. Given the left-skew distribution, the ‘fitdistrplus’ package (version 1.0-14) (<span class="dropBlock reference-citations"><a id="body-ref-sref43-3" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-3" aria-controls="sref43-3">Delignette-Muller and Dutang, 2015</a><div class="dropBlock__holder" data-db-target-of="sref43-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Delignette-Muller, M.L. ∙ Dutang, C.</div><div><strong>fitdistrplus: An R Package for Fitting Distributions</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>64</strong>:1-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v064.i04" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84925685098" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v064.i04" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to determine an additional distribution to fit the data, as described for the accelerated region enrichment analysis above, with a gamma distribution selected on the basis of goodness-of-fit statistics. Distribution parameters were fit using a bootstrap method (‘bootdist’ from ‘fitdistrplus’), with the median parameter estimates from 1000 samples used to define the distribution for significance testing of adjusted variant counts with ‘pgamma’. Results differed negligibly from those obtained using a normal CDF (<a id="crosref1690" href="#mmc5">Table S5</a>, Sheet 28).</div><div id="p0535" role="paragraph"><b>Human knee morphometric analysis:</b> Bone and cartilage measurements of 600 knees were derived from the OA Biomarkers Consortium Project (OABCP) for the National Institutes of Health (FNIH) Project, which used the patient knee MRI images from&nbsp;the Osteoarthritis Initiative (OAI) (see <a id="intref0265" href="http://oai.epi-ucsf.org./datarelease/" target="_blank">http://oai.epi-ucsf.org./datarelease/</a> and <a id="intref0270" href="https://fnih.org/what-we-do/biomarkers-consortium/programs/osteoarthritis-project" target="_blank">https://fnih.org/what-we-do/biomarkers-consortium/programs/osteoarthritis-project</a>). Some of these measurements including the total area of medial and lateral subchondral bone were performed by iMorphics (Manchester, UK), a company focused on advanced image analysis technology for the analysis and interpretation of 3D medical images. The total area of the subchondral bone was measured according to the definition by Eckstein et&nbsp;al., with peripheral osteophytes excluded and central osteophytes included (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Eckstein et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Eckstein, F. ∙ Ateshian, G. ∙ Burgkart, R. ...</div><div><strong>Proposal for a nomenclature for magnetic resonance imaging based measures of articular cartilage in osteoarthritis</strong></div><div><em>Osteoarthritis Cartilage.</em> 2006; <strong>14</strong>:974-983</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2006.03.005&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2006.03.005&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33748160505" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16730462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2006.03.005&amp;pmid=16730462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These measurements were obtained based on the 3D bone shapes that were automatically segmented from 3D DESS MRI images using active appearance models (<span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Hunter et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Hunter, D. ∙ Nevitt, M. ∙ Lynch, J. ..., FNIH OA Biomarkers Consortium</div><div><strong>Longitudinal validation of periarticular bone area and 3D shape as biomarkers for knee OA progression? Data from the FNIH OA Biomarkers Consortium</strong></div><div><em>Ann. Rheum. Dis.</em> 2016; <strong>75</strong>:1607-1614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2015-207602" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26483253/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2015-207602&amp;pmid=26483253" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0540" role="paragraph">The measurements of the volume of medial and lateral cartilage were performed using KneeIQ, a fully automatic computer-based framework developed by Biomediq (Copenhagen, Denmark). Biomediq is a company that specializes in analyses of knee MRI for scoring of cartilage quantity and quality and bone structure in relation to osteoarthritis. The framework comprises a rigid multi-atlas registration method to detect the region of interest for each anatomical structure, and a supervised voxel classification method based on a <i>k</i>-NN classifier for structure-wise classification combined with supervised voxel classification (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Folkesson et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Folkesson, J. ∙ Dam, E.B. ∙ Olsen, O.F. ...</div><div><strong>Segmenting articular cartilage automatically using a voxel classification approach</strong></div><div><em>IEEE Trans. Med. Imaging.</em> 2007; <strong>26</strong>:106-115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/TMI.2006.886808" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33846214324" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17243589/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FTMI.2006.886808&amp;pmid=17243589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The corresponding Kellgren Lawrence (KL) grade/score of each subject was derived from OAI’s baseline clinical dataset (0.2.3). Morphometric data for proximal tibia/distal femur features were taken from all patients with an annotated KL grade of one (n&nbsp;= 75) and tested for homogeneity of variance using Fligner-Killeen and Levene’s tests as implemented in R (version 3.4.4). Significance was defined as p&nbsp;&lt; 0.05; test statistics and resulting p values are shown in <a id="crosref1710" href="#mmc5">Table S5</a>, Sheet 29.</div><div id="p0545" role="paragraph"><b>Description of OAI patient cohort dataset:</b> The OAI dataset processed consisted of 4129 individuals identified as either having, or being at risk of developing, osteoarthritis. This set contains groups of 3366 Caucasian and 763 Black/African-American individuals (on the basis of self-reported ethnicity); in order to control for potential effects of demographic history, only those individuals in the Caucasian group were considered for genetic analyses (see below). This group ranged in age from 45-79 (average of 62), consisted of 1498 males and 1868 females, for which 916 presented with no radiographic OA at entry, 774 had OA in one knee (KL grade &gt; 0), and 1676 had bilateral OA (KL grade &gt; 0).</div><div id="p0550" role="paragraph"><b>Sequence variation in OAI cohort:</b> Genotyping data for study participants in the OAI dataset was obtained through dbGaP with proper permissions (n&nbsp;= 4129); self-identified race was used to separate the dataset into White/Caucasian (n&nbsp;= 3366) and Black/African-American (n&nbsp;= 763) groups. Given the disparate sampling sizes of these two groups, and to avoid potential demographic signals on genotype, only individuals in the White/Caucasian group were further analyzed. Genotyped SNPs were extracted using plink (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Chang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Chang, C.C. ∙ Chow, C.C. ∙ Tellier, L.C. ...</div><div><strong>Second-generation PLINK: rising to the challenge of larger and richer datasets</strong></div><div><em>Gigascience.</em> 2015; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13742-015-0047-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84930213392" target="_blank" aria-label="Open Scopus in new tab">Scopus (1923)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13742-015-0047-8&amp;pmid=25722852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) version 1.9, and subsequently subset to those intersecting 20kb-padded ATAC-seq peaks (pooled from all ATAC-seq sets generated, lifted-over to hg18). Subset hg18 genotyping files were then lifted-over to hg19 using Crossmap (<span class="dropBlock reference-citations"><a id="body-ref-sref204-2" href="#" role="doc-biblioref" data-xml-rid="bib204" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref204-2" aria-controls="sref204-2">Zhao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref204-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">190.</div><div>Zhao, H. ∙ Sun, Z. ∙ Wang, J. ...</div><div><strong>CrossMap: a versatile tool for coordinate conversion between genome assemblies</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:1006-1007</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84897373336" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24351709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt730&amp;pmid=24351709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) version 0.3.3 using chain files obtained from UCSC. BEAGLE5 (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Browning et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Browning, B.L. ∙ Zhou, Y. ∙ Browning, S.R.</div><div><strong>A One-Penny Imputed Genome from Next-Generation Reference Panels</strong></div><div><em>Am. J. Hum. Genet.</em> 2018; <strong>103</strong>:338-348</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2018.07.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2018.07.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85056238160" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2018.07.015&amp;pmid=30100085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to impute variants from the 1KG3 European reference panel using the ‘conform-gt.jar’ and ‘beagle.28Sep18.793.jar’ utilities, leaving all settings to default. Imputed VCF files were intersected with knee-specific ATAC-seq region sets (elements fixed to a size of 500bp) using tabix&nbsp;(Li, 2011) (version 1.7.24) to obtain variants occurring within these putative regulatory regions. Subset files were converted to tab format with the following Unix command: bcftools query -f ‘%CHROM\t%POS\t%ID\t%REF\t%ALT[\t%SAMPLE&nbsp;= %TGT]\n’ -o out.v<i>cf.</i> in.v<i>cf.</i> The patient data was down-sampled to n&nbsp;= 25 (with replacement for 200 re-samples), using a custom R script, to make this analysis comparable to that done on the 1KG3 dataset in the intra-species diversity analysis (see above). Variants were defined using a minor allele frequency (MAF) threshold of &gt;&nbsp;= 0.05 for all subsamples, filtering tab-formatted files using a custom python script. Counts data was defined as the number of variants intersecting a given element and were averaged over re-sampled variant sets. Counts data between proximal tibia-specific, distal femur-specific and knee-common-specific elements were subsequently compared using hurdle modeling as described above, with significance assessed at adjusted p value&nbsp;&lt; 0.05 (<a id="crosref1730" href="#mmc5">Table S5</a>, Sheet 30). To check for consistency across sub-samples when taking averages for hurdle modeling, the same comparisons were done using the first and third quartiles of counts for a particular sequence across sub-sampled sets, to confirm that the statistical relationships between proximal-tibia and distal-femur elements observed are reproduced.</div><div id="p0555" role="paragraph"><b>Analysis of OA GWAS variants:</b> Acquisition of lead variants, as presented in <a id="crosref1735" href="#mmc7">Table S7</a> (Sheet 1), resulted in a final aggregate set of 95 leads. Leads were aggregated from a number of studies (<span class="dropBlock reference-citations"><a id="body-ref-sref103-3" href="#" role="doc-biblioref" data-xml-rid="bib2 bib25 bib26 bib24 bib40 bib50 bib51 bib82 bib101 bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-3" aria-controls="sref103-3">arcOGEN Consortium et&nbsp;al., 2012; Castaño Betancourt et&nbsp;al., 2012, 2016; Casalone et&nbsp;al., 2018; Day-Williams et&nbsp;al., 2011; Evangelou et&nbsp;al., 2011, 2014; Hindorff et&nbsp;al., 2009; Kerkhof et&nbsp;al., 2010; Klein et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref103-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>arcOGEN Consortium, arcOGEN Collaborators ∙ Zeggini, E. ...</div><div><strong>Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study</strong></div><div><em>Lancet.</em> 2012; <strong>380</strong>:815-823</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865595151" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22763110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2812%2960681-3&amp;pmid=22763110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Casalone, E. ∙ Tachmazidou, I. ∙ Zengini, E. ..., arcOGEN Consortium</div><div><strong>A novel variant in GLIS3 is associated with osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2018; <strong>77</strong>:620-623</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2017-211848" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044834428" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29436472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2017-211848&amp;pmid=29436472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Castaño-Betancourt, M.C. ∙ Cailotto, F. ∙ Kerkhof, H.J. ...</div><div><strong>Genome-wide association and functional studies identify the DOT1L gene to be involved in cartilage thickness and hip osteoarthritis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:8218-8223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1119899109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84861422537" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22566624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1119899109&amp;pmid=22566624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Castaño-Betancourt, M.C. ∙ Evans, D.S. ∙ Ramos, Y.F. ...</div><div><strong>Novel genetic variants for cartilage thickness and hip osteoarthritis</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1006260</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1006260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84994339045" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27701424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1006260&amp;pmid=27701424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Day-Williams, A.G. ∙ Southam, L. ∙ Panoutsopoulou, K. ..., arcOGEN Consortium</div><div><strong>A variant in MCF2L is associated with osteoarthritis</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>89</strong>:446-450</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2011.08.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2011.08.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-80052736294" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21871595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.08.001&amp;pmid=21871595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Evangelou, E. ∙ Valdes, A.M. ∙ Kerkhof, H.J. ..., arcOGEN Consortium, Translation Research in Europe Applied Technologies for Osteoarthritis (TreatOA)</div><div><strong>Meta-analysis of genome-wide association studies confirms a susceptibility locus for knee osteoarthritis on chromosome 7q22</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:349-355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.132787" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78751703383" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21068099/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.132787&amp;pmid=21068099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Evangelou, E. ∙ Kerkhof, H.J. ∙ Styrkarsdottir, U. ..., arcOGEN Consortium</div><div><strong>A meta-analysis of genome-wide association studies identifies novel variants associated with osteoarthritis of the hip</strong></div><div><em>Ann. Rheum. Dis.</em> 2014; <strong>73</strong>:2130-2136</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2012-203114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84883445177" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23989986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2012-203114&amp;pmid=23989986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Hindorff, L.A. ∙ Sethupathy, P. ∙ Junkins, H.A. ...</div><div><strong>Potential etiologic and functional implications of genome-wide association loci for human diseases and traits</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:9362-9367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0903103106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-67249117049" target="_blank" aria-label="Open Scopus in new tab">Scopus (2719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19474294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0903103106&amp;pmid=19474294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Kerkhof, H.J. ∙ Lories, R.J. ∙ Meulenbelt, I. ...</div><div><strong>A genome-wide association study identifies an osteoarthritis susceptibility locus on chromosome 7q22</strong></div><div><em>Arthritis Rheum.</em> 2010; <strong>62</strong>:499-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.27184" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-75749110202" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20112360/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.27184&amp;pmid=20112360" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Klein, J.C. ∙ Keith, A. ∙ Rice, S.J. ...</div><div><strong>Functional testing of thousands of osteoarthritis-associated variants for regulatory activity</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10439-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85066610023" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164647/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10439-y&amp;pmid=31164647" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Liu et&nbsp;al., 2017b</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Liu, Y. ∙ Yau, M.S. ∙ Yerges-Armstrong, L.M. ...</div><div><strong>Genetic Determinants of Radiographic Knee Osteoarthritis in African Americans</strong></div><div><em>J.&nbsp;Rheumatol.</em> 2017; <strong>44</strong>:1652-1658</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3899/jrheum.161488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85032637808" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28916551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3899%2Fjrheum.161488&amp;pmid=28916551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref200" href="#" role="doc-biblioref" data-xml-rid="bib126 bib129 bib138 bib152 bib171 bib172 bib199 bib200" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref200" aria-controls="sref200">Miyamoto et&nbsp;al., 2008; Nakajima et&nbsp;al., 2010; Panoutsopoulou et&nbsp;al., 2011; Ramos et&nbsp;al., 2014; Styrkarsdottir et&nbsp;al., 2014, 2017; Zengini et&nbsp;al., 2018; Zhai et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref200"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Miyamoto, Y. ∙ Shi, D. ∙ Nakajima, M. ...</div><div><strong>Common variants in DVWA on chromosome 3p24.3 are associated with susceptibility to knee osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2008; <strong>40</strong>:994-998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.176" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-48349105759" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18622395/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.176&amp;pmid=18622395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Nakajima, M. ∙ Takahashi, A. ∙ Kou, I. ...</div><div><strong>New sequence variants in HLA class II/III region associated with susceptibility to knee osteoarthritis identified by genome-wide association study</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e9723</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0009723" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78149408549" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20305777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0009723&amp;pmid=20305777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Panoutsopoulou, K. ∙ Southam, L. ∙ Elliott, K.S. ..., arcOGEN Consortium</div><div><strong>Insights into the genetic architecture of osteoarthritis from stage 1 of the arcOGEN study</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:864-867</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.141473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79953309875" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21177295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.141473&amp;pmid=21177295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Ramos, Y.F. ∙ Metrustry, S. ∙ Arden, N. ..., arcOGEN Consortium, TreatOA Collaborators</div><div><strong>Meta-analysis identifies loci affecting levels of the potential osteoarthritis biomarkers sCOMP and uCTX-II with genome wide significance</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2014; <strong>51</strong>:596-604</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2014-102478" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84907073808" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25057126/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2014-102478&amp;pmid=25057126" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">161.</div><div>Styrkarsdottir, U. ∙ Thorleifsson, G. ∙ Helgadottir, H.T. ..., TREAT-OA Consortium, arcOGEN Consortium</div><div><strong>Severe osteoarthritis of the hand associates with common variants within the ALDH1A2 gene and with rare variants at 1p31</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:498-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2957" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84899638695" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24728293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2957&amp;pmid=24728293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">162.</div><div>Styrkarsdottir, U. ∙ Helgason, H. ∙ Sigurdsson, A. ..., arcOGEN consortium</div><div><strong>Whole-genome sequencing identifies rare genotypes in COMP and CHADL associated with high risk of hip osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:801-805</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3816" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85015667730" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28319091/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3816&amp;pmid=28319091" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">185.</div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">186.</div><div>Zhai, G. ∙ van Meurs, J.B. ∙ Livshits, G. ...</div><div><strong>A genome-wide association study suggests that a locus within the ataxin 2 binding protein 1 gene is associated with hand osteoarthritis: the Treat-OA consortium</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2009; <strong>46</strong>:614-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.2009.067314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-69749123294" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19508968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.2009.067314&amp;pmid=19508968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Linkage disequilibrium (LD) between lead osteoarthritis SNPs was calculated using vcftools (version 0.1.15) and the EUR subset of 1KG3 (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); leads with R<sup>2</sup> greater than 0.5 were collapsed, retaining the lead with the lowest reported association p value, for a final set of 83 independent osteoarthritis loci. Proxy SNPs were subsequently obtained using ProxyFinder (<a id="intref0275" href="https://github.com/immunogenomics/harmjan/tree/master/ProxyFinder" target="_blank">https://github.com/immunogenomics/harmjan/tree/master/ProxyFinder</a>) on 1KG3 EUR data, using an LD-cutoff of 0.5, a minimum MAF of 0.05, and a window size of 500kb, with all other settings left to defaults. This resulted in a set of 4691 proxy SNPs (<a id="crosref1745" href="#mmc7">Table S7</a>, Sheet 2), which were combined with the leads and subsequently used for intersections with bedtools (version 2.26.0). For each element set (e.g., proximal tibia-specific) overlapping regions were merged with bedtools, with randomized background regions subsequently generated to match the number and average size of peaks within the target set using the bedtools ‘random’ function; each custom background (for each target set) consists of 10,000 sequence sets, with each randomized set intersected with the osteoarthritis variant set to generate a background distribution. These distributions were investigated using the ‘qqnorm’ (R base) and ‘qqPlot’ (car package) functions to look for visible deviations from normality, for which no obvious deviations were observed. The number of variants intersecting the target set was then compared to this background following standardization - statistical significance was assessed using a CDF of the standard normal distribution as implemented in the ‘pnorm’ function in R (version 3.4.4). Significance was defined as p&nbsp;&lt; 0.05, relevant test statistics and values are in <a id="crosref1750" href="#mmc7">Table S7</a> Sheet 8.</div><div id="p0560" role="paragraph"><b>Partitioned heritability of OA:</b> Summary statistics for the UK Biobanks dataset relating to self-reported osteoarthritis incidence was obtained from LDHub (<span class="dropBlock reference-citations"><a id="body-ref-sref206" href="#" role="doc-biblioref" data-xml-rid="bib206" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref206" aria-controls="sref206">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref206"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">192.</div><div>Zheng, J. ∙ Erzurumluoglu, A.M. ∙ Elsworth, B.L. ..., Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium</div><div><strong>LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:272-279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_192_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85028296791" target="_blank" aria-label="Open Scopus in new tab">Scopus (271)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27663502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw613&amp;pmid=27663502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (file ‘20002_1465.ukbb.sumstats.gz’, <a id="intref0280" href="http://ldsc.broadinstitute.org/gwashare" target="_blank">http://ldsc.broadinstitute.org/gwashare</a>) and analyzed using the LDSC software (<a id="intref0285" href="https://github.com/bulik/ldsc" target="_blank">https://github.com/bulik/ldsc</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref55-4" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-4" aria-controls="sref55-4">Finucane et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref55-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Finucane, H.K. ∙ Bulik-Sullivan, B. ∙ Gusev, A. ..., ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, RACI Consortium</div><div><strong>Partitioning heritability by functional annotation using genome-wide association summary statistics</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1228-1235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000443086" target="_blank" aria-label="Open Scopus in new tab">Scopus (535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26414678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3404&amp;pmid=26414678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Summary data was converted to an appropriate format for the ‘munge_numstats.py’ script using custom python code. The latest version of the ‘baseline-LD’ model available from the Alkes group (version 2.2) (<span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Hormozdiari et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Hormozdiari, F. ∙ Gazal, S. ∙ van de Geijn, B. ...</div><div><strong>Leveraging molecular quantitative trait loci to understand the genetic architecture of diseases and complex traits</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1041-1047</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0148-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85049003482" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0148-2&amp;pmid=29942083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0290" href="https://data.broadinstitute.org/alkesgroup/LDSCORE" target="_blank">https://data.broadinstitute.org/alkesgroup/LDSCORE</a>) was downloaded. Initial heritability analysis (using the ‘ldsc.py’ script, ‘–h2′ mode) was performed using 1000 Genomes reference allele frequencies provided by the Alkes group, along with weights extracted from the baseline-LD model using a custom script, yielding an observed scale h<sup>2</sup> estimate of 0.0188, in-line with the value reported for this trait by LDHub (<a id="intref0295" href="http://ldsc.broadinstitute.org/lookup" target="_blank">http://ldsc.broadinstitute.org/lookup</a>). For partitioned heritability analysis custom scripts were used to generate a modified version of the baseline-LD model with a reduced set of features including our element sets (see <a id="crosref1770" href="#mmc7">Table S7</a>, Sheet 13 for full list of features), which was subsequently used to re-calculate LD scores with the ‘ldsc.py’ script with the following options: plink files for the EUR subset of 1000G (obtained from the Alkes group, see above link),–ld-wind-cm 1, constraining to the same set of SNPs used in the original baseline LD model. Partitioned heritability of features was done using the ‘ldsc.py–h2′ mode, using re-calculated LD scores, extracted weights, reference 1KG3 frequencies, the ‘–overlap-annot’ flag, and all other settings left to defaults. P values for significant proportion of heritability captured by particular features was corrected for the number of features tested (n&nbsp;= 31) using BH correction. Results are shown in <a id="crosref1775" href="#mmc7">Table S7</a>, Sheet 13, with statistical significance defined as adjusted p value&nbsp;&lt; 0.05.</div><div id="p0565" role="paragraph"><b>Sequence diversity in 1KG3, EGDP and OAI populations:</b> Knee-specific element sets (i.e., distal-femur-specific, proximal-tibia-specific, and knee-common-specific) were pooled and subsequently used to extract variants from OAI VCF files (as described above) using tabix (<span class="dropBlock reference-citations"><a id="body-ref-sref108-5" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108-5" aria-controls="sref108-5">Li, 2011a</a><div class="dropBlock__holder" data-db-target-of="sref108-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Li, H.</div><div><strong>Tabix: fast retrieval of sequence features from generic TAB-delimited files</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:718-719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79951993896" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21208982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq671&amp;pmid=21208982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.7.24). Given that imputed variant information for the OAI dataset was generated using 1KG3 as a reference, and thus would likely bias apparent differences in alternative-allele occurrence between these two datasets, only variants which were directly genotyped in the OAI study were retained for subsequent analysis. These variants were subsequently subset to those for which information from 1KG3 was also available, resulting in a final set of 1946 variants. The number of reference and alternative alleles (0, 1 or 2) for all variants were summed per-individual using a custom Python script. In order to account for demographic history that may bias trends in allele frequencies between the OAI and 1KG3 populations, the CEU subset of 1KG3 (n&nbsp;= 99) was selected to match the Caucasian subset (n&nbsp;= 3366) of the OAI population. Given the disparities in sample size between the two datasets, subsampling was performed for 200 subsamples for both populations; the alternative-allele counts for individuals within each subsample were averaged, with the final set of averaged data-points compared between OAI and 1KG3 sets using the Student’s t test (one-sided, alternative of greater OAI diversity). Subsampling was done using n&nbsp;= 20 as well as n&nbsp;= 200 to confirm that results held under both conditions. In order to ensure robustness of comparison tests to subsample-averaging, first and third quartile values for alternative-allele counts within each subsample were also compared using the Student’s t test. To confirm that the comparison results were not sensitive to spurious sets of randomized sampling, the above algorithm was applied 1000 times, with t test statistics and p values averaged across replicates. Averages and standard deviations for replicated comparisons are reported in <a id="crosref1780" href="#mmc7">Table S7</a>, Sheet 14, along with all other statistics mentioned above. In order to replicate these results in another independent background population data from the EGDP (<span class="dropBlock reference-citations"><a id="body-ref-sref137-4" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-4" aria-controls="sref137-4">Pagani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref137-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was obtained, with variant files (both OAI and EGDP) subset to match those variants directly genotyped in both datasets. This final intersect set consisted of 1896 variants, for which alternative-allele counts were summed per-individual. To try to minimize biases due to demographic history the Caucasian subset of the OAI population (n&nbsp;= 3366) was compared to the EGDP dataset filtered to include only those individuals of European continent origin (i.e., Northern, Eastern, Western, Southern Europe) (n&nbsp;= 101). Subsampling, comparisons, and robustness checks were performed as described above for the 1KG3-OAI comparison. As an additional check to confirm the observed increase in OAI sequence variation relative to both background populations, the 1KG3 and EGDP subsets were combined (n&nbsp;= 200) and the above described statistical comparison algorithm was applied to compare the subset OAI population to this combined background population.</div><div id="p0570" role="paragraph">In order to confirm that this sequence behavior was a feature of knee-specific chondrocyte regulatory elements, rather than driven by systemic differences in allele frequencies between the OAI cohort and background populations, an independent element set (GM12878 ATAC-seq, described above) was used to subset the OAI dataset, impute, and subsequently filter for directly-genotyped variants falling within GM12878 elements as described above and in section ‘Sequence Variation in OAI Cohort’. This variant set was then filtered to include only those for which genotyping data in the 1KG3 and EGDP datasets was similarly available, resulting in a final consolidated variant set of 3932, for which per-individual alternative-allele counts were generated as above for the three populations. Subsampling, comparisons, and robustness checks for these count data were performed as above, the results of which are shown in <a id="crosref1790" href="#mmc7">Table S7</a>, Sheet 14.</div><div id="p0575" role="paragraph">In order to generate the density histogram shown in <a id="crosref1795" href="#fig4">Figure&nbsp;4</a>F, subsample-averaged alternative allele counts for OAI and 1KG3 subsamples (200 datapoints with n&nbsp;= 20 per sub-sample) were converted to density curves using the ‘density’ function (R base), using min, max values defined by the OAI/1KG3 superset to enforce a shared x axis. These curves were subsequently averaged over 1000 replicates, filtered for extreme x-values (i.e., those for which density approached 0), and plotted. The indicated significance bar represents the robusticity-checked t-test results presented in <a id="crosref1800" href="#mmc7">Table S7</a>, Sheet 14 for the OAI-1KG3 knee-specific element comparison.</div><div id="p0580" role="paragraph"><b>Selection and genetic drift analyses:</b> Variants from the entire 1KGP Phase 3 dataset were extracted for knee-specific element sets using tabix (version 1.7.24), filtered for duplicates and MAF &gt; 0.05 with bcftools (version 1.8), and subsequently separated into individual 1000 Genomes populations, to the exclusion of admixed populations including all Americas (with the exception of GIH), Finns (FIN) and Iberians (IBS) for a total of 18 populations across Europe, Asia and Africa. These populations include the following 1KG3 codes: BEB, CDX, CEU, CHB, CHS, ESN, GBR, GIH, GWD, ITU, JPT, KHV, LWK, MSL, PJL, STU, TSI, YRI. Fst calculations for all SNPs extracted for a given set were performed using the ‘Fst’ function of the pegas (<span class="dropBlock reference-citations"><a id="body-ref-sref139-2" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139-2" aria-controls="sref139-2">Paradis, 2010</a><div class="dropBlock__holder" data-db-target-of="sref139-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Paradis, E.</div><div><strong>pegas: an R package for population genetics with an integrated-modular approach</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:419-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp696" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77949527981" target="_blank" aria-label="Open Scopus in new tab">Scopus (757)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20080509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp696&amp;pmid=20080509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) package in R (version 0.11) which implements the Weir and Cockerham formula for Fst calculation (<span class="dropBlock reference-citations"><a id="body-ref-sref189-2" href="#" role="doc-biblioref" data-xml-rid="bib189" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref189-2" aria-controls="sref189-2">Weir and Cockerham, 1984</a><div class="dropBlock__holder" data-db-target-of="sref189-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">177.</div><div>Weir, B.S. ∙ Cockerham, C.C.</div><div><strong>Estimating F-statistics for the analysis of population structure</strong></div><div><em>Evolution.</em> 1984; <strong>38</strong>:1358-1370</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2307/2408641" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28563791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2307%2F2408641&amp;pmid=28563791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Values were calculated for all population comparisons (n&nbsp;= 153), with the resulting matrix used to perform principal components analysis (PCA) using the prcomp function as implemented in R (version 3.4.4) (<a id="crosref1815" href="#figs5">Figure&nbsp;S5</a>C). To gauge the divergence trends captured by principal components (PCs) Pearson correlations between PCs and population comparisons (e.g., CDX and GIH) were calculated and subsequently clustered (using hierarchical clustering of population comparisons, e.g., BEB-CHS) for visualization using the ComplexHeatmap (<span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Gu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M.</div><div><strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:2847-2849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw313" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84992187637" target="_blank" aria-label="Open Scopus in new tab">Scopus (664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw313&amp;pmid=27207943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) R package (version 1.17.1) (<a id="crosref1825" href="#figs3">Figure&nbsp;S3</a>D). For variant clustering the fviz ‘nbclust’ function from the factoextra (<span class="dropBlock reference-citations"><a id="body-ref-sref99-2" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-2" aria-controls="sref99-2">Kassambara and Mundt, 2017</a><div class="dropBlock__holder" data-db-target-of="sref99-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Kassambara, A. ∙ Mundt, F.</div><div><strong>Factoextra: extract and visualize the results of multivariate data analyses</strong></div><div>R package version 1.0. 4, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KassambaraF.MundtFactoextra%3A+extract+and+visualize+the+results+of+multivariate+data+analyses2017R+package+version+1.0.+4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) package (version 1.0.5) was used to estimate ideal ‘k’ for CLARA clustering (as implemented in cluster [<span class="dropBlock reference-citations"><a id="body-ref-sref119-2" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-2" aria-controls="sref119-2">Maechler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref119-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Maechler, M. ∙ Rousseeuw, P. ∙ Struyf, A. ...</div><div><strong>cluster: Cluster Analysis Basics and Extensions. R Packages Version 1</strong></div><div>2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MaechlerP.RousseeuwA.StruyfM.HubertK.Hornikcluster%3A+Cluster+Analysis+Basics+and+Extensions.+R+Packages+Version+12018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>], 2.0.7-1) of each set of SNPs using the silhouette method (also from cluster), which determines the ideal cluster number that minimizes within-group Euclidean distances, stepping through a range of possible k values (k&nbsp;= 1-10). Silhouette plots were generated for all knee-specific sets separately, consistently indicating an ideal ‘k’ of 2. After cluster generation the clustered pairwise-Fst matrices were visualized using ComplexHeatmap and hierarchical clustering for both population comparisons as well as individual SNPs, as seen in <a id="crosref1840" href="#figs3">Figures S3</a>E and S3F.</div><div id="p0585" role="paragraph"><b>ATAC-seq - selection test intersection:</b> To test whether knee elements intersect regions known to be under selection in humans, we utilized selection data from the 1KGP and the Estonian Genome Diversity Project (EGDP) (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Jagoda et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Jagoda, E. ∙ Lawson, D.J. ∙ Wall, J.D. ...</div><div><strong>Disentangling Immediate Adaptive Introgression from Selection on Standing Introgressed Variation in Humans</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:623-630</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85046466019" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29220488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx314&amp;pmid=29220488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the 1KGP, selected regions were acquired from Grossman and colleagues (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Grossman et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Grossman, S.R. ∙ Andersen, K.G. ∙ Shlyakhter, I. ..., 1000 Genomes Project</div><div><strong>Identifying recent adaptations in large-scale genomic data</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:703-713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874069538" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.035&amp;pmid=23415221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) who used the Genome-Wide Composite of Multiple Signals Test (CMS<sub><i>GW</i></sub>) to identify regions of the genome under selection (FDR 19%) in the Yoruba (YRI), Northern and Western European (CEU), Han Chinese (CHB), and Japanese (JPT) 1KGP populations (see Table S1 of <span class="dropBlock reference-citations"><a id="body-ref-sref69-3" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-3" aria-controls="sref69-3">Grossman et&nbsp;al. [2013</a><div class="dropBlock__holder" data-db-target-of="sref69-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Grossman, S.R. ∙ Andersen, K.G. ∙ Shlyakhter, I. ..., 1000 Genomes Project</div><div><strong>Identifying recent adaptations in large-scale genomic data</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:703-713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874069538" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.035&amp;pmid=23415221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>]). The coordinates of these selected regions from Grossman and colleagues were lifted over from the hg18 to the hg19 human genome build using the UCSC Genome Browser LiftOver tool, with overlap of at least one base pair with an ATAC-seq element was reported.</div><div id="p0590" role="paragraph">Selected regions from the EGDP came from two sources. Tests for regions under recent (&lt; 30kya) positive selection were reported by Pagani and colleagues (<span class="dropBlock reference-citations"><a id="body-ref-sref137-5" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-5" aria-controls="sref137-5">Pagani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref137-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), who identified regions of recent selection using three selection tests: iHS (<span class="dropBlock reference-citations"><a id="body-ref-sref185" href="#" role="doc-biblioref" data-xml-rid="bib185" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref185" aria-controls="sref185">Voight et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref185"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Voight, B.F. ∙ Kudaravalli, S. ∙ Wen, X. ...</div><div><strong>A map of recent positive selection in the human genome</strong></div><div><em>PLoS Biol.</em> 2006; <strong>4</strong>:e72</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0040072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33745160439" target="_blank" aria-label="Open Scopus in new tab">Scopus (138)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16494531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0040072&amp;pmid=16494531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), nSL (<span class="dropBlock reference-citations"><a id="body-ref-sref53" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53" aria-controls="sref53">Ferrer-Admetlla et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref53"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Ferrer-Admetlla, A. ∙ Liang, M. ∙ Korneliussen, T. ...</div><div><strong>On detecting incomplete soft or hard selective sweeps using haplotype structure</strong></div><div><em>Mol. Biol. Evol.</em> 2014; <strong>31</strong>:1275-1291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msu077" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84899542092" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24554778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsu077&amp;pmid=24554778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and Tajima’s D (<span class="dropBlock reference-citations"><a id="body-ref-sref176" href="#" role="doc-biblioref" data-xml-rid="bib176" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref176" aria-controls="sref176">Tajima, 1989</a><div class="dropBlock__holder" data-db-target-of="sref176"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Tajima, F.</div><div><strong>Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</strong></div><div><em>Genetics.</em> 1989; <strong>123</strong>:585-595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/genetics/123.3.585" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2513255/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgenetics%2F123.3.585&amp;pmid=2513255" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). They employed these tests on individuals from 12 populations in the EGDP: West and Central Africa, Middle East, South and West Europe, East and North Europe, Volga-Uralic, South Asia, West Siberia, South Siberia and Mongolia, Central Siberia, Northeast Siberia, Mainland East and Southeast Asia, and Island Southeast Asia. Tests for regions under positive selection in humans but in the more distant past - i.e., after the split between Africans and non-Africans but before the split between East and West Eurasian populations (50-30kya) - were identified by <span class="dropBlock reference-citations"><a id="body-ref-sref93-3" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-3" aria-controls="sref93-3">Jagoda et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref93-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Jagoda, E. ∙ Lawson, D.J. ∙ Wall, J.D. ...</div><div><strong>Disentangling Immediate Adaptive Introgression from Selection on Standing Introgressed Variation in Humans</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:623-630</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85046466019" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29220488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx314&amp;pmid=29220488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> using the 3P-CLR statistic (<span class="dropBlock reference-citations"><a id="body-ref-sref151" href="#" role="doc-biblioref" data-xml-rid="bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151" aria-controls="sref151">Racimo, 2016</a><div class="dropBlock__holder" data-db-target-of="sref151"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>Racimo, F.</div><div><strong>Testing for ancient selection using cross-population allele frequency differentiation</strong></div><div><em>Genetics.</em> 2016; <strong>202</strong>:733-750</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.115.178095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84973624508" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26596347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.115.178095&amp;pmid=26596347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Jagoda and colleagues employed this statistic on EGDP individuals from the South and West Europe, East and North Europe, Mainland East and Southeast Asia and West and Central Africa populations. Both the recent and ancient selection tests were conducted on 200kb windows, with windows in the top 1% of signals considered significant. To determine the overlap between knee elements and these regions, elements were binned into 200kb windows. An element was scored as overlapping a recent selection signal if the 200kb window containing it had been in the top 1% of one of the three recent tests conducted by <span class="dropBlock reference-citations"><a id="body-ref-sref137-6" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-6" aria-controls="sref137-6">Pagani et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref137-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. An element was scored as overlapping an ancient selection signal if the 200kb window containing it had been in the top 1% of one of the 3P-CLR tests conducted by <span class="dropBlock reference-citations"><a id="body-ref-sref93-4" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-4" aria-controls="sref93-4">Jagoda et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref93-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Jagoda, E. ∙ Lawson, D.J. ∙ Wall, J.D. ...</div><div><strong>Disentangling Immediate Adaptive Introgression from Selection on Standing Introgressed Variation in Humans</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:623-630</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85046466019" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29220488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx314&amp;pmid=29220488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Enrichment/depletion for intersections with selection windows was assessed using hypergeometric tests (‘phyper’ from R Base); comparisons were defined as the number of unique genomic windows intersected by any element, and the number of said windows also being under selection, compared to the total number of genomic windows and the total genome-wide number of windows under selection. P values were adjusted for the number of sets compared (n&nbsp;= 20); relevant test-statistics, p values, and fold-changes are presented in <a id="crosref1900" href="#mmc8">Table S8</a>, Sheets 20-21, as well as <a id="crosref1905" href="#fig4">Figure&nbsp;4</a>D. Selected regions defined by <span class="dropBlock reference-citations"><a id="body-ref-sref69-4" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-4" aria-controls="sref69-4">Grossman et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref69-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Grossman, S.R. ∙ Andersen, K.G. ∙ Shlyakhter, I. ..., 1000 Genomes Project</div><div><strong>Identifying recent adaptations in large-scale genomic data</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:703-713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874069538" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.035&amp;pmid=23415221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> were tested for enriched overlap with element sets using regioneR (version 1.8.1) using the ‘permTest’ function, generating 1000 randomized region sets as a background using the ‘circularRandomizeRegions’ option and the ‘count.once’ flag, with all other options set to defaults. Significance was assessed at p&nbsp;&lt; 0.05 (<a id="crosref1915" href="#mmc8">Table S8</a>, Sheet 22).</div><div id="p0595" role="paragraph"><b>Haploblock analysis:</b> 1MB regions around each selection window of interest were defined and used to extract variant data from EGDP (<span class="dropBlock reference-citations"><a id="body-ref-sref137-7" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137-7" aria-controls="sref137-7">Pagani et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref137-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for populations which were identified as being subject to selection in the given locus using bcftools (version 1.8), with duplicate SNPs removed. Files were subsequently re-processed with vcftools (version 0.1.15) and PLINK (<span class="dropBlock reference-citations"><a id="body-ref-sref28-3" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-3" aria-controls="sref28-3">Chang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref28-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Chang, C.C. ∙ Chow, C.C. ∙ Tellier, L.C. ...</div><div><strong>Second-generation PLINK: rising to the challenge of larger and richer datasets</strong></div><div><em>Gigascience.</em> 2015; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13742-015-0047-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84930213392" target="_blank" aria-label="Open Scopus in new tab">Scopus (1923)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13742-015-0047-8&amp;pmid=25722852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.90) (<a id="intref0300" href="http://www.cog-genomics.org/plink/1.9/" target="_blank">http://www.cog-genomics.org/plink/1.9/</a>) for use with Haploview (<span class="dropBlock reference-citations"><a id="body-ref-sref5-2" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5-2" aria-controls="sref5-2">Barrett et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref5-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Barrett, J.C. ∙ Fry, B. ∙ Maller, J. ...</div><div><strong>Haploview: analysis and visualization of LD and haplotype maps</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:263-265</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bth457" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-13444269543" target="_blank" aria-label="Open Scopus in new tab">Scopus (10961)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15297300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbth457&amp;pmid=15297300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 4.2) with a -minMAF cutoff of 0.05 and the default Gabriel (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Gabriel et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Gabriel, S.B. ∙ Schaffner, S.F. ∙ Nguyen, H. ...</div><div><strong>The structure of haplotype blocks in the human genome</strong></div><div><em>Science.</em> 2002; <strong>296</strong>:2225-2229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1069424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-18444369013" target="_blank" aria-label="Open Scopus in new tab">Scopus (4402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12029063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1069424&amp;pmid=12029063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) block generation algorithm, all other flags being left to default settings. To visually assess whether a given osteoarthritis lead and associated proxies were present on the selected or alternate haploblock in the population, a multiple alignment based on SNP sequences (MAF &gt;&nbsp;= 0.05) for all individuals was constructed either across a haploblock or across the full 1MB region. RaxML (<span class="dropBlock reference-citations"><a id="body-ref-sref169-2" href="#" role="doc-biblioref" data-xml-rid="bib169" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref169-2" aria-controls="sref169-2">Stamatakis, 2006</a><div class="dropBlock__holder" data-db-target-of="sref169-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Stamatakis, A.</div><div><strong>RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</strong></div><div><em>Bioinformatics.</em> 2006; <strong>22</strong>:2688-2690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btl446" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750403801" target="_blank" aria-label="Open Scopus in new tab">Scopus (11499)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16928733/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtl446&amp;pmid=16928733" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 7.3.0) was used to generate a maximum-likelihood tree with the GTR-CAT model along with bootstrap (n&nbsp;= 100) support. Chimpanzee reference sequences for all SNPs used in the multiple alignment were taken from panTro4 and used to assign ancestral (red) and derived (blue) states for all aligned sequences. See <a id="crosref1945" href="#figs3">Figures S3</a>G–S3J.</div><div id="p0600" role="paragraph"><b>Motif analysis – de-novo:</b> Sequence sets for the knee-specific regions (with a fixed length of 500&nbsp;bp) were generated using reference sequences from mm10 and hg19 for original and lifted-over regions (respectively). HOMER (version 4.10) <i>de novo</i> motif analysis was performed on each sequence set using a 10x random shuffling as a background set. <i>De novo</i> motifs were compared to a vertebrate motif library included with HOMER, which incorporates the JASPAR (<span class="dropBlock reference-citations"><a id="body-ref-sref120" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120" aria-controls="sref120">Mathelier et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref120"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Mathelier, A. ∙ Fornes, O. ∙ Arenillas, D.J. ...</div><div><strong>JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:D110-D115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1176" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26531826/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1176&amp;pmid=26531826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) database; matches are scored using Pearson’s correlation coefficient of vectorized motif matrices (PWMs), with neutral frequencies (0.25) substituted for non-overlapping (e.g., gapped) positions. Best-matching motif PWMs are shown in <a id="crosref1955" href="#mmc3">Table S3</a>, Sheets 1-6.</div><div id="p0605" role="paragraph"><b>Motif analysis – inter-species:</b> For analysis of inter-species alterations to the predicted binding sites of certain TFs, a set of TFs bearing resemblance to the <i>de-novo</i> motifs identified in our element sets was combined with a previously-defined set of TFs associated with chondrogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Liu et&nbsp;al., 2017a</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Liu, C.-F. ∙ Samsa, W.E. ∙ Zhou, G. ...</div><div><strong>Transcriptional control of chondrocyte specification and differentiation</strong></div><div><em>Semin. Cell Dev. Biol.</em> 2017; <strong>62</strong>:34-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.semcdb.2016.10.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85006055004" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27771362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.semcdb.2016.10.004&amp;pmid=27771362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) for a total of n&nbsp;= 45 transcription factors for which defined PWMs were available from the compendium provided by MotifDb (<span class="dropBlock reference-citations"><a id="body-ref-sref165-2" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref165-2" aria-controls="sref165-2">Shannon and Richards, 2014</a><div class="dropBlock__holder" data-db-target-of="sref165-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Shannon, P. ∙ Richards, M.</div><div><strong>MotifDb: An annotated collection of protein-DNA binding sequence motifs. R Package. Version 1</strong></div><div>2014</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.ShannonM.RichardsMotifDb%3A+An+annotated+collection+of+protein-DNA+binding+sequence+motifs.+R+Package.+Version+12014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which contains matrices from various sources including the JASPAR database.</div><div id="p0610" role="paragraph">A human-chimpanzee whole-genome alignment was obtained from UCSC and used to extract orthologous reference hg38 and panTro4 sequences for all regions in each set (lifted-over from hg19 to hg38), regions fixed to a constant size of 500&nbsp;bp (as done for the <i>de-novo</i> analysis above), with custom code. Aligned sequences were compared to identify human/chimpanzee divergences within elements of a given set. These divergences were subsequently analyzed using motifbreakR (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Coetzee et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Coetzee, S.G. ∙ Coetzee, G.A. ∙ Hazelett, D.J.</div><div><strong>motifbreakR: an R/Bioconductor package for predicting variant effects at transcription factor binding sites</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3847-3849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84950293634" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26272984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv470&amp;pmid=26272984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 1.6.0). Briefly, this software first takes the set of PWMs for TFs of interest and scans the genomic region around altered bases (here, inter-species nucleotide divergences) for sequence matches to a particular PWM (via log-odds scoring). Once potential binding sites are identified via this <i>in-silico</i> scoring, these sites are subsequently re-scored using the altered base (here, replacing the hg38 sequence with the panTro4 equivalent); effect of an altered base is calculated as relative difference in log-odds scores for site, with positive differences indicating improved matching of a sequence to a TF PWM, negative differences indicating diminished matching. Thus, for human-chimp divergences intersecting sequence matches for a particular TF’s PWM, an ‘improving’ or ‘disrupting’ effect classification can be applied.</div><div id="p0615" role="paragraph">This scanning and scoring method was applied to human-chimp divergences within elements from the three knee-specific sets, with the final number of divergences intersecting sequence matches for a particular TF PWM counted.</div><div id="p0620" role="paragraph">A random background of 500 region sets (each consisting of 464 lifted-over human sequences, the size of the smallest set, the knee-common-specific set) with a size of 500&nbsp;bp was generated from genome-wide sampling; human/chimpanzee sequences for these randomized sets were subsequently compared to generate randomized sets of human-chimp divergences. These sets were analyzed with the same scanning/scoring method, with the number of intersecting divergences for a particular TF calculated as for the target sets. Values for both target and randomized sets were corrected as divergences per bp of sequence to account for differences in set size and plotted as histograms (<a id="crosref1970" href="#fig2">Figure&nbsp;2</a>A). These distributions were investigated using the ‘qqnorm’ (R&nbsp;base) and ‘qqPlot’ (car package) functions to look for visible deviations from normality, for which no obvious deviations were observed. Values were standardized and statistical significance was assessed using a CDF of the standard normal distribution as implemented in the ‘pnorm’ function in R (version 3.4.4). P values for significant deviations from the background distribution were corrected for the number of TFs tested using a BH correction. Significance was defined as adjusted p&nbsp;&lt; 0.05 (<a id="crosref1975" href="#mmc3">Table S3</a>, Sheets 7-9).</div><div id="p0625" role="paragraph"><b>Motif analysis – intra-species:</b> Human sequence variants (SNPs) identified with a MAF &gt;&nbsp;= 0.05 in the full 1KGP Phase 3 (<span class="dropBlock reference-citations"><a id="body-ref-sref3-4" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-4" aria-controls="sref3-4">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) set which overlap knee elements were analyzed for intersections with TF PWM sequence matches (i.e., predicted TF binding sites) as described above, using the same set of 45 TF PWMs. Alongside these sets of SNPs, variants clustered by pairwise population Fst (see above) were analyzed with the same methodology. SNPs per bp of sequence values were calculated for each element set (as well as clustered variant sets) to account for differences in set size. The same background region set used in the inter-species motif analysis was used, with counts of human variants intersecting predicted TF sequence matches adjusted for set size. Statistical deviations from the background distribution were similarly assessed using a CDF of the normal distribution as implemented in ‘pnorm’, with BH correction applied to account for the number of TFs tested. Significance was defined as adjusted p&nbsp;&lt; 0.05 (<a id="crosref1985" href="#mmc3">Table S3</a>, Sheets 10-20).</div><div id="p0630" role="paragraph"><b>Motif analysis – OA variants:</b> All defined TF PWMs available from MotifDb (version 1.6.0) were retrieved, constraining results to single PWMs for unique TFs (taking JASPAR-sourced PWMs when available), for a final set of 1686 PWMs used. OA lead variants and their associated proxies (after filtering for inconsistent ref/alt classifications with respect to hg19 reference sequence, a final set of 4172 variants was used) were analyzed using motifbreakR (version 1.6.0), as described above, for intersections with predicted TF sequence motifs (or, ‘binding sequences’) for the TF superset. The distribution of variant intersections with predicted binding sequences is shown in <a id="crosref1990" href="#fig4">Figure&nbsp;4</a>B, with intersections counted per TF PWM, for the entire OA lead/proxy set. This distribution was assessed using the ‘qqnorm’ (R base) and ‘qqPlot’(car version 3.0.3) functions – these suggested substantial deviations from a normal distribution. Accordingly, the ‘fitdistrplus’ package (version 1.0-14) was used to determine an appropriate distribution to fit the data. The ‘descdist’ function was initially used to assess curve behavior; given this, goodness-of-fit statistics (from the ‘gofstat’ function) for gamma, negative-binomial, exponential, log-normal and Weibull distributions were compared, with the gamma distribution subsequently selected given the observed left-skew of the logged distribution. Gamma distribution parameters (‘shape’ and ‘rate’ in the R implementation of ‘pgamma’) were fit using a bootstrap method (‘bootdist’ from ‘fitdistrplus’), with the median parameter estimates from 1000 samples used to define the distribution for significance testing of particular TF PWMs with ‘pgamma’ (upper-tail p values). Significant deviations from this superset distribution were tested for the set of TFs which showed significant biases in their intersection by human-chimp nucleotide divergences (adjusted p value&nbsp;&lt; 0.05; n&nbsp;= 28, see <a id="crosref1995" href="#mmc7">Table S7</a>, Sheets 7-9), with p values accordingly adjusted using BH correction (<a id="crosref2000" href="#mmc7">Table S7</a>, Sheet 10). Additionally, deviations of particular TF PWMs from the superset distribution were also tested for OA variants falling within element sets (e.g., chondrocyte-pooled, knee-specific, GM12878, embryonic brain)– gamma distribution parameters for different element sets (e.g., knee-specific pooled, distal femur-specific) were fit using the ‘bootdist’ function as above, with enrichment p values similarly adjusted for n&nbsp;= 28 TF PWMs tested (<a id="crosref2005" href="#mmc7">Table S7</a>, Sheets 10-11).</div><div id="p0635" role="paragraph">Predicted binding sequences for chondrogenesis-associated and <i>de-novo</i> identified motifs (n&nbsp;= 45, used above in inter- and intra-species analyses) intersected by element-captured OA variants were similarly identified with motifbreakR (<a id="crosref2010" href="#mmc7">Table S7</a>, Sheet 9) and are shown as a per-TF histogram (for the pooled knee-specific set, <a id="crosref2015" href="#fig4">Figure&nbsp;4</a>C).</div><div id="p0640" role="paragraph">To test whether knee elements capture a significant proportion of OA variants intersecting binding sequences for a particular TF (e.g., elements tend to capture OA variants predicted to intersect KLF5 sites) hypergeometric testing was performed using the ‘phyper’ function from R base, testing the set of inter-species biased TFs used earlier (n&nbsp;= 28). Comparisons were defined as: (a) the number of predicted binding sites (for target TF) intersected by element-captured variants, and the total number of predicted binding sites (for all TFs tested) intersected by element-captured variants, (b) the number of predicted binding sites (for target TF) intersected by all OA variants, and the total number of predicted binding sites (for all TFs tested) intersected by all OA variants. P&nbsp;values were adjusted for the number of TFs tested (n&nbsp;= 28); relevant test-statistics, p values and fold-changes are presented in <a id="crosref2020" href="#mmc7">Table S7</a>, Sheet 12.</div><div id="p0645" role="paragraph"><b>TF binding sites HaploregV4.1 and UniProbe:</b> To identify upstream TFs physically bound at the <i>R4</i> enhancer and rs6060369 variant position, we used HaploReg (<span class="dropBlock reference-citations"><a id="body-ref-sref188-2" href="#" role="doc-biblioref" data-xml-rid="bib188" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref188-2" aria-controls="sref188-2">Ward and Kellis, 2012</a><div class="dropBlock__holder" data-db-target-of="sref188-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">176.</div><div>Ward, L.D. ∙ Kellis, M.</div><div><strong>HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D930-D934</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84858779229" target="_blank" aria-label="Open Scopus in new tab">Scopus (1288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22064851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr917&amp;pmid=22064851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (version 4.162) with default parameters (<a id="intref0305" href="https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php" target="_blank">https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php</a>). We also queried the UniProbe database (<span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Hume et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Hume, M.A. ∙ Barrera, L.A. ∙ Gisselbrecht, S.S. ...</div><div><strong>UniPROBE, update 2015: new tools and content for the online database of protein-binding microarray data on protein-DNA interactions</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D117-D122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84922580691" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25378322/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1045&amp;pmid=25378322" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0310" href="http://the_brain.bwh.harvard.edu/uniprobe/about.php/" target="_blank">http://the_brain.bwh.harvard.edu/uniprobe/about.php/</a>), which is based on replicate experimental measurements of <i>in&nbsp;vitro</i> binding interactions between expressed TFs and all possible 8-mer target oligonucleotides, to identify additional factors. The HaploReg analysis identified 5 TFs physically bound via ENCODE ChIP-seq datasets: CJUN, JUND, POL2, STAT3, ZNF263. For the UniProbe analysis (<a id="crosref2035" href="#mmc9">Table S9</a>, Sheets 7-9), two 15 base-pair sequences differing only at the non-risk “C” and risk “T” alleles at rs6060369 were analyzed using Score Threshold: 0.2 and Species: <i>Homo sapiens</i>. This produced 7 distinct TF predictions for non-risk, and 19 distinct TF predictions for risk allele. For a TF binding site to be considered reduced or gained, the site for that TF must exhibit an enrichment score change of greater than at least 0.10, which corresponded to significant changes in Z-score. Next, both lists of potential upstream regulators exhibiting either true physical binding (Haploreg) or marked reduction/gain in binding affinity (UniProbe) were then screened using expression and phenotypic data in order to identify those TFs also expressed in, or functionally required, specifically in hind limb joints. To carry out this analysis we used data in VisiGene (<span class="dropBlock reference-citations"><a id="body-ref-sref98-3" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-3" aria-controls="sref98-3">Karolchik et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref98-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Karolchik, D. ∙ Barber, G.P. ∙ Casper, J. ...</div><div><strong>The UCSC Genome Browser database: 2014 update</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D764-D770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84891771466" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1168&amp;pmid=24270787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0315" href="https://genome.ucsc.edu/cgi-bin/hgVisiGene" target="_blank">http://genome.ucsc.edu/cgi-bin/hgVisigene</a>), Eurexpress (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Diez-Roux et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Diez-Roux, G. ∙ Banfi, S. ∙ Sultan, M. ...</div><div><strong>A high-resolution anatomical atlas of the transcriptome in the mouse embryo</strong></div><div><em>PLoS Biol.</em> 2011; <strong>9</strong>:e1000582</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.1000582" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79851473886" target="_blank" aria-label="Open Scopus in new tab">Scopus (397)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21267068/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.1000582&amp;pmid=21267068" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0320" href="http://www.eurexpress.org/ee/" target="_blank">http://www.eurexpress.org/ee/</a>), Genepaint (<span class="dropBlock reference-citations"><a id="body-ref-sref183" href="#" role="doc-biblioref" data-xml-rid="bib183" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref183" aria-controls="sref183">Visel et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref183"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">171.</div><div>Visel, A. ∙ Thaller, C. ∙ Eichele, G.</div><div><strong>GenePaint.org: an atlas of gene expression patterns in the mouse embryo</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:D552-D556</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkh029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14681479/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkh029&amp;pmid=14681479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0325" href="http://www.genepaint.org/Frameset.html" target="_blank">http://www.genepaint.org/Frameset.html</a>), and the Mouse Genome Informatics expression and phenotypic databases (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Blake et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Blake, J.A. ∙ Eppig, J.T. ∙ Kadin, J.A. ..., the Mouse Genome Database Group</div><div><strong>Mouse Genome Database (MGD)-2017: community knowledge resource for the laboratory mouse</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D723-D729</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1040" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85016041593" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1040&amp;pmid=27899570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="intref0330" href="http://www.informatics.jax.org/" target="_blank">http://www.informatics.jax.org/</a>). Of the listed factors, Pitx1 met the above criteria as it is expressed in hind limbs, is required for normal development of hind limb long bones, particularly knees, in humans and mice (<span class="dropBlock reference-citations"><a id="body-ref-sref130-2" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130-2" aria-controls="sref130-2">Nemec et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref130-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Nemec, S. ∙ Luxey, M. ∙ Jain, D. ...</div><div><strong><i>Pitx1</i> directly modulates the core limb development program to implement hindlimb identity</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:3325-3335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.154864" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85029782232" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28807899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.154864&amp;pmid=28807899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and has roles in osteoarthritis (<span class="dropBlock reference-citations"><a id="body-ref-sref140-2" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-2" aria-controls="sref140-2">Picard et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref140-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Picard, C. ∙ Azeddine, B. ∙ Moldovan, F. ...</div><div><strong>New emerging role of pitx1 transcription factor in osteoarthritis pathogenesis</strong></div><div><em>Clin. Orthop. Relat. Res.</em> 2007; <strong>462</strong>:59-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/BLO.0b013e3180d09d9c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34548400346" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17549029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FBLO.0b013e3180d09d9c&amp;pmid=17549029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><div id="p0650" role="paragraph"><b>MicroCT analyses:</b> In all cases, the measurer (AMK and SH) was blinded to specimens’ genotype. Quantified anatomical indices were compared between the genotypes using multivariate linear regression analysis with anatomy used as dependent variable, and genotype and line as independent variables. Sidak post hoc was used to correct for multiple comparisons between genotypes. Analysis was conducted in SPSS (IBM Corp., Armonk, NY). P values are two-sided and the statistical significance was assessed at alpha&nbsp;= 0.05 (<a id="crosref2070" href="#figs4">Figure&nbsp;S4</a>, <a id="crosref2075" href="#mmc9">Table S9</a>, Sheets 1-4).</div><div id="p0655" role="paragraph"><b>Assessment of osteoarthritis:</b> A Mann-Whitney test was used to compare the OARSI scores between the wild-type (n&nbsp;= 5 at P30; n&nbsp;= 6 at 1 year) and homozygous (n&nbsp;= 5 at P30; n&nbsp;= 14 at 1 year) knees. Linear regression was used to investigate the associations between OARSI score and quantified anatomical indices of the femur and tibia. The analyses were conducted in SPSS (IBM Corp., Armink, NY). P values are two-sided and the statistical significance was assessed at alpha&nbsp;= 0.05.</div><div id="p0660" role="paragraph"><b>Allele-specific expression:</b> Pyrosequencing results for each SNP were used to calculate the allelic ratios of C57BL/6J (<i>R4</i> null allele or <i>R4</i> rs6060369 “T” allele in the heterozygous state) to 129X1/SVJ (wild-type allele). Each ratio of cDNA products found in heterozygous animals was then normalized by the ratio of WT C57BL/6J to 129X1/SVJ genomic products, amplified from known 1:1 mixtures of each sequence. The permutation test, a nonparametric measure, was used to determine significance between the wild-type and heterozygous allelic expression ratio with the {perm} module in R.</div><div id="p0665" role="paragraph"><b>Gene expression experiments:</b> For each gene (<i>GDF5</i>, <i>CEP250</i>, and <i>UQCC1</i>) expression data was normalized relative to house-keeping gene expression and compared between control and experimental condition (i.e., <i>R4</i> enhancer deletion or <i>R4</i> 41&nbsp;bp variant position deletion). All data are presented as the mean ± SEM. Individual pairwise comparisons between control and experimental condition were analyzed by two- sample, two-tailed Student’s t test unless otherwise noted, with p&nbsp;&lt; 0.05 regarded as significant. n numbers listed in figure legends (n&nbsp;= 3 biological replicates per comparison).</div><div id="p0670" role="paragraph"><b>rs6060369 luciferase reporter assay:</b> Upon acquiring luciferase data after transfection of rs6060369 <i>R4</i> enhancer constructs, we first normalized each Firefly luciferase value per well by its corresponding Renilla luciferase value per well, and then compared the mean expression of the reference allele construct (normalized by empty vector) to that of the mean expression of the alternate allele construct (normalized by empty vector). For each, the means ± SEM of multiple independent measurements were calculated. The unpaired two-tailed Student’s t test was used to determine the significance of differences between means. The p values from independent experiments were combined across three experiments using Fisher’s combined probability test.</div></section><section id="sec-4-6"><h3>Data and Code Availability</h3><div id="p0675" role="paragraph">The data supporting the findings of this study, as presented in <a id="crosref2080" href="#fig1">Figures 1</a>, <a id="crosref2085" href="#fig2">2</a>, <a id="crosref2090" href="#fig3">3</a>, <a id="crosref2095" href="#fig4">4</a>, <a id="crosref2100" href="#fig5">5</a>, <a id="crosref2105" href="#fig6">6</a>, <a id="crosref2110" href="#fig7">7</a>, <a id="crosref2115" href="#figs1">S1</a>, <a id="crosref2120" href="#figs2">S2</a>, <a id="crosref2125" href="#figs3">S3</a>, <a id="crosref2130" href="#figs4">S4</a>, and <a id="crosref2135" href="#figs5">S5</a> are available within <a id="crosref2140" href="#mmc1">Tables S1</a>, <a id="crosref2145" href="#mmc2">S2</a>, <a id="crosref2150" href="#mmc3">S3</a>, <a id="crosref2155" href="#mmc4">S4</a>, <a id="crosref2160" href="#mmc5">S5</a>, <a id="crosref2165" href="#mmc6">S6</a>, <a id="crosref2170" href="#mmc7">S7</a>, <a id="crosref2175" href="#mmc8">S8</a>, and <a id="crosref2180" href="#mmc9">S9</a> included in this publication as well as from the Lead Contact upon reasonable request. The accession number for the raw ATAC-seq data and processed peak bed files reported in this paper is GEO: <a id="interref510" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122877" target="_blank">GSE122877</a></div><section id="sec-4-6-1"><h4>Relevant URLs</h4><div id="p0680" role="paragraph">Jackson Laboratory, <a id="intref0335" href="https://www.jax.org/" target="_blank">https://www.jax.org/</a>; MIT CRISPR Tools, <a id="intref0340" href="http://zlab.bio/guide-design-resources" target="_blank">http://zlab.bio/guide-design-resources</a>; HaploRegV.4.1, <a id="interref0515" href="https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php/" target="_blank">https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php/</a>; UniPROBE Database, <a id="intref0350" href="http://the_brain.bwh.harvard.edu/uniprobe/about.php/" target="_blank">http://the_brain.bwh.harvard.edu/uniprobe/about.php/</a>; Visigene,:<a id="intref0355" href="https://genome.ucsc.edu/cgi-bin/hgVisigene" target="_blank">http://genome.ucsc.edu/cgi-bin/hgVisigene</a>; Eurexpress, <a id="intref0360" href="http://www.eurexpress.org/ee/" target="_blank">http://www.eurexpress.org/ee/</a>; GenePaint, <a id="intref0365" href="http://www.genepaint.org/Frameset.html" target="_blank">http://www.genepaint.org/Frameset.html</a>; Mouse Genome Informatics, <a id="intref0370" href="http://www.informatics.jax.org/" target="_blank">http://www.informatics.jax.org/</a>; 1000 Genomes Project raw&nbsp;data, ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/ALL.chr20.phase1_release_v3.20101123.snps_indels_svs.genotypes.vcf.gz; GADP data, <a id="intref0375" href="https://eichlerlab.gs.washington.edu/greatape/data/" target="_blank">https://eichlerlab.gs.washington.edu/greatape/data/</a>; EGDP data, <a id="intref0380" href="https://evolbio.ut.ee/CGgenomes_VCF/" target="_blank">https://evolbio.ut.ee/CGgenomes_VCF/</a>; SAMtools, <a id="intref0385" href="http://samtools.sourceforge.net/" target="_blank">http://samtools.sourceforge.net/</a>; ProxyFinder (<a id="intref0390" href="https://github.com/immunogenomics/harmjan/tree/master/ProxyFinder" target="_blank">https://github.com/immunogenomics/harmjan/tree/master/ProxyFinder</a>); OAI: <a id="intref0395" href="http://oai.epi-ucsf.org./datarelease/" target="_blank">http://oai.epi-ucsf.org./datarelease/</a>; FNIH: <a id="intref0400" href="https://fnih.org/what-we-do/biomarkers-consortium/programs/osteoarthritis-project" target="_blank">https://fnih.org/what-we-do/biomarkers-consortium/programs/osteoarthritis-project</a>; LDSC, <a id="intref0405" href="https://github.com/bulik/ldsc" target="_blank">https://github.com/bulik/ldsc</a>, LDHub Files, <a id="intref0410" href="http://ldsc.broadinstitute.org/gwashare" target="_blank">http://ldsc.broadinstitute.org/gwashare</a>; LDHub results, <a id="intref0415" href="http://ldsc.broadinstitute.org/lookup" target="_blank">http://ldsc.broadinstitute.org/lookup</a>; Alkes Group LDSCORE files, <a id="intref0420" href="https://data.broadinstitute.org/alkesgroup/LDSCORE" target="_blank">https://data.broadinstitute.org/alkesgroup/LDSCORE</a></div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0685" role="paragraph">The authors thank Drs. Bruce, Brooks, and Bouxsein for μCT from the Center for Skeletal Research (NIH P30 AR066261) (Massachusetts General Hospital); EpigenDx for ASE; Applied StemCell for mice; Drs. Sabeti (Harvard University [HU]), Zeng (Tufts University), and Goldring (The Hospital for Special Surgery) for cell lines; the HUGMF for targeting; the HUBCF for sequencing; Tufts Medical Center and Boston Children’s Hospital for histology; Dr. Worthington (HU) for statistics guidance; Drs. Price (Harvard T.H. Chan School of Public Health), Tabin (Harvard Medical School), Doxey (University of Waterloo), Lowe (Duke University), Bowes (iMorphics), Yau (Marcus Institute for Aging Research), Felson (Boston University), Pregizer (Boston Children’s Hospital), and Pilbeam (HU) and the Capellini lab members for insight and manuscript review; three reviewers for their insight into this work. This work was supported by NIH/NIAMS (1R01AR070139&nbsp;to T.D.C. and R01AR065462 to A.M.K.); the Orthopaedic Foundation at Boston Children’s Hospital (to A.M.K.); NIH/NCATS BU-CTSI (1UL1TR001430 to V.B.K. and G.H.C.); an American Heart Association Scientist Development Grant (17SDG33670323 to V.B.K. and G.H.C.); the Hariri Institute for Computing and Computational Science and Engineering at Boston University (Hariri Research Award to V.B.K. and G.H.C.); NIH (5U01AG-018820 to V.B.K. and G.H.C.); HU PRISE (to S.Y.); NSF (BCS1518596 to T.D.C. and N.R.) for mouse research; and HU Dean’s Competitive Fund and Milton Fund for Human Research (to T.D.C.).</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0690" role="paragraph">T.D.C. supervised project. T.D.C., D.R., and N.R. conceived and designed the project. D.R. performed all computational analyses. E.J. performed selection analyses. J.W., M.Y., and P.M. performed ATAC-seq. T.D.C. and J.C. designed mice. Z.L. performed mouse work, μCT, histology, and ASE. A.M.K. and S.H. performed morphometrics. J.T.S. and J.K. performed histology/OA analyses. A.M. and R.K. performed microscopy. Z.L. and S.Y. performed <i>in&nbsp;vitro</i> experiments. P.M. performed ChIP. P.M., D.E.M., and V.R. performed μCT. V.B.K. and G.H.C. performed morphometrics. T.D.C. and D.R. wrote/edited the manuscript with input from all authors.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0695" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (9)</h2><div id="p0705" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/1e00ca9b-b5d9-45e5-a228-b2fb792932b9/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (18.86 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. ATAC-Seq Datasets from Mouse and Human Long Bone Samples with Downstream Analyses, Related to Figure&nbsp;1</div><div id="espara0010" role="paragraph">(Sheets 1-8, “IDR-Filt. Prox. Femur” – “IDR-Filt. Dist. Radius”): narrowPeak-formatted peak (mm10) information for called ATAC-seq peaks from indicated tissue following IDR replicate filtering. (Sheet 9, “IDR Filtering Results”): Summarized numbers of called peaks on individual biological replicates and final peak sets (mm10) following replicate consolidation. (Sheet 10, “IDR Consistency Check”): Results of analyses for within- and between-sample concordance. Columns A-C, E-G similar to those in Sheet 9. Self-consistency and rescue-ratios for the distal femur and proximal tibia replicate pairs used are indicated. (Sheet 11, “Human E59 ATAC-seq”): Summarized numbers of called peaks for ATAC-seq performed on human E59 embryonic tissue. (Sheets 12-25, “General Knee Dist. Femur (hg19)” – “Embryonic Brain (hg19)”): Lifted-over peak regions (hg19) in BED format from indicated tissue. “General” sets refer to original IDR-filtered, brain-filtered peak sets intersected as shown in Sheets 26-27, “Peak Set Intersections.” “Specific” sets refer to knee/elbow peak set intersections, filtered for opposing bone-ends (e.g., distal tibia, proximal femur). (Sheets 26-27, Peak Set Intersections): Intersections between peak sets in mm10/hg19 (indicated) in generating the indicated general/specific joint sets. (Sheet 28, “Fore-Hindlimb Intersections”): Intersections of homologous forelimb/hindlimb peak sets. (Sheet 29, “Human E59 – Mouse Intersections”): Human E59 peak sets intersecting equivalent lifted-over mouse sets. Percent overlaps for mouse (‘A’) and human (‘B’) sets (respectively) are indicated. (Sheet 30, “Human-Mouse Orthology”): Mouse knee-specific sets were lifted-over to hg19, then lifted back to mm10 in order to confirm one-to-one orthology. The same analysis was performed for human E59 region sets. (Sheet 31, “Genomic Annotations”): Elements in each lifted-over set assigned genomic annotations based on distance to nearest TSS. (Sheets 32 – 47, “Distal Femur Brain-filtered” – “Elbow-Common-Specific”): GREAT output using ATAC-seq region sets. GREAT analysis was performed on sets of called ATAC-seq peaks, lifted over to hg19. Sheet titles indicate the set used - ‘brain-filtered’ sets are those in which ATAC-seq peaks were filtered for brain tissue, but not separated from the opposing end of the long bone, nor separated at the knee/elbow. Table shows top hits per each category for which FDR (both binomial and hypergeometric tests) ≤ 0.05. See http://great.stanford.edupublic/html for details on column headings. Sheets reporting no GREAT data indicate that there were no significantly enriched terms using that dataset). (Sheets 48 – 50, “DF - E59 Overlap” – “KC - E59 Overlap”): GREAT results for lifted-over knee-specific sets intersecting human E59 data. (Sheets 51-52, “Mouse_ATAC” – “Human_E59_ATAC”): ATAC-seq quality/control data and primers. Column headings are self-explanatory. “Mouse_ATAC”: ATAC-seq performed on E15.5 mouse embryonic tissue. “Human_E59_ATAC”: ATAC-seq performed on E59 human embryonic tissue.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/8c3d72aa-94fe-4f18-b488-92e8d555fadf/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (3.17 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. ATAC-Seq Dataset Overlaps with Evolutionary and Functional Genomics Datasets, Related to Figure&nbsp;1</div><div id="espara0015" role="paragraph">(Sheet 1, “Col2a-eCFP”): Intersections between gross-dissected proximal and distal femur ATAC-seq region sets and Col2a-eCFP-sorted equivalent region sets were performed using regioneR to test for significance of overlap compared to a randomized background. Columns: “Enrichment p-value”: p value for overlap enrichment test compared to randomized background. “Observed Intersections”: Observed number of overlaps between indicated peak set and chromatin mark regions. “Enrichment Z-score”: Z-score calculated as comparison of observed overlaps compared to randomized distribution. (Sheets 2-4, “Overlaps - E47 Limb Bud - H3K27” – “Overlaps - Human E59 - ATAC”): Intersections between ATAC-seq region sets and chromatin mark data were performed using regioneR to test for significance of overlap compared to a randomized background. Columns as above. Sheet “Overlaps - E47 Limb Bud - H3K27ac” provides overlap enrichment tests for specific and general knee sets, along with knee-specific sets separated by human sequence diversity, for H3K27ac ChIP-seq called regions obtained from human embryonic (E47) forelimb/hind limb bud. Sheet “Overlaps - BMDC - H3K27ac” provides overlap enrichment tests for specific and general knee sets, along with knee-specific sets separated by human sequence diversity, for H3K27ac ChIP-seq called regions obtained from bone marrow derived chondrocytes. Sheet “Overlaps - Human E59 – ATAC-seq” provides overlap enrichment tests for indicated knee-specific sets, for Human E59 ATAC-seq region sets from matched tissues (as indicated by ‘Mouse Set’ and ‘Human Match’ columns). (Sheets 5-7, “ChIP-seq Overlaps - Dist. Femur” – “ChIP-seq Overlaps - KC”): Overlaps between transcription factor ChIP-seq and ATAC-seq peak sets with regioneR, columns as above with additional: “Factor”: TF assayed (mouse experiments indicated), “Source Reference”: ENCODE or GEO Datasets experiment accession code for a given ChIP-seq experiment. (Sheets 8, “Conservation - PhyloP20ways”): phyloP20ways per-bp values for ATAC-seq region sets were compared across sets either through averaging per-bp values across 500bp for all regions (“Per-BP”) or averaging values per-region (“Region-Averaged”). One-sided p values for Wilcoxon rank-sum tests are shown, along with adjusted values. Yellow bars indicate region sets being compared. (Sheets 9-14, “Dist. Femur Bottom 25%” – “Knee-Common Top 25%”): Human-chimp sequence identity for ATAC-seq peaks was calculated for all sequences and used to generate top/bottom 25% slices. GREAT results for hg19 peaks in each of these slices are shown; tables show top hits per each category for which FDR (both binomial and hypergeometric tests) ≤ 0.05. See http://great.stanford.edupublic/html for details on column headings. Sheets reporting no GREAT data indicate that there were no significantly enriched terms using that dataset. (Sheets 15 – 20, “Dist. Femur Bottom 10%” - “Knee-Common Top 10%”): Equivalent GREAT results to above for top/bottom 10% slices. (Sheets 21 – 23, “Dist. Femur - Peak Data” – “Knee-Common - Peak Data”): Region information for human-chimp phyloP20ways analysis, for indicated set. “chromosome,” “start” and “end” columns indicate the location of a given ATAC-seq peak in hg38 coordinates. “hg38.sequence” and “panTro4.sequence” columns contain the extracted sequences from multiz20ways MAF, with <sup>∗</sup>/- indicating gaps. “Raw.Match.Score,” “Total.length” and “Sequence.Identity” refer to the number of matched nucleotides, length of sequence (ignoring shared gaps) and calculated sequence identity, respectively. “phyloP20ways.score” refers to phyloP20ways score averaged over the length of the indicated region. “random.region.phyloP20ways.score” refers to the averaged phyloP20ways score for a randomly-generated, size-matched region, and is used to generate the random distribution as seen in Figure&nbsp;1D.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/3b700148-713e-4597-a0c8-13cc4f5e1b39/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (2.24 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Transcription Factor Binding Analyses Using ATAC-Seq Sets, Related to Figure&nbsp;2</div><div id="espara0020" role="paragraph">(Sheets 1-6, “Distal Femur (mm10)” – “Knee Common (hg19)”): HOMER <i>de-novo</i> motif analysis results for the indicated peak set, using original mm10 and lifted-over hg19 sequence sets. Columns: “Motif Logo”: Logo of generated <i>de novo</i> motif. “Rank”: Motifs ranked by calculated p value. “P-value” and “log p-value”: p values for motif enrichment calculated by HOMER. “% of Targets/Background”: Percent of target and background sequences predicted to contain a given motif. “STD(Bg STD)”: Standard deviation of motif occurrence (bp) away from the center of the 500&nbsp;bp sequence for target and background sets, respectively. “Similar Motif”: Similar motif as identified through search of HOMER vertebrate motif library; similarity score shown in parentheses. (Sheets 7 – 9, “Motif Alteration - Distal Femur” – “Motif Alteration – Knee-Common”): Lifted-over knee region sets were tested for enrichment/depletion of human-chimp divergences predicted to alter potential binding motifs for a set of relevant transcription factors. Columns: “Transcription Factor Motif”: transcription factor PWM for which predicted binding sites are being tested. “Distribution p-value”: p value for enrichment/depletion test against randomized background. “Adjusted p-value”: p value corrected for multiple tests of different TFs. “Direction of Bias”: Whether a given region set had more (enrichment) or less (depletion) divergences predicted to alter a given TF motif than expected for randomized region sets. ChIP-seq signal enrichments for available TFs are shown in Table S2, Sheets 5-7. (Sheets 10-17, “Knee-Specific Pooled Cluster 1” – “SK-Common Cluster 2”): Clusters of common human variants were tested for biases to modify potential binding motifs for a set of relevant transcription factors. Columns as above. “SK” sheets refer to specific-knee sets. (Sheets 18-20, “Distal Femur Intra-species” – “Knee-Common Intra-species”): Common human variants falling within ATAC-seq regions (of a fixed 500bp size) from the three knee-specific sets (unclustered) were tested for biases to modify potential binding motifs. Columns as above.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/eb1858da-39be-4c1c-b356-72f5c0ea8847/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (188.45 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Putative Knee Regulatory Regions and Acceleration Signals, Related to Figure&nbsp;1</div><div id="espara0025" role="paragraph">(Sheet 1, “HAR Region Set”): Genomic coordinates (hg19) of regions showing evidence of acceleration in humans aggregated from multiple studies; citations for particular regions are indicated in the “source” column. (Sheets 2-10, “Embryonic Brain” – “Knee-Common-Specific”): Intersections between regions detailed in Sheet 1 and the indicated ATAC-seq region set. Columns: “chr,” “start,” “end”: hg19 coordinates for a given ATAC-seq region intersecting a region of acceleration. “HAR.chr,” “HAR.start,” “HAR.end”: hg19 coordinates for region of acceleration intersected. “Nearest.Refseq.Gene,” “Entrez.ID,” “Gene.Symbol,”: Nearest annotated TSS to the indicated ATAC-seq region, as RefSeq accession, Entrez ID or human gene symbol, respectively. “Nearest.TSS”: Distance (bp) from intersected region to nearest TSS, negative values denoting elements upstream. “Gene.Description,” “Gene.Type”: Short description and type of nearest annotated gene. (Sheets 11-13, “Dist. Femur - E59 Overlap” – “Knee Common - E59 Overlap”): Intersections between mouse embryonic ATAC-seq regions with equivalent human E59 embryonic data, overlapping acceleration signals. Columns as in sheets 2-10. (Sheet 14, “Intersection Enrichment Testing”): Accelerated region intersections for each set were compared to a 10K random background to test for enrichment of overlap. “Set”: ATAC-seq region set tested. “Set size (bp)”: size of indicated region set in base-pairs. “Number of Accelerated Region Intersections”: Number of accelerated region intersections in this set. “Number of Intersections per bp”: Accelerated region intersections expressed as a percentage of total set size. “Beta Distribution P-value”: Significance value for enrichment test against background using a fitted beta distribution. “Adjusted p-value”: Correction for multiple hypothesis testing using BH correction. (Sheet 15, “Great Annotations”): GREAT output using an aggregate of all lifted-over mouse hindlimb ATAC-seq regions intersecting regions of acceleration. Table shows top 50 hits per each category for which FDR (both binomial and hypergeometric tests) ≤ 0.05. See http://great.stanford.edupublic/html for details on column headings. Sheets reporting no GREAT data indicate that there were no significantly enriched terms using that dataset.</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/3e2e8816-96ce-49e2-ab8e-827b960f5bee/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (816.62 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Analyses of Inter- and Intra-species Diversity in ATAC-Seq Sets, Related to Figure&nbsp;3</div><div id="espara0030" role="paragraph">(Sheets 1 – 24, “Inter-species - DF General” – “Intra-Spec. (Gor) Elbow-Spec.”): Intra-species diversity across and within taxa. Columns: “Model”: Statistical method for which results are indicated. “Comparison”: Species/Sets being compared. “p-value”: Result of Tukey post-hoc testing for the indicated pair. “Direction (wrt first in pair)”: direction of variance between the pair, with respect to the first in the ‘Comparison’ pair. “t.ratio”: Test statistic for Tukey post-hoc test. Note for certain sheets resampling tests are indicated by “BOTTOM 25%” and “TOP 25%” headings, which refer to statistical results based on taking the 25th and 75th percentile (respectively), rather than the mean, for human subsampling. The “n&nbsp;= 100 subsampling” heading refers to statistical results based on increasing the human sub-sample size for testing. Sheets 1-3, 7-9, 13-15, 19-21: Comparisons of intra-species diversity in humans, chimpanzees and gorillas for sequences in the indicated joint region sets, either “general” or “specific.” Sheets 4-6, 10-12, 16-18, 22-24: Comparisons of intra-species diversity in sequences between forelimb/hindlimb joint region sets, either “general” or “specific,” for human, chimpanzee and gorilla. (Sheet 25, “Inter-Spec. E59 Knee Overlap”): Comparisons of intra-species diversity in humans, chimpanzees and gorillas for knee-specific and elbow-specific region sets overlapping with matched human embryonic tissue (E59). (Sheet 26, “Intra-Spec. E59 Knee Overlap”): Comparisons of intra-species diversity in humans, chimpanzees and gorillas (comparing sets within species) for knee-specific and elbow-specific region sets overlapping with matched human embryonic tissue (E59). (Sheet 27, “Human Seq. Constraint”): Elements/regions in each set were intersected with common human variants, with intersections compared to a set of 1000 randomized region sets to test for depletion/enrichment. Columns: “Set”: Genomic feature set tested. Order is equivalent to left-right in Figure&nbsp;3A. “Set size (bp)”: Size of indicated region set in base-pairs. “Number of Common Variants Intersected”: Number of common human variants intersecting this set. “Number of Intersections per bp”: Common variant intersections expressed as a percentage of total set size. “Background Average”: Average SNP/bp value from background region sets. “CDF P-value”: Significance value for testing significant deviation from background using normal CDF. “Corrected CDF p-value”: Correction for multiple hypothesis testing using BH correction. “Direction”: Whether set value falls to the left/right of background distribution. (Sheet 28, “Chimpanzee Seq. Constraint”): Elements/regions in each set were intersected with common chimpanzee variants, with intersections compared to a set of 1000 randomized region sets to test for depletion/enrichment. Columns as indicated in Sheet 27, with the addition of ‘Gamma P value’ and ‘Corrected Gamma P value’, which refer to distribution p values obtained when using a fitted gamma distribution in addition to those obtained using a normal CDF (Methods). (Sheet 29, “Morphological Constraint”): MRI measurement data for femur and tibia from the OAI dataset were tested for significant differences in variation across individuals; average and standard deviations of measurements for tibia/femur are indicated in columns B-E, with test statistics and resulting p values for performed Fligner-Killeen and Levene’s tests shown. (Sheet 30, “OAI Diversity”): Genotype information from the OAI dataset was intersected with knee-specific elements to compare sequence diversity across sets. Column titles as for Sheets 1-24. “BOTTOM 25%” and “TOP 25%” headings refer to statistical results based on taking the 25th and 75th percentile (respectively), rather than the mean, for cohort subsampling. (Sheets 31 – 42, “Dist. Femur Most Cons 25%” – “Elbow-Common Least Cons 25%”): GREAT output using lifted-over ATAC-seq peaks from specific joint region sets, further separated based on the intersection of common human variants (averaged over sub-samples) using most/least constrained (i.e., least/most variable) 25% slices. (Sheets 43-54, “Chimp Dist. Femur Most Cons 25%” – “Chimp Elbow-Comm. Least Cons 25”): Equivalent GREAT output for lifted-over ATAC-seq peak sets separated on basis of chimpanzee variant intersections. Sheets reporting no GREAT data indicate that there were no significantly enriched terms using that dataset).</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/13e79cb6-fe29-43bd-9f37-506a1dde98a0/mmc6.xlsx" download="mmc6.xlsx" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Spreadsheet (31.22 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Table S6. Analyses of Intra-species Diversity in ATAC-Seq Sets, Related to Figure&nbsp;3</div><div id="espara0035" role="paragraph">(Sheets 1 – 36, “Human General Knee Distal Femur” – “Gorilla Elbow-Common-Specific”): Genetic variants from human, chimpanzee and gorilla intersecting each element set. Columns: “chr,” “start,” “end”: ATAC-seq region coordinates (padded to fixed length of 500&nbsp;bp), sorted by sequence diversity. “Mean Values”: Mean number of common variants intersecting the indicated region across sub-samples. “Standard Deviation”: Standard deviation of common variant intersections across sub-samples. “25th Percentile”: First quartile value for subsampled common variant intersections. “75th Percentile”: Third quartile value for subsampled common variants intersections. “Subsample X”: Number of common variants intersecting the indicated region within a particular subsample (X being a particular subsample number). Note that human subsamples refer to 200 subsamples of n&nbsp;= 25 from 1KG3 individuals. Gorilla subsamples consist of 5 subsamples of n&nbsp;= 25 for 31 individuals. For chimpanzee diversity the entire set was used as a single sample for 25 individuals. Chimpanzee/Gorilla region coordinates are in hg18, human sheets are in hg19.</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/39633529-002e-45d6-b043-537f9800adf0/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (399.12 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table S7. Heritable Osteoarthritis Risk in ATAC-Seq Peaks, Related to Figure&nbsp;4</div><div id="espara0040" role="paragraph">(Sheet 1, “OA Lead Set”): Information on the lead OA risk-variants aggregated for analysis. Columns: “rsid”: DBSNP160 accession. “chromosome,” “Chromosome Start (hg38),” “Chromosome End (hg38)”: Coordinates for the indicated SNP in hg38. (Sheet 2, “OA Lead-Proxy Intersections”): Intersections of independent OA risk loci and ATAC-seq region sets. Columns: “lead_RSID”: lead variant identified in GWAS studies. “chrom,” “bp,” “proxy_chr,” “proxy_bp”: position (hg19) of lead variant and its proxies, respectively. “proxy_rsid”: DBSNP accession for proxy variant. Columns G-P: Intersection between indicated ATAC-seq region set (e.g., “Distal Femur-Specific”) and a given proxy variant, ‘1’ indicating overlap. (Sheet 3, “OA - Intersections – Selection”): Those proxy variants intersecting ATAC-seq regions, subsequently intersected with EGDP selection windows. Columns: “Chr,” “Start,” “End”: coordinates for a given selection window (hg19). “pclr”: Results of test for ancient selection via 3P-CLR. Columns E-P: Results of test for recent selection in each of the indicated populations. ’1’ indicates a significant test result. “Genes”: Genes within the indicated selection window. Columns R-AB are equivalent to those described for Sheet 2. (Sheet 4, “rs143383 Pop. Allele Freq.”): Frequency of the OA risk-variant rs143383 in EGDP populations. Columns: “ref,” “alt”: allele frequency of reference and alternate allele, respectively. “reference, alternate”: Reference and alternate allele, respectively. “# of individuals”: Number of individuals in a given EGDP population. “Final_Group”: EGDP population for which allele frequencies are reported. “RSID”: accession of given variant in chr:pos format. Columns H-J: Results (p values) of recent selection tests for the indicated population within the selection window overlapping the given OA risk locus. “proxy”: Proxy variant intersected by ATAC-seq data. “lead”: Lead OA-risk variant in LD with proxy. “selected”: Whether or not the current selection window gives a significant (p&nbsp;&lt; 0.05) selection test for an EGDP population, as demarcated by “<sup>∗</sup>.” (Sheets 5-7, “rs2626053 Pop. Allele Freq.” – “rs7785659 Pop. Allele Freq.”): Population allele frequencies for additional ATAC-seq intersected OA risk loci, with columns as described for “rs143383 Pop. Allele Freq.” (Sheet 8, “OA Variant Enrichments”): Results of enrichment testing for OA risk loci. Columns: “Set”: the ATAC-seq peak set tested. “OA Variants Intersected”: Number of unique OA risk loci intersected by peaks. “Average/SD Intersections in Background”: Average/standard-deviation (of) number of unique OA risk loci intersected in randomized backgrounds (Methods). “Z-score”: Resulting Z-score of target intersections. “Enrichment p-value”: one-sided p value for enrichment (Methods). (Sheet 9, “OA Variant Motif Intersections”): OA variants captured by element sets were assessed for overlap with predicted chondrogenesis-associated TF binding sequences. Columns: “SNP_ID”: String-formatted information for given variant, corresponding to columns C-F. “RSID”: DBSNP identifier. “TF Motif”: Transcription factor PWM matching a given sequence. Columns F-J: Information on PWM sequence match. “Predicted Effect”: Whether the alternate allele results in a higher/lower PWM sequence match. “Element Set Capturing Variant”: The element set intersecting the given variant. Note that some of these sets are overlapping, so duplicate rows are possible. (Sheet 10, “Motif Biases - All OA Vars”): Predicted TF binding sites overlapping any OA variant were counted to look for biases in the incidence of variants within certain predicted sites. Columns: “Factor”: Specific TF PWMs compared to the superset distribution (Methods). “# of Variants”: Number of OA variants which intersect predicted sites for this TF. “Average Intersections for all TFs”: Average number of OA variants intersecting any predicted TF site. “Gamma Distribution p-value”: Significance value for testing significant deviation from superset using a fitted gamma distribution. “Adjusted p-value”: Correction for multiple hypothesis testing using BH correction. “Direction”: Whether set value falls to the left/right of superset distribution. (Sheet 11, “Motif Biases - Element OA Variants”): Predicted TF binding sites overlapping element-captured OA variants were counted for particular TF PWMs, similar to Sheet 10. Only tests for which un-adjusted p&nbsp;&lt; 0.05 are shown. Columns are as in Sheet 10, with “Set” indicating the element set considered. (Sheet 12, “Element – OA Motif Proportions”): The predicted TF binding sites intersected by element-captured OA variants were compared to all OA variants to look for biased occurrence in element sets. Columns: “Set”: OA variants captured by a particular element set. “Factor”: particular TF PWM being tested. “Number of Target TF Predicted Binding Sites”: Number of predicted TF binding sites (of those tested) intersected by variants from this set. “Number of Predicted TF Binding Sites”: Number of predicted sites for the target TF intersected by element variants. “Total Number of Target TF Predicted Binding Sites”: Number of predicted sites for the target TF intersected by any OA variant. “Total Number of Predicted TF Binding Sites”: Number of predicted TF binding sites (of those tested) intersected by any OA variant. “% of Target TF sites of Set”: Percent of target TF sites intersected by element-variants, relative to all TFs tested. “% of Target TF Sites of Total”: Percent of target TF sites intersected by any OA variant, relative to all TFs tested. “% of Target TF Sites Captured in Set”: Percent of OA variant-intersected target TF sites captured by element set. “Hypergeometric Test”: p value for hypergeometric test. “Adjusted P-value”: Adjusting for multiple hypotheses with BH correction. “Direction”: whether target set captures more/less target sites than expected. “Fold-Change”: Enrichment/depletion of sites in target set. (Sheet 13, “OA - Partitioned Heritability”): LDSC analysis was performed on OA GWAS summary statistics, utilizing a custom set of genomic features in addition to a modified baseline LD model. Columns: “Category”: Genomic annotation used to partition genome-wide heritability. Custom element set annotations are highlighted in yellow; other annotations are taken from the LDSC baseline model (<a id="intref0430" href="https://data.broadinstitute.org/alkesgroup/LDSCORE" target="_blank">https://data.broadinstitute.org/alkesgroup/LDSCORE</a>), which are described in their original publications (Finucane et&nbsp;al., 2015; Gazal et&nbsp;al., 2017). “Prop._SNPs”: Proportion of SNPs captured by features; note that feature sets are overlapping, so column sums to &gt; 1. “Prop._h2”: Proportion of genome-wide heritability, similarly sums to &gt; 1.“Prop._h2_std_error”: Standard error of partitioned heritability estimate. “Enrichment”: Proportion of heritability / proportion of SNPs. “Enrichment_p”: Significance of heritability proportion captured by feature. “P_ADJ”: Adjusted p values for multiple hypothesis correction. Columns I – K: Coefficients relating to LD score regression, see Finucane et&nbsp;al. (2015). (Sheet 14, “OAI Cohort Variation”): The number of alternate alleles falling within indicated element sets for subsampled individuals from the OAI cohort as well as additional control populations (1KG3, EGDP) was compared (see STAR Methods). t test statistic, group averages and p values shown were averaged over 1000 permutations (Student’s t test, alternative of greater variation in OAI), with standard-errors similarly reported (‘sd’); headers indicate results when using n&nbsp;= 20 and n&nbsp;= 200 subsets.</div></div></div><div id="mmc8" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/2bb8a659-573a-41d7-a84d-f3e7c7c9b4c7/mmc8.xlsx" download="mmc8.xlsx" aria-labelledby="mmc8-heading mmc8-link"><span id="mmc8-link">Spreadsheet (3.51 MB)</span></a></div><div class="core-description"><div id="mmc8-heading" class="heading">Table S8. Common Human Variant Clustering and Region Selection Tests, Related to Figure&nbsp;4</div><div id="espara0045" role="paragraph">(Sheet 1, “Cluster Properties”): Columns: “Set”: Source of the indicated set of clustered variants from knee-specific ATAC-seq regions. “# of Variants”: Number of common variants grouped in the indicated cluster. “Cluster Mean Fst”: Average Fst value, across all pairwise population comparisons, for all common variants within the cluster. “Cluster SD Fst”: Standard deviation of Fst values across all pairwise population comparisons for all common variants in the cluster. “Cluster Max Fst”: Maximum Fst value, across all pairwise population comparisons, for all common variants within the cluster. (Sheets 2-9, “Knee-Specific Pool Cluster 1” – “Knee-Common-Specific Cluster 2”): Position information for clusters described in “Cluster Properties” (hg19), with variant accessions indicated. (Sheets 10 – 17, “GREAT KS Pool Cluster 1” – “GREAT Knee Common-Sp. Cluster 2”): GREAT output using clustered variants, sheet titles indicate set/cluster used, KS indicating ‘Knee-Specific’, Pool indicating “Pooled,” Sp. Indicating “Specific.” (Sheet 18, “Selection Window Intersects”): Intersects of region sets and EGDP selection windows. Columns: “Chr_Mb,” “Chr,” “Start,” “End”: coordinates for a given selection window (hg19). “pclr”: Results of test for ancient selection via 3P-CLR. Columns F-Q: Results of test for recent selection in each of the indicated populations. ’1’ indicates a significant test result. “Genes”: Genes within the indicated selection window. Columns S-AE: Intersection between indicated ATAC-seq region set (e.g., “Distal Femur-Specific”) and a given selection window, ’0’ indicating no overlap. (Sheet 19, “Grossman Region Intersects”): Intersects of region sets and CMS selection regions. Columns: “chr hg19,” “start,” “end”: Genomic coordinates for a given selection region. Columns E-Q: Intersection between indicated ATAC-seq region set and a given selection region. (Sheet 20, “Ancient Sel. Window Testing”): Intersections between region sets and signals of ancient selection. Columns: “Region Set”: ATAC-seq region set used for testing. “Number.of.Windows.Intersected”: Number of 200kb windows intersected by a given set. “Number.of.Windows.Under.Selection”: Of the windows intersected, the number of windows for which at least one population had a significant selection signal. “Total.Number.of.Windows.Under.Selection”: Of all the windows tested across the genome, how many had at least one population with a significant selection signal. “Total.Number.of.Windows”: Number of 200kb windows across the genome used for testing. “Hypergeometric.Test”: p value for a hyper-geometric test of over/under-representation. “Direction (Over/Under-represented)”: Direction of indicated hyper-geometric test (i.e., more/less selection signal intersections than expected). “Adjusted.P.value”: p value for indicated hyper-geometric test after BH multiple-hypothesis correction. “Fold-Change”: Enrichment/depletion of selection window overlaps in target set. (Sheet 21, “Recent Sel. Window Testing”): Intersections between region sets and signals of recent selection. Columns are as described for Sheet 3. (Sheet 22, “Grossman Selection Enrichments”): Intersections between region sets and CMS selection regions. Columns: “Region Set”: Region sets typically used for overlap testing in this study, but also consisting of the following sets: ‘OA Variants’: Lead OA-risk variants and their proxies; ‘General Knee Pooled - OA Intersections’: Those regions from the general knee pooled set intersecting OA-risk variants and/or their proxies; ‘Limb Bud - OA Intersections’: Those regions from the human E47 limb bud H3K27ac set intersecting OA-risk variants and/or their proxies; ‘BMDC - OA Intersections’: Those regions from the human BMDC H3K27ac set intersecting OA-risk variants and/or their proxies. “Enrichment p-value”: p value for overlap enrichment/depletion test compared to randomized background. “Observed Intersections”: Observed number of overlaps between indicated peak set and ChIP-seq mark regions. “Enrichment/Depletion Z-score”: Z-score calculated as comparison of observed overlaps compared to randomized distribution.</div></div></div><div id="mmc9" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.02.057/attachment/7ed7304f-eeba-4790-b54d-05202cdc2abf/mmc9.xlsx" download="mmc9.xlsx" aria-labelledby="mmc9-heading mmc9-link"><span id="mmc9-link">Spreadsheet (43.94 KB)</span></a></div><div class="core-description"><div id="mmc9-heading" class="heading">Table S9. Experimental and Bioinformatics Results on the <i>GDF5 R4</i> Enhancer, Related to Figures 5, 6, and 7</div><div id="espara0050" role="paragraph">(Sheets 1 – 4, “<i>R4</i> null P30” – “<i>R4</i> rs6060369 P56 t-test”): P30, P56, and 1 year microCT statistics of <i>R4</i> Engineered Mice. (Sheet 1, “<i>R4</i> null P30”): Differences in key morphological features of the <i>R4</i> null knee joint at P30. All outcome measures were defined as continuous variables (n&nbsp;= 5 per genotype) and compared between the genotypes using multivariate linear regression analysis with anatomy used as dependent variable, and genotype and line as independent variables. Sidak post-hoc was used to correct for multiple comparisons between genotypes. P values are two-sided and the statistical significance was assessed at alpha&nbsp;= 0.05 for all the comparisons. Significant differences are highlighted in bold. WT: Wild Type; HET: Heterozygotes; HOM: Homozygotes. (Sheet 2, “<i>R4</i> null 1 year”): Differences in key morphological features of the <i>R4</i> null knee joint at 1 year. Statistical treatment similar to ‘<i>R4</i> null p30’ results, with similarly-defined continuous variables (6 wildtype, 15 heterozygous and 14 homozygous). Formatting as above. (Sheet 3, “<i>R4 rs6060369</i> P56”): Differences in key morphological features of the <i>R4 rs6060369</i> replacement knee joint at P56. All outcome measures were defined as continuous variables (6 wildtype, 8 heterozygous and 6 homozygous) and compared between the genotypes using oneway ANOVA analysis with anatomy used as dependent variable and genotype as independent variable. Sidak post-hoc was used to correct for multiple comparisons between genotypes. P values are two-sided and the statistical significance was assessed at alpha&nbsp;= 0.05 for all the comparisons. Formatting as above. (Sheet 4, “<i>R4 rs6060369</i> P56 t-test”): Differences in key morphological features of the <i>R4 rs6060369</i> knee joint at P56 body size normalized. All outcome measures were determined using paired t test on littermate WT, Homozygotes, and Heterozygotes (n&nbsp;= 6 mixed male and female per genotype). P values are two-sided. Significant differences, which are uncorrected, are highlighted in bold. (Sheet 5, “<i>R4</i> null P30 histology data”): Analysis of osteoarthritis on <i>R4</i> WT and HOM outcomes based on total joint OARSI Score at P30 indicating that significant joint osteoarthritis is absent at P30 in WT and HOM knees. A Mann-Whitney test was used to compare the OARSI Score between the wild type (n&nbsp;= 5) and homozygous (n = 5) knees. P values are two-sided and the statistical significance was assessed at alpha&nbsp;= 0.05. (Sheet 6, “<i>R4</i> null 1 Year histology data”): Analysis of osteoarthritis on <i>R4</i> WT and HOM outcomes based on total joint OARSI Score and Osteophyte Score at 1 year. Significantly more osteoarthritis as measured using OARSI Score was observed in HOM knees. No significant difference was observed in Osteophyte Score between genotypes. A Mann-Whitney test was used to compare the OARSI Scores as well as Osteophyte Scores between the wild type (n&nbsp;= 6) and homozygous (n&nbsp;= 14) knees. P values are two-sided and the statistical significance was assessed at alpha&nbsp;= 0.05. (Sheets 7-9, “UniProbe - Non-Risk TTTAGCCGAGC” – “UniProbe - Risk vs Non-Risk”): UniProbe Analysis of rs6060369 variant position. Two 15&nbsp;bp sequences centered on the non-risk “C” (Sheet 5 - black font) and risk “T” (Sheet 6 - red font) alleles at rs6060369 were investigated for evidence of <i>in&nbsp;vitro</i> experimental TF binding affinities. Comparisons of the output for each allele (Sheet 7) reveals a number of reduced or gained TFs, for the risk allele with the PITX1 TF being highlighted (yellow) as a binding gain (i.e., &gt; 0.10 enrichment change) and factor expressed in and required for knee development. (Sheet 10, “Construct Primers”): Sequences of primers and sgRNAs used in this study to clone and target the <i>GDF5 R4</i> regulatory element in mouse and human cells.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">1.</a></div><div>Andersen, H.</div><div><strong>Histochemical studies on the histogenesis of the knee joint and superior tibio-fibular joint in human foetuses</strong></div><div><em>Acta Anat. (Basel).</em> 1961; <strong>46</strong>:279-303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000141791" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/13861166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000141791&amp;pmid=13861166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">2.</a></div><div>arcOGEN Consortium, arcOGEN Collaborators ∙ Zeggini, E. ...</div><div><strong>Identification of new susceptibility loci for osteoarthritis (arcOGEN): a genome-wide association study</strong></div><div><em>Lancet.</em> 2012; <strong>380</strong>:815-823</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2812%2960681-3&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865595151" target="_blank" aria-label="Open Scopus in new tab">Scopus (230)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22763110/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2812%2960681-3&amp;pmid=22763110" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (4671)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-1" title="View in article">4.</a></div><div>Barr, A.J. ∙ Dube, B. ∙ Hensor, E.M.A. ...</div><div><strong>The relationship between three-dimensional knee MRI bone shape and total knee replacement-a case control study: data from the Osteoarthritis Initiative</strong></div><div><em>Rheumatology (Oxford).</em> 2016; <strong>55</strong>:1585-1593</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/rheumatology/kew191" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84995616558" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27185958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Frheumatology%2Fkew191&amp;pmid=27185958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5-1" title="View in article">5.</a></div><div>Barrett, J.C. ∙ Fry, B. ∙ Maller, J. ...</div><div><strong>Haploview: analysis and visualization of LD and haplotype maps</strong></div><div><em>Bioinformatics.</em> 2005; <strong>21</strong>:263-265</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bth457" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-13444269543" target="_blank" aria-label="Open Scopus in new tab">Scopus (10961)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15297300/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbth457&amp;pmid=15297300" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref164" title="View in article">6.</a></div><div>Basit, S. ∙ Naqvi, S.K.-H. ∙ Wasif, N. ...</div><div><strong>A novel insertion mutation in the cartilage-derived morphogenetic protein-1 (CDMP1) gene underlies Grebe-type chondrodysplasia in a consanguineous Pakistani family</strong></div><div><em>BMC Med. Genet.</em> 2008; <strong>9</strong>:102</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2350-9-102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-58149327059" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19038017/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2350-9-102&amp;pmid=19038017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">7.</a></div><div>Baugé, C. ∙ Girard, N. ∙ Lhuissier, E. ...</div><div><strong>Regulation and Role of TGFβ Signaling Pathway in Aging and Osteoarthritis Joints</strong></div><div><em>Aging Dis.</em> 2013; <strong>5</strong>:394-405</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25489490/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=25489490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Benjamini, Y. ∙ Hochberg, Y.</div><div><strong>Controlling the false discovery rate: a practical and powerful approach to multiple testing</strong></div><div><em>J.&nbsp;R. Stat. Soc. B.</em> 1995; <strong>57</strong>:289-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Berenbaum, F. ∙ Wallace, I.J. ∙ Lieberman, D.E. ...</div><div><strong>Modern-day environmental factors in the pathogenesis of osteoarthritis</strong></div><div><em>Nat. Rev. Rheumatol.</em> 2018; <strong>14</strong>:674-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41584-018-0073-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053529092" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30209413/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41584-018-0073-x&amp;pmid=30209413" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">10.</a></div><div>Bird, C.P. ∙ Stranger, B.E. ∙ Liu, M. ...</div><div><strong>Fast-evolving noncoding sequences in the human genome</strong></div><div><em>Genome Biol.</em> 2007; <strong>8</strong>:R118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2007-8-6-r118" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049157173" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17578567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2007-8-6-r118&amp;pmid=17578567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#fig2" title="View in article">11.</a></div><div>Blake, J.A. ∙ Eppig, J.T. ∙ Kadin, J.A. ..., the Mouse Genome Database Group</div><div><strong>Mouse Genome Database (MGD)-2017: community knowledge resource for the laboratory mouse</strong></div><div><em>Nucleic Acids Res.</em> 2017; <strong>45</strong>:D723-D729</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw1040" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85016041593" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27899570/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw1040&amp;pmid=27899570" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Boyd, J.L. ∙ Skove, S.L. ∙ Rouanet, J.P. ...</div><div><strong>Human-chimpanzee differences in a FZD8 enhancer alter cell-cycle dynamics in the developing neocortex</strong></div><div><em>Curr. Biol.</em> 2015; <strong>25</strong>:772-779</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cub.2015.01.041&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84926420296" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25702574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2015.01.041&amp;pmid=25702574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114-1" title="View in article">13.</a></div><div>Bramble, D.M. ∙ Lieberman, D.E.</div><div><strong>Endurance running and the evolution of Homo</strong></div><div><em>Nature.</em> 2004; <strong>432</strong>:345-352</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03052" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-9244222260" target="_blank" aria-label="Open Scopus in new tab">Scopus (892)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15549097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03052&amp;pmid=15549097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Browning, B.L. ∙ Zhou, Y. ∙ Browning, S.R.</div><div><strong>A One-Penny Imputed Genome from Next-Generation Reference Panels</strong></div><div><em>Am. J. Hum. Genet.</em> 2018; <strong>103</strong>:338-348</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2018.07.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2018.07.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85056238160" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2018.07.015&amp;pmid=30100085" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Buenrostro, J.D. ∙ Wu, B. ∙ Chang, H.Y. ...</div><div><strong>ATAC-seq: a method for assaying chromatin accessibility genome-wide</strong></div><div><em>Curr. Protoc. Mol. Biol.</em> 2015; <strong>109</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/0471142727.mb2129s109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84949212255" target="_blank" aria-label="Open Scopus in new tab">Scopus (441)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F0471142727.mb2129s109" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145-2" title="View in article">17.</a></div><div>Bush, E.C. ∙ Lahn, B.T.</div><div><strong>A genome-wide screen for noncoding elements important in primate evolution</strong></div><div><em>BMC Evol. Biol.</em> 2008; <strong>8</strong>:17</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2148-8-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-39049088628" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18215302/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2148-8-17&amp;pmid=18215302" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref140-1" title="View in article">18.</a></div><div>Butterfield, N.C. ∙ Curry, K.F. ∙ Steinberg, J. ...</div><div><strong>Accelerating functional gene discovery in osteoarthritis</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/836221" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F836221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#fig2" title="View in article">19.</a></div><div>Capellini, T.D. ∙ Di Giacomo, G. ∙ Salsi, V. ...</div><div><strong>Pbx1/Pbx2 requirement for distal limb patterning is mediated by the hierarchical control of Hox gene spatial distribution and Shh expression</strong></div><div><em>Development.</em> 2006; <strong>133</strong>:2263-2273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.02395" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33745584051" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16672333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.02395&amp;pmid=16672333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">20.</a></div><div>Capellini, T.D. ∙ Chen, H. ∙ Cao, J. ...</div><div><strong>Ancient selection for derived alleles at a GDF5 enhancer influencing human growth and osteoarthritis risk</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1202-1210</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3911" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85026427115" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28671685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3911&amp;pmid=28671685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref177-2" title="View in article">21.</a></div><div>Capra, J.A. ∙ Erwin, G.D. ∙ McKinsey, G. ...</div><div><strong>Many human accelerated regions are developmental enhancers</strong></div><div><em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em> 2013; <strong>368</strong>:20130025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84887496761" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24218637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0025&amp;pmid=24218637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Carnes, B.A. ∙ Olshansky, S.J.</div><div><strong>Evolutionary Perspectives on Human Senescence</strong></div><div><em>Popul. Dev. Rev.</em> 1993; <strong>19</strong>:793</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2307/2938414" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2307%2F2938414" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">23.</a></div><div>Casalone, E. ∙ Tachmazidou, I. ∙ Zengini, E. ..., arcOGEN Consortium</div><div><strong>A novel variant in GLIS3 is associated with osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2018; <strong>77</strong>:620-623</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2017-211848" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044834428" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29436472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2017-211848&amp;pmid=29436472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">24.</a></div><div>Castaño-Betancourt, M.C. ∙ Cailotto, F. ∙ Kerkhof, H.J. ...</div><div><strong>Genome-wide association and functional studies identify the DOT1L gene to be involved in cartilage thickness and hip osteoarthritis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:8218-8223</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1119899109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84861422537" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22566624/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1119899109&amp;pmid=22566624" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">25.</a></div><div>Castaño-Betancourt, M.C. ∙ Evans, D.S. ∙ Ramos, Y.F. ...</div><div><strong>Novel genetic variants for cartilage thickness and hip osteoarthritis</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1006260</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1006260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84994339045" target="_blank" aria-label="Open Scopus in new tab">Scopus (39)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27701424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1006260&amp;pmid=27701424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref160" title="View in article">26.</a></div><div>Chan, Y.F. ∙ Marks, M.E. ∙ Jones, F.C. ...</div><div><strong>Adaptive evolution of pelvic reduction in sticklebacks by recurrent deletion of a Pitx1 enhancer</strong></div><div><em>Science.</em> 2010; <strong>327</strong>:302-305</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1182213" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-74549118236" target="_blank" aria-label="Open Scopus in new tab">Scopus (588)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20007865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1182213&amp;pmid=20007865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">27.</a></div><div>Chang, C.C. ∙ Chow, C.C. ∙ Tellier, L.C. ...</div><div><strong>Second-generation PLINK: rising to the challenge of larger and richer datasets</strong></div><div><em>Gigascience.</em> 2015; <strong>4</strong>:7</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13742-015-0047-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84930213392" target="_blank" aria-label="Open Scopus in new tab">Scopus (1923)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13742-015-0047-8&amp;pmid=25722852" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168" title="View in article">28.</a></div><div>Charles, M.D. ∙ Haloman, S. ∙ Chen, L. ...</div><div><strong>Magnetic resonance imaging-based topographical differences between control and recurrent patellofemoral instability patients</strong></div><div><em>Am. J. Sports Med.</em> 2013; <strong>41</strong>:374-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0363546512472441" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84873370967" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23371940/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0363546512472441&amp;pmid=23371940" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148-1" title="View in article">29.</a></div><div>Chen, H. ∙ Capellini, T.D. ∙ Schoor, M. ...</div><div><strong>Heads, shoulders, elbows, knees, and toes: modular Gdf5 enhancers control different joints in the vertebrate skeleton</strong></div><div><em>PLoS Genet.</em> 2016; <strong>12</strong>:e1006454</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1006454" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000624431" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27902701/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1006454&amp;pmid=27902701" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">30.</a></div><div>Chokalingam, K. ∙ Hunter, S. ∙ Gooch, C. ...</div><div><strong>Three-dimensional in&nbsp;vitro effects of compression and time in culture on aggregate modulus and on gene expression and protein content of collagen type II in murine chondrocytes</strong></div><div><em>Tissue Eng. Part A.</em> 2009; <strong>15</strong>:2807-2816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.tea.2008.0560" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-73349122898" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19231914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.tea.2008.0560&amp;pmid=19231914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">31.</a></div><div>Coetzee, S.G. ∙ Coetzee, G.A. ∙ Hazelett, D.J.</div><div><strong>motifbreakR: an R/Bioconductor package for predicting variant effects at transcription factor binding sites</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3847-3849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84950293634" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26272984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv470&amp;pmid=26272984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">32.</a></div><div>Cotney, J. ∙ Leng, J. ∙ Yin, J. ...</div><div><strong>The evolution of lineage-specific regulatory activities in the human embryonic limb</strong></div><div><em>Cell.</em> 2013; <strong>154</strong>:185-196</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.05.056&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84879911829" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23827682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.05.056&amp;pmid=23827682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">33.</a></div><div>Cupit, L.D. ∙ Schmidt, V.A. ∙ Miller, F. ...</div><div><strong>Distinct PAR/IQGAP expression patterns during murine development: implications for thrombin-associated cytoskeletal reorganization</strong></div><div><em>Mamm. Genome.</em> 2004; <strong>15</strong>:618-629</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00335-004-2370-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-3943058758" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15457341/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00335-004-2370-8&amp;pmid=15457341" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">34.</a></div><div>Danecek, P. ∙ Auton, A. ∙ Abecasis, G. ..., 1000 Genomes Project Analysis Group</div><div><strong>The variant call format and VCFtools</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2156-2158</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr330" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79960405019" target="_blank" aria-label="Open Scopus in new tab">Scopus (3479)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr330&amp;pmid=21653522" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">35.</a></div><div>Darwin, C.</div><div><strong>The Descent of Man and Selection in Relation to Sex</strong></div><div>Murray, 1888</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.DarwinThe+Descent+of+Man+and+Selection+in+Relation+to+Sex1888Murray" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">36.</a></div><div>Das Neves Borges, P. ∙ Forte, A.E. ∙ Vincent, T.L. ...</div><div><strong>Rapid, automated imaging of mouse articular cartilage by microCT for early detection of osteoarthritis and finite element modelling of joint mechanics</strong></div><div><em>Osteoarthritis Cartilage.</em> 2014; <strong>22</strong>:1419-1428</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2014.07.014&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2014.07.014&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84908179041" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25278053/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2014.07.014&amp;pmid=25278053" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">37.</a></div><div>Davis, C.A. ∙ Hitz, B.C. ∙ Sloan, C.A. ...</div><div><strong>The Encyclopedia of DNA elements (ENCODE): data portal update</strong></div><div><em>Nucleic Acids Res.</em> 2018; <strong>46</strong>:D794-D801</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkx1081" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040942752" target="_blank" aria-label="Open Scopus in new tab">Scopus (213)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29126249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkx1081&amp;pmid=29126249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">38.</a></div><div>Davydov, E.V. ∙ Goode, D.L. ∙ Sirota, M. ...</div><div><strong>Identifying a high fraction of the human genome to be under selective constraint using GERP++</strong></div><div><em>PLoS Comput. Biol.</em> 2010; <strong>6</strong>:e1001025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1001025" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78651237647" target="_blank" aria-label="Open Scopus in new tab">Scopus (720)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21152010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1001025&amp;pmid=21152010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">39.</a></div><div>Day-Williams, A.G. ∙ Southam, L. ∙ Panoutsopoulou, K. ..., arcOGEN Consortium</div><div><strong>A variant in MCF2L is associated with osteoarthritis</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>89</strong>:446-450</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2011.08.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.ajhg.2011.08.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-80052736294" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21871595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.08.001&amp;pmid=21871595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">40.</a></div><div>Decker, R.S. ∙ Koyama, E. ∙ Pacifici, M.</div><div><strong>Genesis and morphogenesis of limb synovial joints and articular cartilage</strong></div><div><em>Matrix Biol.</em> 2014; <strong>39</strong>:5-10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.matbio.2014.08.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84908300461" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25172830/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.matbio.2014.08.006&amp;pmid=25172830" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">41.</a></div><div>Delignette-Muller, M.L. ∙ Dutang, C.</div><div><strong>fitdistrplus: An R Package for Fitting Distributions</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2015; <strong>64</strong>:1-34</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v064.i04" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84925685098" target="_blank" aria-label="Open Scopus in new tab">Scopus (668)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v064.i04" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">42.</a></div><div>DeMare, L.E. ∙ Leng, J. ∙ Cotney, J. ...</div><div><strong>The genomic landscape of cohesin-associated chromatin interactions</strong></div><div><em>Genome Res.</em> 2013; <strong>23</strong>:1224-1234</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.156570.113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881119091" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23704192/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.156570.113&amp;pmid=23704192" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">43.</a></div><div>Dickinson, M.E. ∙ Flenniken, A.M. ∙ Ji, X. ..., International Mouse Phenotyping Consortium, Jackson Laboratory, Infrastructure Nationale PHENOMIN, Institut Clinique de la Souris (ICS), Charles River Laboratories, MRC Harwell, Toronto Centre for Phenogenomics, Wellcome Trust Sanger Institute, RIKEN BioResource Center</div><div><strong>High-throughput discovery of novel developmental phenotypes</strong></div><div><em>Nature.</em> 2016; <strong>537</strong>:508-514</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84989204804" target="_blank" aria-label="Open Scopus in new tab">Scopus (309)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27626380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19356&amp;pmid=27626380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">44.</a></div><div>Diez-Roux, G. ∙ Banfi, S. ∙ Sultan, M. ...</div><div><strong>A high-resolution anatomical atlas of the transcriptome in the mouse embryo</strong></div><div><em>PLoS Biol.</em> 2011; <strong>9</strong>:e1000582</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.1000582" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79851473886" target="_blank" aria-label="Open Scopus in new tab">Scopus (397)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21267068/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.1000582&amp;pmid=21267068" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">45.</a></div><div>DiLeone, R.J. ∙ Russell, L.B. ∙ Kingsley, D.M.</div><div><strong>An extensive 3′ regulatory region controls expression of Bmp5 in specific anatomical structures of the mouse embryo</strong></div><div><em>Genetics.</em> 1998; <strong>148</strong>:401-408</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9475750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=9475750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">46.</a></div><div>Eckstein, F. ∙ Ateshian, G. ∙ Burgkart, R. ...</div><div><strong>Proposal for a nomenclature for magnetic resonance imaging based measures of articular cartilage in osteoarthritis</strong></div><div><em>Osteoarthritis Cartilage.</em> 2006; <strong>14</strong>:974-983</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2006.03.005&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2006.03.005&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33748160505" target="_blank" aria-label="Open Scopus in new tab">Scopus (181)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16730462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2006.03.005&amp;pmid=16730462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">47.</a></div><div>ENCODE Project Consortium</div><div><strong>An integrated encyclopedia of DNA elements in the human genome</strong></div><div><em>Nature.</em> 2012; <strong>489</strong>:57-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11247" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84865790047" target="_blank" aria-label="Open Scopus in new tab">Scopus (8320)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22955616/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11247&amp;pmid=22955616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">48.</a></div><div>Evangelou, E. ∙ Valdes, A.M. ∙ Kerkhof, H.J. ..., arcOGEN Consortium, Translation Research in Europe Applied Technologies for Osteoarthritis (TreatOA)</div><div><strong>Meta-analysis of genome-wide association studies confirms a susceptibility locus for knee osteoarthritis on chromosome 7q22</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:349-355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.132787" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78751703383" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21068099/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.132787&amp;pmid=21068099" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">49.</a></div><div>Evangelou, E. ∙ Kerkhof, H.J. ∙ Styrkarsdottir, U. ..., arcOGEN Consortium</div><div><strong>A meta-analysis of genome-wide association studies identifies novel variants associated with osteoarthritis of the hip</strong></div><div><em>Ann. Rheum. Dis.</em> 2014; <strong>73</strong>:2130-2136</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2012-203114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84883445177" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23989986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2012-203114&amp;pmid=23989986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref155" title="View in article">50.</a></div><div>Felson, D.T. ∙ Anderson, J.J. ∙ Naimark, A. ...</div><div><strong>Obesity and knee osteoarthritis. The Framingham Study</strong></div><div><em>Ann. Intern. Med.</em> 1988; <strong>109</strong>:18-24</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7326/0003-4819-109-1-18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/3377350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7326%2F0003-4819-109-1-18&amp;pmid=3377350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53" title="View in article">51.</a></div><div>Ferrer-Admetlla, A. ∙ Liang, M. ∙ Korneliussen, T. ...</div><div><strong>On detecting incomplete soft or hard selective sweeps using haplotype structure</strong></div><div><em>Mol. Biol. Evol.</em> 2014; <strong>31</strong>:1275-1291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msu077" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84899542092" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24554778/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsu077&amp;pmid=24554778" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">52.</a></div><div>Finger, F. ∙ Schörle, C. ∙ Zien, A. ...</div><div><strong>Molecular phenotyping of human chondrocyte cell lines T/C-28a2, T/C-28a4, and C-28/I2</strong></div><div><em>Arthritis Rheum.</em> 2003; <strong>48</strong>:3395-3403</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.11341" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0346850618" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14673991/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.11341&amp;pmid=14673991" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">53.</a></div><div>Finucane, H.K. ∙ Bulik-Sullivan, B. ∙ Gusev, A. ..., ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, RACI Consortium</div><div><strong>Partitioning heritability by functional annotation using genome-wide association summary statistics</strong></div><div><em>Nat. Genet.</em> 2015; <strong>47</strong>:1228-1235</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85000443086" target="_blank" aria-label="Open Scopus in new tab">Scopus (535)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26414678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3404&amp;pmid=26414678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">54.</a></div><div>Folkesson, J. ∙ Dam, E.B. ∙ Olsen, O.F. ...</div><div><strong>Segmenting articular cartilage automatically using a voxel classification approach</strong></div><div><em>IEEE Trans. Med. Imaging.</em> 2007; <strong>26</strong>:106-115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1109/TMI.2006.886808" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33846214324" target="_blank" aria-label="Open Scopus in new tab">Scopus (145)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17243589/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1109%2FTMI.2006.886808&amp;pmid=17243589" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">55.</a></div><div>Fox, J. ∙ Weisberg, S.</div><div><strong>An R Companion to Applied Regression</strong></div><div>SAGE Publications, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxS.WeisbergAn+R+Companion+to+Applied+RegressionThird+Edition2019SAGE+Publications" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref177-1" title="View in article">56.</a></div><div>Frelat, M.A. ∙ Shaw, C.N. ∙ Sukhdeo, S. ...</div><div><strong>Evolution of the hominin knee and ankle</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 2017; <strong>108</strong>:147-160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.jhevol.2017.03.006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85019890304" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622927/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jhevol.2017.03.006&amp;pmid=28622927" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">57.</a></div><div>Gabriel, S.B. ∙ Schaffner, S.F. ∙ Nguyen, H. ...</div><div><strong>The structure of haplotype blocks in the human genome</strong></div><div><em>Science.</em> 2002; <strong>296</strong>:2225-2229</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1069424" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-18444369013" target="_blank" aria-label="Open Scopus in new tab">Scopus (4402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12029063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1069424&amp;pmid=12029063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">58.</a></div><div>Gamer, L.W. ∙ Pregizer, S. ∙ Gamer, J. ...</div><div><strong>The Role of Bmp2 in the Maturation and Maintenance of the Murine Knee Joint</strong></div><div><em>J.&nbsp;Bone Miner. Res.</em> 2018; <strong>33</strong>:1708-1717</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/jbmr.3441" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85052506820" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29665134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fjbmr.3441&amp;pmid=29665134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">59.</a></div><div>Gardner, E. ∙ O’Rahilly, R.</div><div><strong>The early development of the knee joint in staged human embryos</strong></div><div><em>J.&nbsp;Anat.</em> 1968; <strong>102</strong>:289-299</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5643844/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=5643844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">60.</a></div><div>Gazal, S. ∙ Finucane, H.K. ∙ Furlotte, N.A. ...</div><div><strong>Linkage disequilibrium-dependent architecture of human complex traits shows action of negative selection</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1421-1427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85030183053" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28892061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3954&amp;pmid=28892061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-3" title="View in article">61.</a></div><div>Gehrke, A.R. ∙ Schneider, I. ∙ de la Calle-Mustienes, E. ...</div><div><strong>Deep conservation of wrist and digit enhancers in fish</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:803-808</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1420208112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84921395685" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25535365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1420208112&amp;pmid=25535365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">62.</a></div><div>Gel, B. ∙ Díez-Villanueva, A. ∙ Serra, E. ...</div><div><strong>regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:289-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv562" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84959886262" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26424858/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv562&amp;pmid=26424858" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib207" class="citations"><div id="sref207" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref207" title="View in article">63.</a></div><div>Gillis, J. ∙ Pavlidis, P.</div><div><strong>Assessing identity, redundancy and confounds in Gene Ontology annotations over time</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:476-482</div><div>In press</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts727" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874298500" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23297035/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts727&amp;pmid=23297035" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145-2" title="View in article">64.</a></div><div>Gittelman, R.M. ∙ Hun, E. ∙ Ay, F. ...</div><div><strong>Comprehensive identification and analysis of human accelerated regulatory DNA</strong></div><div><em>Genome Res.</em> 2015; <strong>25</strong>:1245-1255</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.192591.115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84937476478" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.192591.115&amp;pmid=26104583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">65.</a></div><div>Glasson, S.S. ∙ Chambers, M.G. ∙ Van Den Berg, W.B. ...</div><div><strong>The OARSI histopathology initiative - recommendations for histological assessments of osteoarthritis in the mouse</strong></div><div><em>Osteoarthritis Cartilage.</em> 2010; <strong>18</strong>:S17-S23</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2010.05.025&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2010.05.025&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77956924858" target="_blank" aria-label="Open Scopus in new tab">Scopus (576)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20864019/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2010.05.025&amp;pmid=20864019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref177-2" title="View in article">66.</a></div><div>Gokhman, D. ∙ Agranat-Tamir, L. ∙ Housman, G. ...</div><div><strong>Recent Regulatory Changes Shaped Human Facial and Vocal Anatomy</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/106955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F106955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">67.</a></div><div>Gray, D.J. ∙ Gardner, E.</div><div><strong>Prenatal development of the human knee and superior tibiofibular joints</strong></div><div><em>Am. J. Anat.</em> 1950; <strong>86</strong>:235-287</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/aja.1000860204" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15410671/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Faja.1000860204&amp;pmid=15410671" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">68.</a></div><div>Grossman, S.R. ∙ Andersen, K.G. ∙ Shlyakhter, I. ..., 1000 Genomes Project</div><div><strong>Identifying recent adaptations in large-scale genomic data</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:703-713</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2013.01.035&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84874069538" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415221/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.035&amp;pmid=23415221" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">69.</a></div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M.</div><div><strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:2847-2849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw313" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84992187637" target="_blank" aria-label="Open Scopus in new tab">Scopus (664)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw313&amp;pmid=27207943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">70.</a></div><div>Guenther, C. ∙ Pantalena-Filho, L. ∙ Kingsley, D.M.</div><div><strong>Shaping skeletal growth by modular regulatory elements in the Bmp5 gene</strong></div><div><em>PLoS Genet.</em> 2008; <strong>4</strong>:e1000308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1000308" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-58149163279" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19096511/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1000308&amp;pmid=19096511" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">71.</a></div><div>Guo, M. ∙ Liu, Z. ∙ Willen, J. ...</div><div><strong>Epigenetic profiling of growth plate chondrocytes sheds insight into regulatory genetic variation influencing height</strong></div><div><em>eLife.</em> 2017; <strong>6</strong>:e29329</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.7554/eLife.29329" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85040906101" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29205154/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.7554%2FeLife.29329&amp;pmid=29205154" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168" title="View in article">72.</a></div><div>Hashemi, J. ∙ Chandrashekar, N. ∙ Gill, B. ...</div><div><strong>The geometry of the tibial plateau and its influence on the biomechanics of the tibiofemoral joint</strong></div><div><em>J.&nbsp;Bone Joint Surg. Am.</em> 2008; <strong>90</strong>:2724-2734</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2106/JBJS.G.01358" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-57349117781" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19047719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2106%2FJBJS.G.01358&amp;pmid=19047719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">73.</a></div><div>He, Y. ∙ Gong, L. ∙ Fang, Y. ...</div><div><strong>The role of retinoic acid in hepatic lipid homeostasis defined by genomic binding and transcriptome profiling</strong></div><div><em>BMC Genomics.</em> 2013; <strong>14</strong>:575</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-14-575" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84883128543" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23981290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-14-575&amp;pmid=23981290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">74.</a></div><div>Hedman, A.C. ∙ Smith, J.M. ∙ Sacks, D.B.</div><div><strong>The biology of IQGAP proteins: beyond the cytoskeleton</strong></div><div><em>EMBO Rep.</em> 2015; <strong>16</strong>:427-446</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15252/embr.201439834" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84926252376" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25722290/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15252%2Fembr.201439834&amp;pmid=25722290" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79-1" title="View in article">75.</a></div><div>Heinz, S. ∙ Benner, C. ∙ Spann, N. ...</div><div><strong>Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</strong></div><div><em>Mol. Cell.</em> 2010; <strong>38</strong>:576-589</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.molcel.2010.05.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952567987" target="_blank" aria-label="Open Scopus in new tab">Scopus (3844)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.molcel.2010.05.004&amp;pmid=20513432" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">76.</a></div><div>Hijazi, A.</div><div><strong>Investigating the role of CCCTC-binding factor in osteoarthritis pathogenesis</strong></div><div><em>Osteoarthritis Cartilage.</em> 2018; <strong>26</strong>:S154</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2018.02.332&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2018.02.332&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29222056/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2018.02.332&amp;pmid=29222056" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">77.</a></div><div>Hindorff, L.A. ∙ Sethupathy, P. ∙ Junkins, H.A. ...</div><div><strong>Potential etiologic and functional implications of genome-wide association loci for human diseases and traits</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:9362-9367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0903103106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-67249117049" target="_blank" aria-label="Open Scopus in new tab">Scopus (2719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19474294/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0903103106&amp;pmid=19474294" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">78.</a></div><div>Hormozdiari, F. ∙ Gazal, S. ∙ van de Geijn, B. ...</div><div><strong>Leveraging molecular quantitative trait loci to understand the genetic architecture of diseases and complex traits</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1041-1047</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0148-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85049003482" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0148-2&amp;pmid=29942083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref156" title="View in article">79.</a></div><div>Houard, X. ∙ Goldring, M.B. ∙ Berenbaum, F.</div><div><strong>Homeostatic mechanisms in articular cartilage and role of inflammation in osteoarthritis</strong></div><div><em>Curr. Rheumatol. Rep.</em> 2013; <strong>15</strong>:375</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s11926-013-0375-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84893782724" target="_blank" aria-label="Open Scopus in new tab">Scopus (159)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24072604/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs11926-013-0375-6&amp;pmid=24072604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168" title="View in article">80.</a></div><div>Howell, S.M. ∙ Howell, S.J. ∙ Hull, M.L.</div><div><strong>Assessment of the radii of the medial and lateral femoral condyles in varus and valgus knees with osteoarthritis</strong></div><div><em>J.&nbsp;Bone Joint Surg. Am.</em> 2010; <strong>92</strong>:98-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2106/JBJS.H.01566" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-73949146227" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20048101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2106%2FJBJS.H.01566&amp;pmid=20048101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">81.</a></div><div>Hubisz, M.J. ∙ Pollard, K.S. ∙ Siepel, A.</div><div><strong>PHAST and RPHAST: phylogenetic analysis with space/time models</strong></div><div><em>Brief. Bioinform.</em> 2011; <strong>12</strong>:41-51</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bib/bbq072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79551634894" target="_blank" aria-label="Open Scopus in new tab">Scopus (147)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21278375/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbib%2Fbbq072&amp;pmid=21278375" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">82.</a></div><div>Hume, M.A. ∙ Barrera, L.A. ∙ Gisselbrecht, S.S. ...</div><div><strong>UniPROBE, update 2015: new tools and content for the online database of protein-binding microarray data on protein-DNA interactions</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:D117-D122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku1045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84922580691" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25378322/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku1045&amp;pmid=25378322" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">83.</a></div><div>Hunter, D.J. ∙ Bierma-Zeinstra, S.</div><div><strong>Osteoarthritis</strong></div><div><em>Lancet.</em> 2019; <strong>393</strong>:1745-1759</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2819%2930417-9&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2FS0140-6736%2819%2930417-9&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31034380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2819%2930417-9&amp;pmid=31034380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-2" title="View in article">84.</a></div><div>Hunter, D. ∙ Nevitt, M. ∙ Lynch, J. ..., FNIH OA Biomarkers Consortium</div><div><strong>Longitudinal validation of periarticular bone area and 3D shape as biomarkers for knee OA progression? Data from the FNIH OA Biomarkers Consortium</strong></div><div><em>Ann. Rheum. Dis.</em> 2016; <strong>75</strong>:1607-1614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2015-207602" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26483253/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2015-207602&amp;pmid=26483253" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">85.</a></div><div>Indjeian, V.B. ∙ Kingman, G.A. ∙ Jones, F.C. ...</div><div><strong>Evolving New Skeletal Traits by cis-Regulatory Changes in Bone Morphogenetic Proteins</strong></div><div><em>Cell.</em> 2016; <strong>164</strong>:45-56</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2015.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.cell.2015.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84954262110" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26774823/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2015.12.007&amp;pmid=26774823" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#fig6" title="View in article">86.</a></div><div>Infante, C.R. ∙ Park, S. ∙ Mihala, A.G. ...</div><div><strong>Pitx1 broadly associates with limb enhancers and is enriched on hindlimb cis-regulatory elements</strong></div><div><em>Dev. Biol.</em> 2013; <strong>374</strong>:234-244</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2012.11.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84872376764" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23201014/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2012.11.017&amp;pmid=23201014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref198" title="View in article">87.</a></div><div>Jackman, S.</div><div><strong>{pscl}: Classes and Methods for {R} Developed in the Political Science Computational Laboratory</strong></div><div>R package version 1.5.2. United States Studies Centre, University of Sydney, 2017</div><div><a id="intref0435" href="https://github.com/atahk/pscl/" target="_blank">https://github.com/atahk/pscl/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.Jackman%7Bpscl%7D%3A+Classes+and+Methods+for+%7BR%7D+Developed+in+the+Political+Science+Computational+Laboratory2017R+package+version+1.5.2.+United+States+Studies+Centre%2C+University+of+Sydneyhttps%3A%2F%2Fgithub.com%2Fatahk%2Fpscl%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137-1" title="View in article">88.</a></div><div>Jagoda, E. ∙ Lawson, D.J. ∙ Wall, J.D. ...</div><div><strong>Disentangling Immediate Adaptive Introgression from Selection on Standing Introgressed Variation in Humans</strong></div><div><em>Mol. Biol. Evol.</em> 2018; <strong>35</strong>:623-630</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msx314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85046466019" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29220488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsx314&amp;pmid=29220488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127" title="View in article">89.</a></div><div>Jungers, W.L.</div><div><strong>Relative joint size and hominoid locomotor adaptations with implications for the evolution of hominid bipedalism</strong></div><div><em>J.&nbsp;Hum. Evol.</em> 1988; <strong>17</strong>:247-265</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0047-2484(88)90056-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0047-2484%2888%2990056-5" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">90.</a></div><div>Kannu, P. ∙ Bateman, J.F. ∙ Belluoccio, D. ...</div><div><strong>Employing molecular genetics of chondrodysplasias to inform the study of osteoarthritis</strong></div><div><em>Arthritis Rheum.</em> 2009; <strong>60</strong>:325-334</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.24251" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-59649092384" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19180483/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.24251&amp;pmid=19180483" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref177-2" title="View in article">91.</a></div><div>Kanton, S. ∙ Boyle, M.J. ∙ He, Z. ...</div><div><strong>Organoid single-cell genomic atlas uncovers human-specific features of brain development</strong></div><div><em>Nature.</em> 2019; <strong>574</strong>:418-422</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1654-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85073480809" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31619793/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1654-9&amp;pmid=31619793" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">92.</a></div><div>Karolchik, D. ∙ Hinrichs, A.S. ∙ Furey, T.S. ...</div><div><strong>The UCSC Table Browser data retrieval tool</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:D493-D496</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkh103" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14681465/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkh103&amp;pmid=14681465" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98-1" title="View in article">93.</a></div><div>Karolchik, D. ∙ Barber, G.P. ∙ Casper, J. ...</div><div><strong>The UCSC Genome Browser database: 2014 update</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D764-D770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84891771466" target="_blank" aria-label="Open Scopus in new tab">Scopus (481)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24270787/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1168&amp;pmid=24270787" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99-1" title="View in article">94.</a></div><div>Kassambara, A. ∙ Mundt, F.</div><div><strong>Factoextra: extract and visualize the results of multivariate data analyses</strong></div><div>R package version 1.0. 4, 2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KassambaraF.MundtFactoextra%3A+extract+and+visualize+the+results+of+multivariate+data+analyses2017R+package+version+1.0.+4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100" title="View in article">95.</a></div><div>Kent, W.J. ∙ Sugnet, C.W. ∙ Furey, T.S. ...</div><div><strong>The human genome browser at UCSC</strong></div><div><em>Genome Res.</em> 2002; <strong>12</strong>:996-1006</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.229102.+Article+published+online+before+print+in+May+2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0036079158" target="_blank" aria-label="Open Scopus in new tab">Scopus (5344)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12045153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002&amp;pmid=12045153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-3" title="View in article">96.</a></div><div>Kerkhof, H.J. ∙ Lories, R.J. ∙ Meulenbelt, I. ...</div><div><strong>A genome-wide association study identifies an osteoarthritis susceptibility locus on chromosome 7q22</strong></div><div><em>Arthritis Rheum.</em> 2010; <strong>62</strong>:499-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.27184" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-75749110202" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20112360/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.27184&amp;pmid=20112360" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref182" title="View in article">97.</a></div><div>King, M.-C. ∙ Wilson, A.C.</div><div><strong>Evolution at two levels in humans and chimpanzees</strong></div><div><em>Science.</em> 1975; <strong>188</strong>:107-116</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1090005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1090005/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1090005&amp;pmid=1090005" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref199-1" title="View in article">98.</a></div><div>Klein, J.C. ∙ Keith, A. ∙ Rice, S.J. ...</div><div><strong>Functional testing of thousands of osteoarthritis-associated variants for regulatory activity</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:2434</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-019-10439-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85066610023" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31164647/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-019-10439-y&amp;pmid=31164647" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">99.</a></div><div>Kokenyesi, R. ∙ Tan, L. ∙ Robbins, J.R. ...</div><div><strong>Proteoglycan production by immortalized human chondrocyte cell lines cultured under conditions that promote expression of the differentiated phenotype</strong></div><div><em>Arch. Biochem. Biophys.</em> 2000; <strong>383</strong>:79-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/abbi.2000.2044" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0034326794" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11097179/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fabbi.2000.2044&amp;pmid=11097179" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">100.</a></div><div>Kosicki, M. ∙ Tomberg, K. ∙ Bradley, A.</div><div><strong>Repair of double-strand breaks induced by CRISPR-Cas9 leads to large deletions and complex rearrangements</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:765-771</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt0918-899c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053085220" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30010673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt0918-899c&amp;pmid=30010673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107-1" title="View in article">101.</a></div><div>Langmead, B. ∙ Salzberg, S.L.</div><div><strong>Fast gapped-read alignment with Bowtie 2</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:357-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84859210032" target="_blank" aria-label="Open Scopus in new tab">Scopus (15006)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.1923&amp;pmid=22388286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108-1" title="View in article">102.</a></div><div>Li, H.</div><div><strong>Tabix: fast retrieval of sequence features from generic TAB-delimited files</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:718-719</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79951993896" target="_blank" aria-label="Open Scopus in new tab">Scopus (192)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21208982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq671&amp;pmid=21208982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">103.</a></div><div>Li, H.</div><div><strong>A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</strong></div><div><em>Bioinformatics.</em> 2011; <strong>27</strong>:2987-2993</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btr509" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-80054915847" target="_blank" aria-label="Open Scopus in new tab">Scopus (1639)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21903627/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtr509&amp;pmid=21903627" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">104.</a></div><div>Li, Q. ∙ Brown, J.B. ∙ Huang, H. ...</div><div><strong>Measuring reproducibility of high-throughput experiments</strong></div><div><em>Ann. Appl. Stat.</em> 2011; <strong>5</strong>:1752-1779</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1214/11-AOAS466" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79955564639" target="_blank" aria-label="Open Scopus in new tab">Scopus (359)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1214%2F11-AOAS466" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111-1" title="View in article">105.</a></div><div>Liu, C.-F. ∙ Samsa, W.E. ∙ Zhou, G. ...</div><div><strong>Transcriptional control of chondrocyte specification and differentiation</strong></div><div><em>Semin. Cell Dev. Biol.</em> 2017; <strong>62</strong>:34-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.semcdb.2016.10.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85006055004" target="_blank" aria-label="Open Scopus in new tab">Scopus (49)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27771362/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.semcdb.2016.10.004&amp;pmid=27771362" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">106.</a></div><div>Liu, Y. ∙ Yau, M.S. ∙ Yerges-Armstrong, L.M. ...</div><div><strong>Genetic Determinants of Radiographic Knee Osteoarthritis in African Americans</strong></div><div><em>J.&nbsp;Rheumatol.</em> 2017; <strong>44</strong>:1652-1658</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3899/jrheum.161488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85032637808" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28916551/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3899%2Fjrheum.161488&amp;pmid=28916551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">107.</a></div><div>Loughlin, J.</div><div><strong>Genetic contribution to osteoarthritis development: current state of evidence</strong></div><div><em>Curr. Opin. Rheumatol.</em> 2015; <strong>27</strong>:284-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/BOR.0000000000000171" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84926329234" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25775188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FBOR.0000000000000171&amp;pmid=25775188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114-1" title="View in article">108.</a></div><div>Lovejoy, C.O.</div><div><strong>The natural history of human gait and posture. Part 3. The knee</strong></div><div><em>Gait Posture.</em> 2007; <strong>25</strong>:325-341</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gaitpost.2006.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33846904576" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16766186/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gaitpost.2006.05.001&amp;pmid=16766186" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115-1" title="View in article">109.</a></div><div>Lowenstine, L.J. ∙ McManamon, R. ∙ Terio, K.A.</div><div><strong>Comparative Pathology of Aging Great Apes: Bonobos, Chimpanzees, Gorillas, and Orangutans</strong></div><div><em>Vet. Pathol.</em> 2016; <strong>53</strong>:250-276</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985815612154" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84960119504" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26721908/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985815612154&amp;pmid=26721908" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">110.</a></div><div>Lyons, K.M. ∙ Rosen, V.</div><div><strong>BMPs, TGFβ, and border security at the interzone</strong></div><div><em>Curr. Top. Dev. Biol.</em> 2019; <strong>133</strong>:153-170</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/bs.ctdb.2019.02.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85062191280" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30902251/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fbs.ctdb.2019.02.001&amp;pmid=30902251" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146" title="View in article">111.</a></div><div>Maas, S.A. ∙ Fallon, J.F.</div><div><strong>Single base pair change in the long-range Sonic hedgehog limb-specific enhancer is a genetic basis for preaxial polydactyly</strong></div><div><em>Dev. Dyn.</em> 2005; <strong>232</strong>:345-348</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.20254" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-12344299320" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15637698/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.20254&amp;pmid=15637698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118" title="View in article">112.</a></div><div>Macrini, T.E. ∙ Coan, H.B. ∙ Levine, S.M. ...</div><div><strong>Reproductive status and sex show strong effects on knee OA in a baboon model</strong></div><div><em>Osteoarthritis Cartilage.</em> 2013; <strong>21</strong>:839-848</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2013.03.003&amp;cf=fulltext&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.joca.2013.03.003&amp;cf=pdf&amp;site=yjoca-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84877111473" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23499674/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.joca.2013.03.003&amp;pmid=23499674" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119-1" title="View in article">113.</a></div><div>Maechler, M. ∙ Rousseeuw, P. ∙ Struyf, A. ...</div><div><strong>cluster: Cluster Analysis Basics and Extensions. R Packages Version 1</strong></div><div>2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MaechlerP.RousseeuwA.StruyfM.HubertK.Hornikcluster%3A+Cluster+Analysis+Basics+and+Extensions.+R+Packages+Version+12018" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120" title="View in article">114.</a></div><div>Mathelier, A. ∙ Fornes, O. ∙ Arenillas, D.J. ...</div><div><strong>JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:D110-D115</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1176" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26531826/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1176&amp;pmid=26531826" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121-1" title="View in article">115.</a></div><div>McLean, C.Y. ∙ Bristor, D. ∙ Hiller, M. ...</div><div><strong>GREAT improves functional interpretation of cis-regulatory regions</strong></div><div><em>Nat. Biotechnol.</em> 2010; <strong>28</strong>:495-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1630" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77952214662" target="_blank" aria-label="Open Scopus in new tab">Scopus (1686)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20436461/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1630&amp;pmid=20436461" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref182" title="View in article">116.</a></div><div>McLean, C.Y. ∙ Reno, P.L. ∙ Pollen, A.A. ...</div><div><strong>Human-specific loss of regulatory DNA and the evolution of human-specific traits</strong></div><div><em>Nature.</em> 2011; <strong>471</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09774" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79952386187" target="_blank" aria-label="Open Scopus in new tab">Scopus (274)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21390129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09774&amp;pmid=21390129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">117.</a></div><div>Mérida-Velasco, J.A. ∙ Sánchez-Montesinos, I. ∙ Espín-Ferra, J. ...</div><div><strong>Development of the human knee joint</strong></div><div><em>Anat. Rec.</em> 1997; <strong>248</strong>:269-278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1097-0185(199706)248:2%3C269::AID-AR14%3E3.0.CO;2-N" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0030974656" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9185993/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291097-0185%28199706%29248%3A2%253C269%3A%3AAID-AR14%253E3.0.CO%3B2-N&amp;pmid=9185993" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124" title="View in article">118.</a></div><div>Meyer, M.B. ∙ Benkusky, N.A. ∙ Pike, J.W.</div><div><strong>The RUNX2 cistrome in osteoblasts: characterization, down-regulation following differentiation, and relationship to gene expression</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2014; <strong>289</strong>:16016-16031</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M114.552216" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84902207251" target="_blank" aria-label="Open Scopus in new tab">Scopus (69)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24764292/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M114.552216&amp;pmid=24764292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">119.</a></div><div>Miyamoto, Y. ∙ Mabuchi, A. ∙ Shi, D. ...</div><div><strong>A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2007; <strong>39</strong>:529-533</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/2005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34047103363" target="_blank" aria-label="Open Scopus in new tab">Scopus (336)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17384641/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F2005&amp;pmid=17384641" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">120.</a></div><div>Miyamoto, Y. ∙ Shi, D. ∙ Nakajima, M. ...</div><div><strong>Common variants in DVWA on chromosome 3p24.3 are associated with susceptibility to knee osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2008; <strong>40</strong>:994-998</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.176" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-48349105759" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18622395/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.176&amp;pmid=18622395" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127" title="View in article">121.</a></div><div>Morrison, J.B.</div><div><strong>The mechanics of the knee joint in relation to normal walking</strong></div><div><em>J.&nbsp;Biomech.</em> 1970; <strong>3</strong>:51-61</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0021-9290(70)90050-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0014698584" target="_blank" aria-label="Open Scopus in new tab">Scopus (764)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/5521530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0021-9290%2870%2990050-3&amp;pmid=5521530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">122.</a></div><div>Mun, J.</div><div><strong>Advanced Analytical Models: over 800 Models and 300 Applications from the Basel II Accord to Wall Street and Beyond</strong></div><div>Wiley, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.MunAdvanced+Analytical+Models%3A+over+800+Models+and+300+Applications+from+the+Basel+II+Accord+to+Wall+Street+and+Beyond2008Wiley" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">123.</a></div><div>Nakajima, M. ∙ Takahashi, A. ∙ Kou, I. ...</div><div><strong>New sequence variants in HLA class II/III region associated with susceptibility to knee osteoarthritis identified by genome-wide association study</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e9723</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0009723" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78149408549" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20305777/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0009723&amp;pmid=20305777" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-1" title="View in article">124.</a></div><div>Nemec, S. ∙ Luxey, M. ∙ Jain, D. ...</div><div><strong><i>Pitx1</i> directly modulates the core limb development program to implement hindlimb identity</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:3325-3335</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.154864" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85029782232" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28807899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.154864&amp;pmid=28807899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-1" title="View in article">125.</a></div><div>Neogi, T. ∙ Felson, D.T.</div><div><strong>Osteoarthritis: Bone as an imaging biomarker and treatment target in OA</strong></div><div><em>Nat. Rev. Rheumatol.</em> 2016; <strong>12</strong>:503-504</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrrheum.2016.113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84978174688" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27383914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrrheum.2016.113&amp;pmid=27383914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-1" title="View in article">126.</a></div><div>Neogi, T. ∙ Bowes, M.A. ∙ Niu, J. ...</div><div><strong>Magnetic resonance imaging-based three-dimensional bone shape of the knee predicts onset of knee osteoarthritis: data from the osteoarthritis initiative</strong></div><div><em>Arthritis Rheum.</em> 2013; <strong>65</strong>:2048-2058</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.37987" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881340844" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23650083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.37987&amp;pmid=23650083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134" title="View in article">127.</a></div><div>O’Rahilly, R.</div><div><strong>The early prenatal development of the human knee joint</strong></div><div><em>J.&nbsp;Anat.</em> 1951; <strong>85</strong>:166-170</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14824030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=14824030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136" title="View in article">128.</a></div><div>Ohba, S. ∙ He, X. ∙ Hojo, H. ...</div><div><strong>Distinct Transcriptional Programs Underlie Sox9 Regulation of the Mammalian Chondrocyte</strong></div><div><em>Cell Rep.</em> 2015; <strong>12</strong>:229-243</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.celrep.2015.06.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=10.1016%2Fj.celrep.2015.06.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84937521813" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26146088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2015.06.013&amp;pmid=26146088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137-1" title="View in article">129.</a></div><div>Pagani, L. ∙ Lawson, D.J. ∙ Jagoda, E. ...</div><div><strong>Genomic analyses inform on migration events during the peopling of Eurasia</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:238-242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19792" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84981502573" target="_blank" aria-label="Open Scopus in new tab">Scopus (160)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654910/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19792&amp;pmid=27654910" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">130.</a></div><div>Panoutsopoulou, K. ∙ Southam, L. ∙ Elliott, K.S. ..., arcOGEN Consortium</div><div><strong>Insights into the genetic architecture of osteoarthritis from stage 1 of the arcOGEN study</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:864-867</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.141473" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-79953309875" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21177295/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.141473&amp;pmid=21177295" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref139-1" title="View in article">131.</a></div><div>Paradis, E.</div><div><strong>pegas: an R package for population genetics with an integrated-modular approach</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:419-420</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp696" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77949527981" target="_blank" aria-label="Open Scopus in new tab">Scopus (757)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20080509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp696&amp;pmid=20080509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib140" class="citations"><div id="sref140" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref140-1" title="View in article">132.</a></div><div>Picard, C. ∙ Azeddine, B. ∙ Moldovan, F. ...</div><div><strong>New emerging role of pitx1 transcription factor in osteoarthritis pathogenesis</strong></div><div><em>Clin. Orthop. Relat. Res.</em> 2007; <strong>462</strong>:59-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/BLO.0b013e3180d09d9c" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-34548400346" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17549029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FBLO.0b013e3180d09d9c&amp;pmid=17549029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib141" class="citations"><div id="sref141" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref141" title="View in article">133.</a></div><div>Polk, J.D. ∙ Williams, S.A. ∙ Peterson, J.V.</div><div><strong>Body size and joint posture in primates</strong></div><div><em>Am. J. Phys. Anthropol.</em> 2009; <strong>140</strong>:359-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/ajpa.21083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-70349149919" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19425104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fajpa.21083&amp;pmid=19425104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib142" class="citations"><div id="sref142" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145-2" title="View in article">134.</a></div><div>Pollard, K.S. ∙ Salama, S.R. ∙ King, B. ...</div><div><strong>Forces shaping the fastest evolving regions in the human genome</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e168</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020168" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-35648988566" target="_blank" aria-label="Open Scopus in new tab">Scopus (254)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17040131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020168&amp;pmid=17040131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib143" class="citations"><div id="sref143" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref143" title="View in article">135.</a></div><div>Pollard, K.S. ∙ Salama, S.R. ∙ Lambert, N. ...</div><div><strong>An RNA gene expressed during cortical development evolved rapidly in humans</strong></div><div><em>Nature.</em> 2006; <strong>443</strong>:167-172</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33748675254" target="_blank" aria-label="Open Scopus in new tab">Scopus (626)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16915236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05113&amp;pmid=16915236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib144" class="citations"><div id="sref144" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144-1" title="View in article">136.</a></div><div>Pollard, K.S. ∙ Hubisz, M.J. ∙ Rosenbloom, K.R. ...</div><div><strong>Detection of nonneutral substitution rates on mammalian phylogenies</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:110-121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.097857.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-74949092081" target="_blank" aria-label="Open Scopus in new tab">Scopus (968)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19858363/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.097857.109&amp;pmid=19858363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib145" class="citations"><div id="sref145" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">137.</a></div><div>Prabhakar, S. ∙ Noonan, J.P. ∙ Pääbo, S. ...</div><div><strong>Accelerated evolution of conserved noncoding sequences in humans</strong></div><div><em>Science.</em> 2006; <strong>314</strong>:786</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1130738" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750737061" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17082449/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1130738&amp;pmid=17082449" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib146" class="citations"><div id="sref146" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146" title="View in article">138.</a></div><div>Prabhakar, S. ∙ Visel, A. ∙ Akiyama, J.A. ...</div><div><strong>Human-specific gain of function in a developmental enhancer</strong></div><div><em>Science.</em> 2008; <strong>321</strong>:1346-1350</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1159974" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-51149097366" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18772437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1159974&amp;pmid=18772437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib147" class="citations"><div id="sref147" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref147-1" title="View in article">139.</a></div><div>Prado-Martinez, J. ∙ Sudmant, P.H. ∙ Kidd, J.M. ...</div><div><strong>Great ape genetic diversity and population history</strong></div><div><em>Nature.</em> 2013; <strong>499</strong>:471-475</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12228" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84881159598" target="_blank" aria-label="Open Scopus in new tab">Scopus (402)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23823723/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12228&amp;pmid=23823723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib148" class="citations"><div id="sref148" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148-1" title="View in article">140.</a></div><div>Pregizer, S.K. ∙ Kiapour, A.M. ∙ Young, M. ...</div><div><strong>Impact of broad regulatory regions on Gdf5 expression and function in knee development and susceptibility to osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2018; <strong>77</strong>:450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/annrheumdis-2017-212475" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85052526710" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29311146/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fannrheumdis-2017-212475&amp;pmid=29311146" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib149" class="citations"><div id="sref149" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref149-1" title="View in article">141.</a></div><div>Quinlan, A.R. ∙ Hall, I.M.</div><div><strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:841-842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq033" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-77951770756" target="_blank" aria-label="Open Scopus in new tab">Scopus (6963)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq033&amp;pmid=20110278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib150" class="citations"><div id="sref150" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref150-1" title="View in article">142.</a></div><div>R Development Core Team</div><div><strong>R: A Language and Environment for Statistical Computing.</strong></div><div>R Foundation for Statistical Computing, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Development+Core+TeamR%3A+A+Language+and+Environment+for+Statistical+Computing.2008R+Foundation+for+Statistical+Computing" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib151" class="citations"><div id="sref151" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">143.</a></div><div>Racimo, F.</div><div><strong>Testing for ancient selection using cross-population allele frequency differentiation</strong></div><div><em>Genetics.</em> 2016; <strong>202</strong>:733-750</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.115.178095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84973624508" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26596347/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.115.178095&amp;pmid=26596347" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib152" class="citations"><div id="sref152" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">144.</a></div><div>Ramos, Y.F. ∙ Metrustry, S. ∙ Arden, N. ..., arcOGEN Consortium, TreatOA Collaborators</div><div><strong>Meta-analysis identifies loci affecting levels of the potential osteoarthritis biomarkers sCOMP and uCTX-II with genome wide significance</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2014; <strong>51</strong>:596-604</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2014-102478" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84907073808" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25057126/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2014-102478&amp;pmid=25057126" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib153" class="citations"><div id="sref153" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref153-1" title="View in article">145.</a></div><div>Ran, F.A. ∙ Hsu, P.D. ∙ Wright, J. ...</div><div><strong>Genome engineering using the CRISPR-Cas9 system</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:2281-2308</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84887010498" target="_blank" aria-label="Open Scopus in new tab">Scopus (3965)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24157548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.143&amp;pmid=24157548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib154" class="citations"><div id="sref154" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref154" title="View in article">146.</a></div><div>Rasmussen, M.D. ∙ Hubisz, M.J. ∙ Gronau, I. ...</div><div><strong>Genome-wide inference of ancestral recombination graphs</strong></div><div><em>PLoS Genet.</em> 2014; <strong>10</strong>:e1004342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1004342" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84901640930" target="_blank" aria-label="Open Scopus in new tab">Scopus (99)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24831947/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1004342&amp;pmid=24831947" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib155" class="citations"><div id="sref155" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref155" title="View in article">147.</a></div><div>Reyes, C. ∙ Leyland, K.M. ∙ Peat, G. ...</div><div><strong>Association between overweight and obesity and risk of clinically diagnosed knee, hip, and hand osteoarthritis: a population-based cohort study</strong></div><div><em>Arthritis Rheumatol.</em> 2016; <strong>68</strong>:1869-1875</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.39707" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84979642437" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27059260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.39707&amp;pmid=27059260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib156" class="citations"><div id="sref156" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref156" title="View in article">148.</a></div><div>Richette, P. ∙ Poitou, C. ∙ Garnero, P. ...</div><div><strong>Benefits of massive weight loss on symptoms, systemic inflammation and cartilage turnover in obese patients with knee osteoarthritis</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:139-144</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.134015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78650676944" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20980288/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.134015&amp;pmid=20980288" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib157" class="citations"><div id="sref157" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">149.</a></div><div>Rosen, V.</div><div><strong>BMP2 signaling in bone development and repair</strong></div><div><em>Cytokine Growth Factor Rev.</em> 2009; <strong>20</strong>:475-480</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.cytogfr.2009.10.018" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-70649094222" target="_blank" aria-label="Open Scopus in new tab">Scopus (227)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19892583/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cytogfr.2009.10.018&amp;pmid=19892583" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib158" class="citations"><div id="sref158" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref164" title="View in article">150.</a></div><div>Rountree, R.B. ∙ Schoor, M. ∙ Chen, H. ...</div><div><strong>BMP receptor signaling is required for postnatal maintenance of articular cartilage</strong></div><div><em>PLoS Biol.</em> 2004; <strong>2</strong>:e355</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0020355" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-13944283679" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15492776/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0020355&amp;pmid=15492776" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib160" class="citations"><div id="sref160" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref160" title="View in article">151.</a></div><div>Salazar, V.S. ∙ Gamer, L.W. ∙ Rosen, V.</div><div><strong>BMP signalling in skeletal development, disease and repair</strong></div><div><em>Nat. Rev. Endocrinol.</em> 2016; <strong>12</strong>:203-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrendo.2016.12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84961215919" target="_blank" aria-label="Open Scopus in new tab">Scopus (245)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26893264/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrendo.2016.12&amp;pmid=26893264" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib161" class="citations"><div id="sref161" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168" title="View in article">152.</a></div><div>Schwartz, H.J.</div><div><strong>Effect of chronic chromolyn sodium therapy in a beautician with occupational asthma</strong></div><div><em>J.&nbsp;Occup. Med.</em> 1989; <strong>31</strong>:112-114</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2496209/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=2496209" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib162" class="citations"><div id="sref162" class="citation"><div class="citation-content"><div class="label"><a href="#fig2" title="View in article">153.</a></div><div>Selleri, L. ∙ Depew, M.J. ∙ Jacobs, Y. ...</div><div><strong>Requirement for Pbx1 in skeletal patterning and programming chondrocyte proliferation and differentiation</strong></div><div><em>Development.</em> 2001; <strong>128</strong>:3543-3557</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.128.18.3543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11566859/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.128.18.3543&amp;pmid=11566859" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib163" class="citations"><div id="sref163" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">154.</a></div><div>Sen, M. ∙ Cheng, Y.-H. ∙ Goldring, M.B. ...</div><div><strong>WISP3-dependent regulation of type II collagen and aggrecan production in chondrocytes</strong></div><div><em>Arthritis Rheum.</em> 2004; <strong>50</strong>:488-497</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.20005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-1042290330" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14872491/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.20005&amp;pmid=14872491" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib164" class="citations"><div id="sref164" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref164" title="View in article">155.</a></div><div>Settle, Jr., S.H. ∙ Rountree, R.B. ∙ Sinha, A. ...</div><div><strong>Multiple joint and skeletal patterning defects caused by single and double mutations in the mouse Gdf6 and Gdf5 genes</strong></div><div><em>Dev. Biol.</em> 2003; <strong>254</strong>:116-130</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0012-1606(02)00022-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0037330644" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12606286/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0012-1606%2802%2900022-2&amp;pmid=12606286" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib165" class="citations"><div id="sref165" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref165-1" title="View in article">156.</a></div><div>Shannon, P. ∙ Richards, M.</div><div><strong>MotifDb: An annotated collection of protein-DNA binding sequence motifs. R Package. Version 1</strong></div><div>2014</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.ShannonM.RichardsMotifDb%3A+An+annotated+collection+of+protein-DNA+binding+sequence+motifs.+R+Package.+Version+12014" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib166" class="citations"><div id="sref166" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref166" title="View in article">157.</a></div><div>Shinoda, Y. ∙ Ogata, N. ∙ Higashikawa, A. ...</div><div><strong>Kruppel-like factor 5 causes cartilage degradation through transactivation of matrix metalloproteinase 9</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2008; <strong>283</strong>:24682-24689</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M709857200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-54049154742" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18617520/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M709857200&amp;pmid=18617520" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib167" class="citations"><div id="sref167" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref167" title="View in article">158.</a></div><div>Shlyueva, D. ∙ Stampfel, G. ∙ Stark, A.</div><div><strong>Transcriptional enhancers: from properties to genome-wide predictions</strong></div><div><em>Nat. Rev. Genet.</em> 2014; <strong>15</strong>:272-286</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg3682" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84899450857" target="_blank" aria-label="Open Scopus in new tab">Scopus (584)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24614317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg3682&amp;pmid=24614317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib168" class="citations"><div id="sref168" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168" title="View in article">159.</a></div><div>Sonnery-Cottet, B. ∙ Archbold, P. ∙ Cucurulo, T. ...</div><div><strong>The influence of the tibial slope and the size of the intercondylar notch on rupture of the anterior cruciate ligament</strong></div><div><em>J.&nbsp;Bone Joint Surg. Br.</em> 2011; <strong>93</strong>:1475-1478</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1302/0301-620X.93B11.26905" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-81155125190" target="_blank" aria-label="Open Scopus in new tab">Scopus (78)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22058297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1302%2F0301-620X.93B11.26905&amp;pmid=22058297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib169" class="citations"><div id="sref169" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref169-1" title="View in article">160.</a></div><div>Stamatakis, A.</div><div><strong>RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</strong></div><div><em>Bioinformatics.</em> 2006; <strong>22</strong>:2688-2690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btl446" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33750403801" target="_blank" aria-label="Open Scopus in new tab">Scopus (11499)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16928733/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtl446&amp;pmid=16928733" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib171" class="citations"><div id="sref171" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">161.</a></div><div>Styrkarsdottir, U. ∙ Thorleifsson, G. ∙ Helgadottir, H.T. ..., TREAT-OA Consortium, arcOGEN Consortium</div><div><strong>Severe osteoarthritis of the hand associates with common variants within the ALDH1A2 gene and with rare variants at 1p31</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:498-502</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2957" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84899638695" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24728293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2957&amp;pmid=24728293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib172" class="citations"><div id="sref172" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">162.</a></div><div>Styrkarsdottir, U. ∙ Helgason, H. ∙ Sigurdsson, A. ..., arcOGEN consortium</div><div><strong>Whole-genome sequencing identifies rare genotypes in COMP and CHADL associated with high risk of hip osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:801-805</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3816" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85015667730" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28319091/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3816&amp;pmid=28319091" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib173" class="citations"><div id="sref173" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref199-1" title="View in article">163.</a></div><div>Styrkarsdottir, U. ∙ Lund, S.H. ∙ Thorleifsson, G. ...</div><div><strong>Meta-analysis of Icelandic and UK data sets identifies missense variants in SMO, IL11, COL11A1 and 13 more new loci associated with osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1681-1687</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0247-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85055988857" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30374069/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0247-0&amp;pmid=30374069" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib175" class="citations"><div id="sref175" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref199-1" title="View in article">164.</a></div><div>Tachmazidou, I. ∙ Hatzikotoulas, K. ∙ Southam, L. ..., arcOGEN Consortium</div><div><strong>Identification of new therapeutic targets for osteoarthritis through genome-wide analyses of UK Biobank data</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:230-236</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0327-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85060491699" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30664745/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0327-1&amp;pmid=30664745" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib176" class="citations"><div id="sref176" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref176" title="View in article">165.</a></div><div>Tajima, F.</div><div><strong>Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</strong></div><div><em>Genetics.</em> 1989; <strong>123</strong>:585-595</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/genetics/123.3.585" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2513255/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgenetics%2F123.3.585&amp;pmid=2513255" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib177" class="citations"><div id="sref177" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref177-1" title="View in article">166.</a></div><div>Tardieu, C.</div><div><strong>Ontogeny and phylogeny of femoro-tibial characters in humans and hominid fossils: functional influence and genetic determinism</strong></div><div><em>Am. J. Phys. Anthropol.</em> 1999; <strong>110</strong>:365-377</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/(SICI)1096-8644(199911)110:3%3C365::AID-AJPA8%3E3.0.CO;2-T" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-0032747147" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10516567/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F%28SICI%291096-8644%28199911%29110%3A3%253C365%3A%3AAID-AJPA8%253E3.0.CO%3B2-T&amp;pmid=10516567" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib178" class="citations"><div id="sref178" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref178" title="View in article">167.</a></div><div>Tardieu, C.</div><div><strong>Development of the human hind limb and its importance for the evolution of bipedalism</strong></div><div><em>Evol. Anthropol.</em> 2010; <strong>19</strong>:174-186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/evan.20276" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78049380999" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fevan.20276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib179" class="citations"><div id="sref179" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">168.</a></div><div>Valdes, A.M. ∙ Van Oene, M. ∙ Hart, D.J. ...</div><div><strong>Reproducible genetic associations between candidate genes and clinical knee osteoarthritis in men and women</strong></div><div><em>Arthritis Rheum.</em> 2006; <strong>54</strong>:533-539</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/art.21621" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-32444433647" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16453284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fart.21621&amp;pmid=16453284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib180" class="citations"><div id="sref180" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">169.</a></div><div>Valdes, A.M. ∙ Evangelou, E. ∙ Kerkhof, H.J.M. ...</div><div><strong>The GDF5 rs143383 polymorphism is associated with osteoarthritis of the knee with genome-wide statistical significance</strong></div><div><em>Ann. Rheum. Dis.</em> 2011; <strong>70</strong>:873-875</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/ard.2010.134155" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-78650988509" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20870806/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fard.2010.134155&amp;pmid=20870806" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib182" class="citations"><div id="sref182" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref182" title="View in article">170.</a></div><div>Varki, A. ∙ Altheide, T.K.</div><div><strong>Comparing the human and chimpanzee genomes: searching for needles in a haystack</strong></div><div><em>Genome Res.</em> 2005; <strong>15</strong>:1746-1758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.3737405" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-28844466854" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16339373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.3737405&amp;pmid=16339373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib183" class="citations"><div id="sref183" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref183" title="View in article">171.</a></div><div>Visel, A. ∙ Thaller, C. ∙ Eichele, G.</div><div><strong>GenePaint.org: an atlas of gene expression patterns in the mouse embryo</strong></div><div><em>Nucleic Acids Res.</em> 2004; <strong>32</strong>:D552-D556</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkh029" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14681479/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkh029&amp;pmid=14681479" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib184" class="citations"><div id="sref184" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref184" title="View in article">172.</a></div><div>Vitti, J.J. ∙ Grossman, S.R. ∙ Sabeti, P.C.</div><div><strong>Detecting natural selection in genomic data</strong></div><div><em>Annu. Rev. Genet.</em> 2013; <strong>47</strong>:97-120</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-genet-111212-133526" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84888582669" target="_blank" aria-label="Open Scopus in new tab">Scopus (266)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24274750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-genet-111212-133526&amp;pmid=24274750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib185" class="citations"><div id="sref185" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref185" title="View in article">173.</a></div><div>Voight, B.F. ∙ Kudaravalli, S. ∙ Wen, X. ...</div><div><strong>A map of recent positive selection in the human genome</strong></div><div><em>PLoS Biol.</em> 2006; <strong>4</strong>:e72</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.0040072" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-33745160439" target="_blank" aria-label="Open Scopus in new tab">Scopus (138)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16494531/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.0040072&amp;pmid=16494531" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib186" class="citations"><div id="sref186" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref192" title="View in article">174.</a></div><div>Wang, R.N. ∙ Green, J. ∙ Wang, Z. ...</div><div><strong>Bone Morphogenetic Protein (BMP) signaling in development and human diseases</strong></div><div><em>Genes Dis.</em> 2014; <strong>1</strong>:87-105</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gendis.2014.07.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25401122/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gendis.2014.07.005&amp;pmid=25401122" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib187" class="citations"><div id="sref187" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref187" title="View in article">175.</a></div><div>Wang, J.S. ∙ Infante, C.R. ∙ Park, S. ...</div><div><strong>PITX1 promotes chondrogenesis and myogenesis in mouse hindlimbs through conserved regulatory targets</strong></div><div><em>Dev. Biol.</em> 2018; <strong>434</strong>:186-195</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2017.12.013" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85039043376" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29273440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2017.12.013&amp;pmid=29273440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib188" class="citations"><div id="sref188" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref188-1" title="View in article">176.</a></div><div>Ward, L.D. ∙ Kellis, M.</div><div><strong>HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D930-D934</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84858779229" target="_blank" aria-label="Open Scopus in new tab">Scopus (1288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22064851/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr917&amp;pmid=22064851" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib189" class="citations"><div id="sref189" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref189-1" title="View in article">177.</a></div><div>Weir, B.S. ∙ Cockerham, C.C.</div><div><strong>Estimating F-statistics for the analysis of population structure</strong></div><div><em>Evolution.</em> 1984; <strong>38</strong>:1358-1370</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.2307/2408641" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28563791/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.2307%2F2408641&amp;pmid=28563791" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib191" class="citations"><div id="sref191" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref191" title="View in article">178.</a></div><div>Wittkopp, P.J. ∙ Kalay, G.</div><div><strong>Cis-regulatory elements: molecular mechanisms and evolutionary processes underlying divergence</strong></div><div><em>Nat. Rev. Genet.</em> 2011; <strong>13</strong>:59-69</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrg3095" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-83855163414" target="_blank" aria-label="Open Scopus in new tab">Scopus (419)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22143240/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrg3095&amp;pmid=22143240" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib192" class="citations"><div id="sref192" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref192" title="View in article">179.</a></div><div>Wu, M. ∙ Chen, G. ∙ Li, Y.-P.</div><div><strong>TGF-β and BMP signaling in osteoblast, skeletal development, and bone formation, homeostasis and disease</strong></div><div><em>Bone Res.</em> 2016; <strong>4</strong>:16009</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/boneres.2016.9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84979021935" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27563484/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fboneres.2016.9&amp;pmid=27563484" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib193" class="citations"><div id="sref193" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref205" title="View in article">180.</a></div><div>Xu, S. ∙ Liu, P. ∙ Chen, Y. ...</div><div><strong>Foxp2 regulates anatomical features that may be relevant for vocal behaviors and bipedal locomotion</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:8799-8804</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1721820115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85053629492" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30104377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1721820115&amp;pmid=30104377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib195" class="citations"><div id="sref195" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref195" title="View in article">181.</a></div><div>Ye, W. ∙ Song, Y. ∙ Huang, Z. ...</div><div><strong>A unique stylopod patterning mechanism by Shox2-controlled osteogenesis</strong></div><div><em>Development.</em> 2016; <strong>143</strong>:2548-2560</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.138750" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84978700538" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27287812/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.138750&amp;pmid=27287812" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib196" class="citations"><div id="sref196" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref196" title="View in article">182.</a></div><div>Yue, F. ∙ Cheng, Y. ∙ Breschi, A. ..., Mouse ENCODE Consortium</div><div><strong>A comparative encyclopedia of DNA elements in the mouse genome</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:355-364</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13992" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84911462077" target="_blank" aria-label="Open Scopus in new tab">Scopus (718)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25409824/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13992&amp;pmid=25409824" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib197" class="citations"><div id="sref197" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref197" title="View in article">183.</a></div><div>Zehra, R. ∙ Abbasi, A.A.</div><div><strong>Homo sapiens-Specific Binding Site Variants within Brain Exclusive Enhancers Are Subject to Accelerated Divergence across Human Population</strong></div><div><em>Genome Biol. Evol.</em> 2018; <strong>10</strong>:956-966</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/gbe/evy052" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85054362503" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29608725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgbe%2Fevy052&amp;pmid=29608725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib198" class="citations"><div id="sref198" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref198" title="View in article">184.</a></div><div>Zeileis, A. ∙ Kleiber, C. ∙ Jackman, S.</div><div><strong>Regression Models for Count Data in {R}</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2008; <strong>27</strong>:1-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.18637/jss.v027.i08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.18637%2Fjss.v027.i08" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib199" class="citations"><div id="sref199" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref199-1" title="View in article">185.</a></div><div>Zengini, E. ∙ Hatzikotoulas, K. ∙ Tachmazidou, I. ...</div><div><strong>Genome-wide analyses using UK Biobank data provide insights into the genetic architecture of osteoarthritis</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:549-558</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0079-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85044225612" target="_blank" aria-label="Open Scopus in new tab">Scopus (57)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29559693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0079-y&amp;pmid=29559693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib200" class="citations"><div id="sref200" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref200" title="View in article">186.</a></div><div>Zhai, G. ∙ van Meurs, J.B. ∙ Livshits, G. ...</div><div><strong>A genome-wide association study suggests that a locus within the ataxin 2 binding protein 1 gene is associated with hand osteoarthritis: the Treat-OA consortium</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2009; <strong>46</strong>:614-616</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmg.2009.067314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-69749123294" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19508968/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmg.2009.067314&amp;pmid=19508968" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib201" class="citations"><div id="sref201" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">187.</a></div><div>Zhai, G. ∙ Doré, J. ∙ Rahman, P.</div><div><strong>TGF-β signal transduction pathways and osteoarthritis</strong></div><div><em>Rheumatol. Int.</em> 2015; <strong>35</strong>:1283-1292</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00296-015-3251-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84931569939" target="_blank" aria-label="Open Scopus in new tab">Scopus (32)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25773660/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00296-015-3251-z&amp;pmid=25773660" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib202" class="citations"><div id="sref202" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref202" title="View in article">188.</a></div><div>Zhang, F. ∙ Lupski, J.R.</div><div><strong>Non-coding genetic variants in human disease</strong></div><div><em>Hum. Mol. Genet.</em> 2015; <strong>24</strong>:R102-R110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddv259" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84943752520" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26152199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddv259&amp;pmid=26152199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib203" class="citations"><div id="sref203" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref203-1" title="View in article">189.</a></div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (5513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib204" class="citations"><div id="sref204" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref204-1" title="View in article">190.</a></div><div>Zhao, H. ∙ Sun, Z. ∙ Wang, J. ...</div><div><strong>CrossMap: a versatile tool for coordinate conversion between genome assemblies</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:1006-1007</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84897373336" target="_blank" aria-label="Open Scopus in new tab">Scopus (114)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24351709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt730&amp;pmid=24351709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib205" class="citations"><div id="sref205" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref205" title="View in article">191.</a></div><div>Zhao, H. ∙ Zhou, W. ∙ Yao, Z. ...</div><div><strong>Foxp1/2/4 regulate endochondral ossification as a suppresser complex</strong></div><div><em>Dev. Biol.</em> 2015; <strong>398</strong>:242-254</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2014.12.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-84922711705" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25527076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2014.12.007&amp;pmid=25527076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib206" class="citations"><div id="sref206" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref206" title="View in article">192.</a></div><div>Zheng, J. ∙ Erzurumluoglu, A.M. ∙ Elsworth, B.L. ..., Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium</div><div><strong>LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:272-279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_192_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.02.057&amp;key=2-s2.0-85028296791" target="_blank" aria-label="Open Scopus in new tab">Scopus (271)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27663502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw613&amp;pmid=27663502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (12)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/fd526227-1668-44e6-9757-ec0811af53e2/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/ef8cf5f8-d12b-4722-8305-a1ef22267252/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/d24d3d98-6314-42c0-8c35-cd637ae08718/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/a5d4e21b-adc2-4f49-b380-e9a44db27165/main.assets/gr3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/f192a26e-ddbf-4120-b778-edc444d7087e/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/4ad5fa4f-b9da-411d-9717-97ca0bfd4518/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/d49118e2-6689-4853-be1f-b446f31a8964/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/3449439a-5be1-44fe-876e-427ae331f1de/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/a6934eb5-5284-401f-a77a-03cc07336d7e/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/56b3e442-c772-43e4-a095-c7f66145053f/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/da5bb7c7-8caf-4eaa-a435-260ff6762a13/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/asset/25235b62-a8ee-4a11-b00a-b5859c2baf6a/main.assets/gr7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Barticle%3Aarticle%3Apii%5C%3AS0092867420302348%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00094" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">81</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">166</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">4</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">110</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.02.057&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (9)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/1e00ca9b-b5d9-45e5-a228-b2fb792932b9/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (18.86 MB)</span></a><div class="core-description">Table S1. ATAC-Seq Datasets from Mouse and Human Long Bone Samples with Downstream Analyses, Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/8c3d72aa-94fe-4f18-b488-92e8d555fadf/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (3.17 MB)</span></a><div class="core-description">Table S2. ATAC-Seq Dataset Overlaps with Evolutionary and Functional Genomics Datasets, Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/3b700148-713e-4597-a0c8-13cc4f5e1b39/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (2.24 MB)</span></a><div class="core-description">Table S3. Transcription Factor Binding Analyses Using ATAC-Seq Sets, Related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/eb1858da-39be-4c1c-b356-72f5c0ea8847/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (188.45 KB)</span></a><div class="core-description">Table S4. Putative Knee Regulatory Regions and Acceleration Signals, Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/3e2e8816-96ce-49e2-ab8e-827b960f5bee/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (816.62 KB)</span></a><div class="core-description">Table S5. Analyses of Inter- and Intra-species Diversity in ATAC-Seq Sets, Related to Figure&nbsp;3</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/13e79cb6-fe29-43bd-9f37-506a1dde98a0/mmc6.xlsx" download="mmc6.xlsx"><span id="mmc6-link">Spreadsheet (31.22 MB)</span></a><div class="core-description">Table S6. Analyses of Intra-species Diversity in ATAC-Seq Sets, Related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/39633529-002e-45d6-b043-537f9800adf0/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (399.12 KB)</span></a><div class="core-description">Table S7. Heritable Osteoarthritis Risk in ATAC-Seq Peaks, Related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/2bb8a659-573a-41d7-a84d-f3e7c7c9b4c7/mmc8.xlsx" download="mmc8.xlsx"><span id="mmc8-link">Spreadsheet (3.51 MB)</span></a><div class="core-description">Table S8. Common Human Variant Clustering and Region Selection Tests, Related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.02.057/attachment/7ed7304f-eeba-4790-b54d-05202cdc2abf/mmc9.xlsx" download="mmc9.xlsx"><span id="mmc9-link">Spreadsheet (43.94 KB)</span></a><div class="core-description">Table S9. Experimental and Bioinformatics Results on the <i>GDF5 R4</i> Enhancer, Related to Figures 5, 6, and 7</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420302348" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.02.057","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.02.057/asset/fd526227-1668-44e6-9757-ec0811af53e2/main.assets/gr1.jpg" height="699" width="750" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/39fef6bf-4b87-4a1d-a8bc-f753fad2f1f3/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.02.057/asset/ef8cf5f8-d12b-4722-8305-a1ef22267252/main.assets/figs1.jpg" height="503" width="752" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/b7e0a7a9-129f-4d00-8590-4bad837b64b1/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.02.057/asset/d24d3d98-6314-42c0-8c35-cd637ae08718/main.assets/gr2.jpg" height="939" width="482" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/26e99865-b0f5-4e6f-8b2a-d71427a405a9/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.02.057/asset/a5d4e21b-adc2-4f49-b380-e9a44db27165/main.assets/gr3.jpg" height="830" width="752" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/e04b483d-720f-4d75-8757-202257505b66/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.02.057/asset/f192a26e-ddbf-4120-b778-edc444d7087e/main.assets/figs2.jpg" height="940" width="578" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/d8acb6b0-c9a7-4388-a2ec-2c4d26a8c130/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.02.057/asset/4ad5fa4f-b9da-411d-9717-97ca0bfd4518/main.assets/gr4.jpg" height="923" width="752" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/ce5c71b8-f507-4271-aff1-6a2617d56b50/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.02.057/asset/d49118e2-6689-4853-be1f-b446f31a8964/main.assets/figs3.jpg" height="940" width="553" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/5934fb0b-9456-4267-be08-77a1c32c40fd/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.02.057/asset/3449439a-5be1-44fe-876e-427ae331f1de/main.assets/gr5.jpg" height="939" width="620" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/237f5184-6230-400c-ac77-aa5dd617273f/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.02.057/asset/a6934eb5-5284-401f-a77a-03cc07336d7e/main.assets/figs4.jpg" height="939" width="584" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/754fe31c-95bb-4506-93ed-8f70d604163d/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.02.057/asset/56b3e442-c772-43e4-a095-c7f66145053f/main.assets/figs5.jpg" height="940" width="598" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/574e7c7a-3e80-46a9-a7e2-3ca0cf05d324/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.02.057/asset/da5bb7c7-8caf-4eaa-a435-260ff6762a13/main.assets/gr6.jpg" height="334" width="499" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/a5de51ad-1152-4dc4-84d0-2457f53edd79/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.02.057/asset/25235b62-a8ee-4a11-b00a-b5859c2baf6a/main.assets/gr7.jpg" height="598" width="752" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/a692925a-4a72-4c86-b285-2217bc07813c/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.02.057/asset/fd526227-1668-44e6-9757-ec0811af53e2/main.assets/gr1.jpg" height="699" width="750" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/39fef6bf-4b87-4a1d-a8bc-f753fad2f1f3/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.02.057/asset/ef8cf5f8-d12b-4722-8305-a1ef22267252/main.assets/figs1.jpg" height="503" width="752" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/b7e0a7a9-129f-4d00-8590-4bad837b64b1/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.02.057/asset/d24d3d98-6314-42c0-8c35-cd637ae08718/main.assets/gr2.jpg" height="939" width="482" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/26e99865-b0f5-4e6f-8b2a-d71427a405a9/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.02.057/asset/a5d4e21b-adc2-4f49-b380-e9a44db27165/main.assets/gr3.jpg" height="830" width="752" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/e04b483d-720f-4d75-8757-202257505b66/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.02.057/asset/f192a26e-ddbf-4120-b778-edc444d7087e/main.assets/figs2.jpg" height="940" width="578" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/d8acb6b0-c9a7-4388-a2ec-2c4d26a8c130/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.02.057/asset/4ad5fa4f-b9da-411d-9717-97ca0bfd4518/main.assets/gr4.jpg" height="923" width="752" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/ce5c71b8-f507-4271-aff1-6a2617d56b50/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.02.057/asset/d49118e2-6689-4853-be1f-b446f31a8964/main.assets/figs3.jpg" height="940" width="553" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/5934fb0b-9456-4267-be08-77a1c32c40fd/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.02.057/asset/3449439a-5be1-44fe-876e-427ae331f1de/main.assets/gr5.jpg" height="939" width="620" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/237f5184-6230-400c-ac77-aa5dd617273f/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.02.057/asset/a6934eb5-5284-401f-a77a-03cc07336d7e/main.assets/figs4.jpg" height="939" width="584" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/754fe31c-95bb-4506-93ed-8f70d604163d/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.02.057/asset/56b3e442-c772-43e4-a095-c7f66145053f/main.assets/figs5.jpg" height="940" width="598" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/574e7c7a-3e80-46a9-a7e2-3ca0cf05d324/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.02.057/asset/da5bb7c7-8caf-4eaa-a435-260ff6762a13/main.assets/gr6.jpg" height="334" width="499" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/a5de51ad-1152-4dc4-84d0-2457f53edd79/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2020.02.057/asset/25235b62-a8ee-4a11-b00a-b5859c2baf6a/main.assets/gr7.jpg" height="598" width="752" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2020.02.057/asset/a692925a-4a72-4c86-b285-2217bc07813c/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30234-8';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)30234-8';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749586039199","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"181","issueNumber":"2","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420302348","coverDate":"202004","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Evolutionary Selection and Constraint on Human Knee Chondrocyte Regulation Impacts Osteoarthritis Risk","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db7e7dafd14ee3',t:'MTc0OTU4NjAzOC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db7e7dafd14ee3&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.6.2&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2020.02.057&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>